<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net" />
<link rel="dns-prefetch" href="//www.google.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2019.12.12.873844v4.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2019.12.12.873844v4.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2019.12.12.873844v4.ppt" />
<meta name="type" content="article" />
<meta name="category" content="article" />
<meta name="HW.identifier" content="/biorxiv/early/2022/01/20/2019.12.12.873844.atom" />
<meta name="HW.pisa" content="biorxiv;2019.12.12.873844v4" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A modular RNA interference system for multiplexed gene regulation" />
<meta name="DC.Identifier" content="10.1101/2019.12.12.873844" />
<meta name="DC.Date" content="2022-01-20" />
<meta name="DC.Publisher" content="Cold Spring Harbor Laboratory" />
<meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission." />
<meta name="DC.AccessRights" content="restricted" />
<meta name="DC.Description" content="The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity. To this effect, we introduce m odular A rtificial R NA i nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in Escherichia coli . The regulatory function of mARi was characterised in a range of relevant genetic contexts, demonstrating its independence from other genetic control elements and the gene of interest, and providing new insight into the design rules of RNA based regulation in E. coli , while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type. Importantly, the extensibility and orthogonality of mARi enables the simultaneous post-transcriptional regulation of multi-gene systems as both single-gene cassettes and poly-cistronic operons. To facilitate adoption, mARi was designed to be directly integrated into the modular BASIC DNA assembly framework. We anticipate that mARi-based genetic control within an extensible DNA assembly framework will facilitate metabolic engineering, layered genetic control, and advanced genetic circuit applications.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="DC.Contributor" content="Ari Dwijayanti" />
<meta name="DC.Contributor" content="Marko Storch" />
<meta name="DC.Contributor" content="Guy-Bart Stan" />
<meta name="DC.Contributor" content="Geoff S. Baldwin" />
<meta name="article:published_time" content="2022-01-20" />
<meta name="article:section" content="New Results" />
<meta name="citation_title" content="A modular RNA interference system for multiplexed gene regulation" />
<meta name="citation_abstract" lang="en" content="&lt;h3&gt;ABSTRACT&lt;/h3&gt;
&lt;p&gt;The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity. To this effect, we introduce &lt;b&gt;m&lt;/b&gt;odular &lt;b&gt;A&lt;/b&gt;rtificial &lt;b&gt;R&lt;/b&gt;NA &lt;b&gt;i&lt;/b&gt;nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in &lt;i&gt;Escherichia coli&lt;/i&gt;. The regulatory function of mARi was characterised in a range of relevant genetic contexts, demonstrating its independence from other genetic control elements and the gene of interest, and providing new insight into the design rules of RNA based regulation in &lt;i&gt;E. coli&lt;/i&gt;, while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type. Importantly, the extensibility and orthogonality of mARi enables the simultaneous post-transcriptional regulation of multi-gene systems as both single-gene cassettes and poly-cistronic operons. To facilitate adoption, mARi was designed to be directly integrated into the modular BASIC DNA assembly framework. We anticipate that mARi-based genetic control within an extensible DNA assembly framework will facilitate metabolic engineering, layered genetic control, and advanced genetic circuit applications.&lt;/p&gt;" />
<meta name="citation_journal_title" content="bioRxiv" />
<meta name="citation_publisher" content="Cold Spring Harbor Laboratory" />
<meta name="citation_publication_date" content="2022/01/01" />
<meta name="citation_mjid" content="biorxiv;2019.12.12.873844v4" />
<meta name="citation_id" content="2019.12.12.873844v4" />
<meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4" />
<meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4.abstract" />
<meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4.full" />
<meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844.full.pdf" />
<meta name="citation_doi" content="10.1101/2019.12.12.873844" />
<meta name="citation_num_pages" content="30" />
<meta name="citation_article_type" content="Article" />
<meta name="citation_section" content="New Results" />
<meta name="citation_firstpage" content="2019.12.12.873844" />
<meta name="citation_author" content="Ari Dwijayanti" />
<meta name="citation_author_institution" content="CNRS@CREATE" />
<meta name="citation_author" content="Marko Storch" />
<meta name="citation_author_institution" content="London Biofoundry, Translation &amp; Innovation Hub, White City Campus, Imperial College London" />
<meta name="citation_author_institution" content="Imperial College Centre for Synthetic Biology, Imperial College London" />
<meta name="citation_author" content="Guy-Bart Stan" />
<meta name="citation_author_institution" content="Imperial College Centre for Synthetic Biology, Imperial College London" />
<meta name="citation_author_institution" content="Department of Bioengineering, Bessemer Building, South Kensington Campus, Imperial College London" />
<meta name="citation_author_email" content="g.baldwin@imperial.ac.uk" />
<meta name="citation_author_email" content="g.stan@imperial.ac.uk" />
<meta name="citation_author" content="Geoff S. Baldwin" />
<meta name="citation_author_institution" content="Imperial College Centre for Synthetic Biology, Imperial College London" />
<meta name="citation_author_institution" content="Department of Life Sciences, Sir Alexander Fleming Building, South Kensington Campus, Imperial College London" />
<meta name="citation_author_email" content="g.baldwin@imperial.ac.uk" />
<meta name="citation_author_email" content="g.stan@imperial.ac.uk" />
<meta name="citation_reference" content="Cameron, D.E., Bashor, C.J. and Collins, J.J. (2014) A brief history of synthetic biology. Nat. Rev. Microbiol., 12, 381–390." />
<meta name="citation_reference" content="Purnick, P.E.M. and Weiss, R. (2009) The second wave of synthetic biology: From modules to systems. Nat. Rev. Mol. Cell Biol., 10, 410–422." />
<meta name="citation_reference" content="Slusarczyk, A.L., Lin, A. and Weiss, R. (2012) Foundations for the design and implementation of synthetic genetic circuits. Nat. Rev. Genet., 13, 406–420." />
<meta name="citation_reference" content="Kelwick, R., MacDonald, J.T., Webb, A.J. and Freemont, P. (2014) Developments in the Tools and Methodologies of Synthetic Biology. Front. Bioeng. Biotechnol., 2, 60." />
<meta name="citation_reference" content="Xiang, Y., Dalchau, N. and Wang, B. (2018) Scaling up genetic circuit design for cellular computing: advances and prospects. Nat. Comput., 17, 833–853." />
<meta name="citation_reference" content="Storz, G., Vogel, J. and Wassarman, K.M. (2011) Regulation by Small RNAs in Bacteria: Expanding Frontiers. Mol. Cell, 43, 880–891." />
<meta name="citation_reference" content="Huang, H.-H., Qian, Y. and Del Vecchio, D. (2018) A quasi-integral controller for adaptation of genetic modules to variable ribosome demand. Nat. Commun., 9, 5415." />
<meta name="citation_reference" content="Thompson, K.M., Rhodius, V.A. and Gottesman, S. (2007) σE regulates and is regulated by a small RNA in Escherichia coli. J. Bacteriol., 189, 4243–4256." />
<meta name="citation_reference" content="Tu, K.C., Long, T., Svenningsen, S.L., Wingreen, N.S. and Bassler, B.L. (2010) Negative Feedback Loops Involving Small Regulatory RNAs Precisely Control the Vibrio harveyi Quorum-Sensing Response. Mol. Cell, 37, 567–579." />
<meta name="citation_reference" content="Brosse, A., Korobeinikova, A., Gottesman, S. and Guillier, M. (2016) Unexpected properties of sRNA promoters allow feedback control via regulation of a two-component system. Nucleic Acids Res., 44, 9650–9666." />
<meta name="citation_reference" content="Nitzan, M., Rehani, R. and Margalit, H. (2017) Integration of Bacterial Small RNAs in Regulatory Networks. Annu. Rev. Biophys., 46, 131–148." />
<meta name="citation_reference" content="Beisel, C.L. and Storz, G. (2010) Base pairing small RNAs and their roles in global regulatory networks. FEMS Microbiol. Rev., 34, 866–882." />
<meta name="citation_reference" content="Levine, E., Zhang, Z., Kuhlman, T. and Hwa, T. (2007) Quantitative characteristics of gene regulation by small RNA. PLoS Biol., 5, 1998–2010." />
<meta name="citation_reference" content="Rice, J.B., Balasubramanian, D. and Vanderpool, C.K. (2012) Small RNA binding-site multiplicity involved in translational regulation of a polycistronic mRNA. Proc. Natl. Acad. Sci., 109, E2691– E2698." />
<meta name="citation_reference" content="Morita, T. and Aiba, H. (2011) RNase E action at a distance: Degradation of target mRNAs mediated by an Hfq-binding small RNA in bacteria. Genes Dev., 25, 294–298." />
<meta name="citation_reference" content="Pfeiffer, V., Papenfort, K., Lucchini, S., Hinton, J.C.D. and Vogel, J. (2009) Coding sequence targeting by MicC RNA reveals bacterial mRNA silencing downstream of translational initiation. Nat. Struct. Mol. Biol., 16, 840–846." />
<meta name="citation_reference" content="Yoo, S.M., Na, D. and Lee, S.Y. (2013) Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli. Nat. Protoc., 8, 1694–707." />
<meta name="citation_reference" content="Sharma, V., Yamamura, A. and Yokobayashi, Y. (2012) Engineering artificial small RNAs for Conditional gene silencing in escherichia coli. ACS Synth. Biol., 1, 6–13." />
<meta name="citation_reference" content="Viegas, S.C., Silva, I.J., Saramago, M., Domingues, S. and Arraiano, C.M. (2011) Regulation of the small regulatory RNA MicA by ribonuclease III: a target-dependent pathway. Nucleic Acids Res., 39, 2918–2930." />
<meta name="citation_reference" content="Man, S., Cheng, R., Miao, C., Gong, Q., Gu, Y., Lu, X., Han, F. and Yu, W. (2011) Artificial trans-encoded small non-coding RNAs specifically silence the selected gene expression in bacteria. Nucleic Acids Res., 39, e50–e50." />
<meta name="citation_reference" content="Nakashima, N., Tamura, T. and Good, L. (2006) Paired termini stabilize antisense RNAs and enhance conditional gene silencing in Escherichia coli. Nucleic Acids Res., 34, e138–e138." />
<meta name="citation_reference" content="Engdahl, H.M., Lindell, M. and Wagner, E.G.H. (2001) Introduction of an RNA Stability Element at the 5′-End of an Antisense RNA Cassette Increases the Inhibition of Target RNA Translation. Antisense Nucleic Acid Drug Dev., 11, 29–40." />
<meta name="citation_reference" content="Ellis, M.J., Trussler, R.S. and Haniford, D.B. (2015) A cis-encoded sRNA, Hfq and mRNA secondary structure act independently to suppress IS200 transposition. Nucleic Acids Res., 43, 6511." />
<meta name="citation_reference" content="Ghodasara, A. and Voigt, C.A. (2017) Balancing gene expression without library construction via a reusable sRNA pool. Nucleic Acids Res., 45, 8116–8127." />
<meta name="citation_reference" content="Kelly, C.L., Harris, A.W.K., Steel, H., Hancock, E.J., Heap, J.T. and Papachristodoulou, A. (2018) Synthetic negative feedback circuits using engineered small RNAs. Nucleic Acids Res., 46, 9875– 9889." />
<meta name="citation_reference" content="Ceroni, F., Furini, S., Stefan, A., Hochkoeppler, A. and Giordano, E. (2012) A synthetic post-transcriptional controller to explore the modular design of gene circuits. ACS Synth. Biol., 1, 163– 71." />
<meta name="citation_reference" content="Na, D., Yoo, S.M., Chung, H., Park, H., Park, J.H. and Lee, S.Y. (2013) Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs. Nat. Biotechnol., 31, 170–174." />
<meta name="citation_reference" content="Lahiry, A., Stimple, S.D., Wood, D.W. and Lease, R.A. (2017) Retargeting a Dual-Acting sRNA for Multiple mRNA Transcript Regulation. ACS Synth. Biol., 6, 648–658." />
<meta name="citation_reference" content="Sagawa, S., Shin, J.-E., Hussein, R. and Lim, H.N. (2015) Paradoxical suppression of small RNA activity at high Hfq concentrations due to random-order binding. Nucleic Acids Res., 43, 8502–8515." />
<meta name="citation_reference" content="Hoynes-O’Connor, A. and Moon, T.S. (2016) Development of Design Rules for Reliable Antisense RNA Behavior in E. coli. ACS Synth. Biol., 5, 1441–1454." />
<meta name="citation_reference" content="Lee, Y.J. and Moon, T.S. (2018) Design rules of synthetic non-coding RNAs in bacteria. Methods, 143, 58–69." />
<meta name="citation_reference" content="Massaiu, I., Pasotti, L., Casanova, M., Politi, N., Zucca, S., Cusella De Angelis, M.G. and Magni, P. (2015) Quantification of the gene silencing performances of rationally-designed synthetic small RNAs. Syst. Synth. Biol., 9, 107–123." />
<meta name="citation_reference" content="Storch, M., Haines, M.C. and Baldwin, G.S. (2020) DNA-BOT: a low-cost, automated DNA assembly platform for synthetic biology. Synth. Biol., 5, 1–7." />
<meta name="citation_reference" content="Storch, M., Casini, A., Mackrow, B., Fleming, T., Trewhitt, H., Ellis, T. and Baldwin, G.S. (2015) BASIC: A New Biopart Assembly Standard for Idempotent Cloning Provides Accurate, Single-Tier DNA Assembly for Synthetic Biology. ACS Synth. Biol., 4, 781–787." />
<meta name="citation_reference" content="Casini, A., Christodoulou, G., Freemont, P.S., Baldwin, G.S., Ellis, T. and MacDonald, J.T. (2014) R2oDNA Designer: Computational Design of Biologically Neutral Synthetic DNA Sequences. ACS Synth. Biol., 3, 525–528." />
<meta name="citation_reference" content="Darty, K., Denise, A. and Ponty, Y. (2009) VARNA: Interactive drawing and editing of the RNA secondary structure. Bioinformatics, 25, 1974–1975." />
<meta name="citation_reference" content="Busch, A., Richter, A.S. and Backofen, R. (2008) IntaRNA: Efficient prediction of bacterial sRNA targets incorporating target site accessibility and seed regions. Bioinformatics, 24, 2849–2856." />
<meta name="citation_reference" content="Wright, P.R., Georg, J., Mann, M., Sorescu, D.A., Richter, A.S., Lott, S., Kleinkauf, R., Hess, W.R. and Backofen, R. (2014) CopraRNA and IntaRNA: Predicting small RNA targets, networks and interaction domains. Nucleic Acids Res., 42, 119–123." />
<meta name="citation_reference" content="Markham, N.R. and Zuker, M. (2005) DINAMelt web server for nucleic acid melting prediction. Nucleic Acids Res., 33, 577–581." />
<meta name="citation_reference" content="Bonde, M.T., Pedersen, M., Klausen, M.S., Jensen, S.I., Wulff, T., Harrison, S., Nielsen, A.T., Herrgård, M.J. and Sommer, M.O.A. (2016) Predictable tuning of protein expression in bacteria. Nat. Methods, 13, 233–236." />
<meta name="citation_reference" content="Salis, H.M., Mirsky, E.A. and Voigt, C.A. (2009) Automated design of synthetic ribosome binding sites to control protein expression. Nat. Biotechnol., 27, 946–950." />
<meta name="citation_reference" content="Espah Borujeni, A., Channarasappa, A.S. and Salis, H.M. (2014) Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites. Nucleic Acids Res., 42, 2646–2659." />
<meta name="citation_reference" content="Raden, M., Ali, S.M., Alkhnbashi, O.S., Busch, A., Costa, F., Davis, J.A., Eggenhofer, F., Gelhausen, R., Georg, J., Heyne, S., et al. (2018) Freiburg RNA tools: A central online resource for RNA-focused research and teaching. Nucleic Acids Res., 46, W25–W29." />
<meta name="citation_reference" content="Wright, P.R., Richter, A.S., Papenfort, K., Mann, M., Vogel, J., Hess, W.R., Backofen, R. and Georg, J. (2013) Comparative genomics boosts target prediction for bacterial small RNAs. Proc. Natl. Acad. Sci., 110, E3487–E3496." />
<meta name="citation_reference" content="Storch, M., Casini, A., Mackrow, B., Ellis, T. and Baldwin, G.S. (2017) BASIC: A Simple and Accurate Modular DNA Assembly Method. In Methods in Molecular Biology.Vol. 1472, pp. 79–91." />
<meta name="citation_reference" content="Vytvytska, O., Moll, I., Kaberdin, V.R., Gabain, A. Von and Bla, U. (2000) Hfq (HF1) stimulates ompA mRNA decay by interfering with ribosome binding." />
<meta name="citation_reference" content="Večerek, B., Moll, I. and Bläsi, U. (2005) Translational autocontrol of the Escherichia coli hfq RNA chaperone gene. RNA, 11, 976–984." />
<meta name="citation_reference" content="Franze De Fernandez, M.T., Eoyang, L. and August, J.T. (1968) Factor fraction required for the synthesis of bacteriophage Qβ-RNA. Nature, 219, 588–290." />
<meta name="citation_reference" content="Sun, T., Li, S., Song, X., Pei, G., Diao, J., Cui, J., Shi, M., Chen, L. and Zhang, W. (2018) Re-direction of carbon flux to key precursor malonyl-CoA via artificial small RNAs in photosynthetic Synechocystis sp. PCC 6803. Biotechnol. Biofuels, 11, 1–16." />
<meta name="citation_reference" content="Huis in ‘t Veld, R.A.G., Kramer, G., van der Ende, A., Speijer, D. and Pannekoek, Y. (2017) The Hfq regulon of Neisseria meningitidis. FEBS Open Bio, 7, 777–788." />
<meta name="citation_reference" content="Puerta-Fernández, E. and Vioque, A. (2011) Hfq is required for optimal nitrate assimilation in the cyanobacterium Anabaena sp. strain PCC 7120. J. Bacteriol., 193, 3546–3555." />
<meta name="citation_reference" content="Chen, S., Zhang, A., Blyn, L.B. and Storz, G. (2004) MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli. J. Bacteriol., 186, 6689–6697." />
<meta name="citation_reference" content="Vogel, J. and Luisi, B.F. (2011) Hfq and its constellation of RNA. Nat. Rev. Microbiol., 9, 578–589." />
<meta name="citation_reference" content="Mitarai, N., Benjamin, J.-A.M.J.-A.M., Krishna, S., Semsey, S., Csiszovszki, Z., Massé, E., Sneppen, K., Masse, E. and Sneppen, K. (2009) Dynamic features of gene expression control by small regulatory RNAs. Proc. Natl. Acad. Sci. U. S. A., 106, 10655–10659." />
<meta name="citation_reference" content="Jahn, M., Vorpahl, C., Hübschmann, T., Harms, H. and Müller, S. (2016) Copy number variability of expression plasmids determined by cell sorting and droplet digital PCR. Microb. Cell Fact., 15, 1–12." />
<meta name="citation_reference" content="Teimouri, H., Korkmazhan, E., Stavans, J. and Levine, E. (2017) Sub-cellular mRNA localization modulates the regulation of gene expression by small RNAs in bacteria. Phys. Biol., 14." />
<meta name="citation_reference" content="Nielsen, A.A.K., Segall-Shapiro, T.H. and Voigt, C.A. (2013) Advances in genetic circuit design: Novel biochemistries, deep part mining, and precision gene expression. Curr. Opin. Chem. Biol., 17, 878–892." />
<meta name="citation_reference" content="Bradley, R.W., Buck, M. and Wang, B. (2016) Tools and Principles for Microbial Gene Circuit Engineering. J. Mol. Biol., 428, 862–888." />
<meta name="citation_reference" content="Morita, T. and Aiba, H. (2019) Mechanism and physiological significance of autoregulation of the Escherichia coli hfq gene. RNA, 25, 264–276." />
<meta name="citation_reference" content="Azam, T.A., Iwata, A., Nishimura, A., Ueda, S. and Ishihama, A. (1999) Growth phase-dependent variation in protein composition of the Escherichia coli nucleoid. J. Bacteriol., 181, 6361–6370." />
<meta name="citation_reference" content="Kajitani, M., Kato, A., Wada, A., Inokuchi, Y. and Ishihama, A. (1994) Regulation of the Escherichia coli hfq gene encoding the host factor for phage Q(β). J. Bacteriol., 176, 531–534." />
<meta name="citation_reference" content="Yoon, S.H., Han, M., Jeong, H., Lee, C.H., Xia, X., Lee, D., Shim, J.H., Lee, S.Y., Oh, T.K. and Kim, J.F. (2012) Comparative multi-omics systems analysis of Escherichia coli strains B and K-12. Genome Biol., 13, R37." />
<meta name="citation_reference" content="Moon, K. and Gottesman, S. (2011) Competition among Hfq-binding small RNAs in Escherichia coli. Mol. Microbiol., 82, 1545–1562." />
<meta name="citation_reference" content="Wagner, E.G.H. (2013) Cycling of RNAs on Hfq. RNA Biol., 10, 619–626." />
<meta name="twitter:title" content="A modular RNA interference system for multiplexed gene regulation" />
<meta name="twitter:site" content="@biorxivpreprint" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="twitter:description" content="The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity. To this effect, we introduce m odular A rtificial R NA i nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in Escherichia coli . The regulatory function of mARi was characterised in a range of relevant genetic contexts, demonstrating its independence from other genetic control elements and the gene of interest, and providing new insight into the design rules of RNA based regulation in E. coli , while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type. Importantly, the extensibility and orthogonality of mARi enables the simultaneous post-transcriptional regulation of multi-gene systems as both single-gene cassettes and poly-cistronic operons. To facilitate adoption, mARi was designed to be directly integrated into the modular BASIC DNA assembly framework. We anticipate that mARi-based genetic control within an extensible DNA assembly framework will facilitate metabolic engineering, layered genetic control, and advanced genetic circuit applications.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-title" property="og:title" content="A modular RNA interference system for multiplexed gene regulation" />
<meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4" />
<meta name="og-site-name" property="og:site_name" content="bioRxiv" />
<meta name="og-description" property="og:description" content="The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity. To this effect, we introduce m odular A rtificial R NA i nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in Escherichia coli . The regulatory function of mARi was characterised in a range of relevant genetic contexts, demonstrating its independence from other genetic control elements and the gene of interest, and providing new insight into the design rules of RNA based regulation in E. coli , while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type. Importantly, the extensibility and orthogonality of mARi enables the simultaneous post-transcriptional regulation of multi-gene systems as both single-gene cassettes and poly-cistronic operons. To facilitate adoption, mARi was designed to be directly integrated into the modular BASIC DNA assembly framework. We anticipate that mARi-based genetic control within an extensible DNA assembly framework will facilitate metabolic engineering, layered genetic control, and advanced genetic circuit applications.

### Competing Interest Statement

The authors have declared no competing interest." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png" />
<meta name="citation_date" content="2022-01-20" />
<meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution" />
<meta name="generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4" />
<link rel="shortlink" href="https://www.biorxiv.org/node/2359265" />
    <title>A modular RNA interference system for multiplexed gene regulation | bioRxiv</title>  
    <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]> */
</style>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<style type="text/css" media="all">
/* <![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]> */
</style>

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all" />
    <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&amp;render=explicit&amp;onload=drupalRecaptchaOnload"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"tFiZmN9D-IAIXsYMUJkztvab8Wnm_d0ZegiXLHO-SJU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2359265","apath":"\/biorxiv\/early\/2022\/01\/20\/2019.12.12.873844.atom","pisa":"biorxiv;2019.12.12.873844v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2019.12.12.873844v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2019.12.12.873844v4","title":"A modular RNA interference system for multiplexed gene regulation","identifier":"node\/2359265"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LdQAfJ1ATCNIkYIFdjNo96nWe7lzrbOwuxkVoQ1dkW2Y06\\\\\\\/HywMY7LLdGHFHa\\\\\\\/snZbeIeq9\\\\\\\/51ah9HalWlkAfghfSWHr7frmwPc\\\\\\\/V3YcaIejKQyS1GxOJkJr3v7y3ejDEEL1N9LiE09e04j5XHg+G+JHfIQ9LIn3VmV1tB7y3dwElXuhnigmg0QozmHxTVR3toD4zfYTXXXU+l6h2JkRs0W2qrOGhO2\\\\\\\/3DlhyWsX30C7hJaEjJre\\\\\\\/NMxTFAJ1py9Q3g6nP+wYh7rAbLtJNeteW827K4J7Xdmvayrz+tvwbE0NxTRyRjuYylolowR6MnWBcqSneDznf1kyrOOhjiO7HR\\\\\\\/yCUbToUbcbCgwkBkayxzLAb2OFVPhJZXVg\\\\\\\/zM9X2FnFzcPDwlKs+SpvRPb\\\\\\\/InxkjHdup31deQ+lVMJO4jAdQzVegc6hincHqDN2Q7HtyeCO8YoIhQpR41YXkror1RYbYzo0R6EvpFsW1ZopTVoe0f4Mhk7nkCp8xyXK7oiDud17kymBTai8QHtURN8WimvKnP+shteu4+1cJkp6T\\\\\\\/t0UeHNlJztEJaJws0GkkK0SeQ2Mn5POgCpqCabVdQxr5aSuXLdmgdk3IxUzCCo4mxygmj7dDWlaQSmjphKSgIMmc19+PpHjXDRJsBP\\\\\\\/GbHgRqr6OiKaKRFlZltEeKfvHuobuJG5tkduGBGVMevQ7mqbVmWim59NRlH2rD0UOSKzlp3rrzjWArJj7F83o4gq47VMLejdR0MO1D8zKZ\\\\\\\/rlG+f+SQRPghhXV6TajotAKE32yHnvMqQOa0HZqhYvX0DgFP8UQqYyuCFAccJArKEv5iYRVWldc8qcKeQ0qJf6Ou3f05htf8gl3N0guG08Gxvo\\\\\\\/lWeOsTTtvNelaDo3cCgCnnT2mXLMRv7hjn1D90oHRmyOY0kQTHU49gkFv0K1jGlA4UXp0pJ8z9n86rYPDr1PsY2WL983p1GVcLIK8NP7gtdJTTbrCuVwmnYcL4WPsEWAiIFBEe0bkWJMSTphx1u0+XtEn9lDFRLGafH\\\\\\\/hd3pBrvXWGF45S8JBpTaFyEpF+YHI2AYMzWvYfWyW0sOaRfEYHRWof3jBz+iay+rIDJmRNrPjeHETBvQt7sUjlIxoSWO8\\\\\\\/GY5QIG4rZ5IGZ0Vz6cVpkASKAeWjwS3TP0XiRq+USyC+W7kbeExjNLRgpZ878KwXN16qvWwYpfq4t+JYXtkl4xNoxLPIrWsMQ0jg5R9yfwPLndWjD\\\\\\\/F3GlXAjdW+wVfb4yRiHMCjm6moH5ynZq8lvf\\\\\\\/EqnurzsZG5+Zhi2N7YKRxfwbWm8iYe1+bO6\\\\\\\/2MoxdrScfF4+60usfxbpH26BDox5m30I653OesD75Q3gaTDh98VWPHjz33vSXWnyR2AU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022T0BaZ2cEtihd5EI5Nkcoag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220cb5b685ab645dfcbcc41b5ef2e024d56ecceba2d8f36c5836d46017c75c8e7e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2019.12.12.873844v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-2359265 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
        <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
    
  <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
    <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg" /></a>      </div>
                </div>
          </div>
</div><div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">Home</a></li>
<li class="leaf" role="menuitem"><a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="leaf" role="menuitem"><a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="last leaf" role="menuitem"><a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">ALERTS / RSS</a></li>
</ul></nav>    </div>
  </div>
</div><div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2019.12.12.873844v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="search_rightsidebar_keywords_901159612">Search for this keyword </label>
 <input placeholder="Search" type="text" id="search_rightsidebar_keywords_901159612" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><span class="icon-search"></span><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_831320655" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oMjPyrZmEZ3Al8OunS0AjHPl6SqUZtO-8RqnqcPVgYg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 advanced-search-link" >
  
      
  
  <div class="pane-content">
    <a href="/search">Advanced Search</a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node2359265" data-node-nid="2359265" id="node-2359265--22141499346" data-pisa="biorxiv;2019.12.12.873844v4" data-pisa-master="biorxiv;2019.12.12.873844" data-apath="/biorxiv/early/2022/01/20/2019.12.12.873844.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip" >

      <span class="biorxiv-article-type">
        New Results    </span>
  
        <h1 class="highwire-cite-title" id="page-title">A modular RNA interference system for multiplexed gene regulation</h1>  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ari</span> <span class="nlm-surname">Dwijayanti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marko</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Guy-Bart</span> <span class="nlm-surname">Stan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Geoff S.</span> <span class="nlm-surname">Baldwin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">doi:</span> https://doi.org/10.1101/2019.12.12.873844 </span></div>
  
  
  </div>
<div id="hw-article-author-popups-node-2359265--22141499346" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ari Dwijayanti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>1</span><span class='nlm-institution'>CNRS@CREATE</span>, 1 Create Way, #8-01 Create Tower, <span class='nlm-country'>Singapore</span> 138602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ari%2BDwijayanti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dwijayanti%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAri%2BDwijayanti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marko Storch </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>2</span><span class='nlm-institution'>London Biofoundry, Translation & Innovation Hub, White City Campus, Imperial College London</span>, London W12 0BZ, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Imperial College Centre for Synthetic Biology, Imperial College London</span>, London SW7 2AZ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marko%2BStorch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Storch%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarko%2BStorch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guy-Bart Stan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Imperial College Centre for Synthetic Biology, Imperial College London</span>, London SW7 2AZ, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>4</span><span class='nlm-institution'>Department of Bioengineering, Bessemer Building, South Kensington Campus, Imperial College London</span>, London SW7 2AZ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guy-Bart%2BStan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stan%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuy-Bart%2BStan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:g.baldwin@imperial.ac.uk" class="" data-icon-position="" data-hide-link-title="0">g.baldwin@imperial.ac.uk</a>
<a href="mailto:g.stan@imperial.ac.uk" class="" data-icon-position="" data-hide-link-title="0">g.stan@imperial.ac.uk</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Geoff S. Baldwin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>3</span><span class='nlm-institution'>Imperial College Centre for Synthetic Biology, Imperial College London</span>, London SW7 2AZ, <span class='nlm-country'>UK</span></div><div class='author-affiliation'><span class='nlm-sup'>5</span><span class='nlm-institution'>Department of Life Sciences, Sir Alexander Fleming Building, South Kensington Campus, Imperial College London</span>, London SW7 2AZ, <span class='nlm-country'>UK</span></div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoff%2BS.%2BBaldwin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baldwin%20GS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoff%2BS.%2BBaldwin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:g.baldwin@imperial.ac.uk" class="" data-icon-position="" data-hide-link-title="0">g.baldwin@imperial.ac.uk</a>
<a href="mailto:g.stan@imperial.ac.uk" class="" data-icon-position="" data-hide-link-title="0">g.stan@imperial.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10.1101/2019.12.12.873844v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="" data-url-enabled="1">Abstract</a><a href="/panels_ajax_tab/biorxiv_tab_art/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873844v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="full-text" data-url-enabled="1">Full Text</a><a href="/panels_ajax_tab/article_tab_full_text/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873844v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="article-info" data-url-enabled="1">Info/History</a><a href="/panels_ajax_tab/biorxiv_tab_info/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873844v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/article_tab_metrics/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10.1101/2019.12.12.873844v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="supplementary-material" data-url-enabled="1">Supplementary material</a><a href="/panels_ajax_tab/biorxiv_tab_data/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/10.1101/2019.12.12.873844v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2359265" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> Preview PDF</a><a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2359265/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-article_tab_full_text"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2 class="">ABSTRACT</h2><p id="p-2">The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity. To this effect, we introduce <strong>m</strong>odular <strong>A</strong>rtificial <strong>R</strong>NA <strong>i</strong>nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in <em>Escherichia coli</em>. The regulatory function of mARi was characterised in a range of relevant genetic contexts, demonstrating its independence from other genetic control elements and the gene of interest, and providing new insight into the design rules of RNA based regulation in <em>E. coli</em>, while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type. Importantly, the extensibility and orthogonality of mARi enables the simultaneous post-transcriptional regulation of multi-gene systems as both single-gene cassettes and poly-cistronic operons. To facilitate adoption, mARi was designed to be directly integrated into the modular BASIC DNA assembly framework. We anticipate that mARi-based genetic control within an extensible DNA assembly framework will facilitate metabolic engineering, layered genetic control, and advanced genetic circuit applications.</p></div><div class="section" id="sec-1"><h2 class="">INTRODUCTION</h2><p id="p-4">Synthetic biology aims to make the engineering of biological systems more predictable, efficient and reliable (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>–<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>). With this goal in mind, our ability to predictably combine genetic parts into higher-level functional biological/biochemical systems of increasing complexity is essential to improve the efficiency of the Design-Build-Test-Learn cycle (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>). In this work, we focus on expanding the toolbox of modular biomolecular control elements that function at the post-transcriptional level, specifically small non-coding RNAs (sRNAs). The ubiquity of sRNA-based control in critical cellular processes such as homeostasis and adaptation demonstrates their importance for the design and control of biological systems (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>–<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>). The sRNAs act to coordinate and synchronise multiple signals through sequence-specific and transient RNA-RNA interactions (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>), typically leading to down-regulation of target gene expression (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>–<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>).</p><p id="p-5">Based on the arrangement of sRNAs and their targets, sRNAs can be grouped into <em>cis</em> and <em>trans</em>-encoded sRNAs. The <em>cis</em>-encoded sRNAs are transcribed from the same loci as their target mRNA, but as a reverse complement, thus making a complementary base-pairing to the mRNA they regulate; in contrast, <em>trans</em>-encoded sRNAs are located in a distal site. Despite this, <em>trans</em>-encoded sRNAs can have efficient regulation of their mRNA targets (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>–<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>). In bacterial systems, <em>trans</em>-encoded sRNAs rather than <em>cis</em>-encoded sRNAs have been reported to be especially effective in silencing gene expression due to their longer half-life in the cytoplasm, thought to be due to association with the Hfq chaperone protein (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>–<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>). The <em>trans</em>-encoded sRNAs, therefore, have been widely applied as versatile genetically encoded controllers for the design of synthetic biological circuits for metabolic engineering and synthetic biology purposes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>–<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>).</p><p id="p-6">Previous approaches to implement <em>trans</em>-encoded sRNA were exemplified by the use of native sRNAs and their cognate targets i.e. <em>rpoS, hns, sodB</em>, and <em>ompC</em> (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>). By fusing the leader sequences from these natural sRNA systems to a reporter gene (e.g. GFP and mCherry), the performance of the native sRNA system could be quantified from the fluorescence output. Due to its highly composable structure, the sRNA can also be re-programmed to target any RNA sequence by modification of the seed sequence, whilst retaining the native sRNA scaffold (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>). We term this new type of sRNA an artificial sRNA system. Several rules and guides have been developed to improve the rational design of a set of novel sRNA systems (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>). This strategy has been implemented to create artificial sRNA systems with bespoke designs, where the sRNA is targeted to bind the first 24 bp of the GOIs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>). Artificial sRNAs have also been employed to specifically bind a set of standardised target sequences that are inserted upstream of GOIs (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>). However, the requirement to insert specific target sites upstream of each GOI may impede the implementation and adoption of sRNA-based controllers for broader applications.</p><p id="p-7">To leverage the use of sRNAs in genetic circuits constructed via a modular DNA assembly method, we set out to design a universal, modular framework for the facile implementation of <em>trans</em>-acting sRNA-based control using verified, re-usable target sequences, namely <strong>m</strong>odular <strong>A</strong>rtificial <strong>R</strong>NA <strong>i</strong>nterference (mARi). BASIC DNA assembly provides a modular, standardised and automatable (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">33</a>) framework for genetic structure based on orthogonal, computationally designed linkers (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We exploited this by targeting the DNA linkers used in the construction of gene expression cassettes and operons. In our mARi design, a modified seed sequence specific to the linker upstream of the target Gene of Interest (GOI) was fused to a native sRNA scaffold containing a host factor-1 (Hfq) binding site (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">27</a>) (<strong><a id="xref-fig-1-2" class="xref-fig" href="#F1">Fig. 1a</a>, <a id="xref-fig-2-1" class="xref-fig" href="#F2">2a</a></strong>). The integration of mARi-based regulation into a modular DNA assembly method offers a simple yet powerful strategy for implementing sRNA regulatory systems. Since standardised linkers are used in the assembly process, gene expression can be controlled by expression of an mARi that is cognate to the linker sequence upstream of the target gene (<strong><a id="xref-fig-1-3" class="xref-fig" href="#F1">Fig. 1</a></strong>). We characterise the post-transcriptional regulation of mARi in the context of various genetic design parameters including transcript ratio, molecular copy number, spatial organisation, growth temperatures and media, host strain, and growth phase, thus expanding our understanding of the design rules for new artificial sRNA-based regulators. Finally, mARi sequence variants were further expanded and their implementation was demonstrated for simultaneous regulation in a multi-gene system with different genetic architectures.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="mARi-based regulation integrated into the BASIC modular design and assembly framework. (a) mARi-based regulators composed of a promoter, seed sequence (grey), sRNA scaffold (purple) and terminator. These regulators were created as interchangeable components together with standardised DNA parts (i.e. plasmid backbone, promoter, and coding sequence) and computationally designed DNA linkers (i.e. neutral, UTR-RBS, and Prefix/Suffix linker) used in the BASIC DNA assembly framework (34). (b) Plasmids were constructed from the interchangeable DNA parts using BASIC DNA assembly: a representative assembled plasmid shows a functional UTR-RBS linker controlling expression of a downstream gene of interest (GOI). (c) Expression of mARi leads to repression of a target gene with a cognate UTR by base-pairing between the 5&#x2019;untranslated region and the mARi seed sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-72377722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;mARi-based regulation integrated into the BASIC modular design and assembly framework.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) mARi-based regulators composed of a promoter, seed sequence (grey), sRNA scaffold (purple) and terminator. These regulators were created as interchangeable components together with standardised DNA parts (i.e. plasmid backbone, promoter, and coding sequence) and computationally designed DNA linkers (i.e. neutral, UTR-RBS, and Prefix/Suffix linker) used in the BASIC DNA assembly framework (34). (&lt;strong&gt;b&lt;/strong&gt;) Plasmids were constructed from the interchangeable DNA parts using BASIC DNA assembly: a representative assembled plasmid shows a functional UTR-RBS linker controlling expression of a downstream gene of interest (GOI). (&lt;strong&gt;c&lt;/strong&gt;) Expression of mARi leads to repression of a target gene with a cognate UTR by base-pairing between the 5&#x2019;untranslated region and the mARi seed sequence.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F1.medium.gif" width="440" height="131"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F1.medium.gif" width="440" height="131"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1.</span> <span class="caption-title">mARi-based regulation integrated into the BASIC modular design and assembly framework.</span><p id="p-8" class="first-child">(<strong>a</strong>) mARi-based regulators composed of a promoter, seed sequence (grey), sRNA scaffold (purple) and terminator. These regulators were created as interchangeable components together with standardised DNA parts (i.e. plasmid backbone, promoter, and coding sequence) and computationally designed DNA linkers (i.e. neutral, UTR-RBS, and Prefix/Suffix linker) used in the BASIC DNA assembly framework (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">34</a>). (<strong>b</strong>) Plasmids were constructed from the interchangeable DNA parts using BASIC DNA assembly: a representative assembled plasmid shows a functional UTR-RBS linker controlling expression of a downstream gene of interest (GOI). (<strong>c</strong>) Expression of mARi leads to repression of a target gene with a cognate UTR by base-pairing between the 5’untranslated region and the mARi seed sequence.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section" id="sec-2"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-3" class="subsection"><h3>Bacterial strains and outgrowth conditions</h3><p id="p-9">The following <em>E. coli</em> strains were used for plasmid construction and expression: DH5α (F<sup>−</sup> <em>endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG purB20</em> ϕ80d<em>lacZ</em>ΔM15 Δ (<em>lacZYA-argF</em>)U169, hsdR17 (<em>r</em> <sub><em>K</em></sub> <sup>−</sup><em>m</em><sub><em>K</em></sub> <sup>+</sup>), λ<sup>−</sup>), DH10b [F-<em>mcr</em>A Δ (<em>mrr</em>-<em>hsd</em>RMS-<em>mcr</em>BC) ϕ80<em>lac</em>ZΔM15 Δ<em>lac</em>X74 <em>rec</em>A1 <em>end</em>A1 <em>ara</em>D139 Δ (<em>ara leu</em>) 7697 <em>gal</em>U <em>gal</em>K <em>rps</em>L <em>nup</em>G λ-], BL21 (DE3) (F<sup>−</sup><em>omp</em>T <em>hsd</em>S<sub>B</sub> (r<sub>B</sub><sup>−</sup>, m<sub>B</sub><sup>−</sup>) <em>gal dcm</em> (DE3)), and BL21star (DE3) (F<sup>−</sup><em>omp</em>T <em>hsd</em>S<sub>B</sub> (r<sub>B</sub><sup>−</sup>, mB<sup>−</sup>) <em>gal dcm rne</em>131 (DE3)). <em>E. coli</em> colonies were grown from single-colony isolates in LB (Luria Bertani) medium supplemented with the appropriate antibiotics, shaken at 220 rpm and grown at 37°C (unless specified otherwise). The antibiotics used for plasmid maintenance were carbenicillin and kanamycin at a concentration of 100 μg/ml and 50 μg/ml, respectively.</p></div><div id="sec-4" class="subsection"><h3>Design and analysis of mARi sequences</h3><p id="p-10">The sequences of UTR-RBS linkers used in BASIC DNA assembly (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">34</a>) were designed and validated using R2O DNA designer (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>). Secondary structures of the mARi sequences were predicted using RNAFold WebServer (<a href="http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi">http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi</a>) and visualised using VARNA GUI (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>) (<a href="http://varna.lri.fr/index.php?lang=en&amp;page=tutorial&amp;css=varna">http://varna.lri.fr/index.php?lang=en&amp;page=tutorial&amp;css=varna</a>). The predicted interactions between mARi regulators and their mRNA targets were simulated through IntaRNA (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>) (<a href="https://bio.tools/intarna">https://bio.tools/intarna</a>). The free binding energy of mARi and its mRNA target site was estimated using a web-based service of the Two-State Melting Hybridisation of UNAFold RNA folding software DINAMelt (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>) (<a href="http://www.unafold.org/Dinamelt/applications/two-state-melting-hybridization.php">http://www.unafold.org/Dinamelt/applications/two-state-melting-hybridization.php</a>) using default parameters at 37°C. The percent identity of mARi-UTR pairs was calculated using the EMBOSS needle method (<a href="https://www.ebi.ac.uk/Tools/psa/emboss_needle/">https://www.ebi.ac.uk/Tools/psa/emboss_needle/</a>). The Hamming distance was calculated using a custom Python script (<a href="https://github.com/MsDwijayanti/hamming/blob/main/hamming.py">https://github.com/MsDwijayanti/hamming/blob/main/hamming.py</a>). The predicted expression strength of the UTR-RBS linker was simulated using EMOPEC (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>) and RBS calculator v2.0 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">41</a>, <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">42</a>) with the input sequence starting from the Transcription Start Site (+1) to 100 bp of the downstream CDS. The off-target effect was computed using CopraRNA (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">38</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">43</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">44</a>) (<a href="https://bio.tools/coprarna">https://bio.tools/coprarna</a>). NC_000913 (<em>E. coli</em> MG1655), NC_010473 (<em>E. coli</em> DH10b), NC_012892 and NC_012971 (<em>E. coli</em> BL21 (DE3)) were used as inputs references for host-background off-target prediction.</p></div><div id="sec-5" class="subsection"><h3>Plasmid assembly</h3><p id="p-11">Plasmids were constructed using the BASIC DNA assembly method (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">34</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">45</a>). New DNA parts with prefix and suffix sequences were synthesised as gBlocks by Integrated DNA Technologies (IDT), gene fragments by TWIST Bioscience, or derived by PCR mutagenesis. The gBlocks and gene fragments were directly cloned into pJET1.2 or pUC AmpR_plasmid as BASIC bioparts. PCR mutagenesis for generating new BASIC parts were carried out using Phusion Polymerase (NEB) and phosphorylated oligonucleotides (IDT). Sequence verification of BASIC bioparts was carried out using Sanger sequencing (Source Bioscience). Details of the BASIC bioparts used to construct the plasmids are provided in <strong>Supplementary Table 5</strong>. Linkers for BASIC assembly were provided by Biolegio or IDT (<strong>Supplementary Table 6</strong>). All constructs were transformed into chemically competent <em>E. coli</em> DH5α. Maps of plasmids used in this study are provided in <strong>Supplementary Fig. 9</strong>.</p></div><div id="sec-6" class="subsection"><h3>Flow cytometry assay</h3><p id="p-12">Cellular expression of sfGFP and mCherry was performed in 96 well plates with biological triplicates for each construct. <em>E. coli</em> carrying an empty backbone (containing only an origin of replication and an antibiotic resistance gene) was used as a negative control. <em>E. coli</em> colonies were inoculated from glycerol stock into LB media supplemented with appropriate antibiotics. The colonies were grown for 16 h at 30°C and 600 rpm in a benchtop shaker (Mikura). The culture was then diluted 200x (10 µl into 190 µl, then 10 µl into 90 µl) in LB media using an automated liquid handling robot (CyBio Felix). The plate was grown at 37°C and 600 rpm in a benchtop shaker (Mikura). After 6 h of incubation, the cell density was measured by absorption at 600 nm (Abs<sub>600</sub>; not normalised for path length). Abs<sub>600</sub> typically reached values in the range 0.6–0.7. Two microliters of liquid culture were taken and diluted into 200 µl Phosphate Buffer Saline (PBS) supplemented with 2 mg/ml kanamycin to inhibit further protein synthesis and cell growth. The fluorescence data of single cells for the dual reporters system (<strong><a id="xref-fig-4-1" class="xref-fig" href="#F4">Fig. 4b</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">d</a></strong>) was collected using BD Fortessa flow cytometer with a 488 nm excitation laser and filter 530/30 for sfGFP; 561 nm excitation laser and filter 610/20 for mCherry. For the triple reporters system (<strong><a id="xref-fig-4-3" class="xref-fig" href="#F4">Fig. 4c</a></strong>), fluorescence was measured using Attune NxT flow cytometer with a 488 nm excitation laser and channel BL1 530/30 for sfGFP; 561 nm excitation laser and channel YL2 620/15 for mCherry; 405 nm excitation laser and channel VL1 440/50 for mTagBFP. In total 10,000 events were collected for each sample. The data was stored as an FCS file 3.0 and analysis was done using FlowJo V10. Single cell population gating was performed after plotting FSC-H against SSC-H and histograms of each channel of fluorescence. The chosen gating covered 86-99% of the total population. The fluorescence intensity of the sample was calculated by subtracting the geometric mean fluorescence of the control (strains with empty backbone). The normalised fluorescence was calculated by dividing the background-subtracted mean fluorescence of the sample with mARi by the background-subtracted mean fluorescence of the sample without mARi expression.</p></div><div id="sec-7" class="subsection"><h3>Plate reader assay</h3><p id="p-13">For continuous growth assay, overnight liquid cultures (as for flow cytometry assay) were diluted 200x in LB media (10 µl into 190 µl, then 10 µl into 90 µl) using an automated liquid handling robot (CyBio Felix); the plate was then incubated in a microplate reader (Clariostar, BMG Labtech) with continuous shaking at 37°C and 600 rpm for 12-24 h. For different growth temperatures, the DH5α cells with and without mARi expression plasmids were incubated at 30°C or 37°C in LB medium. For different growth media, the engineered DH5α cell were tested in two mediums: LB and EZ MOPS RDM supplemented with 0.2% glucose. For the EZ MOPS RDM media supplemented with 0.2% glucose, 5 µL samples were diluted into 95 µL of fresh media using an automated liquid handling robot (CyBio Felix). The plate was incubated at 37°C and 600 rpm.</p><p id="p-14">Absorbance at 600 nm and GFP fluorescence (F:482-16/F:530-40) were measured every 15 min. The corrected Abs<sub>600</sub> was calculated by subtracting the mean Abs<sub>600</sub> of media only control from the Abs<sub>600</sub> of each well. The fluorescence intensity was corrected by subtracting the mean fluorescence value of negative control at equivalent Abs<sub>600</sub>. The value Fl/Abs<sub>600</sub> was calculated by dividing the value of corrected fluorescence by corrected Abs<sub>600</sub>. Statistical analysis was calculated in Prism v.8.0 (GraphPad). The values were compared using two-tailed Student’s t-test for unpaired comparisons and one-way ANOVA.</p></div></div><div class="section" id="sec-8"><h2 class="">RESULTS</h2><div id="sec-9" class="subsection"><h3>Design of a modular Artificial RNA interference regulation system</h3><p id="p-15">The mARi sequences were designed with two core components: the seed sequence and an sRNA scaffold containing the Hfq binding site (<strong><a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1a</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">2a</a></strong>). The scaffold sequence is responsible for recognition by the RNA chaperone protein Hfq, which is highly conserved as part of natural cellular regulation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">46</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">47</a>) in a wide range of organisms (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">48</a>–<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">51</a>). One of the intensively-studied Hfq-dependent sRNAs, MicC was chosen from sRNA scaffolds naturally found in <em>E. coli</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">52</a>) and <em>Salmonella enterica</em> (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">16</a>). The inclusion of this RNA chaperone binding site in the sRNA structure has previously been shown to one of the best Hfq binding sites to mediate sRNA-based gene repression (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">30</a>). The nature of the multiple targets of MicC based sRNAs in controlling the expression of outer membrane proteins (OmpC and OmpD) can be exploited to target any mRNA sequence of interest. Based on this architecture, the seed sequences can be modified while preserving the native MicC scaffold (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">30</a>) (<strong><a id="xref-fig-2-3" class="xref-fig" href="#F2">Fig. 2a</a></strong>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Design and development of mARi-based gene regulation. (a) Sequence and secondary structure of mARi containing a seed sequence (grey) and MicC RNA scaffold (purple). (b) Schematic design of the target site selection schema for mARi-mediated repression. Four positions within the translation initiation region were selected as target sites (Positions 1-4; Supplementary Table 1). The numbers indicate the relative position of mARi downstream of the transcription start site (+1). (c) Functional characterisation of mARi-based gene regulation showing the silencing activity of mARi-A for the different target positions vs a sfGFP-only control without mARi. (d) Uniformity of representative expressing strains with and without mARi expression. (e) The fluorescence intensity (top) and normalized fluorescence (bottom) when position 1 mARi-A was used to target UTR-A with varying RBS strengths. All sfGFP fluorescence measurements were performed by flow cytometry after 6 h growth. Data are shown with error bars for the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined using two-tailed Student&#x2019;s t-test (**** represents p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-72377722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Design and development of mARi-based gene regulation.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) Sequence and secondary structure of mARi containing a seed sequence (grey) and MicC RNA scaffold (purple). (&lt;strong&gt;b&lt;/strong&gt;) Schematic design of the target site selection schema for mARi-mediated repression. Four positions within the translation initiation region were selected as target sites (Positions 1-4; &lt;strong&gt;Supplementary Table 1&lt;/strong&gt;). The numbers indicate the relative position of mARi downstream of the transcription start site (+1). &lt;em&gt;(&lt;/em&gt;&lt;strong&gt;c&lt;/strong&gt;) Functional characterisation of mARi-based gene regulation showing the silencing activity of mARi-A for the different target positions vs a sfGFP-only control without mARi. (&lt;strong&gt;d&lt;/strong&gt;) Uniformity of representative expressing strains with and without mARi expression. (&lt;strong&gt;e&lt;/strong&gt;) The fluorescence intensity (top) and normalized fluorescence (bottom) when position 1 mARi-A was used to target UTR-A with varying RBS strengths. All sfGFP fluorescence measurements were performed by flow cytometry after 6 h growth. Data are shown with error bars for the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined using two-tailed Student&#x2019;s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, * represents p&lt;0.1, ns represents not significant).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F2.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F2.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2.</span> <span class="caption-title">Design and development of mARi-based gene regulation.</span><p id="p-16" class="first-child">(<strong>a</strong>) Sequence and secondary structure of mARi containing a seed sequence (grey) and MicC RNA scaffold (purple). (<strong>b</strong>) Schematic design of the target site selection schema for mARi-mediated repression. Four positions within the translation initiation region were selected as target sites (Positions 1-4; <strong>Supplementary Table 1</strong>). The numbers indicate the relative position of mARi downstream of the transcription start site (+1). <em>(</em><strong>c</strong>) Functional characterisation of mARi-based gene regulation showing the silencing activity of mARi-A for the different target positions vs a sfGFP-only control without mARi. (<strong>d</strong>) Uniformity of representative expressing strains with and without mARi expression. (<strong>e</strong>) The fluorescence intensity (top) and normalized fluorescence (bottom) when position 1 mARi-A was used to target UTR-A with varying RBS strengths. All sfGFP fluorescence measurements were performed by flow cytometry after 6 h growth. Data are shown with error bars for the mean ± SD of triplicate measurements (black dots). Statistically significant differences were determined using two-tailed Student’s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, * represents p&lt;0.1, ns represents not significant).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">The seed sequence within mARi creates a specific base-pairing interaction to a target sequence in the mRNA of interest, resulting in its post-transcriptional silencing. To standardise and enhance the modularity of mARi, the seed sequence was designed to target expression cassettes constructed via BASIC DNA assembly (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">34</a>). BASIC assembly relies on standardised linkers between interchangeable DNA parts (<strong><a id="xref-fig-1-5" class="xref-fig" href="#F1">Fig. 1a</a></strong>) (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">34</a>). Expression cassettes and operons can be tuned using functional DNA linkers that encode a ribosome binding site (RBS) within a defined 5’ untranslated region (UTR; UTR-RBS linker; <strong><a id="xref-fig-1-6" class="xref-fig" href="#F1">Fig. 1b</a></strong>). Since BASIC UTRs are computationally designed to be orthogonal to the host genome and exclude secondary structure (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">35</a>), they provide an ideal target for mARi and reduce off-target effects.</p><p id="p-18">We designed mARi sequences to target the translation initiation region of mRNA for three main reasons: (i) the 5′ UTR sequences are relatively AU rich (30-43.8% GC; <strong>Supplementary Table 1</strong>) and AU rich sequences are preferentially bound by Hfq (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">53</a>); (ii) targets located in the translation initiation region have been shown to exhibit a low off-target effect in <em>trans</em>-encoded sRNA-based regulation (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">30</a>), which is important for regulator specificity/orthogonality; (iii) a 5′ UTR target site location offers potential modularity since it is independent of different GOIs and is compatible with the modular UTR-RBS linkers in the BASIC design framework (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">34</a>). To evaluate this approach, seed sequences were designed to target different positions in the translation initiation region (<strong><a id="xref-fig-2-4" class="xref-fig" href="#F2">Fig. 2b</a></strong>). Positions 1-3 addressed gene-independent targets within the BASIC framework, while position 4 was specific to the gene of interest, in this case <em>superfolder-gfp</em> (<em>sfgfp</em>; <strong><a id="xref-fig-2-5" class="xref-fig" href="#F2">Fig. 2b</a></strong>). The reverse complements of the cognate target sequences were combined with the MicC scaffold to create a series of mARi sequences (<strong>Supplementary Table 1</strong>).</p><p id="p-19">The design of the full sequence of each mARi variant was computationally evaluated to ensure that it met the requirements for effective repression (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">30</a>). Firstly, the designed sequence for each mARi was made compatible with BASIC DNA assembly (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">34</a>), specifically avoiding the forbidden restriction site <em>Bsa</em>I. Secondly, to achieve effective repression and to have a high affinity binding between the seed and target sequences, the GC content and binding energy of the seed sequences were calculated and found to vary from 34.62% to 43.8% and -38.6 kcal/mol to -61 kcal/mol, respectively. The binding energy has a positive correlation to the binding affinity and observed repression capability (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">27</a>, <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">30</a>), and previous studies suggested a typical binding energy of -30 to -40 kcal/mol (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">27</a>) was required for effective repression activity. However, lower binding energy (less than -40 kcal/mol) sRNAs have been experimentally validated and suggested to achieve higher repression (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">30</a>). Homology length determines mARi specificity and target discrimination (<a id="xref-ref-30-8" class="xref-bibr" href="#ref-30">30</a>): shorter binding sites have a lower binding affinity and may cause off-target effects (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">27</a>), while longer binding sequences may have a complex secondary structure that could cause binding to the mRNA target to be less thermodynamically favorable (<a id="xref-ref-30-9" class="xref-bibr" href="#ref-30">30</a>). All mARi regulators used here were designed to have perfect complementarity in their seed sequences to their target mRNA. (<strong>Supplementary Fig. 1b</strong>). To investigate the repression efficiency of mARi targets, sfGFP expression cassettes were assembled with a constitutive promoter P<sub>J23101_BASIC</sub> and UTR-A-RBSc linker, while mARi expression was driven by the constitutive P<sub>J23101</sub> (<strong><a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1b</a></strong>). Measurement of reporter expression demonstrated that all mARi target sequences led to repression of sfGFP, compared to control cells lacking mARi (<strong><a id="xref-fig-2-6" class="xref-fig" href="#F2">Fig. 2c</a></strong>). The intra-gene target at position 4 showed significant repression as anticipated (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">25</a>, <a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">27</a>). However, targeting this position would require bespoke design of a specific sequence for the target gene, thus optimisation for each new expression target is required; a less desirable strategy from a modular engineering perspective. Position 1, which targets the UTR upstream of the RBS was as effective as position 4 (<strong><a id="xref-fig-2-7" class="xref-fig" href="#F2">Fig. 2c</a></strong>). The position 1 target shown had been further optimised for length; reducing the length to 25 bp, as shown for the other positions, had only a minor impact, although further reduction to 20 bp resulted in significant loss of repression activity (<strong>Supplementary Fig. 2</strong>). Targeting this standardised UTR component of the BASIC framework is ideal for modular engineering since it is independent of both the GOI and RBS. The homogeneity of representative producing strains with mARi position 1 against the control strains without mARi expression was shown in <strong><a id="xref-fig-2-8" class="xref-fig" href="#F2">Fig. 2d</a></strong>.</p><p id="p-20">To evaluate the independence of position 1 UTR target, a set of constructs with five different RBS strengths (RBSa-RBSe) (<strong>Supplementary Table 2</strong>) controlling sfGFP translation were assembled both with and without the position 1 mARi cassette all within the same UTR-A context. The <em>sfgfp</em> gene-only constructs resulted in a series of fluorescence intensities reflecting RBS strength variation (<strong><a id="xref-fig-2-9" class="xref-fig" href="#F2">Fig. 2e</a></strong>). The position 1 UTR-A targeted mARi cassette reduced the fluorescence signal in all cases and the relative repression was found to be constant across the series (<strong><a id="xref-fig-2-10" class="xref-fig" href="#F2">Fig. 2e</a></strong>). This demonstrated that the repression activity of mARi-A at position 1 was independent of RBS strength, an important feature for RBS design flexibility and modularity of mARi.</p></div><div id="sec-10" class="subsection"><h3>mARi activity is tuneable via genetic design parameters</h3><p id="p-21">To evaluate the features of mARi, we characterised its responsiveness to genetic design parameters and its robustness to different contexts and growth conditions. Two key determinants modulating the repression activity of this post-transcriptional regulator are (i) the degradation rate of RNA complexes and (ii) the ratio of transcript abundance (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">54</a>). We focused on investigating the relative abundance of mARi in relation to its target mRNA. Unlike previous work where inducible expression was used to modulate relative transcript abundance of sRNA to mRNA target (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">13</a>, <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">32</a>, <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">54</a>), here we constitutively expressed both transcripts to test the impact of different transcript levels, which provides more consistent relative expression. Relative transcript abundance of mARi and mRNA was estimated from the relative promoter strength driving both regulator and target (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">54</a>). A set of standardised constitutive promoters (P<sub>J23xxx_BASIC</sub>) were used to express a <em>sfGFP</em> reporter gene using UTR A-RBSc in a p15A backbone (<strong>Supplementary Fig. 3a</strong>). The functional characterisation results were then used as a basis to infer the relative transcript expression ratio (<strong><a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a></strong>), which is calculated by dividing the strength of the promoter used for mARi by the promoter expressing the mRNA target. To assess the impact of the mARi:mRNA ratio, a matrix of designs with four different promoter strengths for both mARi and mRNA were constructed (<strong><a id="xref-fig-3-2" class="xref-fig" href="#F3">Fig. 3a, b</a></strong>). The resulting sfGFP expression showed a consistent trend, with sfGFP fluorescence decreasing as the mARi:mRNA ratio increased (<strong><a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3c, d</a>, Supplementary Fig. 3c, Supplementary Table 3</strong>). The maximum repression was observed when an excess of mARi was present in relation to its target mRNA, presumably because an excess of mARi is required to saturate binding of target mRNA. On the other hand, a low expression ratio led to weak repression, presumably due to an insufficient amount of mARi being available to inactivate mRNA target (<a id="xref-ref-30-10" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">54</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterisation of mARi-A across different genetic architectures and growth conditions. (a) Heat-map showing relative expression ratio calculated from the relative strength of promoters driving mARi and mRNA expression across a matrix of constructs (Supplementary Fig. 3b). The representative conditions used for set-point high and low expression ratios are highlighted with a blue square (PJ23119-mARi:PJ23101BASIC-mRNA) and grey square (PJ23101-mARi:PJ23101BASIC-mRNA), respectively. (b) Schematic of mARi repression constructs used to investigate the effect of transcript ratios using four different strength constitutive promoters driving both mARi and target mRNA in a single plasmid system. (c) Heat-map showing the measured relative expression levels (mean normalized fluorescence) of the mARi expression ratio matrix from (a). The calculated relative expression ratio and repression activity of mARi are provided in Supplementary Table 3. (d) Correlation of transcript expression ratios in a and normalised fluorescence in c. (e) Schematic of the mARi-based repression system with high and low expression ratios constructed in single- and double-plasmid systems with different copy numbers. Reporter expression was measured for the single (f) and double (g) plasmid systems for high and low ratio mARi against a control construct without mARi; sfGFP fluorescence measurements were performed by flow cytometry assay. (h) The robustness of the double-plasmid mARi-based regulation in different incubation temperatures. A schematic of the mARi-based repression system in double-plasmid systems used is illustrated. (i) The robustness of the double-plasmid mARi-based repression system in a variety of growth media. (j) Reporter expression of the single-plasmid high-ratio mARi on p15A was measured in different E. coli strains against a control construct without mARi. (k) The repression activity of single-plasmid high-ratio mARi on p15A in representative strains (DH10b, BL21 (DE3), and BL21star (DE3)) across different growth phases (Supplementary Fig. 5). Assays in h, i, j and k were performed by plate reader assay. All error bars show the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student&#x2019;s t-test (**** represents p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-72377722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;Characterisation of mARi-A across different genetic architectures and growth conditions.&lt;/span&gt; &lt;strong&gt;(a)&lt;/strong&gt; Heat-map showing relative expression ratio calculated from the relative strength of promoters driving mARi and mRNA expression across a matrix of constructs (&lt;strong&gt;Supplementary Fig. 3b&lt;/strong&gt;). The representative conditions used for set-point high and low expression ratios are highlighted with a blue square (P&lt;sub&gt;J23119&lt;/sub&gt;-mARi:P&lt;sub&gt;J23101BASIC&lt;/sub&gt;-mRNA) and grey square (P&lt;sub&gt;J23101&lt;/sub&gt;-mARi:P&lt;sub&gt;J23101BASIC&lt;/sub&gt;-mRNA), respectively. &lt;strong&gt;(b)&lt;/strong&gt; Schematic of mARi repression constructs used to investigate the effect of transcript ratios using four different strength constitutive promoters driving both mARi and target mRNA in a single plasmid system. &lt;strong&gt;(c&lt;/strong&gt;) Heat-map showing the measured relative expression levels (mean normalized fluorescence) of the mARi expression ratio matrix from (&lt;strong&gt;a&lt;/strong&gt;). The calculated relative expression ratio and repression activity of mARi are provided in &lt;strong&gt;Supplementary Table 3&lt;/strong&gt;. (&lt;strong&gt;d&lt;/strong&gt;) Correlation of transcript expression ratios in &lt;strong&gt;a&lt;/strong&gt; and normalised fluorescence in &lt;strong&gt;c&lt;/strong&gt;. (&lt;strong&gt;e&lt;/strong&gt;) Schematic of the mARi-based repression system with high and low expression ratios constructed in single- and double-plasmid systems with different copy numbers. Reporter expression was measured for the single (&lt;strong&gt;f&lt;/strong&gt;) and double (&lt;strong&gt;g&lt;/strong&gt;) plasmid systems for high and low ratio mARi against a control construct without mARi; sfGFP fluorescence measurements were performed by flow cytometry assay. (&lt;strong&gt;h&lt;/strong&gt;) The robustness of the double-plasmid mARi-based regulation in different incubation temperatures. A schematic of the mARi-based repression system in double-plasmid systems used is illustrated. (&lt;strong&gt;i&lt;/strong&gt;) The robustness of the double-plasmid mARi-based repression system in a variety of growth media. (&lt;strong&gt;j&lt;/strong&gt;) Reporter expression of the single-plasmid high-ratio mARi on p15A was measured in different &lt;em&gt;E. coli&lt;/em&gt; strains against a control construct without mARi. (&lt;strong&gt;k&lt;/strong&gt;) The repression activity of single-plasmid high-ratio mARi on p15A in representative strains (DH10b, BL21 (DE3), and BL21star (DE3)) across different growth phases (&lt;strong&gt;Supplementary Fig. 5&lt;/strong&gt;). Assays in &lt;strong&gt;h, i, j&lt;/strong&gt; and &lt;strong&gt;k&lt;/strong&gt; were performed by plate reader assay. All error bars show the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student&#x2019;s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, ** represents p&lt;0.01, * represents p&lt;0.1, ns represents not significant).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F3.medium.gif" width="345" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F3.medium.gif" width="345" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3.</span> <span class="caption-title">Characterisation of mARi-A across different genetic architectures and growth conditions.</span><p id="p-22" class="first-child"><strong>(a)</strong> Heat-map showing relative expression ratio calculated from the relative strength of promoters driving mARi and mRNA expression across a matrix of constructs (<strong>Supplementary Fig. 3b</strong>). The representative conditions used for set-point high and low expression ratios are highlighted with a blue square (P<sub>J23119</sub>-mARi:P<sub>J23101BASIC</sub>-mRNA) and grey square (P<sub>J23101</sub>-mARi:P<sub>J23101BASIC</sub>-mRNA), respectively. <strong>(b)</strong> Schematic of mARi repression constructs used to investigate the effect of transcript ratios using four different strength constitutive promoters driving both mARi and target mRNA in a single plasmid system. <strong>(c</strong>) Heat-map showing the measured relative expression levels (mean normalized fluorescence) of the mARi expression ratio matrix from (<strong>a</strong>). The calculated relative expression ratio and repression activity of mARi are provided in <strong>Supplementary Table 3</strong>. (<strong>d</strong>) Correlation of transcript expression ratios in <strong>a</strong> and normalised fluorescence in <strong>c</strong>. (<strong>e</strong>) Schematic of the mARi-based repression system with high and low expression ratios constructed in single- and double-plasmid systems with different copy numbers. Reporter expression was measured for the single (<strong>f</strong>) and double (<strong>g</strong>) plasmid systems for high and low ratio mARi against a control construct without mARi; sfGFP fluorescence measurements were performed by flow cytometry assay. (<strong>h</strong>) The robustness of the double-plasmid mARi-based regulation in different incubation temperatures. A schematic of the mARi-based repression system in double-plasmid systems used is illustrated. (<strong>i</strong>) The robustness of the double-plasmid mARi-based repression system in a variety of growth media. (<strong>j</strong>) Reporter expression of the single-plasmid high-ratio mARi on p15A was measured in different <em>E. coli</em> strains against a control construct without mARi. (<strong>k</strong>) The repression activity of single-plasmid high-ratio mARi on p15A in representative strains (DH10b, BL21 (DE3), and BL21star (DE3)) across different growth phases (<strong>Supplementary Fig. 5</strong>). Assays in <strong>h, i, j</strong> and <strong>k</strong> were performed by plate reader assay. All error bars show the mean ± SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student’s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, ** represents p&lt;0.01, * represents p&lt;0.1, ns represents not significant).</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="mARi enables multiplex and simultaneous control of gene expression in a multi-gene system. (a) A schematic of experimental design to evaluate the target specificity of modular mARi-mRNA pairs is shown. Fluorescence response matrix of all possible mARi/UTR pairs measured by flow cytometry assay with triplicate repeats (Supplementary Fig. 7d); ^ represents data where the use of mARi-A and mARi-B increased UTR-D-sfGFP expression relative to control. (b) A schematic of reporters in dual transcription units is shown: the fluorescence response of reporters in the double transcriptional unit system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. (c) A schematic of reporters in triple transcription units is shown: the fluorescence response of reporters in the triple transcriptional unit system when combined with all possible mARi-A, mARi-B and mARi-C regulators was measured by flow cytometry assay. (d) A schematic of dual reporters in an operon system is shown: the fluorescence response of dual reporters in the operon system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. All error bars show the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student&#x2019;s t-test (**** represents p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-72377722" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;span class=&quot;caption-title&quot;&gt;mARi enables multiplex and simultaneous control of gene expression in a multi-gene system.&lt;/span&gt; (&lt;strong&gt;a&lt;/strong&gt;) A schematic of experimental design to evaluate the target specificity of modular mARi-mRNA pairs is shown. Fluorescence response matrix of all possible mARi/UTR pairs measured by flow cytometry assay with triplicate repeats (&lt;strong&gt;Supplementary Fig. 7d&lt;/strong&gt;); ^ represents data where the use of mARi-A and mARi-B increased UTR-D-sfGFP expression relative to control. (&lt;strong&gt;b&lt;/strong&gt;) A schematic of reporters in dual transcription units is shown: the fluorescence response of reporters in the double transcriptional unit system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. (&lt;strong&gt;c&lt;/strong&gt;) A schematic of reporters in triple transcription units is shown: the fluorescence response of reporters in the triple transcriptional unit system when combined with all possible mARi-A, mARi-B and mARi-C regulators was measured by flow cytometry assay. (&lt;strong&gt;d&lt;/strong&gt;) A schematic of dual reporters in an operon system is shown: the fluorescence response of dual reporters in the operon system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. All error bars show the mean &#xB1; SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student&#x2019;s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, ** represents p&lt;0.01, represents p&lt;0.1, ns represents not significant).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F4.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F4.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab last"><a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/20/2019.12.12.873844/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4.</span> <span class="caption-title">mARi enables multiplex and simultaneous control of gene expression in a multi-gene system.</span><p id="p-23" class="first-child">(<strong>a</strong>) A schematic of experimental design to evaluate the target specificity of modular mARi-mRNA pairs is shown. Fluorescence response matrix of all possible mARi/UTR pairs measured by flow cytometry assay with triplicate repeats (<strong>Supplementary Fig. 7d</strong>); ^ represents data where the use of mARi-A and mARi-B increased UTR-D-sfGFP expression relative to control. (<strong>b</strong>) A schematic of reporters in dual transcription units is shown: the fluorescence response of reporters in the double transcriptional unit system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. (<strong>c</strong>) A schematic of reporters in triple transcription units is shown: the fluorescence response of reporters in the triple transcriptional unit system when combined with all possible mARi-A, mARi-B and mARi-C regulators was measured by flow cytometry assay. (<strong>d</strong>) A schematic of dual reporters in an operon system is shown: the fluorescence response of dual reporters in the operon system when combined with all possible mARi-A and mARi-B regulators was measured by flow cytometry assay. All error bars show the mean ± SD of triplicate measurements (black dots). Statistically significant differences were determined against control without mARi expression (C) using two-tailed Student’s t-test (**** represents p&lt;0.0001, *** represents p&lt;0.001, ** represents p&lt;0.01, represents p&lt;0.1, ns represents not significant).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">A further variable affecting intracellular transcript abundance is plasmid copy number used in the system. Designs with both high (blue box in <strong><a id="xref-fig-3-4" class="xref-fig" href="#F3">Fig. 3a</a></strong>) and low (grey box in <strong><a id="xref-fig-3-5" class="xref-fig" href="#F3">Fig. 3a</a></strong>) expression ratios were selected to exemplify different levels of transcript abundance. These systems were first assembled with both mARi and its target in a single plasmid, with varying plasmid copy numbers from low to high: pSC101 (∼5 copies per cell), p15A (∼10 copies per cell) and pMB1 (∼15-20 copies per cell) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">55</a>) (<strong><a id="xref-fig-3-6" class="xref-fig" href="#F3">Fig. 3e</a> left panel, Supplementary Fig. 4a</strong>). Expression of mARi in either high or low expression ratio repressed sfGFP expression for all tested plasmid copy numbers, demonstrating that the mARi-based regulatory system is functional at different plasmid copy numbers (<strong><a id="xref-fig-3-7" class="xref-fig" href="#F3">Fig. 3f</a></strong>).</p><p id="p-25">We next reasoned that by placing the mARi on a second, higher copy vector, repression should be more efficient. To test this hypothesis, the mRNA expression cassette was placed on a p15A backbone, while mARi was placed on the higher copy pMB1 and pUC backbones (∼500-700 copies per cell), with both high- and low-expression ratios (<strong><a id="xref-fig-3-8" class="xref-fig" href="#F3">Fig. 3e</a> right panel, Supplementary Fig. 4b</strong>). In this experimental design, the transcript expression ratio was governed by the combination of plasmid copy number and transcription rate of mARi relative to mRNA. Despite mARi being driven by a stronger promoter and being on a higher copy vector, in all cases the double-plasmid systems all showed less repression than that observed with the corresponding single-plasmid systems (<strong><a id="xref-fig-3-9" class="xref-fig" href="#F3">Fig. 3g</a></strong>). This suggests a role for spatial organisation of transcripts in post-transcriptional repression, as anticipated from recent theoretical analysis (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">56</a>). The fact that the sRNA-based regulator has a shorter half-life and stability than mRNA may further reduce the mARi effectiveness when it is used in a double plasmid system with a broader physical transcript distribution.</p><p id="p-26">To investigate the robustness of mARi repression system, we further tested the double plasmid system with both high and low expression ratios in different temperatures and growth media. The repression activity of mARi in low expression ratios was not significantly affected by a combination of different growth temperatures and growth media (<strong><a id="xref-fig-3-10" class="xref-fig" href="#F3">Fig. 3h, 3i</a></strong>). However, a high expression ratio resulted in a slight increase of mARi repression activity in the use of higher incubation temperature and a defined growth medium.</p></div><div id="sec-11" class="subsection"><h3>mARi-based regulation is robust to different genetic contexts</h3><p id="p-27">Unlike other biomolecular regulators (e.g. transcription factors, TALEs, zinc fingers, or CRISPRi) (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">5</a>, <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">57</a>, <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">58</a>), mARi only requires transcriptional activity and endogenous Hfq, which is autoregulated by Hfq binding its own mRNA (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">59</a>). Thus, the expression of a short sequence for mARi would be expected to have a low cellular burden and not significantly impact host cell growth. To test this, the high transcript ratio for mARi in a p15A backbone and control plasmid without mARi expression were used to compare the effect of mARi overexpression on host fitness. We sought to test this repression system in four commonly used <em>E. coli</em> strains: DH5α, DH10b, BL21(DE3), and BL21star(DE3). The high expression of mARi did not affect bacterial growth across different host strains, indicating the mARi-based repression system has a low production cost, and does not impact significantly on host cells (<strong>Supplementary Fig. 5</strong>).</p><p id="p-28">We then sought to evaluate the silencing activity of mARi in these <em>E. coli</em> K12 and B strains. The mARi-based gene regulation utilises the native <em>E. coli</em> Hfq chaperone and degradosome complex (e.g. RNase E) as helper proteins to regulate gene repression. The intracellular abundance of these components is influenced by several factors, including the genotype and growth phase. For instance, the abundance of Hfq is constant during the exponential growth, but reduces when <em>E. coli</em> cells reach stationary phase (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">47</a>, <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">60</a>–<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">63</a>).</p><p id="p-29">Functional characterisation of mARi at early stationary phase, 8 h (<strong>Supplementary Fig. 5</strong>), showed about 60-80% repression activity in both <em>E. coli</em> K-12 and B strains (<strong><a id="xref-fig-3-11" class="xref-fig" href="#F3">Fig. 3j</a></strong>), while the silencing activity in the B strains was higher than in the K-12 strains for comparable plasmid copy numbers. (<strong>Supplementary Fig. 6</strong>). The mARi system was active across different phases of cell growth (<strong><a id="xref-fig-3-12" class="xref-fig" href="#F3">Fig. 3k</a></strong>): the <em>E. coli</em> K-12 strain (DH10b cells) showed relatively stable repression activity in both mid-exponential and early stationary growth, whereas a slightly increased repression activity was observed in early stationary phase for the BL21(DE3) and BL21star(DE3) strains.</p></div><div id="sec-12" class="subsection"><h3>Expanding a set of orthogonal mARi regulators through seed sequence modification</h3><p id="p-30">We sought to create a set of mARi-based regulators based on its underlying modular design principles coupled with the modular UTR-RBS linkers used in BASIC to construct both expression cassettes and operons. Five different UTR-RBS linkers were identified using R2oDNA Designer (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">35</a>), whilst preserving the same medium strength RBS (RBSc). Computational evaluation of the five selected UTR (UTR-A-E) sequences demonstrated minimal similarity/identity scores (<strong>Supplementary Fig. 7a)</strong> and high Hamming distance scores (<strong>Supplementary Fig. 7b)</strong> between the set. Low sequence similarity and high Hamming distance of non-cognate pairs is desirable to achieve minimal cross-talk interaction and off-target effects (<a id="xref-ref-30-11" class="xref-bibr" href="#ref-30">30</a>). The target 35 bp upstream of the RBS sequence were evaluated for their GC content and free binding energy of the target/seed sequences (<strong>Supplementary Table 4)</strong>. The GC content of the seed sequence is in a range of 31% to 43% and resulted in a free binding energy of - 53.9 kcal/mol to -63.9 kcal/mol, which were in the favourable range for sRNA repression (<a id="xref-ref-30-12" class="xref-bibr" href="#ref-30">30</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>). The seed sequences were also predicted to have minimal off-target interactions with three different <em>E. coli</em> genomes: <em>E. coli</em> MG1655, DH10b, and BL21(DE3) <strong>(Supplementary Table 4)</strong>.</p><p id="p-31">To experimentally validate target specificity, we constructed all possible combinations of mARi and target UTR upstream of <em>sfgfp</em>; in each case maintaining the same promoter (<strong><a id="xref-fig-4-4" class="xref-fig" href="#F4">Fig. 4a</a></strong>). The expression of sfGFP was significantly repressed in the presence of the cognate mARi systems relative to the non-cognate pairings (<strong><a id="xref-fig-4-5" class="xref-fig" href="#F4">Fig. 4a</a>, Supplementary Fig. 7d</strong>), confirming their specificity with reduced cross-reactivity between each pair. However, there is an increased expression in UTR-D-sfGFP when mARi-A and mARi-B were used. The lower Hamming score of UTR D towards UTR A (<strong>Supplementary Fig. 7b</strong>) may contribute to this, but it does not completely explain the effect with mARi-B, or other observations; similar effects were not observed for UTR-E, which similarly has lower Hamming scores for UTR-A, -B and -D (<strong>Supplementary Fig. 7b</strong>).</p></div><div id="sec-13" class="subsection"><h3>Modularity and orthogonality of mARi enables simultaneous and multiplexed gene regulation</h3><p id="p-32">After demonstrating that mARi regulators exhibit specific interaction towards cognate UTRs in the context of a single gene, we evaluated their ability to operate in a multiplexed reporter gene design. Initially, this was evaluated with both reporter genes (i.e. <em>sfgfp</em> and <em>mcherry</em>) organised as separate transcriptional units (<strong><a id="xref-fig-4-6" class="xref-fig" href="#F4">Fig. 4b</a></strong>). Specific and independent repression by each mARi towards its cognate target was observed, with UTR-A-<em>sfgfp</em> being selectively targeted by mARi-A and UTR-B-<em>mcherry</em> being targeted by mARi-B (<strong><a id="xref-fig-4-7" class="xref-fig" href="#F4">Fig. 4b</a></strong> and <strong>Supplementary Fig. 8a</strong>). Further, specific and independent repression was also confirmed when three mARi regulators (mARi-A, mARi-B, and mARi-C) were employed to target UTR-A, UTR-B, and UTR-C preceding the <em>sfgfp, mcherry</em>, and <em>mtagbfp</em> genes, respectively (<strong><a id="xref-fig-4-8" class="xref-fig" href="#F4">Fig. 4c</a></strong> and <strong>Supplementary Fig. 8b</strong>).</p><p id="p-33">Two sets of mARi regulators were used to modulate gene expression in a two-gene operon with UTR-A-RBSc driving the expression of <em>sfgfp</em>, upstream of UTR-B-RBSc driving the expression of <em>mcherry</em>, constitutively expressed by the promoter P<sub>J23101_BASIC</sub> (<strong><a id="xref-fig-4-9" class="xref-fig" href="#F4">Fig. 4d</a></strong>). With both genes on a single transcript, the targeted repression for the first gene (<em>sfgfp</em>) is strongly affected by its cognate mARi-A, with only a minor affect from mARi-B, which targets the downstream <em>mcherry</em>; the downstream <em>mcherry</em> is most strongly attenuated by its cognate mARi-B, but it is also significantly attenuated by repression of the upstream <em>sfgfp</em> with full attenuation only reached with both mARi-A and B (<strong><a id="xref-fig-4-10" class="xref-fig" href="#F4">Fig. 4d</a></strong> and <strong>Supplementary Fig. 8c</strong>).</p></div></div><div class="section" id="sec-14"><h2 class="">DISCUSSION</h2><p id="p-34">Here we report the development, characterisation, and implementation of a modular post-transcriptional regulation system based on <em>trans</em>-encoded sRNAs. The modular design of mARi is derived from the highly composable structure of the natural MicC sRNAs scaffold (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">27</a>). A potential limitation of our system is that our mARi regulators utilise shared endogenous Hfq chaperone for their repression activity. However, endogenous Hfq is autoregulated by Hfq binding its own mRNA, even when expressed from plasmid-borne genes (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">59</a>) and the intracellular concentration of available Hfq can adapt rapidly to cellular changes in the RNA pool by facilitated recycling of the protein (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">64</a>). In addition, deviation from the optimum Hfq concentration can lead to suppression of sRNA activity in <em>E. coli</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">29</a>). Efforts to modulate sRNA activity by exogenous expression of Hfq would therefore require careful balancing of the whole system.</p><p id="p-35">Modification of the seed sequence, whilst retaining the sRNA scaffold, has enabled the reprogramming of natural sRNAs with different targets of interests that do not exist in nature (<a id="xref-ref-17-10" class="xref-bibr" href="#ref-17">17</a>, <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">18</a>, <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">28</a>). Here, we utilised this modularity for targeted gene repression by directing mARi regulators to standardised UTRs in the translation initiation region of the mRNA target. We demonstrated that targeted repression by binding to standardised sequences in the 5’-UTR upstream of the RBS were independent from both RBS and GOI contexts. The computationally designed orthogonal UTR sequences of the BASIC UTR-RBS linkers proved to be ideal target sites for mARi regulators (<strong><a id="xref-fig-2-11" class="xref-fig" href="#F2">Fig. 2c, e</a></strong> and <strong><a id="xref-fig-4-11" class="xref-fig" href="#F4">4</a></strong>). The standardised linker sequences that define the UTR have been computationally generated and validated to ensure their orthogonality in the DNA assembly process, as well as to the <em>E. coli</em> host (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">35</a>). Therefore, by targeting the BASIC DNA linkers, it was anticipated that the mARi would exert orthogonal, post-transcriptional gene regulation with minimal cross-interaction to non-cognate targets, bioparts, plasmid backbones and host genotypes (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">35</a>). The sequence composition of seed/target sites can be refined to optimise their binding affinity. This approach may result in higher repression and orthogonality. Therefore, it could also be coupled with an improved algorithm to create an sRNA optimised orthogonal sequence for the assembly process using BASIC DNA assembly.</p><p id="p-36">The mARi-based repression of a polycistronic mRNA provides some insight into its mechanism of action. Facilitated degradation of the mRNA would be expected to induce a similar repression level on both genes, even when only one is targeted, but single-gene targeting in an operon led to a differential effect between the operon’s genes. A non-targeted gene downstream of the mARi gene target exhibited significant repression, while a non-targeted gene upstream of the mARi gene target exhibited only slight repression (<strong><a id="xref-fig-4-12" class="xref-fig" href="#F4">Fig. 4d</a></strong>). This behaviour is consistent with steric hindrance of the ribosome for the UTR-RBS preventing translation initiation being the main mode of action of the mARi sRNA, rather than active degradation. The silencing effects on the downstream <em>mcherry</em> gene were also additive for both the gene’s cognate mARi and the upstream gene’s mARi, suggesting that repression at each UTR-RBS position was independent. It further suggests that read-through was possible from the upstream gene, even when the downstream RBS was repressed, but this was occluded when both UTRs were bound by their respective mARi regulators.</p><p id="p-37">The mechanism of action being based on the inhibition of translation initiation rather than active degradation may also be due to the designed mARi regulators and UTR-RBS linkers omitting an RNase E binding site, which is essential for RNase E-dependent cleavage. The elimination of RNase E in the mARi design was anticipated to reduce background degradation of unpaired mARi regulators and improve their cellular half-lives. The exclusion of an RNase E cleavage site in the mARi design may be beneficial for the implementation of an orthogonal repression system in various <em>E. coli</em> strains, with a response that is independent of the host genotype. Indeed, the system was observed to function across different <em>E. coli</em> strains and growth phases, demonstrating both its portability and robustness, notably with minimal difference between BL21(DE3) and BL21star(DE3), the latter being deficient in RNase E (<strong><a id="xref-fig-3-13" class="xref-fig" href="#F3">Fig. 3</a></strong>).</p><p id="p-38">The modular nature of the design framework meant facile variation of genetic design parameters. Our observations of design characteristics were consistent with previous studies (<a id="xref-ref-30-13" class="xref-bibr" href="#ref-30">30</a>), where repression increased as a function of length and hence free binding energy (<strong>Supplementary Fig. 2</strong>). Importantly, we have also identified spatial organisation and relative transcript abundance as being critical factors in efficient repression by sRNA (<strong><a id="xref-fig-3-14" class="xref-fig" href="#F3">Fig. 3</a></strong>). Expressing mARi in a high expression ratio and a single plasmid system with medium plasmid copy number (p15A) resulted in up to 80% repression activity. With <em>trans</em> expression of mARi from a second plasmid, the repression ratio was always worse, even when it was encoded on a higher plasmid copy number. Also somewhat surprising was the low efficiency of repression in a low-copy (pSC101) single plasmid system. This likely represents the importance of having the correct balance of all components relative to the native Hfq availability, as anticipated from Sagawa et al. (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">29</a>). The importance of spatial organisation is somewhat surprising given the abundance of natural <em>trans</em>-encoded systems, but it is likely to originate from enhanced proximity and hence binding between the mARi and its target.</p><p id="p-39">The modular nature of the mARi design enabled control of different genes and facilitated multiplexing of target genes. In total, five pairs of orthogonal mARi /UTR regulators were created and tested in this work. These mARi regulators were applied to multiplexed and simultaneous post-transcriptional regulation in multi-gene systems, including both multiple transcriptional units and an operon architecture (<strong><a id="xref-fig-4-13" class="xref-fig" href="#F4">Fig. 4</a></strong>). Importantly, the orthogonal basis of the BASIC design framework removed the need to design or insert bespoke target sites upstream of target genes, a drawback of previous works focused on reusable <em>trans</em>-encoded small RNAs (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">7</a>, <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>–<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>, <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">28</a>); it was implemented by simply assembling the mARi regulators and their cognate targets using BASIC DNA assembly without re-optimisation of the regulator for different target genes. This extensibility is essential for the simultaneous regulation of multiple genes in metabolic engineering, layered genetic control, and advanced genetic circuit applications, while scalability is possible through our recently demonstrated low cost automation platform for BASIC assembly (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">33</a>).</p></div><div class="section" id="sec-16"><h2 class="">Author Contributions</h2><p id="p-40">A.D.: Designed and performed all experiments and data analysis, devised the experimental strategy and wrote the manuscript.</p><p id="p-41">M.S.: Devised experimental strategy and design, assisted with the experiments and data interpretation and wrote the manuscript.</p><p id="p-42">G.-B.S.: Contributed to the experimental strategy, assisted with experimental design and data interpretation and wrote the manuscript.</p><p id="p-43">G.S.B.: Contributed to the experimental strategy, assisted with the experimental design and data interpretation and wrote the manuscript.</p><p id="teststring100">All Authors: Inserted a Testring into the HTML Doc.</p></div><div class="section" id="sec-17"><h2 class="">Competing interests</h2><p id="p-44">The authors declare no competing interests.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-45">A.D. received a PhD scholarship from Indonesia Endowment Fund for Education (LPDP). G.-B.S. gratefully acknowledges the support of the UK EPSRC through the EPSRC Fellowship for Growth EP/M002187/1 and of the Royal Academy of Engineering through the Chair in Emerging Technology programme. G.S.B. acknowledges the support of UK Research and Innovation through the Engineering and Physical Sciences Research Council [EP/R034915/1]; as well as the EU through H2020 [820699].</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label">1.</span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.1" data-doi="10.1038/nrmicro3239"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Cameron</span>,  <span class="cit-name-given-names">D.E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Bashor</span>,  <span class="cit-name-given-names">C.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Collins</span>,  <span class="cit-name-given-names">J.J.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">A brief history of synthetic biology</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol</abbr>., <span class="cit-vol">12</span>, <span class="cit-fpage">381</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol%26rft.volume%253D12%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3239%26rft_id%253Dinfo%253Apmid%252F24686414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24686414&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">2.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.2" data-doi="10.1038/nrm2698"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Purnick</span>,  <span class="cit-name-given-names">P.E.M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">The second wave of synthetic biology: From modules to systems</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol</abbr>., <span class="cit-vol">10</span>, <span class="cit-fpage">410</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DPurnick%26rft.auinit1%253DP.%2BE.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D410%26rft.epage%253D422%26rft.atitle%253DThe%2Bsecond%2Bwave%2Bof%2Bsynthetic%2Bbiology%253A%2Bfrom%2Bmodules%2Bto%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2698%26rft_id%253Dinfo%253Apmid%252F19461664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19461664&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266270900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">3.</span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.3" data-doi="10.1038/nrg3227"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Slusarczyk</span>,  <span class="cit-name-given-names">A.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lin</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Weiss</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Foundations for the design and implementation of synthetic genetic circuits</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet</abbr>., <span class="cit-vol">13</span>, <span class="cit-fpage">406</span>–<span class="cit-lpage">420</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSlusarczyk%26rft.auinit1%253DA.%2BL.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D406%26rft.epage%253D420%26rft.atitle%253DFoundations%2Bfor%2Bthe%2Bdesign%2Band%2Bimplementation%2Bof%2Bsynthetic%2Bgenetic%2Bcircuits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3227%26rft_id%253Dinfo%253Apmid%252F22596318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22596318&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">4.</span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.4"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelwick</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">MacDonald</span>,  <span class="cit-name-given-names">J.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Webb</span>,  <span class="cit-name-given-names">A.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Freemont</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Developments in the Tools and Methodologies of Synthetic Biology</span>. <abbr class="cit-jnl-abbrev">Front. Bioeng. Biotechnol</abbr>., <span class="cit-vol">2</span>, <span class="cit-fpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BBioeng.%2BBiotechnol%26rft.volume%253D2%26rft.spage%253D60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">5.</span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.5" data-doi="10.1007/s11047-018-9715-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Xiang</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Dalchau</span>,  <span class="cit-name-given-names">N.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Scaling up genetic circuit design for cellular computing: advances and prospects</span>. <abbr class="cit-jnl-abbrev">Nat. Comput</abbr>., <span class="cit-vol">17</span>, <span class="cit-fpage">833</span>–<span class="cit-lpage">853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BComput%26rft.volume%253D17%26rft.spage%253D833%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11047-018-9715-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11047-018-9715-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">6.</span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.6" data-doi="10.1016/j.molcel.2011.08.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storz</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wassarman</span>,  <span class="cit-name-given-names">K.M.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Regulation by Small RNAs in Bacteria: Expanding Frontiers</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>, <span class="cit-vol">43</span>, <span class="cit-fpage">880</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DStorz%26rft.auinit1%253DG.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D880%26rft.epage%253D891%26rft.atitle%253DRegulation%2Bby%2Bsmall%2BRNAs%2Bin%2Bbacteria%253A%2Bexpanding%2Bfrontiers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.08.022%26rft_id%253Dinfo%253Apmid%252F21925377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.08.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925377&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295309800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">7.</span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.7" data-doi="10.1038/s41467-018-07899-z"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">H.-H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Qian</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Del Vecchio</span>,  <span class="cit-name-given-names">D.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">A quasi-integral controller for adaptation of genetic modules to variable ribosome demand</span>. <abbr class="cit-jnl-abbrev">Nat. Commun</abbr>., <span class="cit-vol">9</span>, <span class="cit-fpage">5415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D9%26rft.spage%253D5415%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07899-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-07899-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">8.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.8" data-doi="10.1128/JB.00020-07"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Thompson</span>,  <span class="cit-name-given-names">K.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rhodius</span>,  <span class="cit-name-given-names">V.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gottesman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">σE regulates and is regulated by a small RNA in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>., <span class="cit-vol">189</span>, <span class="cit-fpage">4243</span>–<span class="cit-lpage">4256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00020-07%26rft_id%253Dinfo%253Apmid%252F17416652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OS8xMS80MjQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">9.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.9" data-doi="10.1016/j.molcel.2010.01.022"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Tu</span>,  <span class="cit-name-given-names">K.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Svenningsen</span>,  <span class="cit-name-given-names">S.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wingreen</span>,  <span class="cit-name-given-names">N.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bassler</span>,  <span class="cit-name-given-names">B.L.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Negative Feedback Loops Involving Small Regulatory RNAs Precisely Control the Vibrio harveyi Quorum-Sensing Response</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr>, <span class="cit-vol">37</span>, <span class="cit-fpage">567</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DTu%26rft.auinit1%253DK.%2BC.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D567%26rft.epage%253D579%26rft.atitle%253DNegative%2Bfeedback%2Bloops%2Binvolving%2Bsmall%2Bregulatory%2BRNAs%2Bprecisely%2Bcontrol%2Bthe%2BVibrio%2Bharveyi%2Bquorum-sensing%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.01.022%26rft_id%253Dinfo%253Apmid%252F20188674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.01.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20188674&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275349000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">10.</span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.10" data-doi="10.1093/nar/gkw642"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Brosse</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korobeinikova</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gottesman</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Guillier</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Unexpected properties of sRNA promoters allow feedback control via regulation of a two-component system</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">44</span>, <span class="cit-fpage">9650</span>–<span class="cit-lpage">9666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw642%26rft_id%253Dinfo%253Apmid%252F27439713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27439713&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">11.</span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.11" data-doi="10.1146/annurev-biophys-070816-034058"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nitzan</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Rehani</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Margalit</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Integration of Bacterial Small RNAs in Regulatory Networks</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Biophys</abbr>., <span class="cit-vol">46</span>, <span class="cit-fpage">131</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiophys%26rft.volume%253D46%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biophys-070816-034058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biophys-070816-034058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">12.</span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.12" data-doi="10.1111/j.1574-6976.2010.00241.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Beisel</span>,  <span class="cit-name-given-names">C.L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Storz</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2010</span>) <span class="cit-article-title">Base pairing small RNAs and their roles in global regulatory networks</span>. <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev</abbr>., <span class="cit-vol">34</span>, <span class="cit-fpage">866</span>–<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2010.00241.x%26rft_id%253Dinfo%253Apmid%252F20662934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6976.2010.00241.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20662934&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280633500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">13.</span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.13"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kuhlman</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Hwa</span>,  <span class="cit-name-given-names">T.</span></span> (<span class="cit-pub-date">2007</span>) <span class="cit-article-title">Quantitative characteristics of gene regulation by small RNA</span>. <abbr class="cit-jnl-abbrev">PLoS Biol</abbr>., <span class="cit-vol">5</span>, <span class="cit-fpage">1998</span>–<span class="cit-lpage">2010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D5%26rft.spage%253D1998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000249552300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">14.</span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.14" data-doi="10.1073/pnas.1207927109"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">J.B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Balasubramanian</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vanderpool</span>,  <span class="cit-name-given-names">C.K.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Small RNA binding-site multiplicity involved in translational regulation of a polycistronic mRNA</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>., <span class="cit-vol">109</span>, <span class="cit-fpage">E2691</span>– <span class="cit-lpage">E2698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1207927109%26rft_id%253Dinfo%253Apmid%252F22988087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQwL0UyNjkxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">15.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.15" data-doi="10.1101/gad.2030311"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morita</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Aiba</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">RNase E action at a distance: Degradation of target mRNAs mediated by an Hfq-binding small RNA in bacteria</span>. <abbr class="cit-jnl-abbrev">Genes Dev</abbr>., <span class="cit-vol">25</span>, <span class="cit-fpage">294</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMorita%26rft.auinit1%253DT.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D294%26rft.epage%253D298%26rft.atitle%253DRNase%2BE%2Baction%2Bat%2Ba%2Bdistance%253A%2Bdegradation%2Bof%2Btarget%2BmRNAs%2Bmediated%2Bby%2Ban%2BHfq-binding%2Bsmall%2BRNA%2Bin%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2030311%26rft_id%253Dinfo%253Apmid%252F21325130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjUvNC8yOTQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8yMC8yMDE5LjEyLjEyLjg3Mzg0NC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">16.</span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.16" data-doi="10.1038/nsmb.1631"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papenfort</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lucchini</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hinton</span>,  <span class="cit-name-given-names">J.C.D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Coding sequence targeting by MicC RNA reveals bacterial mRNA silencing downstream of translational initiation</span>. <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol</abbr>., <span class="cit-vol">16</span>, <span class="cit-fpage">840</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DPfeiffer%26rft.auinit1%253DV.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D840%26rft.epage%253D846%26rft.atitle%253DCoding%2Bsequence%2Btargeting%2Bby%2BMicC%2BRNA%2Breveals%2Bbacterial%2BmRNA%2Bsilencing%2Bdownstream%2Bof%2Btranslational%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.1631%26rft_id%253Dinfo%253Apmid%252F19620966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.1631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19620966&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268738700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">17.</span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.17" data-doi="10.1038/nprot.2013.105"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoo</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Na</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.Y.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc</abbr>., <span class="cit-vol">8</span>, <span class="cit-fpage">1694</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc%26rft.volume%253D8%26rft.spage%253D1694%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.105%26rft_id%253Dinfo%253Apmid%252F23928502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23928502&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">18.</span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.18" data-doi="10.1021/sb200001q"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sharma</span>,  <span class="cit-name-given-names">V.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yamamura</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yokobayashi</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Engineering artificial small RNAs for Conditional gene silencing in escherichia coli</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">1</span>, <span class="cit-fpage">6</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D1%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fsb200001q%26rft_id%253Dinfo%253Apmid%252F23651005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/sb200001q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23651005&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305437000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">19.</span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.19" data-doi="10.1093/nar/gkq1239"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Viegas</span>,  <span class="cit-name-given-names">S.C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Silva</span>,  <span class="cit-name-given-names">I.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Saramago</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Domingues</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Arraiano</span>,  <span class="cit-name-given-names">C.M.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Regulation of the small regulatory RNA MicA by ribonuclease III: a target-dependent pathway</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">39</span>, <span class="cit-fpage">2918</span>–<span class="cit-lpage">2930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1239%26rft_id%253Dinfo%253Apmid%252F21138960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21138960&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289628400045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">20.</span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.20" data-doi="10.1093/nar/gkr034"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Man</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Miao</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gong</span>,  <span class="cit-name-given-names">Q.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lu</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">F.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Artificial trans-encoded small non-coding RNAs specifically silence the selected gene expression in bacteria</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">39</span>, <span class="cit-fpage">e50</span>–<span class="cit-lpage">e50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr034%26rft_id%253Dinfo%253Apmid%252F21296758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21296758&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">21.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.21" data-doi="10.1093/nar/gkl697"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nakashima</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Good</span>,  <span class="cit-name-given-names">L.</span></span> (<span class="cit-pub-date">2006</span>) <span class="cit-article-title">Paired termini stabilize antisense RNAs and enhance conditional gene silencing in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">34</span>, <span class="cit-fpage">e138</span>–<span class="cit-lpage">e138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl697%26rft_id%253Dinfo%253Apmid%252F17062631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17062631&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">22.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.22" data-doi="10.1089/108729001750072100"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Engdahl</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lindell</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wagner</span>,  <span class="cit-name-given-names">E.G.H.</span></span> (<span class="cit-pub-date">2001</span>) <span class="cit-article-title">Introduction of an RNA Stability Element at the 5′-End of an Antisense RNA Cassette Increases the Inhibition of Target RNA Translation</span>. <abbr class="cit-jnl-abbrev">Antisense Nucleic Acid Drug Dev</abbr>., <span class="cit-vol">11</span>, <span class="cit-fpage">29</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntisense%2B%2526%2Bnucleic%2Bacid%2Bdrug%2Bdevelopment%26rft.stitle%253DAntisense%2BNucleic%2BAcid%2BDrug%2BDev%26rft.aulast%253DEngdahl%26rft.auinit1%253DH.%2BM.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D40%26rft.atitle%253DIntroduction%2Bof%2Ban%2BRNA%2Bstability%2Belement%2Bat%2Bthe%2B5%2527-end%2Bof%2Ban%2Bantisense%2BRNA%2Bcassette%2Bincreases%2Bthe%2Binhibition%2Bof%2Btarget%2BRNA%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F108729001750072100%26rft_id%253Dinfo%253Apmid%252F11258619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/108729001750072100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11258619&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167322300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">23.</span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.23" data-doi="10.1093/nar/gkv584"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">M.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trussler</span>,  <span class="cit-name-given-names">R.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Haniford</span>,  <span class="cit-name-given-names">D.B.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">A cis-encoded sRNA, Hfq and mRNA secondary structure act independently to suppress IS200 transposition</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">43</span>, <span class="cit-fpage">6511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv584%26rft_id%253Dinfo%253Apmid%252F26044710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26044710&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">24.</span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.24"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ghodasara</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Voigt</span>,  <span class="cit-name-given-names">C.A.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Balancing gene expression without library construction via a reusable sRNA pool</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">45</span>, <span class="cit-fpage">8116</span>–<span class="cit-lpage">8127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D45%26rft.spage%253D8116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">25.</span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.25" data-doi="10.1093/nar/gky828"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kelly</span>,  <span class="cit-name-given-names">C.L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">A.W.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Steel</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hancock</span>,  <span class="cit-name-given-names">E.J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heap</span>,  <span class="cit-name-given-names">J.T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Papachristodoulou</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Synthetic negative feedback circuits using engineered small RNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">46</span>, <span class="cit-fpage">9875</span>– <span class="cit-lpage">9889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253D9875%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">26.</span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.26" data-doi="10.1021/sb200021s"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Ceroni</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Furini</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stefan</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hochkoeppler</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Giordano</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">A synthetic post-transcriptional controller to explore the modular design of gene circuits</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">1</span>, <span class="cit-fpage">163</span>– <span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D1%26rft.spage%253D163%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fsb200021s%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/sb200021s&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">27.</span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.27" data-doi="10.1038/nbt.2461"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Na</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Yoo</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chung</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">J.H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.Y.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>., <span class="cit-vol">31</span>, <span class="cit-fpage">170</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D31%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2461%26rft_id%253Dinfo%253Apmid%252F23334451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23334451&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">28.</span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.28" data-doi="10.1021/acssynbio.6b00261"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lahiry</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stimple</span>,  <span class="cit-name-given-names">S.D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wood</span>,  <span class="cit-name-given-names">D.W.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lease</span>,  <span class="cit-name-given-names">R.A.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Retargeting a Dual-Acting sRNA for Multiple mRNA Transcript Regulation</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">6</span>, <span class="cit-fpage">648</span>–<span class="cit-lpage">658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D6%26rft.spage%253D648%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.6b00261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.6b00261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">29.</span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.29" data-doi="10.1093/nar/gkv777"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sagawa</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shin</span>,  <span class="cit-name-given-names">J.-E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hussein</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Lim</span>,  <span class="cit-name-given-names">H.N.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Paradoxical suppression of small RNA activity at high Hfq concentrations due to random-order binding</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">43</span>, <span class="cit-fpage">8502</span>–<span class="cit-lpage">8515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv777%26rft_id%253Dinfo%253Apmid%252F26261213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26261213&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">30.</span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.30" data-doi="10.1021/acssynbio.6b00036"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Hoynes-O’Connor</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">T.S.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Development of Design Rules for Reliable Antisense RNA Behavior in E. coli</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">5</span>, <span class="cit-fpage">1441</span>–<span class="cit-lpage">1454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D5%26rft.spage%253D1441%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facssynbio.6b00036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/acssynbio.6b00036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">31.</span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.31" data-doi="10.1016/j.ymeth.2018.01.001"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">Y.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">T.S.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Design rules of synthetic non-coding RNAs in bacteria</span>. <abbr class="cit-jnl-abbrev">Methods</abbr>, <span class="cit-vol">143</span>, <span class="cit-fpage">58</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D143%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2018.01.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2018.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">32.</span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.32"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Massaiu</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pasotti</span>,  <span class="cit-name-given-names">L.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casanova</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Politi</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zucca</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cusella De Angelis</span>,  <span class="cit-name-given-names">M.G.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Magni</span>,  <span class="cit-name-given-names">P.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">Quantification of the gene silencing performances of rationally-designed synthetic small RNAs</span>. <abbr class="cit-jnl-abbrev">Syst. Synth. Biol</abbr>., <span class="cit-vol">9</span>, <span class="cit-fpage">107</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BSynth.%2BBiol%26rft.volume%253D9%26rft.spage%253D107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">33.</span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.33" data-doi="10.1093/synbio/ysaa012"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Haines</span>,  <span class="cit-name-given-names">M.C.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baldwin</span>,  <span class="cit-name-given-names">G.S.</span></span> (<span class="cit-pub-date">2020</span>) <span class="cit-article-title">DNA-BOT: a low-cost, automated DNA assembly platform for synthetic biology</span>. <abbr class="cit-jnl-abbrev">Synth. Biol</abbr>., <span class="cit-vol">5</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSynth.%2BBiol%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsynbio%252Fysaa012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/synbio/ysaa012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">34.</span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.34" data-doi="10.1021/sb500356d"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackrow</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Fleming</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Trewhitt</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baldwin</span>,  <span class="cit-name-given-names">G.S.</span></span> (<span class="cit-pub-date">2015</span>) <span class="cit-article-title">BASIC: A New Biopart Assembly Standard for Idempotent Cloning Provides Accurate, Single-Tier DNA Assembly for Synthetic Biology</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">4</span>, <span class="cit-fpage">781</span>–<span class="cit-lpage">787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D4%26rft.spage%253D781%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fsb500356d%26rft_id%253Dinfo%253Apmid%252F25746445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/sb500356d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25746445&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">35.</span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.35" data-doi="10.1021/sb4001323"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Casini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Christodoulou</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Freemont</span>,  <span class="cit-name-given-names">P.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Baldwin</span>,  <span class="cit-name-given-names">G.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">MacDonald</span>,  <span class="cit-name-given-names">J.T.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">R2oDNA Designer: Computational Design of Biologically Neutral Synthetic DNA Sequences</span>. <abbr class="cit-jnl-abbrev">ACS Synth. Biol</abbr>., <span class="cit-vol">3</span>, <span class="cit-fpage">525</span>–<span class="cit-lpage">528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DACS%2BSynth.%2BBiol%26rft.volume%253D3%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fsb4001323%26rft_id%253Dinfo%253Apmid%252F24933158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/sb4001323&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24933158&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">36.</span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.36" data-doi="10.1093/bioinformatics/btp250"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Darty</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Denise</span>,  <span class="cit-name-given-names">A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ponty</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">VARNA: Interactive drawing and editing of the RNA secondary structure</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">1974</span>–<span class="cit-lpage">1975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp250%26rft_id%253Dinfo%253Apmid%252F19398448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19398448&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">37.</span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.37" data-doi="10.1093/bioinformatics/btn544"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Busch</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">A.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Backofen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2008</span>) <span class="cit-article-title">IntaRNA: Efficient prediction of bacterial sRNA targets incorporating target site accessibility and seed regions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr>, <span class="cit-vol">24</span>, <span class="cit-fpage">2849</span>–<span class="cit-lpage">2856</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn544%26rft_id%253Dinfo%253Apmid%252F18940824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn544&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18940824&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261456700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">38.</span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.38"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">P.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Georg</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sorescu</span>,  <span class="cit-name-given-names">D.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lott</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kleinkauf</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hess</span>,  <span class="cit-name-given-names">W.R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Backofen</span>,  <span class="cit-name-given-names">R.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">CopraRNA and IntaRNA: Predicting small RNA targets, networks and interaction domains</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">42</span>, <span class="cit-fpage">119</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D42%26rft.spage%253D119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">39.</span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.39" data-doi="10.1093/nar/gki206"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Markham</span>,  <span class="cit-name-given-names">N.R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Zuker</span>,  <span class="cit-name-given-names">M.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">DINAMelt web server for nucleic acid melting prediction</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">33</span>, <span class="cit-fpage">577</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLee%26rft.auinit1%253DB.-M.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D577%26rft.epage%253D586%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BXanthomonas%2Boryzae%2Bpathovar%2Boryzae%2BKACC10331%252C%2Bthe%2Bbacterial%2Bblight%2Bpathogen%2Bof%2Brice%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki206%26rft_id%253Dinfo%253Apmid%252F15673718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15673718&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226941000026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">40.</span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.40" data-doi="10.1038/nmeth.3727"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bonde</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pedersen</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Klausen</span>,  <span class="cit-name-given-names">M.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">S.I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wulff</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">A.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Herrgård</span>,  <span class="cit-name-given-names">M.J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sommer</span>,  <span class="cit-name-given-names">M.O.A.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Predictable tuning of protein expression in bacteria</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr>, <span class="cit-vol">13</span>, <span class="cit-fpage">233</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3727%26rft_id%253Dinfo%253Apmid%252F26752768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26752768&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">41.</span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.41" data-doi="10.1038/nbt.1568"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Salis</span>,  <span class="cit-name-given-names">H.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mirsky</span>,  <span class="cit-name-given-names">E.A.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Voigt</span>,  <span class="cit-name-given-names">C.A.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Automated design of synthetic ribosome binding sites to control protein expression</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol</abbr>., <span class="cit-vol">27</span>, <span class="cit-fpage">946</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSalis%26rft.auinit1%253DH.%2BM.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D946%26rft.epage%253D950%26rft.atitle%253DAutomated%2Bdesign%2Bof%2Bsynthetic%2Bribosome%2Bbinding%2Bsites%2Bto%2Bcontrol%2Bprotein%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1568%26rft_id%253Dinfo%253Apmid%252F19801975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1568&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19801975&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271472500026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">42.</span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.42" data-doi="10.1093/nar/gkt1139"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Espah Borujeni</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Channarasappa</span>,  <span class="cit-name-given-names">A.S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Salis</span>,  <span class="cit-name-given-names">H.M.</span></span> (<span class="cit-pub-date">2014</span>) <span class="cit-article-title">Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">42</span>, <span class="cit-fpage">2646</span>–<span class="cit-lpage">2659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1139%26rft_id%253Dinfo%253Apmid%252F24234441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24234441&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332381000053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">43.</span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.43" data-doi="10.1093/nar/gky329"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Raden</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ali</span>,  <span class="cit-name-given-names">S.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Alkhnbashi</span>,  <span class="cit-name-given-names">O.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Busch</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Costa</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Davis</span>,  <span class="cit-name-given-names">J.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eggenhofer</span>,  <span class="cit-name-given-names">F.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gelhausen</span>,  <span class="cit-name-given-names">R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Georg</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Heyne</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-etal">et al.</span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Freiburg RNA tools: A central online resource for RNA-focused research and teaching</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>., <span class="cit-vol">46</span>, <span class="cit-fpage">W25</span>–<span class="cit-lpage">W29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D46%26rft.spage%253DW25%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky329%26rft_id%253Dinfo%253Apmid%252F29788132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29788132&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">44.</span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.44" data-doi="10.1073/pnas.1303248110"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">P.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">A.S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Papenfort</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mann</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hess</span>,  <span class="cit-name-given-names">W.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Backofen</span>,  <span class="cit-name-given-names">R.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Georg</span>,  <span class="cit-name-given-names">J.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Comparative genomics boosts target prediction for bacterial small RNAs</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci</abbr>., <span class="cit-vol">110</span>, <span class="cit-fpage">E3487</span>–<span class="cit-lpage">E3496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1303248110%26rft_id%253Dinfo%253Apmid%252F23980183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzM3L0UzNDg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">45.</span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.45"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Storch</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Casini</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Mackrow</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Baldwin</span>,  <span class="cit-name-given-names">G.S.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">BASIC: A Simple and Accurate Modular DNA Assembly Method</span>. <abbr class="cit-jnl-abbrev">In Methods in Molecular Biology</abbr>.Vol. <span class="cit-vol">1472</span>, pp. <span class="cit-fpage">79</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIn%2BMethods%2Bin%2BMolecular%2BBiology%26rft.volume%253D1472%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">46.</span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.46"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vytvytska</span>,  <span class="cit-name-given-names">O.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moll</span>,  <span class="cit-name-given-names">I.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kaberdin</span>,  <span class="cit-name-given-names">V.R.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Gabain</span>,  <span class="cit-name-given-names">A. Von</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bla</span>,  <span class="cit-name-given-names">U.</span></span> (<span class="cit-pub-date">2000</span>) <abbr class="cit-jnl-abbrev">Hfq (HF1) stimulates ompA mRNA decay by interfering with ribosome binding</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">47.</span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.47" data-doi="10.1261/rna.2360205"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Večerek</span>,  <span class="cit-name-given-names">B.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Moll</span>,  <span class="cit-name-given-names">I.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Bläsi</span>,  <span class="cit-name-given-names">U.</span></span> (<span class="cit-pub-date">2005</span>) <span class="cit-article-title">Translational autocontrol of the Escherichia coli hfq RNA chaperone gene</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">976</span>–<span class="cit-lpage">984</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2360205%26rft_id%253Dinfo%253Apmid%252F15872186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjExLzYvOTc2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">48.</span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.48" data-doi="10.1038/219588a0"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Franze De Fernandez</span>,  <span class="cit-name-given-names">M.T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Eoyang</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">August</span>,  <span class="cit-name-given-names">J.T.</span></span> (<span class="cit-pub-date">1968</span>) <span class="cit-article-title">Factor fraction required for the synthesis of bacteriophage Qβ-RNA</span>. <abbr class="cit-jnl-abbrev">Nature</abbr>, <span class="cit-vol">219</span>, <span class="cit-fpage">588</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFranze%2Bde%2BFernandez%26rft.auinit1%253DM.%2BT.%26rft.volume%253D219%26rft.issue%253D5154%26rft.spage%253D588%26rft.epage%253D590%26rft.atitle%253DFactor%2Bfraction%2Brequired%2Bfor%2Bthe%2Bsynthesis%2Bof%2Bbacteriophage%2BQbeta-RNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F219588a0%26rft_id%253Dinfo%253Apmid%252F4874917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/219588a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4874917&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1968B569000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label">49.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.49" data-doi="10.1186/s13068-018-1097-9"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Song</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Pei</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Diao</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Cui</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">L.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">W.</span></span> (<span class="cit-pub-date">2018</span>) <span class="cit-article-title">Re-direction of carbon flux to key precursor malonyl-CoA via artificial small RNAs in photosynthetic Synechocystis sp. PCC 6803</span>. <abbr class="cit-jnl-abbrev">Biotechnol. Biofuels</abbr>, <span class="cit-vol">11</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BBiofuels%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13068-018-1097-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13068-018-1097-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">50.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.50" data-doi="10.1002/2211-5463.12218"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Huis in ‘t</span> <span class="cit-auth"><span class="cit-name-surname">Veld</span>,  <span class="cit-name-given-names">R.A.G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kramer</span>,  <span class="cit-name-given-names">G.</span></span>, <span class="cit-auth"><span class="cit-name-surname">van der Ende</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Speijer</span>,  <span class="cit-name-given-names">D.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Pannekoek</span>,  <span class="cit-name-given-names">Y.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">The Hfq regulon of Neisseria meningitidis</span>. <abbr class="cit-jnl-abbrev">FEBS Open Bio</abbr>, <span class="cit-vol">7</span>, <span class="cit-fpage">777</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BOpen%2BBio%26rft.volume%253D7%26rft.spage%253D777%26rft_id%253Dinfo%253Adoi%252F10.1002%252F2211-5463.12218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/2211-5463.12218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">51.</span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.51" data-doi="10.1128/JB.00254-11"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Puerta-Fernández</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Vioque</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Hfq is required for optimal nitrate assimilation in the cyanobacterium Anabaena sp. strain PCC 7120</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>., <span class="cit-vol">193</span>, <span class="cit-fpage">3546</span>–<span class="cit-lpage">3555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00254-11%26rft_id%253Dinfo%253Apmid%252F21602329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5My8xNC8zNTQ2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">52.</span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.52" data-doi="10.1128/JB.186.20.6689-6697.2004"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Blyn</span>,  <span class="cit-name-given-names">L.B.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Storz</span>,  <span class="cit-name-given-names">G.</span></span> (<span class="cit-pub-date">2004</span>) <span class="cit-article-title">MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>., <span class="cit-vol">186</span>, <span class="cit-fpage">6689</span>–<span class="cit-lpage">6697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DChen%26rft.auinit1%253DS.%26rft.volume%253D186%26rft.issue%253D20%26rft.spage%253D6689%26rft.epage%253D6697%26rft.atitle%253DMicC%252C%2Ba%2BSecond%2BSmall-RNA%2BRegulator%2Bof%2BOmp%2BProtein%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.20.6689-6697.2004%26rft_id%253Dinfo%253Apmid%252F15466019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ni8yMC82Njg5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">53.</span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.53" data-doi="10.1038/nrmicro2615"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Luisi</span>,  <span class="cit-name-given-names">B.F.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Hfq and its constellation of RNA</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol</abbr>., <span class="cit-vol">9</span>, <span class="cit-fpage">578</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DVogel%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D578%26rft.epage%253D589%26rft.atitle%253DHfq%2Band%2Bits%2Bconstellation%2Bof%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2615%26rft_id%253Dinfo%253Apmid%252F21760622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21760622&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">54.</span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.54" data-doi="10.1073/pnas.0901466106"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Mitarai</span>,  <span class="cit-name-given-names">N.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Benjamin</span>,  <span class="cit-name-given-names">J.-A.M.J.-A.M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Krishna</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Semsey</span>,  <span class="cit-name-given-names">S.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Csiszovszki</span>,  <span class="cit-name-given-names">Z.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Massé</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Sneppen</span>,  <span class="cit-name-given-names">K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Masse</span>,  <span class="cit-name-given-names">E.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Sneppen</span>,  <span class="cit-name-given-names">K.</span></span> (<span class="cit-pub-date">2009</span>) <span class="cit-article-title">Dynamic features of gene expression control by small regulatory RNAs</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. U. S. A</abbr>., <span class="cit-vol">106</span>, <span class="cit-fpage">10655</span>–<span class="cit-lpage">10659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0901466106%26rft_id%253Dinfo%253Apmid%252F19541626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzI2LzEwNjU1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">55.</span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.55" data-doi="10.1186/s12934-015-0402-6"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Jahn</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Vorpahl</span>,  <span class="cit-name-given-names">C.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Hübschmann</span>,  <span class="cit-name-given-names">T.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Harms</span>,  <span class="cit-name-given-names">H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Copy number variability of expression plasmids determined by cell sorting and droplet digital PCR</span>. <abbr class="cit-jnl-abbrev">Microb. Cell Fact</abbr>., <span class="cit-vol">15</span>, <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrob.%2BCell%2BFact%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12934-015-0402-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12934-015-0402-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label">56.</span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.56"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Teimouri</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Korkmazhan</span>,  <span class="cit-name-given-names">E.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Stavans</span>,  <span class="cit-name-given-names">J.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Levine</span>,  <span class="cit-name-given-names">E.</span></span> (<span class="cit-pub-date">2017</span>) <span class="cit-article-title">Sub-cellular mRNA localization modulates the regulation of gene expression by small RNAs in bacteria</span>. <abbr class="cit-jnl-abbrev">Phys. Biol</abbr>., <span class="cit-vol">14</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label">57.</span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.57" data-doi="10.1016/j.cbpa.2013.10.003"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">A.A.K.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Segall-Shapiro</span>,  <span class="cit-name-given-names">T.H.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Voigt</span>,  <span class="cit-name-given-names">C.A.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Advances in genetic circuit design: Novel biochemistries, deep part mining, and precision gene expression</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Chem. Biol</abbr>., <span class="cit-vol">17</span>, <span class="cit-fpage">878</span>–<span class="cit-lpage">892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BChem.%2BBiol%26rft.volume%253D17%26rft.spage%253D878%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbpa.2013.10.003%26rft_id%253Dinfo%253Apmid%252F24268307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbpa.2013.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24268307&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">58.</span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.58"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Bradley</span>,  <span class="cit-name-given-names">R.W.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Buck</span>,  <span class="cit-name-given-names">M.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span> (<span class="cit-pub-date">2016</span>) <span class="cit-article-title">Tools and Principles for Microbial Gene Circuit Engineering</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol</abbr>., <span class="cit-vol">428</span>, <span class="cit-fpage">862</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol%26rft.volume%253D428%26rft.spage%253D862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label">59.</span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.59" data-doi="10.1261/rna.068106.118"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Morita</span>,  <span class="cit-name-given-names">T.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Aiba</span>,  <span class="cit-name-given-names">H.</span></span> (<span class="cit-pub-date">2019</span>) <span class="cit-article-title">Mechanism and physiological significance of autoregulation of the Escherichia coli hfq gene</span>. <abbr class="cit-jnl-abbrev">RNA</abbr>, <span class="cit-vol">25</span>, <span class="cit-fpage">264</span>–<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.068106.118%26rft_id%253Dinfo%253Apmid%252F30487269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjI1LzIvMjY0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">60.</span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.60" data-doi="10.1128/JB.181.20.6361-6370.1999"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Azam</span>,  <span class="cit-name-given-names">T.A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Iwata</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Nishimura</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Ueda</span>,  <span class="cit-name-given-names">S.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ishihama</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1999</span>) <span class="cit-article-title">Growth phase-dependent variation in protein composition of the Escherichia coli nucleoid</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>., <span class="cit-vol">181</span>, <span class="cit-fpage">6361</span>–<span class="cit-lpage">6370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAli%2BAzam%26rft.auinit1%253DT.%26rft.volume%253D181%26rft.issue%253D20%26rft.spage%253D6361%26rft.epage%253D6370%26rft.atitle%253DGrowth%2BPhase-Dependent%2BVariation%2Bin%2BProtein%2BComposition%2Bof%2Bthe%2BEscherichia%2Bcoli%2BNucleoid%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.181.20.6361-6370.1999%26rft_id%253Dinfo%253Apmid%252F10515926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4MS8yMC82MzYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">61.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.61" data-doi="10.1128/jb.176.2.531-534.1994"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Kato</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Wada</span>,  <span class="cit-name-given-names">A.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Inokuchi</span>,  <span class="cit-name-given-names">Y.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Ishihama</span>,  <span class="cit-name-given-names">A.</span></span> (<span class="cit-pub-date">1994</span>) <span class="cit-article-title">Regulation of the Escherichia coli hfq gene encoding the host factor for phage Q(β)</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol</abbr>., <span class="cit-vol">176</span>, <span class="cit-fpage">531</span>–<span class="cit-lpage">534</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKajitani%26rft.auinit1%253DM.%26rft.volume%253D176%26rft.issue%253D2%26rft.spage%253D531%26rft.epage%253D534%26rft.atitle%253DRegulation%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bhfq%2Bgene%2Bencoding%2Bthe%2Bhost%2Bfactor%2Bfor%2Bphage%2BQ%2Bbeta.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.176.2.531-534.1994%26rft_id%253Dinfo%253Apmid%252F8288550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc2LzIvNTMxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMjAvMjAxOS4xMi4xMi44NzM4NDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label">62.</span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2019.12.12.873844v4.62" data-doi="10.1186/gb-2012-13-5-r37"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Yoon</span>,  <span class="cit-name-given-names">S.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">C.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Xia</span>,  <span class="cit-name-given-names">X.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">D.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Shim</span>,  <span class="cit-name-given-names">J.H.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">S.Y.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Oh</span>,  <span class="cit-name-given-names">T.K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">J.F.</span></span> (<span class="cit-pub-date">2012</span>) <span class="cit-article-title">Comparative multi-omics systems analysis of Escherichia coli strains B and K-12</span>. <abbr class="cit-jnl-abbrev">Genome Biol</abbr>., <span class="cit-vol">13</span>, <span class="cit-fpage">R37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DYoon%26rft.auinit1%253DS.%2BH.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253DR37%26rft.epage%253DR37%26rft.atitle%253DComparative%2Bmulti-omics%2Bsystems%2Banalysis%2Bof%2BEscherichia%2Bcoli%2Bstrains%2BB%2Band%2BK-12.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-5-r37%26rft_id%253Dinfo%253Apmid%252F22632713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-5-r37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22632713&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">63.</span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.63" data-doi="10.1111/j.1365-2958.2011.07907.x"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">K.</span></span> and <span class="cit-auth"><span class="cit-name-surname">Gottesman</span>,  <span class="cit-name-given-names">S.</span></span> (<span class="cit-pub-date">2011</span>) <span class="cit-article-title">Competition among Hfq-binding small RNAs in Escherichia coli</span>. <abbr class="cit-jnl-abbrev">Mol. Microbiol</abbr>., <span class="cit-vol">82</span>, <span class="cit-fpage">1545</span>–<span class="cit-lpage">1562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DMoon%26rft.auinit1%253DK.%26rft.volume%253D82%26rft.issue%253D6%26rft.spage%253D1545%26rft.epage%253D1562%26rft.atitle%253DCompetition%2Bamong%2BHfq-binding%2Bsmall%2BRNAs%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07907.x%26rft_id%253Dinfo%253Apmid%252F22040174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07907.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22040174&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label">64.</span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference 64. in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-2019.12.12.873844v4.64" data-doi="10.4161/rna.24044"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Wagner</span>,  <span class="cit-name-given-names">E.G.H.</span></span> (<span class="cit-pub-date">2013</span>) <span class="cit-article-title">Cycling of RNAs on Hfq</span>. <abbr class="cit-jnl-abbrev">RNA Biol</abbr>., <span class="cit-vol">10</span>, <span class="cit-fpage">619</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol%26rft.volume%253D10%26rft.spage%253D619%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.24044%26rft_id%253Dinfo%253Apmid%252F23466677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.24044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23466677&amp;link_type=MED&amp;atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F20%2F2019.12.12.873844.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321701600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span class="related-urls"></span></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-disqus-comment" >
  
      
  
  <div class="pane-content">
    <div id="disqus_thread"><noscript><p><a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2019.12.12.873844v4" class="" data-icon-position="" data-hide-link-title="0">View the discussion thread.</a></p></noscript></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><span class="icon-chevron-up"></span> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10.1101/2021.09.30.462548v2" title="Prime Editing Efficiency and Fidelity are Enhanced in the Absence of Mismatch Repair" rel="prev" class="pager-link-prev link-icon"><span class="icon-circle-arrow-left"></span> <span class="title">Previous</span></a></span><span class="pager-next"><a href="/content/10.1101/2021.12.09.471697v3" title="Long read sequencing reveals sequential complex rearrangements driven by Hepatitis B virus integration" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <span class="icon-circle-arrow-right"></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    Posted&nbsp;January 20, 2022.  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-art-tools" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/10.1101/2019.12.12.873844v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon"><span class="icon-external-link-sign"></span> <span class="title">Download PDF</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-biorxiv-supplementary-fragment" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10.1101/2019.12.12.873844v4.supplementary-material"><i class="icon-file"></i>Supplementary Material </a></p>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10.1101/2019.12.12.873844v4.full" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word about bioRxiv.</p><p>NOTE: Your email address is requested solely to identify you as the sender of this article.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10.1101/2019.12.12.873844v4" class="active" data-icon-position="" data-hide-link-title="0">A modular RNA interference system for multiplexed gene regulation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from bioRxiv
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see this page from the bioRxiv website.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/2359265" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-he-3FBotIG_tWxWr9NcDxKISBIBIugjawY77CFNvLAs" />
<input type="hidden" name="form_id" value="forward_form" />
<fieldset class="captcha form-wrapper"><legend><span class="fieldset-legend">CAPTCHA</span></legend><div class="fieldset-wrapper"><div class="fieldset-description">This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.</div><input type="hidden" name="captcha_sid" value="484465876" />
<input type="hidden" name="captcha_token" value="89805397f15abe412bc5729194ff6144" />
<input type="hidden" name="captcha_response" value="Google no captcha" />
<div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image"></div></div></fieldset>
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><span class="icon-share-alt"></span> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article node2359265--3" data-node-nid="2359265" id="node-2359265--4629301625" data-pisa="biorxiv;2019.12.12.873844v4" data-pisa-master="biorxiv;2019.12.12.873844" data-seqnum="2359265" data-apath="/biorxiv/early/2022/01/20/2019.12.12.873844.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A modular RNA interference system for multiplexed gene regulation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ari</span> <span class="nlm-surname">Dwijayanti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marko</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Guy-Bart</span> <span class="nlm-surname">Stan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Geoff S.</span> <span class="nlm-surname">Baldwin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.12.873844; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.12.873844 </span></div>
  
  
  </div>
</div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;title=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;title=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;text=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;t=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;title=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;title=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation&amp;summary=&amp;source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;title=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools"><span class="icon-globe"></span> <span class="title">Citation Tools</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2359265--5" data-node-nid="2359265" id="citation-node-2359265--61054692343" data-pisa="biorxiv;2019.12.12.873844v4" data-pisa-master="biorxiv;2019.12.12.873844" data-seqnum="2359265" data-apath="/biorxiv/early/2022/01/20/2019.12.12.873844.atom"><div  class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix" >

      <div class="highwire-cite-title" >
      <div class="highwire-cite-title">A modular RNA interference system for multiplexed gene regulation</div>    </div>
  
      <div  class="highwire-cite-authors" ><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Ari</span> <span class="nlm-surname">Dwijayanti</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Marko</span> <span class="nlm-surname">Storch</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Guy-Bart</span> <span class="nlm-surname">Stan</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Geoff S.</span> <span class="nlm-surname">Baldwin</span></span></span></div>
  
      <div  class="highwire-cite-metadata" ><span  class="highwire-cite-metadata-journal highwire-cite-metadata">bioRxiv </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2019.12.12.873844; </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="doi_label">doi:</span> https://doi.org/10.1101/2019.12.12.873844 </span></div>
  
  
  </div>
</div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage&amp;rft.epage&amp;rft.atitle=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation&amp;rft.volume&amp;rft.issue&amp;rft.date=2022-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=bioRxiv&amp;rft.au=Dwijayanti%2C+Ari&amp;rft.au=Storch%2C+Marko&amp;rft.au=Stan%2C+Guy-Bart&amp;rft.au=Baldwin%2C+Geoff+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/2359265/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/2359265/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/2359265/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/2359265/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/2359265/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/2359265/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/2359265/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/2359265/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/2359265/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/2359265/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/2359265/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/2359265/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;count=horizontal&amp;via=&amp;text=A%20modular%20RNA%20interference%20system%20for%20multiplexed%20gene%20regulation&amp;counturl=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Tweet Widget</span></a></li><li><a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2019.12.12.873844v4&amp;layout=button_count&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;width=100&amp;height=21&amp;font=&amp;locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Facebook Like</span></a></li><li class="last"><a href="https://www.biorxiv.org/content/10.1101/2019.12.12.873844v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0"><span class="element-invisible">Google Plus One</span></a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-collections" >
  
        <h2 class="pane-title">Subject Area</h2>
    
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-collections"><div class="highwire-list"><ul class="highwire-article-collection-term-list"><li class="first last odd"><span class="highwire-article-collection-term"><a href="/collection/synthetic-biology" class="highlight" data-icon-position="" data-hide-link-title="0">Synthetic Biology<i class="icon-caret-right"></i>
</a></span></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2" >
  
      
  
  <div class="pane-content">
    <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
<div class="panel-flexible-inside panels-flexible-new-inside">
<div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
  <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
<div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
  
      
  <div class="snippet-content">
    <b>Subject Areas</b>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet" >
  
      
  
  <div class="pane-content">
    <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
  
      
  <div class="snippet-content">
    <a href="/content/early/recent"><strong>All Articles</strong></a>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-subject-collections" >
  
      
  
  <div class="pane-content">
    <ul id="collection" class="collection highwire-list-expand"><li class="outer collection depth-2  child first"><div class = "data-wrapper"><a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">Animal Behavior and Cognition</a> <span class = "article-count">(4123)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">Biochemistry</a> <span class = "article-count">(8837)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">Bioengineering</a> <span class = "article-count">(6539)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">Bioinformatics</a> <span class = "article-count">(23503)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">Biophysics</a> <span class = "article-count">(11822)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">Cancer Biology</a> <span class = "article-count">(9242)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">Cell Biology</a> <span class = "article-count">(13358)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">Clinical Trials</a> <span class = "article-count">(138)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">Developmental Biology</a> <span class = "article-count">(7456)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">Ecology</a> <span class = "article-count">(11438)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">Epidemiology</a> <span class = "article-count">(2066)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">Evolutionary Biology</a> <span class = "article-count">(15186)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">Genetics</a> <span class = "article-count">(10460)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">Genomics</a> <span class = "article-count">(14058)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">Immunology</a> <span class = "article-count">(9196)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">Microbiology</a> <span class = "article-count">(22242)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">Molecular Biology</a> <span class = "article-count">(8840)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">Neuroscience</a> <span class = "article-count">(47702)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">Paleontology</a> <span class = "article-count">(352)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">Pathology</a> <span class = "article-count">(1433)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">Pharmacology and Toxicology</a> <span class = "article-count">(2493)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">Physiology</a> <span class = "article-count">(3742)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">Plant Biology</a> <span class = "article-count">(8103)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">Scientific Communication and Education</a> <span class = "article-count">(1439)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">Synthetic Biology</a> <span class = "article-count">(2230)</span></div></li>
<li class="outer collection depth-2  child"><div class = "data-wrapper"><a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">Systems Biology</a> <span class = "article-count">(6050)</span></div></li>
<li class="outer collection depth-2  child last"><div class = "data-wrapper"><a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">Zoology</a> <span class = "article-count">(1259)</span></div></li>
</ul>  </div>

  
  </div>
  </div>
</div>
</div>
</div>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js"></script>
<script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
function euCookieComplianceLoadScripts() {}
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var eu_cookie_compliance_cookie_name = "";
//--><!]]>
</script>
  </body>
</html>
