{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "!createdb -h fonduer-postgres-dev -U postgres jkracht\n",
    "#!dropdb -h fonduer-postgres-dev -U postgres jkracht"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: nltk in /usr/local/lib/python3.7/site-packages (3.8.1)\n",
      "Requirement already satisfied: click in /usr/local/lib/python3.7/site-packages (from nltk) (8.1.3)\n",
      "Requirement already satisfied: tqdm in /usr/local/lib/python3.7/site-packages (from nltk) (4.65.0)\n",
      "Requirement already satisfied: joblib in /usr/local/lib/python3.7/site-packages (from nltk) (1.2.0)\n",
      "Requirement already satisfied: regex>=2021.8.3 in /usr/local/lib/python3.7/site-packages (from nltk) (2023.5.5)\n",
      "Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.7/site-packages (from click->nltk) (5.2.0)\n",
      "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/site-packages (from importlib-metadata->click->nltk) (3.15.0)\n",
      "Requirement already satisfied: typing-extensions>=3.6.4 in /usr/local/lib/python3.7/site-packages (from importlib-metadata->click->nltk) (4.5.0)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n",
      "\n",
      "[notice] A new release of pip is available: 23.0.1 -> 23.1.2\n",
      "[notice] To update, run: pip install --upgrade pip\n",
      "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
      "[nltk_data]   Package wordnet is already up-to-date!\n",
      "[2023-05-31 14:30:35,796][INFO] fonduer.meta:49 - Setting logging directory to: logs/2023-05-31_14-30-35\n",
      "[2023-05-31 14:30:35,810][INFO] fonduer.meta:135 - Connecting user:postgres to fonduer-postgres-dev:5432/jkracht\n",
      "[2023-05-31 14:30:35,845][INFO] fonduer.meta:162 - Initializing the storage schema\n",
      "[2023-05-31 14:30:37,268][INFO] fonduer.utils.udf:67 - Running UDF...\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "55fcb3be694c4a498dafbdb29ad883c4",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/2 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "%matplotlib inline\n",
    "import os\n",
    "import sys\n",
    "import subprocess\n",
    "import logging\n",
    "from pprint import pprint\n",
    "\n",
    "subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'nltk'])\n",
    "\n",
    "from nltk.corpus import wordnet as wn\n",
    "import nltk\n",
    "nltk.download('wordnet')\n",
    "\n",
    "from fonduer.candidates.matchers import LambdaFunctionMatcher, Union, Intersect, RegexMatchSpan\n",
    "\n",
    "from fonduer import Meta, init_logging\n",
    "from fonduer.parser.preprocessors import HTMLDocPreprocessor\n",
    "from fonduer.parser import Parser\n",
    "from fonduer.parser.models import Document, Sentence\n",
    "\n",
    "PARALLEL = 2\n",
    "ATTRIBUTE = \"jkracht\"\n",
    "conn_string = f'postgresql://postgres@fonduer-postgres-dev:5432/{ATTRIBUTE}'\n",
    "\n",
    "init_logging(log_dir=\"logs\")\n",
    "\n",
    "session = Meta.init(conn_string).Session()\n",
    "\n",
    "# Document parser\n",
    "docs_path = \"data/test_collection2/\"\n",
    "doc_preprocessor = HTMLDocPreprocessor(docs_path)\n",
    "\n",
    "corpus_parser = Parser(session, structural=True, lingual=True)\n",
    "corpus_parser.apply(doc_preprocessor, parallelism=PARALLEL)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Num Docs: 2\n",
      "Num Sections: 4\n",
      "Num Tables: 0\n",
      "Num Cells: 0\n",
      "Num Sentences: 3470\n",
      "Num Paragraphs: 2591\n",
      "Num Figures: 40\n",
      "Num Captions: 0\n"
     ]
    }
   ],
   "source": [
    "from fonduer.parser.models import Document, Section, Table, Cell, Paragraph, Sentence, Figure, Caption\n",
    "print(f\"Num Docs: {session.query(Document).count()}\")\n",
    "\n",
    "print(f\"Num Sections: {session.query(Section).count()}\")\n",
    "print(f\"Num Tables: {session.query(Table).count()}\")\n",
    "print(f\"Num Cells: {session.query(Cell).count()}\")\n",
    "print(f\"Num Sentences: {session.query(Sentence).count()}\")\n",
    "print(f\"Num Paragraphs: {session.query(Paragraph).count()}\")\n",
    "\n",
    "print(f\"Num Figures: {session.query(Figure).count()}\")\n",
    "print(f\"Num Captions: {session.query(Caption).count()}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[2023-05-31 14:39:29,165][INFO] fonduer.candidates.mentions:467 - Clearing table: task\n",
      "[2023-05-31 14:39:29,167][INFO] fonduer.utils.udf:67 - Running UDF...\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "2f86c095de05465f88250a767076c716",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/2 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Tasks: 0\n"
     ]
    }
   ],
   "source": [
    "# mention\n",
    "from fonduer.candidates.models import mention_subclass\n",
    "from fonduer.candidates import MentionExtractor\n",
    "from fonduer.candidates import MentionNgrams\n",
    "from fonduer.candidates.mentions import MentionSentences, MentionCaptions, MentionParagraphs, MentionDocuments, MentionTables\n",
    "from fonduer.utils.data_model_utils.textual import get_neighbor_sentence_ngrams, get_left_ngrams\n",
    "Task = mention_subclass(\"Task\")\n",
    "\n",
    "\n",
    "list_of_headlines = [\n",
    "                     #\"Acknowledgements\", \"Acknowledgement\", \"acknowledgements\", \"acknowledgement\", \n",
    "                     \"Contributions\", \"Contribution\", \"contribution\", \"contributions\",\n",
    "                     \"Author Contributions\",\n",
    "                     #\"Credits\", \"Credit\", \"credits\", \"credit\",\n",
    "                     \"Überschrift 1\"]\n",
    "    \n",
    "# def mention_span_in_acknowledments_matches_verb(mention):\n",
    "#     span_string = mention.get_span()\n",
    "\n",
    "#     try:\n",
    "#         # get last paragraphs first sentence (headline of the paragraph)\n",
    "#         headline_of_last_paragraph = session.query(Paragraph).get(mention.sentence.paragraph_id-1).sentences[0].text\n",
    "\n",
    "#         # check if last headline is listed to extract mentions of\n",
    "#         #if headline_of_last_paragraph in list_of_headlines:\n",
    "#         if any(option in headline_of_last_paragraph for option in list_of_headlines):\n",
    "\n",
    "#             #test if span is a verb\n",
    "#             for word in wn.synsets(span_string):\n",
    "#                 if word.pos() == \"v\": # and word.name().split(\".\")[0] == span_string.lower():\n",
    "#                     return True # case: span is a ver in a wanted paragraph\n",
    "\n",
    "#             return False # case: span is not a verb\n",
    "        \n",
    "#         else:\n",
    "#             return False # case: span not in wanted paragraph\n",
    "#     except:\n",
    "#         return False # case: no prior paragraph\n",
    "\n",
    "\n",
    "def mention_get_verb(mention):\n",
    "    \"\"\"\n",
    "    function checks\n",
    "    \"\"\"\n",
    "    span_string = mention.get_span()\n",
    "    doc_id = mention.sentence.paragraph.document_id\n",
    "    para_pos = mention.sentence.paragraph.position\n",
    "    try:\n",
    "        headline = session.query(Paragraph).filter(Paragraph.position >= para_pos-10, Paragraph.position <= para_pos-1, Paragraph.document_id == doc_id)\n",
    "        for h in headline:\n",
    "            x = h.sentences[0].text\n",
    "            if any(option.lower() in x.lower() for option in list_of_headlines):\n",
    "                return True\n",
    "                # #test if span is a verb\n",
    "                # for word in wn.synsets(span_string):\n",
    "                #     if word.pos() == \"v\": # and word.name().split(\".\")[0] == span_string.lower():\n",
    "                #         # print(\"1\")\n",
    "                #         return True # case: span is a ver in a wanted paragraph\n",
    "                # # print(\"2\")\n",
    "                # return False # case: span is not a verb\n",
    "        \n",
    "        else:\n",
    "            # print(\"3\")\n",
    "            return False # case: span not in wanted paragraph\n",
    "    except:\n",
    "        # print(\"4\")\n",
    "        return False\n",
    "        \n",
    "\n",
    "# matcher_task1 = LambdaFunctionMatcher(func = mention_span_in_acknowledments_matches_verb)\n",
    "matcher_task2 = LambdaFunctionMatcher(func = mention_get_verb)\n",
    "\n",
    "task_space = MentionSentences()\n",
    "\n",
    "matchers_task = Intersect(\n",
    "    # matcher_task1,\n",
    "    matcher_task2\n",
    "\n",
    ")\n",
    "\n",
    "docs = session.query(Document).all()\n",
    "\n",
    "mention_extractor = MentionExtractor(\n",
    "    session,\n",
    "    [Task],\n",
    "    [task_space], #task_sentences #task_space\n",
    "    [matchers_task],\n",
    "    parallelism=PARALLEL,\n",
    ")\n",
    "\n",
    "mention_extractor.apply(docs, parallelism=PARALLEL, clear=True)\n",
    "print(\n",
    "    f\"Number of Tasks: {session.query(Task).count()}\",\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "para_pos = 11\n",
    "a = list(session.query(Paragraph).filter(Paragraph.position <= 11, Paragraph.position >= 1, Paragraph.document_id == 118064))\n",
    "for b in a:\n",
    "    print(b.sentences[0].text)\n",
    "#list(session.query(Paragraph).filter(Paragraph.position == 10, Paragraph.document_id == 118064))[0].sentences[0].text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "A modular RNA interference system for multiplexed gene regulation | bioRxiv\n",
      "0\n",
      "xxxxx\n",
      "Skip to main content\n",
      "1\n",
      "xxxxx\n",
      "Home\n",
      "2\n",
      "xxxxx\n",
      "About\n",
      "3\n",
      "xxxxx\n",
      "Submit\n",
      "4\n",
      "xxxxx\n",
      "ALERTS / RSS\n",
      "5\n",
      "xxxxx\n",
      "Search for this keyword\n",
      "6\n",
      "xxxxx\n",
      "Advanced Search\n",
      "7\n",
      "xxxxx\n",
      "New Results\n",
      "8\n",
      "xxxxx\n",
      "A modular RNA interference system for multiplexed gene regulation\n",
      "9\n",
      "xxxxx\n",
      "Ari Dwijayanti, Marko Storch, Guy-Bart Stan, Geoff S. Baldwin\n",
      "10\n",
      "xxxxx\n",
      "doi:\n",
      "11\n",
      "xxxxx\n",
      "Ari Dwijayanti\n",
      "12\n",
      "xxxxx\n",
      "1CNRS@CREATE, 1 Create Way, #8-01 Create Tower, Singapore 138602\n",
      "13\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "14\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "15\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "16\n",
      "xxxxx\n",
      "Marko Storch\n",
      "17\n",
      "xxxxx\n",
      "2London Biofoundry, Translation & Innovation Hub, White City Campus, Imperial College London, London W12 0BZ, UK\n",
      "18\n",
      "xxxxx\n",
      "3Imperial College Centre for Synthetic Biology, Imperial College London, London SW7 2AZ, UK\n",
      "19\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "20\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "21\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "22\n",
      "xxxxx\n",
      "Guy-Bart Stan\n",
      "23\n",
      "xxxxx\n",
      "3Imperial College Centre for Synthetic Biology, Imperial College London, London SW7 2AZ, UK\n",
      "24\n",
      "xxxxx\n",
      "4Department of Bioengineering, Bessemer Building, South Kensington Campus, Imperial College London, London SW7 2AZ, UK\n",
      "25\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "26\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "27\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "28\n",
      "xxxxx\n",
      "For correspondence:\n",
      "29\n",
      "xxxxx\n",
      "g.baldwin@imperial.ac.uk\n",
      "30\n",
      "xxxxx\n",
      "g.stan@imperial.ac.uk\n",
      "31\n",
      "xxxxx\n",
      "Geoff S. Baldwin\n",
      "32\n",
      "xxxxx\n",
      "3Imperial College Centre for Synthetic Biology, Imperial College London, London SW7 2AZ, UK\n",
      "33\n",
      "xxxxx\n",
      "5Department of Life Sciences, Sir Alexander Fleming Building, South Kensington Campus, Imperial College London, London SW7 2AZ, UK\n",
      "34\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "35\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "36\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "37\n",
      "xxxxx\n",
      "For correspondence:\n",
      "38\n",
      "xxxxx\n",
      "g.baldwin@imperial.ac.uk\n",
      "39\n",
      "xxxxx\n",
      "g.stan@imperial.ac.uk\n",
      "40\n",
      "xxxxx\n",
      "Abstract\n",
      "41\n",
      "xxxxx\n",
      "Full Text\n",
      "42\n",
      "xxxxx\n",
      "Info/History\n",
      "43\n",
      "xxxxx\n",
      "Metrics\n",
      "44\n",
      "xxxxx\n",
      "Supplementary material\n",
      "45\n",
      "xxxxx\n",
      "Preview PDF\n",
      "46\n",
      "xxxxx\n",
      "ABSTRACT\n",
      "47\n",
      "xxxxx\n",
      "The rational design and realisation of simple-to-use genetic control elements that are modular, orthogonal and robust is essential to the construction of predictable and reliable biological systems of increasing complexity.\n",
      "48\n",
      "xxxxx\n",
      "m\n",
      "49\n",
      "xxxxx\n",
      "odular\n",
      "50\n",
      "xxxxx\n",
      "A\n",
      "51\n",
      "xxxxx\n",
      "rtificial\n",
      "52\n",
      "xxxxx\n",
      "R\n",
      "53\n",
      "xxxxx\n",
      "NA\n",
      "54\n",
      "xxxxx\n",
      "i\n",
      "55\n",
      "xxxxx\n",
      "nterference (mARi), a rational, modular and extensible design framework that enables robust, portable and multiplexed post-transcriptional regulation of gene expression in\n",
      "56\n",
      "xxxxx\n",
      "Escherichia coli\n",
      "57\n",
      "xxxxx\n",
      ".\n",
      "58\n",
      "xxxxx\n",
      "E. coli\n",
      "59\n",
      "xxxxx\n",
      ", while a range of cellular contexts also demonstrated it to be independent of growth-phase and strain type.\n",
      "60\n",
      "xxxxx\n",
      "INTRODUCTION\n",
      "61\n",
      "xxxxx\n",
      "Synthetic biology aims to make the engineering of biological systems more predictable, efficient and reliable (\n",
      "62\n",
      "xxxxx\n",
      "1\n",
      "63\n",
      "xxxxx\n",
      "-\n",
      "64\n",
      "xxxxx\n",
      "3\n",
      "65\n",
      "xxxxx\n",
      ").\n",
      "66\n",
      "xxxxx\n",
      "4\n",
      "67\n",
      "xxxxx\n",
      ",\n",
      "68\n",
      "xxxxx\n",
      "5\n",
      "69\n",
      "xxxxx\n",
      ").\n",
      "70\n",
      "xxxxx\n",
      "6\n",
      "71\n",
      "xxxxx\n",
      "-\n",
      "72\n",
      "xxxxx\n",
      "10\n",
      "73\n",
      "xxxxx\n",
      ").\n",
      "74\n",
      "xxxxx\n",
      "11\n",
      "75\n",
      "xxxxx\n",
      ",\n",
      "76\n",
      "xxxxx\n",
      "12\n",
      "77\n",
      "xxxxx\n",
      "), typically leading to down-regulation of target gene expression (\n",
      "78\n",
      "xxxxx\n",
      "13\n",
      "79\n",
      "xxxxx\n",
      "-\n",
      "80\n",
      "xxxxx\n",
      "16\n",
      "81\n",
      "xxxxx\n",
      ").\n",
      "82\n",
      "xxxxx\n",
      "Based on the arrangement of sRNAs and their targets, sRNAs can be grouped into\n",
      "83\n",
      "xxxxx\n",
      "cis\n",
      "84\n",
      "xxxxx\n",
      "and\n",
      "85\n",
      "xxxxx\n",
      "trans\n",
      "86\n",
      "xxxxx\n",
      "-encoded sRNAs.\n",
      "87\n",
      "xxxxx\n",
      "cis\n",
      "88\n",
      "xxxxx\n",
      "-encoded sRNAs are transcribed from the same loci as their target mRNA, but as a reverse complement, thus making a complementary base-pairing to the mRNA they regulate;\n",
      "89\n",
      "xxxxx\n",
      "trans\n",
      "90\n",
      "xxxxx\n",
      "Characterization of cell-to-cell variation in nuclear transport rates and identification of its sources | bioRxiv\n",
      "0\n",
      "xxxxx\n",
      "Skip to main content\n",
      "1\n",
      "xxxxx\n",
      "Home\n",
      "2\n",
      "xxxxx\n",
      "About\n",
      "3\n",
      "xxxxx\n",
      "Submit\n",
      "4\n",
      "xxxxx\n",
      "ALERTS / RSS\n",
      "5\n",
      "xxxxx\n",
      "Search for this keyword\n",
      "6\n",
      "xxxxx\n",
      "Advanced Search\n",
      "7\n",
      "xxxxx\n",
      "New Results\n",
      "8\n",
      "xxxxx\n",
      "Characterization of cell-to-cell variation in nuclear transport rates and identification of its sources\n",
      "9\n",
      "xxxxx\n",
      "View ORCID Profile\n",
      "10\n",
      "xxxxx\n",
      "Lucía Durrieu, Alan Bush, Alicia Grande, Rikard Johansson, David Janzén, Andrea Katz,\n",
      "11\n",
      "xxxxx\n",
      "View ORCID Profile\n",
      "12\n",
      "xxxxx\n",
      "Gunnar Cedersund,\n",
      "13\n",
      "xxxxx\n",
      "View ORCID Profile\n",
      "14\n",
      "xxxxx\n",
      "Alejandro Colman-Lerner\n",
      "15\n",
      "xxxxx\n",
      "doi:\n",
      "16\n",
      "xxxxx\n",
      "Lucía Durrieu\n",
      "17\n",
      "xxxxx\n",
      "1Department of Physiology, Molecular and Cellular Biology, School of Exact and Natural Sciences, University of Buenos Aires (UBA), C1428EGA Buenos Aires, Argentina\n",
      "18\n",
      "xxxxx\n",
      "2Institute of Physiology, Molecular Biology and Neurosciences, National Council of Scientific and Technical Research (IFIBYNE-UBA-CONICET), C1428EGA Buenos Aires, Argentina\n",
      "19\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "20\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "21\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "22\n",
      "xxxxx\n",
      "ORCID record for Lucía Durrieu\n",
      "23\n",
      "xxxxx\n",
      "Alan Bush\n",
      "24\n",
      "xxxxx\n",
      "1Department of Physiology, Molecular and Cellular Biology, School of Exact and Natural Sciences, University of Buenos Aires (UBA), C1428EGA Buenos Aires, Argentina\n",
      "25\n",
      "xxxxx\n",
      "2Institute of Physiology, Molecular Biology and Neurosciences, National Council of Scientific and Technical Research (IFIBYNE-UBA-CONICET), C1428EGA Buenos Aires, Argentina\n",
      "26\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "27\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "28\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "29\n",
      "xxxxx\n",
      "Alicia Grande\n",
      "30\n",
      "xxxxx\n",
      "1Department of Physiology, Molecular and Cellular Biology, School of Exact and Natural Sciences, University of Buenos Aires (UBA), C1428EGA Buenos Aires, Argentina\n",
      "31\n",
      "xxxxx\n",
      "2Institute of Physiology, Molecular Biology and Neurosciences, National Council of Scientific and Technical Research (IFIBYNE-UBA-CONICET), C1428EGA Buenos Aires, Argentina\n",
      "32\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "33\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "34\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "35\n",
      "xxxxx\n",
      "Rikard Johansson\n",
      "36\n",
      "xxxxx\n",
      "4Department of Biomedical Engineering, Linköping University, Linköping, Sweden\n",
      "37\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "38\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "39\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "40\n",
      "xxxxx\n",
      "David Janzén\n",
      "41\n",
      "xxxxx\n",
      "4Department of Biomedical Engineering, Linköping University, Linköping, Sweden\n",
      "42\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "43\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "44\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "45\n",
      "xxxxx\n",
      "Andrea Katz\n",
      "46\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "47\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "48\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "49\n",
      "xxxxx\n",
      "Gunnar Cedersund\n",
      "50\n",
      "xxxxx\n",
      "4Department of Biomedical Engineering, Linköping University, Linköping, Sweden\n",
      "51\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "52\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "53\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "54\n",
      "xxxxx\n",
      "ORCID record for Gunnar Cedersund\n",
      "55\n",
      "xxxxx\n",
      "Alejandro Colman-Lerner\n",
      "56\n",
      "xxxxx\n",
      "1Department of Physiology, Molecular and Cellular Biology, School of Exact and Natural Sciences, University of Buenos Aires (UBA), C1428EGA Buenos Aires, Argentina\n",
      "57\n",
      "xxxxx\n",
      "2Institute of Physiology, Molecular Biology and Neurosciences, National Council of Scientific and Technical Research (IFIBYNE-UBA-CONICET), C1428EGA Buenos Aires, Argentina\n",
      "58\n",
      "xxxxx\n",
      "Find this author on Google Scholar\n",
      "59\n",
      "xxxxx\n",
      "Find this author on PubMed\n",
      "60\n",
      "xxxxx\n",
      "Search for this author on this site\n",
      "61\n",
      "xxxxx\n",
      "ORCID record for Alejandro Colman-Lerner\n",
      "62\n",
      "xxxxx\n",
      "For correspondence:\n",
      "63\n",
      "xxxxx\n",
      "colman-lerner@fbmc.fcen.uba.ar\n",
      "64\n",
      "xxxxx\n",
      "Abstract\n",
      "65\n",
      "xxxxx\n",
      "Full Text\n",
      "66\n",
      "xxxxx\n",
      "Info/History\n",
      "67\n",
      "xxxxx\n",
      "Metrics\n",
      "68\n",
      "xxxxx\n",
      "Supplementary material\n",
      "69\n",
      "xxxxx\n",
      "Preview PDF\n",
      "70\n",
      "xxxxx\n",
      "Abstract\n",
      "71\n",
      "xxxxx\n",
      "Nuclear transport is an essential part of eukaryotic cell function.\n",
      "72\n",
      "xxxxx\n",
      "Introduction\n",
      "73\n",
      "xxxxx\n",
      "Isogenic cells, grown under identical conditions, display heterogeneity in their molecular contents.\n",
      "74\n",
      "xxxxx\n",
      "Elowitz et al.,\n",
      "75\n",
      "xxxxx\n",
      ";\n",
      "76\n",
      "xxxxx\n",
      "Raser and O’Shea, 2004\n",
      "77\n",
      "xxxxx\n",
      ";\n",
      "78\n",
      "xxxxx\n",
      "Rosenfeld et al.,\n",
      "79\n",
      "xxxxx\n",
      "), but also from fluctuations in other processes, such as the mitotic segregation of materials (\n",
      "80\n",
      "xxxxx\n",
      "Huh and Paulsson, 2011\n",
      "81\n",
      "xxxxx\n",
      ").\n",
      "82\n",
      "xxxxx\n",
      "Blake et al.,\n",
      "83\n",
      "xxxxx\n",
      ";\n",
      "84\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "85\n",
      "xxxxx\n",
      ";\n",
      "86\n",
      "xxxxx\n",
      "Gándara et al.,\n",
      "87\n",
      "xxxxx\n",
      ";\n",
      "88\n",
      "xxxxx\n",
      "Mitchell et al.,\n",
      "89\n",
      "xxxxx\n",
      ";\n",
      "90\n",
      "xxxxx\n",
      "Raser and O’Shea, 2004\n",
      "91\n",
      "xxxxx\n",
      ";\n",
      "92\n",
      "xxxxx\n",
      "Roux et al.,\n",
      "93\n",
      "xxxxx\n",
      ";\n",
      "94\n",
      "xxxxx\n",
      "Spencer et al.,\n",
      "95\n",
      "xxxxx\n",
      ";\n",
      "96\n",
      "xxxxx\n",
      "St-Pierre and Endy, 2008\n",
      "97\n",
      "xxxxx\n",
      ").\n",
      "98\n",
      "xxxxx\n",
      "Kaufmann et al.,\n",
      "99\n",
      "xxxxx\n",
      ";\n",
      "100\n",
      "xxxxx\n",
      "Li et al.,\n",
      "101\n",
      "xxxxx\n",
      "-encoded sRNAs are located in a distal site.\n",
      "91\n",
      "xxxxx\n",
      "trans\n",
      "92\n",
      "xxxxx\n",
      "-encoded sRNAs can have efficient regulation of their mRNA targets (\n",
      "93\n",
      "xxxxx\n",
      "14\n",
      "94\n",
      "xxxxx\n",
      ",\n",
      "95\n",
      "xxxxx\n",
      "17\n",
      "96\n",
      "xxxxx\n",
      "-\n",
      "97\n",
      "xxxxx\n",
      "19\n",
      "98\n",
      "xxxxx\n",
      ").\n",
      "99\n",
      "xxxxx\n",
      "trans\n",
      "100\n",
      "xxxxx\n",
      "-encoded sRNAs rather than\n",
      "101\n",
      "xxxxx\n",
      "cis\n",
      "102\n",
      "xxxxx\n",
      "-encoded sRNAs have been reported to be especially effective in silencing gene expression due to their longer half-life in the cytoplasm, thought to be due to association with the Hfq chaperone protein (\n",
      "103\n",
      "xxxxx\n",
      "20\n",
      "104\n",
      "xxxxx\n",
      "-\n",
      "105\n",
      "xxxxx\n",
      "23\n",
      "106\n",
      "xxxxx\n",
      ").\n",
      "107\n",
      "xxxxx\n",
      "trans\n",
      "108\n",
      "xxxxx\n",
      "-encoded sRNAs, therefore, have been widely applied as versatile genetically encoded controllers for the design of synthetic biological circuits for metabolic engineering and synthetic biology purposes (\n",
      "109\n",
      "xxxxx\n",
      "7\n",
      "110\n",
      "xxxxx\n",
      ",\n",
      "111\n",
      "xxxxx\n",
      "24\n",
      "112\n",
      "xxxxx\n",
      "-\n",
      "113\n",
      "xxxxx\n",
      "27\n",
      "114\n",
      "xxxxx\n",
      ").\n",
      "115\n",
      "xxxxx\n",
      "Previous approaches to implement\n",
      "116\n",
      "xxxxx\n",
      "trans\n",
      "117\n",
      "xxxxx\n",
      "-encoded sRNA were exemplified by the use of native sRNAs and their cognate targets i.e.\n",
      "118\n",
      "xxxxx\n",
      "rpoS, hns, sodB\n",
      "119\n",
      "xxxxx\n",
      ", and\n",
      "120\n",
      "xxxxx\n",
      "ompC\n",
      "121\n",
      "xxxxx\n",
      "(\n",
      "122\n",
      "xxxxx\n",
      "28\n",
      "123\n",
      "xxxxx\n",
      ",\n",
      "124\n",
      "xxxxx\n",
      "29\n",
      "125\n",
      "xxxxx\n",
      ").\n",
      "126\n",
      "xxxxx\n",
      "17\n",
      "127\n",
      "xxxxx\n",
      ",\n",
      "128\n",
      "xxxxx\n",
      "18\n",
      "129\n",
      "xxxxx\n",
      ",\n",
      "130\n",
      "xxxxx\n",
      "28\n",
      "131\n",
      "xxxxx\n",
      ").\n",
      "132\n",
      "xxxxx\n",
      "17\n",
      "133\n",
      "xxxxx\n",
      ",\n",
      "134\n",
      "xxxxx\n",
      "27\n",
      "135\n",
      "xxxxx\n",
      ",\n",
      "136\n",
      "xxxxx\n",
      "30\n",
      "137\n",
      "xxxxx\n",
      ",\n",
      "138\n",
      "xxxxx\n",
      "31\n",
      "139\n",
      "xxxxx\n",
      ").\n",
      "140\n",
      "xxxxx\n",
      "25\n",
      "141\n",
      "xxxxx\n",
      ",\n",
      "142\n",
      "xxxxx\n",
      "32\n",
      "143\n",
      "xxxxx\n",
      ").\n",
      "144\n",
      "xxxxx\n",
      "24\n",
      "145\n",
      "xxxxx\n",
      ").\n",
      "146\n",
      "xxxxx\n",
      "To leverage the use of sRNAs in genetic circuits constructed via a modular DNA assembly method, we set out to design a universal, modular framework for the facile implementation of\n",
      "147\n",
      "xxxxx\n",
      "trans\n",
      "148\n",
      "xxxxx\n",
      "-acting sRNA-based control using verified, re-usable target sequences, namely\n",
      "149\n",
      "xxxxx\n",
      "m\n",
      "150\n",
      "xxxxx\n",
      "odular\n",
      "151\n",
      "xxxxx\n",
      "A\n",
      "152\n",
      "xxxxx\n",
      "rtificial\n",
      "153\n",
      "xxxxx\n",
      "R\n",
      "154\n",
      "xxxxx\n",
      "NA\n",
      "155\n",
      "xxxxx\n",
      "i\n",
      "156\n",
      "xxxxx\n",
      "nterference (mARi).\n",
      "157\n",
      "xxxxx\n",
      "33\n",
      "158\n",
      "xxxxx\n",
      ") framework for genetic structure based on orthogonal, computationally designed linkers (\n",
      "159\n",
      "xxxxx\n",
      "34\n",
      "160\n",
      "xxxxx\n",
      ") (\n",
      "161\n",
      "xxxxx\n",
      "Figure 1\n",
      "162\n",
      "xxxxx\n",
      ").\n",
      "163\n",
      "xxxxx\n",
      "17\n",
      "164\n",
      "xxxxx\n",
      ",\n",
      "165\n",
      "xxxxx\n",
      "27\n",
      "166\n",
      "xxxxx\n",
      ") (\n",
      "167\n",
      "xxxxx\n",
      "Fig.\n",
      "168\n",
      "xxxxx\n",
      ",\n",
      "169\n",
      "xxxxx\n",
      "2a\n",
      "170\n",
      "xxxxx\n",
      ").\n",
      "171\n",
      "xxxxx\n",
      "Fig.\n",
      "172\n",
      "xxxxx\n",
      ").\n",
      "173\n",
      "xxxxx\n",
      "Download figure\n",
      "174\n",
      "xxxxx\n",
      "Open in new tab\n",
      "175\n",
      "xxxxx\n",
      "Figure 1.\n",
      "176\n",
      "xxxxx\n",
      "(\n",
      "177\n",
      "xxxxx\n",
      "a\n",
      "178\n",
      "xxxxx\n",
      ") mARi-based regulators composed of a promoter, seed sequence (grey), sRNA scaffold (purple) and terminator.\n",
      "179\n",
      "xxxxx\n",
      "34\n",
      "180\n",
      "xxxxx\n",
      "). (\n",
      "181\n",
      "xxxxx\n",
      "b\n",
      "182\n",
      "xxxxx\n",
      ") Plasmids were constructed from the interchangeable DNA parts using BASIC DNA assembly:\n",
      "183\n",
      "xxxxx\n",
      "c\n",
      "184\n",
      "xxxxx\n",
      ") Expression of mARi leads to repression of a target gene with a cognate UTR by base-pairing between the 5’untranslated region and the mARi seed sequence.\n",
      "185\n",
      "xxxxx\n",
      "MATERIALS AND METHODS\n",
      "186\n",
      "xxxxx\n",
      ";\n",
      "102\n",
      "xxxxx\n",
      "Mitchell, 2021\n",
      "103\n",
      "xxxxx\n",
      ";\n",
      "104\n",
      "xxxxx\n",
      "Mura et al.,\n",
      "105\n",
      "xxxxx\n",
      ";\n",
      "106\n",
      "xxxxx\n",
      "Spencer et al.,\n",
      "107\n",
      "xxxxx\n",
      ";\n",
      "108\n",
      "xxxxx\n",
      "Vashistha et al.,\n",
      "109\n",
      "xxxxx\n",
      ").\n",
      "110\n",
      "xxxxx\n",
      "Regulation of nuclear transport influences important cellular processes, such as gene expression regulation and signal transduction.\n",
      "111\n",
      "xxxxx\n",
      "Saccharomyces cerevisiae\n",
      "112\n",
      "xxxxx\n",
      "consist of 30 different nucleoporins (\n",
      "113\n",
      "xxxxx\n",
      "Rout et al.,\n",
      "114\n",
      "xxxxx\n",
      ") (human NPC are 110 MDa, containing 34 different nucleoporins).\n",
      "115\n",
      "xxxxx\n",
      "Paine and Scherr, 1975\n",
      "116\n",
      "xxxxx\n",
      ") or ∼ 5 nm diameter (\n",
      "117\n",
      "xxxxx\n",
      "Mohr et al.,\n",
      "118\n",
      "xxxxx\n",
      ";\n",
      "119\n",
      "xxxxx\n",
      "Yang and Musser, 2006\n",
      "120\n",
      "xxxxx\n",
      ")), or facilitated translocation for small and big species (\n",
      "121\n",
      "xxxxx\n",
      "Ribbeck and Görlich, 2002\n",
      "122\n",
      "xxxxx\n",
      ").\n",
      "123\n",
      "xxxxx\n",
      "S. cerevisiae\n",
      "124\n",
      "xxxxx\n",
      "cells divide by budding, a polarized process that produces a new cell, the daughter.\n",
      "125\n",
      "xxxxx\n",
      "Bobola et al.,\n",
      "126\n",
      "xxxxx\n",
      ";\n",
      "127\n",
      "xxxxx\n",
      "Nasmyth, 1983\n",
      "128\n",
      "xxxxx\n",
      ";\n",
      "129\n",
      "xxxxx\n",
      "Sil and Herskowitz, 1996\n",
      "130\n",
      "xxxxx\n",
      ") while in the other program the transcription factor Ace2 induces in the early G1 of daughter cells the expression of a group of genes, including chitinase, involved in cell separation (\n",
      "131\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "132\n",
      "xxxxx\n",
      ").\n",
      "133\n",
      "xxxxx\n",
      "Mazanka et al.,\n",
      "134\n",
      "xxxxx\n",
      ";\n",
      "135\n",
      "xxxxx\n",
      "Weiss et al.,\n",
      "136\n",
      "xxxxx\n",
      ").\n",
      "137\n",
      "xxxxx\n",
      "Boettcher et al.,\n",
      "138\n",
      "xxxxx\n",
      ";\n",
      "139\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "140\n",
      "xxxxx\n",
      ";\n",
      "141\n",
      "xxxxx\n",
      "Mazanka et al.,\n",
      "142\n",
      "xxxxx\n",
      ") (note that, as in all fungi, mitosis in yeast does not involve nuclear envelope breakdown (\n",
      "143\n",
      "xxxxx\n",
      "Cavalier-Smith, 2010\n",
      "144\n",
      "xxxxx\n",
      ")), is triggered when the phosphatase Cdc14, released from the nucleolus by the mitotic exit network (\n",
      "145\n",
      "xxxxx\n",
      "O’Conallain et al.,\n",
      "146\n",
      "xxxxx\n",
      "), removes the Cdc28-dependent phosphorylations on Ace2.\n",
      "147\n",
      "xxxxx\n",
      "Brace et al.,\n",
      "148\n",
      "xxxxx\n",
      ").\n",
      "149\n",
      "xxxxx\n",
      "Bourens et al.,\n",
      "150\n",
      "xxxxx\n",
      ";\n",
      "151\n",
      "xxxxx\n",
      "Brace et al.,\n",
      "152\n",
      "xxxxx\n",
      ";\n",
      "153\n",
      "xxxxx\n",
      "Mazanka et al.,\n",
      "154\n",
      "xxxxx\n",
      ";\n",
      "155\n",
      "xxxxx\n",
      "Mazanka and Weiss, 2010\n",
      "156\n",
      "xxxxx\n",
      ";\n",
      "157\n",
      "xxxxx\n",
      "Montpellier and Bourens M Bécam Am, 2008\n",
      "158\n",
      "xxxxx\n",
      ";\n",
      "159\n",
      "xxxxx\n",
      "Sbia et al.,\n",
      "160\n",
      "xxxxx\n",
      ").\n",
      "161\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "162\n",
      "xxxxx\n",
      ").\n",
      "163\n",
      "xxxxx\n",
      "Boettcher et al.,\n",
      "164\n",
      "xxxxx\n",
      ").\n",
      "165\n",
      "xxxxx\n",
      "2012\n",
      "166\n",
      "xxxxx\n",
      ";\n",
      "167\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "168\n",
      "xxxxx\n",
      ";\n",
      "169\n",
      "xxxxx\n",
      "Dohrmann et al.,\n",
      "170\n",
      "xxxxx\n",
      ";\n",
      "171\n",
      "xxxxx\n",
      "Herrero et al.,\n",
      "172\n",
      "xxxxx\n",
      ";\n",
      "173\n",
      "xxxxx\n",
      "Mazanka et al.,\n",
      "174\n",
      "xxxxx\n",
      ";\n",
      "175\n",
      "xxxxx\n",
      "1999\n",
      "176\n",
      "xxxxx\n",
      ";\n",
      "177\n",
      "xxxxx\n",
      "Sbia et al.,\n",
      "178\n",
      "xxxxx\n",
      ";\n",
      "179\n",
      "xxxxx\n",
      "Weiss et al.,\n",
      "180\n",
      "xxxxx\n",
      ").\n",
      "181\n",
      "xxxxx\n",
      "Boettcher and Barral, 2014\n",
      "182\n",
      "xxxxx\n",
      ").\n",
      "183\n",
      "xxxxx\n",
      "Herein, we developed a method to monitor nuclear transport rates in single cells using the optical technique fluorescence recovery after photobleaching (FRAP) (\n",
      "184\n",
      "xxxxx\n",
      "Axelrod et al.,\n",
      "185\n",
      "xxxxx\n",
      ";\n",
      "186\n",
      "xxxxx\n",
      "Cole et al.,\n",
      "187\n",
      "xxxxx\n",
      ").\n",
      "188\n",
      "xxxxx\n",
      "Results\n",
      "189\n",
      "xxxxx\n",
      "A train of FRAPs enables robust estimation of nuclear transport rates in single cells\n",
      "190\n",
      "xxxxx\n",
      "Transport between cellular compartments\n",
      "191\n",
      "xxxxx\n",
      "in vivo\n",
      "192\n",
      "xxxxx\n",
      "is frequently studied using FRAP (\n",
      "193\n",
      "xxxxx\n",
      "Cole et al.,\n",
      "194\n",
      "xxxxx\n",
      ";\n",
      "195\n",
      "xxxxx\n",
      "Goodwin and Kenworthy, 2005\n",
      "196\n",
      "xxxxx\n",
      ";\n",
      "197\n",
      "xxxxx\n",
      "Wachsmuth et al.,\n",
      "198\n",
      "xxxxx\n",
      ").\n",
      "199\n",
      "xxxxx\n",
      "recovery\n",
      "200\n",
      "xxxxx\n",
      ") of the fluorescence signal.\n",
      "201\n",
      "xxxxx\n",
      "Figure 1 A\n",
      "202\n",
      "xxxxx\n",
      ") should help improve the single-cell information (without excessive exposure or intensities), potentially avoiding the need of averaging over many cells.\n",
      "203\n",
      "xxxxx\n",
      "Figure 1 B\n",
      "204\n",
      "xxxxx\n",
      "We considered the photobleaching reaction to be irreversible.\n",
      "205\n",
      "xxxxx\n",
      "Gordon et al.,\n",
      "206\n",
      "xxxxx\n",
      ").\n",
      "207\n",
      "xxxxx\n",
      "-1\n",
      "208\n",
      "xxxxx\n",
      "Bacterial strains and outgrowth conditions\n",
      "187\n",
      "xxxxx\n",
      "The following\n",
      "188\n",
      "xxxxx\n",
      "E. coli\n",
      "189\n",
      "xxxxx\n",
      "DH5α (F\n",
      "190\n",
      "xxxxx\n",
      "-\n",
      "191\n",
      "xxxxx\n",
      "endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG purB20\n",
      "192\n",
      "xxxxx\n",
      "ϕ80d\n",
      "193\n",
      "xxxxx\n",
      "lacZ\n",
      "194\n",
      "xxxxx\n",
      "ΔM15 Δ (\n",
      "195\n",
      "xxxxx\n",
      "lacZYA-argF\n",
      "196\n",
      "xxxxx\n",
      ")U169, hsdR17 (\n",
      "197\n",
      "xxxxx\n",
      "r\n",
      "198\n",
      "xxxxx\n",
      "K\n",
      "199\n",
      "xxxxx\n",
      "-\n",
      "200\n",
      "xxxxx\n",
      "m\n",
      "201\n",
      "xxxxx\n",
      "K\n",
      "202\n",
      "xxxxx\n",
      "+\n",
      "203\n",
      "xxxxx\n",
      "), λ\n",
      "204\n",
      "xxxxx\n",
      "-\n",
      "205\n",
      "xxxxx\n",
      "), DH10b [F-\n",
      "206\n",
      "xxxxx\n",
      "mcr\n",
      "207\n",
      "xxxxx\n",
      "A Δ (\n",
      "208\n",
      "xxxxx\n",
      "mrr\n",
      "209\n",
      "xxxxx\n",
      "-\n",
      "210\n",
      "xxxxx\n",
      "hsd\n",
      "211\n",
      "xxxxx\n",
      "RMS-\n",
      "212\n",
      "xxxxx\n",
      "mcr\n",
      "213\n",
      "xxxxx\n",
      "BC) ϕ80\n",
      "214\n",
      "xxxxx\n",
      "lac\n",
      "215\n",
      "xxxxx\n",
      "ZΔM15 Δ\n",
      "216\n",
      "xxxxx\n",
      "lac\n",
      "217\n",
      "xxxxx\n",
      "X74\n",
      "218\n",
      "xxxxx\n",
      "rec\n",
      "219\n",
      "xxxxx\n",
      "A1\n",
      "220\n",
      "xxxxx\n",
      "end\n",
      "221\n",
      "xxxxx\n",
      "A1\n",
      "222\n",
      "xxxxx\n",
      "ara\n",
      "223\n",
      "xxxxx\n",
      "D139 Δ (\n",
      "224\n",
      "xxxxx\n",
      "ara leu\n",
      "225\n",
      "xxxxx\n",
      ") 7697\n",
      "226\n",
      "xxxxx\n",
      "gal\n",
      "227\n",
      "xxxxx\n",
      "U\n",
      "228\n",
      "xxxxx\n",
      "gal\n",
      "229\n",
      "xxxxx\n",
      "K\n",
      "230\n",
      "xxxxx\n",
      "rps\n",
      "231\n",
      "xxxxx\n",
      "L\n",
      "232\n",
      "xxxxx\n",
      "nup\n",
      "233\n",
      "xxxxx\n",
      "G λ-], BL21 (DE3) (F\n",
      "234\n",
      "xxxxx\n",
      "-\n",
      "235\n",
      "xxxxx\n",
      "omp\n",
      "236\n",
      "xxxxx\n",
      "T\n",
      "237\n",
      "xxxxx\n",
      "hsd\n",
      "238\n",
      "xxxxx\n",
      "S\n",
      "239\n",
      "xxxxx\n",
      "B\n",
      "240\n",
      "xxxxx\n",
      "(r\n",
      "241\n",
      "xxxxx\n",
      "B\n",
      "242\n",
      "xxxxx\n",
      "-\n",
      "243\n",
      "xxxxx\n",
      ", m\n",
      "244\n",
      "xxxxx\n",
      "B\n",
      "245\n",
      "xxxxx\n",
      "-\n",
      "246\n",
      "xxxxx\n",
      ")\n",
      "247\n",
      "xxxxx\n",
      "gal dcm\n",
      "248\n",
      "xxxxx\n",
      "(DE3)), and BL21star (DE3) (F\n",
      "249\n",
      "xxxxx\n",
      "-\n",
      "250\n",
      "xxxxx\n",
      "omp\n",
      "251\n",
      "xxxxx\n",
      "T\n",
      "252\n",
      "xxxxx\n",
      "hsd\n",
      "253\n",
      "xxxxx\n",
      "S\n",
      "254\n",
      "xxxxx\n",
      "B\n",
      "255\n",
      "xxxxx\n",
      "(r\n",
      "256\n",
      "xxxxx\n",
      "B\n",
      "257\n",
      "xxxxx\n",
      "-\n",
      "258\n",
      "xxxxx\n",
      ", mB\n",
      "259\n",
      "xxxxx\n",
      "-\n",
      "260\n",
      "xxxxx\n",
      ")\n",
      "261\n",
      "xxxxx\n",
      "gal dcm rne\n",
      "262\n",
      "xxxxx\n",
      "131 (DE3)).\n",
      "263\n",
      "xxxxx\n",
      "E. coli\n",
      "264\n",
      "xxxxx\n",
      "colonies were grown from single-colony isolates in LB (Luria Bertani) medium supplemented with the appropriate antibiotics, shaken at 220 rpm and grown at 37°C (unless specified otherwise).\n",
      "265\n",
      "xxxxx\n",
      "Design and analysis of mARi sequences\n",
      "266\n",
      "xxxxx\n",
      "The sequences of UTR-RBS linkers used in BASIC DNA assembly (\n",
      "267\n",
      "xxxxx\n",
      "34\n",
      "268\n",
      "xxxxx\n",
      ") were designed and validated using R2O DNA designer (\n",
      "269\n",
      "xxxxx\n",
      "35\n",
      "270\n",
      "xxxxx\n",
      ").\n",
      "271\n",
      "xxxxx\n",
      "http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi\n",
      "272\n",
      "xxxxx\n",
      ") and visualised using VARNA GUI (\n",
      "273\n",
      "xxxxx\n",
      "36\n",
      "274\n",
      "xxxxx\n",
      ") (\n",
      "275\n",
      "xxxxx\n",
      "http://varna.lri.fr/index.php?lang=en&page=tutorial&css=varna\n",
      "276\n",
      "xxxxx\n",
      ").\n",
      "277\n",
      "xxxxx\n",
      "37\n",
      "278\n",
      "xxxxx\n",
      ",\n",
      "279\n",
      "xxxxx\n",
      "38\n",
      "280\n",
      "xxxxx\n",
      ") (\n",
      "281\n",
      "xxxxx\n",
      "https://bio.tools/intarna\n",
      "282\n",
      "xxxxx\n",
      "SD=0.05, true export rate=0.90 sec\n",
      "209\n",
      "xxxxx\n",
      "-1\n",
      "210\n",
      "xxxxx\n",
      ") (\n",
      "211\n",
      "xxxxx\n",
      "Figure 1 C\n",
      "212\n",
      "xxxxx\n",
      ").\n",
      "213\n",
      "xxxxx\n",
      "-1\n",
      "214\n",
      "xxxxx\n",
      "SD=2.37, true export rate=0.90 sec\n",
      "215\n",
      "xxxxx\n",
      "-1\n",
      "216\n",
      "xxxxx\n",
      ") (\n",
      "217\n",
      "xxxxx\n",
      "Figure 1 C\n",
      "218\n",
      "xxxxx\n",
      ").\n",
      "219\n",
      "xxxxx\n",
      "-1\n",
      "220\n",
      "xxxxx\n",
      "SD=0.30, true export rate=0.90 sec\n",
      "221\n",
      "xxxxx\n",
      "-1\n",
      "222\n",
      "xxxxx\n",
      ") (\n",
      "223\n",
      "xxxxx\n",
      "Fig 1C\n",
      "224\n",
      "xxxxx\n",
      ").\n",
      "225\n",
      "xxxxx\n",
      "Fig.\n",
      "226\n",
      "xxxxx\n",
      ").\n",
      "227\n",
      "xxxxx\n",
      "Download figure\n",
      "228\n",
      "xxxxx\n",
      "Open in new tab\n",
      "229\n",
      "xxxxx\n",
      "Figure 1 Trains of FRAPs allows the estimation of nuclear import and export rates in single cells even under low signal-to-noise conditions.\n",
      "230\n",
      "xxxxx\n",
      "(A)\n",
      "231\n",
      "xxxxx\n",
      "Schema of an essay with a train of 2 FRAPs.\n",
      "232\n",
      "xxxxx\n",
      "(B)\n",
      "233\n",
      "xxxxx\n",
      "Reactions schematic in the model used for data simulation and fitting.\n",
      "234\n",
      "xxxxx\n",
      "(C)\n",
      "235\n",
      "xxxxx\n",
      "Box plots show import (left) and export (right) obtained by fitting simulated data (K\n",
      "236\n",
      "xxxxx\n",
      "I\n",
      "237\n",
      "xxxxx\n",
      "= 0.4 sec\n",
      "238\n",
      "xxxxx\n",
      "-1\n",
      "239\n",
      "xxxxx\n",
      ", K\n",
      "240\n",
      "xxxxx\n",
      "EV\n",
      "241\n",
      "xxxxx\n",
      "= 0.9 sec\n",
      "242\n",
      "xxxxx\n",
      "-1\n",
      "243\n",
      "xxxxx\n",
      ") of trains of different number of FRAPs to which noise was added to achieve the signal to noise indicated.\n",
      "244\n",
      "xxxxx\n",
      "I\n",
      "245\n",
      "xxxxx\n",
      "= 0.4 sec\n",
      "246\n",
      "xxxxx\n",
      "-1\n",
      "247\n",
      "xxxxx\n",
      ", K\n",
      "248\n",
      "xxxxx\n",
      "EV\n",
      "249\n",
      "xxxxx\n",
      "= 0.9 sec\n",
      "250\n",
      "xxxxx\n",
      "-1\n",
      "251\n",
      "xxxxx\n",
      ", gray dotted line (same qualitative results were observed with other parameters).\n",
      "252\n",
      "xxxxx\n",
      "Determination of nuclear transport rates in single cells\n",
      "253\n",
      "xxxxx\n",
      "Based on the above analysis, we developed an experimental protocol and analysis pipeline, which we termed scFRAP (single cell FRAP), to estimate nuclear export and import rates independently for single cells (see the Appendix) (\n",
      "254\n",
      "xxxxx\n",
      "Figure 2A\n",
      "255\n",
      "xxxxx\n",
      ").\n",
      "256\n",
      "xxxxx\n",
      "Figure 1B\n",
      "257\n",
      "xxxxx\n",
      "(\n",
      "258\n",
      "xxxxx\n",
      "Figure 2B\n",
      "259\n",
      "xxxxx\n",
      "), in which we included the possibility that a fraction of the species be “fixed” (not shuttling) in the timescale of the experiment (e.g., molecules bound to the DNA in the nucleus).\n",
      "260\n",
      "xxxxx\n",
      "Download figure\n",
      "261\n",
      "xxxxx\n",
      "Open in new tab\n",
      "262\n",
      "xxxxx\n",
      "Figure 2 Experimental determination of nuclear transport rates in single cells.\n",
      "263\n",
      "xxxxx\n",
      "(A)\n",
      "264\n",
      "xxxxx\n",
      "Example of confocal-microscopy images from a representative FRAP experiment.\n",
      "265\n",
      "xxxxx\n",
      "Left:\n",
      "266\n",
      "xxxxx\n",
      "Time lapse images of a mother-daughter cell pair expressing Ace2-CFP and free YFP.\n",
      "267\n",
      "xxxxx\n",
      "Middle:\n",
      "268\n",
      "xxxxx\n",
      "Fluorescence intensity curves (red:\n",
      "269\n",
      "xxxxx\n",
      "Right:\n",
      "270\n",
      "xxxxx\n",
      "Likelihood profile analysis (see Methods).\n",
      "271\n",
      "xxxxx\n",
      "EV\n",
      "272\n",
      "xxxxx\n",
      "vs log10 K\n",
      "273\n",
      "xxxxx\n",
      "I\n",
      "274\n",
      "xxxxx\n",
      ".\n",
      "275\n",
      "xxxxx\n",
      "(B)\n",
      "276\n",
      "xxxxx\n",
      "Diagram of the reactions included in the full model used for fitting the experimental data.\n",
      "277\n",
      "xxxxx\n",
      "c\n",
      "278\n",
      "xxxxx\n",
      ") or nucleus (F\n",
      "279\n",
      "xxxxx\n",
      "c\n",
      "280\n",
      "xxxxx\n",
      ")was added.\n",
      "281\n",
      "xxxxx\n",
      "(C)\n",
      "282\n",
      "xxxxx\n",
      "Diagram of the yeast cell cycle.\n",
      "283\n",
      "xxxxx\n",
      "(D)\n",
      "284\n",
      "xxxxx\n",
      "Ratio of cytosol/nucleosol diffusion-available volumes, estimated assuming K\n",
      "285\n",
      "xxxxx\n",
      "EV\n",
      "286\n",
      "xxxxx\n",
      "= K\n",
      "287\n",
      "xxxxx\n",
      "I\n",
      "288\n",
      "xxxxx\n",
      "for freely diffusible YFP.\n",
      "289\n",
      "xxxxx\n",
      "(E-F)\n",
      "290\n",
      "xxxxx\n",
      "Estimated nuclear transport rates for the proteins YFP (n=58), Fus3-YFP (n=48) and Ace2-YFP (n=49) and Ace2 G128E-YFP (n=20).\n",
      "291\n",
      "xxxxx\n",
      "(G)\n",
      "292\n",
      "xxxxx\n",
      "Fraction of the total amount of proteins that is fixed in the nucleus (not shuttling between nucleus and cytosol).\n",
      "293\n",
      "xxxxx\n",
      "(H)\n",
      "294\n",
      "xxxxx\n",
      "Number of nuclear fixed Ace2 molecules in mother and daughter cells, for WT and G128E mutant strains.\n",
      "295\n",
      "xxxxx\n",
      "(I)\n",
      "296\n",
      "xxxxx\n",
      "Distinct symbol shapes indicate experiment replicates.\n",
      "297\n",
      "xxxxx\n",
      "In order to minimize potential variation in the transport rates due to the cell cycle, we restricted our measurements to cells at the beginning of G1 by assessing the localization of two proteins:\n",
      "298\n",
      "xxxxx\n",
      "Lippincott and Li, 1998\n",
      "299\n",
      "xxxxx\n",
      "), and CFP-Ace2, which translocates to the nucleus at the end of mitosis and remains nuclear during early G1.\n",
      "300\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "301\n",
      "xxxxx\n",
      ") (\n",
      "302\n",
      "xxxxx\n",
      "Figure 2C\n",
      "303\n",
      "xxxxx\n",
      ").\n",
      "304\n",
      "xxxxx\n",
      "Figure 2C\n",
      "305\n",
      "xxxxx\n",
      ", grey bracket).\n",
      "306\n",
      "xxxxx\n",
      "Fitting of the FRAP data yielded parameters K\n",
      "307\n",
      "xxxxx\n",
      "I\n",
      "308\n",
      "xxxxx\n",
      "and K\n",
      "309\n",
      "xxxxx\n",
      "EV\n",
      "310\n",
      "xxxxx\n",
      ".\n",
      "311\n",
      "xxxxx\n",
      "V\n",
      "312\n",
      "xxxxx\n",
      ", the ratio of the diffusion-accessible volumes of the cytoplasm and the nucleus.\n",
      "313\n",
      "xxxxx\n",
      "V\n",
      "314\n",
      "xxxxx\n",
      "The value of\n",
      "315\n",
      "xxxxx\n",
      ").\n",
      "283\n",
      "xxxxx\n",
      "39\n",
      "284\n",
      "xxxxx\n",
      ") (\n",
      "285\n",
      "xxxxx\n",
      "http://www.unafold.org/Dinamelt/applications/two-state-melting-hybridization.php\n",
      "286\n",
      "xxxxx\n",
      ") using default parameters at 37°C.\n",
      "287\n",
      "xxxxx\n",
      "https://www.ebi.ac.uk/Tools/psa/emboss_needle/\n",
      "288\n",
      "xxxxx\n",
      ").\n",
      "289\n",
      "xxxxx\n",
      "https://github.com/MsDwijayanti/hamming/blob/main/hamming.py\n",
      "290\n",
      "xxxxx\n",
      ").\n",
      "291\n",
      "xxxxx\n",
      "40\n",
      "292\n",
      "xxxxx\n",
      ") and RBS calculator v2.0 (\n",
      "293\n",
      "xxxxx\n",
      "41\n",
      "294\n",
      "xxxxx\n",
      ",\n",
      "295\n",
      "xxxxx\n",
      "42\n",
      "296\n",
      "xxxxx\n",
      ") with the input sequence starting from the Transcription Start Site (+1) to 100 bp of the downstream CDS.\n",
      "297\n",
      "xxxxx\n",
      "38\n",
      "298\n",
      "xxxxx\n",
      ",\n",
      "299\n",
      "xxxxx\n",
      "43\n",
      "300\n",
      "xxxxx\n",
      ",\n",
      "301\n",
      "xxxxx\n",
      "44\n",
      "302\n",
      "xxxxx\n",
      ") (\n",
      "303\n",
      "xxxxx\n",
      "https://bio.tools/coprarna\n",
      "304\n",
      "xxxxx\n",
      ").\n",
      "305\n",
      "xxxxx\n",
      "E. coli\n",
      "306\n",
      "xxxxx\n",
      "MG1655), NC_010473 (\n",
      "307\n",
      "xxxxx\n",
      "E. coli\n",
      "308\n",
      "xxxxx\n",
      "DH10b), NC_012892 and NC_012971 (\n",
      "309\n",
      "xxxxx\n",
      "E. coli\n",
      "310\n",
      "xxxxx\n",
      "BL21 (DE3)) were used as inputs references for host-background off-target prediction.\n",
      "311\n",
      "xxxxx\n",
      "Plasmid assembly\n",
      "312\n",
      "xxxxx\n",
      "Plasmids were constructed using the BASIC DNA assembly method (\n",
      "313\n",
      "xxxxx\n",
      "34\n",
      "314\n",
      "xxxxx\n",
      ",\n",
      "315\n",
      "xxxxx\n",
      "45\n",
      "316\n",
      "xxxxx\n",
      ").\n",
      "317\n",
      "xxxxx\n",
      "Supplementary Table 5\n",
      "318\n",
      "xxxxx\n",
      "Linkers for BASIC assembly were provided by Biolegio or IDT (\n",
      "319\n",
      "xxxxx\n",
      "Supplementary Table 6\n",
      "320\n",
      "xxxxx\n",
      ").\n",
      "321\n",
      "xxxxx\n",
      "E. coli\n",
      "322\n",
      "xxxxx\n",
      "DH5α.\n",
      "323\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "324\n",
      "xxxxx\n",
      ".\n",
      "325\n",
      "xxxxx\n",
      "Flow cytometry assay\n",
      "326\n",
      "xxxxx\n",
      "Cellular expression of sfGFP and mCherry was performed in 96 well plates with biological triplicates for each construct.\n",
      "327\n",
      "xxxxx\n",
      "E. coli\n",
      "328\n",
      "xxxxx\n",
      "carrying an empty backbone (containing only an origin of replication and an antibiotic resistance gene) was used as a negative control.\n",
      "329\n",
      "xxxxx\n",
      "E. coli\n",
      "330\n",
      "xxxxx\n",
      "colonies were inoculated from glycerol stock into LB media supplemented with appropriate antibiotics.\n",
      "331\n",
      "xxxxx\n",
      "600\n",
      "332\n",
      "xxxxx\n",
      ";\n",
      "333\n",
      "xxxxx\n",
      "600\n",
      "334\n",
      "xxxxx\n",
      "typically reached values in the range 0.6-0.7.\n",
      "335\n",
      "xxxxx\n",
      "Fig.\n",
      "336\n",
      "xxxxx\n",
      "and\n",
      "337\n",
      "xxxxx\n",
      "d\n",
      "338\n",
      "xxxxx\n",
      "For the triple reporters system (\n",
      "339\n",
      "xxxxx\n",
      "Fig.\n",
      "340\n",
      "xxxxx\n",
      "), fluorescence was measured using Attune NxT flow cytometer with a 488 nm excitation laser and channel BL1 530/30 for sfGFP;\n",
      "341\n",
      "xxxxx\n",
      "Plate reader assay\n",
      "342\n",
      "xxxxx\n",
      "For continuous growth assay, overnight liquid cultures (as for flow cytometry assay) were diluted 200x in LB media (10 µl into 190 µl, then 10 µl into 90 µl) using an automated liquid handling robot (CyBio Felix);\n",
      "343\n",
      "xxxxx\n",
      "Absorbance at 600 nm and GFP fluorescence (F:482-16/F:530-40) were measured every 15 min.\n",
      "344\n",
      "xxxxx\n",
      "600\n",
      "345\n",
      "xxxxx\n",
      "was calculated by subtracting the mean Abs\n",
      "346\n",
      "xxxxx\n",
      "600\n",
      "347\n",
      "xxxxx\n",
      "of media only control from the Abs\n",
      "348\n",
      "xxxxx\n",
      "600\n",
      "349\n",
      "xxxxx\n",
      "of each well.\n",
      "350\n",
      "xxxxx\n",
      "600\n",
      "351\n",
      "xxxxx\n",
      ".\n",
      "352\n",
      "xxxxx\n",
      "600\n",
      "353\n",
      "xxxxx\n",
      "was calculated by dividing the value of corrected fluorescence by corrected Abs\n",
      "354\n",
      "xxxxx\n",
      "600\n",
      "355\n",
      "xxxxx\n",
      ".\n",
      "356\n",
      "xxxxx\n",
      "RESULTS\n",
      "357\n",
      "xxxxx\n",
      "Design of a modular Artificial RNA interference regulation system\n",
      "358\n",
      "xxxxx\n",
      "The mARi sequences were designed with two core components:\n",
      "359\n",
      "xxxxx\n",
      "Fig.\n",
      "360\n",
      "xxxxx\n",
      ",\n",
      "361\n",
      "xxxxx\n",
      "2a\n",
      "362\n",
      "xxxxx\n",
      ").\n",
      "363\n",
      "xxxxx\n",
      "46\n",
      "364\n",
      "xxxxx\n",
      ",\n",
      "365\n",
      "xxxxx\n",
      "47\n",
      "366\n",
      "xxxxx\n",
      ") in a wide range of organisms (\n",
      "367\n",
      "xxxxx\n",
      "48\n",
      "368\n",
      "xxxxx\n",
      "-\n",
      "369\n",
      "xxxxx\n",
      "51\n",
      "370\n",
      "xxxxx\n",
      ").\n",
      "371\n",
      "xxxxx\n",
      "E. coli\n",
      "372\n",
      "xxxxx\n",
      "(\n",
      "373\n",
      "xxxxx\n",
      "52\n",
      "374\n",
      "xxxxx\n",
      ") and\n",
      "375\n",
      "xxxxx\n",
      "Salmonella enterica\n",
      "376\n",
      "xxxxx\n",
      "(\n",
      "377\n",
      "xxxxx\n",
      "16\n",
      "378\n",
      "xxxxx\n",
      "V\n",
      "316\n",
      "xxxxx\n",
      "is of interest by itself, and it should not be confused with the ratio of the cytoplasm to the nucleus geometric volume, since not all of the cytoplasmic and nuclear volumes are available for diffusion, due to the presence of organelles, macromolecules, chromatin and other cellular materials precluding diffusion of the protein of interest.\n",
      "317\n",
      "xxxxx\n",
      "V\n",
      "318\n",
      "xxxxx\n",
      "for each cell can be obtained for a molecule with identical import and export rates, such as YFP, which is small enough to diffuse passively through the nuclear pores, and it does not interact with any importin or exportin.\n",
      "319\n",
      "xxxxx\n",
      "ACT1\n",
      "320\n",
      "xxxxx\n",
      "promoter (P\n",
      "321\n",
      "xxxxx\n",
      "ACT1\n",
      "322\n",
      "xxxxx\n",
      ").\n",
      "323\n",
      "xxxxx\n",
      "EV\n",
      "324\n",
      "xxxxx\n",
      "and K\n",
      "325\n",
      "xxxxx\n",
      "I\n",
      "326\n",
      "xxxxx\n",
      "we calculated the K\n",
      "327\n",
      "xxxxx\n",
      "EV\n",
      "328\n",
      "xxxxx\n",
      "to K\n",
      "329\n",
      "xxxxx\n",
      "I\n",
      "330\n",
      "xxxxx\n",
      "ratio and obtained a\n",
      "331\n",
      "xxxxx\n",
      "V\n",
      "332\n",
      "xxxxx\n",
      "of 7.14 ± 0.53 (mean ± SEM) with an SD of 2.64 (\n",
      "333\n",
      "xxxxx\n",
      "Figure 2D\n",
      "334\n",
      "xxxxx\n",
      ").\n",
      "335\n",
      "xxxxx\n",
      "V\n",
      "336\n",
      "xxxxx\n",
      "of 6.84 ± 0.43 with an SD of 2.08 (\n",
      "337\n",
      "xxxxx\n",
      "Figure 2D\n",
      "338\n",
      "xxxxx\n",
      ").\n",
      "339\n",
      "xxxxx\n",
      "-1\n",
      "340\n",
      "xxxxx\n",
      "with and SD of 0.073 sec\n",
      "341\n",
      "xxxxx\n",
      "-1\n",
      "342\n",
      "xxxxx\n",
      "(\n",
      "343\n",
      "xxxxx\n",
      "Figure 2 E-F\n",
      "344\n",
      "xxxxx\n",
      ",\n",
      "345\n",
      "xxxxx\n",
      "Table 1\n",
      "346\n",
      "xxxxx\n",
      ").\n",
      "347\n",
      "xxxxx\n",
      "V\n",
      "348\n",
      "xxxxx\n",
      "calculated from the YFP measurements.\n",
      "349\n",
      "xxxxx\n",
      "View this table:\n",
      "350\n",
      "xxxxx\n",
      "View inline\n",
      "351\n",
      "xxxxx\n",
      "View popup\n",
      "352\n",
      "xxxxx\n",
      "Download powerpoint\n",
      "353\n",
      "xxxxx\n",
      "Table 1 Estimated parameters from the scFRAP experiments.\n",
      "354\n",
      "xxxxx\n",
      "To assess the statistical significance of the estimated parameters Export rate (K\n",
      "355\n",
      "xxxxx\n",
      "EV\n",
      "356\n",
      "xxxxx\n",
      "), Import Rate (K\n",
      "357\n",
      "xxxxx\n",
      "I\n",
      "358\n",
      "xxxxx\n",
      "), tau and Nuclear fixed fraction, a Welch two sample t-test was performed on the log-transformed values.\n",
      "359\n",
      "xxxxx\n",
      "We applied this method to study the nuclear transport of Ace2 using a strain expressing\n",
      "360\n",
      "xxxxx\n",
      "P\n",
      "361\n",
      "xxxxx\n",
      "ACE2\n",
      "362\n",
      "xxxxx\n",
      "-YFP-ACE2\n",
      "363\n",
      "xxxxx\n",
      ".\n",
      "364\n",
      "xxxxx\n",
      "P\n",
      "365\n",
      "xxxxx\n",
      "FUS3\n",
      "366\n",
      "xxxxx\n",
      "-FUS3-YFP\n",
      "367\n",
      "xxxxx\n",
      ", coding for an unrelated protein kinase Fus3, central in the pheromone response pathway, which also shuttles between the nucleus and the cytoplasm (\n",
      "368\n",
      "xxxxx\n",
      "Dohlman and Thorner, 2001\n",
      "369\n",
      "xxxxx\n",
      ").\n",
      "370\n",
      "xxxxx\n",
      "Paine and Scherr, 1975\n",
      "371\n",
      "xxxxx\n",
      ").\n",
      "372\n",
      "xxxxx\n",
      "van Drogen et al.,\n",
      "373\n",
      "xxxxx\n",
      ").\n",
      "374\n",
      "xxxxx\n",
      "The export rate for Fus3-YFP did not differ from that of YFP, but Fus3 import was faster (\n",
      "375\n",
      "xxxxx\n",
      "Figure 2 E-F\n",
      "376\n",
      "xxxxx\n",
      ",\n",
      "377\n",
      "xxxxx\n",
      "Table 1\n",
      "378\n",
      "xxxxx\n",
      ").\n",
      "379\n",
      "xxxxx\n",
      "Figure 2 E-F\n",
      "380\n",
      "xxxxx\n",
      ",\n",
      "381\n",
      "xxxxx\n",
      "Table 1\n",
      "382\n",
      "xxxxx\n",
      "We also measured the behavior of the mutant Ace2-G128E (\n",
      "383\n",
      "xxxxx\n",
      "2000\n",
      "384\n",
      "xxxxx\n",
      ").\n",
      "385\n",
      "xxxxx\n",
      "Jensen et al.,\n",
      "386\n",
      "xxxxx\n",
      "), and shows an altered cellular localization, persisting in the mother cell nucleus to mid G1 phase, similar to daughter cells (\n",
      "387\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "388\n",
      "xxxxx\n",
      ").\n",
      "389\n",
      "xxxxx\n",
      "Bourens et al.,\n",
      "390\n",
      "xxxxx\n",
      ";\n",
      "391\n",
      "xxxxx\n",
      "Mazanka et al.,\n",
      "392\n",
      "xxxxx\n",
      ").\n",
      "393\n",
      "xxxxx\n",
      "Figure 2 E-F\n",
      "394\n",
      "xxxxx\n",
      ",\n",
      "395\n",
      "xxxxx\n",
      "Table 1\n",
      "396\n",
      "xxxxx\n",
      ").\n",
      "397\n",
      "xxxxx\n",
      "Estimation of the amount of non-shutting molecules in the nucleus\n",
      "398\n",
      "xxxxx\n",
      "The FRAP-analysis method also renders an estimation of the number of non-shuttling (“fixed”) molecules present during the experiment in either compartment (\n",
      "399\n",
      "xxxxx\n",
      "Figure 2B\n",
      "400\n",
      "xxxxx\n",
      ").\n",
      "401\n",
      "xxxxx\n",
      "Figure 2G\n",
      "402\n",
      "xxxxx\n",
      "), as expected.\n",
      "403\n",
      "xxxxx\n",
      "Figure 2G\n",
      "404\n",
      "xxxxx\n",
      ").\n",
      "405\n",
      "xxxxx\n",
      "Dohlman and Thorner, 2001\n",
      "406\n",
      "xxxxx\n",
      ").\n",
      "407\n",
      "xxxxx\n",
      "van Drogen et al.,\n",
      "408\n",
      "xxxxx\n",
      ").\n",
      "409\n",
      "xxxxx\n",
      "Figure 2H\n",
      "410\n",
      "xxxxx\n",
      "), consistent with Ace2 preferential induction of genes in daughter cells (\n",
      "411\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "412\n",
      "xxxxx\n",
      ").\n",
      "413\n",
      "xxxxx\n",
      "Figure 2I\n",
      "414\n",
      "xxxxx\n",
      ",\n",
      "415\n",
      "xxxxx\n",
      "SM Figure 1\n",
      "416\n",
      "xxxxx\n",
      ").\n",
      "417\n",
      "xxxxx\n",
      "Colman-Lerner et al.,\n",
      "418\n",
      "xxxxx\n",
      "), mother cells had a similar number of nuclear fixed YFP-Ace2 molecules than daughters (\n",
      "419\n",
      "xxxxx\n",
      "Figure 2H\n",
      "420\n",
      "xxxxx\n",
      ",\n",
      "421\n",
      "xxxxx\n",
      "SM Figure 1\n",
      "422\n",
      "xxxxx\n",
      ").\n",
      "379\n",
      "xxxxx\n",
      "30\n",
      "380\n",
      "xxxxx\n",
      ").\n",
      "381\n",
      "xxxxx\n",
      "17\n",
      "382\n",
      "xxxxx\n",
      ",\n",
      "383\n",
      "xxxxx\n",
      "25\n",
      "384\n",
      "xxxxx\n",
      ",\n",
      "385\n",
      "xxxxx\n",
      "27\n",
      "386\n",
      "xxxxx\n",
      ",\n",
      "387\n",
      "xxxxx\n",
      "30\n",
      "388\n",
      "xxxxx\n",
      ") (\n",
      "389\n",
      "xxxxx\n",
      "Fig.\n",
      "390\n",
      "xxxxx\n",
      ").\n",
      "391\n",
      "xxxxx\n",
      "Download figure\n",
      "392\n",
      "xxxxx\n",
      "Open in new tab\n",
      "393\n",
      "xxxxx\n",
      "Figure 2.\n",
      "394\n",
      "xxxxx\n",
      "(\n",
      "395\n",
      "xxxxx\n",
      "a\n",
      "396\n",
      "xxxxx\n",
      ") Sequence and secondary structure of mARi containing a seed sequence (grey) and MicC RNA scaffold (purple). (\n",
      "397\n",
      "xxxxx\n",
      "b\n",
      "398\n",
      "xxxxx\n",
      ") Schematic design of the target site selection schema for mARi-mediated repression.\n",
      "399\n",
      "xxxxx\n",
      "Supplementary Table 1\n",
      "400\n",
      "xxxxx\n",
      ").\n",
      "401\n",
      "xxxxx\n",
      "(\n",
      "402\n",
      "xxxxx\n",
      "c\n",
      "403\n",
      "xxxxx\n",
      ") Functional characterisation of mARi-based gene regulation showing the silencing activity of mARi-A for the different target positions vs a sfGFP-only control without mARi. (\n",
      "404\n",
      "xxxxx\n",
      "d\n",
      "405\n",
      "xxxxx\n",
      ") Uniformity of representative expressing strains with and without mARi expression. (\n",
      "406\n",
      "xxxxx\n",
      "e\n",
      "407\n",
      "xxxxx\n",
      ") The fluorescence intensity (top) and normalized fluorescence (bottom) when position 1 mARi-A was used to target UTR-A with varying RBS strengths.\n",
      "408\n",
      "xxxxx\n",
      "The seed sequence within mARi creates a specific base-pairing interaction to a target sequence in the mRNA of interest, resulting in its post-transcriptional silencing.\n",
      "409\n",
      "xxxxx\n",
      "34\n",
      "410\n",
      "xxxxx\n",
      ").\n",
      "411\n",
      "xxxxx\n",
      "Fig.\n",
      "412\n",
      "xxxxx\n",
      ") (\n",
      "413\n",
      "xxxxx\n",
      "34\n",
      "414\n",
      "xxxxx\n",
      ").\n",
      "415\n",
      "xxxxx\n",
      "Fig.\n",
      "416\n",
      "xxxxx\n",
      ").\n",
      "417\n",
      "xxxxx\n",
      "35\n",
      "418\n",
      "xxxxx\n",
      "), they provide an ideal target for mARi and reduce off-target effects.\n",
      "419\n",
      "xxxxx\n",
      "We designed mARi sequences to target the translation initiation region of mRNA for three main reasons: (\n",
      "420\n",
      "xxxxx\n",
      "Supplementary Table 1\n",
      "421\n",
      "xxxxx\n",
      ") and AU rich sequences are preferentially bound by Hfq (\n",
      "422\n",
      "xxxxx\n",
      "53\n",
      "423\n",
      "xxxxx\n",
      "); (\n",
      "424\n",
      "xxxxx\n",
      "trans\n",
      "425\n",
      "xxxxx\n",
      "-encoded sRNA-based regulation (\n",
      "426\n",
      "xxxxx\n",
      "30\n",
      "427\n",
      "xxxxx\n",
      "), which is important for regulator specificity/orthogonality; (\n",
      "428\n",
      "xxxxx\n",
      "34\n",
      "429\n",
      "xxxxx\n",
      ").\n",
      "430\n",
      "xxxxx\n",
      "Fig.\n",
      "431\n",
      "xxxxx\n",
      ").\n",
      "432\n",
      "xxxxx\n",
      "superfolder-gfp\n",
      "433\n",
      "xxxxx\n",
      "(\n",
      "434\n",
      "xxxxx\n",
      "sfgfp\n",
      "435\n",
      "xxxxx\n",
      ";\n",
      "436\n",
      "xxxxx\n",
      "Fig.\n",
      "437\n",
      "xxxxx\n",
      ").\n",
      "438\n",
      "xxxxx\n",
      "Supplementary Table 1\n",
      "439\n",
      "xxxxx\n",
      ").\n",
      "440\n",
      "xxxxx\n",
      "The design of the full sequence of each mARi variant was computationally evaluated to ensure that it met the requirements for effective repression (\n",
      "441\n",
      "xxxxx\n",
      "30\n",
      "442\n",
      "xxxxx\n",
      ").\n",
      "443\n",
      "xxxxx\n",
      "34\n",
      "444\n",
      "xxxxx\n",
      "), specifically avoiding the forbidden restriction site\n",
      "445\n",
      "xxxxx\n",
      "Bsa\n",
      "446\n",
      "xxxxx\n",
      "I. Secondly, to achieve effective repression and to have a high affinity binding between the seed and target sequences, the GC content and binding energy of the seed sequences were calculated and found to vary from 34.62% to 43.8% and -38.6 kcal/mol to -61 kcal/mol, respectively.\n",
      "447\n",
      "xxxxx\n",
      "27\n",
      "448\n",
      "xxxxx\n",
      ",\n",
      "449\n",
      "xxxxx\n",
      "30\n",
      "450\n",
      "xxxxx\n",
      "), and previous studies suggested a typical binding energy of -30 to -40 kcal/mol (\n",
      "451\n",
      "xxxxx\n",
      "17\n",
      "452\n",
      "xxxxx\n",
      ",\n",
      "453\n",
      "xxxxx\n",
      "27\n",
      "454\n",
      "xxxxx\n",
      ") was required for effective repression activity.\n",
      "455\n",
      "xxxxx\n",
      "30\n",
      "456\n",
      "xxxxx\n",
      ").\n",
      "457\n",
      "xxxxx\n",
      "30\n",
      "458\n",
      "xxxxx\n",
      "): shorter binding sites have a lower binding affinity and may cause off-target effects (\n",
      "459\n",
      "xxxxx\n",
      "17\n",
      "460\n",
      "xxxxx\n",
      ",\n",
      "461\n",
      "xxxxx\n",
      "27\n",
      "462\n",
      "xxxxx\n",
      "), while longer binding sequences may have a complex secondary structure that could cause binding to the mRNA target to be less thermodynamically favorable (\n",
      "463\n",
      "xxxxx\n",
      "30\n",
      "464\n",
      "xxxxx\n",
      ").\n",
      "465\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "466\n",
      "xxxxx\n",
      ").\n",
      "467\n",
      "xxxxx\n",
      "J23101_BASIC\n",
      "468\n",
      "xxxxx\n",
      "and UTR-A-RBSc linker, while mARi expression was driven by the constitutive P\n",
      "469\n",
      "xxxxx\n",
      "J23101\n",
      "470\n",
      "xxxxx\n",
      "(\n",
      "471\n",
      "xxxxx\n",
      "Fig.\n",
      "472\n",
      "xxxxx\n",
      ").\n",
      "473\n",
      "xxxxx\n",
      "Fig.\n",
      "474\n",
      "xxxxx\n",
      ").\n",
      "423\n",
      "xxxxx\n",
      "SM Figure 1\n",
      "424\n",
      "xxxxx\n",
      ").\n",
      "425\n",
      "xxxxx\n",
      "Download figure\n",
      "426\n",
      "xxxxx\n",
      "Open in new tab\n",
      "427\n",
      "xxxxx\n",
      "Supplementary Figure 1 (related to Figure 2) Mother- daughter cell asymmetry in the distribution of Ace2 mobile and fixed molecules.\n",
      "428\n",
      "xxxxx\n",
      "(A) Plot shows fixed vs total Ace2-G128E. r is the correlation coefficient.\n",
      "429\n",
      "xxxxx\n",
      "Fig.\n",
      "430\n",
      "xxxxx\n",
      "We found no significant correlation.\n",
      "431\n",
      "xxxxx\n",
      "The high variability in nuclear transport rates is mainly due to variation in the amount of NPCs\n",
      "432\n",
      "xxxxx\n",
      "Interestingly, we found large cell-to-cell variation in the import and export rates of all three proteins studied (\n",
      "433\n",
      "xxxxx\n",
      "Figure 2 E-F\n",
      "434\n",
      "xxxxx\n",
      ",\n",
      "435\n",
      "xxxxx\n",
      "Table 1\n",
      "436\n",
      "xxxxx\n",
      ").\n",
      "437\n",
      "xxxxx\n",
      "Figure 3A\n",
      "438\n",
      "xxxxx\n",
      ").\n",
      "439\n",
      "xxxxx\n",
      "Download figure\n",
      "440\n",
      "xxxxx\n",
      "Open in new tab\n",
      "441\n",
      "xxxxx\n",
      "Figure 3.\n",
      "442\n",
      "xxxxx\n",
      "(\n",
      "443\n",
      "xxxxx\n",
      "A\n",
      "444\n",
      "xxxxx\n",
      ") Individual traces of FRAP experiments in single cells (gray lines).\n",
      "445\n",
      "xxxxx\n",
      "B\n",
      "446\n",
      "xxxxx\n",
      ") Plots show export vs import for the proteins YFP (n=58), Fus3-YFP (n=48) and Ace2-YFP (n=49) and Ace2 G128E-YFP (n=20).\n",
      "447\n",
      "xxxxx\n",
      "C\n",
      "448\n",
      "xxxxx\n",
      ") Plot shows a quantification of the correlated and un-correlated variation between import and export rates.\n",
      "449\n",
      "xxxxx\n",
      "Top:\n",
      "450\n",
      "xxxxx\n",
      "Representative confocal images of the analyzed nuclear pore proteins.\n",
      "451\n",
      "xxxxx\n",
      "Middle:\n",
      "452\n",
      "xxxxx\n",
      "Total fluorescence of GFP fusions to the indicated nuclear envelope proteins per cell, as quantified from fluorescence intensity (AU).\n",
      "453\n",
      "xxxxx\n",
      "Bottom:\n",
      "454\n",
      "xxxxx\n",
      "Density plots as above for Import Rate (sec\n",
      "455\n",
      "xxxxx\n",
      "-1\n",
      "456\n",
      "xxxxx\n",
      ") and Export rate (sec\n",
      "457\n",
      "xxxxx\n",
      "-1\n",
      "458\n",
      "xxxxx\n",
      ") of free YFP.\n",
      "459\n",
      "xxxxx\n",
      "We then wondered what the sources of the measured variability in rates were.\n",
      "460\n",
      "xxxxx\n",
      "Thus, to dissect these two sources of variation, we quantified the correlated and uncorrelated components of the variability in the rates of each protein (\n",
      "461\n",
      "xxxxx\n",
      "Qiuyan Fua and Pachter, 2016\n",
      "462\n",
      "xxxxx\n",
      ") (\n",
      "463\n",
      "xxxxx\n",
      "Figure 3 C\n",
      "464\n",
      "xxxxx\n",
      ").\n",
      "465\n",
      "xxxxx\n",
      "Figure 3B, C\n",
      "466\n",
      "xxxxx\n",
      ").\n",
      "467\n",
      "xxxxx\n",
      "Figure 3 B\n",
      "468\n",
      "xxxxx\n",
      "and\n",
      "469\n",
      "xxxxx\n",
      "C\n",
      "470\n",
      "xxxxx\n",
      "), probably because its import and export are individually regulated.\n",
      "471\n",
      "xxxxx\n",
      "Figure 3C\n",
      "472\n",
      "xxxxx\n",
      ")).\n",
      "473\n",
      "xxxxx\n",
      "Figure 2E\n",
      "474\n",
      "xxxxx\n",
      "), the high Ace2 uncorrelated variation could be attributed to cell-to-cell variability in the export rate.\n",
      "475\n",
      "xxxxx\n",
      "To assess whether the co-variation of import and export rates is due to differences among cells in the number of NPCs, we analyzed, as a proxy, the cell-to-cell variability in the amounts of nuclear pore proteins (\n",
      "476\n",
      "xxxxx\n",
      "Khmelinskii et al.,\n",
      "477\n",
      "xxxxx\n",
      ").\n",
      "478\n",
      "xxxxx\n",
      "2003\n",
      "479\n",
      "xxxxx\n",
      ") (\n",
      "480\n",
      "xxxxx\n",
      "Figure 3 D\n",
      "481\n",
      "xxxxx\n",
      ", top).\n",
      "482\n",
      "xxxxx\n",
      "Figure 3D\n",
      "483\n",
      "xxxxx\n",
      ").\n",
      "484\n",
      "xxxxx\n",
      "Segregation of NPCs, and nuclear transport rates during cell division are highly noisy\n",
      "485\n",
      "xxxxx\n",
      "Comparison of mother vs daughter NPC numbers revealed no significant correlation (\n",
      "486\n",
      "xxxxx\n",
      "Figure 4 A\n",
      "487\n",
      "xxxxx\n",
      "), suggesting that segregation for NPCs is highly noisy.\n",
      "488\n",
      "xxxxx\n",
      "Figure 4 B-C\n",
      "489\n",
      "xxxxx\n",
      ").\n",
      "490\n",
      "xxxxx\n",
      "SM Figure 2 A\n",
      "491\n",
      "xxxxx\n",
      ").\n",
      "492\n",
      "xxxxx\n",
      "Download figure\n",
      "493\n",
      "xxxxx\n",
      "Open in new tab\n",
      "494\n",
      "xxxxx\n",
      "Supplementary Figure 2 (related to Figure 4) Difference in nuclear transport rates between mother and daughter cells.\n",
      "495\n",
      "xxxxx\n",
      "(\n",
      "496\n",
      "xxxxx\n",
      "A) Similarity between export rate in pairs of cells in the case of YFP (left), Fus3-YFP (middle) and YFP-Ace2 (left).\n",
      "497\n",
      "xxxxx\n",
      "Download figure\n",
      "498\n",
      "xxxxx\n",
      "Open in new tab\n",
      "499\n",
      "xxxxx\n",
      "Figure 4.\n",
      "500\n",
      "xxxxx\n",
      "(\n",
      "501\n",
      "xxxxx\n",
      "A\n",
      "502\n",
      "xxxxx\n",
      ") Plots show daughter vs mother normalized nuclear fluorescence (total GFP fluorescence in the nucleus divided by the cell volume) of three NPC components, Mlp1 (left), Mlp2 (middle) and Nup49 (right). (\n",
      "503\n",
      "xxxxx\n",
      "B-C\n",
      "504\n",
      "xxxxx\n",
      ") Plots show daughter vs mother import rate (\n",
      "505\n",
      "xxxxx\n",
      "B\n",
      "506\n",
      "xxxxx\n",
      "), or export rate (\n",
      "507\n",
      "xxxxx\n",
      "C\n",
      "508\n",
      "xxxxx\n",
      "), obtained for YPF (left), Fus3-YFP (middle) and YFP-Ace2 (right, WT:\n",
      "509\n",
      "xxxxx\n",
      "D\n",
      "510\n",
      "xxxxx\n",
      ") Plot of the difference in rates (daughter minus its mother), import vs export for YFP-Ace2 (WT, blue circles) and the G128E mutant (green triangles).\n",
      "511\n",
      "xxxxx\n",
      "E\n",
      "512\n",
      "xxxxx\n",
      ") Ratio of import to export rate for the indicated strains in mothers (red) and daughters (green).\n",
      "513\n",
      "xxxxx\n",
      "Mother-Daughter asymmetry in nuclear transport rates\n",
      "514\n",
      "xxxxx\n",
      "Another mechanism that could cause population variability would be a systematic asymmetry in the segregation of components during division.\n",
      "515\n",
      "xxxxx\n",
      "I\n",
      "516\n",
      "xxxxx\n",
      "p = 0.46) or Fus3-YFP (t-test kE p = 0.48, K\n",
      "517\n",
      "xxxxx\n",
      "I\n",
      "518\n",
      "xxxxx\n",
      "p = 0.21) (\n",
      "519\n",
      "xxxxx\n",
      "Figure 4 B-C\n",
      "520\n",
      "xxxxx\n",
      ", left), indicating that whatever differences might exist between mothers and daughters, these are not sufficiently strong to overcome the noisy segregation of NPCs.\n",
      "521\n",
      "xxxxx\n",
      "In stark contrast with YFP and Fus3-YFP, for YFP-Ace2 the rates of both import and export were significantly faster in daughter cells than in mothers (t-test kE p = 0.06, K\n",
      "522\n",
      "xxxxx\n",
      "I\n",
      "523\n",
      "xxxxx\n",
      "p = 0.01) (\n",
      "524\n",
      "xxxxx\n",
      "Figure 4 B-C\n",
      "525\n",
      "xxxxx\n",
      ", right).\n",
      "526\n",
      "xxxxx\n",
      "Figure 4 D\n",
      "527\n",
      "xxxxx\n",
      "), an intriguing result that would require future studies.\n",
      "528\n",
      "xxxxx\n",
      "Figure 4 E\n",
      "529\n",
      "xxxxx\n",
      ").\n",
      "475\n",
      "xxxxx\n",
      "17\n",
      "476\n",
      "xxxxx\n",
      ",\n",
      "477\n",
      "xxxxx\n",
      "25\n",
      "478\n",
      "xxxxx\n",
      ",\n",
      "479\n",
      "xxxxx\n",
      "27\n",
      "480\n",
      "xxxxx\n",
      ").\n",
      "481\n",
      "xxxxx\n",
      "Fig.\n",
      "482\n",
      "xxxxx\n",
      ").\n",
      "483\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "484\n",
      "xxxxx\n",
      ").\n",
      "485\n",
      "xxxxx\n",
      "2d\n",
      "486\n",
      "xxxxx\n",
      ".\n",
      "487\n",
      "xxxxx\n",
      "To evaluate the independence of position 1 UTR target, a set of constructs with five different RBS strengths (RBSa-RBSe) (\n",
      "488\n",
      "xxxxx\n",
      "Supplementary Table 2\n",
      "489\n",
      "xxxxx\n",
      ") controlling sfGFP translation were assembled both with and without the position 1 mARi cassette all within the same UTR-A context.\n",
      "490\n",
      "xxxxx\n",
      "sfgfp\n",
      "491\n",
      "xxxxx\n",
      "gene-only constructs resulted in a series of fluorescence intensities reflecting RBS strength variation (\n",
      "492\n",
      "xxxxx\n",
      "Fig.\n",
      "493\n",
      "xxxxx\n",
      ").\n",
      "494\n",
      "xxxxx\n",
      "Fig.\n",
      "495\n",
      "xxxxx\n",
      ").\n",
      "496\n",
      "xxxxx\n",
      "mARi activity is tuneable via genetic design parameters\n",
      "497\n",
      "xxxxx\n",
      "To evaluate the features of mARi, we characterised its responsiveness to genetic design parameters and its robustness to different contexts and growth conditions.\n",
      "498\n",
      "xxxxx\n",
      "54\n",
      "499\n",
      "xxxxx\n",
      ").\n",
      "500\n",
      "xxxxx\n",
      "13\n",
      "501\n",
      "xxxxx\n",
      ",\n",
      "502\n",
      "xxxxx\n",
      "32\n",
      "503\n",
      "xxxxx\n",
      ",\n",
      "504\n",
      "xxxxx\n",
      "54\n",
      "505\n",
      "xxxxx\n",
      "), here we constitutively expressed both transcripts to test the impact of different transcript levels, which provides more consistent relative expression.\n",
      "506\n",
      "xxxxx\n",
      "54\n",
      "507\n",
      "xxxxx\n",
      ").\n",
      "508\n",
      "xxxxx\n",
      "J23xxx_BASIC\n",
      "509\n",
      "xxxxx\n",
      ") were used to express a\n",
      "510\n",
      "xxxxx\n",
      "sfGFP\n",
      "511\n",
      "xxxxx\n",
      "reporter gene using UTR A-RBSc in a p15A backbone (\n",
      "512\n",
      "xxxxx\n",
      "3a\n",
      "513\n",
      "xxxxx\n",
      ").\n",
      "514\n",
      "xxxxx\n",
      "Fig.\n",
      "515\n",
      "xxxxx\n",
      "), which is calculated by dividing the strength of the promoter used for mARi by the promoter expressing the mRNA target.\n",
      "516\n",
      "xxxxx\n",
      "Fig.\n",
      "517\n",
      "xxxxx\n",
      ").\n",
      "518\n",
      "xxxxx\n",
      "Fig.\n",
      "519\n",
      "xxxxx\n",
      ", Supplementary Fig.\n",
      "520\n",
      "xxxxx\n",
      ").\n",
      "521\n",
      "xxxxx\n",
      "30\n",
      "522\n",
      "xxxxx\n",
      ",\n",
      "523\n",
      "xxxxx\n",
      "54\n",
      "524\n",
      "xxxxx\n",
      ").\n",
      "525\n",
      "xxxxx\n",
      "Download figure\n",
      "526\n",
      "xxxxx\n",
      "Open in new tab\n",
      "527\n",
      "xxxxx\n",
      "Figure 3.\n",
      "528\n",
      "xxxxx\n",
      "(a)\n",
      "529\n",
      "xxxxx\n",
      "Heat-map showing relative expression ratio calculated from the relative strength of promoters driving mARi and mRNA expression across a matrix of constructs (\n",
      "530\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "531\n",
      "xxxxx\n",
      ").\n",
      "532\n",
      "xxxxx\n",
      "J23119\n",
      "533\n",
      "xxxxx\n",
      "-mARi:\n",
      "534\n",
      "xxxxx\n",
      "J23101BASIC\n",
      "535\n",
      "xxxxx\n",
      "-mRNA) and grey square (P\n",
      "536\n",
      "xxxxx\n",
      "J23101\n",
      "537\n",
      "xxxxx\n",
      "-mARi:\n",
      "538\n",
      "xxxxx\n",
      "J23101BASIC\n",
      "539\n",
      "xxxxx\n",
      "-mRNA), respectively.\n",
      "540\n",
      "xxxxx\n",
      "(b)\n",
      "541\n",
      "xxxxx\n",
      "Schematic of mARi repression constructs used to investigate the effect of transcript ratios using four different strength constitutive promoters driving both mARi and target mRNA in a single plasmid system.\n",
      "542\n",
      "xxxxx\n",
      "(c\n",
      "543\n",
      "xxxxx\n",
      ") Heat-map showing the measured relative expression levels (mean normalized fluorescence) of the mARi expression ratio matrix from (\n",
      "544\n",
      "xxxxx\n",
      "a\n",
      "545\n",
      "xxxxx\n",
      ").\n",
      "546\n",
      "xxxxx\n",
      "Supplementary Table 3\n",
      "547\n",
      "xxxxx\n",
      ". (\n",
      "548\n",
      "xxxxx\n",
      "d\n",
      "549\n",
      "xxxxx\n",
      ") Correlation of transcript expression ratios in\n",
      "550\n",
      "xxxxx\n",
      "a\n",
      "551\n",
      "xxxxx\n",
      "and normalised fluorescence in\n",
      "552\n",
      "xxxxx\n",
      "c\n",
      "553\n",
      "xxxxx\n",
      ". (\n",
      "554\n",
      "xxxxx\n",
      "e\n",
      "555\n",
      "xxxxx\n",
      ") Schematic of the mARi-based repression system with high and low expression ratios constructed in single- and double-plasmid systems with different copy numbers.\n",
      "556\n",
      "xxxxx\n",
      "f\n",
      "557\n",
      "xxxxx\n",
      ") and double (\n",
      "558\n",
      "xxxxx\n",
      "g\n",
      "559\n",
      "xxxxx\n",
      ") plasmid systems for high and low ratio mARi against a control construct without mARi;\n",
      "560\n",
      "xxxxx\n",
      "h\n",
      "561\n",
      "xxxxx\n",
      ") The robustness of the double-plasmid mARi-based regulation in different incubation temperatures.\n",
      "562\n",
      "xxxxx\n",
      "i\n",
      "563\n",
      "xxxxx\n",
      ") The robustness of the double-plasmid mARi-based repression system in a variety of growth media. (\n",
      "564\n",
      "xxxxx\n",
      "j\n",
      "565\n",
      "xxxxx\n",
      ") Reporter expression of the single-plasmid high-ratio mARi on p15A was measured in different\n",
      "566\n",
      "xxxxx\n",
      "E. coli\n",
      "567\n",
      "xxxxx\n",
      "strains against a control construct without mARi. (\n",
      "568\n",
      "xxxxx\n",
      "k\n",
      "569\n",
      "xxxxx\n",
      ") The repression activity of single-plasmid high-ratio mARi on p15A in representative strains (DH10b, BL21 (DE3), and BL21star (DE3)) across different growth phases (\n",
      "570\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "571\n",
      "xxxxx\n",
      ").\n",
      "572\n",
      "xxxxx\n",
      "h, i, j\n",
      "573\n",
      "xxxxx\n",
      "and\n",
      "574\n",
      "xxxxx\n",
      "k\n",
      "575\n",
      "xxxxx\n",
      "were performed by plate reader assay.\n",
      "576\n",
      "xxxxx\n",
      "Download figure\n",
      "577\n",
      "xxxxx\n",
      "Open in new tab\n",
      "578\n",
      "xxxxx\n",
      "Figure 4.\n",
      "579\n",
      "xxxxx\n",
      "(\n",
      "580\n",
      "xxxxx\n",
      "a\n",
      "581\n",
      "xxxxx\n",
      ") A schematic of experimental design to evaluate the target specificity of modular mARi-mRNA pairs is shown.\n",
      "582\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "583\n",
      "xxxxx\n",
      ");\n",
      "584\n",
      "xxxxx\n",
      "b\n",
      "585\n",
      "xxxxx\n",
      ") A schematic of reporters in dual transcription units is shown:\n",
      "586\n",
      "xxxxx\n",
      "c\n",
      "587\n",
      "xxxxx\n",
      ") A schematic of reporters in triple transcription units is shown:\n",
      "588\n",
      "xxxxx\n",
      "d\n",
      "589\n",
      "xxxxx\n",
      ") A schematic of dual reporters in an operon system is shown:\n",
      "590\n",
      "xxxxx\n",
      "A further variable affecting intracellular transcript abundance is plasmid copy number used in the system.\n",
      "591\n",
      "xxxxx\n",
      "Fig.\n",
      "592\n",
      "xxxxx\n",
      ") and low (grey box in\n",
      "593\n",
      "xxxxx\n",
      "Fig.\n",
      "594\n",
      "xxxxx\n",
      ") expression ratios were selected to exemplify different levels of transcript abundance.\n",
      "595\n",
      "xxxxx\n",
      "55\n",
      "596\n",
      "xxxxx\n",
      ") (\n",
      "597\n",
      "xxxxx\n",
      "Fig.\n",
      "598\n",
      "xxxxx\n",
      "left panel, Supplementary Fig.\n",
      "599\n",
      "xxxxx\n",
      ").\n",
      "600\n",
      "xxxxx\n",
      "Fig.\n",
      "601\n",
      "xxxxx\n",
      ").\n",
      "602\n",
      "xxxxx\n",
      "We next reasoned that by placing the mARi on a second, higher copy vector, repression should be more efficient.\n",
      "603\n",
      "xxxxx\n",
      "3e\n",
      "604\n",
      "xxxxx\n",
      "right panel, Supplementary Fig.\n",
      "605\n",
      "xxxxx\n",
      ").\n",
      "606\n",
      "xxxxx\n",
      "Fig.\n",
      "607\n",
      "xxxxx\n",
      ").\n",
      "608\n",
      "xxxxx\n",
      "56\n",
      "609\n",
      "xxxxx\n",
      ").\n",
      "610\n",
      "xxxxx\n",
      "To investigate the robustness of mARi repression system, we further tested the double plasmid system with both high and low expression ratios in different temperatures and growth media.\n",
      "611\n",
      "xxxxx\n",
      "Fig.\n",
      "612\n",
      "xxxxx\n",
      ").\n",
      "613\n",
      "xxxxx\n",
      "mARi-based regulation is robust to different genetic contexts\n",
      "614\n",
      "xxxxx\n",
      "Unlike other biomolecular regulators (e.g. transcription factors, TALEs, zinc fingers, or CRISPRi) (\n",
      "615\n",
      "xxxxx\n",
      "5\n",
      "616\n",
      "xxxxx\n",
      ",\n",
      "617\n",
      "xxxxx\n",
      "57\n",
      "618\n",
      "xxxxx\n",
      ",\n",
      "619\n",
      "xxxxx\n",
      "58\n",
      "620\n",
      "xxxxx\n",
      "), mARi only requires transcriptional activity and endogenous Hfq, which is autoregulated by Hfq binding its own mRNA (\n",
      "621\n",
      "xxxxx\n",
      "59\n",
      "622\n",
      "xxxxx\n",
      ").\n",
      "623\n",
      "xxxxx\n",
      "E. coli\n",
      "624\n",
      "xxxxx\n",
      "strains:\n",
      "625\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "626\n",
      "xxxxx\n",
      ").\n",
      "627\n",
      "xxxxx\n",
      "We then sought to evaluate the silencing activity of mARi in these\n",
      "628\n",
      "xxxxx\n",
      "E. coli\n",
      "629\n",
      "xxxxx\n",
      "K12 and B strains.\n",
      "630\n",
      "xxxxx\n",
      "E. coli\n",
      "631\n",
      "xxxxx\n",
      "Hfq chaperone and degradosome complex (e.g. RNase E) as helper proteins to regulate gene repression.\n",
      "632\n",
      "xxxxx\n",
      "E. coli\n",
      "633\n",
      "xxxxx\n",
      "cells reach stationary phase (\n",
      "634\n",
      "xxxxx\n",
      "47\n",
      "635\n",
      "xxxxx\n",
      ",\n",
      "636\n",
      "xxxxx\n",
      "60\n",
      "637\n",
      "xxxxx\n",
      "-\n",
      "638\n",
      "xxxxx\n",
      "63\n",
      "639\n",
      "xxxxx\n",
      ").\n",
      "640\n",
      "xxxxx\n",
      "Functional characterisation of mARi at early stationary phase, 8 h (\n",
      "641\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "642\n",
      "xxxxx\n",
      "), showed about 60-80% repression activity in both\n",
      "643\n",
      "xxxxx\n",
      "E. coli\n",
      "644\n",
      "xxxxx\n",
      "K-12 and B strains (\n",
      "645\n",
      "xxxxx\n",
      "Fig.\n",
      "646\n",
      "xxxxx\n",
      "), while the silencing activity in the B strains was higher than in the K-12 strains for comparable plasmid copy numbers. (\n",
      "647\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "648\n",
      "xxxxx\n",
      ").\n",
      "649\n",
      "xxxxx\n",
      "Fig.\n",
      "650\n",
      "xxxxx\n",
      "): the\n",
      "651\n",
      "xxxxx\n",
      "E. coli\n",
      "652\n",
      "xxxxx\n",
      "K-12 strain (DH10b cells) showed relatively stable repression activity in both mid-exponential and early stationary growth, whereas a slightly increased repression activity was observed in early stationary phase for the BL21(DE3) and BL21star(DE3) strains.\n",
      "653\n",
      "xxxxx\n",
      "Expanding a set of orthogonal mARi regulators through seed sequence modification\n",
      "654\n",
      "xxxxx\n",
      "We sought to create a set of mARi-based regulators based on its underlying modular design principles coupled with the modular UTR-RBS linkers used in BASIC to construct both expression cassettes and operons.\n",
      "655\n",
      "xxxxx\n",
      "35\n",
      "656\n",
      "xxxxx\n",
      "), whilst preserving the same medium strength RBS (RBSc).\n",
      "657\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "658\n",
      "xxxxx\n",
      "and high Hamming distance scores (\n",
      "659\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "660\n",
      "xxxxx\n",
      "between the set.\n",
      "661\n",
      "xxxxx\n",
      "30\n",
      "662\n",
      "xxxxx\n",
      ").\n",
      "663\n",
      "xxxxx\n",
      "Supplementary Table 4)\n",
      "664\n",
      "xxxxx\n",
      ".\n",
      "665\n",
      "xxxxx\n",
      "30\n",
      "666\n",
      "xxxxx\n",
      ").\n",
      "530\n",
      "xxxxx\n",
      "Figure 4 D-E\n",
      "531\n",
      "xxxxx\n",
      "), free YFP, or Fus3-YFP (\n",
      "532\n",
      "xxxxx\n",
      "Supplementary Figure 2 B\n",
      "533\n",
      "xxxxx\n",
      "), so it appears to be related to the specific mechanisms that cause Ace2 asymmetric localization to the daughter cell.\n",
      "534\n",
      "xxxxx\n",
      "Discussion\n",
      "535\n",
      "xxxxx\n",
      "Quantitative measurements of dynamic processes in single cells are essential for a system-level understanding of cellular function.\n",
      "536\n",
      "xxxxx\n",
      "We estimated nuclear transport rates in the order of 0.1 sec\n",
      "537\n",
      "xxxxx\n",
      "-1\n",
      "538\n",
      "xxxxx\n",
      ".\n",
      "539\n",
      "xxxxx\n",
      "Winey et al.,\n",
      "540\n",
      "xxxxx\n",
      "), 1.2 molecules of X are imported to the nucleus per second per pore.\n",
      "541\n",
      "xxxxx\n",
      "Nemergut and Macara, 2000\n",
      "542\n",
      "xxxxx\n",
      ";\n",
      "543\n",
      "xxxxx\n",
      "Ribbeck et al.,\n",
      "544\n",
      "xxxxx\n",
      ";\n",
      "545\n",
      "xxxxx\n",
      "Siebrasse and Peters, 2002\n",
      "546\n",
      "xxxxx\n",
      ";\n",
      "547\n",
      "xxxxx\n",
      "Timney et al.,\n",
      "548\n",
      "xxxxx\n",
      ").\n",
      "549\n",
      "xxxxx\n",
      "Translocation of the actively transported proteins we tested was faster than the passively diffusing YFP.\n",
      "550\n",
      "xxxxx\n",
      "Kiskin et al.,\n",
      "551\n",
      "xxxxx\n",
      ";\n",
      "552\n",
      "xxxxx\n",
      "Siebrasse and Peters, 2002\n",
      "553\n",
      "xxxxx\n",
      ") as well as by bulk import measurements in permeabilized cells (\n",
      "554\n",
      "xxxxx\n",
      "Frey et al.,\n",
      "555\n",
      "xxxxx\n",
      ";\n",
      "556\n",
      "xxxxx\n",
      "Ribbeck et al.,\n",
      "557\n",
      "xxxxx\n",
      ").\n",
      "558\n",
      "xxxxx\n",
      "van Drogen et al.,\n",
      "559\n",
      "xxxxx\n",
      ") that blocking active transport reduced, but did not eliminate, Fus3 nuclear transport.\n",
      "560\n",
      "xxxxx\n",
      "We found large cell-to-cell variation in transport rates for the three proteins studied.\n",
      "561\n",
      "xxxxx\n",
      "The actively transported proteins Fus3 and Ace2 had higher levels of correlated variation than YFP.\n",
      "562\n",
      "xxxxx\n",
      "Variability in the core transport machinery (leading to the correlated variability import and export rates we showed here) would only affect the shuttling speed of mobile proteins, without affecting their net subcellular localization (which is dictated by the ratio of import to export).\n",
      "563\n",
      "xxxxx\n",
      "According to our estimations, the effective volume in the yeast cytosol available for diffusion is 7 times larger than that volume in the nucleosol.\n",
      "564\n",
      "xxxxx\n",
      "Jorgensen et al.,\n",
      "565\n",
      "xxxxx\n",
      "), it follows that the cytoplasm is twice as occupied as the nucleus, perhaps due to the presence of organelles.\n",
      "566\n",
      "xxxxx\n",
      "Uchida et al.,\n",
      "567\n",
      "xxxxx\n",
      "), we estimated a ratio of 8 subtracting from the cytoplasm volume that of the vacuole, mitochondria, and lipid bodies.\n",
      "568\n",
      "xxxxx\n",
      "Webster et al.,\n",
      "569\n",
      "xxxxx\n",
      ").\n",
      "570\n",
      "xxxxx\n",
      "Cellular division involves the segregation of all its components between descendant cells.\n",
      "571\n",
      "xxxxx\n",
      "Huh and Paulsson, 2011\n",
      "572\n",
      "xxxxx\n",
      "), its counter-side, “cellular heritability”, would limit it (\n",
      "573\n",
      "xxxxx\n",
      "Vashistha et al.,\n",
      "574\n",
      "xxxxx\n",
      ").\n",
      "575\n",
      "xxxxx\n",
      "Kaufmann et al.,\n",
      "576\n",
      "xxxxx\n",
      ";\n",
      "577\n",
      "xxxxx\n",
      "Mura et al.,\n",
      "578\n",
      "xxxxx\n",
      ";\n",
      "579\n",
      "xxxxx\n",
      "Sigal et al.,\n",
      "580\n",
      "xxxxx\n",
      ";\n",
      "581\n",
      "xxxxx\n",
      "Spencer et al.,\n",
      "582\n",
      "xxxxx\n",
      ";\n",
      "583\n",
      "xxxxx\n",
      "Vashistha et al.,\n",
      "584\n",
      "xxxxx\n",
      ").\n",
      "585\n",
      "xxxxx\n",
      "Li et al.,\n",
      "586\n",
      "xxxxx\n",
      ";\n",
      "587\n",
      "xxxxx\n",
      "Suel et al.,\n",
      "588\n",
      "xxxxx\n",
      ").\n",
      "589\n",
      "xxxxx\n",
      "Although the ultimate mechanism that causes Ace2 asymmetric localization to the daughter cell nucleus is still unknown, our FRAP measurements provide notable insights.\n",
      "590\n",
      "xxxxx\n",
      "Altogether, this study sheds light into cell-to-cell variation in the kinetic rates of nuclear transport and its sources.\n",
      "591\n",
      "xxxxx\n",
      "Authors contributions\n",
      "592\n",
      "xxxxx\n",
      "LD performed most of the scFRAPs, constructed the strains and some of the plasmids used in the study, performed modeling and data analysis.\n",
      "593\n",
      "xxxxx\n",
      "Declaration of interests\n",
      "594\n",
      "xxxxx\n",
      "The authors declare no competing interests.\n",
      "595\n",
      "xxxxx\n",
      "STAR Methods\n",
      "596\n",
      "xxxxx\n",
      "RESOURCE AVAILABILITY\n",
      "597\n",
      "xxxxx\n",
      "Lead contact\n",
      "598\n",
      "xxxxx\n",
      "Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Alejandro Colman-Lerner (colman-lerner{at}fbmc.fcen.uba.ar).\n",
      "599\n",
      "xxxxx\n",
      "Materials availability\n",
      "600\n",
      "xxxxx\n",
      "Plasmids and yeast strains generated in this study are available upon request.\n",
      "601\n",
      "xxxxx\n",
      "Data and code availability\n",
      "602\n",
      "xxxxx\n",
      "Raw microscopy images reported in this paper will be shared by the lead contact upon request.\n",
      "603\n",
      "xxxxx\n",
      "The NuclearFRAP R package, has been deposited at Zenodo and is publicly available as of the date of publication.\n",
      "604\n",
      "xxxxx\n",
      "Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.\n",
      "605\n",
      "xxxxx\n",
      "EXPERIMENTAL MODEL AND SUBJECT DETAILS\n",
      "606\n",
      "xxxxx\n",
      "Saccharomyces cerevisiae strains and plasmids\n",
      "607\n",
      "xxxxx\n",
      "We performed general molecular biology procedures, yeast strain manipulation and construction according to previously established methods (Ausubel et al.;\n",
      "608\n",
      "xxxxx\n",
      "Guthrie and Fink, 1991\n",
      "609\n",
      "xxxxx\n",
      ")\n",
      "610\n",
      "xxxxx\n",
      "All strains generated and used in this study are listed in the key resources table.\n",
      "611\n",
      "xxxxx\n",
      "Tagging of Myo1 with mCherry was performed by homologous recombination by transforming the appropriate strains with a PCR fragment containing mCherry followed by a hygromycin resistance cassette, flanked by 40 nucleotides of homology with the end of the MYO1 ORF (except the STOP codon) on its 5’ end and the region of the 3’ UTR from +50 to +90.\n",
      "612\n",
      "xxxxx\n",
      "We selected transformants in plates with YPD plates with hygromycin and confirmed that integration occurred in the right location by visual determination of the presence of the red fluorescent ring at the bud-neck.\n",
      "613\n",
      "xxxxx\n",
      "All plasmids generated and used in this study are listed in the key resources table.\n",
      "614\n",
      "xxxxx\n",
      "Cells preparation for microscopy\n",
      "615\n",
      "xxxxx\n",
      "Yeast cells were prepared for the scFRAPs experiments by growing them at least two consecutive days on selective BSM-TRP LEU URA 2% glucose plates.\n",
      "616\n",
      "xxxxx\n",
      "METHOD DETAILS\n",
      "617\n",
      "xxxxx\n",
      "Imaging and scFRAPs assays\n",
      "618\n",
      "xxxxx\n",
      "The scFRAPs were performed in an Olimpus IX-81 inverted microscope with a FV1000 confocal module with an oil immersion Olympus UplanSapo 63X objective (numerical aperture, NA 1.35).\n",
      "619\n",
      "xxxxx\n",
      "Cells in the desired stage of the cell cycle were identified, then reference images of the transmission, CFP and mCherry channels (to observe Ace2-CFP and Myo1-mCherry) were acquired.\n",
      "620\n",
      "xxxxx\n",
      "Data Analysis and Fitting\n",
      "621\n",
      "xxxxx\n",
      "For a detailed description of the protocols for image acquisition, data correction and fitting, see the Appendix.\n",
      "622\n",
      "xxxxx\n",
      "Team, 2010\n",
      "623\n",
      "xxxxx\n",
      ") with our own R packages, Rcell (\n",
      "624\n",
      "xxxxx\n",
      "Bush et al.,\n",
      "625\n",
      "xxxxx\n",
      ") and NuclearFRAP.\n",
      "626\n",
      "xxxxx\n",
      "QUANTIFICATION AND STATISTICAL ANALYSIS\n",
      "627\n",
      "xxxxx\n",
      "Statistical testing\n",
      "628\n",
      "xxxxx\n",
      "The number of data points -individual cells or mother-daughter pairs of cells depending on the case-for each analysis can be found in the figure legend.\n",
      "629\n",
      "xxxxx\n",
      "All estimated parameters are presented as median ± S.E.M. When indicated, statistical significance was calculated using Student’s two-sample t-test (unpaired two samples for means) over the logarithms of the values (with base 10).\n",
      "630\n",
      "xxxxx\n",
      "Comparisons between variances were performed according to (\n",
      "631\n",
      "xxxxx\n",
      "Lewontin, 1966\n",
      "632\n",
      "xxxxx\n",
      ").\n",
      "633\n",
      "xxxxx\n",
      "The error bars for the correlation and SD in\n",
      "634\n",
      "xxxxx\n",
      "Figure 3\n",
      "635\n",
      "xxxxx\n",
      "were produced by bootstrapping 1000 times with reposition samples of 90% the size of the data for the experiment, estimating either the correlation or the SD according to the case, and then defining the interval that contained 95% of the estimated values.\n",
      "636\n",
      "xxxxx\n",
      ",\n",
      "667\n",
      "xxxxx\n",
      "31\n",
      "668\n",
      "xxxxx\n",
      ").\n",
      "669\n",
      "xxxxx\n",
      "E. coli\n",
      "670\n",
      "xxxxx\n",
      "genomes:\n",
      "671\n",
      "xxxxx\n",
      "E. coli\n",
      "672\n",
      "xxxxx\n",
      "MG1655, DH10b, and BL21(DE3)\n",
      "673\n",
      "xxxxx\n",
      "(Supplementary Table 4)\n",
      "674\n",
      "xxxxx\n",
      ".\n",
      "675\n",
      "xxxxx\n",
      "To experimentally validate target specificity, we constructed all possible combinations of mARi and target UTR upstream of\n",
      "676\n",
      "xxxxx\n",
      "sfgfp\n",
      "677\n",
      "xxxxx\n",
      "in each case maintaining the same promoter (\n",
      "678\n",
      "xxxxx\n",
      "Fig.\n",
      "679\n",
      "xxxxx\n",
      ").\n",
      "680\n",
      "xxxxx\n",
      "Fig.\n",
      "681\n",
      "xxxxx\n",
      ", Supplementary Fig.\n",
      "682\n",
      "xxxxx\n",
      "), confirming their specificity with reduced cross-reactivity between each pair.\n",
      "683\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "684\n",
      "xxxxx\n",
      ") may contribute to this, but it does not completely explain the effect with mARi-B, or other observations;\n",
      "685\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "686\n",
      "xxxxx\n",
      ").\n",
      "687\n",
      "xxxxx\n",
      "Modularity and orthogonality of mARi enables simultaneous and multiplexed gene regulation\n",
      "688\n",
      "xxxxx\n",
      "After demonstrating that mARi regulators exhibit specific interaction towards cognate UTRs in the context of a single gene, we evaluated their ability to operate in a multiplexed reporter gene design.\n",
      "689\n",
      "xxxxx\n",
      "sfgfp\n",
      "690\n",
      "xxxxx\n",
      "and\n",
      "691\n",
      "xxxxx\n",
      "mcherry\n",
      "692\n",
      "xxxxx\n",
      ") organised as separate transcriptional units (\n",
      "693\n",
      "xxxxx\n",
      "Fig.\n",
      "694\n",
      "xxxxx\n",
      ").\n",
      "695\n",
      "xxxxx\n",
      "sfgfp\n",
      "696\n",
      "xxxxx\n",
      "being selectively targeted by mARi-A and UTR-B-\n",
      "697\n",
      "xxxxx\n",
      "mcherry\n",
      "698\n",
      "xxxxx\n",
      "being targeted by mARi-B (\n",
      "699\n",
      "xxxxx\n",
      "Fig.\n",
      "700\n",
      "xxxxx\n",
      "and\n",
      "701\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "702\n",
      "xxxxx\n",
      ").\n",
      "703\n",
      "xxxxx\n",
      "sfgfp, mcherry\n",
      "704\n",
      "xxxxx\n",
      ", and\n",
      "705\n",
      "xxxxx\n",
      "mtagbfp\n",
      "706\n",
      "xxxxx\n",
      "genes, respectively (\n",
      "707\n",
      "xxxxx\n",
      "Fig.\n",
      "708\n",
      "xxxxx\n",
      "and\n",
      "709\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "710\n",
      "xxxxx\n",
      ").\n",
      "711\n",
      "xxxxx\n",
      "Two sets of mARi regulators were used to modulate gene expression in a two-gene operon with UTR-A-RBSc driving the expression of\n",
      "712\n",
      "xxxxx\n",
      "sfgfp\n",
      "713\n",
      "xxxxx\n",
      ", upstream of UTR-B-RBSc driving the expression of\n",
      "714\n",
      "xxxxx\n",
      "mcherry\n",
      "715\n",
      "xxxxx\n",
      ", constitutively expressed by the promoter P\n",
      "716\n",
      "xxxxx\n",
      "J23101_BASIC\n",
      "717\n",
      "xxxxx\n",
      "(\n",
      "718\n",
      "xxxxx\n",
      "Fig.\n",
      "719\n",
      "xxxxx\n",
      ").\n",
      "720\n",
      "xxxxx\n",
      "sfgfp\n",
      "721\n",
      "xxxxx\n",
      ") is strongly affected by its cognate mARi-A, with only a minor affect from mARi-B, which targets the downstream\n",
      "722\n",
      "xxxxx\n",
      "mcherry\n",
      "723\n",
      "xxxxx\n",
      ";\n",
      "724\n",
      "xxxxx\n",
      "mcherry\n",
      "725\n",
      "xxxxx\n",
      "is most strongly attenuated by its cognate mARi-B, but it is also significantly attenuated by repression of the upstream\n",
      "726\n",
      "xxxxx\n",
      "sfgfp\n",
      "727\n",
      "xxxxx\n",
      "with full attenuation only reached with both mARi-A and B (\n",
      "728\n",
      "xxxxx\n",
      "Fig.\n",
      "729\n",
      "xxxxx\n",
      "and\n",
      "730\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "731\n",
      "xxxxx\n",
      ").\n",
      "732\n",
      "xxxxx\n",
      "DISCUSSION\n",
      "733\n",
      "xxxxx\n",
      "Here we report the development, characterisation, and implementation of a modular post-transcriptional regulation system based on\n",
      "734\n",
      "xxxxx\n",
      "trans\n",
      "735\n",
      "xxxxx\n",
      "-encoded sRNAs.\n",
      "736\n",
      "xxxxx\n",
      "17\n",
      "737\n",
      "xxxxx\n",
      ",\n",
      "738\n",
      "xxxxx\n",
      "27\n",
      "739\n",
      "xxxxx\n",
      ").\n",
      "740\n",
      "xxxxx\n",
      "59\n",
      "741\n",
      "xxxxx\n",
      ") and the intracellular concentration of available Hfq can adapt rapidly to cellular changes in the RNA pool by facilitated recycling of the protein (\n",
      "742\n",
      "xxxxx\n",
      "64\n",
      "743\n",
      "xxxxx\n",
      ").\n",
      "744\n",
      "xxxxx\n",
      "E. coli\n",
      "745\n",
      "xxxxx\n",
      "(\n",
      "746\n",
      "xxxxx\n",
      "29\n",
      "747\n",
      "xxxxx\n",
      ").\n",
      "748\n",
      "xxxxx\n",
      "Modification of the seed sequence, whilst retaining the sRNA scaffold, has enabled the reprogramming of natural sRNAs with different targets of interests that do not exist in nature (\n",
      "749\n",
      "xxxxx\n",
      "17\n",
      "750\n",
      "xxxxx\n",
      ",\n",
      "751\n",
      "xxxxx\n",
      "18\n",
      "752\n",
      "xxxxx\n",
      ",\n",
      "753\n",
      "xxxxx\n",
      "28\n",
      "754\n",
      "xxxxx\n",
      ").\n",
      "755\n",
      "xxxxx\n",
      "Fig.\n",
      "756\n",
      "xxxxx\n",
      "and\n",
      "757\n",
      "xxxxx\n",
      "4\n",
      "758\n",
      "xxxxx\n",
      ").\n",
      "759\n",
      "xxxxx\n",
      "E. coli\n",
      "760\n",
      "xxxxx\n",
      "host (\n",
      "761\n",
      "xxxxx\n",
      "35\n",
      "762\n",
      "xxxxx\n",
      "Estimation of correlated and un-correlated variability\n",
      "637\n",
      "xxxxx\n",
      "For the analysis in\n",
      "638\n",
      "xxxxx\n",
      "Figure 3\n",
      "639\n",
      "xxxxx\n",
      ", where the cell-to-cell variability in transport rates was decomposed into correlated and un-correlated, the following estimators were used (after (\n",
      "640\n",
      "xxxxx\n",
      "Qiuyan Fua and Pachter, 2016\n",
      "641\n",
      "xxxxx\n",
      ")).\n",
      "642\n",
      "xxxxx\n",
      "Data inclusion/ exclusion decision\n",
      "643\n",
      "xxxxx\n",
      "Prior to analysis and fitting, the acquired images were manually curated in order to discard experiments that had technical issues, such as cells movement or focus drift.\n",
      "644\n",
      "xxxxx\n",
      "Acknowledgements\n",
      "645\n",
      "xxxxx\n",
      "We are grateful to Peter Pryciak and Alejandra Ventura for feedback and advise.\n",
      "646\n",
      "xxxxx\n",
      "Footnotes\n",
      "647\n",
      "xxxxx\n",
      "This is completely revised version.\n",
      "648\n",
      "xxxxx\n",
      "References\n",
      "649\n",
      "xxxxx\n",
      "New York, N.Y.:\n",
      "650\n",
      "xxxxx\n",
      "↵\n",
      "651\n",
      "xxxxx\n",
      "Axelrod,  D., Koppel,  D.E., Schlessinger,  J., Elson,  E., and Webb,  W.W. (1976).\n",
      "652\n",
      "xxxxx\n",
      "Biophysical Journal\n",
      "653\n",
      "xxxxx\n",
      "16, 1055-1069.\n",
      "654\n",
      "xxxxx\n",
      "OpenUrl\n",
      "655\n",
      "xxxxx\n",
      "CrossRef\n",
      "656\n",
      "xxxxx\n",
      "PubMed\n",
      "657\n",
      "xxxxx\n",
      "Web of Science\n",
      "658\n",
      "xxxxx\n",
      "↵\n",
      "659\n",
      "xxxxx\n",
      "Blake,  W.J., M, K.A., Cantor,  C.R., and Collins,  J.J. (2003).\n",
      "660\n",
      "xxxxx\n",
      "Nature\n",
      "661\n",
      "xxxxx\n",
      "422, 633-637.\n",
      "662\n",
      "xxxxx\n",
      "OpenUrl\n",
      "663\n",
      "xxxxx\n",
      "CrossRef\n",
      "664\n",
      "xxxxx\n",
      "PubMed\n",
      "665\n",
      "xxxxx\n",
      "Web of Science\n",
      "666\n",
      "xxxxx\n",
      "↵\n",
      "667\n",
      "xxxxx\n",
      "Bobola,  N., Jansen,  R.P., Shin,  T.H., and Nasmyth,  K. (1996).\n",
      "668\n",
      "xxxxx\n",
      "Cell\n",
      "669\n",
      "xxxxx\n",
      "84, 699-709.\n",
      "670\n",
      "xxxxx\n",
      "OpenUrl\n",
      "671\n",
      "xxxxx\n",
      "CrossRef\n",
      "672\n",
      "xxxxx\n",
      "PubMed\n",
      "673\n",
      "xxxxx\n",
      "Web of Science\n",
      "674\n",
      "xxxxx\n",
      "↵\n",
      "675\n",
      "xxxxx\n",
      "Boettcher,  B., and Barral,  Y. (2014).\n",
      "676\n",
      "xxxxx\n",
      "Nucleus\n",
      "677\n",
      "xxxxx\n",
      "4, 160-165.\n",
      "678\n",
      "xxxxx\n",
      "OpenUrl\n",
      "679\n",
      "xxxxx\n",
      "↵\n",
      "680\n",
      "xxxxx\n",
      "Boettcher,  B., Marquez-Lago,  T.T., Bayer,  M., Weiss,  E.L., and Barral,  Y. (2012).\n",
      "681\n",
      "xxxxx\n",
      "Journal of Cell Biology\n",
      "682\n",
      "xxxxx\n",
      "197, 921-937.\n",
      "683\n",
      "xxxxx\n",
      "OpenUrl\n",
      "684\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "685\n",
      "xxxxx\n",
      "↵\n",
      "686\n",
      "xxxxx\n",
      "Bourens,  M., Racki,  W., Bécam,  A.-M.M., Panozzo,  C., Boulon,  S., Bertrand,  E., and Herbert,  C.J. (2008).\n",
      "687\n",
      "xxxxx\n",
      "Biology of the cell / under the auspices of the European Cell Biology Organization\n",
      "688\n",
      "xxxxx\n",
      "100, 343-354.\n",
      "689\n",
      "xxxxx\n",
      "OpenUrl\n",
      "690\n",
      "xxxxx\n",
      "↵\n",
      "691\n",
      "xxxxx\n",
      "Brace,  J., Hsu,  J., and Weiss,  E. (2011).\n",
      "692\n",
      "xxxxx\n",
      "Mol Cell Biol\n",
      "693\n",
      "xxxxx\n",
      "31, 721-735.\n",
      "694\n",
      "xxxxx\n",
      "OpenUrl\n",
      "695\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "696\n",
      "xxxxx\n",
      "↵\n",
      "697\n",
      "xxxxx\n",
      "Bush,  A., Chernomoretz,  A., Yu,  R., Gordon,  A., and Colman-Lerner,  A. (2012).\n",
      "698\n",
      "xxxxx\n",
      "Curr Protoc Mol Biol\n",
      "699\n",
      "xxxxx\n",
      "Chapter 14, Unit 14 18.\n",
      "700\n",
      "xxxxx\n",
      "↵\n",
      "701\n",
      "xxxxx\n",
      "Cavalier-Smith,  T. (2010).\n",
      "702\n",
      "xxxxx\n",
      "Biology Direct\n",
      "703\n",
      "xxxxx\n",
      "5, 7.\n",
      "704\n",
      "xxxxx\n",
      "OpenUrl\n",
      "705\n",
      "xxxxx\n",
      "↵\n",
      "706\n",
      "xxxxx\n",
      "Cole,  N.B., Sciaky,  N., Marotta,  A., Song,  J., and Lippincott-Schwartz,  J. (1996a).\n",
      "707\n",
      "xxxxx\n",
      "MolBiolCell\n",
      "708\n",
      "xxxxx\n",
      "7, 631-650.\n",
      "709\n",
      "xxxxx\n",
      "OpenUrl\n",
      "710\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "711\n",
      "xxxxx\n",
      "↵\n",
      "712\n",
      "xxxxx\n",
      "Cole,  N.B., Smith,  C.L., Sciaky,  N., Terasaki,  M., Edidin,  M., and Lippincott-Schwartz,  J. (1996b).\n",
      "713\n",
      "xxxxx\n",
      "Science\n",
      "714\n",
      "xxxxx\n",
      "273, 797-801.\n",
      "715\n",
      "xxxxx\n",
      "OpenUrl\n",
      "716\n",
      "xxxxx\n",
      "Abstract\n",
      "717\n",
      "xxxxx\n",
      "↵\n",
      "718\n",
      "xxxxx\n",
      "Colman-Lerner,  A., Chin,  T.E., and Brent,  R. (2001).\n",
      "719\n",
      "xxxxx\n",
      "Cell\n",
      "720\n",
      "xxxxx\n",
      "107, 739-750.\n",
      "721\n",
      "xxxxx\n",
      "OpenUrl\n",
      "722\n",
      "xxxxx\n",
      "CrossRef\n",
      "723\n",
      "xxxxx\n",
      "PubMed\n",
      "724\n",
      "xxxxx\n",
      "Web of Science\n",
      "725\n",
      "xxxxx\n",
      "↵\n",
      "726\n",
      "xxxxx\n",
      "Colman-Lerner,  A., Gordon,  A., Serra,  E., Chin,  T., Resnekov,  O., Endy,  D., Pesce,  C.G., and Brent,  R. (2005).\n",
      "727\n",
      "xxxxx\n",
      "Nature\n",
      "728\n",
      "xxxxx\n",
      "437, 699-706.\n",
      "729\n",
      "xxxxx\n",
      "OpenUrl\n",
      "730\n",
      "xxxxx\n",
      "CrossRef\n",
      "731\n",
      "xxxxx\n",
      "PubMed\n",
      "732\n",
      "xxxxx\n",
      "Web of Science\n",
      "733\n",
      "xxxxx\n",
      "↵\n",
      "734\n",
      "xxxxx\n",
      "Dohlman,  H.G., and Thorner,  J.W. (2001).\n",
      "735\n",
      "xxxxx\n",
      "AnnuRevBiochem\n",
      "736\n",
      "xxxxx\n",
      "70, 703-754.\n",
      "737\n",
      "xxxxx\n",
      "OpenUrl\n",
      "738\n",
      "xxxxx\n",
      "CrossRef\n",
      "739\n",
      "xxxxx\n",
      "PubMed\n",
      "740\n",
      "xxxxx\n",
      "Web of Science\n",
      "741\n",
      "xxxxx\n",
      "↵\n",
      "742\n",
      "xxxxx\n",
      "Dohrmann,  P.R., Butler,  G., Tamai,  K., Dorland,  S., Greene,  J.R., Thiele,  D.J., and Stillman,  D.J. (1992).\n",
      "743\n",
      "xxxxx\n",
      ").\n",
      "763\n",
      "xxxxx\n",
      "35\n",
      "764\n",
      "xxxxx\n",
      "The sequence composition of seed/target sites can be refined to optimise their binding affinity.\n",
      "765\n",
      "xxxxx\n",
      "The mARi-based repression of a polycistronic mRNA provides some insight into its mechanism of action.\n",
      "766\n",
      "xxxxx\n",
      "4d\n",
      "767\n",
      "xxxxx\n",
      ").\n",
      "768\n",
      "xxxxx\n",
      "mcherry\n",
      "769\n",
      "xxxxx\n",
      "gene were also additive for both the gene’s cognate mARi and the upstream gene’s mARi, suggesting that repression at each UTR-RBS position was independent.\n",
      "770\n",
      "xxxxx\n",
      "The mechanism of action being based on the inhibition of translation initiation rather than active degradation may also be due to the designed mARi regulators and UTR-RBS linkers omitting an RNase E binding site, which is essential for RNase E-dependent cleavage.\n",
      "771\n",
      "xxxxx\n",
      "E. coli\n",
      "772\n",
      "xxxxx\n",
      "strains, with a response that is independent of the host genotype.\n",
      "773\n",
      "xxxxx\n",
      "E. coli\n",
      "774\n",
      "xxxxx\n",
      "strains and growth phases, demonstrating both its portability and robustness, notably with minimal difference between BL21(DE3) and BL21star(DE3), the latter being deficient in RNase E (\n",
      "775\n",
      "xxxxx\n",
      "Fig.\n",
      "776\n",
      "xxxxx\n",
      ").\n",
      "777\n",
      "xxxxx\n",
      "The modular nature of the design framework meant facile variation of genetic design parameters.\n",
      "778\n",
      "xxxxx\n",
      "30\n",
      "779\n",
      "xxxxx\n",
      "), where repression increased as a function of length and hence free binding energy (\n",
      "780\n",
      "xxxxx\n",
      "Supplementary Fig.\n",
      "781\n",
      "xxxxx\n",
      ").\n",
      "782\n",
      "xxxxx\n",
      "Fig.\n",
      "783\n",
      "xxxxx\n",
      ").\n",
      "784\n",
      "xxxxx\n",
      "trans\n",
      "785\n",
      "xxxxx\n",
      "expression of mARi from a second plasmid, the repression ratio was always worse, even when it was encoded on a higher plasmid copy number.\n",
      "786\n",
      "xxxxx\n",
      "29\n",
      "787\n",
      "xxxxx\n",
      ").\n",
      "788\n",
      "xxxxx\n",
      "trans\n",
      "789\n",
      "xxxxx\n",
      "-encoded systems, but it is likely to originate from enhanced proximity and hence binding between the mARi and its target.\n",
      "790\n",
      "xxxxx\n",
      "The modular nature of the mARi design enabled control of different genes and facilitated multiplexing of target genes.\n",
      "791\n",
      "xxxxx\n",
      "Fig.\n",
      "792\n",
      "xxxxx\n",
      ").\n",
      "793\n",
      "xxxxx\n",
      "trans\n",
      "794\n",
      "xxxxx\n",
      "-encoded small RNAs (\n",
      "795\n",
      "xxxxx\n",
      "7\n",
      "796\n",
      "xxxxx\n",
      ",\n",
      "797\n",
      "xxxxx\n",
      "24\n",
      "798\n",
      "xxxxx\n",
      "-\n",
      "799\n",
      "xxxxx\n",
      "26\n",
      "800\n",
      "xxxxx\n",
      ",\n",
      "801\n",
      "xxxxx\n",
      "28\n",
      "802\n",
      "xxxxx\n",
      ");\n",
      "803\n",
      "xxxxx\n",
      "33\n",
      "804\n",
      "xxxxx\n",
      ").\n",
      "805\n",
      "xxxxx\n",
      "Author Contributions\n",
      "806\n",
      "xxxxx\n",
      "A.D.:\n",
      "807\n",
      "xxxxx\n",
      "M.S.:\n",
      "808\n",
      "xxxxx\n",
      "G.-B.S.:\n",
      "809\n",
      "xxxxx\n",
      "G.S.B.:\n",
      "810\n",
      "xxxxx\n",
      "Competing interests\n",
      "811\n",
      "xxxxx\n",
      "The authors declare no competing interests.\n",
      "812\n",
      "xxxxx\n",
      "Acknowledgements\n",
      "813\n",
      "xxxxx\n",
      "A.D. received a PhD scholarship from Indonesia Endowment Fund for Education (LPDP).\n",
      "814\n",
      "xxxxx\n",
      "References\n",
      "815\n",
      "xxxxx\n",
      "1.\n",
      "816\n",
      "xxxxx\n",
      "↵\n",
      "817\n",
      "xxxxx\n",
      "Cameron,  D.E., Bashor,  C.J. and Collins,  J.J. (2014) A brief history of synthetic biology.\n",
      "818\n",
      "xxxxx\n",
      "Nat.\n",
      "819\n",
      "xxxxx\n",
      ".,\n",
      "820\n",
      "xxxxx\n",
      "OpenUrl\n",
      "821\n",
      "xxxxx\n",
      "CrossRef\n",
      "822\n",
      "xxxxx\n",
      "PubMed\n",
      "823\n",
      "xxxxx\n",
      "2.\n",
      "824\n",
      "xxxxx\n",
      "Purnick,  P.E.M. and Weiss,  R. (2009) The second wave of synthetic biology:\n",
      "825\n",
      "xxxxx\n",
      "Nat.\n",
      "826\n",
      "xxxxx\n",
      ".,\n",
      "827\n",
      "xxxxx\n",
      "OpenUrl\n",
      "828\n",
      "xxxxx\n",
      "CrossRef\n",
      "829\n",
      "xxxxx\n",
      "PubMed\n",
      "830\n",
      "xxxxx\n",
      "Web of Science\n",
      "831\n",
      "xxxxx\n",
      "3.\n",
      "832\n",
      "xxxxx\n",
      "↵\n",
      "833\n",
      "xxxxx\n",
      "Slusarczyk,  A.L., Lin,  A. and Weiss,  R. (2012) Foundations for the design and implementation of synthetic genetic circuits.\n",
      "834\n",
      "xxxxx\n",
      "Nat.\n",
      "835\n",
      "xxxxx\n",
      ".,\n",
      "836\n",
      "xxxxx\n",
      "OpenUrl\n",
      "837\n",
      "xxxxx\n",
      "CrossRef\n",
      "838\n",
      "xxxxx\n",
      "PubMed\n",
      "839\n",
      "xxxxx\n",
      "4.\n",
      "840\n",
      "xxxxx\n",
      "↵\n",
      "841\n",
      "xxxxx\n",
      "Kelwick,  R., MacDonald,  J.T., Webb,  A.J. and Freemont,  P. (2014) Developments in the Tools and Methodologies of Synthetic Biology.\n",
      "842\n",
      "xxxxx\n",
      "Front.\n",
      "843\n",
      "xxxxx\n",
      ".,\n",
      "844\n",
      "xxxxx\n",
      "OpenUrl\n",
      "845\n",
      "xxxxx\n",
      "5.\n",
      "846\n",
      "xxxxx\n",
      "↵\n",
      "847\n",
      "xxxxx\n",
      "Xiang,  Y., Dalchau,  N. and Wang,  B. (2018) Scaling up genetic circuit design for cellular computing:\n",
      "848\n",
      "xxxxx\n",
      "Nat.\n",
      "849\n",
      "xxxxx\n",
      ".,\n",
      "850\n",
      "xxxxx\n",
      "OpenUrl\n",
      "851\n",
      "xxxxx\n",
      "CrossRef\n",
      "852\n",
      "xxxxx\n",
      "6.\n",
      "853\n",
      "xxxxx\n",
      "↵\n",
      "854\n",
      "xxxxx\n",
      "Storz,  G., Vogel,  J. and Wassarman,  K.M. (2011) Regulation by Small RNAs in Bacteria:\n",
      "855\n",
      "xxxxx\n",
      "Mol.\n",
      "856\n",
      "xxxxx\n",
      ", 43, 880-891.\n",
      "857\n",
      "xxxxx\n",
      "OpenUrl\n",
      "858\n",
      "xxxxx\n",
      "CrossRef\n",
      "859\n",
      "xxxxx\n",
      "Genes Dev\n",
      "744\n",
      "xxxxx\n",
      "6, 93-104.\n",
      "745\n",
      "xxxxx\n",
      "OpenUrl\n",
      "746\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "747\n",
      "xxxxx\n",
      "↵\n",
      "748\n",
      "xxxxx\n",
      "Elowitz,  M.B., Levine,  A.J., Siggia,  E.D., and Swain,  P.S. (2002).\n",
      "749\n",
      "xxxxx\n",
      "Science\n",
      "750\n",
      "xxxxx\n",
      "297, 1183-1186.\n",
      "751\n",
      "xxxxx\n",
      "OpenUrl\n",
      "752\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "753\n",
      "xxxxx\n",
      "↵\n",
      "754\n",
      "xxxxx\n",
      "Frey,  S., Rees,  R., Schünemann,  J., Ng,  S.C., Fünfgeld,  K., Huyton,  T., and Görlich,  D. (2018).\n",
      "755\n",
      "xxxxx\n",
      "Cell\n",
      "756\n",
      "xxxxx\n",
      "174, 202-217.e209.\n",
      "757\n",
      "xxxxx\n",
      "OpenUrl\n",
      "758\n",
      "xxxxx\n",
      "CrossRef\n",
      "759\n",
      "xxxxx\n",
      "PubMed\n",
      "760\n",
      "xxxxx\n",
      "↵\n",
      "761\n",
      "xxxxx\n",
      "Gándara,  L., Durrieu,  L., Behrensen,  C., and Wappner,  P. (2019).\n",
      "762\n",
      "xxxxx\n",
      "Scientific Reports\n",
      "763\n",
      "xxxxx\n",
      "9, 1-12.\n",
      "764\n",
      "xxxxx\n",
      "OpenUrl\n",
      "765\n",
      "xxxxx\n",
      "↵\n",
      "766\n",
      "xxxxx\n",
      "Goodwin,  J.S., and Kenworthy,  A.K. (2005).\n",
      "767\n",
      "xxxxx\n",
      "Methods\n",
      "768\n",
      "xxxxx\n",
      "37, 154-164.\n",
      "769\n",
      "xxxxx\n",
      "OpenUrl\n",
      "770\n",
      "xxxxx\n",
      "CrossRef\n",
      "771\n",
      "xxxxx\n",
      "PubMed\n",
      "772\n",
      "xxxxx\n",
      "Web of Science\n",
      "773\n",
      "xxxxx\n",
      "↵\n",
      "774\n",
      "xxxxx\n",
      "Gordon,  A., Colman-Lerner,  A., Chin,  T.E., Benjamin,  K.R., Yu,  R.C., and Brent,  R. (2007).\n",
      "775\n",
      "xxxxx\n",
      "Nat Methods\n",
      "776\n",
      "xxxxx\n",
      "4, 175-181.\n",
      "777\n",
      "xxxxx\n",
      "OpenUrl\n",
      "778\n",
      "xxxxx\n",
      "CrossRef\n",
      "779\n",
      "xxxxx\n",
      "PubMed\n",
      "780\n",
      "xxxxx\n",
      "Web of Science\n",
      "781\n",
      "xxxxx\n",
      "↵\n",
      "782\n",
      "xxxxx\n",
      "Guthrie,  C., and Fink,  G.R. (1991).\n",
      "783\n",
      "xxxxx\n",
      "↵\n",
      "784\n",
      "xxxxx\n",
      "Herrero,  E., Stinus,  S., Bellows,  E., Berry,  L., Wood,  H., and Thorpe,  P.H. (2020).\n",
      "785\n",
      "xxxxx\n",
      "Asymmetric transcription factor partitioning during yeast cell division requires the FACT chromatin remodeler and cell cycle progression\n",
      "786\n",
      "xxxxx\n",
      ".\n",
      "787\n",
      "xxxxx\n",
      "↵\n",
      "788\n",
      "xxxxx\n",
      "Huh,  D., and Paulsson,  J. (2011).\n",
      "789\n",
      "xxxxx\n",
      "Nature Genetics\n",
      "790\n",
      "xxxxx\n",
      "43, 95-100.\n",
      "791\n",
      "xxxxx\n",
      "OpenUrl\n",
      "792\n",
      "xxxxx\n",
      "CrossRef\n",
      "793\n",
      "xxxxx\n",
      "PubMed\n",
      "794\n",
      "xxxxx\n",
      "Web of Science\n",
      "795\n",
      "xxxxx\n",
      "↵\n",
      "796\n",
      "xxxxx\n",
      "Huh,  W.K., Falvo,  J.V., Gerke,  L.C., Carroll,  A.S., Howson,  R.W., Weissman,  J.S., and O’Shea,  E.K. (2003).\n",
      "797\n",
      "xxxxx\n",
      "Nature\n",
      "798\n",
      "xxxxx\n",
      "425, 686-691.\n",
      "799\n",
      "xxxxx\n",
      "OpenUrl\n",
      "800\n",
      "xxxxx\n",
      "CrossRef\n",
      "801\n",
      "xxxxx\n",
      "PubMed\n",
      "802\n",
      "xxxxx\n",
      "Web of Science\n",
      "803\n",
      "xxxxx\n",
      "↵\n",
      "804\n",
      "xxxxx\n",
      "Jensen,  T.H., Neville,  M., Legrain,  P., and Rosbash,  M. (2000).\n",
      "805\n",
      "xxxxx\n",
      "Society\n",
      "806\n",
      "xxxxx\n",
      "20, 8047-8058.\n",
      "807\n",
      "xxxxx\n",
      "OpenUrl\n",
      "808\n",
      "xxxxx\n",
      "↵\n",
      "809\n",
      "xxxxx\n",
      "Jorgensen,  P., Edgington,  N.P., Schneider,  B.L., Rupes,  I., Tyers,  M., and Futcher,  B. (2007).\n",
      "810\n",
      "xxxxx\n",
      "Molecular Biology of the Cell\n",
      "811\n",
      "xxxxx\n",
      "18, 3523-3532.\n",
      "812\n",
      "xxxxx\n",
      "OpenUrl\n",
      "813\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "814\n",
      "xxxxx\n",
      "↵\n",
      "815\n",
      "xxxxx\n",
      "Kaufmann,  B.B., Yang,  Q., Mettetal,  J.T., and van Oudenaarden,  A. (2007).\n",
      "816\n",
      "xxxxx\n",
      "PLoS Biol\n",
      "817\n",
      "xxxxx\n",
      "5, e239.\n",
      "818\n",
      "xxxxx\n",
      "OpenUrl\n",
      "819\n",
      "xxxxx\n",
      "CrossRef\n",
      "820\n",
      "xxxxx\n",
      "PubMed\n",
      "821\n",
      "xxxxx\n",
      "↵\n",
      "822\n",
      "xxxxx\n",
      "Khmelinskii,  A., Keller,  P.J., Lorenz,  H., Schiebel,  E., and Knop,  M. (2010).\n",
      "823\n",
      "xxxxx\n",
      "Nature\n",
      "824\n",
      "xxxxx\n",
      "466, E1.\n",
      "825\n",
      "xxxxx\n",
      "OpenUrl\n",
      "826\n",
      "xxxxx\n",
      "CrossRef\n",
      "827\n",
      "xxxxx\n",
      "PubMed\n",
      "828\n",
      "xxxxx\n",
      "↵\n",
      "829\n",
      "xxxxx\n",
      "Kiskin,  N.I., Siebrasse,  J.P., and Peters,  R. (2003).\n",
      "830\n",
      "xxxxx\n",
      "Biophysical Journal\n",
      "831\n",
      "xxxxx\n",
      "85, 2311-2322.\n",
      "832\n",
      "xxxxx\n",
      "OpenUrl\n",
      "833\n",
      "xxxxx\n",
      "CrossRef\n",
      "834\n",
      "xxxxx\n",
      "PubMed\n",
      "835\n",
      "xxxxx\n",
      "↵\n",
      "836\n",
      "xxxxx\n",
      "On the measure of relative variability.\n",
      "837\n",
      "xxxxx\n",
      "Systematic Zoology\n",
      "838\n",
      "xxxxx\n",
      "15, 141-142.\n",
      "839\n",
      "xxxxx\n",
      "OpenUrl\n",
      "840\n",
      "xxxxx\n",
      "CrossRef\n",
      "841\n",
      "xxxxx\n",
      "↵\n",
      "842\n",
      "xxxxx\n",
      "Li,  Y., Shan,  Y., Desai,  R.V., Cox,  K.H., Weinberger,  L.S., and Takahashi,  J.S. (2020).\n",
      "843\n",
      "xxxxx\n",
      "Proceedings of the National Academy of Sciences of the United States of America\n",
      "844\n",
      "xxxxx\n",
      "117, 10350-10356.\n",
      "845\n",
      "xxxxx\n",
      "OpenUrl\n",
      "846\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "847\n",
      "xxxxx\n",
      "↵\n",
      "848\n",
      "xxxxx\n",
      "Lippincott,  J., and Li,  R. (1998).\n",
      "849\n",
      "xxxxx\n",
      "JCell Biol\n",
      "850\n",
      "xxxxx\n",
      "PubMed\n",
      "860\n",
      "xxxxx\n",
      "Web of Science\n",
      "861\n",
      "xxxxx\n",
      "7.\n",
      "862\n",
      "xxxxx\n",
      "↵\n",
      "863\n",
      "xxxxx\n",
      "Huang,  H.-H., Qian,  Y. and Del Vecchio,  D. (2018) A quasi-integral controller for adaptation of genetic modules to variable ribosome demand.\n",
      "864\n",
      "xxxxx\n",
      "Nat.\n",
      "865\n",
      "xxxxx\n",
      ".,\n",
      "866\n",
      "xxxxx\n",
      "OpenUrl\n",
      "867\n",
      "xxxxx\n",
      "CrossRef\n",
      "868\n",
      "xxxxx\n",
      "8.\n",
      "869\n",
      "xxxxx\n",
      "Thompson,  K.M., Rhodius,  V.A. and Gottesman,  S. (2007) σE regulates and is regulated by a small RNA in Escherichia coli.\n",
      "870\n",
      "xxxxx\n",
      "J. Bacteriol\n",
      "871\n",
      "xxxxx\n",
      ".,\n",
      "872\n",
      "xxxxx\n",
      "OpenUrl\n",
      "873\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "874\n",
      "xxxxx\n",
      "9.\n",
      "875\n",
      "xxxxx\n",
      "Tu,  K.C., Long,  T., Svenningsen,  S.L., Wingreen,  N.S. and Bassler,  B.L. (2010) Negative Feedback Loops Involving Small Regulatory RNAs Precisely Control the Vibrio harveyi Quorum-Sensing Response.\n",
      "876\n",
      "xxxxx\n",
      "Mol.\n",
      "877\n",
      "xxxxx\n",
      ", 37, 567-579.\n",
      "878\n",
      "xxxxx\n",
      "OpenUrl\n",
      "879\n",
      "xxxxx\n",
      "CrossRef\n",
      "880\n",
      "xxxxx\n",
      "PubMed\n",
      "881\n",
      "xxxxx\n",
      "Web of Science\n",
      "882\n",
      "xxxxx\n",
      "10.\n",
      "883\n",
      "xxxxx\n",
      "↵\n",
      "884\n",
      "xxxxx\n",
      "Brosse,  A., Korobeinikova,  A., Gottesman,  S. and Guillier,  M. (2016) Unexpected properties of sRNA promoters allow feedback control via regulation of a two-component system.\n",
      "885\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "886\n",
      "xxxxx\n",
      ".,\n",
      "887\n",
      "xxxxx\n",
      "OpenUrl\n",
      "888\n",
      "xxxxx\n",
      "CrossRef\n",
      "889\n",
      "xxxxx\n",
      "PubMed\n",
      "890\n",
      "xxxxx\n",
      "11.\n",
      "891\n",
      "xxxxx\n",
      "↵\n",
      "892\n",
      "xxxxx\n",
      "Nitzan,  M., Rehani,  R. and Margalit,  H. (2017) Integration of Bacterial Small RNAs in Regulatory Networks.\n",
      "893\n",
      "xxxxx\n",
      "Annu.\n",
      "894\n",
      "xxxxx\n",
      ".,\n",
      "895\n",
      "xxxxx\n",
      "OpenUrl\n",
      "896\n",
      "xxxxx\n",
      "CrossRef\n",
      "897\n",
      "xxxxx\n",
      "12.\n",
      "898\n",
      "xxxxx\n",
      "↵\n",
      "899\n",
      "xxxxx\n",
      "Beisel,  C.L. and Storz,  G. (2010) Base pairing small RNAs and their roles in global regulatory networks.\n",
      "900\n",
      "xxxxx\n",
      "FEMS Microbiol.\n",
      "901\n",
      "xxxxx\n",
      ".,\n",
      "902\n",
      "xxxxx\n",
      "OpenUrl\n",
      "903\n",
      "xxxxx\n",
      "CrossRef\n",
      "904\n",
      "xxxxx\n",
      "PubMed\n",
      "905\n",
      "xxxxx\n",
      "Web of Science\n",
      "906\n",
      "xxxxx\n",
      "13.\n",
      "907\n",
      "xxxxx\n",
      "↵\n",
      "908\n",
      "xxxxx\n",
      "Levine,  E., Zhang,  Z., Kuhlman,  T. and Hwa,  T. (2007) Quantitative characteristics of gene regulation by small RNA.\n",
      "909\n",
      "xxxxx\n",
      "PLoS Biol\n",
      "910\n",
      "xxxxx\n",
      ".,\n",
      "911\n",
      "xxxxx\n",
      "OpenUrl\n",
      "912\n",
      "xxxxx\n",
      "Web of Science\n",
      "913\n",
      "xxxxx\n",
      "14.\n",
      "914\n",
      "xxxxx\n",
      "↵\n",
      "915\n",
      "xxxxx\n",
      "Rice,  J.B., Balasubramanian,  D. and Vanderpool,  C.K. (2012) Small RNA binding-site multiplicity involved in translational regulation of a polycistronic mRNA.\n",
      "916\n",
      "xxxxx\n",
      "Proc.\n",
      "917\n",
      "xxxxx\n",
      ".,\n",
      "918\n",
      "xxxxx\n",
      "OpenUrl\n",
      "919\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "920\n",
      "xxxxx\n",
      "15.\n",
      "921\n",
      "xxxxx\n",
      "Degradation of target mRNAs mediated by an Hfq-binding small RNA in bacteria.\n",
      "922\n",
      "xxxxx\n",
      "Genes Dev\n",
      "923\n",
      "xxxxx\n",
      ".,\n",
      "924\n",
      "xxxxx\n",
      "OpenUrl\n",
      "925\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "926\n",
      "xxxxx\n",
      "16.\n",
      "927\n",
      "xxxxx\n",
      "↵\n",
      "928\n",
      "xxxxx\n",
      "Pfeiffer,  V., Papenfort,  K., Lucchini,  S., Hinton,  J.C.D. and Vogel,  J. (2009) Coding sequence targeting by MicC RNA reveals bacterial mRNA silencing downstream of translational initiation.\n",
      "929\n",
      "xxxxx\n",
      "Nat.\n",
      "930\n",
      "xxxxx\n",
      ".,\n",
      "931\n",
      "xxxxx\n",
      "OpenUrl\n",
      "932\n",
      "xxxxx\n",
      "CrossRef\n",
      "933\n",
      "xxxxx\n",
      "PubMed\n",
      "934\n",
      "xxxxx\n",
      "Web of Science\n",
      "935\n",
      "xxxxx\n",
      "17.\n",
      "936\n",
      "xxxxx\n",
      "↵\n",
      "937\n",
      "xxxxx\n",
      "Yoo,  S.M., Na,  D. and Lee,  S.Y. (2013) Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli.\n",
      "938\n",
      "xxxxx\n",
      "Nat.\n",
      "939\n",
      "xxxxx\n",
      ".,\n",
      "940\n",
      "xxxxx\n",
      "OpenUrl\n",
      "941\n",
      "xxxxx\n",
      "CrossRef\n",
      "942\n",
      "xxxxx\n",
      "PubMed\n",
      "943\n",
      "xxxxx\n",
      "18.\n",
      "944\n",
      "xxxxx\n",
      "↵\n",
      "945\n",
      "xxxxx\n",
      "Sharma,  V., Yamamura,  A. and Yokobayashi,  Y. (2012) Engineering artificial small RNAs for Conditional gene silencing in escherichia coli.\n",
      "946\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "947\n",
      "xxxxx\n",
      ".,\n",
      "948\n",
      "xxxxx\n",
      "OpenUrl\n",
      "949\n",
      "xxxxx\n",
      "CrossRef\n",
      "950\n",
      "xxxxx\n",
      "PubMed\n",
      "951\n",
      "xxxxx\n",
      "Web of Science\n",
      "952\n",
      "xxxxx\n",
      "19.\n",
      "953\n",
      "xxxxx\n",
      "↵\n",
      "954\n",
      "xxxxx\n",
      "Viegas,  S.C., Silva,  I.J., Saramago,  M., Domingues,  S. and Arraiano,  C.M. (2011) Regulation of the small regulatory RNA MicA by ribonuclease III:\n",
      "955\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "956\n",
      "xxxxx\n",
      "140, 355-366.\n",
      "851\n",
      "xxxxx\n",
      "OpenUrl\n",
      "852\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "853\n",
      "xxxxx\n",
      "↵\n",
      "854\n",
      "xxxxx\n",
      "Mazanka,  E., Alexander,  J., Yeh,  B.J., Charoenpong,  P., Lowery,  D.M., Yaffe,  M., and Weiss,  E.L. (2008).\n",
      "855\n",
      "xxxxx\n",
      "PLoS Biol\n",
      "856\n",
      "xxxxx\n",
      "6, e203.\n",
      "857\n",
      "xxxxx\n",
      "OpenUrl\n",
      "858\n",
      "xxxxx\n",
      "CrossRef\n",
      "859\n",
      "xxxxx\n",
      "PubMed\n",
      "860\n",
      "xxxxx\n",
      "↵\n",
      "861\n",
      "xxxxx\n",
      "Mazanka,  E., and Weiss,  E.L. (2010).\n",
      "862\n",
      "xxxxx\n",
      "Molecular biology of the cell\n",
      "863\n",
      "xxxxx\n",
      "21, 2809-2820.\n",
      "864\n",
      "xxxxx\n",
      "OpenUrl\n",
      "865\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "866\n",
      "xxxxx\n",
      "↵\n",
      "867\n",
      "xxxxx\n",
      "What Will B Will B:\n",
      "868\n",
      "xxxxx\n",
      "Frontiers in Cell and Developmental Biology\n",
      "869\n",
      "xxxxx\n",
      "8, 1-8.\n",
      "870\n",
      "xxxxx\n",
      "OpenUrl\n",
      "871\n",
      "xxxxx\n",
      "↵\n",
      "872\n",
      "xxxxx\n",
      "Mitchell,  S., Vargas,  J., and Hoffmann,  A. (2016).\n",
      "873\n",
      "xxxxx\n",
      "Wiley interdisciplinary reviews Systems biology and medicine\n",
      "874\n",
      "xxxxx\n",
      "8, 227-241.\n",
      "875\n",
      "xxxxx\n",
      "OpenUrl\n",
      "876\n",
      "xxxxx\n",
      "↵\n",
      "877\n",
      "xxxxx\n",
      "Mohr,  D., Frey,  S., Fischer,  T., Güttler,  T., and Görlich,  D. (2009).\n",
      "878\n",
      "xxxxx\n",
      "Embo J\n",
      "879\n",
      "xxxxx\n",
      "28, 2541-2553.\n",
      "880\n",
      "xxxxx\n",
      "OpenUrl\n",
      "881\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "882\n",
      "xxxxx\n",
      "↵\n",
      "883\n",
      "xxxxx\n",
      "Montpellier,  D., and Bourens M Bécam Am,  P.C.B.S.B.E.H.C.J.R.W. (2008).\n",
      "884\n",
      "xxxxx\n",
      "Biol Cell\n",
      "885\n",
      "xxxxx\n",
      "100, 343-354.\n",
      "886\n",
      "xxxxx\n",
      "OpenUrl\n",
      "887\n",
      "xxxxx\n",
      "CrossRef\n",
      "888\n",
      "xxxxx\n",
      "PubMed\n",
      "889\n",
      "xxxxx\n",
      "Web of Science\n",
      "890\n",
      "xxxxx\n",
      "↵\n",
      "891\n",
      "xxxxx\n",
      "Mura,  M., Feillet,  C., Bertolusso,  R., Delaunay,  F., and Kimmel,  M. (2019).\n",
      "892\n",
      "xxxxx\n",
      "PLoS Computational Biology\n",
      "893\n",
      "xxxxx\n",
      "15, 1-26.\n",
      "894\n",
      "xxxxx\n",
      "OpenUrl\n",
      "895\n",
      "xxxxx\n",
      "CrossRef\n",
      "896\n",
      "xxxxx\n",
      "↵\n",
      "897\n",
      "xxxxx\n",
      "Nasmyth,  K. (1983).\n",
      "898\n",
      "xxxxx\n",
      "Nature\n",
      "899\n",
      "xxxxx\n",
      "302, 670-676.\n",
      "900\n",
      "xxxxx\n",
      "OpenUrl\n",
      "901\n",
      "xxxxx\n",
      "CrossRef\n",
      "902\n",
      "xxxxx\n",
      "PubMed\n",
      "903\n",
      "xxxxx\n",
      "↵\n",
      "904\n",
      "xxxxx\n",
      "Nemergut,  M.E., and Macara,  I.G. (2000).\n",
      "905\n",
      "xxxxx\n",
      "The Journal of cell biology\n",
      "906\n",
      "xxxxx\n",
      "149, 835-850.\n",
      "907\n",
      "xxxxx\n",
      "OpenUrl\n",
      "908\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "909\n",
      "xxxxx\n",
      "↵\n",
      "910\n",
      "xxxxx\n",
      "O’Conallain,  C., Doolin,  M.T., Taggart,  C., Thornton,  F., and Butler,  G. (1999).\n",
      "911\n",
      "xxxxx\n",
      "MolGenGenet\n",
      "912\n",
      "xxxxx\n",
      "262, 275-282.\n",
      "913\n",
      "xxxxx\n",
      "OpenUrl\n",
      "914\n",
      "xxxxx\n",
      "↵\n",
      "915\n",
      "xxxxx\n",
      "Paine,  P., and Scherr,  P. (1975).\n",
      "916\n",
      "xxxxx\n",
      "BiophysJ\n",
      "917\n",
      "xxxxx\n",
      "15, 1087-1091.\n",
      "918\n",
      "xxxxx\n",
      "OpenUrl\n",
      "919\n",
      "xxxxx\n",
      "PubMed\n",
      "920\n",
      "xxxxx\n",
      "↵\n",
      "921\n",
      "xxxxx\n",
      "Qiuyan Fua,  A., and Pachter,  L. (2016).\n",
      "922\n",
      "xxxxx\n",
      "Stat Appl Genet Mol Biol\n",
      "923\n",
      "xxxxx\n",
      "176, 139-148.\n",
      "924\n",
      "xxxxx\n",
      "OpenUrl\n",
      "925\n",
      "xxxxx\n",
      "↵\n",
      "926\n",
      "xxxxx\n",
      "Racki,  W.J., Becam,  A.M., Nasr,  F., and Herbert,  C.J. (2000).\n",
      "927\n",
      "xxxxx\n",
      "EMBO J\n",
      "928\n",
      "xxxxx\n",
      "19, 4524-4532.\n",
      "929\n",
      "xxxxx\n",
      "OpenUrl\n",
      "930\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "931\n",
      "xxxxx\n",
      "Rasband,  W.S. (1997-2005).\n",
      "932\n",
      "xxxxx\n",
      "http://rsbinfonihgov/ij/\n",
      "933\n",
      "xxxxx\n",
      ".\n",
      "934\n",
      "xxxxx\n",
      "↵\n",
      "935\n",
      "xxxxx\n",
      "Raser,  J.M., and O’Shea,  E.K. (2004).\n",
      "936\n",
      "xxxxx\n",
      "Science\n",
      "937\n",
      "xxxxx\n",
      "304, 1811-1814.\n",
      "938\n",
      "xxxxx\n",
      "OpenUrl\n",
      "939\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "940\n",
      "xxxxx\n",
      "↵\n",
      "941\n",
      "xxxxx\n",
      "Ribbeck,  K., and Görlich,  D. (2002).\n",
      "942\n",
      "xxxxx\n",
      "Embo J\n",
      "943\n",
      "xxxxx\n",
      "21, 2664-2671.\n",
      "944\n",
      "xxxxx\n",
      "OpenUrl\n",
      "945\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "946\n",
      "xxxxx\n",
      "↵\n",
      "947\n",
      "xxxxx\n",
      "Ribbeck,  K., Gorlich,  D., and Go,  D. (2001).\n",
      "948\n",
      "xxxxx\n",
      "EMBO J\n",
      "949\n",
      "xxxxx\n",
      "20, 1320-1330.\n",
      "950\n",
      "xxxxx\n",
      "OpenUrl\n",
      "951\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "952\n",
      "xxxxx\n",
      "↵\n",
      "953\n",
      "xxxxx\n",
      "Rosenfeld,  N., Elowitz,  M.B., and Alon,  U. (2002).\n",
      "954\n",
      "xxxxx\n",
      "J Mol Biol\n",
      "955\n",
      "xxxxx\n",
      "323, 785-793.\n",
      "956\n",
      "xxxxx\n",
      "OpenUrl\n",
      "957\n",
      "xxxxx\n",
      ".,\n",
      "957\n",
      "xxxxx\n",
      "OpenUrl\n",
      "958\n",
      "xxxxx\n",
      "CrossRef\n",
      "959\n",
      "xxxxx\n",
      "PubMed\n",
      "960\n",
      "xxxxx\n",
      "Web of Science\n",
      "961\n",
      "xxxxx\n",
      "20.\n",
      "962\n",
      "xxxxx\n",
      "↵\n",
      "963\n",
      "xxxxx\n",
      "Man,  S., Cheng,  R., Miao,  C., Gong,  Q., Gu,  Y., Lu,  X., Han,  F. and Yu,  W. (2011) Artificial trans-encoded small non-coding RNAs specifically silence the selected gene expression in bacteria.\n",
      "964\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "965\n",
      "xxxxx\n",
      ".,\n",
      "966\n",
      "xxxxx\n",
      "OpenUrl\n",
      "967\n",
      "xxxxx\n",
      "CrossRef\n",
      "968\n",
      "xxxxx\n",
      "PubMed\n",
      "969\n",
      "xxxxx\n",
      "21.\n",
      "970\n",
      "xxxxx\n",
      "Nakashima,  N., Tamura,  T. and Good,  L. (2006) Paired termini stabilize antisense RNAs and enhance conditional gene silencing in Escherichia coli.\n",
      "971\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "972\n",
      "xxxxx\n",
      ".,\n",
      "973\n",
      "xxxxx\n",
      "OpenUrl\n",
      "974\n",
      "xxxxx\n",
      "CrossRef\n",
      "975\n",
      "xxxxx\n",
      "PubMed\n",
      "976\n",
      "xxxxx\n",
      "22.\n",
      "977\n",
      "xxxxx\n",
      "Engdahl,  H.M., Lindell,  M. and Wagner,  E.G.H. (2001) Introduction of an RNA Stability Element at the 5′-End of an Antisense RNA Cassette Increases the Inhibition of Target RNA Translation.\n",
      "978\n",
      "xxxxx\n",
      "Antisense Nucleic Acid Drug Dev\n",
      "979\n",
      "xxxxx\n",
      ".,\n",
      "980\n",
      "xxxxx\n",
      "OpenUrl\n",
      "981\n",
      "xxxxx\n",
      "CrossRef\n",
      "982\n",
      "xxxxx\n",
      "PubMed\n",
      "983\n",
      "xxxxx\n",
      "Web of Science\n",
      "984\n",
      "xxxxx\n",
      "23.\n",
      "985\n",
      "xxxxx\n",
      "↵\n",
      "986\n",
      "xxxxx\n",
      "Ellis,  M.J., Trussler,  R.S. and Haniford,  D.B. (2015) A cis-encoded sRNA, Hfq and mRNA secondary structure act independently to suppress IS200 transposition.\n",
      "987\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "988\n",
      "xxxxx\n",
      ".,\n",
      "989\n",
      "xxxxx\n",
      "OpenUrl\n",
      "990\n",
      "xxxxx\n",
      "CrossRef\n",
      "991\n",
      "xxxxx\n",
      "PubMed\n",
      "992\n",
      "xxxxx\n",
      "24.\n",
      "993\n",
      "xxxxx\n",
      "↵\n",
      "994\n",
      "xxxxx\n",
      "Ghodasara,  A. and Voigt,  C.A. (2017) Balancing gene expression without library construction via a reusable sRNA pool.\n",
      "995\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "996\n",
      "xxxxx\n",
      ".,\n",
      "997\n",
      "xxxxx\n",
      "OpenUrl\n",
      "998\n",
      "xxxxx\n",
      "25.\n",
      "999\n",
      "xxxxx\n",
      "↵\n",
      "1000\n",
      "xxxxx\n",
      "Kelly,  C.L., Harris,  A.W.K., Steel,  H., Hancock,  E.J., Heap,  J.T. and Papachristodoulou,  A. (2018) Synthetic negative feedback circuits using engineered small RNAs.\n",
      "1001\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1002\n",
      "xxxxx\n",
      ".,\n",
      "1003\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1004\n",
      "xxxxx\n",
      "CrossRef\n",
      "1005\n",
      "xxxxx\n",
      "26.\n",
      "1006\n",
      "xxxxx\n",
      "↵\n",
      "1007\n",
      "xxxxx\n",
      "Ceroni,  F., Furini,  S., Stefan,  A., Hochkoeppler,  A. and Giordano,  E. (2012) A synthetic post-transcriptional controller to explore the modular design of gene circuits.\n",
      "1008\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "1009\n",
      "xxxxx\n",
      ".,\n",
      "1010\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1011\n",
      "xxxxx\n",
      "CrossRef\n",
      "1012\n",
      "xxxxx\n",
      "27.\n",
      "1013\n",
      "xxxxx\n",
      "↵\n",
      "1014\n",
      "xxxxx\n",
      "Na,  D., Yoo,  S.M., Chung,  H., Park,  H., Park,  J.H. and Lee,  S.Y. (2013) Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs.\n",
      "1015\n",
      "xxxxx\n",
      "Nat.\n",
      "1016\n",
      "xxxxx\n",
      ".,\n",
      "1017\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1018\n",
      "xxxxx\n",
      "CrossRef\n",
      "1019\n",
      "xxxxx\n",
      "PubMed\n",
      "1020\n",
      "xxxxx\n",
      "28.\n",
      "1021\n",
      "xxxxx\n",
      "↵\n",
      "1022\n",
      "xxxxx\n",
      "Lahiry,  A., Stimple,  S.D., Wood,  D.W. and Lease,  R.A. (2017) Retargeting a Dual-Acting sRNA for Multiple mRNA Transcript Regulation.\n",
      "1023\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "1024\n",
      "xxxxx\n",
      ".,\n",
      "1025\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1026\n",
      "xxxxx\n",
      "CrossRef\n",
      "1027\n",
      "xxxxx\n",
      "29.\n",
      "1028\n",
      "xxxxx\n",
      "↵\n",
      "1029\n",
      "xxxxx\n",
      "Sagawa,  S., Shin,  J.-E., Hussein,  R. and Lim,  H.N. (2015) Paradoxical suppression of small RNA activity at high Hfq concentrations due to random-order binding.\n",
      "1030\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1031\n",
      "xxxxx\n",
      ".,\n",
      "1032\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1033\n",
      "xxxxx\n",
      "CrossRef\n",
      "1034\n",
      "xxxxx\n",
      "PubMed\n",
      "1035\n",
      "xxxxx\n",
      "30.\n",
      "1036\n",
      "xxxxx\n",
      "↵\n",
      "1037\n",
      "xxxxx\n",
      "Hoynes-O’Connor,  A. and Moon,  T.S. (2016) Development of Design Rules for Reliable Antisense RNA Behavior in E. coli.\n",
      "1038\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "1039\n",
      "xxxxx\n",
      ".,\n",
      "1040\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1041\n",
      "xxxxx\n",
      "CrossRef\n",
      "1042\n",
      "xxxxx\n",
      "31.\n",
      "1043\n",
      "xxxxx\n",
      "↵\n",
      "1044\n",
      "xxxxx\n",
      "Lee,  Y.J. and Moon,  T.S. (2018) Design rules of synthetic non-coding RNAs in bacteria.\n",
      "1045\n",
      "xxxxx\n",
      "Methods\n",
      "1046\n",
      "xxxxx\n",
      ", 143, 58-69.\n",
      "1047\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1048\n",
      "xxxxx\n",
      "CrossRef\n",
      "1049\n",
      "xxxxx\n",
      "32.\n",
      "1050\n",
      "xxxxx\n",
      "↵\n",
      "1051\n",
      "xxxxx\n",
      "Massaiu,  I., Pasotti,  L., Casanova,  M., Politi,  N., Zucca,  S., Cusella De Angelis,  M.G. and Magni,  P. (2015) Quantification of the gene silencing performances of rationally-designed synthetic small RNAs.\n",
      "1052\n",
      "xxxxx\n",
      "Synth.\n",
      "1053\n",
      "xxxxx\n",
      "CrossRef\n",
      "958\n",
      "xxxxx\n",
      "PubMed\n",
      "959\n",
      "xxxxx\n",
      "Web of Science\n",
      "960\n",
      "xxxxx\n",
      "↵\n",
      "961\n",
      "xxxxx\n",
      "Rout,  M.P., Aitchison,  J.D., Suprapto,  A., Hjertaas,  K., Zhao,  Y., and Chait,  B.T. (2000).\n",
      "962\n",
      "xxxxx\n",
      "The Journal of cell biology\n",
      "963\n",
      "xxxxx\n",
      "148, 635-651.\n",
      "964\n",
      "xxxxx\n",
      "OpenUrl\n",
      "965\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "966\n",
      "xxxxx\n",
      "↵\n",
      "967\n",
      "xxxxx\n",
      "Roux,  J., Hafner,  M., Bandara,  S., Sims,  J.J., Hudson,  H., Chai,  D., and Sorger,  P.K. (2015).\n",
      "968\n",
      "xxxxx\n",
      "Molecular systems biology\n",
      "969\n",
      "xxxxx\n",
      "11, 803-803.\n",
      "970\n",
      "xxxxx\n",
      "OpenUrl\n",
      "971\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "972\n",
      "xxxxx\n",
      "↵\n",
      "973\n",
      "xxxxx\n",
      "Sbia,  M., Parnell,  E.J., Yu,  Y., Olsen,  A.E., Kretschmann,  K.L., Voth,  W.P., and Stillman,  D.J. (2008).\n",
      "974\n",
      "xxxxx\n",
      "J Biol Chem\n",
      "975\n",
      "xxxxx\n",
      "283, 11135-11145.\n",
      "976\n",
      "xxxxx\n",
      "OpenUrl\n",
      "977\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "978\n",
      "xxxxx\n",
      "↵\n",
      "979\n",
      "xxxxx\n",
      "Siebrasse,  J.P., and Peters,  R. (2002).\n",
      "980\n",
      "xxxxx\n",
      "EMBO reports\n",
      "981\n",
      "xxxxx\n",
      "3, 887-892.\n",
      "982\n",
      "xxxxx\n",
      "OpenUrl\n",
      "983\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "984\n",
      "xxxxx\n",
      "↵\n",
      "985\n",
      "xxxxx\n",
      "Sigal,  A., Milo,  R., Cohen,  A., Geva-Zatorsky,  N., Klein,  Y., Liron,  Y., Rosenfeld,  N., Danon,  T., Perzov,  N., and Alon,  U. (2006).\n",
      "986\n",
      "xxxxx\n",
      "Nature\n",
      "987\n",
      "xxxxx\n",
      "444, 643-646.\n",
      "988\n",
      "xxxxx\n",
      "OpenUrl\n",
      "989\n",
      "xxxxx\n",
      "CrossRef\n",
      "990\n",
      "xxxxx\n",
      "PubMed\n",
      "991\n",
      "xxxxx\n",
      "Web of Science\n",
      "992\n",
      "xxxxx\n",
      "↵\n",
      "993\n",
      "xxxxx\n",
      "Sil,  A., and Herskowitz,  I. (1996).\n",
      "994\n",
      "xxxxx\n",
      "Cell\n",
      "995\n",
      "xxxxx\n",
      "84, 711-722.\n",
      "996\n",
      "xxxxx\n",
      "OpenUrl\n",
      "997\n",
      "xxxxx\n",
      "CrossRef\n",
      "998\n",
      "xxxxx\n",
      "PubMed\n",
      "999\n",
      "xxxxx\n",
      "Web of Science\n",
      "1000\n",
      "xxxxx\n",
      "↵\n",
      "1001\n",
      "xxxxx\n",
      "Spencer,  S.L., Gaudet,  S., Albeck,  J.G., Burke,  J.M., and Sorger,  P.K. (2009).\n",
      "1002\n",
      "xxxxx\n",
      "Nature\n",
      "1003\n",
      "xxxxx\n",
      "459, 428-432.\n",
      "1004\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1005\n",
      "xxxxx\n",
      "CrossRef\n",
      "1006\n",
      "xxxxx\n",
      "PubMed\n",
      "1007\n",
      "xxxxx\n",
      "Web of Science\n",
      "1008\n",
      "xxxxx\n",
      "↵\n",
      "1009\n",
      "xxxxx\n",
      "St-Pierre,  F., and Endy,  D. (2008).\n",
      "1010\n",
      "xxxxx\n",
      "Proc Natl Acad Sci U S A\n",
      "1011\n",
      "xxxxx\n",
      "105, 20705-20710.\n",
      "1012\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1013\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1014\n",
      "xxxxx\n",
      "↵\n",
      "1015\n",
      "xxxxx\n",
      "Suel,  G.M., Garcia-Ojalvo,  J., Liberman,  L.M., and Elowitz,  M. (2006).\n",
      "1016\n",
      "xxxxx\n",
      "Nature\n",
      "1017\n",
      "xxxxx\n",
      "440, 545-550.\n",
      "1018\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1019\n",
      "xxxxx\n",
      "CrossRef\n",
      "1020\n",
      "xxxxx\n",
      "PubMed\n",
      "1021\n",
      "xxxxx\n",
      "Web of Science\n",
      "1022\n",
      "xxxxx\n",
      "↵\n",
      "1023\n",
      "xxxxx\n",
      "Team,  R.D.C. (2010).\n",
      "1024\n",
      "xxxxx\n",
      "R:\n",
      "1025\n",
      "xxxxx\n",
      ".\n",
      "1026\n",
      "xxxxx\n",
      "↵\n",
      "1027\n",
      "xxxxx\n",
      "Timney,  B.L., Tetenbaum-Novatt,  J., Agate,  D.S., Williams,  R., Zhang,  W., Chait,  B.T., and Rout,  M.P. (2006).\n",
      "1028\n",
      "xxxxx\n",
      "The Journal of cell biology\n",
      "1029\n",
      "xxxxx\n",
      "175, 579-593.\n",
      "1030\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1031\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1032\n",
      "xxxxx\n",
      "↵\n",
      "1033\n",
      "xxxxx\n",
      "Uchida,  M., Sun,  Y., McDermott,  G., Knoechel,  C., Le Gros,  M.A., Parkinson,  D., Drubin,  D.G., and Larabell,  C.A. (2011).\n",
      "1034\n",
      "xxxxx\n",
      "Yeast\n",
      "1035\n",
      "xxxxx\n",
      "28, 227-236.\n",
      "1036\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1037\n",
      "xxxxx\n",
      "CrossRef\n",
      "1038\n",
      "xxxxx\n",
      "PubMed\n",
      "1039\n",
      "xxxxx\n",
      "Web of Science\n",
      "1040\n",
      "xxxxx\n",
      "↵\n",
      "1041\n",
      "xxxxx\n",
      "van Drogen,  F., Stucke,  V.M., Jorritsma,  G., and Peter,  M. (2001).\n",
      "1042\n",
      "xxxxx\n",
      "NatCell Biol\n",
      "1043\n",
      "xxxxx\n",
      "3, 1051-1059.\n",
      "1044\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1045\n",
      "xxxxx\n",
      "CrossRef\n",
      "1046\n",
      "xxxxx\n",
      "PubMed\n",
      "1047\n",
      "xxxxx\n",
      "Web of Science\n",
      "1048\n",
      "xxxxx\n",
      "↵\n",
      "1049\n",
      "xxxxx\n",
      "Vashistha,  H., Kohram,  M., and Salman,  H. (2021).\n",
      "1050\n",
      "xxxxx\n",
      "eLife\n",
      "1051\n",
      "xxxxx\n",
      ", 1-16.\n",
      "1052\n",
      "xxxxx\n",
      "↵\n",
      "1053\n",
      "xxxxx\n",
      "Wachsmuth,  M., Weidemann,  T., Müller,  G., Hoffmann-Rohrer,  U.W., Knoch,  T.A., Waldeck,  W., and Langowski,  J. (2003).\n",
      "1054\n",
      "xxxxx\n",
      "Biophysical Journal\n",
      "1055\n",
      "xxxxx\n",
      "84, 3353-3363.\n",
      "1056\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1057\n",
      "xxxxx\n",
      "CrossRef\n",
      "1058\n",
      "xxxxx\n",
      "PubMed\n",
      "1059\n",
      "xxxxx\n",
      "Web of Science\n",
      "1060\n",
      "xxxxx\n",
      "↵\n",
      "1061\n",
      "xxxxx\n",
      "Webster,  M.T., McCaffery,  J.M., and Cohen-Fix,  O. (2010).\n",
      "1062\n",
      "xxxxx\n",
      "Journal of Cell Biology\n",
      "1063\n",
      "xxxxx\n",
      "191, 1079-1088.\n",
      "1064\n",
      "xxxxx\n",
      ".,\n",
      "1054\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1055\n",
      "xxxxx\n",
      "33.\n",
      "1056\n",
      "xxxxx\n",
      "↵\n",
      "1057\n",
      "xxxxx\n",
      "Storch,  M., Haines,  M.C. and Baldwin,  G.S. (2020) DNA-BOT:\n",
      "1058\n",
      "xxxxx\n",
      "Synth.\n",
      "1059\n",
      "xxxxx\n",
      ".,\n",
      "1060\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1061\n",
      "xxxxx\n",
      "CrossRef\n",
      "1062\n",
      "xxxxx\n",
      "34.\n",
      "1063\n",
      "xxxxx\n",
      "↵\n",
      "1064\n",
      "xxxxx\n",
      "Storch,  M., Casini,  A., Mackrow,  B., Fleming,  T., Trewhitt,  H., Ellis,  T. and Baldwin,  G.S. (2015) BASIC:\n",
      "1065\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "1066\n",
      "xxxxx\n",
      ".,\n",
      "1067\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1068\n",
      "xxxxx\n",
      "CrossRef\n",
      "1069\n",
      "xxxxx\n",
      "PubMed\n",
      "1070\n",
      "xxxxx\n",
      "35.\n",
      "1071\n",
      "xxxxx\n",
      "↵\n",
      "1072\n",
      "xxxxx\n",
      "Casini,  A., Christodoulou,  G., Freemont,  P.S., Baldwin,  G.S., Ellis,  T. and MacDonald,  J.T. (2014) R2oDNA Designer:\n",
      "1073\n",
      "xxxxx\n",
      "ACS Synth.\n",
      "1074\n",
      "xxxxx\n",
      ".,\n",
      "1075\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1076\n",
      "xxxxx\n",
      "CrossRef\n",
      "1077\n",
      "xxxxx\n",
      "PubMed\n",
      "1078\n",
      "xxxxx\n",
      "36.\n",
      "1079\n",
      "xxxxx\n",
      "↵\n",
      "1080\n",
      "xxxxx\n",
      "Darty,  K., Denise,  A. and Ponty,  Y. (2009) VARNA:\n",
      "1081\n",
      "xxxxx\n",
      "Bioinformatics\n",
      "1082\n",
      "xxxxx\n",
      ", 25, 1974-1975.\n",
      "1083\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1084\n",
      "xxxxx\n",
      "CrossRef\n",
      "1085\n",
      "xxxxx\n",
      "PubMed\n",
      "1086\n",
      "xxxxx\n",
      "Web of Science\n",
      "1087\n",
      "xxxxx\n",
      "37.\n",
      "1088\n",
      "xxxxx\n",
      "↵\n",
      "1089\n",
      "xxxxx\n",
      "Busch,  A., Richter,  A.S. and Backofen,  R. (2008) IntaRNA:\n",
      "1090\n",
      "xxxxx\n",
      "Bioinformatics\n",
      "1091\n",
      "xxxxx\n",
      ", 24, 2849-2856.\n",
      "1092\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1093\n",
      "xxxxx\n",
      "CrossRef\n",
      "1094\n",
      "xxxxx\n",
      "PubMed\n",
      "1095\n",
      "xxxxx\n",
      "Web of Science\n",
      "1096\n",
      "xxxxx\n",
      "38.\n",
      "1097\n",
      "xxxxx\n",
      "↵\n",
      "1098\n",
      "xxxxx\n",
      "Wright,  P.R., Georg,  J., Mann,  M., Sorescu,  D.A., Richter,  A.S., Lott,  S., Kleinkauf,  R., Hess,  W.R. and Backofen,  R. (2014) CopraRNA and IntaRNA:\n",
      "1099\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1100\n",
      "xxxxx\n",
      ".,\n",
      "1101\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1102\n",
      "xxxxx\n",
      "39.\n",
      "1103\n",
      "xxxxx\n",
      "↵\n",
      "1104\n",
      "xxxxx\n",
      "Markham,  N.R. and Zuker,  M. (2005) DINAMelt web server for nucleic acid melting prediction.\n",
      "1105\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1106\n",
      "xxxxx\n",
      ".,\n",
      "1107\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1108\n",
      "xxxxx\n",
      "CrossRef\n",
      "1109\n",
      "xxxxx\n",
      "PubMed\n",
      "1110\n",
      "xxxxx\n",
      "Web of Science\n",
      "1111\n",
      "xxxxx\n",
      "40.\n",
      "1112\n",
      "xxxxx\n",
      "↵\n",
      "1113\n",
      "xxxxx\n",
      "Bonde,  M.T., Pedersen,  M., Klausen,  M.S., Jensen,  S.I., Wulff,  T., Harrison,  S., Nielsen,  A.T., Herrgård,  M.J. and Sommer,  M.O.A. (2016) Predictable tuning of protein expression in bacteria.\n",
      "1114\n",
      "xxxxx\n",
      "Nat.\n",
      "1115\n",
      "xxxxx\n",
      ", 13, 233-236.\n",
      "1116\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1117\n",
      "xxxxx\n",
      "CrossRef\n",
      "1118\n",
      "xxxxx\n",
      "PubMed\n",
      "1119\n",
      "xxxxx\n",
      "41.\n",
      "1120\n",
      "xxxxx\n",
      "↵\n",
      "1121\n",
      "xxxxx\n",
      "Salis,  H.M., Mirsky,  E.A. and Voigt,  C.A. (2009) Automated design of synthetic ribosome binding sites to control protein expression.\n",
      "1122\n",
      "xxxxx\n",
      "Nat.\n",
      "1123\n",
      "xxxxx\n",
      ".,\n",
      "1124\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1125\n",
      "xxxxx\n",
      "CrossRef\n",
      "1126\n",
      "xxxxx\n",
      "PubMed\n",
      "1127\n",
      "xxxxx\n",
      "Web of Science\n",
      "1128\n",
      "xxxxx\n",
      "42.\n",
      "1129\n",
      "xxxxx\n",
      "↵\n",
      "1130\n",
      "xxxxx\n",
      "Espah Borujeni,  A., Channarasappa,  A.S. and Salis,  H.M. (2014) Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.\n",
      "1131\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1132\n",
      "xxxxx\n",
      ".,\n",
      "1133\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1134\n",
      "xxxxx\n",
      "CrossRef\n",
      "1135\n",
      "xxxxx\n",
      "PubMed\n",
      "1136\n",
      "xxxxx\n",
      "Web of Science\n",
      "1137\n",
      "xxxxx\n",
      "43.\n",
      "1138\n",
      "xxxxx\n",
      "↵\n",
      "1139\n",
      "xxxxx\n",
      "Raden,  M., Ali,  S.M., Alkhnbashi,  O.S., Busch,  A., Costa,  F., Davis,  J.A., Eggenhofer,  F., Gelhausen,  R., Georg,  J., Heyne,  S., et al. (\n",
      "1140\n",
      "xxxxx\n",
      "Nucleic Acids Res\n",
      "1141\n",
      "xxxxx\n",
      ".,\n",
      "1142\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1143\n",
      "xxxxx\n",
      "CrossRef\n",
      "1144\n",
      "xxxxx\n",
      "PubMed\n",
      "1145\n",
      "xxxxx\n",
      "44.\n",
      "1146\n",
      "xxxxx\n",
      "↵\n",
      "1147\n",
      "xxxxx\n",
      "Wright,  P.R., Richter,  A.S., Papenfort,  K., Mann,  M., Vogel,  J., Hess,  W.R., Backofen,  R. and Georg,  J. (2013) Comparative genomics boosts target prediction for bacterial small RNAs.\n",
      "1148\n",
      "xxxxx\n",
      "Proc.\n",
      "1149\n",
      "xxxxx\n",
      ".,\n",
      "1150\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1065\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1066\n",
      "xxxxx\n",
      "↵\n",
      "1067\n",
      "xxxxx\n",
      "Weiss,  E.L., Kurischko,  C., Zhang,  C., Shokat,  K., Drubin,  D.G., and Luca,  F.C. (2002).\n",
      "1068\n",
      "xxxxx\n",
      "J Cell Biol\n",
      "1069\n",
      "xxxxx\n",
      "158, 885-900.\n",
      "1070\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1071\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1072\n",
      "xxxxx\n",
      "↵\n",
      "1073\n",
      "xxxxx\n",
      "Winey,  M., Yarar,  D., Giddings,  T.H., and Mastronarde,  D.N. (1997).\n",
      "1074\n",
      "xxxxx\n",
      "Molecular biology of the cell\n",
      "1075\n",
      "xxxxx\n",
      "8, 2119-2132.\n",
      "1076\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1077\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1078\n",
      "xxxxx\n",
      "↵\n",
      "1079\n",
      "xxxxx\n",
      "Yang,  W., and Musser,  S.M. (2006).\n",
      "1080\n",
      "xxxxx\n",
      "Journal of Cell Biology\n",
      "1081\n",
      "xxxxx\n",
      "174, 951-961.\n",
      "1082\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1083\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1084\n",
      "xxxxx\n",
      "View the discussion thread.\n",
      "1085\n",
      "xxxxx\n",
      "Back to top\n",
      "1086\n",
      "xxxxx\n",
      "Previous\n",
      "1087\n",
      "xxxxx\n",
      "Next\n",
      "1088\n",
      "xxxxx\n",
      "Posted June 24, 2022.\n",
      "1089\n",
      "xxxxx\n",
      "Download PDF\n",
      "1090\n",
      "xxxxx\n",
      "Supplementary Material\n",
      "1091\n",
      "xxxxx\n",
      "Email\n",
      "1092\n",
      "xxxxx\n",
      "Thank you for your interest in spreading the word about bioRxiv.\n",
      "1093\n",
      "xxxxx\n",
      "NOTE:\n",
      "1094\n",
      "xxxxx\n",
      "Your Email *\n",
      "1095\n",
      "xxxxx\n",
      "Your Name *\n",
      "1096\n",
      "xxxxx\n",
      "Send To *\n",
      "1097\n",
      "xxxxx\n",
      "Enter multiple addresses on separate lines or separate them with commas.\n",
      "1098\n",
      "xxxxx\n",
      "You are going to email the following\n",
      "1099\n",
      "xxxxx\n",
      "Characterization of cell-to-cell variation in nuclear transport rates and identification of its sources\n",
      "1100\n",
      "xxxxx\n",
      "Message Subject\n",
      "1101\n",
      "xxxxx\n",
      "(Your Name) has forwarded a page to you from bioRxiv\n",
      "1102\n",
      "xxxxx\n",
      "Message Body\n",
      "1103\n",
      "xxxxx\n",
      "(Your Name) thought you would like to see this page from the bioRxiv website.\n",
      "1104\n",
      "xxxxx\n",
      "Your Personal Message\n",
      "1105\n",
      "xxxxx\n",
      "CAPTCHA\n",
      "1106\n",
      "xxxxx\n",
      "This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.\n",
      "1107\n",
      "xxxxx\n",
      "Share\n",
      "1108\n",
      "xxxxx\n",
      "Characterization of cell-to-cell variation in nuclear transport rates and identification of its sources\n",
      "1109\n",
      "xxxxx\n",
      "Lucía Durrieu, Alan Bush, Alicia Grande, Rikard Johansson, David Janzén, Andrea Katz, Gunnar Cedersund, Alejandro Colman-Lerner\n",
      "1110\n",
      "xxxxx\n",
      "bioRxiv 001768;\n",
      "1111\n",
      "xxxxx\n",
      "Share This Article:\n",
      "1112\n",
      "xxxxx\n",
      "Copy\n",
      "1113\n",
      "xxxxx\n",
      "Citation Tools\n",
      "1114\n",
      "xxxxx\n",
      "Characterization of cell-to-cell variation in nuclear transport rates and identification of its sources\n",
      "1115\n",
      "xxxxx\n",
      "Lucía Durrieu, Alan Bush, Alicia Grande, Rikard Johansson, David Janzén, Andrea Katz, Gunnar Cedersund, Alejandro Colman-Lerner\n",
      "1116\n",
      "xxxxx\n",
      "bioRxiv 001768;\n",
      "1117\n",
      "xxxxx\n",
      "Citation Manager Formats\n",
      "1118\n",
      "xxxxx\n",
      "BibTeX\n",
      "1119\n",
      "xxxxx\n",
      "Bookends\n",
      "1120\n",
      "xxxxx\n",
      "EasyBib\n",
      "1121\n",
      "xxxxx\n",
      "EndNote (tagged)\n",
      "1122\n",
      "xxxxx\n",
      "EndNote 8 (xml)\n",
      "1123\n",
      "xxxxx\n",
      "Medlars\n",
      "1124\n",
      "xxxxx\n",
      "Mendeley\n",
      "1125\n",
      "xxxxx\n",
      "Papers\n",
      "1126\n",
      "xxxxx\n",
      "RefWorks Tagged\n",
      "1127\n",
      "xxxxx\n",
      "Ref Manager\n",
      "1128\n",
      "xxxxx\n",
      "RIS\n",
      "1129\n",
      "xxxxx\n",
      "Zotero\n",
      "1130\n",
      "xxxxx\n",
      "Tweet Widget\n",
      "1131\n",
      "xxxxx\n",
      "Facebook Like\n",
      "1132\n",
      "xxxxx\n",
      "Google Plus One\n",
      "1133\n",
      "xxxxx\n",
      "Subject Area\n",
      "1134\n",
      "xxxxx\n",
      "Systems Biology\n",
      "1135\n",
      "xxxxx\n",
      "Biophysics\n",
      "1136\n",
      "xxxxx\n",
      "Subject Areas\n",
      "1137\n",
      "xxxxx\n",
      "All Articles\n",
      "1138\n",
      "xxxxx\n",
      "Animal Behavior and Cognition\n",
      "1139\n",
      "xxxxx\n",
      "(4123)\n",
      "1140\n",
      "xxxxx\n",
      "Biochemistry\n",
      "1141\n",
      "xxxxx\n",
      "(8837)\n",
      "1142\n",
      "xxxxx\n",
      "Bioengineering\n",
      "1143\n",
      "xxxxx\n",
      "(6539)\n",
      "1144\n",
      "xxxxx\n",
      "Bioinformatics\n",
      "1145\n",
      "xxxxx\n",
      "(23503)\n",
      "1146\n",
      "xxxxx\n",
      "Biophysics\n",
      "1147\n",
      "xxxxx\n",
      "(11821)\n",
      "1148\n",
      "xxxxx\n",
      "Cancer Biology\n",
      "1149\n",
      "xxxxx\n",
      "(9242)\n",
      "1150\n",
      "xxxxx\n",
      "Cell Biology\n",
      "1151\n",
      "xxxxx\n",
      "(13358)\n",
      "1152\n",
      "xxxxx\n",
      "Clinical Trials\n",
      "1153\n",
      "xxxxx\n",
      "(138)\n",
      "1154\n",
      "xxxxx\n",
      "Developmental Biology\n",
      "1155\n",
      "xxxxx\n",
      "(7456)\n",
      "1156\n",
      "xxxxx\n",
      "Ecology\n",
      "1157\n",
      "xxxxx\n",
      "(11437)\n",
      "1158\n",
      "xxxxx\n",
      "Epidemiology\n",
      "1159\n",
      "xxxxx\n",
      "(2066)\n",
      "1160\n",
      "xxxxx\n",
      "Evolutionary Biology\n",
      "1161\n",
      "xxxxx\n",
      "(15186)\n",
      "1162\n",
      "xxxxx\n",
      "Genetics\n",
      "1163\n",
      "xxxxx\n",
      "(10460)\n",
      "1164\n",
      "xxxxx\n",
      "Genomics\n",
      "1165\n",
      "xxxxx\n",
      "(14058)\n",
      "1166\n",
      "xxxxx\n",
      "Immunology\n",
      "1167\n",
      "xxxxx\n",
      "(9196)\n",
      "1168\n",
      "xxxxx\n",
      "Microbiology\n",
      "1169\n",
      "xxxxx\n",
      "(22242)\n",
      "1170\n",
      "xxxxx\n",
      "Molecular Biology\n",
      "1171\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1151\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1152\n",
      "xxxxx\n",
      "45.\n",
      "1153\n",
      "xxxxx\n",
      "↵\n",
      "1154\n",
      "xxxxx\n",
      "Storch,  M., Casini,  A., Mackrow,  B., Ellis,  T. and Baldwin,  G.S. (2017) BASIC:\n",
      "1155\n",
      "xxxxx\n",
      "In Methods in Molecular Biology\n",
      "1156\n",
      "xxxxx\n",
      ".Vol.\n",
      "1157\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1158\n",
      "xxxxx\n",
      "46.\n",
      "1159\n",
      "xxxxx\n",
      "↵\n",
      "1160\n",
      "xxxxx\n",
      "Vytvytska,  O., Moll,  I., Kaberdin,  V.R., Gabain,  A. Von and Bla,  U. (2000)\n",
      "1161\n",
      "xxxxx\n",
      "Hfq (HF1) stimulates ompA mRNA decay by interfering with ribosome binding\n",
      "1162\n",
      "xxxxx\n",
      ".\n",
      "1163\n",
      "xxxxx\n",
      "47.\n",
      "1164\n",
      "xxxxx\n",
      "↵\n",
      "1165\n",
      "xxxxx\n",
      "Večerek,  B., Moll,  I. and Bläsi,  U. (2005) Translational autocontrol of the Escherichia coli hfq RNA chaperone gene.\n",
      "1166\n",
      "xxxxx\n",
      "RNA\n",
      "1167\n",
      "xxxxx\n",
      ", 11, 976-984.\n",
      "1168\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1169\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1170\n",
      "xxxxx\n",
      "48.\n",
      "1171\n",
      "xxxxx\n",
      "↵\n",
      "1172\n",
      "xxxxx\n",
      "Franze De Fernandez,  M.T., Eoyang,  L. and August,  J.T. (1968) Factor fraction required for the synthesis of bacteriophage Qβ-RNA.\n",
      "1173\n",
      "xxxxx\n",
      "Nature\n",
      "1174\n",
      "xxxxx\n",
      ", 219, 588-290.\n",
      "1175\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1176\n",
      "xxxxx\n",
      "CrossRef\n",
      "1177\n",
      "xxxxx\n",
      "PubMed\n",
      "1178\n",
      "xxxxx\n",
      "Web of Science\n",
      "1179\n",
      "xxxxx\n",
      "49.\n",
      "1180\n",
      "xxxxx\n",
      "Sun,  T., Li,  S., Song,  X., Pei,  G., Diao,  J., Cui,  J., Shi,  M., Chen,  L. and Zhang,  W. (2018) Re-direction of carbon flux to key precursor malonyl-CoA via artificial small RNAs in photosynthetic Synechocystis sp.\n",
      "1181\n",
      "xxxxx\n",
      "Biotechnol.\n",
      "1182\n",
      "xxxxx\n",
      ", 11, 1-16.\n",
      "1183\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1184\n",
      "xxxxx\n",
      "CrossRef\n",
      "1185\n",
      "xxxxx\n",
      "50.\n",
      "1186\n",
      "xxxxx\n",
      "Huis in ‘t Veld,  R.A.G., Kramer,  G., van der Ende,  A., Speijer,  D. and Pannekoek,  Y. (2017) The Hfq regulon of Neisseria meningitidis.\n",
      "1187\n",
      "xxxxx\n",
      "FEBS Open Bio\n",
      "1188\n",
      "xxxxx\n",
      ", 7, 777-788.\n",
      "1189\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1190\n",
      "xxxxx\n",
      "CrossRef\n",
      "1191\n",
      "xxxxx\n",
      "51.\n",
      "1192\n",
      "xxxxx\n",
      "↵\n",
      "1193\n",
      "xxxxx\n",
      "strain PCC 7120.\n",
      "1194\n",
      "xxxxx\n",
      "J. Bacteriol\n",
      "1195\n",
      "xxxxx\n",
      ".,\n",
      "1196\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1197\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1198\n",
      "xxxxx\n",
      "52.\n",
      "1199\n",
      "xxxxx\n",
      "↵\n",
      "1200\n",
      "xxxxx\n",
      "Chen,  S., Zhang,  A., Blyn,  L.B. and Storz,  G. (2004) MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli MicC, a Second Small-RNA Regulator of Omp Protein Expression in Escherichia coli.\n",
      "1201\n",
      "xxxxx\n",
      "J. Bacteriol\n",
      "1202\n",
      "xxxxx\n",
      ".,\n",
      "1203\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1204\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1205\n",
      "xxxxx\n",
      "53.\n",
      "1206\n",
      "xxxxx\n",
      "↵\n",
      "1207\n",
      "xxxxx\n",
      "Vogel,  J. and Luisi,  B.F. (2011) Hfq and its constellation of RNA.\n",
      "1208\n",
      "xxxxx\n",
      "Nat.\n",
      "1209\n",
      "xxxxx\n",
      ".,\n",
      "1210\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1211\n",
      "xxxxx\n",
      "CrossRef\n",
      "1212\n",
      "xxxxx\n",
      "PubMed\n",
      "1213\n",
      "xxxxx\n",
      "54.\n",
      "1214\n",
      "xxxxx\n",
      "↵\n",
      "1215\n",
      "xxxxx\n",
      "Mitarai,  N., Benjamin,  J.-A.M.J.-A.M., Krishna,  S., Semsey,  S., Csiszovszki,  Z., Massé,  E., Sneppen,  K., Masse,  E. and Sneppen,  K. (2009) Dynamic features of gene expression control by small regulatory RNAs.\n",
      "1216\n",
      "xxxxx\n",
      "Proc.\n",
      "1217\n",
      "xxxxx\n",
      ".,\n",
      "1218\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1219\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1220\n",
      "xxxxx\n",
      "55.\n",
      "1221\n",
      "xxxxx\n",
      "↵\n",
      "1222\n",
      "xxxxx\n",
      "Jahn,  M., Vorpahl,  C., Hübschmann,  T., Harms,  H. and Müller,  S. (2016) Copy number variability of expression plasmids determined by cell sorting and droplet digital PCR.\n",
      "1223\n",
      "xxxxx\n",
      "Microb.\n",
      "1224\n",
      "xxxxx\n",
      ".,\n",
      "1225\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1226\n",
      "xxxxx\n",
      "CrossRef\n",
      "1227\n",
      "xxxxx\n",
      "56.\n",
      "1228\n",
      "xxxxx\n",
      "↵\n",
      "1229\n",
      "xxxxx\n",
      "Teimouri,  H., Korkmazhan,  E., Stavans,  J. and Levine,  E. (2017) Sub-cellular mRNA localization modulates the regulation of gene expression by small RNAs in bacteria.\n",
      "1230\n",
      "xxxxx\n",
      "Phys.\n",
      "1231\n",
      "xxxxx\n",
      ".,\n",
      "1232\n",
      "xxxxx\n",
      "57.\n",
      "1233\n",
      "xxxxx\n",
      "↵\n",
      "1234\n",
      "xxxxx\n",
      "Nielsen,  A.A.K., Segall-Shapiro,  T.H. and Voigt,  C.A. (2013) Advances in genetic circuit design:\n",
      "1235\n",
      "xxxxx\n",
      "Curr.\n",
      "1236\n",
      "xxxxx\n",
      ".,\n",
      "1237\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1238\n",
      "xxxxx\n",
      "CrossRef\n",
      "1239\n",
      "xxxxx\n",
      "PubMed\n",
      "1240\n",
      "xxxxx\n",
      "58.\n",
      "1241\n",
      "xxxxx\n",
      "↵\n",
      "1242\n",
      "xxxxx\n",
      "Bradley,  R.W., Buck,  M. and Wang,  B. (2016) Tools and Principles for Microbial Gene Circuit Engineering.\n",
      "1243\n",
      "xxxxx\n",
      "J. Mol.\n",
      "1244\n",
      "xxxxx\n",
      ".,\n",
      "1245\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1246\n",
      "xxxxx\n",
      "59.\n",
      "1247\n",
      "xxxxx\n",
      "(8840)\n",
      "1172\n",
      "xxxxx\n",
      "Neuroscience\n",
      "1173\n",
      "xxxxx\n",
      "(47701)\n",
      "1174\n",
      "xxxxx\n",
      "Paleontology\n",
      "1175\n",
      "xxxxx\n",
      "(352)\n",
      "1176\n",
      "xxxxx\n",
      "Pathology\n",
      "1177\n",
      "xxxxx\n",
      "(1433)\n",
      "1178\n",
      "xxxxx\n",
      "Pharmacology and Toxicology\n",
      "1179\n",
      "xxxxx\n",
      "(2493)\n",
      "1180\n",
      "xxxxx\n",
      "Physiology\n",
      "1181\n",
      "xxxxx\n",
      "(3742)\n",
      "1182\n",
      "xxxxx\n",
      "Plant Biology\n",
      "1183\n",
      "xxxxx\n",
      "(8103)\n",
      "1184\n",
      "xxxxx\n",
      "Scientific Communication and Education\n",
      "1185\n",
      "xxxxx\n",
      "(1439)\n",
      "1186\n",
      "xxxxx\n",
      "Synthetic Biology\n",
      "1187\n",
      "xxxxx\n",
      "(2230)\n",
      "1188\n",
      "xxxxx\n",
      "Systems Biology\n",
      "1189\n",
      "xxxxx\n",
      "(6050)\n",
      "1190\n",
      "xxxxx\n",
      "Zoology\n",
      "1191\n",
      "xxxxx\n",
      "(1259)\n",
      "1192\n",
      "xxxxx\n",
      "↵\n",
      "1248\n",
      "xxxxx\n",
      "Morita,  T. and Aiba,  H. (2019) Mechanism and physiological significance of autoregulation of the Escherichia coli hfq gene.\n",
      "1249\n",
      "xxxxx\n",
      "RNA\n",
      "1250\n",
      "xxxxx\n",
      ", 25, 264-276.\n",
      "1251\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1252\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1253\n",
      "xxxxx\n",
      "60.\n",
      "1254\n",
      "xxxxx\n",
      "↵\n",
      "1255\n",
      "xxxxx\n",
      "Azam,  T.A., Iwata,  A., Nishimura,  A., Ueda,  S. and Ishihama,  A. (1999) Growth phase-dependent variation in protein composition of the Escherichia coli nucleoid.\n",
      "1256\n",
      "xxxxx\n",
      "J. Bacteriol\n",
      "1257\n",
      "xxxxx\n",
      ".,\n",
      "1258\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1259\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1260\n",
      "xxxxx\n",
      "61.\n",
      "1261\n",
      "xxxxx\n",
      "Kajitani,  M., Kato,  A., Wada,  A., Inokuchi,  Y. and Ishihama,  A. (1994) Regulation of the Escherichia coli hfq gene encoding the host factor for phage Q(β).\n",
      "1262\n",
      "xxxxx\n",
      "J. Bacteriol\n",
      "1263\n",
      "xxxxx\n",
      ".,\n",
      "1264\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1265\n",
      "xxxxx\n",
      "Abstract/FREE Full Text\n",
      "1266\n",
      "xxxxx\n",
      "62.\n",
      "1267\n",
      "xxxxx\n",
      "Yoon,  S.H., Han,  M., Jeong,  H., Lee,  C.H., Xia,  X., Lee,  D., Shim,  J.H., Lee,  S.Y., Oh,  T.K. and Kim,  J.F. (2012) Comparative multi-omics systems analysis of Escherichia coli strains B and K-12.\n",
      "1268\n",
      "xxxxx\n",
      "Genome Biol\n",
      "1269\n",
      "xxxxx\n",
      ".,\n",
      "1270\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1271\n",
      "xxxxx\n",
      "CrossRef\n",
      "1272\n",
      "xxxxx\n",
      "PubMed\n",
      "1273\n",
      "xxxxx\n",
      "63.\n",
      "1274\n",
      "xxxxx\n",
      "↵\n",
      "1275\n",
      "xxxxx\n",
      "Moon,  K. and Gottesman,  S. (2011) Competition among Hfq-binding small RNAs in Escherichia coli.\n",
      "1276\n",
      "xxxxx\n",
      "Mol.\n",
      "1277\n",
      "xxxxx\n",
      ".,\n",
      "1278\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1279\n",
      "xxxxx\n",
      "CrossRef\n",
      "1280\n",
      "xxxxx\n",
      "PubMed\n",
      "1281\n",
      "xxxxx\n",
      "64.\n",
      "1282\n",
      "xxxxx\n",
      "↵\n",
      "1283\n",
      "xxxxx\n",
      "Wagner,  E.G.H. (2013) Cycling of RNAs on Hfq.\n",
      "1284\n",
      "xxxxx\n",
      "RNA Biol\n",
      "1285\n",
      "xxxxx\n",
      ".,\n",
      "1286\n",
      "xxxxx\n",
      "OpenUrl\n",
      "1287\n",
      "xxxxx\n",
      "CrossRef\n",
      "1288\n",
      "xxxxx\n",
      "PubMed\n",
      "1289\n",
      "xxxxx\n",
      "Web of Science\n",
      "1290\n",
      "xxxxx\n",
      "View the discussion thread.\n",
      "1291\n",
      "xxxxx\n",
      "Back to top\n",
      "1292\n",
      "xxxxx\n",
      "Previous\n",
      "1293\n",
      "xxxxx\n",
      "Next\n",
      "1294\n",
      "xxxxx\n",
      "Posted January 20, 2022.\n",
      "1295\n",
      "xxxxx\n",
      "Download PDF\n",
      "1296\n",
      "xxxxx\n",
      "Supplementary Material\n",
      "1297\n",
      "xxxxx\n",
      "Email\n",
      "1298\n",
      "xxxxx\n",
      "Thank you for your interest in spreading the word about bioRxiv.\n",
      "1299\n",
      "xxxxx\n",
      "NOTE:\n",
      "1300\n",
      "xxxxx\n",
      "Your Email *\n",
      "1301\n",
      "xxxxx\n",
      "Your Name *\n",
      "1302\n",
      "xxxxx\n",
      "Send To *\n",
      "1303\n",
      "xxxxx\n",
      "Enter multiple addresses on separate lines or separate them with commas.\n",
      "1304\n",
      "xxxxx\n",
      "You are going to email the following\n",
      "1305\n",
      "xxxxx\n",
      "A modular RNA interference system for multiplexed gene regulation\n",
      "1306\n",
      "xxxxx\n",
      "Message Subject\n",
      "1307\n",
      "xxxxx\n",
      "(Your Name) has forwarded a page to you from bioRxiv\n",
      "1308\n",
      "xxxxx\n",
      "Message Body\n",
      "1309\n",
      "xxxxx\n",
      "(Your Name) thought you would like to see this page from the bioRxiv website.\n",
      "1310\n",
      "xxxxx\n",
      "Your Personal Message\n",
      "1311\n",
      "xxxxx\n",
      "CAPTCHA\n",
      "1312\n",
      "xxxxx\n",
      "This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.\n",
      "1313\n",
      "xxxxx\n",
      "Share\n",
      "1314\n",
      "xxxxx\n",
      "A modular RNA interference system for multiplexed gene regulation\n",
      "1315\n",
      "xxxxx\n",
      "Ari Dwijayanti, Marko Storch, Guy-Bart Stan, Geoff S. Baldwin\n",
      "1316\n",
      "xxxxx\n",
      "doi:\n",
      "1317\n",
      "xxxxx\n",
      "Share This Article:\n",
      "1318\n",
      "xxxxx\n",
      "Copy\n",
      "1319\n",
      "xxxxx\n",
      "Citation Tools\n",
      "1320\n",
      "xxxxx\n",
      "A modular RNA interference system for multiplexed gene regulation\n",
      "1321\n",
      "xxxxx\n",
      "Ari Dwijayanti, Marko Storch, Guy-Bart Stan, Geoff S. Baldwin\n",
      "1322\n",
      "xxxxx\n",
      "bioRxiv 2019.12.12.873844;\n",
      "1323\n",
      "xxxxx\n",
      "Citation Manager Formats\n",
      "1324\n",
      "xxxxx\n",
      "BibTeX\n",
      "1325\n",
      "xxxxx\n",
      "Bookends\n",
      "1326\n",
      "xxxxx\n",
      "EasyBib\n",
      "1327\n",
      "xxxxx\n",
      "EndNote (tagged)\n",
      "1328\n",
      "xxxxx\n",
      "EndNote 8 (xml)\n",
      "1329\n",
      "xxxxx\n",
      "Medlars\n",
      "1330\n",
      "xxxxx\n",
      "Mendeley\n",
      "1331\n",
      "xxxxx\n",
      "Papers\n",
      "1332\n",
      "xxxxx\n",
      "RefWorks Tagged\n",
      "1333\n",
      "xxxxx\n",
      "Ref Manager\n",
      "1334\n",
      "xxxxx\n",
      "RIS\n",
      "1335\n",
      "xxxxx\n",
      "Zotero\n",
      "1336\n",
      "xxxxx\n",
      "Tweet Widget\n",
      "1337\n",
      "xxxxx\n",
      "Facebook Like\n",
      "1338\n",
      "xxxxx\n",
      "Google Plus One\n",
      "1339\n",
      "xxxxx\n",
      "Subject Area\n",
      "1340\n",
      "xxxxx\n",
      "Synthetic Biology\n",
      "1341\n",
      "xxxxx\n",
      "Subject Areas\n",
      "1342\n",
      "xxxxx\n",
      "All Articles\n",
      "1343\n",
      "xxxxx\n",
      "Animal Behavior and Cognition\n",
      "1344\n",
      "xxxxx\n",
      "(4123)\n",
      "1345\n",
      "xxxxx\n",
      "Biochemistry\n",
      "1346\n",
      "xxxxx\n",
      "(8837)\n",
      "1347\n",
      "xxxxx\n",
      "Bioengineering\n",
      "1348\n",
      "xxxxx\n",
      "(6539)\n",
      "1349\n",
      "xxxxx\n",
      "Bioinformatics\n",
      "1350\n",
      "xxxxx\n",
      "(23503)\n",
      "1351\n",
      "xxxxx\n",
      "Biophysics\n",
      "1352\n",
      "xxxxx\n",
      "(11822)\n",
      "1353\n",
      "xxxxx\n",
      "Cancer Biology\n",
      "1354\n",
      "xxxxx\n",
      "(9242)\n",
      "1355\n",
      "xxxxx\n",
      "Cell Biology\n",
      "1356\n",
      "xxxxx\n",
      "(13358)\n",
      "1357\n",
      "xxxxx\n",
      "Clinical Trials\n",
      "1358\n",
      "xxxxx\n",
      "(138)\n",
      "1359\n",
      "xxxxx\n",
      "Developmental Biology\n",
      "1360\n",
      "xxxxx\n",
      "(7456)\n",
      "1361\n",
      "xxxxx\n",
      "Ecology\n",
      "1362\n",
      "xxxxx\n",
      "(11438)\n",
      "1363\n",
      "xxxxx\n",
      "Epidemiology\n",
      "1364\n",
      "xxxxx\n",
      "(2066)\n",
      "1365\n",
      "xxxxx\n",
      "Evolutionary Biology\n",
      "1366\n",
      "xxxxx\n",
      "(15186)\n",
      "1367\n",
      "xxxxx\n",
      "Genetics\n",
      "1368\n",
      "xxxxx\n",
      "(10460)\n",
      "1369\n",
      "xxxxx\n",
      "Genomics\n",
      "1370\n",
      "xxxxx\n",
      "(14058)\n",
      "1371\n",
      "xxxxx\n",
      "Immunology\n",
      "1372\n",
      "xxxxx\n",
      "(9196)\n",
      "1373\n",
      "xxxxx\n",
      "Microbiology\n",
      "1374\n",
      "xxxxx\n",
      "(22242)\n",
      "1375\n",
      "xxxxx\n",
      "Molecular Biology\n",
      "1376\n",
      "xxxxx\n",
      "(8840)\n",
      "1377\n",
      "xxxxx\n",
      "Neuroscience\n",
      "1378\n",
      "xxxxx\n",
      "(47702)\n",
      "1379\n",
      "xxxxx\n",
      "Paleontology\n",
      "1380\n",
      "xxxxx\n",
      "(352)\n",
      "1381\n",
      "xxxxx\n",
      "Pathology\n",
      "1382\n",
      "xxxxx\n",
      "(1433)\n",
      "1383\n",
      "xxxxx\n",
      "Pharmacology and Toxicology\n",
      "1384\n",
      "xxxxx\n",
      "(2493)\n",
      "1385\n",
      "xxxxx\n",
      "Physiology\n",
      "1386\n",
      "xxxxx\n",
      "(3742)\n",
      "1387\n",
      "xxxxx\n",
      "Plant Biology\n",
      "1388\n",
      "xxxxx\n",
      "(8103)\n",
      "1389\n",
      "xxxxx\n",
      "Scientific Communication and Education\n",
      "1390\n",
      "xxxxx\n",
      "(1439)\n",
      "1391\n",
      "xxxxx\n",
      "Synthetic Biology\n",
      "1392\n",
      "xxxxx\n",
      "(2230)\n",
      "1393\n",
      "xxxxx\n",
      "Systems Biology\n",
      "1394\n",
      "xxxxx\n",
      "(6050)\n",
      "1395\n",
      "xxxxx\n",
      "Zoology\n",
      "1396\n",
      "xxxxx\n",
      "(1259)\n",
      "1397\n",
      "xxxxx\n"
     ]
    }
   ],
   "source": [
    "for i in session.query(Paragraph).all():\n",
    "    print(i.sentences[0].text)\n",
    "    print(i.position)\n",
    "    print(\"xxxxx\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = 0\n",
    "for mention in session.query(Task).all():\n",
    "    print(mention.context.get_span())\n",
    "    #print(\"\\n\")\n",
    "    if a > 100:\n",
    "        break\n",
    "    a += 1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['__bool__', '__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__']\n"
     ]
    }
   ],
   "source": [
    "a = session.query(Paragraph).get(3325)\n",
    "\n",
    "print(dir(a))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "66205\n",
      "66209\n",
      "21\n",
      "----\n",
      "1\n",
      "66205\n",
      "66211\n",
      "22 23 24 25 26!\n",
      "----\n",
      "2\n",
      "66205\n",
      "66213\n",
      "35\n",
      "----\n",
      "3\n",
      "66205\n",
      "66215\n",
      "36 37 38 39 40.\n",
      "----\n",
      "0\n",
      "66206\n",
      "66210\n",
      "1\n",
      "----\n",
      "1\n",
      "66206\n",
      "66212\n",
      "2 3 4 5 6!\n",
      "----\n",
      "2\n",
      "66206\n",
      "66214\n",
      "15\n",
      "----\n",
      "3\n",
      "66206\n",
      "66216\n",
      "16 17 18 19 20.\n",
      "----\n"
     ]
    }
   ],
   "source": [
    "for a in session.query(Paragraph).all():\n",
    "    print(a.position)\n",
    "    print(a.document_id)\n",
    "    print(a.id)\n",
    "    print(a.sentences[0].text)\n",
    "    print(\"----\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 229,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1869\n",
      "1870\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<generator object get_neighbor_sentence_ngrams at 0x7f456fb490d0>"
      ]
     },
     "execution_count": 229,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = session.query(Task).all()\n",
    "for b in a:\n",
    "    print(b.id)\n",
    "\n",
    "\n",
    "# b = a.sentences[0].text\n",
    "\n",
    "get_neighbor_sentence_ngrams(b)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Bla Blub.'"
      ]
     },
     "execution_count": 214,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = session.query(Sentence).get(29011)\n",
    "a.text\n",
    "a.get_neighbor_sentence_ngrams(window = 1, n= 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'NoneType' object has no attribute 'sentence_id'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_813606/3424386646.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0mmention\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mquery\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mTask\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfilter\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mTask\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mid\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mmention_id\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfirst\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 6\u001b[0;31m \u001b[0msentence\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msession\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mquery\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mSentence\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfilter\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mSentence\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mid\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mmention\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msentence_id\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfirst\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      7\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      8\u001b[0m \u001b[0mngrams\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msentence\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_neighbor_sentence_ngrams\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mwindow\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mwindow\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mn\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mn\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mAttributeError\u001b[0m: 'NoneType' object has no attribute 'sentence_id'"
     ]
    }
   ],
   "source": [
    "from fonduer.candidates.models import candidate_subclass\n",
    "from definitions.mentions.mention_subclasses import NameFull, NameAbbr, Task\n",
    "\n",
    "NameFullAbbr = candidate_subclass(\"NameFullAbbr\", [NameFull, NameAbbr])\n",
    "NameFullTask = candidate_subclass(\"NameFullTask\", [NameFull, Task])\n",
    "\n",
    "from fonduer.candidates import CandidateExtractor\n",
    "from fonduer.parser.models import Document\n",
    "from pipeline.utils import split_documents, get_session, PARALLEL\n",
    "from pipeline.throttler.NameFullTask import NameFullTask_throttlers\n",
    "\n",
    "\n",
    "candidate_extractor = CandidateExtractor(\n",
    "                                session,\n",
    "                                [NameFullAbbr,NameFullTask], #NameFullAbbr,\n",
    "                                throttlers=NameFullTask_throttlers,\n",
    "                            )\n",
    "\n",
    "# for split, docs in enumerate(split_documents(session)):\n",
    "for split, docs in enumerate(session.query(Document).all()): \n",
    "    docs = [docs]\n",
    "    print(f\"\\nSplit {split}: {len(docs)} documents\")\n",
    "    candidate_extractor.apply(docs, split=split, parallelism=PARALLEL)\n",
    "    print(f\"Split {split}: Number of NameFull + NameAbbrv candidates:\", session.query(NameFullAbbr).filter(NameFullAbbr.split==split).count())\n",
    "    print(f\"Split {split}: Number of NameFull + Task candidates:\", session.query(NameFullTask).filter(NameFullTask.split==split).count())\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Testbereich\n",
    "\n",
    "def mention_span_in_acknowledments_matches_verb(mention):\n",
    "    # read in span/word\n",
    "    span_string = mention\n",
    "\n",
    "    # Task(SpanMention(\"RefWorks Tagged\", sentence=61332, chars=[0,14], words=[0,1]))\n",
    "    \n",
    "    for word in span_string.split(): # mention.get_span()[0]\n",
    "        #print(word)\n",
    "        word_filtered = word.replace(\"!\",\"\")\n",
    "        try:\n",
    "            if wn.synsets(word_filtered)[0].pos() == \"v\":\n",
    "                #print(word)\n",
    "                return word_filtered\n",
    "        except:\n",
    "            pass\n",
    "    return False\n",
    "\n",
    "a = \"I played soccer!\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'played'"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mention_span_in_acknowledments_matches_verb(a)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'n'"
      ]
     },
     "execution_count": 109,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wn.synsets(\"want\")[0].pos()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "n\n",
      "privation\n",
      "n\n",
      "lack\n",
      "n\n",
      "need\n",
      "n\n",
      "wish\n",
      "v\n",
      "desire\n",
      "v\n",
      "want\n",
      "v\n",
      "want\n",
      "v\n",
      "want\n",
      "v\n",
      "want\n"
     ]
    }
   ],
   "source": [
    "for a in wn.synsets(\"want\"):\n",
    "    print(a.pos())\n",
    "    print(a.name().split(\".\")[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "    sentence_of_span = mention.sentence.text\n",
    "    #print(sentence_of_span)\n",
    "    #x = mention._get_table()\n",
    "    # print(mention.sentence.paragraph_id)\n",
    "    # #print(mention.document_id)\n",
    "    # print(\"x\")\n",
    "    # print(mention.sentence.paragraph_id-1)\n",
    "    #print(mention.sentence.paragraph_id-1)\n",
    "    # a = session.query(Paragraph).get(mention.sentence.paragraph_id-1)\n",
    "    #print(a.sentences[0].text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 186,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "F\n"
     ]
    }
   ],
   "source": [
    "\n",
    "# if any in \n",
    "\n",
    "a = [\"abc\", \"bbb\", \"adf\"]\n",
    "headline_of_last_paragraph = \"Acknowledgment\"\n",
    "\n",
    "if any(option in headline_of_last_paragraph for option in list_of_headlines):\n",
    "    print(\"T\")\n",
    "else:\n",
    "    print(\"F\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "a = \"Auto\"\n",
    "b = \"auto\"\n",
    "print(b.islower())"
   ]
  },
  {
   "attachments": {},
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Idee\n",
    "Throttler\n",
    "    - alle candidates mit task mention die Groß geschrieben sind aussortieren\n",
    "        - Fall Satzanfang: sollten keine Verben stehen???\n",
    "\n",
    "# Probleme\n",
    "1. get_neighbor_sentence_ngrams\n",
    "bekomme mit 'get_neighbor_sentence_ngrams' keine Paragraph Sprünge hin\n",
    "    - Alternativweg: n_max riesig groß wählen, den ngram durchiterieren, bis eine andere paragraph_id vorkommt\n",
    "    - weiter: ersten satz dieser paragraph_id auf 'acknowledgment' checken\n",
    "    - dann auf verb testen\n",
    "\n",
    "2. PARALLEL > 1 Problem für paragraph_id\n",
    "- Ursprungsproblem: Paragraph_ids werden anders verteilt bei PARALLEL > 1\n",
    "    - vorherige id ist nicht id-1\n",
    "\n",
    "3. Bedenken: zu viel im Throttler\n",
    "- sortiert alles außerhalb des gewünschten paragraphen aus\n",
    "- check auf verb (vielleicht erst im throttler)\n",
    "\n",
    "\n",
    "# Getan\n",
    "Matcher geschrieben, der bei Parallel = 1 funktioniert\n",
    "- guckt ob eine Form von 'acknowledgment', 'contribution', etc. im ersten Satz (Überschrift) des vorherigen Paragraphen steht\n",
    "- checkt ob eingelesenes Wort ein Verb ist\n",
    "    - importiert nltk.corpus\n",
    "    - synset-wortlist für jeden span, checken ob pos() == \"v\" (word ist ein Verb) -> match\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.16"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
