<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.16.951657v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.16.951657v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.16.951657v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/07/30/2020.02.16.951657.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.16.951657v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.16.951657"/>
  <meta name="DC.Date" content="2022-07-30"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background The eukaryotic genome is capable of producing multiple isoforms from a gene by alternative polyadenylation (APA) during pre-mRNA processing. APA in the 3’-untranslated region (3’-UTR) of mRNA produces transcripts with shorter or longer 3’-UTR. Often, 3’-UTR serves as a binding platform for microRNAs and RNA-binding proteins, which affect the fate of the mRNA transcript. Thus, 3’-UTR APA is known to modulate translation and provides a mean to regulate gene expression at the post-transcriptional level. Current bioinformatics pipelines have limited capability in profiling 3’-UTR APA events due to incomplete annotations and a low-resolution analyzing power: widely available bioinformatics pipelines do not reference actionable polyadenylation (cleavage) sites but simulate 3’-UTR APA only using RNA-seq read coverage, causing false positive identifications. To overcome these limitations, we developed APA-Scan, a robust program that identifies 3’-UTR APA events and visualizes the RNA-seq short-read coverage with gene annotations.

Methods APA-Scan utilizes either predicted or experimentally validated actionable polyadenylation signals as a reference for polyadenylation sites and calculates the quantity of long and short 3’-UTR transcripts in the RNA-seq data. APA-Scan works in three major steps: (i) calculate the read coverage of the 3’-UTR regions of genes; (ii) identify the potential APA sites and evaluate the significance of the events among two biological conditions; (iii) graphical representation of user specific event with 3’-UTR annotation and read coverage on the 3’-UTR regions. APA-Scan is implemented in Python3. Source code and a comprehensive user’s manual are freely available at <https://github.com/compbiolabucf/APA-Scan>.

Result APA-Scan was applied to both simulated and real RNA-seq datasets and compared with two widely used baselines DaPars and APAtrap. In simulation APA-Scan significantly improved the accuracy of 3’-UTR APA identification compared to the other baselines. The performance of APA-Scan was also validated by 3’-end-seq data and qPCR on mouse embryonic fibroblast cells. The experiments confirm that APA-Scan can detect unannotated 3’ -UTR APA events and improve genome annotation.

Conclusion APA-Scan is a comprehensive computational pipeline to detect transcriptome-wide 3’-UTR APA events. The pipeline integrates both RNA-seq and 3’-end-seq data information and can efficiently identify the significant events with a high-resolution short reads coverage plots.

### Competing Interest Statement

The authors have declared no competing interest.

*   APA
    :   alternative polyadenylation
    3’-UTR
    :   3’-untranslated region
    mTOR
    :   mechanistic target of rapamycin
    PAS
    :   polyadenylation signal
    MEFs
    :   mouse embryonic fibroblasts
    WT
    :   wild-type
    ROC
    :   receiver operating characteristic
    AUC
    :   area under the ROC curve
    qPCR
    :   quantitative polymerase chain reaction"/>
  <meta name="DC.Contributor" content="Naima Ahmed Fahmi"/>
  <meta name="DC.Contributor" content="Khandakar Tanvir Ahmed"/>
  <meta name="DC.Contributor" content="Jae-Woong Chang"/>
  <meta name="DC.Contributor" content="Heba Nassereddeen"/>
  <meta name="DC.Contributor" content="Deliang Fan"/>
  <meta name="DC.Contributor" content="Jeongsik Yong"/>
  <meta name="DC.Contributor" content="Wei Zhang"/>
  <meta name="article:published_time" content="2022-07-30"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<h3>Background</h3>
<p>The eukaryotic genome is capable of producing multiple isoforms from a gene by alternative polyadenylation (APA) during pre-mRNA processing. APA in the 3’-untranslated region (3’-UTR) of mRNA produces transcripts with shorter or longer 3’-UTR. Often, 3’-UTR serves as a binding platform for microRNAs and RNA-binding proteins, which affect the fate of the mRNA transcript. Thus, 3’-UTR APA is known to modulate translation and provides a mean to regulate gene expression at the post-transcriptional level. Current bioinformatics pipelines have limited capability in profiling 3’-UTR APA events due to incomplete annotations and a low-resolution analyzing power: widely available bioinformatics pipelines do not reference actionable polyadenylation (cleavage) sites but simulate 3’-UTR APA only using RNA-seq read coverage, causing false positive identifications. To overcome these limitations, we developed APA-Scan, a robust program that identifies 3’-UTR APA events and visualizes the RNA-seq short-read coverage with gene annotations.</p><h3>Methods</h3>
<p>APA-Scan utilizes either predicted or experimentally validated actionable polyadenylation signals as a reference for polyadenylation sites and calculates the quantity of long and short 3’-UTR transcripts in the RNA-seq data. APA-Scan works in three major steps: (i) calculate the read coverage of the 3’-UTR regions of genes; (ii) identify the potential APA sites and evaluate the significance of the events among two biological conditions; (iii) graphical representation of user specific event with 3’-UTR annotation and read coverage on the 3’-UTR regions. APA-Scan is implemented in Python3. Source code and a comprehensive user’s manual are freely available at https://github.com/compbiolabucf/APA-Scan.</p><h3>Result</h3>
<p>APA-Scan was applied to both simulated and real RNA-seq datasets and compared with two widely used baselines DaPars and APAtrap. In simulation APA-Scan significantly improved the accuracy of 3’-UTR APA identification compared to the other baselines. The performance of APA-Scan was also validated by 3’-end-seq data and qPCR on mouse embryonic fibroblast cells. The experiments confirm that APA-Scan can detect unannotated 3’ -UTR APA events and improve genome annotation.</p><h3>Conclusion</h3>
<p>APA-Scan is a comprehensive computational pipeline to detect transcriptome-wide 3’-UTR APA events. The pipeline integrates both RNA-seq and 3’-end-seq data information and can efficiently identify the significant events with a high-resolution short reads coverage plots.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.16.951657v3"/>
  <meta name="citation_id" content="2020.02.16.951657v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.16.951657"/>
  <meta name="citation_num_pages" content="35"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.16.951657"/>
  <meta name="citation_author" content="Naima Ahmed Fahmi"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of Central Florida"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2745-2130"/>
  <meta name="citation_author" content="Khandakar Tanvir Ahmed"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of Central Florida"/>
  <meta name="citation_author" content="Jae-Woong Chang"/>
  <meta name="citation_author_institution" content="Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota Twin Cities"/>
  <meta name="citation_author" content="Heba Nassereddeen"/>
  <meta name="citation_author_institution" content="Department of Computer Engineering, University of Central Florida"/>
  <meta name="citation_author" content="Deliang Fan"/>
  <meta name="citation_author_institution" content="School of Electrical, Computer and Energy Engineering, Arizona State University"/>
  <meta name="citation_author" content="Jeongsik Yong"/>
  <meta name="citation_author_institution" content="Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota Twin Cities"/>
  <meta name="citation_author" content="Wei Zhang"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of Central Florida"/>
  <meta name="citation_author_email" content="wzhang.cs@ucf.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3605-9373"/>
  <meta name="citation_reference" content="Proudfoot, N.J.: Ending the message: poly (A) signals then and now. Genes & development 25(17), 1770–1782 (2011)"/>
  <meta name="citation_reference" content="Tian, B., Manley, J.L.: Alternative cleavage and polyadenylation: the long and short of it. Trends in Biochemical Sciences 38(6), 312–320 (2013)"/>
  <meta name="citation_reference" content="Elkon, R., Ugalde, A.P., Agami, R.: Alternative cleavage and polyadenylation: extent, regulation and function. Nature Reviews Genetics 14(7), 496 (2013)"/>
  <meta name="citation_reference" content="Yeh, H.-S., Zhang, W., Yong, J.: Analyses of alternative polyadenylation: from old school biochemistry to high-throughput technologies. BMB reports 50(4), 201 (2017)"/>
  <meta name="citation_reference" content="Mayr, C., Bartel, D.P.: Widespread shortening of 3’ UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells. Cell 138(4), 673–684 (2009)"/>
  <meta name="citation_reference" content="Lembo, A., Di Cunto, F., Provero, P.: Shortening of 3 UTRs correlates with poor prognosis in breast and lung cancer. PloS one 7(2), 31129 (2012)"/>
  <meta name="citation_reference" content="Morris, A.R., Bos, A., Diosdado, B., Rooijers, K., Elkon, R., Bolijn, A.S., Carvalho, B., Meijer, G.A., Agami, R.: Alternative cleavage and polyadenylation during colorectal cancer development. Clinical Cancer Research 18(19), 5256–5266 (2012)"/>
  <meta name="citation_reference" content="Chang, J.-W., Zhang, W., Yeh, H.-S., De Jong, E.P., Jun, S., Kim, K.-H., Bae, S.S., Beckman, K., Hwang, T.H., Kim, K.-S., et al.: mRNA 3-UTR shortening is a molecular signature of mTORC1 activation. Nature communications 6(1), 1–9 (2015)"/>
  <meta name="citation_reference" content="Chang, J.-W., Zhang, W., Yeh, H.-S., Park, M., Yao, C., Shi, Y., Kuang, R., Yong, J.: An integrative model for alternative polyadenylation, IntMAP, delineates mTOR-modulated endoplasmic reticulum stress response. Nucleic Acids Research 46(12), 5996–6008 (2018)"/>
  <meta name="citation_reference" content="Hoffman, Y., Bublik, D.R., P. Ugalde, A., Elkon, R., Biniashvili, T., Agami, R., Oren, M., Pilpel, Y.: 3’UTR shortening potentiates microRNA-based repression of pro-differentiation genes in proliferating human cells. PLoS genetics 12(2), 1005879 (2016)"/>
  <meta name="citation_reference" content="Sandberg, R., Neilson, J.R., Sarma, A., Sharp, P.A., Burge, C.B.: Proliferating cells express mRNAs with shortened 3’untranslated regions and fewer microRNA target sites. Science 320(5883), 1643–1647 (2008)"/>
  <meta name="citation_reference" content="Xia, Z., Donehower, L.A., Cooper, T.A., Neilson, J.R., Wheeler, D.A., Wagner, E.J., Li, W.: Dynamic Analyses of Alternative Polyadenylation from RNA-Seq Reveal 3’-UTR Landscape Across 7 Tumor Types. Nature Communications 5, 5274 (2014)"/>
  <meta name="citation_reference" content="Wang, W., Wei, Z., Li, H.: A change-point model for identifying 3’ UTR switching by nextgeneration RNA sequencing. Bioinformatics 30(15), 2162–2170 (2014)"/>
  <meta name="citation_reference" content="Le Pera, L., Mazzapioda, M., Tramontano, A.: 3USS: a web server for detecting alternative 3’ UTRs from RNA-seq experiments. Bioinformatics 31(11), 1845–1847 (2015)"/>
  <meta name="citation_reference" content="Ye, C., Long, Y., Ji, G., Li, Q.Q., Wu, X.: APAtrap: identification and quantification of alternative polyadenylation sites from RNA-seq data. Bioinformatics 34(11), 1841–1849 (2018)"/>
  <meta name="citation_reference" content="Magana-Mora, A., Kalkatawi, M., Bajic, V.B.: Omni-PolyA: a method and tool for accurate recognition of Poly(A) signals in human genomic DNA. BMC genomics 18(1), 1–13 (2017)"/>
  <meta name="citation_reference" content="Shepard, P.J., Choi, E.-A., Lu, J., Flanagan, L.A., Hertel, K.J., Shi, Y.: Complex and dynamic landscape of rna polyadenylation revealed by pas-seq. Rna 17(4), 761–772 (2011)"/>
  <meta name="citation_reference" content="Griebel, T., Zacher, B., Ribeca, P., Raineri, E., Lacroix, V., Guigó, R., Sammeth, M.: Modelling and simulating generic RNA-Seq experiments with the flux simulator. Nucleic acids research 40(20), 10073–10083 (2012)"/>
  <meta name="citation_reference" content="Oshlack, A., Robinson, M.D., Young, M.D.: From RNA-seq reads to differential expression results. Genome biology 11(12), 220 (2010)"/>
  <meta name="citation_reference" content="Gruber, A.J., Schmidt, R., Gruber, A.R., Martin, G., Ghosh, S., Belmadani, M., Keller, W., Zavolan, M.: A comprehensive analysis of 3 end sequencing data sets reveals novel polyadenylation signals and the repressive role of heterogeneous ribonucleoprotein C on cleavage and polyadenylation. Genome research 26(8), 1145–1159 (2016)"/>
  <meta name="citation_reference" content="Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R.: The sequence alignment/map format and SAMtools. Bioinformatics 25(16), 2078–2079 (2009)"/>
  <meta name="citation_reference" content="Kim, D., Pertea, G., Trapnell, C., Pimentel, H., Kelley, R., Salzberg, S.L.: TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome biology 14(4), 1–13 (2013)"/>
  <meta name="citation_reference" content="Langmead, B., Trapnell, C., Pop, M., Salzberg, S.L.: Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome biology 10(3), 1–10 (2009)"/>
  <meta name="twitter:title" content="APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background The eukaryotic genome is capable of producing multiple isoforms from a gene by alternative polyadenylation (APA) during pre-mRNA processing. APA in the 3’-untranslated region (3’-UTR) of mRNA produces transcripts with shorter or longer 3’-UTR. Often, 3’-UTR serves as a binding platform for microRNAs and RNA-binding proteins, which affect the fate of the mRNA transcript. Thus, 3’-UTR APA is known to modulate translation and provides a mean to regulate gene expression at the post-transcriptional level. Current bioinformatics pipelines have limited capability in profiling 3’-UTR APA events due to incomplete annotations and a low-resolution analyzing power: widely available bioinformatics pipelines do not reference actionable polyadenylation (cleavage) sites but simulate 3’-UTR APA only using RNA-seq read coverage, causing false positive identifications. To overcome these limitations, we developed APA-Scan, a robust program that identifies 3’-UTR APA events and visualizes the RNA-seq short-read coverage with gene annotations.

Methods APA-Scan utilizes either predicted or experimentally validated actionable polyadenylation signals as a reference for polyadenylation sites and calculates the quantity of long and short 3’-UTR transcripts in the RNA-seq data. APA-Scan works in three major steps: (i) calculate the read coverage of the 3’-UTR regions of genes; (ii) identify the potential APA sites and evaluate the significance of the events among two biological conditions; (iii) graphical representation of user specific event with 3’-UTR annotation and read coverage on the 3’-UTR regions. APA-Scan is implemented in Python3. Source code and a comprehensive user’s manual are freely available at <https://github.com/compbiolabucf/APA-Scan>.

Result APA-Scan was applied to both simulated and real RNA-seq datasets and compared with two widely used baselines DaPars and APAtrap. In simulation APA-Scan significantly improved the accuracy of 3’-UTR APA identification compared to the other baselines. The performance of APA-Scan was also validated by 3’-end-seq data and qPCR on mouse embryonic fibroblast cells. The experiments confirm that APA-Scan can detect unannotated 3’ -UTR APA events and improve genome annotation.

Conclusion APA-Scan is a comprehensive computational pipeline to detect transcriptome-wide 3’-UTR APA events. The pipeline integrates both RNA-seq and 3’-end-seq data information and can efficiently identify the significant events with a high-resolution short reads coverage plots.

### Competing Interest Statement

The authors have declared no competing interest.

*   APA
    :   alternative polyadenylation
    3’-UTR
    :   3’-untranslated region
    mTOR
    :   mechanistic target of rapamycin
    PAS
    :   polyadenylation signal
    MEFs
    :   mouse embryonic fibroblasts
    WT
    :   wild-type
    ROC
    :   receiver operating characteristic
    AUC
    :   area under the ROC curve
    qPCR
    :   quantitative polymerase chain reaction"/>
  <meta name="og-title" property="og:title" content="APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background The eukaryotic genome is capable of producing multiple isoforms from a gene by alternative polyadenylation (APA) during pre-mRNA processing. APA in the 3’-untranslated region (3’-UTR) of mRNA produces transcripts with shorter or longer 3’-UTR. Often, 3’-UTR serves as a binding platform for microRNAs and RNA-binding proteins, which affect the fate of the mRNA transcript. Thus, 3’-UTR APA is known to modulate translation and provides a mean to regulate gene expression at the post-transcriptional level. Current bioinformatics pipelines have limited capability in profiling 3’-UTR APA events due to incomplete annotations and a low-resolution analyzing power: widely available bioinformatics pipelines do not reference actionable polyadenylation (cleavage) sites but simulate 3’-UTR APA only using RNA-seq read coverage, causing false positive identifications. To overcome these limitations, we developed APA-Scan, a robust program that identifies 3’-UTR APA events and visualizes the RNA-seq short-read coverage with gene annotations.

Methods APA-Scan utilizes either predicted or experimentally validated actionable polyadenylation signals as a reference for polyadenylation sites and calculates the quantity of long and short 3’-UTR transcripts in the RNA-seq data. APA-Scan works in three major steps: (i) calculate the read coverage of the 3’-UTR regions of genes; (ii) identify the potential APA sites and evaluate the significance of the events among two biological conditions; (iii) graphical representation of user specific event with 3’-UTR annotation and read coverage on the 3’-UTR regions. APA-Scan is implemented in Python3. Source code and a comprehensive user’s manual are freely available at <https://github.com/compbiolabucf/APA-Scan>.

Result APA-Scan was applied to both simulated and real RNA-seq datasets and compared with two widely used baselines DaPars and APAtrap. In simulation APA-Scan significantly improved the accuracy of 3’-UTR APA identification compared to the other baselines. The performance of APA-Scan was also validated by 3’-end-seq data and qPCR on mouse embryonic fibroblast cells. The experiments confirm that APA-Scan can detect unannotated 3’ -UTR APA events and improve genome annotation.

Conclusion APA-Scan is a comprehensive computational pipeline to detect transcriptome-wide 3’-UTR APA events. The pipeline integrates both RNA-seq and 3’-end-seq data information and can efficiently identify the significant events with a high-resolution short reads coverage plots.

### Competing Interest Statement

The authors have declared no competing interest.

*   APA
    :   alternative polyadenylation
    3’-UTR
    :   3’-untranslated region
    mTOR
    :   mechanistic target of rapamycin
    PAS
    :   polyadenylation signal
    MEFs
    :   mouse embryonic fibroblasts
    WT
    :   wild-type
    ROC
    :   receiver operating characteristic
    AUC
    :   area under the ROC curve
    qPCR
    :   quantitative polymerase chain reaction"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-07-30"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2670172"/>
  <title>
   APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GN8CF9U2xtKqmk-fd-zQ6MOFvAUiIHFZ5W07mwIt9Tc"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.02.16.951657v3"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.02.16.951657v3"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.16.951657v3"}],"nid":"2670172","apath":"\/biorxiv\/early\/2022\/07\/30\/2020.02.16.951657.atom","pisa":"biorxiv;2020.02.16.951657v3","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.16.951657v3","title":"APA-Scan: Detection and Visualization of 3\u2019-UTR Alternative Polyadenylation with RNA-seq and 3\u2019-end-seq Data","identifier":"node\/2670172"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qiE03JFyoyA+jMPA4ETCRAasFvUJ2W4+fYxiw6NWH8bo+LiFRLcIf760tqv3NBsQXcTDxR5Xtqdtk5a43rn+VKC1t0x1vtDsyMJq4pFSukzMF\\\\\\\/dvKDljCieWb1CGj6\\\\\\\/u6dz0CeI33NDe+Sr06B1l5Y8rFs9Fd74vbVSvtg\\\\\\\/omb8kLb75C\\\\\\\/dGdl+7IiRqto2Ax+UDT0\\\\\\\/IM\\\\\\\/Er7y3TuXGzkDftGQFBE+iSEfgbWwZrcBKk6adZau4EJgIfrnlK4WK3kBfdYy22x7uEEAZxW1LRh5JXS43TcQp6UoF7gZNTPjlH+NbAw4TSIR3T2GCcsax\\\\\\\/VQKLZykY6xh+0DVt3djqY33Zxp1G7Lly2Q9N3hyFEYVkGAvouwzEk2n59qt\\\\\\\/r\\\\\\\/oIueas8SJAgRgWDflEbC5bNik03nH9uEq1Oh3elxzBNmIK7RqgAgJZvkz6Z9QriAPp6Oy\\\\\\\/Q9mHr8fv0B1+VZEoNHuxfZAUpkpbfipYhYOkC08YTYT1xyzm\\\\\\\/Bm5rysBldKd\\\\\\\/akGUnA0WlM9bP+S28yqA\\\\\\\/suHQoHoYobpwfc1ZOtgZcFPiJNFpfqQhh2uetFwz9fzruungG22thx0w15Jtyyl5OZg5CWAn+3f0fbB64Bu1xg+880Frx\\\\\\\/ORCT\\\\\\\/dzur4lFSgwXTt0sO6Cnp28HdSx26lhrmlx87H6jeGLHyykQsRuVXXzf5VWYqhyHR6FX80PzhwHdukrKgRkKYPutVKpnSxwmq+XYasBoTHWmT7mx\\\\\\\/TQPfR7SsJ4hyWOaytI2Nii+dxRuP09dM0ACdNx8EMjamI+MB+cEyz\\\\\\\/NpFI6ryXXYIb2G3LgcQzxSbAVMLtdrYcdPVHE5S4Y9VPH7+lOwbE44mScczpgr4NprzSsVqALcCBVHjMCk1lZtpHTPPvnIFNJLYV2Yx4bobaykVnnGz4bIX6wOA\\\\\\\/\\\\\\\/x6QvgTeCTb4zE2DsBQsp1BRdDiApzY222ZxaA0\\\\\\\/Q10kcig2yyRwOg5JsZgqYfX7egHiYM1xj7Fd+jQhA834HQ\\\\\\\/ic7cY9JAApLH0F4tSRkXjPt8d7FvdShE7p6Pb1q2vQtMREXNR6QhoP7t31Suo5PnnxGyb2FxY\\\\\\\/EQXjXsn4spm0b6O\\\\\\\/gdrTePbSdqc+0gxV+nxdzrQyF5lTd5OBEzKnN2XQlap5HZa51EmCr\\\\\\\/ypCrX+7JJuyM4jMD9njH6Lg6IfZ2kxgQhmJomZC1I1G7t9lYvYQZIeikwY8wYt5obExVqeyQlqal3\\\\\\\/uebPMrdw8PEkMH0vcqRtBeYp0N\\\\\\\/XbYuyz3mpWx8+towLE6QTa14PCXNIfzoBUxlOxyXVSnTuSMa2ZAe6Wtt\\\\\\\/jxrdyICnn+0h13pNjVaByVR70lshmcD0cSjNwkDCn+Ed1ciMoIEQlg1xjumiaEjNF3aWiCM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022D2Yol1tMt+oFxB64V2nKLw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u002244fe565b2a944064c58c5a8e89395626e8428472695e0cb89d45ce18b7d0c23a\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.16.951657v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2670172 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.16.951657v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1026435706">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1026435706" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1253299612" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-tLl6agCmlyJH5G2H3PdpX74pucp2um5Ze9P-jlSC1NA"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2670172" data-node-nid="2670172" id="node-2670172--2375582701" data-pisa="biorxiv;2020.02.16.951657v3" data-pisa-master="biorxiv;2020.02.16.951657" data-apath="/biorxiv/early/2022/07/30/2020.02.16.951657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-2745-2130" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Naima Ahmed
                      </span>
                      <span class="nlm-surname">
                       Fahmi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Khandakar Tanvir
                      </span>
                      <span class="nlm-surname">
                       Ahmed
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Jae-Woong
                      </span>
                      <span class="nlm-surname">
                       Chang
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Heba
                      </span>
                      <span class="nlm-surname">
                       Nassereddeen
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Deliang
                      </span>
                      <span class="nlm-surname">
                       Fan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Jeongsik
                      </span>
                      <span class="nlm-surname">
                       Yong
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0003-3605-9373" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Wei
                      </span>
                      <span class="nlm-surname">
                       Zhang
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.02.16.951657
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2670172--2375582701" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Naima Ahmed Fahmi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of Central Florida
                       </span>
                       , 4000 Central Florida Blvd, Orlando, FL 32816,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Naima%2BAhmed%2BFahmi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fahmi%20NA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ANaima%2BAhmed%2BFahmi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2745-2130" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Naima Ahmed Fahmi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Khandakar Tanvir Ahmed
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of Central Florida
                       </span>
                       , 4000 Central Florida Blvd, Orlando, FL 32816,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Khandakar%2BTanvir%2BAhmed%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ahmed%20KT&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKhandakar%2BTanvir%2BAhmed%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Jae-Woong Chang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota Twin Cities
                       </span>
                       , 420 Washington Ave. S.E., Minneapolis, MN 55455,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jae-Woong%2BChang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chang%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJae-Woong%2BChang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Heba Nassereddeen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Engineering, University of Central Florida
                       </span>
                       , 4000 Central Florida Blvd, Orlando, FL 32816,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Heba%2BNassereddeen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Nassereddeen%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AHeba%2BNassereddeen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Deliang Fan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        School of Electrical, Computer and Energy Engineering, Arizona State University
                       </span>
                       , 650 E Tyler Mall, Tempe, AZ 85287,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Deliang%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Fan%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADeliang%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Jeongsik Yong
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Biochemistry, Molecular Biology and Biophysics, University of Minnesota Twin Cities
                       </span>
                       , 420 Washington Ave. S.E., Minneapolis, MN 55455,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jeongsik%2BYong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Yong%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJeongsik%2BYong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Wei Zhang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of Central Florida
                       </span>
                       , 4000 Central Florida Blvd, Orlando, FL 32816,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wei%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhang%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AWei%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-3605-9373" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Wei Zhang
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:wzhang.cs@ucf.edu" class="" data-icon-position="" data-hide-link-title="0">
                        wzhang.cs@ucf.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.02.16.951657v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.16.951657v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.16.951657v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.16.951657v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.16.951657v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.02.16.951657v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2670172" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2670172/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <div id="sec-1" class="subsection">
                             <p id="p-2">
                              <strong>
                               Background
                              </strong>
                              The eukaryotic genome is capable of producing multiple isoforms from a gene by alternative polyadenylation (APA) during pre-mRNA processing. APA in the 3’-untranslated region (3’-UTR) of mRNA produces transcripts with shorter or longer 3’-UTR. Often, 3’-UTR serves as a binding platform for microRNAs and RNA-binding proteins, which affect the fate of the mRNA transcript. Thus, 3’-UTR APA is known to modulate translation and provides a mean to regulate gene expression at the post-transcriptional level. Current bioinformatics pipelines have limited capability in profiling 3’-UTR APA events due to incomplete annotations and a low-resolution analyzing power: widely available bioinformatics pipelines do not reference actionable polyadenylation (cleavage) sites but simulate 3’-UTR APA only using RNA-seq read coverage, causing false positive identifications. To overcome these limitations, we developed APA-Scan, a robust program that identifies 3’-UTR APA events and visualizes the RNA-seq short-read coverage with gene annotations.
                             </p>
                            </div>
                            <div id="sec-2" class="subsection">
                             <p id="p-3">
                              <strong>
                               Methods
                              </strong>
                              APA-Scan utilizes either predicted or experimentally validated actionable polyadenylation signals as a reference for polyadenylation sites and calculates the quantity of long and short 3’-UTR transcripts in the RNA-seq data. APA-Scan works in three major steps: (i) calculate the read coverage of the 3’-UTR regions of genes; (ii) identify the potential APA sites and evaluate the significance of the events among two biological conditions; (iii) graphical representation of user specific event with 3’-UTR annotation and read coverage on the 3’-UTR regions. APA-Scan is implemented in Python3. Source code and a comprehensive user’s manual are freely available at
                              <a href="https://github.com/compbiolabucf/APA-Scan">
                               https://github.com/compbiolabucf/APA-Scan
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <p id="p-4">
                              <strong>
                               Result
                              </strong>
                              APA-Scan was applied to both simulated and real RNA-seq datasets and compared with two widely used baselines DaPars and APAtrap. In simulation APA-Scan significantly improved the accuracy of 3’-UTR APA identification compared to the other baselines. The performance of APA-Scan was also validated by 3’-end-seq data and qPCR on mouse embryonic fibroblast cells. The experiments confirm that APA-Scan can detect unannotated 3’ -UTR APA events and improve genome annotation.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <p id="p-5">
                              <strong>
                               Conclusion
                              </strong>
                              APA-Scan is a comprehensive computational pipeline to detect transcriptome-wide 3’-UTR APA events. The pipeline integrates both RNA-seq and 3’-end-seq data information and can efficiently identify the significant events with a high-resolution short reads coverage plots.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-9">
                             Poly(A)-tails are added to pre-mRNA after the polyadenylation signal (PAS) during the 3’-end processing of pre-mRNA [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. The last exon of mRNA contains a non-coding region, 3’-untranslated region (3’-UTR), which spans from the termination codon to the polyadenylation site. The 3’-UTR acts as a molecular scaffold to bind microRNAs and RNA-binding proteins and functions in regulatory gene expression [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ]. In human and mouse, more than 70% of genes contain multiple PASs in their 3’ - UTRs and polyadenylation using upstream PASs leads to the production of mRNA with shortened 3’-UTRs (3’-UTR APA) [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. 3’-UTR APA is known to increase the efficiency of translation and is associated with T cell activation, oncogene activation, and poor prognosis in many diseases [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]. Recent study has demonstrated that 3’ - UTR APA is one way to increase protein synthesis without increasing the quantities of mRNAs, indicating that it is an important element in gene expression which cannot be understood by conventional differential gene or transcript expression analysis [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. Up-regulation of mTOR signaling pathway can lead to transcriptome-wide 3’-UTR APA [
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ].
                            </p>
                            <p id="p-10">
                             3’-UTR APA has gained much attention recently and the importance of the 3’ - UTR APA in human diseases has been demonstrated as mentioned above. Some recent studies show that both proliferating cells and transformed cells favor expression of shorter 3’-UTR through APA and lead to the activation of oncogenes [
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ]. Some other research shows the trend in cancer cells for highly expressed genes to exhibit shorter 3’-UTR with fewer microRNA binding sites, decreasing microRNA-mediated translation repression [
                             <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. All these studies imply that 3’-UTR APA may serve as a new layer of prognostic biomarker. A scalable computational model is highly needed to detect the genome-wide unannotated 3’-UTR APA in different phenotypes.
                            </p>
                            <p id="p-11">
                             Several bioinformatics pipelines are available for the analysis of UTR-APA using RNA-seq data [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. In general, all these methods measure the changes in 3’-UTR lengths by modeling the RNA-seq read density change near the 3’-end of mRNAs. Indeed, with the aid of these methods, RNA-seq experiments became a powerful approach to investigate 3’-UTR APA. However, in many cases the identified APA sites are not functionally and physiologically relevant because most pipelines do not reference actionable PASs in their 3’-UTR APA simulation. RNA-seq is not particularly accurate when it comes to identifying polyadenylation sites, making novel APA transcript identification rather difficult. Therefore, 3’-end-seq data has been developed to address these issues by enriching for 3’-end reads in high-throughput sequencing experiment [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ] and provides the accurate polyadenylation sites. In addition to the limitations of the current bioinformatics pipelines mentioned above, none of them can provide high-resolution read coverage plots of the APA events with an accurate annotation. We have developed APA-Scan (
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             ), a bioinformatics program, to detect and visualize genome-wide 3’-UTR APA events. APA-Scan integrates both 3’-end-seq data and the location information of predicted canonical PASs with RNA-seq data to improve the quantitative definition of genome-wide UTR APA events. APA-Scan efficiently manages large-scale alignment files and generates a comprehensive analysis for UTR APA events. It is also advantageous in producing high quality plots of the events.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F1.large.jpg?width=800&height=600&carousel=1" title="Workflow of APA-Scan. Starting with aligned RNA-seq and 3’-end-seq (optional) bam files, APA-Scan consists of three steps and generates high quality graphical illustration of aligned sequences with the indication of 3’-UTR APA events. (A) Read coverage files are generated for RNA-seq and 3’-end-seq (if provided) input samples. (B) APA-Scan identifies potential cleavage sites according to polyadenylation signal (PAS) hexamer: ATTAAA or AATAAA, or 3’-end peaks (if 3’-end-seq data is available). (C) Graphical illustration of the identified events. The illustration also highlights unannotated short 3’-UTR transcript identified from this task. The vertical red lines show the corresponding cleavage sites." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup">Workflow of APA-Scan. Starting with aligned RNA-seq and 3’-end-seq (optional) bam files, APA-Scan consists of three steps and generates high quality graphical illustration of aligned sequences with the indication of 3’-UTR APA events. (A) Read coverage files are generated for RNA-seq and 3’-end-seq (if provided) input samples. (B) APA-Scan identifies potential cleavage sites according to polyadenylation signal (PAS) hexamer: ATTAAA or AATAAA, or 3’-end peaks (if 3’-end-seq data is available). (C) Graphical illustration of the identified events. The illustration also highlights unannotated short 3’-UTR transcript identified from this task. The vertical red lines show the corresponding cleavage sites.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F1.medium.gif" width="298" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F1.medium.gif" width="298" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <p id="p-12" class="first-child">
                               Workflow of APA-Scan. Starting with aligned RNA-seq and 3’-end-seq (optional) bam files, APA-Scan consists of three steps and generates high quality graphical illustration of aligned sequences with the indication of 3’-UTR APA events. (A) Read coverage files are generated for RNA-seq and 3’-end-seq (if provided) input samples. (B) APA-Scan identifies potential cleavage sites according to polyadenylation signal (PAS) hexamer: ATTAAA or AATAAA, or 3’-end peaks (if 3’-end-seq data is available). (C) Graphical illustration of the identified events. The illustration also highlights unannotated short 3’-UTR transcript identified from this task. The vertical red lines show the corresponding cleavage sites.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-13">
                             APA-Scan is designed to identify both annotated and de novo 3 ‘ -UTR APA events between different biological conditions. To access the performance of APA-Scan, it was compared with two baseline methods on both simulated and real RNA-seq datasets. In the simulation experiment, we first generated synthetic dataset with pre-defined 3’-UTR APA events (ground truth) to test if the APA-Scan and baseline methods can detect them. Next, we performed experiments on two mouse embryonic fibroblast (MEF) cells to evaluate the performance of APA-Scan. The results of analyzing real MEF RNA-seq datasets were validated using both qPCR and 3’-end-seq data.
                            </p>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Experimental results with simulated RNA-seq data
                             </h3>
                             <p id="p-14">
                              In the simulation experiment, we generated synthetic RNA-seq short reads with flux-simulator [
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              ]. 1000 pre-defined 3’-UTR APA events were simulated as the ground truth between two different conditions. In each condition, three technical replicates were generated by repeating the experiment three times with the same parameter setting in the flux simulator. The details of the parameters used in this experiment are provided in the Additional file 2. For both conditions, the gene expressions were sampled from a Poisson distribution to reflect a real RNA-seq data [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ]. For each gene, one proximal polyadenylation site was synthesized to represent the end of the short isoform and the end of the annotated transcript was applied to define the end of the long isoform of that gene. To generate the ground truth profile of the 3’-UTR APA events, the expression proportions of the short and long isoforms in the same gene were assigned significantly different values in two conditions (i.e., the proportion difference was larger than 10%) to represent the existence of the APA event.
                             </p>
                             <p id="p-15">
                              In the simulation experiment, two sets of synthetic data were generated by flux-simulator. One with 30M (30 million) paired-end reads in each replicate and one with 50M paired-end read. In both cases, the read length is 76 bps of each end. APA-Scan was compared with DaPars and APAtrap on the simulated RNA-seq datasets. To detect the significant 3’-UTR APA events, APA-Scan used p-value < 0.05 (χ2-test) as the cutoff. DaPars identified APA events according to the difference in PDUI (Percentage of Distal polyA Usage Index) values between two conditions > 0.1 and FDR < 0.05; whereas APAtrap selected events using the cutoff values of two parameters: percentage difference of APA site usage between two conditions > 0.1 and FDR < 0.05. The performance of the methods is then evaluated using AUC score, sensitivity and specificity.
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              shows that, APA-Scan outperformed the two baselines in terms of AUC scores and got the best score of 0.94 in both sequence depths (30M reads and 50M reads) and followed by APAtrap (0.73 in 30 million reads case and 0.75 in 50 million reads case). DaPars did not work very well compared to the other two methods and the AUC scores were below 0.7 in both cases, though there was an improvement in the case with more reads. We also report the sensitivity and specificity for each method with two different sequencing depths in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              . APA-Scan gets the highest sensitivity and specificity scores for both cases, which indicates that APA-Scan outperformed the baseline methods in detecting the true 3’-UTR APA events and eliminating the true negative ones.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F2.large.jpg?width=800&height=600&carousel=1" title="Simulation experiment to assess the performance of APA-Scan and the baseline methods (DaPars and APAtrap). (a) Results on the simulation experiment with 30 million (30M) short reads. (b) Results on the simulation experiment with 50M short reads. The receiver operating characteristic (ROC) curves, i.e., true positive rate against false positive rate, are plotted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup">Simulation experiment to assess the performance of APA-Scan and the baseline methods (DaPars and APAtrap). (a) Results on the simulation experiment with 30 million (30M) short reads. (b) Results on the simulation experiment with 50M short reads. The receiver operating characteristic (ROC) curves, i.e., true positive rate against false positive rate, are plotted.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F2.medium.gif" width="440" height="170"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F2.medium.gif" width="440" height="170"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-16" class="first-child">
                                Simulation experiment to assess the performance of APA-Scan and the baseline methods (DaPars and APAtrap). (a) Results on the simulation experiment with 30 million (30M) short reads. (b) Results on the simulation experiment with 50M short reads. The receiver operating characteristic (ROC) curves, i.e., true positive rate against false positive rate, are plotted.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2672865/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2672865/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2672865" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <p id="p-17" class="first-child">
                                Categorized overview of the technical parameters of APA-Scan
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2672859/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2672859/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2672859" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <p id="p-18" class="first-child">
                                Comparison among APA-Scan, DaPars and APAtrap on simulated RNA-seq data with two different sequencing depths (30 million reads and 50 million reads). AUC (the area under the ROC curve) score, sensitivity, specificity of the three methods are reported. The best results across the three methods are bold.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-19">
                              As different sequencing depths may affect the performance of APA-Scan, we generated five simulation experiments with different read depths, i.e., 2M, 5M, 10M, 30M, and 50M paired-end reads by flux-simulator with the same parameter setting to learn the impact of sequencing depths in the analysis of 3’-UTR APA with APA-Scan. In this experiment, the read length was also 76 bps for each end and three replicates were generated for each condition in each read depth using the same procedures as mentioned in the previous section.
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              shows the ROC curves for different sequencing depth on detecting the 3’-UTR APA events. APA-Scan shows moderate performance with low sequencing depths (i.e., 2M and 5M). However, the performance of APA-Scan improved drastically (AUC = 0.94) after it reached to a certain sequencing depth (i.e., 10M in this study) and holds that performance across read depths above that threshold. This result suggests that APA-Scan is quite robust in detecting APA events on lowly expressed genes and relatively low read coverage samples.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F3.large.jpg?width=800&height=600&carousel=1" title="Simulation experiment to assess the performance of APA-Scan on different sequencing depths. The ROC curves for the results of different RNA-seq read depth are plotted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup">Simulation experiment to assess the performance of APA-Scan on different sequencing depths. The ROC curves for the results of different RNA-seq read depth are plotted.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F3.medium.gif" width="440" height="322"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F3.medium.gif" width="440" height="322"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-20" class="first-child">
                                Simulation experiment to assess the performance of APA-Scan on different sequencing depths. The ROC curves for the results of different RNA-seq read depth are plotted.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Experimental results with MEFs samples
                             </h3>
                             <p id="p-21">
                              In the real RNA-seq experiments, two MEFs samples Tsc1
                              <sup>
                               -/-
                              </sup>
                              and WT were used in the analysis to evaluate the performance of APA-Scan and baseline methods. Knockout of Tsc1, a negative regulator of mTOR pathway, leads to uncontrolled mTOR hyper-activation compared with WT. For the comparison and evaluation purposes, the APA-Scan was run on two different setups. One used PASs in the 3’-UTRs as potential cleavage sites, and we denote it by APA-Scan
                              <sup>
                               PAS
                              </sup>
                              . The other one considered 3’-end-seq peaks as candidate sites, and it is denoted as APA-Scan
                              <sup>
                               peaks
                              </sup>
                              . First, APA-Scan
                              <sup>
                               PAS
                              </sup>
                              was applied to detect 3’-UTR APA events between the two MEFs samples with p-value < 0.05. APA-Scan
                              <sup>
                               PAS
                              </sup>
                              detected 265 events, whereas DaPars and APAtrap detected 785 and 1130 significant events, respectively. These events were then verified by the polyadenylation sites reported by 3’-end-seq data. If a predicted 3’-UTR APA event is within 50 bps upstream or downstream of the loci of the peak(s) in 3’-end-seq data, then this APA event is considered overlapping with the 3’-end-seq signals. Though APA-Scan
                              <sup>
                               PAS
                              </sup>
                              detected less number of significant events compared to the baseline methods, 87.92% (233) of the events were validated by the 3’-end-seq signals according to the result shown in
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              and
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Table 3
                              </a>
                              . DaPars and APAtrap identified more events than APA-Scan
                              <sup>
                               PAS
                              </sup>
                              , however, both the number and ratio of the overlapping events with the 3’-end-seq signals are significantly lower than the events detected by APA-Scan
                              <sup>
                               PAS
                              </sup>
                              . Note that APA-Scan
                              <sup>
                               PAS
                              </sup>
                              did not use any information from 3’-end-seq data to identify the APA events. These results concur with our findings in the simulation experiment that APA-Scan not only do better detection on the true APA events but also prevent the false positives.
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              shows the number of overlapped genes with the 3’-UTR APA events detected by the three methods. From the results, we can conclude that the agreement of the three methods is not high and most identified events were only detected by one method.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F4.large.jpg?width=800&height=600&carousel=1" title="Evidence of polyadenylation sites supported by 3’-end-seq data for 3’-UTR APA events detected by different methods in MEFs samples. The number of events predicted by each method are shown in purple and the number of events validated by the signals in the 3’-end-seq data are shown in blue. The x-axis shows the percentage of the identified events is validated by 3’-end-seq." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup">Evidence of polyadenylation sites supported by 3’-end-seq data for 3’-UTR APA events detected by different methods in MEFs samples. The number of events predicted by each method are shown in purple and the number of events validated by the signals in the 3’-end-seq data are shown in blue. The x-axis shows the percentage of the identified events is validated by 3’-end-seq.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F4.medium.gif" width="440" height="322"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F4.medium.gif" width="440" height="322"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-22" class="first-child">
                                Evidence of polyadenylation sites supported by 3’-end-seq data for 3’-UTR APA events detected by different methods in MEFs samples. The number of events predicted by each method are shown in purple and the number of events validated by the signals in the 3’-end-seq data are shown in blue. The x-axis shows the percentage of the identified events is validated by 3’-end-seq.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F5.large.jpg?width=800&height=600&carousel=1" title="Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods (i.e., APA-Scan, DaPars and APAtrap) between two MEFs samples (WT vs Tsc1-/-)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods (i.e., APA-Scan, DaPars and APAtrap) between two MEFs samples (WT vs Tsc1<sup>-/-</sup>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F5.medium.gif" width="432" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F5.medium.gif" width="432" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <p id="p-23" class="first-child">
                                Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods (i.e., APA-Scan, DaPars and APAtrap) between two MEFs samples (WT vs Tsc1
                                <sup>
                                 -/-
                                </sup>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2672858/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2672858/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2672858" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 3.
                               </span>
                               <p id="p-24" class="first-child">
                                Number of events detected by APA-Scan, DaPars and APAtrap and validated by 3’-end-seq data for MEFs samples.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              To further validate the analysis results by APA-Scan, we conducted qPCR experiments for Srsf3 and Rpl22 transcripts from Tsc1
                              <sup>
                               -/-
                              </sup>
                              and WT MEFs based on the significant 3’-UTR APA events reported by APA-Scan
                              <sup>
                               peaks
                              </sup>
                              . These genes were selected due to the design of PCR (polymerase chain reaction) primers for wet-lab validation. As shown in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              , both Srsf3 and Rpl22 showed the increase of the short 3’-UTR transcript by APA in Tsc1
                              <sup>
                               -/-
                              </sup>
                              compared to WT MEFs, which is consistent with our observations on the RNA-seq and 3’-end-seq read coverage plots. These results further confirm that APA-Scan can identify the true 3’-UTR APA events with RNA-seq and 3’-end-seq samples from two different biological contexts. The more details of the qPCR analysis and the primer sequences of the two genes are available in the Additional file 1.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F6.large.jpg?width=800&height=600&carousel=1" title="Experimental results: (a) RNA-seq and 3’-end-seq read coverage plots of the 3’-UTR in Srsf3 and Rpl22 gene in the two samples with isoform annotation. (b) The level of total, short 3’-UTR, and long 3’-UTR transcripts from Srsf3 and Rpl22 was measured by qPCR. Because it is not possible to design specific primers for the qPCR analysis of short 3’-UTR transcript, the amount of short 3’-UTR transcripts were calculated by subtracting the quantity of long 3’-UTR transcripts from total." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Experimental results: (a) RNA-seq and 3’-end-seq read coverage plots of the 3’-UTR in <em>Srsf3</em> and <em>Rpl22</em> gene in the two samples with isoform annotation. (b) The level of total, short 3’-UTR, and long 3’-UTR transcripts from <em>Srsf3</em> and <em>Rpl22</em> was measured by qPCR. Because it is not possible to design specific primers for the qPCR analysis of short 3’-UTR transcript, the amount of short 3’-UTR transcripts were calculated by subtracting the quantity of long 3’-UTR transcripts from total.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F6.medium.gif" width="431" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F6.medium.gif" width="431" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <p id="p-26" class="first-child">
                                Experimental results: (a) RNA-seq and 3’-end-seq read coverage plots of the 3’-UTR in Srsf3 and Rpl22 gene in the two samples with isoform annotation. (b) The level of total, short 3’-UTR, and long 3’-UTR transcripts from Srsf3 and Rpl22 was measured by qPCR. Because it is not possible to design specific primers for the qPCR analysis of short 3’-UTR transcript, the amount of short 3’-UTR transcripts were calculated by subtracting the quantity of long 3’-UTR transcripts from total.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-27">
                              Generally, the nucleotide profiles surrounding the polyadenylation sites are dominated by two motifs and their variants: AATAAA and ATTAAA and these two hexamers are observed upstream of the cleavage sites [
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ]. This phenomenon leads us to explore the nucleotide composition near the predicted polyadenylation sites by APA-Scan
                              <sup>
                               peaks
                              </sup>
                              .
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Figure 7
                              </a>
                              shows a high concentration/cluster of nucleotide ‘A’ in the polyadenylation site, positioned at 0. The upstream surrounding region is also dominated by ‘A’ and ‘T’, which clearly indicates the existence of potential 3’-UTR APA events.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F7.large.jpg?width=800&height=600&carousel=1" title="Nucleotide composition of the sequence surrounding the polyadenylation sites identified by APA-Scanpeaks for MEFs samples. 50bp up and downstream region is plotted with base sequences. x-axis denotes the position in the region, 0 is the location of the identified polyadenylation site. y-axis shows the fraction of the nucleotides content at each position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Nucleotide composition of the sequence surrounding the polyadenylation sites identified by APA-Scan<sup><em>peaks</em></sup> for MEFs samples. 50bp up and downstream region is plotted with base sequences. x-axis denotes the position in the region, 0 is the location of the identified polyadenylation site. y-axis shows the fraction of the nucleotides content at each position.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F7.medium.gif" width="440" height="321"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F7.medium.gif" width="440" height="321"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <p id="p-28" class="first-child">
                                Nucleotide composition of the sequence surrounding the polyadenylation sites identified by APA-Scan
                                <sup>
                                 peaks
                                </sup>
                                for MEFs samples. 50bp up and downstream region is plotted with base sequences. x-axis denotes the position in the region, 0 is the location of the identified polyadenylation site. y-axis shows the fraction of the nucleotides content at each position.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-29">
                             APA is one mechanism for post-transcriptional regulation of mRNA expression, and it is defined as use of more than one polyadenylation sites. 3’-UTR APA is one of the most frequent APA forms, which contains more than one polyadenylation sites in the 3’-UTR. It generates multiple mRNA transcripts with different 3’-UTR lengths without affecting the protein encoded by the gene. Since the 3’-UTR of mRNA of-ten contains binding sites for microRNAs, 3’-UTR APA potentially leads to altered mRNA stability or protein translation efficiency due to variation of 3’-UTR length. Identification and assessment of APA sites has been a major goal in understanding transcriptomic diversity. Several bioinformatics tools have been developed to predict transcriptome-wide polyadenylation sites with RNA-seq data. However, our experimental results on simulated and real samples indicate that the current methods (e.g., DaPars and APAtrap) can detect large number of APA events, but significant portion of the events are false positives. A similar data analysis on BT549 breast cancer cells (mock vs. torin 1 treated) in
                             <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                              Figure S1
                             </a>
                             in the Additional file 1 illustrates a similar pattern. By integrating 3’-end-seq and RNA-seq data, APA-Scan can potentially reduce the number of false positive events. To evaluate the performance of APA-Scan on real cancer patient samples, one pair (tumor vs. matched normal tissue) of The Cancer Genome Atlas (TCGA) breast cancer samples are also analyzed and reported in the Additional file 1.
                             <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                              Figure S2
                             </a>
                             shows that a significant portion (>72%) of the 3’-UTR APA events are not differentially expressed. There-fore, APA-based molecular signatures could provide additional predictive power of cancer outcomes by combining the differently expressed genes.
                            </p>
                            <p id="p-30">
                             APA-Scan not only can accurately detect the splicing events compared to the baseline methods, but also provides reasonable running time.
                             <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">
                              Table 4
                             </a>
                             shows a com- parison of the CPU time of each method on Tsc1
                             <sup>
                              -/-
                             </sup>
                             and WT MEFs. The CPU time was measured on an Intel(R) Xeon(R) CPU E5-2620 v4 @ 2.10GHz machine. Both APA-Scan and APAtrap completed the analysis in a similar amount of time. However, DaPars is much slower than the other two methods which is not suit- able to be applied on large-scale experiment in terms of running time. Overall, this study reports an efficient and precise framework for 3’-UTR APA identification with RNA-seq and 3’-end-seq data.
                            </p>
                            <div id="T4" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2672870/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2672870/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2672870" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Table 4.
                              </span>
                              <p id="p-31" class="first-child">
                               CPU time of APA-Scan, DaPars and APAtrap on two MEFs samples (WT vs Tsc1
                               <sup>
                                -/-
                               </sup>
                               ).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Conclusion
                            </h2>
                            <p id="p-32">
                             We developed APA-Scan, which offers a comprehensive computational pipeline to identify transcriptome-wide 3’-UTR APA events. By integrating RNA-seq data and 3’-end-seq information (experimentally verified or computationally predicted). APA-Scan can efficiently identify significant APA events and also, can illustrate the events with read coverage plots. 3’-end-seq signals and the wet-lab experiment using qPCR demonstrate that APA-Scan provides high-accuracy and quantitative profiling of 3’-UTR APA events. Therefore, we expect that, APA-Scan will serve as a useful tool for APA site analysis.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              APA-Scan pipeline
                             </h3>
                             <p id="p-33">
                              APA-Scan workflow comprises of three major steps: (i) read coverage estimation; (ii) identification of polyadenylation sites and the calculation of APA; (iii) graphical illustration of UTR APA events (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ). First, APA-Scan takes aligned RNA- seq and 3’-end-seq data from two different biological conditions as input. Each biological condition can have multiples samples or replicates. The read coverage files are generated by SAMtools [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ]. In this step, the 3’-end-seq data is an optional input.
                             </p>
                             <p id="p-34">
                              In the second step, APA-Scan starts the analysis by extracting 3’-UTR frames for each gene. APA-Scan is designed in two modes: (a) Default, and (b) Extended. All the aligned reads from 3’-end-seq data are pooled together to identify peaks and the corresponding unannotated cleavage sites in 3’-UTR regions and downstream of the 3’-UTR regions (i.e., APA-Scan
                              <sup>
                               peaks
                              </sup>
                              ). In the Default mode, 3’-UTR regions are selected according to the end of the longest annotated transcript of the gene. The loci of peaks identified in the 3’-end-seq data are considered as potential cleavage sites. If the 3’-end-seq data is not provided by the user, detected PASs (generally two variations of the hexamers: AATAAA, ATTAAA) in 3’-UTRs are considered as the potential cleavage sites (i.e., APA-Scan
                              <sup>
                               PAS
                              </sup>
                              ) follow the ideas in Omni-PolyA [
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ] which use 12 most common PAS variants to determine the cleavage sites. In the Extended mode of APA-Scan, the potential peaks/PAS signals are searched up to 10kb downstream of the end of transcript to discover de novo distal polyadenylation sites. The locations detected from all input samples are merged to get a combined list of potential cleavage sites. The major commands and general terminologies to run APA-Scan are listed in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              .
                             </p>
                             <p id="p-35">
                              APA-Scan evaluates each empirical cleavage site in the 3’-UTR of a transcript by contrasting the RNA-seq short reads coverage up and downstream of the candidate site between the two biological conditions. n and N denote the average read coverage up and downstream of the site. They are determined by estimating the number of reads mapped to upstream and downstream of the cleavage site, r
                              <sub>
                               u
                              </sub>
                              and r
                              <sub>
                               d
                              </sub>
                              , divided by their effective length, l
                              <sub>
                               u
                              </sub>
                              and l
                              <sub>
                               d
                              </sub>
                              , respectively (i.e.,
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). For each potential polyadenylation site, the ratio differences between the samples in two conditions are calculated based on the following equation
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/07/30/2020.02.16.951657/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where 1 and 2 represent the two conditions. Ratio difference indicates the change in read coverage between two conditions and only the absolute ratio difference > 0.1 is considered as candidate site for the further analysis. After that, the canonical 2 x 2 χ2-test is applied to report the p-value for each candidate site. The χ2-test measures how much the observation deviates from the null hypothesis. In our experiment, we set the null hypothesis as the average read coverage before and after the cleavage sites are consistent among the two biological conditions. For any true 3’-UTR APA event, there must be a significant read coverage drop-off around the cleavage sites, and the ratios of the average read coverages before and after the cleavage sites are crucially different in the two conditions. In such cases, the χ2-test precisely reports significant p-values to reject our null hypothesis. APA-Scan will report both significant and insignificant in an Excel file. A comprehensive user’s manual is provided in the Additional file 2.
                             </p>
                             <p id="p-36">
                              In the third step, based on the significance of 3’ -UTR APA events calculated in the previous step, APA-Scan generates RNA-seq and 3’-end-seq (if provided) read coverage plots with the 3’-UTR annotations for one or more user-specific events. Users may specify the region of the genome locus to generate the read alignment plot.
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              (Step 3) illustrates an example of the read coverage plot generated by APA-Scan.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Baselines and evaluation methods
                             </h3>
                             <p id="p-37">
                              In this study, two widely used 3’-UTR APA identification approaches, DaPars [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ] and APAtrap [
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ] were applied to compare the performance with APA-Scan. The command lines to run the baseline methods are available in the Additional file 1. To evaluate the performance of APA-Scan and baseline methods, the area under the ROC curve (AUC), sensitivity and specificity were used on the identified lists of 3’-UTR APA events.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Short read alignments and peak identification
                             </h3>
                             <p id="p-38">
                              In this study, two mouse embryonic fibroblasts (MEFs) samples and two breast cancer cell lines (BT549) were used in the analysis to evaluate the performance of APA-Scan and baseline methods. For the MEFs samples, we performed RNA-seq and 3’-end-seq analyses of poly(A+) RNAs isolated from Tsc1
                              <sup>
                               -/-
                              </sup>
                              and wild-type (WT) MEFs. In the RNA-seq analysis, 63,742,790 paired-end reads for WT and 74,251,891 paired-end reads for Tsc1
                              <sup>
                               -/-
                              </sup>
                              MEFs were produced from Hi-Seq pipeline with length of 50 bps of each end. The short reads were aligned to the mm10 reference genome by TopHat2 [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ], allowing up to two mismatches. Finally, 87.1% of short reads from WT and 87.5% of sequence reads from Tsc1
                              <sup>
                               -/-
                              </sup>
                              MEFs were mapped to the reference genome for APA analysis in the study. In the 3’-end-seq analysis, the reads from WT and Tsc1
                              <sup>
                               -/-
                              </sup>
                              MEFs were preprocessed to trim A’s off the 3’-ends and then filtered by removing the reads of low-quality 3’-end (Phred score < 30) and shorter than 25 bps. The remaining reads were aligned to the mm10 reference genome by Bowtie [
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] without allowing any mismatches. In total, 6,186,893 paired-end reads were aligned for WT and 5,382,111 reads were aligned for Tsc1
                              <sup>
                               -/-
                              </sup>
                              . All aligned reads from 3’-end-seq were pooled together in order to identify peaks and the corresponding cleavage sites in the reference genome by the read coverage signals. In each read alignment ‘hill’, the location with the highest read coverage between two zero coverage positions was considered as the peak of the ‘hill’. The 3’-end of the peak is chosen as the potential corresponding cleavage sites where the read coverage at the peak quantifies the cleavage at the site. For the breast cancer cell lines, we performed RNA-seq analysis of poly(A+) RNAs isolated from BT549 mock and Torin1 treated cells. 131,955,082 paired-end reads for BT549 mock, and 138,127,113 paired-end reads for BT549 treated with Torin1 were produced from Hi-Seq pipeline with length of 51 bps of each end. The short reads were aligned to the hg38 reference genome by TopHat2, allowing up to two mismatches. Finally, 85.2% of short reads from BT549 mock and 84.7% of sequence reads from BT549 treated with Torin1 were mapped to the reference genome for APA analysis in the study.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             Declarations
                            </h2>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Ethics approval and consent to participate
                            </h2>
                            <p id="p-48">
                             Not applicable.
                            </p>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             Consent for publication
                            </h2>
                            <p id="p-49">
                             Not applicable.
                            </p>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             Availability of data and materials
                            </h2>
                            <p id="p-50">
                             The source code in this study is available at:
                             <a href="https://github.com/compbiolabucf/APA-Scan">
                              https://github.com/compbiolabucf/APA-Scan
                             </a>
                             . The accession number for the MEFs RNA-seq data in this study is
                             <span class="underline">
                              SRP056624
                             </span>
                             . The accession number for the 3’-end-seq data in this study is
                             <span class="underline">
                              SRP133833
                             </span>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Competing interests
                            </h2>
                            <p id="p-51">
                             The authors declare that they have no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-20">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-52">
                             The study was supported by the National Science Foundation grant FET2003749 and National Institutes of Health 1R01GM113952-01A1 and DK097771. Publication costs are funded by the National Science Foundation grant FET2003749. The funding bodies had no role in study design, data collection, data analysis and interpretation of data and in writing the manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Author’s contributions
                            </h2>
                            <p id="p-53">
                             NAF, DF, JY, and WZ conceived the study and planned the analysis. NAF, KTA, and HN performed data analysis. JWC and JY designed and performed qPCR experiments. NAF, KTA, JY, and WZ wrote the manuscript. All authors read and approved the final manuscript.
                            </p>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Additional Files
                            </h2>
                            <div class="section" id="sec-23">
                             <h3>
                              Additional file 1
                             </h3>
                             <p id="p-54">
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Figure S1
                              </a>
                              ;
                              <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                               Figure S2
                              </a>
                              ; The command lines used for running the baseline methods; Parameters to run flux-simulator; qPCR analysis and primer sequences.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Additional file 2
                             </h3>
                             <p id="p-55">
                              User’s manual of APA-Scan
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Supplementary
                            </h2>
                            <div class="section" id="sec-26">
                             <h3>
                              1 Running the baselines
                             </h3>
                             <div class="section" id="sec-27">
                              <h4>
                               1.1 DaPars
                              </h4>
                              <p>
                               Inputs:
                              </p>
                              <ul class="list-unord" id="list-1">
                               <li id="list-item-1">
                                <p id="p-57">
                                 BED files. Flux-simulator simulated three fastq files for three replicates in each condition. Using SAMtools (v0.1.8), read coverage files for each chromosome are generated in BAM format from the fastq files. Six bedgraph files in two conditions are generated from the BAM files using BEDtools.
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-58">
                                 Gene annotation in .bed format: mm10_Refseq.bed
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-59">
                                 Configuration file: configure.txt
                                </p>
                                <p id="p-60">
                                 Annotated_3UTR=mm10_Refseq_extracted_3UTR.bed
                                </p>
                                <p id="p-61">
                                 Group1_Tophat_aligned_Wig = case_1.bedgraph, case_2.bedgraph, case_3.bedgraph Group2_Tophat_aligned_Wig = control_1.bedgraph, control_2.bedgraph, control_3.bedgraph
                                </p>
                                <p id="p-62">
                                 Output_directory = DaPars_out/
                                </p>
                                <p id="p-63">
                                 Output_result_file = Dapars_out
                                </p>
                                <p id="p-64">
                                 Num_least_in_group1 = 1
                                </p>
                                <p id="p-65">
                                 Num_least_in_group2 = 1
                                </p>
                                <p id="p-66">
                                 Coverage_cutoff = 30
                                </p>
                                <p id="p-67">
                                 FDR_cutoff = 0.05
                                </p>
                                <p id="p-68">
                                 PDUI_cutoff = 0.1
                                </p>
                                <p id="p-69">
                                 Fold_change_cutoff = 0.59
                                </p>
                               </li>
                              </ul>
                              <ul class="list-simple" id="list-2">
                               <li id="list-item-4">
                                <p id="p-70">
                                 <strong>
                                  Command 1:
                                 </strong>
                                </p>
                                <p id="p-71">
                                 python DaPars_Extract_Anno.py -b mm10_Refseq.bed
                                </p>
                                <p id="p-72">
                                 -s mm10_Refseq_id.txt -o mm10_Refseq_extracted_3UTR.bed
                                </p>
                               </li>
                               <li id="list-item-5">
                                <p id="p-73">
                                 <strong>
                                  Command 2:
                                 </strong>
                                </p>
                                <p id="p-74">
                                 python DaPars_main.py configure.txt
                                </p>
                               </li>
                              </ul>
                             </div>
                             <div class="section" id="sec-28">
                              <h4>
                               1.2 APAtrap
                              </h4>
                              <p>
                               Inputs:
                              </p>
                              <ul class="list-unord" id="list-3">
                               <li id="list-item-6">
                                <p id="p-76">
                                 BED files. Flux-simulator simulated three fastq files for three replicates in each condition. Using SAMtools (v0.1.8), read coverage files for each chromosome are generated in BAM format from the fastq files. Six bedgraph files in two conditions are generated from the BAM files using BEDtools.
                                </p>
                               </li>
                               <li id="list-item-7">
                                <p id="p-77">
                                 Gene annotation in .bed format: mm10_Refseq.bed
                                </p>
                               </li>
                              </ul>
                              <ul class="list-simple" id="list-4">
                               <li id="list-item-8">
                                <p id="p-78">
                                 <strong>
                                  Command 1:
                                 </strong>
                                </p>
                                <p id="p-79">
                                 identifyDistal3UTR -i A1.bedgraph A2.bedgraph A3.bedgraph B1.bedgraph B2.bedgraph B3.bedgraph -m mm10_Refseq.bed -o mm10.utr.bed
                                </p>
                               </li>
                               <li id="list-item-9">
                                <p id="p-80">
                                 <strong>
                                  Command 2:
                                 </strong>
                                </p>
                                <p id="p-81">
                                 predictAPA -i A1.bedgraph, A2.bedgraph, A3.bedgraph B1.bedgraph, B2.bedgraph, B3.bedgraph -g 2 -n 3 3 -u mm10.utr.bed -o APA_output.txt
                                </p>
                               </li>
                               <li id="list-item-10">
                                <p id="p-82">
                                 <strong>
                                  Command 3:
                                 </strong>
                                </p>
                                <p id="p-83">
                                 deAPA(‘APA_output.txt’, ‘APA_output.stat.txt’, 1, 2, 1, 1, 20)
                                </p>
                               </li>
                              </ul>
                             </div>
                            </div>
                            <div class="section" id="sec-29">
                             <h3>
                              2 Parameters to run flux-simulator (30 million reads)
                             </h3>
                             <div id="T5" class="table pos-anchor">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T5.medium.gif" width="440" height="416"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="2" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T5.medium.gif" width="440" height="416"/>
                                  </noscript>
                                 </span>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-30">
                             <h3>
                              3 Realtime quantitative PCR (RT-qPCR) analysis and primer sequences
                             </h3>
                             <p>
                              Total RNAs from TSC1 WT or TSC1-/- MEF cells were isolated by Trizol method according to manufacturer’s protocol (
                              <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/trizol_reagent.pdf">
                               https://assets.thermofisher.com/TFS-Assets/LSG/manuals/trizol_reagent.pdf
                              </a>
                              ). Reverse transcription reaction using Oligo-d(T) priming and NxGen M-MuLV Reverse transcriptase (Lucigen) was carried out according to the manufacturer’s protocol (
                              <a href="https://www.lucigen.com/docs/manuals/MA115-M-MuLV.pdf">
                               https://www.lucigen.com/docs/manuals/MA115-M-MuLV.pdf
                              </a>
                              ). SYBR Green was used to detect and quantitate the PCR products in real-time reactions. Quantitation of the real-time PCR results was done using standard curve method for accuracy and reliability of the analysis. The primer sequences used to measure the RSI for each transcript are as follows:
                             </p>
                             <ul class="list-simple" id="list-5">
                              <li id="list-item-11">
                               <p id="p-85">
                                mRpl22 Total forward 5’-AAGTTCAC CCTGGACTGC AC-3’
                               </p>
                              </li>
                              <li id="list-item-12">
                               <p id="p-86">
                                mRpl22 Total reverse 5’-GTGATCTT GCTCTTGCTG CG-3’
                               </p>
                              </li>
                              <li id="list-item-13">
                               <p id="p-87">
                                mRPL22 Long Forward 5’-TGGGCATC TGGGCTTTTA GG-3’
                               </p>
                              </li>
                              <li id="list-item-14">
                               <p id="p-88">
                                mRPL22 Long reverse 5’-GCTTGTTGCA GACTTGCTCA-3’
                               </p>
                              </li>
                              <li id="list-item-15">
                               <p id="p-89">
                                mSRSF3 Total forward 5’ - GCTGCCGTGTAAGAGTGGAA-3’
                               </p>
                              </li>
                              <li id="list-item-16">
                               <p id="p-90">
                                mSRSF3 Total reverse 5’- AGGACTCCTCCTGCGGTAAT-3’
                               </p>
                              </li>
                              <li id="list-item-17">
                               <p id="p-91">
                                mSRSF3 Long forward 5’ - TGCAACAGTCTTGTGGCTTA-3’
                               </p>
                              </li>
                              <li id="list-item-18">
                               <p id="p-92">
                                mSRSF3 Long reverse 5’-TGCAATGGCTCTTACATAGACC-3’
                               </p>
                              </li>
                             </ul>
                            </div>
                            <div class="section" id="sec-31">
                             <h3>
                              4 Venn diagram for BT549 mock vs BT549 Torin1 treated
                             </h3>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F8.large.jpg?width=800&height=600&carousel=1" title="Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods between two breast cancer cell lines (BT549 mock vs BT549 Torin1 treated)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup">Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods between two breast cancer cell lines (BT549 mock vs BT549 Torin1 treated).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F8.medium.gif" width="402" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F8.medium.gif" width="402" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1:
                               </span>
                               <p id="p-93" class="first-child">
                                Venn diagram shows the overlapped genes with the 3’-UTR APA events identified by three methods between two breast cancer cell lines (BT549 mock vs BT549 Torin1 treated).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              5 Experimental results with TCGA BRCA samples
                             </h3>
                             <p id="p-94">
                              One pair of TCGA breast cancer normal and tumor samples were selected and ran through APA-Scan for the detection of 3’-UTR APA. A total of 1266 APA events were detected between the two samples, whereas 1170 (92%) significant APA events were identified in tumor and 96 (8%) significant APA events were found in normal tissue sample. To inspect the correlation between 3’-UTR APA events and the gene expression profiles, we did the differential gene expression analysis and identified the genes in both tumor and normal samples. The result is illustrated in
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              . In the scatter plot, the y-axis denotes the Log
                              <sub>
                               2
                              </sub>
                              fold-change in the differential gene expression analysis and the x-axis shows the significance of UTR-APA (Log
                              <sub>
                               10
                              </sub>
                              p-value). The left three sections and the right three sections show the 3’-UTR truncated genes in tumor and normal sample, respectively. The top three sections and the bottom three sections represent the up-regulated and down-regulated genes in normal tissue over tumor sample. This plot leads us to the observation that majority (>72%) of the 3’-UTR APA genes are not differentially expressed.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F9.large.jpg?width=800&height=600&carousel=1" title="Scatter plot of APA and differentially expressed genes for TCGA tumor (TCGA-BH-A0BQ-01A) vs. matched normal tissue (TCGA-BH-A0BQ-11A) sample. Red dots represent individual gene in the analysis. Horizontal blue-dashed lines represent the cutoff values for two-fold changes in differential gene expression. Vertical green-dashed lines represent the cutoff values for log10(p-value) of 3’-UTR APA determined by the Chi-squared test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-402770489" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Scatter plot of APA and differentially expressed genes for TCGA tumor (TCGA-BH-A0BQ-01A) vs. matched normal tissue (TCGA-BH-A0BQ-11A) sample. Red dots represent individual gene in the analysis. Horizontal blue-dashed lines represent the cutoff values for two-fold changes in differential gene expression. Vertical green-dashed lines represent the cutoff values for log<sub>10</sub>(p-value) of 3’-UTR APA determined by the Chi-squared test.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F9.medium.gif" width="440" height="329"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F9.medium.gif" width="440" height="329"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2:
                               </span>
                               <p id="p-95" class="first-child">
                                Scatter plot of APA and differentially expressed genes for TCGA tumor (TCGA-BH-A0BQ-01A) vs. matched normal tissue (TCGA-BH-A0BQ-11A) sample. Red dots represent individual gene in the analysis. Horizontal blue-dashed lines represent the cutoff values for two-fold changes in differential gene expression. Vertical green-dashed lines represent the cutoff values for log
                                <sub>
                                 10
                                </sub>
                                (p-value) of 3’-UTR APA determined by the Chi-squared test.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-33">
                            <h2 class="">
                             APA-Scan User Manual
                            </h2>
                            <div class="section" id="sec-34">
                             <h3>
                              1 About
                             </h3>
                             <p id="p-96">
                              APA-Scan is a computational tool which can detect and visualize genome-wide 3’-UTR APA events. APA-Scan integrates both 3’-end-seq (an RNA-seq method with a specific enrichment of 3’-ends ofmRNA) data and the location information of predicted canonical PASs with RNA-seq data to improve the quantitative definition of genome-wide UTR-APA events. It is also advantageous in producing high quality plots of the user defined events.
                             </p>
                            </div>
                            <div class="section" id="sec-35">
                             <h3>
                              2 Download
                             </h3>
                             <p id="p-97">
                              APA-Scan is downloadable directly from
                              <a href="https://github.com/compbiolabucf/APA-Scan">
                               https://github.com/compbiolabucf/APA-Scan
                              </a>
                              . Users need to have python (version 3.0 or higher) installed in their machine to run APA-Scan.
                             </p>
                            </div>
                            <div class="section" id="sec-36">
                             <h3>
                              3 Required Softwares
                             </h3>
                             <ol class="list-ord" id="list-6">
                              <li id="list-item-19">
                               <p id="p-98">
                                Python (version 3.0 or higher)
                               </p>
                              </li>
                              <li id="list-item-20">
                               <p id="p-99">
                                Samtools 0.1.8* [This specific version]
                               </p>
                              </li>
                             </ol>
                             <div class="section" id="sec-37">
                              <h4>
                               Required python packages
                              </h4>
                              <ol class="list-ord" id="list-7">
                               <li id="list-item-21">
                                <p id="p-100">
                                 Pandas: $ pip install pandas
                                </p>
                               </li>
                               <li id="list-item-22">
                                <p id="p-101">
                                 Bio: $ pip install biopython
                                </p>
                               </li>
                               <li id="list-item-23">
                                <p id="p-102">
                                 Scipy: $ pip install scipy
                                </p>
                               </li>
                               <li id="list-item-24">
                                <p id="p-103">
                                 Numpy: $ pip install numpy
                                </p>
                               </li>
                               <li id="list-item-25">
                                <p id="p-104">
                                 Peakutils: $ pip install PeakUtils
                                </p>
                               </li>
                              </ol>
                             </div>
                            </div>
                            <div class="section" id="sec-38">
                             <h3>
                              4 Run APA-Scan
                             </h3>
                             <p>
                              APA-Scan can handle both human and mouse data for detecting potential APA truncation sites. The tool is designed to follow the format of Refseq annotation and genome file from UCSC Genome Browser. Users need to have the following two files in the parent directory in order to run APA-Scan:
                             </p>
                             <ol class="list-ord" id="list-8">
                              <li id="list-item-26">
                               <p id="p-106">
                                Refseq annotation (.txt format)
                               </p>
                              </li>
                              <li id="list-item-27">
                               <p id="p-107">
                                Genome fasta file (downloaded from UCSC genome browser)
                               </p>
                              </li>
                             </ol>
                             <div class="section" id="sec-39">
                              <h4>
                               4.1 Required files
                              </h4>
                              <p id="p-108">
                               APA-Scan has two python scripts:
                               <strong>
                                APA-Scan.py, Make-Plots.py
                               </strong>
                              </p>
                              <p id="p-109">
                               And 1 configuration file:
                               <strong>
                                configuration.ini
                               </strong>
                              </p>
                              <p id="p-110">
                               The configuration file allows the user to specify the directories of the input samples, the species to be analyzed and the directory where all output files will be stored.
                              </p>
                              <p id="p-111">
                               APA-Scan supports the analysis of multiple samples that belong to two different groups-all BAM files inside the input1 directory will be considered as part of the first group, and all BAM files inside the input2 directory will be considered as part of the second group. It is required to have at least one BAM file in each input directory.
                              </p>
                             </div>
                             <div class="section" id="sec-40">
                              <h4>
                               4.2 Running with parameters in the configuration.ini file
                              </h4>
                              <div id="T6" class="table pos-anchor">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="4.2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T6.medium.gif" width="440" height="297"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="4.2" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T6.medium.gif" width="440" height="297"/>
                                   </noscript>
                                  </span>
                                 </div>
                                </div>
                               </div>
                              </div>
                              <p id="p-112">
                               An example of the congiration.ini file is provided below:
                              </p>
                              <div id="F10" class="fig pos-anchor odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F10.medium.gif" width="440" height="269"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F10.medium.gif" width="440" height="269"/>
                                   </noscript>
                                  </span>
                                 </div>
                                </div>
                               </div>
                              </div>
                              <p id="p-113">
                               Once the parameters have been specified in the configuration file, the user will open a terminal and enter the following command to run APA-Scan:
                              </p>
                              <div class="section" id="sec-41">
                               <h5>
                                $ python3 APA-Scan.py
                               </h5>
                               <p id="p-114">
                                APA-Scan.py will generate several intermediary files in the output directory. After computing the significance of the association between the two groups of samples, the final results will be written in the file named
                                <strong>
                                 Group1_Vs_Group2.csv
                                </strong>
                                . The following image shows some of the generated fields in Group1_Vs_Group2.csv:
                               </p>
                               <div id="T7" class="table pos-anchor">
                                <div class="highwire-figure">
                                 <div class="fig-inline-img-wrapper">
                                  <div class="fig-inline-img">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Table" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T7.medium.gif" width="440" height="246"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Table" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T7.medium.gif" width="440" height="246"/>
                                    </noscript>
                                   </span>
                                  </div>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-42">
                             <h3>
                              5 Run Make-plots.py
                             </h3>
                             <p id="p-115">
                              Make-plots.py also requires the same configuration file to run. It will use the input and output directories listed in the configuration file and prepare a read coverage plot along with the 3’-UTR annotation based on user defined region.
                             </p>
                             <div id="sec-43" class="subsection">
                              <div class="section" id="sec-44">
                               <h5>
                                python3 Make-plots.py
                               </h5>
                               <p id="p-116">
                                After executing this command above for a few seconds, Make-plots.py will ask the user to insert the region of interest in a specific format:
                               </p>
                              </div>
                              <div class="section" id="sec-45">
                               <h5>
                                Chrom:GeneName:RegionStart-RegionEnd
                               </h5>
                              </div>
                             </div>
                             <div class="section" id="sec-46">
                              <h4>
                               5.1 Make-plots.py parameter descriptions
                              </h4>
                              <div id="T8" class="table pos-anchor">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="5.1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T8.medium.gif" width="440" height="144"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="5.1" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/T8.medium.gif" width="440" height="144"/>
                                   </noscript>
                                  </span>
                                 </div>
                                </div>
                               </div>
                              </div>
                              <p id="p-117">
                               Example:
                               <strong>
                                chr1:Tceb1:16641724-16643478
                               </strong>
                              </p>
                              <p id="p-118">
                               Make-Plots.py will generate a visual representation of the results shown for each of the regions entered. The plot will illustrate the most significant transcript cleavage site with a red vertical bar on top of RNA-seq read data (and 3’end-seq if available). If the input parameters have 3’end-seq information along with the RNA-seq, then it will generate plots for both cases (See figure below). It will also show the UTR truncation point (annotated and unannotated) at the bottom panel.
                              </p>
                              <div id="F11" class="fig pos-anchor even">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F11.medium.gif" width="440" height="399"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/30/2020.02.16.951657/F11.medium.gif" width="440" height="399"/>
                                   </noscript>
                                  </span>
                                 </div>
                                </div>
                               </div>
                              </div>
                              <p id="p-119">
                               The first two subplots of the figure represent the read coverage of the two biological conditions. The bottom subplot shows the gene annotation and the exon information of that gene.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-7">
                               A more comprehensive experiment is conducted in this revision
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-8">
                               <a href="https://github.com/compbiolabucf/APA-Scan">
                                https://github.com/compbiolabucf/APA-Scan
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section glossary">
                            <h2 class="">
                             Abbreviations
                            </h2>
                            <dl id="def-list-1">
                             <dt id="def-item-1">
                              APA
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              alternative polyadenylation
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-2">
                              3’-UTR
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              3’-untranslated region
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-3">
                              mTOR
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              mechanistic target of rapamycin
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-4">
                              PAS
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              polyadenylation signal
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-5">
                              MEFs
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              mouse embryonic fibroblasts
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-6">
                              WT
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              wild-type
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-7">
                              ROC
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              receiver operating characteristic
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-8">
                              AUC
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              area under the ROC curve
                              <span class="def-item-dd-sep">
                              </span>
                             </dd>
                             <dt id="def-item-9">
                              qPCR
                              <span class="def-item-dt-sep">
                              </span>
                             </dt>
                             <dd>
                              quantitative polymerase chain reaction
                             </dd>
                            </dl>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.1" data-doi="10.1101/gad.17268411">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Proudfoot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.J.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Ending the message: poly (A) signals then and now
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes & development
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  17
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1770
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1782
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DProudfoot%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D25%26rft.issue%253D17%26rft.spage%253D1770%26rft.epage%253D1782%26rft.atitle%253DEnding%2Bthe%2Bmessage%253A%2Bpoly%2528A%2529%2Bsignals%2Bthen%2Band%2Bnow%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.17268411%26rft_id%253Dinfo%253Apmid%252F21896654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI1LzE3LzE3NzAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8zMC8yMDIwLjAyLjE2Ljk1MTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.2" data-doi="10.1016/j.tibs.2013.03.005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Manley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.L.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Alternative cleavage and polyadenylation: the long and short of it
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in Biochemical Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  312
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BBiochemical%2BSciences%26rft.volume%253D38%26rft.spage%253D312%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2013.03.005%26rft_id%253Dinfo%253Apmid%252F23632313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tibs.2013.03.005&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23632313&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000320480100006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.3" data-doi="10.1038/nrg3482">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elkon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ugalde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Agami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Alternative cleavage and polyadenylation: extent, regulation and function
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  496
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D14%26rft.spage%253D496%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3482%26rft_id%253Dinfo%253Apmid%252F23774734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3482&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23774734&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yeh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Analyses of alternative polyadenylation: from old school biochemistry to high-throughput technologies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMB reports
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  201
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMB%2Breports%26rft.volume%253D50%26rft.spage%253D201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.5" data-doi="10.1016/j.cell.2009.06.016">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mayr
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.P.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Widespread shortening of 3’ UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  138
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  673
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  684
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMayr%26rft.auinit1%253DC.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D673%26rft.epage%253D684%26rft.atitle%253DWidespread%2Bshortening%2Bof%2B3%2527UTRs%2Bby%2Balternative%2Bcleavage%2Band%2Bpolyadenylation%2Bactivates%2Boncogenes%2Bin%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.06.016%26rft_id%253Dinfo%253Apmid%252F19703394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.06.016&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19703394&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000269156100010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lembo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Di Cunto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Provero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Shortening of 3 UTRs correlates with poor prognosis in breast and lung cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  31129
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D7%26rft.spage%253D31129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.7" data-doi="10.1158/1078-0432.CCR-12-0543">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Diosdado
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rooijers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elkon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bolijn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carvalho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meijer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Agami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Alternative cleavage and polyadenylation during colorectal cancer development
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clinical Cancer Research
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  5256
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5266
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClinical%2BCancer%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1078-0432.CCR-12-0543%26rft_id%253Dinfo%253Apmid%252F22874640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNsaW5jYW5yZXMiO3M6NToicmVzaWQiO3M6MTA6IjE4LzE5LzUyNTYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNy8zMC8yMDIwLjAyLjE2Ljk1MTY1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yeh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Jong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.-H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bae
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beckman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hwang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.-S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  mRNA 3-UTR shortening is a molecular signature of mTORC1 activation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.9" data-doi="10.1093/nar/gky340">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yeh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  An integrative model for alternative polyadenylation, IntMAP, delineates mTOR-modulated endoplasmic reticulum stress response
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  5996
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6008
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D46%26rft.spage%253D5996%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky340%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky340&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoffman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bublik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   P. Ugalde
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elkon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Biniashvili
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Agami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pilpel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  3’UTR shortening potentiates microRNA-based repression of pro-differentiation genes in proliferating human cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1005879
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bgenetics%26rft.volume%253D12%26rft.spage%253D1005879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.11" data-doi="10.1126/science.1155390">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sandberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.B.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Proliferating cells express mRNAs with shortened 3’untranslated regions and fewer microRNA target sites
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  320
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5883
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1643
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1647
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSandberg%26rft.auinit1%253DR.%26rft.volume%253D320%26rft.issue%253D5883%26rft.spage%253D1643%26rft.epage%253D1647%26rft.atitle%253DProliferating%2BCells%2BExpress%2BmRNAs%2Bwith%2BShortened%2B3%2527%2BUntranslated%2BRegions%2Band%2BFewer%2BMicroRNA%2BTarget%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155390%26rft_id%253Dinfo%253Apmid%252F18566288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4My8xNjQzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMzAvMjAyMC4wMi4xNi45NTE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donehower
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cooper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wheeler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wagner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Dynamic Analyses of Alternative Polyadenylation from RNA-Seq Reveal 3’-UTR Landscape Across 7 Tumor Types
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5274
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D5%26rft.spage%253D5274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.13" data-doi="10.1093/bioinformatics/btu189">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  A change-point model for identifying 3’ UTR switching by nextgeneration RNA sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  15
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2162
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2170
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu189%26rft_id%253Dinfo%253Apmid%252F24728858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu189&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24728858&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000340049100011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.14" data-doi="10.1093/bioinformatics/btv035">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Le Pera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mazzapioda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tramontano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  3USS: a web server for detecting alternative 3’ UTRs from RNA-seq experiments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1845
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1847
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv035%26rft_id%253Dinfo%253Apmid%252F25617413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv035&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25617413&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.15" data-doi="10.1093/bioinformatics/bty029">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ye
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Long
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ji
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  APAtrap: identification and quantification of alternative polyadenylation sites from RNA-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1841
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1849
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D1841%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty029%26rft_id%253Dinfo%253Apmid%252F29360928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty029&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29360928&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.16" data-doi="10.1186/s12864-017-3899-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Magana-Mora
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kalkatawi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bajic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.B.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Omni-PolyA: a method and tool for accurate recognition of Poly(A) signals in human genomic DNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3899-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12864-017-3899-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.17" data-doi="10.1261/rna.2581711">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shepard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Choi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.-A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flanagan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hertel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Complex and dynamic landscape of rna polyadenylation revealed by pas-seq
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Rna
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  772
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRna%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2581711%26rft_id%253Dinfo%253Apmid%252F21343387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNzYxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDcvMzAvMjAyMC4wMi4xNi45NTE2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.18" data-doi="10.1093/nar/gks666">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Griebel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zacher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ribeca
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raineri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lacroix
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guigó
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sammeth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Modelling and simulating generic RNA-Seq experiments with the flux simulator
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic acids research
                                 </abbr>
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  10073
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10083
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2Bacids%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks666%26rft_id%253Dinfo%253Apmid%252F22962361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks666&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22962361&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000310970700018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.19" data-doi="10.1186/gb-2010-11-12-220">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oshlack
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Young
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  From RNA-seq reads to differential expression results
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  220
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DOshlack%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D220%26rft.epage%253D220%26rft.atitle%253DFrom%2BRNA-seq%2Breads%2Bto%2Bdifferential%2Bexpression%2Bresults.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-12-220%26rft_id%253Dinfo%253Apmid%252F21176179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-12-220&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21176179&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.20" data-doi="10.1101/gr.202432.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gruber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmidt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gruber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belmadani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zavolan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  A comprehensive analysis of 3 end sequencing data sets reveals novel polyadenylation signals and the repressive role of heterogeneous ribonucleoprotein C on cleavage and polyadenylation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1145
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1159
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.202432.115%26rft_id%253Dinfo%253Apmid%252F27382025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMTE0NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA3LzMwLzIwMjAuMDIuMTYuOTUxNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.21" data-doi="10.1093/bioinformatics/btp352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Handsaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wysoker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fennell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Homer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abecasis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  The sequence alignment/map format and SAMtools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  16
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2078
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2079
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19505943&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268808600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.22" data-doi="10.1186/gb-2013-14-1-r1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pertea
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trapnell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pimentel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kelley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.L.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-1-r1%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-1-r1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F30%2F2020.02.16.951657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.16.951657v3.23" data-doi="10.1186/gb-2009-10-1-r1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trapnell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pop
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.L.
                                  </span>
                                 </span>
                                 :
                                 <span class="cit-article-title">
                                  Ultrafast and memory-efficient alignment of short DNA sequences to the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 )
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-1-r1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-1-r1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.16.951657v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.07.29.502011v1" title="Establishing Physalis as a new Solanaceae model system enables genetic reevaluation of the inflated calyx syndrome" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted July 30, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.02.16.951657v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/07/30/2020.02.16.951657.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.02.16.951657v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.02.16.951657v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2670172"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-i1Pev4mX9yS2Yfu-WvZ1H1b3oi1Lhm7FsAy41SAkHeA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484467275"/>
                                   <input type="hidden" name="captcha_token" value="59182adaea44b3ef3fd5b112a9ec7f02"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2670172--3" data-node-nid="2670172" id="node-2670172--4868179102" data-pisa="biorxiv;2020.02.16.951657v3" data-pisa-master="biorxiv;2020.02.16.951657" data-seqnum="2670172" data-apath="/biorxiv/early/2022/07/30/2020.02.16.951657.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Naima Ahmed
                                  </span>
                                  <span class="nlm-surname">
                                   Fahmi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Khandakar Tanvir
                                  </span>
                                  <span class="nlm-surname">
                                   Ahmed
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Jae-Woong
                                  </span>
                                  <span class="nlm-surname">
                                   Chang
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Heba
                                  </span>
                                  <span class="nlm-surname">
                                   Nassereddeen
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Deliang
                                  </span>
                                  <span class="nlm-surname">
                                   Fan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Jeongsik
                                  </span>
                                  <span class="nlm-surname">
                                   Yong
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Wei
                                  </span>
                                  <span class="nlm-surname">
                                   Zhang
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.02.16.951657;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.02.16.951657
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&title=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&title=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&text=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&t=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&title=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&title=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&title=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2670172--5" data-node-nid="2670172" id="citation-node-2670172--61918432109" data-pisa="biorxiv;2020.02.16.951657v3" data-pisa-master="biorxiv;2020.02.16.951657" data-seqnum="2670172" data-apath="/biorxiv/early/2022/07/30/2020.02.16.951657.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     APA-Scan: Detection and Visualization of 3’-UTR Alternative Polyadenylation with RNA-seq and 3’-end-seq Data
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Naima Ahmed
                                      </span>
                                      <span class="nlm-surname">
                                       Fahmi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Khandakar Tanvir
                                      </span>
                                      <span class="nlm-surname">
                                       Ahmed
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Jae-Woong
                                      </span>
                                      <span class="nlm-surname">
                                       Chang
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Heba
                                      </span>
                                      <span class="nlm-surname">
                                       Nassereddeen
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Deliang
                                      </span>
                                      <span class="nlm-surname">
                                       Fan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Jeongsik
                                      </span>
                                      <span class="nlm-surname">
                                       Yong
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Wei
                                      </span>
                                      <span class="nlm-surname">
                                       Zhang
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.02.16.951657;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.02.16.951657
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Fahmi%2C+Naima+Ahmed&rft.au=Ahmed%2C+Khandakar+Tanvir&rft.au=Chang%2C+Jae-Woong&rft.au=Nassereddeen%2C+Heba&rft.au=Fan%2C+Deliang&rft.au=Yong%2C+Jeongsik&rft.au=Zhang%2C+Wei">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2670172/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2670172/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2670172/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2670172/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2670172/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2670172/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2670172/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2670172/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2670172/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2670172/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2670172/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2670172/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&count=horizontal&via=&text=APA-Scan%3A%20Detection%20and%20Visualization%20of%203%E2%80%99-UTR%20Alternative%20Polyadenylation%20with%20RNA-seq%20and%203%E2%80%99-end-seq%20Data&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.16.951657v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.02.16.951657v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
