<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.10.24.353599v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.10.24.353599v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.10.24.353599v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/11/11/2020.10.24.353599.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.10.24.353599v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process"/>
  <meta name="DC.Identifier" content="10.1101/2020.10.24.353599"/>
  <meta name="DC.Date" content="2022-11-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Background The human microbiome comprises the microorganisms that inhabit the various locales of the human body and plays a vital role in human health. The composition of a microbial population is often quantified through measures of species diversity, which summarize the number of species along with their relative abundances into a single value. In a microbiome sample there will certainly be species missing from the target population which will affect the diversity estimates.

Methods We employ a model based on the hierarchical Pitman-Yor (HPY) process to model the species abundance distributions over multiple populations. The model parameters are estimated using a Gibbs sampler. We also derive estimates of species diversity, conditional and unconditional on the observed data, as a function of the HPY parameters Finally, we derive a general formula for the Hill numbers in the HPY context.

Results We show that the Gibbs sampler for the HPY model performs well in simulations. We also show that the conditional estimates of diversity from the HPY model improve over naïve estimates when species are missing. Similarly the conditional HPY estimates tend to perform better than the naïve estimates especially when the number of individuals sampled from a population is small.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Kevin Mcgregor"/>
  <meta name="DC.Contributor" content="Aurélie Labbe"/>
  <meta name="DC.Contributor" content="Celia M.T. Greenwood"/>
  <meta name="DC.Contributor" content="Todd Parsons"/>
  <meta name="DC.Contributor" content="Christopher Quince"/>
  <meta name="article:published_time" content="2022-11-11"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process"/>
  <meta name="citation_abstract" lang="en" content="<h3>Background</h3>
<p>The human microbiome comprises the microorganisms that inhabit the various locales of the human body and plays a vital role in human health. The composition of a microbial population is often quantified through measures of species diversity, which summarize the number of species along with their relative abundances into a single value. In a microbiome sample there will certainly be species missing from the target population which will affect the diversity estimates.</p><h3>Methods</h3>
<p>We employ a model based on the hierarchical Pitman-Yor (HPY) process to model the species abundance distributions over multiple populations. The model parameters are estimated using a Gibbs sampler. We also derive estimates of species diversity, conditional and unconditional on the observed data, as a function of the HPY parameters Finally, we derive a general formula for the Hill numbers in the HPY context.</p><h3>Results</h3>
<p>We show that the Gibbs sampler for the HPY model performs well in simulations. We also show that the conditional estimates of diversity from the HPY model improve over naïve estimates when species are missing. Similarly the conditional HPY estimates tend to perform better than the naïve estimates especially when the number of individuals sampled from a population is small.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.10.24.353599v2"/>
  <meta name="citation_id" content="2020.10.24.353599v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.10.24.353599"/>
  <meta name="citation_num_pages" content="15"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.10.24.353599"/>
  <meta name="citation_author" content="Kevin Mcgregor"/>
  <meta name="citation_author_institution" content="Department of Mathematics and Statistics, York University"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, Biostatistics, and Occupational Health, McGill University"/>
  <meta name="citation_author_institution" content="Warwick Medical School, University of Warwick"/>
  <meta name="citation_author_institution" content="Lady Davis Institute for Medical Research"/>
  <meta name="citation_author" content="Aurélie Labbe"/>
  <meta name="citation_author_institution" content="Department of Decision Sciences, HEC Montréal"/>
  <meta name="citation_author" content="Celia M.T. Greenwood"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, Biostatistics, and Occupational Health, McGill University"/>
  <meta name="citation_author_institution" content="Lady Davis Institute for Medical Research"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2427-5696"/>
  <meta name="citation_author" content="Todd Parsons"/>
  <meta name="citation_author_institution" content="Laboratoire de Probabilités, Statistique et Modélisation, Sorbonne Université"/>
  <meta name="citation_author" content="Christopher Quince"/>
  <meta name="citation_author_institution" content="Warwick Medical School, University of Warwick"/>
  <meta name="citation_author_institution" content="High-Resolution Microbiomics Group, Earlham Institute"/>
  <meta name="citation_author_email" content="c.quince@warwick.ac.uk"/>
  <meta name="citation_reference" content="Arbel, J., Mengersen, K., Rousseau, J. et al. (2016). Bayesian nonparametric dependent model for partially replicated data: the influence of fuel spills on species diversity. The Annals of Applied Statistics 10 1496–1516."/>
  <meta name="citation_reference" content="Battiston, M., Favaro, S. and Teh, Y. W. (2018). Multi-armed bandit for species discovery: a Bayesian nonparametric approach. Journal of the American Statistical Association 113 455–466."/>
  <meta name="citation_reference" content="Buntine, W. (2012). Hyper-parameter Estimation for the Dirichlet Prior."/>
  <meta name="citation_reference" content="Camerlenghi, F., Dumitrascu, B., Ferrari, F., Engelhardt, B. E. and Favaro, S. (2020). Nonpara-metric Bayesian multiarmed bandits for single-cell experiment design. The Annals of Applied Statistics 14 2003–2019."/>
  <meta name="citation_reference" content="Cerquetti, A. (2015). Bayesian Estimation of Gini-Simpson’s Index Under Mainland-Island Community Struc-ture. In Convegno della Società Italiana di Statistica 61–70. Springer."/>
  <meta name="citation_reference" content="Chesson, P. (2000). Mechanisms of maintenance of species diversity. Annual review of Ecology and Systematics 31 343–366."/>
  <meta name="citation_reference" content="Clauset, A., Shalizi, C. R. and Newman, M. E. (2009). Power-law distributions in empirical data. SIAM review 51 661–703."/>
  <meta name="citation_reference" content="Crane, H. et al. (2016). The ubiquitous Ewens sampling formula. Statistical science 31 1–19."/>
  <meta name="citation_reference" content="Escobar, M. D. and West, M. (1995). Bayesian density estimation and inference using mixtures. Journal of the american statistical association 90 577–588."/>
  <meta name="citation_reference" content="Etienne, R. S. (2005). A new sampling formula for neutral biodiversity. Ecology letters 8 253–260."/>
  <meta name="citation_reference" content="Ewens, W. J. (1972). The sampling theory of selectively neutral alleles. Theoretical population biology 3 87–112."/>
  <meta name="citation_reference" content="Goldwater, S., Johnson, M. and Griffiths, T. L. (2006). Interpolating between types and tokens by esti-mating power-law generators. In Advances in neural information processing systems 459–466."/>
  <meta name="citation_reference" content="Gorelick, R. (2006). Combining richness and abundance into a single diversity index using matrix analogues of Shannon’s and Simpson’s indices. Ecography 29 525–530."/>
  <meta name="citation_reference" content="Harris, K., Parsons, T. L., Ijaz, U. Z., Lahti, L., Holmes, I. and Quince, C. (2015). Linking statistical and ecological theory: Hubbell’s unified neutral theory of biodiversity as a hierarchical Dirichlet process. Proceedings of the IEEE 105 516–529."/>
  <meta name="citation_reference" content="Hill, M. O. (1973). Diversity and evenness: a unifying notation and its consequences. Ecology 54 427–432."/>
  <meta name="citation_reference" content="Hubbell, S. P. (2001). The unified neutral theory of biodiversity and biogeography (MPB-32) 32. Princeton University Press."/>
  <meta name="citation_reference" content="Huynh, V., Phung, D. and Venkatesh, S. (2016). Streaming variational inference for dirichlet process mix-tures. In Asian Conference on Machine Learning 237–252."/>
  <meta name="citation_reference" content="Jeraldo, P., Sipos, M., Chia, N., Brulc, J. M., Dhillon, A. S., Konkel, M. E., Larson, C. L., Nel-Son, K. E., Qu, A., Schook, L. B. et al. (2012). Quantification of the relative roles of niche and neutral processes in structuring gastrointestinal microbiomes. Proceedings of the National Academy of Sciences 109 9692–9698."/>
  <meta name="citation_reference" content="Kimura, M. (1968). Evolutionary rate at the molecular level. Nature 217 624–626."/>
  <meta name="citation_reference" content="Kurihara, K., Welling, M. and Teh, Y. W. (2007). Collapsed Variational Dirichlet Process Mixture Models. In IJCAI 7 2796–2801."/>
  <meta name="citation_reference" content="Li, K., Bihan, M. and MethÉ, B. A. (2013). Analyses of the stability and core taxonomic memberships of the human microbiome. PloS one 8 e63139."/>
  <meta name="citation_reference" content="Ren, B., Bacallado, S., Favaro, S., Holmes, S. and Trippa, L. (2017). Bayesian nonparametric ordination for the analysis of microbial communities. Journal of the American Statistical Association 112 1430–1442."/>
  <meta name="citation_reference" content="Turnbaugh, P. J., Hamady, M., Yatsunenko, T., Cantarel, B. L., Duncan, A., Ley, R. E., So-gin, M. L., Jones, W. J., Roe, B. A., Affourtit, J. P. et al. (2009). A core gut microbiome in obese and lean twins. nature 457 480–484."/>
  <meta name="citation_reference" content="Yamato, H., Sibuya, M. and Nomachi, T. (2001). Ordered sample from two-parameter GEM distribution. Statistics & probability letters 55 19–27."/>
  <meta name="twitter:title" content="Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Background The human microbiome comprises the microorganisms that inhabit the various locales of the human body and plays a vital role in human health. The composition of a microbial population is often quantified through measures of species diversity, which summarize the number of species along with their relative abundances into a single value. In a microbiome sample there will certainly be species missing from the target population which will affect the diversity estimates.

Methods We employ a model based on the hierarchical Pitman-Yor (HPY) process to model the species abundance distributions over multiple populations. The model parameters are estimated using a Gibbs sampler. We also derive estimates of species diversity, conditional and unconditional on the observed data, as a function of the HPY parameters Finally, we derive a general formula for the Hill numbers in the HPY context.

Results We show that the Gibbs sampler for the HPY model performs well in simulations. We also show that the conditional estimates of diversity from the HPY model improve over naïve estimates when species are missing. Similarly the conditional HPY estimates tend to perform better than the naïve estimates especially when the number of individuals sampled from a population is small.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Background The human microbiome comprises the microorganisms that inhabit the various locales of the human body and plays a vital role in human health. The composition of a microbial population is often quantified through measures of species diversity, which summarize the number of species along with their relative abundances into a single value. In a microbiome sample there will certainly be species missing from the target population which will affect the diversity estimates.

Methods We employ a model based on the hierarchical Pitman-Yor (HPY) process to model the species abundance distributions over multiple populations. The model parameters are estimated using a Gibbs sampler. We also derive estimates of species diversity, conditional and unconditional on the observed data, as a function of the HPY parameters Finally, we derive a general formula for the Hill numbers in the HPY context.

Results We show that the Gibbs sampler for the HPY model performs well in simulations. We also show that the conditional estimates of diversity from the HPY model improve over naïve estimates when species are missing. Similarly the conditional HPY estimates tend to perform better than the naïve estimates especially when the number of individuals sampled from a population is small.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-11-11"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2844652"/>
  <title>
   Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WTUiOFLCBCySUgjiAbUxV37TvHRuwThAAhbhTAR1ogU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.10.24.353599v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.10.24.353599v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.10.24.353599v2"}],"nid":"2844652","apath":"\/biorxiv\/early\/2022\/11\/11\/2020.10.24.353599.atom","pisa":"biorxiv;2020.10.24.353599v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.10.24.353599v2","title":"Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process","identifier":"node\/2844652"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221yC\\\\\\\/lbC+D3oENxm+xtX1qNNpiiDHfPSytAWOaL8UPG6b1X69xHps7ljr9SsuY4yBUqYaUPAs0kBT59ytK60stQKq61cVnmxDlCrYxnTkDxmIow1H08nNreNgDCQ7OS+Wo\\\\\\\/OGR+quXx53V++VlxNHFYrYlWmlXKA+RrqZVRfHzr7SuBxjg7KWm6kZpugVjAGEPPZVvvAxnqc7tiuUREdlpuUdkRuINeRxcVVgHCauMtUQli2ZnuJOMy0duaS92vMgUxXGqFI39HYaLqXjASEr\\\\\\\/ZBjJSeRwh1dfCXfRik15+qDcpCWJEN6F1A2fK6mAe1VaFyJgN4w7fEulGwV\\\\\\\/juMBN\\\\\\\/MM+x9pfV06YkeZdr9JKj28EBerVDsbLZ9iM1GzRLIL6E7QkPXDL7beeb8bcRa6Cxqm8cd\\\\\\\/Jm17ViypmewqPkgQTGou0WGPIZLaD91F4mJyoOVF6cIAfHR2SW\\\\\\\/1HpBLsKjLp3kBZHVJQ4xmUu45+OXqpAfuFr1nwU7IaGJge0Mm8znhZH6iBfE2ewBe0p\\\\\\\/Ic2WMPlljkz7FzE6s6eScJTgGaHBQfdJqvOcAFVQEfwblhOxLTJgNEpYVTgvy\\\\\\\/w5XwV12wmAtWoZHScmO78LskLMcZRH2EMzRtN+TmAf7gY8JvaYt8GKqb2qfrZF3KIypxF3Y1ueBKe4ObPWrTlcaG\\\\\\\/jGXfcrUkdciRP6+RzKUOvjiQV6K6xeNxyObYxV3JJCTWwzPoivNhrlKehxwkrVjfcqQFlgcH+GfBpCCeBJyRv6e69\\\\\\\/MaJ+wieOHyiBFof343K4pzpugkLq1hUU4Tz0d0apK3jnUGia0M1xqrpurTN\\\\\\\/jYELcoDisD37SDtmgl4OBu\\\\\\\/+Q+TFyGGzRsvMfmKLA76ZsiRZbUXNuOxM438sg5dbLoFAhcasV754Pukg6bAMRVWo1aVF+KOc+KJNCYbggSTlUvcikPVVo0gqGFkMTYuDZMWLxuuZpuk72avRR5\\\\\\\/A+47HqqVR9qhrlpuRrbeGNDXNEWQOW80EEPC94sskQ14vu\\\\\\\/NxTtYQ2fwJqjjTGQMG7m61l12s6KlQczVlyBer55SpLB2lor2fhX0SB2oU1i0EA8yQRjGOFavXTgFv679Z4U+OzPhJcLJm+6CyN1iND6j8k4BCGU8ZbZKRclcS20WvgOAJm4OWTevcM7IFwxX+TWjrdAjQrcVoQrAu9A\\\\\\\/M1XCA8qgk14lnumo8\\\\\\\/4r7kYRBn4G\\\\\\\/fOzNsrLB\\\\\\\/EHJFoRYQjqnAZUyHAKMWw1CswQYskeFn9TmO2nBXwh0sCgO+MpGf0PSPwravXBDclna\\\\\\\/Vn4ZSLkoFWCx3rm7xuRawghn5uS4W0HzaToVI2HtzFR95vWdwH7NvDJc52DC1pKtpwcB56e8ySgHpex1\\\\\\\/mSlCEXSc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022XMJ\\\\\\\/\\\\\\\/TnQL+2dA4+9Bwxz9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226382162a495bc9805345a1d04fcbbccb2c26922fba379cff7dba3bf13856fcbd\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.10.24.353599v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2844652 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.10.24.353599v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1880623783">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1880623783" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1421933499" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-nWkEkTZ_JF4_I7IPVjl_10l0MArA2uHL1Dk-49akFgg"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2844652" data-node-nid="2844652" id="node-2844652--21664795588" data-pisa="biorxiv;2020.10.24.353599v2" data-pisa-master="biorxiv;2020.10.24.353599" data-apath="/biorxiv/early/2022/11/11/2020.10.24.353599.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Kevin
                      </span>
                      <span class="nlm-surname">
                       Mcgregor
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Aurélie
                      </span>
                      <span class="nlm-surname">
                       Labbe
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-2427-5696" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Celia M.T.
                      </span>
                      <span class="nlm-surname">
                       Greenwood
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Todd
                      </span>
                      <span class="nlm-surname">
                       Parsons
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Christopher
                      </span>
                      <span class="nlm-surname">
                       Quince
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.10.24.353599
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2844652--21664795588" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Kevin Mcgregor
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Mathematics and Statistics, York University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Epidemiology, Biostatistics, and Occupational Health, McGill University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Warwick Medical School, University of Warwick
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Lady Davis Institute for Medical Research
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kevin%2BMcgregor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Mcgregor%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKevin%2BMcgregor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Aurélie Labbe
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Department of Decision Sciences, HEC Montréal
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Aur%C3%A9lie%2BLabbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Labbe%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAur%25C3%25A9lie%2BLabbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Celia M.T. Greenwood
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Epidemiology, Biostatistics, and Occupational Health, McGill University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Lady Davis Institute for Medical Research
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Celia%2BM.T.%2BGreenwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Greenwood%20CM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACelia%2BM.T.%2BGreenwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2427-5696" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Celia M.T. Greenwood
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Todd Parsons
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Laboratoire de Probabilités, Statistique et Modélisation, Sorbonne Université
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Todd%2BParsons%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Parsons%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ATodd%2BParsons%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Christopher Quince
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Warwick Medical School, University of Warwick
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        High-Resolution Microbiomics Group, Earlham Institute
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Christopher%2BQuince%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Quince%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AChristopher%2BQuince%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:c.quince@warwick.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        c.quince@warwick.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.10.24.353599v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.24.353599v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.24.353599v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.24.353599v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.10.24.353599v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.10.24.353599v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2844652" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2844652/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2>
                             Abstract
                            </h2>
                            <div id="sec-1" class="subsection">
                             <p id="p-7">
                              <strong>
                               Background
                              </strong>
                              The human microbiome comprises the microorganisms that inhabit the various locales of the human body and plays a vital role in human health. The composition of a microbial population is often quantified through measures of species diversity, which summarize the number of species along with their relative abundances into a single value. In a microbiome sample there will certainly be species missing from the target population which will affect the diversity estimates.
                             </p>
                            </div>
                            <div id="sec-2" class="subsection">
                             <p id="p-8">
                              <strong>
                               Methods
                              </strong>
                              We employ a model based on the hierarchical Pitman-Yor (HPY) process to model the species abundance distributions over multiple populations. The model parameters are estimated using a Gibbs sampler. We also derive estimates of species diversity, conditional and unconditional on the observed data, as a function of the HPY parameters Finally, we derive a general formula for the Hill numbers in the HPY context.
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <p id="p-9">
                              <strong>
                               Results
                              </strong>
                              We show that the Gibbs sampler for the HPY model performs well in simulations. We also show that the conditional estimates of diversity from the HPY model improve over naïve estimates when species are missing. Similarly the conditional HPY estimates tend to perform better than the naïve estimates especially when the number of individuals sampled from a population is small.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-4">
                            <h2 class="">
                             1. Introduction
                            </h2>
                            <p id="p-12">
                             Human microbiome studies attempt to quantify the makeup of the different species that occupy the human body. The composition of the gut microbiome in particular plays an important part in human health. Modelling species abundance distributions in this framework has long been a goal in metagenomic datasets. There have been a number of methods based on Bayesian non-parametrics developed to model these kinds of community dynamics. In particular, the Dependent Dirichlet process has been applied in this context to species abundance distribution and diversity estimation, and has additionally been used to address the effects of covariates on these entities (
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Ren et al., 2017
                             </a>
                             ;
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              Arbel et al., 2016
                             </a>
                             ). However, these models were created from a statistical standpoint, and do not reflect the underlying principles of community ecology. In contrast, there are alternative statistical methods that attempt to describe and capture the dynamics of such distributions in a manner that is faithful to principles of ecological theory. Nevertheless, there are competing perspectives on how evolutionary processes and the environment affect species distributions (
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              Jeraldo et al., 2012
                             </a>
                             ). For example, one theory suggests that species assemblage is characterized by niches, which are defined by the allocation of resources in the population (
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              Chesson, 2000
                             </a>
                             ). There also exists a competing theory that describes “neutral” models, which assume that species are functionally equivalent and that processes such as immigration and birth/death events primarily contribute to community diversity (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Etienne, 2005
                             </a>
                             ). The theory of neutrality claims that most genetic diversity observed in a population is a result of chance, as opposed to Darwinian selection (
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              Kimura, 1968
                             </a>
                             ); that is, the birth rate that a particular species depends on the number of individuals of that species present in the population, rather than on the species’ ability to survive in the environment. This is analogous to the theory of neutrality for gene alleles that originated in the field of population genetics (
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Ewens, 1972
                             </a>
                             ). One important doctrine in neutral theory is Hubbell’s Unified Neutral Theory of Biodiversity (
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Hubbell, 2001
                             </a>
                             ). In Hubbell’s theory, it is assumed that there are a number of distinct local communities (sometimes called local populations) that are subject to immigration and birth/death processes. Immigration into each community is independent of the other communities, and all individuals that immigrate to a community do so from a conceptual metacommunity shared by all local communities. Immigration can happen at different rates across populations. The birth rate in the neutral model is on a per capita basis—births are more likely to occur for more highly abundant species. Additionally, the rate of speciation, i.e. the frequency at which new species appear (i.e. species that have not yet been observed in any population), is a defining parameter of the top-level metacommunity. Note that the metacommunity and the local communities in Hubbell’s model are sometimes referred to in the context of the mainland-island model, where immigration occurs from a mainland (analogous to metacommunity) to local islands surrounded by uninhabitable space (analagous to the local communities).
                            </p>
                            <p id="p-13">
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              Harris et al. (2015)
                             </a>
                             showed that the class of neutral models, when multiple local communities with differing population dynamics are considered and under certain conditions on individual mean reproductive success, converges to the hierarchical Dirichlet process (HDP) with large local population size. They developed a Gibbs sampler for the hierarchical Dirichlet process given a matrix of species counts among multiple local communities. The authors also presented a means for testing for neutrality in the populations after fitting the model. However, microbial abundance data has suggested that ranked species abundance distributions follow a power-law tail (
                             <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                              Li, Bihan and Methé, 2013
                             </a>
                             ). That is, if we consider the ranked proportions of species in a population, p
                             <sub>
                              k
                             </sub>
                             , k = 1, 2, … such that p
                             <sub>
                              1
                             </sub>
                             > p
                             <sub>
                              2
                             </sub>
                             > p
                             <sub>
                              3
                             </sub>
                             > …, then we have the following relationship for the ranked abundances:
                             <span class="disp-formula" id="disp-formula-1">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-1.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-1.gif"/>
                               </noscript>
                              </span>
                             </span>
                             for some a > 0 (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              Clauset, Shalizi and Newman, 2009
                             </a>
                             ). One limitation of the Dirichlet process model is it cannot accommodate a power-law tail in its species distribution. Conversely, a related process called the Pitman-Yor process, can indeed generate a species distribution that exhibits a power-law tail (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Goldwater, Johnson and Griffiths, 2006
                             </a>
                             ). It is then of value to pursue the question as to whether the Pitman-Yor model is more appropriate for modelling the species frequency distribution than the Dirichlet model. In this paper we investigate the use of the Pitman-Yor process in a hierarchical formulation called the hierarchical Pitman-Yor process (HPY process) to model abundance data in multiple microbial populations. The HPY model has been employed outside the context of community modelling (REFS). It has also been recently used in species discovery (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Battiston, Favaro and Teh, 2018
                             </a>
                             ) as well as in single-cell RNA-Seq data (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              Camerlenghi et al., 2020
                             </a>
                             ). However, to the authors’ knowledge, this model has not previously been used in a species diversity estimation framework, nor have estimates of diversity metrics been derived in terms of the HPY model parameters. Therefore, in this article we provide such a derivation of diversity metrics in this context; in particular, we derive a general formula for the Hill numbers as a function of HPY parameters.
                            </p>
                            <p id="p-14">
                             The HPY process is analagous to the mainland-island model, but it can also accommodate a departure from the assumption of neutrality. We develop a Gibbs sampler to fit the model parameters. In addition to the Gibbs sampler, we derive expressions for two measures of species diveristy in the context of the HPY process. We perform an extensive simulation study to investigate the performance of the HPY model under different situations, as well as the efficacy of the measures of species diversity defined in the HPY context.
                            </p>
                            <p id="p-15">
                             We begin in the next section with the definition of the Pitman-Yor Process. In the Methods section (
                             <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">
                              Section 2.1
                             </a>
                             ), we introduce the hierarchical Pitman-Yor model and give details for the Gibbs sampler as well as a description on calculating diversity in this model. In
                             <a id="xref-sec-13-1" class="xref-sec" href="#sec-13">
                              Section 3
                             </a>
                             we present results for an extensive simulation study that investigates the performance of the HPY model. Finally, in
                             <a id="xref-sec-14-1" class="xref-sec" href="#sec-14">
                              Section 4
                             </a>
                             we show results from applying the HPY model to a gut microbiome 16S sequencing dataset from a study on lean and obese twin pairs.
                            </p>
                            <div id="sec-5" class="subsection">
                             <h3>
                              1.1. Pitman-Yor process
                             </h3>
                             <p id="p-16">
                              Assume that, for 0 ≤ α < 1 and γ > −α, we generate a sequence of random variables V
                              <sub>
                               k
                              </sub>
                              , for k = 1, 2, … such that V
                              <sub>
                               k
                              </sub>
                              ∼ Beta(1 −α, γ + kα). Furthermore, define the following:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Then the Griffiths, Engen, and McCloskey distribution is defined as the joint distribution of (p
                              <sub>
                               1
                              </sub>
                              , p
                              <sub>
                               2
                              </sub>
                              , …) and is abbreviated as GEM(γ, α) (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Yamato, Sibuya and Nomachi, 2001
                              </a>
                              ).
                             </p>
                             <p id="p-17">
                              Let
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,… be a sequence of independent samples from some distribution H. If we draw the vector p ∼ GEM(γ, α) independently from each
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , then the Pitman-Yor process (or two-parameter Poisson-Dirichlet process) is defined as:
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is a discrete measure at
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               Buntine and Hutter, 2010
                              </a>
                              ). The Pitman-Yor process is abbreviated as PY(α, γ, H). α and γ are called the discount and concentration parameters, respectively. H is referred to as the base distribution. The Dirichlet process is a special case of the Pitman-Yor process where the discount parameter α = 0.
                             </p>
                             <p id="p-18">
                              A convenient way of drawing from the Pitman-Yor process is through the Chinese restaurant process representation. Assume that we have already drawn X
                              <sub>
                               1
                              </sub>
                              , …, X
                              <sub>
                               n
                              </sub>
                              among which we have drawn the values
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              directly from the base distribution H, for some 1 ≤ K ≤ n. If H is continuous, then all of
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are distinct, however if H is discrete then some of the values may not be distinct. Then the distribution of X
                              <sub>
                               n+1
                              </sub>
                              conditional on X
                              <sub>
                               1
                              </sub>
                              , …, X
                              <sub>
                               n
                              </sub>
                              is:
                              <span class="disp-formula" id="disp-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where n
                              <sub>
                               k
                              </sub>
                              represents the number of times the value
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              appears among X
                              <sub>
                               1
                              </sub>
                              , …, X
                              <sub>
                               n
                              </sub>
                              . More simply, X
                              <sub>
                               n+1
                              </sub>
                              is sampled either from an existing value
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with probability proportional to n
                              <sub>
                               k
                              </sub>
                              , or from the base distribution H. The values
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are called “tables” in the Chinese restaurant configuration. The tables may also be referred to simply by their indices 1, …, K.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              1.2. Diversity
                             </h3>
                             <p id="p-19">
                              One of the most important descriptive tools in microbial community estimation is species diversity. Diversity aims to create a quantitative description of a population that incorporates information such as the total number of species in a population and the relative frequencies of those species. For example, one commonly applied measure of diversity is Simpson’s index, which concerns the probability of sampling the same species twice from a population. Assume a population contains K distinct species and that the proportion of species k is represented by p
                              <sub>
                               k
                              </sub>
                              . Simpson’s index is sometimes calculated as the probability of sampling two different species in subsequent draws (i.e. the complement of the above definition). The latter definition is used in this paper. Simpson’s index, denoted by D, is then written as:
                              <span class="disp-formula" id="disp-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               Gorelick, 2006
                              </a>
                              ). A larger value of D implies the probability of obtaining the same species in two subsequent samples from the population is small. Thus, the closer the value of D is to one, the more diverse the population under consideration.
                             </p>
                             <p id="p-20">
                              The fundamental problem in estimating diversity is that it is unlikely that a sample from a population will include all K species that exist in the population. To illustrate the problem, consider a population with three species (K = 3), where the counts of the three species are (n
                              <sub>
                               1
                              </sub>
                              , n
                              <sub>
                               2
                              </sub>
                              , n
                              <sub>
                               3
                              </sub>
                              ) = (5, 4, 1). If we calculate Simpson’s index naïvely based on
                              <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                               Equation 4
                              </a>
                              we get D = 0.58. Now, consider a sample from this population where we obtain the counts (n
                              <sub>
                               1
                              </sub>
                              , n
                              <sub>
                               2
                              </sub>
                              , n
                              <sub>
                               3
                              </sub>
                              ) = (3, 2, 0). Calculating Simpson’s index on the sample proportions gives D = 0.48. Thus, the diversity in the population has been underestimated, simply because we failed to sample individuals from species 3.
                             </p>
                             <p id="p-21">
                              Model assumptions can help to estimate species diversity in the presence of missing species. Methods for estimating Simpson’s index in the mainland-island model assumed in Hubbell’s neutral theory of biodiversity have been developed by
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               Cerquetti (2015)
                              </a>
                              . Expressions to estimate Simpson’s index have not yet been developed in the HPY context, however. Thus, in this paper we propose the derivation of an expression of Simpson’s index in this context. The idea is that the HPY Gibbs sampler proposed in this paper could be applied to species abundance data, and estimates of the HPY model parameters could be obtained. From there, Simpson’s index could be calculated using the estimated HPY model parameters. The advantage of this approach is that, even if a particular population has missing species, the model structure assumed in the HPY framework can help to model the probability of discovering a new species (i.e. one that has not yet been observed). There is also a sense of “borrowing strength” between populations, in that we can account for sampling probabilities for a species that has a zero count in a population by considering the frequencies of that species in another population.
                             </p>
                             <p id="p-22">
                              The framework that we pursue to estimate Simpson’s index using the HPY model depends on calculation of the Hill numbers (
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               Hill, 1973
                              </a>
                              ), which are defined for integers q ≥ 2 as:
                              <span class="disp-formula" id="disp-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              so that Simpson’s index as defined in
                              <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                               Equation 4
                              </a>
                              can be written as
                              <span class="disp-formula" id="disp-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In this paper we show that the Chinese restaurant franchise representation of the HPY process admits a convenient way to derive the reciprocal of
                              <sup>
                               2
                              </sup>
                              D. Additionally, we provide a formula for
                              <sup>
                               q
                              </sup>
                              D for general q ≥ 2 in the HPY context.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             2. Methods
                            </h2>
                            <p id="p-23">
                             In this section we outline the model for the HPY process and describe the Gibbs sampler used to fit the model. Additionally we present a formula for Simpson’s index in the context of the HPY model parameters. In the mainland-island model, each individual in a local community can be traced back to an ancestor that immigrated to the community. In the hierarchical Pitman-Yor model, the “tables” in the Chinese restaurant representation of the Pitman-Yor process are analogous to individuals’ ancestors that immigrated in the mainland-island model. Multiple individuals in a community could have descended from the same ancestor and there can be multiple ancestors of the same species. The makeup of the ancestors are not directly observable and are therefore treated as latent variables in this framework. In this paper we use the following notation:
                            </p>
                            <ul class="list-unord" id="list-1">
                             <li id="list-item-1">
                              <p id="p-24">
                               J is the number of observed local communities
                              </p>
                             </li>
                             <li id="list-item-2">
                              <p id="p-25">
                               K is the total number of observed species
                              </p>
                             </li>
                             <li id="list-item-3">
                              <p id="p-26">
                               Y
                               <sub>
                                jk
                               </sub>
                               is the observed count of species k in population j
                              </p>
                             </li>
                             <li id="list-item-4">
                              <p id="p-27">
                               t
                               <sub>
                                jp
                               </sub>
                               is the ancestor (“table”) from which individual p in community j descends
                              </p>
                             </li>
                             <li id="list-item-5">
                              <p id="p-28">
                               m
                               <sub>
                                jk
                               </sub>
                               is the number of ancestors in community j corresponding to species k
                              </p>
                             </li>
                             <li id="list-item-6">
                              <p id="p-29">
                               n
                               <sub>
                                jtk
                               </sub>
                               is the number of observations in community j that descended from ancestor t, corresponding to species k.
                              </p>
                             </li>
                             <li id="list-item-7">
                              <p id="p-30">
                               k
                               <sub>
                                jp
                               </sub>
                               is the species of individual p in community j (a number from 1 to K)
                              </p>
                             </li>
                             <li id="list-item-8">
                              <p id="p-31">
                               ψ
                               <sub>
                                jt
                               </sub>
                               is the species of the t
                               <sup>
                                th
                               </sup>
                               ancestor in community j (a number from 1 to K)
                              </p>
                             </li>
                            </ul>
                            <p id="p-32">
                             We use the symbol · in a subscript to denote the summation of all values over a particular dimension. For example,
                             <span class="inline-formula" id="inline-formula-10">
                              <span class="highwire-responsive-lazyload">
                               <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-10.gif"/>
                               <noscript>
                                <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/inline-graphic-10.gif"/>
                               </noscript>
                              </span>
                             </span>
                             is the number of ancestors in population j over all species k = 1, …, K.
                            </p>
                            <div id="sec-8" class="subsection">
                             <h3>
                              2.1. Hierarchical Pitman-Yor process model
                             </h3>
                             <p id="p-33">
                              In this section we introduce the hierarchical Pitman-Yor process and describe how it is connected to the mainland-island model. The model is defined as follows:
                              <span class="disp-formula" id="disp-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for j = 1, …, J. The vector of proportions p
                              <sub>
                               0
                              </sub>
                              represents the abundance of species in the metacommunity which is assumed to follow a GEM distribution. Each vector p
                              <sub>
                               j
                              </sub>
                              represents the abundance distribution of local population j, and is assumed to follow a Pitman-Yor process with base distribution p
                              <sub>
                               0
                              </sub>
                              . Therefore, sampling a new ancestor in local community j is akin to sampling from the abundance distribution p
                              <sub>
                               0
                              </sub>
                              of the metacommunity. Finally, the vector of observed species counts in population j, denoted by Y
                              <sub>
                               j
                              </sub>
                              , is sampled from a multinomial distribution with its proportion vector set to p
                              <sub>
                               j
                              </sub>
                              . This final step represents the sampling procedure from the local community. The hierarchical Dirichlet process developed in
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               Harris et al. (2015)
                              </a>
                              , was shown to be the large population size limiting distribution for Hubbell’s neutral model under certain conditions on individual mean reproductive success. In the hierarchical Pitman-Yor model, we include the discount parameters α and σ
                              <sub>
                               j
                              </sub>
                              ; it has been shown that a non-zero value for the discount parameter corresponds to non-neutral assemblage (Crane et al., 2016). Thus, the discount parameter could be used as a measure of departure from neutrality.
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              gives a schematic of the assumed HPY model assuming two local populations.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F1.large.jpg?width=800&height=600&carousel=1" title="Schematic showing the HPY model assuming two local populations. A red node labelled Aij corresponds to ancestor j in local population i. The microbes pictured at the bottom correspond to descendants of the ancestors; i.e. the microbes present in the local communities. The observed species abundance table is a sample from the descendant microbes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-57972501" data-figure-caption='<div class="highwire-markup">Schematic showing the HPY model assuming two local populations. A red node labelled Aij corresponds to ancestor j in local population i. The microbes pictured at the bottom correspond to descendants of the ancestors; i.e. the microbes present in the local communities. The observed species abundance table is a sample from the descendant microbes.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F1.medium.gif" width="440" height="167"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F1.medium.gif" width="440" height="167"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig 1.
                               </span>
                               <p id="p-34" class="first-child">
                                Schematic showing the HPY model assuming two local populations. A red node labelled Aij corresponds to ancestor j in local population i. The microbes pictured at the bottom correspond to descendants of the ancestors; i.e. the microbes present in the local communities. The observed species abundance table is a sample from the descendant microbes.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-35">
                              Priors for the top- and local-level discount and concentration parameters also must be specified in the HPY model:
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.10.24.353599/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for j = 1, …, J. Here, a
                              <sub>
                               0
                              </sub>
                              and b
                              <sub>
                               0
                              </sub>
                              are respectively the shape and scale hyperparameters of the gamma distribution for the top-level concentration parameter γ. Likewise, a
                              <sub>
                               j
                              </sub>
                              and b
                              <sub>
                               j
                              </sub>
                              are the shape and scale hyperparameters in the prior for the local-level concentration parameter θ
                              <sub>
                               j
                              </sub>
                              .
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              2.2. Diversity estimation
                             </h3>
                             <p id="p-36">
                              In this paper we consider Simpson’s index in the context of the HPY model. The idea is, rather than calculating the Hill number reciprocal 1/(
                              <sup>
                               2
                              </sup>
                              D) directly from the observed proportions, we can infer in a given population the probability of sampling the same species twice in a row based on the sampling probabilities prescribed by the hierarchical Pitman-Yor process. This theoretical calculation would allow the inclusion of sampling species that have not yet been observed in a particular sample. We can do this by considering the Chinese restaurant process representation of the Pitman-Yor process. That is, when a particular species is sampled in a population, the probability of observing that same species a second time is slightly altered, since the corresponding ancestral counts will have been modified. Thus, the joint probability of sampling the same species twice can be calculated by considering all possible configurations of sampling the same species twice. We consider two forms of the expression for Simpson’s index: the first form, which we call the unconditional Simpson’s index, uses only the discount and concentration parameters from the HPY model; the second form, which we call the conditional Simpson’s index, uses the HPY parameters as well as the observed species counts and ancestral states t
                              <sub>
                               jp
                              </sub>
                              . The unconditional index represents the overall probability of sampling the same species twice in the HPY framework, and the conditional index represents the probability that the next two samples are of the same species, given a particular number of individuals have already been sampled. Considering the error in estimating the HPY discount and concentration parameters, we expect that additional leveraging of the observed counts and estimated ancestral states will aid in estimation accuracy for Simpson’s index. In addition to providing a means of calculating 1/(
                              <sup>
                               2
                              </sup>
                              D) in this way, we also provide a formula for calculating 1/(
                              <sup>
                               q
                              </sup>
                              D) for general q ≥ 2; these results can be seen in Section S2.1.1.
                             </p>
                             <p id="p-37">
                              Simpson’s index is a measure of alpha-diversity—in other words it measures the amount of diversity in a single population. In this paper we also consider the equivalent probability calculation across populations, i.e. the probability of sampling one species in one population and the same species in another population. This could be applied as a measure of beta-diversity, which measures the discrepancies in species abundance distributions between populations. This probability can similarly be calculated in the context of the hierarchical Pitman-Yor model. We have have derived expressions for Simpson’s index (conditional and unconditional) relating to both alpha-diversity and beta-diversity. As the expressions are quite complicated, they can be found in the supplement (Section S2).
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              2.3 Gibbs sampler for the HPY model
                             </h3>
                             <p id="p-38">
                              In order to fit the HPY model defined in
                              <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">
                               Section 2.1
                              </a>
                              , we use a Gibbs sampler. The parameters that must be updated in each step of the Gibbs sampler are the top-level GEM parameters α and γ; the local-level PY parameters σ
                              <sub>
                               j
                              </sub>
                              and θ
                              <sub>
                               j
                              </sub>
                              , for j = 1, …, J ; and the ancestor indicators t
                              <sub>
                               jp
                              </sub>
                              for each j = 1, …, J and p = 1, …, n
                              <sub>
                               j··
                              </sub>
                              .
                             </p>
                             <p id="p-39">
                              As the local-level abundances are represented by a Pitman-Yor process, updates for the parameters σ
                              <sub>
                               j
                              </sub>
                              and θ
                              <sub>
                               j
                              </sub>
                              can be obtained by slice-sampling procedures defined by
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               Buntine (2012)
                              </a>
                              . This algorithm is outlined in Algorithm S1 in the supplement. Note that the full conditionals for the concentration parameters are not log-concave. Buntine uses the technique from
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               Escobar and West (1995)
                              </a>
                              to generate an auxiliary beta-distributed random variable. The joint distribution of the concentration parameter and the auxiliary variable is log-concave (see Section S1.2.2) which allows the use of a slice-sampler. At the top level we have a GEM distribution, however the slice samplers from Buntine can still be applied with a slight modification to the table count vectors. Details on these procedures are given in Section S1.1 for the discount parameters, and Section S1.2 for the concentration parameters. For the ancestral indicators t
                              <sub>
                               jp
                              </sub>
                              , j = 1, …, J and p = 1, …, n
                              <sub>
                               j··
                              </sub>
                              , we apply the method used in
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               Battiston, Favaro and Teh (2018)
                              </a>
                              . In their algorithm, each individual (i.e. each organism) is removed from the population and reassigned to a new or existing ancestor with probabilities defined by the Chinese restaurant process. Details of that procedure are outlined in Section S1.3.
                              <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">
                               Algorithm 1
                              </a>
                              outlines a single iteration of the Gibbs sampler used to fit the model.
                             </p>
                             <p id="p-40">
                              It is not immediately obvious what choices would be best for the hyperparameters a
                              <sub>
                               0
                              </sub>
                              , b
                              <sub>
                               0
                              </sub>
                              , a
                              <sub>
                               j
                              </sub>
                              , and b
                              <sub>
                               j
                              </sub>
                              . To guarantee the log-concavity of the joint distribution of the concentration parameters, we need to choose the shape parameters so that a
                              <sub>
                               0
                              </sub>
                              ≥ 1 and each a
                              <sub>
                               j
                              </sub>
                              ≥ 1 (see Section S1.2.2). We find that setting these shape parameters to 1.1 works well in practice. The sampler is fairly sensitive to the choice of the scale parameters b
                              <sub>
                               0
                              </sub>
                              and b
                              <sub>
                               j
                              </sub>
                              . However, we do find in practice that setting all the scale parameters equal to the number of observed species K leads to good estimation accuracy. This allows the prior means for the concentration parameters to scale as a function of the number of observed species, which is necessary since we have observed that a poorly specified setting for this prior mean can negatively affect the posterior distributions of these parameters. This configuration corresponds to the recommendation in Buntine’s
                              <kbd>
                               libstb
                              </kbd>
                              C library, which is used to sample the Pitman-Yor parameters (
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Buntine, 2012
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              2.4. Simulation study
                             </h3>
                             <p id="p-41">
                              To validate the Gibbs sampler for the HPY model, we have designed an extensive simulation study. We simulate data from the hierarchical Pitman-Yor process using its Chinese restaurant process representation. That is, for every new individual sampled in local population j, we will perform one of three options (1) sample from an existing ancestor, thus assigning that individual the same species as the ancestor; (2) sample from an existing species at the top level; or (3) sample a new species at the top level.
                             </p>
                             <div id="statement-1" class="statement">
                              <span class="statement-label">
                               Algorithm 1:
                              </span>
                              <p id="p-42">
                               Iteration i of the HPY Sampler
                              </p>
                              <div id="F2" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F2.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-57972501" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F2.medium.gif" width="440" height="325"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F2.medium.gif" width="440" height="325"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure2" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-43">
                              </p>
                             </div>
                             <p id="p-44">
                              In the simulation study, we simulate data under multiple realistic configurations of the parameters in the HPY model. For the local-level parameters, we consider the values σ
                              <sub>
                               j
                              </sub>
                              ∈ {0.2, 0.5} and θ
                              <sub>
                               j
                              </sub>
                              ∈ {5, 25}. For the top-level parameters, we consider the values α ∈ {0.2, 0.5, 0.8} and γ ∈ {5, 25, 50}. We generate 10 local communities, and assume the same values for the local parameters across all communities. In all cases, the total number of individuals generated from the HPY process is 100,000 in each community; however, the final number of individuals sampled in each community in the multinomial step is n
                              <sub>
                               j··
                              </sub>
                              ∈ {500, 1000, 5000} (i.e. this is the number of individuals that would be observed in the species abundance vector Y
                              <sub>
                               j
                              </sub>
                              in each population j). We generated 50 replications of each simulation scenario. For each of the simulation replications the HPY Gibbs sampler was run for 2000 steps along with 1000 burn-in steps. To obtain parameter estimates we take the posterior mode of the MCMC samples. The posterior mode is more appropriate than the posterior mean in this context as the posterior distributions of the concentration and discount parameters are usually quite skewed due to the constraints on their supports.
                             </p>
                             <p id="p-45">
                              Results from the simulation study are shown in
                              <a id="xref-sec-13-2" class="xref-sec" href="#sec-13">
                               Section 3
                              </a>
                              . We examine the accuracy of the estimated parameters by comparing the posterior mode of the parameter samples from the Gibbs sampler to the true simulated values. We also calculate Simpson’s index to measure alpha-diversity in each population (see Equations S.13 and S.14) and beta-diversity to compare across populations (see Equations S.15 and S.16); we do this for both the unconditional and conditional versions of Simpson’s index. We compare Simpson’s index from the HPY model to the naïvely calculated indices, i.e. summing the squared observed proportions in each sample.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              2.5. Data analysis in twin study
                             </h3>
                             <p id="p-46">
                              We apply the hierarchical Pitman-Yor model Gibbs sampler on a 16S sequencing dataset from the Missouri Adolescent Female Twin Study (
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               Turnbaugh et al., 2009
                              </a>
                              ). The study was undertaken to investigate the differences in the composition of the gut microbiota between lean and obese hosts. The study considered monozygotic and dizygotic twin pairs that were concordant for obesity as well as their mothers, though we only include data from the twins in the HPY analysis. The study population consisted of 21-32 year-old women of European and African ancestry. 16S sequencing from fecal samples was performed using multiplex pyrosequencing on the V2 and V6 regions. Fecal samples were taken at two time points for each twin, however we consider samples only from the first time point, as there would likely be strong correlation between the compositions of the intestinal microbiota within each twin between the two time points. Most twin pairs lived apart, though 29% of the twin paris did live in the same household. In this analysis we consider a sample size of 102 samples from 29 monozygotic and 22 dizygotic twin pairs. In the cleaned dataset the are 28 women in the lean category (BMI 18.5–24.9 kg/m
                              <sup>
                               2
                              </sup>
                              ), 7 women in the overweight category (BMI 25–30 kg/m
                              <sup>
                               2
                              </sup>
                              ), and 70 women in the obese category (BMI > 30 kg/m
                              <sup>
                               2
                              </sup>
                              ).
                             </p>
                             <p id="p-47">
                              In this paper we wish to estimate diversity using the HPY model and compare diversity within the adiposity groups. We also explore some technical details such as the appropriateness of the fitted HPY model to the observed data and the effect of sequencing depth on the diversity estimates. From the HPY sampler we obtain 2000 MCMC samples after running 1000 burn-in samples. Note that, in consideration of the results of the simulation study, we only calculate the conditional version of Simpson’s index in this data application.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             3. Simulation results
                            </h2>
                            <p id="p-48">
                             In this section we outline the results of the simulation study described in
                             <a id="xref-sec-11-1" class="xref-sec" href="#sec-11">
                              Section 2.4
                             </a>
                             . First we consider the estimation accuracy of the various parameters in the HPY model (γ, α, θ
                             <sub>
                              j
                             </sub>
                             , σ
                             <sub>
                              j
                             </sub>
                             ). Results from these investigations are shown in Section S3. Each of these plots shows estimation errors as the difference between the estimated parameter (posterior mode) and the true simulated value. These errors are normalized by dividing by the true simulated values to facilitate comparisons across simulation scenarios.
                            </p>
                            <p id="p-49">
                             In Figure S1 we consider the normalized errors from estimating the top-level concentration parameter γ. Estimates generally appear to be unbiased. There is little improvement in estimation accuracy with increasing sample size in the local populations. Conversely, there does appear to be better estimation accuracy when the true value of the top-level discount parameter (α) is smaller. Higher values of α correspond to a heavier tail when considering the species abundance distribution aggregated across populations. That is, a higher top-level discount parameter implies many singleton species, which could contribute to the difficulties in estimation accuracy in this particular parameter configuration.
                            </p>
                            <p id="p-50">
                             Simulation results for the top-level discount parameter α are shown in Figure S2. Again, the estimates do not appear to have any particular bias. In many cases there does appear to be a slight improvement in estimation accuracy with increasing sample size. There is a substantial increase in estimation accuracy for larger values of the true simulated value of α. In the n
                             <sub>
                              j··
                             </sub>
                             = 500 configuration, there is generally better estimation accuracy for σ
                             <sub>
                              j
                             </sub>
                             = 0.2 than for σ
                             <sub>
                              j
                             </sub>
                             = 0.5.
                            </p>
                            <p id="p-51">
                             In Figure S3 we show the estimation accuracy in simulations for the local-level concentration parameters θ
                             <sub>
                              j
                             </sub>
                             , j ∈ {1, …, J}. There is generally better estimation accuracy when the true value of the local concentration parameters are θ
                             <sub>
                              j
                             </sub>
                             = 5 compared to θ
                             <sub>
                              j
                             </sub>
                             = 25. It also seems that there is a slight positive bias in estimates for this parameter, though the reason for this potential bias is still unclear. Similarly in Figure S4 we show results for the local-level concentration parameters σ
                             <sub>
                              j
                             </sub>
                             , j ∈ {1, …, J}. In most cases there is a noticeable improvement with increasing sample size. In some cases there is an improvement for θ = 25 as opposed to θ = 5, however, this improvement is not consistent across all simulation scenarios. There also appears to be a slight negative bias for these parameters. In the MCMC samples there is typically a strong negative correlation between the local concentration and discount parameters due to the fact that both parameters are related to the total number of observed species. This could explain why the biases for the local concentrations are of the opposite sign of the biases of the local discount parameters.
                            </p>
                            <p id="p-52">
                             Next, we investigate the results of estimating Simpson’s index using the HPY model. First, the unconditional and conditional versions of Simpson’s index are calculated from Equations S.13 and S.14, respectively. We calculate each index for each of the 2000 MCMC samples from the Gibbs sampler. Then, then final estimate for each version of Simpson’s index is taken as the mode of those values. It should be noted that the posterior distribution of Simpson’s index does not have a high variance, so the actual estimated value would not change much if the posterior mean or median were used. We compare the values of Simpson’s index from the HPY model to a naïvely calculated Simpson’s index from the observed proportions in each subsampled population. The true value of Simpson’s index is considered to be the result of applying
                             <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">
                              Equation 4
                             </a>
                             to the proportions in the full simulated HPY dataset (which would be unobserved in practice).
                            </p>
                            <p id="p-53">
                             In
                             <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                              Figure 2
                             </a>
                             we compare the errors for the conditional and naïve estimation methods; the unconditional version also appears in Figure S5, but performs significantly worse than other methods. There is consistently greater variation in the amount of error for the unconditional version of Simpson’s index. For the sake of more easily making a comparison between unconditional and naïve, we have omitted the unconditional from
                             <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                              Figure 2
                             </a>
                             . The naïvely estimated Simpson’s index is often underestimated, especially when γ = 50, whereas the HPY conditional calculation was more accurate, though it did slightly overestimate in some cases, in particular when the sample size in the local populations was smaller. Recall that higher values of γ generally correspond to a higher species richness across all populations (i.e. in the metacommunity). Thus, the conditional Simpson’s index calculated using the HPY framework is more accurate when there is high species richness in the metacommunity. The conditional index is also generally more accurate for smaller population sizes. It should be noted that, as the number of samples taken within each population increases, both the naïvely calculated and HPY calculated values of Simpson’s index are consistently accurate. This shows that it is important to consider the number of sequences when calculating diversity in a population, and that the conditional estimates from the HPY model are valuable in samples with a low number of sequences.
                            </p>
                            <div id="F3" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F3.large.jpg?width=800&height=600&carousel=1" title="Estimation error for Simpson’s index comparing conditional Simpson’s index (HPY) and naïve estimates in simulated data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-57972501" data-figure-caption='<div class="highwire-markup">Estimation error for Simpson’s index comparing conditional Simpson’s index (HPY) and naïve estimates in simulated data.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F3.medium.gif" width="440" height="439"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F3.medium.gif" width="440" height="439"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig 2.
                              </span>
                              <p id="p-54" class="first-child">
                               Estimation error for Simpson’s index comparing conditional Simpson’s index (HPY) and naïve estimates in simulated data.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-55">
                             We also explore the results of Simpson’s index for beta-diversity. In the HPY model we estimate this in the same way as for the alpha-diversity measure of Simpson’s index shown above, only we now use Equation S.15 for the unconditional index and Equation S.16 for the conditional index. Figure S6 shows the error for Simpson’s index for beta-diversity estimated from the HPY model (unconditional and conditional) and naïvely. There are not many discernible differences between the three estimation methods. This is likely due to the fact that, when calculating the underlying true beta-diversity values, it is probable that one or both of the proportions from a pair of populations will be zero. Thus, differences in the true beta-diversity values will be greatly affected by more abundant species, whereas differences in estimated values tend to occur due to differences in rarer species. Like before, estimation improves substantially for both methods with an increasing number of sequences.
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             4. Data analysis results
                            </h2>
                            <p id="p-56">
                             We now outline the results of applying the HPY model in the lean/obese twin study as described in
                             <a id="xref-sec-12-1" class="xref-sec" href="#sec-12">
                              Section 2.5
                             </a>
                             . In this section we only consider the conditional HPY version of Simpson’s index, due to its clear advantage over the unconditional version as seen in the simulation study. First we consider the HPY estimated values of Simpson’s index within the obesity categories.
                            </p>
                            <p id="p-57">
                             <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                              Figure 3(a)
                             </a>
                             gives the distributions of Simpson’s index from the HPY sampler within each of the obesity categories. There is no apparent difference in alpha-diversity between subjects within the three categories using this measure. Similarly, we consider Simpson’s index for beta-diversity in
                             <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                              Figure 3(b)
                             </a>
                             calculated between subject pairs within the same obesity category and across obesity categories. Again, there is very little difference in beta-diversity using this metric in subject pairs concordant for obesity category compared to discordant.
                            </p>
                            <div id="F4" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F4.large.jpg?width=800&height=600&carousel=1" title="(a) Simpson’s index estimated from HPY model across overweight categories. (b) Simpson’s index for beta-diversity between subject pairs that are concordant vs. discordant for overweight categories." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-57972501" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Simpson’s index estimated from HPY model across overweight categories. <strong>(b)</strong> Simpson’s index for beta-diversity between subject pairs that are concordant vs. discordant for overweight categories.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F4.medium.gif" width="440" height="217"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F4.medium.gif" width="440" height="217"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig 3.
                              </span>
                              <p id="p-58" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Simpson’s index estimated from HPY model across overweight categories.
                               <strong>
                                (b)
                               </strong>
                               Simpson’s index for beta-diversity between subject pairs that are concordant vs. discordant for overweight categories.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-59">
                             We also check the distributions of the estimated values of the local-level HPY parameters in the obesity categories in Figure S7. As in the above results for diversity, there do not appear to be differences in the distributions of either the local concentrations θ
                             <sub>
                              j
                             </sub>
                             or the local discounts σ
                             <sub>
                              j
                             </sub>
                             between the obesity categories. Next, we check whether the estimated locallevel parameter estimates correlate with the age of the host. Though there is no apparent relationship for θ
                             <sub>
                              j
                             </sub>
                             , there does appear to be a positive association between the age of the host and the local discount σ
                             <sub>
                              j
                             </sub>
                             , as shown in Figure S8. However, this association is rather weak with linear regression estimating a slope of only 0.0064, p = 0.026. Recall that a positive discount parameter reflects a non-neutral community. The range of estimated local discount parameters is from about 0.3 to 0.67, suggesting non-neutral assemblage within all the local communities.
                            </p>
                            <p id="p-60">
                             Next we consider the effect of twin pairs on the results of the HPY model. In Figure S9 we examine scatterplots comparing (a) local concentration θ
                             <sub>
                              j
                             </sub>
                             , (b) local discount σ
                             <sub>
                              j
                             </sub>
                             , and (c) Simpson’s index between twin pairs. There is no apparent concordance for any of the three quantities between twin pairs. We also look at Simpson’s index for beta-diversity between and across twin pairs from the HPY model in Figure S10. There is a subtle difference in means of beta-diversity within and between twin pairs—the mean was 0.9098 for unrelated pairs, and 0.9027 within twin pairs. A two-sample t-test for comparing these means resulted in p = 0.0402. This means that the probability of sampling the same species twice is higher for twin pairs, suggesting slightly more similar diversity within twin pairs compared to unrelated pairs. Finally, we check whether alpha-diversity or beta-diversity differs with respect to monozygotic (MZ) or dizygotic (DZ) twin pairs in Figure S11. There does not appear to be any significant difference in either alpha or beta-diveristy between MZ and DZ twin pairs.
                            </p>
                            <p id="p-61">
                             It is also important to check the appropriateness of the HPY model on this dataset. To do so, we compare the curve of ranked species abundances (averaged over all subjects in the dataset) to multiple datasets simulated under the HPY model. In each simulation we ran the HPY process using parameters from one of the MCMC samples from the Gibbs sampler run on the lean/obese twins dataset. We also did the same using a hierarchical Dirichlet process for comparison.
                             <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                              Figure 4
                             </a>
                             shows the range of the the ranked species abundances curves for the HPY and Dirichlet models along with the observed species abundance curve from the dataset. It is immediately apparent that the HPY model fits the observed data much better than the hierarchical Dirichlet model. In particular, the HPY model better captures the tail of the distribution, which is unsurprising given the model’s ability to handle a power-law tail.
                            </p>
                            <div id="F5" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F5.large.jpg?width=800&height=600&carousel=1" title="Ranked abundances of species in the twin study. The blue dots represent the true abundances and the shaded region represents the range of the ranked abundances in the simulated data from both the hierarchical Dirichlet and Pitman-Yor processes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-57972501" data-figure-caption='<div class="highwire-markup">Ranked abundances of species in the twin study. The blue dots represent the true abundances and the shaded region represents the range of the ranked abundances in the simulated data from both the hierarchical Dirichlet and Pitman-Yor processes.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F5.medium.gif" width="440" height="257"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F5.medium.gif" width="440" height="257"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.10.24.353599/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig 4.
                              </span>
                              <p id="p-62" class="first-child">
                               Ranked abundances of species in the twin study. The blue dots represent the true abundances and the shaded region represents the range of the ranked abundances in the simulated data from both the hierarchical Dirichlet and Pitman-Yor processes.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-63">
                             Finally we see the effect on the number of sequences in a sample on the difference in Simpson’s index (both alpha and beta-diversity) between the HPY model and naïvely estimated. In Figure S12 we see the differences in both indices as a function of the number of sequences; in the plot for beta-diversity we consider the sum of the sequences in both samples. In both cases there is more agreement between the HPY and naïve estimates with increasing sequence counts. This result is consistent with the simulation results presented in
                             <a id="xref-sec-13-3" class="xref-sec" href="#sec-13">
                              Section 3
                             </a>
                             and again underscores the fact that the HPY model is useful for estimating diversity in populations with a smaller number of sequences sampled.
                            </p>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             5. Discussion
                            </h2>
                            <p id="p-64">
                             In this paper we have applied the hierarchical Pitman-Yor process to model species abundance distributions in microbiome sequencing data. We have developed a Gibbs sampler to fit the model and shown that the parameters in the HPY process are well estimated in this context. We have additionally provided a general formula for calculation of Hill numbers in the context of the HPY process. Finally, we have derived an estimator of Simpson’s index that leverages both HPY estimates from the Gibbs sampler as well as the observed data and inferred ancestral states. This conditional estimator was shown to outperform the unconditional estimator as well as the naïve estimator in most simulation cases.
                            </p>
                            <p id="p-65">
                             The main limitation of the method is the computational inefficiency of the Gibbs sampler, in particular resampling the ancestral states t
                             <sub>
                              jp
                             </sub>
                             . Since sequencing depths could be quite high, there could be tens of thousands of ancestral states to update in each iteration of the Gibbs sampler. However, because all individuals of the same species within a population could be considered interchangeable to an extent, we do not have to keep track of the individuals’ ancestral states, rather we only need to keep track of the total number of ancestors corresponding to each species. In this case, ancestors can be added or subtracted from each population at a rate that depends on HPY parameters and current ancestor frequencies.
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              Buntine and Hutter (2010)
                             </a>
                             developed a sampler in this style which could be applied instead of the table indicator sampler as described by
                             <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                              Battiston, Favaro and Teh (2018)
                             </a>
                             . Similarly, there have also been developments in variational approximations in Dirichlet mixtures that could be extended for the hierarchical Pitman-Yor model (
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Kurihara, Welling and Teh, 2007
                             </a>
                             ;
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Huynh, Phung and Venkatesh, 2016
                             </a>
                             ). Such approximations could greatly reduce the computational times in this model.
                            </p>
                            <p id="p-66">
                             In this paper, when calculating Simpson’s index from the hierarhical Pitman-Yor model, we consider the results of sampling the next two individuals from the Chinese restaurant representation of the process, after having observed a sample from the target population. However this is distinct from the “true” diversity in a microbiome population, since the frequencies of the Chinese restaurant process will be different in the true underlying population. To get better estimates of diversity as well as species richness, individuals can be up-sampled within each population using the Chinese restaurant process. From there various measures of diversity could be calculated to estimate their true values in the underlying population. This could be useful in particular for diversity measures that would be difficult or impossible to write directly in terms of the HPY parameters.
                            </p>
                            <p id="p-67">
                             One last point is that one of the main advantages of using the Pitman-Yor process instead of the Dirichlet process was to better capture the tail of the ranked species abundance distribution. However, since Simpson’s index is a diversity measure of second order, it is not sensitive to changes in the tail of this distribution. This means that Simpson’s index is not necessarily the best diversity measure for investigating the tail. However, the calculation of Simpson’s index can be expressed very nicely in terms of the probabilities defined by the Chinese restaurant representation. We believe using this index is a good starting point for demonstrating how diversity measures could be estimated using the HPY process, though future work should focus on alternative metrics to better capture the tail of the ranked species abundance distribution.
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             6. Conclusion
                            </h2>
                            <p id="p-68">
                             We have developed a Gibbs sampler to fit a model based on the hierarchical Pitman-Yor process given species abundance data across multiple microbial populations. This process is more appropriate for use on microbiota species abundance data than the previously established hierarchical Dirichlet process due to its ability to accommodate a power-law tail. Additionally, we have provided new expressions for Simpson’s index measuring alpha-diversity and beta-diversity in the context of the hierarchical Pitman-Yor process and showed their usefulness in simulation and in a data application. In doing so, we also derived a general formula for calculation of the Hill numbers in the HPY framework.
                            </p>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             SUPPLEMENTARY MATERIAL
                            </h2>
                            <div class="section" id="sec-19">
                             <h3>
                              supplement.pdf
                             </h3>
                             <p id="p-70">
                              Supplementary file containing additional information relevant to the manuscript.
                             </p>
                            </div>
                            <div class="section" id="sec-20">
                             <h3>
                              R-package
                             </h3>
                             <p id="p-71">
                              Implementation of the Gibbs sampler and calculation of Simpson’s index in the HPY model. Package
                              <kbd>
                               hpy
                              </kbd>
                              is available at
                              <a href="https://github.com/kevinmcgregor/hpy">
                               https://github.com/kevinmcgregor/hpy
                              </a>
                             </p>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              Twin study data
                             </h3>
                             <p id="p-72">
                              This dataset is publicly available:
                              <a href="https://pubmed.ncbi.nlm.nih.gov/19043404/">
                               https://pubmed.ncbi.nlm.nih.gov/19043404/
                              </a>
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-69">
                             K.M. would like to acknowledge support from an NSERC Discovery Grant (RGPIN-2021-03634). K.M. would like to acknowledge travel support from the Queen Elizabeth Scholars Program (“Quantitative Biology and Medical Genetics for the World”) as well as a McGill Graduate Mobility Award. K.M. was also supported by a Fonds de recherche du Québec-Santé doctoral award, as well as the McGill University Faculty of Medicine’s Cameron-Davis and Davis Fellowship and Gerald Clavet Fellowship. C.G. and A.L. acknowledge the CIHR operating grant (MOP-130344). C.G. would also like to acknowledge support from Compute Canada (ID 2541) as well as support from the Ludmer Centre for Neuroinformatics and Mental Health. C.Q. is funded through an MRC fellowship (MR/M50161X/1) as part of the Cloud Infrastructure for Microbial Genomics (CLIMB) consortium (MR/L015080/1). Images in
                             <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             used under CC BY 3.0 license. Attribution: microbe bacterium, bacillus, infusoria, microorgan by Alina Oleynik and Microbe by Sidiq Fathurochman from the Noun Project.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <span class="em-link">
                                <span class="em-addr">
                                 kevinmcg{at}yorku.ca
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-2">
                              <p id="p-2">
                               <a class="rev-xref" href="#xref-fn-2-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                *
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 celia.greenwood{at}mcgill.ca
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-3">
                              <p id="p-3">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                †
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 aurelie.labbe{at}hec.ca
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-4">
                              <p id="p-4">
                               <a class="rev-xref" href="#xref-fn-4-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                §
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 todd.parsons{at}upmc.fr
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-others" id="fn-5">
                              <p id="p-5">
                               <a class="rev-xref" href="#xref-fn-5-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                ¶
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 Christopher.Quince{at}earlham.ac.uk
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-6">
                              <p id="p-11">
                               Restructuring of the manuscript and a revised derivation of theoretical results for diversity metrics in the hierarchical Pitman-Yor process.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             REFERENCES
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arbel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mengersen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rousseau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bayesian nonparametric dependent model for partially replicated data: the influence of fuel spills on species diversity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Annals of Applied Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 <span class="cit-fpage">
                                  1496
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1516
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D10%26rft.spage%253D1496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Battiston
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Favaro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Multi-armed bandit for species discovery: a Bayesian nonparametric approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Statistical Association
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 <span class="cit-fpage">
                                  455
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  466
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2BAssociation%26rft.volume%253D113%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buntine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  Hyper-parameter Estimation for the Dirichlet Prior
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buntine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hutter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A Bayesian view of the Poisson-Dirichlet process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1007.0296
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Camerlenghi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dumitrascu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferrari
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Engelhardt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Favaro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Nonpara-metric Bayesian multiarmed bandits for single-cell experiment design
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Annals of Applied Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 <span class="cit-fpage">
                                  2003
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BAnnals%2Bof%2BApplied%2BStatistics%26rft.volume%253D14%26rft.spage%253D2003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.24.353599v2.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cerquetti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  Bayesian Estimation of Gini-Simpson’s Index Under Mainland-Island Community Struc-ture
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In Convegno della Società Italiana di Statistica
                                 </span>
                                 <span class="cit-fpage">
                                  61
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  70
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.7" data-doi="10.1146/annurev.ecolsys.31.1.343">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chesson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Mechanisms of maintenance of species diversity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual review of Ecology and Systematics
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 <span class="cit-fpage">
                                  343
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  366
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DChesson%26rft.auinit1%253DP.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D343%26rft.epage%253D366%26rft.atitle%253DMECHANISMS%2BOF%2BMAINTENANCE%2BOF%2BSPECIES%2BDIVERSITY%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.31.1.343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.31.1.343&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000166011500015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clauset
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shalizi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. R.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Newman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Power-law distributions in empirical data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  SIAM review
                                 </abbr>
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 <span class="cit-fpage">
                                  661
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  703
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSIAM%2Breview%26rft.volume%253D51%26rft.spage%253D661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.24.353599v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crane
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The ubiquitous Ewens sampling formula
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Statistical science
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStatistical%2Bscience%26rft.volume%253D31%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.10" data-doi="10.2307/2291069">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Escobar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   West
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bayesian density estimation and inference using mixtures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the american statistical association
                                 </abbr>
                                 <span class="cit-vol">
                                  90
                                 </span>
                                 <span class="cit-fpage">
                                  577
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  588
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2Bamerican%2Bstatistical%2Bassociation%26rft.volume%253D90%26rft.spage%253D577%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2291069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2291069&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995RA10400023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.11" data-doi="10.1111/j.1461-0248.2004.00717.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Etienne
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A new sampling formula for neutral biodiversity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ecology letters
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 <span class="cit-fpage">
                                  253
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  260
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEcology%2Bletters%26rft.volume%253D8%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1461-0248.2004.00717.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1461-0248.2004.00717.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000227249400003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.12" data-doi="10.1016/0040-5809(72)90035-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ewens
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1972
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The sampling theory of selectively neutral alleles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Theoretical population biology
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 <span class="cit-fpage">
                                  87
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  112
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DEwens%26rft.auinit1%253DW.%2BJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D112%26rft.atitle%253DThe%2Bsampling%2Btheory%2Bof%2Bselectively%2Bneutral%2Balleles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252872%252990035-4%26rft_id%253Dinfo%253Apmid%252F4667078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0040-5809(72)90035-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=4667078&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F11%2F11%2F2020.10.24.353599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1972L864600005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldwater
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Griffiths
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Interpolating between types and tokens by esti-mating power-law generators
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Advances in neural information processing systems
                                 </abbr>
                                 <span class="cit-fpage">
                                  459
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  466
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.14" data-doi="10.1111/j.0906-7590.2006.04601.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gorelick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Combining richness and abundance into a single diversity index using matrix analogues of Shannon’s and Simpson’s indices
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ecography
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 <span class="cit-fpage">
                                  525
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  530
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D29%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0906-7590.2006.04601.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.0906-7590.2006.04601.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parsons
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ijaz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U. Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lahti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quince
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Linking statistical and ecological theory: Hubbell’s unified neutral theory of biodiversity as a hierarchical Dirichlet process
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the IEEE
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 <span class="cit-fpage">
                                  516
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  529
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BIEEE%26rft.volume%253D105%26rft.spage%253D516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.16" data-doi="10.2307/1934352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. O.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1973
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Diversity and evenness: a unifying notation and its consequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ecology
                                 </abbr>
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 <span class="cit-fpage">
                                  427
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  432
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D54%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1934352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/1934352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1973P453400020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.10.24.353599v2.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hubbell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  The unified neutral theory of biodiversity and biogeography (MPB-32)
                                 </span>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Princeton University Press
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huynh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Venkatesh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Streaming variational inference for dirichlet process mix-tures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Asian Conference on Machine Learning
                                 </abbr>
                                 <span class="cit-fpage">
                                  237
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  252
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.19" data-doi="10.1073/pnas.1206721109">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeraldo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sipos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brulc
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dhillon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Konkel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Larson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nel-Son
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schook
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantification of the relative roles of niche and neutral processes in structuring gastrointestinal microbiomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 <span class="cit-fpage">
                                  9692
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9698
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1206721109%26rft_id%253Dinfo%253Apmid%252F22615407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzI1Lzk2OTIiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8xMS8xMS8yMDIwLjEwLjI0LjM1MzU5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.20" data-doi="10.1038/217624a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kimura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1968
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evolutionary rate at the molecular level
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  217
                                 </span>
                                 <span class="cit-fpage">
                                  624
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  626
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D217%26rft.issue%253D5129%26rft.spage%253D624%26rft.epage%253D626%26rft.atitle%253DEvolutionary%2Brate%2Bat%2Bthe%2Bmolecular%2Blevel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F217624a0%26rft_id%253Dinfo%253Apmid%252F5637732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/217624a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=5637732&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F11%2F11%2F2020.10.24.353599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1968A646300008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kurihara
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welling
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Teh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Collapsed Variational Dirichlet Process Mixture Models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In IJCAI
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 <span class="cit-fpage">
                                  2796
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2801
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%2BIJCAI%26rft.volume%253D7%26rft.spage%253D2796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.10.24.353599v2.22" data-doi="10.1371/journal.pone.0063139">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bihan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MethÉ
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Analyses of the stability and core taxonomic memberships of the human microbiome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 <span class="cit-fpage">
                                  e63139
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D8%26rft.spage%253De63139%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063139%26rft_id%253Dinfo%253Apmid%252F23671663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063139&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23671663&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F11%2F11%2F2020.10.24.353599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bacallado
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Favaro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holmes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trippa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bayesian nonparametric ordination for the analysis of microbial communities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of the American Statistical Association
                                 </abbr>
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 <span class="cit-fpage">
                                  1430
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1442
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BAmerican%2BStatistical%2BAssociation%26rft.volume%253D112%26rft.spage%253D1430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.24" data-doi="10.1038/nature07540">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Turnbaugh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hamady
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yatsunenko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cantarel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duncan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   So-gin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Affourtit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A core gut microbiome in obese and lean twins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  nature
                                 </abbr>
                                 <span class="cit-vol">
                                  457
                                 </span>
                                 <span class="cit-fpage">
                                  480
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  484
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTurnbaugh%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D457%26rft.issue%253D7228%26rft.spage%253D480%26rft.epage%253D484%26rft.atitle%253DA%2Bcore%2Bgut%2Bmicrobiome%2Bin%2Bobese%2Band%2Blean%2Btwins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07540%26rft_id%253Dinfo%253Apmid%252F19043404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature07540&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19043404&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F11%2F11%2F2020.10.24.353599.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000262519200047&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.10.24.353599v2.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yamato
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sibuya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nomachi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Ordered sample from two-parameter GEM distribution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Statistics & probability letters
                                 </abbr>
                                 <span class="cit-vol">
                                  55
                                 </span>
                                 <span class="cit-fpage">
                                  19
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStatistics%2B%2526%2Bprobability%2Bletters%26rft.volume%253D55%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.10.24.353599v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.02.11.480074v2" title="Predictive link between systemic metabolism and cytokine signatures in the brain of apolipoprotein E ε4 mice" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.12.05.471324v3" title="Bat teeth illuminate the diversification of mammalian tooth classes" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted November 11, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.10.24.353599v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.10.24.353599v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.10.24.353599v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.10.24.353599v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2844652"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-4Mct4YDyiGXdlcOs-n0HyAQI8bfYzBBMPe0KKG1XUDw"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484483274"/>
                                   <input type="hidden" name="captcha_token" value="28f53c88541d647ed6371853d901ae6a"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2844652--3" data-node-nid="2844652" id="node-2844652--41750406890" data-pisa="biorxiv;2020.10.24.353599v2" data-pisa-master="biorxiv;2020.10.24.353599" data-seqnum="2844652" data-apath="/biorxiv/early/2022/11/11/2020.10.24.353599.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Kevin
                                  </span>
                                  <span class="nlm-surname">
                                   Mcgregor
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Aurélie
                                  </span>
                                  <span class="nlm-surname">
                                   Labbe
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Celia M.T.
                                  </span>
                                  <span class="nlm-surname">
                                   Greenwood
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Todd
                                  </span>
                                  <span class="nlm-surname">
                                   Parsons
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Christopher
                                  </span>
                                  <span class="nlm-surname">
                                   Quince
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.10.24.353599;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.10.24.353599
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&title=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&title=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&text=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&t=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&title=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&title=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&title=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2844652--5" data-node-nid="2844652" id="citation-node-2844652--61571671837" data-pisa="biorxiv;2020.10.24.353599v2" data-pisa-master="biorxiv;2020.10.24.353599" data-seqnum="2844652" data-apath="/biorxiv/early/2022/11/11/2020.10.24.353599.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Microbial diversity estimation and hill number calculation using the hierarchical Pitman-Yor process
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Kevin
                                      </span>
                                      <span class="nlm-surname">
                                       Mcgregor
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Aurélie
                                      </span>
                                      <span class="nlm-surname">
                                       Labbe
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Celia M.T.
                                      </span>
                                      <span class="nlm-surname">
                                       Greenwood
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Todd
                                      </span>
                                      <span class="nlm-surname">
                                       Parsons
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Christopher
                                      </span>
                                      <span class="nlm-surname">
                                       Quince
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.10.24.353599;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.10.24.353599
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Mcgregor%2C+Kevin&rft.au=Labbe%2C+Aur%C3%A9lie&rft.au=Greenwood%2C+Celia+M.T.&rft.au=Parsons%2C+Todd&rft.au=Quince%2C+Christopher">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2844652/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2844652/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2844652/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2844652/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2844652/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2844652/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2844652/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2844652/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2844652/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2844652/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2844652/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2844652/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&count=horizontal&via=&text=Microbial%20diversity%20estimation%20and%20hill%20number%20calculation%20using%20the%20hierarchical%20Pitman-Yor%20process&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.10.24.353599v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.10.24.353599v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
