<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.09.10.21263388v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.09.10.21263388v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.09.10.21263388v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/embed/graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/11/21/2021.09.10.21263388.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.09.10.21263388v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma"/>
  <meta name="DC.Identifier" content="10.1101/2021.09.10.21263388"/>
  <meta name="DC.Date" content="2022-11-21"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="The incredible variety of proteins associated with immune responses presents a major challenge in immune monitoring. When combinations of these proteins are measured, cell types that influence disease can be precisely identified. Here, we introduce TerraFlow, a novel data analysis tool that performs an exhaustive search of disease-associated cell populations from high-parameter flow cytometry experiments. Using a newly generated dataset, from 24-color immune checkpoint-focused and 18-color immune function-focused experiments, we apply TerraFlow to classical Hodgkin lymphoma (cHL), where systemic T-cell immunity has not been investigated in detail. We reveal novel immune perturbations in newly diagnosed cHL, as well as persistent immune perturbations after treatment. Newly diagnosed patients have elevated levels of activated (CD278+), exhausted (e.g., CD366+ and CD152+ phenotypes), and IL17-expressing cells, along with diminished levels of naïve and central memory (CD127+) T-cells and fewer IFNγ+ and TNF+ T-cells. Exhaustion signatures are reduced with treatment, but compared to healthy individuals, treated patients still exhibit more activated (CD278+ phenotypes), exhausted (CD366+), and IL17-expressing cells. Notably, TerraFlow identifies more phenotypic differences between patient groups than FlowSOM and CellCNN, often with better predictive power. Finally, we introduce a new non-gating approach for data analysis that obviates the need for time-consuming and subjective setting of fluorescence thresholds. Our results benchmark TerraFlow against common methods, provide mechanistic support for past reports of immune deficiency in cHL, and provide a roadmap for future immunotherapy and biomarker studies.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

This work was funded by NYU Langone Health, the Feinberg family, and the American Cancer Society MRSG.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Informed consent was obtained from cHL patients (N=44) treated at the Perlmutter Cancer Center at NYU Langone Health and New York Presbyterian Weil Cornell (NYP) between 2011 and 2016. The study was reviewed by both IRBs.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data is not publicly available yet."/>
  <meta name="DC.Contributor" content="Daniel Freeman"/>
  <meta name="DC.Contributor" content="Catherine Diefenbach"/>
  <meta name="DC.Contributor" content="Linda Lam"/>
  <meta name="DC.Contributor" content="Tri Le"/>
  <meta name="DC.Contributor" content="Jason Alexandre"/>
  <meta name="DC.Contributor" content="Bruce Raphael"/>
  <meta name="DC.Contributor" content="Michael Grossbard"/>
  <meta name="DC.Contributor" content="David Kaminetzky"/>
  <meta name="DC.Contributor" content="Jia Ruan"/>
  <meta name="DC.Contributor" content="Pratip K. Chattopadhyay"/>
  <meta name="article:published_time" content="2022-11-21"/>
  <meta name="citation_title" content="TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>The incredible variety of proteins associated with immune responses presents a major challenge in immune monitoring. When combinations of these proteins are measured, cell types that influence disease can be precisely identified. Here, we introduce TerraFlow, a novel data analysis tool that performs an exhaustive search of disease-associated cell populations from high-parameter flow cytometry experiments. Using a newly generated dataset, from 24-color immune checkpoint-focused and 18-color immune function-focused experiments, we apply TerraFlow to classical Hodgkin lymphoma (cHL), where systemic T-cell immunity has not been investigated in detail. We reveal novel immune perturbations in newly diagnosed cHL, as well as persistent immune perturbations after treatment. Newly diagnosed patients have elevated levels of activated (CD278+), exhausted (e.g., CD366+ and CD152+ phenotypes), and IL17-expressing cells, along with diminished levels of naïve and central memory (CD127+) T-cells and fewer IFNγ+ and TNF+ T-cells. Exhaustion signatures are reduced with treatment, but compared to healthy individuals, treated patients still exhibit more activated (CD278+ phenotypes), exhausted (CD366+), and IL17-expressing cells. Notably, TerraFlow identifies more phenotypic differences between patient groups than FlowSOM and CellCNN, often with better predictive power. Finally, we introduce a new non-gating approach for data analysis that obviates the need for time-consuming and subjective setting of fluorescence thresholds. Our results benchmark TerraFlow against common methods, provide mechanistic support for past reports of immune deficiency in cHL, and provide a roadmap for future immunotherapy and biomarker studies.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.09.10.21263388v2"/>
  <meta name="citation_id" content="2021.09.10.21263388v2"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.09.10.21263388"/>
  <meta name="citation_num_pages" content="44"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.09.10.21263388"/>
  <meta name="citation_author" content="Daniel Freeman"/>
  <meta name="citation_author_institution" content="TerraFlow Bioinformatics"/>
  <meta name="citation_author_institution" content="Biology and Biomedical Sciences, Harvard Medical School"/>
  <meta name="citation_author_institution" content="Perlmutter Cancer Center, NYU Langone Health"/>
  <meta name="citation_author" content="Catherine Diefenbach"/>
  <meta name="citation_author_institution" content="TerraFlow Bioinformatics"/>
  <meta name="citation_author" content="Linda Lam"/>
  <meta name="citation_author_institution" content="TerraFlow Bioinformatics"/>
  <meta name="citation_author" content="Tri Le"/>
  <meta name="citation_author_institution" content="Perlmutter Cancer Center, NYU Langone Health"/>
  <meta name="citation_author" content="Jason Alexandre"/>
  <meta name="citation_author_institution" content="TerraFlow Bioinformatics"/>
  <meta name="citation_author" content="Bruce Raphael"/>
  <meta name="citation_author_institution" content="Perlmutter Cancer Center, NYU Langone Health"/>
  <meta name="citation_author" content="Michael Grossbard"/>
  <meta name="citation_author_institution" content="Perlmutter Cancer Center, NYU Langone Health"/>
  <meta name="citation_author" content="David Kaminetzky"/>
  <meta name="citation_author_institution" content="Perlmutter Cancer Center, NYU Langone Health"/>
  <meta name="citation_author" content="Jia Ruan"/>
  <meta name="citation_author_institution" content="Division of Hematology & Medical Oncology, Weill Cornell Medicine"/>
  <meta name="citation_author" content="Pratip K. Chattopadhyay"/>
  <meta name="citation_author_institution" content="Talon Biomarkers"/>
  <meta name="citation_author_email" content="pratip@talonbiomarkers.com"/>
  <meta name="citation_reference" content="Steidl, C., Connors, J. M. & Gascoyne, R. D. Molecular pathogenesis of Hodgkin’s lymphoma: increasing evidence of the importance of the microenvironment. J Clin Oncol 29, 1812–1826, doi:10.1200/JCO.2010.32.8401 (2011)."/>
  <meta name="citation_reference" content="Aldinucci, D., Gloghini, A., Pinto, A., De Filippi, R. & Carbone, A. The classical Hodgkin’s lymphoma microenvironment and its role in promoting tumour growth and immune escape. J Pathol 221, 248–263, doi:10.1002/path.2711 (2010)."/>
  <meta name="citation_reference" content="Kuppers, R. The biology of Hodgkin’s lymphoma. Nat Rev Cancer 9, 15–27, doi:10.1038/nrc2542 (2009)."/>
  <meta name="citation_reference" content="Hsi, E. D. Biologic features of Hodgkin lymphoma and the development of biologic prognostic factors in Hodgkin lymphoma: tumor and microenvironment. Leuk Lymphoma 49, 1668–1680, doi:10.1080/10428190802163339 (2008)."/>
  <meta name="citation_reference" content="Khan, G. Epstein-Barr virus, cytokines, and inflammation: a cocktail for the pathogenesis of Hodgkin’s lymphoma? Exp Hematol 34, 399–406, doi:10.1016/j.exphem.2005.11.008 (2006)."/>
  <meta name="citation_reference" content="Aldinucci, D. et al. Expression of CCR5 receptors on Reed-Sternberg cells and Hodgkin lymphoma cell lines: involvement of CCL5/Rantes in tumor cell growth and microenvironmental interactions. Int J Cancer 122, 769–776, doi:10.1002/ijc.23119 (2008)."/>
  <meta name="citation_reference" content="Tanijiri, T. et al. Hodgkin’s reed-sternberg cell line (KM-H2) promotes a bidirectional differentiation of CD4+CD25+Foxp3+ T cells and CD4+ cytotoxic T lymphocytes from CD4+ naive T cells. J Leukoc Biol 82, 576–584, doi:10.1189/jlb.0906565 (2007)."/>
  <meta name="citation_reference" content="van den Berg, A., Visser, L. & Poppema, S. High expression of the CC chemokine TARC in Reed-Sternberg cells. A possible explanation for the characteristic T-cell infiltratein Hodgkin’s lymphoma. Am J Pathol 154, 1685–1691, doi:10.1016/S0002-9440(10)65424-7 (1999)."/>
  <meta name="citation_reference" content="van Rijswijk, R. E., de Meijer, A., Sybesma, J. P. & Kater, L. Five-year survival in Hodgkin’s disease. The prospective value of immune status at diagnosis. Cancer 57, 1489–1496, doi:10.1002/1097-0142(19860415)57:8<1489::aid-cncr2820570809>3.0.co;2-n (1986)."/>
  <meta name="citation_reference" content="Weihrauch, M. R. et al. Elevated serum levels of CC thymus and activation-related chemokine (TARC) in primary Hodgkin’s disease: potential for a prognostic factor. Cancer Res 65, 5516–5519, doi:10.1158/0008-5472.CAN-05-0100 (2005)."/>
  <meta name="citation_reference" content="Ansell, S. M. Annual clinical updates in hematological malignancies: a continuing medical education series. Hodgkin lymphoma: 2011 update on diagnosis, risk-stratification, and management. Am J Hematol 86, 851–858, doi:10.1002/ajh.22105 (2011)."/>
  <meta name="citation_reference" content="Ouyang J, P. A., Pogge E, Ponader S, Rabinovich G, Neuberg DS, et al. Galectin-1 Serum Levels Reflect Tumor Burden and Adverse Clinical Features in Hodgkin Lymphoma. ASH Annual Meeting Abstracts 120, 1 (2012)."/>
  <meta name="citation_reference" content="Levin, L. I. et al. Elevated Serum Levels of sCD30 and IL6 and Detectable IL10 Precede Classical Hodgkin Lymphoma Diagnosis. Cancer Epidemiol Biomarkers Prev 26, 1114–1123, doi:10.1158/1055-9965.EPI-16-1012 (2017)."/>
  <meta name="citation_reference" content="Chattopadhyay, P. K., Winters, A. F., Lomas, W. E., 3rd, Laino, A. S. & Woods, D. M. High-Parameter Single-Cell Analysis. Annu Rev Anal Chem (Palo Alto Calif) 12, 411–430, doi:10.1146/annurev-anchem-061417-125927 (2019)."/>
  <meta name="citation_reference" content="Woods, D. M. et al. Nivolumab and ipilimumab are associated with distinct immune landscape changes and response-associated immunophenotypes. JCI Insight 5, doi:10.1172/jci.insight.137066 (2020)."/>
  <meta name="citation_reference" content="Arvaniti, E., Claassen, M. Sensitive detection of rare disease-associated cell subsets via representation learning. Nat Commun 8, 14825, doi:10.1038/ncomms14825 (2017)."/>
  <meta name="citation_reference" content="Nettey, L., Giles, A. J. & Chattopadhyay, P. K. OMIP-050: A 28-color/30-parameter Fluorescence Flow Cytometry Panel to Enumerate and Characterize Cells Expressing a Wide Array of Immune Checkpoint Molecules. Cytometry A 93, 1094–1096, doi:10.1002/cyto.a.23608 (2018)."/>
  <meta name="twitter:title" content="TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="The incredible variety of proteins associated with immune responses presents a major challenge in immune monitoring. When combinations of these proteins are measured, cell types that influence disease can be precisely identified. Here, we introduce TerraFlow, a novel data analysis tool that performs an exhaustive search of disease-associated cell populations from high-parameter flow cytometry experiments. Using a newly generated dataset, from 24-color immune checkpoint-focused and 18-color immune function-focused experiments, we apply TerraFlow to classical Hodgkin lymphoma (cHL), where systemic T-cell immunity has not been investigated in detail. We reveal novel immune perturbations in newly diagnosed cHL, as well as persistent immune perturbations after treatment. Newly diagnosed patients have elevated levels of activated (CD278+), exhausted (e.g., CD366+ and CD152+ phenotypes), and IL17-expressing cells, along with diminished levels of naïve and central memory (CD127+) T-cells and fewer IFNγ+ and TNF+ T-cells. Exhaustion signatures are reduced with treatment, but compared to healthy individuals, treated patients still exhibit more activated (CD278+ phenotypes), exhausted (CD366+), and IL17-expressing cells. Notably, TerraFlow identifies more phenotypic differences between patient groups than FlowSOM and CellCNN, often with better predictive power. Finally, we introduce a new non-gating approach for data analysis that obviates the need for time-consuming and subjective setting of fluorescence thresholds. Our results benchmark TerraFlow against common methods, provide mechanistic support for past reports of immune deficiency in cHL, and provide a roadmap for future immunotherapy and biomarker studies.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

This work was funded by NYU Langone Health, the Feinberg family, and the American Cancer Society MRSG.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Informed consent was obtained from cHL patients (N=44) treated at the Perlmutter Cancer Center at NYU Langone Health and New York Presbyterian Weil Cornell (NYP) between 2011 and 2016. The study was reviewed by both IRBs.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data is not publicly available yet."/>
  <meta name="og-title" property="og:title" content="TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="The incredible variety of proteins associated with immune responses presents a major challenge in immune monitoring. When combinations of these proteins are measured, cell types that influence disease can be precisely identified. Here, we introduce TerraFlow, a novel data analysis tool that performs an exhaustive search of disease-associated cell populations from high-parameter flow cytometry experiments. Using a newly generated dataset, from 24-color immune checkpoint-focused and 18-color immune function-focused experiments, we apply TerraFlow to classical Hodgkin lymphoma (cHL), where systemic T-cell immunity has not been investigated in detail. We reveal novel immune perturbations in newly diagnosed cHL, as well as persistent immune perturbations after treatment. Newly diagnosed patients have elevated levels of activated (CD278+), exhausted (e.g., CD366+ and CD152+ phenotypes), and IL17-expressing cells, along with diminished levels of naïve and central memory (CD127+) T-cells and fewer IFNγ+ and TNF+ T-cells. Exhaustion signatures are reduced with treatment, but compared to healthy individuals, treated patients still exhibit more activated (CD278+ phenotypes), exhausted (CD366+), and IL17-expressing cells. Notably, TerraFlow identifies more phenotypic differences between patient groups than FlowSOM and CellCNN, often with better predictive power. Finally, we introduce a new non-gating approach for data analysis that obviates the need for time-consuming and subjective setting of fluorescence thresholds. Our results benchmark TerraFlow against common methods, provide mechanistic support for past reports of immune deficiency in cHL, and provide a roadmap for future immunotherapy and biomarker studies.

### Competing Interest Statement

The authors have declared no competing interest.

### Funding Statement

This work was funded by NYU Langone Health, the Feinberg family, and the American Cancer Society MRSG.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

Informed consent was obtained from cHL patients (N=44) treated at the Perlmutter Cancer Center at NYU Langone Health and New York Presbyterian Weil Cornell (NYP) between 2011 and 2016. The study was reviewed by both IRBs.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

Data is not publicly available yet."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-11-21"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/585542"/>
  <title>
   TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UwwvYPU4S8VyzYxoQpS9xG-47NNnYS5fLX-TVz51Fp8"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"585542","apath":"\/medrxiv\/early\/2022\/11\/21\/2021.09.10.21263388.atom","pisa":"medrxiv;2021.09.10.21263388v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.09.10.21263388v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.09.10.21263388v2","title":"TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma","identifier":"node\/585542"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022orJK6ANd0UVAwdK9vKbesTJoxPpvm0P\\\\\\\/qW4Su35\\\\\\\/gl5eoycwQ1dU5P\\\\\\\/3hOOwaabwqlypcFYI7Xfyc0TM75zHP9k9LYfGb6glxrFsOHstb2uzdKymhCIcMdWay0eYzUWPLEOC2cNukxXWMqcCr3K3QP828Xkl5MmTGJ5+C1DaQfe5rPm3oriOefwC4x51ZaRPem16K+CinmB\\\\\\\/flsdQmyXhInIkTm7ii4cTYqzbQNdPeJsWmX4umqVgnSeDuzPWmj4IT\\\\\\\/VsX96Xlp1JnhEIcIOXvsvlZ6Z2eaRAuGlURItO9eJyaVRreC7yf3V4ZhNn0bPJrPdA9t1qu38g2FPlSHkJHln3UUTsRoochoIkh9Sq1tbFtgoXhl8MUc2lBm8aWFYI2rb6ctjc6fmixnhNmLxxYvKPYmSE7V9FN8+OkkmUh1vPv5YuG4DbNIE31e5HPED+XFbansgYrnn7MNcIjdm5lCaxWSRgG2oSZhdwLG6ry+Sgzw0uoYIuRtKHG+8GCdUGnHDCAGGeRRiMTz7Xj+f2CAg86So2lJMhazOMG\\\\\\\/z\\\\\\\/qHRdGXm\\\\\\\/VLEruk82DhQ5neaQZ2RBGJXviSIjypiVA1SfjDLRr97wg8Yr2T1qTfqOQrbb5YKfGW9FWpCONL3vVu6pk3uBkc2bwiAovXFc7m3dlhMe5JYCzzDPqHRX0RMHxME+B80vNIBSX6sUg16q2RW52XobtxciX3s44yIs9laorS0YPdaMOmkjMrw9K+xNMPaF05V5CbwO6fHnaBNZl2aEKCYIUkVl8sZZT6Zz\\\\\\\/pmqIXdi0cvFbsx18z8UFj2kWh0qHpzEo5BLQL29YSEy4qfVieMrxu2DGMa9TlKA5yjUd8iSBsiwcBkQgbI5\\\\\\\/imQ9gxJGlhLEBda2e2kxzCdEB\\\\\\\/nRhUNwgOtscWYD33RiW9vH2Vfv4BluBbpFy0kBV5dgtvrkAoSPYjEU+7XO14UtL3Lynt4nKmd6Wv\\\\\\\/qnO3wEbVCXyPbUkNgcDC13utZYokS+OgxM1oiaylolD28n1abgUSZ\\\\\\\/KyaC17sN25COlDwwVZpbMCX2RWbg+ODxJj4JLWj83UE7BQ612FN61VrkGCVHot64f4+BZSQJ+M\\\\\\\/X6c88QMxGS54\\\\\\\/b\\\\\\\/pndeDCc5Kd5YqS+GdJjPw\\\\\\\/FVciJ1PgHBf2Qt9RyEy1cgyqpOtp8uPUiwbafHCKCT7IAbFlBkubHO\\\\\\\/urfepjEf4ZdvE+w\\\\\\\/UQGDO7aUcsCV2gpm3QKb4hq0DV+uiSZhPx+CSFcqbUCPd5mrz4PMs1UjdrQeZVeZqFvGNGCGY7ze3GtLnIO8\\\\\\\/nqltb5wu+gYMDhnHLIUNX2LMsq6ZadEb1xkf9Tpm4YEvCB1TegFH3Ae50\\\\\\\/qYTLM+aZzOW+3yb9H7MkbH5v8QlERVqjfU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZW81wre\\\\\\\/EkkrD+xKCgNdNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00224923d3ae110ca45fa8198f5d8fcf4716aae2c5b7b6df5d0c50d5139af506175e\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.09.10.21263388v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-585542 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.09.10.21263388v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2037640676">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2037640676" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_2055169759" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-ECBs43e3MKhu-w5m7GHfIks4jTPJN3PLrzyz7IRijmo"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node585542" data-node-nid="585542" id="node-585542--21685830399" data-pisa="medrxiv;2021.09.10.21263388v2" data-pisa-master="medrxiv;2021.09.10.21263388" data-apath="/medrxiv/early/2022/11/21/2021.09.10.21263388.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Daniel
                      </span>
                      <span class="nlm-surname">
                       Freeman
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Catherine
                      </span>
                      <span class="nlm-surname">
                       Diefenbach
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Linda
                      </span>
                      <span class="nlm-surname">
                       Lam
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Tri
                      </span>
                      <span class="nlm-surname">
                       Le
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Jason
                      </span>
                      <span class="nlm-surname">
                       Alexandre
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Bruce
                      </span>
                      <span class="nlm-surname">
                       Raphael
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="6">
                      <span class="nlm-given-names">
                       Michael
                      </span>
                      <span class="nlm-surname">
                       Grossbard
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       David
                      </span>
                      <span class="nlm-surname">
                       Kaminetzky
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Jia
                      </span>
                      <span class="nlm-surname">
                       Ruan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">
                       Pratip K.
                      </span>
                      <span class="nlm-surname">
                       Chattopadhyay
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.09.10.21263388
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-585542--21685830399" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Daniel Freeman
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        TerraFlow Bioinformatics
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Biology and Biomedical Sciences, Harvard Medical School
                       </span>
                       , Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Perlmutter Cancer Center, NYU Langone Health
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Daniel%2BFreeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Freeman%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADaniel%2BFreeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Catherine Diefenbach
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        TerraFlow Bioinformatics
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Catherine%2BDiefenbach%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Diefenbach%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ACatherine%2BDiefenbach%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Linda Lam
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        TerraFlow Bioinformatics
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Linda%2BLam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lam%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALinda%2BLam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Tri Le
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Perlmutter Cancer Center, NYU Langone Health
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Tri%2BLe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Le%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ATri%2BLe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Jason Alexandre
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        TerraFlow Bioinformatics
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jason%2BAlexandre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Alexandre%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJason%2BAlexandre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Bruce Raphael
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Perlmutter Cancer Center, NYU Langone Health
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Bruce%2BRaphael%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Raphael%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABruce%2BRaphael%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Michael Grossbard
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Perlmutter Cancer Center, NYU Langone Health
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Michael%2BGrossbard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Grossbard%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMichael%2BGrossbard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     David Kaminetzky
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Perlmutter Cancer Center, NYU Langone Health
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=David%2BKaminetzky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kaminetzky%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADavid%2BKaminetzky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Jia Ruan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Division of Hematology & Medical Oncology, Weill Cornell Medicine
                       </span>
                       , New York, NY,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jia%2BRuan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ruan%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AJia%2BRuan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Pratip K. Chattopadhyay
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Talon Biomarkers
                       </span>
                       , Mendham, NJ,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Pratip%2BK.%2BChattopadhyay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chattopadhyay%20PK&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3APratip%2BK.%2BChattopadhyay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:pratip@talonbiomarkers.com" class="" data-icon-position="" data-hide-link-title="0">
                        pratip@talonbiomarkers.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.09.10.21263388v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.10.21263388v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.10.21263388v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.10.21263388v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.09.10.21263388v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.09.10.21263388v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:585542" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:585542/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             The incredible variety of proteins associated with immune responses presents a major challenge in immune monitoring. When combinations of these proteins are measured, cell types that influence disease can be precisely identified. Here, we introduce TerraFlow, a novel data analysis tool that performs an exhaustive search of disease-associated cell populations from high-parameter flow cytometry experiments. Using a newly generated dataset, from 24-color immune checkpoint-focused and 18-color immune function-focused experiments, we apply TerraFlow to classical Hodgkin lymphoma (cHL), where systemic T-cell immunity has not been investigated in detail. We reveal novel immune perturbations in newly diagnosed cHL, as well as persistent immune perturbations after treatment. Newly diagnosed patients have elevated levels of activated (CD278+), exhausted (e.g., CD366+ and CD152+ phenotypes), and IL17-expressing cells, along with diminished levels of naïve and central memory (CD127+) T-cells and fewer IFNγ+ and TNF+ T-cells. Exhaustion signatures are reduced with treatment, but compared to healthy individuals, treated patients still exhibit more activated (CD278+ phenotypes), exhausted (CD366+), and IL17-expressing cells. Notably, TerraFlow identifies more phenotypic differences between patient groups than FlowSOM and CellCNN, often with better predictive power. Finally, we introduce a new non-gating approach for data analysis that obviates the need for time-consuming and subjective setting of fluorescence thresholds. Our results benchmark TerraFlow against common methods, provide mechanistic support for past reports of immune deficiency in cHL, and provide a roadmap for future immunotherapy and biomarker studies.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-17">
                             Immune responses are coordinated by a myriad of proteins distributed across a wide variety of cell types. The presence or absence of particular cell types may significantly influence the immune response in malignancy. In lymphoma, for example, the immune landscape of the tumor microenvironment (TME) plays a clear role in disease. Patients with classical Hodgkin lymphoma (cHL) possess rare, malignant Hodgkin/Reed Sternberg (HRS) cells that shape their microenvironment to prevent immune surveillance and inhibit cytotoxic immune responses
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              -
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             . Various mechanisms underlying this phenomenon have been proposed including the secretion of inhibitory cytokines such as TARC (CCL17), attraction of suppressive T helper 2 (Th2) and regulatory T (Treg) cells to the TME, and differentiation of naïve CD4+ T-cells into forkhead box P3 (FoxP3+) regulatory (Treg) T-cells
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              -
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-18">
                             Less, however, is known about the systemic immune system of patients with cHL. Specifically, are immune perturbations in cHL global or local, and what influence does TME exert on systemic immunity? Several studies suggest systemic immune dysregulation in early and advanced cHL, as demonstrated by poor responses to recall antigens in delayed-type hypersensitivity testing
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             and systemic elevations of a variety of secreted immune modulators, including CCL17 (TARC)
                             <sup>
                              <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              -
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             , IL6
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             , IL2R
                             <sup>
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             , galectin-1
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                             </sup>
                             , and soluble CD30
                             <sup>
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             . Importantly, these studies did not detail the specific cell subsets that are altered in cHL. Characterization of immune cell subsets in lymphoma is important, because systemic immune dysfunction can influence anti-tumor immunity, treatment responses, or immunity to other diseases or to vaccination. Furthermore, the development of prognostic and treatment related biomarkers is most efficient when candidates are identified from peripheral blood, since this sample type is amenable to routine monitoring. Characterizing the systemic immune landscape of cHL will significantly advance immune biomarker development in cHL.
                            </p>
                            <p id="p-19">
                             High-parameter flow cytometry is a particularly established and robust platform for characterizing immunophenotypes but can be limited by bottlenecks in data analysis
                             <sup>
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             . Unsupervised methods such as k-means and FlowSOM partition cells into clusters based on similar protein expression profiles. Downstream analyses then compare cluster abundance between patient groups. While unsupervised methods are useful for detecting natural groupings, they do not account for heterogeneity within clusters. Moreover, many antibody panels do not resolve biologically distinct subtypes
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             . Recently, supervised methods have begun to couple cell representation and disease association into one iterative process. For example, CellCNN uses a convolutional neural network to automatically learn the molecular features of disease-associated cell types. Because supervised models are directly optimized to predict patient groups, they are more sensitive to rare populations than unsupervised methods
                             <sup>
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             . However, models may only report a subset of cell types affected by disease. Moreover, extensive downstream analysis is needed to interpret selected populations and develop biomarkers to define them. There remains a need for a method that can clearly define a complete and interpretable set of cell phenotypes associated with disease.
                            </p>
                            <p id="p-20">
                             In this paper, we introduce TerraFlow (
                             <a href="http://www.terraflow.app">
                              www.terraflow.app
                             </a>
                             ), a novel data analysis tool that performs an exhaustive search of disease-associated cell populations and returns results in a directly interpretable format. Past tools we (and others) developed systematically measured every possible phenotype generated from Boolean combinations of all markers in an antibody panel
                             <sup>
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             . Complete enumeration can produce hundreds of significant phenotypes, many of which contain extraneous or overlapping markers. TerraFlow resolves these redundancies by selecting the smallest set of phenotypes that capture major cohort differences. Each phenotype contains the minimum number of markers needed to define the target population; the addition or subtraction of additional markers reduces the phenotype’s association with outcome. Meanwhile, a recursive feature elimination (RFE) module identifies the smallest set of markers that can be used to discriminate patient groups. This information can be used to design large-scale validation or correlates studies on lower-parameter instruments. Because the output of TerraFlow consists of precisely-defined phenotypes, rather than clusters of cells on a plot, cell populations of importance can be easily interpreted, purified by cell sorting, or developed as clinical biomarkers.
                            </p>
                            <p id="p-21">
                             TerraFlow also introduces a novel non-gating approach for generating phenotypes. Traditional flow analyses use hand-drawn gates to define populations of interest. While a convenient method for measuring complex phenotypes, manual gates can obscure expression patterns that do not conform to a strict on/off binary. Take, for example, a setting where cells expressing high levels of both IFNγ and TNF are important in disease outcome (like what is observed for polyfunctional T-cells). Whereas traditional gates treat each population as an on/off binary, we introduce a non-gating approach that incorporates the number of expressing cells as well as their relative brightness. Because IFNγ and TNF are each relatively rare, we first transform fluorescent intensities with a sigmoidal function that inflates the value of bright cells over neutral or dim cells. Because the double-positive population may be rare compared to the single-positive populations, we next apply a root product calculation that favors the double-positive population. Finally, we calculate the average cell weight within each patient and compare population abundance between cohorts. We show that the non-gating method approximates Boolean phenotypes while also capturing important variation within the positive and negative regions.
                            </p>
                            <p id="p-22">
                             By combining the power of high-parameter flow cytometry with our novel data analysis platform, we investigated whether newly-diagnosed cHL was associated with perturbations in the T-cell compartment and whether these perturbations resolved after treatment. We assayed cellular proteins associated with activation, exhaustion, and suppression of peripheral T-cells. We also studied cytokine expression after in vitro polyclonal re-stimulation in the context of cell differentiation, activation, and exhaustion. Our results catalog peripheral immunity in cHL patients with unprecedented depth, revealing previously unappreciated systemic immune abnormalities in cHL patients.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Cell processing and high-parameter flow cytometry
                             </h3>
                             <p id="p-23">
                              PBMCs were derived from whole blood using density-gradient centrifugation, resuspended at 10 million cells/mL, and cryopreserved at -135° C. Samples were thawed, washed in RPMI (Invitrogen, Carlsbad, CA), and split into two equal aliquots. The first aliquot was stained with a panel of antibodies described in previous work
                              <sup>
                               <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               ,
                               <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              and
                              <strong>
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Supplemental Table 1
                               </a>
                              </strong>
                              (“Immune Checkpoint Panel”). The second was stimulated with phorbol myristate acetate (PMA) and ionomycin (Sigma, St. Louis, MO) in the presence of Golgi Plug containing Brefeldin A (BD Biosciences, San Jose, CA). After four hours, cells were stained with the flow cytometry antibodies described in
                              <strong>
                               <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                                Supplemental Table 2
                               </a>
                              </strong>
                              (“Cytokine Panel”). Samples were immediately analyzed on a Symphony Flow Cytometer (BD Biosciences, San Jose, CA). Flow cytometry staining from a representative patient is shown in
                              <strong>
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Supplemental Figure 1A
                               </a>
                              </strong>
                              .
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/585840/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup last">
                                  <a href="/highwire/markup/585840/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Supplemental Table 1
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/585836/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/585836/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/585836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Supplemental Table 2
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F7.large.jpg?width=800&height=600&carousel=1" title="A) Antibody staining from a representative patient. B) Left panel: As the number of parameters in a combinatorial phenotype increase, many of the putative cell populations are undetectable. Beyond five markers, >95% of the theoretical phenotypes produced by exhaustive combinatorics are not represented in the dataset. Right panel: Discrimination of cHL versus healthy patients does not improve as phenotypes contain more than five markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Antibody staining from a representative patient. B) Left panel: As the number of parameters in a combinatorial phenotype increase, many of the putative cell populations are undetectable. Beyond five markers, >95% of the theoretical phenotypes produced by exhaustive combinatorics are not represented in the dataset. Right panel: Discrimination of cHL versus healthy patients does not improve as phenotypes contain more than five markers.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F7.medium.gif" width="440" height="350"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplemental Figure 1:" src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F7.medium.gif" width="440" height="350"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Supplemental Figure 1:
                               </span>
                               <p id="p-75" class="first-child">
                                A) Antibody staining from a representative patient. B) Left panel: As the number of parameters in a combinatorial phenotype increase, many of the putative cell populations are undetectable. Beyond five markers, >95% of the theoretical phenotypes produced by exhaustive combinatorics are not represented in the dataset. Right panel: Discrimination of cHL versus healthy patients does not improve as phenotypes contain more than five markers.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Human subjects
                             </h3>
                             <p id="p-24">
                              Informed consent was obtained from 44 cHL patients treated at NYU Langone Health and New York-Presbyterian Weill Cornell between 2011 and 2016. Blood samples were drawn before treatment and at a three-month follow up. 25 age-matched, cryopreserved, healthy donor PBMCs were also obtained from STEMCELL Technologies (Cambridge, MA). Each cohort compared in this study was represented by 25-33 individuals (
                              <strong>
                               <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                Supplemental Table 2
                               </a>
                              </strong>
                              ). Typical of cHL, patients had a median age of 34.5 and a range of 18-90 years. 52% were male and 48% female. Patients had nodular sclerosing (80%), mixed cellularity (10%), lymphocyte rich (3%), and unspecified (2%) histology. Most had stage II disease (64%) followed by stage III (14%) and IV (21%). Patients with active viral infection or autoimmune disease were excluded. For post-treatment analyses, patients had received ABVD +/-consolidative radiation.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Non-gating combinatorics to generate phenotypes
                             </h3>
                             <p id="p-25">
                              Fluorescent intensities are linearized with the logicle method (F), rescaled to a variable range [α, β] and transformed with a sigmoidal function (θ) that inflates the expression of brighter cells. For negative markers, the transformation is reversed to favor dimmer cells. Sample expression (s) was calculated by taking the root mean product of the transformed cell values. Formally, expression of a given combinatoric phenotype was defined as:
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/11/21/2021.09.10.21263388/embed/graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/11/21/2021.09.10.21263388/embed/graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where F = Linearized flourescent signal
                             </p>
                             <p id="p-26">
                              K = Number of proteins
                             </p>
                             <p id="p-27">
                              N = Number of cells
                             </p>
                             <p id="p-28">
                              For example, CD95+CCR7-expression in sample 1 was defined as:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/11/21/2021.09.10.21263388/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.medrxiv.org/sites/default/files/highwire/medrxiv/early/2022/11/21/2021.09.10.21263388/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              In this study, we used α = −9 and β = 5 for both panels.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Detection limit
                             </h3>
                             <p id="p-29">
                              The number of possible combinations increases factorially with the number of proteins. However, fewer cell types are detectable as phenotypes become more complex. TerraFlow excludes phenotypes represented by fewer than 100 cells/sample on average. Samples can have fewer than 100 cells if frequencies follow a non-uniform distribution. For example, a detectable phenotype could be represented by 200 cells/sample in one cohort and completely absent in the other. TerraFlow also excludes gates containing more than 95% of parent events. After filtering out sparse and redundant phenotypes, we found that combinations of 1-5 proteins were sufficient to capture ∼95% of all detectable phenotypes in the Hodgkin’s checkpoint dataset (
                              <strong>
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Supplemental Figure 1B
                               </a>
                              </strong>
                              ). Stratification of healthy and cHL patients did not improve when Elastic Net models were trained on higher-order combinations (alpha=0.1,
                              <strong>
                               <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                                Supplemental Figure 1B
                               </a>
                              </strong>
                              ). Based on these results, all subsequent results are based on combinations of 1-5 proteins.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Identification of phenotypes enriched in a patient group
                             </h3>
                             <p id="p-30">
                              Many canonical immune populations are defined by the combination of proteins they express on their surface. For example, naïve T-cells are defined by CCR7+CD45RA+ while central memory cells are defined by CCR7+CD45RA−. Each additional marker resolves subtypes with deeper levels of granularity. TerraFlow extends this intuition by systematically evaluating every combination of 1 to 5 markers that could be measured within a given panel, generating ∼200,000 phenotypes per dataset. Here, we introduce a novel search algorithm then arranges features into a network that captures gradient changes as markers are successively added, removed, and swapped between related phenotypes. First, linear correlation is calculated between the abundance of each phenotype and patient outcome. Next, phenotypes are arranged into a network by adding edges where nodes differ by the addition or removal of one marker. For example, one path through the network might pass through nodes CCR7+ → CCR7+CD4+ → CCR7+CD4+CD8−. Finally, each node is queried to determine if its correlation is stronger than any adjacent node. These optima are selected to represent unique differences between patient cohorts.
                             </p>
                             <p id="p-31">
                              Selected phenotypes are queried again, this time to determine if any markers can be removed without compromising correlation. If a simpler version of a phenotype has a correlation within 97% of the optimal, the extra marker is removed. For example, CD95+CD4+ may have a correlation of 0.90 but CD95+ alone may have a correlation of 0.89. The extra CD4 marker is dropped in favor of the simpler representation. Phenotypes are pruned to convergence, further reducing the total number of selected populations.
                             </p>
                             <p id="p-32">
                              Finally, phenotypes are grouped together if they define overlapping cell populations. For example, CD4+ and CD8− both describe helper T-cells. TerraFlow resolves redundant phenotypes by hierarchically clustering populations with Jaccard similarity indices of 50% or greater. Each group is represented as a set of alternative gating strategies or collapsed into the phenotype with the strongest correlation.
                             </p>
                             <p id="p-33">
                              Selected phenotypes meet a rigorous, independently verifiable set of criteria. First, each phenotype is represented by an optimal phenotype. Adding additional markers to these phenotypes will not improve association between population abundance and disease status. Conversely, removing any one marker will severely weaken disease association. Second, each phenotype represents a unique cell type. Overlapping cell populations are grouped together, even if defined by distinct molecular features. Finally, each phenotype represents a statistically significant correlate of disease. Together, these criteria allow TerraFlow to return a tractable set of phenotypes without sacrificing important disease information.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Recursive feature elimination to guide development of simpler antibody panels
                             </h3>
                             <p id="p-34">
                              While large panels are useful for exploratory purposes, they often contain more markers than are needed to predict the phenotype of interest. TerraFlow uses recursive feature elimination (RFE) to identify the smallest set of markers that allow accurate predictions of patient group. A regularized logistic regression model uses the full set of combinatoric features to predict patient group. Once baseline performance is established, every combination containing the first marker is removed from the feature set and a new model is trained on the remaining phenotypes. The first marker was restored to the feature set and a second marker was removed. The process is repeated for every marker in the panel. At each iteration, the model’s ability to predict sample labels is reevaluated using 10-fold cross-validation and compared to the baseline. The marker whose removal has the least detrimental impact on performance is permanently removed from the feature set. The process repeats until one marker remains. In many cases, performance held constant or even improved until a critical set of markers remained. Removing any of these markers from the panel results in a sharp drop in accuracy. Conversely, restoring any one marker does not significantly improve performance.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Weighted Lasso maximizes accuracy and interpretability
                             </h3>
                             <p id="p-35">
                              A custom logistic regression model predicted patient groups during RFE. As in Lasso, an L2 penalty encouraged sparse models by eliminating predictors that were not relevant to the classification task or were highly correlated to each other. Lasso models tend to select complex phenotypes that are overrepresented in the feature set. Here, an additional tiebreaker term penalized predictors based on the number of markers in the phenotype. If two predictors were highly correlated to clinical outcome and each other, the tiebreaker term ensured that the simpler phenotype prevailed.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Evaluating and confirming model results
                             </h3>
                             <p id="p-36">
                              Models were evaluated using 10-fold stratified cross-validation. Test set predictions were pooled from each fold before calculating accuracy or area under the curve (AUC). At each fold, we enumerated every combination of 1-5 markers that was detectable in the training set. We then used TerraFlow to identify unique disease-associated cell types. From those, we subset the 20 phenotypes with strongest correlation to patient label or those with an FDR-adjusted P-value smaller than 0.05 (whichever was more stringent). Finally, we trained a ridge logistic regression model to use selected phenotypes to predict patient labels in the test set. Lambda was optimized by performing 10-fold CV within each training set and selecting a value one standard deviation greater than the optimal.
                             </p>
                             <p id="p-37">
                              For comparison to existing algorithms, we used the FlowSOM package in R to partition cells into 8 clusters. Fluorescent intensities were logicly-transformed and Z-score normalized. We decreased grid dimensions until the largest cluster contained fewer than 50% of total events. The final model used xdim=5 and ydim=5 for both panels. A simple logistic regression model used cluster frequencies to predict patient labels.
                             </p>
                             <p id="p-38">
                              We also used the Python implementation of CellCNN to automatically learn disease-associated cell types. The same preprocessed data was used for FlowSOM and CellCNN analysis. Within each training set, we used nested 3-fold CV to select from the following hyperparameters: maxpool percentages=(0.01, 1.0, 5.0, 20.0, 100.0), nfilters=(3, 4, 5, 6, 7, 8, 9), learning rate=[0.001, 0.01]. We used 3,000 cells per multi-cell input and 200 multi-cell inputs based on previous experiments on PBMCs. To describe selected populations, CellCNN automatically compiled a matrix of filter weights from all runs achieving a validation accuracy above 95%. It then performed hierarchical clustering with a cosine similarity cutoff of 0.4. One representative filter was selected from each cluster to display in the heatmap. We obtained population frequencies by transforming FCS data with each selected filter and measuring the percent of cells with a response greater than 0.
                             </p>
                             <p id="p-39">
                              For validation results from the newly-diagnosed comparison to healthy patients, an independent dataset was generated from a new experiment with unique patient samples.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              TerraFlow: A New High Parameter Data Analysis Pipeline
                             </h3>
                             <p id="p-40">
                              TerraFlow is a multi-step pipeline for analysis of high parameter single-cell data. It uses an efficient combinatorics approach to enumerate cells expressing every possible combination of up to five markers at a time.
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                              </strong>
                              describes each step in the pipeline. First, cell populations are constructed using TerraFlow’s novel “non-gating” approach. Raw fluorescent intensities are transformed with a set of functions that favor bright, multi-positive cells, allowing rare events to affect the overall mean. Expression levels for each marker are automatically translated into the familiar “positive” and “negative” terminology, reflecting relative levels of expression. Thus, cells with high expression are “positive,” while cells with low levels of (or no) expression are called “negative.” TerraFlow can also create Boolean combinations of traditional, user-defined thresholds (i.e., gates,
                              <strong>
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Figure 1A
                               </a>
                              </strong>
                              ). The process is repeated for every possible combination of 1-5 markers, generating ∼200,000 phenotypes per panel.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F1.large.jpg?width=800&height=600&carousel=1" title="A) TerraFlow introduces a new method for constructing populations by transforming fluorescence signal by a sigmoid function. The non-gating approach weights bright, multi-positive cells heavily. TerraFlow also constructs populations from combinations of gates defined by manual thresholds. B) Populations constructed with either the non-gating or traditional Boolean approaches are tested for their association with study groups, and arranged into a network by adding edges where nodes differ by the addition or removal of one marker. Each node is then queried to determine if its correlation is stronger than any adjacent node. Nodes with the highest correlation to outcome are selected as an optimum representative of a family of cell populations. C) Recursive Feature Elimination (RFE) iteratively tests machine learning models, beginning with a model containing all markers, followed by models that remove one marker at a time. The markers whose removal adversely impacts AUC are those deemed necessary to discriminate the patient groups. D) TerraFlow filters the dataset from ∼200,000 cell populations to identify unique, disease-associated cell types and the minimal set of markers that define the difference between patient groups. E) Comparison between non-gating and threshold-based gating shows that non-gating captures more variation in expression levels from within traditional positive and negative gates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) TerraFlow introduces a new method for constructing populations by transforming fluorescence signal by a sigmoid function. The non-gating approach weights bright, multi-positive cells heavily. TerraFlow also constructs populations from combinations of gates defined by manual thresholds. B) Populations constructed with either the non-gating or traditional Boolean approaches are tested for their association with study groups, and arranged into a network by adding edges where nodes differ by the addition or removal of one marker. Each node is then queried to determine if its correlation is stronger than any adjacent node. Nodes with the highest correlation to outcome are selected as an optimum representative of a family of cell populations. C) Recursive Feature Elimination (RFE) iteratively tests machine learning models, beginning with a model containing all markers, followed by models that remove one marker at a time. The markers whose removal adversely impacts AUC are those deemed necessary to discriminate the patient groups. D) TerraFlow filters the dataset from ∼200,000 cell populations to identify unique, disease-associated cell types and the minimal set of markers that define the difference between patient groups. E) Comparison between non-gating and threshold-based gating shows that non-gating captures more variation in expression levels from within traditional positive and negative gates.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F1.medium.gif" width="334" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F1.medium.gif" width="334" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <p id="p-41" class="first-child">
                                A) TerraFlow introduces a new method for constructing populations by transforming fluorescence signal by a sigmoid function. The non-gating approach weights bright, multi-positive cells heavily. TerraFlow also constructs populations from combinations of gates defined by manual thresholds. B) Populations constructed with either the non-gating or traditional Boolean approaches are tested for their association with study groups, and arranged into a network by adding edges where nodes differ by the addition or removal of one marker. Each node is then queried to determine if its correlation is stronger than any adjacent node. Nodes with the highest correlation to outcome are selected as an optimum representative of a family of cell populations. C) Recursive Feature Elimination (RFE) iteratively tests machine learning models, beginning with a model containing all markers, followed by models that remove one marker at a time. The markers whose removal adversely impacts AUC are those deemed necessary to discriminate the patient groups. D) TerraFlow filters the dataset from ∼200,000 cell populations to identify unique, disease-associated cell types and the minimal set of markers that define the difference between patient groups. E) Comparison between non-gating and threshold-based gating shows that non-gating captures more variation in expression levels from within traditional positive and negative gates.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-42">
                              To identify phenotypes differentially expressed between patient groups, we use a novel form of network analysis (
                              <strong>
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Figure 1B
                               </a>
                              </strong>
                              ). To demonstrate, we simulate a flow cytometry dataset in which there are no differences between two cohorts aside from random noise. We then inject CCR7+CD95+ cells into the treatment cohort. Injection affects the frequency of CCR7+CD95+ but also of related phenotypes such as CCR7+ and CCR7+CD95+GITR+ (boxplots,
                              <strong>
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Figure 1B
                               </a>
                              </strong>
                              ). Rather than report out all three populations as significant, TerraFlow finds the phenotype that best represents cohort differences. First, linear correlation is calculated between population abundance and patient outcome. Next, phenotypes are arranged into a network by adding edges where nodes differ by the addition or removal of one marker. For example, one path through the network might pass through CCR7+ → CCR7+CD95+ → CCR7+CD95+GITR+ (
                              <strong>
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Figure 1B
                               </a>
                              </strong>
                              ). Finally, each node is queried to determine if its correlation is stronger than any adjacent node. These optima are selected to represent unique differences between patient cohorts.
                             </p>
                             <p id="p-43">
                              Finally, while large panels are useful for exploratory purposes, they often contain more markers than are needed to predict the clinical outcome of interest. TerraFlow uses recursive feature elimination to identify the smallest set of markers that allow accurate predictions of clinical outcome. A machine learning model uses the entire combinatoric feature set to classify samples (Baseline,
                              <strong>
                               <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                                Figure 1C
                               </a>
                              </strong>
                              ). The model then iteratively removes the least important marker from the panel and reevaluates performance using 10-fold cross-validation. The process continues until one marker remains. In the simulated dataset, performance remains stable until two markers are left, correctly highlighting the importance of the injected CCR7 and CD95 cells in the simulated dataset (red box,
                              <strong>
                               <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                                Figure 1C
                               </a>
                              </strong>
                              ). To summarize, the algorithm (
                              <strong>
                               <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                                Figure 1D
                               </a>
                              </strong>
                              ) constructs ∼200,000 cell populations based on combinations of protein expression, from these about 5,000 are detectable in a typical dataset (see Methods), and network analysis typically identifies around 30 unique, disease associated cell types (as described in the results below). The RFE module also identifies the minimal set of markers that can be used to define the difference between patient groups.
                             </p>
                             <p id="p-44">
                              Using data generated from our study of cHL, we compare non-gating and Boolean analyses. For various 1-4 marker phenotypes from the Checkpoint panel (see Methods), cells identified as positive by user-defined threshold-based gating have higher expression values on the non-gating scale (
                              <strong>
                               <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                                Figure 1E
                               </a>
                              </strong>
                              ). The non-gating approach also captures extensive variation in expression level within the positive and negative regions. Patient-level expression, as defined by the non-gating approach or the threshold-based approach, is highly correlated for phenotypes containing 1-3 markers, and slightly less correlated for higher-order phenotypes (
                              <strong>
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                Supplemental Figure 2A
                               </a>
                              </strong>
                              ). Finally, cell populations (i.e., phenotypes) associated with healthy controls or newly-diagnosed-cHL patients have similar associations with outcome, regardless of whether they are defined by the non-gating or threshold-based approaches. These correlations are very high for populations defined by a single marker (1N,
                              <strong>
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                Supplemental Figure 2B
                               </a>
                              </strong>
                              ), and good for populations defined by two (2N) or three (3N) markers. Populations defined by more markers show less correlation. Nevertheless, since various features of the algorithm favor simpler phenotypes, the non-gating approach performance is strong, saving the time needed for manual, threshold-based gating.
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F8.large.jpg?width=800&height=600&carousel=1" title="A) Correlation between frequencies of cells identified by non-gating and Boolean approaches, for single, two, three, and four marker phenotypes. B) The association with patient group is correlated for single marker phenotypes regardless of whether non-gating or Boolean gating is used. For more complex phenotypes, defined by 3-5 markers (3N-5N), results are less correlated for the approaches. TerraFlow favors the identification and reporting of simpler phenotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Correlation between frequencies of cells identified by non-gating and Boolean approaches, for single, two, three, and four marker phenotypes. B) The association with patient group is correlated for single marker phenotypes regardless of whether non-gating or Boolean gating is used. For more complex phenotypes, defined by 3-5 markers (3N-5N), results are less correlated for the approaches. TerraFlow favors the identification and reporting of simpler phenotypes.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F8.medium.gif" width="440" height="303"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplemental Figure 2:" src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F8.medium.gif" width="440" height="303"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplemental Figure 2:
                               </span>
                               <p id="p-76" class="first-child">
                                A) Correlation between frequencies of cells identified by non-gating and Boolean approaches, for single, two, three, and four marker phenotypes. B) The association with patient group is correlated for single marker phenotypes regardless of whether non-gating or Boolean gating is used. For more complex phenotypes, defined by 3-5 markers (3N-5N), results are less correlated for the approaches. TerraFlow favors the identification and reporting of simpler phenotypes.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-45">
                              We use TerraFlow to explore several clinical research questions in the setting of cHL. First, we asked whether measures of T-cell phenotype and function, such as cell activation, exhaustion, and/or cytokine production, are impaired in newly diagnosed cHL patients, compared to healthy controls. Next, we asked whether any differences emerged or persisted after treatment. These analyses compared pre- and post-treatment patients, as well as post-treatment patients and healthy donors. Results are benchmarked against popular methods such as UMAP, FlowSOM, and CellCNN.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Mapping the topology of T-cell phenotype and function in newly-diagnosed cHL patients
                             </h3>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Immunophenotypes
                              </h4>
                              <p id="p-46">
                               We first ask whether systemic T-cell functions such as activation, exhaustion, and suppression were impaired in newly diagnosed cHL patients compared to healthy controls. Our examination begins with TerraFlow’s network analysis (
                               <strong>
                                <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                                 Figure 1B
                                </a>
                               </strong>
                               ). Our non-gating approach evaluated every combination of 1-5 markers that could be formed within the immune checkpoint flow cytometry panel (i.e., Checkpoint dataset), generating approximately 230,000 phenotypes. Of those, approximately 4,800 phenotypes were expressed at detectable levels (see Methods for description of detection threshold). 313 were significantly overexpressed in healthy or newly-diagnosed cHL patients (FDR-adjusted P<0.01). Of those, TerraFlow defined 30 optimal phenotypes. Overlapping populations were further grouped together to produce 27 unique disease-associated cell types.
                               <strong>
                                <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                 Figure 2A
                                </a>
                               </strong>
                               describes the top eight phenotypes with the strongest correlation to patient groups (in this case, healthy donors vs. pre-treatment cHL patients).
                               <strong>
                                <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                 Figure 2B
                                </a>
                               </strong>
                               defines the expression level of other markers (columns) for 12 immunophenotypes (rows) within a heatmap. The heatmap feature allows investigators to quickly scan expression of other markers, beyond those within the 1-5 parameter phenotypes initially defined by Terraflow. This feature provides more biological insight into each population. For each of the phenotypes, the correlation with outcome is also depicted (right side of the panel); the phenotypes are ordered by the strength and directionality of their correlation.
                              </p>
                              <div id="F2" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F2.large.jpg?width=800&height=600&carousel=1" title="A) Distributions for most statistically significant immunophenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Validation of model with training-test set approach." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions for most statistically significant immunophenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Validation of model with training-test set approach.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F2.medium.gif" width="281" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F2.medium.gif" width="281" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 2.
                                </span>
                                <p id="p-47" class="first-child">
                                 A) Distributions for most statistically significant immunophenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Validation of model with training-test set approach.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-48">
                               Models are evaluated using 10-fold cross-validation. TerraFlow achieves excellent separation between healthy and cHL patients, outperforming FlowSOM and approaching CellCNN (AUC=0.96, P<0.001,
                               <strong>
                                <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                 Figures 2C
                                </a>
                                and
                                <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                 2D
                                </a>
                               </strong>
                               ). Additional validation steps are described later in this manuscript.
                              </p>
                              <p id="p-49">
                               Our results reveal that cell populations expressing combinations of GITR, CD366, CD152, CD272, and PD1 are the most enriched in newly diagnosed cHL patients versus healthy individuals (e.g., GITR+CD45RO-CD366+,
                               <strong>
                                <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                 Figure 2A
                                </a>
                               </strong>
                               ). In contrast, cells expressing CD127 are enriched in healthy individuals (and thus diminished in cHL patients, e.g., CD127+GITR-CD272-,
                               <strong>
                                <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                                 Figures 2A
                                </a>
                                and
                                <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                                 2B
                                </a>
                               </strong>
                               ). In sum, these results suggest increased exhaustion (elevated frequencies of GITR+, CD366+, CD152+, and/or PD1+ subsets), activation (CD272+ cells), and differentiation (reduced CD127) of peripheral T-cells in cHL patients.
                              </p>
                              <p id="p-50">
                               TerraFlow can also identify the minimal combination of markers that distinguish two study groups through Recursive Feature Elimination (RFE). TerraFlow first trains a regularized logistic regression model to use the full non-gating combinatoric feature set to classify healthy and newly-diagnosed cHL patients. It then iteratively removes the least predictive marker from the panel and reevaluates performance using 10-fold cross-validation. Performance improved as markers were removed from the panel until eight remained. Six markers achieved performance within 95% of the optimal, highlighting the importance of PD1, CD103, CCR7, and GITR (
                               <strong>
                                <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                 Figure 3A
                                </a>
                               </strong>
                               ). A machine learning model that only includes the RFE-selected markers in combination (
                               <strong>
                                <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                 Figure 3B
                                </a>
                               </strong>
                               ) distinguishes newly-diagnosed cHL patients from healthy donors in an independent cohort of 20 patients (
                               <strong>
                                <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                 Figure 3C
                                </a>
                               </strong>
                               ; AUC = 0.97; p<0.001,
                               <strong>
                                <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                 Figure 3D
                                </a>
                               </strong>
                               ). Cells that are GITR+PD1+ (
                               <strong>
                                <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                 Figure 3E
                                </a>
                               </strong>
                               ) are significantly higher in cHL patients than healthy controls. Thus, the ensemble of PD1, CD103, CCR7, and GITR represent the simplest set of markers that could be incorporated into a flow cytometry panel to distinguish cHL patients from healthy donors, with GITR+PD1+ cells particularly valuable for identification of newly-diagnosed patients.
                              </p>
                              <div id="F3" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F3.large.jpg?width=800&height=600&carousel=1" title="A) RFE identifies PD1, CD103, CCR7, and GITR as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. B) Select phenotypes that contain RFE-selected markers, and their correlation with patient group. C) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. D) Results from training and validation sets. E) Difference in abundance of GITR+PD1+ cells across patient groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) RFE identifies PD1, CD103, CCR7, and GITR as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. B) Select phenotypes that contain RFE-selected markers, and their correlation with patient group. C) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. D) Results from training and validation sets. E) Difference in abundance of GITR+PD1+ cells across patient groups.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F3.medium.gif" width="440" height="408"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F3.medium.gif" width="440" height="408"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 3.
                                </span>
                                <p id="p-51" class="first-child">
                                 A) RFE identifies PD1, CD103, CCR7, and GITR as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. B) Select phenotypes that contain RFE-selected markers, and their correlation with patient group. C) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. D) Results from training and validation sets. E) Difference in abundance of GITR+PD1+ cells across patient groups.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               Traditional Boolean Gating
                              </h4>
                              <p id="p-52">
                               <strong>
                                <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                 Supplemental Figure 3
                                </a>
                               </strong>
                               depicts TerraFlow analysis of cell populations defined by combinations of manually gated thresholds. TerraFlow’s network analysis found that cell populations expressing combinations of GITR, CD152, CD366, CD272, CD278, and HLADR are enriched in newly diagnosed patients (
                               <strong>
                                <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                                 Supplemental Figures 3A
                                </a>
                                and
                                <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                                 B
                                </a>
                               </strong>
                               ). TerraFlow models trained on Boolean frequencies demonstrate lower performance than models trained on non-gating expression levels (cross-validated AUC=0.88, p<0.0001;
                               <strong>
                                <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                                 Supplemental Figures 3C
                                </a>
                                and
                                <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                                 3D
                                </a>
                               </strong>
                               ). Like the non-gating approach, the phenotypes identified in this analysis also suggest that cHL patients exhibit increased exhaustion (GITR+, CD152+, CD366+ phenotypes) and activation (CD278+ and HLADR+ phenotypes).
                              </p>
                              <div id="F9" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F9.large.jpg?width=800&height=600&carousel=1" title="A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) RFE analysis identifies a broad ensemble of markers necessary to identify differences between patient groups. E) Correlation with study group for select phenotypes that contain RFE-selected markers. F) Model, validation, and data for selected RFE-defined phenotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) RFE analysis identifies a broad ensemble of markers necessary to identify differences between patient groups. E) Correlation with study group for select phenotypes that contain RFE-selected markers. F) Model, validation, and data for selected RFE-defined phenotypes.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F9.medium.gif" width="347" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplemental Figure 3." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F9.medium.gif" width="347" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 3." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplemental Figure 3.
                                </span>
                                <p id="p-77" class="first-child">
                                 A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Checkpoint Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) RFE analysis identifies a broad ensemble of markers necessary to identify differences between patient groups. E) Correlation with study group for select phenotypes that contain RFE-selected markers. F) Model, validation, and data for selected RFE-defined phenotypes.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-53">
                               Recursive feature elimination shows that CD152, CD95, PD1, TIGIT, CCR7, CD8, and GITR are important to define the difference between healthy donors and cHL patients (
                               <strong>
                                <a id="xref-fig-9-4" class="xref-fig" href="#F9">
                                 Supplemental Figure 3E
                                </a>
                               </strong>
                               ). Amongst the cell populations defined by these markers, CD8+CD95+ and CCR7+CD95+ cells have the largest coefficients in the final logistic regression model (
                               <strong>
                                <a id="xref-fig-9-5" class="xref-fig" href="#F9">
                                 Supplemental Figure 3F
                                </a>
                               </strong>
                               ); the complete set of phenotypes formed from these markers can distinguish newly diagnosed patients from healthy donors with high separation (AUC=0.97, p<0.001 for the independent validation study). However, the individual phenotypes with the largest coefficients do not describe statistically significant differences alone (
                               <strong>
                                <a id="xref-fig-9-6" class="xref-fig" href="#F9">
                                 Supplemental Figure 3F
                                </a>
                               </strong>
                               ), suggesting that the full ensemble of RFE-selected markers (CD152, CD95, PD1, TIGIT, CCR7, CD8, and GITR) are required to discriminate patient groups when traditional Boolean gating is used.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Immune Function
                              </h4>
                              <p id="p-54">
                               In the cytokine panel, our non-gating approach generates approximately 1,100 phenotypes significantly overexpressed in healthy or newly-diagnosed cHL patients (FDR-adjusted p<8.5E-6). TerraFlow’s network approach reduces these to 25 unique disease phenotypes (
                               <strong>
                                <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                 Figures 4A
                                </a>
                                and
                                <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                 B
                                </a>
                               </strong>
                               shows the top phenotypes). TerraFlow distinguishes healthy and newly-diagnosed cHL patients with a cross-validated AUC of 0.82 (not shown), and p<0.001), comparable to FlowSOM (AUC=0.85) but lower than CellCNN (AUC=0.96, not shown). Still, Terraflow defines more disease-associated cell types than either alternative.
                              </p>
                              <div id="F4" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F4.large.jpg?width=800&height=600&carousel=1" title="A) Distributions for most statistically significant immune function phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies IL4 and CD278 as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Results from training and independent validation cohort of 20 patients. G) Difference in abundance of IL4+ cells across patient groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions for most statistically significant immune function phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies IL4 and CD278 as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Results from training and independent validation cohort of 20 patients. G) Difference in abundance of IL4+ cells across patient groups.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F4.medium.gif" width="440" height="323"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F4.medium.gif" width="440" height="323"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 4.
                                </span>
                                <p id="p-55" class="first-child">
                                 A) Distributions for most statistically significant immune function phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies IL4 and CD278 as a minimal set of markers needed to distinguish healthy donors from newly-diagnosed patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Results from training and independent validation cohort of 20 patients. G) Difference in abundance of IL4+ cells across patient groups.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-56">
                               Our results reveal that cell populations expressing combinations of CD152, CD366, CD57, CD95, CD278, CD134 and IL17 are the most enriched in newly diagnosed cHL patients versus healthy individuals. In contrast, cells expressing IFNγ, TNF, and/or IL4 are enriched in healthy individuals (
                               <strong>
                                <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                 Figures 4A
                                </a>
                                and
                                <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                 4B
                                </a>
                               </strong>
                               ). In sum, these results suggest that peripheral T-cells in cHL patients are exhausted and skewed toward Th17 and Tc17 responses, with a loss of IFNγ, TNF, and IL4-producing cells.
                              </p>
                              <p id="p-57">
                               RFE analysis shows that just two markers, CD278 and IL4 (
                               <strong>
                                <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                                 Figure 4C
                                </a>
                               </strong>
                               ), are sufficient to distinguish newly-diagnosed cHL patients and healthy donors (AUC=0.978, p<0.0001 in the independent validation data;
                               <strong>
                                <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                                 Figure 4D-F
                                </a>
                               </strong>
                               ). IL4 distinguishes patients particularly well (
                               <strong>
                                <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                                 Figure 4G
                                </a>
                                )
                               </strong>
                               . Manual, threshold-based Boolean analysis provided similar results (
                               <strong>
                                <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                 Supplemental Figure 4A-D
                                </a>
                               </strong>
                               ) but did not identify a reduced set of markers for identification of newly-diagnosed patients (data not shown).
                              </p>
                              <div id="F10" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F10.large.jpg?width=800&height=600&carousel=1" title="A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Results for 10-fold cross-validation. E) UMAP projections of healthy donor data, pre-treatment patient data, and overlay of FlowSOM-generated clusters. F) Distributions of frequencies for FlowSOM-defined clusters (top) and CellCNN-define populations (bottom) across patient groups. G) Marker expression across clusters (FlowSOM) and Kernels (CellCNN), and correlation with outcome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Results for 10-fold cross-validation. E) UMAP projections of healthy donor data, pre-treatment patient data, and overlay of FlowSOM-generated clusters. F) Distributions of frequencies for FlowSOM-defined clusters (top) and CellCNN-define populations (bottom) across patient groups. G) Marker expression across clusters (FlowSOM) and Kernels (CellCNN), and correlation with outcome.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F10.medium.gif" width="355" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplemental Figure 4." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F10.medium.gif" width="355" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 4." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplemental Figure 4.
                                </span>
                                <p id="p-78" class="first-child">
                                 A) Distributions of the most statistically significant phenotypes across patient groups (healthy vs. newly-diagnosed cHL; Cytokine Panel; traditional Boolean Gating). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of healthy and newly-diagnosed patients using phenotypes identified by TerraFlow in a ridge logistic regression model; results are compared to CellCNN and FlowSOM. D) Results for 10-fold cross-validation. E) UMAP projections of healthy donor data, pre-treatment patient data, and overlay of FlowSOM-generated clusters. F) Distributions of frequencies for FlowSOM-defined clusters (top) and CellCNN-define populations (bottom) across patient groups. G) Marker expression across clusters (FlowSOM) and Kernels (CellCNN), and correlation with outcome.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               Comparison to Common Analysis Approach
                              </h4>
                              <p id="p-58">
                               We next compared TerraFlow to popular methods such as UMAP, FlowSOM, and CellCNN. UMAP requires visual inspection to identify differences between patient groups, a subjective and time-consuming process. FlowSOM introduces more rigor by automatically clustering cells with similar attributes. FlowSOM results were sensitive to multiple tuning parameters. Furthermore, clusters were contiguous or overlapping in the UMAP projection, reflecting the lack of obvious subtypes in the checkpoint dataset
                               <strong>
                                (
                                <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                 Supplemental Figure 4E
                                </a>
                                )
                               </strong>
                               . FlowSOM identified three clusters significantly enriched in newly-diagnosed cHL patients. Of those, only one cluster validated in the follow-up experiment (P<0.05). CellCNN learned populations that were stronger correlates of cHL but only reported two phenotypes (
                               <strong>
                                <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                                 Supplemental Figure 4F
                                </a>
                               </strong>
                               ). Populations are described using mean fluorescent intensities (for FlowSOM) or learned filter weights (for CellCNN,
                               <strong>
                                <a id="xref-fig-10-4" class="xref-fig" href="#F10">
                                 Supplemental Figure 4G
                                </a>
                               </strong>
                               ). However, because populations are defined with complex transformations of the entire panel, it is not clear if a smaller set of markers would have been sufficient to capture the population of interest. Phenotypes could not be validated by manually gating populations in traditional flow cytometry software.
                              </p>
                             </div>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Changes in T-cell phenotype and function with treatment for cHL
                             </h3>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Immunophenotype and Function
                              </h4>
                              <p id="p-59">
                               TerraFlow’s non-gating approach identifies 387 phenotypes that change significantly (FDR-adjusted p<0.05) between the paired comparison of patients before and after their treatment. Amongst these phenotypes, 12 are unique, including those expressing combinations of PD1+ and CD366+ (elevated pre-treatment) and those expressing HLA-DR+, CD95+, or TIGIT+ (elevated post-treatment;
                               <strong>
                                <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                 Supplemental Figures 5A
                                </a>
                                and
                                <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                 5B
                                </a>
                               </strong>
                               ). During cross-validation, TerraFlow correctly identified the post-treatment sample in 83.3% of individuals, rivaling CellCNN (85.2%) and outperforming FlowSOM (62.3%,
                               <strong>
                                <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                                 Supplemental Figure 5C
                                </a>
                                ;
                               </strong>
                               TerraFlow validation results shown in
                               <strong>
                                <a id="xref-fig-11-4" class="xref-fig" href="#F11">
                                 Supplemental Figure 5D
                                </a>
                               </strong>
                               ). These results suggest that circulating exhausted cells before treatment are replaced by activated (HLADR+) cells, with one exhausted TIGIT+ cell population persisting post-treatment.
                              </p>
                              <div id="F11" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F11.large.jpg?width=800&height=600&carousel=1" title="A) Distributions of the most statistically significant phenotypes across patient groups (pre- vs. post-treatment; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of pre- and post-treatment patients using phenotypes identified by TerraFlow; results are compared to CellCNN and FlowSOM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions of the most statistically significant phenotypes across patient groups (pre- vs. post-treatment; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of pre- and post-treatment patients using phenotypes identified by TerraFlow; results are compared to CellCNN and FlowSOM.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F11.medium.gif" width="244" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplemental Figure 5." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F11.medium.gif" width="244" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 5." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplemental Figure 5.
                                </span>
                                <p id="p-79" class="first-child">
                                 A) Distributions of the most statistically significant phenotypes across patient groups (pre- vs. post-treatment; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) Classification of pre- and post-treatment patients using phenotypes identified by TerraFlow; results are compared to CellCNN and FlowSOM.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-60">
                               RFE analysis (data not shown) could not identify a minimal set of markers from the Checkpoint panel that could distinguish pre- and post-treatment time points. Traditional Boolean analysis, based on investigator-defined thresholds, also could not identify an ensemble of Checkpoint panel phenotypes that distinguished pre- and post-treatment in a machine learning model in the validation dataset (data not shown, mean accuracy = 55%). For the cytokine panel, there was no statistically significant difference observed between pre- and post-treatment, with either the non-gating or traditional Boolean gating approaches (data not shown).
                              </p>
                             </div>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              Do T-cell phenotype and function normalize after treatment for cHL?
                             </h3>
                             <div id="sec-21" class="subsection">
                              <h4>
                               Immunophenotypes
                              </h4>
                              <p id="p-61">
                               For the Checkpoint panel, strong differences were observed between cHL patients after treatment and healthy individuals (26 unique phenotypes detected, TerraFlow AUC=0.94, p<0.0001 with 10-fold cross-validation; data not shown). The performance of TerraFlow’s machine learning model rivaled CellCNN (AUC = 0.96) and exceeded FlowSOM (AUC = 0.79). After treatment, cHL patients continue to exhibit higher levels of activated and exhausted cells than healthy donors, including various cell populations expressing CD272, GITR, or CD152 (
                               <strong>
                                <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                 Supplemental Figures 6A
                                </a>
                                and
                                <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                 6B
                                </a>
                               </strong>
                               ), as defined with our non-gating approach. Notably, cell populations expressing PD1 are lower in patients after treatment than healthy donors, suggesting heterogeneity in checkpoint responses to treatment, and reinforcing the importance of high-parameter, single-cell analysis of multiple markers of exhaustion (
                               <strong>
                                <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                                 Supplemental Figures 6A
                                </a>
                                and
                                <a id="xref-fig-12-4" class="xref-fig" href="#F12">
                                 6B
                                </a>
                               </strong>
                               ). RFE of non-gating data did not identify an interpretable minimal set of markers that distinguish post-treatment patients from healthy donors (data not shown).
                              </p>
                              <div id="F12" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F12.large.jpg?width=800&height=600&carousel=1" title="A) Distributions of the most statistically significant phenotypes across patient groups (post-treatment cHL patients vs. healthy donors; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE analysis identifies CD4, CCR7, CD152, and CD57 as markers necessary to identify differences between patient groups. D) Model, validation, and data for selected RFE-defined phenotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions of the most statistically significant phenotypes across patient groups (post-treatment cHL patients vs. healthy donors; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE analysis identifies CD4, CCR7, CD152, and CD57 as markers necessary to identify differences between patient groups. D) Model, validation, and data for selected RFE-defined phenotypes.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F12.medium.gif" width="255" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplemental Figure 6." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F12.medium.gif" width="255" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 6." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplemental Figure 6.
                                </span>
                                <p id="p-80" class="first-child">
                                 A) Distributions of the most statistically significant phenotypes across patient groups (post-treatment cHL patients vs. healthy donors; Checkpoint Panel; non-gating approach). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE analysis identifies CD4, CCR7, CD152, and CD57 as markers necessary to identify differences between patient groups. D) Model, validation, and data for selected RFE-defined phenotypes.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-62">
                               Traditional, threshold-based comparison of post-treatment patients and healthy donors gave largely similar results, highlighting in addition the elevation of a CD366+ cell phenotype (data not shown). RFE of the threshold-based data showed that the most important markers for describing immunophenotypic differences between post-treatment patients and healthy controls were CD4, CCR7, CD152, and CD57 (
                               <strong>
                                <a id="xref-fig-12-5" class="xref-fig" href="#F12">
                                 Supplemental Figure 6C
                                </a>
                               </strong>
                               ); models built from the phenotypes that include these markers have an AUC of 0.726, with high statistical significance in the test set (p<0.01;
                               <strong>
                                <a id="xref-fig-12-6" class="xref-fig" href="#F12">
                                 Supplemental Figure 6D
                                </a>
                               </strong>
                               ). In particular, CD152+CD57-cells are elevated post-treatment. The overall pattern from both analysis approaches reveals continued exhaustion of cells post-treatment (as evidenced by CD152, and CD57 phenotypes), without normalization to healthy donor levels.
                              </p>
                             </div>
                             <div id="sec-22" class="subsection">
                              <h4>
                               Immune Function
                              </h4>
                              <p id="p-63">
                               TerraFlow revealed 25 disease-associated differences in functional phenotypes between healthy donors and post-treatment cHL patients. Cell types enriched post-treatment included stimulated cells expressing multiple activation and exhaustion markers (CD366, CD95, CD57, CD278, CD152, CD134;
                               <strong>
                                <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                 Figure 5A
                                </a>
                               </strong>
                               ), as well as cell populations expressing IL17 (e.g., CD366+ IL17+;
                               <strong>
                                <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                 Figure 5B
                                </a>
                               </strong>
                               ) and IL4 (CD57+ CD4+ CD278+ CD152+ IL4+;
                               <strong>
                                <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                 Figure 5B
                                </a>
                               </strong>
                               ). In contrast, cell populations expressing TNF were diminished post-treatment (
                               <strong>
                                <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                                 Figures 5A
                                </a>
                                and
                                <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                                 5B
                                </a>
                               </strong>
                               ). Similar results were found in the threshold-based analysis (data not shown). In sum, the data suggests that after treatment, cHL patients have increased polarization of cells toward Th2 and Th17 functions, rather than Th1 function, and cells expressing cytokines in post-treatment patients may be more prone to exhaustion than healthy donors. Our model achieved a cross-validated AUC of 0.91 (p<0.0001), rivaling CellCNN (AUC=0.92) and outperforming FlowSOM (AUC=0.83).
                              </p>
                              <div id="F5" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F5.large.jpg?width=800&height=600&carousel=1" title="A) Distributions for most statistically significant immune function phenotypes across patient groups (newly-diagnosed vs. post-treatment cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies CD95, TNF, and IL17 as a minimal set of markers needed to distinguish pre- and post-treatment patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Difference in abundance of IL17+ cells across patient groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Distributions for most statistically significant immune function phenotypes across patient groups (newly-diagnosed vs. post-treatment cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies CD95, TNF, and IL17 as a minimal set of markers needed to distinguish pre- and post-treatment patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Difference in abundance of IL17+ cells across patient groups.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F5.medium.gif" width="232" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F5.medium.gif" width="232" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 5.
                                </span>
                                <p id="p-64" class="first-child">
                                 A) Distributions for most statistically significant immune function phenotypes across patient groups (newly-diagnosed vs. post-treatment cHL; Cytokine Panel). B) Heat map depicting marker frequency (columns) within each phenotype. Adjacent bar graph shows correlation between population frequency and patient group. C) RFE identifies CD95, TNF, and IL17 as a minimal set of markers needed to distinguish pre- and post-treatment patients. D) Machine learning model including only RFE-selected markers distinguishes healthy from newly-diagnosed patients in an independent validation cohort of 20 patients. E) Results from 10-fold cross validation with training and test datasets. F) Difference in abundance of IL17+ cells across patient groups.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-65">
                               Recursive feature analysis (
                               <strong>
                                <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                                 Figure 5C
                                </a>
                               </strong>
                               ) of the non-gating data shows that CD95, TNF, and IL17 are the minimal set of markers needed to identify differences between post-treatment patients and healthy donors (cross-validated AUC=0.80; p<0.001;
                               <strong>
                                <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                 Figure 5D
                                </a>
                               </strong>
                               ). In particular, IL17+ cells are elevated post-treatment compared to healthy donors (p<0.0001,
                               <strong>
                                <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                 Figure 5D
                                </a>
                               </strong>
                               ). These findings are complementary to, and consistent with, the output of earlier steps in the algorithm: post-treatment patients have reduced Th1 (i.e., IFNγ or TNF) responses, and more exhausted cells primed for apoptosis (CD95+) than healthy donors.
                              </p>
                             </div>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Selected phenotypes validate to new data
                             </h3>
                             <p id="p-66">
                              To further demonstrate the concordance between our novel non-gating approach and traditional threshold-based analysis, we selected the 12 phenotypes most significantly over or under-expressed in newly-diagnosed cHL patients compared to healthy donors. We then manually gated cells with the same phenotype in an independent cohort of 20 individuals. Of the 12 phenotypes generated by our non-gating TerraFlow algorithm, 11 validated to new data when measured with traditional, threshold-based gating (p<0.05). Individual populations from the Checkpoint panel (
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Figure 6A
                               </a>
                              </strong>
                              ) differed strongly between newly-diagnosed cHL patients and healthy donors with a median fold-change of 5.4 and p-value of 0.0044 in the validation set (
                              <strong>
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Figure 6A
                               </a>
                              </strong>
                              ). We then trained a logistic regression model on non-gating expression in the original dataset and applied it to the traditional frequencies in the validation dataset. The model achieved perfect separation between healthy and cHL patients using the 12 selected cell types (p<0.0001), outperforming FlowSOM- and CellCNN-based analyses (
                              <strong>
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Figure 6A
                               </a>
                              </strong>
                              ). Similar results were observed with populations identified from the Cytokine panel analyses, with all three algorithms achieving perfect separation (
                              <strong>
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Figure 6B
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F6.large.jpg?width=800&height=600&carousel=1" title="A) Immune checkpoint phenotypes identified in the training dataset with the non-gating approach can be identified by manual gating in the independent validation cohort, and then frequencies can be compared across patient groups to show that results from the non-gating approach are faithfully replicated. B) Immune function profiles, identified with the Cytokine Panel, can also be replicated across non-gating and traditional approaches." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1374257999" data-figure-caption='<div class="highwire-markup">A) Immune checkpoint phenotypes identified in the training dataset with the non-gating approach can be identified by manual gating in the independent validation cohort, and then frequencies can be compared across patient groups to show that results from the non-gating approach are faithfully replicated. B) Immune function profiles, identified with the Cytokine Panel, can also be replicated across non-gating and traditional approaches.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F6.medium.gif" width="418" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F6.medium.gif" width="418" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.medrxiv.org/content/medrxiv/early/2022/11/21/2021.09.10.21263388/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <p id="p-67" class="first-child">
                                A) Immune checkpoint phenotypes identified in the training dataset with the non-gating approach can be identified by manual gating in the independent validation cohort, and then frequencies can be compared across patient groups to show that results from the non-gating approach are faithfully replicated. B) Immune function profiles, identified with the Cytokine Panel, can also be replicated across non-gating and traditional approaches.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-24">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-68">
                             TerraFlow provides several advantages over current data analysis approaches. First, TerraFlow performs an exhaustive search for disease-associated cell types. In the checkpoint panel, FlowSOM identified one cluster that was weakly associated with cHL in a follow-up experiment. CellCNN identified populations that were stronger correlates of cHL but only reported two phenotypes. By contrast, TerraFlow consistently found ten or more unique cell types that were each strongly correlated to cHL and validated to new data.
                            </p>
                            <p id="p-69">
                             Second, TerraFlow defines each population with an explicit phenotype. FlowSOM and CellCNN represent populations with complex transformations of the entire panel. By contrast, TerraFlow selects phenotypes that can be described with just one or two markers, only adding three or more if necessary to define the target population. Gating strategies can be directly implemented in traditional flow cytometry software. They can be validated using smaller panels typically used in large typical trials. For deeper characterization, gating strategies can be used to sort populations for downstream experiments such as functional assays or whole-transcriptome sequencing. Whereas traditional methods require extensive downstream interpretation, TerraFlow populations can be directly isolated and developed as putative biomarkers.
                            </p>
                            <p id="p-70">
                             Finally, TerraFlow provides superior ease of use. Existing methods require users to anticipate the number of clusters or tune arcane machine learning parameters. By contrast, TerraFlow doesn’t require any input beyond clean FCS data and patient labels. Our non-gating approach even obviates the need for manual thresholds, approximating Boolean gates without using fluorescent cutoffs at all. We show that that the non-gating approach captures expression changes within the target population, increasing overall predictive power. Selected phenotypes easily translate to classical hand-drawn gates.
                            </p>
                            <p id="p-71">
                             Our study of patients with cHL provides a rich and finely detailed analysis of the immunophenotypes and functional features of T-cells before and after treatment. Many cell types expressing markers of T-cell exhaustion and activation are elevated in newly-diagnosed (untreated) patients, revealing extensive, systemic alterations in T-cell subset representation in patients. These alterations include changes in the polarization of function in T-cell subsets, as cells are more likely to be TH17 and TC17 cells than TH1 cells in untreated patients (compared to healthy donors). The loss of IFNγ+ cells in cHL patients may release the brakes on IL17-responses. Our results offer potential mechanistic explanations for immune dysfunction in cHL patients. Our results also suggest that other immune checkpoint targets beyond PD1 may be valuable in cHL treatment, such as CD152 (CTLA), CD366 (TIM3), CD278 (ICOS), CD272 (BTLA), TIGIT, GITR, or cell surface CCR4 and CCR6 (to target TH17 cells). Interestingly, around three months post-treatment, patients still exhibit an altered T-cell immune checkpoint and functional landscape, even though the levels of PD1-expressing cells have normalized; future studies will test whether particular alterations in immune checkpoints are associated with cHL relapse.
                            </p>
                           </div>
                           <div class="section data-availability" id="sec-25">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-72">
                             Data is not publicly available yet.
                            </p>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-74">
                             DF performed experiments, developed TerraFlow, analyzed data, and wrote manuscript. CD conceived of and designed studies, led the clinical study and recruited patients, analyzed and interpreted data, and wrote manuscript. TL performed experiments. LL organized study and performed experiments. JA organized study and performed experiments. BR, MG, DK, and JR conducted the clinical study and recruited patients. PKC conceived of and designed studies, developed and benchmarked TerraFlow, analyzed and interpreted data, and wrote manuscript.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-73">
                             The authors would like to thank Suraj Saksena for coordinating the BD Bioscieces/Precision Immunology Laboratory visiting scientist program under which TL began these experiments. We would also like to thank additional investigators who contributed patients to this study: Tibor Moscovits and Kenneth Hymes (NYU Langone Health), as well as Peter Martin and John Leonard (Weill Cornell). We also acknowledge funding support from the Feinberg family, and the American Cancer Society MRSG (CD).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-16">
                               This revision contains the following updates to the original paper:
1) Comparison to existing approaches
2) Updates to the algorithm
3) Validation on an independent cohort
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list">
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.1" data-doi="10.1200/JCO.2010.32.8401">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steidl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Connors
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gascoyne
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Molecular pathogenesis of Hodgkin’s lymphoma: increasing evidence of the importance of the microenvironment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Clin Oncol
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1812
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1826
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1200/JCO.2010.32.8401
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BClin%2BOncol%26rft_id%253Dinfo%253Adoi%252F10.1200%252FJCO.2010.32.8401%26rft_id%253Dinfo%253Apmid%252F21483001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNvIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xNC8xODEyIjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMTEvMjEvMjAyMS4wOS4xMC4yMTI2MzM4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.10.21263388v2.2" data-doi="10.1002/path.2711">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aldinucci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gloghini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pinto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Filippi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carbone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The classical Hodgkin’s lymphoma microenvironment and its role in promoting tumour growth and immune escape
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Pathol
                                 </abbr>
                                 <span class="cit-vol">
                                  221
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  248
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  263
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/path.2711
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bpathology%26rft.stitle%253DJ%2BPathol%26rft.aulast%253DAldinucci%26rft.auinit1%253DD.%26rft.volume%253D221%26rft.issue%253D3%26rft.spage%253D248%26rft.epage%253D263%26rft.atitle%253DThe%2Bclassical%2BHodgkin%2527s%2Blymphoma%2Bmicroenvironment%2Band%2Bits%2Brole%2Bin%2Bpromoting%2Btumour%2Bgrowth%2Band%2Bimmune%2Bescape.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpath.2711%26rft_id%253Dinfo%253Apmid%252F20527019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/path.2711&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20527019&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000279461000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.10.21263388v2.3" data-doi="10.1038/nrc2542">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kuppers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  The biology of Hodgkin’s lymphoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  27
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/nrc2542
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BCancer%26rft.stitle%253DNat%2BRev%2BCancer%26rft.aulast%253DKuppers%26rft.auinit1%253DR.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D15%26rft.epage%253D27%26rft.atitle%253DThe%2Bbiology%2Bof%2BHodgkin%2527s%2Blymphoma.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc2542%26rft_id%253Dinfo%253Apmid%252F19078975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrc2542&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19078975&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000261929700016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.10.21263388v2.4" data-doi="10.1080/10428190802163339">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hsi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. D.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Biologic features of Hodgkin lymphoma and the development of biologic prognostic factors in Hodgkin lymphoma: tumor and microenvironment
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Leuk Lymphoma
                                 </abbr>
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1668
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1680
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1080/10428190802163339
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLeukemia%2B%2526%2Blymphoma%26rft.stitle%253DLeuk%2BLymphoma%26rft.aulast%253DHsi%26rft.auinit1%253DE.%2BD.%26rft.volume%253D49%26rft.issue%253D9%26rft.spage%253D1668%26rft.epage%253D1680%26rft.atitle%253DBiologic%2Bfeatures%2Bof%2BHodgkin%2Blymphoma%2Band%2Bthe%2Bdevelopment%2Bof%2Bbiologic%2Bprognostic%2Bfactors%2Bin%2BHodgkin%2Blymphoma%253A%2Btumor%2Band%2Bmicroenvironment.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10428190802163339%26rft_id%253Dinfo%253Apmid%252F18798102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/10428190802163339&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18798102&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000259240500008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.5" data-doi="10.1016/j.exphem.2005.11.008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Epstein-Barr virus, cytokines, and inflammation: a cocktail for the pathogenesis of Hodgkin’s lymphoma?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Exp Hematol
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  399
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  406
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/j.exphem.2005.11.008
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bhematology%26rft.stitle%253DExp%2BHematol%26rft.aulast%253DKhan%26rft.auinit1%253DG.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D399%26rft.epage%253D406%26rft.atitle%253DEpstein-Barr%2Bvirus%252C%2Bcytokines%252C%2Band%2Binflammation%253A%2Ba%2Bcocktail%2Bfor%2Bthe%2Bpathogenesis%2Bof%2BHodgkin%2527s%2Blymphoma%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.exphem.2005.11.008%26rft_id%253Dinfo%253Apmid%252F16569586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.exphem.2005.11.008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16569586&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000236838200001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.6" data-doi="10.1002/ijc.23119">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aldinucci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Expression of CCR5 receptors on Reed-Sternberg cells and Hodgkin lymphoma cell lines: involvement of CCL5/Rantes in tumor cell growth and microenvironmental interactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Int J Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  122
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  769
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  776
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/ijc.23119
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Bcancer.%2B%2BJournal%2Binternational%2Bdu%2Bcancer%26rft.stitle%253DInt%2BJ%2BCancer%26rft.aulast%253DAldinucci%26rft.auinit1%253DD.%26rft.volume%253D122%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DExpression%2Bof%2BCCR5%2Breceptors%2Bon%2BReed-Sternberg%2Bcells%2Band%2BHodgkin%2Blymphoma%2Bcell%2Blines%253A%2Binvolvement%2Bof%2BCCL5%252FRantes%2Bin%2Btumor%2Bcell%2Bgrowth%2Band%2Bmicroenvironmental%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fijc.23119%26rft_id%253Dinfo%253Apmid%252F17935139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/ijc.23119&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17935139&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000252522700008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2021.09.10.21263388v2.7" data-doi="10.1189/jlb.0906565">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanijiri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Hodgkin’s reed-sternberg cell line (KM-H2) promotes a bidirectional differentiation of CD4+CD25+Foxp3+ T cells and CD4+ cytotoxic T lymphocytes from CD4+ naive T cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Leukoc Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  82
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  576
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  584
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1189/jlb.0906565
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BLeukoc%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1189%252Fjlb.0906565%26rft_id%253Dinfo%253Apmid%252F17545218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1189/jlb.0906565&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17545218&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000249022200015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.8" data-doi="10.1016/S0002-9440(10)65424-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van den Berg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Visser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Poppema
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  High expression of the CC chemokine TARC in Reed-Sternberg cells. A possible explanation for the characteristic T-cell infiltratein Hodgkin’s lymphoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am J Pathol
                                 </abbr>
                                 <span class="cit-vol">
                                  154
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1685
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1691
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1016/S0002-9440(10)65424-7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2BOf%2BPathology%26rft.stitle%253DAm.%2BJ.%2BPathol.%26rft.aulast%253Dvan%2Bden%2BBerg%26rft.auinit1%253DA.%26rft.volume%253D154%26rft.issue%253D6%26rft.spage%253D1685%26rft.epage%253D1691%26rft.atitle%253DHigh%2BExpression%2Bof%2Bthe%2BCC%2BChemokine%2BTARC%2Bin%2BReed-Sternberg%2BCells%2B%253A%2BA%2BPossible%2BExplanation%2Bfor%2Bthe%2BCharacteristic%2BT-Cell%2BInfiltratein%2BHodgkin%2527s%2BLymphoma%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0002-9440%252810%252965424-7%26rft_id%253Dinfo%253Apmid%252F10362793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0002-9440(10)65424-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10362793&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000080678800010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.9" data-doi="10.1002/1097-0142(19860415)57:8&lt;1489::aid-cncr2820570809&gt;3.0.co;2-n">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Rijswijk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Meijer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sybesma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kater
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Five-year survival in Hodgkin’s disease. The prospective value of immune status at diagnosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  57
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1489
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1496
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/1097-0142(19860415)57:8<1489::aid-cncr2820570809>3.0.co;2-n
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%26rft.stitle%253DCancer%26rft.aulast%253Dvan%2BRijswijk%26rft.auinit1%253DR.%2BE.%26rft.volume%253D57%26rft.issue%253D8%26rft.spage%253D1489%26rft.epage%253D1496%26rft.atitle%253DFive-year%2Bsurvival%2Bin%2BHodgkin%2527s%2Bdisease.%2BThe%2Bprospective%2Bvalue%2Bof%2Bimmune%2Bstatus%2Bat%2Bdiagnosis.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1097-0142%252819860415%252957%253A8%2526lt%253B1489%253A%253Aaid-cncr2820570809%2526gt%253B3.0.co%253B2-n%26rft_id%253Dinfo%253Apmid%252F2936444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/1097-0142(19860415)57:8&lt;1489::aid-cncr2820570809&gt;3.0.co;2-n&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2936444&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.10" data-doi="10.1158/0008-5472.CAN-05-0100">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weihrauch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Elevated serum levels of CC thymus and activation-related chemokine (TARC) in primary Hodgkin’s disease: potential for a prognostic factor
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Res
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5516
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5519
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1158/0008-5472.CAN-05-0100
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DWeihrauch%26rft.auinit1%253DM.%2BR.%26rft.volume%253D65%26rft.issue%253D13%26rft.spage%253D5516%26rft.epage%253D5519%26rft.atitle%253DElevated%2BSerum%2BLevels%2Bof%2BCC%2BThymus%2Band%2BActivation-Related%2BChemokine%2B%2528TARC%2529%2Bin%2BPrimary%2BHodgkin%2527s%2BDisease%253A%2BPotential%2Bfor%2Ba%2BPrognostic%2BFactor%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-05-0100%26rft_id%253Dinfo%253Apmid%252F15994922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2NS8xMy81NTE2IjtzOjQ6ImF0b20iO3M6NTA6Ii9tZWRyeGl2L2Vhcmx5LzIwMjIvMTEvMjEvMjAyMS4wOS4xMC4yMTI2MzM4OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.11" data-doi="10.1002/ajh.22105">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ansell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Annual clinical updates in hematological malignancies: a continuing medical education series. Hodgkin lymphoma: 2011 update on diagnosis, risk-stratification, and management
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am J Hematol
                                 </abbr>
                                 <span class="cit-vol">
                                  86
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  851
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  858
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/ajh.22105
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhematology%26rft.stitle%253DAm%2BJ%2BHematol%26rft.aulast%253DAnsell%26rft.auinit1%253DS.%2BM.%26rft.volume%253D86%26rft.issue%253D10%26rft.spage%253D851%26rft.epage%253D858%26rft.atitle%253DAnnual%2Bclinical%2Bupdates%2Bin%2Bhematological%2Bmalignancies%253A%2Ba%2Bcontinuing%2Bmedical%2Beducation%2Bseries.%2BHodgkin%2Blymphoma%253A%2B2011%2Bupdate%2Bon%2Bdiagnosis%252C%2Brisk-stratification%252C%2Band%2Bmanagement.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajh.22105%26rft_id%253Dinfo%253Apmid%252F21922525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/ajh.22105&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21922525&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F11%2F21%2F2021.09.10.21263388.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouyang J
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pogge
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ponader
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rabinovich
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neuberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   DS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Galectin-1 Serum Levels Reflect Tumor Burden and Adverse Clinical Features in Hodgkin Lymphoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ASH Annual Meeting Abstracts
                                 </abbr>
                                 <span class="cit-vol">
                                  120
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DASH%2BAnnual%2BMeeting%2BAbstracts%26rft.stitle%253DASH%2BANNUAL%2BMEETING%2BABSTRACTS%26rft.aulast%253DStanworth%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D120%26rft.issue%253D21%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DThe%2BEffect%2Bof%2Ba%2BNo-Prophylactic%2BVersus%2BProphylactic%2BPlatelet%2BTransfusion%2BStrategy%2BOn%2BBleeding%2Bin%2BPatients%2Bwith%2BHematological%2BMalignancies%2Band%2BSevere%2BThrombocytopenia%2B%2528TOPPS%2Btrial%2529.%2BA%2BRandomized%2BControlled%252C%2BNon-Inferiority%2BTrial%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.13" data-doi="10.1158/1055-9965.EPI-16-1012">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Elevated Serum Levels of sCD30 and IL6 and Detectable IL10 Precede Classical Hodgkin Lymphoma Diagnosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Epidemiol Biomarkers Prev
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1114
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1123
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1158/1055-9965.EPI-16-1012
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BEpidemiol%2BBiomarkers%2BPrev%26rft_id%253Dinfo%253Adoi%252F10.1158%252F1055-9965.EPI-16-1012%26rft_id%253Dinfo%253Apmid%252F28341757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiY2VicCI7czo1OiJyZXNpZCI7czo5OiIyNi83LzExMTQiO3M6NDoiYXRvbSI7czo1MDoiL21lZHJ4aXYvZWFybHkvMjAyMi8xMS8yMS8yMDIxLjA5LjEwLjIxMjYzMzg4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.14" data-doi="10.1146/annurev-anchem-061417-125927">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chattopadhyay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winters
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. E.
                                  </span>
                                  ,
                                  <span class="cit-name-suffix">
                                   3rd
                                  </span>
                                  .
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laino
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Woods
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  High-Parameter Single-Cell Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu Rev Anal Chem (Palo Alto Calif)
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  411
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  430
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1146/annurev-anchem-061417-125927
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BAnal%2BChem%2B%2528Palo%2BAlto%2BCalif%2529%26rft.volume%253D12%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-anchem-061417-125927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-anchem-061417-125927&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.15" data-doi="10.1172/jci.insight.137066">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Woods
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nivolumab and ipilimumab are associated with distinct immune landscape changes and response-associated immunophenotypes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  JCI Insight
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1172/jci.insight.137066
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJCI%2BInsight%26rft_id%253Dinfo%253Adoi%252F10.1172%252Fjci.insight.137066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1172/jci.insight.137066&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.16" data-doi="10.1038/ncomms14825">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arvaniti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Claassen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Sensitive detection of rare disease-associated cell subsets via representation learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  14825
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1038/ncomms14825
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D14825%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ncomms14825&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.09.10.21263388v2.17" data-doi="10.1002/cyto.a.23608">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nettey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Giles
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chattopadhyay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  OMIP-050: A 28-color/30-parameter Fluorescence Flow Cytometry Panel to Enumerate and Characterize Cells Expressing a Wide Array of Immune Checkpoint Molecules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cytometry A
                                 </abbr>
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1094
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1096
                                 </span>
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                  ,
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1002/cyto.a.23608
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCytometry%2BA%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcyto.a.23608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/cyto.a.23608&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.09.10.21263388v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.08.08.22278547v6" title="Infectiousness of SARS-CoV-2 breakthrough infections and reinfections during the Omicron wave" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.11.19.22282451v1" title="Catch-up Routine Immunization to Restore Childhood Immunization Coverage following COVID-induced Declines in Zamboanga Peninsula, Philippines" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted November 21, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.09.10.21263388v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/11/21/2021.09.10.21263388.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.09.10.21263388v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.09.10.21263388v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/585542"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-zbPsrNe70pUsuOYT2Zxbv-t2FFJk0GhY7p4ikxznLfY"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277159466"/>
                                   <input type="hidden" name="captcha_token" value="fe2e13a1974b4cf74b7044cc7c58e5a0"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node585542--3" data-node-nid="585542" id="node-585542--41412412027" data-pisa="medrxiv;2021.09.10.21263388v2" data-pisa-master="medrxiv;2021.09.10.21263388" data-seqnum="585542" data-apath="/medrxiv/early/2022/11/21/2021.09.10.21263388.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Daniel
                                  </span>
                                  <span class="nlm-surname">
                                   Freeman
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Catherine
                                  </span>
                                  <span class="nlm-surname">
                                   Diefenbach
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Linda
                                  </span>
                                  <span class="nlm-surname">
                                   Lam
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Tri
                                  </span>
                                  <span class="nlm-surname">
                                   Le
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Jason
                                  </span>
                                  <span class="nlm-surname">
                                   Alexandre
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Bruce
                                  </span>
                                  <span class="nlm-surname">
                                   Raphael
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Michael
                                  </span>
                                  <span class="nlm-surname">
                                   Grossbard
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   David
                                  </span>
                                  <span class="nlm-surname">
                                   Kaminetzky
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Jia
                                  </span>
                                  <span class="nlm-surname">
                                   Ruan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Pratip K.
                                  </span>
                                  <span class="nlm-surname">
                                   Chattopadhyay
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.09.10.21263388;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.09.10.21263388
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&title=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&title=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&text=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&t=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&title=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&title=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&title=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node585542--5" data-node-nid="585542" id="citation-node-585542--61592019237" data-pisa="medrxiv;2021.09.10.21263388v2" data-pisa-master="medrxiv;2021.09.10.21263388" data-seqnum="585542" data-apath="/medrxiv/early/2022/11/21/2021.09.10.21263388.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     TerraFlow, A New High Parameter Data Analysis Tool, Reveals Systemic T-cell Exhaustion and Dysfunctional Cytokine Production in Classical Hodgkin Lymphoma
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Daniel
                                      </span>
                                      <span class="nlm-surname">
                                       Freeman
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Catherine
                                      </span>
                                      <span class="nlm-surname">
                                       Diefenbach
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Linda
                                      </span>
                                      <span class="nlm-surname">
                                       Lam
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Tri
                                      </span>
                                      <span class="nlm-surname">
                                       Le
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Jason
                                      </span>
                                      <span class="nlm-surname">
                                       Alexandre
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Bruce
                                      </span>
                                      <span class="nlm-surname">
                                       Raphael
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Michael
                                      </span>
                                      <span class="nlm-surname">
                                       Grossbard
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       David
                                      </span>
                                      <span class="nlm-surname">
                                       Kaminetzky
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Jia
                                      </span>
                                      <span class="nlm-surname">
                                       Ruan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Pratip K.
                                      </span>
                                      <span class="nlm-surname">
                                       Chattopadhyay
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.09.10.21263388;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.09.10.21263388
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Freeman%2C+Daniel&rft.au=Diefenbach%2C+Catherine&rft.au=Lam%2C+Linda&rft.au=Le%2C+Tri&rft.au=Alexandre%2C+Jason&rft.au=Raphael%2C+Bruce&rft.au=Grossbard%2C+Michael&rft.au=Kaminetzky%2C+David&rft.au=Ruan%2C+Jia&rft.au=Chattopadhyay%2C+Pratip+K.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/585542/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/585542/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/585542/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/585542/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/585542/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/585542/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/585542/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/585542/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/585542/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/585542/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/585542/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/585542/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&count=horizontal&via=&text=TerraFlow%2C%20A%20New%20High%20Parameter%20Data%20Analysis%20Tool%2C%20Reveals%20Systemic%20T-cell%20Exhaustion%20and%20Dysfunctional%20Cytokine%20Production%20in%20Classical%20Hodgkin%20Lymphoma&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.09.10.21263388v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.09.10.21263388v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/allergy-and-immunology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Allergy and Immunology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
