<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.01.10.901504v6.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.01.10.901504v6.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.01.10.901504v6.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/06/14/2020.01.10.901504.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.01.10.901504v6"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1"/>
  <meta name="DC.Identifier" content="10.1101/2020.01.10.901504"/>
  <meta name="DC.Date" content="2022-06-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Animals frequently make decisions based on sensory cues. In such a setting, the overlap in the information on the stimulus and on the choice is crucial for the formation of informed behavioral decisions. Yet, how the information on the stimulus and on the choice interact in the brain is poorly understood. Here, we study the representation of a binary decision variable in the primary visual cortex (V1) while macaque monkeys perform delayed match-to-sample task on naturalistic visual stimuli close to psychophysical threshold. Using population vectors, we demonstrate the overlap in decoding spaces on binary stimulus classes “match/non-match” and binary choices “same /different” of the animal. Leveraging this overlap, we use learning from the invariant information across the two classification problems to predict the choice of the animal as a time-dependent population signal. We show the importance of the across-neuron organization and the temporal structure of spike trains for the decision signal and suggest how noise correlations between neurons with similar decoding selectivity are helpful for the accumulation of the decision signal. Finally, we show that decision signal is primarily carried by bursting neurons in the superficial layers of the cortex.

Author summary V1 is necessary for normal visual processing and is known to process features of visual stimuli such as orientation, but whether V1 also encodes behavioral decisions is an unresolved issue, with conflicting evidence. Here, we demonstrate that V1 encodes a mixed variable that contains the information about the stimulus as well as about the choice. We learn the structure of population responses in trials pertaining to the variable “stimulus+choice”, and apply the resulting population vectors to trials that differ only about the choice of the animal, but not about the stimulus class. Moreover, we learn structure of population responses on time-averaged data and then apply it on time-dependent (spiking) data. During the late phase of the trial, this procedure allows to predict the upcoming choice of the animal with a time-dependent population signal. The spiking signal of small neural population is sparse, and we hypothesize that positive correlations between neurons in the same decoding pool help the transmission of the decision-related information downstream. We find that noise correlations in the same decoding pool are significantly stronger than across coding pools, which corroborates our hypothesis on the benefit of noise correlations for the read-out of a time-dependent population signal.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Veronika Koren"/>
  <meta name="DC.Contributor" content="Ariana R. Andrei"/>
  <meta name="DC.Contributor" content="Ming Hu"/>
  <meta name="DC.Contributor" content="Valentin Dragoi"/>
  <meta name="DC.Contributor" content="Klaus Obermayer"/>
  <meta name="article:published_time" content="2022-06-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Animals frequently make decisions based on sensory cues. In such a setting, the overlap in the information on the stimulus and on the choice is crucial for the formation of informed behavioral decisions. Yet, how the information on the stimulus and on the choice interact in the brain is poorly understood. Here, we study the representation of a binary decision variable in the primary visual cortex (V1) while macaque monkeys perform delayed match-to-sample task on naturalistic visual stimuli close to psychophysical threshold. Using population vectors, we demonstrate the overlap in decoding spaces on binary stimulus classes “match/non-match” and binary choices “same /different” of the animal. Leveraging this overlap, we use learning from the invariant information across the two classification problems to predict the choice of the animal as a time-dependent population signal. We show the importance of the across-neuron organization and the temporal structure of spike trains for the decision signal and suggest how noise correlations between neurons with similar decoding selectivity are helpful for the accumulation of the decision signal. Finally, we show that decision signal is primarily carried by bursting neurons in the superficial layers of the cortex.</p><h3>Author summary</h3>
<p>V1 is necessary for normal visual processing and is known to process features of visual stimuli such as orientation, but whether V1 also encodes behavioral decisions is an unresolved issue, with conflicting evidence. Here, we demonstrate that V1 encodes a mixed variable that contains the information about the stimulus as well as about the choice. We learn the structure of population responses in trials pertaining to the variable “stimulus+choice”, and apply the resulting population vectors to trials that differ only about the choice of the animal, but not about the stimulus class. Moreover, we learn structure of population responses on time-averaged data and then apply it on time-dependent (spiking) data. During the late phase of the trial, this procedure allows to predict the upcoming choice of the animal with a time-dependent population signal. The spiking signal of small neural population is sparse, and we hypothesize that positive correlations between neurons in the same decoding pool help the transmission of the decision-related information downstream. We find that noise correlations in the same decoding pool are significantly stronger than across coding pools, which corroborates our hypothesis on the benefit of noise correlations for the read-out of a time-dependent population signal.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.01.10.901504v6"/>
  <meta name="citation_id" content="2020.01.10.901504v6"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.01.10.901504"/>
  <meta name="citation_num_pages" content="32"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.01.10.901504"/>
  <meta name="citation_author" content="Veronika Koren"/>
  <meta name="citation_author_institution" content="Institute of Software Engineering and Theoretical Computer Science, Technische Universität Berlin"/>
  <meta name="citation_author_institution" content="Bernstein Center for Computational Neuroscience Berlin"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2920-2717"/>
  <meta name="citation_author" content="Ariana R. Andrei"/>
  <meta name="citation_author_institution" content="Department of Neurobiology and Anatomy, University of Texas Medical School"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2152-2580"/>
  <meta name="citation_author" content="Ming Hu"/>
  <meta name="citation_author_institution" content="Department of Neurobiology and Anatomy, University of Texas Medical School"/>
  <meta name="citation_author" content="Valentin Dragoi"/>
  <meta name="citation_author_institution" content="Department of Neurobiology and Anatomy, University of Texas Medical School"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9526-0926"/>
  <meta name="citation_author" content="Klaus Obermayer"/>
  <meta name="citation_author_institution" content="Institute of Software Engineering and Theoretical Computer Science, Technische Universität Berlin"/>
  <meta name="citation_author_institution" content="Bernstein Center for Computational Neuroscience Berlin"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0471-8170"/>
  <meta name="citation_reference" content="Shadlen MN, Newsome WT. Motion perception: seeing and deciding. Proceedings of the national academy of sciences. 1996;93(2):628–633."/>
  <meta name="citation_reference" content="Britten KH, Newsome WT, Shadlen MN, Celebrini S, Movshon JA, et al. A relationship between behavioral choice and the visual responses of neurons in macaque MT. Visual neuroscience. 1996;13(1):87–100."/>
  <meta name="citation_reference" content="Shadlen MN, Britten KH, Newsome WT, Movshon JA. A computational analysis of the relationship between neuronal and behavioral responses to visual motion. Journal of Neuroscience. 1996;16(4):1486–1510."/>
  <meta name="citation_reference" content="Logothetis NK, Schall JD. Neuronal correlates of subjective visual perception. Science. 1989;245(4919):761–763."/>
  <meta name="citation_reference" content="Roitman JD, Shadlen MN. Response of neurons in the lateral intraparietal area during a combined visual discrimination reaction time task. Journal of neuroscience. 2002;22(21):9475–9489."/>
  <meta name="citation_reference" content="Nienborg H, Cumming BG. Decision-related activity in sensory neurons reflects more than a neuron’s causal effect. Nature. 2009;459(7243):89–92."/>
  <meta name="citation_reference" content="Goris RL, Ziemba CM, Stine GM, Simoncelli EP, Movshon JA. Dissociation of choice formation and choice-correlated activity in macaque visual cortex. Journal of Neuroscience. 2017;37(20):5195–5203."/>
  <meta name="citation_reference" content="Seidemann E, Geisler WS. Linking V1 activity to behavior. Annual review of vision science. 2018;4:287–310."/>
  <meta name="citation_reference" content="Jasper AI, Tanabe S, Kohn A. Predicting perceptual decisions using visual cortical population responses and choice history. Journal of Neuroscience. 2019;39(34):6714–6727."/>
  <meta name="citation_reference" content="Krishna A, Tanabe S, Kohn A. Decision Signals in the Local Field Potentials of Early and Mid-Level Macaque Visual Cortex. Cerebral Cortex. 2021;31(1):169–183."/>
  <meta name="citation_reference" content="Uka T, DeAngelis GC. Contribution of area MT to stereoscopic depth perception: choice-related response modulations reflect task strategy. Neuron. 2004;42(2):297–310."/>
  <meta name="citation_reference" content="Nienborg H, Cumming BG. Macaque V2 neurons, but not V1 neurons, show choice-related activity. Journal of Neuroscience. 2006;26(37):9567–9578."/>
  <meta name="citation_reference" content="Nienborg H, Cumming BG. Decision-related activity in sensory neurons may depend on the columnar architecture of cerebral cortex. Journal of Neuroscience. 2014;34(10):3579–3585."/>
  <meta name="citation_reference" content="Palmer C, Seidemann E. Choice probability and reaction-Time correlations in Macaque V1. Journal of Vision. 2008;8(6):815–815."/>
  <meta name="citation_reference" content="Yang H, Kwon SE, Severson KS, O’connor DH. Origins of choice-related activity in mouse somatosensory cortex. Nature neuroscience. 2016;19(1):127–134."/>
  <meta name="citation_reference" content="Nienborg H, R Cohen M, Cumming BG. Decision-related activity in sensory neurons: correlations among neurons and with behavior. Annual review of neuroscience. 2012;35:463–483."/>
  <meta name="citation_reference" content="Gao P, Ganguli S. On simplicity and complexity in the brave new world of large-scale neuroscience. Current opinion in neurobiology. 2015;32:148–155."/>
  <meta name="citation_reference" content="Saxena S, Cunningham JP. Towards the neural population doctrine. Current opinion in neurobiology. 2019;55:103–111."/>
  <meta name="citation_reference" content="Boerlin M, Machens CK, Denève S. Predictive coding of dynamical variables in balanced spiking networks. PLoS Comput Biol. 2013;9(11):e1003258."/>
  <meta name="citation_reference" content="Harris KD, Bartho P, Chadderton P, Curto C, de la Rocha J, Hollender L, et al. How do neurons work together? Lessons from auditory cortex. Hearing research. 2011;271(1-2):37–53."/>
  <meta name="citation_reference" content="Luczak A, Bartho P, Harris KD. Gating of sensory input by spontaneous cortical activity. Journal of Neuroscience. 2013;33(4):1684–1695."/>
  <meta name="citation_reference" content="Churchland MM, Cunningham JP, Kaufman MT, Foster JD, Nuyujukian P, Ryu SI, et al. Neural population dynamics during reaching. Nature. 2012;487(7405):51–56."/>
  <meta name="citation_reference" content="Rigotti M, Barak O, Warden MR, Wang XJ, Daw ND, Miller EK, et al. The importance of mixed selectivity in complex cognitive tasks. Nature. 2013;497(7451):585–590."/>
  <meta name="citation_reference" content="Ni AM, Ruff DA, Alberts JJ, Symmonds J, Cohen MR. Learning and attention reveal a general relationship between population activity and behavior. Science. 2018;359(6374):463–465."/>
  <meta name="citation_reference" content="Kafashan M, Jaffe AW, Chettih SN, Nogueira R, Arandia-Romero I, Harvey CD, et al. Scaling of sensory information in large neural populations shows signatures of information-limiting correlations. Nature communications. 2021;12(1):1–16."/>
  <meta name="citation_reference" content="Engel TA, Wang XJ. Same or different? A neural circuit mechanism of similarity-based pattern match decision making. Journal of Neuroscience. 2011;31(19):6982–6996."/>
  <meta name="citation_reference" content="Muckli L, De Martino F, Vizioli L, Petro LS, Smith FW, Ugurbil K, et al. Contextual feedback to superficial layers of V1. Current Biology. 2015;25(20):2690–2695."/>
  <meta name="citation_reference" content="Zipser K, Lamme VA, Schiller PH. Contextual modulation in primary visual cortex. Journal of Neuroscience. 1996;16(22):7376–7389."/>
  <meta name="citation_reference" content="Juan CH, Walsh V. Feedback to V1: a reverse hierarchy in vision. Experimental brain research. 2003;150(2):259–263."/>
  <meta name="citation_reference" content="Spoerer CJ, Kietzmann TC, Mehrer J, Charest I, Kriegeskorte N. Recurrent neural networks can explain flexible trading of speed and accuracy in biological vision. PLoS computational biology. 2020;16(10):e1008215."/>
  <meta name="citation_reference" content="Silvanto J, Cowey A, Lavie N, Walsh V. Striate cortex (V1) activity gates awareness of motion. Nature neuroscience. 2005;8(2):143–144."/>
  <meta name="citation_reference" content="Hupé J, James A, Payne B, Lomber S, Girard P, Bullier J. Cortical feedback improves discrimination between figure and background by V1, V2 and V3 neurons. Nature. 1998;394(6695):784–787."/>
  <meta name="citation_reference" content="Panzeri S, Harvey CD, Piasini E, Latham PE, Fellin T. Cracking the neural code for sensory perception by combining statistics, intervention, and behavior. Neuron. 2017;93(3):491–507."/>
  <meta name="citation_reference" content="Zhao Y, Yates JL, Levi AJ, Huk AC, Park IM. Stimulus-choice (mis) alignment in primate area MT. PLoS computational biology. 2020;16(5):e1007614."/>
  <meta name="citation_reference" content="Vapnik V, Izmailov R. Rethinking statistical learning theory: learning using statistical invariants. Machine Learning. 2019;108(3):381–423."/>
  <meta name="citation_reference" content="Blainey P, Krzywinski M, Altman N. Points of significance: replication. Nature methods. 2014;11(9):879."/>
  <meta name="citation_reference" content="Vapnik V. The nature of statistical learning theory. Springer science & business media; 1999."/>
  <meta name="citation_reference" content="Koren V. Uncovering structured responses of neural populations recorded from macaque monkeys with linear support vector machines. STAR protocols. 2021;2(3):100746."/>
  <meta name="citation_reference" content="Koren V, Andrei AR, Hu M, Dragoi V, Obermayer K. Pairwise Synchrony and Correlations Depend on the Structure of the Population Code in Visual Cortex. Cell Reports. 2020;33(6):108367."/>
  <meta name="citation_reference" content="Compte A, Constantinidis C, Tegnér J, Raghavachari S, Chafee MV, Goldman-Rakic PS, et al. Temporally irregular mnemonic persistent activity in prefrontal neurons of monkeys during a delayed response task. Journal of neurophysiology. 2003;90(5):3441–3454."/>
  <meta name="citation_reference" content="Pesaran B, Pezaris JS, Sahani M, Mitra PP, Andersen RA. Temporal structure in neuronal activity during working memory in macaque parietal cortex. Nature neuroscience. 2002;5(8):805–811."/>
  <meta name="citation_reference" content="Hansen BJ, Chelaru MI, Dragoi V. Correlated variability in laminar cortical circuits. Neuron. 2012;76(3):590–602."/>
  <meta name="citation_reference" content="Koren V, Andrei AR, Hu M, Dragoi V, Obermayer K. Reading-out task variables as a low-dimensional reconstruction of neural spike trains in single trials. PloS one. 2019;14(10):e0222649."/>
  <meta name="citation_reference" content="Bair W, Zohary E, Newsome WT. Correlated firing in macaque visual area MT: time scales and relationship to behavior. Journal of Neuroscience. 2001;21(5):1676–1697."/>
  <meta name="citation_reference" content="Shahidi N, Andrei AR, Hu M, Dragoi V. High-order coordination of cortical spiking activity modulates perceptual accuracy. Nature neuroscience. 2019;22(7):1148–1158."/>
  <meta name="citation_reference" content="Belousov A, Verzakov S, Von Frese J. A flexible classification approach with optimal generalisation performance: support vector machines. Chemometrics and intelligent laboratory systems. 2002;64(1):15–25."/>
  <meta name="citation_reference" content="Cohen MR, Maunsell JH. Attention improves performance primarily by reducing interneuronal correlations. Nature neuroscience. 2009;12(12):1594."/>
  <meta name="citation_reference" content="Brunton BW, Botvinick MM, Brody CD. Rats and humans can optimally accumulate evidence for decision-making. Science. 2013;340(6128):95–98."/>
  <meta name="citation_reference" content="Beck JM, Ma WJ, Pitkow X, Latham PE, Pouget A. Not noisy, just wrong: the role of suboptimal inference in behavioral variability. Neuron. 2012;74(1):30–39."/>
  <meta name="citation_reference" content="Callaway EM. Feedforward, feedback and inhibitory connections in primate visual cortex. Neural Networks. 2004;17(5-6):625–632."/>
  <meta name="citation_reference" content="Averbeck BB, Latham PE, Pouget A. Neural correlations, population coding and computation. Nature reviews neuroscience. 2006;7(5):358–366."/>
  <meta name="citation_reference" content="Chettih SN, Harvey CD. Single-neuron perturbations reveal feature-specific competition in V1. Nature. 2019;567(7748):334–340."/>
  <meta name="citation_reference" content="Valente M, Pica G, Bondanelli G, Moroni M, Runyan CA, Morcos AS, et al. Correlations enhance the behavioral readout of neural population activity in association cortex. Nature neuroscience. 2021;24(7):975–986."/>
  <meta name="citation_reference" content="Payeur A, Guerguiev J, Zenke F, Richards BA, Naud R. Burst-dependent synaptic plasticity can coordinate learning in hierarchical circuits. Nature neuroscience. 2021;24(7):1010–1019."/>
  <meta name="citation_reference" content="Felleman DJ, Van Essen DC. Distributed hierarchical processing in the primate cerebral cortex. Cerebral cortex (New York, NY: 1991). 1991;1(1):1–47."/>
  <meta name="citation_reference" content="Pica G, Piasini E, Safaai H, Runyan C, Harvey C, Diamond M, et al. Quantifying how much sensory information in a neural code is relevant for behavior. Advances in Neural Information Processing Systems. 2017;30."/>
  <meta name="citation_reference" content="Steinmetz NA, Zatka-Haas P, Carandini M, Harris KD. Distributed coding of choice, action and engagement across the mouse brain. Nature. 2019;576(7786):266–273."/>
  <meta name="citation_reference" content="Zylberberg J, Pouget A, Latham PE, Shea-Brown E. Robust information propagation through noisy neural circuits. PLoS computational biology. 2017;13(4):e1005497."/>
  <meta name="citation_reference" content="Bondy AG, Haefner RM, Cumming BG. Feedback determines the structure of correlated variability in primary visual cortex. Nature neuroscience. 2018;21(4):598–606."/>
  <meta name="citation_reference" content="Ren SQ, Li Z, Lin S, Bergami M, Shi SH. Precise Long-Range Microcircuit-to-Microcircuit Communication Connects the Frontal and Sensory Cortices in the Mammalian Brain. Neuron. 2019;104(2):385–401."/>
  <meta name="citation_reference" content="Onorato I, Neuenschwander S, Hoy J, Lima B, Rocha KS, Broggini AC, et al. A distinct class of bursting neurons with strong gamma synchronization and stimulus selectivity in monkey V1. Neuron. 2020;105(1):180–197."/>
  <meta name="citation_reference" content="Larkum M. A cellular mechanism for cortical associations: an organizing principle for the cerebral cortex. Trends in neurosciences. 2013;36(3):141–151."/>
  <meta name="citation_reference" content="Naud R, Sprekeler H. Sparse bursts optimize information transmission in a multiplexed neural code. Proceedings of the National Academy of Sciences. 2018;115(27):E6329–E6338."/>
  <meta name="twitter:title" content="Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Animals frequently make decisions based on sensory cues. In such a setting, the overlap in the information on the stimulus and on the choice is crucial for the formation of informed behavioral decisions. Yet, how the information on the stimulus and on the choice interact in the brain is poorly understood. Here, we study the representation of a binary decision variable in the primary visual cortex (V1) while macaque monkeys perform delayed match-to-sample task on naturalistic visual stimuli close to psychophysical threshold. Using population vectors, we demonstrate the overlap in decoding spaces on binary stimulus classes “match/non-match” and binary choices “same /different” of the animal. Leveraging this overlap, we use learning from the invariant information across the two classification problems to predict the choice of the animal as a time-dependent population signal. We show the importance of the across-neuron organization and the temporal structure of spike trains for the decision signal and suggest how noise correlations between neurons with similar decoding selectivity are helpful for the accumulation of the decision signal. Finally, we show that decision signal is primarily carried by bursting neurons in the superficial layers of the cortex.

Author summary V1 is necessary for normal visual processing and is known to process features of visual stimuli such as orientation, but whether V1 also encodes behavioral decisions is an unresolved issue, with conflicting evidence. Here, we demonstrate that V1 encodes a mixed variable that contains the information about the stimulus as well as about the choice. We learn the structure of population responses in trials pertaining to the variable “stimulus+choice”, and apply the resulting population vectors to trials that differ only about the choice of the animal, but not about the stimulus class. Moreover, we learn structure of population responses on time-averaged data and then apply it on time-dependent (spiking) data. During the late phase of the trial, this procedure allows to predict the upcoming choice of the animal with a time-dependent population signal. The spiking signal of small neural population is sparse, and we hypothesize that positive correlations between neurons in the same decoding pool help the transmission of the decision-related information downstream. We find that noise correlations in the same decoding pool are significantly stronger than across coding pools, which corroborates our hypothesis on the benefit of noise correlations for the read-out of a time-dependent population signal.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Animals frequently make decisions based on sensory cues. In such a setting, the overlap in the information on the stimulus and on the choice is crucial for the formation of informed behavioral decisions. Yet, how the information on the stimulus and on the choice interact in the brain is poorly understood. Here, we study the representation of a binary decision variable in the primary visual cortex (V1) while macaque monkeys perform delayed match-to-sample task on naturalistic visual stimuli close to psychophysical threshold. Using population vectors, we demonstrate the overlap in decoding spaces on binary stimulus classes “match/non-match” and binary choices “same /different” of the animal. Leveraging this overlap, we use learning from the invariant information across the two classification problems to predict the choice of the animal as a time-dependent population signal. We show the importance of the across-neuron organization and the temporal structure of spike trains for the decision signal and suggest how noise correlations between neurons with similar decoding selectivity are helpful for the accumulation of the decision signal. Finally, we show that decision signal is primarily carried by bursting neurons in the superficial layers of the cortex.

Author summary V1 is necessary for normal visual processing and is known to process features of visual stimuli such as orientation, but whether V1 also encodes behavioral decisions is an unresolved issue, with conflicting evidence. Here, we demonstrate that V1 encodes a mixed variable that contains the information about the stimulus as well as about the choice. We learn the structure of population responses in trials pertaining to the variable “stimulus+choice”, and apply the resulting population vectors to trials that differ only about the choice of the animal, but not about the stimulus class. Moreover, we learn structure of population responses on time-averaged data and then apply it on time-dependent (spiking) data. During the late phase of the trial, this procedure allows to predict the upcoming choice of the animal with a time-dependent population signal. The spiking signal of small neural population is sparse, and we hypothesize that positive correlations between neurons in the same decoding pool help the transmission of the decision-related information downstream. We find that noise correlations in the same decoding pool are significantly stronger than across coding pools, which corroborates our hypothesis on the benefit of noise correlations for the read-out of a time-dependent population signal.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-06-14"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2594161"/>
  <title>
   Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1 | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"T6w7DxCTy5neEE5QlSLYEslGVQ7Zo40EI2mDBtxtxg0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.01.10.901504v6"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.01.10.901504v6"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.01.10.901504v6"}],"nid":"2594161","apath":"\/biorxiv\/early\/2022\/06\/14\/2020.01.10.901504.atom","pisa":"biorxiv;2020.01.10.901504v6","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.01.10.901504v6","title":"Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1","identifier":"node\/2594161"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MsJPJ\\\\\\\/GvefnPkruyxwZJxnyWJSGQyzRSc5JYt9GAaLctsNCBKpdzd4ocmMcohIMEZq6yvOUs7K8yw0wDmjBA9rWGwtzw8E0fzGvk4fvHGCs5hj9NMOjua9Ht5rw7rc6x1kYiBrTEVm7uJ9Ry6K+NB1e3YLgRhEaFYs1aINTSx9yPadrzfkWadFWABHk2S3AR9pVFn4SJdn3hxOPM1kr7pR0dHxd8INLoQ\\\\\\\/g+FolbV3+buROktaXMZlMN5GjPOvkBQO7esIaIPH2Dox\\\\\\\/Mh0nbPGCKqAXAEKfe0LdW1Vuav5P6bOsEr1vfs8SX5Qg\\\\\\\/cJz1D69s0HeEHhxEb5XqOzdIPf6d\\\\\\\/L5XJvZhebfzKcuCEFxrdik1C05YvcV98OyB3M2Z5LTWUp1h9Kr71tGfkfh6k40h9q2Rw6Y2GI8NX6tPYO+1OE3Q\\\\\\\/nGaruHWmIqEOvDOThB4xDoODBv2XCQ1GMPY\\\\\\\/x6EVF\\\\\\\/c2IJ8fReZzlGpqDki0WmyBeZp7yKu42t4Cv+ClucTGafYOarSN7phU6t4PI5M\\\\\\\/XEM\\\\\\\/NpuYRPC6ClIb4yy1ZvInP+PkSg+iPsCcc5d9LDfUAHAR4DasOoGLMdmYBH+AxHGiqDHHcO0djB5M85VKxvdgpNphXa86idtzMT85O0HKQLq2VYeasQzpcQwIbEEiTiyKvoTNVumiRmhe+U1mIpGOokS009TmpVufq6b9MUH5rNHPvWkjwQbvinfvheYMOT74KkuWex2tw5ed1sy\\\\\\\/aDkEz6miHxU9ZORPa8RC3cDxLgvCgxWPHuu8h87pxoEedYhykU\\\\\\\/20keNtSGsUJYLQC3l1HGIWT+93EI0VUvdZNO2dPgrlzItAGn0k+PxwaQ9yeN95t+Sv9JUNWV1c3Ooji0Buql68Hhjvb2kkWomDbUris7EFxnApbE4bj26emypFBORoBCbC2RnyN8Z1xW5GC2JM2lbv3nPCwxTd1OUHtKA6lxkuqTf86YxEMQ\\\\\\\/5Ln+3i2al\\\\\\\/KsUMrXyPWihweC8r36OhFyxkcQIa+\\\\\\\/OXZHcMEI7koGjc2n3CS7d9QOdd3SE0drx4he1chqEdKIaiL5XbDaFmQXB4+AqsIjcMxOA9moVobMySNwTWPBZp0KCh9qGFG45DOqB8N\\\\\\\/bxlbPehLsrFIkI5bwfyBksp2kdcYlLwD6Qh7gO+\\\\\\\/b+cke9lJu8K6aXO+rYQKqQMGhykzzjLj2x4LB9QxhZtQ5QlN7LD+peeU01LJSgZ\\\\\\\/d+4ZAL\\\\\\\/DCgjCgvgsNuklU1heDXpEQdM42RZgaanZFQ+8XsZ5ZzrxvRXvGL\\\\\\\/sviWttTGjexNQ0iYQlQ1W8g\\\\\\\/O1HwDCYWowCI7JXDuM6EZDQADyyzwYSmyTrbr1IJU8uk6ITF6sld2UeXkWFMcrYA3UTzCa4MYYo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022MMecjHGtEGKLuRXdIRy6DQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022902f1ff4908e6232901051878e0f85e91d4d981811235fcd566da2e178bbe8fd\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.01.10.901504v6.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2594161 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.01.10.901504v6.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2082702830">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2082702830" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1077551937" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-AV5KSpt4tybwZ-0V1pyZQleiPTIqtoW68MnYaW9ErjM"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2594161" data-node-nid="2594161" id="node-2594161--21952229423" data-pisa="biorxiv;2020.01.10.901504v6" data-pisa-master="biorxiv;2020.01.10.901504" data-apath="/biorxiv/early/2022/06/14/2020.01.10.901504.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-2920-2717" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Veronika
                      </span>
                      <span class="nlm-surname">
                       Koren
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-2152-2580" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ariana R.
                      </span>
                      <span class="nlm-surname">
                       Andrei
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Ming
                      </span>
                      <span class="nlm-surname">
                       Hu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-9526-0926" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Valentin
                      </span>
                      <span class="nlm-surname">
                       Dragoi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0002-0471-8170" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Klaus
                      </span>
                      <span class="nlm-surname">
                       Obermayer
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.01.10.901504
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2594161--21952229423" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Veronika Koren
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Software Engineering and Theoretical Computer Science, Technische Universität Berlin
                       </span>
                       , Berlin,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Bernstein Center for Computational Neuroscience Berlin
                       </span>
                       ,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Veronika%2BKoren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Koren%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AVeronika%2BKoren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2920-2717" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Veronika Koren
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Ariana R. Andrei
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Neurobiology and Anatomy, University of Texas Medical School
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        United States
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ariana%2BR.%2BAndrei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Andrei%20AR&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAriana%2BR.%2BAndrei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2152-2580" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ariana R. Andrei
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Ming Hu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Neurobiology and Anatomy, University of Texas Medical School
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        United States
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ming%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hu%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMing%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Valentin Dragoi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Neurobiology and Anatomy, University of Texas Medical School
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        United States
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Valentin%2BDragoi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dragoi%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AValentin%2BDragoi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-9526-0926" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Valentin Dragoi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Klaus Obermayer
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institute of Software Engineering and Theoretical Computer Science, Technische Universität Berlin
                       </span>
                       , Berlin,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Bernstein Center for Computational Neuroscience Berlin
                       </span>
                       ,
                       <span class="nlm-country">
                        Germany
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Klaus%2BObermayer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Obermayer%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKlaus%2BObermayer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0471-8170" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Klaus Obermayer
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.01.10.901504v6" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.10.901504v6.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.10.901504v6.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.10.901504v6.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.01.10.901504v6.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.01.10.901504v6.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2594161" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2594161/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-5">
                             Animals frequently make decisions based on sensory cues. In such a setting, the overlap in the information on the stimulus and on the choice is crucial for the formation of informed behavioral decisions. Yet, how the information on the stimulus and on the choice interact in the brain is poorly understood. Here, we study the representation of a binary decision variable in the primary visual cortex (V1) while macaque monkeys perform delayed match-to-sample task on naturalistic visual stimuli close to psychophysical threshold. Using population vectors, we demonstrate the overlap in decoding spaces on binary stimulus classes “match/non-match” and binary choices “same /different” of the animal. Leveraging this overlap, we use learning from the invariant information across the two classification problems to predict the choice of the animal as a time-dependent population signal. We show the importance of the across-neuron organization and the temporal structure of spike trains for the decision signal and suggest how noise correlations between neurons with similar decoding selectivity are helpful for the accumulation of the decision signal. Finally, we show that decision signal is primarily carried by bursting neurons in the superficial layers of the cortex.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-6">
                              <strong>
                               Author summary
                              </strong>
                              V1 is necessary for normal visual processing and is known to process features of visual stimuli such as orientation, but whether V1 also encodes behavioral decisions is an unresolved issue, with conflicting evidence. Here, we demonstrate that V1 encodes a mixed variable that contains the information about the stimulus as well as about the choice. We learn the structure of population responses in trials pertaining to the variable “stimulus+choice”, and apply the resulting population vectors to trials that differ only about the choice of the animal, but not about the stimulus class. Moreover, we learn structure of population responses on time-averaged data and then apply it on time-dependent (spiking) data. During the late phase of the trial, this procedure allows to predict the upcoming choice of the animal with a time-dependent population signal. The spiking signal of small neural population is sparse, and we hypothesize that positive correlations between neurons in the same decoding pool help the transmission of the decision-related information downstream. We find that noise correlations in the same decoding pool are significantly stronger than across coding pools, which corroborates our hypothesis on the benefit of noise correlations for the read-out of a time-dependent population signal.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-9">
                             Animals frequently make decisions based on the observation of the stimuli in their environment. Decision-making based on sensory information is traditionally studied with the measure of choice probability (CP) [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ]. The CP measures the covariation between neural activity and the upcoming choice behavior of the animal. As we observe spike trains of single neurons, the CP is computed from distributions of spike counts that lead to a particular choice. If distributions are far apart, the choice can be predicted with high probability while highly overlapping distributions of spike counts give weak or non-significant CP. Weak but significantly better than chance CPs have been reported in a variety of sensory areas [
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ] including V2 [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ]. The evidence on CP in V1 is disputed, since some studies found no evidence for CP in V1 [
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ], but the same authors subsequently found significant CPs in V1 during a coarse orientation-discrimination task with artificial stimuli [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ], arguing that, in order to observe CP, neurons must be topologically organized with respect to the task feature. Weak CPs in V1 have been reported by other studies [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ], while robust choice signals have been reported in V1 when decoding from Local Field Potentials in a coarse orientation difference task with drifting gratings [
                             <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ], as well as from spike counts of single neurons [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ], together demonstrating that the activity of neural populations in V1 does carry the information about the choice of the animal.
                            </p>
                            <p id="p-10">
                             The hypothesis behind the CP framework is that it captures the choice related sensory evidence. As it is measured in single neurons, what informs the CP is the change in the spike count of the neuron, conditioned on the choice of the animal. Sensory neurons are tuned to features of sensory stimuli and within the CP framework, pooling the activity across neurons with a particular tuning corresponds to collecting the evidence about the sensory variable that leads to a decision signal and allows the brain to generate an appropriate behavioral response [
                             <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. As a simple example with a binary stimulus orientation “vertical” and “horizontal” and a corresponding binary choice of the animal, an increase in the spike count of neurons with the preference for the vertical orientation increases the probability of animal’s decision for “vertical” compared to “horizontal” while an increase in the spike count of neurons with preferred orientation for “horizontal” increases the probability for the choice “horizontal” compared to the choice “vertical”. As such, the CP framework is tightly linked to the notion of tuning of single neurons to the stimulus feature that is informative for the choice, and there has to be a positive correlation between the activity of tuned neurons and the choice. The CP can be though of as a “vote” of a particular neuron that is always consistent with its tuning. A recent study, however, has questioned the CPs by reporting a negative correlation between the activity of V1 neurons and animal’s choice [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ], as neurons fired more vigorously when the animal chose the orientation that was the opposite to their preferred orientation. This result is in contradiction with the main hypothesis of the CP framework that predicts an increase in the spike count in neurons with preferred, and not with the anti-preferred orientation.
                            </p>
                            <p id="p-11">
                             The CP framework is based on the assumption that single neurons carry behaviorally relevant signals while a body of theoretical insights and data analyses suggest that behaviorally relevant signals in the cortex are encoded by neural populations rather than by single neurons. Recent years have seen the revival and development of the idea that behaviorally relevant variables are encoded by collective dynamics of neural ensembles [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             –
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ], and population codes seem to be insightful about the neural function in the auditory [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ,
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ], motor [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ], prefrontal [
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ], and visual cortex [
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ]. In particular, distributed population codes have been suggested by the analysis of large neural populations in V1 [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ]. Moreover, distributed population codes implemented as spiking neural networks have been demonstrated as optimal neural architectures by the theory [
                             <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ]. If cortical networks operate with (distributed) population codes, a single neuron measure of choice-related signals, such as the CP, might only capture a limited amount of information about the choice.
                            </p>
                            <p id="p-12">
                             Often, choice-related signals in sensory areas are studied in experimental paradigms where a sensory variable is changed in a continuous fashion, leading to a categorical decision variable with a limited number of options - typically two as the decision variable is binary. To better capture the relation between the representation of the stimulus and the choice in V1, we here impose a behavioral paradigm with a binary and categorical stimulus class instead of a continuous sensory variable. Previous work on decision-making with categorical variables suggested that such variables might be computed in a higher brain area, possibly through a comparison circuit [
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ], and represented in low-level areas as a contextual cue through a feedback signal areas [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             –
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ]. Such feedback signals have been shown to be precisely timed [
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ] with the timing of feedback signals being crucial for perception [
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ,
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]. Since choice signals in V1 are likely caused by top-down projections from higher-order brain areas [
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ], we reason that the neural representation of the choice is build in in real time to contribute to animal’s behavior as it unfolds. This evidence favors time-dependent models of decoding to measures that require averaging of spiking activity across time.
                            </p>
                            <p id="p-13">
                             In the context of decision-making based on sensory cues, encoding of the sensory information and reading-out of the sensory information that is relevant for the choice behavior are two distinct, but connected computational problems. In particular, to generate choice signals that are informed by sensory evidence, there has to be an intersection of information between the stimulus and the choice [
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ]. While the activity of higher cortical areas might carry partially independent signals about the stimuli and the choice in order to dissociate these two variables [
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ], it is unlikely that V1 can afford the representation of the choice that is independent of the stimulus. Here, we hypothesize that, in face of a binary stimulus class, V1 can learn to implement perhaps the simplest case of intersection between the stimulus and the choice. We suggest that V1 neurons modify their synapses with high-level cortical areas and learn a set of decoding weights to represent the stimulus class and the correct choice of the animal. Following this hypothesis, we learn population decoding weights in trials with correct choice, where there is the information about the stimulus class and about animal’s correct choice. We then utilize these decoding weights to compute the population signal in trials that only differ about the choice, but not about the stimulus. Such a generalized learning scheme is based on the idea of learning from invariants [
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              35
                             </a>
                             ], where intelligent agents are able to generalize the learned structure between distinct datasets with overlapping information, as to avoid the necessity to re-learn similar structures over and over. The generalized learning of decoding weights allows to predict the future choice of the animal with a time-dependent population read-out of spiking activity.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Materials and methods
                            </h2>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Data and code availability
                             </h3>
                             <p id="p-14">
                              The analysis was done with Matlab R2019 (Mathworks). Code is available in a public GitHub repository:
                              <a href="https://github.com/VeronikaKoren/transfer_learning">
                               https://github.com/VeronikaKoren/transfer_learning
                              </a>
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Experimental model and subject details
                             </h3>
                             <p id="p-15">
                              All experiments were conducted in accordance with protocols approved by The Animal Welfare Committee (AWC) and the Institutional Animal Care and Use Committee (IACUC) for McGovern Medical School at The University of Texas Health Science Center at Houston (UTHealth), and met or exceeded the standards proposed by the National Institutes of Health’s Guide for the Care and Use of Laboratory Animals.
                             </p>
                             <p id="p-16">
                              Two male rhesus macaques (Macaca mulatta; M1, 7 years old, 15kg; M2, 11 years old, 13kg) were used in this study. Animals were trained on a delayed match-to-sample task with visual stimuli. As the animal successfully fixated within the fixation area, this automatically triggered the trial. Two naturalistic images appeared consecutively, with a delay period in between. The target and the test stimulus were shown for 300 ms each, with a variable delay period in between (800 to 1000 ms). The identity of stimuli, complex naturalistic images depicting an outdoor scene, changed from one recording session to the other. Within the same session, the target and the test stimuli could be either the same (condition “match”) or else the test stimulus was rotated with respect to the target (condition “non-match”). The difference in orientation was such as to keep the performance stable at around 70 percent correct on non-matching stimuli, and ranged between 3 and 10 degrees. Animal subjects were trained to communicate their choice by holding the bar for the choice “different” and releasing the bar for “same”. For the trial to be valid, the behavioral response had to be communicated between 200 and 1200 ms after the offset of the test stimulus.
                             </p>
                             <p id="p-17">
                              Multi-unit signal was measured with laminar arrays comprising 16 channels with 100 μ– meter spacing between adjacent contacts. Electrodes captured neuronal activity across the cortical depth, from superficial to deep layers. The multi-unit signal was spike sorted and we analyzed all cells that responded to the stimulus with at least a 4-fold increase of the firing rate with respect to the baseline. The data was collected in 20 recording sessions, which gave 160 neurons. We analyzed 3 conditions, “correct match” (CM), “correct non-match” (CNM) and “incorrect non-match” (INM), where “correct/incorrect” refers to the behavioral performance and “match/non-match” refers to the stimulus class. In condition “incorrect match”, there were not enough trials to perform the analysis.
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              reports the summary statistics on the number of trials (biological replicates) in each condition.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2596685/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2596685/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2596685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <span class="caption-title">
                                The minimal, maximal, and average number of trials across recording sessions, for each condition.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Quantification and statistical analysis
                             </h3>
                             <div id="sec-7" class="subsection">
                              <h4>
                               Single neuron analysis
                              </h4>
                              <p id="p-18">
                               The spike train of a single neuron n in trial j is defined as a binary vector of zeros and ones,
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where n = 1,…, N is the neural index, k = 1,…, K, is the time index with step of 1 millisecond (ms), and j = 1,…, J
                               <sub>
                                1
                               </sub>
                               , J
                               <sub>
                                1
                               </sub>
                               + 1,…, J
                               <sub>
                                2
                               </sub>
                               , J
                               <sub>
                                2
                               </sub>
                               + 1,…J is the trial index. Trials were collected in conditions CM (j = 1,…, J
                               <sub>
                                1
                               </sub>
                               ), CNM (j = J
                               <sub>
                                1
                               </sub>
                               + 1,…, J
                               <sub>
                                2
                               </sub>
                               ) and INM (j = J
                               <sub>
                                2
                               </sub>
                               + 1,…, J).
                              </p>
                              <p id="p-19">
                               The sensitivity index of the neuron n to the non-match of stimuli is defined as follows:
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the average spike count during the presentation of the target (test) stimulus in condition CNM, and σ
                               <sub>
                                n
                               </sub>
                               is the standard deviation of spike counts comprising trials from both target and test. To estimate the significance of d’, we randomly permuted (without repetition) trial labels for “target” and “test” and computed d’ with randomly labeled data N
                               <sub>
                                perm
                               </sub>
                               = 1000 times. The p-value is then calculated by ranking the true d’ among the distribution of N
                               <sub>
                                perm
                               </sub>
                               values of d’ computed with permuted labels. Here, as well as throughout the paper, random permutations are technical replicates [
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                               ].
                              </p>
                             </div>
                             <div id="sec-8" class="subsection">
                              <h4>
                               Multivariate classification with Support Vector Machine
                              </h4>
                              <p id="p-20">
                               For multivariate classification, we use spike counts of neurons that were recorded in parallel. Spike counts were computed in the time window of [0, K] ms with respect to the onset of the stimulus (target or test),
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . For each neuron, spike counts were z-scored across trials:
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where 〈s
                               <sub>
                                nj
                               </sub>
                               〉
                               <sub>
                                j
                               </sub>
                               and Var
                               <sub>
                                j
                               </sub>
                               (s
                               <sub>
                                nj
                               </sub>
                               ) are the mean and the variance of spike counts for the neuron n across trials from conditions that are used for classification.
                              </p>
                              <p id="p-21">
                               Z-scored spike counts of N neurons recorded in parallel were utilized as input to the linear Support Vector Machine (SVM; [
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ]). Prediction accuracy of the classifier is computed as the balanced accuracy (BAC) on the hold-out test set:
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with TP, TN, FP and FN the number of true positive, true negative, false positive and false negative classifications, respectively. The performance measure of BAC corrects for the imbalance across conditions (see
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               ). The BAC is cross-validated N
                               <sup>
                                CV
                               </sup>
                               = 100-times and reported values are averaged across cross-validations. The significance of the balanced accuracy is evaluated with the permutation test, utilizing N
                               <sub>
                                perm
                               </sub>
                               = 1000 random permutations of class labels without repetition (see [
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                               ]).
                              </p>
                              <p id="p-22">
                               When the population is divided in groups (see below), the prediction accuracy for the group of interest is computed by utilizing all simultaneously recorded neurons and removing the information from neurons that do not belong to the group of interest (see [
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                               ]). While class labels for the group of interest are correct, they are randomly assigned for the remaining of neurons, making those neurons contribute only chance-level signal. The advantage of this method is to keep the dimensionality of the dataset intact, thus avoiding the possible confound of imbalanced subpopulations. The significance of the difference in BAC between groups is tested with t-test on 100 instances of the BAC with random permutation of the class label (without repetition). The use of the t-test is justified as the distribution of BAC across random permutations is normal.
                              </p>
                             </div>
                             <div id="sec-9" class="subsection">
                              <h4>
                               Estimation of the population vector
                              </h4>
                              <p id="p-23">
                               We estimate the population vector as the normalized vector of feature weights of the linear SVM. The population vector is computed using the activity of neurons recorded in parallel and refers to a specific binary classification problem.
                              </p>
                              <p id="p-24">
                               One sample for the classifier is the vector of z-scored spike counts (eq. 3) of N simultaneously recorded neurons in trial j,
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Linear SVM searches for an N – 1-dimensional plane (a hyperplane) that optimally separates data points [
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ] in conditions CNM and CM. The hyperplane is defined as
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <strong>
                                w
                               </strong>
                               is the vector of feature weights and b is the offset of the hyperplane from the origin. On each side of H
                               <sub>
                                0
                               </sub>
                               , we can define a hyperplane that verifies the following:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-25">
                               If the problem is linearly separable, all training samples verify the following the inequality:
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where y
                               <sub>
                                j
                               </sub>
                               ∈ { −1, 1} is the class label (y
                               <sub>
                                j
                               </sub>
                               = −1 in condition CNM and y
                               <sub>
                                j
                               </sub>
                               = 1 in condition CM). Training the linear SVM consists in maximizing the number of correctly classified samples and, at the same time, minimizing the distance between H
                               <sub>
                                1
                               </sub>
                               and H
                               <sub>
                                2
                               </sub>
                               , which can be expressed with the Lagrangian as follows:
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-26">
                               The first term on the right hand side of
                               <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                                Eq (8)
                               </a>
                               ensures the maximal distance between hyperplanes H
                               <sub>
                                1
                               </sub>
                               and H
                               <sub>
                                2
                               </sub>
                               , and the second term on the right ensures correct classification. To find the minimum of the Lagrangian with respect to the vector
                               <strong>
                                w
                               </strong>
                               , we take the derivative of the Lagrangian with respect to
                               <strong>
                                w
                               </strong>
                               and set it to zero. This gives the expression for the vector of feature weights:
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-27">
                               The vector of feature weights is normalized with the Euclidean (L
                               <sup>
                                2
                               </sup>
                               ) norm,
                               <span class="disp-formula" id="disp-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We refer to the vector
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as the population vector and to the n–th entry of
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ,
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , as the decoding weight of the neuron n.
                              </p>
                             </div>
                             <div id="sec-10" class="subsection">
                              <h4>
                               Comparison of population vectors across classification problems
                              </h4>
                              <p id="p-28">
                               We estimate the population vector separately for classification problems on stimulus + choice and on choice, getting vectors
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Classification of stimulus + choice utilizes trials from conditions CM and CNM, while classification of choice utilizes trials from conditions CNM and INM. The number of trials is imbalanced across conditions (namely, there are much less trials in condition INM compared to CM; see
                               <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               ), and such imbalance can affect the population vector. We balance the number of trials with the bootstrap method. In each recording session, we find the number of trials of the condition with most trials, and randomly sample, with repetition, the same number of trials from the two other distributions. The bootstrap method applies only to comparison of population vectors, where reported results are averaged across bootstraps.
                              </p>
                              <p id="p-29">
                               We measure the similarity of the population vectors
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               by computing the angle between them. From the dot product of vectors
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ,
                               <span class="disp-formula" id="disp-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where α is the angle between the vectors, we compute the angle α:
                               <span class="disp-formula" id="disp-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-30">
                               Notice that, since vectors are normalized, their Euclidean norm is unity,
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . If the vectors
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are similarly oriented, the angle between them is small. If, conversely, the orientation of two vectors is random between 0 and π, the angle between them is, on average, orthogonal (the average is across bootstrapped samples). The significance of the angle is evaluated with the permutation test. To construct the null model, we draw random vectors from the uniform distribution that have the same range and number of elements as the true population vectors, and compute the angle α
                               <sub>
                                p
                               </sub>
                               between the two random vectors. The p-value is computed by ranking the angle of true population vectors, α, among the distribution of N
                               <sub>
                                perm
                               </sub>
                               angles of random vectors, α
                               <sub>
                                p
                               </sub>
                               . The test is significant if the following is true:
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where the division with the number of tests implements the Bonferroni correction for multiple testing.
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               Population signal
                              </h4>
                              <p id="p-31">
                               Our decoding procedure comprises two steps, the estimation of the population vector (see the section “Estimation of the population vector”), and the computation of the population signal. These two steps implement an equivalent to the training and the validation step in a classical classification scheme. This is why the split of trials for the two steps is non-overlapping, such that no trials that have been used for the estimation of the population vector are used when computing the population signal.
                              </p>
                              <p id="p-32">
                               The population vectors are estimated using all trials in condition CM and half of the trials in condition CNM (trials with index
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ). The population signal is computed on a hold-out set, utilizing the remaining half of the trials from condition CNM and all trials from condition INM (trials with index
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ). The split of trials in training and validation set in condition CNM is cross-validated with Monte-Carlo method, using N
                               <sub>
                                CV
                               </sub>
                               = 100 random splits. All reported results are averaged across cross-validations.
                              </p>
                              <p id="p-33">
                               In the validation step, the population vector
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is utilized to compute the population signal as a weighted sum of spike trains. Consider the vector of spike trains of N simultaneously recorded neurons.
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-34">
                               The population signal at time is the dot product of the vector of spike trains at time t
                               <sub>
                                k
                               </sub>
                               with the population vector (the latter does not change over time),
                               <span class="disp-formula" id="disp-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the trial average, utilizing all trials of the validation set. By subtracting the trial average, we compute the deviation of the signal from the mean. Arguably, the deviation of the signal from its mean, rather than the absolute value of the signal, is a signal of biological relevance. As the transfer function, F(y(t
                               <sub>
                                k
                               </sub>
                               )), we use a convolution with an exponential filter,
                               <span class="disp-formula" id="disp-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with u(τ) = exp(−λτ), τ ∈ T, with support T ms. The convolution with an exponential filter models the causal effect of the presynaptic spike on the neural membrane of the read-out neuron. The population signal x
                               <sub>
                                j
                               </sub>
                               (t
                               <sub>
                                k
                               </sub>
                               ) is a time-resolved, low-dimensional representation of parallel spike trains in trial j.
                              </p>
                              <p id="p-35">
                               Next, we average the choice signal across trials, distinguishing conditions CNM (decision “different”) and INM (decision “same”),
                               <span class="disp-formula" id="disp-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and [z] is the ceiling function. We obtain two time-dependent decision signals as in
                               <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                                Eq 16
                               </a>
                               in every recording session. To have a more compact result, we compute the difference of the population signals in each recording session,
                               <span class="disp-formula" id="disp-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and average across sessions.
                              </p>
                              <p id="p-36">
                               To assess the significance of the difference signal Δx(t
                               <sub>
                                k
                               </sub>
                               ), we use the permutation test, based on random permutation of class labels. We compute N
                               <sub>
                                perm
                               </sub>
                               instances of the chance-level difference signal
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and compare the distribution of these values with the original difference signal Δx(t
                               <sub>
                                k
                               </sub>
                               ) in each timestep. When the difference signal of the true model, Δx(t
                               <sub>
                                k
                               </sub>
                               ), is outside of the distribution of results of the null model,
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for p = 1,…, N
                               <sub>
                                perm
                               </sub>
                               , we consider that signals for x
                               <sup>
                                same
                               </sup>
                               (t
                               <sub>
                                k
                               </sub>
                               ) and x
                               <sup>
                                different
                               </sup>
                               (t
                               <sub>
                                k
                               </sub>
                               ) are significantly different.
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Population signal with univariate decoding weights
                              </h4>
                              <p id="p-37">
                               To test decoding with univariate decoding weights, we replace population vectors from
                               <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                                Eq 9
                               </a>
                               –
                               <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">
                                10
                               </a>
                               with weights that are computed from the area under the ROC curve of single neurons. The area under the ROC curve (AUC, [
                               <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ]) is computed from distributions of spike counts of single neurons, distinguishing conditions CNM and CM. From the measure of AUC, we subtract the baseline,
                               <span class="inline-formula" id="inline-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , with AUC
                               <sub>
                                baseline
                               </sub>
                               = 0.5. The weight
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is positive if the neuron increased the spike count in condition CM compared to CNM, and negative if the neuron increased the spike count in condition CNM compared to CM. As we collect univariate decoding weights
                               <span class="inline-formula" id="inline-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               across neurons recorded on the same electrode, we get a vector
                               <span class="inline-formula" id="inline-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . As with multivariate vector of decoding weights (see
                               <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">
                                eq 10
                               </a>
                               ), we normalize the weight vector with the Euclidean norm,
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-38">
                               Normalized vector of univariate decoding weights
                               <span class="inline-formula" id="inline-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               can now replace the population vector
                               <span class="inline-formula" id="inline-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in
                               <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">
                                eq 14
                               </a>
                               and be used to compute the population signal. With normalization in
                               <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                                Eq 18
                               </a>
                               , the length of the vector
                               <span class="inline-formula" id="inline-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is unity, hence we can directly compare the population signals computed with multivariate and univariate decoding weights.
                              </p>
                             </div>
                             <div id="sec-13" class="subsection">
                              <h4>
                               Criteria for division in subpopulations
                              </h4>
                              <p id="p-39">
                               We separate simultaneously recorded neurons into subpopulation with respect to the following four criteria: 1) sign and 2) strength of decoding weight, 3) burstiness and 4) location in the cortical layer. For criteria that take into account the population vector
                               <span class="inline-formula" id="inline-formula-32">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (i.e. the sign and strength of decoding weight), we utilize trials pertaining to the variable stimulus + choice (conditions CM and CNM). For all criteria, we used the length of the time window of K = 400 ms.
                              </p>
                              <p id="p-40">
                               We separate the population into neurons with positive (
                               <span class="inline-formula" id="inline-formula-33">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-33.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-33.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ; plus neurons) and negative decoding weight (
                               <span class="inline-formula" id="inline-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-34.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-34.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ; minus neurons) simply by considering the sign of the decoding weight of each neuron.
                              </p>
                              <p id="p-41">
                               Neurons with strong and weak weight are distinguished by ranking the absolute value of decoding weight,
                               <span class="inline-formula" id="inline-formula-35">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-35.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-35.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , among the distribution of the same measure as decoding weights are computed with SVMs with permuted class labels. The strength of the weight of a single neuron n is a scalar, and the same result for models with permuted class labels is a distribution of N
                               <sub>
                                perm
                               </sub>
                               values, where N
                               <sub>
                                perm
                               </sub>
                               is the number of random permutations of class labels. If the strength of the weight of the neuron n is ranked within the first 25 % of weights from the null model, we assume that the neuron n has a strong weight, and we assume it has weak weight otherwise.
                              </p>
                              <p id="p-42">
                               We distinguish bursting and non-bursting neurons utilizing a criterion based on the power spectrum (PS) of spike trains. It has been shown that bursting neurons have decreased PS in low and middle frequency ranges [
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                               ]. We compute the power spectrum of spike trains for every single neuron, using multiplication of the spike train with 5 Slepian tapers to increase the reliability of the estimation [
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ]. Power spectra are divided with neuron’s average firing rate, which, in case the spike train is a homogeneous Poisson process, gives a constant power spectrum of 1. We then compute the mean power spectrum for frequencies between 10 and 200 Hz, discarding frequencies below 10 Hz due to the short time window K. As a reference, the mean PS of a homogeneous Poisson process is 1, while the mean PS of a bursting neuron is typically below 1. We determine whether a neuron is bursting or non-bursting with a permutation test. We randomly permute N
                               <sub>
                                perm
                               </sub>
                               -times, without repetition, the spike timing of the neuron, and compute the distribution of mean PS from these data. We then rank the mean PS of the true model model among the distribution of corresponding results from spike trains with permuted spike timing. The neuron n is classified as bursting if its mean PS ranks within the lowest 5 percent of equivalent results of spike trains with permuted time index. If the neuron does not fulfill this criterion, it is classified as non-bursting.
                              </p>
                              <p id="p-43">
                               Finally, we distinguish three cortical layers [
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ], the superficial (supragranular), the middle (granular) and the deep (infragranular) layer. The method for determining cortical layers utilizes the covariance matrix of the current source density and has been described previously (see [
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                               ]).
                              </p>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Choice-related signal in subpopulations
                              </h4>
                              <p id="p-44">
                               To gain better understanding of the information about the choice carried by subpopulations (as defined in the section “Criteria for division in subpopulations”), we compute population signals using only the information from a specific group of neurons that is of interest. The population signal of the group of interest is computed by removing the information about the choice in neurons that do not belong to the group of interest. This method ensures that the imbalance in the number of neuron across subpopulations does not influence the population signal. The information is removed by randomly permuting the class labels for “same” and “different” in the validation step, for neurons that do not belong to the group of interest. With permutation of the class label, the association between decoding weight of the neuron and the spike train is random, resulting in a signal that is close to zero at all times.
                              </p>
                              <p id="p-45">
                               As an example, we detail this method for the case of decision signals in neurons with positive and negative decoding weights, but the same methods is used to compute the decision signal for subpopulations of neurons with strong and weak weights, for bursting and non-bursting subpopulations and for subpopulations in cortical layers.
                              </p>
                              <p id="p-46">
                               The decision signal of plus neurons is computed with the spike train
                               <span class="inline-formula" id="inline-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where the label of the spike train is correct for the plus neurons, and random (i.e., correct or incorrect with equal probability) for the minus neurons,
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-47">
                               Next steps are as described previously (see
                               <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">
                                Eq 16
                               </a>
                               ). Iterating this procedure N
                               <sub>
                                perm
                               </sub>
                               –times, with p = 1,…, N
                               <sub>
                                perm
                               </sub>
                               random permutations of class labels for neurons that are not of interest, and then averaging across permutations, we get the population signal for plus neurons. Similarly, the population signal of minus neurons is computed by utilizing the spike train
                               <span class="inline-formula" id="inline-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where the label of the spike train is now correct for the minus neurons, and random for the plus neurons.
                              </p>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               Cross-correlation of population signals
                              </h4>
                              <p id="p-48">
                               The cross-correlation is used to measure the time-dependent relation of population signals between two subpopulations of simultaneously recorded neurons. Here we detail the procedure for computing the cross-correlation between population signals of plus and minus neurons, but the same procedure is used for bursting and non-bursting neurons and for neurons with weak and strong decoding weights.
                              </p>
                              <p id="p-49">
                               The cross-correlation between population signals of plus and minus neurons in trial j and for the permutation instance p (see the section “Choice-related signal in subpopulations”) is:
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with time lag τ = 1, 2,…, 2K – 1. The cross-correlation is normalized with the autocorrelation at zero time lag,
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where R
                               <sup>
                                ++
                               </sup>
                               (R
                               <sup>
                                −−
                               </sup>
                               ) is the autocorrelation function for plus (minus) neurons,
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-50">
                               The correlation function in
                               <a id="xref-disp-formula-21-1" class="xref-disp-formula" href="#disp-formula-21">
                                eq 21
                               </a>
                               is then averaged across trials and across permutations. Since there was no difference in the correlation across conditions, we used all trials from the validation set.
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-51">
                               The significance of the cross-correlation in
                               <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">
                                eq 23
                               </a>
                               is evaluated with the permutation test, where signals
                               <span class="inline-formula" id="inline-formula-38">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-38.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-38.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-39">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-39.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-39.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are computed with permutation of class labels. Additionally, we use random assignment to the group of plus and minus neurons in
                               <a id="xref-disp-formula-20-1" class="xref-disp-formula" href="#disp-formula-20">
                                Eq 20
                               </a>
                               .
                              </p>
                              <p id="p-52">
                               The cross-correlation between population signals in cortical layers is computed by taking into account the population signals from two layers at a time, i.e.,
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with (c
                               <sub>
                                1
                               </sub>
                               , c
                               <sub>
                                2
                               </sub>
                               ) ∈ {(SG, G), (SG, IG), (G, IG)}. The rest of the procedure is the same as in
                               <a id="xref-disp-formula-21-2" class="xref-disp-formula" href="#disp-formula-21">
                                Eq 21
                               </a>
                               –
                               <a id="xref-disp-formula-23-2" class="xref-disp-formula" href="#disp-formula-23">
                                23
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Noise correlation of spike timing
                              </h4>
                              <p id="p-53">
                               Let n, m ∈ {1,…, N} be fixed indices of two neurons, n = m. We define the spike trains of neurons n and m in trial j by f
                               <sub>
                                j
                               </sub>
                               (t
                               <sub>
                                k
                               </sub>
                               ):= o
                               <sub>
                                nj
                               </sub>
                               (t
                               <sub>
                                k
                               </sub>
                               ) and g
                               <sub>
                                j
                               </sub>
                               (t
                               <sub>
                                k
                               </sub>
                               ):= o
                               <sub>
                                mj
                               </sub>
                               (t
                               <sub>
                                k
                               </sub>
                               ). The cross-correlation measures the co-occurence of spikes as a function of time lag τ between the two spike trains
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-54">
                               To make the cross-correlation independent of firing rates, we normalize it with autocorrelation of neurons n and m at zero time lag:
                               <span class="disp-formula" id="disp-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the autocorrelation function is computed as follows:
                               <span class="disp-formula" id="disp-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-55">
                               We now subtract the correlation that is attributed to trial-invariant processes:
                               <span class="disp-formula" id="disp-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the trial-invariant correlation function is computed by randomly permuting the order of trials of the spike train, without repetition, of one of the neurons:
                               <span class="disp-formula" id="disp-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-40">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-40.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-40.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the average of N
                               <sub>
                                perm
                               </sub>
                               instances of random permutation. Finally, we sum the correlation function across the time lags [
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                               ]:
                               <span class="disp-formula" id="disp-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where τ’ is the maximal time lag.
                              </p>
                              <p id="p-56">
                               To compute noise correlations, we use spike trains at the resolution of experimental recordings (Δt =1 ms) and do not apply any further binning. When noise correlations are measured within a subpopulation of neurons, the entire procedure (
                               <a id="xref-disp-formula-25-1" class="xref-disp-formula" href="#disp-formula-25">
                                Eq 25
                               </a>
                               –
                               <a id="xref-disp-formula-29-1" class="xref-disp-formula" href="#disp-formula-29">
                                29
                               </a>
                               ) applies to neuronal pairs from the specific subpopulation. The subpopulations are defined based on decoding weights from the classification problem stimulus + choice, while the correlation function, is computed using trials corresponding to the variable choice.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Decoding spaces are similar between classification problems on stimulus + choice and choice
                             </h3>
                             <p id="p-57">
                              Two adult male macaques were trained on a delayed match-to-sample visual task. The subject visualized the target and the test stimuli, with a delay period in between (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig 1A
                              </a>
                              ). The stimuli were complex naturalistic images in black and white, depicting an outdoor scene, and their identity changed from one recording session to another [
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ]. The target and test stimulus were either identical (condition match) or not (condition non-match), with the only possible difference being the change in the orientation of the test stimulus. In condition non-match, the angle of rotation of the test stimulus was random (clockwise or anti-clockwise), close to psychophysical threshold (between 3 and 12 deg) and was kept fixed throughout the recording session. The task of the subject was to communicate its decision about the stimulus class (“same” or “different”). The multiunit signal was captured with linear arrays, inserted perpendicularly to the cortical surface, recording the activity of neuronal populations across the cortical depth (see methods). Example spike rasters for three simultaneously recorded neurons, belonging to the superficial (supragranular or SG), middle (granular or G) and deep (infragranular or IG) cortical layer, are shown on
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig 1C
                              </a>
                              . The difficulty of the task (i.e., the angle of rotation of the test stimulus) was calibrated during the experiment, as to keep the performance at 75% correct (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig 1C
                              </a>
                              ).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F1.large.jpg?width=800&height=600&carousel=1" title="Single neuron statistics, PSTH and multivariate classifier do not predict animal’s choice. A: Experimental paradigm. During the experimental trial, a target and a test image consecutively appear on the screen, interleaved with a delay period. B: Spike rasters of three neurons from the same recording session centered on the onset of the target (left) and the test stimulus. Neurons are located in the superficial cortical layer (magenta), middle layer (blue) deep layer (black). The window when the stimulus is is on is marked by the yellow region. C: Percent correct trials on non-matching stimuli. D: Sensitivity index (d’) of single neurons to non-match of the stimuli, sorted from the strongest to the weakest. The gray area shows 95 % of results with permuted labels for “target” and “test”. E: Scatter plot of the mean firing rate for the choice “different” (x-axis) and “same” (y-axis). R is the Pearson correlation coefficient. F: Left: CP of single neurons (black), and the distribution of CPs for models with permuted class labels (gray). Right: Distribution of CPs from the left. G: Population Peri-Stimulus time histogram for choices “same” (green) and “different” (blue), with the mean ± SEM for the variability across sessions. The yellow window indicates the presence of the stimulus and the black bar indicates the time window we typically used for the analysis. The red arrow marks the time of the zero crossing of the decision signal in Fig. 3E. H: Prediction accuracy of the linear SVM for the classification problem on stimulus + choice (orange) and on choice (black). Left plot shows results in recording sessions and right plot shows session-averaged results. Gray boxplots show the distribution of results of models with permuted class labels. Parameter for D-H: Time window of K = [0, 400] ms with respect to the onset of the test stimulus (D-F,H). out of 160 neurons had a significant CP (permutation test). Counting also the neurons with non-significant CP, there was an excess of neurons with CP > 0.5 (p = 0.0140, permutation test), however, seen the weakness of this effect, the population CP did not seem to be a strong predictor of animal’s decision. The population Peri-Stimulus Time Histogram also did not allow to discriminate animal’s choice (Fig 1G)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Single neuron statistics, PSTH and multivariate classifier do not predict animal’s choice.</span> A: Experimental paradigm. During the experimental trial, a target and a test image consecutively appear on the screen, interleaved with a delay period. B: Spike rasters of three neurons from the same recording session centered on the onset of the target (left) and the test stimulus. Neurons are located in the superficial cortical layer (magenta), middle layer (blue) deep layer (black). The window when the stimulus is is on is marked by the yellow region. C: Percent correct trials on non-matching stimuli. D: Sensitivity index (<em>d</em>’) of single neurons to non-match of the stimuli, sorted from the strongest to the weakest. The gray area shows 95 % of results with permuted labels for “target” and “test”. E: Scatter plot of the mean firing rate for the choice “different” (x-axis) and “same” (y-axis). <em>R</em> is the Pearson correlation coefficient. F: Left: CP of single neurons (black), and the distribution of CPs for models with permuted class labels (gray). Right: Distribution of CPs from the left. G: Population Peri-Stimulus time histogram for choices “same” (green) and “different” (blue), with the mean ± SEM for the variability across sessions. The yellow window indicates the presence of the stimulus and the black bar indicates the time window we typically used for the analysis. The red arrow marks the time of the zero crossing of the decision signal in Fig. 3E. H: Prediction accuracy of the linear SVM for the classification problem on stimulus + choice (orange) and on choice (black). Left plot shows results in recording sessions and right plot shows session-averaged results. Gray boxplots show the distribution of results of models with permuted class labels. Parameter for D-H: Time window of <em>K</em> = [0, 400] ms with respect to the onset of the test stimulus (D-F,H). out of 160 neurons had a significant CP (permutation test). Counting also the neurons with non-significant CP, there was an excess of neurons with <em>CP</em> > 0.5 (<em>p</em> = 0.0140, permutation test), however, seen the weakness of this effect, the population CP did not seem to be a strong predictor of animal’s decision. The population Peri-Stimulus Time Histogram also did not allow to discriminate animal’s choice (Fig 1G).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F1.medium.gif" width="440" height="395"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F1.medium.gif" width="440" height="395"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig 1.
                               </span>
                               <span class="caption-title">
                                Single neuron statistics, PSTH and multivariate classifier do not predict animal’s choice.
                               </span>
                               <p id="p-58" class="first-child">
                                A: Experimental paradigm. During the experimental trial, a target and a test image consecutively appear on the screen, interleaved with a delay period. B: Spike rasters of three neurons from the same recording session centered on the onset of the target (left) and the test stimulus. Neurons are located in the superficial cortical layer (magenta), middle layer (blue) deep layer (black). The window when the stimulus is is on is marked by the yellow region. C: Percent correct trials on non-matching stimuli. D: Sensitivity index (d’) of single neurons to non-match of the stimuli, sorted from the strongest to the weakest. The gray area shows 95 % of results with permuted labels for “target” and “test”. E: Scatter plot of the mean firing rate for the choice “different” (x-axis) and “same” (y-axis). R is the Pearson correlation coefficient. F: Left: CP of single neurons (black), and the distribution of CPs for models with permuted class labels (gray). Right: Distribution of CPs from the left. G: Population Peri-Stimulus time histogram for choices “same” (green) and “different” (blue), with the mean ± SEM for the variability across sessions. The yellow window indicates the presence of the stimulus and the black bar indicates the time window we typically used for the analysis. The red arrow marks the time of the zero crossing of the decision signal in
                                <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                 Fig. 3E
                                </a>
                                . H: Prediction accuracy of the linear SVM for the classification problem on stimulus + choice (orange) and on choice (black). Left plot shows results in recording sessions and right plot shows session-averaged results. Gray boxplots show the distribution of results of models with permuted class labels. Parameter for D-H: Time window of K = [0, 400] ms with respect to the onset of the test stimulus (D-F,H). out of 160 neurons had a significant CP (permutation test). Counting also the neurons with non-significant CP, there was an excess of neurons with CP > 0.5 (p = 0.0140, permutation test), however, seen the weakness of this effect, the population CP did not seem to be a strong predictor of animal’s decision. The population Peri-Stimulus Time Histogram also did not allow to discriminate animal’s choice (
                                <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                 Fig 1G
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-59">
                              Single neuron statistics and population averages are not predictive of the choice of the animal. Most single neurons were sensitive to the non-match of stimuli (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig 1D
                              </a>
                              ), with the sensitivity index d’ (see methods) of 133 out of 160 neurons being significant to the change in orientation between the target and the test stimulus (permutation tests with permutation of class labels; all permutation tests in the paper use N
                              <sub>
                               perm
                              </sub>
                              = 1000 permutations). Firing rates varied strongly across units, but not across the decision variable for the choices “same” vs. “different” (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig 1E
                              </a>
                              ). CPs of single neurons were both bigger and smaller than the baseline (CP
                              <sub>
                               baseline
                              </sub>
                              = 0.5), with an increase (CP > 0.5) as well as a decrease (CP < 0.5) in the firing rate with the choice “same” compared to the choice “different” (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig 1F
                              </a>
                              ). After correction for multiple testing, only 4 Patterns of spike counts across the population predict the variable stimulus + choice, but not the variable choice. We applied a linear Support Vector Machine (SVM), the classifier with optimal generalization performance [
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                              ], on spike counts of populations of simultaneously recorded neurons during the presentation of the test stimulus. Prediction of the variable choice was not significantly different from chance (average balanced accuracy (BAC) of BAC = 0.507; not significant; permutation test with random permutations of class labels). Variable stimulus + choice, meanwhile, is predicted with better than chance accuracy (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig 1H
                              </a>
                              ), demonstrating that patterns of spike counts are predictive of the choice when both choices are correct. The variable stimulus + choice, changes about the choice (“same”/”different”) as well as about the stimulus class (“match”/”non-match”; see
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ), and from current results, one is tempted to conclude that the activity of neural populations in V1 contains the information about the stimulus class, but not about animal’s decision. However, the prediction of the stimulus class without the information about the choice is significantly lower than the prediction of the stimulus that is accompanied with correct choice (p < 0.001, permutation test; BAC
                              <sup>
                               s+c
                              </sup>
                              = 0.557, BAC
                              <sup>
                               S
                              </sup>
                              = 0.536), suggesting that the decision variable nevertheless has a significant impact on the neural activity in V1. The variable choice might not be possible to predict with classical decoding paradigms, because it comes with a mixture of correct and incorrect behavior (
                              <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). When the choice is correct, we expect it to be mainly caused by correct inference on sensory information by the brain. Causes of an incorrect choice, however, can be many, for example, fluctuation of attention [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ], failure of transmission of the sensory information downstream [
                              <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ], noise in the feedforward input [
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                              ] or an error in the process of inference [
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              ]. The coexistence of different causes in the same data can result in a chance-level prediction. Nevertheless, the choice of the animal can still be a latent variable that influences neural activity.
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2596687/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2596687/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2596687" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2.
                               </span>
                               <p id="p-60" class="first-child">
                                Binary classification problems. The condition “incorrect match” has not been considered due to the insufficient number of trials.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-61">
                              If animal’s choice influences neural activity in the classification problem on stimulus + choice, we expect partially overlapping decoding spaces in the classification problems on stimulus + choice and on choice (see
                              <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ). Assuming a linear classification model, we use again the linear SVM to compare the orientation of linear separation boundaries (hyperplanes) across the two classification problems. Let us have N neurons observed simultaneously from which we collect data samples
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is a vector of z-scored spike counts of the N neurons in trial j (see methods). In general, the primal problem of the linear SVM can be expressed as follows:
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <strong>
                               w
                              </strong>
                              = [w
                              <sub>
                               1
                              </sub>
                              ,…,w
                              <sub>
                               N
                              </sub>
                              ]
                              <sup>
                               T
                              </sup>
                              is the vector of feature weights, λ
                              <sub>
                               j
                              </sub>
                              and y
                              <sub>
                               j
                              </sub>
                              ∈ { −1, 1} are the Lagrange multiplier and the binary label in trial j, respectively, and b is the offset of the separation boundary from the origin. The optimal solution to the primal problem is a linear separation boundary in the N-dimensional space of neural activities that is optimal for separating binary classes “correct match” and “correct non-match”. We are interested in the orientation of the separation boundary, since the offset is not relevant for our problem. The offset would be relevant in case of a global change in the firing rate with the decision variable, which is not the case here (see
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig 1E-G
                              </a>
                              ). The orientation of the separation boundary in the N–dimensional space of neural activities is fully determined by the vector of feature weights
                              <strong>
                               W
                              </strong>
                              , with the separating boundary being perpendicular to the vector of feature weights. The expression for the vector
                              <strong>
                               W
                              </strong>
                              is given by minimizing the primal problem in
                              <a id="xref-disp-formula-31-1" class="xref-disp-formula" href="#disp-formula-31">
                               Eq (31
                              </a>
                              with respect to the vector
                              <strong>
                               w
                              </strong>
                              . This gives:
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-62">
                              After normalizing the vector of feature weights with the Euclidean norm to keep the vectors in the same range across the recording sessions, we get the population vector
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              :
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-63">
                              Note that normalization in
                              <a id="xref-disp-formula-33-1" class="xref-disp-formula" href="#disp-formula-33">
                               Eq (33)
                              </a>
                              does not change the orientation of the vector in
                              <a id="xref-disp-formula-32-1" class="xref-disp-formula" href="#disp-formula-32">
                               Eq (32)
                              </a>
                              , but only normalizes its length. The n–th entry of the population vector
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ,
                              <span class="inline-formula" id="inline-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , is the decoding weight of the neuron n, that summarizes the contribution of the neuron n to the separation boundary. A strong vs. weak decoding weight
                              <span class="inline-formula" id="inline-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              imply a strong vs. weak contribution of a particular neuron to the classification boundary. For example, if a neuron increases its spike count with respect to the baseline equally likely across conditions “correct match” and “correct non-match”, the activity of such a neuron is not helpful for the classifier and its decoding weight will be close to 0, but a neuron that consistently changes its spike count across conditions and is useful for classification will have a strong decoding weight. The sign of the weight,
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , is informative about the increase/decrease of the spike count across conditions, and neurons with the same sign of the weight respond similarly to a particular condition.
                             </p>
                             <p id="p-64">
                              We estimate the similarity of decoding spaces in classification problems on stimulus + choice and on choice by computing the angle between the population vectors. We separately estimate population vectors for classification problems stimulus + choice and choice, getting population vectors
                              <span class="inline-formula" id="inline-formula-48">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-49">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in each recording session. The angle between population vectors (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig 2A
                              </a>
                              ) tells about the similarity of the orientation of separating boundaries in the two classification problems. If population vectors were independent, the angle between them would on average be orthogonal, α
                              <sub>
                               independent
                              </sub>
                              = π/2, as it is when we compute population vectors from data with randomly permuted class labels (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig 2B
                              </a>
                              , gray boxplots). Deviation of the average angle from orthogonality means that the separation boundaries are partially overlapping. We find that the average angle between the population vectors deviates significantly from orthogonality, consistently for different time windows (α = [80.2, 80.8, 80.0] deg for the length of the time window of K = [300, 400, 500] ms, respectively;
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig 2B
                              </a>
                              ) and significant in all cases (p < 0.001, in all cases, permutation tests).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F2.large.jpg?width=800&height=600&carousel=1" title="Similarity of decoding spaces in classification problems on stimulus + choice and on choice. A: Schema of the angle between population vectors in the context of stimulus + choice and choice. B: Average angle between population vectors (red cross) and the distribution of corresponding results by models with random weights (gray boxplots), for the length of the time window K = 300 (left), K = 400 (middle) and K = 500 ms (right). C: Top: Scatter plot of the absolute value of weights for the classification problem on stimulus + choice and on choice, utilizing the length of the time window of K = [300, 400, 500] ms. Bottom: Average correlation coefficient (green) for the data on the top. Gray boxplots show the distribution of corresponding results when decoding weights are computed with randomly permuted class labels. Parameter: Nperm = 1000" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Similarity of decoding spaces in classification problems on <em>stimulus</em> + <em>choice</em> and on <em>choice</em>.</span> A: Schema of the angle between population vectors in the context of <em>stimulus</em> + <em>choice</em> and <em>choice</em>. B: Average angle between population vectors (red cross) and the distribution of corresponding results by models with random weights (gray boxplots), for the length of the time window <em>K</em> = 300 (left), <em>K</em> = 400 (middle) and <em>K</em> = 500 ms (right). C: Top: Scatter plot of the absolute value of weights for the classification problem on <em>stimulus</em> + <em>choice</em> and on <em>choice</em>, utilizing the length of the time window of <em>K</em> = [300, 400, 500] ms. Bottom: Average correlation coefficient (green) for the data on the top. Gray boxplots show the distribution of corresponding results when decoding weights are computed with randomly permuted class labels. Parameter: <em>N</em><sub>perm</sub> = 1000</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F2.medium.gif" width="440" height="141"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F2.medium.gif" width="440" height="141"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 2.
                               </span>
                               <span class="caption-title">
                                Similarity of decoding spaces in classification problems on stimulus + choice and on choice.
                               </span>
                               <p id="p-65" class="first-child">
                                A: Schema of the angle between population vectors in the context of stimulus + choice and choice. B: Average angle between population vectors (red cross) and the distribution of corresponding results by models with random weights (gray boxplots), for the length of the time window K = 300 (left), K = 400 (middle) and K = 500 ms (right). C: Top: Scatter plot of the absolute value of weights for the classification problem on stimulus + choice and on choice, utilizing the length of the time window of K = [300, 400, 500] ms. Bottom: Average correlation coefficient (green) for the data on the top. Gray boxplots show the distribution of corresponding results when decoding weights are computed with randomly permuted class labels. Parameter: N
                                <sub>
                                 perm
                                </sub>
                                = 1000
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-66">
                              An alternative method to test the similarity of decoding spaces across the classification problems on stimulus + choice and on choice is to compare decoding weights of single neurons. To do so, we collect decoding weights across recording sessions, which gives two sets of decoding weights,
                              <span class="inline-formula" id="inline-formula-50">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-51">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , with neuron index n =1,…, N
                              <sub>
                               total
                              </sub>
                              (N
                              <sub>
                               total
                              </sub>
                              is the total number of neurons across recording sessions). We find a significant positive correlation (r = [0.13, 0.12, 0.13] across the two sets of weights, for the length of decoding window of K = [300, 400, 500] ms, respectively;
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig 2C
                              </a>
                              ). Positive correlation of decoding weights and similarity of population vectors across the two classification problems demonstrate similarity of decoding spaces in classification problems on stimulus + choice and on choice.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              Learning from invariants predicts the choice of the animal
                             </h3>
                             <p id="p-67">
                              After establishing the similarity of decoding spaces between classification problems on stimulus + choice and on choice, we exploit this similarity to learn from invariants and predict the variable choice as a time-resolved population signal. Learning from statistical invariants is a type of learning attributed to intelligent agents that goes beyond minimization of the expected error in a specific classification problem [
                              <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ]. In intelligent systems, we expect learning to generalize on yet unseen but related problems. Learning from invariants allows an intelligent system to generalize a learned structure across distinct classification problems that share invariant statistical structure. In our case, classification problems on stimulus + choice and on choice both vary about the choice of the animal (
                              <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ), and have overlapping decoding spaces (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig 2
                              </a>
                              ), hence, we hypothesize that learning on stimulus + choice could generalize over the variable choice. To implement this idea, we use decoding weights on the classification problem stimulus + choice (s+c),
                              <span class="inline-formula" id="inline-formula-52">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , to weights parallel spike trains in trials that differ about the variable choice (c):
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with t
                              <sub>
                               k
                              </sub>
                              the time index with resolution of the recorded spiking signal (Δt
                              <sub>
                               k
                              </sub>
                              = 1 ms), n = 1,…, N the neural index for neurons recorded in parallel, j the trial index and F(y) = [y * u](t) the convolution with an exponential filter u(τ) = exp – λτ. From the absolute signal
                              <span class="inline-formula" id="inline-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                              we subtract the running mean,
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where 〈x
                              <sub>
                               j
                              </sub>
                              (t
                              <sub>
                               k
                              </sub>
                              )〉
                              <sub>
                               j
                              </sub>
                              is the average across all trials. The time-dependent linear decoder in
                              <a id="xref-disp-formula-34-1" class="xref-disp-formula" href="#disp-formula-34">
                               Eq (34
                              </a>
                              –
                              <a id="xref-disp-formula-35-1" class="xref-disp-formula" href="#disp-formula-35">
                               35
                              </a>
                              ) models the synaptic current at a hypothetical read-out neuron, generated by the spiking activity of the observed neural population. The convolution models the filtering of synaptic inputs at the membrane of the read-out neuron, with (λ)
                              <sup>
                               -1
                              </sup>
                              the membrane time constant of the read-out neuron. Such a decoding procedure implements learning from invariants and differs from the regular training/validation scheme in two ways (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig 3A
                              </a>
                              ). First, we learned the decoding weights in the classification problem that minimizes predicted error on the variable stimulus + choice, while we validated it on the variable choice. These are two related (
                              <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ) but nevertheless distinct classification problems. Second, we learned decoding weights on spike counts while we apply them on spike trains (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3B
                              </a>
                              ). This is because decoding weights
                              <span class="inline-formula" id="inline-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-54.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-54.gif"/>
                                </noscript>
                               </span>
                              </span>
                              model changes in synaptic connections between observed neurons and the read-out neuron, a process that presumably takes place across days or weeks while the animal is training to perform the behavioral task. The signal about the choice, on the other hand, is presumably read-out by the brain in real time.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F3.large.jpg?width=800&height=600&carousel=1" title="Learning from invariants allows the prediction of the upcoming choice. A: Learning from invariants (schematic). In the learning phase, the classifier learns from samples of parallel spike trains (left). Following the regular validation scheme, the same type of data in the same conditions would be used for validating the predictive power of the model on a hold-out set (top right). Here, we test learning from invariants (bottom right), where the training and the test set differ in the time scale of the data (from spike counts in the training set to spike trains in the validation set) and the classification problems (stimulus + choice in the training set and choice in the validation set). B: Computation of time-dependent population signal by weighting spikes with decoding weights (schematic). Every spike causes a jump of the population signal, followed by an exponential decay towards 0. Spikes of neurons with positive weights push the signal up and spikes of neurons with negative weights pull the signal down. C: The population signal for the decision “same” (green) and “different” (blue) with learning from invariants. We show the mean ± SEM for the variability across recording sessions. Results are cross-validated with 100 cross-validations and the learning and validation set are non-overlapping (see methods). D: Session-averaged difference of population signals for the regular model (red) and the model with permuted class labels (gray). E: Difference of decision signals for three values of the parameter λ. The exponential kernel used for convolution is normalized such that the area under the kernel is 1. F: Same as in C, but replacing decoding weights from Eq 32 with centered and normalized area under the ROC curve. Parameters: λ-1 = 20 ms, time window for C is [0, 500] ms, time window for D-F is [0, 400] ms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Learning from invariants allows the prediction of the upcoming choice.</span> A: Learning from invariants (schematic). In the learning phase, the classifier learns from samples of parallel spike trains (left). Following the regular validation scheme, the same type of data in the same conditions would be used for validating the predictive power of the model on a hold-out set (top right). Here, we test learning from invariants (bottom right), where the training and the test set differ in the time scale of the data (from spike counts in the training set to spike trains in the validation set) and the classification problems (stimulus + choice in the training set and choice in the validation set). B: Computation of time-dependent population signal by weighting spikes with decoding weights (schematic). Every spike causes a jump of the population signal, followed by an exponential decay towards 0. Spikes of neurons with positive weights push the signal up and spikes of neurons with negative weights pull the signal down. C: The population signal for the decision “same” (green) and “different” (blue) with learning from invariants. We show the mean ± SEM for the variability across recording sessions. Results are cross-validated with 100 cross-validations and the learning and validation set are non-overlapping (see methods). D: Session-averaged difference of population signals for the regular model (red) and the model with permuted class labels (gray). E: Difference of decision signals for three values of the parameter <em>λ</em>. The exponential kernel used for convolution is normalized such that the area under the kernel is 1. F: Same as in C, but replacing decoding weights from Eq 32 with centered and normalized area under the ROC curve. Parameters: <em>λ</em><sup>-1</sup> = 20 ms, time window for C is [0, 500] ms, time window for D-F is [0, 400] ms.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F3.medium.gif" width="440" height="232"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F3.medium.gif" width="440" height="232"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 3.
                               </span>
                               <span class="caption-title">
                                Learning from invariants allows the prediction of the upcoming choice.
                               </span>
                               <p id="p-68" class="first-child">
                                A: Learning from invariants (schematic). In the learning phase, the classifier learns from samples of parallel spike trains (left). Following the regular validation scheme, the same type of data in the same conditions would be used for validating the predictive power of the model on a hold-out set (top right). Here, we test learning from invariants (bottom right), where the training and the test set differ in the time scale of the data (from spike counts in the training set to spike trains in the validation set) and the classification problems (stimulus + choice in the training set and choice in the validation set). B: Computation of time-dependent population signal by weighting spikes with decoding weights (schematic). Every spike causes a jump of the population signal, followed by an exponential decay towards 0. Spikes of neurons with positive weights push the signal up and spikes of neurons with negative weights pull the signal down. C: The population signal for the decision “same” (green) and “different” (blue) with learning from invariants. We show the mean ± SEM for the variability across recording sessions. Results are cross-validated with 100 cross-validations and the learning and validation set are non-overlapping (see methods). D: Session-averaged difference of population signals for the regular model (red) and the model with permuted class labels (gray). E: Difference of decision signals for three values of the parameter λ. The exponential kernel used for convolution is normalized such that the area under the kernel is 1. F: Same as in C, but replacing decoding weights from
                                <a id="xref-disp-formula-32-2" class="xref-disp-formula" href="#disp-formula-32">
                                 Eq 32
                                </a>
                                with centered and normalized area under the ROC curve. Parameters: λ
                                <sup>
                                 -1
                                </sup>
                                = 20 ms, time window for C is [0, 500] ms, time window for D-F is [0, 400] ms.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-69">
                              With learning from invariants, a time-dependent decision variable can be predicted during the majority of the trial. Averaging the population signals in
                              <a id="xref-disp-formula-35-2" class="xref-disp-formula" href="#disp-formula-35">
                               Eq 35
                              </a>
                              across trials that lead the same decision, we get distinct signals for animal’s choice “same” vs. “different” that persists even after 150 ms after the offset of the stimulus (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig 3C
                              </a>
                              ). Since the variable choice does not per se vary about the stimulus class (both conditions in the variable choice are with non-matching stimuli; see
                              <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ), the population signal computed in
                              <a id="xref-disp-formula-34-2" class="xref-disp-formula" href="#disp-formula-34">
                               Eq 34
                              </a>
                              –
                              <a id="xref-disp-formula-35-3" class="xref-disp-formula" href="#disp-formula-35">
                               35
                              </a>
                              reflects the information about the choice of the animal. To estimate the significance of the difference of the population signals pertaining to decisions “same” and “different”, we compute the difference of trial-averaged signals:
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and compare it to same results computed with permutations of class labels (see methods). The difference signal Δx(t
                              <sub>
                               k
                              </sub>
                              ) is significant during the majority of the trial (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig 3D
                              </a>
                              ), and the read-out of animal’s decision is stable starting from 140 ms after the onset of the stimulus (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig 3C-D
                              </a>
                              ). Feedforward currents are expected to drive neural activity at the stimulus onset, while top-down and lateral inputs are expected to dominate the neural activity during the late phase of the trial [
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ]. The timing of 140 ms after the stimulus onset coincides with the end of transient firing at the stimulus onset (
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Fig 1G
                              </a>
                              , red arrow), suggesting the relevance of top-down inputs, as opposed to bottom-up inputs, for the decision signal in the present setting. Note that the variable choice contains correct and incorrect behavior (
                              <a id="xref-table-wrap-2-7" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ), which is a likely explanation for unstable representation of the choice at the beginning of the trial. Properties of the time-dependent population signal decoded from the spiking activity are compatible with an internally generated decision variable resulting from a top-down projection from another cortical area.
                             </p>
                             <p id="p-70">
                              Prediction of animal’s choice is robust to the time window used for decoding (S1 Fig), the time constant of convolution (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig 3E
                              </a>
                              ), as well as the method for estimating decoding weights. As we replace decoding weights in
                              <a id="xref-disp-formula-31-2" class="xref-disp-formula" href="#disp-formula-31">
                               Eq 31
                              </a>
                              –
                              <a id="xref-disp-formula-33-2" class="xref-disp-formula" href="#disp-formula-33">
                               33
                              </a>
                              with centered and normalized Area Under the ROC curve that we estimated for each neuron (see methods) the choice of the animal can still be predicted, even though less accurately (
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Fig 3F
                              </a>
                              ). This shows that optimal decoding weights estimated with the linear SVM are not required to predict the choice. Learning on trials with correct behavior (i.e., stimulus + choice), however, is necessary for prediction of the choice. When the learning and the validation step both utilize the variable choice (instead of learning on stimulus + choice), the resulting signals are not significant (S2 Fig). As expected, prediction of the choice during the visualization of target stimulus is at chance (S3 Fig), because during the presentation of the target stimulus, the information necessary for discrimination of matching from non-matching stimuli is not yet available to the visual system.
                             </p>
                             <p id="p-71">
                              The correct sign of decoding weights,
                              <span class="inline-formula" id="inline-formula-55">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-55.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-55.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , is necessary for predicting animal’s decision. The sign as well as the amplitude of decoding weights determine the orientation of the population vector in the N–dimensional space of neural activities. However, flipping the sign of the weight typically causes a bigger change in orientation than perturbing the amplitude (or strength) of the weight (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig 4A
                              </a>
                              , left). To test the importance of the sign and the amplitude of decoding weights, we replace the correct sign and amplitude with values randomly drawn from the uniform distribution. As we randomize the sign of decoding weights, prediction of animal’s decision is no longer possible, while using random amplitude still allows the predict animal’s decision (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig 4B
                              </a>
                              ), even though the decision signal is now weaker with respect to the unperturbed signal. Further on, we compute the decision signals separately in subpopulations with positive and negative sign of decoding weight and estimate their time-lagged cross-correlation in the same trial (see methods). The decision signals of subpopulations with positive and negative weights are negatively correlated (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig 4C
                              </a>
                              ), with half-width at the trough value at 18 ms (using the membrane time constant of the read-out unit of λ
                              <sup>
                               -1
                              </sup>
                              = 20 ms). Negative correlation suggests competition between these subpopulations.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F4.large.jpg?width=800&height=600&carousel=1" title="Correct sign of decoding weights is necessary for prediction of the choice. A: Effect of changing the sign of a decoding weight (left) and the amplitude of a decoding weight (right) on the orientation of the population vector in the space of inputs of N=2 neurons (schematic). Both types of perturbation change the orientation of the population vector (red arrow), but flipping the sign typically induces bigger changes. B: The difference of decision signals computed with random sign of weights (top) and random amplitude of weights (bottom). The gray area marks the distribution of models with permuted class labels. Signals are scaled with respect to the unperturbed signal (in Fig 3C) at maximum deviation from 0 (211 ms after the stimulus onset). C: Top: Cross-correlation of decision signals between neurons with positive and negative weights, for different values of the inverse time constant of convolution λ). We show the mean ± SEM across recording sessions. Bottom: Session-averaged cross-correlation for the model with λ-1 =20 ms (red) and the distribution of results for models with permuted class labels (gray area). D: Same as in B, but using the information from neurons with strong weights (top) and with weak weights (bottom). E: Same as in C, but for subpopulations of neurons with strong and weak weights. Parameters (when applicable): λ-1 = 20 ms, Nperm = 1000, Δt = 1 ms." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Correct sign of decoding weights is necessary for prediction of the choice.</span> A: Effect of changing the sign of a decoding weight (left) and the amplitude of a decoding weight (right) on the orientation of the population vector in the space of inputs of N=2 neurons (schematic). Both types of perturbation change the orientation of the population vector (red arrow), but flipping the sign typically induces bigger changes. B: The difference of decision signals computed with random sign of weights (top) and random amplitude of weights (bottom). The gray area marks the distribution of models with permuted class labels. Signals are scaled with respect to the unperturbed signal (in Fig 3C) at maximum deviation from 0 (211 ms after the stimulus onset). C: Top: Cross-correlation of decision signals between neurons with positive and negative weights, for different values of the inverse time constant of convolution <em>λ</em>). We show the mean ± SEM across recording sessions. Bottom: Session-averaged cross-correlation for the model with <em>λ</em><sup>-1</sup> =20 ms (red) and the distribution of results for models with permuted class labels (gray area). D: Same as in B, but using the information from neurons with strong weights (top) and with weak weights (bottom). E: Same as in C, but for subpopulations of neurons with strong and weak weights. Parameters (when applicable): <em>λ</em><sup>-1</sup> = 20 ms, <em>N</em><sub>perm</sub> = 1000, Δ<em>t</em> = 1 ms.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F4.medium.gif" width="425" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F4.medium.gif" width="425" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 4.
                               </span>
                               <span class="caption-title">
                                Correct sign of decoding weights is necessary for prediction of the choice.
                               </span>
                               <p id="p-72" class="first-child">
                                A: Effect of changing the sign of a decoding weight (left) and the amplitude of a decoding weight (right) on the orientation of the population vector in the space of inputs of N=2 neurons (schematic). Both types of perturbation change the orientation of the population vector (red arrow), but flipping the sign typically induces bigger changes. B: The difference of decision signals computed with random sign of weights (top) and random amplitude of weights (bottom). The gray area marks the distribution of models with permuted class labels. Signals are scaled with respect to the unperturbed signal (in
                                <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                                 Fig 3C
                                </a>
                                ) at maximum deviation from 0 (211 ms after the stimulus onset). C: Top: Cross-correlation of decision signals between neurons with positive and negative weights, for different values of the inverse time constant of convolution λ). We show the mean ± SEM across recording sessions. Bottom: Session-averaged cross-correlation for the model with λ
                                <sup>
                                 -1
                                </sup>
                                =20 ms (red) and the distribution of results for models with permuted class labels (gray area). D: Same as in B, but using the information from neurons with strong weights (top) and with weak weights (bottom). E: Same as in C, but for subpopulations of neurons with strong and weak weights. Parameters (when applicable): λ
                                <sup>
                                 -1
                                </sup>
                                = 20 ms, N
                                <sub>
                                 perm
                                </sub>
                                = 1000, Δt = 1 ms.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-73">
                              The decision signal relies on the activity of a subpopulation of informative neurons. We split the population of simultaneously observed neurons in two groups with respect to the amplitude (or strength) of the decoding weight
                              <span class="inline-formula" id="inline-formula-56">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-56.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-56.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , distinguishing neurons with strong and weak decoding weights (see methods). The groups are approximately balanced, with 79 and 81 neurons with strong and weak weights, respectively (as we collect the weights across recording sessions). As it follows from the definition of the linear SVM (see
                              <a id="xref-disp-formula-31-3" class="xref-disp-formula" href="#disp-formula-31">
                               Eq 31
                              </a>
                              –
                              <a id="xref-disp-formula-32-3" class="xref-disp-formula" href="#disp-formula-32">
                               32
                              </a>
                              ), neurons with strong decoding weights are those that contribute the most to the population vector, while neurons with weak decoding weights contribute little or nothing. The subpopulation with strong decoding weights carries the decision signal, while the decision signal of the subpopulation with weak weights does not reach significance (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig 4D
                              </a>
                              ). The cross-correlation between decision signals of subpopulations with strong and weak weights is positive (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig 4E
                              </a>
                              ), with half-width of the peak value at 39 ms (using the membrane time constant of λ
                              <sup>
                               -1
                              </sup>
                              = 20 ms).
                             </p>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              Positive noise correlations between neurons from the same coding pool strengthen the time-dependent decision signal
                             </h3>
                             <p id="p-74">
                              An unresolved question in neuroscience is the role of noise correlations for neural coding. Positive noise correlations among neurons with similar tuning harm the quantity of information that a neural population transmits with a time-averaged signal [
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ]. In a population code based on time-averaged neural activity, one would therefore expect that noise correlations among neuronal pairs with similar tuning are weak, or at least weaker than noise correlations among pairs with different or opposite tuning. In contrast to this prediction of a theoretical model, positive interactions between signal and noise correlations in V1 have been observed in experiments [
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                              ]. Moreover, if positive correlations among neurons with similar selectivity were detrimental to the neural code, we would expect them to be weaker with correct compared to incorrect behavior. In contrast, stronger noise correlations [
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                              ] and across-neuron coordination [
                              <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ] were reported in correct compared to incorrect behavior. This evidence suggest that, in some cases at least, noise correlations among neurons with similar selectivity might be useful for the read-out. While across-neuron noise correlations might harm the transmission of information when reading-out from samples of time-averaged neural activity (i.e., parallel spike counts), this is not necessarily the case when reading out from a time-dependent signal (i.e., parallel spike trains). A spike train of a single neuron is a signal that is transient and sparse in time. The presynaptic spike has an impact on the membrane potential of the read-out neuron for the duration of one membrane time constant of the read-out neuron, which is around 10-20 ms. Due to the sparsity and the transient effect of spikes on the postsynaptic neurons, the beneficial relation between neuronal selectivity and noise correlations in a time-dependent read-out is reversed with respect to the time-averaged read-out.
                             </p>
                             <p id="p-75">
                              With a time-dependent read-out as in
                              <a id="xref-disp-formula-34-3" class="xref-disp-formula" href="#disp-formula-34">
                               Eq (34)
                              </a>
                              , positive noise correlations among neurons with similar selectivity and negative noise correlations between neurons with different selectivity strengthen the population signal and increase its impact at the read-out neuron(s). In our case with a binary classification problem, we therefore expect neuronal pairs with the same sign of decoding weight (same decoding pool) to have stronger noise correlations than pairs with the opposite sign of decoding weight (opposite decoding pool). This organization of noise correlations is indeed observed in the data (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig 5A,B
                              </a>
                              ). Then, due to normalization in
                              <a id="xref-disp-formula-35-4" class="xref-disp-formula" href="#disp-formula-35">
                               Eq (35)
                              </a>
                              where we subtracted the baseline of the neural signals, we get positive effective interaction between neurons from the same pool, and negative effective interaction across neurons from the opposite pools. Such spatio-temporal organization of spiking allows the population signal to accumulate over time and raise above the noisy background (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig 5C
                              </a>
                              , top), which improves detection of the population signal at the read-out neuron. If, on the contrary, neurons from the opposite pools fire in a random sequence, the population signal remains close to zero and can only convey noise to the read-out neuron (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig 5C
                              </a>
                              , bottom).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F5.large.jpg?width=800&height=600&carousel=1" title="Across-neurons and across-time organization of spike trains helps the transmission of the decision signal A: Noise correlation of spike timing for neuronal pairs with the same sign of the weight (within pools; orange) and with the opposite sign of the weight (across pools; ochre). Noise correlations are computed as the area under the cross-correlogram (rccg), and the x-axis is the maximal time lag used for computing the area. We show the mean rccg± SEM for the variability across pairs (Nwithin = 207,Nacross = 514). B: Difference of average noise correlation (rwithin – racross; green) and the distribution of corresponding results results with Nperm random permutations of labels for label {“within”, “across”}. C: Schematic of spike patterns that either help or hinder the accumulation of a time-resolved population signal. D: Same as in A, but showing noise correlations among neuronal pairs with strong weights (orange) and weak weights (black). E: Same as in B, but showing the difference of average noise correlation (rstrong – rweak; black) and the distribution of results for Nperm random permutations of labels for labels { “strong”, “weak” } in gray. F: Difference of decision signals with permuted spikes across neurons. We permute spikes such as to maintain the coarse organization in 2 pools of neurons with positive and negative sign of the weight (black), and across all neurons (ochre). The gray area is for the distribution of signals with permuted class labels (the null model). The difference signals is scaled with respect to the unperturbed signal at maximum deviation from 0 (i.e., the decision signal in Fig 3C at 211 ms after the stimulus onset). We permuted neuronal indices, randomly in each cross-validation cycle. G: Same as in F, showing decision signals with temporal jitter in the spike timing. H: Same as in F, showing the decision signal with permuted spike timing over the entire time window ([0,400] ms) or in two time windows before and after the zero crossing at 140 ms after the stimulus onset ([0,140],[141,400] ms). Parameters: time window for measuring correlations (A,B,D,E) and the effect of across-neurons structure is [140, 500] ms, time window for G and H is [0, 400] ms after the onset of the test stimulus; λ-1 = 20 ms, Nperm = 1000." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Across-neurons and across-time organization of spike trains helps the transmission of the decision signal</span> A: Noise correlation of spike timing for neuronal pairs with the same sign of the weight (within pools; orange) and with the opposite sign of the weight (across pools; ochre). Noise correlations are computed as the area under the cross-correlogram (<em>r<sup>ccg</sup></em>), and the x-axis is the maximal time lag used for computing the area. We show the mean <em>r<sup>ccg</sup></em>± SEM for the variability across pairs (<em>N</em><sup>within</sup> = 207,<em>N</em><sup>across</sup> = 514). B: Difference of average noise correlation (<em>r</em><sup>within</sup> – <em>r</em><sup>across</sup>; green) and the distribution of corresponding results results with <em>N</em><sub>perm</sub> random permutations of labels for label {“within”, “across”}. C: Schematic of spike patterns that either help or hinder the accumulation of a time-resolved population signal. D: Same as in A, but showing noise correlations among neuronal pairs with strong weights (orange) and weak weights (black). E: Same as in B, but showing the difference of average noise correlation (<em>r</em><sup>strong</sup> – <em>r</em><sup>weak</sup>; black) and the distribution of results for <em>N</em><sub>perm</sub> random permutations of labels for labels { “strong”, “weak” } in gray. F: Difference of decision signals with permuted spikes across neurons. We permute spikes such as to maintain the coarse organization in 2 pools of neurons with positive and negative sign of the weight (black), and across all neurons (ochre). The gray area is for the distribution of signals with permuted class labels (the null model). The difference signals is scaled with respect to the unperturbed signal at maximum deviation from 0 (i.e., the decision signal in Fig 3C at 211 ms after the stimulus onset). We permuted neuronal indices, randomly in each cross-validation cycle. G: Same as in F, showing decision signals with temporal jitter in the spike timing. H: Same as in F, showing the decision signal with permuted spike timing over the entire time window ([0,400] ms) or in two time windows before and after the zero crossing at 140 ms after the stimulus onset ([0,140],[141,400] ms). Parameters: time window for measuring correlations (A,B,D,E) and the effect of across-neurons structure is [140, 500] ms, time window for G and H is [0, 400] ms after the onset of the test stimulus; <em>λ</em><sup>-1</sup> = 20 ms, <em>N</em><sub>perm</sub> = 1000.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F5.medium.gif" width="380" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F5.medium.gif" width="380" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 5.
                               </span>
                               <span class="caption-title">
                                Across-neurons and across-time organization of spike trains helps the transmission of the decision signal
                               </span>
                               <p id="p-76" class="first-child">
                                A: Noise correlation of spike timing for neuronal pairs with the same sign of the weight (within pools; orange) and with the opposite sign of the weight (across pools; ochre). Noise correlations are computed as the area under the cross-correlogram (r
                                <sup>
                                 ccg
                                </sup>
                                ), and the x-axis is the maximal time lag used for computing the area. We show the mean r
                                <sup>
                                 ccg
                                </sup>
                                ± SEM for the variability across pairs (N
                                <sup>
                                 within
                                </sup>
                                = 207,N
                                <sup>
                                 across
                                </sup>
                                = 514). B: Difference of average noise correlation (r
                                <sup>
                                 within
                                </sup>
                                – r
                                <sup>
                                 across
                                </sup>
                                ; green) and the distribution of corresponding results results with N
                                <sub>
                                 perm
                                </sub>
                                random permutations of labels for label {“within”, “across”}. C: Schematic of spike patterns that either help or hinder the accumulation of a time-resolved population signal. D: Same as in A, but showing noise correlations among neuronal pairs with strong weights (orange) and weak weights (black). E: Same as in B, but showing the difference of average noise correlation (r
                                <sup>
                                 strong
                                </sup>
                                – r
                                <sup>
                                 weak
                                </sup>
                                ; black) and the distribution of results for N
                                <sub>
                                 perm
                                </sub>
                                random permutations of labels for labels { “strong”, “weak” } in gray. F: Difference of decision signals with permuted spikes across neurons. We permute spikes such as to maintain the coarse organization in 2 pools of neurons with positive and negative sign of the weight (black), and across all neurons (ochre). The gray area is for the distribution of signals with permuted class labels (the null model). The difference signals is scaled with respect to the unperturbed signal at maximum deviation from 0 (i.e., the decision signal in
                                <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                                 Fig 3C
                                </a>
                                at 211 ms after the stimulus onset). We permuted neuronal indices, randomly in each cross-validation cycle. G: Same as in F, showing decision signals with temporal jitter in the spike timing. H: Same as in F, showing the decision signal with permuted spike timing over the entire time window ([0,400] ms) or in two time windows before and after the zero crossing at 140 ms after the stimulus onset ([0,140],[141,400] ms). Parameters: time window for measuring correlations (A,B,D,E) and the effect of across-neurons structure is [140, 500] ms, time window for G and H is [0, 400] ms after the onset of the test stimulus; λ
                                <sup>
                                 -1
                                </sup>
                                = 20 ms, N
                                <sub>
                                 perm
                                </sub>
                                = 1000.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-77">
                              If noise correlations are helpful for the information transfer, we expect them to be stronger within subpopulations of neurons that are particularly informative for the task. By definition of the decoding model, neurons that are particularly informative for the task are neurons with strong decoding weights and neurons that are less informative have weak decoding weights (see
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig 4D
                              </a>
                              ). We therefore compare noise correlations among neuronal pairs with strong weights and weak weights. In line with our hypothesis on the benefit of noise correlations for the read-out of the time-dependent population signal, neurons with strong decoding weights show stronger noise correlations of spike timing than neurons with weak weights (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig 5D,E
                              </a>
                              ).
                             </p>
                             <p id="p-78">
                              Furthermore, across-neuron structure of spike trains is beneficial for the read-out of the decision signal. The benefit of noise correlations for coding suggested by results above (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig 5A,D
                              </a>
                              ) implies that across-neuron and across-time organization in parallel spike trains carries information about animal’s decision. We test the effect of across-neuron structure in spike trains on the decision signal by removing this structure from spike trains, computing the decision signal on such perturbed data, and comparing the perturbed decision signal with the unperturbed one. First, we keep the coarse organization in two neuronal pools but selectively remove across-neuron structure within neuronal pools. Removing the structure within pools weakens the decision signal for about 31 % with respect to the original signal (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig 5F
                              </a>
                              , black). Next, we perturb the structure across all neurons. This further weakens the decision signal for another 9 %. In total, the across-neuron organization of spike trains therefore contributes a substantial 40 % to the read-out of the decision signal. This contribution gets partitioned into about 31 % for the organization in two neuronal pools (following the sign of decoding weight) and in 9 % for the fine organization within pools.
                             </p>
                             <p id="p-79">
                              Further on, we test the effect of fine temporal structure of spike trains on the decision signal. Do do so, we jitter the timing of every spike within the time window of (± [20,100,200] ms) and then recompute the decision signal. The longer the jittering window, the weaker the decision signal (
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig 5G
                              </a>
                              ). More precisely, a jitter of 20,100 and 200 ms reduce the decision signal with respect to the signal with intact spike timing for 34, 57 and 75 %, respectively, measuring the difference at the maximal deviation of the original signal from zero. Note that the decision signal with 200 ms jittering window is close to losing significance. Even short jittering window of 5 and 10 ms reduce the decision signal for 16 and 22 %, respectively. We now test an even stronger perturbation of the temporal structure of spike trains on the read-out and permute the spike timing (for every neuron independently) over the entire time window between [0,400] ms after the stimulus onset. This weakens the decision signal for 79 % with respect to the decision signal with intact spike timing, and brings it close to noise (
                              <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                               Fig 5F
                              </a>
                              , red). We reason that a strong effect of shuffling the spike timing over a long time window might be partially due to different sources of the decision signal in the initial and late phase of the trial (see
                              <a id="xref-fig-3-11" class="xref-fig" href="#F3">
                               Fig 3D
                              </a>
                              ). If this is so, limiting the permutation to the early and late time window could improve the read-out. Indeed, limiting the permutation of the spike timing to before and after 140 ms after the onset of the test stimulus, we get a stronger decision signal (
                              <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                               Fig 5F
                              </a>
                              , black), that is however still 55 % weaker than the original signal.
                             </p>
                            </div>
                            <div id="sec-21" class="subsection">
                             <h3>
                              Activity of bursting neurons and of neurons in the superficial cortical layer is particularly informative about the choice of the animal
                             </h3>
                             <p id="p-80">
                              In the present setting, the read-out of the decision signal relies on decoding weights
                              <span class="inline-formula" id="inline-formula-57">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-57.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-57.gif"/>
                                </noscript>
                               </span>
                              </span>
                              that have here been learned with a supervised learning method (using the linear SVM). In a biological network, learning of decoding weights requires plasticity of synapses between V1 neurons and a downstream brain area that is providing the top-down input. A recent study demonstrated a biologically relevant mechanism of synaptic plasticity that operates across the brain hierarchy [
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ], where learning of synaptic weights between a low- and high-level brain area critically relies on bursting of neurons in the low-level brain area. Seen these results, we hypothesize that bursting neurons might be important for the read-out of the choice signal in the present setting. We divide neurons into bursting and non-bursting, based on characteristics of the power spectrum (PS) of spike trains. The PS of bursting neurons are characterized by a reduced power in middle range frequencies [
                              <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ] (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig 6A
                              </a>
                              ), and we capture this effect by computing the area under the PS of the spike train and compare it with the same result as the spike train has been randomly permuted over time. A random permutation of the spike train results in a Poisson process, since each timestep is now equally likely to contain a spike. We classify a neuron as bursting if it has the area under PS smaller than the lower bound of models with permuted spike timing (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig 6B
                              </a>
                              ; see methods), and as non-bursting otherwise.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F6.large.jpg?width=800&height=600&carousel=1" title="Bursting neurons carry more information about the choice than non-bursting neurons. A: Power spectrum for 4 representative neurons from the same recording session. We show the power spectrum for the true model (black) and the distribution of power spectra for models with randomly permuted spike timing (blue). Bursting neurons are those with reduced power in lower and medium range frequencies (middle right and right). B: Area under the power spectrum for all neurons, sorted from the lowest to the highest value (black), and the lower bound of the area under the power spectrum for models with permuted spike timing (blue). Red asterisk marks significance (permutation test, α = 0.05). There are 68 bursting and 92 non-bursting neurons. C: The difference of mean decision signals for bursting (top) and non-bursting subpopulations (bottom). Corresponding results of models with permuted class labels are shown as the gray area. D: Top: Cross-correlation of decision signals of bursting and non-bursting subpopulations. We show the mean ± SEM for the variability across recording sessions. Bottom: Session-averaged cross-correlation for the regular model (red) and the distribution of results of models with permuted class labels (gray area). E: Scatter plot of the area under the power spectrum vs. strength of decoding weight, distinguishing neurons with positive weights (top) and neurons with negative weights (bottom). R is the Pearson correlation coefficient. F: Balanced accuracy of the linear SVM on the classification problem stimulus + choice, using the information from bursting (magenta) or non-bursting subpopulations (black). G: Distribution of average firing rates for bursting and non-bursting neurons. H: Noise correlations of spike timing among bursting (N=198 pairs) and non-bursting neurons (N=231 pairs). We show the mean ± SEM for variability across pairs. I: Average difference of noise correlations (rbursting – rnon-bursting; blue) as a function of the maximal time lag, and the distribution of corresponding results of models with random assignment of the label {“bursting”, “non-bursting”}). Parameters: λ-1 = 20 ms (C), time window is [0, 400] ms after the onset of the test stimulus for B-G, and [140,500] ms for H-I; Nperm = 1000." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Bursting neurons carry more information about the choice than non-bursting neurons.</span> A: Power spectrum for 4 representative neurons from the same recording session. We show the power spectrum for the true model (black) and the distribution of power spectra for models with randomly permuted spike timing (blue). Bursting neurons are those with reduced power in lower and medium range frequencies (middle right and right). B: Area under the power spectrum for all neurons, sorted from the lowest to the highest value (black), and the lower bound of the area under the power spectrum for models with permuted spike timing (blue). Red asterisk marks significance (permutation test, <em>α</em> = 0.05). There are 68 bursting and 92 non-bursting neurons. C: The difference of mean decision signals for bursting (top) and non-bursting subpopulations (bottom). Corresponding results of models with permuted class labels are shown as the gray area. D: Top: Cross-correlation of decision signals of bursting and non-bursting subpopulations. We show the mean ± SEM for the variability across recording sessions. Bottom: Session-averaged cross-correlation for the regular model (red) and the distribution of results of models with permuted class labels (gray area). E: Scatter plot of the area under the power spectrum vs. strength of decoding weight, distinguishing neurons with positive weights (top) and neurons with negative weights (bottom). R is the Pearson correlation coefficient. F: Balanced accuracy of the linear SVM on the classification problem stimulus + choice, using the information from bursting (magenta) or non-bursting subpopulations (black). G: Distribution of average firing rates for bursting and non-bursting neurons. H: Noise correlations of spike timing among bursting (N=198 pairs) and non-bursting neurons (N=231 pairs). We show the mean ± SEM for variability across pairs. I: Average difference of noise correlations (<em>r</em><sup>bursting</sup> – <em>r</em><sup>non-bursting</sup>; blue) as a function of the maximal time lag, and the distribution of corresponding results of models with random assignment of the label {“bursting”, “non-bursting”}). Parameters: <em>λ</em><sup>-1</sup> = 20 ms (C), time window is [0, 400] ms after the onset of the test stimulus for B-G, and [140,500] ms for H-I; <em>N</em><sub>perm</sub> = 1000.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F6.medium.gif" width="318" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F6.medium.gif" width="318" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 6.
                               </span>
                               <span class="caption-title">
                                Bursting neurons carry more information about the choice than non-bursting neurons.
                               </span>
                               <p id="p-81" class="first-child">
                                A: Power spectrum for 4 representative neurons from the same recording session. We show the power spectrum for the true model (black) and the distribution of power spectra for models with randomly permuted spike timing (blue). Bursting neurons are those with reduced power in lower and medium range frequencies (middle right and right). B: Area under the power spectrum for all neurons, sorted from the lowest to the highest value (black), and the lower bound of the area under the power spectrum for models with permuted spike timing (blue). Red asterisk marks significance (permutation test, α = 0.05). There are 68 bursting and 92 non-bursting neurons. C: The difference of mean decision signals for bursting (top) and non-bursting subpopulations (bottom). Corresponding results of models with permuted class labels are shown as the gray area. D: Top: Cross-correlation of decision signals of bursting and non-bursting subpopulations. We show the mean ± SEM for the variability across recording sessions. Bottom: Session-averaged cross-correlation for the regular model (red) and the distribution of results of models with permuted class labels (gray area). E: Scatter plot of the area under the power spectrum vs. strength of decoding weight, distinguishing neurons with positive weights (top) and neurons with negative weights (bottom). R is the Pearson correlation coefficient. F: Balanced accuracy of the linear SVM on the classification problem stimulus + choice, using the information from bursting (magenta) or non-bursting subpopulations (black). G: Distribution of average firing rates for bursting and non-bursting neurons. H: Noise correlations of spike timing among bursting (N=198 pairs) and non-bursting neurons (N=231 pairs). We show the mean ± SEM for variability across pairs. I: Average difference of noise correlations (r
                                <sup>
                                 bursting
                                </sup>
                                – r
                                <sup>
                                 non-bursting
                                </sup>
                                ; blue) as a function of the maximal time lag, and the distribution of corresponding results of models with random assignment of the label {“bursting”, “non-bursting”}). Parameters: λ
                                <sup>
                                 -1
                                </sup>
                                = 20 ms (C), time window is [0, 400] ms after the onset of the test stimulus for B-G, and [140,500] ms for H-I; N
                                <sub>
                                 perm
                                </sub>
                                = 1000.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-82">
                              As we now compute the decision signal for bursting and non-bursting subpopulations, we find that the decision signal in bursting subpopulations is clearly stronger than in non-bursting subpopulations (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig 6C
                              </a>
                              ). Decision signals of bursting and non-bursting subpopulations are positively correlated and have a peak at zero time-lag (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Fig 6D
                              </a>
                              ). Importantly, for neurons with positive weight
                              <span class="inline-formula" id="inline-formula-58">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-58.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/06/14/2020.01.10.901504/embed/inline-graphic-58.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the strength of decoding weight is negatively correlated with the area under the PS (
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Fig 6E
                              </a>
                              ), meaning that the more bursting the neuron, the stronger decoding weight it tends to have. The correlation between the strength of the decoding weight and the are area under PS is significant for neurons with positive weights, but not for neurons with negative weights (p = 0.012 for neurons with positive weights, p = 0.610 for neurons with negative weight; permutation test with 1000 random assignment of labels {“bursting”, “non-bursting”}).
                             </p>
                             <p id="p-83">
                              Since bursting is related to learning of synaptic weights, we reason that bursting neurons might be better than non-bursting neurons also in predicting stimulus + choice with time-averaged data. Using again the linear SVM on parallel spike counts, but this time separating the information carried by bursting and non-bursting subpopulations, we find that subpopulations of bursting neurons predict the variable stimulus + choice better than the non-bursting neurons (p < 10
                              <sup>
                               -17
                              </sup>
                              , two-tailed Wilcoxon signed-rank test;
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               Fig 6F
                              </a>
                              ). This shows that the time-averaged activity in bursting neurons is more informative about stimulus + choice than in non-bursting neurons. Even though the absolute firing rate does not inform the classifier in
                              <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                               Fig 6F
                              </a>
                              , it is noteworthy that bursting neurons have substantially stronger firing rates than non-bursting neurons (
                              <a id="xref-fig-6-8" class="xref-fig" href="#F6">
                               Fig 6G
                              </a>
                              ). Furthermore, consistent with our hypothesis on the benefit of noise correlations for a time-dependent population read-out, bursting neurons have markedly stronger noise correlations of spike timing compared to non-bursting neurons (
                              <a id="xref-fig-6-9" class="xref-fig" href="#F6">
                               Fig 6H-I
                              </a>
                              ).
                             </p>
                             <p id="p-84">
                              At last, we divide neurons with respect to their location along the cortical depth in three cortical layers. Cortical layers differ in the source of their inputs, as the middle layer receives the strongest feed-forward input, while the superficial layer has been suggested as a likely target of top-down inputs [
                              <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                              ,
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                              ]. Moreover, cortical layers were shown to differ in their correlation structure [
                              <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ], which has direct consequences on the population code used here (see
                              <a id="xref-fig-5-10" class="xref-fig" href="#F5">
                               Fig 5C
                              </a>
                              ). We distinguish the superficial layer (N=48), the middle layer (N=51) and the deep layer of the cortex (N=61), using a method based on the current source density [
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ]. As we compute the decision signal in each layer, we find substantial and qualitative differences across layers (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig 7A
                              </a>
                              ), and in particular, the strongest decision signal is carried by the superficial layer. Decision signals are weakly positively correlated across all pairs of layers, and cross-correlations peak at 0 time-lag in all cases (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig 7B
                              </a>
                              ). If the decision signal originated from a bottom-up input, we would expect to see a significant decision signal in the middle layer, with potential decision signals in other layers following after a time delay. The fact that the decision signal in the middle layer is weak, but strong in the superficial layer is in line with our hypothesis about the top-down origin of the decision signal. Moreover, the fact that the cross-correlation of decision signals has a peak at zero time lag indicates that decision signals originate from a common (top-down) input rather than from interaction across cortical layers.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F7.large.jpg?width=800&height=600&carousel=1" title="Spiking activity in the superficial cortical layer carries the strongest decision signal. A: Session-averaged difference of decision signals (red) and the distribution of corresponding signals by read-out models with permuted class labels (gray). B: Cross-correlation of decision signals across pairs of cortical layers (red) and the corresponding result of models with permuted class labels (gray area). C: Percentage of bursting versus non-bursting neurons in cortical layers. D: Noise correlation of spike timing for neuronal pairs within specific layers. We show the mean ± SEM across pairs. We used the time window [140,500] ms after the onset of the test stimulus. Parameters: λ-1 = 20 ms, Nperm = 1000, time window for A-C is [0, 400], time window for D is [140,500] ms after the onset of the test stimulus." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-227165868" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Spiking activity in the superficial cortical layer carries the strongest decision signal.</span> A: Session-averaged difference of decision signals (red) and the distribution of corresponding signals by read-out models with permuted class labels (gray). B: Cross-correlation of decision signals across pairs of cortical layers (red) and the corresponding result of models with permuted class labels (gray area). C: Percentage of bursting versus non-bursting neurons in cortical layers. D: Noise correlation of spike timing for neuronal pairs within specific layers. We show the mean ± SEM across pairs. We used the time window [140,500] ms after the onset of the test stimulus. Parameters: <em>λ</em><sup>-1</sup> = 20 ms, <em>N</em><sub>perm</sub> = 1000, time window for A-C is [0, 400], time window for D is [140,500] ms after the onset of the test stimulus.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F7.medium.gif" width="440" height="281"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F7.medium.gif" width="440" height="281"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/06/14/2020.01.10.901504/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 7.
                               </span>
                               <span class="caption-title">
                                Spiking activity in the superficial cortical layer carries the strongest decision signal.
                               </span>
                               <p id="p-85" class="first-child">
                                A: Session-averaged difference of decision signals (red) and the distribution of corresponding signals by read-out models with permuted class labels (gray). B: Cross-correlation of decision signals across pairs of cortical layers (red) and the corresponding result of models with permuted class labels (gray area). C: Percentage of bursting versus non-bursting neurons in cortical layers. D: Noise correlation of spike timing for neuronal pairs within specific layers. We show the mean ± SEM across pairs. We used the time window [140,500] ms after the onset of the test stimulus. Parameters: λ
                                <sup>
                                 -1
                                </sup>
                                = 20 ms, N
                                <sub>
                                 perm
                                </sub>
                                = 1000, time window for A-C is [0, 400], time window for D is [140,500] ms after the onset of the test stimulus.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-86">
                              The superficial layer has an excess of bursting neurons (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig 7D
                              </a>
                              ), which explains stronger signal in the superficial layer compared to other layers. In the superficial layer, 54 % of neurons are bursting neurons (while the remainder are non-bursting). In the middle and deep layer, 39 and 36 % of neurons are bursting, respectively. We have shown above that bursting neurons have stronger correlations than non-bursting neurons. Consistently with the excess of bursting neuron in the superficial layer of the cortex, noise correlations of spike timing are the strongest among neurons in the superficial layer compared to middle and deep cortical layers (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Fig 7D
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-87">
                             In the present study, we examined the stimulus- and the choice-related population signals in V1 and predicted the future behavioral choice from the spiking activity in the primary visual cortex. We showed that the choice can be decoded with a time-dependent linear population read-out, and that decoding weights need to be learned in correct trials. We argued that the generalization of learning from the variable stimulus + choice to choice relies on the (partial) overlap in the information on the stimuli and the choice. Furthermore, we found that bursting neurons and neurons in the superficial layer of the cortex are particularly informative for the task. Our results suggested that correlated spiking of neurons with similar decoding selectivity helps the accumulation of the time-resolved choice signal and thus enhances the population signals about the categorical stimulus class and the congruent behavioral choice of the animal.
                            </p>
                            <p id="p-88">
                             Current results raise a question, why is learning in correct trials necessary for the prediction of the choice? The following scenario during the condition “incorrect non-match” is compatible with all our observations: 1) the test stimulus is “non-match” and V1 responds with a spiking pattern corresponding to the stimulus class “non-match” during the first 100 or so milliseconds after the onset of the stimulus, 2) there is an error in transmission of the signal to downstream areas that will result in incorrect choice [
                             <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">
                              45
                             </a>
                             ], 3) due to the error in transmission, a high-level area erroneously signals the stimulus class “match” and the choice “same” and projects this signal back to V1, and 4) as the projection is received by V1, the spiking pattern in V1 switches and signals the stimulus class “match” and the choice “same”. This scenario allows the activity in V1 to be predictive of the choice simply through the alignment of the representation about the stimuli and the choice, activated by a top-down projection.
                            </p>
                            <p id="p-89">
                             If the population signal in the context of stimulus + choice would rely exclusively on the difference in stimuli, we would not be able to predict the variable choice by learning population decoding weights on stimulus + choice. On the contrary, if the representations of the choice and the stimulus class in V1 are mixed [
                             <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ] and the information about the two variables is overlapping [
                             <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ], learning in the context of stimulus + choice can generalize to the representation of choice, leveraging the part of information that is invariant across the two classification problems. In the brain, such alignment can be instrumental in the sense that it reinforces the representation of the upcoming behavioral choice across brain networks. The information about the behavioral choice was shown to be distributed across multiple brain areas [
                             <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                              57
                             </a>
                             ], and alignment of the signals about the stimulus and the choice could serve the purpose of reverberating and amplifying the choice-related signal within the cortical circuitry, to help the brain converge to a single behavioral outcome. Future work could address the overlap of the stimulus and choice-related information with information-theoretic measures [
                             <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">
                              56
                             </a>
                             ] that can disentangle the contribution of these sources of information more precisely than the read-out based methods that we used in the present analysis.
                            </p>
                            <p id="p-90">
                             Another result that merits the discussion is the role of noise correlations for the time-dependent population signal. Positive noise correlations of spike counts between neurons with similar selectivity are known to decrease the quantity of information in a rate code [
                             <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                              51
                             </a>
                             ], and they do so also in the present dataset [
                             <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                              39
                             </a>
                             ]. However, in the present decision-making task, rather than transmitting all the information about the complex naturalistic stimuli, neural networks are faced with the task of extracting task-relevant information (“match/non-match” of stimuli) and creating a variable that can inform the binary choice of the animal. The binary variable related to animal’s choice “same/different” only consists of 1 bit of information. Rather than maximizing the quantity of information about the stimulus, it might be important to create a variable that is robust to trial-to-trial variability of spiking patterns and can be read-out consistently over the course of the trial. Recent theoretical work pointed out that noise correlations among neurons with similar tuning are useful for improving the robustness of a rate code [
                             <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                              58
                             </a>
                             ] and the temporal consistency of the read-out [
                             <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">
                              53
                             </a>
                             ]. In the present work, we show how structured noise correlations can be useful in a time-dependent population read-out. Correlated spiking of neurons with similar decoding selectivity allows accumulation of the population signal and makes the signal rise from the noisy background. Note that if the spiking activity was entirely uncorrelated across neurons, a time-dependent read-out of such uncorrelated spiking would be a noisy oscillation around zero. From such a signal it would indeed be impossible to read-out the choice of the animal or any other behaviorally relevant variable. Present results therefore suggest a simple and biologically relevant role of correlated spiking between neurons with similar selectivity - to allow the accumulation of a time-dependent low-dimensional signal and help its transmission downstream.
                            </p>
                            <p id="p-91">
                             How can a top-down signal simultaneously drive neural responses and determine the structure of correlations? Previous work has suggested that the structure of noise correlations in V1 is primarily determined by a common input [
                             <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ]. Moreover, correlated variability in V1 was shown to change with task instructions [
                             <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                              59
                             </a>
                             ], strongly suggesting that correlated variability in V1 can be driven by a feedback from higher cortical area(s). In line with these results, we showed that noise correlations are stronger between neurons that contribute the most to the choice signal. As long as top-down projections are specific about the sign of decoding weights of V1 neurons, such projections could at the same time drive the choice signal and create the observed pattern of noise correlations where neurons with similar decoding selectivity are more strongly correlated. Top-down input could target neurons with positive weights in trials with decision “same” and neurons with negative weights in trials with decision “different” through precise long-range connections. Precise long range microcircuit-to-microcircuit connections between the prefrontal cortex and V1, recently put in evidence [
                             <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                              60
                             </a>
                             ], are a good candidate for conveying such a precise feedback signal. Future work could tell if such a simple mechanism can explain present results.
                            </p>
                            <p id="p-92">
                             Finally, our results showed that, in the present behavioral task, bursting neurons are particularly informative about the choice of the animal. Bursting neurons reported here likely correspond to recently reported class of bursting excitatory neurons in the V1 of the macaque, located predominantly in the superficial layers of the cortex [
                             <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                              61
                             </a>
                             ]. Bursting neurons have been proposed to integrate bottom-up and top-down inputs [
                             <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                              62
                             </a>
                             ,
                             <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                              63
                             </a>
                             ], and be involved in learning of synaptic weights across cortical hierarchy [
                             <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                              54
                             </a>
                             ]. Encoding of binary stimulus classes and the binary choice is not a build-in feature of V1 neurons and in order for V1 neurons to encode such variables, their synapses with the top-down projecting neurons have to be modified. Bursting neurons might therefore be particularly informative about the choice of the animal in the current experimental setting because their synapses with top-down projecting neurons have been modified in such a way as to support coding of a binary stimulus and choice.
                            </p>
                           </div>
                           <div class="section" id="sec-23">
                            <h2 class="">
                             Conclusion
                            </h2>
                            <p id="p-93">
                             In the present setting, macaque monkeys learned to classify binary categories “match” vs. “non-match” on succession of two complex naturalistic stimuli. In such a setting, a binary choice variable “same”/“different” in the absence of the information about the stimulus class can be predicted from the spiking activity of neural populations in the primary visual cortex using learning from invariants, a learning method that is likely used in intelligent systems. The time-dependent signal about animal’s decision can be computed as a population read-out of spike trains by the observed neural population and models a variable proportional to the synaptic current in hypothetical read-out neurons. Such a signal carries the information about animal’s choice in decoding weights of bursting neurons in superficial cortical layers, as well as in temporal and across-neuron organization of spike trains. Temporal and across-neuron organization of spike trains is captured by noise correlations of spike timing that seem to be such as to help the accumulation of the decision signal.
                            </p>
                           </div>
                           <div class="section" id="sec-25">
                            <h2 class="">
                             Supporting information
                            </h2>
                            <p id="p-94">
                             <strong>
                              S1 Fig. Robustness of the decision signal with respect to the time window used for decoding.
                             </strong>
                            </p>
                            <p id="p-95">
                             <strong>
                              S2 Fig. Learning and validation on the variable
                             </strong>
                             choice
                             <strong>
                              result in poor prediction of the decision variable.
                             </strong>
                            </p>
                            <p id="p-96">
                             <strong>
                              S3 Fig. Animal’s decision cannot be predicted during the target stimulus.
                             </strong>
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-97">
                             This project was supported by Deutsche Forschungsgemeinschaft, grant GRK 1589/2, Technische Universität Berlin and Berlin Equal Opportunity Programm (Berliner Chancengleichheitsprogramm).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-3">
                              <p id="p-3">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                *
                               </span>
                               <span class="em-link">
                                <span class="em-addr">
                                 valentin.dragoi{at}uth.tmc.edu
                                </span>
                               </span>
                               ,
                               <span class="em-link">
                                <span class="em-addr">
                                 klaus.obermayer{at}tu-berlin.de
                                </span>
                               </span>
                               ,
                               <span class="em-link">
                                <span class="em-addr">
                                 vkoren{at}uke.de
                                </span>
                               </span>
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-4">
                              <p id="p-8">
                               major revision
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.1" data-doi="10.1073/pnas.93.2.628">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Newsome
                                  </span>
                                  <span class="cit-name-given-names">
                                   WT.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Motion perception: seeing and deciding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the national academy of sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  628
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  633
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShadlen%26rft.auinit1%253DM.%2BN.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D628%26rft.epage%253D633%26rft.atitle%253DMotion%2Bperception%253A%2Bseeing%2Band%2Bdeciding.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.2.628%26rft_id%253Dinfo%253Apmid%252F8570606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5My8yLzYyOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE0LzIwMjAuMDEuMTAuOTAxNTA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.2" data-doi="10.1017/S095252380000715X">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Britten
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Newsome
                                  </span>
                                  <span class="cit-name-given-names">
                                   WT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Celebrini
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Movshon
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A relationship between behavioral choice and the visual responses of neurons in macaque MT
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Visual neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  87
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  100
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DVisual%2Bneuroscience%26rft.stitle%253DVis%2BNeurosci%26rft.aulast%253DBritten%26rft.auinit1%253DK.%2BH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D100%26rft.atitle%253DA%2Brelationship%2Bbetween%2Bbehavioral%2Bchoice%2Band%2Bthe%2Bvisual%2Bresponses%2Bof%2Bneurons%2Bin%2Bmacaque%2BMT.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS095252380000715X%26rft_id%253Dinfo%253Apmid%252F8730992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1017/S095252380000715X&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8730992&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996TN93200009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.10.901504v6.3" data-doi="10.1523/JNEUROSCI.16-04-01486.1996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Britten
                                  </span>
                                  <span class="cit-name-given-names">
                                   KH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Newsome
                                  </span>
                                  <span class="cit-name-given-names">
                                   WT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Movshon
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A computational analysis of the relationship between neuronal and behavioral responses to visual motion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1486
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1510
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DShadlen%26rft.auinit1%253DM.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D1486%26rft.epage%253D1510%26rft.atitle%253DA%2Bcomputational%2Banalysis%2Bof%2Bthe%2Brelationship%2Bbetween%2Bneuronal%2Band%2Bbehavioral%2Bresponses%2Bto%2Bvisual%2Bmotion%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.16-04-01486.1996%26rft_id%253Dinfo%253Apmid%252F8778300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjE2LzQvMTQ4NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE0LzIwMjAuMDEuMTAuOTAxNTA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.10.901504v6.4" data-doi="10.1126/science.2772635">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Logothetis
                                  </span>
                                  <span class="cit-name-given-names">
                                   NK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schall
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Neuronal correlates of subjective visual perception
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  245
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4919
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  763
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLogothetis%26rft.auinit1%253DN.%26rft.volume%253D245%26rft.issue%253D4919%26rft.spage%253D761%26rft.epage%253D763%26rft.atitle%253DNeuronal%2Bcorrelates%2Bof%2Bsubjective%2Bvisual%2Bperception%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.2772635%26rft_id%253Dinfo%253Apmid%252F2772635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNDUvNDkxOS83NjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNi8xNC8yMDIwLjAxLjEwLjkwMTUwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.5" data-doi="10.1523/JNEUROSCI.22-21-09475.2002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roitman
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MN.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Response of neurons in the lateral intraparietal area during a combined visual discrimination reaction time task
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  21
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  9475
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9489
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DRoitman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D22%26rft.issue%253D21%26rft.spage%253D9475%26rft.epage%253D9489%26rft.atitle%253DResponse%2Bof%2BNeurons%2Bin%2Bthe%2BLateral%2BIntraparietal%2BArea%2Bduring%2Ba%2BCombined%2BVisual%2BDiscrimination%2BReaction%2BTime%2BTask%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.22-21-09475.2002%26rft_id%253Dinfo%253Apmid%252F12417672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yMS85NDc1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.10.901504v6.6" data-doi="10.1038/nature07821">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nienborg
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   BG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Decision-related activity in sensory neurons reflects more than a neuron’s causal effect
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  459
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7243
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  89
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  92
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNienborg%26rft.auinit1%253DH.%26rft.volume%253D459%26rft.issue%253D7243%26rft.spage%253D89%26rft.epage%253D92%26rft.atitle%253DDecision-related%2Bactivity%2Bin%2Bsensory%2Bneurons%2Breflects%2Bmore%2Bthan%2Ba%2Bneuron%2527s%2Bcausal%2Beffect.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07821%26rft_id%253Dinfo%253Apmid%252F19270683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature07821&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19270683&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000265801300035&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.7" data-doi="10.1523/JNEUROSCI.3331-16.2017">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goris
                                  </span>
                                  <span class="cit-name-given-names">
                                   RL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ziemba
                                  </span>
                                  <span class="cit-name-given-names">
                                   CM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stine
                                  </span>
                                  <span class="cit-name-given-names">
                                   GM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simoncelli
                                  </span>
                                  <span class="cit-name-given-names">
                                   EP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Movshon
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Dissociation of choice formation and choice-correlated activity in macaque visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  5195
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5203
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.3331-16.2017%26rft_id%253Dinfo%253Apmid%252F28432137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzNy8yMC81MTk1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seidemann
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Geisler
                                  </span>
                                  <span class="cit-name-given-names">
                                   WS.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Linking V1 activity to behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual review of vision science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  287
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bvision%2Bscience%26rft.volume%253D4%26rft.spage%253D287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.9" data-doi="10.1523/JNEUROSCI.0035-19.2019">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jasper
                                  </span>
                                  <span class="cit-name-given-names">
                                   AI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanabe
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kohn
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Predicting perceptual decisions using visual cortical population responses and choice history
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  34
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6714
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6727
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.0035-19.2019%26rft_id%253Dinfo%253Apmid%252F31235648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8zNC82NzE0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krishna
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tanabe
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kohn
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Decision Signals in the Local Field Potentials of Early and Mid-Level Macaque Visual Cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cerebral Cortex
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  169
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  183
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCerebral%2BCortex%26rft.volume%253D31%26rft.spage%253D169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.11" data-doi="10.1016/S0896-6273(04)00186-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uka
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   DeAngelis
                                  </span>
                                  <span class="cit-name-given-names">
                                   GC.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Contribution of area MT to stereoscopic depth perception: choice-related response modulations reflect task strategy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  42
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  297
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DUka%26rft.auinit1%253DT.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D297%26rft.epage%253D310%26rft.atitle%253DContribution%2Bof%2Barea%2BMT%2Bto%2Bstereoscopic%2Bdepth%2Bperception%253A%2Bchoice-related%2Bresponse%2Bmodulations%2Breflect%2Btask%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0896-6273%252804%252900186-2%26rft_id%253Dinfo%253Apmid%252F15091344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0896-6273(04)00186-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15091344&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000221458500013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.12" data-doi="10.1523/JNEUROSCI.2256-06.2006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nienborg
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   BG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Macaque V2 neurons, but not V1 neurons, show choice-related activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  37
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  9567
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9578
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DNienborg%26rft.auinit1%253DH.%26rft.volume%253D26%26rft.issue%253D37%26rft.spage%253D9567%26rft.epage%253D9578%26rft.atitle%253DMacaque%2Bv2%2Bneurons%252C%2Bbut%2Bnot%2Bv1%2Bneurons%252C%2Bshow%2Bchoice-related%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2256-06.2006%26rft_id%253Dinfo%253Apmid%252F16971541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8zNy85NTY3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.13" data-doi="10.1523/JNEUROSCI.2340-13.2014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nienborg
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   BG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Decision-related activity in sensory neurons may depend on the columnar architecture of cerebral cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3579
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3585
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DNienborg%26rft.auinit1%253DH.%26rft.volume%253D34%26rft.issue%253D10%26rft.spage%253D3579%26rft.epage%253D3585%26rft.atitle%253DDecision-Related%2BActivity%2Bin%2BSensory%2BNeurons%2BMay%2BDepend%2Bon%2Bthe%2BColumnar%2BArchitecture%2Bof%2BCerebral%2BCortex%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2340-13.2014%26rft_id%253Dinfo%253Apmid%252F24599457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzNC8xMC8zNTc5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.14" data-doi="10.1167/8.6.815">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Palmer
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seidemann
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Choice probability and reaction-Time correlations in Macaque V1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Vision
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  815
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  815
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVision%26rft_id%253Dinfo%253Adoi%252F10.1167%252F8.6.815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoiam92IjtzOjU6InJlc2lkIjtzOjc6IjgvNi84MTUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNi8xNC8yMDIwLjAxLjEwLjkwMTUwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.15" data-doi="10.1038/nn.4183">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kwon
                                  </span>
                                  <span class="cit-name-given-names">
                                   SE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Severson
                                  </span>
                                  <span class="cit-name-given-names">
                                   KS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’connor
                                  </span>
                                  <span class="cit-name-given-names">
                                   DH.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Origins of choice-related activity in mouse somatosensory cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  134
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D19%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4183%26rft_id%253Dinfo%253Apmid%252F26642088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.4183&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26642088&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.16" data-doi="10.1146/annurev-neuro-062111-150403">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nienborg
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   R Cohen
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   BG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Decision-related activity in sensory neurons: correlations among neurons and with behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual review of neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  463
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  483
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bneuroscience%26rft.stitle%253DAnnu%2BRev%2BNeurosci%26rft.aulast%253DNienborg%26rft.auinit1%253DH.%26rft.volume%253D35%26rft.spage%253D463%26rft.epage%253D483%26rft.atitle%253DDecision-related%2Bactivity%2Bin%2Bsensory%2Bneurons%253A%2Bcorrelations%2Bamong%2Bneurons%2Band%2Bwith%2Bbehavior.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-neuro-062111-150403%26rft_id%253Dinfo%253Apmid%252F22483043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-neuro-062111-150403&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22483043&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000307960400023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.17" data-doi="10.1016/j.conb.2015.04.003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ganguli
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  On simplicity and complexity in the brave new world of large-scale neuroscience
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in neurobiology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  148
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  155
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.volume%253D32%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2015.04.003%26rft_id%253Dinfo%253Apmid%252F25932978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2015.04.003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25932978&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.10.901504v6.18" data-doi="10.1016/j.conb.2019.02.002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saxena
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Towards the neural population doctrine
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in neurobiology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  55
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  103
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  111
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.volume%253D55%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2019.02.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2019.02.002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.19" data-doi="10.1371/journal.pcbi.1003258">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boerlin
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Machens
                                  </span>
                                  <span class="cit-name-given-names">
                                   CK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Denève
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Predictive coding of dynamical variables in balanced spiking networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1003258
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D9%26rft.spage%253De1003258%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003258%26rft_id%253Dinfo%253Apmid%252F24244113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003258&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24244113&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.20" data-doi="10.1016/j.heares.2010.06.006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  <span class="cit-name-given-names">
                                   KD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartho
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chadderton
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Curto
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de la Rocha
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hollender
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  How do neurons work together? Lessons from auditory cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Hearing research
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  271
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1-2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  37
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  53
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHearing%2Bresearch%26rft.stitle%253DHear%2BRes%26rft.aulast%253DHarris%26rft.auinit1%253DK.%2BD.%26rft.volume%253D271%26rft.issue%253D1-2%26rft.spage%253D37%26rft.epage%253D53%26rft.atitle%253DHow%2Bdo%2Bneurons%2Bwork%2Btogether%253F%2BLessons%2Bfrom%2Bauditory%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.heares.2010.06.006%26rft_id%253Dinfo%253Apmid%252F20603208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.heares.2010.06.006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20603208&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.21" data-doi="10.1523/JNEUROSCI.2928-12.2013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luczak
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartho
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  <span class="cit-name-given-names">
                                   KD.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Gating of sensory input by spontaneous cortical activity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1684
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1695
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DLuczak%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D4%26rft.spage%253D1684%26rft.epage%253D1695%26rft.atitle%253DGating%2Bof%2BSensory%2BInput%2Bby%2BSpontaneous%2BCortical%2BActivity%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2928-12.2013%26rft_id%253Dinfo%253Apmid%252F23345241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjMzLzQvMTY4NCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE0LzIwMjAuMDEuMTAuOTAxNTA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.22" data-doi="10.1038/nature11129">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Churchland
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                  <span class="cit-name-given-names">
                                   JP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaufman
                                  </span>
                                  <span class="cit-name-given-names">
                                   MT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                  <span class="cit-name-given-names">
                                   JD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nuyujukian
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ryu
                                  </span>
                                  <span class="cit-name-given-names">
                                   SI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Neural population dynamics during reaching
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  487
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7405
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  51
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  56
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChurchland%26rft.auinit1%253DM.%2BM.%26rft.volume%253D487%26rft.issue%253D7405%26rft.spage%253D51%26rft.epage%253D56%26rft.atitle%253DNeural%2Bpopulation%2Bdynamics%2Bduring%2Breaching.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11129%26rft_id%253Dinfo%253Apmid%252F22722855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature11129&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22722855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000305982900048&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.23" data-doi="10.1038/nature12160">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rigotti
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barak
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Warden
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   XJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Daw
                                  </span>
                                  <span class="cit-name-given-names">
                                   ND
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                  <span class="cit-name-given-names">
                                   EK
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The importance of mixed selectivity in complex cognitive tasks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  497
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7451
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  585
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  590
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12160%26rft_id%253Dinfo%253Apmid%252F23685452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12160&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23685452&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000319556100036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.24" data-doi="10.1126/science.aao0284">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ni
                                  </span>
                                  <span class="cit-name-given-names">
                                   AM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruff
                                  </span>
                                  <span class="cit-name-given-names">
                                   DA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alberts
                                  </span>
                                  <span class="cit-name-given-names">
                                   JJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Symmonds
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Learning and attention reveal a general relationship between population activity and behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  359
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6374
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  463
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  465
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNi%26rft.auinit1%253DA.%2BM.%26rft.volume%253D359%26rft.issue%253D6374%26rft.spage%253D463%26rft.epage%253D465%26rft.atitle%253DLearning%2Band%2Battention%2Breveal%2Ba%2Bgeneral%2Brelationship%2Bbetween%2Bpopulation%2Bactivity%2Band%2Bbehavior%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aao0284%26rft_id%253Dinfo%253Apmid%252F29371470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTkvNjM3NC80NjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNi8xNC8yMDIwLjAxLjEwLjkwMTUwNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.25" data-doi="10.1038/s41467-021-27201-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kafashan
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jaffe
                                  </span>
                                  <span class="cit-name-given-names">
                                   AW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chettih
                                  </span>
                                  <span class="cit-name-given-names">
                                   SN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nogueira
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arandia-Romero
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harvey
                                  </span>
                                  <span class="cit-name-given-names">
                                   CD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scaling of sensory information in large neural populations shows signatures of information-limiting correlations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D12%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-021-27201-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-021-27201-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.26" data-doi="10.1523/JNEUROSCI.6150-10.2011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Engel
                                  </span>
                                  <span class="cit-name-given-names">
                                   TA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  <span class="cit-name-given-names">
                                   XJ.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Same or different? A neural circuit mechanism of similarity-based pattern match decision making
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6982
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6996
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DEngel%26rft.auinit1%253DT.%2BA.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D6982%26rft.epage%253D6996%26rft.atitle%253DSame%2Bor%2BDifferent%253F%2BA%2BNeural%2BCircuit%2BMechanism%2Bof%2BSimilarity-Based%2BPattern%2BMatch%2BDecision%2BMaking%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.6150-10.2011%26rft_id%253Dinfo%253Apmid%252F21562260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xOS82OTgyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.27" data-doi="10.1016/j.cub.2015.08.057">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muckli
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Martino
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vizioli
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petro
                                  </span>
                                  <span class="cit-name-given-names">
                                   LS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  <span class="cit-name-given-names">
                                   FW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ugurbil
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Contextual feedback to superficial layers of V1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2690
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2695
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BBiology%26rft.volume%253D25%26rft.spage%253D2690%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2015.08.057%26rft_id%253Dinfo%253Apmid%252F26441356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cub.2015.08.057&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26441356&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.01.10.901504v6.28" data-doi="10.1523/JNEUROSCI.16-22-07376.1996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zipser
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lamme
                                  </span>
                                  <span class="cit-name-given-names">
                                   VA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schiller
                                  </span>
                                  <span class="cit-name-given-names">
                                   PH.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Contextual modulation in primary visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  7376
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7389
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DZipser%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D22%26rft.spage%253D7376%26rft.epage%253D7389%26rft.atitle%253DContextual%2BModulation%2Bin%2BPrimary%2BVisual%2BCortex%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.16-22-07376.1996%26rft_id%253Dinfo%253Apmid%252F8929444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8yMi83Mzc2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.29" data-doi="10.1007/s00221-003-1478-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Juan
                                  </span>
                                  <span class="cit-name-given-names">
                                   CH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walsh
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Feedback to V1: a reverse hierarchy in vision
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental brain research
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  150
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  259
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  263
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bbrain%2Bresearch.%2B%2BExperimentelle%2BHirnforschung.%2B%2BExp%25C3%25A9rimentation%2Bc%25C3%25A9r%25C3%25A9brale%26rft.stitle%253DExp%2BBrain%2BRes%26rft.aulast%253DJuan%26rft.auinit1%253DC.%2BH.%26rft.volume%253D150%26rft.issue%253D2%26rft.spage%253D259%26rft.epage%253D263%26rft.atitle%253DFeedback%2Bto%2BV1%253A%2Ba%2Breverse%2Bhierarchy%2Bin%2Bvision.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00221-003-1478-5%26rft_id%253Dinfo%253Apmid%252F12682809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s00221-003-1478-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12682809&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000183165100015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spoerer
                                  </span>
                                  <span class="cit-name-given-names">
                                   CJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kietzmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   TC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mehrer
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Charest
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kriegeskorte
                                  </span>
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Recurrent neural networks can explain flexible trading of speed and accuracy in biological vision
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1008215
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D16%26rft.spage%253D1008215e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.31" data-doi="10.1038/nn1379">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Silvanto
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cowey
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lavie
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walsh
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Striate cortex (V1) activity gates awareness of motion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  143
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  144
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DSilvanto%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D143%26rft.epage%253D144%26rft.atitle%253DStriate%2Bcortex%2B%2528V1%2529%2Bactivity%2Bgates%2Bawareness%2Bof%2Bmotion.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn1379%26rft_id%253Dinfo%253Apmid%252F15643428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn1379&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15643428&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000226638200010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.32" data-doi="10.1038/29537">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hupé
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   James
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payne
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lomber
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Girard
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bullier
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cortical feedback improves discrimination between figure and background by V1, V2 and V3 neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  394
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6695
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  784
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  787
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHup%25C3%25A9%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D394%26rft.issue%253D6695%26rft.spage%253D784%26rft.epage%253D787%26rft.atitle%253DCortical%2Bfeedback%2Bimproves%2Bdiscrimination%2Bbetween%2Bfigure%2Band%2Bbackground%2Bby%2BV1%252C%2BV2%2Band%2BV3%2Bneurons.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F29537%26rft_id%253Dinfo%253Apmid%252F9723617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/29537&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=9723617&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000075503600046&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.33" data-doi="10.1016/j.neuron.2016.12.036">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Panzeri
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harvey
                                  </span>
                                  <span class="cit-name-given-names">
                                   CD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Piasini
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Latham
                                  </span>
                                  <span class="cit-name-given-names">
                                   PE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fellin
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cracking the neural code for sensory perception by combining statistics, intervention, and behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  491
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  507
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D93%26rft.spage%253D491%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2016.12.036%26rft_id%253Dinfo%253Apmid%252F28182905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2016.12.036&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28182905&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yates
                                  </span>
                                  <span class="cit-name-given-names">
                                   JL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levi
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huk
                                  </span>
                                  <span class="cit-name-given-names">
                                   AC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  <span class="cit-name-given-names">
                                   IM.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Stimulus-choice (mis) alignment in primate area MT
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1007614
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D16%26rft.spage%253De1007614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vapnik
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Izmailov
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Rethinking statistical learning theory: learning using statistical invariants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Machine Learning
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  108
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  381
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  423
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%26rft.volume%253D108%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blainey
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krzywinski
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Altman
                                  </span>
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Points of significance: replication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  879
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D11%26rft.spage%253D879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.01.10.901504v6.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vapnik
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  The nature of statistical learning theory
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer science & business media
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Uncovering structured responses of neural populations recorded from macaque monkeys with linear support vector machines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  STAR protocols
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  100746
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSTAR%2Bprotocols%26rft.volume%253D2%26rft.spage%253D100746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrei
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dragoi
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Obermayer
                                  </span>
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Pairwise Synchrony and Correlations Depend on the Structure of the Population Code in Visual Cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Reports
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  108367
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D33%26rft.spage%253D108367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.40" data-doi="10.1152/jn.00949.2002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Compte
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Constantinidis
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tegnér
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raghavachari
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chafee
                                  </span>
                                  <span class="cit-name-given-names">
                                   MV
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goldman-Rakic
                                  </span>
                                  <span class="cit-name-given-names">
                                   PS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Temporally irregular mnemonic persistent activity in prefrontal neurons of monkeys during a delayed response task
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  90
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  3441
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3454
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00949.2002%26rft_id%253Dinfo%253Apmid%252F12773500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00949.2002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12773500&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000186547600054&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.41" data-doi="10.1038/nn890">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pesaran
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pezaris
                                  </span>
                                  <span class="cit-name-given-names">
                                   JS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahani
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitra
                                  </span>
                                  <span class="cit-name-given-names">
                                   PP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andersen
                                  </span>
                                  <span class="cit-name-given-names">
                                   RA.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Temporal structure in neuronal activity during working memory in macaque parietal cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  805
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  811
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DPesaran%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D805%26rft.epage%253D811%26rft.atitle%253DTemporal%2Bstructure%2Bin%2Bneuronal%2Bactivity%2Bduring%2Bworking%2Bmemory%2Bin%2Bmacaque%2Bparietal%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn890%26rft_id%253Dinfo%253Apmid%252F12134152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn890&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12134152&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000177088900025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.42" data-doi="10.1016/j.neuron.2012.08.029">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  <span class="cit-name-given-names">
                                   BJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chelaru
                                  </span>
                                  <span class="cit-name-given-names">
                                   MI
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dragoi
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Correlated variability in laminar cortical circuits
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  76
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  590
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  602
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D76%26rft.spage%253D590%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2012.08.029%26rft_id%253Dinfo%253Apmid%252F23141070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2012.08.029&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23141070&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000311063700014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrei
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dragoi
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Obermayer
                                  </span>
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Reading-out task variables as a low-dimensional reconstruction of neural spike trains in single trials
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e0222649
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPloS%2Bone%26rft.volume%253D14%26rft.spage%253De0222649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.44" data-doi="10.1523/JNEUROSCI.21-05-01676.2001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bair
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zohary
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Newsome
                                  </span>
                                  <span class="cit-name-given-names">
                                   WT.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Correlated firing in macaque visual area MT: time scales and relationship to behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1676
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1697
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DBair%26rft.auinit1%253DW.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D1676%26rft.epage%253D1697%26rft.atitle%253DCorrelated%2BFiring%2Bin%2BMacaque%2BVisual%2BArea%2BMT%253A%2BTime%2BScales%2Band%2BRelationship%2Bto%2BBehavior%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.21-05-01676.2001%26rft_id%253Dinfo%253Apmid%252F11222658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjIxLzUvMTY3NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE0LzIwMjAuMDEuMTAuOTAxNTA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.45" data-doi="10.1038/s41593-019-0406-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shahidi
                                  </span>
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrei
                                  </span>
                                  <span class="cit-name-given-names">
                                   AR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dragoi
                                  </span>
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  High-order coordination of cortical spiking activity modulates perceptual accuracy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1148
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1158
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D22%26rft.spage%253D1148%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-019-0406-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41593-019-0406-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Belousov
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verzakov
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Von Frese
                                  </span>
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A flexible classification approach with optimal generalisation performance: support vector machines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Chemometrics and intelligent laboratory systems
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  15
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DChemometrics%2Band%2Bintelligent%2Blaboratory%2Bsystems%26rft.volume%253D64%26rft.spage%253D15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.47" data-doi="10.1038/nn.2439">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  <span class="cit-name-given-names">
                                   MR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maunsell
                                  </span>
                                  <span class="cit-name-given-names">
                                   JH.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Attention improves performance primarily by reducing interneuronal correlations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1594
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DCohen%26rft.auinit1%253DM.%2BR.%26rft.volume%253D12%26rft.issue%253D12%26rft.spage%253D1594%26rft.epage%253D1600%26rft.atitle%253DAttention%2Bimproves%2Bperformance%2Bprimarily%2Bby%2Breducing%2Binterneuronal%2Bcorrelations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.2439%26rft_id%253Dinfo%253Apmid%252F19915566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.2439&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19915566&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000272065600020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.48" data-doi="10.1126/science.1233912">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brunton
                                  </span>
                                  <span class="cit-name-given-names">
                                   BW
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Botvinick
                                  </span>
                                  <span class="cit-name-given-names">
                                   MM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brody
                                  </span>
                                  <span class="cit-name-given-names">
                                   CD.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Rats and humans can optimally accumulate evidence for decision-making
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  340
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6128
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  95
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBrunton%26rft.auinit1%253DB.%2BW.%26rft.volume%253D340%26rft.issue%253D6128%26rft.spage%253D95%26rft.epage%253D98%26rft.atitle%253DRats%2Band%2BHumans%2BCan%2BOptimally%2BAccumulate%2BEvidence%2Bfor%2BDecision-Making%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1233912%26rft_id%253Dinfo%253Apmid%252F23559254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDAvNjEyOC85NSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA2LzE0LzIwMjAuMDEuMTAuOTAxNTA0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.49" data-doi="10.1016/j.neuron.2012.03.016">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beck
                                  </span>
                                  <span class="cit-name-given-names">
                                   JM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  <span class="cit-name-given-names">
                                   WJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pitkow
                                  </span>
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Latham
                                  </span>
                                  <span class="cit-name-given-names">
                                   PE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pouget
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Not noisy, just wrong: the role of suboptimal inference in behavioral variability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  30
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  39
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D74%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2012.03.016%26rft_id%253Dinfo%253Apmid%252F22500627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2012.03.016&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22500627&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000302893800006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.50" data-doi="10.1016/j.neunet.2004.04.004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Callaway
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Feedforward, feedback and inhibitory connections in primate visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural Networks
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5-6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  625
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  632
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2Bnetworks%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BNeural%2BNetwork%2BSociety%26rft.stitle%253DNeural%2BNetw%26rft.aulast%253DCallaway%26rft.auinit1%253DE.%2BM.%26rft.volume%253D17%26rft.issue%253D5-6%26rft.spage%253D625%26rft.epage%253D632%26rft.atitle%253DFeedforward%252C%2Bfeedback%2Band%2Binhibitory%2Bconnections%2Bin%2Bprimate%2Bvisual%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neunet.2004.04.004%26rft_id%253Dinfo%253Apmid%252F15288888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neunet.2004.04.004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15288888&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000223495600003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.51" data-doi="10.1038/nrn1888">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Averbeck
                                  </span>
                                  <span class="cit-name-given-names">
                                   BB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Latham
                                  </span>
                                  <span class="cit-name-given-names">
                                   PE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pouget
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Neural correlations, population coding and computation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature reviews neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  358
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  366
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BNeuroscience%26rft.stitle%253DNat%2BRev%2BNeurosci%26rft.aulast%253DAverbeck%26rft.auinit1%253DB.%2BB.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D358%26rft.epage%253D366%26rft.atitle%253DNeural%2Bcorrelations%252C%2Bpopulation%2Bcoding%2Band%2Bcomputation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn1888%26rft_id%253Dinfo%253Apmid%252F16760916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrn1888&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16760916&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000237072600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.52" data-doi="10.1038/s41586-019-0997-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chettih
                                  </span>
                                  <span class="cit-name-given-names">
                                   SN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harvey
                                  </span>
                                  <span class="cit-name-given-names">
                                   CD.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Single-neuron perturbations reveal feature-specific competition in V1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  567
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7748
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  334
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  340
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D567%26rft.spage%253D334%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-0997-6%26rft_id%253Dinfo%253Apmid%252F30842660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-0997-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30842660&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Valente
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pica
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bondanelli
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moroni
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Runyan
                                  </span>
                                  <span class="cit-name-given-names">
                                   CA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morcos
                                  </span>
                                  <span class="cit-name-given-names">
                                   AS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Correlations enhance the behavioral readout of neural population activity in association cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  975
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  986
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D24%26rft.spage%253D975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.54">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Payeur
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guerguiev
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zenke
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richards
                                  </span>
                                  <span class="cit-name-given-names">
                                   BA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naud
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Burst-dependent synaptic plasticity can coordinate learning in hierarchical circuits
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1010
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D24%26rft.spage%253D1010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.55" data-doi="10.1093/cercor/1.1.1-a">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Felleman
                                  </span>
                                  <span class="cit-name-given-names">
                                   DJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Essen
                                  </span>
                                  <span class="cit-name-given-names">
                                   DC.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Distributed hierarchical processing in the primate cerebral cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cerebral cortex (New York, NY: 1991)
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  47
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCerebral%2Bcortex%2B%2528New%2BYork%252C%2BNY%253A%2B1991%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcercor%252F1.1.1-a%26rft_id%253Dinfo%253Apmid%252F1822724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/cercor/1.1.1-a&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1822724&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000208047200002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pica
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Piasini
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Safaai
                                  </span>
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Runyan
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harvey
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Diamond
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Quantifying how much sensory information in a neural code is relevant for behavior
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Neural Information Processing Systems
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.57" data-doi="org/10.1038/s41586-019-1787-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                  <span class="cit-name-given-names">
                                   NA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zatka-Haas
                                  </span>
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carandini
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  <span class="cit-name-given-names">
                                   KD.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Distributed coding of choice, action and engagement across the mouse brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  576
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7786
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  266
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  273
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D576%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1038%252Fs41586-019-1787-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=org/10.1038/s41586-019-1787-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zylberberg
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pouget
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Latham
                                  </span>
                                  <span class="cit-name-given-names">
                                   PE
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shea-Brown
                                  </span>
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Robust information propagation through noisy neural circuits
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1005497
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D13%26rft.spage%253De1005497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bondy
                                  </span>
                                  <span class="cit-name-given-names">
                                   AG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haefner
                                  </span>
                                  <span class="cit-name-given-names">
                                   RM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   BG.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Feedback determines the structure of correlated variability in primary visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  598
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  606
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D21%26rft.spage%253D598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.60">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                  <span class="cit-name-given-names">
                                   SQ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergami
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  <span class="cit-name-given-names">
                                   SH.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Precise Long-Range Microcircuit-to-Microcircuit Communication Connects the Frontal and Sensory Cortices in the Mammalian Brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  385
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  401
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D104%26rft.spage%253D385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.61">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Onorato
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Neuenschwander
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoy
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lima
                                  </span>
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rocha
                                  </span>
                                  <span class="cit-name-given-names">
                                   KS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Broggini
                                  </span>
                                  <span class="cit-name-given-names">
                                   AC
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  A distinct class of bursting neurons with strong gamma synchronization and stimulus selectivity in monkey V1
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  180
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  197
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D105%26rft.spage%253D180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.62" data-doi="10.1016/j.tins.2012.11.006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Larkum
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  A cellular mechanism for cortical associations: an organizing principle for the cerebral cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in neurosciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  141
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  151
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bneurosciences%26rft.volume%253D36%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tins.2012.11.006%26rft_id%253Dinfo%253Apmid%252F23273272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tins.2012.11.006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23273272&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F06%2F14%2F2020.01.10.901504.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000316710400001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               63.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference 63. in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.01.10.901504v6.63" data-doi="10.1073/pnas.1720995115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naud
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sprekeler
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Sparse bursts optimize information transmission in a multiplexed neural code
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  27
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  E6329
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E6338
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1720995115%26rft_id%253Dinfo%253Apmid%252F29934400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzI3L0U2MzI5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDYvMTQvMjAyMC4wMS4xMC45MDE1MDQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.01.10.901504v6" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.13.472356v6" title="Spontaneous parthenogenesis in the parasitoid wasp Cotesia typhae: low frequency anomaly or evolving process?" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.06.10.495369v1" title="AAV-mediated neuronal expression of a scFv antibody selective for Aβ oligomers protects synapses and rescues memory in Alzheimer models" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted June 14, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.01.10.901504v6.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.01.10.901504v6.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.01.10.901504v6.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.01.10.901504v6" class="active" data-icon-position="" data-hide-link-title="0">
                                   Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2594161"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-TP8CVOax_m6IYjjT4dZLPNWV0UkA508dRM5mliD1eok"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484466402"/>
                                   <input type="hidden" name="captcha_token" value="45ae904b4a8fdb1f9ff3c48cdda729c9"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2594161--3" data-node-nid="2594161" id="node-2594161--41136301809" data-pisa="biorxiv;2020.01.10.901504v6" data-pisa-master="biorxiv;2020.01.10.901504" data-seqnum="2594161" data-apath="/biorxiv/early/2022/06/14/2020.01.10.901504.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Veronika
                                  </span>
                                  <span class="nlm-surname">
                                   Koren
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Ariana R.
                                  </span>
                                  <span class="nlm-surname">
                                   Andrei
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Ming
                                  </span>
                                  <span class="nlm-surname">
                                   Hu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Valentin
                                  </span>
                                  <span class="nlm-surname">
                                   Dragoi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Klaus
                                  </span>
                                  <span class="nlm-surname">
                                   Obermayer
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.01.10.901504;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.01.10.901504
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&title=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&title=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&text=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&t=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&title=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&title=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&title=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2594161--5" data-node-nid="2594161" id="citation-node-2594161--672779443" data-pisa="biorxiv;2020.01.10.901504v6" data-pisa-master="biorxiv;2020.01.10.901504" data-seqnum="2594161" data-apath="/biorxiv/early/2022/06/14/2020.01.10.901504.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Learning from invariants predicts upcoming behavioral choice from spiking activity in monkey V1
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Veronika
                                      </span>
                                      <span class="nlm-surname">
                                       Koren
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Ariana R.
                                      </span>
                                      <span class="nlm-surname">
                                       Andrei
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Ming
                                      </span>
                                      <span class="nlm-surname">
                                       Hu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Valentin
                                      </span>
                                      <span class="nlm-surname">
                                       Dragoi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Klaus
                                      </span>
                                      <span class="nlm-surname">
                                       Obermayer
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.01.10.901504;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.01.10.901504
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Koren%2C+Veronika&rft.au=Andrei%2C+Ariana+R.&rft.au=Hu%2C+Ming&rft.au=Dragoi%2C+Valentin&rft.au=Obermayer%2C+Klaus">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2594161/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2594161/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2594161/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2594161/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2594161/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2594161/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2594161/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2594161/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2594161/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2594161/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2594161/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2594161/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&count=horizontal&via=&text=Learning%20from%20invariants%20predicts%20upcoming%20behavioral%20choice%20from%20spiking%20activity%20in%20monkey%20V1&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.01.10.901504v6&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.01.10.901504v6" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
