<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.02.27.968933v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.02.27.968933v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.02.27.968933v4.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/04/07/2020.02.27.968933.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.02.27.968933v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems"/>
  <meta name="DC.Identifier" content="10.1101/2020.02.27.968933"/>
  <meta name="DC.Date" content="2022-04-07"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="ZFP-, TALE-, and CRISPR-based methods for genome, epigenome editing and imaging have provided powerful tools to interrogate functions of genomes. Targeting sequence design is vital to the success of these experiments. While existing design software mainly focus on designing target sequence for specific elements, we report here the implementation of JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator), a suite of software for enumerating all single- and multi-copy sites in the genome that can be incorporated for genome-scale designs as well as loaded onto genome browsers alongside other tracks for convenient web-based graphic-user-interface (GUI)-enabled design. We also implement fast algorithms to identify sequence neighborhoods or off-target counts of targeting sequences so that designs with low probability of off-target can be identified among millions of design sequences in reasonable time. We demonstrate the application of JACKIE-designed CRISPR site clusters for genome imaging.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Jacqueline Jufen Zhu"/>
  <meta name="DC.Contributor" content="Albert Wu Cheng"/>
  <meta name="article:published_time" content="2022-04-07"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems"/>
  <meta name="citation_abstract" lang="en" content="<h3>ABSTRACT</h3>
<p>ZFP-, TALE-, and CRISPR-based methods for genome, epigenome editing and imaging have provided powerful tools to interrogate functions of genomes. Targeting sequence design is vital to the success of these experiments. While existing design software mainly focus on designing target sequence for specific elements, we report here the implementation of JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator), a suite of software for enumerating all single- and multi-copy sites in the genome that can be incorporated for genome-scale designs as well as loaded onto genome browsers alongside other tracks for convenient web-based graphic-user-interface (GUI)-enabled design. We also implement fast algorithms to identify sequence neighborhoods or off-target counts of targeting sequences so that designs with low probability of off-target can be identified among millions of design sequences in reasonable time. We demonstrate the application of JACKIE-designed CRISPR site clusters for genome imaging.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.02.27.968933v4"/>
  <meta name="citation_id" content="2020.02.27.968933v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.02.27.968933"/>
  <meta name="citation_num_pages" content="17"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.02.27.968933"/>
  <meta name="citation_author" content="Jacqueline Jufen Zhu"/>
  <meta name="citation_author_institution" content="School of Biological and Health Systems Engineering, Arizona State University"/>
  <meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6416-4563"/>
  <meta name="citation_author" content="Albert Wu Cheng"/>
  <meta name="citation_author_institution" content="School of Biological and Health Systems Engineering, Arizona State University"/>
  <meta name="citation_author_institution" content="The Jackson Laboratory for Genomic Medicine"/>
  <meta name="citation_author_institution" content="The Jackson Laboratory Cancer Center"/>
  <meta name="citation_author_institution" content="Department of Genetics and Genome Sciences, University of Connecticut Health Center"/>
  <meta name="citation_author_institution" content="Institute for Systems Genomics, University of Connecticut Health Center"/>
  <meta name="citation_author_email" content="albert@cheng.bio"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2653-5743"/>
  <meta name="citation_reference" content="Higashikuni, Y. and T.K. Lu, Advancing CRISPR-Based Programmable Platforms beyond Genome Editing in Mammalian Cells. ACS Synth Biol, 2019. 8(12): p. 2607–2619."/>
  <meta name="citation_reference" content="Wang, F., et al., Advances in CRISPR-Cas systems for RNA targeting, tracking and editing. Biotechnol Adv, 2019. 37(5): p. 708–729."/>
  <meta name="citation_reference" content="Sander, J.D. and J.K. Joung, CRISPR-Cas systems for editing, regulating and targeting genomes. Nat Biotechnol, 2014. 32(4): p. 347–55."/>
  <meta name="citation_reference" content="Cheng, A.W., et al., Casilio: a versatile CRISPR-Cas9-Pumilio hybrid for gene regulation and genomic labeling. Cell Res, 2016. 26(2): p. 254–7."/>
  <meta name="citation_reference" content="Maass, P.G., et al., Spatiotemporal allele organization by allele-specific CRISPR live-cell imaging (SNP-CLING). Nat Struct Mol Biol, 2018. 25(2): p. 176–184."/>
  <meta name="citation_reference" content="Ma, H., et al., CRISPR-Sirius: RNA scaffolds for signal amplification in genome imaging. Nat Methods, 2018. 15(11): p. 928–931."/>
  <meta name="citation_reference" content="Qin, P., et al., Live cell imaging of low- and non-repetitive chromosome loci using CRISPR-Cas9. Nat Commun, 2017. 8: p. 14725."/>
  <meta name="citation_reference" content="Zhu, J.J. and A.W. Cheng, JACKIE: Fast enumeration of single- and multi-copy CRISPR binding sites. bioRxiv, 2020. 2020.02.27.968933."/>
  <meta name="citation_reference" content="Bae, S., J. Park, and J.S. Kim, Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics, 2014. 30(10): p. 1473–5."/>
  <meta name="citation_reference" content="Haeussler, M., et al., Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR. Genome Biol, 2016. 17(1): p. 148."/>
  <meta name="citation_reference" content="Bradford, J. and D. Perrin, A benchmark of computational CRISPR-Cas9 guide design methods. PLoS Comput Biol, 2019. 15(8): p. e1007274."/>
  <meta name="citation_reference" content="Kent, W.J., et al., The human genome browser at UCSC. Genome Res, 2002. 12(6): p. 996–1006."/>
  <meta name="citation_reference" content="McKenna, A. and J. Shendure, FlashFry: a fast and flexible tool for large-scale CRISPR target design. BMC Biol, 2018. 16(1): p. 74."/>
  <meta name="citation_reference" content="Cancellieri, S., et al., CRISPRitz: rapid, high-throughput and variant-aware in silico off-target site identification for CRISPR genome editing. Bioinformatics, 2020. 36(7): p. 2001–2008."/>
  <meta name="twitter:title" content="JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="ZFP-, TALE-, and CRISPR-based methods for genome, epigenome editing and imaging have provided powerful tools to interrogate functions of genomes. Targeting sequence design is vital to the success of these experiments. While existing design software mainly focus on designing target sequence for specific elements, we report here the implementation of JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator), a suite of software for enumerating all single- and multi-copy sites in the genome that can be incorporated for genome-scale designs as well as loaded onto genome browsers alongside other tracks for convenient web-based graphic-user-interface (GUI)-enabled design. We also implement fast algorithms to identify sequence neighborhoods or off-target counts of targeting sequences so that designs with low probability of off-target can be identified among millions of design sequences in reasonable time. We demonstrate the application of JACKIE-designed CRISPR site clusters for genome imaging.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="ZFP-, TALE-, and CRISPR-based methods for genome, epigenome editing and imaging have provided powerful tools to interrogate functions of genomes. Targeting sequence design is vital to the success of these experiments. While existing design software mainly focus on designing target sequence for specific elements, we report here the implementation of JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator), a suite of software for enumerating all single- and multi-copy sites in the genome that can be incorporated for genome-scale designs as well as loaded onto genome browsers alongside other tracks for convenient web-based graphic-user-interface (GUI)-enabled design. We also implement fast algorithms to identify sequence neighborhoods or off-target counts of targeting sequences so that designs with low probability of off-target can be identified among millions of design sequences in reasonable time. We demonstrate the application of JACKIE-designed CRISPR site clusters for genome imaging.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-04-07"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2485400"/>
  <title>
   JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"f4dpwarXjvEp2n5VeRc7Uy1G-zMT1pTz5LhE0NHP4b0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2485400","apath":"\/biorxiv\/early\/2022\/04\/07\/2020.02.27.968933.atom","pisa":"biorxiv;2020.02.27.968933v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.02.27.968933v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.02.27.968933v4","title":"JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems","identifier":"node\/2485400"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CAfhvh77JdfR9KeRS46K9UYGIO4Cg25VCMX3FJcbvX9QobiG99poDPNH72WfUxeL38vt85fmA8b\\\\\\\/\\\\\\\/foJ\\\\\\\/neAJPtILFv7izqiwFaXdxGC1veEKi8FbBJvzp\\\\\\\/f8GAsrZV9ZtxW3zi0F3Pd17TrzhYit6RUAXo+P07IqIppw1V0mJ7anwRWYyMMmWO4eILUE6dMVnMQq4PlYuB5N8yknyS68\\\\\\\/+Aikom\\\\\\\/EedvRnYJPh9PrMcxRmZ6maZDEEEN7qDTHqt8tDOo\\\\\\\/e3cddAjIiA4xjtVPjhzV1HKkyswoL8MAXHo23DxyLulOEcr7tPDeDF1oyWweJ1trqNcadX6BLM41V1xqYy280M7YRH+3nrW+uIC3MoilvnpJD4mkCEozNl9xH6onb7J0uxhHGRbii4L0k\\\\\\\/TMx13A2DZT8cNAOre4AnYL7sWaHyl8pGAk8jU1WMBRTDC3H1sFCd57uE2p9SN8LqmDuWRPynTjyUPv7lygvBCJP9A1VT\\\\\\\/sIRjC4AxvaT0zHBDXgjL09Y7R2RX4ogRyje\\\\\\\/Gu24v7Jx\\\\\\\/KNa5QpS107\\\\\\\/5dwftwCHPEhYaemxckXK9sKrejLoXCqxsRpYucBFBCIed8A3Kd1eHA8DkXpH3Ny4Z0fcQ3rn4wyDk\\\\\\\/Dufy0T+q6p8diLhUYn9TPayDSFi\\\\\\\/0OOGMz1R22muNLx7oz6LpRsU0fibZiBpWI5KKburYkAdZr\\\\\\\/6vrSUfhl+8EuXPAGtKUKFrIHTO8BGmw2GhBjaZB3RT\\\\\\\/veV\\\\\\\/OMyGLXaGZ\\\\\\\/B87yWSg3z5\\\\\\\/lUh+\\\\\\\/y9Ch6ltbYj5laoM74oZmhONYR4KsNa1cggds6g0xi2K9j1ue9RnS6XB8eOQbnIEjXYzZYlE8GiVyucGsePOpUYjxTNutj3QHqUqfBeYIYOBWGEaHkonvvVxfHr6h5xZUCVbdeFmE8C79e9QRT2SIXf35mLw8QWV9yebz7v\\\\\\\/UppDZc\\\\\\\/fvfclM3W9aIjufCxdhWQFmlA\\\\\\\/rOEb\\\\\\\/e45QSr1ri4+F+HOch0xw8Iu1e65re0BLdP1rrbWmO15WAm51T8GE\\\\\\\/hQJTRFT0TyQsZuaq2\\\\\\\/SvKOPAzGbBEwIuYQhaGnf7fDfVFqQEVStA6st+htUfqt8g\\\\\\\/RUbtkhyk2\\\\\\\/I1\\\\\\\/DEwLBNe0c6PfyY1UWZqdqEQuz8Z7DhGsrenf0ZNVHsR1W9Hg8CA1DuDNLWmcFctY2rp3sYGtFv7CG75bu\\\\\\\/aEQ6cbCi3rNAKBpXJsnqgwSHvnhXei2to08XMwHrICu4r\\\\\\\/OOr\\\\\\\/EdjENHFtF7QntiX90Dpyd5XPMLeOMTuyg\\\\\\\/McxL9DdrrNoQ\\\\\\\/alBvwMPcF23a5fWo5QUH1iOhaIMoKzN9f3UgUujT3lTr6g\\\\\\\/7fzPNcNRmcH6uMtmhSkLMSddMGWGRG0HL7XIQJb3zLSH330=\\\u0022,\\\u0022iv\\\u0022:\\\u0022rbazUmW3gpFYGP8BSopK0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222c2b7e84c315fb4818d48120d5f90bfa7d9ae6119f7121eceb50f257d627be8f\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.02.27.968933v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2485400 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.02.27.968933v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_994596613">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_994596613" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1865609282" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-4H945uqih0BBoO7UA2B3qDq09V0ksMMYFMRabaublDM"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2485400" data-node-nid="2485400" id="node-2485400--2966488972" data-pisa="biorxiv;2020.02.27.968933v4" data-pisa-master="biorxiv;2020.02.27.968933" data-apath="/biorxiv/early/2022/04/07/2020.02.27.968933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-6416-4563" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jacqueline Jufen
                      </span>
                      <span class="nlm-surname">
                       Zhu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-2653-5743" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Albert Wu
                      </span>
                      <span class="nlm-surname">
                       Cheng
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.02.27.968933
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2485400--2966488972" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Jacqueline Jufen Zhu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Biological and Health Systems Engineering, Arizona State University
                       </span>
                       , Tempe, AZ 85281,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        The Jackson Laboratory for Genomic Medicine
                       </span>
                       , Farmington, CT 06032,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jacqueline%2BJufen%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhu%20JJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJacqueline%2BJufen%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6416-4563" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jacqueline Jufen Zhu
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Albert Wu Cheng
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        School of Biological and Health Systems Engineering, Arizona State University
                       </span>
                       , Tempe, AZ 85281,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        The Jackson Laboratory for Genomic Medicine
                       </span>
                       , Farmington, CT 06032,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        The Jackson Laboratory Cancer Center
                       </span>
                       , Bar Harbor, ME 04609,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Genetics and Genome Sciences, University of Connecticut Health Center
                       </span>
                       , Farmington, CT 06030,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Institute for Systems Genomics, University of Connecticut Health Center
                       </span>
                       , Farmington, CT 06030,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Albert%2BWu%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Cheng%20AW&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlbert%2BWu%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-2653-5743" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Albert Wu Cheng
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:albert@cheng.bio" class="" data-icon-position="" data-hide-link-title="0">
                        albert@cheng.bio
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.02.27.968933v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.27.968933v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.27.968933v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.27.968933v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.02.27.968933v4.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.02.27.968933v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2485400" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2485400/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             ABSTRACT
                            </h2>
                            <p id="p-2">
                             ZFP-, TALE-, and CRISPR-based methods for genome, epigenome editing and imaging have provided powerful tools to interrogate functions of genomes. Targeting sequence design is vital to the success of these experiments. While existing design software mainly focus on designing target sequence for specific elements, we report here the implementation of JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator), a suite of software for enumerating all single- and multi-copy sites in the genome that can be incorporated for genome-scale designs as well as loaded onto genome browsers alongside other tracks for convenient web-based graphic-user-interface (GUI)-enabled design. We also implement fast algorithms to identify sequence neighborhoods or off-target counts of targeting sequences so that designs with low probability of off-target can be identified among millions of design sequences in reasonable time. We demonstrate the application of JACKIE-designed CRISPR site clusters for genome imaging.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             INTRODUCTION
                            </h2>
                            <p id="p-6">
                             Zinc finger protein (ZFP), Transcription Activator Like Effector (TALE) and CRISPR-Cas system have revolutionized genome biology and synthetic biology by providing versatile DNA and RNA-targeting modalities for engineering [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             -
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. These technologies have revolutionized genome research by providing versatile tools for “writing” the genome, epigenome and transcriptome as well as “reading” the dynamics of the genome architecture through live-cell genome imaging. The success of these experiments depends heavily on the design of target sequences within the genome. While genome and epigenome editing experiments usually requires target sequences that occur only once in the genome, genome imaging require clustered repetitive sequences [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             -
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ]. Most of the existing design software packages provide small scale or one-by-one design [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             -
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. For large-scale design, these software packages are inefficient. Furthermore, for genome imaging where clustered repetitive sequences are needed, no design software is available.
                            </p>
                            <p id="p-7">
                             For enumeration of all potential target sites of length k (k-mers) and their locations in the genome, we implemented JACKIE (Jackie and Albert’s Comprehensive K-mer Instances Enumerator) software package that is compatible with high performance computing (HPC) clusters. This allow the generation of databases of single- or multi-copy target sites that can not only be used in genome-wide library design but can also be loaded as genome tracks on genome browsers [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ] alongside other genomic tracks to allow for GUI-enabled design of targeting experiments. To demonstrate application, we generated databases of CRISPR target sites in human hg38 and mouse mm10 genomes and selected several clustered gRNAs for genomic imaging experiments.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             MATERIALS AND METHODS
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Implementation of JACKIE
                             </h3>
                             <p id="p-8">
                              JACKIE consists of four sets of programs implemented in C++,
                              <kbd>
                               JACKIE.bin
                              </kbd>
                              ,
                              <kbd>
                               JACKIE.sortToBed
                              </kbd>
                              , (
                              <kbd>
                               JACKIE.encodeSeqSpace, JACKIE.encodeSeqSpaceNGG, JACKIE.encodeSeqSpace.prefixed, JACKIE.encodeSeqCountDatabase
                              </kbd>
                              ) and (
                              <kbd>
                               JACKIE.countSeqNeighbors, JACKIE.countSeqNeighbors.pmulti, JACKIE.countOffSites
                              </kbd>
                              ), as well as Python and Bash scripts for job batching and downstream processing (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig 1
                              </a>
                              ).
                              <kbd>
                               JACKIE.bin
                              </kbd>
                              scans the genome and enumerates any potential targeting sites (k-mers) on both strands (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig 1a
                              </a>
                              ). A pattern-matching parameter allows the specification of length k of the k-mer binding sites, as well as motif constraints (such as NGG in the case of CRISPR-SpCas9). To enhance memory and computation efficiency, k-mers are mapped to an unsigned 64-bit integer (
                              <kbd>
                               uint64_t NucKey
                              </kbd>
                              ) by encoding each nucleotide and an end-of-string (EOS) indicator in 3 bits, A->000, C->001, T->010, G->011 and EOS->111 (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig 1a
                              </a>
                              ). The chromosome (unsigned 32-bit integer,
                              <kbd>
                               uint32_t chrID
                              </kbd>
                              ) and location for each sequence instance (signed 32-bit integer,
                              <kbd>
                               int32_t pos
                              </kbd>
                              ) are recorded such that chID maps to a chromosome through a tab-delimited table while the sign of
                              <kbd>
                               pos
                              </kbd>
                              registers the positive (+) and negative (-) strands. The aggregate
                              <kbd>
                               KeyedPosition
                              </kbd>
                              data (
                              <kbd>
                               NucKey, ChrID, pos
                              </kbd>
                              ) generated at each position during the genome scan are written into files prefixed by a predefined prefix length (default to 6, e.g., ATGAGC.bin contains all
                              <kbd>
                               KeyedPosition
                              </kbd>
                              of k-mers prefixed by ATGAGC) so that the next steps can be highly parallelized. The
                              <kbd>
                               JACKIE.bin
                              </kbd>
                              operations can be divided into four subtasks each working on k-mers prefixed by each of the four bases (e.g., the “A” subtask generates all AXXXXX.bin files and an A.ref.txt chromosome table). Upon the completion of the
                              <kbd>
                               JACKIE.bin
                              </kbd>
                              operations,
                              <kbd>
                               JACKIE.sortToBed
                              </kbd>
                              is run on each bin file by loading the
                              <kbd>
                               KeyedPosition
                              </kbd>
                              records into a list in memory, and sorting them by
                              <kbd>
                               NucKey
                              </kbd>
                              , thus placing
                              <kbd>
                               KeyedPosition
                              </kbd>
                              records of identical k-mers adjacent to each other in the list (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig 1b
                              </a>
                              ). The
                              <kbd>
                               NucKey
                              </kbd>
                              -sorted list is then traversed to output k-mer sites in bed interval files that can be concatenated to generate a genome-wide target site database (JACKIEdb) (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig 1c
                              </a>
                              ). The extensive divide-and-conquer design and binary encoding of sequences allow
                              <kbd>
                               JACKIE
                              </kbd>
                              to be highly parallelized on an HPC cluster as well as efficient sorting of sequences. A python script (
                              <kbd>
                               chainExonBedsToTranscriptBed.py
                              </kbd>
                              ) can be used to take bed file outputs from
                              <kbd>
                               JACKIE
                              </kbd>
                              to further collapse same-chromosome target sites into an extended bed interval file format with each record containing locations of sites with the same k-mer encoded as bed file blocks to appear as chains in genome browsers (see
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig 2b
                              </a>
                              ). Other helper scripts (
                              <kbd>
                               fastjoinBedByOverlap.py
                              </kbd>
                              ,
                              <kbd>
                               joinBedItemsWithinRadius.py
                              </kbd>
                              ) can be used for downstream processing, such as for the fast extraction of target sites overlapping defined set of intervals. target sequences, number of occurrences, and span sizes (the distance between the 5’ most site to 3’ most site on the same chromosome) are encoded in the output files so that target sites can be filtered by simple text processing software (e.g., awk) to identify sites or site clusters with desired properties.
                             </p>
                             <div id="F1" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F1.large.jpg?width=800&height=600&carousel=1" title="Implementation of JACKIE. (a) JACKIE.bin scans the genome for k-mers matching a specified motif (e.g., XXXXXXXXXXXXXXXXXXXXNGG matches 20-mer denoted by 20X followed by NGG proto-adjacent motif (PAM) required by the CRISPR/SpCas9 system), and encode each occurrence in a data structure called KeyedPosition, consisting of NucKey (unsigned 64-bit integer, uint64_t), chrID (unsigned 32-bit integer, uint32_t), and pos (signed 32-bit integer, int32_t). NucKey records a binary representation of the k-mer sequence using 3 bits per nucleotide. chrID records an integer representing a chromosome. A chrID-chromosome mapping file is generated. pos records the location of the binding site, with negative and positive integers representing the minus strand and positive strand, respectively. JACKIE.bin outputs KeyedPosition to files as it scans the genome. To parallelize this step, four processes are started as separate cluster jobs, each focusing on the A, C, T, or G as the first nucleotide of k-mer. To allow for parallelization in subsequent steps, KeyedPosition records are appended to files per 6-mer prefix of the k-mer (.bin, e.g., ATGAGC.bin contains all records for all k-mer starting with ATGAGC). (b) JACKIE.sortToBed loads each .bin file and performs a sort on the KeyedPosition records on the NucKey variable and then traverse the sorted list of KeyedPosition to output a bed interval file containing the coordinate position of each k-mer site with item named by ./, and score field recording the copy number. Parallelization is achieved by either starting separate jobs on individual .bin files, or by starting batch jobs on 2mer prefices (e.g., AA AT AC AG operating on AA*.bin, AT*.bin, AC*.bin, AG*.bin, etc). (c) The outputs (.bed) are then merged into a combined bed file (JACKIEdb) using cat Unix command. (d) Evaluation of off-target effects starts with generating a bit array representation of sequence space (SeqSpace) of the target genome. JACKIE.encodeSeqSpace initializes a 4k-bit (i.e., 42k−3 bytes) array to zeros as an “empty” SeqSpace. The program scans the genome and record the presence of each encountered k-mer by first translating the sequence to a binary index, and then setting the bit on the SeqSpace referenced by the index to one. The reverse-complement k-mer is handled similarly. A sliding mechanism that shifts two bits as the genome advance to the next nucleotide is used for both indices for time efficiency. (e) JACKIE.countSeqNeighbors reads in SeqSpace and a table of query sequences (e.g., target site design bed files from JACKIE.sortToBed) and compute the numbers of sequence neighbors with different number of mismatches up to a specified threshold. At start, JACKIE.countSeqNeighbors builds a collection of position AND-masks and pos_mut OR-masks that when applied to a query sequence encoded in the binary representation generates all possible neighbor sequences up to the specified mismatch number. The resultant query_mut values then serve as the indices to query the SeqSpace array. If the corresponding bit of the array is 1, the sequence neighbor count for that particular mismatch number is incremented by one. After going through all pairs of AND-mask and OR-mask, the sequence neighbor counts for each mismatch number up to the specified threshold are reported for that particular query sequence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-84547977" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Implementation of JACKIE.</span> <strong>(a)</strong> <kbd>JACKIE.bin</kbd> scans the genome for k-mers matching a specified motif (e.g., XXXXXXXXXXXXXXXXXXXXNGG matches 20-mer denoted by 20X followed by NGG proto-adjacent motif (PAM) required by the CRISPR/SpCas9 system), and encode each occurrence in a data structure called <kbd>KeyedPosition</kbd>, consisting of NucKey (unsigned 64-bit integer, <kbd>uint64_t</kbd>), <kbd>chrID</kbd> (unsigned 32-bit integer, <kbd>uint32_t</kbd>), and <kbd>pos</kbd> (signed 32-bit integer, <kbd>int32_t</kbd>). NucKey records a binary representation of the k-mer sequence using 3 bits per nucleotide. <kbd>chrID</kbd> records an integer representing a chromosome. A <kbd>chrID</kbd>-chromosome mapping file is generated. pos records the location of the binding site, with negative and positive integers representing the minus strand and positive strand, respectively. <kbd>JACKIE.bin</kbd> outputs <kbd>KeyedPosition</kbd> to files as it scans the genome. To parallelize this step, four processes are started as separate cluster jobs, each focusing on the A, C, T, or G as the first nucleotide of k-mer. To allow for parallelization in subsequent steps, <kbd>KeyedPosition</kbd> records are appended to files per 6-mer prefix of the k-mer (<6merPrefix>.bin, e.g., ATGAGC.bin contains all records for all k-mer starting with ATGAGC). <strong>(b)</strong> <kbd>JACKIE.sortToBed</kbd> loads each <6merPrefix>.bin file and performs a sort on the <kbd>KeyedPosition</kbd> records on the <kbd>NucKey</kbd> variable and then traverse the sorted list of <kbd>KeyedPosition</kbd> to output a bed interval file containing the coordinate position of each k-mer site with item named by <NucKey>.<copy number>/<sequence>, and score field recording the copy number. Parallelization is achieved by either starting separate jobs on individual <6merPrefix>.bin files, or by starting batch jobs on 2mer prefices (e.g., AA AT AC AG operating on AA*.bin, AT*.bin, AC*.bin, AG*.bin, etc). <strong>(c)</strong> The outputs (<6merPrefix>.bed) are then merged into a combined bed file (JACKIEdb) using cat Unix command. <strong>(d)</strong> Evaluation of off-target effects starts with generating a bit array representation of sequence space (<kbd>SeqSpace</kbd>) of the target genome. <kbd>JACKIE.encodeSeqSpace</kbd> initializes a 4<sup><em>k</em></sup>-bit (i.e., 4<sup>2<em>k</em>−3</sup> bytes) array to zeros as an “empty” <kbd>SeqSpace</kbd>. The program scans the genome and record the presence of each encountered k-mer by first translating the sequence to a binary index, and then setting the bit on the <kbd>SeqSpace</kbd> referenced by the index to one. The reverse-complement k-mer is handled similarly. A sliding mechanism that shifts two bits as the genome advance to the next nucleotide is used for both indices for time efficiency. <strong>(e)</strong> <kbd>JACKIE.countSeqNeighbors</kbd> reads in <kbd>SeqSpace</kbd> and a table of query sequences (e.g., target site design bed files from <kbd>JACKIE.sortToBed</kbd>) and compute the numbers of sequence neighbors with different number of mismatches up to a specified threshold. At start, <kbd>JACKIE.countSeqNeighbors</kbd> builds a collection of position AND-masks and pos_mut OR-masks that when applied to a query sequence encoded in the binary representation generates all possible neighbor sequences up to the specified mismatch number. The resultant query_mut values then serve as the indices to query the <kbd>SeqSpace</kbd> array. If the corresponding bit of the array is 1, the sequence neighbor count for that particular mismatch number is incremented by one. After going through all pairs of AND-mask and OR-mask, the sequence neighbor counts for each mismatch number up to the specified threshold are reported for that particular query sequence.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F1.medium.gif" width="440" height="295"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F1.medium.gif" width="440" height="295"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig 1.
                               </span>
                               <span class="caption-title">
                                Implementation of JACKIE.
                               </span>
                               <p id="p-9" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                <kbd>
                                 JACKIE.bin
                                </kbd>
                                scans the genome for k-mers matching a specified motif (e.g., XXXXXXXXXXXXXXXXXXXXNGG matches 20-mer denoted by 20X followed by NGG proto-adjacent motif (PAM) required by the CRISPR/SpCas9 system), and encode each occurrence in a data structure called
                                <kbd>
                                 KeyedPosition
                                </kbd>
                                , consisting of NucKey (unsigned 64-bit integer,
                                <kbd>
                                 uint64_t
                                </kbd>
                                ),
                                <kbd>
                                 chrID
                                </kbd>
                                (unsigned 32-bit integer,
                                <kbd>
                                 uint32_t
                                </kbd>
                                ), and
                                <kbd>
                                 pos
                                </kbd>
                                (signed 32-bit integer,
                                <kbd>
                                 int32_t
                                </kbd>
                                ). NucKey records a binary representation of the k-mer sequence using 3 bits per nucleotide.
                                <kbd>
                                 chrID
                                </kbd>
                                records an integer representing a chromosome. A
                                <kbd>
                                 chrID
                                </kbd>
                                -chromosome mapping file is generated. pos records the location of the binding site, with negative and positive integers representing the minus strand and positive strand, respectively.
                                <kbd>
                                 JACKIE.bin
                                </kbd>
                                outputs
                                <kbd>
                                 KeyedPosition
                                </kbd>
                                to files as it scans the genome. To parallelize this step, four processes are started as separate cluster jobs, each focusing on the A, C, T, or G as the first nucleotide of k-mer. To allow for parallelization in subsequent steps,
                                <kbd>
                                 KeyedPosition
                                </kbd>
                                records are appended to files per 6-mer prefix of the k-mer (<6merPrefix>.bin, e.g., ATGAGC.bin contains all records for all k-mer starting with ATGAGC).
                                <strong>
                                 (b)
                                </strong>
                                <kbd>
                                 JACKIE.sortToBed
                                </kbd>
                                loads each <6merPrefix>.bin file and performs a sort on the
                                <kbd>
                                 KeyedPosition
                                </kbd>
                                records on the
                                <kbd>
                                 NucKey
                                </kbd>
                                variable and then traverse the sorted list of
                                <kbd>
                                 KeyedPosition
                                </kbd>
                                to output a bed interval file containing the coordinate position of each k-mer site with item named by <NucKey>.<copy number>/<sequence>, and score field recording the copy number. Parallelization is achieved by either starting separate jobs on individual <6merPrefix>.bin files, or by starting batch jobs on 2mer prefices (e.g., AA AT AC AG operating on AA*.bin, AT*.bin, AC*.bin, AG*.bin, etc).
                                <strong>
                                 (c)
                                </strong>
                                The outputs (<6merPrefix>.bed) are then merged into a combined bed file (JACKIEdb) using cat Unix command.
                                <strong>
                                 (d)
                                </strong>
                                Evaluation of off-target effects starts with generating a bit array representation of sequence space (
                                <kbd>
                                 SeqSpace
                                </kbd>
                                ) of the target genome.
                                <kbd>
                                 JACKIE.encodeSeqSpace
                                </kbd>
                                initializes a 4
                                <sup>
                                 k
                                </sup>
                                -bit (i.e., 4
                                <sup>
                                 2k−3
                                </sup>
                                bytes) array to zeros as an “empty”
                                <kbd>
                                 SeqSpace
                                </kbd>
                                . The program scans the genome and record the presence of each encountered k-mer by first translating the sequence to a binary index, and then setting the bit on the
                                <kbd>
                                 SeqSpace
                                </kbd>
                                referenced by the index to one. The reverse-complement k-mer is handled similarly. A sliding mechanism that shifts two bits as the genome advance to the next nucleotide is used for both indices for time efficiency.
                                <strong>
                                 (e)
                                </strong>
                                <kbd>
                                 JACKIE.countSeqNeighbors
                                </kbd>
                                reads in
                                <kbd>
                                 SeqSpace
                                </kbd>
                                and a table of query sequences (e.g., target site design bed files from
                                <kbd>
                                 JACKIE.sortToBed
                                </kbd>
                                ) and compute the numbers of sequence neighbors with different number of mismatches up to a specified threshold. At start,
                                <kbd>
                                 JACKIE.countSeqNeighbors
                                </kbd>
                                builds a collection of position AND-masks and pos_mut OR-masks that when applied to a query sequence encoded in the binary representation generates all possible neighbor sequences up to the specified mismatch number. The resultant query_mut values then serve as the indices to query the
                                <kbd>
                                 SeqSpace
                                </kbd>
                                array. If the corresponding bit of the array is 1, the sequence neighbor count for that particular mismatch number is incremented by one. After going through all pairs of AND-mask and OR-mask, the sequence neighbor counts for each mismatch number up to the specified threshold are reported for that particular query sequence.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-fig odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F2.large.jpg?width=800&height=600&carousel=1" title="Example use cases for JACKIE designs. (a) A screenshot of UCSC genome browser showing CRISPR/SpCas9 gRNA clusters. Top: Example track filter settings for 30 or fewer total off-targets up to 3-mismatches, 40-60%GC spacer, no 1-mismatch off-targets, 20 or more sites in cluster, spacer not containing more than runs of 4 or more T, cluster smaller than or equal to 5kb. Middle: Example gRNA clusters after filtering. Bottom: Information page of the first cluster shown after clicking on the item. (b) screenshot of UCSC genome browser showing single-copy CRISPR-SpCas9 gRNA sites. Top: Example track filter settings for up to 2 off-target sites, 40-60%GC spacer, no 1-mismatch off-targets, spacer not containing more than runs of 4 or more T. (c) To demonstrate genomic imaging use case, we selected three gRNA clusters and performed CRISPR genome imaging experiment. UCSC genome browser screenshots are shown on the left with the line-bar annotating the CRISPR sites, and representative fluorescence microscopy images shown on the right. Microscopy images were derived by merging green (CRISPR) and blue (DAPI: nucleus stain). Arrows point to fluorescent loci of interest. Scale bars, 5μm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-84547977" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Example use cases for JACKIE designs.</span> <strong>(a)</strong> A screenshot of UCSC genome browser showing CRISPR/SpCas9 gRNA clusters. Top: Example track filter settings for 30 or fewer total off-targets up to 3-mismatches, 40-60%GC spacer, no 1-mismatch off-targets, 20 or more sites in cluster, spacer not containing more than runs of 4 or more T, cluster smaller than or equal to 5kb. Middle: Example gRNA clusters after filtering. Bottom: Information page of the first cluster shown after clicking on the item. <strong>(b)</strong> screenshot of UCSC genome browser showing single-copy CRISPR-SpCas9 gRNA sites. Top: Example track filter settings for up to 2 off-target sites, 40-60%GC spacer, no 1-mismatch off-targets, spacer not containing more than runs of 4 or more T. <strong>(c)</strong> To demonstrate genomic imaging use case, we selected three gRNA clusters and performed CRISPR genome imaging experiment. UCSC genome browser screenshots are shown on the left with the line-bar annotating the CRISPR sites, and representative fluorescence microscopy images shown on the right. Microscopy images were derived by merging green (CRISPR) and blue (DAPI: nucleus stain). Arrows point to fluorescent loci of interest. Scale bars, 5μm.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F2.medium.gif" width="375" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F2.medium.gif" width="375" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/04/07/2020.02.27.968933/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig 2.
                               </span>
                               <span class="caption-title">
                                Example use cases for JACKIE designs.
                               </span>
                               <p id="p-10" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                A screenshot of UCSC genome browser showing CRISPR/SpCas9 gRNA clusters. Top: Example track filter settings for 30 or fewer total off-targets up to 3-mismatches, 40-60%GC spacer, no 1-mismatch off-targets, 20 or more sites in cluster, spacer not containing more than runs of 4 or more T, cluster smaller than or equal to 5kb. Middle: Example gRNA clusters after filtering. Bottom: Information page of the first cluster shown after clicking on the item.
                                <strong>
                                 (b)
                                </strong>
                                screenshot of UCSC genome browser showing single-copy CRISPR-SpCas9 gRNA sites. Top: Example track filter settings for up to 2 off-target sites, 40-60%GC spacer, no 1-mismatch off-targets, spacer not containing more than runs of 4 or more T.
                                <strong>
                                 (c)
                                </strong>
                                To demonstrate genomic imaging use case, we selected three gRNA clusters and performed CRISPR genome imaging experiment. UCSC genome browser screenshots are shown on the left with the line-bar annotating the CRISPR sites, and representative fluorescence microscopy images shown on the right. Microscopy images were derived by merging green (CRISPR) and blue (DAPI: nucleus stain). Arrows point to fluorescent loci of interest. Scale bars, 5μm.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-11">
                              Off-target effects of engineered nucleases happen when non-target sites with similar sequence to the target sites are recognized and whereat unwanted effects are induced. To prevent or reduce off-target effects, the simplest method is to filter for target sites with fewer predicted off-target locations having same or similar sequences. One of the most popular software for CRISPR off-target predictions is Cas-OFFinder which enumerates sequences and their occurrences with up to a certain number of mismatches to the designed gRNAs [
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ]. While very efficient in identifying off-target sites for a few gRNAs, evaluation of millions of gRNA designs using Cas-OFFinder would be impractical. In many cases, the goal in identifying a specific target sequence is not about the exact locations of the off-target sites, but the number of off-target sites with a different number of mismatches. And in the case of genome imaging and epigenome editing where there is a relatively higher degree of freedom for target site selection, i.e., the target sites only need to be at close proximity of the element of interest or within a relatively large genomic region, the problem then becomes finding target sites with few or no off-target sequences with up to a certain number of mismatches. Motivated by this, we implement off-target evaluation strategy in terms of “neighborhood” sequences of target sites in the sequence space. The goal is to identify the number of k-mers sequences (in terms of their presence or absence but not their actual locations nor copy numbers) that are certain number of mismatches (e.g., 3), or hamming distances, away from the query design sequence. The off-target prediction routines consist of two programs:
                              <kbd>
                               JACKIE.encodeSeqSpace
                              </kbd>
                              (or
                              <kbd>
                               JACKIE.encodeSeqSpaceNGG
                              </kbd>
                              for CRISPR-SpCas9-PAM constrained sequence subspace) and
                              <kbd>
                               JACKIE.countSeqNeighbors
                              </kbd>
                              (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig 1d,e
                              </a>
                              ).
                              <kbd>
                               JACKIE.encodeSeqSpace
                              </kbd>
                              scans genome sequence files and record the presence or absence of each possible k-mer in a bit array representation of the k-mer sequence space (SeqSpace) (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig 1d
                              </a>
                              ). At the beginning, SeqSpace is sized at 4
                              <sup>
                               k
                              </sup>
                              bits, i.e., 4
                              <sup>
                               k
                              </sup>
                              /8=2
                              <sup>
                               2k−3
                              </sup>
                              bytes, and initialized with zeros. As the genome sequence is scanned, each encountered k-mer sequence is mapped per nucleotide to 2-bit representation (A→00, C→01, G→10, T→11), and packed into a unsigned 64-bit integer (
                              <kbd>
                               uint64_t
                              </kbd>
                              ). The unused most significant bits are zero-masked. The forward strand sequence is mapped to the
                              <kbd>
                               fwd_idx
                              </kbd>
                              variable while the reverse complementary sequence is mapped to the
                              <kbd>
                               rev_idx
                              </kbd>
                              variable. To ensure computational efficiency, a “sliding” scheme is used such that the forward index (
                              <kbd>
                               fwd_idx
                              </kbd>
                              ) is shifted left by 2 bits while the reverse complement index (
                              <kbd>
                               rev_idx
                              </kbd>
                              ) is shifted right by 2 bits as the scanning process advances to the next nucleotide, and two bits for the new encountered nucleotide (or the reverse complement) are then added to the least or most significant bit positions, respectively, for
                              <kbd>
                               fwd_idx
                              </kbd>
                              and
                              <kbd>
                               rev_idx
                              </kbd>
                              . The bits of SeqSpace at the positions indexed by
                              <kbd>
                               fwd_idx
                              </kbd>
                              and
                              <kbd>
                               rev_idx
                              </kbd>
                              is then set to 1, recording the presence of the k-mer and its reverse complement in the SeqSpace. At completion, SeqSpace will have recorded in each bit the presence (bit value 1) or absence (bit value 0) of a corresponding indexed k-mer in the genome. The SeqSpace array will then be outputted to disk as compressed
                              <kbd>
                               seqbits.gz
                              </kbd>
                              to allow for future uses.
                              <kbd>
                               JACKIE.countSeqNeighbors
                              </kbd>
                              loads SeqSpace from the
                              <kbd>
                               seqbits.gz
                              </kbd>
                              file and a list of target site sequences such as the ones in JACKIEdb, and count the number of sequence neighbors each target site sequence has for each number of mismatches up to a user-provided threshold (e.g., 3). The binary representation of k-mers and the indexed SeqSpace allow fast computation of sequence neighbors.
                              <kbd>
                               JACKIE.countSeqNeighbors
                              </kbd>
                              first generates all position bitwise AND-masks and OR-masks that allow all permutation of sequence changes of a query k-mer through a bitwise AND and a bitwise OR operation, respectively. An example pair of AND- and OR-mask for changing three bases of the query in the binary representation is shown in
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig 1e
                              </a>
                              . For each query sequence,
                              <kbd>
                               JACKIE.countSeqNeighbors
                              </kbd>
                              applies the AND,OR-mask pairs one-by-one, and lookup the bit indexed by the resultant
                              <kbd>
                               Query_mut
                              </kbd>
                              variable in the
                              <kbd>
                               SeqSpace
                              </kbd>
                              array, incrementing the count of sequence neighbors for the corresponding number of mismatches by one. After completing all searches for the query sequence, the number of sequence neighbors is outputted per mismatch number. For computers that do not support 128GB memory for 20-mer
                              <kbd>
                               SeqSpace
                              </kbd>
                              , we implemented “divide- and-conquer” versions,
                              <kbd>
                               JACKIE.encodeSeqSpace.prefixed
                              </kbd>
                              and
                              <kbd>
                               JACKIE.countSeqNeighbors.pmulti
                              </kbd>
                              which divide the problems into prefixed subspaces. For example, using 2 nucleotide (nt) prefix, we divided the problem into 16 subspaces (AA, AC, AG, AT, …), each requiring only 8GB memory for k=20. If needed, the programs allow further division of subspaces. For users interested to obtain the exact off-target site counts, we implemented
                              <kbd>
                               JACKIE.encodeSeqCountDatabase
                              </kbd>
                              and
                              <kbd>
                               JACKIE.countOffSites
                              </kbd>
                              which encode SeqSpace with exact copy numbers and report exact off-target site counts, respectively. A user-specified number of bits (b) are used to record the copy number of a particular k-mer in the genome up to 2
                              <sup>
                               b
                              </sup>
                              −2 copies. If the copy number exceeds 2
                              <sup>
                               b
                              </sup>
                              −2, the SeqSpace record is set to 2
                              <sup>
                               b
                              </sup>
                              −1, and the copy number is recorded in a “overflow” map in the C++ Standard Template Library (STL) map class, by setting the key to the 2-bit representation of the k-mer (uint64_t) and the value to the copy number.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              CRISPR imaging experiments
                             </h3>
                             <p id="p-12">
                              gRNA spacer sequences (as listed in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig 2c
                              </a>
                              ) were cloned into gRNA-15xPBSc expression vector via an oligo-annealing protocol as previously described [
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ]. HEK293T cells were cultivated in Dulbecco’s modified Eagle’s medium (DMEM) (Sigma) with 10% fetal bovine serum (FBS)(Lonza), 4% Glutamax (Gibco), 1% Sodium Pyruvate (Gibco) and penicillin-streptomycin (Gibco). Incubator conditions were 37 °C and 5% CO2. Cells were seeded into 24-well plates the day before being transfected with 50ng pAC1445-dCas9 plasmid (Addgene #73169), 25ng pAC1447-Clover-PUFc plasmid (Addgene #73689) and 250ng sgRNA-15xPBSc plasmid. Cells were fixed, mounted on slides, and imaged 48 hours after transfection on a Leica SP8 confocal microscope.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             RESULTS
                            </h2>
                            <p id="p-13">
                             We ran
                             <kbd>
                              JACKIE.bin
                             </kbd>
                             (4 jobs) and
                             <kbd>
                              JACKIE.sortToBed
                             </kbd>
                             (16 jobs) on a Unix HPC cluster with Intel Xeon compute cores for hg38 and mm10 mouse genome, which took totals of ~35 minutes and ~22 minutes, respectively. On a MacBook Pro laptop (2021; MacBookPro18,3; 14-inch; Apple M1 Pro chip; 16GB ram; 1TB SSD; MacOS 12.2.1 Monterey), running
                             <kbd>
                              JACKIE.bin
                             </kbd>
                             on hg38 human genome with 4 processes took ~12 minutes, and
                             <kbd>
                              JACKIE.sortToBed
                             </kbd>
                             with 16 processes, took ~27 minutes. Thus,
                             <kbd>
                              JACKIE.bin
                             </kbd>
                             and
                             <kbd>
                              JACKIE.sortToBed
                             </kbd>
                             ran on the MacBook Pro laptop for a total of ~40 minutes.
                             <kbd>
                              JACKIE.encodeSeqSpaceNGG
                             </kbd>
                             on hg38 took 45 minutes (25 minutes spent on encoding, 20 minutes spent on compressing to seqbits.gz file) on one node of the Unix HPC cluster. We ran
                             <kbd>
                              JACKIE.countSeqNeighbors
                             </kbd>
                             on 52,666,035 single-copy CRISPR query sequences for up to 3 mismatches which took 12.5 hours on one node of the Unix HPC cluster. On a MacBook Pro laptop (2021; MacBookPro18,3; 14-inch; Apple M1 Pro chip; 16GB ram; 1TB SSD; MacOS 12.2.1 Monterey), running
                             <kbd>
                              JACKIE.encodeSeqCountDatabase
                             </kbd>
                             on hg38 with 3-bit SeqSpace took 4645 seconds (~1.3 hours) total when 16 runs of 2-nt subspaces (k=20) were executed in serial. On the same machine, running
                             <kbd>
                              JACKIE.encodeSeqSpace.prefixed
                             </kbd>
                             took a total of 29 min. Sequence Count Database or SeqSpace encoding (“indexing”) is only needed once for a particular genome for a particular k and PAM. We compared speeds of the off-target prediction portion of JACKIE with
                             <kbd>
                              Cas-OFFinder
                             </kbd>
                             [
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ] and
                             <kbd>
                              FlashFry
                             </kbd>
                             [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ]. Cas-OFFinder is one of the first and most popular off-target prediction software while FlashFry was evaluated to be the fastest among existing off-target prediction software packages by a previous paper [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. Indexing of
                             <kbd>
                              FlashFry
                             </kbd>
                             database took 2 hours. Running
                             <kbd>
                              Cas-OFFinder, FlashFry, JACKIE.countOffSites
                             </kbd>
                             and
                             <kbd>
                              JACKIE.countSeqNeighbors.pmulti
                             </kbd>
                             on 100,000 queries took 26873 seconds (~7.5 hours), 3687 seconds (~1 hour), 657 seconds (~11 minutes), and 295 seconds (~5 minutes), respectively. We ran
                             <kbd>
                              JACKIE.countOffSites
                             </kbd>
                             and
                             <kbd>
                              JACKIE.countSeqNeighbors.pmulti
                             </kbd>
                             on 1,000,000 queries which took 6224 seconds (~1.72 hours), 2439 seconds (~40 minutes), respectively. Note these are not completely fair comparisons as Cas-OFFinder reports the locations of off-target sites allowing DNA/RNA bulges in addition to mismatches whereas
                             <kbd>
                              FlashFry
                             </kbd>
                             and
                             <kbd>
                              JACKIE.countOffSites
                             </kbd>
                             report the number of off-target sites with mismatches only. The fastest approach taken by
                             <kbd>
                              JACKIE.countSeqNeighbors.pmulti
                             </kbd>
                             reports the number of neighborhood sequences with mismatches only. For large-scale target design projects only requiring target sites with few or no off-targets up to a certain mismatch threshold, off-target prediction approach taken by JACKIE would suffice and provide a time-effective solution. Bed files generated by JACKIE can be uploaded to genome browser alongside other tracks for convenient GUI-enabled target designs. We generated JACKIE database (JACKIEdb) for CRISPR/SpCas9 hg38 and mm10 and created UCSC genome sessions with gRNA tracks on which users can filter for specific gRNA properties, including the numbers of 1-,2-, and 3-mismatch sites, total off-target site number, GC% of spacer sequence, longest run of T as well as number of sites and size of the gRNA clusters (
                             <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                              Fig 2a,b
                             </a>
                             ). Not only can users filter for desired gRNA properties, but also can they select gRNAs alongside hundreds of tracks already available on the UCSC genome browser. We selected three CRISPR binding clusters and applied CRISPR genome imaging to visualize them via fluorescence microscopy (
                             <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                              Fig 2c
                             </a>
                             ), producing three fluorescent foci in each case, consistent with the ploidy of the HEK293T cells used in the experiments.
                            </p>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             DISCUSSION
                            </h2>
                            <p id="p-14">
                             Advances in engineered nuclease technologies have fueled an explosion of genome, epigenome, transcriptome editing and imaging tools. The easily programmable feature as exemplified by the CRISPR systems allows large-scale genome-wide experiments. While some applications, such as genome and epigenome editing, require target sites that are unique, i.e., with sequence occurring only once in the genome, applications such as imaging requires multiple binding sites that are clustered within a particular region. Existing target design software packages focus on individual designs but are inefficient in large-scale design. In this paper, we described a new software package, JACKIE, that enumerates all single- and multi-copy k-mers in a target genome as well as provides fast evaluation of off-target effects. The off-target prediction algorithm is nearly a hundred fold more time-efficient than the most popular software for off-target prediction, making the off-target prediction of millions of sequences practical. Genome-wide design by JACKIE can be conveniently loaded onto GUI-enabled genome browsers alongside other tracks so that users can design target sequences with respect to annotated elements or epigenomic features. The genome-wide design in bed format also allows high-throughput computational selection of target sites using command line tools. We computed CRISPR-SpCas9 single-copy and multi-copy binding sites (JACKIEdb) that can be directly downloaded or visualized on UCSC genome browser. As a demonstration for the utility of gRNA designs by JACKIE, we selected three <10kb clusters of >30 gRNAs and performed CRISPR genome imaging. We believe that JACKIE and JACKIEdb will facilitate designs of genome-wide target libraries or large-scale engineered nuclease projects.
                            </p>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             DATA AVAILABILITY
                            </h2>
                            <p id="p-15">
                             Source codes and precomputed tracks for hg38 and mm10 are available for download at
                             <a href="http://cheng.bio/JACKIE">
                              http://cheng.bio/JACKIE
                             </a>
                             and
                             <a href="https://github.com/albertwcheng/JACKIE2">
                              https://github.com/albertwcheng/JACKIE2
                             </a>
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             ACKNOWLEDGEMENTS
                            </h2>
                            <p id="p-16">
                             This work has been supported by the National Human Genome Research Institute grant R01HG009900 (to A.W.C). Arizona State University startup grant (to A.W.C.) and Jackson Laboratory internal grants (to A.W.C.).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-dataset fn-group-external-links" id="fn-1">
                              <p id="p-4">
                               <a href="https://github.com/albertwcheng/JACKIE2">
                                https://github.com/albertwcheng/JACKIE2
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="http://cheng.bio/jackie">
                                http://cheng.bio/jackie
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Higashikuni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Advancing CRISPR-Based Programmable Platforms beyond Genome Editing in Mammalian Cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACS Synth Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  2607
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2619
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACS%2BSynth%2BBiol%26rft.volume%253D8%26rft.spage%253D2607%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968933v4.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Advances in CRISPR-Cas systems for RNA targeting, tracking and editing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biotechnol Adv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  708
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  729
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiotechnol%2BAdv%26rft.volume%253D37%26rft.spage%253D708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.3" data-doi="10.1038/nbt.2842">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Joung
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  CRISPR-Cas systems for editing, regulating and targeting genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Biotechnol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  347
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  55
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BBiotechnol%26rft.volume%253D32%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2842%26rft_id%253Dinfo%253Apmid%252F24584096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nbt.2842&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24584096&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.4" data-doi="10.1038/cr.2016.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Casilio: a versatile CRISPR-Cas9-Pumilio hybrid for gene regulation and genomic labeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  254
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BRes%26rft.volume%253D26%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2016.3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/cr.2016.3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968933v4.5" data-doi="10.1038/s41594-017-0015-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maass
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Spatiotemporal allele organization by allele-specific CRISPR live-cell imaging (SNP-CLING)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Struct Mol Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  176
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  184
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BStruct%2BMol%2BBiol%26rft.volume%253D25%26rft.spage%253D176%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41594-017-0015-3%26rft_id%253Dinfo%253Apmid%252F29343869%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41594-017-0015-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29343869&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968933v4.6" data-doi="10.1038/s41592-018-0174-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  CRISPR-Sirius: RNA scaffolds for signal amplification in genome imaging
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  928
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  931
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D15%26rft.spage%253D928%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41592-018-0174-0%26rft_id%253Dinfo%253Apmid%252F30377374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41592-018-0174-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30377374&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.7" data-doi="10.1038/ncomms14725">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Live cell imaging of low- and non-repetitive chromosome loci using CRISPR-Cas9
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 : p.
                                 <span class="cit-fpage">
                                  14725
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D8%26rft.spage%253D14725%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms14725%26rft_id%253Dinfo%253Apmid%252F28290446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ncomms14725&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28290446&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheng
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  JACKIE: Fast enumeration of single- and multi-copy CRISPR binding sites
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                 <span class="cit-fpage">
                                  2020.02.27.968933
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.9" data-doi="10.1093/bioinformatics/btu048">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bae
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  1473
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu048%26rft_id%253Dinfo%253Apmid%252F24463181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu048&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24463181&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000336530000021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.02.27.968933v4.10" data-doi="10.1186/s13059-016-1012-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haeussler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  Evaluation of off-target and on-target scoring algorithms and integration into the guide RNA selection tool CRISPOR
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  148
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft.volume%253D17%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1012-2%26rft_id%253Dinfo%253Apmid%252F27380939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-016-1012-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27380939&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F04%2F07%2F2020.02.27.968933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bradford
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Perrin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  A benchmark of computational CRISPR-Cas9 guide design methods
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  e1007274
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D15%26rft.spage%253De1007274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.12" data-doi="10.1101/gr.229102. Article published online before print in May 2002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kent
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  The human genome browser at UCSC
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Res
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  996
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDQvMDcvMjAyMC4wMi4yNy45Njg5MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.13" data-doi="10.1186/s12915-018-0545-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McKenna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shendure
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  FlashFry: a fast and flexible tool for large-scale CRISPR target design
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Biol
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  74
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol%26rft.volume%253D16%26rft.spage%253D74%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-018-0545-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s12915-018-0545-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.02.27.968933v4.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cancellieri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 ,
                                 <span class="cit-article-title">
                                  CRISPRitz: rapid, high-throughput and variant-aware in silico off-target site identification for CRISPR genome editing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ): p.
                                 <span class="cit-fpage">
                                  2001
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D2001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.02.27.968933v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.17.476625v2" title="Quantitative Models of Lipid Transfer and Membrane Contact Formation" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.02.05.429898v3" title="Cell Rearrangement Generates Pattern Emergence as a Function of Temporal Morphogen Exposure" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted April 07, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.02.27.968933v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/04/07/2020.02.27.968933.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.02.27.968933v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.02.27.968933v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2485400"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-l92tNXXn8YzJBqMtr2aS_ZdzYgjNHeq7WuDceKeltoU"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484467440"/>
                                   <input type="hidden" name="captcha_token" value="1826d1949766c967e478ddf70a2dd2fd"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2485400--3" data-node-nid="2485400" id="node-2485400--42037392855" data-pisa="biorxiv;2020.02.27.968933v4" data-pisa-master="biorxiv;2020.02.27.968933" data-seqnum="2485400" data-apath="/biorxiv/early/2022/04/07/2020.02.27.968933.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Jacqueline Jufen
                                  </span>
                                  <span class="nlm-surname">
                                   Zhu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Albert Wu
                                  </span>
                                  <span class="nlm-surname">
                                   Cheng
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.02.27.968933;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.02.27.968933
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&title=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&title=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&text=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&t=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&title=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&title=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&title=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2485400--5" data-node-nid="2485400" id="citation-node-2485400--6956547373" data-pisa="biorxiv;2020.02.27.968933v4" data-pisa-master="biorxiv;2020.02.27.968933" data-seqnum="2485400" data-apath="/biorxiv/early/2022/04/07/2020.02.27.968933.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     JACKIE: Fast enumeration of genomic single- and multi-copy target sites and their off-targets for CRISPR and other engineered nuclease systems
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Jacqueline Jufen
                                      </span>
                                      <span class="nlm-surname">
                                       Zhu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Albert Wu
                                      </span>
                                      <span class="nlm-surname">
                                       Cheng
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.02.27.968933;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.02.27.968933
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Zhu%2C+Jacqueline+Jufen&rft.au=Cheng%2C+Albert+Wu">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2485400/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2485400/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2485400/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2485400/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2485400/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2485400/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2485400/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2485400/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2485400/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2485400/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2485400/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2485400/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&count=horizontal&via=&text=JACKIE%3A%20Fast%20enumeration%20of%20genomic%20single-%20and%20multi-copy%20target%20sites%20and%20their%20off-targets%20for%20CRISPR%20and%20other%20engineered%20nuclease%20systems&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.02.27.968933v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.02.27.968933v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15186)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9196)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
