<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.07.30.228965v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.07.30.228965v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.07.30.228965v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/30/2020.07.30.228965.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.07.30.228965v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Nucleotide usage biases distort inferences of the species tree"/>
  <meta name="DC.Identifier" content="10.1101/2020.07.30.228965"/>
  <meta name="DC.Date" content="2022-05-30"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Despite the importance of natural selection in species’ evolutionary history, phylogenetic methods that take into account population-level processes typically ignore selection. The assumption of neutrality is often based on the idea that selection occurs at a minority of loci in the genome and is unlikely to compromise phylogenetic inferences significantly. However, genome-wide processes like GC-bias and some variation segregating at the coding regions are known to evolve in the nearly neutral range. As we are now using genome-wide data to estimate species trees, it is natural to ask whether weak but pervasive selection is likely to blur species tree inferences. We developed a polymorphism-aware phylogenetic model tailored for measuring signatures of nucleotide usage biases to test the impact of selection in the species tree. Our analyses indicate that while the inferred relationships among species are not significantly compromised, the genetic distances are systematically underestimated in a node-height dependent manner: i.e., the deeper nodes tend to be more underestimated than the shallow ones. Such biases have implications for molecular dating. We dated the evolutionary history of 30 worldwide fruit fly populations, and we found signatures of GC-bias considerably affecting the estimated divergence times (up to 23%) in the neutral model. Our findings call for the need to account for selection when quantifying divergence or dating species evolution.

Significance statement Although little is known about the impact of natural selection on species tree estimation, expectations are that it occurs at a minority of loci in eukaryotic genomes and is thus unlikely to affect the divergence process. However, growing evidence suggests that a large amount of the genomic variation evolves under weak but pervasive selection (e.g., fixation biases created by GC-bias gene conversion). We tested the impact of unaccounted-for nearly neutral selection on species tree estimation and found that the estimated branch lengths are systematically biased. Our results highlight the need for selection-aware models in species tree estimation and molecular dating.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Rui Borges"/>
  <meta name="DC.Contributor" content="Bastien Boussau"/>
  <meta name="DC.Contributor" content="Gergely J. Szöllősi"/>
  <meta name="DC.Contributor" content="Carolin Kosiol"/>
  <meta name="article:published_time" content="2022-05-30"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Nucleotide usage biases distort inferences of the species tree"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Despite the importance of natural selection in species’ evolutionary history, phylogenetic methods that take into account population-level processes typically ignore selection. The assumption of neutrality is often based on the idea that selection occurs at a minority of loci in the genome and is unlikely to compromise phylogenetic inferences significantly. However, genome-wide processes like GC-bias and some variation segregating at the coding regions are known to evolve in the nearly neutral range. As we are now using genome-wide data to estimate species trees, it is natural to ask whether weak but pervasive selection is likely to blur species tree inferences. We developed a polymorphism-aware phylogenetic model tailored for measuring signatures of nucleotide usage biases to test the impact of selection in the species tree. Our analyses indicate that while the inferred relationships among species are not significantly compromised, the genetic distances are systematically underestimated in a node-height dependent manner: i.e., the deeper nodes tend to be more underestimated than the shallow ones. Such biases have implications for molecular dating. We dated the evolutionary history of 30 worldwide fruit fly populations, and we found signatures of GC-bias considerably affecting the estimated divergence times (up to 23%) in the neutral model. Our findings call for the need to account for selection when quantifying divergence or dating species evolution.</p><h3>Significance statement</h3>
<p>Although little is known about the impact of natural selection on species tree estimation, expectations are that it occurs at a minority of loci in eukaryotic genomes and is thus unlikely to affect the divergence process. However, growing evidence suggests that a large amount of the genomic variation evolves under weak but pervasive selection (e.g., fixation biases created by GC-bias gene conversion). We tested the impact of unaccounted-for nearly neutral selection on species tree estimation and found that the estimated branch lengths are systematically biased. Our results highlight the need for selection-aware models in species tree estimation and molecular dating.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.07.30.228965v3"/>
  <meta name="citation_id" content="2020.07.30.228965v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.07.30.228965"/>
  <meta name="citation_num_pages" content="24"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.07.30.228965"/>
  <meta name="citation_author" content="Rui Borges"/>
  <meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5905-3778"/>
  <meta name="citation_author" content="Bastien Boussau"/>
  <meta name="citation_author_institution" content="Univ Lyon, Université Claude Bernard Lyon 1"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0776-4460"/>
  <meta name="citation_author" content="Gergely J. Szöllősi"/>
  <meta name="citation_author_institution" content="Department of Biological Physics, Eötvös University"/>
  <meta name="citation_author_institution" content="MTA-ELTE “Lendület” Evolutionary Genomics Research Group"/>
  <meta name="citation_author_institution" content="Evolutionary Systems Research Group, Centre for Ecological Research, Hungarian Academy of Sciences"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8556-845X"/>
  <meta name="citation_author" content="Carolin Kosiol"/>
  <meta name="citation_author_institution" content="Institut für Populationsgenetik, Vetmeduni Vienna"/>
  <meta name="citation_author_institution" content="Centre for Biological Diversity, University of St Andrews"/>
  <meta name="citation_author_email" content="ck202@st-andrews.ac.uk"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3219-6648"/>
  <meta name="citation_reference" content="Adams, R. H., Schield, D. R., Card, D. C., and Castoe, T. A. (2018). Assessing the Impacts of Positive Selection on Coalescent-Based Species Tree Estimation and Species Delimitation. Systematic Biology, 67(6):1076–1090."/>
  <meta name="citation_reference" content="Begun, D. J., Holloway, A. K., Stevens, K., Hillier, L. W., Poh, Y.-P., Hahn, M. W., Nista, P. M., Jones, C. D., Kern, A. D., Dewey, C. N., Pachter, L., Myers, E., and Langley, C. H. (2007). Population Genomics: Whole-Genome Analysis of Polymorphism and Divergence in Drosophila simulans. PLoS Biology, 5(11):e310."/>
  <meta name="citation_reference" content="Borges, R., Szöllősi, G. J., and Kosiol, C. (2019). Quantifying GC-Biased Gene Conversion in Great Ape Genomes Using Polymorphism-Aware Models. Genetics, 212(4):1321–1336."/>
  <meta name="citation_reference" content="Castoe, T. A., de Koning, A. P. J., Kim, H.-M., Gu, W., Noonan, B. P., Naylor, G., Jiang, Z. J., Parkinson, C. L., and Pollock, D. D. (2009). Evidence for an ancient adaptive episode of convergent molecular evolution. Proceedings of the National Academy of Sciences, 106(22):8986–8991."/>
  <meta name="citation_reference" content="De Maio, N., Schlötterer, C., and Kosiol, C. (2013). Linking great apes genome evolution across time scales using polymorphism-aware phylogenetic models. Molecular Biology and Evolution, 30(10):2249–2262."/>
  <meta name="citation_reference" content="De Maio, N., Schrempf, D., and Kosiol, C. (2015). PoMo: An allele frequency-based approach for species tree estimation. Systematic Biology, 64(6):1018–1031."/>
  <meta name="citation_reference" content="Duret, L. and Galtier, N. (2009). Biased gene conversion and the evolution of mammalian genomic landscapes. Annual Review of Genomics and Human Genetics, 10:285–311."/>
  <meta name="citation_reference" content="Edwards, S. V. (2009). Natural selection and phylogenetic analysis. Proceedings of the National Academy of Sciences, 106(22):8799–8800."/>
  <meta name="citation_reference" content="Edwards, S. V., Xi, Z., Janke, A., Faircloth, B. C., McCormack, J. E., Glenn, T. C., Zhong, B., Wu, S., Lemmon, E. M., Lemmon, A. R., Leaché, A. D., Liu, L., and Davis, C. C. (2016). Implementing and testing the multispecies coalescent model: A valuable paradigm for phylogenomics. Molecular Phylogenetics and Evolution, 94:447–462."/>
  <meta name="citation_reference" content="Flouri, T., Jiao, X., Rannala, B., and Yang, Z. (2020). A bayesian implementation of the multispecies coalescent model with introgression for phylogenomic analysis. Molecular Biology and Evolution, 37(4)."/>
  <meta name="citation_reference" content="Galtier, N., Duret, L., Glémin, S., and Ranwez, V. (2009). GC-biased gene conversion promotes the fixation of deleterious amino acid changes in primates."/>
  <meta name="citation_reference" content="Galtier, N., Piganeau, G., Mouchiroud, D., and Duret, L. (2001). GC-content evolution in mammalian genomes: the biased gene conversion hypothesis. Genetics, 159(2):907–11."/>
  <meta name="citation_reference" content="Glémin, S. (2010). Surprising Fitness Consequences of GC-Biased Gene Conversion: I. Mutation Load and Inbreeding Depression. Genetics, 185(3):939–959."/>
  <meta name="citation_reference" content="Glémin, S., Arndt, P. F., Messer, P. W., Petrov, D., Galtier, N., and Duret, L. (2015). Quantification of GC-biased gene conversion in the human genome. Genome Research, 25(8):1215–1228."/>
  <meta name="citation_reference" content="Hervas, S., Sanz, E., Casillas, S., Pool, J. E., and Barbadilla, A. (2017). PopFly: the Drosophila population genomics browser. Bioinformatics, 33(17):2779–2780."/>
  <meta name="citation_reference" content="Höhna, S., Landis, M. J., Heath, T. A., Boussau, B., Lartillot, N., Moore, B. R., Huelsenbeck, J. P., and Ronquist, F. (2016). RevBayes: Bayesian Phylogenetic Inference Using Graphical Models and an Interactive Model-Specification Language. Systematic Biology, 65(4):726–736."/>
  <meta name="citation_reference" content="Hughes, A. L. (2005). Evidence for Abundant Slightly Deleterious Polymorphisms in Bacterial Populations. Genetics, 169(2):533–538."/>
  <meta name="citation_reference" content="Hughes, A. L., Packer, B., Welch, R., Bergen, A. W., Chanock, S. J., and Yeager, M. (2003). Widespread purifying selection at polymorphic sites in human protein-coding loci. Proceedings of the National Academy of Sciences, 100(26):15754–15757."/>
  <meta name="citation_reference" content="Kingman, J. (1982a). The coalescent. Stochastic Processes and their Applications, 13(3):235–248."/>
  <meta name="citation_reference" content="Kingman, J. F. C. (1982b). On the genealogy of large populations. Journal of Applied Probability, 19(A):27–43."/>
  <meta name="citation_reference" content="Lachaise, D., Cariou, M.-L., David, J. R., Lemeunier, F., Tsacas, L., and Ashburner, M. (1988). Historical Biogeography of the Drosophila melanogaster Species Subgroup. In Evolutionary Biology, pages 159–225. Springer US, Boston, MA."/>
  <meta name="citation_reference" content="Lartillot, N. (2013). Phylogenetic Patterns of GC-Biased Gene Conversion in Placental Mammals and the Evolutionary Dynamics of Recombination Landscapes. Molecular Biology and Evolution, 30(3):489–502."/>
  <meta name="citation_reference" content="Lassalle, F., Périan, S., Bataillon, T., Nesme, X., Duret, L., and Daubin, V. (2015). GC-Content Evolution in Bacterial Genomes: The Biased Gene Conversion Hypothesis Expands. PLOS Genetics, 11(2):e1004941."/>
  <meta name="citation_reference" content="Laurent, S. J., Werzner, A., Excoffier, L., and Stephan, W. (2011). Approximate Bayesian Analysis of Drosophila melanogaster Polymorphism Data Reveals a Recent Colonization of Southeast Asia. Molecular Biology and Evolution, 28(7):2041–2051."/>
  <meta name="citation_reference" content="Leaché, A. D. and Oaks, J. R. (2017). The Utility of Single Nucleotide Polymorphism (SNP) Data in Phylogenetics. Annual Review of Ecology, Evolution, and Systematics, 48(1):69–84."/>
  <meta name="citation_reference" content="Li, H. and Stephan, W. (2006). Inferring the demographic history and rate of adaptive substitution in Drosophila. PLoS Genetics."/>
  <meta name="citation_reference" content="Lynch, M., Ackerman, M. S., Gout, J.-F., Long, H., Sung, W., Thomas, W. K., and Foster, P. L. (2016). Genetic drift, selection and the evolution of the mutation rate. Nature Reviews Genetics, 17(11):704–714."/>
  <meta name="citation_reference" content="Maddison, W. P. and Knowles, L. L. (2006). Inferring Phylogeny Despite Incomplete Lineage Sorting. Systematic Biology, 55(1):21–30."/>
  <meta name="citation_reference" content="Nagylaki, T. (1983). Evolution of a Finite Population under Gene Conversion. Proceedings of the National Academy of Sciences of the United States of America, 80(20):6278–6281."/>
  <meta name="citation_reference" content="Nei, M. (2005). Selectionism and Neutralism in Molecular Evolution. Molecular Biology and Evolution, 22(12):2318–2342."/>
  <meta name="citation_reference" content="Ohta, T. (1973). Slightly Deleterious Mutant Substitutions in Evolution. Nature, 246(5428):96–98."/>
  <meta name="citation_reference" content="Ohta, T. (2002). Near-neutrality in evolution of genes and gene regulation. Proceedings of the National Academy of Sciences, 99(25):16134–16137."/>
  <meta name="citation_reference" content="Pessia, E., Popa, A., Mousset, S., Rezvoy, C., Duret, L., and Marais, G. A. B. (2012). Evidence for Widespread GC-biased Gene Conversion in Eukaryotes. Genome Biology and Evolution, 4(7):675–682."/>
  <meta name="citation_reference" content="Pollard, D. A., Iyer, V. N., Moses, A. M., and Eisen, M. B. (2006). Widespread Discordance of Gene Trees with Species Tree in Drosophila: Evidence for Incomplete Lineage Sorting. PLoS Genetics, 2(10):e173."/>
  <meta name="citation_reference" content="Prado-Martinez, J., Sudmant, P. H., Kidd, J. M., Li, H., Kelley, J. L., Lorente-Galdos, B., Veeramah, K. R., Woerner, A. E., O’Connor, T. D., Santpere, G., Cagan, A., Theunert, C., Casals, F., Laayouni, H., Munch, K., Hobolth, A., Halager, A. E., Malig, M., Hernandez-Rodriguez, J., Hernando-Herraez, I., Prüfer, K., Pybus, M., Johnstone, L., Lachmann, M., Alkan, C., Twigg, D., Petit, N., Baker, C., Hormozdiari, F., Fernandez-Callejo, M., Dabad, M., Wilson, M. L., Stevison, L., Camprubí, C., Carvalho, T., Ruiz-Herrera, A., Vives, L., Mele, M., Abello, T., Kondova, I., Bontrop, R. E., Pusey, A., Lankester, F., Kiyang, J. A., Bergl, R. A., Lonsdorf, E., Myers, S., Ventura, M., Gagneux, P., Comas, D., Siegismund, H., Blanc, J., Agueda-Calpena, L., Gut, M., Fulton, L., Tishkoff, S. A., Mullikin, J. C., Wilson, R. K., Gut, I. G., Gonder, M. K., Ryder, O. A., Hahn, B. H., Navarro, A., Akey, J. M., Bertranpetit, J., Reich, D., Mailund, T., Schierup, M. H., Hvilsom, C., Andrés, A. M., Wall, J. D., Bustamante, C. D., Hammer, M. F., Eichler, E. E., and Marques-Bonet, T. (2013). Great ape genetic diversity and population history. Nature, 499(7459):471–475."/>
  <meta name="citation_reference" content="R Core Team (2021). R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria."/>
  <meta name="citation_reference" content="citation_journal_title=Phylogenetics in the Genomic Era, pages 3.3:1–3.3:21. No commercial publisher — Authors open access book;citation_title=The Multi-species Coalescent Model and Species Tree Inference;citation_year=2020"/>
  <meta name="citation_reference" content="Rannala, B. and Yang, Z. (2003). Bayes Estimation of Species Divergence Times and Ancestral Population Sizes Using DNA Sequences From Multiple Loci. Genetics, 164(4):1645–1656."/>
  <meta name="citation_reference" content="Robinson, M. C., Stone, E. A., and Singh, N. D. (2014). Population Genomic Analysis Reveals No Evidence for GC-Biased Gene Conversion in Drosophila melanogaster. Molecular Biology and Evolution, 31(2):425–433."/>
  <meta name="citation_reference" content="Romiguier, J. and Roux, C. (2017). Analytical biases associated with gc-content in molecular evolution. Frontiers in Genetics, 8:16."/>
  <meta name="citation_reference" content="Sawyer, S. A., Parsch, J., Zhang, Z., and Hartl, D. L. (2007). Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila. Proceedings of the National Academy of Sciences, 104(16):6504–6510."/>
  <meta name="citation_reference" content="Schrempf, D., Minh, B. Q., De Maio, N., von Haeseler, A., and Kosiol, C. (2016). Reversible polymorphism-aware phylogenetic models and their application to tree inference. Journal of Theoretical Biology, 407:362–370."/>
  <meta name="citation_reference" content="Schrempf, D., Minh, B. Q., von Haeseler, A., and Kosiol, C. (2019). Polymorphism-Aware Species Trees with Advanced Mutation Models, Bootstrap, and Rate Heterogeneity. Molecular Biology and Evolution, 36(6):1294–1301."/>
  <meta name="citation_reference" content="Stephan, W. and Li, H. (2007). The recent demographic and adaptive history of Drosophila melanogaster. Heredity, 98(2):65–68."/>
  <meta name="citation_reference" content="Szöllősi, G. J., Tannier, E., Daubin, V., and Boussau, B. (2015). The Inference of Gene Trees with Species Trees. Systematic Biology, 64(1):e42–e62."/>
  <meta name="citation_reference" content="Tavaré, S. (1986). Some Probabilistic and Statistical Problems in the Analysis of DNA Sequences. Lectures on Mathematics in the Life Sciences, 17:57–86."/>
  <meta name="citation_reference" content="Xu, B. and Yang, Z. (2016). Challenges in Species Tree Estimation Under the Multispecies Coalescent Model. Genetics, 204(4):1353–1368."/>
  <meta name="twitter:title" content="Nucleotide usage biases distort inferences of the species tree"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Despite the importance of natural selection in species’ evolutionary history, phylogenetic methods that take into account population-level processes typically ignore selection. The assumption of neutrality is often based on the idea that selection occurs at a minority of loci in the genome and is unlikely to compromise phylogenetic inferences significantly. However, genome-wide processes like GC-bias and some variation segregating at the coding regions are known to evolve in the nearly neutral range. As we are now using genome-wide data to estimate species trees, it is natural to ask whether weak but pervasive selection is likely to blur species tree inferences. We developed a polymorphism-aware phylogenetic model tailored for measuring signatures of nucleotide usage biases to test the impact of selection in the species tree. Our analyses indicate that while the inferred relationships among species are not significantly compromised, the genetic distances are systematically underestimated in a node-height dependent manner: i.e., the deeper nodes tend to be more underestimated than the shallow ones. Such biases have implications for molecular dating. We dated the evolutionary history of 30 worldwide fruit fly populations, and we found signatures of GC-bias considerably affecting the estimated divergence times (up to 23%) in the neutral model. Our findings call for the need to account for selection when quantifying divergence or dating species evolution.

Significance statement Although little is known about the impact of natural selection on species tree estimation, expectations are that it occurs at a minority of loci in eukaryotic genomes and is thus unlikely to affect the divergence process. However, growing evidence suggests that a large amount of the genomic variation evolves under weak but pervasive selection (e.g., fixation biases created by GC-bias gene conversion). We tested the impact of unaccounted-for nearly neutral selection on species tree estimation and found that the estimated branch lengths are systematically biased. Our results highlight the need for selection-aware models in species tree estimation and molecular dating.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Nucleotide usage biases distort inferences of the species tree"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Despite the importance of natural selection in species’ evolutionary history, phylogenetic methods that take into account population-level processes typically ignore selection. The assumption of neutrality is often based on the idea that selection occurs at a minority of loci in the genome and is unlikely to compromise phylogenetic inferences significantly. However, genome-wide processes like GC-bias and some variation segregating at the coding regions are known to evolve in the nearly neutral range. As we are now using genome-wide data to estimate species trees, it is natural to ask whether weak but pervasive selection is likely to blur species tree inferences. We developed a polymorphism-aware phylogenetic model tailored for measuring signatures of nucleotide usage biases to test the impact of selection in the species tree. Our analyses indicate that while the inferred relationships among species are not significantly compromised, the genetic distances are systematically underestimated in a node-height dependent manner: i.e., the deeper nodes tend to be more underestimated than the shallow ones. Such biases have implications for molecular dating. We dated the evolutionary history of 30 worldwide fruit fly populations, and we found signatures of GC-bias considerably affecting the estimated divergence times (up to 23%) in the neutral model. Our findings call for the need to account for selection when quantifying divergence or dating species evolution.

Significance statement Although little is known about the impact of natural selection on species tree estimation, expectations are that it occurs at a minority of loci in eukaryotic genomes and is thus unlikely to affect the divergence process. However, growing evidence suggests that a large amount of the genomic variation evolves under weak but pervasive selection (e.g., fixation biases created by GC-bias gene conversion). We tested the impact of unaccounted-for nearly neutral selection on species tree estimation and found that the estimated branch lengths are systematically biased. Our results highlight the need for selection-aware models in species tree estimation and molecular dating.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-30"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2569744"/>
  <title>
   Nucleotide usage biases distort inferences of the species tree | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"I3YJjxfbHVi_gNEt0U3FFQMIjZ3u-MOOO7FirAs4nn4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.07.30.228965v3"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.07.30.228965v3"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.07.30.228965v3"}],"nid":"2569744","apath":"\/biorxiv\/early\/2022\/05\/30\/2020.07.30.228965.atom","pisa":"biorxiv;2020.07.30.228965v3","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.07.30.228965v3","title":"Nucleotide usage biases distort inferences of the species tree","identifier":"node\/2569744"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022toONHBJwzVMpDmsVttVRr1SO1pcpZabAtsUz5PITBLohyjwadgTmRm\\\\\\\/Mbl1+fQOJFIC7dXwx6HwQgYo\\\\\\\/arJ5AfKD7EHCccEtjT0UymR3IrCO3Z98Tjp7xHnSUvis8CyKrxnIIjjmpnhftbyOy3CIJlp5FNy\\\\\\\/YmBnAPgVfoAT\\\\\\\/6ziez7ZzJjHIONfpuRAnbMuj37P83nry1dEdWJ99VsFPB\\\\\\\/3Hk1voJtA5neIqKeaU3EPLRBAlHuOgk30gJF4qoTmBzZx5rvAzcATNCDn+IF5fCPEouZV45A0QX1dSFpUNnrW4+GE7CGH\\\\\\\/1qYvq4DejhHng4yO2bY\\\\\\\/KeaR\\\\\\\/IRJ9UbeVl+zsmg0AHIKCEd8ozWciv\\\\\\\/x1U\\\\\\\/+CzMsa4WoIBZTOjFenCEKSb4erpGmJVyiWJtBwi0dYJZ5ONaWvlR5W2Q2qJTLvWmz32plPgtSOIBE0Vcj+SniaGbyk\\\\\\\/Gz3a8ID9jzt7P5QkfTKl6dLy7gdrki0fHpZ5KAYywKhLK2dKH1IlGpWgP+fKCoDJIupL3jrbY82ns30SmOMksq8hEORKATzKLar7pwg9KnEBSlx91o01FloP2akGvVOWz2IYh1931JtaJRQTX+AnmGNMCB3WwamLtD5QvTNhcI8+c7E\\\\\\\/RZPxgJHV+faGP37RJOlaLiydQ2BFgUy5Ljlu0NOJNp8qx0a+1HH+WvDjLY7nL2yWB6Kx0d7LXg9iuUGR9DIQ8RkBO8fVEZLyPkbxH1VB1HoiazPUgAgLuq3yWvDeDltZGzSfPWncAu89PC1oUhyycveiRn8VA4SUjjyCX3Wv4hV+iV8I3KT81SZYccPKEd6OhTwb5c5fTaSBA579oNCCAQlHQ7vWd3x+mr6l43frzQ0OvxnhIYP7CHaS1+\\\\\\\/xB6pPa87uyP6\\\\\\\/PWuFNgwI7OwJL7VfFZLvaAqAtHqWkuVj8r7RzGhqkfLWI739zvCZGQP+k\\\\\\\/MLqw76GtYmc1LVROyD6FHgfpCQ3VphelHL9XmoDZyxs4mg\\\\\\\/pLgjTy8oN\\\\\\\/WwsrPgznojhU+A8+bwS0HrzLlD8mM+tJTV7Zq+bLx9VoMEHQy3Vd9sRfwE9hR1ec1jMKOuHOCQdHyEc67V5ogLxYgax9FPGSpqiYRHSkszcJ7QD28yS4BjX9\\\\\\\/BefucT4EEnGrAbNVQ03UiXyxgwmyCrjjbBVQqbtGnZjw+QBxKWXLkNXn\\\\\\\/3n7xos9yGUuqFFKmWNvPaeh\\\\\\\/awyBTxQsP85Repu95Ac24qr8AIOsy9rAYxvHkpm\\\\\\\/8D4Wm120oP515jNuUIAU5DmDAG9f4pvASoaGx5awASZiuKbrdDVLYdTAPEDEtwFDUqiQ\\\\\\\/jibZO9lXp4U818nNLV5SjuBmUimE4FQjDnZg\\\\\\\/HrKThNEcZeIijK4JPh4nDbJR4=\\\u0022,\\\u0022iv\\\u0022:\\\u00224hnX5QsyVvs+ZNqxaiXazQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022355623c3bfdeb8fb10c76a8999eccd3f8530d0fcbd24be3dbd3bdbf736e650ad\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.07.30.228965v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2569744 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.07.30.228965v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_225174722">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_225174722" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1282762140" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-nM3rZJrdjTLcK5Ad7HzCS79yjiAZV6Y6B5hk8KU96Do"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2569744" data-node-nid="2569744" id="node-2569744--21312416936" data-pisa="biorxiv;2020.07.30.228965v3" data-pisa-master="biorxiv;2020.07.30.228965" data-apath="/biorxiv/early/2022/05/30/2020.07.30.228965.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Nucleotide usage biases distort inferences of the species tree
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-5905-3778" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Rui
                      </span>
                      <span class="nlm-surname">
                       Borges
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-0776-4460" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Bastien
                      </span>
                      <span class="nlm-surname">
                       Boussau
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-8556-845X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Gergely J.
                      </span>
                      <span class="nlm-surname">
                       Szöllősi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-3219-6648" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Carolin
                      </span>
                      <span class="nlm-surname">
                       Kosiol
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.07.30.228965
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2569744--21312416936" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Rui Borges
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institut für Populationsgenetik, Vetmeduni Vienna
                       </span>
                       , Veterinärplatz 1, 1210 Wien,
                       <span class="nlm-country">
                        Austria
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Rui%2BBorges%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Borges%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ARui%2BBorges%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-5905-3778" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Rui Borges
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Bastien Boussau
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Univ Lyon, Université Claude Bernard Lyon 1
                       </span>
                       , CNRS UMR 5558, LBBE, F-69100, Villeurbanne,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Bastien%2BBoussau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Boussau%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABastien%2BBoussau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0776-4460" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Bastien Boussau
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Gergely J. Szöllősi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Biological Physics, Eötvös University
                       </span>
                       , Pázmány P. stny. 1A, H-1117 Budapest,
                       <span class="nlm-country">
                        Hungary
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        MTA-ELTE “Lendület” Evolutionary Genomics Research Group
                       </span>
                       , Pázmány P. stny. 1A, H-1117 Budapest,
                       <span class="nlm-country">
                        Hungary
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Evolutionary Systems Research Group, Centre for Ecological Research, Hungarian Academy of Sciences
                       </span>
                       , 8237 Tihany,
                       <span class="nlm-country">
                        Hungary
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gergely%2BJ.%2BSz%C3%B6ll%C5%91si%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Sz%C3%B6ll%C5%91si%20GJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGergely%2BJ.%2BSz%25C3%25B6ll%25C5%2591si%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-8556-845X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Gergely J. Szöllősi
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Carolin Kosiol
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Institut für Populationsgenetik, Vetmeduni Vienna
                       </span>
                       , Veterinärplatz 1, 1210 Wien,
                       <span class="nlm-country">
                        Austria
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Centre for Biological Diversity, University of St Andrews
                       </span>
                       , St Andrews, Fife KY16 9TH,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Carolin%2BKosiol%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kosiol%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACarolin%2BKosiol%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-3219-6648" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Carolin Kosiol
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:ck202@st-andrews.ac.uk" class="" data-icon-position="" data-hide-link-title="0">
                        ck202@st-andrews.ac.uk
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.07.30.228965v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.228965v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.228965v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.228965v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.228965v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.228965v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.07.30.228965v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2569744" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2569744/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Despite the importance of natural selection in species’ evolutionary history, phylogenetic methods that take into account population-level processes typically ignore selection. The assumption of neutrality is often based on the idea that selection occurs at a minority of loci in the genome and is unlikely to compromise phylogenetic inferences significantly. However, genome-wide processes like GC-bias and some variation segregating at the coding regions are known to evolve in the nearly neutral range. As we are now using genome-wide data to estimate species trees, it is natural to ask whether weak but pervasive selection is likely to blur species tree inferences. We developed a polymorphism-aware phylogenetic model tailored for measuring signatures of nucleotide usage biases to test the impact of selection in the species tree. Our analyses indicate that while the inferred relationships among species are not significantly compromised, the genetic distances are systematically underestimated in a node-height dependent manner: i.e., the deeper nodes tend to be more underestimated than the shallow ones. Such biases have implications for molecular dating. We dated the evolutionary history of 30 worldwide fruit fly populations, and we found signatures of GC-bias considerably affecting the estimated divergence times (up to 23%) in the neutral model. Our findings call for the need to account for selection when quantifying divergence or dating species evolution.
                            </p>
                            <div id="sec-1" class="subsection">
                             <p id="p-3">
                              <strong>
                               Significance statement
                              </strong>
                              Although little is known about the impact of natural selection on species tree estimation, expectations are that it occurs at a minority of loci in eukaryotic genomes and is thus unlikely to affect the divergence process. However, growing evidence suggests that a large amount of the genomic variation evolves under weak but pervasive selection (e.g., fixation biases created by GC-bias gene conversion). We tested the impact of unaccounted-for nearly neutral selection on species tree estimation and found that the estimated branch lengths are systematically biased. Our results highlight the need for selection-aware models in species tree estimation and molecular dating.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-7">
                             Species trees provide a framework to understand the divergence process and speciation and are nowadays used routinely in integrative research to address many biological questions. Despite their generalized use, modeling species evolution using DNA sequences poses significant challenges. The main one being that different genes or genomic regions narrate different evolutionary histories, leading to discordant gene and species topologies (
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              Maddison and Knowles, 2006
                             </a>
                             ). Incomplete lineage sorting (ILS), i.e., the maintenance of ancestral polymorphisms due to random genetic drift, is a primary cause of such discordance (
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Pollard et al., 2006
                             </a>
                             ). However, other processes were also described, such as gene gain or loss, horizontal gene transfer across the species boundaries, and gene flow among diverging populations (reviewed in
                             <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                              Szöllősi et al. (2015)
                             </a>
                             ). Apart from the gene/species tree discordance, difficulties arise when quantifying divergence.
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              Xu and Yang (2016)
                             </a>
                             reported that calendar times and evolutionary rates can be particularly challenging to untie for deeper evolutionary scales where the molecular clock is violated.
                            </p>
                            <p id="p-8">
                             Despite the challenges posed by modeling species evolution, the last two decades have seen an explosion of sophisticated statistical methods for inferring species trees. The multispecies coalescent model (MSC) has arisen as a leading framework for inferring species phylogenies while accounting for ILS and gene tree-species tree conflict (
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              Rannala and Yang, 2003
                             </a>
                             ). The coalescent traces the genealogical history of a sample of sequences from a population backward in time, describing the stochastic process of lineage joining (
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              Kingman, 1982a
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              b
                             </a>
                             ).
                            </p>
                            <p id="p-9">
                             Nevertheless, alternative approaches to the MSC have been proposed, as the polymorphismaware phylogenetic models (PoMos;
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              fig. 1
                             </a>
                             ) (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              De Maio et al., 2013
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              2015
                             </a>
                             ). PoMos model the allele content of a set of populations over time at a particular locus, thus naturally integrating over all the possible allelic histories to directly estimate the species tree. (To be technically accurate, PoMos estimate population trees. However, because of PoMo ability to model closely related populations as well as diverged populations it most prominent application are populations sampled to estimate species trees. Therefore, we will adhere to this terminology in this article.) PoMos naturally account for ILS while avoiding using genealogy samplers that are computationally costly. Different from the MSC, PoMos assume independence between sites, which allows easily gathering information from multiple individuals and several populations to testing hypotheses genome-wide (
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              Schrempf et al., 2016
                             </a>
                             ,
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              2019
                             </a>
                             ). More importantly, PoMos are versatile, for they can be straightforwardly rebuilt to account for other population forces (e.g., allelic selection (
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Borges et al., 2019
                             </a>
                             )). MSC-based methods with selection are notoriously difficult.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F1.large.jpg?width=800&height=600&carousel=1" title="State-space and parameter scaling of the virtual PoMos. The tetrahedrons represent the state-space of the virtual PoMos: the fixed sites {Mai} are placed in the vertices, while the edges represent the polymorphic states {mai, (M − m)aj}. Black and grey arrows distinguish respectively mutation from genetic drift plus selection events. PoMo is a particular case of the four-variate Moran model with boundary mutations and selection, in which the alleles are the four nucleotide bases. The formulas represent the parameter scaling between the effective and the virtual populations (supplementary text S1).  is the mutation rate from allele ai to aj; and  is the fitness coefficient of allele ai on the effective dynamic of population size N ( relates to the selection coefficient of ai by the equation ).  and  are the corresponding parameters in the virtual dynamic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">State-space and parameter scaling of the virtual PoMos.</span> The tetrahedrons represent the state-space of the virtual PoMos: the fixed sites {<em>Ma</em><sub><em>i</em></sub>} are placed in the vertices, while the edges represent the polymorphic states {<em>ma</em><sub><em>i</em></sub>, (<em>M</em> − <em>m</em>)<em>a</em><sub><em>j</em></sub>}. Black and grey arrows distinguish respectively mutation from genetic drift plus selection events. PoMo is a particular case of the four-variate Moran model with boundary mutations and selection, in which the alleles are the four nucleotide bases. The formulas represent the parameter scaling between the effective and the virtual populations (supplementary text S1).  is the mutation rate from allele <em>a</em><sub><em>i</em></sub> to <em>a</em><sub><em>j</em></sub>; and  is the fitness coefficient of allele <em>a</em><sub><em>i</em></sub> on the effective dynamic of population size <em>N</em> ( relates to the selection coefficient of <em>a</em><sub><em>i</em></sub> by the equation ).  and  are the corresponding parameters in the virtual dynamic.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F1.medium.gif" width="440" height="280"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F1.medium.gif" width="440" height="280"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1:
                              </span>
                              <span class="caption-title">
                               State-space and parameter scaling of the virtual PoMos.
                              </span>
                              <p id="p-10" class="first-child">
                               The tetrahedrons represent the state-space of the virtual PoMos: the fixed sites {Ma
                               <sub>
                                i
                               </sub>
                               } are placed in the vertices, while the edges represent the polymorphic states {ma
                               <sub>
                                i
                               </sub>
                               , (M − m)a
                               <sub>
                                j
                               </sub>
                               }. Black and grey arrows distinguish respectively mutation from genetic drift plus selection events. PoMo is a particular case of the four-variate Moran model with boundary mutations and selection, in which the alleles are the four nucleotide bases. The formulas represent the parameter scaling between the effective and the virtual populations (supplementary text S1).
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the mutation rate from allele a
                               <sub>
                                i
                               </sub>
                               to a
                               <sub>
                                j
                               </sub>
                               ; and
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the fitness coefficient of allele a
                               <sub>
                                i
                               </sub>
                               on the effective dynamic of population size N (
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               relates to the selection coefficient of a
                               <sub>
                                i
                               </sub>
                               by the equation
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ).
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F1/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are the corresponding parameters in the virtual dynamic.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-11">
                             Despite the remarkable effort made to account for forces that can detail the species evolutionary history, models of species evolution generally ignore natural selection. While the literature acknowledges the need for selection-aware models of species evolution, the expectation is that most forms of natural selection are not greatly compromising phylogenetic analysis (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              Edwards, 2009
                             </a>
                             ). Two main arguments justify this expectation. The first argument is that selection often occurs at a minority of loci in the genome, in which case any spurious signals would likely be swamped out by the many neutral loci that are sampled (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Edwards et al., 2016
                             </a>
                             ). The second argument is that the forms of selection that most likely affect the species tree are rare. This is the case of selection-driven convergent evolution and balancing selection, which tend to preserve beneficial alleles at a gene for long periods of time (
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Castoe et al., 2009
                             </a>
                             ;
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              Edwards, 2009
                             </a>
                             ). As corroboration for these expectations, a recent study found that species-specific positive selection has mild effects on phylogeny estimation across an extensive range of conditions encountered in empirical data sets (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              Adams et al., 2018
                             </a>
                             ).
                            </p>
                            <p id="p-12">
                             However, such claims have not been properly tested, especially for the case of pervasive weak selection. Theoretical expectations predict the existence of an intermediate category between the neutral and the selected modes of evolution, known as nearly neutral. The study of variants with small selection coefficients, particularly slightly deleterious mutations, has been the focus of the so-called nearly neutral theory (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Ohta, 1973
                             </a>
                             ). This theory provided criteria to define the fate of nearly neutral variants, which depends on both selection and random genetic drift.
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              Ohta (2002)
                             </a>
                             has suggested that if the relative advantage or disadvantage, σ, of a particular allele is less than twice the reciprocal of the effective population size N (i.e., the scaled selection coefficient N |σ| < 2) the allele’s trajectory is nearly neutral. However, more lenient thresholds were also reasoned (e.g.,
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              Nei (2005)
                             </a>
                             ).
                            </p>
                            <p id="p-13">
                             Empirical evidence for nearly neutral evolution has become more substantial within the last few years due to the possibility of sequencing many genomes and multiple individuals. One such example is GC-biased gene conversion (gBGC), a mutation bias favoring G and C alleles over A and T alleles at mismatch positions during recombination (
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Galtier et al., 2001
                             </a>
                             ). gBGC affects the fixation probability of GC alleles and is best modeled as selection (
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Nagylaki, 1983
                             </a>
                             ). Integrative analysis considering the recombination landscape and nucleotide substitution patterns along the genome provided evidence that gBGC acts in both eukaryotes and bacteria (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Galtier et al., 2009
                             </a>
                             ;
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Lassalle et al., 2015
                             </a>
                             ). In humans, gBGC was estimated one order of magnitude lower than the reciprocal of the effective population size (∼ 1.17 × 10
                             <sup>
                              −5
                             </sup>
                             ) (
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Glémin, 2010
                             </a>
                             ), thus, gBGC produces patterns of nucleotide substitution that resemble weak selection. The same was observed for other apes where most of their exons evolve under weak gBGC (Nγ < 1; where γ is the GC-bias rate) (
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              Lartillot, 2013
                             </a>
                             ). In fruit flies, most amino acid replacements have weak signatures of positive selection. However, most of the selective effects are nearly neutral, with around 46% of amino acid replacements exhibiting scaled selection coefficients lower than two and 84% lower than four (
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              Sawyer et al., 2007
                             </a>
                             ). Also, many non-synonymous polymorphisms in functionally important sites of human and bacterial populations were shown to segregate at frequencies of around 1–10%; i.e., much more frequent than variants associated with classic Mendelian diseases (
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Hughes et al., 2003
                             </a>
                             ;
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Hughes, 2005
                             </a>
                             ). Such an observation suggests that ongoing purifying selection among segregating amino acids is weak.
                            </p>
                            <p id="p-14">
                             As a substantial part of the genome evolves nearly neutrally, this form of weak but pervasive selection has the potential to impact species’ evolutionary history. Here, we present novel PoMos specially tailored to measure nucleotide usage bias throughout the genome and use them to assess the impact of unaccounted-for weak selection (or selection-like processes) on species tree estimation. We show that near neutrality significantly blurs inferences of the species tree by biasing estimations of the species divergence and discuss the implications of these biases on molecular dating.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Modelling species evolution with selection
                             </h3>
                             <p id="p-15">
                              PoMos offer a versatile approach to describe species evolution (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Leaché and Oaks, 2017
                              </a>
                              ). They add a new layer of complexity to the standard models of sequence evolution by accounting for population-level processes (such as mutations, genetic drift, and selection) to describe sequence evolution. To do that, PoMos expand the 4 × 4 state-space to model a population of individuals in which changes in allele content and frequency are both possible. The PoMo state-space includes fixed (or boundary) states {Na
                              <sub>
                               i
                              </sub>
                              }, in which all N individuals have the same allele a
                              <sub>
                               i
                              </sub>
                              ∈ {A, C, G, T}, but also polymorphic states {na
                              <sub>
                               i
                              </sub>
                              , (N − n)a
                              <sub>
                               j
                              </sub>
                              }, if two alleles a
                              <sub>
                               i
                              </sub>
                              and a
                              <sub>
                               j
                              </sub>
                              are present in the population with absolute frequencies n and N − n (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               fig. 1
                              </a>
                              ). PoMos do not consider triallelic sites, a simplification that is generally acceptable for eukaryotes, for which the levels of polymorphism are low (
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               Lynch et al., 2016
                              </a>
                              ). Furthermore, like many phylogenetic methods, PoMos assume that sites are independent (i.e., linkage equilibrium).
                             </p>
                             <p id="p-16">
                              One of the main challenges of PoMos is the size of their state-space grows with the effective population size: i.e., 4 + 6(N − 1) states. This prohibits inferences with realistic population sizes as they can easily have 10
                              <sup>
                               4
                              </sup>
                              − 10
                              <sup>
                               6
                              </sup>
                              individuals (
                              <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                               Lynch et al., 2016
                              </a>
                              ), PoMos would typically employ 10 individuals because it results in manageable state-space and robust estimates of population properties such as the genetic diversity θ = 4Nµ (
                              <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                               De Maio et al., 2015
                              </a>
                              ;
                              <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                               Schrempf et al., 2016
                              </a>
                              ). Recently, we developed theory that permitted us to distinguish between the effective and virtual population size (
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Borges et al., 2019
                              </a>
                              ). Here, we use those results to develop the virtual PoMos. The idea is to mimic a population dynamic that unfolds on the effective population N, using a virtual population of smaller size M. It can be shown that by matching the expected diversity (i.e., the proportion of fixed and polymorphic sites) in both the effective and the virtual populations (
                              <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                               Borges et al., 2019
                              </a>
                              ), one can obtain scaling laws for the mutation rates and fitness coefficients (supplementary Text S1 and
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               fig. 1
                              </a>
                              ):
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the mutation rate from allele a
                              <sub>
                               i
                              </sub>
                              to a
                              <sub>
                               j
                              </sub>
                              and
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              the fitness coefficient of allele a
                              <sub>
                               i
                              </sub>
                              on the effective dynamic, while
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are the corresponding parameters in the virtual one.
                             </p>
                             <p id="p-17">
                              Here, we defined the virtual PoMos Two and Three, by setting the virtual population size M to two and three individuals, respectively. PoMoTwo has only one polymorphic state per allelic pair, i.e., {1a
                              <sub>
                               i
                              </sub>
                              , 1a
                              <sub>
                               j
                              </sub>
                              }, and is used to describe species evolution under neutrality (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               fig. 1
                              </a>
                              ) while still accounting for mutation bias. For the neutral case, the mutation rates are scaled by the ratio of harmonic numbers (denoted here by H.) of the virtual and effective population sizes:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              PoMoThree includes two polymorphic states per allelic pair, i..e, {1a
                              <sub>
                               i
                              </sub>
                              , 2a
                              <sub>
                               j
                              </sub>
                              } and {2a
                              <sub>
                               i
                              </sub>
                              , 1a
                              <sub>
                               j
                              </sub>
                              }, and additionally allows one to infer allelic selection (at least two polymorphic states are necessary to make the selection coefficients identifiable;
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               fig. 1
                              </a>
                              ). The scalings in
                              <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                               equation (1)
                              </a>
                              simplify to:
                              <span class="disp-formula" id="disp-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The virtual PoMoThree, in particular, allows quantifying signatures of selection and its impact on evolution.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Virtual population dynamics provide a good fit for their effective counterpart
                             </h3>
                             <p id="p-18">
                              To validate the concept behind the virtual PoMos, we assessed their fit and computational performance using simulated data. We simulated phylogenetic data sets with signatures of weak but pervasive nucleotide usage biases as observed in populations of great apes and fruit flies (
                              <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                               Borges et al., 2019
                              </a>
                              ;
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               Prado-Martinez et al., 2013
                              </a>
                              ;
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               Begun et al., 2007
                              </a>
                              ). In particular, we simulated the typically observed mutation bias from G or C (also known as strong alleles) to A and T (also known as weak alleles) opposing gBGC (
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               Glémin et al., 2015
                              </a>
                              ).
                             </p>
                             <p id="p-19">
                              Mutation rates were assumed reversible, where the mutation rate is defined by the product of an exchangeability term and an allele frequency under no selection: e.g., the mutation rate from a strong (S) to a weak (W) allele is defined as µ
                              <sub>
                               SW
                              </sub>
                              = ρ
                              <sub>
                               SW
                              </sub>
                              π
                              <sub>
                               W
                              </sub>
                              . The four possible mutation rates are thus fully determined by three parameters: π
                              <sub>
                               W
                              </sub>
                              = 1 − π
                              <sub>
                               S
                              </sub>
                              , ρ
                              <sub>
                               SW
                              </sub>
                              = ρ
                              <sub>
                               W S
                              </sub>
                              and ρ
                              <sub>
                               SS
                              </sub>
                              = ρ
                              <sub>
                               W W
                              </sub>
                              . The GC-bias preference was modeled by assigning a fitness of 1 to the weak alleles and a fitness ϕ
                              <sub>
                               S
                              </sub>
                              = 1+γ to the strong alleles. A four-allelic Moran model was employed to simulate the evolutionary history of four populations across eight evolutionary scenarios by combining dynamics with high and low genetic diversity, weak and strong regimes of GC-bias, and shallow and deep divergence times. The specific parameters used in each simulated scenario are listed in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               table 1
                              </a>
                              .
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2572739/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2572739/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2572739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1.
                               </span>
                               <span class="caption-title">
                                Description of the simulated scenarios.
                               </span>
                               <p id="p-20" class="first-child">
                                θ represents the genetic diversity, γ the GC-bias coefficient (which relates to the GC-bias fitness by ϕ
                                <sub>
                                 S
                                </sub>
                                = 1 + γ) and τ the time of the most recent common ancestor in the phylogeny (defined in units of generations).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-21">
                              Phylogenetic data from the eight simulated scenarios and the virtual PoMoThree model were employed to infer the mutation rates and the GC-bias fitness on the virtual population. These estimates were recalibrated back to the effective dynamic using
                              <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                               equation (1)
                              </a>
                              , and the relative bias was calculated to evaluate their accuracy. The GC-bias fitness shows the highest accuracy, being estimated with a consistent relative error smaller than 1% across the eight simulated scenarios. This result demonstrates that PoMoThree is particularly suited to quantify selection bias on genomic sequences. For most of the simulated scenarios, the mutation rate parameters are reasonably well estimated, with an average relative error smaller than 5% (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               fig. 2
                              </a>
                              ). These analyses indicate that the virtual PoMos provide a good approximation for an effective dynamic with mutation and selection biases. We have also estimated the mutation rate parameters using the virtual PoMoTwo and observed that the equilibrium base composition of strong alleles is systematically overestimated (i.e., π
                              <sub>
                               S
                              </sub>
                              = 1 − π
                              <sub>
                               W
                              </sub>
                              ), particularly in scenarios where GC-bias is stronger (supplementary fig. S1). This shows that the mutation rates try to accommodate the effect of selection towards the strong alleles by increasing the rate of weak to strong mutations (notice that µ
                              <sub>
                               W S
                              </sub>
                              = ρ
                              <sub>
                               SW
                              </sub>
                              π
                              <sub>
                               S
                              </sub>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F2.large.jpg?width=800&height=600&carousel=1" title="Estimating mutation rates and selection coefficients with a virtual dynamic. The simulated data from a four-allelic Moran dynamic with mutation bias and selection was fed to the virtual PoMoThree and the population parameters estimated using the Bayesian phylogenetic tool RevBayes (Höhna et al., 2016). As the population parameters were estimated in a population of three virtual individuals, we recalibrated them back to the effective population via the parameter scalings in equation (1). The estimated parameters are four: πW, ρSW and ρSS are terms of the mutation rates that were assumed reversible (i.e., µSW = ρSW πW); ϕS is the fitness coefficient of the strong alleles (i.e., the GC-bias fitness). The average bias of the estimates was obtained based on alignments of 105 sites and 25 replicates per scenario. The specific parameters used in each simulated scenario are listed in table 1. The red dotted line represents a 5% relative error threshold." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating mutation rates and selection coefficients with a virtual dynamic.</span> The simulated data from a four-allelic Moran dynamic with mutation bias and selection was fed to the virtual PoMoThree and the population parameters estimated using the Bayesian phylogenetic tool RevBayes (Höhna et al., 2016). As the population parameters were estimated in a population of three virtual individuals, we recalibrated them back to the effective population via the parameter scalings in equation (1). The estimated parameters are four: <em>π</em><sub><em>W</em></sub>, <em>ρ</em><sub><em>SW</em></sub> and <em>ρ</em><sub><em>SS</em></sub> are terms of the mutation rates that were assumed reversible (i.e., <em>µ</em><sub><em>SW</em></sub> = <em>ρ</em><sub><em>SW</em></sub> <em>π</em><sub><em>W</em></sub>); <em>ϕ</em><sub><em>S</em></sub> is the fitness coefficient of the strong alleles (i.e., the GC-bias fitness). The average bias of the estimates was obtained based on alignments of 10<sup>5</sup> sites and 25 replicates per scenario. The specific parameters used in each simulated scenario are listed in table 1. The red dotted line represents a 5% relative error threshold.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F2.medium.gif" width="440" height="130"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F2.medium.gif" width="440" height="130"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                Estimating mutation rates and selection coefficients with a virtual dynamic.
                               </span>
                               <p id="p-22" class="first-child">
                                The simulated data from a four-allelic Moran dynamic with mutation bias and selection was fed to the virtual PoMoThree and the population parameters estimated using the Bayesian phylogenetic tool RevBayes (
                                <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                 Höhna et al., 2016
                                </a>
                                ). As the population parameters were estimated in a population of three virtual individuals, we recalibrated them back to the effective population via the parameter scalings in
                                <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">
                                 equation (1)
                                </a>
                                . The estimated parameters are four: π
                                <sub>
                                 W
                                </sub>
                                , ρ
                                <sub>
                                 SW
                                </sub>
                                and ρ
                                <sub>
                                 SS
                                </sub>
                                are terms of the mutation rates that were assumed reversible (i.e., µ
                                <sub>
                                 SW
                                </sub>
                                = ρ
                                <sub>
                                 SW
                                </sub>
                                π
                                <sub>
                                 W
                                </sub>
                                ); ϕ
                                <sub>
                                 S
                                </sub>
                                is the fitness coefficient of the strong alleles (i.e., the GC-bias fitness). The average bias of the estimates was obtained based on alignments of 10
                                <sup>
                                 5
                                </sup>
                                sites and 25 replicates per scenario. The specific parameters used in each simulated scenario are listed in
                                <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                 table 1
                                </a>
                                . The red dotted line represents a 5% relative error threshold.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-23">
                              We further compared the computational efficiency of the virtual PoMos, by comparing it to the standard general time-reversible nucleotide substitution model (GTR;
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               Tavaré (1986)
                              </a>
                              ) in multiple sequence alignments, including 10 to 100 species and 100 to 1 million sites. For the three methods, similar settings for the phylogenetic inferences were employed in RevBayes (
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               Höhna et al., 2016
                              </a>
                              ), including a standard MCMC chain with 50 000 generations and no parallelization. The computational performance of these three methods was assessed using three criteria: memory usage, CPU time, and average effective sample size of the sampled posterior parameters (ESS; i.e., the number of effectively independent draws from the posterior distribution that the Markov chain is equivalent to).
                             </p>
                             <p id="p-24">
                              Despite PoMoThree and Two having a larger state-space (16 and 10 states respectively) and therefore more possibilities for different site patterns at the tips, the memory usage is relatively similar among the methods (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               fig. 3A
                              </a>
                              ). The memory usage of PoMoThree is on average 16% higher than that of PoMoTwo and 39% higher than that of GTR. However, we observed that in some cases, RevBayes could not start the analyses because the memory reached the computers’ maximum. Thus, while memory usage does not differ greatly among the methods, it certainly might be a limitation for genome-wide applications. Different from memory usage, the CPU time varies considerably among methods. We observed that the time consumed by each method increases exponentially with the alignment size (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               fig. 3B
                              </a>
                              ). PoMoThree requires, on average, 10.5 times more CPU time than the GTR and 2.3 times than PoMoTwo. This is because the PoMoThree has a larger state-space and consequently more site patterns to evaluate the likelihood. Regarding the average ESS, PoMoThree is the slowest mixing method with an average ESS of 2000 (supplementary table S1), 21.9% and 17.6% smaller than the average ESS of GTR and PoMoTwo, respectively.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F3.large.jpg?width=800&height=600&carousel=1" title="Computational performance of the virtual PoMos. The computational performance of the virtual PoMos Two and Three was compared with the general time-reversible model (GTR) based on (A) memory usage and (B) CPU time. The Bayesian inferences consisted of a standard MCMC chain with 50 000 generations and on simulated alignments with 10 to 100 species and 100 to 1 million sites, with levels of diversity similar to those of great apes (using the parameters defined for the simulated scenario 5). These analyses ran on an iMac desktop with an Intel Core i5 processor of 3.4 GHz of speed and 32GB of memory, without any kind of parallelization. Missing dots represent situations where the computer’s maximum memory was reached, and RevBayes was not able to start the analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Computational performance of the virtual PoMos.</span> The computational performance of the virtual PoMos Two and Three was compared with the general time-reversible model (GTR) based on <strong>(A)</strong> memory usage and <strong>(B)</strong> CPU time. The Bayesian inferences consisted of a standard MCMC chain with 50 000 generations and on simulated alignments with 10 to 100 species and 100 to 1 million sites, with levels of diversity similar to those of great apes (using the parameters defined for the simulated scenario 5). These analyses ran on an iMac desktop with an Intel Core i5 processor of 3.4 GHz of speed and 32GB of memory, without any kind of parallelization. Missing dots represent situations where the computer’s maximum memory was reached, and RevBayes was not able to start the analyses.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F3.medium.gif" width="440" height="294"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F3.medium.gif" width="440" height="294"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                Computational performance of the virtual PoMos.
                               </span>
                               <p id="p-25" class="first-child">
                                The computational performance of the virtual PoMos Two and Three was compared with the general time-reversible model (GTR) based on
                                <strong>
                                 (A)
                                </strong>
                                memory usage and
                                <strong>
                                 (B)
                                </strong>
                                CPU time. The Bayesian inferences consisted of a standard MCMC chain with 50 000 generations and on simulated alignments with 10 to 100 species and 100 to 1 million sites, with levels of diversity similar to those of great apes (using the parameters defined for the simulated scenario 5). These analyses ran on an iMac desktop with an Intel Core i5 processor of 3.4 GHz of speed and 32GB of memory, without any kind of parallelization. Missing dots represent situations where the computer’s maximum memory was reached, and RevBayes was not able to start the analyses.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Ignored selection affects species tree topology mildly
                             </h3>
                             <p id="p-26">
                              The impact of unaccounted-for selection on the reconstructed species tree can only be truly known by assessing the correctness of the estimation of both the topology and the branch lengths. Such aspects can be easily tested using simulated data on a known species tree. We used a species tree with four evolving populations, including two closely related populations and a third one recently diverging from the other two:
                              <kbd>
                               (A,(B,(C,D)))
                              </kbd>
                              . We set the evolutionary distances matching so that the first, second, and third divergence times are equal to τ, τ/2 and τ/4 respectively, where τ is the time of the most recent common ancestor. We thus expect higher levels of ILS when τ is equal to N generations than when it is equal to 10N (
                              <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                               table 1
                              </a>
                              ).
                             </p>
                             <p id="p-27">
                              To investigate the impact of selection on the species tree topology, we compared the trees inferred by the virtual PoMos (i.e., neutral PoMoTwo and the selection-aware PoMoThree) and the standard GTR model. If not stated otherwise, fixed and polymorphic counts from a sampled 10 individuals per population was used to perform the phylogenetic inferences with the virtual PoMos. As the GTR model only includes fixed states, a single random individual was used. The posterior probability of the correct topology was used to evaluate the accuracy of each method. Our results show that the virtual PoMos are more accurate than the GTR in all the tested scenarios, especially for alignments with fewer sites (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               fig. 4
                              </a>
                              and supplementary fig. S2). Because there are 15 different topologies, the expected posterior probability of the true tree if there is no signal in the data is around 6.7%, which is close to the performance of our methods in some conditions when there is little data. An interesting feature is that the topologies estimated by the GTR model tend to pick the first diverging branch incorrectly with greater probability than PoMos, especially for polymorphism-poor data sets (scenarios 1 and 5;
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               fig. 4
                              </a>
                              ). This suggests that apart from being able to correct for ILS among closely related taxa, the PoMos are also superior in establishing deeper relationships. Despite that, we observed that the GTR’s accuracy increases with the alignment size, showing that the GTR topologies should converge to the correct species tree topology with genome-scale data.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F4.large.jpg?width=800&height=600&carousel=1" title="Estimating the species tree topology with selection. Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the topology by assessing the posterior probability of the true topology ((A,(B,(C,D))); blue sub-bars) under these three methods. The remaining probability corresponds to wrongly estimated topologies. The posterior probabilities of wrong topologies that are still able to correctly identify the first taxon to diverge are represented in shades of gray: (A,(C,(B,D))) in dark gray and (A,(D,(C,B))) in light gray. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences). The results for the eight scenarios can be found in supplementary fig. S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating the species tree topology with selection.</span> Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the topology by assessing the posterior probability of the true topology (<kbd>(A,(B,(C,D)))</kbd>; blue sub-bars) under these three methods. The remaining probability corresponds to wrongly estimated topologies. The posterior probabilities of wrong topologies that are still able to correctly identify the first taxon to diverge are represented in shades of gray: <kbd>(A,(C,(B,D)))</kbd> in dark gray and <kbd>(A,(D,(C,B)))</kbd> in light gray. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences). The results for the eight scenarios can be found in supplementary fig. S2.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F4.medium.gif" width="440" height="286"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F4.medium.gif" width="440" height="286"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Estimating the species tree topology with selection.
                               </span>
                               <p id="p-28" class="first-child">
                                Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the topology by assessing the posterior probability of the true topology (
                                <kbd>
                                 (A,(B,(C,D)))
                                </kbd>
                                ; blue sub-bars) under these three methods. The remaining probability corresponds to wrongly estimated topologies. The posterior probabilities of wrong topologies that are still able to correctly identify the first taxon to diverge are represented in shades of gray:
                                <kbd>
                                 (A,(C,(B,D)))
                                </kbd>
                                in dark gray and
                                <kbd>
                                 (A,(D,(C,B)))
                                </kbd>
                                in light gray. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences). The results for the eight scenarios can be found in supplementary fig. S2.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-29">
                              PoMoThree is only slightly more accurate than PoMoTwo, which indicates that neutral models may already provide fairly good estimates of the species tree topology (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               fig. 4
                              </a>
                              ). This is still valid when we increase the intensity of GC-selection (scenario 5 vs. 7;
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               fig. 4
                              </a>
                              )), suggesting that weak selection regimes should not greatly impact inferences of the relationships among species. Overall, our results confirm that the three methods perform similarly in estimating the species tree topology if enough genomic data is available. The specific number of sites necessary to return the correct topology will depend on several aspects, as the number of taxa and the complexity of the phylogenetic relationships among them. In any case, our results suggest that regardless of the employed method, better estimates of the species tree topology are expected for well-diverged taxa and polymorphism-rich data sets (scenario 1 vs. 2 and 7;
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               fig. 4
                              </a>
                              ).
                             </p>
                             <p id="p-30">
                              Similar conclusions can be drawn from when a sample of two individuals per population is taken instead of 10. Though the overall accuracy of PoMos decreases (supplementary fig. S3), we again observed that the posterior probability of the correct topology increases with the alignment size. This result is not surprising if we consider the results we have already discussed for the GTR model. Choosing a single virtual sequence to represent the whole population, as is often done when employing the standard nucleotide models with multi-individual data, corresponds to sample a single individual for the population (i.e., the situation where no polymorphisms are observed). Nevertheless, despite the dramatic reduction of the observed diversity, we observed that the GTR model can estimate the correct topology with enough genomic data.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Neglected selection distorts the measured divergence
                             </h3>
                             <p id="p-31">
                              We investigated the impact of selection on the branch lengths by comparing the genetic distances estimated by the virtual PoMos and the standard GTR model. To assess the accuracy of the branch length estimates, we evaluated the absolute error between the estimated and expected pairwise distances among the four taxa. This strategy has the advantage of permitting comparing methods even if the estimated topologies are not exactly matching. Normalized distances were used for this comparison, as the GTR and PoMos operate in different evolutionary units: number of substitutions and Moran events (i.e., mutations plus frequency shifts), respectively.
                             </p>
                             <p id="p-32">
                              Our results show that the error in the estimated branch lengths decreases as the size of the alignments increases (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               fig. 5
                              </a>
                              and supplementary fig. S4). Again, we observed that the GTR is the model of evolution producing the least accurate estimates, with an average error 2.5 times higher than PoMoThree, indicating that polymorphisms have a considerable impact on branch length estimation. PoMoThree outperforms PoMoTwo for all the observed scenarios, with PoMoTwo having an increased error of 61% on average. In some scenarios, PoMoThree has a negligible error (e.g., scenarios 2 and 8;
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              and supplementary figs. S4 and S5), demonstrating that the virtual PoMos provide a good fit for the divergence process operating in an effective dynamic with mutation and selection biases. If we only consider the scenarios where selection is more intense (i.e., Nγ = 1.5), the error increases to 75%, indicating that ignored selection significantly contributes to bias the estimated divergence (e.g., scenarios 6 vs. 8;
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              and supplementary figs. S4 and S5).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F5.large.jpg?width=800&height=600&carousel=1" title="Estimating the species tree branch lengths with selection. Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the branch lengths by calculating the absolute error of all six pairwise comparisons of genetic distances among the four taxa. As the three methods employ different evolutionary units, the branch lengths were normalized so that the three models could be compared. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Estimating the species tree branch lengths with selection.</span> Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the branch lengths by calculating the absolute error of all six pairwise comparisons of genetic distances among the four taxa. As the three methods employ different evolutionary units, the branch lengths were normalized so that the three models could be compared. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F5.medium.gif" width="440" height="281"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F5.medium.gif" width="440" height="281"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Estimating the species tree branch lengths with selection.
                               </span>
                               <p id="p-33" class="first-child">
                                Data sets of different sizes (x-axis) and simulated under different scenarios were used to perform species tree inferences with three methods: GTR, PoMoTwo (P2 in the figure), and PoMoThree (P3 in the figure). We investigated the accuracy of the branch lengths by calculating the absolute error of all six pairwise comparisons of genetic distances among the four taxa. As the three methods employ different evolutionary units, the branch lengths were normalized so that the three models could be compared. PoMoTwo and Three trees were inferred based on a sample of 10 individuals per taxa (totaling 40 sequences).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-34">
                              We have also observed that the error on the estimated branch lengths does not decay to zero as fast as for the topology case, persisting in some cases even when the topologies are perfectly estimated (inferences with 10
                              <sup>
                               5
                              </sup>
                              sites in scenarios 1 and 7;
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               fig. 5
                              </a>
                              ). Because of the sampling of individuals, polymorphisms in the original population may appear as fixed differences and provide misleading phylogenetic signal that could contribute to this slow decay. We observed that decreasing the sample size from 10 to two individuals increases the error of the estimated branch lengths by 75% on average in the virtual PoMos (supplementary fig. S5). As the patterns of diversity on the original and sampled populations are different, the models of evolution are unable to determine the exact contribution of mutations, fixations, and frequency shifts for the divergence process. The impact of such sampling bias should be particularly problematic among recently diverged populations, where several populations likely share segregating polymorphisms. Indeed, our simulations corroborate this expectation: scenarios 1 vs. 2 (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               fig. 5
                              </a>
                              ).
                             </p>
                             <p id="p-35">
                              To establish that the superior performance of PoMoThree was not due to a larger state space with more polymorphic states, we have calculated the accuracy of the estimated topologies and branch lengths by employing the PoMoThree model with selection coefficients fixed to 0.0. We observed that this neutral PoMoThree has an intermediate accuracy between PoMoTwo and PoMoThree with selection (supplementary figs. S6 and S7), which suggest that modeling selection does increase the fit of PoMos beyond that what would have been expected by having a finer state space with more polymorphic states.
                             </p>
                             <p id="p-36">
                              To further investigate the source of bias on the branch lengths, we tested whether the placement of the estimated divergence varies across evolutionary scales. To do that, we compare the height of each node on the true tree as estimated by the neutral PoMoTwo and the selection-aware PoMoThree models. For this comparison, we have only used node heights estimated under the 100 000 sites alignments, where both methods estimate the true topology correctly. We observed that PoMoTwo produces smaller node heights on average than PoMoThree (clouds always below the identity line in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               fig. 6A
                              </a>
                              ). This result suggests that weak selection signatures can mislead neutral models to estimate species histories with more recent divergence events. However, and more importantly, this bias is not uniform across nodes. The shallow nodes are less underestimated than the deeper nodes, with the ratio of node heights moving away from 1.0 from the shallow to the deeper nodes in all the tested scenarios (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               fig. 6A
                              </a>
                              and supplementary figs. S8 and S9). The rate of evolution is known to be challenging to untie at deeper evolutionary scales (
                              <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                               Xu and Yang, 2016
                              </a>
                              ); here, we show that ignoring pervasive weak selection may contribute to it.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F6.large.jpg?width=800&height=600&carousel=1" title="Estimating divergence in the presence of selection. (A) We investigated the association between the node heights as estimated by the virtual PoMos Two and Three. The phylogenetic tree depicts the three nodes’ placement in the true phylogeny: the node heights double as we approach the root. Values accompanying each cloud denote the average ratio of the node heights in PoMoTwo to those in PoMoThree; these ratios are always smaller than 1.0 for all the other simulated scenarios (supplementary figs. S8 and S9). Each cloud represents 25 replicated alignments. The depicted node heights were all estimated using the 100 000 alignments, where both methods return the correct topology with maximum accuracy. (B) The expected divergence was calculated for different regimes of selection and mutation. For the biallelic case, the base composition π is proportional to the mutation bias: , where µ is the mutation rate. If the base composition is equal (gray line), there is no mutation bias (supplementary text S2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Estimating divergence in the presence of selection.</span> <strong>(A)</strong> We investigated the association between the node heights as estimated by the virtual PoMos Two and Three. The phylogenetic tree depicts the three nodes’ placement in the true phylogeny: the node heights double as we approach the root. Values accompanying each cloud denote the average ratio of the node heights in PoMoTwo to those in PoMoThree; these ratios are always smaller than 1.0 for all the other simulated scenarios (supplementary figs. S8 and S9). Each cloud represents 25 replicated alignments. The depicted node heights were all estimated using the 100 000 alignments, where both methods return the correct topology with maximum accuracy. <strong>(B)</strong> The expected divergence was calculated for different regimes of selection and mutation. For the biallelic case, the base composition <em>π</em> is proportional to the mutation bias: , where <em>µ</em> is the mutation rate. If the base composition is equal (gray line), there is no mutation bias (supplementary text S2).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F6.medium.gif" width="440" height="279"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F6.medium.gif" width="440" height="279"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <span class="caption-title">
                                Estimating divergence in the presence of selection.
                               </span>
                               <p id="p-37" class="first-child">
                                <strong>
                                 (A)
                                </strong>
                                We investigated the association between the node heights as estimated by the virtual PoMos Two and Three. The phylogenetic tree depicts the three nodes’ placement in the true phylogeny: the node heights double as we approach the root. Values accompanying each cloud denote the average ratio of the node heights in PoMoTwo to those in PoMoThree; these ratios are always smaller than 1.0 for all the other simulated scenarios (supplementary figs. S8 and S9). Each cloud represents 25 replicated alignments. The depicted node heights were all estimated using the 100 000 alignments, where both methods return the correct topology with maximum accuracy.
                                <strong>
                                 (B)
                                </strong>
                                The expected divergence was calculated for different regimes of selection and mutation. For the biallelic case, the base composition π is proportional to the mutation bias:
                                <span class="inline-formula" id="inline-formula-11">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F6/embed/inline-graphic-11.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/F6/embed/inline-graphic-11.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , where µ is the mutation rate. If the base composition is equal (gray line), there is no mutation bias (supplementary text S2).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-38">
                              While our simulations intend to represent realistic scenarios, one might want to know what is the overall impact of selection (not necessarily nearly neutral) on the branch lengths. An advantage of PoMos is that some quantities can be formally obtained, such as the expected divergence per generation in diverse mutation-selection regimes (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               fig. 6B
                              </a>
                              and supplementary text S2).
                             </p>
                             <p id="p-39">
                              We observed that the expected divergence is generally higher for the neutral case and tends to decrease as selection becomes more intense (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               fig. 6B
                              </a>
                              ; such trend is not qualitatively affected by the effective population size as shown in supplementary fig. S10). Directional selection fades alleles that could potentially be drifting in the population to fixation, overall reducing the expected divergence: i.e., for the same amount of time, we expect more homogeneous sequences if directional selection is acting. Consequently, to match the same patterns of diversity, selection-aware models tend to return larger branch lengths than the neutral models, corroborating the results obtained in
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               fig. 6A
                              </a>
                              . Furthermore, the underlying population dynamic strongly determines the magnitude of biases on the estimated branch lengths. For instance, the expected divergence is generally higher when selection counteracts mutation (i.e., rarer alleles are favored and vice versa, as is the case with gBGC) than when selection and the mutational bias act concordantly (i.e., rarer alleles are disfavoured and vice versa;
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               fig. 6B
                              </a>
                              ).
                             </p>
                             <p id="p-40">
                              To assess the impact of weak selection on the species trees, we have here simulated and estimated genome-wide effects of GC preference. We note that this strategy is a simplification of the means by which gBGC acts, as it behaves transiently along the genome, being stronger close to recombination hotspots that rapidly change their location at phylogenetic scales (
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               Duret and Galtier, 2009
                              </a>
                              ;
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               Romiguier and Roux, 2017
                              </a>
                              ). Nevertheless, we note that our RevBayes implementation allows for setting more realistic scenarios, including branch-specific models or regional variation on GC-bias rates along the sequence alignment.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Molecular dating with fruit fly populations
                             </h3>
                             <p id="p-41">
                              The geographic origins of the globally distributed fruit fly (Drosophila melanogaster) are still not fully understood. This is certainly not due to the lack of genomic data, as hundreds of sequenced genomes are available for the fruit flies (e.g.,
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Hervas et al. (2017)
                              </a>
                              ), but primarily due to the methods of species tree estimation not scaling with multiple populations and genome-wide data. We employed the virtual PoMos to estimate and date the evolutionary history of 30 fruit fly populations. We used 966 sequences (supplementary table S2) from 1 million distantly located genomic sites, acknowledging the sites’ independence assumption. We also assumed a global clock for molecular dating analyses, which is generally valid for short time scales. We set a uniform prior of 60 000 ± 15 000 years dating the African population expansion, as is suggested in the field’s literature (e.g.,
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               Li and Stephan (2006)
                              </a>
                              ;
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               Stephan and Li (2007)
                              </a>
                              ;
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               Laurent et al. (2011)
                              </a>
                              ). The phylogenies estimated by the virtual PoMos agreed for most of the major clades (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               fig. 7
                              </a>
                              ), overall supporting a similar phylogeographic history for the fruit fly populations. Our analyses showed that the fruit fly populations have flourished in the southeast of the African continent. It is generally accepted that fruit flies originated in equatorial Africa (
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               Lachaise et al., 1988
                              </a>
                              ;
                              <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">
                               Stephan and Li, 2007
                              </a>
                              ). Our analyses suggested further that the species spread to the north, establishing in western equatorial Africa. From there, fruit flies colonized the western equatorial and the northern regions of Africa, with the northern wave leading to the spread of fruit flies worldwide. This phylogeography is not captured by the GTR model (supplementary fig. S11), probably due to high levels of ILS among fruit flies. Indeed, we found that 4.8% of polymorphisms are shared by three or more populations, and 2.0% by 10 or more populations.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F7.large.jpg?width=800&height=600&carousel=1" title="Dating the evolutionary history of 30 worldwide fruit fly populations. We dated the evolutionary history of 30 populations with 4 to 205 individuals (966 sequences in total, supplementary table S1) of Drosophila melanogaster distributed across the globe (data taken from Hervas et al. (2017)). The phylogenetic analyses were employed in RevBayes (Höhna et al., 2016), using alignments of 1 million distantly located genomic sites and the PoMoTwo and PoMoThree models of evolution, the latter being selection-aware. The gray lines between the trees indicate disagreement between the two models. The maximum a posteriori tree is depicted in both cases. The blue rectangles represent the 95% credibility intervals of the age of each node." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-322372946" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Dating the evolutionary history of 30 worldwide fruit fly populations.</span> We dated the evolutionary history of 30 populations with 4 to 205 individuals (966 sequences in total, supplementary table S1) of <em>Drosophila melanogaster</em> distributed across the globe (data taken from Hervas et al. (2017)). The phylogenetic analyses were employed in RevBayes (Höhna et al., 2016), using alignments of 1 million distantly located genomic sites and the PoMoTwo and PoMoThree models of evolution, the latter being selection-aware. The gray lines between the trees indicate disagreement between the two models. The maximum <em>a posteriori</em> tree is depicted in both cases. The blue rectangles represent the 95% credibility intervals of the age of each node.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F7.medium.gif" width="440" height="250"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F7.medium.gif" width="440" height="250"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/30/2020.07.30.228965/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7:
                               </span>
                               <span class="caption-title">
                                Dating the evolutionary history of 30 worldwide fruit fly populations.
                               </span>
                               <p id="p-42" class="first-child">
                                We dated the evolutionary history of 30 populations with 4 to 205 individuals (966 sequences in total, supplementary table S1) of Drosophila melanogaster distributed across the globe (data taken from
                                <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                 Hervas et al. (2017)
                                </a>
                                ). The phylogenetic analyses were employed in RevBayes (
                                <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                                 Höhna et al., 2016
                                </a>
                                ), using alignments of 1 million distantly located genomic sites and the PoMoTwo and PoMoThree models of evolution, the latter being selection-aware. The gray lines between the trees indicate disagreement between the two models. The maximum a posteriori tree is depicted in both cases. The blue rectangles represent the 95% credibility intervals of the age of each node.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-43">
                              The time inferences are less consistent between the virtual PoMos, with PoMoThree providing, as expected by our formal results and simulations, more ancient node ages (around 23%;
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               figs. 7
                              </a>
                              and supplementary figs. S12 and S13). Our divergence times are also comparatively more ancient than other studies. For example, some studies suggested a split of the African lineage at about 16 000 years ago (
                              <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">
                               Stephan and Li, 2007
                              </a>
                              ;
                              <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                               Laurent et al., 2011
                              </a>
                              ), while our analyses estimated it around 38 000 years ago. While unaccounted signatures of selection can underestimate the divergence times, we also notice that the non-monophyly of the out-of-Africa fruit fly populations (i.e., the Egyptian population (EG in
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               fig. 7
                              </a>
                              ) clades together with these populations). This can cause such volatility in the estimated divergence times among studies. We note that the narrow time intervals obtained in our dating analyses should be taken with care, as we are using a sample size of one million sites and a while more complex still simplifying models of evolution and molecular clock. The influence of large data on the time divergence intervals is most clear with the GTR model, which infers a completely different phylogeny than PoMos but with quite accurate time intervals (supplementary fig. S11).
                             </p>
                             <p id="p-44">
                              An additional advantage of using selection-aware methods is that we can quantify selection coefficients. We measured the fruit flies’ GC-bias rate at around 6.17 × 10
                              <sup>
                               −7
                              </sup>
                              per site per generation, one order of magnitude lower than the reciprocal of their effective population size (approximately 1 million individuals according to
                              <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">
                               Lynch et al. (2016)
                              </a>
                              ) and matching the expectations of gBGC. It remains unclear whether fruit flies have gBGC (
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               Robinson et al., 2014
                              </a>
                              ). While it is difficult to think of any other process than gBGC favoring G and C alleles genome-wide, further studies associating these signatures with the recombination landscape are still needed.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-45">
                             Methods of species tree estimation have become an essential tool in studies of molecular evolution. Despite their widespread use, most of these methods assume neutrality. Here, we showed that unaccounted-for pervasive selection and selection-like processes such as gBGC significantly impact the inferences of the species tree. Our extensive analyses in both simulated and real data have two main implications for phylogenomic studies.
                            </p>
                            <p id="p-46">
                             The first one is that assumed neutrality is likely to bias genetic distances in typical population genetic data sets. Modeling natural selection within the species evolution framework is not theoretically or computationally trivial, which has undoubtedly contributed to most tree inference methods making the neutral assumption. The virtual PoMos are full-likelihood methods that include selection and scale with genome-wide data from multiple populations and individuals. Other species tree inference methods, like the MSC, experience a data bottleneck for a few sequences and a handful of populations. Constraints between the species tree and the genealogical histories make it difficult to perform their joint inference. Further-more, the space of unknown genealogies is large, and the genealogy samplers computationally intensive (
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              Flouri et al., 2020
                             </a>
                             ;
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              Rannala et al., 2020
                             </a>
                             ). In contrast, PoMos directly estimate the species tree and naturally integrate over all possible genealogies. Hence, we expect that our methods will permit us to understand better the practical consequences of assuming neutrality in real contexts. Here, we have focused on the case of weak pervasive selection for populations with patterns of diversity similar to fruit flies and great apes. However, our results are likely to extend to several other biological groups, as GC-biased gene conversion is found all across the major groups of eukaryotes (
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              Pessia et al., 2012
                             </a>
                             ).
                            </p>
                            <p id="p-47">
                             The second one is that the bias created by ignoring selection is not uniform across nodes; it is more significant in the deeper nodes than in the shallow ones. Previous reports reasoned that directional selection should only mildly affect the rate of evolution (
                             <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                              Edwards, 2009
                             </a>
                             ). By demonstrating that this effect is node-height dependent, we show that merely recalibrating the overall expected divergence is insufficient to correct divergence time inferences. Therefore, selection-aware models are particularly crucial for molecular dating or other phylogenetic analyses relying on the estimated branch lengths. By showing that one cannot ignore the net effect of selection in the evolutionary process (even in its weakest forms as tested here), we question the neutral assumption’s suitability in species tree inference.
                            </p>
                            <p id="p-48">
                             As the number of genomes sequenced increases rapidly, the development of methods describing the evolutionary process in all its complexity emerges as of fundamental importance. The methods presented here represent an important extension of existing approaches by allowing selection and permitting inference with large-scale data. In the future, we envision that our methods will be particularly useful for molecular dating using genome-wide data sets.
                            </p>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Materials and Methods
                            </h2>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Simulations
                             </h3>
                             <p id="p-49">
                              As our methods aim at estimating species trees with genomic sequences, our simulations were designed to imitate general patterns of molecular evolution. In particular, we set a molecular dynamic with GC-bias favoring the fixation of G and C alleles (or strong alleles) and mutational bias to A and T alleles (or weak alleles) (
                              <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                               Prado-Martinez et al., 2013
                              </a>
                              ;
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               Begun et al., 2007
                              </a>
                              ). Mutation rates were assumed reversible and defined by the product of an exchangeability term and an allele frequency: e.g., the mutation rate from a strong to a weak allele is µ
                              <sub>
                               SW
                              </sub>
                              = ρ
                              <sub>
                               SW
                              </sub>
                              π
                              <sub>
                               W
                              </sub>
                              . Exchangeabilities between weak and strong alleles (i.e., ρ
                              <sub>
                               SW
                              </sub>
                              = ρ
                              <sub>
                               W S
                              </sub>
                              ) were set three times more likely than between similar alleles (i.e., ρ
                              <sub>
                               SS
                              </sub>
                              = ρ
                              <sub>
                               W W
                              </sub>
                              ). We have also set the frequency of the weak alleles to be higher than the strong alleles (i.e., π
                              <sub>
                               W
                              </sub>
                              = 0.7 and π
                              <sub>
                               S
                              </sub>
                              = 0.3). By combining the two exchangeabilities and allele frequencies, four mutation rates can be computed: µ
                              <sub>
                               SW
                              </sub>
                              = ρ
                              <sub>
                               SW
                              </sub>
                              π
                              <sub>
                               W
                              </sub>
                              , µ
                              <sub>
                               W S
                              </sub>
                              = ρ
                              <sub>
                               SW
                              </sub>
                              π
                              <sub>
                               S
                              </sub>
                              , µ
                              <sub>
                               SS
                              </sub>
                              = ρ
                              <sub>
                               SS
                              </sub>
                              π
                              <sub>
                               S
                              </sub>
                              and µ
                              <sub>
                               W W
                              </sub>
                              = ρ
                              <sub>
                               SS
                              </sub>
                              π
                              <sub>
                               W
                              </sub>
                              . A fitness of ϕ
                              <sub>
                               S
                              </sub>
                              = 1 + γ was set to the strong bases, where γ is the GC-bias rate, while the weak bases were assumed neutral with fitness 1.
                             </p>
                             <p id="p-50">
                              We simulated the evolution of a population of four populations of 100 individuals using the PoMo transition matrix, which embedded Markov chain is equivalent to the four-variate Moran model with boundary mutations and allelic selection. The simulator was implemented in R (
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               R Core Team, 2021
                              </a>
                              ) and is provide in the additional files (see data availability statement). The mutation rates and the selection coefficient were set according to the previously defined molecular dynamic and appropriately rescaled. To make our simulation more realistic, we defined different evolutionary scenarios. Following the patterns of diversity observed in populations of fruit flies and great apes (
                              <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                               Prado-Martinez et al., 2013
                              </a>
                              ;
                              <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                               Begun et al., 2007
                              </a>
                              )), we varied the levels of genetic diversity by setting θ = 0.02 or 0.002. We added two regimes of GC-bias with scaled selection coefficient Nγ = 0.5 or 1.5, both under the nearly neutral range (i.e., Nγ< 2). We have also considered different divergence times by setting the time of the most recent common ancestor to τ = N or 10N generations. The evolution of four populations was simulated according to the following phylogenetic tree
                              <kbd>
                               (((pop4:
                              </kbd>
                              τ/4
                              <kbd>
                               , pop3:
                              </kbd>
                              τ/4
                              <kbd>
                               ):
                              </kbd>
                              τ/4
                              <kbd>
                               , pop2:
                              </kbd>
                              τ/2
                              <kbd>
                               ):
                              </kbd>
                              τ/2
                              <kbd>
                               , pop1:
                              </kbd>
                              τ
                              <kbd>
                               );
                              </kbd>
                              . As some populations are closely related, we expect considerable levels of ILS for shorter divergence times. The simulated scenarios and respective parameters are listed in
                              <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                               table 1
                              </a>
                              . For each scenario, we produced alignments of 100 to 10
                              <sup>
                               5
                              </sup>
                              sites and 25 repetitions. We further generated two additional data sets by sampling 10 and 2 individuals per population. To assess the computational performance of the virtual PoMos, we simulated phylogenetic data for 10 to 100 populations and alignments of 100 to 1 million sites based on the parameters of scenario 5.
                             </p>
                             <p id="p-51">
                              Because the simulations were performed in terms of the effective dynamic, the simulated data was converted to match the GTR, PoMoTwo, and PoMoThree state spaces. The fixed states in the original population {Na
                              <sub>
                               i
                              </sub>
                              } were directly converted to the fixed states {3a
                              <sub>
                               i
                              </sub>
                              } and {2a
                              <sub>
                               i
                              </sub>
                              } in the virtual PoMoThree and Two respectively. The polymorphic state {na
                              <sub>
                               i
                              </sub>
                              , (N − n)a
                              <sub>
                               j
                              </sub>
                              } in the original population was directly converted to the state {1a
                              <sub>
                               i
                              </sub>
                              , 1a
                              <sub>
                               j
                              </sub>
                              } PoMoTwo state, as this model has only one polymorphic state per polymorphic type. Differently, PoMoThree has two polymorphic states per polymorphic type. In this case, we used the allele frequency
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in the original state and attributed the state {2a
                              <sub>
                               i
                              </sub>
                              , 1a
                              <sub>
                               j
                              </sub>
                              } with probability
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and {1a
                              <sub>
                               i
                              </sub>
                              ,2a
                              <sub>
                               j
                              </sub>
                              } with probability 1 −
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/30/2020.07.30.228965/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . As the GTR state-space only includes the four alleles, fixed states were set to the fixed allele {a
                              <sub>
                               i
                              </sub>
                              } while polymorphic ones to one of the alleles, {a
                              <sub>
                               i
                              </sub>
                              } or {a
                              <sub>
                               j
                              </sub>
                              }, by randomly drawing according to the allele frequency. The state conversion for the GTR model corresponds to sample a single individual from the population.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Bayesian phylogenetic inferences
                             </h3>
                             <p id="p-52">
                              Phylogenetic trees were estimated using Bayesian inference in the RevBayes software (
                              <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">
                               Höhna et al., 2016
                              </a>
                              ). The virtual PoMos are implemented in RevBayes and can be employed using the functions
                              <kbd>
                               fnReversiblePoMoTwo4N
                              </kbd>
                              and
                              <kbd>
                               fnReversiblePoMoThree4N
                              </kbd>
                              , which can be found in the RevBayes GitHub project (
                              <a href="https://github.com/revbayes">
                               https://github.com/revbayes
                              </a>
                              ). Alternatively, the general function
                              <kbd>
                               fnReversiblePoMo4N
                              </kbd>
                              can also be used by setting the population size to two or three. In total, four free parameters were estimated during the Bayesian inferences: π
                              <sub>
                               W
                              </sub>
                              , ρ
                              <sub>
                               W S
                              </sub>
                              , ρ
                              <sub>
                               SS
                              </sub>
                              = ρ
                              <sub>
                               W W
                              </sub>
                              and ϕ
                              <sub>
                               S
                              </sub>
                              . We set a Beta prior on π
                              <sub>
                               W
                              </sub>
                              and an exponential prior on the exchangeabilities and the GC-bias fitness ϕ
                              <sub>
                               S
                              </sub>
                              . We assumed a strict global clock rate for the molecular clock, drawn from an exponential prior with a root age fixed to 1.0. We fixed the root age to allow distance comparisons between the GTR and the virtual PoMos, as they all operate with different evolutionary units. A uniform time tree prior was set for the phylogeny. MCMC estimation was carried out with two chains with 50 000 generations for the GTR model and 75 000 for the virtual PoMos. The ESS was checked for convergence. The MCMC samples were accepted provided that the ESS of all model parameters was above 200. Similar priors were used for the GTR model, with the exception that the exchangeabilites were also set with a beta prior instead of an exponential one.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Application to fruit fly populations
                             </h3>
                             <p id="p-53">
                              Genome-wide data from 30 worldwide distributed populations of Drosophila melanogaster with 4 to 205 individuals (966 individuals) were extracted from the PopFly database (
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               Hervas et al., 2017
                              </a>
                              ). We have only used genomic data from the 2R, 2L, 3R, and 3L autosomes. In total, one million genomic sites were randomly sampled for the phylogenetic inferences. The fruit flies allele counts were converted to the GTR, PoMoTwo, and Three state spaces following the same procedure as for the simulated data. The resulting multiple sequence alignments were then used to perform molecular dating analyses in RevBayes (
                              <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">
                               Höhna et al., 2016
                              </a>
                              ). We employed a global clock model together with a uniform prior of 60 000 ± 15 000 years dating the African population expansion, following the results of
                              <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">
                               Laurent et al. (2011)
                              </a>
                              . For the remaining parameters, we set the same priors as for the Bayesian phylogenetic inferences with the simulated data.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-15">
                            <h2 class="">
                             Data Availability Statement
                            </h2>
                            <p id="p-54">
                             The simulated data, RevBayes scripts, and fruit fly alignments generated and analyzed during the current study are available at Zenodo (
                             <a href="https://doi.org/10.5281/zenodo.5202827">
                              https://doi.org/10.5281/zenodo.5202827
                             </a>
                             ).
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-55">
                             R.B., C.K., and G.S. designed research; R.B. and B.B. implemented the PoMo models in RevBayes; R.B. and C.K. analyzed data; R.B., C.K., G.S. and B.B. wrote and revised the paper.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-56">
                             The molecular dating analyses of the fruit fly data set have been achieved using the Vienna Scientific Cluster (VSC). This work was supported by the Vienna Science and Technology Fund (WWTF) [MA16-061] and partially supported by the Austrian Science Fund (FWF) [P34524-B]. GJS received funding from the European Research Council under the European Union’s Horizon 2020 research and innovation program under grant agreement no. 714774 and the grant GINOP-2.3.2.-15-2016-00057. We thank two anonymous reviewers and the associate editor Mario dos Reis for their comments and suggestions, which helped us to improve the manuscript.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-5">
                               In this new version, we have changed the titles and corrected other minor typos.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-6">
                               <a href="https://doi.org/10.5281/zenodo.5202827">
                                https://doi.org/10.5281/zenodo.5202827
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Adams
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schield
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Card
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castoe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Assessing the Impacts of Positive Selection on Coalescent-Based Species Tree Estimation and Species Delimitation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1076
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1090
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.volume%253D67%26rft.spage%253D1076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.2" data-doi="10.1371/journal.pbio.0050310">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Begun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Holloway
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stevens
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hillier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Poh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.-P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nista
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kern
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dewey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pachter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Myers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Population Genomics: Whole-Genome Analysis of Polymorphism and Divergence in Drosophila simulans
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e310
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBegun%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De310%26rft.epage%253De310%26rft.atitle%253DPopulation%2Bgenomics%253A%2Bwhole-genome%2Banalysis%2Bof%2Bpolymorphism%2Band%2Bdivergence%2Bin%2BDrosophila%2Bsimulans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050310%26rft_id%253Dinfo%253Apmid%252F17988176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050310&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17988176&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.3" data-doi="10.1534/genetics.119.302074">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Borges
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szöllősi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosiol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantifying GC-Biased Gene Conversion in Great Ape Genomes Using Polymorphism-Aware Models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  212
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1321
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1336
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.119.302074%26rft_id%253Dinfo%253Apmid%252F31147380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMi80LzEzMjEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.4" data-doi="10.1073/pnas.0900233106">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castoe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   de Koning
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. P. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.-M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noonan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Naylor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parkinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pollock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evidence for an ancient adaptive episode of convergent molecular evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8986
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8991
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0900233106%26rft_id%253Dinfo%253Apmid%252F19416880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIyLzg5ODYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.5" data-doi="10.1093/molbev/mst131">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schlötterer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosiol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Linking great apes genome evolution across time scales using polymorphism-aware phylogenetic models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2262
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst131%26rft_id%253Dinfo%253Apmid%252F23906727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst131&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23906727&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000326744700002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.6" data-doi="10.1093/sysbio/syv048">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrempf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosiol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  PoMo: An allele frequency-based approach for species tree estimation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1018
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1031
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv048%26rft_id%253Dinfo%253Apmid%252F26209413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syv048&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26209413&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.7" data-doi="10.1146/annurev-genom-082908-150001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Galtier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Biased gene conversion and the evolution of mammalian genomic landscapes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual Review of Genomics and Human Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  285
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  311
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DDuret%26rft.auinit1%253DL.%26rft.volume%253D10%26rft.spage%253D285%26rft.epage%253D311%26rft.atitle%253DBiased%2Bgene%2Bconversion%2Band%2Bthe%2Bevolution%2Bof%2Bmammalian%2Bgenomic%2Blandscapes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150001%26rft_id%253Dinfo%253Apmid%252F19630562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19630562&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000270313200014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.8" data-doi="10.1073/pnas.0904103106">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. V.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Natural selection and phylogenetic analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  106
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  8799
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8800
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0904103106%26rft_id%253Dinfo%253Apmid%252F19470454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIyLzg3OTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.9" data-doi="10.1016/j.ympev.2015.10.027">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Faircloth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCormack
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glenn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lemmon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lemmon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leaché
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Implementing and testing the multispecies coalescent model: A valuable paradigm for phylogenomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Phylogenetics and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  94
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  447
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  462
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%26rft.volume%253D94%26rft.spage%253D447%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.10.027%26rft_id%253Dinfo%253Apmid%252F26518740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.10.027&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26518740&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flouri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rannala
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A bayesian implementation of the multispecies coalescent model with introgression for phylogenomic analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Galtier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glémin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ranwez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  GC-biased gene conversion promotes the fixation of deleterious amino acid changes in primates
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Galtier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Piganeau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mouchiroud
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  GC-content evolution in mammalian genomes: the biased gene conversion hypothesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  159
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  907
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGaltier%26rft.auinit1%253DN.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D907%26rft.epage%253D911%26rft.atitle%253DGC-Content%2BEvolution%2Bin%2BMammalian%2BGenomes%253A%2BThe%2BBiased%2BGene%2BConversion%2BHypothesis%26rft_id%253Dinfo%253Apmid%252F11693127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvOTA3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMzAvMjAyMC4wNy4zMC4yMjg5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.13" data-doi="10.1534/genetics.110.116368">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glémin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Surprising Fitness Consequences of GC-Biased Gene Conversion: I. Mutation Load and Inbreeding Depression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  185
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  939
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  959
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116368%26rft_id%253Dinfo%253Apmid%252F20421602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg1LzMvOTM5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMzAvMjAyMC4wNy4zMC4yMjg5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.14" data-doi="10.1101/gr.185488.114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glémin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arndt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Messer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petrov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Galtier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantification of GC-biased gene conversion in the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1215
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1228
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185488.114%26rft_id%253Dinfo%253Apmid%252F25995268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI1LzgvMTIxNSI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA1LzMwLzIwMjAuMDcuMzAuMjI4OTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.15" data-doi="10.1093/bioinformatics/btx301">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hervas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sanz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casillas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pool
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barbadilla
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  PopFly: the Drosophila population genomics browser
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  17
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2779
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2780
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2779%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx301%26rft_id%253Dinfo%253Apmid%252F28472360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx301&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28472360&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.16" data-doi="10.1093/sysbio/syw021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Höhna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heath
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boussau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lartillot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huelsenbeck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. P.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ronquist
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  RevBayes: Bayesian Phylogenetic Inference Using Graphical Models and an Interactive Model-Specification Language
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  726
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  736
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyw021%26rft_id%253Dinfo%253Apmid%252F27235697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syw021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27235697&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.17" data-doi="10.1534/genetics.104.036939">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evidence for Abundant Slightly Deleterious Polymorphisms in Bacterial Populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  169
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  533
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  538
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036939%26rft_id%253Dinfo%253Apmid%252F15545641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvNTMzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMzAvMjAyMC4wNy4zMC4yMjg5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.18" data-doi="10.1073/pnas.2536718100">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hughes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Packer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chanock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yeager
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Widespread purifying selection at polymorphic sites in human protein-coding loci
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  100
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  26
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  15754
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15757
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2536718100%26rft_id%253Dinfo%253Apmid%252F14660790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI2LzE1NzU0IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMzAvMjAyMC4wNy4zMC4yMjg5NjUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.19" data-doi="10.1016/0304-4149(82)90011-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982a
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The coalescent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stochastic Processes and their Applications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  235
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStochastic%2BProcesses%2Band%2Btheir%2BApplications%26rft.volume%253D13%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4149%252882%252990011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0304-4149(82)90011-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.20" data-doi="10.2307/3213548">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. F. C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982b
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  On the genealogy of large populations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Applied Probability
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  A
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  27
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  43
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BProbability%26rft.volume%253D19%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3213548%26rft_id%253Dinfo%253Apmid%252F11338422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/3213548&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11338422&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.30.228965v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lachaise
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cariou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   David
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lemeunier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsacas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashburner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  Historical Biogeography of the Drosophila melanogaster Species Subgroup
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In Evolutionary Biology
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  159
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  225
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer US
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Boston, MA
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.22" data-doi="10.1093/molbev/mss239">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lartillot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Phylogenetic Patterns of GC-Biased Gene Conversion in Placental Mammals and the Evolutionary Dynamics of Recombination Landscapes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  489
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  502
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss239%26rft_id%253Dinfo%253Apmid%252F23079417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mss239&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23079417&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000314891800001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lassalle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Périan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bataillon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nesme
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Daubin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  GC-Content Evolution in Bacterial Genomes: The Biased Gene Conversion Hypothesis Expands
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1004941
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BGenetics%26rft.volume%253D11%26rft.spage%253D1004941e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.24" data-doi="10.1093/molbev/msr031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laurent
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Werzner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Excoffier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stephan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Approximate Bayesian Analysis of Drosophila melanogaster Polymorphism Data Reveals a Recent Colonization of Southeast Asia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2041
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2051
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr031%26rft_id%253Dinfo%253Apmid%252F21300986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msr031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21300986&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000291756400009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leaché
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oaks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The Utility of Single Nucleotide Polymorphism (SNP) Data in Phylogenetics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annual Review of Ecology, Evolution, and Systematics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  69
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%252C%2BEvolution%252C%2Band%2BSystematics%26rft.volume%253D48%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stephan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Inferring the demographic history and rate of adaptive substitution in Drosophila
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genetics
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.27" data-doi="10.1038/nrg.2016.104">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lynch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ackerman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gout
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Long
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thomas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. K.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Foster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genetic drift, selection and the evolution of the mutation rate
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  704
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  714
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D17%26rft.spage%253D704%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.104%26rft_id%253Dinfo%253Apmid%252F27739533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg.2016.104&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27739533&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.28" data-doi="10.1080/10635150500354928">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maddison
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. P.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knowles
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Inferring Phylogeny Despite Incomplete Lineage Sorting
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  55
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  21
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft.stitle%253DSyst%2BBiol%26rft.aulast%253DMaddison%26rft.auinit1%253DW.%2BP.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D30%26rft.atitle%253DInferring%2Bphylogeny%2Bdespite%2Bincomplete%2Blineage%2Bsorting.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150500354928%26rft_id%253Dinfo%253Apmid%252F16507521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/10635150500354928&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16507521&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000235622700002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.29" data-doi="10.1073/pnas.80.20.6278">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagylaki
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1983
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evolution of a Finite Population under Gene Conversion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences of the United States of America
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  80
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  20
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6278
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6281
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D80%26rft.issue%253D20%26rft.spage%253D6278%26rft.epage%253D6281%26rft.atitle%253DEvolution%2Bof%2Ba%2Bfinite%2Bpopulation%2Bunder%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.20.6278%26rft_id%253Dinfo%253Apmid%252F6578508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMjAvNjI3OCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA1LzMwLzIwMjAuMDcuMzAuMjI4OTY1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.30" data-doi="10.1093/molbev/msi242">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Selectionism and Neutralism in Molecular Evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2318
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2342
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi242%26rft_id%253Dinfo%253Apmid%252F16120807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msi242&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16120807&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000233361500002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.31" data-doi="10.1038/246096a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1973
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Slightly Deleterious Mutant Substitutions in Evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  246
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5428
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  96
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D246%26rft.issue%253D5428%26rft.spage%253D96%26rft.epage%253D98%26rft.atitle%253DSlightly%2Bdeleterious%2Bmutant%2Bsubstitutions%2Bin%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F246096a0%26rft_id%253Dinfo%253Apmid%252F4585855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/246096a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=4585855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1973R229300031&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.32" data-doi="10.1073/pnas.252626899">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Near-neutrality in evolution of genes and gene regulation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  99
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  25
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  16134
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  16137
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.252626899%26rft_id%253Dinfo%253Apmid%252F12461171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjUvMTYxMzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.33" data-doi="10.1093/gbe/evs052">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pessia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Popa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mousset
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rezvoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marais
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. A. B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evidence for Widespread GC-biased Gene Conversion in Eukaryotes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  675
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  682
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs052%26rft_id%253Dinfo%253Apmid%252F22628461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/gbe/evs052&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22628461&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pollard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iyer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moses
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eisen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Widespread Discordance of Gene Trees with Species Tree in Drosophila: Evidence for Incomplete Lineage Sorting
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e173
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenetics%26rft.volume%253D2%26rft.spage%253De173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.35" data-doi="10.1038/nature12228">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prado-Martinez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sudmant
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kidd
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kelley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lorente-Galdos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Veeramah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Woerner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Connor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Santpere
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cagan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Theunert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Casals
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Laayouni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Munch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hobolth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Halager
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hernandez-Rodriguez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hernando-Herraez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prüfer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pybus
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnstone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lachmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alkan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Twigg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hormozdiari
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fernandez-Callejo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dabad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stevison
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Camprubí
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carvalho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruiz-Herrera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vives
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mele
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abello
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kondova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bontrop
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pusey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lankester
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiyang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bergl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lonsdorf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Myers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ventura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gagneux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Comas
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siegismund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blanc
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Agueda-Calpena
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gut
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fulton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tishkoff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mullikin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gut
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gonder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ryder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navarro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Akey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bertranpetit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mailund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schierup
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hvilsom
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrés
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bustamante
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hammer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eichler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marques-Bonet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Great ape genetic diversity and population history
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  499
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7459
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  471
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  475
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D499%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12228%26rft_id%253Dinfo%253Apmid%252F23823723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12228&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23823723&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000322157900039&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.30.228965v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  R Core Team
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  R: A Language and Environment for Statistical Computing
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  R Foundation for Statistical Computing
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Vienna, Austria
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.37">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Scornavacca
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    C.
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Delsuc
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    F.
                                   </span>
                                  </span>
                                  , and
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Galtier
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    N.
                                   </span>
                                  </span>
                                  , editors
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rannala
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. V. S. V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leaché
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The Multi-species Coalescent Model and Species Tree Inference
                                 </span>
                                 . In
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Scornavacca
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Delsuc
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Galtier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , editors,
                                 <abbr class="cit-jnl-abbrev">
                                  Phylogenetics in the Genomic Era, pages 3.3:1–3.3:21. No commercial publisher — Authors open access book
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rannala
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Bayes Estimation of Species Divergence Times and Ancestral Population Sizes Using DNA Sequences From Multiple Loci
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  164
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1645
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1656
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRannala%26rft.auinit1%253DB.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1645%26rft.epage%253D1656%26rft.atitle%253DBayes%2BEstimation%2Bof%2BSpecies%2BDivergence%2BTimes%2Band%2BAncestral%2BPopulation%2BSizes%2BUsing%2BDNA%2BSequences%2BFrom%2BMultiple%2BLoci%26rft_id%253Dinfo%253Apmid%252F12930768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE2NDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.39" data-doi="10.1093/molbev/mst220">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stone
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Population Genomic Analysis Reveals No Evidence for GC-Biased Gene Conversion in Drosophila melanogaster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  425
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  433
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst220%26rft_id%253Dinfo%253Apmid%252F24214536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst220&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24214536&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Romiguier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Analytical biases associated with gc-content in molecular evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  16
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D8%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.41" data-doi="10.1073/pnas.0701572104">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sawyer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parsch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  16
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  6504
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6510
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0701572104%26rft_id%253Dinfo%253Apmid%252F17409186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzE2LzY1MDQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.42" data-doi="10.1016/j.jtbi.2016.07.042.">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrempf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Minh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   De Maio
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Haeseler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosiol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Reversible polymorphism-aware phylogenetic models and their application to tree inference
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Theoretical Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  407
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  362
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  370
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.volume%253D407%26rft.spage%253D362%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2016.07.042.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2016.07.042.&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.43" data-doi="10.1093/molbev/msz043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schrempf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Minh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   von Haeseler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kosiol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Polymorphism-Aware Species Trees with Advanced Mutation Models, Bootstrap, and Rate Heterogeneity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1294
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1301
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D36%26rft.spage%253D1294%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/msz043&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.44" data-doi="10.1038/sj.hdy.6800901">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stephan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The recent demographic and adaptive history of Drosophila melanogaster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Heredity
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  98
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  65
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  68
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DStephan%26rft.auinit1%253DW.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D65%26rft.epage%253D68%26rft.atitle%253DThe%2Brecent%2Bdemographic%2Band%2Badaptive%2Bhistory%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800901%26rft_id%253Dinfo%253Apmid%252F17006533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800901&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17006533&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000243730100003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.45" data-doi="10.1093/sysbio/syu048">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szöllősi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tannier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Daubin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boussau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The Inference of Gene Trees with Species Trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e42
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  e62
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyu048%26rft_id%253Dinfo%253Apmid%252F25070970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syu048&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25070970&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F30%2F2020.07.30.228965.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tavaré
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Some Probabilistic and Statistical Problems in the Analysis of DNA Sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Lectures on Mathematics in the Life Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  57
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  86
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLectures%2Bon%2BMathematics%2Bin%2Bthe%2BLife%2BSciences%26rft.volume%253D17%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.228965v3.47" data-doi="10.1534/genetics.116.190173">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Challenges in Species Tree Estimation Under the Multispecies Coalescent Model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  204
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1353
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1368
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DB.%26rft.volume%253D204%26rft.issue%253D4%26rft.spage%253D1353%26rft.epage%253D1368%26rft.atitle%253DChallenges%2Bin%2BSpecies%2BTree%2BEstimation%2BUnder%2Bthe%2BMultispecies%2BCoalescent%2BModel%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.190173%26rft_id%253Dinfo%253Apmid%252F27927902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNC80LzEzNTMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8zMC8yMDIwLjA3LjMwLjIyODk2NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.07.30.228965v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2020.01.16.908889v5" title="The hippocampal formation as a hierarchical generative model supporting generative replay and continual learning" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.11.480164v2" title="Strength of stabilizing selection on the amino-acid sequence is associated with the amount of non-additive variance in gene expression" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 30, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.07.30.228965v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.07.30.228965v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/05/30/2020.07.30.228965.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.07.30.228965v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.07.30.228965v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Nucleotide usage biases distort inferences of the species tree
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2569744"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-hNsmk1Rg4yoAgha1R6WwoYSQxdI5-3DF6UDL2pb1tSA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484475264"/>
                                   <input type="hidden" name="captcha_token" value="f0a23483903a064b224962ffb2286fbd"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2569744--3" data-node-nid="2569744" id="node-2569744--42071035730" data-pisa="biorxiv;2020.07.30.228965v3" data-pisa-master="biorxiv;2020.07.30.228965" data-seqnum="2569744" data-apath="/biorxiv/early/2022/05/30/2020.07.30.228965.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Nucleotide usage biases distort inferences of the species tree
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Rui
                                  </span>
                                  <span class="nlm-surname">
                                   Borges
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Bastien
                                  </span>
                                  <span class="nlm-surname">
                                   Boussau
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Gergely J.
                                  </span>
                                  <span class="nlm-surname">
                                   Szöllősi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Carolin
                                  </span>
                                  <span class="nlm-surname">
                                   Kosiol
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.07.30.228965;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.07.30.228965
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&title=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&title=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&text=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&t=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&title=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&title=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&title=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2569744--5" data-node-nid="2569744" id="citation-node-2569744--61852934928" data-pisa="biorxiv;2020.07.30.228965v3" data-pisa-master="biorxiv;2020.07.30.228965" data-seqnum="2569744" data-apath="/biorxiv/early/2022/05/30/2020.07.30.228965.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Nucleotide usage biases distort inferences of the species tree
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Rui
                                      </span>
                                      <span class="nlm-surname">
                                       Borges
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Bastien
                                      </span>
                                      <span class="nlm-surname">
                                       Boussau
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Gergely J.
                                      </span>
                                      <span class="nlm-surname">
                                       Szöllősi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Carolin
                                      </span>
                                      <span class="nlm-surname">
                                       Kosiol
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.07.30.228965;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.07.30.228965
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Borges%2C+Rui&rft.au=Boussau%2C+Bastien&rft.au=Sz%C3%B6ll%C5%91si%2C+Gergely+J.&rft.au=Kosiol%2C+Carolin">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2569744/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2569744/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2569744/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2569744/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2569744/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2569744/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2569744/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2569744/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2569744/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2569744/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2569744/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2569744/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&count=horizontal&via=&text=Nucleotide%20usage%20biases%20distort%20inferences%20of%20the%20species%20tree&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.228965v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.07.30.228965v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/evolutionary-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Evolutionary Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
