<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.11.378141v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.11.378141v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.11.378141v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/15/2020.11.11.378141.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.11.378141v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="A model of egocentric to allocentric understanding in mammalian brains"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.11.378141"/>
  <meta name="DC.Date" content="2022-03-15"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="In the mammalian brain, allocentric representations support efficient self-location and flexible navigation. A number of distinct populations of these spatial responses have been identified but no unified function has been shown to account for their emergence. Here we developed a network, trained with a simple predictive objective, that was capable of mapping egocentric information into an allocentric spatial reference frame. The prediction of visual inputs was sufficient to drive the appearance of spatial representations resembling those observed in rodents: head direction, boundary vector, and place cells, along with the recently discovered egocentric boundary cells, suggesting predictive coding as a principle for their emergence in animals. Strikingly, the network learned a solution for head direction tracking and stabilisation convergent with known biological connectivity. Moreover, like mammalian representations, responses were robust to environmental manipulations, including exposure to novel settings. In contrast to existing reinforcement learning approaches, agents equipped with this network were able to flexibly reuse learnt behaviours —adapting rapidly to unfamiliar environments. Thus, our results indicate that these representations, derived from a simple egocentric predictive framework, form an efficient basis-set for cognitive mapping.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Benigno Uria"/>
  <meta name="DC.Contributor" content="Borja Ibarz"/>
  <meta name="DC.Contributor" content="Andrea Banino"/>
  <meta name="DC.Contributor" content="Vinicius Zambaldi"/>
  <meta name="DC.Contributor" content="Dharshan Kumaran"/>
  <meta name="DC.Contributor" content="Demis Hassabis"/>
  <meta name="DC.Contributor" content="Caswell Barry"/>
  <meta name="DC.Contributor" content="Charles Blundell"/>
  <meta name="article:published_time" content="2022-03-15"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="A model of egocentric to allocentric understanding in mammalian brains"/>
  <meta name="citation_abstract" lang="en" content="<p>In the mammalian brain, allocentric representations support efficient self-location and flexible navigation. A number of distinct populations of these spatial responses have been identified but no unified function has been shown to account for their emergence. Here we developed a network, trained with a simple predictive objective, that was capable of mapping egocentric information into an allocentric spatial reference frame. The prediction of visual inputs was sufficient to drive the appearance of spatial representations resembling those observed in rodents: head direction, boundary vector, and place cells, along with the recently discovered egocentric boundary cells, suggesting predictive coding as a principle for their emergence in animals. Strikingly, the network learned a solution for head direction tracking and stabilisation convergent with known biological connectivity. Moreover, like mammalian representations, responses were robust to environmental manipulations, including exposure to novel settings. In contrast to existing reinforcement learning approaches, agents equipped with this network were able to flexibly reuse learnt behaviours —adapting rapidly to unfamiliar environments. Thus, our results indicate that these representations, derived from a simple egocentric predictive framework, form an efficient basis-set for cognitive mapping.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.11.378141v2"/>
  <meta name="citation_id" content="2020.11.11.378141v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.11.378141"/>
  <meta name="citation_num_pages" content="51"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.11.378141"/>
  <meta name="citation_author" content="Benigno Uria"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author_email" content="buria@google.com"/>
  <meta name="citation_author_email" content="cblundell@google.com"/>
  <meta name="citation_author" content="Borja Ibarz"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3476-9953"/>
  <meta name="citation_author" content="Andrea Banino"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author" content="Vinicius Zambaldi"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author" content="Dharshan Kumaran"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5995-5264"/>
  <meta name="citation_author" content="Demis Hassabis"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author" content="Caswell Barry"/>
  <meta name="citation_author_institution" content="University College London"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6718-0649"/>
  <meta name="citation_author" content="Charles Blundell"/>
  <meta name="citation_author_institution" content="DeepMind"/>
  <meta name="citation_author_email" content="buria@google.com"/>
  <meta name="citation_author_email" content="cblundell@google.com"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0336-0696"/>
  <meta name="citation_reference" content="Tolman, E. C. Cognitive maps in rats and men. Psychological review 55, 189 (1948)."/>
  <meta name="citation_reference" content="O’Keefe, J. & Dostrovsky, J. The hippocampus as a spatial map: Preliminary evidence from unit activity in the freely-moving rat. Brain research (1971)."/>
  <meta name="citation_reference" content="Hafting, T., Fyhn, M., Molden, S., Moser, M.-B. & Moser, E. I. Microstructure of a spatial map in the entorhinal cortex. Nature 436, 801 (2005)."/>
  <meta name="citation_reference" content="Taube, J. S., Muller, R. U. & Ranck, J. B. Head-direction cells recorded from the postsubiculum in freely moving rats. i. description and quantitative analysis. Journal of Neuroscience 10, 420–435 (1990)."/>
  <meta name="citation_reference" content="Solstad, T., Boccara, C. N., Kropff, E., Moser, M.-B. & Moser, E. I. Representation of geometric borders in the entorhinal cortex. Science 322, 1865–1868 (2008)."/>
  <meta name="citation_reference" content="Lever, C., Burton, S., Jeewajee, A., O’Keefe, J. & Burgess, N. Boundary vector cells in the subiculum of the hippocampal formation. Journal of Neuroscience 29, 9771–9777 (2009)."/>
  <meta name="citation_reference" content="Stachenfeld, K. L., Botvinick, M. M. & Gershman, S. J. The hippocampus as a predictive map. Nature neuroscience 20, 1643 (2017)."/>
  <meta name="citation_reference" content="Schiller, D. et al. Memory and space: towards an understanding of the cognitive map. Journal of Neuroscience 35, 13904–13911 (2015)."/>
  <meta name="citation_reference" content="Moser, M.-B., Rowland, D. C. & Moser, E. I. Place cells, grid cells, and memory. Cold Spring Harbor perspectives in biology 7, a021808 (2015)."/>
  <meta name="citation_reference" content="Zipser, D. A computational model of hippocampal place fields. Behavioral neuroscience 99, 1006 (1985)."/>
  <meta name="citation_reference" content="Sharp, P. E. Computer simulation of hippocampal place cells. Psychobiology 19, 103–115 (1991)."/>
  <meta name="citation_reference" content="Fuhs, M. C. & Touretzky, D. S. Synaptic learning models of map separation in the hippocampus. Neurocomputing 32, 379–384 (2000)."/>
  <meta name="citation_reference" content="Franzius, M., Sprekeler, H. & Wiskott, L. Slowness and sparseness lead to place, head-direction, and spatial-view cells. PLoS computational biology 3, e166 (2007)."/>
  <meta name="citation_reference" content="Byrne, P., Becker, S. & Burgess, N. Remembering the past and imagining the future: a neural model of spatial memory and imagery. Psychological review 114, 340 (2007)."/>
  <meta name="citation_reference" content="Wang, C. et al. Egocentric coding of external items in the lateral entorhinal cortex. Science 362, 945–949 (2018)."/>
  <meta name="citation_reference" content="Hinman, J. R., Chapman, G. W. & Hasselmo, M. E. Neuronal representation of environmental boundaries in egocentric coordinates. Nature Communications 10, 2772 (2019)."/>
  <meta name="citation_reference" content="Alexander, A. S. et al. Egocentric boundary vector tuning of the retrosplenial cortex. Science Advances 6, eaaz2322 (2020)."/>
  <meta name="citation_reference" content="Dordek, Y., Soudry, D., Meir, R. & Derdikman, D. Extracting grid cell characteristics from place cell inputs using non-negative principal component analysis. Elife 5, e10094 (2016)."/>
  <meta name="citation_reference" content="Banino, A. et al. Vector-based navigation using grid-like representations in artificial agents. Nature 557, 429 (2018)."/>
  <meta name="citation_reference" content="Brandon, M. P., Koenig, J., Leutgeb, J. K. & Leutgeb, S. New and distinct hippocampal place codes are generated in a new environment during septal inactivation. Neuron 82, 789–796 (2014)."/>
  <meta name="citation_reference" content="Tan, H. M., Wills, T. J. & Cacucci, F. The development of spatial and memory circuits in the rat. Wiley Interdisciplinary Reviews: Cognitive Science 8, e1424 (2017)."/>
  <meta name="citation_reference" content="Yamins, D. L. & DiCarlo, J. J. Using goal-driven deep learning models to understand sensory cortex. Nature neuroscience 19, 356–365 (2016)."/>
  <meta name="citation_reference" content="Zhuang, C. et al. Unsupervised neural network models of the ventral visual stream. Proceedings of the National Academy of Sciences 118 (2021)."/>
  <meta name="citation_reference" content="Whittington, J. C. et al. The tolman-eichenbaum machine: Unifying space and relational memory through generalization in the hippocampal formation. Cell 183, 1249–1263 (2020)."/>
  <meta name="citation_reference" content="George, D. et al. Clone-structured graph representations enable flexible learning and vicarious evaluation of cognitive maps. Nature communications 12, 1–17 (2021)."/>
  <meta name="citation_reference" content="Eichenbaum, H. Hippocampus: cognitive processes and neural representations that underlie declarative memory. Neuron 44, 109–120 (2004)."/>
  <meta name="citation_reference" content="Kumaran, D. & Maguire, E. A. An unexpected sequence of events: mismatch detection in the human hippocampus. PLoS biology 4, e424 (2006)."/>
  <meta name="citation_reference" content="Lisman, J. & Redish, A. D. Prediction, sequences and the hippocampus. Philosophical Transactions of the Royal Society B: Biological Sciences 364, 1193–1201 (2009)."/>
  <meta name="citation_reference" content="Rao, R. P. & Ballard, D. H. Predictive coding in the visual cortex: a functional interpretation of some extra-classical receptive-field effects. Nature neuroscience 2, 79 (1999)."/>
  <meta name="citation_reference" content="Friston, K. & Kiebel, S. Predictive coding under the free-energy principle. Philosophical Transactions of the Royal Society B: Biological Sciences 364, 1211–1221 (2009)."/>
  <meta name="citation_reference" content="Raudies, F. & Hasselmo, M. E. Modeling boundary vector cell firing given optic flow as a cue. PLoS computational biology 8, e1002553 (2012)."/>
  <meta name="citation_reference" content="Ranck, J. B. Head direction cells in the deep cell layer of dorsolateral pre-subiculum in freely moving rats. Electrical activity of the archicortex (1985)."/>
  <meta name="citation_reference" content="Bicanski, A. & Burgess, N. A neural-level model of spatial memory and imagery. ELife 7, e33752 (2018)."/>
  <meta name="citation_reference" content="Wallace, D. J. et al. Rats maintain an overhead binocular field at the expense of constant fusion. Nature 498, 65–69 (2013)."/>
  <meta name="citation_reference" content="Hartley, T., Burgess, N., Lever, C., Cacucci, F. & O’Keefe, J. Modeling place fields in terms of the cortical inputs to the hippocampus. Hippocampus 10, 369–379 (2000)."/>
  <meta name="citation_reference" content="Barry, C. et al. The boundary vector cell model of place cell firing and spatial memory. Reviews in the Neurosciences 17, 71–98 (2006)."/>
  <meta name="citation_reference" content="O’Keefe, J. & Burgess, N. Geometric determinants of the place fields of hippocampal neurons. Nature 381, 425 (1996)."/>
  <meta name="citation_reference" content="Barry, C., Hayman, R., Burgess, N. & Jeffery, K. J. Experience-dependent rescaling of entorhinal grids. Nature neuroscience 10, 682 (2007)."/>
  <meta name="citation_reference" content="Zhang, K. Representation of spatial orientation by the intrinsic dynamics of the head-direction cell ensemble: a theory. Journal of Neuroscience 16, 2112–2126 (1996)."/>
  <meta name="citation_reference" content="Seelig, J. D. & Jayaraman, V. Neural dynamics for landmark orientation and angular path integration. Nature 521, 186–191 (2015)."/>
  <meta name="citation_reference" content="Redish, A. D., Elga, A. N. & Touretzky, D. S. A coupled attractor model of the rodent head direction system. Network: Computation in Neural Systems 7, 671–685 (1996)."/>
  <meta name="citation_reference" content="Espeholt, L. et al. Impala: Scalable distributed deep-rl with importance weighted actor-learner architectures (2018). 1802.01561."/>
  <meta name="citation_reference" content="Morris, R. G. Spatial localization does not require the presence of local cues. Learning and motivation 12, 239–260 (1981)."/>
  <meta name="citation_reference" content="Bicanski, A. & Burgess, N. A neural-level model of spatial memory and imagery. ELife 7, e33752 (2018)."/>
  <meta name="citation_reference" content="Muller, R. U. & Kubie, J. L. The effects of changes in the environment on the spatial firing of hippocampal complex-spike cells. Journal of Neuroscience 7, 1951–1968 (1987)."/>
  <meta name="citation_reference" content="Gothard, K. M., Skaggs, W. E., Moore, K. M. & McNaughton, B. L. Binding of hippocampal ca1 neural activity to multiple reference frames in a landmark-based navigation task. Journal of Neuroscience 16, 823–835 (1996)."/>
  <meta name="citation_reference" content="Blum, K. I. & Abbott, L. A model of spatial map formation in the hippocampus of the rat. Neural computation 8, 85–93 (1996)."/>
  <meta name="citation_reference" content="Hassabis, D. & Maguire, E. A. The construction system of the brain. Philosophical Transactions of the Royal Society B: Biological Sciences 364, 1263–1271 (2009)."/>
  <meta name="citation_reference" content="Hayman, R. M., Chakraborty, S., Anderson, M. I. & Jeffery, K. J. Context-specific acquisition of location discrimination by hippocampal place cells. European Journal of Neuro-science 18, 2825–2834 (2003)."/>
  <meta name="citation_reference" content="Leutgeb, S. et al. Independent codes for spatial and episodic memory in hippocampal neuronal ensembles. Science 309, 619–623 (2005)."/>
  <meta name="citation_reference" content="Sharp, P. E., Blair, H. T. & Cho, J. The anatomical and computational basis of the rat head-direction cell signal. Trends in neurosciences 24, 289–294 (2001)."/>
  <meta name="citation_reference" content="Kim, S. S., Rouault, H., Druckmann, S. & Jayaraman, V. Ring attractor dynamics in the drosophila central brain. Science 356, 849–853 (2017)."/>
  <meta name="citation_reference" content="Fisher, Y. E., Lu, J., D’Alessandro, I. & Wilson, R. I. Sensorimotor experience remaps visual input to a heading-direction network. Nature 576, 121–125 (2019)."/>
  <meta name="citation_reference" content="Poulter, S., Lee, S. A., Dachtler, J., Wills, T. J. & Lever, C. Vector trace cells in the subiculum of the hippocampal formation. bioRxiv 805242 (2019)."/>
  <meta name="citation_reference" content="Kramer, M. A. Nonlinear principal component analysis using autoassociative neural networks. AIChE journal 37, 233–243 (1991)."/>
  <meta name="citation_reference" content="Beattie, C. et al. Deepmind lab (2016). 1612.03801."/>
  <meta name="citation_reference" content="Rivard, B., Li, Y., Lenck-Santini, P.-P., Poucet, B. & Muller, R. U. Representation of objects in space by two classes of hippocampal pyramidal cells. The Journal of general physiology 124, 9–25 (2004)."/>
  <meta name="citation_reference" content="Knight, R. et al. Weighted cue integration in the rodent head direction system. Philosophical Transactions of the Royal Society B: Biological Sciences 369, 20120512 (2014)."/>
  <meta name="citation_reference" content="Alexander, A. et al. Egocentric boundary vector tuning of the retrosplenial cortex. bioRxiv 702712 (2019)."/>
  <meta name="twitter:title" content="A model of egocentric to allocentric understanding in mammalian brains"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="In the mammalian brain, allocentric representations support efficient self-location and flexible navigation. A number of distinct populations of these spatial responses have been identified but no unified function has been shown to account for their emergence. Here we developed a network, trained with a simple predictive objective, that was capable of mapping egocentric information into an allocentric spatial reference frame. The prediction of visual inputs was sufficient to drive the appearance of spatial representations resembling those observed in rodents: head direction, boundary vector, and place cells, along with the recently discovered egocentric boundary cells, suggesting predictive coding as a principle for their emergence in animals. Strikingly, the network learned a solution for head direction tracking and stabilisation convergent with known biological connectivity. Moreover, like mammalian representations, responses were robust to environmental manipulations, including exposure to novel settings. In contrast to existing reinforcement learning approaches, agents equipped with this network were able to flexibly reuse learnt behaviours —adapting rapidly to unfamiliar environments. Thus, our results indicate that these representations, derived from a simple egocentric predictive framework, form an efficient basis-set for cognitive mapping.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="A model of egocentric to allocentric understanding in mammalian brains"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="In the mammalian brain, allocentric representations support efficient self-location and flexible navigation. A number of distinct populations of these spatial responses have been identified but no unified function has been shown to account for their emergence. Here we developed a network, trained with a simple predictive objective, that was capable of mapping egocentric information into an allocentric spatial reference frame. The prediction of visual inputs was sufficient to drive the appearance of spatial representations resembling those observed in rodents: head direction, boundary vector, and place cells, along with the recently discovered egocentric boundary cells, suggesting predictive coding as a principle for their emergence in animals. Strikingly, the network learned a solution for head direction tracking and stabilisation convergent with known biological connectivity. Moreover, like mammalian representations, responses were robust to environmental manipulations, including exposure to novel settings. In contrast to existing reinforcement learning approaches, agents equipped with this network were able to flexibly reuse learnt behaviours —adapting rapidly to unfamiliar environments. Thus, our results indicate that these representations, derived from a simple egocentric predictive framework, form an efficient basis-set for cognitive mapping.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-15"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2448255"/>
  <title>
   A model of egocentric to allocentric understanding in mammalian brains | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"htDcEO76QLzVuRU-oSEQtu6Eu4YxrdPGEvRhrPM1xqk"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2448255","apath":"\/biorxiv\/early\/2022\/03\/15\/2020.11.11.378141.atom","pisa":"biorxiv;2020.11.11.378141v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.11.378141v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.11.378141v2","title":"A model of egocentric to allocentric understanding in mammalian brains","identifier":"node\/2448255"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vO1vlmC4U0pdhvHBgfeWi+rSjp+xdUK3yYv\\\\\\\/DhQg6fy2Fjo3mtSkOkqGC8SVBef4WL4wL1sZqWovimE4RYoppiEtXYGhFOawEPKLsNfVQ\\\\\\\/75WnxK6MIHsa2VZH7seLHW0x3GL05PAtNIjl96rvI8i5cIwnsVZ0TSgferXou1JtEz\\\\\\\/5O2y5oga1I3Y8ykTKIoXue4TA3umqRjVDRcruVEzzkaCB2oeddpjt32tamf5aHNPzHS6Y2iHZkssbfeRbknqRnS+KG\\\\\\\/OOAh9a3FXf226pqgfZ5pVqp8oubAmjXhtBNIpNjrBGOc+pofKWgbOSBhO9WBthAKSy8gICVmMBUJZ1Gpr\\\\\\\/p\\\\\\\/rrloQVEzSjiwOZXt8peca32PON+a+zpFM41pUG0ZqatX7p5ZX\\\\\\\/6nyEza6kijEhYfAbLrnA+yXIM5f9dj1DDe3uIS4d2EN1MRqua82RzJKvuoxlQ\\\\\\\/z\\\\\\\/itRWVYV9v4p1NvqdJtihlT\\\\\\\/WUDvDK8uCWdbdg3uPx1IynX67rMbO3dpY0WE4zE4c1Rjjq1H9pSIjhsl4dhDpff8B1K2nE7AbTio0OvYb\\\\\\\/mnq+fQtKYM125UGlC1z4H8gPcXMUrYrkc9b2m4DBZNmC3G1Wu0UJR4dCrix4C703XdffyLroLZAKRyKCm2I244r6CZaNPVRWd7KzABYlL38ghQU2RNegEBKAS0z\\\\\\\/ubErUFKCJXaoVYVXJzSgZYFIogwIl2BRCIbxI4Q1vEjJ2cGr30tTeh+kDHG1bAke+VNN8KK7jhMrEMS9Qhyl\\\\\\\/wSTeCYaPXKX7uf+akxegBJFl0sB1RLKa0Mtl\\\\\\\/\\\\\\\/U0RDmMnb+I3DodUeKqQ0Gp5vHJh7pLa+64F4NJW4tbQwLk+xWarzknM8RcurujzRGhwOispRgptWM6p44SoEvRafDHR5cDikuyRyWoH09FcGw3kdVlul4zua1x6R2yTqe688964nXA0iV6KLAyA\\\\\\\/G8iFxhV9QssP2sEOF9NSE8LSuihmlCThujwllK2x4g0UPv7l4HMCm217zosBO8SkqBiHZrlPNIW+T15j7+a+qNDjybR2MnKEU18A2ZjkudZON2aQUgmZ96mIo9WPS9hsGyOmpnFFQRY4zwiXqg54sNaa5cbQcREzFo7t2FBpBEJ8vvQUITJNTqZg9IsqlUMOLziQidtS26GidWBc+K1WWLRTg\\\\\\\/SLNvjGq8izu2AB14ayj6Bk6cmeHnu4zrAJuwOg5jGZmZc+6gLYCn5r7oVzXpaBVq5lFHnttYnL0PnAk\\\\\\\/PhPD4fC5tQt8dIpLnoLJaqtOa7b8JYqkzCTE8rUXBL4AlowdsgCN6GNSXRsqnWQY4CM4NqWygGMe4FVGbsrV9c0pVgkyF7cjhLWwbAw7dC+\\\\\\\/GcnhcG\\\\\\\/5D90fjYUUXJc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022aTaWzcaioSUnOw5VTFWGGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022acb3e2038bf0572f64731a6939cb684e74439d5b85d8c8f6c75be97bba70b599\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.11.378141v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2448255 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.11.378141v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_452916847">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_452916847" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_337947989" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-mJHWZ44omQv2CEpV-jzuDTG-yFDi6kruZUCWdMEpxjk"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2448255" data-node-nid="2448255" id="node-2448255--21807966550" data-pisa="biorxiv;2020.11.11.378141v2" data-pisa-master="biorxiv;2020.11.11.378141" data-apath="/biorxiv/early/2022/03/15/2020.11.11.378141.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    A model of egocentric to allocentric understanding in mammalian brains
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Benigno
                      </span>
                      <span class="nlm-surname">
                       Uria
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-3476-9953" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Borja
                      </span>
                      <span class="nlm-surname">
                       Ibarz
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Andrea
                      </span>
                      <span class="nlm-surname">
                       Banino
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Vinicius
                      </span>
                      <span class="nlm-surname">
                       Zambaldi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0001-5995-5264" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Dharshan
                      </span>
                      <span class="nlm-surname">
                       Kumaran
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Demis
                      </span>
                      <span class="nlm-surname">
                       Hassabis
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0001-6718-0649" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Caswell
                      </span>
                      <span class="nlm-surname">
                       Barry
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-0336-0696" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Charles
                      </span>
                      <span class="nlm-surname">
                       Blundell
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.11.378141
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2448255--21807966550" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Benigno Uria
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Benigno%2BUria%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Uria%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABenigno%2BUria%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:buria@google.com" class="" data-icon-position="" data-hide-link-title="0">
                        buria@google.com
                       </a>
                       <a href="mailto:cblundell@google.com" class="" data-icon-position="" data-hide-link-title="0">
                        cblundell@google.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Borja Ibarz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Borja%2BIbarz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ibarz%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABorja%2BIbarz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3476-9953" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Borja Ibarz
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Andrea Banino
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrea%2BBanino%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Banino%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrea%2BBanino%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Vinicius Zambaldi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Vinicius%2BZambaldi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zambaldi%20V&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AVinicius%2BZambaldi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Dharshan Kumaran
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Dharshan%2BKumaran%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kumaran%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADharshan%2BKumaran%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5995-5264" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Dharshan Kumaran
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Demis Hassabis
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Demis%2BHassabis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hassabis%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADemis%2BHassabis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Caswell Barry
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        University College London
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Caswell%2BBarry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Barry%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACaswell%2BBarry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6718-0649" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Caswell Barry
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Charles Blundell
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        DeepMind
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Charles%2BBlundell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Blundell%20C&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ACharles%2BBlundell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-0336-0696" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Charles Blundell
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:buria@google.com" class="" data-icon-position="" data-hide-link-title="0">
                        buria@google.com
                       </a>
                       <a href="mailto:cblundell@google.com" class="" data-icon-position="" data-hide-link-title="0">
                        cblundell@google.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.11.378141v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2448255" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2448255/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.11.378141v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2448255" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2448255/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.11.378141v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2448255" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2448255/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.11.378141v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2448255" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2448255/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.11.378141v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2448255" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2448255/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2>
                             Abstract
                            </h2>
                            <p id="p-3">
                             In the mammalian brain, allocentric representations support efficient self-location and flexible navigation. A number of distinct populations of these spatial responses have been identified but no unified function has been shown to account for their emergence. Here we developed a network, trained with a simple predictive objective, that was capable of mapping egocentric information into an allocentric spatial reference frame. The prediction of visual inputs was sufficient to drive the appearance of spatial representations resembling those observed in rodents: head direction, boundary vector, and place cells, along with the recently discovered egocentric boundary cells, suggesting predictive coding as a principle for their emergence in animals. Strikingly, the network learned a solution for head direction tracking and stabilisation convergent with known biological connectivity. Moreover, like mammalian representations, responses were robust to environmental manipulations, including exposure to novel settings. In contrast to existing reinforcement learning approaches, agents equipped with this network were able to flexibly reuse learnt behaviours —adapting rapidly to unfamiliar environments. Thus, our results indicate that these representations, derived from a simple egocentric predictive framework, form an efficient basis-set for cognitive mapping.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Animals navigate easily and efficiently through the world, learning quickly about new places they encounter
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             . Yet artificial agents struggle with even simple spatial tasks requiring self-localisation and goal directed behaviours. In mammals, the neural circuits supporting spatial memory have been studied extensively. Empirical work has generated a detailed knowledge about populations of spatially modulated neurons – place
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             , grid
                             <sup>
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             , and head direction
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             cells represent body position and direction of facing, while border responsive neurons encode immediate environmental topography
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ,
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                             </sup>
                             . Current thinking sees these networks as components of a cognitive map thought to provide an efficient spatial basis, allowing the structure of novel environments to be learnt rapidly and subsequently supporting flexible navigation, such as short-cuts, detours, and novel routes to remembered goals
                             <sup>
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ,
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-7">
                             Surprisingly, there are significant gaps in our understanding. In particular, it is unclear how these populations interact to support spatial behaviour and how they are themselves derived and updated by incoming sensory information. Fundamentally, we lack a unifying computational account for the emergence of allocentric (world centred) representations from egocentric (self centred) sensory experience
                             <sup>
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             . This poses a problem for neuroscientists, who seek simplifying normative accounts of the brain, and for AI practitioners who aim to build systems with navigational abilities that match those of animals.
                            </p>
                            <p id="p-8">
                             Historically, theoretical approaches to this problem largely presented hand-coded designs focused on a single layer of the biological network
                             <sup>
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                              ,
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             . A smaller number of models have presented a more extensive description of hippocampal networks
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                             </sup>
                             , including the process by which representations might be learnt
                             <sup>
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             . Notably a potential architecture for ego–allocentric transformation
                             <sup>
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             successfully anticipated the existence of egocentric border responsive neurons
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              –
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                             </sup>
                             . Still, in almost all cases existing models are the product of extensive human oversight.
                            </p>
                            <p id="p-9">
                             Recently, deep neural networks have provided a normative explanation for the relationship between two allocentric representations – grid patterns emerge when predicting idealised place cell inputs
                             <sup>
                              <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ,
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               18
                              </a>
                              –
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                             </sup>
                             – adding to experimental
                             <sup>
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                             </sup>
                             and developmental
                             <sup>
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                             </sup>
                             evidence that points to place-cells as an input to downstream grid-cells. Similar approaches have seen considerable success as models of the initial feed-forward processing conducted by the mammalian visual system
                             <sup>
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ,
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                             </sup>
                             . However, although impressive, these models have tended to be limited in scope – often being focused on a single biological circuit, taking either idealised neural activity as an input or being constrained to representations bound to a single reference frame.
                            </p>
                            <p id="p-10">
                             Here we show that a deep learning model trained to predict visual experience from self-motion is sufficient to explain the hierarchy of egocentric to allocentric representations found in mammals, bridging the gap from visual experience to place cells. In particular, we identify head-direction cells, border cells with both ego and allocentric responses, and place cells. Critically, unlike other recent models of spatial representations
                             <sup>
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ,
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                             </sup>
                             , we do so without the need for any allocentric inputs, using only information that is plausibly available to the brain. Thus, our network links this diverse array of cells to a simple predictive framework, it poses hypotheses about the circuit dynamics that produce these responses, and embodies a model linking sensory information, neural activity, and spatial behaviour. Like their biological counterparts, the firing fields of these units are shaped by the surrounding visual cues and boundaries, but retain their characteristics across different environments. When embodied in artificial agents, this redeployment of existing responses supports rapid adaptation to novel environments, enabling effective navigation to remembered goals with limited experience of the test enclosure – demonstrating the flexible transfer of skills at which animals excel but artificial systems have struggled.
                            </p>
                            <div id="sec-2" class="subsection">
                             <h3>
                              Spatial Memory Pipeline
                             </h3>
                             <p id="p-11">
                              The hippocampal formation has been characterised as a predictor-comparator, comparing incoming perceptual stimuli with predictions derived from memory
                              <sup>
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                27
                               </a>
                               –
                               <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                                29
                               </a>
                              </sup>
                              . With this in mind we trained a deep neural network to predict the forthcoming visual experience of an agent exploring virtual environments. Specifically, we developed a model composed of a modular recurrent neural network (RNN) with an external memory – the ‘Spatial Memory Pipeline’ (see supplementary text,
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Supplementary Fig 1
                              </a>
                              ). At each time-step visual inputs entering the pipeline were compressed using a convolutional neural network and compared to previous embeddings stored in the slots of a memory store, the best matching being most strongly reactivated. The remainder of the pipeline was trained to predict which visual memory slot would be reactivated next (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig 1A
                              </a>
                              ). Predictions were generated solely on the basis of self-motion information, provided differentially to RNN modules as angular and linear velocity, in addition to visual corrections from previous steps. There was no direct pathway from detailed visual features to latter parts of the pipeline – visual information being communicated only by the activation of slots in the memory stores (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig 1B
                              </a>
                              ). Thus the Spatial Memory Pipeline forms a predictive code
                              <sup>
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ,
                               <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                              </sup>
                              , anticipating which future visual slot will be activated, based solely on self-motion.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F5.large.jpg?width=800&height=600&carousel=1" title="Diagram of the Spatial Memory Pipeline. A) Computational diagram showing the encoder and decoder networks of the convolutional autencoder used during its training. B) Computational diagram showing how the loss is calculated at every time step for an example model with three predictive RNNs. Preact is the target distribution and Ppred the predicted distribution over memories. C) Computational diagram of the model dynamics for a time step with visual correction (10% of steps). A correction code  is calculated as a visually-dependent weighted average of visual memory embeddings . D) Computational diagram of the model dynamics for a time step without visual correction (90% of steps)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Diagram of the <em>Spatial Memory Pipeline</em>. <strong>A)</strong> Computational diagram showing the encoder and decoder networks of the convolutional autencoder used during its training. <strong>B)</strong> Computational diagram showing how the loss is calculated at every time step for an example model with three predictive RNNs. <em>P</em><sub><em>react</em></sub> is the target distribution and <em>P</em><sub><em>pred</em></sub> the predicted distribution over memories. <strong>C)</strong> Computational diagram of the model dynamics for a time step with visual correction (10% of steps). A correction code  is calculated as a visually-dependent weighted average of visual memory embeddings . <strong>D)</strong> Computational diagram of the model dynamics for a time step without visual correction (90% of steps).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F5.medium.gif" width="369" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F5.medium.gif" width="369" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Supplementary Figure 1:
                               </span>
                               <p id="p-150" class="first-child">
                                Diagram of the Spatial Memory Pipeline.
                                <strong>
                                 A)
                                </strong>
                                Computational diagram showing the encoder and decoder networks of the convolutional autencoder used during its training.
                                <strong>
                                 B)
                                </strong>
                                Computational diagram showing how the loss is calculated at every time step for an example model with three predictive RNNs. P
                                <sub>
                                 react
                                </sub>
                                is the target distribution and P
                                <sub>
                                 pred
                                </sub>
                                the predicted distribution over memories.
                                <strong>
                                 C)
                                </strong>
                                Computational diagram of the model dynamics for a time step with visual correction (10% of steps). A correction code
                                <span class="inline-formula" id="inline-formula-22">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/F5/embed/inline-graphic-22.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/F5/embed/inline-graphic-22.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                is calculated as a visually-dependent weighted average of visual memory embeddings
                                <span class="inline-formula" id="inline-formula-23">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/F5/embed/inline-graphic-23.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/F5/embed/inline-graphic-23.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                                <strong>
                                 D)
                                </strong>
                                Computational diagram of the model dynamics for a time step without visual correction (90% of steps).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F1.large.jpg?width=800&height=600&carousel=1" title="A, B) Computational diagrams for the Spatial Memory Pipeline. A) In the prediction step, a set of RNNs anticipate the reactivation of past visual memories by integrating egocentric velocities – AE denotes autoencoder. B) In the correction step, the reactivated visual memories correct the RNNs’ state to reduce accumulated errors. C) External view of white square enclosure with distal cues. D) Agent’s view from the middle of the enclosure (left) and close to a wall (right). E) Prediction loss along training. The model reduces the uncertainty in visual reactivations by 80% with respect to a uniform distribution. F) Average decoding error of position and head direction from visual slot activations and spatial memory pipeline RNNs, black bars show one standard error of the mean." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A, B)</strong> Computational diagrams for the Spatial Memory Pipeline. A) In the prediction step, a set of RNNs anticipate the reactivation of past visual memories by integrating egocentric velocities – AE denotes autoencoder. B) In the correction step, the reactivated visual memories correct the RNNs’ state to reduce accumulated errors. <strong>C)</strong> External view of white square enclosure with distal cues. <strong>D)</strong> Agent’s view from the middle of the enclosure (left) and close to a wall (right). <strong>E)</strong> Prediction loss along training. The model reduces the uncertainty in visual reactivations by 80% with respect to a uniform distribution. <strong>F)</strong> Average decoding error of position and head direction from visual slot activations and spatial memory pipeline RNNs, black bars show one standard error of the mean.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F1.medium.gif" width="440" height="259"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F1.medium.gif" width="440" height="259"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <p id="p-12" class="first-child">
                                <strong>
                                 A, B)
                                </strong>
                                Computational diagrams for the Spatial Memory Pipeline. A) In the prediction step, a set of RNNs anticipate the reactivation of past visual memories by integrating egocentric velocities – AE denotes autoencoder. B) In the correction step, the reactivated visual memories correct the RNNs’ state to reduce accumulated errors.
                                <strong>
                                 C)
                                </strong>
                                External view of white square enclosure with distal cues.
                                <strong>
                                 D)
                                </strong>
                                Agent’s view from the middle of the enclosure (left) and close to a wall (right).
                                <strong>
                                 E)
                                </strong>
                                Prediction loss along training. The model reduces the uncertainty in visual reactivations by 80% with respect to a uniform distribution.
                                <strong>
                                 F)
                                </strong>
                                Average decoding error of position and head direction from visual slot activations and spatial memory pipeline RNNs, black bars show one standard error of the mean.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Emergence of allocentric representations
                             </h3>
                             <p id="p-13">
                              In our first unsupervised-learning experiment, the Spatial Memory Pipeline was trained in a simulated square environment (2.2 by 2.2 meters) resembling those used in rodent studies – plain white walls with visible distal cues (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig 1C, D
                              </a>
                              ), using a rat-like motion model
                              <sup>
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                              </sup>
                              . After training, the network was able to accurately predict the reactivation of visual memories (80% loss reduction with respect to uniform distribution,
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig 1E
                              </a>
                              ), effectively integrating self-motion information to anticipate the visual scene. To understand how the network performed this task we inspected the activity profile of units in the modular RNNs. These units exhibited a range of spatially stable allocentric responses strongly resembling those found in the mammalian hippocampal spatial memory system, including head-direction (HD) cells, boundary vector cells (BVCs), and place cells, as well as egocentric boundary vector cells (egoBVCs) (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig 2
                              </a>
                              , see
                              <a id="xref-sec-9-1" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ). Indeed, in the trained network, the agent’s position and direction of facing could be accurately decoded from the RNN activations (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig 1F
                              </a>
                              , see
                              <a id="xref-sec-39-1" class="xref-sec" href="#sec-39">
                               Supplementary Methods
                              </a>
                              ). These representations were complementary to one another – each RNN module developed distinct response patterns, dependent on the form of self-motion input it received (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig 2A–D
                              </a>
                              ). Similar results were observed in environments with different geometries (see supplementary text,
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Supplementary Fig 2
                              </a>
                              ). The results persisted for different random initializations of network parameters (see
                              <a id="xref-sec-30-1" class="xref-sec" href="#sec-30">
                               Supplementary Results
                              </a>
                              ). In contrast, a single RNN receiving all the velocity inputs did not develop the whole range of representations (see
                              <a id="xref-sec-30-2" class="xref-sec" href="#sec-30">
                               Supplementary Results
                              </a>
                              ).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F6.large.jpg?width=800&height=600&carousel=1" title="Spa-t12i0al representat12i0ons in the Spatial Memory Pipeline trained in a circular open field environment w-1i5t0h wBachk it1e50 walls. A) Polar pSlot of activity by heading direction of five cells classified as HD cells. B) Spatial ratemaps of five cells shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent is facing in the north direction). C) Spatial ratemaps of five cells classified as egocentric-boundary cells (egoBVCs). D) Spatial ratemaps of five cells classified as boundary-vector cells (BVCs). E) Spatial ratemaps of five memory slots reactivated by RNN-3. F) Spatial stability of cells in each RNN (see Supplementary Methods). G) Resultant vector length of cells in each RNN. H) Resultant vector of each cell in RNN-1. I) Comparison of cells responses to egocentric versus allocentric boundaries in each RNN." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Spa-t12i0al representat12i0ons in the <em>Spatial Memory Pipeline</em> trained in a circular open field environment w-1i5t0h wBachk it1e50 walls. <strong>A)</strong> Polar pSlot of activity by heading direction of five cells classified as HD cells. <strong>B)</strong> Spatial ratemaps of five cells shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent is facing in the north direction). <strong>C)</strong> Spatial ratemaps of five cells classified as egocentric-boundary cells (egoBVCs). <strong>D)</strong> Spatial ratemaps of five cells classified as boundary-vector cells (BVCs). <strong>E)</strong> Spatial ratemaps of five memory slots reactivated by RNN-3. <strong>F)</strong> Spatial stability of cells in each RNN (see Supplementary Methods). <strong>G)</strong> Resultant vector length of cells in each RNN. <strong>H)</strong> Resultant vector of each cell in RNN-1. <strong>I)</strong> Comparison of cells responses to egocentric versus allocentric boundaries in each RNN.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F6.medium.gif" width="209" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F6.medium.gif" width="209" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 2:
                               </span>
                               <p id="p-151" class="first-child">
                                Spa-t12i0al representat12i0ons in the Spatial Memory Pipeline trained in a circular open field environment w-1i5t0h wBachk it1e50 walls.
                                <strong>
                                 A)
                                </strong>
                                Polar pSlot of activity by heading direction of five cells classified as HD cells.
                                <strong>
                                 B)
                                </strong>
                                Spatial ratemaps of five cells shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent is facing in the north direction).
                                <strong>
                                 C)
                                </strong>
                                Spatial ratemaps of five cells classified as egocentric-boundary cells (egoBVCs).
                                <strong>
                                 D)
                                </strong>
                                Spatial ratemaps of five cells classified as boundary-vector cells (BVCs).
                                <strong>
                                 E)
                                </strong>
                                Spatial ratemaps of five memory slots reactivated by RNN-3.
                                <strong>
                                 F)
                                </strong>
                                Spatial stability of cells in each RNN (see
                                <a id="xref-sec-39-7" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ).
                                <strong>
                                 G)
                                </strong>
                                Resultant vector length of cells in each RNN.
                                <strong>
                                 H)
                                </strong>
                                Resultant vector of each cell in RNN-1.
                                <strong>
                                 I)
                                </strong>
                                Comparison of cells responses to egocentric versus allocentric boundaries in each RNN.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F2.large.jpg?width=800&height=600&carousel=1" title="Representations in a spatial memory pipeline trained in the square enclosure shown in Fig 1C. A) Polar plot of average activity by heading direction of five units classified as HD cells. B) Spatial ratemaps of the units shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent was facing in the north direction). C) Spatial ratemaps of five units classified as egoBVCs. D) Spatial ratemaps of five units classified as BVCs. E) Spatial ratemaps of five memory slots reactivated by RNN-3 resembling hippocampal place-cell activity. F) Spatial stability of units in each RNN. G) Resultant vector length of units in each RNN. H) Resultant vector of each unit in RNN-1. I) Comparison of egocentric versus allocentric scores for units in each RNN. Dashed lines indicate cell-type classification thresholds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Representations in a spatial memory pipeline trained in the square enclosure shown in Fig 1C. <strong>A)</strong> Polar plot of average activity by heading direction of five units classified as HD cells. <strong>B)</strong> Spatial ratemaps of the units shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent was facing in the north direction). <strong>C)</strong> Spatial ratemaps of five units classified as egoBVCs. <strong>D)</strong> Spatial ratemaps of five units classified as BVCs. <strong>E)</strong> Spatial ratemaps of five memory slots reactivated by RNN-3 resembling hippocampal place-cell activity. <strong>F)</strong> Spatial stability of units in each RNN. <strong>G)</strong> Resultant vector length of units in each RNN. <strong>H)</strong> Resultant vector of each unit in RNN-1. <strong>I)</strong> Comparison of egocentric versus allocentric scores for units in each RNN. Dashed lines indicate cell-type classification thresholds.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F2.medium.gif" width="208" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F2.medium.gif" width="208" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <p id="p-14" class="first-child">
                                Representations in a spatial memory pipeline trained in the square enclosure shown in
                                <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                                 Fig 1C
                                </a>
                                .
                                <strong>
                                 A)
                                </strong>
                                Polar plot of average activity by heading direction of five units classified as HD cells.
                                <strong>
                                 B)
                                </strong>
                                Spatial ratemaps of the units shown in A. Central plot shows average activation for each location across all head directions. The eight plots around each central plot show location-specific activity restricted to times when the heading direction of the agent was in the the corresponding 45° range (e.g. plot located above the central plot shows average activity when the agent was facing in the north direction).
                                <strong>
                                 C)
                                </strong>
                                Spatial ratemaps of five units classified as egoBVCs.
                                <strong>
                                 D)
                                </strong>
                                Spatial ratemaps of five units classified as BVCs.
                                <strong>
                                 E)
                                </strong>
                                Spatial ratemaps of five memory slots reactivated by RNN-3 resembling hippocampal place-cell activity.
                                <strong>
                                 F)
                                </strong>
                                Spatial stability of units in each RNN.
                                <strong>
                                 G)
                                </strong>
                                Resultant vector length of units in each RNN.
                                <strong>
                                 H)
                                </strong>
                                Resultant vector of each unit in RNN-1.
                                <strong>
                                 I)
                                </strong>
                                Comparison of egocentric versus allocentric scores for units in each RNN. Dashed lines indicate cell-type classification thresholds.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-15">
                              The majority of units in the first RNN module (RNN-1), which received only angular velocities and corrections from the visual memories, exhibited activity that was modulated by the agent’s direction of facing (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig 2A
                              </a>
                              ). These responses were strikingly similar to those of head-direction cells
                              <sup>
                               <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                               ,
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                33
                               </a>
                              </sup>
                              – individual units having a single preferred firing direction invariant across spatial locations (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig 2B
                              </a>
                              ). In total 91% (29/32) of units were classified as HD cells (resultant vector length >0.58, 99th percentile of shuffled data, see
                              <a id="xref-sec-9-2" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ) with unimodal responses (average tuning width 76.2°) distributed uniformly in the unit circle (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig 2H
                              </a>
                              , uniform distribution was selected under Bayes Information Criteria (BIC) over mixtures of Von Mises, see
                              <a id="xref-sec-39-2" class="xref-sec" href="#sec-39">
                               Supplementary Methods
                              </a>
                              ).
                             </p>
                             <p id="p-16">
                              The second module (RNN-2) received angular velocity and speed. Here units exhibited more complex patterns of activity (mean resultant vector = 0.02, 0/128 units classified as HD cells), encoding distances to boundaries at a particular heading (
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig 2C
                              </a>
                              ). Thus, they appeared to be similar to egocentric BVCs
                              <sup>
                               <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                               –
                               <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              which are theorised to play a central role in transforming between ego and allocentric reference frames
                              <sup>
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                               ,
                               <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                34
                               </a>
                              </sup>
                              . To quantify this impression we adopted an egoBVC metric applied previously
                              <sup>
                               <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              (see
                              <a id="xref-sec-9-3" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ). In this way 52 of 128 units (41%) were identified as egoBVCs (ego-boundary score >0.07, 99th percentile of bin-shuffled distribution,
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig 2I
                              </a>
                              ), with the remaining cells displaying mixed patterns of head-direction and egocentric distance to a wall. Rodent egoBVCs exhibit a uniform distribution of preferred firing directions - albeit with a slight tendency to cluster to the animal’s left and right side - while preferred distances have been reported up to 50cm, with shorter range responses being more numerous
                              <sup>
                               <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              . Analysis of the model egoBVCs (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Supplementary Fig 3A–C
                              </a>
                              ) revealed a similar pattern of distances, but a cluster of cells responded to a wall directly in front - plausibly reflecting the monocular input and narrower field of view available to the model (60° vs >180° in rodents)
                              <sup>
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                              </sup>
                              .
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F7.large.jpg?width=800&height=600&carousel=1" title="Detail of boundary cells in Fig 2. A) First row, ratemap of five cells classified as egoBVCs. Second row, egocentric-boundary ratemap (see Supplementary Methods). Third row, allocentric-boundary ratemap (see Supplementary Methods). B) Histogram of preferred egocentric direction to boundary of all units classified as egoBVCs. Inset, Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A mixture of two Von Mises distributions (shown in green), with a mode for boundaries in front agent, was selected (lowest BIC) (see Supplementary Methods). C) Histogram of preferred distance to boundary of all units classified as egoBVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 35 cm. D) First row, ratemap of five cells classified as (allocentric) BVCs. Second row, egocentric-boundary ratemap. Third row, allocentric-boundary ratemap. E) Histogram of preferred allocentric direction to boundary of all units classified as BVCs. Inset shows Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A uniform distribution (shown in green) was selected (lowest BIC). F) Histogram of preferred distance to boundary of all units classified as BVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 29 cm." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Detail of boundary cells in Fig 2. <strong>A)</strong> First row, ratemap of five cells classified as egoBVCs. Second row, egocentric-boundary ratemap (see Supplementary Methods). Third row, allocentric-boundary ratemap (see Supplementary Methods). <strong>B)</strong> Histogram of preferred egocentric direction to boundary of all units classified as egoBVCs. Inset, Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A mixture of two Von Mises distributions (shown in green), with a mode for boundaries in front agent, was selected (lowest BIC) (see Supplementary Methods). <strong>C)</strong> Histogram of preferred distance to boundary of all units classified as egoBVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 35 cm. <strong>D)</strong> First row, ratemap of five cells classified as (allocentric) BVCs. Second row, egocentric-boundary ratemap. Third row, allocentric-boundary ratemap. <strong>E)</strong> Histogram of preferred allocentric direction to boundary of all units classified as BVCs. Inset shows Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A uniform distribution (shown in green) was selected (lowest BIC). <strong>F)</strong> Histogram of preferred distance to boundary of all units classified as BVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 29 cm.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F7.medium.gif" width="440" height="242"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F7.medium.gif" width="440" height="242"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 3:
                               </span>
                               <p id="p-152" class="first-child">
                                Detail of boundary cells in
                                <a id="xref-fig-2-16" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                .
                                <strong>
                                 A)
                                </strong>
                                First row, ratemap of five cells classified as egoBVCs. Second row, egocentric-boundary ratemap (see
                                <a id="xref-sec-39-8" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ). Third row, allocentric-boundary ratemap (see
                                <a id="xref-sec-39-9" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ).
                                <strong>
                                 B)
                                </strong>
                                Histogram of preferred egocentric direction to boundary of all units classified as egoBVCs. Inset, Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A mixture of two Von Mises distributions (shown in green), with a mode for boundaries in front agent, was selected (lowest BIC) (see
                                <a id="xref-sec-39-10" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ).
                                <strong>
                                 C)
                                </strong>
                                Histogram of preferred distance to boundary of all units classified as egoBVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 35 cm.
                                <strong>
                                 D)
                                </strong>
                                First row, ratemap of five cells classified as (allocentric) BVCs. Second row, egocentric-boundary ratemap. Third row, allocentric-boundary ratemap.
                                <strong>
                                 E)
                                </strong>
                                Histogram of preferred allocentric direction to boundary of all units classified as BVCs. Inset shows Bayes information criterion numbers for a uniform distribution over angles (U) and mixtures of Von Mises distributions with different numbers of components. A uniform distribution (shown in green) was selected (lowest BIC).
                                <strong>
                                 F)
                                </strong>
                                Histogram of preferred distance to boundary of all units classified as BVCs. The distribution over distances was well fit by an exponential distribution (shown in green) with mean distance of 29 cm.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-17">
                              In contrast to the first two RNNs, the third (RNN-3) received no self-motion inputs, thus being dependent upon temporal coherence
                              <sup>
                               <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                              </sup>
                              , and corrections from mispredictions as its sole input and learning signal. Units in this layer were not modulated by heading direction (mean resultant vector = 0.11, 0/128 units classified as HD cells), but were characterised by spatially stable responses (
                              <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                               Fig 2F
                              </a>
                              ) often extending parallel to particular walls of the environment (
                              <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                               Fig 2D
                              </a>
                              ). As such, the activity profile of this module was reminiscent of border cells or boundary vector cells (BVCs), which form an allocentric representation of space defined by environmental boundaries
                              <sup>
                               <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                               ,
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                               ,
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . To assess the units’ activity we applied a similar approach to that used for RNN-2 – calculating the resultant vector of the units’ activity projected into allocentric boundary space (see
                              <a id="xref-sec-9-4" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ). Applying this measure to RNN-3 confirmed that 84% (108/128) of units were classified as BVCs (BVC score >0.11, 99th percentile of shuffled distribution) – 37% (48/128) met the criteria for egoBVCs, but 37 of those 48 had a higher BVC score than egoBVC score (
                              <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                               Fig 2I
                              </a>
                              ). The same approach applied to RNN-2 identified 0 of 128 units as BVCs. Analysis of the preferred firing directions and distances of the model BVCs revealed a uniform distribution of directions (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Supplementary Fig 3E
                              </a>
                              ).
                             </p>
                             <p id="p-18">
                              In the brain, BVCs are hypothesised to be a core contributor to the spatial activity of place cells
                              <sup>
                               <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                              </sup>
                              . Consistent with this, BVC-like responses are found in mEC, afferent to hippocampus, as well as within the hippocampus proper
                              <sup>
                               <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                               ,
                               <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              . In our model, the allocentric representations of RNN-3 were stored in the external memory slots as a second set of targets - being reactivated at each time step by comparison to the current state of the RNN-3. The activity of these memory slots strongly resembled place cells with sparse, spatially localised responses (average spatial field size of 0.21 m
                              <sup>
                               2
                              </sup>
                              SD:0.26 to a total environment area of 4.84 m
                              <sup>
                               2
                              </sup>
                              ) that were stable across trajectories, and independent of head direction (see
                              <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                               Fig 2E
                              </a>
                              and
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Supplementary Fig 4
                              </a>
                              ).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F8.large.jpg?width=800&height=600&carousel=1" title="Reactivation of memory slots (containing past RNN-3 states) resembles place-cell responses. A) Spatial ratemap of the reactivation of eight memory slots in the experiment described in Fig 2. Activity resembles that of biological place cells in the hippocampus. B) Spatial stability of memory slot reactivations (see Supplementary Methods). C) Resultant vector of memory slot reactivations. D) Cumulative total activity of place cells (memory slot reactivations) ordered by their total activity (blue). Place cells showed a high sparsity: 25% of cells account for 75% of activity. Activity in the three RNNs shows almost no sparsity (cells activated equally on average). E) Number of fields per place cell (see Supplementary Methods). F) Area of each activity field in square meters (total environment area is 4.8 m2). G) Eccentricity of fields. A score of 1.0 indicates a perfectly round field; lower scores indicate elongation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Reactivation of memory slots (containing past RNN-3 states) resembles place-cell responses. <strong>A)</strong> Spatial ratemap of the reactivation of eight memory slots in the experiment described in Fig 2. Activity resembles that of biological place cells in the hippocampus. <strong>B)</strong> Spatial stability of memory slot reactivations (see Supplementary Methods). <strong>C)</strong> Resultant vector of memory slot reactivations. <strong>D)</strong> Cumulative total activity of place cells (memory slot reactivations) ordered by their total activity (blue). Place cells showed a high sparsity: 25% of cells account for 75% of activity. Activity in the three RNNs shows almost no sparsity (cells activated equally on average). <strong>E)</strong> Number of fields per place cell (see Supplementary Methods). <strong>F)</strong> Area of each activity field in square meters (total environment area is 4.8 <em>m</em><sup>2</sup>). <strong>G)</strong> Eccentricity of fields. A score of 1.0 indicates a perfectly round field; lower scores indicate elongation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F8.medium.gif" width="440" height="351"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F8.medium.gif" width="440" height="351"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 4:
                               </span>
                               <p id="p-153" class="first-child">
                                Reactivation of memory slots (containing past RNN-3 states) resembles place-cell responses.
                                <strong>
                                 A)
                                </strong>
                                Spatial ratemap of the reactivation of eight memory slots in the experiment described in
                                <a id="xref-fig-2-17" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                . Activity resembles that of biological place cells in the hippocampus.
                                <strong>
                                 B)
                                </strong>
                                Spatial stability of memory slot reactivations (see
                                <a id="xref-sec-39-11" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ).
                                <strong>
                                 C)
                                </strong>
                                Resultant vector of memory slot reactivations.
                                <strong>
                                 D)
                                </strong>
                                Cumulative total activity of place cells (memory slot reactivations) ordered by their total activity (blue). Place cells showed a high sparsity: 25% of cells account for 75% of activity. Activity in the three RNNs shows almost no sparsity (cells activated equally on average).
                                <strong>
                                 E)
                                </strong>
                                Number of fields per place cell (see
                                <a id="xref-sec-39-12" class="xref-sec" href="#sec-39">
                                 Supplementary Methods
                                </a>
                                ).
                                <strong>
                                 F)
                                </strong>
                                Area of each activity field in square meters (total environment area is 4.8 m
                                <sup>
                                 2
                                </sup>
                                ).
                                <strong>
                                 G)
                                </strong>
                                Eccentricity of fields. A score of 1.0 indicates a perfectly round field; lower scores indicate elongation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Responses to geometric transformations
                             </h3>
                             <p id="p-19">
                              Simple manipulations of the test environment made without retraining the model, such as rotating distal cues or stretching the enclosure, produced commensurate changes in the receptive fields of spatially modulated cells - resembling the responses of rodent spatial neurons after similar transformations
                              <sup>
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                               ,
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                              </sup>
                              (
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Supplementary Fig 5
                              </a>
                              ,
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Fig 6
                              </a>
                              , and Supplemental Results). Similarly, an extra barrier inserted into a familiar environment becomes a additional target for BVC and egoBVC responses (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Supplementary Fig 7
                              </a>
                              , and
                              <a id="xref-sec-30-3" class="xref-sec" href="#sec-30">
                               Supplementary Results
                              </a>
                              ), causing some place fields to duplicate and others to be suppressed
                              <sup>
                               <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                               ,
                               <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              .
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F9.large.jpg?width=800&height=600&carousel=1" title="Effects of distal cue rotation (45°) on the representations of the model shown in Fig 2. There was no training after the manipulation. A) Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. HD activity followed the rotation of distal cues. B) Number of RNN units identified as HD cells before and after the environment manipulation. C) Histogram of change in phase of the resultant vectors of all HD cells. D) Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Rotation of distal cues did not affect egoBVCs’ activity. E) Number of RNN units identified as egoBVCs before and after the environment manipulation. F) Histogram of change in preferred egocentric angle to boundary of egoBVCs. G) Histogram of change in preferred distance to boundary of egoBVCs. H) Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. BVCs followed the rotation of distal cues. I) Number of RNN units identified as BVCs before and after the environment manipulation. J) Histogram of change in preferred allocentric angle to boundary of BVCs. K) Histogram of change in preferred distance to boundary of BVCs. L) Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation. M) Number of fields for each place cell before and after the environment manipulation. N) Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation. O) Histogram of change in phase of the place-cell centroid of activity when calculated in polar coordinates taking as origin the centre of the enclosure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Effects of distal cue rotation (45°) on the representations of the model shown in Fig 2. There was no training after the manipulation. <strong>A)</strong> Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. HD activity followed the rotation of distal cues. <strong>B)</strong> Number of RNN units identified as HD cells before and after the environment manipulation. <strong>C)</strong> Histogram of change in phase of the resultant vectors of all HD cells. <strong>D)</strong> Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Rotation of distal cues did not affect egoBVCs’ activity. <strong>E)</strong> Number of RNN units identified as egoBVCs before and after the environment manipulation. <strong>F)</strong> Histogram of change in preferred egocentric angle to boundary of egoBVCs. <strong>G)</strong> Histogram of change in preferred distance to boundary of egoBVCs. <strong>H)</strong> Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. BVCs followed the rotation of distal cues. <strong>I)</strong> Number of RNN units identified as BVCs before and after the environment manipulation. <strong>J)</strong> Histogram of change in preferred allocentric angle to boundary of BVCs. <strong>K)</strong> Histogram of change in preferred distance to boundary of BVCs. <strong>L)</strong> Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation. <strong>M)</strong> Number of fields for each place cell before and after the environment manipulation. <strong>N)</strong> Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation. <strong>O)</strong> Histogram of change in phase of the place-cell centroid of activity when calculated in polar coordinates taking as origin the centre of the enclosure.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F9.medium.gif" width="440" height="314"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F9.medium.gif" width="440" height="314"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 5:
                               </span>
                               <p id="p-154" class="first-child">
                                Effects of distal cue rotation (45°) on the representations of the model shown in
                                <a id="xref-fig-2-18" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                . There was no training after the manipulation.
                                <strong>
                                 A)
                                </strong>
                                Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. HD activity followed the rotation of distal cues.
                                <strong>
                                 B)
                                </strong>
                                Number of RNN units identified as HD cells before and after the environment manipulation.
                                <strong>
                                 C)
                                </strong>
                                Histogram of change in phase of the resultant vectors of all HD cells.
                                <strong>
                                 D)
                                </strong>
                                Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Rotation of distal cues did not affect egoBVCs’ activity.
                                <strong>
                                 E)
                                </strong>
                                Number of RNN units identified as egoBVCs before and after the environment manipulation.
                                <strong>
                                 F)
                                </strong>
                                Histogram of change in preferred egocentric angle to boundary of egoBVCs.
                                <strong>
                                 G)
                                </strong>
                                Histogram of change in preferred distance to boundary of egoBVCs.
                                <strong>
                                 H)
                                </strong>
                                Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. BVCs followed the rotation of distal cues.
                                <strong>
                                 I)
                                </strong>
                                Number of RNN units identified as BVCs before and after the environment manipulation.
                                <strong>
                                 J)
                                </strong>
                                Histogram of change in preferred allocentric angle to boundary of BVCs.
                                <strong>
                                 K)
                                </strong>
                                Histogram of change in preferred distance to boundary of BVCs.
                                <strong>
                                 L)
                                </strong>
                                Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation.
                                <strong>
                                 M)
                                </strong>
                                Number of fields for each place cell before and after the environment manipulation.
                                <strong>
                                 N)
                                </strong>
                                Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation.
                                <strong>
                                 O)
                                </strong>
                                Histogram of change in phase of the place-cell centroid of activity when calculated in polar coordinates taking as origin the centre of the enclosure.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F10.large.jpg?width=800&height=600&carousel=1" title="Effects of enclosure resizing (width was doubled) on the representations of the model shown in Fig 2. There was no training after the manipulation. A) Polar plots of five cells identified as head-direction cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD-cells. B) Number of RNN units identified as head-direction cells before and after the environment manipulation. C) Histogram of change in phase of the resultant vectors of all HD cells. D) Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response after the stretch. E) Number of RNN units identified as egoBVCs before and after the environment manipulation. F) Histogram of change in preferred egocentric angle to boundary of egoBVCs. G) Histogram of change in preferred distance to boundary of egoBVCs. H) Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response after the stretch. I) Number of RNN units identified as BVCs before and after the environment manipulation. J) Histogram of change in preferred allocentric angle to boundary of BVCs. K) Histogram of change in preferred distance to boundary of BVCs. L) Ratemap of five place-cell-like units. Top, original environment; below, after manipulation. The manipulation caused the activity fields of most cells to stretch with the stretched environment axis. However some cells’ activity field split into two fields. M) Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation. N) Number of fields for each place cell before and after the environment manipulation. O) Change in size and eccentricity of each place cell’s activity field." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Effects of enclosure resizing (width was doubled) on the representations of the model shown in Fig 2. There was no training after the manipulation. <strong>A)</strong> Polar plots of five cells identified as head-direction cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD-cells. <strong>B)</strong> Number of RNN units identified as head-direction cells before and after the environment manipulation. <strong>C)</strong> Histogram of change in phase of the resultant vectors of all HD cells. <strong>D)</strong> Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response after the stretch. <strong>E)</strong> Number of RNN units identified as egoBVCs before and after the environment manipulation. <strong>F)</strong> Histogram of change in preferred egocentric angle to boundary of egoBVCs. <strong>G)</strong> Histogram of change in preferred distance to boundary of egoBVCs. <strong>H)</strong> Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response after the stretch. <strong>I)</strong> Number of RNN units identified as BVCs before and after the environment manipulation. <strong>J)</strong> Histogram of change in preferred allocentric angle to boundary of BVCs. <strong>K)</strong> Histogram of change in preferred distance to boundary of BVCs. <strong>L)</strong> Ratemap of five place-cell-like units. Top, original environment; below, after manipulation. The manipulation caused the activity fields of most cells to stretch with the stretched environment axis. However some cells’ activity field split into two fields. <strong>M)</strong> Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation. <strong>N)</strong> Number of fields for each place cell before and after the environment manipulation. <strong>O)</strong> Change in size and eccentricity of each place cell’s activity field.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F10.medium.gif" width="440" height="391"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F10.medium.gif" width="440" height="391"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 6:
                               </span>
                               <p id="p-155" class="first-child">
                                Effects of enclosure resizing (width was doubled) on the representations of the model shown in
                                <a id="xref-fig-2-19" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                . There was no training after the manipulation.
                                <strong>
                                 A)
                                </strong>
                                Polar plots of five cells identified as head-direction cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD-cells.
                                <strong>
                                 B)
                                </strong>
                                Number of RNN units identified as head-direction cells before and after the environment manipulation.
                                <strong>
                                 C)
                                </strong>
                                Histogram of change in phase of the resultant vectors of all HD cells.
                                <strong>
                                 D)
                                </strong>
                                Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response after the stretch.
                                <strong>
                                 E)
                                </strong>
                                Number of RNN units identified as egoBVCs before and after the environment manipulation.
                                <strong>
                                 F)
                                </strong>
                                Histogram of change in preferred egocentric angle to boundary of egoBVCs.
                                <strong>
                                 G)
                                </strong>
                                Histogram of change in preferred distance to boundary of egoBVCs.
                                <strong>
                                 H)
                                </strong>
                                Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response after the stretch.
                                <strong>
                                 I)
                                </strong>
                                Number of RNN units identified as BVCs before and after the environment manipulation.
                                <strong>
                                 J)
                                </strong>
                                Histogram of change in preferred allocentric angle to boundary of BVCs.
                                <strong>
                                 K)
                                </strong>
                                Histogram of change in preferred distance to boundary of BVCs.
                                <strong>
                                 L)
                                </strong>
                                Ratemap of five place-cell-like units. Top, original environment; below, after manipulation. The manipulation caused the activity fields of most cells to stretch with the stretched environment axis. However some cells’ activity field split into two fields.
                                <strong>
                                 M)
                                </strong>
                                Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation.
                                <strong>
                                 N)
                                </strong>
                                Number of fields for each place cell before and after the environment manipulation.
                                <strong>
                                 O)
                                </strong>
                                Change in size and eccentricity of each place cell’s activity field.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F11.large.jpg?width=800&height=600&carousel=1" title="Effects of extra barrier insertion on the representations of the model shown in Fig 2. There was no training after the manipulation. A) Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD cells. B) Number of RNN units identified as HD cells before and after the environment manipulation. C) Histogram of change in phase of the resultant vectors of all HD cells. D) Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response to the new barrier. E) Number of RNN units identified as egoBVCs before and after the environment manipulation. F) Histogram of change in preferred egocentric angle to boundary of egoBVCs. G) Histogram of change in preferred distance to boundary of egoBVCs. H) Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response to the new barrier. I) Number of RNN units identified as BVCs before and after the environment manipulation. J) Histogram of change in preferred allocentric angle to boundary of BVCs. K) Histogram of change in preferred distance to boundary of BVCs. L) Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation. The extra barrier caused cells in the region to duplicate their fields. Cells with distant activity fields were not affected. M) Number of fields for each place cell before and after the environment manipulation. N) Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Effects of extra barrier insertion on the representations of the model shown in Fig 2. There was no training after the manipulation. <strong>A)</strong> Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD cells. <strong>B)</strong> Number of RNN units identified as HD cells before and after the environment manipulation. <strong>C)</strong> Histogram of change in phase of the resultant vectors of all HD cells. <strong>D)</strong> Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response to the new barrier. <strong>E)</strong> Number of RNN units identified as egoBVCs before and after the environment manipulation. <strong>F)</strong> Histogram of change in preferred egocentric angle to boundary of egoBVCs. <strong>G)</strong> Histogram of change in preferred distance to boundary of egoBVCs. <strong>H)</strong> Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response to the new barrier. <strong>I)</strong> Number of RNN units identified as BVCs before and after the environment manipulation. <strong>J)</strong> Histogram of change in preferred allocentric angle to boundary of BVCs. <strong>K)</strong> Histogram of change in preferred distance to boundary of BVCs. <strong>L)</strong> Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation. The extra barrier caused cells in the region to duplicate their fields. Cells with distant activity fields were not affected. <strong>M)</strong> Number of fields for each place cell before and after the environment manipulation. <strong>N)</strong> Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F11.medium.gif" width="440" height="318"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F11.medium.gif" width="440" height="318"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 7:
                               </span>
                               <p id="p-156" class="first-child">
                                Effects of extra barrier insertion on the representations of the model shown in
                                <a id="xref-fig-2-20" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                . There was no training after the manipulation.
                                <strong>
                                 A)
                                </strong>
                                Polar plots of five cells identified as HD cells. Top row, original environment. Second row, after manipulation. The manipulation did not affect HD cells.
                                <strong>
                                 B)
                                </strong>
                                Number of RNN units identified as HD cells before and after the environment manipulation.
                                <strong>
                                 C)
                                </strong>
                                Histogram of change in phase of the resultant vectors of all HD cells.
                                <strong>
                                 D)
                                </strong>
                                Ratemap of five cells identified as egoBVCs. Top row, original environment. Second row, after manipulation. Cells maintained their egocentric response to the new barrier.
                                <strong>
                                 E)
                                </strong>
                                Number of RNN units identified as egoBVCs before and after the environment manipulation.
                                <strong>
                                 F)
                                </strong>
                                Histogram of change in preferred egocentric angle to boundary of egoBVCs.
                                <strong>
                                 G)
                                </strong>
                                Histogram of change in preferred distance to boundary of egoBVCs.
                                <strong>
                                 H)
                                </strong>
                                Ratemap of five cells identified as (allocentric) BVCs. Top row, original environment. Second row, after manipulation. Cells maintained their allocentric response to the new barrier.
                                <strong>
                                 I)
                                </strong>
                                Number of RNN units identified as BVCs before and after the environment manipulation.
                                <strong>
                                 J)
                                </strong>
                                Histogram of change in preferred allocentric angle to boundary of BVCs.
                                <strong>
                                 K)
                                </strong>
                                Histogram of change in preferred distance to boundary of BVCs.
                                <strong>
                                 L)
                                </strong>
                                Ratemap of five place-cell-like units. Top row, original environment. Second row, after manipulation. The extra barrier caused cells in the region to duplicate their fields. Cells with distant activity fields were not affected.
                                <strong>
                                 M)
                                </strong>
                                Number of fields for each place cell before and after the environment manipulation.
                                <strong>
                                 N)
                                </strong>
                                Displacement of place-cell field centroid after manipulation. Blue cross indicates the original centroid, red arrows show their displacement. Black arrows show the displacement when a single field in the original environment split into several fields after manipulation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Learned attractor dynamics in the head-direction system
                             </h3>
                             <p id="p-20">
                              In the mammalian brain, head-direction tuning is widely believed to originate from a neural ring attractor which constrains activity to lie on a 1D manifold - this provides a mechanism to integrate head turns when appropriately connected with neurons responsive to angular velocity
                              <sup>
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                              </sup>
                              . In our model, when RNN-1 was instantiated separately and provided only with angular velocities (see
                              <a id="xref-sec-9-5" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ), it displayed a single activity bump that closely tracked the apparent head direction for several hundred steps, effectively integrating angular velocity over periods that greatly exceeded the duration of training trajectories (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig 3A
                              </a>
                              ). Projecting the observed activity vectors onto their first two principal components revealed that they resided close to a 1-D circular manifold (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig 3B
                              </a>
                              ), a characteristic associated with linear continuous attractor systems. Indeed, with zero angular velocity inputs, random initialisations of the network state quickly converged to a set of point attractors (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig 3C–D
                              </a>
                              , see
                              <a id="xref-sec-9-6" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ).
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F3.large.jpg?width=800&height=600&carousel=1" title="Head-direction cell dynamics. A) Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to π/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps π/40 rad/step. B) Projection of the 1000 activity vectors shown in A onto their first two principal components. C) Euclidean distance between successive states in 1000 trajectories with zero angular velocity and random initialisation (see Methods). Thick blue shows the average over one-thousand different initialisations, the thin lines show the 5-th and 95-th percentiles. D) Grey shows the projection of the random initial states used in the simulations of panel C onto the space of panel B. Green shows the state after 10 steps of dynamics with zero angular velocity. Black crosses show the states after 1000 steps. All the trajectories converged to a discrete set of attractor states. E) Projection on the space of panel B of two 50-step trajectories with constant angular velocity input, one positive and one negative, each starting from a random initialisation. F) Autocovariance of states as a function of time lag computed over 1000 randomly initialised 200-step trajectories with positive and negative velocities. G) Histogram of steps to convergence to a single attractor state for 512 correction images. Insets show examples with quick convergence (blue) and some that do not converge to a single state (red). H) Several example state trajectories, initialised from the attractor states in panel D (different colours), providing visual inputs with unambiguous cues (top) for 200 steps. Black crosses show the final states. I) Same as H with ambiguous visual input of a wall." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Head-direction cell dynamics. <strong>A)</strong> Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to <em>π</em>/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps <em>π</em>/40 rad/step. <strong>B)</strong> Projection of the 1000 activity vectors shown in A onto their first two principal components. <strong>C)</strong> Euclidean distance between successive states in 1000 trajectories with zero angular velocity and random initialisation (see Methods). Thick blue shows the average over one-thousand different initialisations, the thin lines show the 5-th and 95-th percentiles. <strong>D)</strong> Grey shows the projection of the random initial states used in the simulations of panel C onto the space of panel B. Green shows the state after 10 steps of dynamics with zero angular velocity. Black crosses show the states after 1000 steps. All the trajectories converged to a discrete set of attractor states. <strong>E)</strong> Projection on the space of panel B of two 50-step trajectories with constant angular velocity input, one positive and one negative, each starting from a random initialisation. <strong>F)</strong> Autocovariance of states as a function of time lag computed over 1000 randomly initialised 200-step trajectories with positive and negative velocities. <strong>G)</strong> Histogram of steps to convergence to a single attractor state for 512 correction images. Insets show examples with quick convergence (blue) and some that do not converge to a single state (red). <strong>H)</strong> Several example state trajectories, initialised from the attractor states in panel D (different colours), providing visual inputs with unambiguous cues (top) for 200 steps. Black crosses show the final states. <strong>I)</strong> Same as H with ambiguous visual input of a wall.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F3.medium.gif" width="440" height="339"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F3.medium.gif" width="440" height="339"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <p id="p-21" class="first-child">
                                Head-direction cell dynamics.
                                <strong>
                                 A)
                                </strong>
                                Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to π/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps π/40 rad/step.
                                <strong>
                                 B)
                                </strong>
                                Projection of the 1000 activity vectors shown in A onto their first two principal components.
                                <strong>
                                 C)
                                </strong>
                                Euclidean distance between successive states in 1000 trajectories with zero angular velocity and random initialisation (see
                                <a id="xref-sec-9-7" class="xref-sec" href="#sec-9">
                                 Methods
                                </a>
                                ). Thick blue shows the average over one-thousand different initialisations, the thin lines show the 5-th and 95-th percentiles.
                                <strong>
                                 D)
                                </strong>
                                Grey shows the projection of the random initial states used in the simulations of panel C onto the space of panel B. Green shows the state after 10 steps of dynamics with zero angular velocity. Black crosses show the states after 1000 steps. All the trajectories converged to a discrete set of attractor states.
                                <strong>
                                 E)
                                </strong>
                                Projection on the space of panel B of two 50-step trajectories with constant angular velocity input, one positive and one negative, each starting from a random initialisation.
                                <strong>
                                 F)
                                </strong>
                                Autocovariance of states as a function of time lag computed over 1000 randomly initialised 200-step trajectories with positive and negative velocities.
                                <strong>
                                 G)
                                </strong>
                                Histogram of steps to convergence to a single attractor state for 512 correction images. Insets show examples with quick convergence (blue) and some that do not converge to a single state (red).
                                <strong>
                                 H)
                                </strong>
                                Several example state trajectories, initialised from the attractor states in panel D (different colours), providing visual inputs with unambiguous cues (top) for 200 steps. Black crosses show the final states.
                                <strong>
                                 I)
                                </strong>
                                Same as H with ambiguous visual input of a wall.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              In contrast, positive or negative angular velocities drove the state to periodic orbits along 1D cyclic attractors (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig 3E–F
                              </a>
                              ). In the mammalian head direction system these dynamics
                              <sup>
                               <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              are hypothesised to be supported by a double ring network with each ring having counter rotated tuning - a similar solution has been observed in the fly
                              <sup>
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              . A strikingly similar connectivity can be learned by simple RNN formulations
                              <sup>
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                              </sup>
                              (
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Supplementary Fig 8
                              </a>
                              ,
                              <a id="xref-sec-30-4" class="xref-sec" href="#sec-30">
                               Supplementary Results
                              </a>
                              ). Angular velocity integration alone is not sufficient to maintain a stable representation of head direction – visual cues are required to reset the activity of units and correct for drift. To investigate how our model incorporates visual information in its representation of heading, we simulated the input of visual corrections (512 images from the training environment) and zero angular velocity (see
                              <a id="xref-sec-9-8" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ). The majority of images (352/512) resulted in network dynamics with a single attractor point per image, regardless of the initial network state (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig 3G
                              </a>
                              ). These images typically display unambiguous distal cues (
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig 3H
                              </a>
                              ). The remaining images (160/512) did not result in a single attractor point, possibly reflecting the geometrical symmetries of the environment. Interestingly, upon visual examination, these images usually lack distal cues and correspond to views of walls and corners (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig 3I
                              </a>
                              ).
                             </p>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F12.large.jpg?width=800&height=600&carousel=1" title="Head direction cell connectivity learnt by a Vanilla-RNN network. A) Polar plot of the resultant vectors of cells in RNN-1. B) Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to π/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps π/40 rad/step. C) Weights in the RNN dynamics matrix, W. Columns and rows ordered by the phase of their resultant vector. D) Histogram of preferred angular velocities for each cell. E) Matrix of weights from CCW cells to all other cells. F) Matrix of weights from CW cells to all other cells. G) Average weights connecting cells in each of the two rings to all other cells ordered by their angular offset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Head direction cell connectivity learnt by a Vanilla-RNN network. <strong>A)</strong> Polar plot of the resultant vectors of cells in RNN-1. <strong>B)</strong> Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to <em>π</em>/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps <em>π</em>/40 rad/step. <strong>C)</strong> Weights in the RNN dynamics matrix, <strong>W</strong>. Columns and rows ordered by the phase of their resultant vector. <strong>D)</strong> Histogram of preferred angular velocities for each cell. <strong>E)</strong> Matrix of weights from CCW cells to all other cells. <strong>F)</strong> Matrix of weights from CW cells to all other cells. <strong>G)</strong> Average weights connecting cells in each of the two rings to all other cells ordered by their angular offset.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F12.medium.gif" width="440" height="195"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F12.medium.gif" width="440" height="195"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 8:
                               </span>
                               <p id="p-157" class="first-child">
                                Head direction cell connectivity learnt by a Vanilla-RNN network.
                                <strong>
                                 A)
                                </strong>
                                Polar plot of the resultant vectors of cells in RNN-1.
                                <strong>
                                 B)
                                </strong>
                                Activation of each unit over 1000 steps of blind integration. Top: True integrated angle. Bottom: Activation of HD-units in RNN-1 through time (units ordered by the phase of their resultant vectors). For the first 250 steps the angular velocity was set to π/20 rad/step, the following 250 steps 0 rad/step, the remaining 500 steps π/40 rad/step.
                                <strong>
                                 C)
                                </strong>
                                Weights in the RNN dynamics matrix,
                                <strong>
                                 W
                                </strong>
                                . Columns and rows ordered by the phase of their resultant vector.
                                <strong>
                                 D)
                                </strong>
                                Histogram of preferred angular velocities for each cell.
                                <strong>
                                 E)
                                </strong>
                                Matrix of weights from CCW cells to all other cells.
                                <strong>
                                 F)
                                </strong>
                                Matrix of weights from CW cells to all other cells.
                                <strong>
                                 G)
                                </strong>
                                Average weights connecting cells in each of the two rings to all other cells ordered by their angular offset.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Spatial characteristics are preserved across environments
                             </h3>
                             <p id="p-23">
                              Next, we sought to establish how stable the different representations in our model were between environments that changed both in terms of their geometry and visual composition. To this end the network was trained concurrently in three environments inspired by empirical studies – a square, a circle, and a trapezoid, all with distinct distal cues, floor and wall textures (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig 4A
                              </a>
                              ). The same RNNs were used throughout but different external memory stores were provided for each environment. Despite the different training environments, we again found similar proportions of head direction, egoBVC, BVC, and place cells in the network (see
                              <a id="xref-sec-30-5" class="xref-sec" href="#sec-30">
                               Supplementary Results
                              </a>
                              ). Indeed, although the environments were visually and geometrically distinct, the classification of units did not materially change between them: 84% (27/32) RNN-1 units were classified as HD cells in all three environments, similarly 78% (100/128) of RNN-2 units where classified as egoBVCs, and 61% (78/128) of RNN-3 units as BVCs. Notably, like biological cells
                              <sup>
                               <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">
                                4
                               </a>
                              </sup>
                              , head direction units maintained their relative angular tuning between environments, while egoBVCs and BVCs maintained their preferred distance to walls (
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Supplementary Fig 9
                              </a>
                              , see
                              <a id="xref-sec-9-9" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ).
                             </p>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F13.large.jpg?width=800&height=600&carousel=1" title="Stability of representations learnt across three environments: square (1), circular (2) and trapezoidal (3). First column compares environments 1 and 2, second column environments 1 and 3, third column environments 2 and 3. A) Histogram of HD cell preferred direction differences between environment pairs. HD cells rotate coherently between environments. B) Histogram of egoBVC preferred egocentric direction differences between environments. The egoBVCs preserve their angular tuning between environments. C) Histogram of BVC preferred allocentric direction differences between environments. The BVCs do not rotate coherently between environments. D) Histogram of egoBVC preferred distance differences between environments (red) and normalised histogram of differences between randomly shuffled units (blue). The egoBVCs preserve their distance tuning across environments. E) Same as D) for BVCs. The distance tuning is significantly preserved across environments, although not as tightly as in the case of egoBVCs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Stability of representations learnt across three environments: square (1), circular (2) and trapezoidal (3). First column compares environments 1 and 2, second column environments 1 and 3, third column environments 2 and 3. <strong>A)</strong> Histogram of HD cell preferred direction differences between environment pairs. HD cells rotate coherently between environments. <strong>B)</strong> Histogram of egoBVC preferred egocentric direction differences between environments. The egoBVCs preserve their angular tuning between environments. <strong>C)</strong> Histogram of BVC preferred allocentric direction differences between environments. The BVCs do not rotate coherently between environments. <strong>D)</strong> Histogram of egoBVC preferred distance differences between environments (red) and normalised histogram of differences between randomly shuffled units (blue). The egoBVCs preserve their distance tuning across environments. <strong>E)</strong> Same as D) for BVCs. The distance tuning is significantly preserved across environments, although not as tightly as in the case of egoBVCs.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F13.medium.gif" width="339" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Supplementary Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F13.medium.gif" width="339" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 9:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Supplementary Figure 9:
                               </span>
                               <p id="p-158" class="first-child">
                                Stability of representations learnt across three environments: square (1), circular (2) and trapezoidal (3). First column compares environments 1 and 2, second column environments 1 and 3, third column environments 2 and 3.
                                <strong>
                                 A)
                                </strong>
                                Histogram of HD cell preferred direction differences between environment pairs. HD cells rotate coherently between environments.
                                <strong>
                                 B)
                                </strong>
                                Histogram of egoBVC preferred egocentric direction differences between environments. The egoBVCs preserve their angular tuning between environments.
                                <strong>
                                 C)
                                </strong>
                                Histogram of BVC preferred allocentric direction differences between environments. The BVCs do not rotate coherently between environments.
                                <strong>
                                 D)
                                </strong>
                                Histogram of egoBVC preferred distance differences between environments (red) and normalised histogram of differences between randomly shuffled units (blue). The egoBVCs preserve their distance tuning across environments.
                                <strong>
                                 E)
                                </strong>
                                Same as D) for BVCs. The distance tuning is significantly preserved across environments, although not as tightly as in the case of egoBVCs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F4.large.jpg?width=800&height=600&carousel=1" title="A) Stability of spatial representations across three environments, one per row. Left to right: view of the enclosure; allocentric orientation tuning of four units classified as HD cells; spatial ratemap of three units classified as egoBVCs; and spatial ratemap of three units classified as BVCs. Same plotting conventions as Fig 2. B) Top, reinforcement learning task in a 4-room enclosure. Spawning points marked with an S. The initial trajectory to the reward is dotted, later trajectories solid. Bottom, visual input from the 3 training enclosures. C) Distribution of training times needed to reach human performance in the reinforcement learning task. In red Spatial Memory Pipeline agents, in black baseline agents. Dashed indicates training from scratch, solid indicates transfer to a visually different environment from an agent previously trained in the environments shown in B. On top, schematic of enclosure and example visual input. D) Same as C but transfer is to an enclosure with a different layout (cross-shaped). E) Same as C but transfer is to a different task, where in each episode a randomly chosen corridor is blocked. F) Same as C but transfer is to a conceptual representation of the environment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>A)</strong> Stability of spatial representations across three environments, one per row. Left to right: view of the enclosure; allocentric orientation tuning of four units classified as HD cells; spatial ratemap of three units classified as egoBVCs; and spatial ratemap of three units classified as BVCs. Same plotting conventions as Fig 2. <strong>B)</strong> Top, reinforcement learning task in a 4-room enclosure. Spawning points marked with an S. The initial trajectory to the reward is dotted, later trajectories solid. Bottom, visual input from the 3 training enclosures. <strong>C)</strong> Distribution of training times needed to reach human performance in the reinforcement learning task. In red Spatial Memory Pipeline agents, in black baseline agents. Dashed indicates training from scratch, solid indicates transfer to a visually different environment from an agent previously trained in the environments shown in B. On top, schematic of enclosure and example visual input. <strong>D)</strong> Same as C but transfer is to an enclosure with a different layout (cross-shaped). <strong>E)</strong> Same as C but transfer is to a different task, where in each episode a randomly chosen corridor is blocked. <strong>F)</strong> Same as C but transfer is to a conceptual representation of the environment.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F4.medium.gif" width="440" height="274"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F4.medium.gif" width="440" height="274"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <p id="p-24" class="first-child">
                                <strong>
                                 A)
                                </strong>
                                Stability of spatial representations across three environments, one per row. Left to right: view of the enclosure; allocentric orientation tuning of four units classified as HD cells; spatial ratemap of three units classified as egoBVCs; and spatial ratemap of three units classified as BVCs. Same plotting conventions as
                                <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                                 Fig 2
                                </a>
                                .
                                <strong>
                                 B)
                                </strong>
                                Top, reinforcement learning task in a 4-room enclosure. Spawning points marked with an S. The initial trajectory to the reward is dotted, later trajectories solid. Bottom, visual input from the 3 training enclosures.
                                <strong>
                                 C)
                                </strong>
                                Distribution of training times needed to reach human performance in the reinforcement learning task. In red Spatial Memory Pipeline agents, in black baseline agents. Dashed indicates training from scratch, solid indicates transfer to a visually different environment from an agent previously trained in the environments shown in B. On top, schematic of enclosure and example visual input.
                                <strong>
                                 D)
                                </strong>
                                Same as C but transfer is to an enclosure with a different layout (cross-shaped).
                                <strong>
                                 E)
                                </strong>
                                Same as C but transfer is to a different task, where in each episode a randomly chosen corridor is blocked.
                                <strong>
                                 F)
                                </strong>
                                Same as C but transfer is to a conceptual representation of the environment.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Representation re-use allows transfer of behaviour
                             </h3>
                             <p id="p-25">
                              In animals, allocentric representations such as head-direction cells and BVCs are rapidly redeployed in novel settings - likely as a result of being constrained to low-dimensional manifolds which decouple the integration of self-motion from high-dimensional perceptual experience
                              <sup>
                               <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                              </sup>
                              . We hypothesised that this self-consistency is an adaptive characteristic allowing spatial behaviour learned in one environment to be quickly transferred to novel environments.
                             </p>
                             <p id="p-26">
                              To test this proposal we incorporated the Spatial Memory Pipeline into a deep reinforcement learning agent
                              <sup>
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                              </sup>
                              (see
                              <a id="xref-sec-9-10" class="xref-sec" href="#sec-9">
                               Methods
                              </a>
                              ) trained to find an invisible goal in a 4-room enclosure (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig 4B
                              </a>
                              ), a task inspired by the classic Morris water maze
                              <sup>
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              . This task captured two levels of localisation: locally within a room (where in the room?) and globally across rooms (which room?). When the agent reached the goal, it received a reward and was teleported to a random location in the enclosure - to maximise reward it had to reach the goal as many times as possible within each episode. Three visually different enclosures were used simultaneously for training (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig 4B
                              </a>
                              ). As before, separate memory stores were used for each enclosure but RNNs were shared.
                             </p>
                             <p id="p-27">
                              Once trained, the agent reached a high degree of proficiency in the task, routinely following the shortest path to the goal (better than human performance). Inspection of the memory pipeline confirmed that the RNNs contained spatial representations consistent across the three visually different environments (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Supplementary Fig 12
                              </a>
                              ). To test our claim that these allocentric representations provide efficient bases for transfer, the trained agent was placed in a visually different 4-room enclosure (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig 4C
                              </a>
                              ), an enclosure with different room layout and visual aspect (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig 4D
                              </a>
                              ), a variation of the 4-room task where corridors were blocked at random (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig 4E
                              </a>
                              ), and a 4-room enclosure presented conceptually to the agent as a two-dimensional plan view (
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Fig 4F
                              </a>
                              ). In all transfer tasks the agent reached human performance significantly faster than an agent trained from scratch (80x, 12x, 13x and 4x respectively). In contrast, if generic recurrent networks were used instead of the Spatial Memory Pipeline, the agent was still able to learn the initial task but achieved only modest performance gains in transfer (1.7x, 1.5x, 1.3x and 1.3x) (
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               Supplementary Fig 11
                              </a>
                              ). Thus, the spatial responses present in the Spatial Memory Pipeline’s RNN support rapid generalisation to novel settings, an ability that is commonly observed in mammals but has been an elusive target for artificial agents.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-28">
                             The Spatial Memory Pipeline described by our model develops an array of sensory-derived allocentric spatial representations strongly resembling those found in the mammalian brain. These representations are learnt solely using a predictive coding principle from egocentric visual perception and self-motion inputs. Like the brain, the network does not have access to allocentric information, and differs markedly from prior work in which brain-inspired agents were limited by the need for allocentric input during training
                             <sup>
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               19
                              </a>
                              ,
                              <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ,
                              <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ,
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               46
                              </a>
                             </sup>
                             . In contrast, the forms of egocentric representation used here are known to contribute to the activity of spatially modulated neurons in the hippocampal region
                             <sup>
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ,
                              <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                               48
                              </a>
                             </sup>
                             while the predictive framework agrees with empirical and theoretical work linking the hippocampus with anticipation of the future
                             <sup>
                              <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                              ,
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              ,
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               50
                              </a>
                             </sup>
                             . Our core contribution then is the provision of a normative model that explains the appearance of known mammalian spatial representations solely from interaction with the sensory world. This model poses hypotheses about the interactions between these representations, the neural dynamics that generate them, and their role in spatial navigation.
                            </p>
                            <p id="p-29">
                             The cell-types that we identified were segregated between different sub-networks, being defined by the forms of information available to them: angular velocity and visual corrections for head-direction cells; angular velocity, speed, and visual corrections for egoBVCs; and only visual corrections for BVCs. External memories reactivated by BVC inputs strongly resembled place cell activity. This framework implicitly captures several properties of the mammalian spatial system - distinct functional populations combining to form a sparse, spatially precise representation of self-location similar to that seen in the hippocampus. Like their biological counterparts, spatial responses in the RNNs were robust to environmental manipulations, responding predictably to the changes made to local cues while retaining their fundamental firing correlates between entirely different enclosures. Our work demonstrates that these representations constitute a flexible and efficient spatial basis, available to be quickly redeployed in a novel setting - sufficient to support rapid transfer learning in a reinforcement learning agent. On the other hand, activity in the memory stores was, by design, entirely distinct between environments, emulating hippocampal remapping, a phenomenon that occurs in response to significant manipulations of environmental cues
                             <sup>
                              <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                               51
                              </a>
                              ,
                              <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                               52
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-30">
                             In the case of head-direction cells in the first RNN, the activity of units resulted from learnt attractor dynamics – both in the integration of velocities and in the anchoring to visual cues. Low dimensional manifolds of this form are widely accepted to provide the preeminent account of the biological head-direction system, explaining the fixed relative offset seen in biological head-direction cells as well as in our model
                             <sup>
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ,
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               53
                              </a>
                             </sup>
                             . Notably a similar head-direction attractor circuit is instantiated topographically in flies
                             <sup>
                              <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                               54
                              </a>
                              ,
                              <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                               55
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-31">
                             A full understanding of the spatial circuits in the mammalian brain will require resolving the functional dependence between different representations. In our model the relationship between the head-direction and BVC systems is determined by their joint association to visual memories (
                             <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                              Supplementary Fig 10
                             </a>
                             ) - thus they rotate coherently to follow changes in the angular location of a familiar distal cue
                             <sup>
                              <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                               56
                              </a>
                             </sup>
                             (
                             <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                              Supplementary Fig 5
                             </a>
                             ). Conversely, in different enclosures with non-overlapping sets of visual memories, biological BVCs have been observed to decouple from the head-direction system while retaining their relative tuning
                             <sup>
                              <a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             . This mechanism of head-direction and BVC dependence contrasts with the hypothesis that postulates BVC activity as the result of direct conjunction of head-direction and egoBVC activations
                             <sup>
                              <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ,
                              <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                             </sup>
                             . Hence, new experimental studies on animals jointly recording the head-direction and BVC systems across enclosures that elicit global remapping will be required to ascertain the mechanisms that give rise to BVCs.
                            </p>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F14.large.jpg?width=800&height=600&carousel=1" title="Hypothesised BVC mechanisms. A) Traditional BVC mechanism: BVCs result from the conjunction of HD and egoBVC signals. B) Proposed BVC mechanism: BVCs are driven by the reactivation of visual memories and temporal coherence." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Hypothesised BVC mechanisms. <strong>A)</strong> Traditional BVC mechanism: BVCs result from the conjunction of HD and egoBVC signals. <strong>B)</strong> Proposed BVC mechanism: BVCs are driven by the reactivation of visual memories and temporal coherence.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F14.medium.gif" width="440" height="145"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F14.medium.gif" width="440" height="145"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 10:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 10:
                              </span>
                              <p id="p-159" class="first-child">
                               Hypothesised BVC mechanisms.
                               <strong>
                                A)
                               </strong>
                               Traditional BVC mechanism: BVCs result from the conjunction of HD and egoBVC signals.
                               <strong>
                                B)
                               </strong>
                               Proposed BVC mechanism: BVCs are driven by the reactivation of visual memories and temporal coherence.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-32">
                             In conclusion, we show that an artificial network replicates both the form and function of a biological network central to self-localisation and navigation. A simple training objective is sufficient, in this case, to approximate the development of neural circuits that have been shaped by both selective pressure applied over evolutionary time as well as by direct experience during an animal’s life.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Materials and methods
                             </h3>
                             <div id="sec-11" class="subsection">
                              <h4>
                               The Spatial Memory Pipeline
                              </h4>
                              <p id="p-33">
                               The Spatial Memory Pipeline consists of two submodules where learning occurs independently. The first level is a visual feature extraction network that reduces the dimensionality of visual inputs. Following visual feature extraction, there is a level of integration through time that encodes estimates of the agent’s location by integrating egocentric velocities. We refer to this integration level as Memory-Map. It is composed of a fast-binding slot-based associative memory that plays the role of an idealised hippocampus, and a set of recurrent neural networks that receive as inputs egocentric velocity signals and play the role of an artificial neocortex.
                              </p>
                             </div>
                             <div id="sec-12" class="subsection">
                              <h4>
                               Visual-feature extraction
                              </h4>
                              <p id="p-34">
                               The first level in the Spatial Memory Pipeline hierarchy employs a convolutional autoencoder
                               <sup>
                                <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                                 57
                                </a>
                               </sup>
                               to reduce the dimensionality of the raw visual input (
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Supplementary Fig 1A
                               </a>
                               ). The autoencoder is composed of an encoder network that transforms the raw visual input,
                               <strong>
                                y
                               </strong>
                               <sup>
                                (raw)
                               </sup>
                               into a low-dimensional visual encoding vector,
                               <strong>
                                y
                               </strong>
                               <sup>
                                (enc)
                               </sup>
                               , that summarises the structure of the input. To learn such compressed code,
                               <strong>
                                y
                               </strong>
                               <sup>
                                (enc)
                               </sup>
                               is passed trough a decoder that produces a reconstruction of the original input,
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . All of the encoder and decoder parameters are optimised to minimise the reconstruction error
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Extended Data Table 4 summarises the autoencoder configuration, see
                               <a id="xref-sec-39-3" class="xref-sec" href="#sec-39">
                                Supplementary Methods
                               </a>
                               for more details.
                              </p>
                             </div>
                             <div id="sec-13" class="subsection">
                              <h4>
                               Memory-maps
                              </h4>
                              <p id="p-35">
                               A Memory-map module consists of two components: a set of R recurrent neural networks, {F
                               <sub>
                                r
                               </sub>
                               }
                               <sub>
                                r ∈ 1…R
                               </sub>
                               , that integrate egocentric velocities, and a slot-based associative memory,
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               that binds upstream inputs,
                               <strong>
                                y
                               </strong>
                               , to RNN state values:
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the variable s indexes the different memory slots, while the super-index denotes the type of information (
                               <strong>
                                y
                               </strong>
                               for upstream inputs, and
                               <strong>
                                x
                               </strong>
                               for the state of RNNs).
                              </p>
                              <p id="p-36">
                               At each time step, t, the current upstream input,
                               <strong>
                                y
                               </strong>
                               <sub>
                                t
                               </sub>
                               , reactivates the memory slots with the most similar contents (
                               <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                Supplementary Fig 1B
                               </a>
                               ). We formalise this concept using a categorical distribution over memory slots:
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where β is a positive scalar that is automatically adjusted to match an entropy target, H
                               <sub>
                                react
                               </sub>
                               –enforcing sparse activity (see
                               <a id="xref-sec-39-4" class="xref-sec" href="#sec-39">
                                Supplementary Methods
                               </a>
                               ).
                              </p>
                              <p id="p-37">
                               At every time step, the states corresponding to each of the predictive RNNs, {
                               <strong>
                                x
                               </strong>
                               <sub>
                                r
                               </sub>
                               }
                               <sub>
                                r∈1…R
                               </sub>
                               where
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , are updated using the current egocentric velocities,
                               <strong>
                                v
                               </strong>
                               <sub>
                                r,t
                               </sub>
                               :
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-38">
                               The type of velocity inputs used for each RNN is reported in Extended Data Table 5. For instance, in our unsupervised experiments the first RNN takes as input the sine and cosine of the angular velocity, ω, whereas the second RNN takes the same inputs as the first RNN and also the linear speed, s. Finally, the third RNN receives no velocity inputs and relies only of temporal correlations.
                              </p>
                              <p id="p-39">
                               These RNN states also induce a predictive categorical distribution over memory slots (
                               <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                                Supplementary Fig 1B
                               </a>
                               ):
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where π
                               <sub>
                                r
                               </sub>
                               are positive scalar parameters that determine the relative importance of each RNN in the predictive distribution and its entropy.
                              </p>
                              <p id="p-40">
                               Model parameters are optimised to minimise the cross-entropy from this predictive distribution to the distribution of visually driven reactivations:
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-41">
                               Thus, the model is trained so P
                               <sub>
                                pred
                               </sub>
                               anticipates the distribution of memory reactivations P
                               <sub>
                                react
                               </sub>
                               .
                              </p>
                              <p id="p-42">
                               Note that at time t, P
                               <sub>
                                pred
                               </sub>
                               has not yet received any information from the current upstream visual input,
                               <strong>
                                y
                               </strong>
                               <sub>
                                t
                               </sub>
                               , forcing the RNNs to use previous egocentric velocity inputs to produce good predictions.
                              </p>
                              <p id="p-43">
                               However, in order for the RNN representations to be allocentrically grounded, and to correct for the accumulation of integration errors, the RNNs must incorporate positional and directional information from upstream visual inputs as well. This correction step should not be performed at every time step, or the integration of velocities would be unnecessary; in our experiments, it was performed at random timesteps with probability P
                               <sub>
                                correction
                               </sub>
                               = 0.1. The incorporation of visual information is implemented by calculating a correction code for each RNN:
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where γ is a positive scalar parameter that determines the entropy of the distribution of weights (
                               <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                                Supplementary Fig 1C
                               </a>
                               ). Each
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               can be thought of as the result of a weighted reactivation of the RNN memory embeddings by the current visual input,
                               <strong>
                                y
                               </strong>
                               <sub>
                                t
                               </sub>
                               .
                              </p>
                              <p id="p-44">
                               In steps when corrections are provided, these correction codes are input to correction RNN cells, G
                               <sub>
                                r
                               </sub>
                               , that combine the predictive state with the correction code (
                               <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                                Supplementary Fig 1C
                               </a>
                               ):
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-45">
                               While in steps when no correction is available, the input to the next time step is simply the predicted state (
                               <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                                Supplementary Fig 1D
                               </a>
                               ):
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-46">
                               Model parameters
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are trained by gradient descent on the prediction loss of
                               <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                equation (5)
                               </a>
                               .
                              </p>
                              <p id="p-47">
                               Note that the contents of the memory store corresponding to the upstream inputs,
                               <strong>
                                m
                               </strong>
                               <sup>
                                (
                                <strong>
                                 y
                                </strong>
                                )
                               </sup>
                               ., are not part of the optimised parameters,
                               <strong>
                                Θ
                               </strong>
                               , as they are used to calculate the target distribution. Therefore, in order to fill the slots of
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with memories of upstream inputs, at every timestep with small probability, P
                               <sub>
                                storage
                               </sub>
                               , a slot s is chosen at random and assigned
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-48">
                               Due to the sparse activation of memory slots there is low interference between memories, i.e. modifying a
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . only has a local effect. For this reason these associative memory dings can be optimised at a higher learning rate (see Extended Data Table 2), resulting in fast-binding of new memories once the RNN dynamics have been learned.
                              </p>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Architecture used in our experiments
                              </h4>
                              <p id="p-49">
                               The experimental results in this paper used the architectures described in Extended Data Tables 4-5.
                              </p>
                             </div>
                             <div id="sec-15" class="subsection">
                              <h4>
                               Visual autoencoders
                              </h4>
                              <p id="p-50">
                               The visual-encoding vectors
                               <strong>
                                y
                               </strong>
                               <sup>
                                (enc)
                               </sup>
                               were obtained by inputting RGB images,
                               <strong>
                                y
                               </strong>
                               <sup>
                                (raw)
                               </sup>
                               , of the environment to an encoder network with 4 convolutional layers with bias and ReLU output nonlinearities, chained to a flat output layer (
                               <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                                Supplementary Fig 1A
                               </a>
                               ). The architecture parameters are summarised in Extended Data Table 4. The encoders used for the reinforcement learning experiments had greater capacity than the encoders used for the unsupervised-learning experiments, since they were trained to encode a variety of environments, as explained below.
                              </p>
                              <p id="p-51">
                               In our experiments, the visual autoencoders were not affected by the prediction loss in
                               <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">
                                equation (5)
                               </a>
                               . Therefore, the autoencoders were trained beforehand, separately from the rest of the model. During the autoencoder training, images were passed through the convolutional layers and flat output layer to produce the vector of visual encodings,
                               <strong>
                                y
                               </strong>
                               <sup>
                                (enc)
                               </sup>
                               . This vector was then passed through a decoder network of de-convolutional layers (with transposed architecture from the encoder but independent parameters) to produce a reconstruction,
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , of the input image (
                               <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                                Supplementary Fig 1A
                               </a>
                               ). All parameters in the autoencoder were optimised to minimise the mean square distance between the input and reconstructed images.
                              </p>
                              <p id="p-52">
                               For all experiments, the autoencoders were trained by minibatch gradient descent using an Adam optimiser with learning rate 10
                               <sup>
                                −4
                               </sup>
                               .
                              </p>
                              <p id="p-53">
                               For unsupervised experiments, training minibatches consisted of 50 images taken at random from the same trajectories used to train the spatial memory pipeline. Training was complete after 200,000 minibatches. We trained a separate autoencoder for each environment (square, circular, and trapezoid cages). Note that the manipulated environments in
                               <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                                Supplementary Fig 5
                               </a>
                               ,
                               <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                Supplementary Fig 7
                               </a>
                               , and
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Supplementary Fig 6
                               </a>
                               , used the same autoencoder as their original, non-manipulated square environment.
                              </p>
                              <p id="p-54">
                               For reinforcement learning experiments the training minibatches consisted of 216 images mixed at random from trajectories generated in the training and testing environments using a rat-like motion model (see below). Training was complete after 200,000 minibatches. A separate autoencoder was trained for the 4-room enclosure represented conceptually (
                               <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                                Fig 4F
                               </a>
                               ), with the same training procedure and architecture.
                              </p>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Rat-like motion model
                              </h4>
                              <p id="p-55">
                               The rat-like motion model used to generate trajectories for unsupervised experiments (and for training the visual autoencoders for reinforcement learning experiments, see above) was based on published work
                               <sup>
                                <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                               </sup>
                               , with parameters specified in Extended Data Table 1. At each time step a linear velocity was generated from a Rayleigh distribution and a rotational velocity from a Gaussian distribution. However, if the trajectory was closer than 3 cm to a wall at an angle narrower than 90°, a deterministic rotation was added that turned it to continue to move parallel to the wall, and the linear velocity was reduced by a factor of 4. Additionally, the trajectory switched between periods of movement (linear velocity Rayleigh parameter 0.13) and periods of pure rotation (linear velocity Rayleigh parameter 0.0). The move and stop periods lasted for an exponentially distributed number of steps with mean 50. The trajectories used to train the spatial memory pipeline were sampled every 7 simulation steps. The angular and linear velocity signals fed to the RNNs in unsupervised experiments were computed dividing the total spatial displacement and angular rotation of the agent over each 7-simulation-step interval by the simulation time of those 7 steps. The corresponding visual embedding came from the image of the environment at the given position and orientation, with the camera 20 cm above and parallel to the ground. Each trajectory consisted of 500 samples, corresponding to 3500 simulation steps.
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               Environment dimensions for unsupervised experiments
                              </h4>
                              <p id="p-56">
                               For the single-environment unsupervised-learning experiments a square enclosure 2.2 m in side was used. The multi-environment experiments, additionally, included a circular enclosure of radius 1.5 m, and an isosceles trapezoidal enclosure with altitude 4.4 m and parallel sides 2.2 m and 0.75 m long. Single-environment experiments were also carried out in the circular enclosure of 1.5 m radius (see
                               <a id="xref-sec-30-6" class="xref-sec" href="#sec-30">
                                Supplementary Results
                               </a>
                               ). The height of the walls in all environments was 0.25 m.
                              </p>
                             </div>
                             <div id="sec-18" class="subsection">
                              <h4>
                               Sigmoid-LSTM and Sigmoid-Vanilla
                              </h4>
                              <p id="p-57">
                               As our aim was to compare the activation of artificial RNNs to firing rate data from biological neurons, we limited the activations of all RNNs to positive values. In order to do this, we modified the commonly used LSTM and Vanilla-RNN cells by substituting their tanh output non-linearity for a sigmoid. We called these cells Sigmoid-LSTM and Sigmoid-Vanilla-RNN respectively. As it learns faster, we used Sigmoid-LSTM in all our experiments; with the exception of the supplementary HD double ring analysis (see
                               <a id="xref-sec-9-11" class="xref-sec" href="#sec-9">
                                Methods
                               </a>
                               ), where we used a Sigmoid-Vanilla-RNN for ease of analysis. In a Vanilla-RNN the mapping from the current state to the next is extremely simple. Namely, the current vector of cell activations,
                               <strong>
                                h
                               </strong>
                               <sub>
                                t
                               </sub>
                               , depends on the previous activations,
                               <strong>
                                h
                               </strong>
                               <sub>
                                t–1
                               </sub>
                               , and inputs,
                               <strong>
                                x
                               </strong>
                               <sub>
                                t
                               </sub>
                               , through equation:
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where σ is the element-wise sigmoid function, and we refer to
                               <strong>
                                W
                               </strong>
                               as the weight matrix of dynamics and
                               <strong>
                                V
                               </strong>
                               as the weight matrix of inputs.
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               Sparse reactivation
                              </h4>
                              <p id="p-58">
                               The inverse-temperature parameter in the reactivation of memory slots, β in
                               <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">
                                equation (2)
                               </a>
                               , is constantly regulated so the entropy of P
                               <sub>
                                react
                               </sub>
                               matches the hyperparameter H
                               <sub>
                                react
                               </sub>
                               (0.5 nats in unsupervised-learning experiments, 1.0 nats in reinforcement learning). In order to do this, β was parameterised as
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and after every trajectory, β
                               <sub>
                                logit
                               </sub>
                               was increased/ decreased by 0.001 when the average entropy of P
                               <sub>
                                react
                               </sub>
                               was lower/higher than H
                               <sub>
                                react
                               </sub>
                               .
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Training details
                              </h4>
                              <p id="p-59">
                               The training parameters for the unsupervised experiments are summarised in Extended Data Table 2. The training was implemented with the TensorFlow platform. We used back-propagation through time (BPTT) with an unroll length of 50 trajectory steps; since trajectories consisted of 500 steps, each trajectory produced 10 BPTT unrolls. Batch size was 32, each batch consisting of unrolls from different trajectories.
                              </p>
                              <p id="p-60">
                               The single-environment experiments used the Adam optimisation algorithm. Different learning rates were used for the memory slot contents and the rest of the network parameters: since the target distribution over slots was very sparse, we could apply a larger learning rate for the memory slot contents.
                              </p>
                              <p id="p-61">
                               For reasons of ease of implementation, the multi-environment experiments used the RMSProp optimisation algorithm. Each of the three environments trained in a separate process, updating the shared parameters synchronously with Distributed TensorFlow. The shared parameters were the weights and biases of the RNNs (both the F
                               <sub>
                                r
                               </sub>
                               prediction RNNs and the G
                               <sub>
                                r
                               </sub>
                               correction RNNs). The memory contents
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and the entropy-scaling variables γ and π
                               <sub>
                                r
                               </sub>
                               were separate for each environment.
                              </p>
                              <p id="p-62">
                               Similarly to previous work
                               <sup>
                                <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                                 19
                                </a>
                               </sup>
                               , dropout was used in the RNN outputs when predicting the memory reactivations.
                              </p>
                             </div>
                             <div id="sec-21" class="subsection">
                              <h4>
                               Assessment of attractor dynamics in the head-direction system
                              </h4>
                              <p id="p-63">
                               For the evaluation of inte-gration dynamics in
                               <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                               the state of RNN-1 was initialised to
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and angular integration (
                               <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">
                                equation 3
                               </a>
                               ) iterated for a thousand steps using the angular velocity
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               rad/step for 250 steps, 0 rad/step for the following 250 steps, and
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               rad/step for the final 500 steps. Visual input was not given to the model.
                               <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                               shows the evolution of the RNN state,
                               <strong>
                                x
                               </strong>
                               <sub>
                                1,t
                               </sub>
                               , ordering its units by the phase of its resultant vector.
                              </p>
                              <p id="p-64">
                               The principal-component space displayed in
                               <a id="xref-fig-3-10" class="xref-fig" href="#F3">
                                Fig 3B
                               </a>
                               was calculated by taking the two principal eigenvectors of the covariance matrix of unit activity in
                               <a id="xref-fig-3-11" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                               . Repeated activity vectors were discarded from the calculation of the mean and covariance matrix.
                              </p>
                              <p id="p-65">
                               For the assessment of fixed attractor points in
                               <a id="xref-fig-3-12" class="xref-fig" href="#F3">
                                Fig 3C–D
                               </a>
                               , the state of each unit in RNN-1 was independently initialised by drawing a sample from a Gaussian with four times the standard deviation of its activations in
                               <a id="xref-fig-3-13" class="xref-fig" href="#F3">
                                Fig 3A
                               </a>
                               . A thousand different random initialisations were used and each run for 1000 steps of angular integration (
                               <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">
                                equation 3
                               </a>
                               ) inputting 0 rad/step angular velocity.
                              </p>
                              <p id="p-66">
                               For the assessment of periodic orbits in
                               <a id="xref-fig-3-14" class="xref-fig" href="#F3">
                                Fig 3E–F
                               </a>
                               , the same one thousand initialisations were used, but each was run for 200 steps of angular integration (
                               <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">
                                equation 3
                               </a>
                               ) using either a clockwise
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               rad/step or counter-clockwise
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               rad/step angular velocity. The 50 first steps of two arbitrary trajectories with opposing velocities were shown in
                               <a id="xref-fig-3-15" class="xref-fig" href="#F3">
                                Fig 3E
                               </a>
                               . Each of the two autocovariance curves in
                               <a id="xref-fig-3-16" class="xref-fig" href="#F3">
                                Fig 3F
                               </a>
                               was calculated using all 1000 trajectories.
                              </p>
                              <p id="p-67">
                               For the assessment of point-attractors under the influence of visual inputs in
                               <a id="xref-fig-3-17" class="xref-fig" href="#F3">
                                Fig 3G–I
                               </a>
                               , we took 512 random images from the training dataset, and run the model for 200 steps of prediction and correction. Each of these 200 steps consisted of a step of visual correction,
                               <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                equations 7
                               </a>
                               and
                               <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                                8
                               </a>
                               , followed by zero angular-velocity integration,
                               <a id="xref-disp-formula-3-4" class="xref-disp-formula" href="#disp-formula-3">
                                equation 3
                               </a>
                               . For every image we run 18 trajectories initialised from each the fixed points found in
                               <a id="xref-fig-3-18" class="xref-fig" href="#F3">
                                Fig 3D
                               </a>
                               . We considered that a single fixed-point had been reached at a particular time step if the maximum distance among the 18 trajectories was less than 0.1 in the PCA space. The bottom plots in
                               <a id="xref-fig-3-19" class="xref-fig" href="#F3">
                                Fig 3H–I
                               </a>
                               show the trajectory of each of these 18 initialisations for thre examples of visual inputs.
                              </p>
                             </div>
                             <div id="sec-22" class="subsection">
                              <h4>
                               Preservation of spatial characteristics across environments
                              </h4>
                              <p id="p-68">
                               We trained the Spatial Memory Pipeline simultaneously in three environments, sharing the RNN parameters across the environments but keeping a separate memory store for each. The separation of memories simulates hippocampal remapping, since there is no correspondence between the slots for the different environments. The RNN units, on the other hand, behaved coherently across the environments, giving further credence to their role as analogues of HD, egoBVC and BVC cells.
                               <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                                Supplementary Fig 9
                               </a>
                               summarises the stability of these representations across the three environments (1, square; 2, circle; 3, trapezoid). The relative preferred angles between HD units in RNN-1 were preserved with great accuracy between environments (Extended Data Fig 9A), i.e., the head direction system rotated as a whole without change in functionality. In contrast, the relative directional tuning of BVC units in RNN-3, although not entirely random (Kuiper test rejected the uniform distribution for the angle differences, p-value< 0.01 for all environment pairs), displayed high variability across units (Extended Data Fig 9C). This means that BVC units in our model, under remapping, are not locked to the head direction system, unlike the case without remapping (see rotation manipulations,
                               <a id="xref-fig-9-4" class="xref-fig" href="#F9">
                                Supplementary Fig 5
                               </a>
                               ). The preferred angles of EgoBVC units in RNN-2 stayed the same in different environments (inter-environment angular differences clustered tightly around zero, Extended Data Fig 9B), as expected for a system of egocentric coordinates. Distance tuning of egoBVCs was very stable across environments (Extended Data Fig 9D), while BVC distance tuning, although significantly preserved (mean across units of the absolute difference between environments significantly smaller than the mean across shuffled pairs, p-value< 10
                               <sup>
                                −4
                               </sup>
                               ), again showed more variability (Extended Data Fig 9E).
                              </p>
                             </div>
                             <div id="sec-23" class="subsection">
                              <h4>
                               Environment for reinforcement learning experiments
                              </h4>
                              <p id="p-69">
                               We assessed the performance of reinforcement learning agents in the DeepMind Laboratory platform
                               <sup>
                                <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                                 58
                                </a>
                               </sup>
                               . The layout of the 4-room enclosures (
                               <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                                Fig 4B
                               </a>
                               ) consisted of four 5 x 5-tile rooms (1.25 x 1.25 m assuming an agent speed of 15 cm/s) linked by four 5 x 1 corridors. The enclosure was surrounded by a sky-box at infinity - so as to provide directional but not distance information - textured with buildings, clouds and trees, which provided distal cues. There were no special markings on the walls or floors of the enclosure, making all rooms and corridors identical except for their relative orientation with respect to the distal cues. At the start of each episode the agent (described below) was placed in a random location and was required to explore in order to find an unmarked goal, paralleling the task of rodents in the classic Morris water maze. When the goal was reached, the agent received a reward of 10 points and was teleported to a random location in the enclosure. The goal remained fixed for the length of the episode, so a well-trained agent would first explore the enclosure in order to discover the goal location and then, after each teleportation, orient itself to return to the same goal as quickly as possible. The length of the episodes was 3000 agent steps (12000 environment frames, see below).
                              </p>
                              <p id="p-70">
                               The agent received observations in the form of camera input (96 x 72 pixels), rotation velocity, and egocentric translation velocity (components parallel and perpendicular to the agent’s orientation).
                              </p>
                              <p id="p-71">
                               The agent could start at any position in the enclosure. The action space was discrete (six actions) but afforded fine-grained motor control (that is, the agent could rotate in small increments, accelerate forwards or backwards, or effect rotational acceleration while moving forwards). Agent actions were repeated for 4 consecutive environment frames
                               <sup>
                                <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                               </sup>
                               ; one actor step consisted therefore of 4 environment steps.
                              </p>
                              <p id="p-72">
                               The visual appearance of the environment was determined by the floor, wall and distal cues. We used 8 different sets of textures, three of them for training and the other five to evaluate transfer. Transfer was evaluated on the same layout as training (
                               <a id="xref-fig-4-10" class="xref-fig" href="#F4">
                                Fig 4C
                               </a>
                               ), on a cross-shaped layout with 5 x 5-tile rooms linked by two 5 x 1 corridors (
                               <a id="xref-fig-4-11" class="xref-fig" href="#F4">
                                Fig 4D
                               </a>
                               ), in the same training layout where in each episode one corridor at random was blocked by a wall (
                               <a id="xref-fig-4-12" class="xref-fig" href="#F4">
                                Fig 4E
                               </a>
                               ), and, finally, on the same training layout provided to the agent not from a first-person camera view but as a top-down schematic representation centered on the agent position and rotated according to the agent’s head direction (
                               <a id="xref-fig-4-13" class="xref-fig" href="#F4">
                                Fig 4F
                               </a>
                               ). In this schematic representation, each of the 4 boundaries of the layout were filled with a different colour in lieu of distal cues. The image size was 96 x 72 pixels, as in the camera-input experiments.
                              </p>
                             </div>
                             <div id="sec-24" class="subsection">
                              <h4>
                               Reinforcement learning agent architecture and training
                              </h4>
                              <p id="p-73">
                               We used importance-weighted actor-learner agent (IMPALA)
                               <sup>
                                <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                               </sup>
                               to learn the task. IMPALA is an actor-critic setup to learn a policy distribution π over the available actions and a baseline function V
                               <sup>
                                π
                               </sup>
                               . Our IMPALA setup consisted of 96 actors, repeatedly generating trajectories of experience, and one learner that used the trajectories sent by actors to learn the policy. At the beginning of each episode actors updated their local policy-network, value-network and Spatial Memory Pipeline parameters to the latest learner parameters and, at the end of each episode, sent the trajectory of observations, rewards, states, and policy distributions to the learner. The learner continuously updated its policy, value and Spatial Memory Pipeline parameters via back-propagation through time (BPTT) on batches of 64 100-step chunks of trajectories collected from different actors. Both the actor-critic parameters and the Spatial Memory Pipeline parameters were optimised simultaneously, but with respect to separate losses. The Spatial Memory Pipeline loss is just as described previously in the unsupervised experiments, and determines the gradients for the Spatial Memory Pipeline parameters. The actor-critic loss is composed of three terms: a policy gradient loss to maximise expected advantage, a baseline value loss to predict the episode returns, and an entropy bonus to prevent premature convergence. The possible policy lag between the actors and the learner was corrected in the policy gradient and baseline losses with V-trace
                               <sup>
                                <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                               </sup>
                               . We used Adam
                               <sup>
                                <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                                 59
                                </a>
                               </sup>
                               , a stochastic gradient descent optimiser, for both the Spatial Memory Pipeline and the actor-critic loss. A schematic of the agent network is displayed in
                               <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                                Supplementary Fig 11A
                               </a>
                               .
                              </p>
                              <div id="F15" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F15.large.jpg?width=800&height=600&carousel=1" title="Reinforcement learning architectures and baselines. A) Agent training with the Spatial Memory Pipeline. The visual inputs from three visually different training environments were stored in separate memory slot banks. The Spatial Memory Pipeline (light blue frame) was trained as in the unsupervised experiments to minimise the loss in prediction of memory reactivations. The policy (purple frame) was trained to minimise a combination of policy gradient, value and entropy losses. B) Agent training baseline with an LSTM replacing the Spatial Memory Pipeline. C) Agent training baseline with one LSTM per training environment and a common LSTM replacing the Spatial Memory Pipeline. D-G) Learning curves in tasks shown in Fig 4C–F. Error regions represent the standard error of the mean episode returns across 5 (D-F) or 1 (G) transfer environment and 25 trained agents (transfer) or 5 seeds (scratch). Red, Spatial Memory Pipeline; black, RL baseline. Dashed, learning from scratch; solid, transferred from trained agent. In D), additionally, transfer curves for the training baseline with one LSTM per training environment (magenta), and for a Spatial Memory Pipeline with correction at every time step (blue)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Reinforcement learning architectures and baselines. <strong>A)</strong> Agent training with the <em>Spatial Memory Pipeline</em>. The visual inputs from three visually different training environments were stored in separate memory slot banks. The <em>Spatial Memory Pipeline</em> (light blue frame) was trained as in the unsupervised experiments to minimise the loss in prediction of memory reactivations. The policy (purple frame) was trained to minimise a combination of policy gradient, value and entropy losses. <strong>B)</strong> Agent training baseline with an LSTM replacing the <em>Spatial Memory Pipeline</em>. <strong>C)</strong> Agent training baseline with one LSTM per training environment and a common LSTM replacing the <em>Spatial Memory Pipeline</em>. <strong>D-G)</strong> Learning curves in tasks shown in Fig 4C–F. Error regions represent the standard error of the mean episode returns across 5 (D-F) or 1 (G) transfer environment and 25 trained agents (transfer) or 5 seeds (scratch). Red, <em>Spatial Memory Pipeline</em>; black, RL baseline. Dashed, learning from scratch; solid, transferred from trained agent. In D), additionally, transfer curves for the training baseline with one LSTM per training environment (magenta), and for a <em>Spatial Memory Pipeline</em> with correction at every time step (blue).</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F15.medium.gif" width="440" height="370"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplementary Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F15.medium.gif" width="440" height="370"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 11:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplementary Figure 11:
                                </span>
                                <p id="p-160" class="first-child">
                                 Reinforcement learning architectures and baselines.
                                 <strong>
                                  A)
                                 </strong>
                                 Agent training with the Spatial Memory Pipeline. The visual inputs from three visually different training environments were stored in separate memory slot banks. The Spatial Memory Pipeline (light blue frame) was trained as in the unsupervised experiments to minimise the loss in prediction of memory reactivations. The policy (purple frame) was trained to minimise a combination of policy gradient, value and entropy losses.
                                 <strong>
                                  B)
                                 </strong>
                                 Agent training baseline with an LSTM replacing the Spatial Memory Pipeline.
                                 <strong>
                                  C)
                                 </strong>
                                 Agent training baseline with one LSTM per training environment and a common LSTM replacing the Spatial Memory Pipeline.
                                 <strong>
                                  D-G)
                                 </strong>
                                 Learning curves in tasks shown in
                                 <a id="xref-fig-4-22" class="xref-fig" href="#F4">
                                  Fig 4C–F
                                 </a>
                                 . Error regions represent the standard error of the mean episode returns across 5 (D-F) or 1 (G) transfer environment and 25 trained agents (transfer) or 5 seeds (scratch). Red, Spatial Memory Pipeline; black, RL baseline. Dashed, learning from scratch; solid, transferred from trained agent. In D), additionally, transfer curves for the training baseline with one LSTM per training environment (magenta), and for a Spatial Memory Pipeline with correction at every time step (blue).
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F16" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F16.large.jpg?width=800&height=600&carousel=1" title="Representations across three reinforcement learning training enclosures, one enclosure in each row. A) Agent view of the enclosures. B) Ratemaps of two egoBVCs in RNN-2, with their egocentric-boundary ratemap (see Supplementary Methods). C) Ratemaps of two BVCs in RNN-3, with their allocentric-boundary ratemap (see Supplementary Methods). D) Polar plots of four HD cells in RNN-1. E) Ratemaps of five place cells in each environment. Note that, unlike the egoBVC, BVC or HD units, there is no relationship between the place cell units across environments, since they correspond to separate memory banks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2053297704" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Representations across three reinforcement learning training enclosures, one enclosure in each row. <strong>A)</strong> Agent view of the enclosures. <strong>B)</strong> Ratemaps of two egoBVCs in RNN-2, with their egocentric-boundary ratemap (see Supplementary Methods). <strong>C)</strong> Ratemaps of two BVCs in RNN-3, with their allocentric-boundary ratemap (see Supplementary Methods). <strong>D)</strong> Polar plots of four HD cells in RNN-1. <strong>E)</strong> Ratemaps of five place cells in each environment. Note that, unlike the egoBVC, BVC or HD units, there is no relationship between the place cell units across environments, since they correspond to separate memory banks.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F16.medium.gif" width="440" height="296"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Supplementary Figure 12:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F16.medium.gif" width="440" height="296"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 12:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/15/2020.11.11.378141/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Supplementary Figure 12:
                                </span>
                                <p id="p-161" class="first-child">
                                 Representations across three reinforcement learning training enclosures, one enclosure in each row.
                                 <strong>
                                  A)
                                 </strong>
                                 Agent view of the enclosures.
                                 <strong>
                                  B)
                                 </strong>
                                 Ratemaps of two egoBVCs in RNN-2, with their egocentric-boundary ratemap (see
                                 <a id="xref-sec-39-13" class="xref-sec" href="#sec-39">
                                  Supplementary Methods
                                 </a>
                                 ).
                                 <strong>
                                  C)
                                 </strong>
                                 Ratemaps of two BVCs in RNN-3, with their allocentric-boundary ratemap (see
                                 <a id="xref-sec-39-14" class="xref-sec" href="#sec-39">
                                  Supplementary Methods
                                 </a>
                                 ).
                                 <strong>
                                  D)
                                 </strong>
                                 Polar plots of four HD cells in RNN-1.
                                 <strong>
                                  E)
                                 </strong>
                                 Ratemaps of five place cells in each environment. Note that, unlike the egoBVC, BVC or HD units, there is no relationship between the place cell units across environments, since they correspond to separate memory banks.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-74">
                               The inputs to the actor-critic network were the previous time-step reward and one-hot encoded action, the concatenation of the current outputs of all the Spatial Memory Pipeline’s RNNs (current allocentric code), and the the concatenation of the outputs of all the Spatial Memory Pipeline’s RNNs observed last time the goal was reached (goal allocentric code, which was set to zero as long as the goal had not been reached in the episode). These inputs were fed to a 256-unit LSTM whose output in turn went through a policy MLP (one 256-unit hidden layer with ReLU activation) to produce the policy, and a value MLP (one 256-unit hidden layer with ReLU activation) to predict the value. Note that the actor-critic network did not receive direct visual input to learn the task - only representations from the Spatial Memory Pipeline.
                              </p>
                              <p id="p-75">
                               The inputs to the Spatial Memory Pipeline, similarly to the unsupervised learning experiments, consisted of egocentric velocities and vision. Because the DeepMind Laboratory agent has inertia, it could shift sideways while moving, even though the action set did not allow strafing. Therefore we provided as inputs not only the translation velocity in the heading direction of the agent, but also the component perpendicular to it. These, together with the sine and cosine of the rotational velocity of the agent, formed the 4-component velocity input vector (as opposed to the 3-component vector used in unsupervised experiments, that lacked a sideways translation component). The visual input was the embedding vector (length 128) of a convolutional autoencoder trained offline to reconstruct images of all the training and transfer environments as explained previously, except the top-down schematic view, for which a separate autoencoder was trained. Compared to the unsupervised experiments, the Spatial Memory Pipeline for the reinforcement learning experiments used a bigger RNN-2 size and more memory slots. The hyperparameters of the architecture are described in Extended Data Tables 4-5, while the hyperparameters for training are described in Extended Data Table 3.
                              </p>
                              <p id="p-76">
                               In the training phase, each actor was assigned one of the 3 visually different training environments. The Spatial Memory Pipeline had correspondingly three sets of memory stores, and experiences coming from each environment were channelled automatically to the corresponding bank. The RNN parameters, in contrast, were shared among all the environments. The agent trained for a combined 2 billion actor steps, then it was tested on the transfer environments. 25 different seeds were used for training, and transfer was evaluated once from each of them.
                              </p>
                              <p id="p-77">
                               We evaluated the trained agents in each of the transfer environments separately. In each transfer experiment all the IMPALA actors experienced only the transfer environment, and memory stores were initialised blank. The memory store contents were overwritten at a quicker pace (probability 0.01 per step) at the beginning of the transfer experiments, until full, then continued to be overwritten at the normal rate (probability 0.0001 per step). The rest of the agent parameters started from their values in the trained agent.
                              </p>
                             </div>
                             <div id="sec-25" class="subsection">
                              <h4>
                               Reinforcement learning baselines
                              </h4>
                              <p id="p-78">
                               To support the hypothesis that it is the Spatial Memory Pipeline representations that allow the agent to transfer the behaviour to a novel environment with little training, we repeated the transfer experiments replacing the Spatial Memory Pipeline with networks of comparable capacity: 1) a generic recurrent network (LSTM) (
                               <a id="xref-fig-15-3" class="xref-fig" href="#F15">
                                Supplementary Fig 11B
                               </a>
                               ) that integrated visual and velocity inputs, trained with the agent policy loss. 2) A two-layer network (
                               <a id="xref-fig-15-4" class="xref-fig" href="#F15">
                                Supplementary Fig 11C
                               </a>
                               ) where the first layer consisted of three LSTMs, each integrating the visual and velocity inputs from one of the three training environments, and the second layer consisted of a single LSTM integrating the batched outputs of the first-layer LSTMs; this network paralleled the Spatial Memory Pipeline architecture, where each training environment had its own separate memory stores. And 3) a Spatial Memory Pipeline identical to the main experiment pipeline, except the visual correction was performed at every time step (P
                               <sub>
                                correction
                               </sub>
                               = 1.0); this Spatial Memory Pipeline did not need to integrate velocities over multiple agent steps in order to predict the slot activations, and consequently did not develop any useful spatial representations. For replacements 2) and 3), transfer was evaluated only on the same layout - but different textures - as training (
                               <a id="xref-fig-15-5" class="xref-fig" href="#F15">
                                Supplementary Fig 11D
                               </a>
                               ). All baselines were trained to the same performance as the Spatial Memory Pipeline agent before transfer. Replacement 1) had somewhat unstable learning in the top-down schematic representation of the environment (
                               <a id="xref-fig-4-14" class="xref-fig" href="#F4">
                                Fig 4F
                               </a>
                               ). A smaller learning rate was used in this case, see Extended Data Table 3, since it removed the instability while preserving the average learning curves.
                              </p>
                              <p id="p-79">
                               The distributions shown in
                               <a id="xref-fig-4-15" class="xref-fig" href="#F4">
                                Fig 4C–F
                               </a>
                               compare the learning time (in millions of actor steps) to reach human performance in each of the 4 transfer settings either training from scratch or transferring from a trained agent, either with the Spatial Memory Pipeline-endowed agent or with baseline network 1). Distributions in
                               <a id="xref-fig-4-16" class="xref-fig" href="#F4">
                                Fig 4C–E
                               </a>
                               consist of 25 points for the scratch box plots (5 environments x 5 seeds) and 125 points for the transfer box plots (5 transfer environments x 25 original seeds), distributions in
                               <a id="xref-fig-4-17" class="xref-fig" href="#F4">
                                Fig 4F
                               </a>
                               consist of 25 points (1 environments x 25 seeds both for scratch and transfer). For each seed, the time to human performance is measured as the steps required for the average episode reward of the actors to surpass the average human performance and remain above it more than 80% of the following training steps. Human performance was evaluated as the average of 5 episodes played by one of the authors after another 5 episodes of getting acquainted with the task. Computing bootstrapped distributions of the median learning time to reach human performance in each condition yield the following means and standard deviations of the ratios between scratch and transfer medians for the transfer settings in
                               <a id="xref-fig-4-18" class="xref-fig" href="#F4">
                                Fig 4C, D, E, F
                               </a>
                               , respectively: for the Spatial Memory Pipeline, 85.6 ± 8.9, 11.4 ± 0.6, 13.7 ± 1.0 and 4.4 ± 0.9. For baseline network 1), 1.7 ± 0.1, 1.5 ± 0.1, 1.3 ± 0.1 and 1.4 ± 0.1.
                              </p>
                              <p id="p-80">
                               <a id="xref-fig-15-6" class="xref-fig" href="#F15">
                                Supplementary Fig 11D–G
                               </a>
                               shows the average learning curves with the standard error of the mean returns across seeds and environments, comparing the Spatial Memory Pipeline-endowed agent with baseline network 1). Additionally,
                               <a id="xref-fig-15-7" class="xref-fig" href="#F15">
                                Supplementary Fig 11D
                               </a>
                               shows learning curves for baselines 2) and 3) as described above.
                              </p>
                              <p id="p-81">
                               The three replacement networks, while able to learn the Morris water-maze task in the training environments, failed to transfer their learned behaviour to visually novel environments (
                               <a id="xref-fig-15-8" class="xref-fig" href="#F15">
                                Supplementary Fig 11D
                               </a>
                               ). After 50 million steps of transfer training in a novel environment, practically all the Spatial Memory Pipeline agents had reached human performance, while none of the agents with generic recurrent networks had, needing a number of training steps for transfer comparable to those needed to learn in the first place (>250 million steps). The Spatial Memory Pipeline that was purposefully trained with continuous vision (P
                               <sub>
                                correction
                               </sub>
                               = 1.0) to disrupt the development of spatial representations took much longer, demonstrating that it is not an architectural bias, but the spatial representations learned that assist in the transfer of behaviour.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-26">
                            <h2 class="">
                             Data and code availability
                            </h2>
                            <p id="p-82">
                             The data and code used to generate
                             <a id="xref-fig-2-13" class="xref-fig" href="#F2">
                              figures 2
                             </a>
                             and
                             <a id="xref-fig-3-20" class="xref-fig" href="#F3">
                              3
                             </a>
                             will be available under the open source Apache 2.0 license at the following URL:
                             <a href="https://github.com/deepmind/deepmind-research/tree/master/spatial_memory_pipeline">
                              https://github.com/deepmind/deepmind-research/tree/master/spatial_memory_pipeline
                             </a>
                             .
                            </p>
                            <p id="p-83">
                             There is Supplemental Information that contains additional results, discussion and methods.
                            </p>
                           </div>
                           <div class="section COI-statement" id="sec-27">
                            <h2 class="">
                             Competing Interests
                            </h2>
                            <p id="p-85">
                             The authors declare that they have no competing financial interests.
                            </p>
                           </div>
                           <div class="section" id="sec-28">
                            <h2 class="">
                             Author Contributions
                            </h2>
                            <p id="p-86">
                             Conceived project: B.U, A.B, B.I, C.Ba, C.Bl, D.K, D.H.
                            </p>
                            <p id="p-87">
                             Contributed ideas to experiments: B.U, C.Ba, B.I, C.Bl, A.B, V.Z, D.K
                            </p>
                            <p id="p-88">
                             Performed experiments and analysis: B.U, B.I, V.Z, A.B
                            </p>
                            <p id="p-89">
                             Development of testing platform and environments: B.I, V.Z, B.U, A.B
                            </p>
                            <p id="p-90">
                             Wrote paper: C.Ba, C.Bl, B.U, B.I, A.B, D.K, D.H
                            </p>
                            <p id="p-91">
                             Managed project:C.Bl, C.Ba, D.H
                            </p>
                           </div>
                           <div class="section" id="sec-29">
                            <h2 class="">
                             Supplemental Information for A model of egocentric to allocentric understanding in mammalian brains
                            </h2>
                            <p>
                             This section contains:
                            </p>
                            <ul class="list-unord" id="list-1">
                             <li id="list-item-1">
                              <p id="p-93">
                               Supplementary Figures
                              </p>
                             </li>
                             <li id="list-item-2">
                              <p>
                               <a id="xref-sec-30-7" class="xref-sec" href="#sec-30">
                                Supplementary Results
                               </a>
                              </p>
                              <ul class="list-simple" id="list-2">
                               <li id="list-item-3">
                                <p id="p-95">
                                 – Emergent representations in a circular environment
                                </p>
                               </li>
                               <li id="list-item-4">
                                <p id="p-96">
                                 – Effects of distal cue rotation on the model’s representations
                                </p>
                               </li>
                               <li id="list-item-5">
                                <p id="p-97">
                                 – Effects of enclosure stretch on the model’s representations
                                </p>
                               </li>
                               <li id="list-item-6">
                                <p id="p-98">
                                 – Effects of barrier insertion on the model’s representations
                                </p>
                               </li>
                               <li id="list-item-7">
                                <p id="p-99">
                                 – Preservation of spatial characteristics across environments
                                </p>
                               </li>
                               <li id="list-item-8">
                                <p id="p-100">
                                 – RNN-1 ring attractor analysis
                                </p>
                               </li>
                              </ul>
                             </li>
                             <li id="list-item-9">
                              <p>
                               <a id="xref-sec-39-5" class="xref-sec" href="#sec-39">
                                Supplementary Methods
                               </a>
                              </p>
                              <ul class="list-simple" id="list-3">
                               <li id="list-item-10">
                                <p id="p-102">
                                 – Decoding of position and heading angle
                                </p>
                               </li>
                               <li id="list-item-11">
                                <p id="p-103">
                                 – Quantitative categorisation of spatial representations
                                </p>
                               </li>
                               <li id="list-item-12">
                                <p id="p-104">
                                 – Spatial ratemaps
                                </p>
                               </li>
                               <li id="list-item-13">
                                <p id="p-105">
                                 – Resultant vector of binned data
                                </p>
                               </li>
                               <li id="list-item-14">
                                <p id="p-106">
                                 – Head-direction cells
                                </p>
                               </li>
                               <li id="list-item-15">
                                <p id="p-107">
                                 – Egocentric boundary score
                                </p>
                               </li>
                               <li id="list-item-16">
                                <p id="p-108">
                                 – Allocentric boundary score
                                </p>
                               </li>
                               <li id="list-item-17">
                                <p id="p-109">
                                 – Spatial stability score
                                </p>
                               </li>
                               <li id="list-item-18">
                                <p id="p-110">
                                 – Place cell field characterisation
                                </p>
                               </li>
                               <li id="list-item-19">
                                <p id="p-111">
                                 – Model selection criterion
                                </p>
                               </li>
                               <li id="list-item-20">
                                <p id="p-112">
                                 – Reinforcement learning baselines
                                </p>
                               </li>
                              </ul>
                             </li>
                            </ul>
                           </div>
                           <div class="section" id="sec-30">
                            <h2 class="">
                             Supplementary Results for A model of egocentric to allocentric understanding in mammalian brains
                            </h2>
                            <div class="section" id="sec-31">
                             <h3>
                              Emergent representations in a circular environment
                             </h3>
                             <p id="p-113">
                              The appearance of spatial representations shown in
                              <a id="xref-fig-2-14" class="xref-fig" href="#F2">
                               Fig 2
                              </a>
                              does not depend on the square geometry of the environment. The emergence of such representations is robust to training in enclosures with different environment shapes. To demonstrate this, the model was trained from scratch in a simulated circular environment of radius 1.5 meters, white walls, no internal cues, and distal cues consisting of a city-like scene. The model developed similar representations (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Supplementary Fig 2
                              </a>
                              ) to those appearing in the square enclosure.
                             </p>
                             <p id="p-114">
                              The majority of units in the first RNN module (RNN-1), which received only angular velocities and corrections from the visual memories, exhibited activity modulated by the agent’s direction of facing (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Supplementary Fig 2A–B
                              </a>
                              ). In total 94% (30/32) of units were classified as head direction cells (resultant vector length >0.48) with unimodal responses distributed uniformly in the unit circle (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Supplementary Fig 2H
                              </a>
                              , uniform distribution was selected under BIC over mixtures of Von Mises).
                             </p>
                             <p id="p-115">
                              The second module (RNN-2) received angular velocity and speed, together with corrections from the visual memories. Units in this RNN encoded distances relative to boundaries at a particular heading (
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Supplementary Fig 2C
                              </a>
                              ). As in the square arena, they resembled egoBVC responses, with 94% (120/128) of units identified as egoBVCs (ego-boundary score >0.1, 99th percentile of bin-shuffled distribution,
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               Supplementary Fig 2I
                              </a>
                              ). None of the 128 units were identified as BVCs. Here cells did not exhibit HD cell-like patterns of activity (mean resultant vector 0.01, 0/128 units classified as HD cells).
                             </p>
                             <p id="p-116">
                              The third (RNN-3) received no self-motion inputs, thus was dependent upon corrections from visual reactivations as its sole input and learning signal. Units in this RNN were characterised by spatially stable responses (
                              <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                               Supplementary Fig 2F
                              </a>
                              ). As in the square enclosure, the activity profile of this module was reminiscent of BVCs (
                              <a id="xref-fig-6-8" class="xref-fig" href="#F6">
                               Supplementary Fig 2D
                              </a>
                              ). 83% (106/128) of units were classified as BVCs (BVC score>0.11, 99th percentile of shuffled distribution) – 16% (20/128) met the criteria for egoBVCs, but 13 of those 20 had a higher BVC score than egoBVC score. Units in this layer were not modulated by heading direction (mean resultant vector 0.11, 0/128 units classified as HD cells).
                             </p>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              Effects of distal cue rotation on the model’s representations
                             </h3>
                             <p id="p-117">
                              First, we rotated the visible distal cues by 45° clockwise (
                              <a id="xref-fig-9-5" class="xref-fig" href="#F9">
                               Supplementary Fig 5
                              </a>
                              ). The preferred orientation of HD units in RNN-1 rotated en masse, tracking the cue rotation the same way that rodent head direction cells are controlled by visual cues (resvec phase rotated by 43° on average, 1° SD) – tuning width was unchanged (average 76.5°, 17.8° SD). The manipulation did not significantly affect egoBVC activity (preferred egocentric directions to wall rotated by 3° on average, 6°SD). The firing correlates of BVCs rotated with the head direction system (35°, 9°SD). The activity field of the place-cell-like memory slots rotated around the centre of the environment (27° on average, 20° SD).
                             </p>
                            </div>
                            <div class="section" id="sec-33">
                             <h3>
                              Effects of enclosure stretch on the model’s representations
                             </h3>
                             <p id="p-118">
                              We transformed the training environment stretching it by 100% along the horizontal axis to form a 4.4 m by 2.2 m rectangle (
                              <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                               Supplementary Fig 6
                              </a>
                              ). Again, the response characteristics of HD cells as well as egoBVCs and BVCs were unchanged, and simply extended along the elongated walls like their biological counterparts
                              <sup>
                               <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                               ,
                               <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                               ,
                               <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              . Place cell responses closely mirrored those observed in rodents
                              <sup>
                               <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                              </sup>
                              , tending to be stretched along the axis of elongation and in some cases becoming bimodal (
                              <a id="xref-fig-10-4" class="xref-fig" href="#F10">
                               Supplementary Fig 6L–O
                              </a>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-34">
                             <h3>
                              Effects of barrier insertion on the model’s representations
                             </h3>
                             <p id="p-119">
                              In a similar fashion, we introduced a barrier into the virtual environment (
                              <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                               Supplementary Fig 7
                              </a>
                              ). The responses of HD cells were largely unaffected, maintaining the same direction of tuning (0° average change in phase, 0.2° SD). Similarly, egoBVCs and BVCs retained their basic firing correlates, responding to the new barrier as they had to the existing walls, resulting in the inception of additional firing fields (
                              <a id="xref-fig-11-4" class="xref-fig" href="#F11">
                               Supplementary Fig 7D,H
                              </a>
                              ). Biological BVCs and egoBVCs are known to respond similarly; indeed, the predictable response of these cells to extra walls is considered to be a diagnostic feature
                              <sup>
                               <a id="xref-ref-5-8" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">
                                6
                               </a>
                              </sup>
                              . Place cell responses were more complex, some – typically those further from the barrier – were unaffected (60%), whereas 22% formed a duplicate field on either side of the barrier (
                              <a id="xref-fig-11-5" class="xref-fig" href="#F11">
                               Supplementary Fig 7L–N
                              </a>
                              ). Notably, similar outcomes have been reported in empirical studies
                              <sup>
                               <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                                60
                               </a>
                               ,
                               <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                              </sup>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-35">
                             <h3>
                              Preservation of spatial characteristics across environments
                             </h3>
                             <p id="p-120">
                              To study the properties of the Spatial Memory Pipeline representations across environments we trained concurrently in three different enclosures (
                              <a id="xref-fig-4-19" class="xref-fig" href="#F4">
                               Fig 4A
                              </a>
                              ). In RNN-1, although the preferred firing directions of HD cells changed between environments, the responses of all units were rotated by a similar amount, maintaining the relative angular tuning between cell pairs (resultant vector phase rotation between environments 1 and 2 was 179° on average, 0.5° SD; between environments 1 and 3, 41° on average, 0.6° SD;
                              <a id="xref-fig-4-20" class="xref-fig" href="#F4">
                               Fig 4A
                              </a>
                              ,
                              <a id="xref-fig-13-3" class="xref-fig" href="#F13">
                               Supplementary Fig 9A
                              </a>
                              ). The preservation of HD-cell characteristics across environments is consistent with the presence of ring-attractor dynamics, as analysed in single-environment experiments (
                              <a id="xref-fig-3-21" class="xref-fig" href="#F3">
                               Fig 3
                              </a>
                              ).
                             </p>
                             <p id="p-121">
                              The ensemble response properties of egoBVCs in RNN-2 were similarly preserved. Between the three different environments individual egoBVCs maintained their distance and directional tuning, without rotation (mean resultant vector phase rotation 2° on average, 25° SD, between environments 1 and 2; 0° on average, 6° SD between 1 and 3,
                              <a id="xref-fig-13-4" class="xref-fig" href="#F13">
                               Supplementary Fig 9B
                              </a>
                              ; mean distance tuning difference 0.07 m between 1 and 2, 0.04 m between 1 and 3, p < 10
                              <sup>
                               −4
                              </sup>
                              compared to differences with shuffled units,
                              <a id="xref-fig-13-5" class="xref-fig" href="#F13">
                               Supplementary Fig 9D
                              </a>
                              ). These representations do not depend on arbitrary distal cues, and are therefore preserved.
                             </p>
                             <p id="p-122">
                              The BVCs in RNN-3 tended also to preserve distance tuning (0.19 m mean difference in distance tuning between environments 1 and 2, 0.26 m between 1 and 3, SD 0.22, p < 10
                              <sup>
                               −4
                              </sup>
                              compared to differences with shuffled units,
                              <a id="xref-fig-13-6" class="xref-fig" href="#F13">
                               Supplementary Fig 9E
                              </a>
                              ), but not directional tuning across environments (phase rotation had SD 69° between environments 1 and 2, and 89° between 1 and 3;
                              <a id="xref-fig-13-7" class="xref-fig" href="#F13">
                               Supplementary Fig 9C
                              </a>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-36">
                             <h3>
                              Robustness to random initialization
                             </h3>
                             <p id="p-123">
                              The representations exemplified in
                              <a id="xref-fig-2-15" class="xref-fig" href="#F2">
                               Fig 2A–E
                              </a>
                              appear independently of the initial values of the parameters of the Spatial Memory Pipeline. In all of 20 experiments run with identical hyperparameters but different random parameter initializations, HDs, egoBVCs and BVCs appeared in numbers similar to those reported in the main text. BVCs always appear exclusively in RNN-3. HDs appear typically in RNN-1 as described (>31 out of 32 units classified as HD in 12 out of 20 experiments), but sometimes they are also found in RNN-2. Specifically, in 7/20 experiments HDs appeared exclusively in RNN-1, as in the main text example; in 5/20 they appeared both in RNN-1 and RNN-2; and in the remaining 8/20 they appeared exclusively in RNN-2. In such cases, all RNN-1 units were classified as egoBVCs. Since angular velocity information is available to RNN-2, it is not unexpected that HDs can be learned there too, and, to the effects of prediction optimization, separating RNN-1 and RNN-2 is unnecessary. Doing so, however, allows us to simplify the analysis of the attractor dynamics of the HD system.
                             </p>
                             <p id="p-124">
                              Similarly, the experiment described in
                              <a id="xref-fig-4-21" class="xref-fig" href="#F4">
                               Fig 4A
                              </a>
                              and
                              <a id="xref-fig-13-8" class="xref-fig" href="#F13">
                               Supplementary Fig 9
                              </a>
                              was typical, but in some of the experiments HDs appeared also in RNN-2. Specifically, across 15 multi-environment experiments (using 5 different random seeds and 3 different RNN learning rates, 10
                              <sup>
                               −3
                              </sup>
                              , 3 . 10
                              <sup>
                               −4
                              </sup>
                              , and 10
                              <sup>
                               −4
                              </sup>
                              ), 8 had HD cells purely in RNN-1, as in the experiment shown, while the other 7 had a significant number of them in RNN-2 instead.
                             </p>
                            </div>
                            <div class="section" id="sec-37">
                             <h3>
                              Single-RNN ablation
                             </h3>
                             <p id="p-125">
                              Splitting the model into three separate RNNs with different velocity inputs is important to obtain all of the representations. If a single RNN with both linear and angular velocity input is used, with the same number of units as the total in our 3-RNN model (288), approximately 1/3 of the units will be HDs and 1/3 will be egoBVCs; the remaining 1/3 do not satisfy any of the classification criteria. Remarkably, no BVCs appear. Conversely, if the single RNN receives no velocity inputs, approximately 60% of the units are classified as BVCs, and nearly 30% of units classify as egoBVCs, but no HDs appear. Thus, having both an RNN with velocity inputs and one without is necessary to produce HDs, egoBVCs and alloBVCs in the model. As noted earlier, the split between RNN-1 and RNN-2 is not strictly necessary but simplifies the analysis of the HD system.
                             </p>
                            </div>
                            <div class="section" id="sec-38">
                             <h3>
                              RNN-1 ring attractor analysis
                             </h3>
                             <p id="p-126">
                              To understand the mechanism by which our model tracks head direction, we repeated the experiment shown in
                              <a id="xref-fig-3-22" class="xref-fig" href="#F3">
                               Fig 3
                              </a>
                              substituting the efficient but complex LSTM integrators with Vanilla-RNNs. In Vanilla-RNNs each activity state is a simple function of the previous state and inputs, thus is amenable to a mechanistic analysis
                              <sup>
                               <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                              </sup>
                              (see
                              <a id="xref-sec-39-6" class="xref-sec" href="#sec-39">
                               Supplementary Methods
                              </a>
                              ). As expected, this new experiment also developed head-direction cells in RNN-1, all 32 units being classified as head direction cells (resultant vector length >0.4, 99-th percentile of shuffled data,
                              <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                               Supplementary Fig 8A
                              </a>
                              ). Like the LSTM-based model, the Vanilla-RNN effectively integrated angular velocity over many time steps (
                              <a id="xref-fig-12-3" class="xref-fig" href="#F12">
                               Supplementary Fig 8B
                              </a>
                              ).
                             </p>
                             <p id="p-127">
                              The simplicity of Vanilla-RNNs allowed us to examine the connectivity that supported angular velocity integration, revealing a striking similarity with that found in the fly
                              <sup>
                               <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                              </sup>
                              and hypothesised for mammals
                              <sup>
                               <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                              </sup>
                              . The weight matrix of dynamics,
                              <strong>
                               W
                              </strong>
                              , when displayed ordered according to each units’ preferred firing direction (
                              <a id="xref-fig-12-4" class="xref-fig" href="#F12">
                               Supplementary Fig 8C
                              </a>
                              ), resembled a circulant matrix, with a diagonal band of excitatory connections surrounded by diagonal bands of inhibition. That is, the connectivity between units forms a ring with local excitation and long-range inhibition. The angular-velocity tuning of the cells, calculated as the arc-tangent of the 2-dimensional vector of control weights,
                              <strong>
                               V
                              </strong>
                              , that receive as input the cosine and sine of the angular velocity, showed a clear split of the units in two groups, 18/32 units being activated by positive angular velocities (ccw-cells) and 14/32 activated by negative angular velocities (cw-cells) (
                              <a id="xref-fig-12-5" class="xref-fig" href="#F12">
                               Supplementary Fig 8D
                              </a>
                              ). Plotting the weight matrix of dynamics separately for each group revealed the mechanism by which angular velocity was integrated (
                              <a id="xref-fig-12-6" class="xref-fig" href="#F12">
                               Supplementary Fig 8E–F
                              </a>
                              ). Each cell preferentially excited other cells offset around the ring in the same direction as their angular-velocity tuning, an asymmetry that became more obvious when the average weights to units with different relative firing directions were calculated (
                              <a id="xref-fig-12-7" class="xref-fig" href="#F12">
                               Supplementary Fig 8G
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-39">
                            <h2 class="">
                             Supplementary Methods for A model of egocentric to allocentric understanding in mammalian brains
                            </h2>
                            <div class="section" id="sec-40">
                             <h3>
                              Decoding of position and heading angle
                             </h3>
                             <p id="p-128">
                              For decoding of position and head direction (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig 1D
                              </a>
                              ) we trained a single-layer MLP (256 hidden units with tanh non-linearity) to predict either the true (x, y) position of the agent in the environment, or the cosine and sine of its true head direction, from either the vector of log probabilities of activation of the visual slots or the concatenation of all the RNN outputs. We used RMSProp as optimiser (learning rate 10
                              <sup>
                               −4
                              </sup>
                              , decay 0.9, no momentum) with square loss. The decoder was trained on 10 million batches of 25 frames sampled at random out of 300,000 frames from the same trajectories used for unsupervised training of the Spatial Memory Pipeline. The decoding error shown in
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig 1D
                              </a>
                              for position is the mean Euclidean distance over the whole training set between the (x, y) output of the decoder and the true position. For head direction, it is the mean absolute difference between the true head direction and the arc-tangent of the cosine and sine prediction of the decoder. Error bars show the standard error of the mean calculated as the standard deviation of 100 bootstrapped means computed from 5000 decoded samples.
                             </p>
                            </div>
                            <div class="section" id="sec-41">
                             <h3>
                              Quantitative categorisation of spatial representations
                             </h3>
                             <p id="p-129">
                              Where possible we have used the same functional characterisations of cells used for rodent data. Resultant vector lengths of allocentric direction-of-facing were used to characterise head-direction cells
                              <sup>
                               <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                                61
                               </a>
                              </sup>
                              . Resultant vector lengths of egocentric-boundary ratemaps were used to characterise egocentric boundary cells (egoBVCs)
                              <sup>
                               <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                                62
                               </a>
                              </sup>
                              . We used the same procedure with allocentric-boundary ratemaps to detect allocentric boundary cells (BVCs). Our results were based on datasets of 800 trajectories each consisting of 500 timesteps.
                             </p>
                             <p id="p-130">
                              Our simulation datasets of 400,000 timesteps correspond to much longer duration than is common in rodent studies. For this reason we avoided using thresholds based on random shifts of the data, as this results in very low thresholds due to a null hypothesis that removes most of the dependency of activations from any environmental features. We used a bin-shuffling procedure that results in much more stringent, and qualitatively pleasing, thresholds. Thresholds were calculated as the 99-th percentile of resultant vector lengths for 1000 bin shuffles for each unit. All units from all RNNs were used to calculate the threshold of each cell category.
                             </p>
                            </div>
                            <div class="section" id="sec-42">
                             <h3>
                              Spatial ratemaps
                             </h3>
                             <p id="p-131">
                              We measured and plotted the dependence of a cell’s activity on the agent’s location by using spatial ratemaps. We partitioned the environment into a grid of 14cm by 14cms square bins aligned with the cardinal directions. The spatial ratemap for a cell is the matrix of its average activity when the agent is located inside each of these bins.
                             </p>
                             <p id="p-132">
                              When we aimed to show the dependence of cell’s activity on both spatial location and head-direction, the per-octant spatial ratemaps were calculated and plotted surrounding the spatial-ratemap. These are eight spatial ratemaps where the data is restricted to times when the agent’s head direction is contained in each of the eight intervals of 45° centered at the cardinal and inter-cardinal directions.
                             </p>
                            </div>
                            <div class="section" id="sec-43">
                             <h3>
                              Resultant vector of binned data
                             </h3>
                             <p id="p-133">
                              Resultant vectors were calculated as a single complex number from mean activities binned by angle:
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where F
                              <sub>
                               b,u
                              </sub>
                              is the average activity of unit u for the b-th bin out of a total of B bins.
                             </p>
                             <p id="p-134">
                              The length of the resultant vector corresponds to the modulus of R
                              <sub>
                               u
                              </sub>
                              and serves as a statistical measure of circular concentration (inverse variance). While the argument of R
                              <sub>
                               u
                              </sub>
                              is a statistical measure of preferred direction of activation.
                             </p>
                            </div>
                            <div class="section" id="sec-44">
                             <h3>
                              Head-direction cells
                             </h3>
                             <p id="p-135">
                              Head direction cells characteristically fire in a small range of allocentric direction. This preference for a single direction can be measured by the length of the resultant vector of activations.
                             </p>
                             <p id="p-136">
                              In order to calculate the resultant vector of each unit, we partitioned allocentric directions into 20 bins. The angular ratemap, F
                              <sub>
                               b,u
                              </sub>
                              , of the u-th unit, with instantaneous activity a
                              <sub>
                               u,t
                              </sub>
                              at time t, was calculated as:
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where α
                              <sub>
                               t
                              </sub>
                              the agent’s direction of facing at time t, and α
                              <sub>
                               t
                              </sub>
                              ∈ b-th bin if
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="inline-formula" id="inline-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-137">
                              The tuning-curve width for units classified as head-direction cells was calculated as
                              <span class="inline-formula" id="inline-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/inline-graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the width of a single bin, times the number of bins where the mean activity was greater than 0.5 of the greatest mean-bin activity for that unit.
                             </p>
                            </div>
                            <div class="section" id="sec-45">
                             <h3>
                              Egocentric boundary score
                             </h3>
                             <p id="p-138">
                              Egocentric boundary cells were characterised by the resultant vector of a the egocentric-boundary ratemap (EBR)
                              <sup>
                               <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">
                                62
                               </a>
                              </sup>
                              . The EBR uses the agent’s position and direction of facing as frame of reference and computes the average instantaneous activity when a boundary is present at a particular distance and egocentric direction. In our unsupervised experiments the EBR was calculated for bins of 4° by 2.5cm. The maximum distance considered was half of the maximum distance between opposing walls. Examples of EBRs can be seen in the middle row of
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Supplementary Fig 3A
                              </a>
                              and
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               D
                              </a>
                              .
                             </p>
                             <p id="p-139">
                              The length of resultant vector of the EBR (averaged over distance) was used to detect egocentric boundary cells. For a cell classified as an egoBVC, the angle of the resultant vector was taken as its preferred direction and the distance of the maximum activity along this direction in the EBR as its preferred distance.
                             </p>
                            </div>
                            <div class="section" id="sec-46">
                             <h3>
                              Allocentric boundary score
                             </h3>
                             <p id="p-140">
                              We characterised allocentric boundary cells using an allocentric-boundary ratemap (ABR). In the ABR the agent’s position is used as frame of reference but not its direction of facing, the relative angle to boundaries is locked to the allocentric north direction. The ABR computes the average instantaneous activity where a boundary is present at a particular distance and allocentric direction. In our unsupervised experiments the ABR was calculated for bins of 4° by 2.5cm. The maximum distance considered was half of the maximum distance between opposing walls. Examples of ABRs can be seen in the bottom row of
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Supplementary Fig 3A
                              </a>
                              and
                              <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                               D
                              </a>
                             </p>
                             <p id="p-141">
                              The length of resultant vector of the ABR (averaged over distance) was used to detect allocentric boundary cells (BVC). For a cell classified as a BVC, the angle of the resultant vector was taken as its preferred direction and the distance of the maximum activity along this direction in the ABR as its preferred distance.
                             </p>
                            </div>
                            <div class="section" id="sec-47">
                             <h3>
                              Spatial stability score
                             </h3>
                             <p id="p-142">
                              Where reported, the spatial stability of a unit was measured as the Pearson correlation of its spatial ratemap for two halves of the data. In our experiments each half of the data was comprised of 200,000 data points, which under the motion model used would amount to 3000 seconds for each half.
                             </p>
                            </div>
                            <div class="section" id="sec-48">
                             <h3>
                              Place cell field characterisation
                             </h3>
                             <p id="p-143">
                              The activity field of a cell was calculated using the following procedure on its spatial ratemap: (1) a Gaussian filter of radius 1 bin was applied, (2) bins with value higher than half the maximum value were considered part of the activity fields, (3) fields were segmented using the
                              <kbd>
                               skimage.measure.label
                              </kbd>
                              function, (4) fields smaller than 3 adjacent bins were discarded, and (5) the attributes of each field (position, size, and eccentricity) were calculated using the
                              <kbd>
                               skimage.measure.regionprops
                              </kbd>
                              function with the original ratemap as
                              <kbd>
                               intensityimage
                              </kbd>
                              parameter. We used version 0.14.0 of
                              <kbd>
                               skimage
                              </kbd>
                              throughout.
                             </p>
                            </div>
                            <div class="section" id="sec-49">
                             <h3>
                              Model selection criterion
                             </h3>
                             <p id="p-144">
                              Where a probability distribution is reported as fitting data (e.g. the distribution of HD angles, or the angular preferences of egoBVCs) we used the Bayesian information criterion to compare several alternative distributions. This criterion penalises the log-likelihood of the model, L, by a term that depends on the number of parameters, k, and number of data points, n:
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-145">
                              A lower BIC signals a better model fit.
                             </p>
                            </div>
                            <div class="section" id="sec-50">
                             <h3>
                              Reinforcement learning baselines
                             </h3>
                             <p>
                              <a id="xref-fig-15-9" class="xref-fig" href="#F15">
                               Supplementary Fig 11A
                              </a>
                              depicts the architecture of our agent as described in Methods. The baselines we used to compare performance in the water maze transfer tasks are the following:
                             </p>
                             <ul class="list-unord" id="list-4">
                              <li id="list-item-21">
                               <p id="p-147">
                                An agent with the same architecture, receiving visual corrections at every time step (instead of every 10 steps on average). This is the baseline labelled SMP transfer no integration in
                                <a id="xref-fig-15-10" class="xref-fig" href="#F15">
                                 Supplementary Fig 11D
                                </a>
                                .
                               </p>
                              </li>
                              <li id="list-item-22">
                               <p id="p-148">
                                An agent where the Spatial Memory Pipeline was replaced by a generic recurrent network (LSTM) (
                                <a id="xref-fig-15-11" class="xref-fig" href="#F15">
                                 Supplementary Fig 11B
                                </a>
                                ) integrating visual and velocity inputs. The size of the LSTM output was 928 units, to make it the same as the concatenation of all the RNN outputs in the Spatial Memory Pipeline. Unlike the memory pipeline, the LSTM did not have a separate unsupervised loss for training, but was trained directly from the gradients of the policy loss.
                               </p>
                              </li>
                              <li id="list-item-23">
                               <p id="p-149">
                                An agent where the Spatial Memory Pipeline was replaced by a two-layer network of LSTMs (
                                <a id="xref-fig-15-12" class="xref-fig" href="#F15">
                                 Supplementary Fig 11C
                                </a>
                                ), where the first layer consisted of three LSTMs (output size 512 units), each integrating the visual and velocity inputs from one of the three training environments, and the second layer consisted of a single LSTM (output size 928 units) integrating the batched outputs of the first-layer LSTMs. This network paralleled the Spatial Memory Pipeline architecture, with the first-layer LSTMs playing the role of the per-environment memory banks and the second-layer LSTM the role of the memory pipeline RNNs. For transfer, the first layer was replaced by a single 512-unit LSTM, and the rest of parameters were kept from the trained agent. As with the single-layer LSTM baseline, the LSTMs did not have a separate unsupervised loss for training, and were trained directly from the gradients of the policy loss.
                               </p>
                              </li>
                             </ul>
                            </div>
                           </div>
                           <div id="sec-51" class="subsection display-objects">
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2451151/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2451151/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2451151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 1:
                              </span>
                              <p id="p-162" class="first-child">
                               Hyperparameters of rat-like motion model
                               <sup>
                                <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">
                                 32
                                </a>
                               </sup>
                               to generate trajectories for unsupervised experiments.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T2" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2451149/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2451149/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2451149" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 2:
                              </span>
                              <p id="p-163" class="first-child">
                               Hyperparameters for training in unsupervised experiments.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T3" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2451141/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2451141/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2451141" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 3:
                              </span>
                              <p id="p-164" class="first-child">
                               Hyperparameters for training in reinforcement learning experiments. The learning rate for the actor-critic loss was 1.10
                               <sup>
                                −4
                               </sup>
                               for all experiments except the top-down schematic representation with RL-baseline, where a slower learning rate of 5.10
                               <sup>
                                −5
                               </sup>
                               was needed to curb instability.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T4" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2451144/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2451144/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2451144" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 4:
                              </span>
                              <p id="p-165" class="first-child">
                               Configuration of the visual encoding level of the Spatial Memory Pipeline in unsupervised, and reinforcement learning experiments.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T5" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2451158/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2451158/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2451158" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 5:
                              </span>
                              <p id="p-166" class="first-child">
                               Configuration of the first integration level of the Spatial Memory Pipeline in unsupervised and reinforcement learning experiments. Where
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/T5/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/15/2020.11.11.378141/T5/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the ofuttphuet visual encoder, ω
                               <sub>
                                t
                               </sub>
                               is the angular velocity, s
                               <sub>
                                t
                               </sub>
                               speed parallel to the direction of heading, and (only for reinforcement learning experiments) s
                               <sub>
                                ⊥,t
                               </sub>
                               the speed perpendicular to the direction of heading.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-84">
                             We thank Matt Botvinick, Vivek Jayaraman, Tim Behrens, Daan Wierstra, Sergei Lebedev, Christopher Summerfield, Kimberly Stachenfeld, and Charlie Beattie for their valuable advice.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-5">
                               New reinforcement learning transfer results.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2>
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.1" data-doi="10.1037/h0061626">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tolman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cognitive maps in rats and men
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Psychological review
                                 </abbr>
                                 <span class="cit-vol">
                                  55
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  189
                                 </span>
                                 <span class="cit-pub-date">
                                  (1948)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPsychological%2Breview%26rft.stitle%253DPsychol%2BRev%26rft.aulast%253DTolman%26rft.auinit1%253DE.%2BC.%26rft.volume%253D55%26rft.issue%253D4%26rft.spage%253D189%26rft.epage%253D208%26rft.atitle%253DCognitive%2Bmaps%2Bin%2Brats%2Band%2Bmen.%26rft_id%253Dinfo%253Adoi%252F10.1037%252Fh0061626%26rft_id%253Dinfo%253Apmid%252F18870876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1037/h0061626&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18870876&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Keefe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dostrovsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The hippocampus as a spatial map: Preliminary evidence from unit activity in the freely-moving rat
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brain research
                                 </abbr>
                                 <span class="cit-pub-date">
                                  (1971)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.3" data-doi="10.1038/nature03721">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafting
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fyhn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Molden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-B
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. I
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Microstructure of a spatial map in the entorhinal cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  436
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  801
                                 </span>
                                 <span class="cit-pub-date">
                                  (2005)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHafting%26rft.auinit1%253DT.%26rft.volume%253D436%26rft.issue%253D7052%26rft.spage%253D801%26rft.epage%253D806%26rft.atitle%253DMicrostructure%2Bof%2Ba%2Bspatial%2Bmap%2Bin%2Bthe%2Bentorhinal%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03721%26rft_id%253Dinfo%253Apmid%252F15965463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature03721&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15965463&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000231116500035&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.4" data-doi="10.1523/JNEUROSCI.10-02-00420.1990">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taube
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. U
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ranck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Head-direction cells recorded from the postsubiculum in freely moving rats. i. description and quantitative analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  420
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  435
                                 </span>
                                 <span class="cit-pub-date">
                                  (1990)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DTaube%26rft.auinit1%253DJ.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D420%26rft.epage%253D435%26rft.atitle%253DHead-direction%2Bcells%2Brecorded%2Bfrom%2Bthe%2Bpostsubiculum%2Bin%2Bfreely%2Bmoving%2Brats.%2BI.%2BDescription%2Band%2Bquantitative%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.10-02-00420.1990%26rft_id%253Dinfo%253Apmid%252F2303851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjg6IjEwLzIvNDIwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTUvMjAyMC4xMS4xMS4zNzgxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.5" data-doi="10.1126/science.1166466">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Solstad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boccara
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kropff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-B
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. I
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Representation of geometric borders in the entorhinal cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  322
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1865
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1868
                                 </span>
                                 <span class="cit-pub-date">
                                  (2008)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSolstad%26rft.auinit1%253DT.%26rft.volume%253D322%26rft.issue%253D5909%26rft.spage%253D1865%26rft.epage%253D1868%26rft.atitle%253DRepresentation%2Bof%2BGeometric%2BBorders%2Bin%2Bthe%2BEntorhinal%2BCortex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166466%26rft_id%253Dinfo%253Apmid%252F19095945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjIvNTkwOS8xODY1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTUvMjAyMC4xMS4xMS4zNzgxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.6" data-doi="10.1523/JNEUROSCI.1319-09.2009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lever
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeewajee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Keefe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Boundary vector cells in the subiculum of the hippocampal formation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  9771
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  9777
                                 </span>
                                 <span class="cit-pub-date">
                                  (2009)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DLever%26rft.auinit1%253DC.%26rft.volume%253D29%26rft.issue%253D31%26rft.spage%253D9771%26rft.epage%253D9777%26rft.atitle%253DBoundary%2BVector%2BCells%2Bin%2Bthe%2BSubiculum%2Bof%2Bthe%2BHippocampal%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.1319-09.2009%26rft_id%253Dinfo%253Apmid%252F19657030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8zMS85NzcxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTUvMjAyMC4xMS4xMS4zNzgxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.7" data-doi="10.1038/nn.4650">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stachenfeld
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Botvinick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. M
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gershman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The hippocampus as a predictive map
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1643
                                 </span>
                                 <span class="cit-pub-date">
                                  (2017)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D20%26rft.spage%253D1643%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4650%26rft_id%253Dinfo%253Apmid%252F28967910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.4650&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28967910&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.8" data-doi="10.1523/JNEUROSCI.2618-15.2015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schiller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Memory and space: towards an understanding of the cognitive map
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  13904
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13911
                                 </span>
                                 <span class="cit-pub-date">
                                  (2015)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DSchiller%26rft.auinit1%253DD.%26rft.volume%253D35%26rft.issue%253D41%26rft.spage%253D13904%26rft.epage%253D13911%26rft.atitle%253DMemory%2Band%2BSpace%253A%2BTowards%2Ban%2BUnderstanding%2Bof%2Bthe%2BCognitive%2BMap%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.2618-15.2015%26rft_id%253Dinfo%253Apmid%252F26468191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjExOiIzNS80MS8xMzkwNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE1LzIwMjAuMTEuMTEuMzc4MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.9" data-doi="10.1101/cshperspect.a021808">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rowland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. C
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. I
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Place cells, grid cells, and memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cold Spring Harbor perspectives in biology
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  a021808
                                 </span>
                                 <span class="cit-pub-date">
                                  (2015)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DMoser%26rft.auinit1%253DM.-B.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253Da021808%26rft.epage%253Da021808%26rft.atitle%253DPlace%2BCells%252C%2BGrid%2BCells%252C%2Band%2BMemory%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a021808%26rft_id%253Dinfo%253Apmid%252F25646382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI3LzIvYTAyMTgwOCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE1LzIwMjAuMTEuMTEuMzc4MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.10" data-doi="10.1037/0735-7044.99.5.1006">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zipser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A computational model of hippocampal place fields
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Behavioral neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  99
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1006
                                 </span>
                                 <span class="cit-pub-date">
                                  (1985)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBehavioral%2Bneuroscience%26rft.stitle%253DBehav%2BNeurosci%26rft.aulast%253DZipser%26rft.auinit1%253DD.%26rft.volume%253D99%26rft.issue%253D5%26rft.spage%253D1006%26rft.epage%253D1018%26rft.atitle%253DA%2Bcomputational%2Bmodel%2Bof%2Bhippocampal%2Bplace%2Bfields.%26rft_id%253Dinfo%253Adoi%252F10.1037%252F0735-7044.99.5.1006%26rft_id%253Dinfo%253Apmid%252F3843299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1037/0735-7044.99.5.1006&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=3843299&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1985ASJ7900018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.11" data-doi="10.1002/hipo.450010303">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Computer simulation of hippocampal place cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Psychobiology
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  103
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  115
                                 </span>
                                 <span class="cit-pub-date">
                                  (1991)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPsychobiology%26rft.volume%253D19%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhipo.450010303%26rft_id%253Dinfo%253Apmid%252F1669295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/hipo.450010303&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1669295&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991FQ33000001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fuhs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. C
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Touretzky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Synaptic learning models of map separation in the hippocampus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neurocomputing
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  379
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  384
                                 </span>
                                 <span class="cit-pub-date">
                                  (2000)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeurocomputing%26rft.volume%253D32%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Franzius
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sprekeler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiskott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Slowness and sparseness lead to place, head-direction, and spatial-view cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e166
                                 </span>
                                 <span class="cit-pub-date">
                                  (2007)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D3%26rft.spage%253D166e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.14" data-doi="10.1037/0033-295X.114.2.340">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Byrne
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Remembering the past and imagining the future: a neural model of spatial memory and imagery
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Psychological review
                                 </abbr>
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  340
                                 </span>
                                 <span class="cit-pub-date">
                                  (2007)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPsychological%2Breview%26rft.stitle%253DPsychol%2BRev%26rft.aulast%253DByrne%26rft.auinit1%253DP.%26rft.volume%253D114%26rft.issue%253D2%26rft.spage%253D340%26rft.epage%253D375%26rft.atitle%253DRemembering%2Bthe%2Bpast%2Band%2Bimagining%2Bthe%2Bfuture%253A%2Ba%2Bneural%2Bmodel%2Bof%2Bspatial%2Bmemory%2Band%2Bimagery.%26rft_id%253Dinfo%253Adoi%252F10.1037%252F0033-295X.114.2.340%26rft_id%253Dinfo%253Apmid%252F17500630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1037/0033-295X.114.2.340&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17500630&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000245764200005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.15" data-doi="10.1126/science.aau4940">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Egocentric coding of external items in the lateral entorhinal cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  362
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  945
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  949
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D362%26rft.issue%253D6417%26rft.spage%253D945%26rft.epage%253D949%26rft.atitle%253DEgocentric%2Bcoding%2Bof%2Bexternal%2Bitems%2Bin%2Bthe%2Blateral%2Bentorhinal%2Bcortex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aau4940%26rft_id%253Dinfo%253Apmid%252F30467169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjIvNjQxNy85NDUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNS8yMDIwLjExLjExLjM3ODE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chapman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. W
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hasselmo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Neuronal representation of environmental boundaries in egocentric coordinates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2772
                                 </span>
                                 <span class="cit-pub-date">
                                  (2019)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D10%26rft.spage%253D2772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.17" data-doi="10.1126/sciadv.aaz2322">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Egocentric boundary vector tuning of the retrosplenial cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science Advances
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  eaaz2322
                                 </span>
                                 <span class="cit-pub-date">
                                  (2020)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DAlexander%26rft.auinit1%253DA.%2BS.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253Deaaz2322%26rft.epage%253Deaaz2322%26rft.atitle%253DEgocentric%2Bboundary%2Bvector%2Btuning%2Bof%2Bthe%2Bretrosplenial%2Bcortex%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aaz2322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNi84L2VhYXoyMzIyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTUvMjAyMC4xMS4xMS4zNzgxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.18" data-doi="10.7554/eLife.10094">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dordek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Soudry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Derdikman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Extracting grid cell characteristics from place cell inputs using non-negative principal component analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e10094
                                 </span>
                                 <span class="cit-pub-date">
                                  (2016)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De10094%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.10094%26rft_id%253Dinfo%253Apmid%252F26952211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.10094&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26952211&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.19" data-doi="10.1038/s41586-018-0102-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Banino
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Vector-based navigation using grid-like representations in artificial agents
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  557
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  429
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D557%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0102-6%26rft_id%253Dinfo%253Apmid%252F29743670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0102-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29743670&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cueva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.-X
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Emergence of grid-like representations by training recurrent neural networks to perform spatial localization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1803.07770
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.21" data-doi="10.1016/j.neuron.2014.04.013">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brandon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koenig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leutgeb
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. K
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leutgeb
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  New and distinct hippocampal place codes are generated in a new environment during septal inactivation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  82
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  789
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  796
                                 </span>
                                 <span class="cit-pub-date">
                                  (2014)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D82%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2014.04.013%26rft_id%253Dinfo%253Apmid%252F24853939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2014.04.013&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24853939&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000336379800010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wills
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cacucci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The development of spatial and memory circuits in the rat
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Wiley Interdisciplinary Reviews: Cognitive Science
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1424
                                 </span>
                                 <span class="cit-pub-date">
                                  (2017)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%253A%2BCognitive%2BScience%26rft.volume%253D8%26rft.spage%253D1424e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.23" data-doi="10.1038/nn.4244">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. L
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   DiCarlo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Using goal-driven deep learning models to understand sensory cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  356
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  365
                                 </span>
                                 <span class="cit-pub-date">
                                  (2016)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D19%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4244%26rft_id%253Dinfo%253Apmid%252F26906502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.4244&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26906502&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhuang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Unsupervised neural network models of the ventral visual stream
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 <span class="cit-pub-date">
                                  (2021)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.25" data-doi="10.1016/j.cell.2020.10.024">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whittington
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The tolman-eichenbaum machine: Unifying space and relational memory through generalization in the hippocampal formation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  183
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1263
                                 </span>
                                 <span class="cit-pub-date">
                                  (2020)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D183%26rft.spage%253D1249%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2020.10.024%26rft_id%253Dinfo%253Apmid%252F33181068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2020.10.024&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33181068&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   George
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Clone-structured graph representations enable flexible learning and vicarious evaluation of cognitive maps
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  17
                                 </span>
                                 <span class="cit-pub-date">
                                  (2021)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D12%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.27" data-doi="10.1016/j.neuron.2004.08.028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eichenbaum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Hippocampus: cognitive processes and neural representations that underlie declarative memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuron
                                 </abbr>
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  109
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  120
                                 </span>
                                 <span class="cit-pub-date">
                                  (2004)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DEichenbaum%26rft.auinit1%253DH.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D120%26rft.atitle%253DHippocampus%253A%2Bcognitive%2Bprocesses%2Band%2Bneural%2Brepresentations%2Bthat%2Bunderlie%2Bdeclarative%2Bmemory.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2004.08.028%26rft_id%253Dinfo%253Apmid%252F15450164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuron.2004.08.028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15450164&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000224232600011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.11.378141v2.28" data-doi="10.1371/journal.pbio.0040424">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kumaran
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maguire
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  An unexpected sequence of events: mismatch detection in the human hippocampus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS biology
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e424
                                 </span>
                                 <span class="cit-pub-date">
                                  (2006)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DKumaran%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De424%26rft.epage%253De424%26rft.atitle%253DAn%2Bunexpected%2Bsequence%2Bof%2Bevents%253A%2Bmismatch%2Bdetection%2Bin%2Bthe%2Bhuman%2Bhippocampus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040424%26rft_id%253Dinfo%253Apmid%252F17132050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040424&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17132050&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.29" data-doi="10.1098/rstb.2008.0316">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lisman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Redish
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. D
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Prediction, sequences and the hippocampus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Philosophical Transactions of the Royal Society B: Biological Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  364
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1193
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1201
                                 </span>
                                 <span class="cit-pub-date">
                                  (2009)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DLisman%26rft.auinit1%253DJ.%26rft.volume%253D364%26rft.issue%253D1521%26rft.spage%253D1193%26rft.epage%253D1201%26rft.atitle%253DPrediction%252C%2Bsequences%2Band%2Bthe%2Bhippocampus%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2008.0316%26rft_id%253Dinfo%253Apmid%252F19528000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rstb.2008.0316&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19528000&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.30" data-doi="10.1038/4580">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. P
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ballard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. H
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Predictive coding in the visual cortex: a functional interpretation of some extra-classical receptive-field effects
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  79
                                 </span>
                                 <span class="cit-pub-date">
                                  (1999)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DRao%26rft.auinit1%253DR.%2BP.%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D87%26rft.atitle%253DPredictive%2Bcoding%2Bin%2Bthe%2Bvisual%2Bcortex%253A%2Ba%2Bfunctional%2Binterpretation%2Bof%2Bsome%2Bextra-classical%2Breceptive-field%2Beffects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F4580%26rft_id%253Dinfo%253Apmid%252F10195184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/4580&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10195184&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000078409300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.31" data-doi="10.1098/rstb.2008.0300">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friston
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kiebel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Predictive coding under the free-energy principle
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Philosophical Transactions of the Royal Society B: Biological Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  364
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1211
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1221
                                 </span>
                                 <span class="cit-pub-date">
                                  (2009)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DFriston%26rft.auinit1%253DK.%26rft.volume%253D364%26rft.issue%253D1521%26rft.spage%253D1211%26rft.epage%253D1221%26rft.atitle%253DPredictive%2Bcoding%2Bunder%2Bthe%2Bfree-energy%2Bprinciple%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2008.0300%26rft_id%253Dinfo%253Apmid%252F19528002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rstb.2008.0300&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19528002&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Raudies
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hasselmo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Modeling boundary vector cell firing given optic flow as a cue
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS computational biology
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e1002553
                                 </span>
                                 <span class="cit-pub-date">
                                  (2012)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2Bcomputational%2Bbiology%26rft.volume%253D8%26rft.spage%253De1002553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ranck
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Head direction cells in the deep cell layer of dorsolateral pre-subiculum in freely moving rats
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Electrical activity of the archicortex
                                 </abbr>
                                 <span class="cit-pub-date">
                                  (1985)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.34" data-doi="10.7554/eLife.33752">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bicanski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A neural-level model of spatial memory and imagery
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ELife
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e33752
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DELife%26rft.volume%253D7%26rft.spage%253De33752%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.33752%26rft_id%253Dinfo%253Apmid%252F30176988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.33752&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30176988&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.35" data-doi="10.1038/nature12153">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wallace
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Rats maintain an overhead binocular field at the expense of constant fusion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  498
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  65
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  69
                                 </span>
                                 <span class="cit-pub-date">
                                  (2013)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D498%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12153%26rft_id%253Dinfo%253Apmid%252F23708965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12153&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23708965&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000319947800033&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.36" data-doi="10.1002/1098-1063(2000)10:4<369::AID-HIPO3>3.0.CO;2-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lever
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cacucci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Keefe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Modeling place fields in terms of the cortical inputs to the hippocampus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Hippocampus
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  369
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  379
                                 </span>
                                 <span class="cit-pub-date">
                                  (2000)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHippocampus%26rft.stitle%253DHippocampus%26rft.aulast%253DHartley%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D369%26rft.epage%253D379%26rft.atitle%253DModeling%2Bplace%2Bfields%2Bin%2Bterms%2Bof%2Bthe%2Bcortical%2Binputs%2Bto%2Bthe%2Bhippocampus.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1098-1063%25282000%252910%253A4%253C369%253A%253AAID-HIPO3%253E3.0.CO%253B2-0%26rft_id%253Dinfo%253Apmid%252F10985276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/1098-1063(2000)10:4<369::AID-HIPO3>3.0.CO;2-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10985276&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000089292000003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.37" data-doi="10.1515/REVNEURO.2006.17.1-2.71">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The boundary vector cell model of place cell firing and spatial memory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Reviews in the Neurosciences
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  71
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 <span class="cit-pub-date">
                                  (2006)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DReviews%2Bin%2Bthe%2Bneurosciences%26rft.stitle%253DRev%2BNeurosci%26rft.aulast%253DBarry%26rft.auinit1%253DC.%26rft.volume%253D17%26rft.issue%253D1-2%26rft.spage%253D71%26rft.epage%253D97%26rft.atitle%253DThe%2Bboundary%2Bvector%2Bcell%2Bmodel%2Bof%2Bplace%2Bcell%2Bfiring%2Band%2Bspatial%2Bmemory.%26rft_id%253Dinfo%253Adoi%252F10.1515%252FREVNEURO.2006.17.1-2.71%26rft_id%253Dinfo%253Apmid%252F16703944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1515/REVNEURO.2006.17.1-2.71&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16703944&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000237307800007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.38" data-doi="10.1038/381425a0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   O’Keefe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Geometric determinants of the place fields of hippocampal neurons
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  381
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  425
                                 </span>
                                 <span class="cit-pub-date">
                                  (1996)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DO%2527Keefe%26rft.auinit1%253DJ.%26rft.volume%253D381%26rft.issue%253D6581%26rft.spage%253D425%26rft.epage%253D428%26rft.atitle%253DGeometric%2Bdeterminants%2Bof%2Bthe%2Bplace%2Bfields%2Bof%2Bhippocampal%2Bneurons.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F381425a0%26rft_id%253Dinfo%253Apmid%252F8632799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/381425a0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8632799&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996UN47900052&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.39" data-doi="10.1038/nn1905">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hayman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeffery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Experience-dependent rescaling of entorhinal grids
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  682
                                 </span>
                                 <span class="cit-pub-date">
                                  (2007)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DBarry%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D682%26rft.epage%253D684%26rft.atitle%253DExperience-dependent%2Brescaling%2Bof%2Bentorhinal%2Bgrids.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn1905%26rft_id%253Dinfo%253Apmid%252F17486102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn1905&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17486102&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000246799800008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.40" data-doi="10.1523/JNEUROSCI.16-06-02112.1996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Representation of spatial orientation by the intrinsic dynamics of the head-direction cell ensemble: a theory
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2112
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2126
                                 </span>
                                 <span class="cit-pub-date">
                                  (1996)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DZhang%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D2112%26rft.epage%253D2126%26rft.atitle%253DRepresentation%2Bof%2Bspatial%2Borientation%2Bby%2Bthe%2Bintrinsic%2Bdynamics%2Bof%2Bthe%2Bhead-direction%2Bcell%2Bensemble%253A%2Ba%2Btheory%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.16-06-02112.1996%26rft_id%253Dinfo%253Apmid%252F8604055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjk6IjE2LzYvMjExMiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzE1LzIwMjAuMTEuMTEuMzc4MTQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.41" data-doi="10.1038/nature14446">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seelig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. D
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayaraman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Neural dynamics for landmark orientation and angular path integration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  521
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  186
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  191
                                 </span>
                                 <span class="cit-pub-date">
                                  (2015)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D521%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14446%26rft_id%253Dinfo%253Apmid%252F25971509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature14446&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25971509&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cueva
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.-X
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Emergence of functional and structural properties of the head direction system by optimization of recurrent neural networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1912.10189
                                 </span>
                                 <span class="cit-pub-date">
                                  (2019)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Redish
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elga
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. N
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Touretzky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A coupled attractor model of the rodent head direction system
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Network: Computation in Neural Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  671
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  685
                                 </span>
                                 <span class="cit-pub-date">
                                  (1996)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNetwork%253A%2BComputation%2Bin%2BNeural%2BSystems%26rft.volume%253D7%26rft.spage%253D671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Espeholt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Impala: Scalable distributed deep-rl with importance weighted actor-learner architectures
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                 <span class="cit-fpage">
                                  1802.01561
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.45" data-doi="10.1016/0023-9690(81)90020-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Spatial localization does not require the presence of local cues
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Learning and motivation
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  239
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  260
                                 </span>
                                 <span class="cit-pub-date">
                                  (1981)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLearning%2Band%2Bmotivation%26rft.volume%253D12%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0023-9690%252881%252990020-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0023-9690(81)90020-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1981LS31700006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.46" data-doi="10.7554/eLife.33752">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bicanski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burgess
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A neural-level model of spatial memory and imagery
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ELife
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e33752
                                 </span>
                                 <span class="cit-pub-date">
                                  (2018)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DELife%26rft.volume%253D7%26rft.spage%253De33752%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.33752%26rft_id%253Dinfo%253Apmid%252F30176988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.33752&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30176988&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.47" data-doi="10.1523/JNEUROSCI.07-07-01951.1987">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. U
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kubie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The effects of changes in the environment on the spatial firing of hippocampal complex-spike cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1951
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1968
                                 </span>
                                 <span class="cit-pub-date">
                                  (1987)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DMuller%26rft.auinit1%253DR.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D1951%26rft.epage%253D1968%26rft.atitle%253DThe%2Beffects%2Bof%2Bchanges%2Bin%2Bthe%2Benvironment%2Bon%2Bthe%2Bspatial%2Bfiring%2Bof%2Bhippocampal%2Bcomplex-spike%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.07-07-01951.1987%26rft_id%253Dinfo%253Apmid%252F3612226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8xOTUxIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMTUvMjAyMC4xMS4xMS4zNzgxNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.48" data-doi="10.1523/JNEUROSCI.16-02-00823.1996">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gothard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Skaggs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. M
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McNaughton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Binding of hippocampal ca1 neural activity to multiple reference frames in a landmark-based navigation task
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  823
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  835
                                 </span>
                                 <span class="cit-pub-date">
                                  (1996)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DGothard%26rft.auinit1%253DK.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D823%26rft.epage%253D835%26rft.atitle%253DBinding%2Bof%2Bhippocampal%2BCA1%2Bneural%2Bactivity%2Bto%2Bmultiple%2Breference%2Bframes%2Bin%2Ba%2Blandmark-based%2Bnavigation%2Btask%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.16-02-00823.1996%26rft_id%253Dinfo%253Apmid%252F8551362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqbmV1cm8iO3M6NToicmVzaWQiO3M6ODoiMTYvMi84MjMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNS8yMDIwLjExLjExLjM3ODE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.49" data-doi="10.1162/neco.1996.8.1.85">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blum
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. I
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abbott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A model of spatial map formation in the hippocampus of the rat
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neural computation
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  85
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  93
                                 </span>
                                 <span class="cit-pub-date">
                                  (1996)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeural%2BComputation%26rft.stitle%253DNeural%2BComput.%26rft.aulast%253DBlum%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D93%26rft.atitle%253DA%2Bmodel%2Bof%2Bspatial%2Bmap%2Bformation%2Bin%2Bthe%2Bhippocampus%2Bof%2Bthe%2Brat%26rft_id%253Dinfo%253Adoi%252F10.1162%252Fneco.1996.8.1.85%26rft_id%253Dinfo%253Apmid%252F8564805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1162/neco.1996.8.1.85&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8564805&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996TP51600005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.50" data-doi="10.1098/rstb.2008.0296">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hassabis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maguire
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The construction system of the brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Philosophical Transactions of the Royal Society B: Biological Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  364
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1263
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1271
                                 </span>
                                 <span class="cit-pub-date">
                                  (2009)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DHassabis%26rft.auinit1%253DD.%26rft.volume%253D364%26rft.issue%253D1521%26rft.spage%253D1263%26rft.epage%253D1271%26rft.atitle%253DThe%2Bconstruction%2Bsystem%2Bof%2Bthe%2Bbrain%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2008.0296%26rft_id%253Dinfo%253Apmid%252F19528007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rstb.2008.0296&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19528007&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.51" data-doi="10.1111/j.1460-9568.2003.03035.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hayman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chakraborty
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. I
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jeffery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Context-specific acquisition of location discrimination by hippocampal place cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  European Journal of Neuro-science
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2825
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2834
                                 </span>
                                 <span class="cit-pub-date">
                                  (2003)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BEuropean%2Bjournal%2Bof%2Bneuroscience%26rft.stitle%253DEur%2BJ%2BNeurosci%26rft.aulast%253DHayman%26rft.auinit1%253DR.%2BM.%26rft.volume%253D18%26rft.issue%253D10%26rft.spage%253D2825%26rft.epage%253D2834%26rft.atitle%253DContext-specific%2Bacquisition%2Bof%2Blocation%2Bdiscrimination%2Bby%2Bhippocampal%2Bplace%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1460-9568.2003.03035.x%26rft_id%253Dinfo%253Apmid%252F14656331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1460-9568.2003.03035.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=14656331&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000186799700016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.52" data-doi="10.1126/science.1114037">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leutgeb
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Independent codes for spatial and episodic memory in hippocampal neuronal ensembles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  309
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  619
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  623
                                 </span>
                                 <span class="cit-pub-date">
                                  (2005)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLeutgeb%26rft.auinit1%253DS.%26rft.volume%253D309%26rft.issue%253D5734%26rft.spage%253D619%26rft.epage%253D623%26rft.atitle%253DIndependent%2BCodes%2Bfor%2BSpatial%2Band%2BEpisodic%2BMemory%2Bin%2BHippocampal%2BNeuronal%2BEnsembles%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1114037%26rft_id%253Dinfo%253Apmid%252F16040709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDkvNTczNC82MTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNS8yMDIwLjExLjExLjM3ODE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.53" data-doi="10.1016/S0166-2236(00)01797-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blair
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. T
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The anatomical and computational basis of the rat head-direction cell signal
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in neurosciences
                                 </abbr>
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  289
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  294
                                 </span>
                                 <span class="cit-pub-date">
                                  (2001)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bneurosciences%26rft.stitle%253DTrends%2BNeurosci%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BE.%26rft.volume%253D24%26rft.issue%253D5%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DThe%2Banatomical%2Band%2Bcomputational%2Bbasis%2Bof%2Bthe%2Brat%2Bhead-direction%2Bcell%2Bsignal.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0166-2236%252800%252901797-5%26rft_id%253Dinfo%253Apmid%252F11311382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0166-2236(00)01797-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11311382&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000168766200014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               54.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference 54. in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.54" data-doi="10.1126/science.aal4835">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rouault
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Druckmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayaraman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Ring attractor dynamics in the drosophila central brain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  356
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  849
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  853
                                 </span>
                                 <span class="cit-pub-date">
                                  (2017)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aal4835%26rft_id%253Dinfo%253Apmid%252F28473639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTYvNjM0MC84NDkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNS8yMDIwLjExLjExLjM3ODE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               55.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference 55. in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.55" data-doi="10.1038/s41586-019-1772-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fisher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   D’Alessandro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. I
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Sensorimotor experience remaps visual input to a heading-direction network
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  576
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  121
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  125
                                 </span>
                                 <span class="cit-pub-date">
                                  (2019)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D576%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-019-1772-4%26rft_id%253Dinfo%253Apmid%252F31748749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-019-1772-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31748749&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               56.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference 56. in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.56">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Poulter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dachtler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wills
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T. J
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lever
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Vector trace cells in the subiculum of the hippocampal formation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 <span class="cit-fpage">
                                  805242
                                 </span>
                                 <span class="cit-pub-date">
                                  (2019)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               57.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference 57. in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.57" data-doi="10.1002/aic.690370209">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kramer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nonlinear principal component analysis using autoassociative neural networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  AIChE journal
                                 </abbr>
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  233
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  243
                                 </span>
                                 <span class="cit-pub-date">
                                  (1991)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAIChE%2Bjournal%26rft.volume%253D37%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1002%252Faic.690370209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/aic.690370209&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991EX55000008&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               58.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference 58. in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.58">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Beattie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Deepmind lab
                                 </abbr>
                                 <span class="cit-pub-date">
                                  (2016)
                                 </span>
                                 .
                                 <span class="cit-fpage">
                                  1612.03801
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               59.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference 59. in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.59">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. P
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ba
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adam: A method for stochastic optimization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1412.6980
                                 </span>
                                 <span class="cit-pub-date">
                                  (2014)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               60.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference 60. in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.60" data-doi="10.1085/jgp.200409015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rivard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenck-Santini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.-P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Poucet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 . &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Muller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. U
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Representation of objects in space by two classes of hippocampal pyramidal cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of general physiology
                                 </abbr>
                                 <span class="cit-vol">
                                  124
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  9
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  25
                                 </span>
                                 <span class="cit-pub-date">
                                  (2004)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bgeneral%2Bphysiology%26rft_id%253Dinfo%253Adoi%252F10.1085%252Fjgp.200409015%26rft_id%253Dinfo%253Apmid%252F15197223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamdwIjtzOjU6InJlc2lkIjtzOjc6IjEyNC8xLzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNS8yMDIwLjExLjExLjM3ODE0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               61.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference 61. in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.61" data-doi="10.1098/rstb.2012.0512">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Weighted cue integration in the rodent head direction system
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Philosophical Transactions of the Royal Society B: Biological Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  369
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  20120512
                                 </span>
                                 <span class="cit-pub-date">
                                  (2014)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0512%26rft_id%253Dinfo%253Apmid%252F24366127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0512&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24366127&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F15%2F2020.11.11.378141.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               62.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference 62. in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.11.378141v2.62">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alexander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Egocentric boundary vector tuning of the retrosplenial cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 <span class="cit-fpage">
                                  702712
                                 </span>
                                 <span class="cit-pub-date">
                                  (2019)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.11.378141v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.07.12.452061v2" title="Novel Apelin-expressing gCap Endothelial Stem-like Cells Orchestrate Lung Microvascular Repair" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.05.29.446266v4" title="Effect of SSRI discontinuation on anxiety-like behaviours in mice" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 15, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.11.378141v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.11.378141v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.11.378141v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   A model of egocentric to allocentric understanding in mammalian brains
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2448255"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-WIk6QmO1PJyhyXFpfb72jnTGyPmWEdQCy64O19Y27WA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484485436"/>
                                   <input type="hidden" name="captcha_token" value="1005fcfc3c7d053043277eafa6171b23"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2448255--3" data-node-nid="2448255" id="node-2448255--41374125947" data-pisa="biorxiv;2020.11.11.378141v2" data-pisa-master="biorxiv;2020.11.11.378141" data-seqnum="2448255" data-apath="/biorxiv/early/2022/03/15/2020.11.11.378141.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 A model of egocentric to allocentric understanding in mammalian brains
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Benigno
                                  </span>
                                  <span class="nlm-surname">
                                   Uria
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Borja
                                  </span>
                                  <span class="nlm-surname">
                                   Ibarz
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Andrea
                                  </span>
                                  <span class="nlm-surname">
                                   Banino
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Vinicius
                                  </span>
                                  <span class="nlm-surname">
                                   Zambaldi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Dharshan
                                  </span>
                                  <span class="nlm-surname">
                                   Kumaran
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Demis
                                  </span>
                                  <span class="nlm-surname">
                                   Hassabis
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Caswell
                                  </span>
                                  <span class="nlm-surname">
                                   Barry
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Charles
                                  </span>
                                  <span class="nlm-surname">
                                   Blundell
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.11.378141;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.11.378141
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&title=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&title=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&text=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&t=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&title=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&title=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&title=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2448255--5" data-node-nid="2448255" id="citation-node-2448255--61330843654" data-pisa="biorxiv;2020.11.11.378141v2" data-pisa-master="biorxiv;2020.11.11.378141" data-seqnum="2448255" data-apath="/biorxiv/early/2022/03/15/2020.11.11.378141.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     A model of egocentric to allocentric understanding in mammalian brains
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Benigno
                                      </span>
                                      <span class="nlm-surname">
                                       Uria
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Borja
                                      </span>
                                      <span class="nlm-surname">
                                       Ibarz
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Andrea
                                      </span>
                                      <span class="nlm-surname">
                                       Banino
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Vinicius
                                      </span>
                                      <span class="nlm-surname">
                                       Zambaldi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Dharshan
                                      </span>
                                      <span class="nlm-surname">
                                       Kumaran
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Demis
                                      </span>
                                      <span class="nlm-surname">
                                       Hassabis
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Caswell
                                      </span>
                                      <span class="nlm-surname">
                                       Barry
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Charles
                                      </span>
                                      <span class="nlm-surname">
                                       Blundell
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.11.378141;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.11.378141
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Uria%2C+Benigno&rft.au=Ibarz%2C+Borja&rft.au=Banino%2C+Andrea&rft.au=Zambaldi%2C+Vinicius&rft.au=Kumaran%2C+Dharshan&rft.au=Hassabis%2C+Demis&rft.au=Barry%2C+Caswell&rft.au=Blundell%2C+Charles">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2448255/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2448255/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2448255/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2448255/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2448255/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2448255/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2448255/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2448255/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2448255/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2448255/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2448255/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2448255/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&count=horizontal&via=&text=A%20model%20of%20egocentric%20to%20allocentric%20understanding%20in%20mammalian%20brains&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.11.378141v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.11.378141v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
