<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.07.13.201673v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.07.13.201673v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.07.13.201673v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/02/02/2020.07.13.201673.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.07.13.201673v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Neural Network Poisson Models for Behavioural and Neural Spike Train Data"/>
  <meta name="DC.Identifier" content="10.1101/2020.07.13.201673"/>
  <meta name="DC.Date" content="2022-02-02"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="One of the most important and challenging application areas for complex machine learning methods is to predict, characterize and model rich, multi-dimensional, neural data. Recent advances in neural recording techniques have made it possible to monitor the activities of a large number of neurons across different brain regions as animals perform behavioural tasks. This poses the critical challenge of establishing links between neural activity at a microscopic scale, which might for instance represent sensory input, and at a macroscopic scale, which then generates behaviour. Predominant modeling methods apply rather disjoint techniques to these scales; by contrast, we suggest an end-to-end model which exploits recent developments of flexible, but tractable, neural network point-process models to characterize dependencies between stimuli, actions, and neural data. We apply this model to a public dataset collected using Neuropixel probes in mice performing a visually-guided behavioural task as well as a synthetic dataset produced from a hierarchical network model with reciprocally connected sensory and integration circuits intended to characterize animal behaviour in a fixed-duration motion discrimination task. We show that our model outperforms previous approaches and contributes novel insights into the relationships between neural activities and behaviour.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Moein Khajehnejad"/>
  <meta name="DC.Contributor" content="Forough Habibollahi"/>
  <meta name="DC.Contributor" content="Richard Nock"/>
  <meta name="DC.Contributor" content="Ehsan Arabzadeh"/>
  <meta name="DC.Contributor" content="Peter Dayan"/>
  <meta name="DC.Contributor" content="Amir Dezfouli"/>
  <meta name="article:published_time" content="2022-02-02"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Neural Network Poisson Models for Behavioural and Neural Spike Train Data"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>One of the most important and challenging application areas for complex machine learning methods is to predict, characterize and model rich, multi-dimensional, neural data. Recent advances in neural recording techniques have made it possible to monitor the activities of a large number of neurons across different brain regions as animals perform behavioural tasks. This poses the critical challenge of establishing links between neural activity at a microscopic scale, which might for instance represent sensory input, and at a macroscopic scale, which then generates behaviour. Predominant modeling methods apply rather disjoint techniques to these scales; by contrast, we suggest an end-to-end model which exploits recent developments of flexible, but tractable, neural network point-process models to characterize dependencies between stimuli, actions, and neural data. We apply this model to a public dataset collected using Neuropixel probes in mice performing a visually-guided behavioural task as well as a synthetic dataset produced from a hierarchical network model with reciprocally connected sensory and integration circuits intended to characterize animal behaviour in a fixed-duration motion discrimination task. We show that our model outperforms previous approaches and contributes novel insights into the relationships between neural activities and behaviour.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.07.13.201673v2"/>
  <meta name="citation_id" content="2020.07.13.201673v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.07.13.201673"/>
  <meta name="citation_num_pages" content="24"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.07.13.201673"/>
  <meta name="citation_author" content="Moein Khajehnejad"/>
  <meta name="citation_author_institution" content="Monash University"/>
  <meta name="citation_author" content="Forough Habibollahi"/>
  <meta name="citation_author_institution" content="The University of Melbourne"/>
  <meta name="citation_author" content="Richard Nock"/>
  <meta name="citation_author_institution" content="Google Research"/>
  <meta name="citation_author" content="Ehsan Arabzadeh"/>
  <meta name="citation_author_institution" content="Australian National University"/>
  <meta name="citation_author" content="Peter Dayan"/>
  <meta name="citation_author_institution" content="MPI for Biological Cybernetics the University of Tübingen"/>
  <meta name="citation_author" content="Amir Dezfouli"/>
  <meta name="citation_author_institution" content="Data61, CSIRO"/>
  <meta name="citation_reference" content="Abadi, Martín et al. (2015). TensorFlow: Large-Scale Machine Learning on Heterogeneous Systems. Software available from tensorflow.org."/>
  <meta name="citation_reference" content="Balleine, Bernard W and John P O’Doherty (2010). “Human and rodent homologies in action control: corticostriatal determinants of goal-directed and habitual action”. In: Neuropsychopharmacology 35.1, pp. 48–69."/>
  <meta name="citation_reference" content="Barak, Omri (2017). “Recurrent neural networks as versatile tools of neuroscience research”. In: Current opinion in neurobiology 46, pp. 1–6."/>
  <meta name="citation_reference" content="Britten, Kenneth H et al. (1996). “A relationship between behavioral choice and the visual responses of neurons in macaque MT”. In: Visual neuroscience 13.1, pp. 87–100."/>
  <meta name="citation_reference" content="Cebrian, AC (2015). “NHPoisson: Modelling and Validation of Non Homogeneous Poisson Processes”. In: R package version 3."/>
  <meta name="citation_reference" content="Chichilnisky, EJ (2001). “A simple white noise analysis of neuronal light responses”. In: Network: Computation in Neural Systems 12.2, pp. 199–213."/>
  <meta name="citation_reference" content="Churchland, Mark M et al. (2010). “Stimulus onset quenches neural variability: a widespread cortical phenomenon”. In: Nature neuroscience 13.3, pp. 369–378."/>
  <meta name="citation_reference" content="Cunningham, John P and M Byron Yu (2014). “Dimensionality reduction for large-scale neural recordings”. In: Nature neuroscience 17.11, p. 1500."/>
  <meta name="citation_reference" content="Daley, D.J. and D. Vere-Jones (2006). An Introduction to the Theory of Point Processes: Volume I: Elementary Theory and Methods. Probability and Its Applications. Springer New York. ISBN: 9780387215648."/>
  <meta name="citation_reference" content="Dayan, Peter and Laurence F Abbott (2001). “Theoretical neuroscience: computational and mathematical modeling of neural systems”. In."/>
  <meta name="citation_reference" content="Dezfouli, Amir et al. (2018). “Integrated accounts of behavioral and neuroimaging data using flexible recurrent neural network models”. In: Advances in Neural Information Processing Systems, pp. 4228–4237."/>
  <meta name="citation_reference" content="Ganguli, Surya and Haim Sompolinsky (2012). “Compressed sensing, sparsity, and dimensionality in neuronal information processing and data analysis”. In: Annual review of neuroscience 35, pp. 485–508."/>
  <meta name="citation_reference" content="Gold, Joshua I and Michael N Shadlen (2007). “The neural basis of decision making”. In: Annual review of neuroscience 30."/>
  <meta name="citation_reference" content="Goris, Robbe LT, J Anthony Movshon, and Eero P Simoncelli (2014). “Partitioning neuronal variability”. In: Nature neuroscience 17.6, pp. 858–865."/>
  <meta name="citation_reference" content="Grimmett, Geoffrey and David Stirzaker (2020). Probability and random processes. Oxford university press."/>
  <meta name="citation_reference" content="Jun, James J et al. (2017). “Fully integrated silicon probes for high-density recording of neural activity”. In: Nature 551.7679, pp. 232–236."/>
  <meta name="citation_reference" content="Kass, Robert E, Uri T Eden, and Emery N Brown (2014). Analysis of neural data. Vol. 491. Springer."/>
  <meta name="citation_reference" content="Kobak, Dmitry et al. (2016). “Demixed principal component analysis of neural population data”. In: Elife 5, e10989."/>
  <meta name="citation_reference" content="Kriegeskorte, Nikolaus and Rogier A Kievit (2013). “Representational geometry: integrating cognition, computation, and the brain”. In: Trends in cognitive sciences 17.8, pp. 401–412."/>
  <meta name="citation_reference" content="Liu, David and Máté Lengyel (2021). “A universal probabilistic spike count model reveals ongoing modulation of neural variability”. In: Advances in Neural Information Processing Systems 34."/>
  <meta name="citation_reference" content="Mante, Valerio et al. (2013). “Context-dependent computation by recurrent dynamics in prefrontal cortex”. In: nature 503.7474, pp. 78–84."/>
  <meta name="citation_reference" content="Meyer, Arne F et al. (2017). “Models of neuronal stimulus-response functions: elaboration, estimation, and evaluation”. In: Frontiers in systems neuroscience 10, p. 109."/>
  <meta name="citation_reference" content="Omi, Takahiro, Kazuyuki Aihara, et al. (2019). “Fully Neural Network based Model for General Temporal Point Processes”. In: Advances in Neural Information Processing Systems, pp. 2120–2129."/>
  <meta name="citation_reference" content="Paninski, Liam (2004). “Maximum likelihood estimation of cascade point-process neural encoding models”. In: Network: Computation in Neural Systems 15.4, pp. 243–262."/>
  <meta name="citation_reference" content="Paninski, Liam, Jonathan Pillow, and Jeremy Lewi (2007). “Statistical models for neural encoding, decoding, and optimal stimulus design”. In: Progress in brain research 165, pp. 493–507."/>
  <meta name="citation_reference" content="Paninski, Liam et al. (2010). “A new look at state-space models for neural data”. In: Journal of computational neuroscience 29.1-2, pp. 107–126."/>
  <meta name="citation_reference" content="Richards, Blake A et al. (2019). “A deep learning framework for neuroscience”. In: Nature neuroscience 22.11, pp. 1761–1770."/>
  <meta name="citation_reference" content="Rieke, Fred et al. (1999). Spikes: exploring the neural code. MIT Press."/>
  <meta name="citation_reference" content="Ross, Sheldon M et al. (1996). Stochastic processes. Vol. 2. Wiley New York."/>
  <meta name="citation_reference" content="Schaeffer, Rylan et al. (2020). “Reverse-engineering recurrent neural network solutions to a hierarchical inference task for mice”. In: Advances in Neural Information Processing Systems 33."/>
  <meta name="citation_reference" content="Shadlen, Michael N and William T Newsome (1998). “The variable discharge of cortical neurons: implications for connectivity, computation, and information coding”. In: Journal of neuroscience 18.10, pp. 3870–3896."/>
  <meta name="citation_reference" content="Sill, Joseph (1998). “Monotonic networks”. In: Advances in neural information processing systems, pp. 661–667."/>
  <meta name="citation_reference" content="Steinmetz, Nicholas A et al. (2019). “Distributed coding of choice, action and engagement across the mouse brain”. In: Nature 576.7786, pp. 266–273."/>
  <meta name="citation_reference" content="Sussillo, David (2014). “Neural circuits as computational dynamical systems”. In: Current opinion in neurobiology 25, pp. 156–163."/>
  <meta name="citation_reference" content="Truccolo, Wilson et al. (2005). “A point process framework for relating neural spiking activity to spiking history, neural ensemble, and extrinsic covariate effects”. In: Journal of neurophysiology 93.2, pp. 1074–1089."/>
  <meta name="citation_reference" content="Whiteway, Matthew R and Daniel A Butts (2019). “The quest for interpretable models of neural population activity”. In: Current opinion in neurobiology 58, pp. 86–93."/>
  <meta name="citation_reference" content="Wimmer, Klaus et al. (2015). “Sensory integration dynamics in a hierarchical network explains choice probabilities in cortical area MT”. In: Nature communications 6.1, pp. 1–13."/>
  <meta name="citation_reference" content="Wu, Anqi et al. (2017). “Gaussian process based nonlinear latent structure discovery in multivariate spike train data”. In: Proceedings of the 31st International Conference on Neural Information Processing Systems, pp. 3499–3508."/>
  <meta name="citation_reference" content="Yamins, Daniel LK et al. (2014). “Performance-optimized hierarchical models predict neural responses in higher visual cortex”. In: Proceedings of the national academy of sciences 111.23, pp. 8619–8624."/>
  <meta name="citation_reference" content="Yu, M Byron et al. (2006). “Extracting dynamical structure embedded in neural activity”. In: Advances in neural information processing systems, pp. 1545–1552."/>
  <meta name="twitter:title" content="Neural Network Poisson Models for Behavioural and Neural Spike Train Data"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="One of the most important and challenging application areas for complex machine learning methods is to predict, characterize and model rich, multi-dimensional, neural data. Recent advances in neural recording techniques have made it possible to monitor the activities of a large number of neurons across different brain regions as animals perform behavioural tasks. This poses the critical challenge of establishing links between neural activity at a microscopic scale, which might for instance represent sensory input, and at a macroscopic scale, which then generates behaviour. Predominant modeling methods apply rather disjoint techniques to these scales; by contrast, we suggest an end-to-end model which exploits recent developments of flexible, but tractable, neural network point-process models to characterize dependencies between stimuli, actions, and neural data. We apply this model to a public dataset collected using Neuropixel probes in mice performing a visually-guided behavioural task as well as a synthetic dataset produced from a hierarchical network model with reciprocally connected sensory and integration circuits intended to characterize animal behaviour in a fixed-duration motion discrimination task. We show that our model outperforms previous approaches and contributes novel insights into the relationships between neural activities and behaviour.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Neural Network Poisson Models for Behavioural and Neural Spike Train Data"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="One of the most important and challenging application areas for complex machine learning methods is to predict, characterize and model rich, multi-dimensional, neural data. Recent advances in neural recording techniques have made it possible to monitor the activities of a large number of neurons across different brain regions as animals perform behavioural tasks. This poses the critical challenge of establishing links between neural activity at a microscopic scale, which might for instance represent sensory input, and at a macroscopic scale, which then generates behaviour. Predominant modeling methods apply rather disjoint techniques to these scales; by contrast, we suggest an end-to-end model which exploits recent developments of flexible, but tractable, neural network point-process models to characterize dependencies between stimuli, actions, and neural data. We apply this model to a public dataset collected using Neuropixel probes in mice performing a visually-guided behavioural task as well as a synthetic dataset produced from a hierarchical network model with reciprocally connected sensory and integration circuits intended to characterize animal behaviour in a fixed-duration motion discrimination task. We show that our model outperforms previous approaches and contributes novel insights into the relationships between neural activities and behaviour.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-02-02"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2381225"/>
  <title>
   Neural Network Poisson Models for Behavioural and Neural Spike Train Data | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"wjorEJMYzKo2yQ2-bnr8CODxZKqt7plPwjHYDejvciw"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2381225","apath":"\/biorxiv\/early\/2022\/02\/02\/2020.07.13.201673.atom","pisa":"biorxiv;2020.07.13.201673v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.07.13.201673v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.07.13.201673v2","title":"Neural Network Poisson Models for Behavioural and Neural Spike Train Data","identifier":"node\/2381225"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xjMxaq0bUYOX3qSYiwjfOvhCkygGNiAoBHPllw5IYS2QLSa9P7yv02p5veex3AvpwKLcAbWEt\\\\\\\/\\\\\\\/gB+a\\\\\\\/1RwqOweEUTMhvbePgleA2uwVQSG5pfa02uabowhy1pNCWw8spj9JnsM+IY4BIv+XWGmHO1UQJfdv42x9Eido3TlPhETMSBb3ig95QI9Tfma+r+EBJpkgyYIGh6dkZEelNPsGFOFj7UjyykciSqe0b9pnHRJP1SV0Zidf5i9HMSpVYwbSFbfswPddB+rUmRLm4J+c6qKw8z2abuayxynT0nMPW2wIHc3rJVDU4WqebRUa1U6zW1xfh3JRIDI6QFVnijmjhfDqGUMWlXOkI\\\\\\\/5HnjGShpqUtwOhmBtsBG+5NGrHx8msohjWFSdRkeAk+G123rqsQhm8dTYAkgulFLNWosp2C\\\\\\\/9X4lOIkCBNuTWBXSDnV8ItHd\\\\\\\/xt9WfjEetvrm4YTNY+X6g\\\\\\\/ESNOOq3ZAYuAluBmlM1XHTHxvyl\\\\\\\/0q2FsBSPuM3HL5GDjEqMU9oFU8YMMUQUfMf6ho8frdbLMOpaCVKRbiIe5wOC1APgcVj0l9zUxtI1n0fNs0lJCSJJA2Z8ZaCPlGbHuXDWOuj6kiK6L3\\\\\\\/Cw9NA8KyAIYHtDQZGhhIPPEsVgwQPyGEJcnjIwpRzpI6Di\\\\\\\/8QLcF4z62Vs8xGLZcF397txiSs3vDZ9gHk\\\\\\\/hMMk6yJ6YPK2eAl+FRxBFxX+OhCh2LCCTws0ujdGJtro7rLU8rE0700173ESmVjnT2wi+PDzVHRA6GC\\\\\\\/mTwDZE4XegRicPorF0LcuZtYTDdfOFV5JalNwTzpGmPvTlmLrF5Ma934zpfqiUb4sEVQFXn4nXRboLi25W8ZzloJsl7KWDJLVxbUKeTzagXFyF514HHe467816YWd0GzYBQhJBFbVcgxPZcydOOpAaSxRxMK\\\\\\\/OunqvcJ7PmXzoF5e+WtmhrnUrF2\\\\\\\/jeHu61tNIo9cY05DtSXI7oqYnkyn1kArljAjSvSblvLbN4Z56tEbttpAEqWzy5uf9\\\\\\\/xxdNkk29DXOrmTyxsxUXWjR\\\\\\\/kAWCt2QCIyoKGqnltYnMh3yt5pwAHfpsPYso3aMGIpdFxT0wnFpnaWDXmsERHRceadl8Uk5DMSHLRPdqO51a4dsvvslMrkzbJcRoQdndn0zOwzA8pdQ3wjmmtdXsS77Ych1Yr4tUcP6W4vIJXKWI4BTCeSNq63bKMeLtAlpuGHD2wUC1dWmZoHyR5HaihALT8iyumo\\\\\\\/TP8swiUA8Nl7mbqStDB+9\\\\\\\/eMKQCG\\\\\\\/II2wSbm19L+wodjv7DRgB22U3KNV6jJuXaQucLizdWotECFNas\\\\\\\/vAeNxfVhlJXYlqjaJyaeI6AaIY27t4ylMYPpv2GPZxHg6nJkwmOUEyU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022gC6FJd76GJ7q6K4kXiyLNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00223a10062188599ede01a4f6d943dbfbd28620ab5018734b50d3fd167cd03c27e5\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.07.13.201673v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2381225 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.07.13.201673v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_185917578">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_185917578" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_750456986" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-mhxSpb06g-VIlQOD8WtepPe3doWdCIowU25LxxmNDUg"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2381225" data-node-nid="2381225" id="node-2381225--2880325380" data-pisa="biorxiv;2020.07.13.201673v2" data-pisa-master="biorxiv;2020.07.13.201673" data-apath="/biorxiv/early/2022/02/02/2020.07.13.201673.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Neural Network Poisson Models for Behavioural and Neural Spike Train Data
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Moein
                      </span>
                      <span class="nlm-surname">
                       Khajehnejad
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Forough
                      </span>
                      <span class="nlm-surname">
                       Habibollahi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Richard
                      </span>
                      <span class="nlm-surname">
                       Nock
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Ehsan
                      </span>
                      <span class="nlm-surname">
                       Arabzadeh
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Peter
                      </span>
                      <span class="nlm-surname">
                       Dayan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Amir
                      </span>
                      <span class="nlm-surname">
                       Dezfouli
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.07.13.201673
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2381225--2880325380" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Moein Khajehnejad
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Monash University
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Moein%2BKhajehnejad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Khajehnejad%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMoein%2BKhajehnejad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Forough Habibollahi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        The University of Melbourne
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Forough%2BHabibollahi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Habibollahi%20F&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AForough%2BHabibollahi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Richard Nock
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Google Research
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Richard%2BNock%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Nock%20R&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ARichard%2BNock%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Ehsan Arabzadeh
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Australian National University
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ehsan%2BArabzadeh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Arabzadeh%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AEhsan%2BArabzadeh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Peter Dayan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        MPI for Biological Cybernetics the University of Tübingen
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Peter%2BDayan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dayan%20P&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3APeter%2BDayan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Amir Dezfouli
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Data61, CSIRO
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Amir%2BDezfouli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dezfouli%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAmir%2BDezfouli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.07.13.201673v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2381225" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2381225/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.13.201673v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2381225" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2381225/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.13.201673v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2381225" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2381225/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.13.201673v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2381225" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2381225/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.07.13.201673v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2381225" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2381225/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             A
                             <span class="sc">
                              bstract
                             </span>
                            </h2>
                            <p id="p-3">
                             One of the most important and challenging application areas for complex machine learning methods is to predict, characterize and model rich, multi-dimensional, neural data. Recent advances in neural recording techniques have made it possible to monitor the activities of a large number of neurons across different brain regions as animals perform behavioural tasks. This poses the critical challenge of establishing links between neural activity at a microscopic scale, which might for instance represent sensory input, and at a macroscopic scale, which then generates behaviour. Predominant modeling methods apply rather disjoint techniques to these scales; by contrast, we suggest an end-to-end model which exploits recent developments of flexible, but tractable, neural network point-process models to characterize dependencies between stimuli, actions, and neural data. We apply this model to a public dataset collected using Neuropixel probes in mice performing a visually-guided behavioural task as well as a synthetic dataset produced from a hierarchical network model with reciprocally connected sensory and integration circuits intended to characterize animal behaviour in a fixed-duration motion discrimination task. We show that our model outperforms previous approaches and contributes novel insights into the relationships between neural activities and behaviour.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-5">
                             Recent developments in neural recording techniques such as Neuropixel probes allow the activities of large numbers of neurons across the brain to be monitored as animals perform behavioural tasks [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Jun et al. 2017
                             </a>
                             ]. This allows us to study how the brain represents past and present sensory inputs across areas, how these representations evolve over time and ultimately lead to behaviour.
                            </p>
                            <p id="p-6">
                             Very coarsely, supervised, reinforcement learning and unsupervised methods have been applied to examine the relationships between activity and behaviour [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              Paninski, Pillow, and Lewi 2007
                             </a>
                             ;
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Mante et al. 2013
                             </a>
                             ;
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              Ganguli and Sompolinsky 2012
                             </a>
                             ;
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              Kass, Eden, and Brown 2014
                             </a>
                             ;
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              Sussillo 2014
                             </a>
                             ;
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Richards et al. 2019
                             </a>
                             ;
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              Schaeffer et al. 2020
                             </a>
                             ]. Encoding and decoding models are examples of supervised learning. Encoding models use linear or non-linear methods to predict the activity of individual neurons based on task-related variables such as stimuli, actions, rewards and the like. Decoding models use linear or non-linear methods to predict the values of these task-related variables from the conjoint activities of multiple neurons within or between areas. Both types of model have been hugely influential from the earliest days of the application of computational methods to understand neural representation and processing [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Dayan and Abbott 2001
                             </a>
                             ;
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              Rieke et al. 1999
                             </a>
                             ;
                             <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                              Kass, Eden, and Brown 2014
                             </a>
                             ;
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Meyer et al. 2017
                             </a>
                             ]. However, simple reflections of the computational constraints of the task are often insufficient to capture complex neural representations of the sensory inputs and actions that are distributed across different brain regions [
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Steinmetz et al. 2019
                             </a>
                             ] and can also evolve over time in ‘null’ neural modes that have no behavioural consequence. Moreover, neural responses are variable and a population’s response will often differ from trial to trial and over time, even under the same experimental conditions [
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              Shadlen and Newsome 1998
                             </a>
                             ]. Most traditional approaches do not generalize to these more naturalistic conditions where trials with identical stimuli do not exhibit identical behavioural schemes [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              Goris, Movshon, and Simoncelli 2014
                             </a>
                             ;
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              Churchland et al. 2010
                             </a>
                             ]. Hence, they can not account for the temporal irregularities in behaviour and neural recordings between trials.
                            </p>
                            <p id="p-7">
                             Reinforcement learning (or in some cases, supervised learning) methods have more recently been used to learn potentially complex feedforward and recurrent neural network (RNN) models that are themselves capable of performing the same behavioural task as the subjects, based on assumptions about sensory noise and processing architecture [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Barak 2017
                             </a>
                             ;
                             <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                              Sussillo 2014
                             </a>
                             ; Richards et al. 2019; Schaeffer et al. 2020;
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              Yamins et al. 2014
                             </a>
                             ]. These have been highly revealing about neural coding. However, they are also ill-suited to capture the myriad complexities of null modes, or the particular sub-optimalities expressed by individual subjects, reflecting their particular incompetence, training history and more.
                            </p>
                            <p id="p-8">
                             Unsupervised methods have also been applied – often ways of mapping very high dimensional population activity into lower dimensional spaces [
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              Paninski et al. 2010
                             </a>
                             ;
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Cunningham and Yu 2014
                             </a>
                             ;
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              Whiteway and Butts 2019
                             </a>
                             ;
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              Yu et al. 2006
                             </a>
                             ]; with the structure in these spaces, and perhaps the dynamical evolution of the states in these spaces, subsequently being related to task variables. These methods are typically useful since the number of dimensions of task input and/or output variability is often rather modest, implying that much of the high dimensional space that could potentially be occupied is either empty or at least not relevant for behaviour. However, they typically have to use intrinsic metrics such as variance to specify which low dimensional projections should be considered – and this again begs the question as to what is important.
                            </p>
                            <p id="p-9">
                             Of course, there are methods that combine various of these approaches [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              Kobak et al. 2016
                             </a>
                             ;
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Kriegeskorte and Kievit 2013
                             </a>
                             ], and continual innovations. Along these lines, [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              Dezfouli et al. 2018
                             </a>
                             ] recently suggested a combined approach, with an RNN being trained to tie fMRI BOLD activity across the brain directly with ongoing behaviour. fMRI data allowed for a form of model inversion, pinning down the RNN state and so implying how behaviour would be realized neurally. However, this approach is licensed by the invertibility that is at least plausible because of the high dimensionality of fMRI. It is not currently guaranteed to be available on a trial-by-trial basis in neural recordings.
                            </p>
                            <p id="p-10">
                             Here, motivated by previous approaches and recent neural network point-process models [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              Omi, Aihara, et al. 2019
                             </a>
                             ], we suggest a novel neural network Poisson process model which: (i) flexibly learns the connections between environmental stimuli and neural representations, and between neural representations and behavioural responses; (ii) jointly fits both behavioural and neural data; (iii) handles variabilities between response times in different trials of an experiments by a temporal rescaling mechanism, and (iv) derives spike count statistics disentangled from chosen temporal bin sizes. The framework allows efficient training of the model without making assumptions about the functional form of the relationship between input stimuli and neural and behavioural processes. We apply the method to two neural/behavioural datasets concerning visual discrimination tasks: one collected using Neuropixel probes [
                             <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">
                              Steinmetz et al. 2019
                             </a>
                             ] from mice, and the other the output of a hierarchical network model with reciprocally connected sensory and integration circuits that was designed to model behaviour in a motion-based task [
                             <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                              Wimmer et al. 2015
                             </a>
                             ]. We show that our method is able in both cases to link behavioural data with their underlying neural processes and input stimuli; the synthetic dataset allows us to compare our results against ground truth.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 The model
                            </h2>
                            <div id="sec-3" class="subsection">
                             <div id="sec-4" class="subsection">
                              <h4>
                               Data description
                              </h4>
                              <p id="p-11">
                               We model canonical visual discrimination experiments. In our case, on each trial, subjects are presented with a stimulus and have to choose an option (or keep still; NoGo). We consider two datasets in this setting. The first is the visual discrimination experiment of [
                               <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">
                                Steinmetz et al. 2019
                               </a>
                               ] (
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Figure 1a
                               </a>
                               ). On each trial, mice are presented with a stimulus (visual contrast on the left or right side) and have to make a simple response by turning a wheel left or right or keeping it still. The second dataset is synthetic and based on the work of [
                               <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">
                                Wimmer et al. 2015
                               </a>
                               ] (
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Figure 1b
                               </a>
                               ). A hierarchical spiking neural network model is built to capture the essence of evidence integration and decision-making of monkeys in a standard two-alternative forced-choice motion discrimination task [
                               <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                                Gold and Shadlen 2007
                               </a>
                               ;
                               <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                                Britten et al. 1996
                               </a>
                               ].
                              </p>
                              <div id="F1" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F1.large.jpg?width=800&height=600&carousel=1" title="a) Steinmetz’ visual discrimination task [Steinmetz et al. 2019]. On each trial, visual stimuli of different contrasts were potentially presented on the left and right sides. If both sides had zero contrast, the mouse earned reward by NoGo. If both had equal, non-zero, contrast, it was rewarded at random. Otherwise, it was rewarded for reporting (a ∈ {left, right}) which contrast was larger. The figure is adapted from [Steinmetz et al. 2019]. b) Synthetic network model illustrating the sensory (E1; E2) and integrator (D1; D2) circuits enjoying feed-forward and top-down feedback connections as well as lateral excitatory and inhibitory (population I) recurrent connections within each circuit. The figure is adapted from [Wimmer et al. 2015]. c) Illustrative example illustrating the occurrence of systematic bias in firing rate estimation when aggregating trials with different end times. Red curves show firing rates for six sample trials. The dashed line shows the average of firing rates based on the unfinished trials at each point in time. Looking at the blue curve, one might conclude that neural activities increase and then decrease over times, which is not true for any individual trial." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>a)</strong> Steinmetz’ visual discrimination task [Steinmetz et al. 2019]. On each trial, visual stimuli of different contrasts were potentially presented on the left and right sides. If both sides had zero contrast, the mouse earned reward by NoGo. If both had equal, non-zero, contrast, it was rewarded at random. Otherwise, it was rewarded for reporting (<em>a</em> ∈ {left, right}) which contrast was larger. The figure is adapted from [Steinmetz et al. 2019]. <strong>b)</strong> Synthetic network model illustrating the sensory (E1; E2) and integrator (D1; D2) circuits enjoying feed-forward and top-down feedback connections as well as lateral excitatory and inhibitory (population I) recurrent connections within each circuit. The figure is adapted from [Wimmer et al. 2015]. <strong>c)</strong> Illustrative example illustrating the occurrence of systematic bias in firing rate estimation when aggregating trials with different end times. Red curves show firing rates for six sample trials. The dashed line shows the average of firing rates based on the unfinished trials at each point in time. Looking at the blue curve, one might conclude that neural activities increase and then decrease over times, which is not true for any individual trial.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F1.medium.gif" width="440" height="176"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F1.medium.gif" width="440" height="176"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                                <span class="fig-label">
                                 Figure 1:
                                </span>
                                <p id="p-12" class="first-child">
                                 <strong>
                                  a)
                                 </strong>
                                 Steinmetz’ visual discrimination task [
                                 <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">
                                  Steinmetz et al. 2019
                                 </a>
                                 ]. On each trial, visual stimuli of different contrasts were potentially presented on the left and right sides. If both sides had zero contrast, the mouse earned reward by NoGo. If both had equal, non-zero, contrast, it was rewarded at random. Otherwise, it was rewarded for reporting (a ∈ {
                                 <span class="sc">
                                  left, right
                                 </span>
                                 }) which contrast was larger. The figure is adapted from [
                                 <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">
                                  Steinmetz et al. 2019
                                 </a>
                                 ].
                                 <strong>
                                  b)
                                 </strong>
                                 Synthetic network model illustrating the sensory (E1; E2) and integrator (D1; D2) circuits enjoying feed-forward and top-down feedback connections as well as lateral excitatory and inhibitory (population I) recurrent connections within each circuit. The figure is adapted from [
                                 <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">
                                  Wimmer et al. 2015
                                 </a>
                                 ].
                                 <strong>
                                  c)
                                 </strong>
                                 Illustrative example illustrating the occurrence of systematic bias in firing rate estimation when aggregating trials with different end times. Red curves show firing rates for six sample trials. The dashed line shows the average of firing rates based on the unfinished trials at each point in time. Looking at the blue curve, one might conclude that neural activities increase and then decrease over times, which is not true for any individual trial.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div id="sec-5" class="subsection">
                              <h4>
                               Formalisation
                              </h4>
                              <p id="p-13">
                               The total number of trials in an experiment is denoted by
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               ; the stimuli on trial
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are (generically) denoted by vector
                               <strong>
                                x
                               </strong>
                               <sub>
                                n
                               </sub>
                               . If a response was made on trial n (at a time relative to stimulus onset we call r
                               <sub>
                                n
                               </sub>
                               ), we denote it by
                               <span class="inline-formula" id="inline-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (where, here,
                               <span class="inline-formula" id="inline-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-4.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-4.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . After an observation window of W = 400ms (for the Steinmetz dataset; same as the window size chosen in [
                               <a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">
                                Steinmetz et al. 2019
                               </a>
                               ]) and W = 2000ms (for the synthetic data) expires, then we consider the subject to have chosen NoGo.
                              </p>
                              <p id="p-14">
                               Different neurons in different areas and even different animals (in the Steinmetz dataset) may be recorded and contribute in separate trials. In total,
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               spikes are observed from unit u at times
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , relative to stimulus onset in the corresponding trials.
                              </p>
                              <p id="p-15">
                               In the following, we first discuss how we model the neural data; and then how we couple this model to predict behaviour.
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Figure 2
                               </a>
                               provides an overview of the designed framework. For more details on the model architecture, please refer to
                               <a id="xref-sec-19-1" class="xref-sec" href="#sec-19">
                                Supplementary Materials
                               </a>
                               .
                              </p>
                              <div id="F2" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F2.large.jpg?width=800&height=600&carousel=1" title="Network architecture. ① Embedding of input stimulus into vector hn. ② Transformation of embedding hn into neural activity of each region for each time point t since the stimulus onset. The monotonic transformation function, zn (t′) = t., is applied to the input spike time series is step. Neural activities arecharacterised by the rescaled cumulative intensity function of spike train for each region, denoted by  for regions . The intensity function  is obtained by differentiating the cumulative intensity hunction  with respect to t. Component ② structurally ensures that  increases with time, so . ③ Neural activities are mapped to behavioural responses which are represented by the rescaled cumulative intensity function  for making each action  at each time t since stimulus onset (t). The data likelihood is computed over both recorded spike trains and behavioural reaponses. This yields a neural log-likelihood function  and a behavioural log-likelihood function ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Network architecture. ① Embedding of input stimulus into vector <strong>h</strong><sub><em>n</em></sub>. ② Transformation of embedding <strong>h</strong><sub><em>n</em></sub> into neural activity of each region for each time point <em>t</em> since the stimulus onset. The monotonic transformation function, <em>z<sub>n</sub></em> (<em>t′</em>) = <em>t</em>., is applied to the input spike time series is step. Neural activities arecharacterised by the rescaled <em>cumulative</em> intensity function of spike train for each region, denoted by  for regions . The intensity function  is obtained by differentiating the cumulative intensity hunction  with respect to <em>t</em>. Component ② structurally ensures that  increases with time, so . ③ Neural activities are mapped to behavioural responses which are represented by the rescaled cumulative intensity function  for making each action  at each time <em>t</em> since stimulus onset (<em>t</em>). The data likelihood is computed over both recorded spike trains and behavioural reaponses. This yields a neural log-likelihood function  and a behavioural log-likelihood function .</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F2.medium.gif" width="440" height="217"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F2.medium.gif" width="440" height="217"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 2:
                                </span>
                                <p id="p-16" class="first-child">
                                 Network architecture. ① Embedding of input stimulus into vector
                                 <strong>
                                  h
                                 </strong>
                                 <sub>
                                  n
                                 </sub>
                                 . ② Transformation of embedding
                                 <strong>
                                  h
                                 </strong>
                                 <sub>
                                  n
                                 </sub>
                                 into neural activity of each region for each time point t since the stimulus onset. The monotonic transformation function, z
                                 <sub>
                                  n
                                 </sub>
                                 (t′) = t., is applied to the input spike time series is step. Neural activities arecharacterised by the rescaled cumulative intensity function of spike train for each region, denoted by
                                 <span class="inline-formula" id="inline-formula-7">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-7.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-7.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 for regions
                                 <span class="inline-formula" id="inline-formula-8">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-8.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-8.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 . The intensity function
                                 <span class="inline-formula" id="inline-formula-9">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-9.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-9.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 is obtained by differentiating the cumulative intensity hunction
                                 <span class="inline-formula" id="inline-formula-10">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-10.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-10.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 with respect to t. Component ② structurally ensures that
                                 <span class="inline-formula" id="inline-formula-11">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-11.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-11.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 increases with time, so
                                 <span class="inline-formula" id="inline-formula-12">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-12.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-12.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 . ③ Neural activities are mapped to behavioural responses which are represented by the rescaled cumulative intensity function
                                 <span class="inline-formula" id="inline-formula-13">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-13.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-13.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 for making each action
                                 <span class="inline-formula" id="inline-formula-14">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-14.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-14.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 at each time t since stimulus onset (t). The data likelihood is computed over both recorded spike trains and behavioural reaponses. This yields a neural log-likelihood function
                                 <span class="inline-formula" id="inline-formula-15">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-15.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-15.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 and a behavioural log-likelihood function
                                 <span class="inline-formula" id="inline-formula-16">
                                  <span class="highwire-responsive-lazyload">
                                   <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-16.gif"/>
                                   <noscript>
                                    <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F2/embed/inline-graphic-16.gif"/>
                                   </noscript>
                                  </span>
                                 </span>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.1 Spike train models
                             </h3>
                             <p id="p-17">
                              We make the simplification that the spikes of each neuron u in trial n can be modelled as the output of an inhomogeneous Poisson process [
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               Daley and Vere-Jones 2006
                              </a>
                              ] with a latent intensity function
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (the superscript N indicates that the intensity function is for the Neural data). This can be interpreted as the instantaneous probability of observing a spike at time τ, where
                              <strong>
                               h
                              </strong>
                              <sub>
                               n
                              </sub>
                              is a function of the stimulus
                              <strong>
                               x
                              </strong>
                              <sub>
                               n
                              </sub>
                              . The Poisson process assumption is that successive spike times are independent, given
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . Note that we seek to capture signal correlations but not noise correlations, and so
                              <strong>
                               h
                              </strong>
                              n does not depend on the spikes observed during a trial.
                             </p>
                             <div id="sec-7" class="subsection">
                              <h4>
                               2.1.1 Time rescaling of spike trains
                              </h4>
                              <p id="p-18">
                               We consider spikes from neuron u until either a response a
                               <sub>
                                n
                               </sub>
                               was made at time r
                               <sub>
                                n
                               </sub>
                               , or to the end of time window W, whichever comes first, i.e., up to W
                               <sub>
                                n
                               </sub>
                               = min(W, r
                               <sub>
                                n
                               </sub>
                               ). The reason for restricting the observation period to r
                               <sub>
                                n
                               </sub>
                               is because the aim is to model the neural processes that lead to behavioural responses, rather than what happens post-response. The joint probability density of observing spike trains from neuron u in trial n is then,
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-3.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-3.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p>
                               Intuitively, the term
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               represents the probability density of observing a spike at time
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . and the exponential term represents the probability of not observing spikes at other times in the observation period. We aim to estimate a single function
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to model the neural activities across all trials. However, note that the duration of trials can be different (based on response times) and only trials that ended after τ can contribute to the estimation of
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , which means that this quantity is implicitly conditioned on r
                               <sub>
                                n
                               </sub>
                               > τ. This property makes the interpretation of
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               rather inconvenient since it will no longer represent how neural activities evolve over time, but is confounded by the distribution of response times (see
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Fig 1c
                               </a>
                               ). To address this issue, it is tempting to merely condition
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               on response times (in addition to
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               ) to get a picture of the spike trains that lead to each specific response time (rather than all the response times after τ). This, however, only partially addresses the issue. To address it more fully, we aim to map all trials with different duration to the same time span. To achieve this goal, we propose the following theorem and proposition:
                              </p>
                              <div id="statement-1" class="statement">
                               <span class="statement-label">
                                Theorem 1.
                               </span>
                               <p id="p-20">
                                Let 0 < s
                                <sup>
                                 ′1
                                </sup>
                                < s
                                <sup>
                                 ′2
                                </sup>
                                <,…,< s
                                <sup>
                                 ′j
                                </sup>
                                ≤ W
                                <sub>
                                 n
                                </sub>
                                ≤ W be a realization from an inhomogeneous Poisson point process, n, with an intensity function λ
                                <sub>
                                 n
                                </sub>
                                (t′) satisfying 0 < λ
                                <sub>
                                 n
                                </sub>
                                (t′) for all t′ ∈ (0, W
                                <sub>
                                 n
                                </sub>
                                ]. Define a one-to-one monotonic transformation function, where:
                                <span class="disp-formula" id="disp-formula-2">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-4.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-4.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                               </p>
                               <p id="p-21">
                                Assume 0 < s
                                <sup>
                                 1
                                </sup>
                                < s
                                <sup>
                                 2
                                </sup>
                                <,…, < s
                                <sup>
                                 j
                                </sup>
                                ≤ W where ∀k ∈ {1,…, j}; s
                                <sup>
                                 k
                                </sup>
                                = z
                                <sub>
                                 n
                                </sub>
                                (s′
                                <sup>
                                 k
                                </sup>
                                ). Then s
                                <sup>
                                 k
                                </sup>
                                are a realization from a second inhomogeneous Poisson point process with λ(t) = λ
                                <sub>
                                 n
                                </sub>
                                (t′) where t = z
                                <sub>
                                 n
                                </sub>
                                (t′).
                               </p>
                              </div>
                              <div id="statement-2" class="statement">
                               <span class="statement-label">
                                Proposition 1.
                               </span>
                               <p id="p-22">
                                For a linear transformation function z
                                <sub>
                                 n
                                </sub>
                                (.), as defined in
                                <strong>
                                 Theorem 1.
                                </strong>
                                , the cumulative intensity function of the original and second point process realizations are related as:
                                <span class="inline-formula" id="inline-formula-25">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-25.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-25.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                               </p>
                               <p id="p-23">
                                Note that
                                <strong>
                                 Theorem 1.
                                </strong>
                                is a special case under the Mapping Theorem [
                                <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                 Grimmett and Stirzaker 2020
                                </a>
                                ]. Please see
                                <a id="xref-sec-19-2" class="xref-sec" href="#sec-19">
                                 Supplementary Materials
                                </a>
                                for proofs.
                               </p>
                              </div>
                             </div>
                             <div id="sec-8" class="subsection">
                              <h4>
                               2.1.2 Parametrising the intensity function
                              </h4>
                              <p id="p-24">
                               Here, we define
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               to represent canonical neural activities (prior to the response) defined over t ∈ [0, W]. Based on the above theorem, the neural activities for a certain trial with duration W
                               <sub>
                                n
                               </sub>
                               can be obtained by applying the time rescaling on the original spike time series using a monotonic function z
                               <sub>
                                n
                               </sub>
                               : [0, W
                               <sub>
                                n
                               </sub>
                               ] → [0, W], t = z
                               <sub>
                                n
                               </sub>
                               (t′),
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-25">
                               The dependence of the intensity function on the embedding
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               plays a crucial role in determining how neural activities are shaped by the stimulus and has to be characterized in a flexible manner. To achieve this, one option is to use a multi-layer feed-forward network which takes t and
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               as inputs, and outputs
                               <span class="inline-formula" id="inline-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Unfortunately, it is then intractable to calculate the integral in
                               <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">
                                equation 1
                               </a>
                               . An elegant solution to this problem is to parameterize the cumulative intensity function
                               <span class="inline-formula" id="inline-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               instead of
                               <span class="inline-formula" id="inline-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               [
                               <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                                Omi, Aihara, et al. 2019
                               </a>
                               ]. This is
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and can be (automatically) differentiated to produce the intensity function:
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-26">
                               We represent
                               <span class="inline-formula" id="inline-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               using a feed-forward network. Note that the gradient of the cumulative intensity function w.r.t to t must always be non-negative (see
                               <a id="xref-sec-9-1" class="xref-sec" href="#sec-9">
                                Section Model structure
                               </a>
                               ). In this work, we chose time to be rescaled uniformly using z
                               <sub>
                                n
                               </sub>
                               (t′) = t′W/W
                               <sub>
                                n
                               </sub>
                               . This makes a substantive assumption that the activities in slow and fast trials are stretched versions of each other. Its advantage is to reduce the task of learning trial-specific intensities to learning the canonical intensity function
                               <span class="inline-formula" id="inline-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Based on Theorem 1. and Proposition 1., trial specific intensity and cumulative intensity functions (defined on t′ ∈ [0, W
                               <sub>
                                n
                               </sub>
                               ]) are then given by,
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-27">
                               The relatively simple form of transformed cumulative intensities is a consequence of the uniform time rescaling (see
                               <a id="xref-sec-19-3" class="xref-sec" href="#sec-19">
                                Supplementary Materials
                               </a>
                               for proof). Using these two related functions, the data log-likelihood implied by
                               <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">
                                equation 1
                               </a>
                               is
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               which retains the required flexibility, while obviating the calculation of the intractable integral. The total data likelihood for all the trials and neurons is then
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                              <p id="p-28">
                               In principle, each recorded neuron in the experiment could be assigned a separate intensity function. However, given the experimental methodology of only recording some neurons on some trials in the Steinmetz dataset, the problem of missing data would be radically acute, and the model would be uninterpretable. The computational cost would also be prohibitive. Instead, we make the simplifying assumptions that all the neurons in each of the 42 brain regions identified by [
                               <a id="xref-ref-35-7" class="xref-bibr" href="#ref-35">
                                Steinmetz et al. 2019
                               </a>
                               ] and (as is true by design) the 4 regions in the synthetic model share common intensity functions. As such, we consider
                               <span class="inline-formula" id="inline-formula-32">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-33">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-33.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-33.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                             </div>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              2.2 Behavioural response models
                             </h3>
                             <p id="p-29">
                              Having provided a way of characterizing the neural response to the embedding
                              <strong>
                               h
                              </strong>
                              <sub>
                               n
                              </sub>
                              , we next need to model the link between neural representations and behaviour. We assume that the probability of making a behavioural response at each point in time depends on the activity of the neurons at that time. In turn, these are driven by the stimulus
                              <strong>
                               h
                              </strong>
                              <sub>
                               n
                              </sub>
                              on that trial. That is, the behavioural responses are indirectly affected by the stimulus via neural activities. However, rather than model this dependence explicitly, which is hard given the punctate nature of the response, we approximate it implicitly, via smooth intensity functions that in turn depend on Λ
                              <sup>
                               N
                              </sup>
                              .
                             </p>
                             <p id="p-30">
                              The intensity function for an action a is denoted by
                              <span class="inline-formula" id="inline-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , which specifies the instantaneous probability of taking the action at time t relative to stimulus onset. The superscript B indicates that the intensity function is for the Behavioural data. A key simplification is to allow for the theoretical possibility that the animal performs the same action more than once on a trial; or performs both actions. However, that actions are actually sparse implies that this approximation is not too costly. We write the canonical behavioural cumulative intensity function as a function of the canonical neural cumulative intensity functions.
                              <span class="disp-formula" id="disp-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-31">
                              Function Φ
                              <sub>
                               a
                              </sub>
                              (.) can be realised using a deep feed-forward network which can represent arbitrary dependencies between neural activities and behavioural responses.
                             </p>
                             <p id="p-32">
                              Then, differentiating:
                              <span class="disp-formula" id="disp-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              which implies that the behavioural response probability at each point in time is indirectly dependent on the stimulus through the spike rate of different neural activities, as desired. Function Φ(.) is also designed to be increasing in t to ensure that
                              <span class="inline-formula" id="inline-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (see
                              <a id="xref-sec-9-2" class="xref-sec" href="#sec-9">
                               Section Model structure
                              </a>
                              ).
                             </p>
                             <p id="p-33">
                              These response rates are presented as a function of canonical neural intensity functions. Similar to 5 for each trial n we have,
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-34">
                              The subjects can either act or not on a trial; the latter is determined by a censoring window W. Write
                              <span class="inline-formula" id="inline-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              as the set of trials on which action a was taken before W, at reaction time r
                              <sub>
                               n
                              </sub>
                              . The simplified joint probability distribution of the behavioural observations is then:
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and, taking logs, the log-likelihood for those observations is,
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-35">
                              Over the whole experiment and actions, the behavioural likelihood can be defined as,
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              in which
                              <span class="inline-formula" id="inline-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is the set of available actions.
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              2.3 Model structure
                             </h3>
                             <p id="p-36">
                              We implement the model using the neural network architecture shown in
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              . This has three components. The first maps the stimulus
                              <strong>
                               x
                              </strong>
                              <sub>
                               n
                              </sub>
                              that was presented through a series of fully connected layers to realize an input embedding denoted by
                              <strong>
                               h
                              </strong>
                              <sub>
                               n
                              </sub>
                              . The second component takes the embedding
                              <strong>
                               h
                              </strong>
                              <sub>
                               n
                              </sub>
                              and t and outputs the modelled activity of each neural region u at time t in the form of cumulative intensity functions for
                              <span class="inline-formula" id="inline-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . This component is designed such that the outputs of the network, i.e.,
                              <span class="inline-formula" id="inline-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , are monotonic functions of t to ensure that their gradients with respect to t (which are neural intensity functions) are always positive. To achieve this, following ideas from [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               Sill 1998
                              </a>
                              ;
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               Chilinski and Silva 2018
                              </a>
                              ;
                              <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">
                               Omi, Aihara, et al. 2019
                              </a>
                              ], the weights of the network are constrained to be positive and tanh activation functions are used in the middle layers and soft-plus in the output layers.
                             </p>
                             <p id="p-37">
                              The third component of the model takes the neural cumulative intensity functions and maps them to the behavioural cumulative intensity functions (function Φ in
                              <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                               equation 7
                              </a>
                              ). We used the same method as for the second component to ensure that the gradient of
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              with respect to t is positive.
                             </p>
                             <p id="p-38">
                              For training the model, the neural loss function
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is used to train all the weights from stimulus to neural cumulative intensity functions (blue and red rectangles in
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              ). Given these trained neural cumulative intensity functions, then the weights connecting neural outputs to behavioural outputs are trained using
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The gradients were obtained using automatic differentiation in Tensorflow [
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               Abadi et al. 2015
                              </a>
                              ]. See
                              <a id="xref-sec-19-4" class="xref-sec" href="#sec-19">
                               Supplementary Materials
                              </a>
                              for details.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             3 Results
                            </h2>
                            <div id="sec-12" class="subsection">
                             <h3>
                              3.1 Data structure
                             </h3>
                             <div id="sec-13" class="subsection">
                              <h4>
                               Synthetic dataset
                              </h4>
                              <p id="p-39">
                               For the synthetic data, we use data generated from the model introduced in [
                               <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">
                                Wimmer et al. 2015
                               </a>
                               ]. Activities from two direction-selective sensory regions (E1 and E2; e.g. V5/MT) as well as two integrator regions (D1 and D2; e.g. LIP, FEF) are modeled and then observed. Each region has 240 neurons and the whole experiment consists of 1800 trials (1200 trials for training and 600 trials for testing). Left and right sensory regions prefer leftwards and rightwards motion respectively; time-varying activity in these regions inspired by stimuli with coherence levels varying from completely obscure: 0%, to rather definite: 50% or 80% (which are encoded using one-hot encoding), are accumulated by populations in the integration region. The latter has attractor dynamics; a response is realized when the activity state is sufficiently close to one of the two attractors. A choice is modelled as being made when the average activity of a (trial-)random subset of neurons in D1 or D2 over a window of 50ms reaches 40Hz. This corresponds to strong evidence in favour of motion in the corresponding direction which can be
                               <span class="sc">
                                right
                               </span>
                               (for D1) or
                               <span class="sc">
                                left
                               </span>
                               (for D2).
                              </p>
                             </div>
                             <div id="sec-14" class="subsection">
                              <h4>
                               Steinmetz dataset
                              </h4>
                              <p id="p-40">
                               We use the data reported in [
                               <a id="xref-ref-35-8" class="xref-bibr" href="#ref-35">
                                Steinmetz et al. 2019
                               </a>
                               ]
                               <a id="xref-fn-2-1" class="xref-fn" href="#fn-2">
                                <sup>
                                 2
                                </sup>
                               </a>
                               . The experiments consist of 38 sessions of a visual discrimination task (
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Figure 1a
                               </a>
                               ). Activities from 42 brain regions from the left hemisphere of the brain were recorded (not all regions were recorded in all the sessions). Overall the data from 30,000 neurons were recorded and the whole experiment consisted of 10011 trials. We used the data from 12 sessions for testing and the rest for training the model.
                              </p>
                              <p id="p-41">
                               On each trial, the animals were presented with stimuli on the left and right and were required to turn a wheel LEFT,
                               <span class="sc">
                                right
                               </span>
                               or keep it fixed (NoGo), based on the contrast input (four possible levels of contrast on each side: 0, 25, 50, 100%; 0% on both sides requires NoGo). We encoded stimulus contrast using one-hot encoding based on which side had a higher contrast, or whether they had equal contrasts (
                               <strong>
                                x
                               </strong>
                               <sub>
                                n
                               </sub>
                               of dimension 3). The reaction time r
                               <sub>
                                n
                               </sub>
                               corresponds to the beginning of the wheel turn if this happens before the end of the response window.
                              </p>
                             </div>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              3.2 Training process
                             </h3>
                             <p id="p-42">
                              All the weights in the model were trained using the Adam optimiser [
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               Kingma and Ba 2014
                              </a>
                              ]. Stimulus and integrator components (blue and red rectangles in
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              ) each were composed of three fully connected hidden layers with 20 neurons in each layer. The third layer was then connected to the output layer consisting of one neuron per region in the dataset. The neural component was followed by two fully connected behavioural layers for each action (with 10 neurons). ‘softplus’ and ‘tanh’ activation functions were used to ensure positive of intensity functions. See
                              <a id="xref-sec-19-5" class="xref-sec" href="#sec-19">
                               Supplementary Materials
                              </a>
                              for more details about the model architecture and training process.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              3.3 Experiments
                             </h3>
                             <p id="p-43">
                              We show statistics of the quality of the fit of the model later. However, we first illustrate the neural and behavioural properties of the model by freezing the weights and performing simulations with different values of Wn for sample regions of both synthetic and Steinmetz datasets. Results showing the performance of the model on all the available test regions of both datasets are presented in
                              <a id="xref-sec-19-6" class="xref-sec" href="#sec-19">
                               Supplementary Materials
                              </a>
                              .
                             </p>
                             <p id="p-44">
                              The solid lines in the upper panels of
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              illustrate the neural responses for the synthetic dataset for 0.3 ≤ W
                              <sub>
                               n
                              </sub>
                              ≤ 0.6 when the stimulus had highest coherence level (0.8) and moved
                              <span class="sc">
                               right
                              </span>
                              . The chosen interval includes more than 90% of all trials in the coherence level of 0.8. These results are compared with the empirical activity derived from the data (dashed lines) for both integrator regions D1 and D2. Note the change in y-scale between the plots, given the strong left stimulus. These values are closely related to the learned intensities and capture the observed variability in response times between different trials given the time rescaling of input spike trains.
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F3.large.jpg?width=800&height=600&carousel=1" title="Synthetic Data (Upper panels). The activity rate of neural population in the two integrator regions of the synthetic dataset. The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with Wn in a specific interval illustrated by the colorbar (see section F in Supplementary Materials). (Lower panels). Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with highest coherence level. The plots show the average between trials with a right reaction. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Synthetic Data (Upper panels).</span> The activity rate of neural population in the two integrator regions of the synthetic dataset. The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with <em>W<sub>n</sub></em> in a specific interval illustrated by the colorbar (see section F in Supplementary Materials). <strong>(Lower panels).</strong> Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with highest coherence level. The plots show the average between trials with a right reaction. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F3.medium.gif" width="440" height="281"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F3.medium.gif" width="440" height="281"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                Synthetic Data (Upper panels).
                               </span>
                               <p id="p-45" class="first-child">
                                The activity rate of neural population in the two integrator regions of the synthetic dataset. The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with W
                                <sub>
                                 n
                                </sub>
                                in a specific interval illustrated by the colorbar (see
                                <a id="xref-sec-32-1" class="xref-sec" href="#sec-32">
                                 section F
                                </a>
                                in
                                <a id="xref-sec-19-7" class="xref-sec" href="#sec-19">
                                 Supplementary Materials
                                </a>
                                ).
                                <strong>
                                 (Lower panels).
                                </strong>
                                Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with highest coherence level. The plots show the average between trials with a
                                <span class="sc">
                                 right
                                </span>
                                reaction. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-46">
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              lower panels show the mean rate of behavioural responses for each interval of learned neural intensities for the complete set of trials. As expected, the highest rate of behavioural action is observed for neural intensities close to 40 spikes/sec for D1. By contrast, region D2 shows very low behavioural activity rates due to the non-favorable direction of stimulus in these trials. Note that the shown empirical firing rates are averaged across all responses, while the intensities correspond to different response times, which helps explain the differences between the two measures. For example, in very fast responses, there is an initial burst in the firing rates, which is captured by the initial sharp rise in the intensities (see
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              ), but this is invisible in the averaged firing rates.
                             </p>
                             <p id="p-47">
                              The upper panels of
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              show modelled and actual neural activities in the Steinmetz dataset for two example brain regions with high contrast levels of stimulus on
                              <span class="sc">
                               right
                              </span>
                              : one the subiculum (
                              <span class="sc">
                               sub
                              </span>
                              ), which [
                              <a id="xref-ref-35-9" class="xref-bibr" href="#ref-35">
                               Steinmetz et al. 2019
                              </a>
                              ] reported as containing neurons that consistently fired before wheel turns regardless of their direction (arguably a surprising feature of this dataset, given the relationship of the subiculum with areas such as the hippocampus and entorhinal cortex rather than motor regions), and the other, visual (
                              <span class="sc">
                               vis
                              </span>
                              p; primary visual area), which is reported to have the highest portion of visual encoding neurons. The activities were all recorded in the left hemisphere, and therefore, the right stimulus/action are contralateral to the recording sites. Comparing the lower and upper panels of
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              show, we generally see more activity on the left side in
                              <span class="sc">
                               vis
                              </span>
                              p for stimuli with high contrast levels on the
                              <span class="sc">
                               right
                              </span>
                              which is consistent with previous analyses [
                              <a id="xref-ref-35-10" class="xref-bibr" href="#ref-35">
                               Steinmetz et al. 2019
                              </a>
                              ]. It is clear that the region shows lower neural activity levels for ipsilateral stimuli. The subiculum (
                              <span class="sc">
                               sub
                              </span>
                              ), is reported in [
                              <a id="xref-ref-35-11" class="xref-bibr" href="#ref-35">
                               Steinmetz et al. 2019
                              </a>
                              ] as containing neurons that consistently fired before wheel turns regardless of their direction. We can see in
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              lower panels that this indeed is captured by our model for the SUB region where it reaches similar firing rates no matter the direction of stimulus and motion are. The relative timing of the activities in the areas are also consistent with expectations. The solid lines in the panels show estimated firing rates, which are consistent with the data (dashed lines) in particular for
                              <span class="sc">
                               vis
                              </span>
                              p which had a peak in firing rate early after stimulus was presented (see below) and closely related to the presented visual stimuli in the task. Note that in our framework, estimation of the neural activities (firing rates) does not rely on selecting a temporal bin size. This is unlike most previous state-of-the-art works [
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               Liu and Lengyel 2021
                              </a>
                              ] where the output firing rates are substantially affected by the choice of bin size.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F4.large.jpg?width=800&height=600&carousel=1" title="Steinmetz (Upper panels). The activity rate of neural population in the two sample regions of the Steinmetz dataset for the highest contrast level on right. The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with Wn in a specific interval illustrated by the colorbar (see section F in Supplementary Materials). (Middle panels). Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with high contrast on right. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval. (Lower panels). Neural activities for contralateral and ipsilateral stimuli. The visp shows a direction selective activity pattern, preferring the contralateral stimuli on the right. However this is not the case for SUB which is in agreement with reports in [Steinmetz et al. 2019] where this region is known to contain firing neurons before the motion initiation regardless of the direction of stimulus and movement." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Steinmetz (Upper panels).</span> The activity rate of neural population in the two sample regions of the Steinmetz dataset for the highest contrast level on right. The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with <em>W<sub>n</sub></em> in a specific interval illustrated by the colorbar (see section F in Supplementary Materials). <strong>(Middle panels).</strong> Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with high contrast on right. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval. <strong>(Lower panels).</strong> Neural activities for contralateral and ipsilateral stimuli. The visp shows a direction selective activity pattern, preferring the contralateral stimuli on the right. However this is not the case for SUB which is in agreement with reports in [Steinmetz et al. 2019] where this region is known to contain firing neurons before the motion initiation regardless of the direction of stimulus and movement.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F4.medium.gif" width="440" height="403"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F4.medium.gif" width="440" height="403"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Steinmetz (Upper panels).
                               </span>
                               <p id="p-48" class="first-child">
                                The activity rate of neural population in the two sample regions of the Steinmetz dataset for the highest contrast level on
                                <span class="sc">
                                 right
                                </span>
                                . The dashed lines show the empirically derived firing rates compared to the solid lines which are estimated using the proposed model. Each line plot corresponds to the average activity rate of trials with W
                                <sub>
                                 n
                                </sub>
                                in a specific interval illustrated by the colorbar (see
                                <a id="xref-sec-32-2" class="xref-sec" href="#sec-32">
                                 section F
                                </a>
                                in
                                <a id="xref-sec-19-8" class="xref-sec" href="#sec-19">
                                 Supplementary Materials
                                </a>
                                ).
                                <strong>
                                 (Middle panels).
                                </strong>
                                Illustrating the average response rate for each interval of neural activity rates. The plots correspond to the trials with high contrast on
                                <span class="sc">
                                 right
                                </span>
                                . Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval.
                                <strong>
                                 (Lower panels).
                                </strong>
                                Neural activities for contralateral and ipsilateral stimuli. The
                                <span class="sc">
                                 vis
                                </span>
                                p shows a direction selective activity pattern, preferring the contralateral stimuli on the
                                <span class="sc">
                                 right
                                </span>
                                . However this is not the case for SUB which is in agreement with reports in [
                                <a id="xref-ref-35-12" class="xref-bibr" href="#ref-35">
                                 Steinmetz et al. 2019
                                </a>
                                ] where this region is known to contain firing neurons before the motion initiation regardless of the direction of stimulus and movement.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-49">
                              Next, examining the behavioural predictions of our model, the neural activities presented in
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              show that sooner responses (with high probabilities) are strongly related to the peak intensities in
                              <span class="sc">
                               vis
                              </span>
                              p. In agreement with these findings, middle right panel in
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              shows higher probability for the occurrence of reactions when the neural activity in
                              <span class="sc">
                               vis
                              </span>
                              p peaks.
                              <span class="sc">
                               sub
                              </span>
                              is also coupled to behaviour in the middle panel – in fact for both directions of movement.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              3.4 Baseline Methods
                             </h3>
                             <p>
                              Finally,
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              shows a comparison of the performance of our proposed framework to those of recent prominent baseline point process estimators. We compare the negative log likelihoods (NLL) for the neural activity on sample regions of Steinmetz dataset as well as the 4 regions of the synthetic dataset. For details on the utilized settings for implementing the baseline methods, please see the
                              <a id="xref-sec-19-9" class="xref-sec" href="#sec-19">
                               Supplementary Materials
                              </a>
                              .
                             </p>
                             <ul class="list-unord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-51">
                                <strong>
                                 GLM Model [
                                 <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                  Truccolo et al. 2005
                                 </a>
                                 ;
                                 <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                  Paninski 2004
                                 </a>
                                 ]
                                </strong>
                                : Generalized-linear models (GLM) also known as Poisson regression, are used to model the intensity of the input data as a linear combination of time-dependent covariates. Here, the total spike counts for all trials are calculated and concatenated for count windows of 5ms as inputs to the GLM model.
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-52">
                                <strong>
                                 NHPoisson Model [
                                 <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                                  Cebrian 2015
                                 </a>
                                 ]
                                </strong>
                                : NHPoisson is a method for the modelling non homogeneous Poisson processes in time estimating maximum likelihood. The model is based on formulating the intensity as a function of time-dependent covariates.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-53">
                                <strong>
                                 Universal Count Model [
                                 <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                                  Liu and Lengyel 2021
                                 </a>
                                 ]
                                </strong>
                                : This model builds on sparse Gaussian processes (GP) to capture arbitrary spike count distributions flexibly relying on both observed and latent covariates. It uses scalable variational inference and can jointly infer the covariate-to-spike count distribution mappings and latent trajectories. We also examine a second variant of this model which replaces the GP-based approaches with an artificial neural network (ANN) mapping. We denote the two variations by U-GP and U-ANN respectively.
                               </p>
                              </li>
                              <li id="list-item-4">
                               <p id="p-54">
                                <strong>
                                 Poisson Gaussian-Process Latent Variable Model (P-GPL) [
                                 <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                  Wu et al. 2017
                                 </a>
                                 ]
                                </strong>
                                : In this model, Poisson spiking observations are accompanied by two underlying Gaussian processes: One governing a temporal latent variable, the other governing a set of nonlinear tuning curves. The model learns using a decoupled Laplace approximation which is a fast approximate inference method. The same set of temporal covariates as above are also utilized in the implementation of this method.
                               </p>
                              </li>
                             </ul>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2385477/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2385477/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2385477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1:
                               </span>
                               <p id="p-55" class="first-child">
                                Comparison of the Negative Log-Likelihood (NLL) measure for the neural intensity function estimations in example regions of the synthetic and Steinmetz datasets.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-56">
                              It is important to mention that the performance of all the baseline methods depend on the length of the selected time bin for spike count calculation; a constraining dependency causing non-robust results that our proposed method overcomes. We selected the time bin length for optimal performance by performing a heuristic search in a rational range.
                             </p>
                             <p id="p-57">
                              Note that the average spike counts per region in the Steinmetz dataset is roughly 60 times less than in the synthetic one. This is due to the difference between the lengths of the experiment and the numbers of neurons per region in the Steinmetz and synthetic dataset. Thereby, the NLL measures differ by two orders of magnitude.
                             </p>
                             <p id="p-58">
                              Finally,
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Figure 5
                              </a>
                              shows the comparison of the performance of the proposed model with the baseline methods in estimating neural activities summed over all the 37 regions in the test data of Steinmetz dataset (including the 4 regions reported in
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F5.large.jpg?width=800&height=600&carousel=1" title="Total NLL of the estimated neural intensity function on Steinmetz test set. Illustrating the sum of NLL values evaluated over all the 37 test regions in the Steinmetz dataset using different baselines compared to the performance of the proposed method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Total NLL of the estimated neural intensity function on Steinmetz test set.</span> Illustrating the sum of NLL values evaluated over all the 37 test regions in the Steinmetz dataset using different baselines compared to the performance of the proposed method.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F5.medium.gif" width="440" height="305"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F5.medium.gif" width="440" height="305"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Total NLL of the estimated neural intensity function on Steinmetz test set.
                               </span>
                               <p id="p-59" class="first-child">
                                Illustrating the sum of NLL values evaluated over all the 37 test regions in the Steinmetz dataset using different baselines compared to the performance of the proposed method.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-18">
                            <h2 class="">
                             4 Discussion
                            </h2>
                            <p id="p-60">
                             We presented a novel framework for linking neural spike trains to sensory inputs and behaviour. The framework extended previous works on fMRI data [
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              Dezfouli et al. 2018
                             </a>
                             ] by using a flexible Point process framework. The model was able to learn a suitable encoding of the stimulus and provided a joint explanation for both behavioural and neural data that could be used to recover correlational links between neural and behavioural activities. Unlike previous efforts, the learning process of the proposed model is independent of the selection of a time bin for spike count calculations obtaining higher robustness. The current method represents the dependency of neural activities on stimulus and trial duration, but not on previous neural activities – thus capturing signal rather than noise correlations (although the latter are an obvious target for future work). There are many additional directions for future work: capturing richer aspects of behaviour that are known to couple to neural activity [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Balleine and O’Doherty 2010
                             </a>
                             ]; integrating and/or substituting spiking activity with calcium imaging; like auto-regressive linear-nonlinear-Poisson (LNP) models [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              Chichilnisky 2001
                             </a>
                             ]; differentiating more finely the activity in different regions (including neurons with opposite stimulus coding). Nevertheless, we suggest that our method casts brain and behaviour interactions in a compelling new light.
                            </p>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Supplementary Materials
                            </h2>
                            <div class="section" id="sec-20">
                             <h3>
                              A Proof of Theorem 1
                             </h3>
                             <p id="p-61">
                              Following the Mapping Theorem of an inhomogeneous Poisson process [
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               Grimmett and Stirzaker 2020
                              </a>
                              ], and given a one-to-one monotonic transformation function, z
                              <sub>
                               n
                              </sub>
                              (t′) = t, between the original inhomogeneous Poisson point process 0 < s′
                              <sup>
                               1
                              </sup>
                              < s′
                              <sup>
                               2
                              </sup>
                              <,…,< s′
                              <sup>
                               j
                              </sup>
                              ≤ W
                              <sub>
                               n
                              </sub>
                              ≤ W and 0 < s
                              <sup>
                               1
                              </sup>
                              < s
                              <sup>
                               2
                              </sup>
                              <,…, < s
                              <sup>
                               j
                              </sup>
                              ≤ W, let 0 < s
                              <sup>
                               1
                              </sup>
                              < s
                              <sup>
                               2
                              </sup>
                              <,…, < s
                              <sup>
                               j
                              </sup>
                              ≤ W represent a set of event (spike) times from a second inhomogeneous Poisson point process. Let g(t) represent the corresponding event time probability density. Given g(t) is a density function (measurable and non-negative function), following the push-forward probability density, we get:
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where z
                              <sup>
                               -1
                              </sup>
                              is the inverse of the transformation, z
                              <sub>
                               n
                              </sub>
                              (.), and f (t′) is the event time probability density function of the original Poisson point process.
                             </p>
                             <p id="p-62">
                              Now, for t ∈ (0, W], let N(t) be the sample path of the associated counting process. The sample path is a right continuous function that jumps 1 at the event times and is constant otherwise. Then, we compute the probability that a spike s
                              <sup>
                               k
                              </sup>
                              occurs in [t, t+Δt) where k = N(t) + 1. Note that events {N(t+Δt)–N(t) = 1} and {s
                              <sup>
                               k
                              </sup>
                              < t+Δt |s
                              <sup>
                               k
                              </sup>
                              > t} are equivalent. Thereby,
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-63">
                              From the definition of conditional probability:
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-64">
                              Therefore, we get:
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-65">
                              Using
                              <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                               equation 13
                              </a>
                              , we obtain:
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-25.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-25.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-66">
                              We also have
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and hence:
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-67">
                              Thereby, inserting the above in
                              <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                               equation 17
                              </a>
                              and by a change of variable u to u′, noting that
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we get:
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where N(t′) is the sample path of the associated counting process for t′ ∈ (0, W
                              <sub>
                               n
                              </sub>
                              ]. The intensity function of a point process, λ(t), can be written as:
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              hence, since we proved P(N(t + Δt) < N(t) = 1) = P(N(t + Δt)′ – N(t′) = 1), we get:
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-68">
                              Finally, the intensity function of the new count process λ(t) equals λ
                              <sub>
                               n
                              </sub>
                              (t′) of the original Poisson point process, thereby satisfying the four properties of a Poisson point process [
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               Ross et al. 1996
                              </a>
                              ] and we have now established our result.
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Figure 6
                              </a>
                              is schematically illustrating this result in case of a linear transformation function.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F6.large.jpg?width=800&height=600&carousel=1" title="Illustrating the transformation of the Poisson point process in real time , to the rescaled time domain which results a point process with intensity function λ(t). The transformation function  performs a one-to-one monotonic mapping of the events in real time (s′j) to the events on the rescaled time axis (sj)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Illustrating the transformation of the Poisson point process in real time , to the rescaled time domain which results a point process with intensity function λ(<em>t</em>). The transformation function  performs a one-to-one monotonic mapping of the events in real time (<em>s<sup>′j</sup></em>) to the events on the rescaled time axis (<em>s<sup>j</sup></em>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F6.medium.gif" width="440" height="439"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F6.medium.gif" width="440" height="439"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <p id="p-69" class="first-child">
                                Illustrating the transformation of the Poisson point process in real time
                                <span class="inline-formula" id="inline-formula-45">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F6/embed/inline-graphic-45.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F6/embed/inline-graphic-45.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , to the rescaled time domain which results a point process with intensity function λ(t). The transformation function
                                <span class="inline-formula" id="inline-formula-46">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F6/embed/inline-graphic-46.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/F6/embed/inline-graphic-46.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                performs a one-to-one monotonic mapping of the events in real time (s
                                <sup>
                                 ′j
                                </sup>
                                ) to the events on the rescaled time axis (s
                                <sup>
                                 j
                                </sup>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F7.large.jpg?width=800&height=600&carousel=1" title="Model Architecture. Schematic representing the details of the implemented neural network to be trained on the Steinmetz dataset. The neural and behavioural outputs of this network are multiplied by a factor of Wn/W for the selected linear transformation function, z, to obtain the final model readouts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Model Architecture.</span> Schematic representing the details of the implemented neural network to be trained on the Steinmetz dataset. The neural and behavioural outputs of this network are multiplied by a factor of <em>W<sub>n</sub>/W</em> for the selected linear transformation function, <em>z</em>, to obtain the final model readouts.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F7.medium.gif" width="440" height="210"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F7.medium.gif" width="440" height="210"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7:
                               </span>
                               <span class="caption-title">
                                Model Architecture.
                               </span>
                               <p id="p-70" class="first-child">
                                Schematic representing the details of the implemented neural network to be trained on the Steinmetz dataset. The neural and behavioural outputs of this network are multiplied by a factor of W
                                <sub>
                                 n
                                </sub>
                                /W for the selected linear transformation function, z, to obtain the final model readouts.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-21">
                             <h3>
                              B Proof of Proposition 1
                             </h3>
                             <p id="p-71">
                              The cumulative intensity function of a point process with intensity function λ
                              <sub>
                               n
                              </sub>
                              (t′) is given by:
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for t′ ∈ [0, W
                              <sub>
                               n
                              </sub>
                              ], which can be (automatically) differentiated to produce the intensity function,
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-72">
                              Given a monotonic transformation function z
                              <sub>
                               n
                              </sub>
                              (t′) = t, and since λ
                              <sub>
                               n
                              </sub>
                              (t′) = λ(t) (see A), we get:
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-73">
                              Now, following integration by parts,
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-74">
                              Inserting the special case of a linear transformation function where
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is a constant, the above equation reduces to:
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <p id="p-75">
                              In this work, we rescaled time using function z
                              <sub>
                               n
                              </sub>
                              (t′) = t′W/W
                              <sub>
                               n
                              </sub>
                              where z
                              <sub>
                               n
                              </sub>
                              : [0, W
                              <sub>
                               n
                              </sub>
                              ] → [0, W] and we chose to stretch the firing rates. Thereby, trial specific cumulative intensity functions are simply defined as:
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                            </div>
                            <div class="section" id="sec-22">
                             <h3>
                              C Model architecture and training
                             </h3>
                             <p id="p-76">
                              In this section we explain the details of the model architecture.
                             </p>
                             <div class="section" id="sec-23">
                              <h4>
                               C.1 Steinmetz dataset
                              </h4>
                              <p id="p-77">
                               The input stimulus
                               <strong>
                                x
                               </strong>
                               <sub>
                                n
                               </sub>
                               is passed through two dense layers with 20 units (
                               <span class="sc">
                                softplus
                               </span>
                               activation). The output is then passed through a layer with |
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               | =50 units using a linear activation function to create the stimulus embedding,
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               . There is no constraint on the weights in these layers.
                              </p>
                              <p id="p-78">
                               As mentioned in the main text, we assumed the time rescaling function is linear. The elapsed time t′ ∈ [0, W
                               <sub>
                                n
                               </sub>
                               ] (since the stimulus) is first scaled by a factor of W/W
                               <sub>
                                n
                               </sub>
                               to obtain t ∈ [0, W] and is then passed through a linear layer (with the same number of units as |
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               | = 50) with the weights constrained to be non-negative. The resulting output is added to
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               , with the sum then being passed through a
                               <span class="sc">
                                tanh
                               </span>
                               activation function.
                              </p>
                              <p id="p-79">
                               The output of this
                               <span class="sc">
                                tanh
                               </span>
                               activation is passed through a dense layer with 20 units and a further
                               <span class="sc">
                                tanh
                               </span>
                               activation function (first neural layer) and then through another dense layer with 42 units and
                               <span class="sc">
                                softplus
                               </span>
                               activation function (second neural layer). The weights of these layers are all constrained to be non-negative. The outputs of this layer correspond to
                               <span class="inline-formula" id="inline-formula-48">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-48.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-48.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , referred to as neural outputs, and are multiplied by W
                               <sub>
                                n
                               </sub>
                               /W in the network readout to obtain
                               <span class="inline-formula" id="inline-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-49.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-49.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for u =1… 42 which is then passed to the loss function.
                              </p>
                              <p id="p-80">
                               For modelling behavioural data, the neural output
                               <span class="inline-formula" id="inline-formula-50">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-50.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-50.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is passed through a dense layer with 10 units (
                               <span class="sc">
                                tanh
                               </span>
                               activation) and then through another dense layer with 1 unit (
                               <span class="sc">
                                softplus
                               </span>
                               activation) to produce
                               <span class="inline-formula" id="inline-formula-51">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-51.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-51.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The weights are all constrained to be non-negative.
                              </p>
                              <p id="p-81">
                               The neural output
                               <span class="inline-formula" id="inline-formula-52">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-52.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-52.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is also passed through a dense layer with 10 units (
                               <span class="sc">
                                tanh
                               </span>
                               activation) and then through another dense layer with 1 unit (
                               <span class="sc">
                                softplus
                               </span>
                               activation) to produce
                               <span class="inline-formula" id="inline-formula-53">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-53.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-53.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The weights of all the layers are constrained to be non-negative.
                              </p>
                              <p id="p-82">
                               Note that the path from t to neural and behavioural outputs only contains positive weights which, together with monotonic activation functions, is a sufficient condition to guarantee that the outputs (neural and behavioural) are monotonic functions of t [
                               <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                                Sill 1998
                               </a>
                               ;
                               <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                                Chilinski and Silva 2018
                               </a>
                               ;
                               <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">
                                Omi, Aihara, et al. 2019
                               </a>
                               ].
                              </p>
                              <p id="p-83">
                               <span class="inline-formula" id="inline-formula-54">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-54.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-54.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-55">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               were multiplied by factor of W
                               <sub>
                                n
                               </sub>
                               /W to obtain trial specific values for
                               <span class="inline-formula" id="inline-formula-56">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-56.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-56.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-57">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-57.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/02/2020.07.13.201673/embed/inline-graphic-57.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-84">
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Figure 7
                               </a>
                               visualizes the details of the model structure implemented to be trained on the Steinmetz dataset.
                              </p>
                             </div>
                             <div class="section" id="sec-24">
                              <h4>
                               C.2 Synthetic dataset
                              </h4>
                              <p id="p-85">
                               The same model architecture was used for this dataset as explained above, but with the following differences: (i) the size of output neural layer was set to four for the four regions involved in this dataset; (ii) the size of the embedding was set to |
                               <strong>
                                h
                               </strong>
                               <sub>
                                n
                               </sub>
                               | = 20, which was smaller than for the Steinmetz dataset, since the synthetic dataset had a smaller number of regions.
                              </p>
                             </div>
                             <div class="section" id="sec-25">
                              <h4>
                               C.3 Training
                              </h4>
                              <p id="p-86">
                               Parameter adaptation was performed using the Adam optimizer with learning rates of 0.01 (Steinmetz) and 0.001 (synthetic). The neural loss function was used to train all the weightsup tothe neural outputs, and behavioural loss was used to train all the weights connecting neural outputs to the behavioural outputs. For the Steinmetz dataset, in each iteration of training the weights wereupdated using neural loss for 10 steps, and then the behavioural loss was used to update the weights for 420 steps (eac.i step is one update to the weights). The training process continued until no significant improvement op the losses (ontraining datn) was observed. For the synthetic dataset, in each iteration the neural loss was used to update the weights for 10 sieps, which was followed by training using the behavioural loss for 40 steps.
                              </p>
                              <p id="p-87">
                               For the generation of the synthetic data we used the code provided in this link
                               <a id="xref-fn-3-1" class="xref-fn" href="#fn-3">
                                <sup>
                                 3
                                </sup>
                               </a>
                               . We generated 1200 trials for training and 600 trials for lesfing. For the Steinmelz datateti wrused 12 sessionsfor testing and the rest for training. Note that since in each session only a subset of regions wnre recorded (42 regions in total), we had 42 regions for training the model and 37 regions for testing the model. The split of sessionsbetween training and test was chosen so as to obtain the maximum number of regions in bosh training and test datasets. The data was downloaded from this link
                               <a id="xref-fn-4-1" class="xref-fn" href="#fn-4">
                                <sup>
                                 4
                                </sup>
                               </a>
                               .
                              </p>
                              <p id="p-88">
                               Automatic differentiation in Tensorflow was used to calculate the intensities.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              D Effects of spike time rescaling on model performance
                             </h3>
                             <p id="p-89">
                              To illustrate the effectiveness of our proposed model and the increase in model performance compared to the case where the time rescaling block is disabled, we reproduced both networks and evaluated them on the synthetic dataset. The two networks were identical in all other aspects. The NLL measures computed on the test data for both networks in all 4 regions as well as the scores of the
                              <span class="sc">
                               no scaling
                              </span>
                              version of the model relative to the original model are listed in
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table 2
                              </a>
                              ; the positive score means that the proposed model (with time rescaling) is better than the case without rescaling.
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2385470/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2385470/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2385470" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 2:
                               </span>
                               <p id="p-90" class="first-child">
                                Comparison of the Negative Log-Likelihood (NLL) measure for the neural intensity function estimations in the original model with and without time rescaling.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              E Baseline Methods
                             </h3>
                             <p id="p-91">
                              For the comparison of our method with the available baselines, we use Generalized-linear models (GLM) [
                              <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                               Truccolo et al. 2005
                              </a>
                              ;
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               Paninski 2004
                              </a>
                              ] as well as a recent method for modelling non homogeneous Poisson processes (NHPoisson) [
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               Cebrian 2015
                              </a>
                              ].
                             </p>
                             <div class="section" id="sec-28">
                              <h4>
                               E.1 GLM
                              </h4>
                              <p id="p-92">
                               For comparison with a Poisson family GLM, one Poisson GLM with a continuous predictor (elapsed time), a categorical predictor with three/four levels for synthetic/Steinmetz dataset (stimuli coherence/contrast), as well as a behavioral predictor (the reaction times) is used to estimate the neural intensity function of each region. As the continuous predictor, we include a list of sinusoidal, tanh, and exponential functions of the elapsed time as covariates. The observed behavioral covariates are in the form of temporal arrays consisting of
                               <span class="sc">
                                no-action
                               </span>
                               :0,
                               <span class="sc">
                                right
                               </span>
                               reaction time, or -
                               <span class="sc">
                                left
                               </span>
                               reaction time in each time bin. Note that the sign of the provided reaction time is indicative of the direction of the motion. The total spike counts for all trials were then calculated and concatenated for count windows of 5ms. Due to the large data size and the use of aggregated spike counts from all trials, we did not use self-coupling terms for spike events. The spike counts were then used as targets for the GLM model, leading to a specific neural intensity function. The derived intensity function was then used to calculate the negative log-likelihood (NLL) based on
                               <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">
                                equation 1
                               </a>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-29">
                              <h4>
                               E.2 NHPoisson
                              </h4>
                              <p id="p-93">
                               We used the same procedure as above with the count windows equal to 0.5 ms for the Steinmetz and synthetic datasets to extract the regional activity rates exploiting the elapsed time, trial stimulus type, and the behavioral reaction times. The NHPoisson R package [
                               <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                                Cebrian 2015
                               </a>
                               ] requires binary information about whether a given event has occurred in each time bin. Thereby, we used a smaller count window with an order of magnitude similar to inter spike intervals. This enabled us to keep the average spike events per non empty time bin equal to 0.97 and 0.99 for Steinmetz and synthetic datasets respectively. A binary vector then stored the presence or absence of spikes in each time bin and then non-zero indices were fed to the NHPoisson model. Repeatedly, in addition to elapsed time arrays and the categorical array of stimuli coherence/contrast levels for synthetic/Steinmetz dataset, temporal arrays consisting of
                               <span class="sc">
                                no-action
                               </span>
                               :0,
                               <span class="sc">
                                right
                               </span>
                               reaction time, or -
                               <span class="sc">
                                left
                               </span>
                               reaction time in each time bin were also provided as behavioural covariates. We also examined a list of sinusoidal, tanh, and exponential functions of the elapsed time as covariates. Using the embedded Akaike information criterion (AIC) calculator in the package, the best covariate was selected and added to the model (lowest AIC score). The extracted neural intensity functions for each region were then obtained. The derived intensity functions were plugged into
                               <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">
                                equation 1
                               </a>
                               to obtain NLL values for the NHPoisson model.
                              </p>
                             </div>
                             <div class="section" id="sec-30">
                              <h4>
                               E.3 Universal Count Model
                              </h4>
                              <p id="p-94">
                               This universal probabilistic spike count model uses sparse Gaussian processes to derive spike count distributions. It consists of C Gaussian process (GP) priors, a basis expansion, and a linear-softmax mapping [
                               <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">
                                Liu and Lengyel 2021
                               </a>
                               ]. Using 1 ms time bins, the spike counts of all trials associated with each neuron (in the Steinmetz dataset) or the neuronal population (in the synthetic dataset) are calculated and concatenated together. Note that for the case of the synthetic dataset, since the generated spike time sequences are not associated with specific neuron IDs and represent the population activity, a single cumulative spike count sequence was fed to the model for training. Once again, we use the elapsed time, the categorical stimulus type associated with each trial repeated for the duration of the trial, as well as the behavioural temporal array (consisting of 0 (i.e. NoGo),
                               <span class="sc">
                                right
                               </span>
                               reaction time, or
                               <span class="sc">
                                left
                               </span>
                               reaction time in each time bin) as the observed covariates for the model. To implement the original model (U-GP), we set hyperparameter C = 3 as suggested in [
                               <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">
                                Liu and Lengyel 2021
                               </a>
                               ] and choose an elementwise linear-exponential basis expansion. We use 5 fold-cross validation on the data from each region and we cross-validate over the neuron dimension by using the train set to infer the latent states in the test data, and then evaluate the cross-validated log-likelihood of the fitted model. For the case of synthetic dataset, the cross validation is across the trials. The learning rate is set to 0.001 and we choose a tuple batch size (to indicate the trial structure of the data; for details please refer to [
                               <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">
                                Liu and Lengyel 2021
                               </a>
                               ]) equal to the number of time bins accumulated over all trials associated with each region. In this method, an inference model with log likelihood based objectives using variational inference is built upon the data and spike couplings are computed and added as well.
                              </p>
                              <p id="p-95">
                               In the second variation of the model (U-ANN), the same setting is implemented by replacing the GP mapping with an artificial neural network (ANN) mapping. In this model, a 10-fold cross validation is utilized. The remainder of the parameters and model settings are tuned according to the suggestions in [
                               <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">
                                Liu and Lengyel 2021
                               </a>
                               ].
                              </p>
                             </div>
                             <div class="section" id="sec-31">
                              <h4>
                               E.4 Poisson Gaussian-Process Latent Variable (P-GPL)
                              </h4>
                              <p id="p-96">
                               This model uses Poisson spiking observations and two underlying Gaussian processes [
                               <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                                Wu et al. 2017
                               </a>
                               ]. A fast approximate inference method called the decoupled Laplace approximation is applied to learn the model from data. A Gaussian process is first used to extract the nonlinear evolution of the latent dynamic in the form of a latent variable. A second GP then generates the log of the tuning curve as a nonlinear function of the latent variable. This curve is then mapped to a final tuning curve via an exponential link function to estimate the spike rates of each neuron (λ
                               <sub>
                                i
                               </sub>
                               (t) for the ith neuron) and henceforth obtain the population activity rate in each discrete time bin. Here, the spike count matrix consisting of spike counts in time bins of 1 ms for each neuron (or the whole population in the synthetic dataset) is used to construct a generative model of the latent structure underlying these data. The data from all trials for each neuron are once again concatenated. Both Sinusoid and deterministic Gaussian bump tuning curves are examined to estimate the latent processes. The deterministic Gaussian bump tuning curve was then selected to report the results due to higher performance on our data which was expected given the naturally 2D motion space which is present in the Steinmetz dataset. We first estimated the parameters for the mapping function using spike trains from all the neurons within the training dataset. Then these parameters were fixed and the latent process using spike trains from 70% of the test data were inferred (as suggested in [
                               <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">
                                Wu et al. 2017
                               </a>
                               ]). We then report the NLL measure comparing the estimated latent process generating λ
                               <sub>
                                i
                               </sub>
                               (t) values and the known empirical rates from the remaining test data averaged over all neurons in each region. The rest of the parameters are chosen according to [
                               <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">
                                Wu et al. 2017
                               </a>
                               ].
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-32">
                             <h3>
                              F Calculation of empirical rates
                             </h3>
                             <p id="p-97">
                              For the empirical firing rates, we divided the observation period into equally-sized bins each having 0.008s and 0.05s width for the Steintmetz and synthetic datasets respectively. Then, we calculated the total number of spikes in each period and normalized that by the total number of neurons and trials contributed to the spike set and also by the period duration (0.008s or 0.05s). Note that only spikes which were made before the response were included in the analysis. Moreover, given the selected stimulus type for each examined W
                              <sub>
                               n
                              </sub>
                              in
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Figure 4
                              </a>
                              , only trials which had end times in an interval of 37.5 ms before the W
                              <sub>
                               n
                              </sub>
                              were included. This interval was chosen to be 60 ms for the results in
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Figure 3
                              </a>
                              . This interval’s width was adjusted so that for each region in both datasets, at least 15% of all trials with the chosen stimulus type would fall within the interval before each of the 5 selected W
                              <sub>
                               n
                              </sub>
                              s in
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Figures 8
                              </a>
                              and
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               10
                              </a>
                              .
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F8.large.jpg?width=800&height=600&carousel=1" title="Synthetic Dataset. The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. As expected from the dataset architecture, for trials when the right choice is made, the activity in D1 represents an increasing trend until the response is made. This is in contrast with region D2 for these trials where there is little to no activity detected in this region. Neurons in both sensory regions, E1 and E2 show constant activity throughout each trial with higher activity detected by E1 neurons which prefer rightward motion." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Synthetic Dataset.</span> The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. As expected from the dataset architecture, for trials when the right choice is made, the activity in D1 represents an increasing trend until the response is made. This is in contrast with region D2 for these trials where there is little to no activity detected in this region. Neurons in both sensory regions, E1 and E2 show constant activity throughout each trial with higher activity detected by E1 neurons which prefer rightward motion.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F8.medium.gif" width="440" height="261"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F8.medium.gif" width="440" height="261"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8:
                               </span>
                               <span class="caption-title">
                                Synthetic Dataset.
                               </span>
                               <p id="p-98" class="first-child">
                                The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on
                                <span class="sc">
                                 right
                                </span>
                                . As expected from the dataset architecture, for trials when the
                                <span class="sc">
                                 right
                                </span>
                                choice is made, the activity in D1 represents an increasing trend until the response is made. This is in contrast with region D2 for these trials where there is little to no activity detected in this region. Neurons in both sensory regions, E1 and E2 show constant activity throughout each trial with higher activity detected by E1 neurons which prefer rightward motion.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-33">
                             <h3>
                              G Additional results
                             </h3>
                             <div class="section" id="sec-34">
                              <h4>
                               G.1 Synthetic dataset
                              </h4>
                              <p id="p-99">
                               Using the procedure explained in
                               <a id="xref-sec-32-3" class="xref-sec" href="#sec-32">
                                Section F
                               </a>
                               ,
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                Figure 8
                               </a>
                               shows the comparison of the model estimation to the empirically derived firing rates for all the regions in the synthetic dataset.
                              </p>
                              <p id="p-100">
                               To evaluate the performance of the model in terms of predicting the behaviour of the neural population and the reaction times, we used all trials with rightward motion and fed them to the trained network of the model to get the estimated neural and behavioural response rates. The average of the estimated behavioural response rates corresponding to each of a set of potential intervals of the estimated neural responses were then calculated over trials.
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Figure 9
                               </a>
                               shows the results for all 4 regions. The purple bars show the average behavioural response rate in each occurred neural activity interval. The orange bars indicate the proportion of trials which achieve each specific neural activity range among all
                               <span class="sc">
                                right
                               </span>
                               trials, hence its occurrence frequency. Comparing the sensory regions in this figure, the behavioural correlate is with a low firing rate for E2 and with significantly higher firing rates in E1 which is the rightward selective sensory region.
                              </p>
                              <div id="F9" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F9.large.jpg?width=800&height=600&carousel=1" title="Synthetic Dataset Illustrating the average estimated response rate for each interval of estimated neural activity. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval. As expected in the right trials and represented by purple bars, highest behaviour rates are detected in D1 when neural activities approach 40 spikes/s in the (32,36) and (36,40) range. On the other hand, there is little to no activity estimated for D2 which is a leftward preferring integration region. Highest response rates are achieved for when E1 neural activity is in the vicinity of 40 spikes/s (in the (36,40) range) and when the neural activity in E2 is in (12,14) range. These are in agreement with results from Figure 8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Synthetic Dataset</span> Illustrating the average estimated response rate for each interval of estimated neural activity. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval. As expected in the right trials and represented by purple bars, highest behaviour rates are detected in D1 when neural activities approach 40 spikes/s in the (32,36) and (36,40) range. On the other hand, there is little to no activity estimated for D2 which is a leftward preferring integration region. Highest response rates are achieved for when E1 neural activity is in the vicinity of 40 spikes/s (in the (36,40) range) and when the neural activity in E2 is in (12,14) range. These are in agreement with results from Figure 8.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F9.medium.gif" width="440" height="321"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 9:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F9.medium.gif" width="440" height="321"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 9:
                                </span>
                                <span class="caption-title">
                                 Synthetic Dataset
                                </span>
                                <p id="p-101" class="first-child">
                                 Illustrating the average estimated response rate for each interval of estimated neural activity. Error bars show the standard error. Purple bars show the average behavioural response rates. Orange bars indicate the proportion of trials with the occurrence of each neural response interval. As expected in the
                                 <span class="sc">
                                  right
                                 </span>
                                 trials and represented by purple bars, highest behaviour rates are detected in D1 when neural activities approach 40 spikes/s in the (32,36) and (36,40) range. On the other hand, there is little to no activity estimated for D2 which is a leftward preferring integration region. Highest response rates are achieved for when E1 neural activity is in the vicinity of 40 spikes/s (in the (36,40) range) and when the neural activity in E2 is in (12,14) range. These are in agreement with results from
                                 <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                                  Figure 8
                                 </a>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div class="section" id="sec-35">
                              <h4>
                               G.2 Steinmetz dataset
                              </h4>
                              <p id="p-102">
                               With a similar procedure as explained above,
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Figure 10
                               </a>
                               shows the comparison of the model estimation with the empirically derived firing rates for all the regions in the Steinmetz dataset. The occasional observed underfitting for some regions may be overcome by further forms of regularization in future works. The plots highlight the performance quality of the trained network on test regions. The temporal neural pattern in each region is well captured by the model outputs for different observation windows W
                               <sub>
                                n
                               </sub>
                               .
                              </p>
                              <p id="p-103">
                               The behavioural performances were also evaluated similar to above for the available test regions in the Steinmetz dataset.
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                Figure 11
                               </a>
                               illustrates the results. In each region, the highest response rates correspond to the neural activity values achieved near the end of trial in
                               <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                                Figure 10
                               </a>
                               . The purple bars show the average behavioural response rate in each neural activity interval. The orange bars show how frequently these were observed in each
                               <span class="sc">
                                right
                               </span>
                               trial; meaning the occurrence ratio of that specific neural response range over all
                               <span class="sc">
                                right
                               </span>
                               test trials.
                              </p>
                              <div id="F10" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-41.large.jpg?width=800&height=600&carousel=1" title="Steinmetz Dataset. The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. The results are obtained only using trials with the right side contrast higher than left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Steinmetz Dataset.</span> The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. The results are obtained only using trials with the right side contrast higher than left.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-41.medium.gif" width="329" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-41.medium.gif" width="329" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                  <ul class="highwire-figure-links inline">
                                   <li class="download-fig first">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-41.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                     Download figure
                                    </a>
                                   </li>
                                   <li class="new-tab last">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-41.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                     Open in new tab
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-42.large.jpg?width=800&height=600&carousel=1" title="Steinmetz Dataset. The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. The results are obtained only using trials with the right side contrast higher than left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Steinmetz Dataset.</span> The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on right. The results are obtained only using trials with the right side contrast higher than left.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-42.medium.gif" width="330" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-42.medium.gif" width="330" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                  <ul class="highwire-figure-links inline">
                                   <li class="download-fig first">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-42.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                     Download figure
                                    </a>
                                   </li>
                                   <li class="new-tab last">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F10/graphic-42.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                     Open in new tab
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 10:
                                </span>
                                <span class="caption-title">
                                 Steinmetz Dataset.
                                </span>
                                <p id="p-104" class="first-child">
                                 The red dashed lines are the empirically derived firing rates compared to the blue solid lines which are estimated using the proposed model. The plots correspond to the trials with high contrast on
                                 <span class="sc">
                                  right
                                 </span>
                                 . The results are obtained only using trials with the
                                 <span class="sc">
                                  right
                                 </span>
                                 side contrast higher than
                                 <span class="sc">
                                  left
                                 </span>
                                 .
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <div id="F11" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-43.large.jpg?width=800&height=600&carousel=1" title="Steinmetz Dataset. Illustrating the average estimated response rate for each interval of estimated neural activity. The purple bars show the average behavioural response rate and orange bars represent the occurrence ratio of that specific neural response interval among all right test trials. Error bars show the standard error. The size of error bars is invisible compared to the bar sizes in most cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Steinmetz Dataset.</span> Illustrating the average estimated response rate for each interval of estimated neural activity. The purple bars show the average behavioural response rate and orange bars represent the occurrence ratio of that specific neural response interval among all right test trials. Error bars show the standard error. The size of error bars is invisible compared to the bar sizes in most cases.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-43.medium.gif" width="297" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-43.medium.gif" width="297" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                  <ul class="highwire-figure-links inline">
                                   <li class="download-fig first">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-43.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                     Download figure
                                    </a>
                                   </li>
                                   <li class="new-tab last">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-43.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                     Open in new tab
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-44.large.jpg?width=800&height=600&carousel=1" title="Steinmetz Dataset. Illustrating the average estimated response rate for each interval of estimated neural activity. The purple bars show the average behavioural response rate and orange bars represent the occurrence ratio of that specific neural response interval among all right test trials. Error bars show the standard error. The size of error bars is invisible compared to the bar sizes in most cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48322722" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Steinmetz Dataset.</span> Illustrating the average estimated response rate for each interval of estimated neural activity. The purple bars show the average behavioural response rate and orange bars represent the occurrence ratio of that specific neural response interval among all right test trials. Error bars show the standard error. The size of error bars is invisible compared to the bar sizes in most cases.</div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-44.medium.gif" width="295" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 11:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-44.medium.gif" width="295" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                  <ul class="highwire-figure-links inline">
                                   <li class="download-fig first">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-44.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                     Download figure
                                    </a>
                                   </li>
                                   <li class="new-tab last">
                                    <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/02/2020.07.13.201673/F11/graphic-44.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                     Open in new tab
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 11:
                                </span>
                                <span class="caption-title">
                                 Steinmetz Dataset.
                                </span>
                                <p id="p-105" class="first-child">
                                 Illustrating the average estimated response rate for each interval of estimated neural activity. The purple bars show the average behavioural response rate and orange bars represent the occurrence ratio of that specific neural response interval among all
                                 <span class="sc">
                                  right
                                 </span>
                                 test trials. Error bars show the standard error. The size of error bars is invisible compared to the bar sizes in most cases.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn" id="fn-2">
                              <p id="p-106">
                               <a class="rev-xref" href="#xref-fn-2-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                2
                               </span>
                               <a href="https://github.com/nsteinme/steinmetz-et-al-2019/wiki/data-files">
                                https://github.com/nsteinme/steinmetz-et-al-2019/wiki/data-files
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-3">
                              <p id="p-107">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                3
                               </span>
                               <a href="https://senselab.med.yale.edu/MicrocircuitDB/showModel.cshtml?model=168867&file=%2Fhierarchical_network%2Freadme.html#tabs-2">
                                https://senselab.med.yale.edu/MicrocircuitDB/showModel.cshtml?model=168867&file=%2Fhierarchical_network%2Freadme.html#tabs-2
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-4">
                              <p id="p-108">
                               <a class="rev-xref" href="#xref-fn-4-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                4
                               </span>
                               <a href="https://github.com/nsteinme/steinmetz-et-al-2019/wiki/data-flles">
                                https://github.com/nsteinme/steinmetz-et-al-2019/wiki/data-flles
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-web" id="cit-2020.07.13.201673v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Martín
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  TensorFlow: Large-Scale Machine Learning on Heterogeneous Systems
                                 </span>
                                 . Software available from
                                 <a href="http://tensorflow.org">
                                  tensorflow.org
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.2" data-doi="10.1038/npp.2009.131">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Balleine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Bernard W
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John P
                                  </span>
                                  <span class="cit-name-surname">
                                   O’Doherty
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Human and rodent homologies in action control: corticostriatal determinants of goal-directed and habitual action
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Neuropsychopharmacology
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  48
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  69
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuropsychopharmacology%2B%253A%2B%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BCollege%2Bof%2BNeuropsychopharmacology%26rft.stitle%253DNeuropsychopharmacology%26rft.aulast%253DBalleine%26rft.auinit1%253DB.%2BW.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D69%26rft.atitle%253DHuman%2Band%2Brodent%2Bhomologies%2Bin%2Baction%2Bcontrol%253A%2Bcorticostriatal%2Bdeterminants%2Bof%2Bgoal-directed%2Band%2Bhabitual%2Baction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnpp.2009.131%26rft_id%253Dinfo%253Apmid%252F19776734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/npp.2009.131&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19776734&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000272784600004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.3" data-doi="10.1016/j.conb.2017.06.003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Omri
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Recurrent neural networks as versatile tools of neuroscience research
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in neurobiology
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.volume%253D46%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2017.06.003%26rft_id%253Dinfo%253Apmid%252F28668365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2017.06.003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28668365&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.4" data-doi="10.1017/S095252380000715X">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Britten
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Kenneth H
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A relationship between behavioral choice and the visual responses of neurons in macaque MT
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Visual neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  87
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  100
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DVisual%2Bneuroscience%26rft.stitle%253DVis%2BNeurosci%26rft.aulast%253DBritten%26rft.auinit1%253DK.%2BH.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D100%26rft.atitle%253DA%2Brelationship%2Bbetween%2Bbehavioral%2Bchoice%2Band%2Bthe%2Bvisual%2Bresponses%2Bof%2Bneurons%2Bin%2Bmacaque%2BMT.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS095252380000715X%26rft_id%253Dinfo%253Apmid%252F8730992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1017/S095252380000715X&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=8730992&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1996TN93200009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cebrian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   AC
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  NHPoisson: Modelling and Validation of Non Homogeneous Poisson Processes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  R package version
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chichilnisky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   EJ
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A simple white noise analysis of neuronal light responses
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Network: Computation in Neural Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  199
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  213
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNetwork%253A%2BComputation%2Bin%2BNeural%2BSystems%26rft.volume%253D12%26rft.spage%253D199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chilinski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Pawel
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ricardo
                                  </span>
                                  <span class="cit-name-surname">
                                   Silva
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Neural likelihoods via cumulative distribution functions
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1811.00974
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.8" data-doi="10.1038/nn.2501">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Churchland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Mark M
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Stimulus onset quenches neural variability: a widespread cortical phenomenon
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  369
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  378
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.stitle%253DNat%2BNeurosci%26rft.aulast%253DChurchland%26rft.auinit1%253DM.%2BM.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D369%26rft.epage%253D378%26rft.atitle%253DStimulus%2Bonset%2Bquenches%2Bneural%2Bvariability%253A%2Ba%2Bwidespread%2Bcortical%2Bphenomenon.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.2501%26rft_id%253Dinfo%253Apmid%252F20173745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.2501&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20173745&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000274860100020&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.9" data-doi="10.1038/nn.3776">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   John P
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Byron Yu
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Dimensionality reduction for large-scale neural recordings
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 , p.
                                 <span class="cit-fpage">
                                  1500
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D17%26rft.spage%253D1500%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.3776%26rft_id%253Dinfo%253Apmid%252F25151264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.3776&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25151264&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.13.201673v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Daley
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Vere-Jones
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  An Introduction to the Theory of Point Processes: Volume I: Elementary Theory and Methods
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Probability and Its Applications
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 <span class="cit-publ-loc">
                                  New York
                                 </span>
                                 . ISBN:
                                 <span class="cit-isbn">
                                  9780387215648
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dayan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Peter
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Laurence F
                                  </span>
                                  <span class="cit-name-surname">
                                   Abbott
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Theoretical neuroscience: computational and mathematical modeling of neural systems
                                 </span>
                                 ”. In.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dezfouli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Amir
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Integrated accounts of behavioral and neuroimaging data using flexible recurrent neural network models
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Neural Information Processing Systems
                                 </abbr>
                                 , pp.
                                 <span class="cit-fpage">
                                  4228
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4237
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.13" data-doi="10.1146/annurev-neuro-062111-150410">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ganguli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Surya
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Haim
                                  </span>
                                  <span class="cit-name-surname">
                                   Sompolinsky
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Compressed sensing, sparsity, and dimensionality in neuronal information processing and data analysis
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Annual review of neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  485
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  508
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bneuroscience%26rft.stitle%253DAnnu%2BRev%2BNeurosci%26rft.aulast%253DGanguli%26rft.auinit1%253DS.%26rft.volume%253D35%26rft.spage%253D485%26rft.epage%253D508%26rft.atitle%253DCompressed%2Bsensing%252C%2Bsparsity%252C%2Band%2Bdimensionality%2Bin%2Bneuronal%2Binformation%2Bprocessing%2Band%2Bdata%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-neuro-062111-150410%26rft_id%253Dinfo%253Apmid%252F22483042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-neuro-062111-150410&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22483042&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000307960400024&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gold
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Joshua I
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael N
                                  </span>
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  The neural basis of decision making
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Annual review of neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.15" data-doi="10.1038/nn.3711">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Robbe LT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J Anthony
                                  </span>
                                  <span class="cit-name-surname">
                                   Movshon
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eero P
                                  </span>
                                  <span class="cit-name-surname">
                                   Simoncelli
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Partitioning neuronal variability
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  858
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  865
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D17%26rft.spage%253D858%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.3711%26rft_id%253Dinfo%253Apmid%252F24777419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.3711&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24777419&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.13.201673v2.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grimmett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Geoffrey
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Stirzaker
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Probability and random processes
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Oxford university press
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.17" data-doi="10.1038/nature24636">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   James J
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Fully integrated silicon probes for high-density recording of neural activity
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  551
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7679
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  232
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  236
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D551%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature24636%26rft_id%253Dinfo%253Apmid%252F29120427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature24636&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29120427&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.13.201673v2.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kass
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Robert E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Uri T
                                  </span>
                                  <span class="cit-name-surname">
                                   Eden
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Emery N
                                  </span>
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Analysis of neural data
                                 </span>
                                 . Vol.
                                 <span class="cit-vol">
                                  491
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Diederik P
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jimmy
                                  </span>
                                  <span class="cit-name-surname">
                                   Ba
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Adam: A method for stochastic optimization
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1412.6980
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.20" data-doi="10.7554/eLife.10989">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kobak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Dmitry
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Demixed principal component analysis of neural population data
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e10989
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D5%26rft.spage%253De10989%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.10989%26rft_id%253Dinfo%253Apmid%252F27067378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.10989&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27067378&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.21" data-doi="10.1016/j.tics.2013.06.007">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kriegeskorte
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Nikolaus
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Rogier A
                                  </span>
                                  <span class="cit-name-surname">
                                   Kievit
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Representational geometry: integrating cognition, computation, and the brain
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in cognitive sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  401
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  412
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcognitive%2Bsciences%26rft.volume%253D17%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tics.2013.06.007%26rft_id%253Dinfo%253Apmid%252F23876494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.tics.2013.06.007&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23876494&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000323458700007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Máté
                                  </span>
                                  <span class="cit-name-surname">
                                   Lengyel
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A universal probabilistic spike count model reveals ongoing modulation of neural variability
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Neural Information Processing Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.23" data-doi="10.1038/nature12742">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mante
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Valerio
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Context-dependent computation by recurrent dynamics in prefrontal cortex
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  nature
                                 </abbr>
                                 <span class="cit-vol">
                                  503
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7474
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  78
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  84
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253Dnature%26rft.volume%253D503%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12742%26rft_id%253Dinfo%253Apmid%252F24201281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12742&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24201281&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000326585600035&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Arne F
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Models of neuronal stimulus-response functions: elaboration, estimation, and evaluation
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in systems neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 , p.
                                 <span class="cit-fpage">
                                  109
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bsystems%2Bneuroscience%26rft.volume%253D10%26rft.spage%253D109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Omi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Takahiro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kazuyuki
                                  </span>
                                  <span class="cit-name-surname">
                                   Aihara
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Fully Neural Network based Model for General Temporal Point Processes
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Neural Information Processing Systems
                                 </abbr>
                                 , pp.
                                 <span class="cit-fpage">
                                  2120
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2129
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paninski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Liam
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Maximum likelihood estimation of cascade point-process neural encoding models
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Network: Computation in Neural Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  243
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  262
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNetwork%253A%2BComputation%2Bin%2BNeural%2BSystems%26rft.volume%253D15%26rft.spage%253D243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.27" data-doi="10.1016/S0079-6123(06)65031-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paninski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Liam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Pillow
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jeremy
                                  </span>
                                  <span class="cit-name-surname">
                                   Lewi
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Statistical models for neural encoding, decoding, and optimal stimulus design
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Progress in brain research
                                 </abbr>
                                 <span class="cit-vol">
                                  165
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  493
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  507
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Bbrain%2Bresearch%26rft.stitle%253DProg%2BBrain%2BRes%26rft.aulast%253DPaninski%26rft.auinit1%253DL.%26rft.volume%253D165%26rft.spage%253D493%26rft.epage%253D507%26rft.atitle%253DStatistical%2Bmodels%2Bfor%2Bneural%2Bencoding%252C%2Bdecoding%252C%2Band%2Boptimal%2Bstimulus%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0079-6123%252806%252965031-0%26rft_id%253Dinfo%253Apmid%252F17925266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0079-6123(06)65031-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17925266&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000280608900032&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.28" data-doi="10.1007/s10827-009-0179-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paninski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Liam
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A new look at state-space models for neural data
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of computational neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  29
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1-2
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  107
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  126
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bneuroscience%26rft.stitle%253DJ%2BComput%2BNeurosci%26rft.aulast%253DPaninski%26rft.auinit1%253DL.%26rft.volume%253D29%26rft.issue%253D1-2%26rft.spage%253D107%26rft.epage%253D126%26rft.atitle%253DA%2Bnew%2Blook%2Bat%2Bstate-space%2Bmodels%2Bfor%2Bneural%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10827-009-0179-x%26rft_id%253Dinfo%253Apmid%252F19649698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10827-009-0179-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19649698&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000281563500010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.29" data-doi="10.1038/s41593-019-0520-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Blake A
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A deep learning framework for neuroscience
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1761
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1770
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D22%26rft.spage%253D1761%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41593-019-0520-2%26rft_id%253Dinfo%253Apmid%252F31659335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41593-019-0520-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31659335&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.13.201673v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rieke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Fred
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Spikes: exploring the neural code
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  MIT Press
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.13.201673v2.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ross
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Sheldon M
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Stochastic processes
                                 </span>
                                 . Vol.
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Wiley New York
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schaeffer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Rylan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Reverse-engineering recurrent neural network solutions to a hierarchical inference task for mice
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in Neural Information Processing Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.33" data-doi="10.1523/JNEUROSCI.18-10-03870.1998">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shadlen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Michael N
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William T
                                  </span>
                                  <span class="cit-name-surname">
                                   Newsome
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  The variable discharge of cortical neurons: implications for connectivity, computation, and information coding
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neuroscience
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  3870
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3896
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DShadlen%26rft.auinit1%253DM.%2BN.%26rft.volume%253D18%26rft.issue%253D10%26rft.spage%253D3870%26rft.epage%253D3896%26rft.atitle%253DThe%2BVariable%2BDischarge%2Bof%2BCortical%2BNeurons%253A%2BImplications%2Bfor%2BConnectivity%252C%2BComputation%252C%2Band%2BInformation%2BCoding%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.18-10-03870.1998%26rft_id%253Dinfo%253Apmid%252F9570816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8zODcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMDIvMjAyMC4wNy4xMy4yMDE2NzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Joseph
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Monotonic networks
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in neural information processing systems
                                 </abbr>
                                 , pp.
                                 <span class="cit-fpage">
                                  661
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  667
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.35" data-doi="org/10.1038/s41586-019-1787-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Nicholas A
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Distributed coding of choice, action and engagement across the mouse brain
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  576
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7786
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  266
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  273
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D576%26rft.spage%253D266%26rft_id%253Dinfo%253Adoi%252Forg%252F10.1038%252Fs41586-019-1787-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=org/10.1038/s41586-019-1787-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.36" data-doi="10.1016/j.conb.2014.01.008">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sussillo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Neural circuits as computational dynamical systems
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in neurobiology
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  156
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  163
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.volume%253D25%26rft.spage%253D156%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2014.01.008%26rft_id%253Dinfo%253Apmid%252F24509098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2014.01.008&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24509098&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.37" data-doi="10.1152/jn.00697.2004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Truccolo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Wilson
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  A point process framework for relating neural spiking activity to spiking history, neural ensemble, and extrinsic covariate effects
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neurophysiology
                                 </abbr>
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1074
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1089
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneurophysiology%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fjn.00697.2004%26rft_id%253Dinfo%253Apmid%252F15356183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1152/jn.00697.2004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15356183&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F02%2F2020.07.13.201673.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000226342000039&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.38" data-doi="10.1016/j.conb.2019.07.004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whiteway
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Matthew R
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel A
                                  </span>
                                  <span class="cit-name-surname">
                                   Butts
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  The quest for interpretable models of neural population activity
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Current opinion in neurobiology
                                 </abbr>
                                 <span class="cit-vol">
                                  58
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  86
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  93
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.volume%253D58%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2019.07.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.conb.2019.07.004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wimmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Klaus
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Sensory integration dynamics in a hierarchical network explains choice probabilities in cortical area MT
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D6%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-confproc" id="cit-2020.07.13.201673v2.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Anqi
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Gaussian process based nonlinear latent structure discovery in multivariate spike train data
                                 </span>
                                 ”. In:
                                 <span class="cit-conf-name">
                                  Proceedings of the 31st International Conference on Neural Information Processing Systems
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  3499
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3508
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.41" data-doi="10.1073/pnas.1403112111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yamins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Daniel LK
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Performance-optimized hierarchical models predict neural responses in higher visual cortex
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the national academy of sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  23
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  8619
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8624
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2Bnational%2Bacademy%2Bof%2Bsciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1403112111%26rft_id%253Dinfo%253Apmid%252F24812127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzIzLzg2MTkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8wMi8yMDIwLjA3LjEzLjIwMTY3My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.13.201673v2.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M Byron
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ). “
                                 <span class="cit-article-title">
                                  Extracting dynamical structure embedded in neural activity
                                 </span>
                                 ”. In:
                                 <abbr class="cit-jnl-abbrev">
                                  Advances in neural information processing systems
                                 </abbr>
                                 , pp.
                                 <span class="cit-fpage">
                                  1545
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1552
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.07.13.201673v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2020.10.08.331850v3" title="A general decoding strategy explains the relationship between behavior and correlated variability" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2020.07.30.228759v3" title="RIP140 inhibits glycolysis-dependent proliferation of breast cancer cells by regulating GLUT3 expression through transcriptional crosstalk between hypoxia induced factor and p53" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 02, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.07.13.201673v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.07.13.201673v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.07.13.201673v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Neural Network Poisson Models for Behavioural and Neural Spike Train Data
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2381225"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-VU5b3GqyBeTj9C_3gqc1obhfwUNw_tc87rLlWLDRp8A"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484473902"/>
                                   <input type="hidden" name="captcha_token" value="077f3b3f2800e70c96e797854e6ec7d1"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2381225--3" data-node-nid="2381225" id="node-2381225--4466410964" data-pisa="biorxiv;2020.07.13.201673v2" data-pisa-master="biorxiv;2020.07.13.201673" data-seqnum="2381225" data-apath="/biorxiv/early/2022/02/02/2020.07.13.201673.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Neural Network Poisson Models for Behavioural and Neural Spike Train Data
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Moein
                                  </span>
                                  <span class="nlm-surname">
                                   Khajehnejad
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Forough
                                  </span>
                                  <span class="nlm-surname">
                                   Habibollahi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Richard
                                  </span>
                                  <span class="nlm-surname">
                                   Nock
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Ehsan
                                  </span>
                                  <span class="nlm-surname">
                                   Arabzadeh
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Peter
                                  </span>
                                  <span class="nlm-surname">
                                   Dayan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Amir
                                  </span>
                                  <span class="nlm-surname">
                                   Dezfouli
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.07.13.201673;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.07.13.201673
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&title=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&title=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&text=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&t=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&title=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&title=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&title=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2381225--5" data-node-nid="2381225" id="citation-node-2381225--6427529188" data-pisa="biorxiv;2020.07.13.201673v2" data-pisa-master="biorxiv;2020.07.13.201673" data-seqnum="2381225" data-apath="/biorxiv/early/2022/02/02/2020.07.13.201673.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Neural Network Poisson Models for Behavioural and Neural Spike Train Data
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Moein
                                      </span>
                                      <span class="nlm-surname">
                                       Khajehnejad
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Forough
                                      </span>
                                      <span class="nlm-surname">
                                       Habibollahi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Richard
                                      </span>
                                      <span class="nlm-surname">
                                       Nock
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Ehsan
                                      </span>
                                      <span class="nlm-surname">
                                       Arabzadeh
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Peter
                                      </span>
                                      <span class="nlm-surname">
                                       Dayan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Amir
                                      </span>
                                      <span class="nlm-surname">
                                       Dezfouli
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.07.13.201673;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.07.13.201673
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Khajehnejad%2C+Moein&rft.au=Habibollahi%2C+Forough&rft.au=Nock%2C+Richard&rft.au=Arabzadeh%2C+Ehsan&rft.au=Dayan%2C+Peter&rft.au=Dezfouli%2C+Amir">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2381225/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2381225/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2381225/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2381225/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2381225/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2381225/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2381225/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2381225/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2381225/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2381225/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2381225/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2381225/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&count=horizontal&via=&text=Neural%20Network%20Poisson%20Models%20for%20Behavioural%20and%20Neural%20Spike%20Train%20Data&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.07.13.201673v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.07.13.201673v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
