<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.18.101402v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.18.101402v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.18.101402v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/08/23/2020.05.18.101402.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.18.101402v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.18.101402"/>
  <meta name="DC.Date" content="2022-08-23"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="RNA molecules can form secondary and tertiary structures that can regulate their localization and function. Using enzymatic or chemical probing together with high-throughput sequencing, secondary structure can be mapped across the entire transcriptome. However, a limiting factor is that only population averages can be obtained since each read is an independent measurement. Although long-read sequencing has recently been used to determine RNA structure, these methods still used aggregate signals across the strands to detect structure. Averaging across the population also means that only limited information about structural heterogeneity across molecules or dependencies within each molecule can be obtained. Here, we present Single-Molecule Structure sequencing (SMS-seq) that combines structural probing with native RNA sequencing to provide non-amplified, structural profiles of individual molecules with novel analysis methods. Our new approach using mutual information enabled single molecule structural interrogation. Each RNA is probed at numerous bases enabling the discovery of dependencies and heterogeneity of structural features. We also show that SMS-seq can capture tertiary interactions, dynamics of riboswitch ligand binding, and mRNA structural features.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Teshome Tilahun Bizuayehu"/>
  <meta name="DC.Contributor" content="Kornel Labun"/>
  <meta name="DC.Contributor" content="Martin Jakubec"/>
  <meta name="DC.Contributor" content="Kirill Jefimov"/>
  <meta name="DC.Contributor" content="Adnan Niazi"/>
  <meta name="DC.Contributor" content="Eivind Valen"/>
  <meta name="article:published_time" content="2022-08-23"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>RNA molecules can form secondary and tertiary structures that can regulate their localization and function. Using enzymatic or chemical probing together with high-throughput sequencing, secondary structure can be mapped across the entire transcriptome. However, a limiting factor is that only population averages can be obtained since each read is an independent measurement. Although long-read sequencing has recently been used to determine RNA structure, these methods still used aggregate signals across the strands to detect structure. Averaging across the population also means that only limited information about structural heterogeneity across molecules or dependencies within each molecule can be obtained. Here, we present Single-Molecule Structure sequencing (SMS-seq) that combines structural probing with native RNA sequencing to provide non-amplified, structural profiles of individual molecules with novel analysis methods. Our new approach using mutual information enabled single molecule structural interrogation. Each RNA is probed at numerous bases enabling the discovery of dependencies and heterogeneity of structural features. We also show that SMS-seq can capture tertiary interactions, dynamics of riboswitch ligand binding, and mRNA structural features.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.18.101402v2"/>
  <meta name="citation_id" content="2020.05.18.101402v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.18.101402"/>
  <meta name="citation_num_pages" content="30"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.18.101402"/>
  <meta name="citation_author" content="Teshome Tilahun Bizuayehu"/>
  <meta name="citation_author_institution" content="Computational Biology Unit, Department of Informatics, University of Bergen"/>
  <meta name="citation_author_institution" content="Sars International Center for Marine Molecular Biology, University of Bergen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8182-6562"/>
  <meta name="citation_author" content="Kornel Labun"/>
  <meta name="citation_author_institution" content="Computational Biology Unit, Department of Informatics, University of Bergen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1262-2611"/>
  <meta name="citation_author" content="Martin Jakubec"/>
  <meta name="citation_author_institution" content="Department of Chemistry, University of Tromsø"/>
  <meta name="citation_author" content="Kirill Jefimov"/>
  <meta name="citation_author_institution" content="Computational Biology Unit, Department of Informatics, University of Bergen"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0916-4442"/>
  <meta name="citation_author" content="Adnan Niazi"/>
  <meta name="citation_author_institution" content="Computational Biology Unit, Department of Informatics, University of Bergen"/>
  <meta name="citation_author" content="Eivind Valen"/>
  <meta name="citation_author_institution" content="Computational Biology Unit, Department of Informatics, University of Bergen"/>
  <meta name="citation_author_institution" content="Sars International Center for Marine Molecular Biology, University of Bergen"/>
  <meta name="citation_author_email" content="eivind.valen@uib.no"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1840-6108"/>
  <meta name="citation_reference" content="Strobel, E.J., Yu, A.M. and Lucks, J.B. (2018) High-throughput determination of RNA structures. Nature Reviews Genetics, 19, 615–634."/>
  <meta name="citation_reference" content="Smola, M.J., Rice, G.M., Busan, S., Siegfried, N.A. and Weeks, K.M. (2015) Selective 2′-hydroxyl acylation analyzed by primer extension and mutational profiling (SHAPE-MaP) for direct, versatile and accurate RNA structure analysis. Nature Protocols, 10, 1643–1669."/>
  <meta name="citation_reference" content="Homan, P.J., Favorov, O.V., Lavender, C.A., Kursun, O., Ge, X., Busan, S., Dokholyan, N.V. and Weeks, K.M. (2014) Single-molecule correlated chemical probing of RNA. Proc. Natl. Acad. Sci. U. S. A., 111, 13858–13863."/>
  <meta name="citation_reference" content="Flynn, R.A., Zhang, Q.C., Spitale, R.C., Lee, B., Mumbach, M.R. and Chang, H.Y. (2016) Transcriptome-wide interrogation of RNA secondary structure in living cells with icSHAPE. Nat. Protoc., 11, 273–290."/>
  <meta name="citation_reference" content="Siegfried, N.A., Busan, S., Rice, G.M., Nelson, J.A.E. and Weeks, K.M. (2014) RNA motif discovery by SHAPE and mutational profiling (SHAPE-MaP). Nat. Methods, 11, 959–965."/>
  <meta name="citation_reference" content="Tomezsko, P.J., Corbin, V.D.A., Gupta, P., Swaminathan, H., Glasgow, M., Persad, S., Edwards, M.D., Mcintosh, L., Papenfuss, A.T., Emery, A., et al. (2020) Determination of RNA structural diversity and its role in HIV-1 RNA splicing. Nature, 582, 438–442."/>
  <meta name="citation_reference" content="Morandi, E., Manfredonia, I., Simon, L.M., Anselmi, F., van Hemert, M.J., Oliviero, S. and Incarnato, D. (2021) Genome-scale deconvolution of RNA structure ensembles. Nat. Methods, 18, 249–252."/>
  <meta name="citation_reference" content="Sexton, A.N., Wang, P.Y., Rutenberg-Schoenberg, M. and Simon, M.D. (2017) Interpreting Reverse Transcriptase Termination and Mutation Events for Greater Insight into the Chemical Probing of RNA. Biochemistry, 56, 4713–4721."/>
  <meta name="citation_reference" content="Stephenson, W., Razaghi, R., Busan, S., Weeks, K.M., Timp, W. and Smibert, P. (2022) Direct detection of RNA modifications and structure using single-molecule nanopore sequencing. Cell Genom, 2."/>
  <meta name="citation_reference" content="Aw, J.G.A., Lim, S.W., Wang, J.X., Lambert, F.R.P., Tan, W.T., Shen, Y., Zhang, Y., Kaewsapsak, P., Li, C., Ng, S.B., et al. (2020) Determination of isoform-specific RNA structure with nanopore long reads. Nat. Biotechnol."/>
  <meta name="citation_reference" content="Serganov, A., Huang, L. and Patel, D.J. (2009) Coenzyme recognition and gene regulation by a flavin mononucleotide riboswitch. Nature, 458, 233–237."/>
  <meta name="citation_reference" content="Leonard, N.J., McDonald, J.J., Henderson, R.E. and Reichmann, M.E. (1971) Reaction of diethyl pyrocarbonate with nucleic acid components. Adenosine. Biochemistry, 10, 3335–3342."/>
  <meta name="citation_reference" content="Oxford Nanopore Technologies (2017) Tombo: detection of non-standard nucleotides using the genome-resolved raw nanopore signal."/>
  <meta name="citation_reference" content="Li, H. (2018) Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics, 34, 3094–3100."/>
  <meta name="citation_reference" content="Rairkar, A., Rubino, H.M. and Lockard, R.E. (1988) Chemical probing of adenine residues within the secondary structure of rabbit 18S ribosomal RNA. Biochemistry, 27, 582–592."/>
  <meta name="citation_reference" content="Peattie, D.A. and Gilbert, W. (1980) Chemical probes for higher-order structure in RNA. Proceedings of the National Academy of Sciences, 77, 4679–4682."/>
  <meta name="citation_reference" content="Moazed, D., Van Stolk, B.J., Douthwaite, S. and Noller, H.F. (1986) Interconversion of active and inactive 30 S ribosomal subunits is accompanied by a conformational change in the decoding region of 16 S rRNA. J. Mol. Biol., 191, 483–493."/>
  <meta name="citation_reference" content="Van Stolk, B.J. and Noller, H.F. (1984) Chemical probing of conformation in large RNA molecules. Journal of Molecular Biology, 180, 151–177."/>
  <meta name="citation_reference" content="Brunel, C., Romby, P., Westhof, E., Ehresmann, C. and Ehresmann, B. (1991) Three-dimensional model of Escherichia coli ribosomal 5 S RNA as deduced from structure probing in solution and computer modeling. J. Mol. Biol., 221, 293–308."/>
  <meta name="citation_reference" content="Reineke, L.C., Komar, A.A., Caprara, M.G. and Merrick, W.C. (2008) A small stem loop element directs internal initiation of the URE2 internal ribosome entry site in Saccharomyces cerevisiae. J. Biol. Chem., 283, 19011–19025."/>
  <meta name="citation_reference" content="Pedregosa, F., Varoquaux, G., Gramfort, A., Michel, V., Thirion, B., Grisel, O., Blondel, M., Prettenhofer, P., Weiss, R., Dubourg, V., et al. (2011) Scikit-learn: Machine Learning in Python. J. Mach. Learn. Res., 12, 2825–2830."/>
  <meta name="citation_reference" content="Lorenz, R., Bernhart, S.H., Höner Zu Siederdissen, C., Tafer, H., Flamm, C., Stadler, P.F. and Hofacker, I.L. (2011) ViennaRNA Package 2.0. Algorithms Mol. Biol., 6, 26."/>
  <meta name="citation_reference" content="Serganov, A., Polonskaia, A., Phan, A.T., Breaker, R.R. and Patel, D.J. (2006) Structural basis for gene regulation by a thiamine pyrophosphate-sensing riboswitch. Nature, 441, 1167–1171."/>
  <meta name="citation_reference" content="Spitale, R.C., Crisalli, P., Flynn, R.A., Torre, E.A., Kool, E.T. and Chang, H.Y. (2013) RNA SHAPE analysis in living cells. Nature Chemical Biology, 9, 18–20."/>
  <meta name="citation_reference" content="Leppek, K., Das, R. and Barna, M. (2018) Functional 5′ UTR mRNA structures in eukaryotic translation regulation and how to find them. Nature Reviews Molecular Cell Biology, 19, 158–174."/>
  <meta name="citation_reference" content="Winkler, W., Nahvi, A. and Breaker, R.R. (2002) Thiamine derivatives bind messenger RNAs directly to regulate bacterial gene expression. Nature, 419, 952–956."/>
  <meta name="citation_reference" content="Haller, A., Altman, R.B., Soulière, M.F., Blanchard, S.C. and Micura, R. (2013) Folding and ligand recognition of the TPP riboswitch aptamer at single-molecule resolution. Proc. Natl. Acad. Sci. U. S. A., 110, 4188–4193."/>
  <meta name="citation_reference" content="Misra, V.K. and Draper, D.E. (2002) The linkage between magnesium binding and RNA folding. J. Mol. Biol., 317, 507–521."/>
  <meta name="citation_reference" content="Vicens, Q., Mondragón, E. and Batey, R.T. (2011) Molecular sensing by the aptamer domain of the FMN riboswitch: a general model for ligand binding by conformational selection. Nucleic Acids Res., 39, 8586–8598."/>
  <meta name="citation_reference" content="RNAcentral Consortium (2021) RNAcentral 2021: secondary structure integration, improved sequence search and new member databases. Nucleic Acids Res., 49, D212–D220."/>
  <meta name="citation_reference" content="Kertesz, M., Wan, Y., Mazor, E., Rinn, J.L., Nutter, R.C., Chang, H.Y. and Segal, E. (2010) Genome-wide measurement of RNA secondary structure in yeast. Nature, 467, 103–107."/>
  <meta name="citation_reference" content="Rouskin, S., Zubradt, M., Washietl, S., Kellis, M. and Weissman, J.S. (2014) Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo. Nature, 505, 701–705."/>
  <meta name="citation_reference" content="Rüegsegger, U., Leber, J.H. and Walter, P. (2001) Block of HAC1 mRNA translation by long-range base pairing is released by cytoplasmic splicing upon induction of the unfolded protein response. Cell, 107, 103–114."/>
  <meta name="citation_reference" content="Busan, S., Weidmann, C.A., Sengupta, A. and Weeks, K.M. (2019) Guidelines for SHAPE Reagent Choice and Detection Strategy for RNA Structure Probing Studies. Biochemistry, 58, 2655–2664."/>
  <meta name="citation_reference" content="The RNAcentral Consortium (2019) RNAcentral: a hub of information for non-coding RNA sequences. Nucleic Acids Res., 47, D1250–D1251."/>
  <meta name="twitter:title" content="Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="RNA molecules can form secondary and tertiary structures that can regulate their localization and function. Using enzymatic or chemical probing together with high-throughput sequencing, secondary structure can be mapped across the entire transcriptome. However, a limiting factor is that only population averages can be obtained since each read is an independent measurement. Although long-read sequencing has recently been used to determine RNA structure, these methods still used aggregate signals across the strands to detect structure. Averaging across the population also means that only limited information about structural heterogeneity across molecules or dependencies within each molecule can be obtained. Here, we present Single-Molecule Structure sequencing (SMS-seq) that combines structural probing with native RNA sequencing to provide non-amplified, structural profiles of individual molecules with novel analysis methods. Our new approach using mutual information enabled single molecule structural interrogation. Each RNA is probed at numerous bases enabling the discovery of dependencies and heterogeneity of structural features. We also show that SMS-seq can capture tertiary interactions, dynamics of riboswitch ligand binding, and mRNA structural features.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="RNA molecules can form secondary and tertiary structures that can regulate their localization and function. Using enzymatic or chemical probing together with high-throughput sequencing, secondary structure can be mapped across the entire transcriptome. However, a limiting factor is that only population averages can be obtained since each read is an independent measurement. Although long-read sequencing has recently been used to determine RNA structure, these methods still used aggregate signals across the strands to detect structure. Averaging across the population also means that only limited information about structural heterogeneity across molecules or dependencies within each molecule can be obtained. Here, we present Single-Molecule Structure sequencing (SMS-seq) that combines structural probing with native RNA sequencing to provide non-amplified, structural profiles of individual molecules with novel analysis methods. Our new approach using mutual information enabled single molecule structural interrogation. Each RNA is probed at numerous bases enabling the discovery of dependencies and heterogeneity of structural features. We also show that SMS-seq can capture tertiary interactions, dynamics of riboswitch ligand binding, and mRNA structural features.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-08-23"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2705725"/>
  <title>
   Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8l9Iv97duWzyU1IBFBN-i2MZPER1xqlI0cln6j1G1yU"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"markup":[{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.05.18.101402v2"},{"requested":"short-text","variant":"abstract","view":"abstract","pisa":"biorxiv;2020.05.18.101402v2"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.18.101402v2"}],"nid":"2705725","apath":"\/biorxiv\/early\/2022\/08\/23\/2020.05.18.101402.atom","pisa":"biorxiv;2020.05.18.101402v2","processed":["highwire_math"],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.18.101402v2","title":"Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles","identifier":"node\/2705725"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022plT8kUfJ1zewbCkiO0mafvprVeTi9nBNwDRGxE9gmeLe1SlpNRdROPf0a8i55CJivQjMZ8Az7p1Tu5p8x98RgYeR7o9U71hHvp5N43oKWrxiH8Qs7IWpH20Vt6obmVJWCo3thko4bSP27FkOqMlwh+Q3WotEXxx8UVPlEVgZnsp+Shw0Wq7IzSgduCfXaj8vwDpncsbqWMTOBDSfrl+3FaMIf+ErbZnDBRAZ94Sbu7exYEE9Ed1gYVEGIPHNRfihg+tvi5Lg9EbjpsrdLtYeonLSGRgeAh9nqPGVLbh6a6tjI2jcux2+VyNCwpFPR9g1HtjTh3airCeL0VgYDV1OsRAllmKmhmCKCq1ZC6UkQsVIpFGorFhtLMeak\\\\\\\/SsFXwWuCqOmdGtl5qLnKTGlcmuEL2cpQXjFC0X92+fY3q1286T6WIJ8Re21yV7fBzLLlljNnsPXOziJ9mMYekFYi3zXwiAkSrMXl012SxzZx\\\\\\\/nNgK5VL21nfMv3v+Z49YE5nZWItcPnrHqXCTOtPQrnjCBMH6UhrzU3+z3wCQCsqpxtYoX8ZCOpJ2g7n5eSFQyISaYww1QokNLyBFgMqmoV\\\\\\\/p7qygez66hruX5sBDynCHUYsXefmaGBEQjA\\\\\\\/90efruoZjUh0\\\\\\\/ubIgdQGLOQ4A0kRmX3cdE4AtsnBX3xTt2Z3I3t5\\\\\\\/H5tLN7uu6iohxGJVRAgY5HUGlQ2uDxl3OTTb0xAAeuw97lf80UWy\\\\\\\/X7H1a\\\\\\\/kdf7fQz+jrkQf\\\\\\\/Q1KGuZ5zGD2\\\\\\\/sCSMd5er4Rbm1Ie7XgbV1iZJj4hNvoGUjF5zaThBZsSUC8Af1CZdt+UTX9SK38HdNEkqfH67RGoJuMG4JEA7\\\\\\\/MJOs\\\\\\\/M\\\\\\\/ujqQ6HImeIXxrQuITzoMW8ePMzDymBa0S9oJYWUlicIQHgnksAt2v8CugQeOitWWwVcd4ZD4zfjETJnXLpbomodXJf5QVx7oeRCqp\\\\\\\/5tA9BanwK92YV5snd8YOq\\\\\\\/qkzh0MxfOMgO6FPZ5BLxasCfy1zjbEA4eBqcWFxpiwg1Ybd6rYQ9cE7iK9wi60lnR8cP8Wj+w9H+Dy8pTA+8yPzqBPU0r6XqV6WU01AXmEMDnuJM61DxjJ7XkHo3yPMthlGroynY92WP7D+yITw1TT7N6jb1Ra9j0HcsHNWLmswi3RHsKNdhPTXTpwVO+yuEnFspR5zGytJeWgftGIX2M4uHMSIDS7EXQEpuegjP3HD76yi2F9BiuB76Fdb2A4rZBoSJmmOVLawrDhjBoNPpZAHSbZu654VjUt3mlJQ2DMWbu0I3GFGQvLwy9qogi\\\\\\\/jdl8o6x0Y0h3HSVKqV0zb8gkL\\\\\\\/Sq\\\\\\\/HV0MmG+Q5KKpz7gFXl0KecdM4GusJL2mi2C+oHJvrFwErGVykvJhIdHI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z5udL5wGo7oVyyvMmnFrng==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220ff323943f82772fa6f080d5ad61b59e4740259d1d6d1d4b82201bc6bd3ec246\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.18.101402v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2705725 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.18.101402v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2050954899">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2050954899" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_968289291" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-tuizZ0iP_u2N-sD0GI15X_KUUpXXQRPtKmURf1XE5t0"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2705725" data-node-nid="2705725" id="node-2705725--22063865419" data-pisa="biorxiv;2020.05.18.101402v2" data-pisa-master="biorxiv;2020.05.18.101402" data-apath="/biorxiv/early/2022/08/23/2020.05.18.101402.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-8182-6562" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Teshome Tilahun
                      </span>
                      <span class="nlm-surname">
                       Bizuayehu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0002-1262-2611" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kornel
                      </span>
                      <span class="nlm-surname">
                       Labun
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Martin
                      </span>
                      <span class="nlm-surname">
                       Jakubec
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0003-0916-4442" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kirill
                      </span>
                      <span class="nlm-surname">
                       Jefimov
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Adnan
                      </span>
                      <span class="nlm-surname">
                       Niazi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0003-1840-6108" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Eivind
                      </span>
                      <span class="nlm-surname">
                       Valen
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.18.101402
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2705725--22063865419" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Teshome Tilahun Bizuayehu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Biology Unit, Department of Informatics, University of Bergen
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Sars International Center for Marine Molecular Biology, University of Bergen
                       </span>
                       , Bergen 5008,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Teshome%2BTilahun%2BBizuayehu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bizuayehu%20TT&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ATeshome%2BTilahun%2BBizuayehu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-8182-6562" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Teshome Tilahun Bizuayehu
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Kornel Labun
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Biology Unit, Department of Informatics, University of Bergen
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kornel%2BLabun%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Labun%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKornel%2BLabun%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1262-2611" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kornel Labun
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Martin Jakubec
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Chemistry, University of Tromsø
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Martin%2BJakubec%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jakubec%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMartin%2BJakubec%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Kirill Jefimov
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Biology Unit, Department of Informatics, University of Bergen
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kirill%2BJefimov%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jefimov%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKirill%2BJefimov%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0916-4442" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kirill Jefimov
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Adnan Niazi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Biology Unit, Department of Informatics, University of Bergen
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Adnan%2BNiazi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Niazi%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAdnan%2BNiazi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Eivind Valen
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Biology Unit, Department of Informatics, University of Bergen
                       </span>
                       ,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Sars International Center for Marine Molecular Biology, University of Bergen
                       </span>
                       , Bergen 5008,
                       <span class="nlm-country">
                        Norway
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Eivind%2BValen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Valen%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AEivind%2BValen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-1840-6108" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Eivind Valen
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:eivind.valen@uib.no" class="" data-icon-position="" data-hide-link-title="0">
                        eivind.valen@uib.no
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.18.101402v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.18.101402v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.18.101402v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.18.101402v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.18.101402v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.18.101402v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.18.101402v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2705725" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2705725/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             RNA molecules can form secondary and tertiary structures that can regulate their localization and function. Using enzymatic or chemical probing together with high-throughput sequencing, secondary structure can be mapped across the entire transcriptome. However, a limiting factor is that only population averages can be obtained since each read is an independent measurement. Although long-read sequencing has recently been used to determine RNA structure, these methods still used aggregate signals across the strands to detect structure. Averaging across the population also means that only limited information about structural heterogeneity across molecules or dependencies within each molecule can be obtained. Here, we present Single-Molecule Structure sequencing (SMS-seq) that combines structural probing with native RNA sequencing to provide non-amplified, structural profiles of individual molecules with novel analysis methods. Our new approach using mutual information enabled single molecule structural interrogation. Each RNA is probed at numerous bases enabling the discovery of dependencies and heterogeneity of structural features. We also show that SMS-seq can capture tertiary interactions, dynamics of riboswitch ligand binding, and mRNA structural features.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Chemical probing combined with sequencing is broadly used for RNA structural profiling (
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ). Most of these experiments use chemicals that modify accessible non-paired bases that result in reverse transcription termination and/or misincorporation of bases during cDNA synthesis (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             –
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ). These sites can then be captured through sequencing the cDNA, resulting in an average reactivity score for each base. Computational tools based on mutational profiling have been developed to determine single molecule RNA structure determination (
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ,
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ). While these methods have been instrumental in mapping structures across the transcriptome, they are susceptible to multiple biases (
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ). First, the synthesis of cDNA gives rise to biases through the processivity of reverse transcriptase, sequence biases during the random priming used to initiate the RT, and PCR amplification. Second, these methods are further constrained by the read length of short-read sequencers that prohibit full-length structural profiling of larger molecules such as messenger RNAs and ribosomal RNAs. Third, most of these methods only capture a single or very few modified bases per sequenced read. Finally, since most of these methods rely on amplification, the data obtained represents the average reactivity at a position across all molecules in the sample. These approaches are therefore limited in their ability to capture information about isoform-specific structures and structural heterogeneity.
                            </p>
                            <p id="p-7">
                             Direct sequencing of long native RNA molecules after chemical probing can reveal insights into the complete structural profiles of longer RNA molecules (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). This could lead to better prediction algorithms and, in turn, more accurate interpretations of RNA structures and functions. Here, we developed a novel method based on nanopore sequencing technology that directly detects the chemical adducts from structural probing on both short and long RNA molecules without the need for reverse transcription and PCR. Unlike two recent nanopore-based structure methods, which only consider average nucleotide accessibility and consensus structure (
                             <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ), our single-molecule approach probes multiple bases on individual molecules and describes their co-occurrence. We demonstrate this method on known structures and show that it can open the way for studying structural co-dependence of bases and conformational heterogeneity of RNA structures. We analyze the dynamics of riboswitches revealing conformational changes upon ligand binding, short- and long-range interactions with cooperative structural domains, and tertiary RNA interactions that have previously been described by crystal structures. In summary, we have developed a succinct structure probing approach that can provide a dynamic single-molecule view of RNA structure.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Materials and methods
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Sample preparation
                             </h3>
                             <p id="p-8">
                              In vitro synthesis of RNA. Custom sense and antisense hairpin oligos with T7 promoter sequence were purchased from Sigma (Sigma-Aldrich) and in vitro transcribed using T7 RNA polymerase (NEB). F. nucleatum FMN riboswitch RNA was synthesized in vitro using T7 RNA polymerase from a pUT7 vector that carries 112-nucleotides FMN sensing domain with hammerhead ribozyme at the end to obtain identical 3′ ends (
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ). Coenzyme thiamine pyrophosphate binding riboswitch (TPP) RNA was synthesized in vitro using T7 RNA polymerase. TPP template was prepared using sense and antisense TPP oligos with T7 promoter sequence (
                              <strong>
                               Supplementary Table 1
                              </strong>
                              ) purchased from Sigma (Sigma-Aldrich) that was assembled using PCR, cloned to pCR 2.1-TOPO vector (Invitrogen), and digested using EcoR I (NEB). Similarly, k-mer sequences were assembled using PCR from oligos purchased from Sigma.
                             </p>
                             <p id="p-9">
                              Cell culture. Yeast (Saccharomyces cerevisiae) strain Y187 was grown in YPD medium at 30 °C. Saturated cultures were diluted to OD600 of ∼0.1 and grown to a final OD600 of 0.7 followed by RNA extraction using hot phenol-chloroform method. Poly(A) and non-poly(A) separation was performed using Poly(A)Purist MAG Kit (Invitrogen) following manufacturer protocol. The remaining non-poly(A) RNA was precipitated overnight at -80 °C following the RNA ethanol precipitation protocol.
                             </p>
                             <p id="p-10">
                              Diethyl pyrocarbonate (DEPC) treatment of RNA. All structural probing was carried out using DEPC (Sigma), which carbethoxylates unpaired adenosine at N-6 or N-7 by opening the imidazole ring (
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ) (
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Fig 1A
                               </a>
                              </strong>
                              ). 1μg RNA was denatured at 90 °C for 3 minutes, cooled on ice, and renatured in 200 mM Hepes pH 7.8, 100 mM KCl, 10 mM MgCl2 for 20 minutes at 20 °C. For (+)DEPC, RNA was treated with 10% (final volume) DEPC for 45 min at 20 °C. DEPC reaction was performed in the absence and presence of six-fold molar excess of riboflavin 5′-monophosphate and thiamine pyrophosphate (Sigma-Aldrich) for FMN and TPP riboswitch, respectively. The reactions were immediately purified using RNA Clean & Concentrator (Zymo Research) or RNA XP bead (Beckman Coulter).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F1.large.jpg?width=800&height=600&carousel=1" title="Oxford nanopore direct RNA structure profiling of a hairpin RNA. A) Molecular structure of Adenosine modification by DEPC that modifies structurally unconstrained adenosine by opening the imidazole ring. B) SMS-seq workflow. Structures are probed with DEPC and the current signal of each nucleotide that passes through the pore is recorded. Modified sites (red) are detected by comparing the current signal from treated samples to the control samples. C) Normalized current mean and standard deviation for a nucleotide at a single-stranded region of RNA modified with DEPC (red) and unmodified (gray). D) ROC curves for unmodified, modified, and denatured 5-mer RNA sequences. The 5-mer RNAs are treated with 1 % (black), 5 % (blue), 10 % (orange) DEPC as well as denatured RNA treated with 5 % DEPC (green). E) Denaturing 15% polyacrylamide gel electrophoresis of 5′-end radiolabeled RNA fragments generated from DEPC treated hairpin RNA (top) and untreated hairpin RNA (bottom), which were treated with borohydride and aniline. Arrows mark positions that were likely to be identified by SMS-seq as modified. F) Proton NMR spectra of non-treated (blue) and DEPC treated (red) for hairpin RNA. Singlet of protons on positions 8 and 2 on the purine ring could be quantified. G) Tombo modification statistics for adenosine bases. Bases are called modified (open) when their statistic is below 0.2 (dotted line) corresponding to an FDR of 15%. H) Heatmap showing 3093 full-length reads (y-axis) with modified (red) and unmodified (gray) nucleotides at each position (x-axis) for each read. Reads are ordered by the overall modification rate. The legend above (Structure) shows which nucleotides are accessible (white) and inaccessible (blue) in the hairpin structure. I) Modification frequency per base over the hairpin structure. Position 29 is uncharacteristic of a stem and is called as modified to a greater degree. This high modification rate could be due to RNA conformational heterogeneity, RNA breathing, or a problematic k-mer. J) ROC curve for the hairpin at the consensus level (black) and individual bases (orange). A random model is shown with the dotted line. K) Groups of reads folding to the same structure are shown by abundance (y-axis) and minimum folding energy (x-axis). The inset contains selected structures with modified bases colored (red circles)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851280623" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Oxford nanopore direct RNA structure profiling of a hairpin RNA.</span> <strong>A</strong>) Molecular structure of Adenosine modification by DEPC that modifies structurally unconstrained adenosine by opening the imidazole ring. <strong>B</strong>) SMS-seq workflow. Structures are probed with DEPC and the current signal of each nucleotide that passes through the pore is recorded. Modified sites (red) are detected by comparing the current signal from treated samples to the control samples. <strong>C</strong>) Normalized current mean and standard deviation for a nucleotide at a single-stranded region of RNA modified with DEPC (red) and unmodified (gray). <strong>D</strong>) ROC curves for unmodified, modified, and denatured 5-mer RNA sequences. The 5-mer RNAs are treated with 1 % (black), 5 % (blue), 10 % (orange) DEPC as well as denatured RNA treated with 5 % DEPC (green). <strong>E</strong>) Denaturing 15% polyacrylamide gel electrophoresis of 5′-end radiolabeled RNA fragments generated from DEPC treated hairpin RNA (top) and untreated hairpin RNA (bottom), which were treated with borohydride and aniline. Arrows mark positions that were likely to be identified by SMS-seq as modified. <strong>F</strong>) Proton NMR spectra of non-treated (blue) and DEPC treated (red) for hairpin RNA. Singlet of protons on positions 8 and 2 on the purine ring could be quantified. <strong>G</strong>) Tombo modification statistics for adenosine bases. Bases are called modified (open) when their statistic is below 0.2 (dotted line) corresponding to an FDR of 15%. <strong>H</strong>) Heatmap showing 3093 full-length reads (y-axis) with modified (red) and unmodified (gray) nucleotides at each position (x-axis) for each read. Reads are ordered by the overall modification rate. The legend above (Structure) shows which nucleotides are accessible (white) and inaccessible (blue) in the hairpin structure. <strong>I</strong>) Modification frequency per base over the hairpin structure. Position 29 is uncharacteristic of a stem and is called as modified to a greater degree. This high modification rate could be due to RNA conformational heterogeneity, RNA breathing, or a problematic k-mer. <strong>J</strong>) ROC curve for the hairpin at the consensus level (black) and individual bases (orange). A random model is shown with the dotted line. <strong>K</strong>) Groups of reads folding to the same structure are shown by abundance (y-axis) and minimum folding energy (x-axis). The inset contains selected structures with modified bases colored (red circles).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F1.medium.gif" width="396" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F1.medium.gif" width="396" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Fig. 1.
                               </span>
                               <span class="caption-title">
                                Oxford nanopore direct RNA structure profiling of a hairpin RNA.
                               </span>
                               <p id="p-11" class="first-child">
                                <strong>
                                 A
                                </strong>
                                ) Molecular structure of Adenosine modification by DEPC that modifies structurally unconstrained adenosine by opening the imidazole ring.
                                <strong>
                                 B
                                </strong>
                                ) SMS-seq workflow. Structures are probed with DEPC and the current signal of each nucleotide that passes through the pore is recorded. Modified sites (red) are detected by comparing the current signal from treated samples to the control samples.
                                <strong>
                                 C
                                </strong>
                                ) Normalized current mean and standard deviation for a nucleotide at a single-stranded region of RNA modified with DEPC (red) and unmodified (gray).
                                <strong>
                                 D
                                </strong>
                                ) ROC curves for unmodified, modified, and denatured 5-mer RNA sequences. The 5-mer RNAs are treated with 1 % (black), 5 % (blue), 10 % (orange) DEPC as well as denatured RNA treated with 5 % DEPC (green).
                                <strong>
                                 E
                                </strong>
                                ) Denaturing 15% polyacrylamide gel electrophoresis of 5′-end radiolabeled RNA fragments generated from DEPC treated hairpin RNA (top) and untreated hairpin RNA (bottom), which were treated with borohydride and aniline. Arrows mark positions that were likely to be identified by SMS-seq as modified.
                                <strong>
                                 F
                                </strong>
                                ) Proton NMR spectra of non-treated (blue) and DEPC treated (red) for hairpin RNA. Singlet of protons on positions 8 and 2 on the purine ring could be quantified.
                                <strong>
                                 G
                                </strong>
                                ) Tombo modification statistics for adenosine bases. Bases are called modified (open) when their statistic is below 0.2 (dotted line) corresponding to an FDR of 15%.
                                <strong>
                                 H
                                </strong>
                                ) Heatmap showing 3093 full-length reads (y-axis) with modified (red) and unmodified (gray) nucleotides at each position (x-axis) for each read. Reads are ordered by the overall modification rate. The legend above (Structure) shows which nucleotides are accessible (white) and inaccessible (blue) in the hairpin structure.
                                <strong>
                                 I
                                </strong>
                                ) Modification frequency per base over the hairpin structure. Position 29 is uncharacteristic of a stem and is called as modified to a greater degree. This high modification rate could be due to RNA conformational heterogeneity, RNA breathing, or a problematic k-mer.
                                <strong>
                                 J
                                </strong>
                                ) ROC curve for the hairpin at the consensus level (black) and individual bases (orange). A random model is shown with the dotted line.
                                <strong>
                                 K
                                </strong>
                                ) Groups of reads folding to the same structure are shown by abundance (y-axis) and minimum folding energy (x-axis). The inset contains selected structures with modified bases colored (red circles).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Oxford nanopore sequencing and mapping
                             </h3>
                             <p id="p-12">
                              Library preparation: Sequencing libraries were prepared following the direct RNA sequencing protocol (Oxford Nanopore Technologies, ONT) by replacing the 3′ adapters (RTA) with custom-made barcodes for multiplexing (
                              <strong>
                               Supplementary Table 1
                              </strong>
                              ) (
                              <a href="https://github.com/hyeshik/poreplex">
                               https://github.com/hyeshik/poreplex
                              </a>
                              ). After renaturation and treatment with DEPC, in vitro synthesized and yeast non-poly(A) RNA were polyadenylated using E.coli polyA polymerase (NEB) at 37 °C for 15 min. To obtain full-length reads and a molecule long enough to be processed, the hairpin and riboswitches were ligated to the 3′ ends of a longer RNA molecule (
                              <strong>
                               Supplementary Table 1
                              </strong>
                              ). RTA adapter-ligated 500 ng RNAs were reverse-transcribed using superscript III. Different barcodes were used for the (+)DEPC and (−)DEPC libraries (
                              <strong>
                               Supplementary Table 1
                              </strong>
                              ). The first-strand complementary DNAs were pooled and a motor protein ligated, which were subjected to Oxford Nanopore sequencing using MinION Flow Cell (R9.4) and run for different durations until we obtained what we deemed to be a sufficient number of reads.
                             </p>
                             <p id="p-13">
                              Read mapping: Reads were demultiplexed using poreplex (v 0.5)(
                              <a href="https://github.com/hyeshik/poreplex">
                               https://github.com/hyeshik/poreplex
                              </a>
                              ) and resquiggled using Tombo v1.5 (ONT)(
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ) by mapping to their respective references. Yeast reference transcriptome was obtained from Ensembl (version R64-1-1). Reads were mapped by allowing mismatches, insertions, and deletions to account for nanopore sequencing errors using minimap2 (v 2.9)(
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Detection of DEPC modification
                             </h3>
                             <p id="p-14">
                              Chemical probing of adenine residues with DEPC. Several studies indicate that DEPC can be used for detecting single-stranded adenosine in RNA molecules (
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              –
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               20
                              </a>
                              ). Chemical reactivity is generally indicative of an absence of base stacking. Lack of reactivity, however, is more difficult to interpret since it could result not only from base pairing and tertiary interactions but also from the presence of stabilizing divalent cations (
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ).
                             </p>
                             <p id="p-15">
                              End labeling: The ring opening and the percentage of modifications in RNA by DEPC were validated using cleavage reaction and NMR. The cleavage reaction was performed on DEPC treated and untreated hairpin RNA as follows: 50 μL 1 M Tris, pH 8.0, and 25 μL of 0.4 M NaBH
                              <sub>
                               4
                              </sub>
                              (prepared fresh) and incubated on ice for 30 min in the dark followed by ethanol precipitation. The pellet was resuspended in 100 μL of 1 M aniline-acetate buffer (Aniline and Glacial Acetic acid, prepared fresh) and incubated at 60°C for 15 min in the dark followed by ethanol precipitation. The resulting fragments of RNA were end-labeled by
                              <sup>
                               32
                              </sup>
                              P-ATP using T4 polynucleotide kinase and run on 15 % denaturing PAGE.
                             </p>
                             <p id="p-16">
                              NMR: The abundance of adenosine in the DEPC treated and control hairpin RNAs sequence was quantified by proton NMR spectrometry using a Bruker BioSpin NEO600 spectrometer equipped with a cryogenic probe operating at 298 K for all measurements. We have standard pulse sequence noesygppr1d with a delay of 60 seconds to allow complete relaxation of nuclei between individual scans. Assignment of proton scans was confirmed by literature and by supplemental HSQC scans.
                             </p>
                             <p id="p-17">
                              Nucleotide modification analysis: We compared the (+)DEPC and (−)DEPC data sets and derived the final DEPC modification for adenosine using Tombo(
                              <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ) method model_sample_compare with --fishers-method-context 3. The modification status of a nucleotide was assigned in Tombo by comparing the current signal distribution between (+)DEPC and (−)DEPC samples (
                              <strong>
                               <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                Fig. 1B, C
                               </a>
                              </strong>
                              ). Next, the Tombo statistic was extracted for each read and each base using custom scripts (
                              <a href="https://git.app.uib.no/valenlab/sms-seq-pipeline">
                               https://git.app.uib.no/valenlab/sms-seq-pipeline
                              </a>
                              ). Due to Tombo working on the consensus level (average of all reads per position) we had to identify a threshold for the Tombo statistic which will allow us to work at the read level. We identified the false discovery rate (FDR) as a function of threshold using a synthetic RNA, which has adenine bases in a large number of different 5-kmer configurations (
                              <strong>
                               Supplementary Table 1
                              </strong>
                              ). This was achieved by detecting modifications on control replicate 1 against control replicate 2 in the pipeline - and evaluating FDR for different threshold values. As the most fitting selection of a threshold is arbitrary when its relation to FDR is linear, we decided on a threshold of 0.2 (equivalent to an FDR of 15%) (
                              <strong>
                               Supplementary Fig. 1)
                              </strong>
                              . To evaluate it further we checked that we can still separate modified adenosine from the unmodified at the consensus level when using our threshold on individual reads (ROC curve
                              <strong>
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Fig. 1D
                               </a>
                              </strong>
                              ), resulting in AUC values ∼0.95 for different modification rates. Furthermore, we calculated FDR for the unmodified ligated RNA part of the (+)DEPC hairpin (
                              <strong>
                               <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                                Fig. 1E
                               </a>
                              </strong>
                              ) to be 0.16 for the cutoff of 0.2 on the Tombo statistic (
                              <strong>
                               Supplementary Fig. 2A, B
                              </strong>
                              ). This indicates that the variance between different replicates is not influencing the FDR greatly and the Tombo method is stable. The modification rate for the hairpin was calculated to be around 31%. When accounting for 15% of false positives in the data we estimate the real modification rate to be around 26 %, similarly to the NMR result (24.7% ± 4.27%) (
                              <strong>
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Fig. 1F
                               </a>
                              </strong>
                              ). Next, we focused on evaluating the quality of calling open-closed regions using modified-unmodified states. To achieve that we calculated ROC curves for the highly stable hairpin construct, with a well-defined loop region (
                              <strong>
                               Supplementary Fig. 2
                              </strong>
                              ). For determining the accuracy of modification calling on the hairpin (ROC curves) we excluded reads with <4 and >18 modifications, which we deemed to be under- and over-modified, respectively (
                              <strong>
                               Supplementary Fig. 2C
                              </strong>
                              ). Additionally, Tombo does not output any statistics for the 10 last bases of the read, and due to the --fishers-method-context set to 3 we lose an additional three bases of the read.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Determination of dependencies
                             </h3>
                             <p id="p-18">
                              Adjusted mutual information is a variation of mutual information correcting for agreement due to chance (see
                              <strong>
                               Eq. 1-2
                              </strong>
                              , and ‘adjusted_mutual_info_score’ function in scikit-learn 0.22.1)(
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ) which can be calculated for pairs of nucleotides to see how informative knowing the modification state of one base is on predicting the state of the other base. Due to the high dependency of bases in close proximity, we normalize the AMI score between two bases by scaling AMI for each distance using z-score (dAMI). Therefore when multiple distances have high dAMI a cluster will become visible on the grid, indicating the mutual information.
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/23/2020.05.18.101402/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/23/2020.05.18.101402/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <strong>
                               Equation 1
                              </strong>
                              . Mutual information score (MI) is calculated between two positions U and V which are represented by vectors of modified/unmodified states (corresponding to 1 and 0) for each of the reads that span both positions U and V. We required at least 10 reads between positions U and V for calculations.
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/23/2020.05.18.101402/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/08/23/2020.05.18.101402/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <strong>
                               Equation 2
                              </strong>
                              . Adjusted mutual information score (AMI) normalizes MI to account for chance.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Deciphering RNA structural heterogeneity
                             </h3>
                             <p id="p-19">
                              The hairpin RNA and snoRNAs were folded using the ViennaRNA Package (
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ). To determine hairpin heterogeneity, all reads were folded in silico with modifications used as hard constraints (not allowed to basepair). The number of reads resulting in the same structure was counted. Yeast non-coding RNAs were folded – based on sequence minimum free energy (MFE), maximum expected accuracy (MEA), and centroid – and reads were clustered to them using the binary distance metric, i.e., the proportion of reads in which both of the two positions are modified/open relative to the number of reads where at least one is modified/open. The structures of the riboswitches were obtained from previous studies (
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-20">
                             High-throughput RNA structure probing commonly uses reagents that react with nucleotides to interrogate the chemical accessibility of a region. The position of chemical modification is then detected by reverse transcription drop-off and/or mutation (
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ). Among RNA structure probing reagents 2-methylnicotinic acid imidazolide (NAI) and dimethyl sulfate (DMS) have been used for both in vivo and in vitro RNA structure determination (
                             <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ), whereas diethyl pyrocarbonate (DEPC) has been used to map RNA structures in vitro (
                             <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ,
                             <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ). NAI acylates the 2’OH, DMS alkylates the N-7 position of guanosine and N-3 position of cytidine (
                             <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ), and DEPC carbethoxylates unpaired adenosine at the N-7 position (
                             <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ). To determine whether we could detect NAI, DMS and DEPC-modified bases using the Oxford Nanopore MinION sequencer we extracted E. coli 16S rRNA and probed it with the three reagents. The sequence quality and read length were drastically lower for NAI and DMS treated RNA compared to DEPC probed RNA (
                             <strong>
                              Supplementary Fig. 3
                             </strong>
                             ) similar to previous reports for NAI and DMS (
                             <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). The current signal in the sequencing pore is affected by 5-mer sequences occupying the reading head at a time. Thus, we assembled and cloned a 5-mer construct from oligos containing adenine bases in a diverse range of neighboring nucleotide sequence contexts. We synthesized RNA from this construct to probe using DEPC (see
                             <strong>
                              Methods
                             </strong>
                             ). Using this construct we compared the output from sequencing unmodified RNA molecules to RNA modified with DEPC at different concentrations (1%, 5%, 10%, and 5% for denatured RNA). The comparison was performed using Tombo which can detect significant differences in the ionic current running through pores when presented with a modified versus unmodified base (
                             <strong>
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                             </strong>
                             )(
                             <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ). This comparison demonstrated that modified bases could be distinguished from unmodified bases (
                             <strong>
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1C
                              </a>
                             </strong>
                             ) at high accuracy (AUC values ∼0.95,
                             <strong>
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                             </strong>
                             ). This is consistent with other studies that demonstrated chemical reagent-induced RNA modifications can be detected using Oxford nanopore sequencing (
                             <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ).
                            </p>
                            <p id="p-21">
                             To assess how well this approach could capture structural information, we synthesized and probed a hairpin RNA with a known, stable structure (
                             <strong>
                              Supplementary Fig. 2A
                             </strong>
                             ). We first determined DEPC modification through footprinting analysis where cleavage patterns indicate modifications in a single-stranded region of the hairpin (
                             <strong>
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig. 1E
                              </a>
                             </strong>
                             ). We then quantified the modification level of the hairpin RNA treated with DEPC using proton NMR. This showed a decrease of adenosine integral by 24.7% ± 4.27% (
                             <strong>
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Fig. 1F
                              </a>
                             </strong>
                             ). We sequenced hairpin RNA treated with DEPC and compared these with control samples obtaining 3093 and 4687 full-length reads, respectively. All hairpins were ligated to a longer unmodified molecule and the transition from the unmodified to the modified hairpin was readily distinguishable as a deviation from the expected output from the Oxford Nanopore MinION sequencer (
                             <strong>
                              Supplementary Fig. 2B
                             </strong>
                             ) confirming that modifications are readily detectable using nanopore sequencing.
                            </p>
                            <p id="p-22">
                             Recently, a nanopore sequencing-based method demonstrated consensus calling of RNA structure (
                             <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). Consensus calling calculates an average across all reads and therefore produces consensus structures, but RNA can potentially form a heterogeneous ensemble of structures (
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ). We, therefore, sought to call modifications at the level of individual bases on single molecules as this makes it possible to detect heterogeneous structures formed by the same sequence, modification dependencies across the molecule, and RNA breathing – the transient opening of RNA structure. Using Tombo we set a threshold for detecting individual bases on individual reads based on the false discovery rate (FDR). The FDR was calculated by predicting modifications in the untreated sequences ligated to the hairpins and reads from the unmodified 5-mer construct (see
                             <strong>
                              Methods
                             </strong>
                             ). We selected a threshold of 0.2, corresponding to a false discovery rate of ∼15%, which when applied to the hairpin (
                             <strong>
                              <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                               Fig. 1G
                              </a>
                             </strong>
                             ), resulted in an average of 7.43 modified bases over full-length reads (
                             <strong>
                              Supplementary Fig. 2C
                             </strong>
                             ). This corresponds to a modification rate of 26% and is consistent with the modification rate obtained from NMR. We observed a significant enrichment of modified bases in the open region versus the stem of the hairpin (
                             <strong>
                              <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                               Fig. 1H, I
                              </a>
                             </strong>
                             ) and obtained near-perfect consensus classification across all reads when considering open versus closed (stem) adenines (
                             <strong>
                              <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                               Fig. 1J
                              </a>
                              , Supplementary Fig. 4, 5A
                             </strong>
                             ). More interestingly, individual bases on individual reads achieved a sensitivity of 0.50 and specificity of 0.74 (
                             <strong>
                              <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                               Fig. 1J
                              </a>
                             </strong>
                             ). However, since the NMR results indicate that most accessible bases are not modified in every read, the number of false negatives is highly inflated and the sensitivity is therefore underestimated. False positives in the inaccessible regions come, at least partially, due to the nature of nanopore sequencing. Some k-mers are more sensitive than others to be detected as modified and modifications of neighboring nucleotides can influence the signal (
                             <strong>
                              Supplementary Fig. 5A
                             </strong>
                             ). However, some modifications within stems are likely to represent real, low-frequency conformations where these bases are transiently exposed and modified (“RNA breathing”) (
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ). To explore this, we folded individual reads with the ViennaRNA Package (
                             <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ) using hard constraints prohibiting modified bases from being part of a stem. Reassuringly, the most abundant structure was the expected hairpin structure indicating that 45.1% of reads have no modification conflicting with this structure. These were however heavily modified in the expected open regions (
                             <strong>
                              <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                               Fig. 1K
                              </a>
                             </strong>
                             inset,
                             <strong>
                              Supplementary Fig. 5B
                             </strong>
                             ), showing that the hairpin structure is indeed the most favorable conformation. Most alternative structures are only from a single read, and potentially represent noise in the calling of modifications (
                             <strong>
                              <a id="xref-fig-1-16" class="xref-fig" href="#F1">
                               Fig. 1K
                              </a>
                             </strong>
                             ). However, some structures were observed more frequently and showed bases modified in the hairpin stem and forming larger loop regions (
                             <strong>
                              <a id="xref-fig-1-17" class="xref-fig" href="#F1">
                               Fig. 1K
                              </a>
                             </strong>
                             , inset). The frequency of these more abundant structures was correlated with the minimum free energy (MFE) (
                             <strong>
                              <a id="xref-fig-1-18" class="xref-fig" href="#F1">
                               Fig. 1K
                              </a>
                             </strong>
                             ) indicating that these could represent transient structures. We explored this heterogeneity using an enzymatic cleavage assay (
                             <strong>
                              Supplementary Fig. 5C
                             </strong>
                             ) which revealed the presence of low-abundance open regions across the stem. Together these observations are consistent with the heterogeneity predicted from SMS-seq. This suggests that SMS-seq can measure structures beyond the consensus structure and reveal conformational heterogeneity or RNA breathing.
                            </p>
                            <p id="p-23">
                             Methods based on enzymatic cleavage or termination during reverse transcription can only capture independent measurements from a single location. Unlike these, SMS-seq can modify and record numerous exposed bases per molecule. SMS-seq is therefore capable of capturing dependencies of open regions across individual molecules. We first assessed the extent to which we could obtain information on the dependencies by calculating the adjusted mutual information (AMI) between all pairs of bases (see
                             <strong>
                              Methods
                             </strong>
                             ). Mutual information measures how much information the state of one base informs of the state of another base. Since calling the modification state of a base using nanopore sequencing is influenced by its neighboring bases, we scaled the AMI according to the distance between the pair of bases considered. Using z-score normalization (see
                             <strong>
                              Methods
                             </strong>
                             ), this provided us with a distance-normalized AMI (dAMI). Applying this metric to the hairpin revealed that bases that are part of the same structural unit share a high dAMI (
                             <strong>
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                             </strong>
                             ). In particular, we observed that the two base-paired regions forming the stem of the hairpin had a modification state that depends on each other (
                             <strong>
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              , black rectangle
                             </strong>
                             ) and to a lesser extent the bases in the loop (
                             <strong>
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              , black triangle
                             </strong>
                             ). While the dAMI can infer dependencies it does not distinguish between open and closed states, but only estimates the extent of information captured by one position in relation to the other. We, therefore, also calculated to what extent bases are co-observed in the same state. Specifically, we calculated co-openness, the probability that one base is modified given that the other is modified; and co-agreement, the fraction of reads where the pair share the same state (
                             <strong>
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                             </strong>
                             ). These metrics highlighted the open and stem regions, respectively. Together this demonstrated that SMS-seq can capture structural co-dependencies in stable structures.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F2.large.jpg?width=800&height=600&carousel=1" title="SMS-seq reveals structural dependencies in RNA. A) dAMI dependencies for pairs of bases in the synthetic hairpin construct. B) Z-score normalized co-openness and co-agreement scores for the synthetic hairpin construct. C) The predicted secondary structure obtained from a previous study(23) for the TPP riboswitch in unbound (left) and bound (right) configurations. Domains are indicated in colors: first stem (yellow), aptamer domain (blue), tertiary interaction of loop-stem (purple), and ribosome binding site (green). Only adenosine residues are colored. D) Log2 fold change in accessibility between bound and unbound states of TPP. Only adenosine residues are depicted. Red bars indicate significant changes (FDR adjusted Cochran–Mantel–Haenszel test q < 0.1) in the ligand-binding domain. E) dAMI dependencies between pairs of bases for the bound (upper) and unbound (lower) state of the TPP riboswitch. Domains are colored according to subfigure C. Three black rectangle labels as I, II, and III show some of the differences between bound and unbound TPP." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851280623" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">SMS-seq reveals structural dependencies in RNA.</span> <strong>A)</strong> dAMI dependencies for pairs of bases in the synthetic hairpin construct. <strong>B)</strong> Z-score normalized co-openness and co-agreement scores for the synthetic hairpin construct. <strong>C)</strong> The predicted secondary structure obtained from a previous study(23) for the TPP riboswitch in unbound (left) and bound (right) configurations. Domains are indicated in colors: first stem (yellow), aptamer domain (blue), tertiary interaction of loop-stem (purple), and ribosome binding site (green). Only adenosine residues are colored. <strong>D)</strong> Log2 fold change in accessibility between bound and unbound states of TPP. Only adenosine residues are depicted. Red bars indicate significant changes (FDR adjusted Cochran–Mantel–Haenszel test q < 0.1) in the ligand-binding domain. <strong>E)</strong> dAMI dependencies between pairs of bases for the bound (upper) and unbound (lower) state of the TPP riboswitch. Domains are colored according to subfigure C. Three black rectangle labels as I, II, and III show some of the differences between bound and unbound TPP.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F2.medium.gif" width="440" height="316"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F2.medium.gif" width="440" height="316"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig. 2.
                              </span>
                              <span class="caption-title">
                               SMS-seq reveals structural dependencies in RNA.
                              </span>
                              <p id="p-24" class="first-child">
                               <strong>
                                A)
                               </strong>
                               dAMI dependencies for pairs of bases in the synthetic hairpin construct.
                               <strong>
                                B)
                               </strong>
                               Z-score normalized co-openness and co-agreement scores for the synthetic hairpin construct.
                               <strong>
                                C)
                               </strong>
                               The predicted secondary structure obtained from a previous study(
                               <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                                23
                               </a>
                               ) for the TPP riboswitch in unbound (left) and bound (right) configurations. Domains are indicated in colors: first stem (yellow), aptamer domain (blue), tertiary interaction of loop-stem (purple), and ribosome binding site (green). Only adenosine residues are colored.
                               <strong>
                                D)
                               </strong>
                               Log2 fold change in accessibility between bound and unbound states of TPP. Only adenosine residues are depicted. Red bars indicate significant changes (FDR adjusted Cochran–Mantel–Haenszel test q < 0.1) in the ligand-binding domain.
                               <strong>
                                E)
                               </strong>
                               dAMI dependencies between pairs of bases for the bound (upper) and unbound (lower) state of the TPP riboswitch. Domains are colored according to subfigure C. Three black rectangle labels as I, II, and III show some of the differences between bound and unbound TPP.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-25">
                             We next asked whether SMS-seq could capture the dynamics and dependencies of structured riboswitches. We probed two riboswitches in their ligand-bound and unbound states: thiamine pyrophosphate (TPP (
                             <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ),
                             <strong>
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig. 2C
                              </a>
                             </strong>
                             ) and F. nucleatum Flavin mononucleotide (FMN (
                             <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ),
                             <strong>
                              Supplementary Fig. 6A
                             </strong>
                             ). As expected, both riboswitches became overall less accessible upon ligand-binding (
                             <strong>
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                              , Supplementary Fig. 6B
                             </strong>
                             ). Comparing between bound and unbound states of TPP revealed that, in particular, the aptamer domain (positions 11–88) exhibited significant differences in accessibility (
                             <strong>
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                             </strong>
                             ). The structural dependencies predicted by dAMI further showed that the aptamer domain and those that encompass the ribosome binding site (positions 125-146) are interdependent. This observation is consistent with the reduced access for ribosomes and translational inhibition upon ligand binding as previously observed (
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ). The high accessibility of the P1 helix, consisting of the pairing of base 11-14 with 85-88 (represented by position 12,
                             <strong>
                              <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                               Fig. 2C
                              </a>
                             </strong>
                             , yellow dot, and
                             <strong>
                              <a id="xref-fig-2-9" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                             </strong>
                             ) in the unbound state showed a strong decrease in accessibility in the bound state indicating its role in conformational change upon ligand recognition. Our dAMI analysis further supported this by showing the role of the first stem (represented by position 12) on the overall architecture of TPP (
                             <strong>
                              <a id="xref-fig-2-10" class="xref-fig" href="#F2">
                               Fig. 2E
                              </a>
                             </strong>
                             , a diagonal line extending from 12) including the ribosome binding site (
                             <strong>
                              <a id="xref-fig-2-11" class="xref-fig" href="#F2">
                               Fig. 2E, box I
                              </a>
                             </strong>
                             ). The dependency of the expression domain on the conformational change of the TPP binding pocket in the presence of TPP is also evident (
                             <strong>
                              <a id="xref-fig-2-12" class="xref-fig" href="#F2">
                               Fig. 2E, box II
                              </a>
                             </strong>
                             ). Interestingly, dAMI also revealed that SMS-seq is capable of capturing the tertiary interactions between the two helices (
                             <strong>
                              <a id="xref-fig-2-13" class="xref-fig" href="#F2">
                               Fig. 2E, box III
                              </a>
                             </strong>
                             ), which has been demonstrated in mutational profiling (RING-MaP) and high-resolution TPP structure mapping (
                             <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ), where the interaction between nucleotides in the L5 loop (bases from 67 to 72) and the P3 helix (a stem structure formed by base pairing of nucleotide 21-27 and 33-38) enables the formation of the ligand-binding pocket. Although RING-Map and SMS-seq agree on loop-helix interaction, they are not completely consistent in their predictions (
                             <strong>
                              Supplementary Fig. 7
                             </strong>
                             ). For instance, RING-MaP suggests RNA interaction groups of L5 with P2 (base pairing between 15-18 and 48-51) and L5 with P4 (base pairing between 56-60 and 80-83), which might not be true since the hinge-like movement of helix P5 relative to P4 (
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ) may not allow such an interaction (
                             <strong>
                              Supplementary Fig. 7
                             </strong>
                             ). Similar observations can be made for the FMN where SMS-seq captures the extensive dependencies (stem-loop and loop-loop) of the two wings of FMN (
                             <strong>
                              Supplementary Fig. 6C, D
                             </strong>
                             ). Together, these results demonstrate that SMS-seq can capture secondary and tertiary dependencies and the conformational changes of riboswitches upon ligand binding.
                            </p>
                            <p id="p-26">
                             To investigate the structural heterogeneity of RNA molecules in solution, we folded the FMN riboswitch in different concentrations of magnesium ion (
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ,
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ). Similar to a previous report (
                             <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ) SMSseq revealed different modification patterns, suggesting heterogeneous conformations of FMN (
                             <strong>
                              Supplementary Fig. 6E
                             </strong>
                             ).
                            </p>
                            <p id="p-27">
                             Encouraged by the results on synthetic RNA structures, we set out to investigate the structure of non-coding RNAs and generated non-poly(A) RNA libraries from yeast (Saccharomyces cerevisiae) and folded these in vitro in the presence and absence of DEPC. We first aligned the reads to in silico folded structures and to known structures from RNACentral (
                             <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                              30
                             </a>
                             ). For structures such as the small nucleolar RNA (snoRNA) snR37 and snR17a where we had many reads, we observed overall agreement between reads and their reported or energetically favorable structures (
                             <strong>
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3A, B
                              </a>
                             </strong>
                             showing snR37). To investigate whether reads could represent structural heterogeneity we attempted to cluster reads to multiple alternative conformations predicted by ViennaRNA: the minimum free energy, maximum expected accuracy, and centroid structure for snR17a. We observed that reads clustered preferentially to these specific structures (
                             <strong>
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                             </strong>
                             ) and rejected randomly shuffled versions of these (t-test p < 2.2e-16, see Methods). We similarly obtained thousands of reads for rRNAs showing modification patterns in agreement with sub-structures of this long molecule (
                             <strong>
                              Supplementary Fig. 8)
                             </strong>
                             . Together these findings highlight that SMS-seq can capture multiple structures of an RNA.
                            </p>
                            <div id="F3" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F3.large.jpg?width=800&height=600&carousel=1" title="Structural probing of non-coding RNA. A) Two structures of snR37, in silico-folded maximum expected accuracy (MEA, left) and structure from RNACentral(35) (right). Bases are colored according to their accessibility relative to the mean accessibility for this molecule. B) Accessibility per position relative to mean (upper panel), Structure by position for MEA and Reported structures (mid panel) where stem (blue) and open (gray), and all reads mapping to snR37 with bases colored according to whether they are modified (red) or not (black), (lower panel). C) Reads mapping to snR17a clustered by their similarity to three in silico predicted structures. On the left, structures represent folding based on minimum free energy (MFE), maximum expected accuracy (MAE), and centroid. On the right, nanopore reads are clustered towards each of the predicted structures (the red in the heatmap and the gray bars on the top of it mark the open regions). To assess the concordance of read modifications to in silico predicted structures we clustered reads against these and randomly permuted structures ten-thousand times. This demonstrated that reads overwhelmingly agreed with predicted structures compared to permuted (t-test p-value < 2.2e-16)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851280623" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Structural probing of non-coding RNA.</span> <strong>A)</strong> Two structures of snR37, <em>in silico</em>-folded maximum expected accuracy (MEA, left) and structure from RNACentral(35) (right). Bases are colored according to their accessibility relative to the mean accessibility for this molecule. <strong>B)</strong> Accessibility per position relative to mean (upper panel), Structure by position for MEA and Reported structures (mid panel) where stem (blue) and open (gray), and all reads mapping to snR37 with bases colored according to whether they are modified (red) or not (black), (lower panel). <strong>C)</strong> Reads mapping to snR17a clustered by their similarity to three <em>in silico</em> predicted structures. On the left, structures represent folding based on minimum free energy (MFE), maximum expected accuracy (MAE), and centroid. On the right, nanopore reads are clustered towards each of the predicted structures (the red in the heatmap and the gray bars on the top of it mark the open regions). To assess the concordance of read modifications to <em>in silico</em> predicted structures we clustered reads against these and randomly permuted structures ten-thousand times. This demonstrated that reads overwhelmingly agreed with predicted structures compared to permuted (t-test p-value < 2.2e-16).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F3.medium.gif" width="313" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F3.medium.gif" width="313" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig. 3.
                              </span>
                              <span class="caption-title">
                               Structural probing of non-coding RNA.
                              </span>
                              <p id="p-28" class="first-child">
                               <strong>
                                A)
                               </strong>
                               Two structures of snR37, in silico-folded maximum expected accuracy (MEA, left) and structure from RNACentral(
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                               ) (right). Bases are colored according to their accessibility relative to the mean accessibility for this molecule.
                               <strong>
                                B)
                               </strong>
                               Accessibility per position relative to mean (upper panel), Structure by position for MEA and Reported structures (mid panel) where stem (blue) and open (gray), and all reads mapping to snR37 with bases colored according to whether they are modified (red) or not (black), (lower panel).
                               <strong>
                                C)
                               </strong>
                               Reads mapping to snR17a clustered by their similarity to three in silico predicted structures. On the left, structures represent folding based on minimum free energy (MFE), maximum expected accuracy (MAE), and centroid. On the right, nanopore reads are clustered towards each of the predicted structures (the red in the heatmap and the gray bars on the top of it mark the open regions). To assess the concordance of read modifications to in silico predicted structures we clustered reads against these and randomly permuted structures ten-thousand times. This demonstrated that reads overwhelmingly agreed with predicted structures compared to permuted (t-test p-value < 2.2e-16).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-29">
                             We finally expanded our SMS-seq approach to endogenous mRNA and isolated poly(A) RNA from yeast and probed these in vitro using DEPC. Average modification rates were highly reproducible across replicates (
                             <strong>
                              Supplementary Fig. 9
                             </strong>
                             ) and similar modification patterns were observed along the mRNAs (
                             <strong>
                              Supplementary Fig. 10
                             </strong>
                             ). We compared these mRNA-selected libraries with libraries based on enzyme-based PARS-assisted folding (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ), and observed a significant difference in modification rates between nucleotides classified by PARS as open and closed (
                             <strong>
                              Supplementary Fig. 11A
                             </strong>
                             ). We could also observe known structural elements in mRNAs (
                             <strong>
                              Supplementary Fig. 11B
                             </strong>
                             ) in line with a previous study (
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ). Globally, our approach was able to confirm known properties of mRNA structure showing an increase in modifications upstream of translation start sites and translation stop sites consistent with these regions being open (
                             <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ) (
                             <strong>
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                             </strong>
                             ). Furthermore, 5’UTRs overall showed a higher modification rate and higher accessibility compared to CDSs, while 3’UTRs were overall less accessible (
                             <strong>
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                             </strong>
                             ) At the global level the average modification rate among mRNAs varies considerably (
                             <strong>
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4C
                              </a>
                             </strong>
                             ). Some genes like HAC1 are predicted to form extensive base pairing consistent with reports that HAC1 form elaborate structures through long-range base pairing for intron splicing (
                             <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                              33
                             </a>
                             ). Other transcripts appear to be overall unstructured. To demonstrate the utility of SMS-seq we investigated more specific regions to study their structural features. For instance, by splitting translation termination sites into the 3 different stop codons (UAA, UGA, UAG), SMS-seq revealed that modification rates decreased at the beginning of 3’UTR for UGA stop codons specifically (
                             <strong>
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4D
                              </a>
                             </strong>
                             ). SMS-seq also predicted more accessible regions around several polyadenylation signal motifs (
                             <strong>
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4E
                              </a>
                             </strong>
                             ). Together these observations show that SMS-seq can be used to probe structural features of the transcriptome.
                            </p>
                            <div id="F4" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F4.large.jpg?width=800&height=600&carousel=1" title="mRNA accessibility and features. A) Metaplot of the genes over translation initiation site and translation termination sites. Each gene is individually z-score normalized and then averaged across all genes. Positions with a z-score below 0 are less accessible than average. B) Comparison of CDS and UTR modification rate. Significant differences were detected using Student’s t-test. C) Accessibility around translation termination sites grouped by stop codons. D) Genes ranked by their average modification rate show a range of accessibilities. The five lowest and highest are labeled by their gene name. E) Polyadenylation signal sequences of yeast mRNA and their modification rate 50 nt upstream and 20 nts downstream. The zero position represents the start of the sequence motif." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1851280623" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">mRNA accessibility and features.</span> A) Metaplot of the genes over translation initiation site and translation termination sites. Each gene is individually z-score normalized and then averaged across all genes. Positions with a z-score below 0 are less accessible than average. B) Comparison of CDS and UTR modification rate. Significant differences were detected using Student’s t-test. C) Accessibility around translation termination sites grouped by stop codons. D) Genes ranked by their average modification rate show a range of accessibilities. The five lowest and highest are labeled by their gene name. E) Polyadenylation signal sequences of yeast mRNA and their modification rate 50 nt upstream and 20 nts downstream. The zero position represents the start of the sequence motif.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F4.medium.gif" width="440" height="434"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F4.medium.gif" width="440" height="434"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/08/23/2020.05.18.101402/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig. 4.
                              </span>
                              <span class="caption-title">
                               mRNA accessibility and features.
                              </span>
                              <p id="p-30" class="first-child">
                               A) Metaplot of the genes over translation initiation site and translation termination sites. Each gene is individually z-score normalized and then averaged across all genes. Positions with a z-score below 0 are less accessible than average. B) Comparison of CDS and UTR modification rate. Significant differences were detected using Student’s t-test. C) Accessibility around translation termination sites grouped by stop codons. D) Genes ranked by their average modification rate show a range of accessibilities. The five lowest and highest are labeled by their gene name. E) Polyadenylation signal sequences of yeast mRNA and their modification rate 50 nt upstream and 20 nts downstream. The zero position represents the start of the sequence motif.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-31">
                             We here introduced a method for single-molecule probing of RNA structures, SMS-seq, that does not rely on amplification or size selection, and does not need to be normalized based on position in the molecule – common sources of biases in previous methods (
                             <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              34
                             </a>
                             ). Unlike many methods, SMS-seq does not rely on reverse transcription thus avoiding another source of bias. We have shown that SMS-seq can detect transient and stable RNA structures, secondary and tertiary structural elements, structural dependencies of bases, and mRNA structural features.
                            </p>
                            <p id="p-32">
                             Since SMS-seq is a single-molecule method it obtains structural information from individual RNA molecules and can therefore be used to uncover heterogeneity and the dynamic nature of RNA structures in different conditions. A previous method, RING-MaP, could capture within molecule correlations of modified bases at the consensus level (
                             <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ); however, they are incapable of capturing longer RNA-RNA interactions. Other clustering-based algorithms like DREEM (
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ) and DRACO (
                             <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ) are able to detect heterogeneity of RNA structure but suffer from experimental artifacts of mutational profiling and short-read sequences when used to determine larger RNA structures.
                            </p>
                            <p id="p-33">
                             Previously, nanopore direct RNA sequencing was demonstrated in the determination of structures of pre-miRNA clusters and mRNA isoforms (
                             <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). While these results revealed consensus structures, SMS-seq probes multiple bases in each individual molecule and uses this to investigate the interdependency of these positions and the overall structure. While obtaining these dependencies requires substantial coverage, we expect that with the rapid improvement to modification-calling algorithms, the sensitivity of our approach will increase and can eventually be used as a constraint for RNA structure prediction algorithms.
                            </p>
                            <p id="p-34">
                             An important aspect of SMS-seq and other nanopore-based structure probing methods is that the sensitivity of the method to detect long-range interaction largely depends on a high modification rate combined with high integrity of the RNA. RNA structure probing using SHAPE reagents (2-methylnicotinic acid imidazolide (NAI), 1-methyl-7-nitroisatoic anhydride (1M7), and N-methylisatoic anhydride (NMIA)), dimethyl sulfate (DMS) and nanopore sequencing resulted in low quality, truncated and unmappable sequences (
                             <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ). In contrast, DEPC resulted in multiple modifications per read and a higher read integrity at the expense of only being applicable for in vitro experiments. However, as a method, SMS-seq can be effectively adapted for RNA-structure in vivo probing assays when in vivo probing agents obtain comparable high-modification, high-integrity reads.
                            </p>
                            <p id="p-35">
                             Another shortcoming of DEPC is its lack of ability to modify other bases than adenosine. Although a new probing reagent that could target all 4 nucleotides, acetylimidazole, has been synthesized to use in nanopore-based structure determination, it is shown that at the higher concentrations necessary to obtain extensive modification rate the quality and fraction of full-length reads were reduced (
                             <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ). Further improvements in RNA probing reagents without compromising the RNA integrity are expected to advance RNA structure determination.
                            </p>
                            <p id="p-36">
                             Building on our and other long-read-based approaches with increased accuracy and structure prediction, we expect full-length single-molecule structural probing of native RNA by direct RNA sequencing will eventually become a standard tool in predicting secondary structure and inferring structural dependencies and tertiary interactions.
                            </p>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Data and materials availability
                             </h3>
                             <p id="p-37">
                              The sequencing files are available on the European Nucleotide Archive website, under accession code PRJEB36658. Scripts used to process the data are available on
                              <a href="https://git.app.uib.no/valenlab/sms-seq">
                               https://git.app.uib.no/valenlab/sms-seq
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-39">
                             T.T.B. conceived the study, performed the principal experiments, analyzed part of the results, and wrote the manuscript. K.L. analyzed the data, wrote part of the manuscript. M.J. performed NMR analysis. K.J. performed part of the experiments. A.N. contributed for data analysis. EV conceived the study, supervised the study, analyzed part of the results, and wrote the manuscript. All authors read and approved the manuscript.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-38">
                             We would like to acknowledge Ruth Brenk (University of Bergen) for providing FMN riboswitch RNA. We also acknowledge our lab members for detailed and constructive comments that helped to improve this study. The study was funded by Norwegian Research Council (project number: 250049) and Trond Mohn Foundation.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               All sections, figures and supplemental files are updated using additional experimental data. New authors are added.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://www.ebi.ac.uk/ena/data/search?query=+PRJEB36658">
                                https://www.ebi.ac.uk/ena/data/search?query=+PRJEB36658
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.1" data-doi="10.1038/s41576-018-0034-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Strobel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lucks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  High-throughput determination of RNA structures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  615
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  634
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BGenetics%26rft.volume%253D19%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-018-0034-x%26rft_id%253Dinfo%253Apmid%252F30054568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41576-018-0034-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30054568&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smola
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rice
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siegfried
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Selective 2′-hydroxyl acylation analyzed by primer extension and mutational profiling (SHAPE-MaP) for direct, versatile and accurate RNA structure analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Protocols
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1643
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1669
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BProtocols%26rft.volume%253D10%26rft.spage%253D1643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.3" data-doi="10.1073/pnas.1407306111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Homan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Favorov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lavender
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kursun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dokholyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.V.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Single-molecule correlated chemical probing of RNA. Proc
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Natl. Acad. Sci. U. S. A
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  13858
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13863
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1407306111%26rft_id%253Dinfo%253Apmid%252F25205807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzM4LzEzODU4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDgvMjMvMjAyMC4wNS4xOC4xMDE0MDIuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.18.101402v2.4" data-doi="10.1038/nprot.2016.011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flynn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spitale
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mumbach
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Transcriptome-wide interrogation of RNA secondary structure in living cells with icSHAPE
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Protoc
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  273
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  290
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc%26rft.volume%253D11%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2016.011%26rft_id%253Dinfo%253Apmid%252F26766114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nprot.2016.011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26766114&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.5" data-doi="10.1038/nmeth.3029">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siegfried
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rice
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nelson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A.E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  RNA motif discovery by SHAPE and mutational profiling (SHAPE-MaP)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  959
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  965
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D959%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3029%26rft_id%253Dinfo%253Apmid%252F25028896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3029&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25028896&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000341409700026&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tomezsko
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Corbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.D.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gupta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Swaminathan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Glasgow
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Persad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edwards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mcintosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Papenfuss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Emery
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Determination of RNA structural diversity and its role in HIV-1 RNA splicing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  582
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  438
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  442
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D582%26rft.spage%253D438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morandi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Manfredonia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anselmi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Hemert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oliviero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Incarnato
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Genome-scale deconvolution of RNA structure ensembles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  252
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D18%26rft.spage%253D249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.8" data-doi="10.1021/acs.biochem.7b00323">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sexton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rutenberg-Schoenberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Interpreting Reverse Transcriptase Termination and Mutation Events for Greater Insight into the Chemical Probing of RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  56
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4713
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4721
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D56%26rft.spage%253D4713%26rft_id%253Dinfo%253Adoi%252F10.1021%252Facs.biochem.7b00323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/acs.biochem.7b00323&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stephenson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Razaghi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Timp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smibert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Direct detection of RNA modifications and structure using single-molecule nanopore sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Genom
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.R.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaewsapsak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Determination of isoform-specific RNA structure with nanopore long reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biotechnol
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.11" data-doi="10.1038/nature07642">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Serganov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Coenzyme recognition and gene regulation by a flavin mononucleotide riboswitch
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  458
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  233
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  237
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSerganov%26rft.auinit1%253DA.%26rft.volume%253D458%26rft.issue%253D7235%26rft.spage%253D233%26rft.epage%253D237%26rft.atitle%253DCoenzyme%2Brecognition%2Band%2Bgene%2Bregulation%2Bby%2Ba%2Bflavin%2Bmononucleotide%2Briboswitch.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07642%26rft_id%253Dinfo%253Apmid%252F19169240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature07642&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19169240&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000264059700050&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.12" data-doi="10.1021/bi00794a003">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leonard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McDonald
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henderson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reichmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1971
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Reaction of diethyl pyrocarbonate with nucleic acid components. Adenosine
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3335
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3342
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D10%26rft.spage%253D3335%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00794a003%26rft_id%253Dinfo%253Apmid%252F5118618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1021/bi00794a003&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=5118618&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1971K190400003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Oxford Nanopore Technologies
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 )
                                 <abbr class="cit-jnl-abbrev">
                                  Tombo: detection of non-standard nucleotides using the genome-resolved raw nanopore signal
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.14" data-doi="10.1093/bioinformatics/bty191">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Minimap2: pairwise alignment for nucleotide sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3094
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3100
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29750242&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rairkar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rubino
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lockard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1988
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Chemical probing of adenine residues within the secondary structure of rabbit 18S ribosomal RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  582
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  592
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D27%26rft.spage%253D582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.16" data-doi="10.1073/pnas.77.8.4679">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Peattie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gilbert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1980
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Chemical probes for higher-order structure in RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  77
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4679
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4682
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DPeattie%26rft.auinit1%253DD.%2BA.%26rft.volume%253D77%26rft.issue%253D8%26rft.spage%253D4679%26rft.epage%253D4682%26rft.atitle%253DChemical%2Bprobes%2Bfor%2Bhigher-order%2Bstructure%2Bin%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.8.4679%26rft_id%253Dinfo%253Apmid%252F6159633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny84LzQ2NzkiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wOC8yMy8yMDIwLjA1LjE4LjEwMTQwMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.18.101402v2.17" data-doi="10.1016/0022-2836(86)90143-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moazed
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Stolk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Douthwaite
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.F.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1986
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Interconversion of active and inactive 30 S ribosomal subunits is accompanied by a conformational change in the decoding region of 16 S rRNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mol. Biol
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  191
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  483
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  493
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMoazed%26rft.auinit1%253DD.%26rft.volume%253D191%26rft.issue%253D3%26rft.spage%253D483%26rft.epage%253D493%26rft.atitle%253DInterconversion%2Bof%2Bactive%2Band%2Binactive%2B30%2BS%2Bribosomal%2Bsubunits%2Bis%2Baccompanied%2Bby%2Ba%2Bconformational%2Bchange%2Bin%2Bthe%2Bdecoding%2Bregion%2Bof%2B16%2BS%2BrRNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252886%252990143-9%26rft_id%253Dinfo%253Apmid%252F2434656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0022-2836(86)90143-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=2434656&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1986E317300017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.05.18.101402v2.18" data-doi="10.1016/0022-2836(84)90435-2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Stolk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.J.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.F.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1984
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Chemical probing of conformation in large RNA molecules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Molecular Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  180
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  151
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  177
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DVan%2BStolk%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D180%26rft.issue%253D1%26rft.spage%253D151%26rft.epage%253D177%26rft.atitle%253DChemical%2Bprobing%2Bof%2Bconformation%2Bin%2Blarge%2BRNA%2Bmolecules.%2BAnalysis%2Bof%2B16%2BS%2Bribosomal%2BRNA%2Busing%2Bdiethylpyrocarbonate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252884%252990435-2%26rft_id%253Dinfo%253Apmid%252F6210372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0022-2836(84)90435-2&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=6210372&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1984TW97300009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.19" data-doi="10.1016/0022-2836(91)80220-O">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brunel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Romby
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Westhof
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ehresmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ehresmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Three-dimensional model of Escherichia coli ribosomal 5 S RNA as deduced from structure probing in solution and computer modeling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mol. Biol
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  221
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  293
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  308
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBrunel%26rft.auinit1%253DC.%26rft.volume%253D221%26rft.issue%253D1%26rft.spage%253D293%26rft.epage%253D308%26rft.atitle%253DThree-dimensional%2Bmodel%2Bof%2BEscherichia%2Bcoli%2Bribosomal%2B5%2BS%2BRNA%2Bas%2Bdeduced%2Bfrom%2Bstructure%2Bprobing%2Bin%2Bsolution%2Band%2Bcomputer%2Bmodeling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252891%252980220-O%26rft_id%253Dinfo%253Apmid%252F1717695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0022-2836(91)80220-O&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1717695&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991GG37200027&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.20" data-doi="10.1074/jbc.M803109200">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reineke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Komar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caprara
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.G.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Merrick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  A small stem loop element directs internal initiation of the URE2 internal ribosome entry site in Saccharomyces cerevisiae
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Biol. Chem
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  283
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  19011
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19025
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M803109200%26rft_id%253Dinfo%253Apmid%252F18460470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvMjcvMTkwMTEiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wOC8yMy8yMDIwLjA1LjE4LjEwMTQwMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.21" data-doi="10.1016/j.cpc.2010.04.018">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pedregosa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Varoquaux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gramfort
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Michel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirion
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grisel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blondel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Prettenhofer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weiss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dubourg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Scikit-learn: Machine Learning in Python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mach. Learn. Res
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2825
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2830
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BMach.%2BLearn.%2BRes%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23755062&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.22" data-doi="10.1186/1748-7188-6-26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lorenz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bernhart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Höner Zu Siederdissen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tafer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flamm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stadler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.F.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hofacker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  ViennaRNA Package 2.0
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Algorithms Mol. Biol
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  26
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DAlgorithms%2BMol%2BBiol%26rft.aulast%253DLorenz%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DViennaRNA%2BPackage%2B2.0.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1748-7188-6-26%26rft_id%253Dinfo%253Apmid%252F22115189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1748-7188-6-26&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22115189&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.23" data-doi="10.1038/nature04740">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Serganov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Polonskaia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Breaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.R.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Structural basis for gene regulation by a thiamine pyrophosphate-sensing riboswitch
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  441
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1167
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1171
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSerganov%26rft.auinit1%253DA.%26rft.volume%253D441%26rft.issue%253D7097%26rft.spage%253D1167%26rft.epage%253D1171%26rft.atitle%253DStructural%2Bbasis%2Bfor%2Bgene%2Bregulation%2Bby%2Ba%2Bthiamine%2Bpyrophosphate-sensing%2Briboswitch.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04740%26rft_id%253Dinfo%253Apmid%252F16728979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature04740&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16728979&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000238615500054&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spitale
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crisalli
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Flynn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Torre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kool
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.T.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  RNA SHAPE analysis in living cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Chemical Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  18
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  20
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BChemical%2BBiology%26rft.volume%253D9%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.25" data-doi="10.1038/nrm.2017.103">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leppek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Das
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Functional 5′ UTR mRNA structures in eukaryotic translation regulation and how to find them
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Molecular Cell Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  158
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  174
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BMolecular%2BCell%2BBiology%26rft.volume%253D19%26rft.spage%253D158%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm.2017.103%26rft_id%253Dinfo%253Apmid%252F29165424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrm.2017.103&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29165424&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.26" data-doi="10.1038/nature01145">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winkler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nahvi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Breaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Thiamine derivatives bind messenger RNAs directly to regulate bacterial gene expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  419
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  952
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  956
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWinkler%26rft.auinit1%253DW.%26rft.volume%253D419%26rft.issue%253D6910%26rft.spage%253D952%26rft.epage%253D956%26rft.atitle%253DThiamine%2Bderivatives%2Bbind%2Bmessenger%2BRNAs%2Bdirectly%2Bto%2Bregulate%2Bbacterial%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01145%26rft_id%253Dinfo%253Apmid%252F12410317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature01145&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12410317&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000178909700050&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.27" data-doi="10.1073/pnas.1218062110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Altman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Soulière
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blanchard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.C.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Micura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Folding and ligand recognition of the TPP riboswitch aptamer at single-molecule resolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U. S. A
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  110
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4188
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4193
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.%2BS.%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1218062110%26rft_id%253Dinfo%253Apmid%252F23440214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzExLzQxODgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wOC8yMy8yMDIwLjA1LjE4LjEwMTQwMi5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.28" data-doi="10.1006/jmbi.2002.5422">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Misra
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.K.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Draper
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  The linkage between magnesium binding and RNA folding
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Mol. Biol
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  317
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  507
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  521
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DMisra%26rft.auinit1%253DV.%2BK.%26rft.volume%253D317%26rft.issue%253D4%26rft.spage%253D507%26rft.epage%253D521%26rft.atitle%253DThe%2Blinkage%2Bbetween%2Bmagnesium%2Bbinding%2Band%2BRNA%2Bfolding.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2002.5422%26rft_id%253Dinfo%253Apmid%252F11955006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/jmbi.2002.5422&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11955006&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000175070100003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.29" data-doi="10.1093/nar/gkr565">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vicens
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mondragón
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Batey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.T.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Molecular sensing by the aptamer domain of the FMN riboswitch: a general model for ligand binding by conformational selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  8586
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8598
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr565%26rft_id%253Dinfo%253Apmid%252F21745821%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkr565&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21745821&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000296341100032&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  RNAcentral Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  RNAcentral 2021: secondary structure integration, improved sequence search and new member databases
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D212
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D220
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D49%26rft.spage%253DD212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.31" data-doi="10.1038/nature09322">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kertesz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mazor
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rinn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nutter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.Y.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Segal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Genome-wide measurement of RNA secondary structure in yeast
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  467
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  103
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  107
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKertesz%26rft.auinit1%253DM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D103%26rft.epage%253D107%26rft.atitle%253DGenome-wide%2Bmeasurement%2Bof%2BRNA%2Bsecondary%2Bstructure%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09322%26rft_id%253Dinfo%253Apmid%252F20811459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature09322&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20811459&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000281461200044&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.32" data-doi="10.1038/nature12894">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rouskin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zubradt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Washietl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kellis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weissman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  505
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  701
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  705
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12894%26rft_id%253Dinfo%253Apmid%252F24336214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature12894&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24336214&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000330321000045&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.33" data-doi="10.1016/S0092-8674(01)00505-0">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rüegsegger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Leber
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.H.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Block of HAC1 mRNA translation by long-range base pairing is released by cytoplasmic splicing upon induction of the unfolded protein response
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  103
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  114
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DR%25C3%25BCegsegger%26rft.auinit1%253DU.%26rft.volume%253D107%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D114%26rft.atitle%253DBlock%2Bof%2BHAC1%2BmRNA%2Btranslation%2Bby%2Blong-range%2Bbase%2Bpairing%2Bis%2Breleased%2Bby%2Bcytoplasmic%2Bsplicing%2Bupon%2Binduction%2Bof%2Bthe%2Bunfolded%2Bprotein%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900505-0%26rft_id%253Dinfo%253Apmid%252F11595189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00505-0&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11595189&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F08%2F23%2F2020.05.18.101402.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000171453100013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weidmann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sengupta
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeks
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  Guidelines for SHAPE Reagent Choice and Detection Strategy for RNA Structure Probing Studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochemistry
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  58
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2655
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2664
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D58%26rft.spage%253D2655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.18.101402v2.35" data-doi="10.1093/nar/gky1206">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  The RNAcentral Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 )
                                 <span class="cit-article-title">
                                  RNAcentral: a hub of information for non-coding RNA sequences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D1250
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D1251
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD1250%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1206&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.18.101402v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.01.03.473214v4" title="Nm-Nano: Predicting 2´-O-Methylation (Nm) Sites in Nanopore RNA Sequencing Data" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted August 23, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.18.101402v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.05.18.101402v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/08/23/2020.05.18.101402.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.18.101402v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.18.101402v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2705725"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-diaxrZ1UJOTV2iO-4nRTaXtDESmdv7TyovPwHEZy8r0"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484470910"/>
                                   <input type="hidden" name="captcha_token" value="bb97a9a99eb8fdf08520b5275cd6d3b0"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2705725--3" data-node-nid="2705725" id="node-2705725--4287796128" data-pisa="biorxiv;2020.05.18.101402v2" data-pisa-master="biorxiv;2020.05.18.101402" data-seqnum="2705725" data-apath="/biorxiv/early/2022/08/23/2020.05.18.101402.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Teshome Tilahun
                                  </span>
                                  <span class="nlm-surname">
                                   Bizuayehu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Kornel
                                  </span>
                                  <span class="nlm-surname">
                                   Labun
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Martin
                                  </span>
                                  <span class="nlm-surname">
                                   Jakubec
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Kirill
                                  </span>
                                  <span class="nlm-surname">
                                   Jefimov
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Adnan
                                  </span>
                                  <span class="nlm-surname">
                                   Niazi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Eivind
                                  </span>
                                  <span class="nlm-surname">
                                   Valen
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.18.101402;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.18.101402
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&title=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&title=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&text=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&t=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&title=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&title=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&title=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2705725--5" data-node-nid="2705725" id="citation-node-2705725--61361179401" data-pisa="biorxiv;2020.05.18.101402v2" data-pisa-master="biorxiv;2020.05.18.101402" data-seqnum="2705725" data-apath="/biorxiv/early/2022/08/23/2020.05.18.101402.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Single molecule structure sequencing reveals RNA structural dependencies, breathing and ensembles
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Teshome Tilahun
                                      </span>
                                      <span class="nlm-surname">
                                       Bizuayehu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Kornel
                                      </span>
                                      <span class="nlm-surname">
                                       Labun
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Martin
                                      </span>
                                      <span class="nlm-surname">
                                       Jakubec
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Kirill
                                      </span>
                                      <span class="nlm-surname">
                                       Jefimov
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Adnan
                                      </span>
                                      <span class="nlm-surname">
                                       Niazi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Eivind
                                      </span>
                                      <span class="nlm-surname">
                                       Valen
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.18.101402;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.18.101402
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Bizuayehu%2C+Teshome+Tilahun&rft.au=Labun%2C+Kornel&rft.au=Jakubec%2C+Martin&rft.au=Jefimov%2C+Kirill&rft.au=Niazi%2C+Adnan&rft.au=Valen%2C+Eivind">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2705725/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2705725/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2705725/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2705725/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2705725/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2705725/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2705725/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2705725/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2705725/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2705725/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2705725/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2705725/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&count=horizontal&via=&text=Single%20molecule%20structure%20sequencing%20reveals%20RNA%20structural%20dependencies%2C%20breathing%20and%20ensembles&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.18.101402v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.18.101402v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/molecular-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Molecular Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
