<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.29.123497v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.29.123497v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.29.123497v3.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/07/25/2020.05.29.123497.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.29.123497v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Integrating multimodal data through interpretable heterogeneous ensembles"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.29.123497"/>
  <meta name="DC.Date" content="2022-07-25"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Motivation Integrating multimodal data represents an effective approach to predicting biomedical characteristics, such as protein functions and disease outcomes. However, existing data integration approaches do not sufficiently address the heterogeneous semantics of multimodal data. In particular, early and intermediate approaches that rely on a uniform integrated representation reinforce the consensus among the modalities, but may lose exclusive local information. The alternative late integration approach that can address this challenge has not been systematically studied for biomedical problems.

Results We propose Ensemble Integration (EI) as a novel systematic implementation of the late integration approach. EI infers local predictive models from the individual data modalities using appropriate algorithms, and uses effective heterogeneous ensemble algorithms to integrate these local models into a global predictive model. We also propose a novel interpretation method for EI models. We tested EI on the problems of predicting protein function from multimodal STRING data, and mortality due to COVID-19 from multimodal data in electronic health records. We found that EI accomplished its goal of producing significantly more accurate predictions than each individual modality. It also performed better than several established early integration methods for each of these problems. The interpretation of a representative EI model for COVID-19 mortality prediction identified several disease-relevant features, such as laboratory test (blood urea nitrogen (BUN) and calcium) and vital sign measurements (minimum oxygen saturation) and demographics (age). These results demonstrated the effectiveness of the EI framework for biomedical data integration and predictive modeling.

Availability Code and data are available at <https://github.com/GauravPandeyLab/ensemble_integration>.

Contact gaurav.pandey{at}mssm.edu

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Yan Chak Li"/>
  <meta name="DC.Contributor" content="Linhua Wang"/>
  <meta name="DC.Contributor" content="Jeffrey N. Law"/>
  <meta name="DC.Contributor" content="T. M. Murali"/>
  <meta name="DC.Contributor" content="Gaurav Pandey"/>
  <meta name="article:published_time" content="2022-07-25"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Integrating multimodal data through interpretable heterogeneous ensembles"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<h3>Motivation</h3>
<p>Integrating multimodal data represents an effective approach to predicting biomedical characteristics, such as protein functions and disease outcomes. However, existing data integration approaches do not sufficiently address the heterogeneous semantics of multimodal data. In particular, early and intermediate approaches that rely on a uniform integrated representation reinforce the consensus among the modalities, but may lose exclusive local information. The alternative late integration approach that can address this challenge has not been systematically studied for biomedical problems.</p><h3>Results</h3>
<p>We propose Ensemble Integration (EI) as a novel systematic implementation of the late integration approach. EI infers local predictive models from the individual data modalities using appropriate algorithms, and uses effective heterogeneous ensemble algorithms to integrate these local models into a global predictive model. We also propose a novel interpretation method for EI models. We tested EI on the problems of predicting protein function from multimodal STRING data, and mortality due to COVID-19 from multimodal data in electronic health records. We found that EI accomplished its goal of producing significantly more accurate predictions than each individual modality. It also performed better than several established early integration methods for each of these problems. The interpretation of a representative EI model for COVID-19 mortality prediction identified several disease-relevant features, such as laboratory test (blood urea nitrogen (BUN) and calcium) and vital sign measurements (minimum oxygen saturation) and demographics (age). These results demonstrated the effectiveness of the EI framework for biomedical data integration and predictive modeling.</p><h3>Availability</h3>
<p>Code and data are available at https://github.com/GauravPandeyLab/ensemble_integration.</p><h3>Contact</h3>
<p>gaurav.pandey@mssm.edu</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.29.123497v3"/>
  <meta name="citation_id" content="2020.05.29.123497v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.29.123497"/>
  <meta name="citation_num_pages" content="9"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.29.123497"/>
  <meta name="citation_author" content="Yan Chak Li"/>
  <meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6554-5958"/>
  <meta name="citation_author" content="Linhua Wang"/>
  <meta name="citation_author_institution" content="Graduate Program in Quantitative and Computational Biosciences, Baylor College of Medicine"/>
  <meta name="citation_author" content="Jeffrey N. Law"/>
  <meta name="citation_author_institution" content="National Renewable Energy Laboratory"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2828-1273"/>
  <meta name="citation_author" content="T. M. Murali"/>
  <meta name="citation_author_institution" content="Department of Computer Science, Virginia Tech"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3688-4672"/>
  <meta name="citation_author" content="Gaurav Pandey"/>
  <meta name="citation_author_institution" content="Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai"/>
  <meta name="citation_author_email" content="pandey.gaurav@gmail.com"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1939-679X"/>
  <meta name="citation_reference" content="Ashburner, M. et al. (2000). Gene ontology: tool for the unification of biology. the gene ontology consortium. Nat. Genet., 25(1), 25–29."/>
  <meta name="citation_reference" content="Basheer, M. et al. (2021). Clinical predictors of mortality and critical illness in patients with covid-19 pneumonia. Metabolites, 11(10)."/>
  <meta name="citation_reference" content="Berenguer, J. et al. (2020). Characteristics and predictors of death among 4035 consecutively hospitalized patients with covid-19 in spain. Clin Microbiol Infect, 26(11), 1525–1536."/>
  <meta name="citation_reference" content="Boehm, K. M. et al. (2021). Harnessing multimodal data integration to advance precision oncology. Nature reviews. Cancer."/>
  <meta name="citation_reference" content="Breiman, L. (2001). Random forests. Machine Learning, 45(1), 5–32."/>
  <meta name="citation_reference" content="Caldera, M. et al. (2017). Interactome-based approaches to human disease. Current Opinion in Systems Biology, 3, 88–94."/>
  <meta name="citation_reference" content="Caruana, R. et al. (2004). Ensemble selection from libraries of models. Proceedings of the twenty-first international conference on Machine learning."/>
  <meta name="citation_reference" content="Caruana, R. et al. (2006). Getting the most out of ensemble selection. In Sixth International Conference on Data Mining (ICDM’06), pages 828–833."/>
  <meta name="citation_reference" content="Chen, T. and Guestrin, C. (2016). Xgboost: A scalable tree boosting system. In Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, KDD ‘16, pages 785–794, New York, NY, USA. Association for Computing Machinery."/>
  <meta name="citation_reference" content="Cho, H. et al. (2016). Compact integration of multi-network topology for functional analysis of genes. Cell Syst, 3(6), 540–548."/>
  <meta name="citation_reference" content="Cowen, L. et al. (2017). Network propagation: a universal amplifier of genetic associations. Nat Rev Genet, 18(9), 551–562."/>
  <meta name="citation_reference" content="Demšar, J. (2006). Statistical comparisons of classifiers over multiple data sets. Journal of Machine Learning Research, 7(1), 1–30."/>
  <meta name="citation_reference" content="Dong, E. et al. (2020). An interactive web-based dashboard to track COVID-19 in real time. The Lancet Infectious Diseases, 20(5), 533–534."/>
  <meta name="citation_reference" content="Doshi-Velez, F. and Kim, B. (2017). Towards a rigorous science of interpretable machine learning. arXiv: Machine Learning."/>
  <meta name="citation_reference" content="Frank, E. et al. (2005). Weka: A machine learning workbench for data mining., pages 1305–1314. Springer, Berlin."/>
  <meta name="citation_reference" content="Geraci, J. et al. (2017). Applying deep neural networks to unstructured text notes in electronic medical records for phenotyping youth depression. Evidence-Based Mental Health, 20(3), 83–87."/>
  <meta name="citation_reference" content="Gligorijević, V. and Pržulj, N. (2015). Methods for biological data integration: perspectives and challenges. J R Soc Interface, 12(112)."/>
  <meta name="citation_reference" content="Gligorijević, V. et al. (2018). deepNF: deep network fusion for protein function prediction. Bioinformatics, 34(22), 3873–3881."/>
  <meta name="citation_reference" content="citation_title=Machine Learning and Knowledge Discovery in Databases;citation_pages=423-438;citation_year=2009"/>
  <meta name="citation_reference" content="Hasin, Y. et al. (2017). Multi-omics approaches to disease. Genome Biology, 18(1), 83."/>
  <meta name="citation_reference" content="Jensen, P. B. et al. (2012). Mining electronic health records: towards better research applications and clinical care. Nat. Rev. Genet., 13(6), 395–405."/>
  <meta name="citation_reference" content="Klopfenstein, D. V. et al. (2018). Goatools: A python library for gene ontology analyses. Scientific Reports, 8(1), 10872."/>
  <meta name="citation_reference" content="Krassowski, M. et al. (2020). State of the field in multi-omics research: From computational needs to data mining and sharing. Frontiers in Genetics, 11, 1598."/>
  <meta name="citation_reference" content="Libbrecht, M. W. and Noble, W. S. (2015). Machine learning applications in genetics and genomics. Nat. Rev. Genet., 16(6), 321–332."/>
  <meta name="citation_reference" content="Lippi, G. et al. (2020). Electrolyte imbalances in patients with severe coronavirus disease 2019 (COVID-19). Ann Clin Biochem, 57(3), 262–265."/>
  <meta name="citation_reference" content="Liu, R. et al. (2020). Supervised learning is an accurate method for network-based gene classification. Bioinformatics, 36, 3457 –3465."/>
  <meta name="citation_reference" content="Lundberg, S. M. et al. (2020). From local explanations to global understanding with explainable ai for trees. Nature Machine Intelligence, 2(1), 56–67."/>
  <meta name="citation_reference" content="Mostafavi, S. et al. (2008). GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function. Genome Biology, 9(Suppl 1), S4."/>
  <meta name="citation_reference" content="Pandey, G. et al. (2006). Computational Approaches for Protein Function Prediction: A Survey. Technical Report 06-028, University of Minnesota."/>
  <meta name="citation_reference" content="Pedregosa, F. et al. (2011). Scikit-learn: Machine learning in python. Journal of Machine Learning Research, 12(85), 2825–2830."/>
  <meta name="citation_reference" content="Price-Haywood, E. G. et al. (2020). Hospitalization and mortality among black patients and white patients with covid-19. N Engl J Med, 382(26), 2534–2543."/>
  <meta name="citation_reference" content="Pun, B. T. et al. (2021). Prevalence and risk factors for delirium in critically ill patients with COVID-19 (COVID-D): a multicentre cohort study. Lancet Respir Med, 9(3), 239–250."/>
  <meta name="citation_reference" content="Radivojac, P. et al. (2013). A large-scale evaluation of computational protein function prediction. Nature Methods, 10(3), 221–227."/>
  <meta name="citation_reference" content="Ray, E. L. and Reich, N. G. (2018). Prediction of infectious disease epidemics via weighted density ensembles. PLOS Computational Biology, 14(2), 1–23."/>
  <meta name="citation_reference" content="Saeys, Y. et al. (2007). A review of feature selection techniques in bioinformatics. Bioinformatics, 23(19), 2507–2517."/>
  <meta name="citation_reference" content="Schapire, R. E. and Freund, Y. (2013). Boosting: Foundations and algorithms. Kybernetes."/>
  <meta name="citation_reference" content="Sesmero, M. P. et al. (2015). Generating ensembles of heterogeneous classifiers using stacked generalization. WIREs Data Mining and Knowledge Discovery, 5(1), 21–34."/>
  <meta name="citation_reference" content="Shen, D. et al. (2017). Deep learning in medical image analysis. Annu Rev Biomed Eng, 19, 221–248."/>
  <meta name="citation_reference" content="Shwartz-Ziv, R. and Armon, A. (2021). Tabular data: Deep learning is not all you need."/>
  <meta name="citation_reference" content="Sieberts, S. K. et al. (2016). Crowdsourced assessment of common genetic contribution to predicting anti-tnf treatment response in rheumatoid arthritis. Nature Communications, 7(1), 12460."/>
  <meta name="citation_reference" content="Sieberts, S. K. et al. (2021). Developing better digital health measures of parkinson’s disease using free living data and a crowdsourced data analysis challenge. medRxiv."/>
  <meta name="citation_reference" content="Stanescu, A. and Pandey, G. (2017). Learning parsimonious ensembles for unbalanced computational genomics problems. Pac Symp Biocomput, 22, 288–299."/>
  <meta name="citation_reference" content="Szklarczyk, D. et al. (2021). The STRING database in 2021: customizable protein–protein networks, and functional characterization of user-uploaded gene/measurement sets. Nucleic Acids Research, 49(D1), D605–D612."/>
  <meta name="citation_reference" content="The Gene Ontology Consortium (2017). Expansion of the gene ontology knowledgebase and resources. Nucleic Acids Res., 45(D1), D331–D338."/>
  <meta name="citation_reference" content="Troyanskaya, O. et al. (2001). Missing value estimation methods for DNA microarrays. Bioinformatics, 17(6), 520–525."/>
  <meta name="citation_reference" content="Vaid, A. et al. (2020). Machine learning to predict mortality and critical events in a cohort of patients with covid-19 in new york city: Model development and validation. J Med Internet Res, 22(11), e24018."/>
  <meta name="citation_reference" content="Walsh, I. et al. (2021). DOME: recommendations for supervised machine learning validation in biology. Nature methods, 18(10), 1122–1127."/>
  <meta name="citation_reference" content="Wang, B. et al. (2014). Similarity network fusion for aggregating data types on a genomic scale. Nat Methods, 11(3), 333–337."/>
  <meta name="citation_reference" content="Wang, L. et al. (2018). Large-scale protein function prediction using heterogeneous ensembles. F1000Res, 7."/>
  <meta name="citation_reference" content="Whalen, S. et al. (2016). Predicting protein function and other biomedical characteristics with heterogeneous ensembles. Methods, 93, 92–102."/>
  <meta name="citation_reference" content="Wynants, L. et al. (2020). Prediction models for diagnosis and prognosis of covid-19: systematic review and critical appraisal. BMJ, 369."/>
  <meta name="citation_reference" content="Yadaw, A. S. et al. (2020). Clinical features of COVID-19 mortality: development and validation of a clinical prediction model. Lancet Digit Health, 2(10), e516–e525."/>
  <meta name="citation_reference" content="Zhou, N. et al. (2019). The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens. Genome Biology, 20(1)."/>
  <meta name="citation_reference" content="Zitnik, M. et al. (2019). Machine learning for integrating data in biology and medicine: Principles, practice, and opportunities. Information Fusion, 50, 71–91."/>
  <meta name="twitter:title" content="Integrating multimodal data through interpretable heterogeneous ensembles"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Motivation Integrating multimodal data represents an effective approach to predicting biomedical characteristics, such as protein functions and disease outcomes. However, existing data integration approaches do not sufficiently address the heterogeneous semantics of multimodal data. In particular, early and intermediate approaches that rely on a uniform integrated representation reinforce the consensus among the modalities, but may lose exclusive local information. The alternative late integration approach that can address this challenge has not been systematically studied for biomedical problems.

Results We propose Ensemble Integration (EI) as a novel systematic implementation of the late integration approach. EI infers local predictive models from the individual data modalities using appropriate algorithms, and uses effective heterogeneous ensemble algorithms to integrate these local models into a global predictive model. We also propose a novel interpretation method for EI models. We tested EI on the problems of predicting protein function from multimodal STRING data, and mortality due to COVID-19 from multimodal data in electronic health records. We found that EI accomplished its goal of producing significantly more accurate predictions than each individual modality. It also performed better than several established early integration methods for each of these problems. The interpretation of a representative EI model for COVID-19 mortality prediction identified several disease-relevant features, such as laboratory test (blood urea nitrogen (BUN) and calcium) and vital sign measurements (minimum oxygen saturation) and demographics (age). These results demonstrated the effectiveness of the EI framework for biomedical data integration and predictive modeling.

Availability Code and data are available at <https://github.com/GauravPandeyLab/ensemble_integration>.

Contact gaurav.pandey{at}mssm.edu

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Integrating multimodal data through interpretable heterogeneous ensembles"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Motivation Integrating multimodal data represents an effective approach to predicting biomedical characteristics, such as protein functions and disease outcomes. However, existing data integration approaches do not sufficiently address the heterogeneous semantics of multimodal data. In particular, early and intermediate approaches that rely on a uniform integrated representation reinforce the consensus among the modalities, but may lose exclusive local information. The alternative late integration approach that can address this challenge has not been systematically studied for biomedical problems.

Results We propose Ensemble Integration (EI) as a novel systematic implementation of the late integration approach. EI infers local predictive models from the individual data modalities using appropriate algorithms, and uses effective heterogeneous ensemble algorithms to integrate these local models into a global predictive model. We also propose a novel interpretation method for EI models. We tested EI on the problems of predicting protein function from multimodal STRING data, and mortality due to COVID-19 from multimodal data in electronic health records. We found that EI accomplished its goal of producing significantly more accurate predictions than each individual modality. It also performed better than several established early integration methods for each of these problems. The interpretation of a representative EI model for COVID-19 mortality prediction identified several disease-relevant features, such as laboratory test (blood urea nitrogen (BUN) and calcium) and vital sign measurements (minimum oxygen saturation) and demographics (age). These results demonstrated the effectiveness of the EI framework for biomedical data integration and predictive modeling.

Availability Code and data are available at <https://github.com/GauravPandeyLab/ensemble_integration>.

Contact gaurav.pandey{at}mssm.edu

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-07-25"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2662353"/>
  <title>
   Integrating multimodal data through interpretable heterogeneous ensembles | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"fewfzHLf5sjCwtcqdjMQCL33iVAEyd_4fYUp_KqjhJE"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2662353","apath":"\/biorxiv\/early\/2022\/07\/25\/2020.05.29.123497.atom","pisa":"biorxiv;2020.05.29.123497v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.29.123497v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.29.123497v3","title":"Integrating multimodal data through interpretable heterogeneous ensembles","identifier":"node\/2662353"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kZPfMZEdVLLDh4zSvINrGKRFpN\\\\\\\/ZZeAVgHRwzBpcx0GIcSPkP\\\\\\\/a2EVnKan8ZvIe7AHhTOMtteyaJe+VWtD\\\\\\\/PGIhEThijBEkc6yNhQyv4ng8w0myrlpmtnrW1tzwDZvbTw5Oi77OYp22Gxa+PTz6p3xq4j028t3wzl0ykq2o\\\\\\\/YRKPvZooYEeUw22opyAolW4ww06a4gRUjfYDBF\\\\\\\/UOnWQHzNreuhbQ5Qj1Vay6so5dU8PPpo7KpDKAo1bIiVrJDaRpyzubfLLRVukieXFXv5DJbS1Ry4prc6UhDPBc7yVeZh7sZPrzMGyxvCqglWXB32341tVpS8B1bWEFUcAiSch\\\\\\\/sXPkcS\\\\\\\/XpME2jT2HFb8kxNrlUGljDFVJHlirYT4GcenmZBDkzoenPmKy8dXzsbsqyTdMC+p2zh0BBQ1ppBjVAcsSjZH+UB\\\\\\\/uh4kufx75i4GYTr0\\\\\\\/vzPfJSe0Frj6wXtLKpuoitex071PE43z9aVQfD+jmjwjT1WMmSxSN\\\\\\\/kCNs3Y7H58GGb+jukq2\\\\\\\/hzD+5NSZy99E1D5tDLE8ryolrePmlFLseXP9gDKWvOEWomyovRuybAE5C4aXRxJQpy1PEUeLX2sArssGpaOvfip0bzbrCZl+7rxVDF3IsuPRktVgPmnYdZlbwXRtoS5\\\\\\\/8gdQMqhe1SpL2RL3niWNNBP1jSgjZFxOsukhQ0nFnTjL2cijhGEYOJnThJNvzvYVBelUaFlqhBxjO6dWeaT5dWg4Uj+IL\\\\\\\/LHGOWvRSOLLn7uYKuoVE5ZcaOYO1lkYn2zBxkyw4VxCVepcIvOe7Cr+\\\\\\\/334hCXLzyU8n5ARAllkVnnpMok0EFSgrNvFHkfSQ5xlGsFFXQpn91w1Q5nzr4OAwTpywD2dOCnKk3hbp5\\\\\\\/QLHIzuEMrACZ6GoO+qE5DvheztQglsE+PhvDLiralt72T\\\\\\\/LCE4iFIsE4IaZlW3Jf2qOJ+LC7rByv3ijRdyVrlO4+nHtpeohx\\\\\\\/+m8vMMBbePSj8b9iqQQhnJIrx3sS\\\\\\\/rq8I3DS4vmwmbmtzRdwyTrz+uiSueZB6s4CDjEdtH8wJW3K8VreSmzXWCma8\\\\\\\/0cFpYzueo2JFVVCBx5hd+I0tGufiZ5UypJmiB3obzqmwiWKLrJwf\\\\\\\/6QLq0i0YEKjXmYBf3t82Aij7RdStE7VQx+EUsZYMZ8JLeKsLMRQPfq4v4MpULxctjuFZeRgpkqcOp5Fo9F4GUOuofPS\\\\\\\/n4cM72LHvKSlbKpf0vME+r8vTmpM3jiGkIZyj\\\\\\\/Nyid3zMjgg6YXUdLlsYD1kzHWF31Agr\\\\\\\/Lu\\\\\\\/3i9a7tRk356xGev8RRNQ5oPaz5ByjxDhmNw39bUrpjmiPFF5atRsB7uJjyStOJD4U9rXmxXfNj1f1XPQaZQpxrPShuqKynA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022DZlc0JobSNiMNyKDnwng+g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00222c30fb3c236662a6132b708384ea1896d29b382b89b899d8aa5b0fbf1bfcd7d3\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.29.123497v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2662353 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.29.123497v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_2068074259">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_2068074259" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1188073089" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-pWfjOwETPxlx1z_a7g-2POGiqT5ZL_BKeWTYzdo2SR0"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2662353" data-node-nid="2662353" id="node-2662353--2617657043" data-pisa="biorxiv;2020.05.29.123497v3" data-pisa-master="biorxiv;2020.05.29.123497" data-apath="/biorxiv/early/2022/07/25/2020.05.29.123497.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Integrating multimodal data through interpretable heterogeneous ensembles
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-6554-5958" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Yan Chak
                      </span>
                      <span class="nlm-surname">
                       Li
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Linhua
                      </span>
                      <span class="nlm-surname">
                       Wang
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0003-2828-1273" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jeffrey N.
                      </span>
                      <span class="nlm-surname">
                       Law
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0003-3688-4672" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       T. M.
                      </span>
                      <span class="nlm-surname">
                       Murali
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-1939-679X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Gaurav
                      </span>
                      <span class="nlm-surname">
                       Pandey
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.29.123497
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2662353--2617657043" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Yan Chak Li
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai
                       </span>
                       , New York, New York,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yan%2BChak%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Li%20YC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AYan%2BChak%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6554-5958" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Yan Chak Li
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Linhua Wang
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Graduate Program in Quantitative and Computational Biosciences, Baylor College of Medicine
                       </span>
                       , Houston, Texas,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Linhua%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wang%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ALinhua%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Jeffrey N. Law
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        National Renewable Energy Laboratory
                       </span>
                       , Golden, Colorado,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jeffrey%2BN.%2BLaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Law%20JN&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJeffrey%2BN.%2BLaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2828-1273" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jeffrey N. Law
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     T. M. Murali
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, Virginia Tech
                       </span>
                       , Blacksburg, Virginia,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=T.%2BM.%2BMurali%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Murali%20TM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AT.%2BM.%2BMurali%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3688-4672" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for T. M. Murali
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Gaurav Pandey
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai
                       </span>
                       , New York, New York,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gaurav%2BPandey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Pandey%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGaurav%2BPandey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-1939-679X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Gaurav Pandey
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:pandey.gaurav@gmail.com" class="" data-icon-position="" data-hide-link-title="0">
                        pandey.gaurav@gmail.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.29.123497v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.29.123497v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.29.123497v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.29.123497v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.29.123497v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.29.123497v3.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.29.123497v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2662353" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2662353/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <div id="sec-1" class="subsection">
                             <p id="p-2">
                              <strong>
                               Motivation
                              </strong>
                              Integrating multimodal data represents an effective approach to predicting biomedical characteristics, such as protein functions and disease outcomes. However, existing data integration approaches do not sufficiently address the heterogeneous semantics of multimodal data. In particular, early and intermediate approaches that rely on a uniform integrated representation reinforce the consensus among the modalities, but may lose exclusive local information. The alternative late integration approach that can address this challenge has not been systematically studied for biomedical problems.
                             </p>
                            </div>
                            <div id="sec-2" class="subsection">
                             <p id="p-3">
                              <strong>
                               Results
                              </strong>
                              We propose Ensemble Integration (EI) as a novel systematic implementation of the late integration approach. EI infers local predictive models from the individual data modalities using appropriate algorithms, and uses effective heterogeneous ensemble algorithms to integrate these local models into a global predictive model. We also propose a novel interpretation method for EI models. We tested EI on the problems of predicting protein function from multimodal STRING data, and mortality due to COVID-19 from multimodal data in electronic health records. We found that EI accomplished its goal of producing significantly more accurate predictions than each individual modality. It also performed better than several established early integration methods for each of these problems. The interpretation of a representative EI model for COVID-19 mortality prediction identified several disease-relevant features, such as laboratory test (blood urea nitrogen (BUN) and calcium) and vital sign measurements (minimum oxygen saturation) and demographics (age). These results demonstrated the effectiveness of the EI framework for biomedical data integration and predictive modeling.
                             </p>
                            </div>
                            <div id="sec-3" class="subsection">
                             <p id="p-4">
                              <strong>
                               Availability
                              </strong>
                              Code and data are available at
                              <a href="https://github.com/GauravPandeyLab/ensemble_integration">
                               https://github.com/GauravPandeyLab/ensemble_integration
                              </a>
                              .
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <p id="p-5">
                              <strong>
                               Contact
                              </strong>
                              <span class="em-link">
                               <span class="em-addr">
                                gaurav.pandey{at}mssm.edu
                               </span>
                              </span>
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-5">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-9">
                             Multimodal data are a collection of diverse types of data that capture complementary aspects of a biomedical entity and its characteristics of interest (
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Boehm et al. (2021)
                             </a>
                             ). For instance, a protein may be characterized by its amino acid sequence, three-dimensional structure, evolutionary history and interactions with other proteins. These different types of information may be used to infer the function(s) of the protein (
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              Pandey et al. (2006)
                             </a>
                             ). Similarly, Electronic Health Record (EHR) systems contain diverse types of clinical data, such as from questionnaires, imaging and laboratory tests, that collectively provide a comprehensive view of a patient’s health (
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              Jensen et al. (2012)
                             </a>
                             ). These multimodal data are expected to be complementary. For instance, data from questionnaires provides baseline information of a patient’s health, while laboratory test measurements indicate their current health state. Due to this complementarity, integrating these multimodal data can result in a more comprehensive understanding and accurate predictions of biomedical characteristics or outcomes, such as the functions of proteins and disease phenotypes of patients (
                             <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                              Boehm et al. (2021)
                             </a>
                             ;
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              Krassowski et al. (2020)
                             </a>
                             ;
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              Hasin et al. (2017)
                             </a>
                             ).
                            </p>
                            <p id="p-10">
                             However, the integration and predictive modeling of multimodal data are still challenging because of the heterogeneity of the individual data modalities, which are usually structured differently and have different semantics (
                             <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                              Zitnik et al. (2019)
                             </a>
                             ;
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Gligorijević and Pržulj (2015)
                             </a>
                             ). For instance, gene expression data can be structured as matrices, each entry of which individually denotes the expression level of a gene in a sample. In contrast, string-formatted amino acid sequences are unstructured, since each position in these sequences needs to be studied in the context of its surrounding positions to infer its biological role. Several data types also present their own challenges, such as the high dimensionality of gene expression profiles, which can make their analysis difficult (
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Saeys et al. (2007)
                             </a>
                             ). Similar issues exist with the multimodal data in EHR systems as well. Due to the varying semantics and challenges of the individual modalities, it has been challenging to identify a uniformly effective prediction method for diverse multimodal data (
                             <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                              Zitnik et al. (2019)
                             </a>
                             ).
                            </p>
                            <p id="p-11">
                             Several data integration methods have been proposed to address the challenge of heterogeneity among multimodal data (
                             <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">
                              Zitnik et al. (2019)
                             </a>
                             ;
                             <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                              Gligorijević and Pržulj (2015)
                             </a>
                             ). These methods can be generally categorized into three groups, namely early, intermediate and late integration (
                             <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                              Fig. 1
                             </a>
                             ). Early integration strategies (
                             <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                              Fig. 1a
                             </a>
                             ) first combine the multimodal data into a uniform intermediate representation, such as a network (
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              Caldera et al. (2017)
                             </a>
                             ;
                             <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                              Wang et al. (2014)
                             </a>
                             ), which is then used for prediction and other analysis purposes. Intermediate strategies (
                             <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                              Fig. 1b
                             </a>
                             ) jointly model the multiple datasets and their elements, such as genes, proteins, etc., also through a uniform intermediate representation. These uniform representations reinforce the consensus among modalities, but may obfuscate the local signal exclusive to each individual modality, and thus adversely affect prediction performance (
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              Greene and Cunningham (2009)
                             </a>
                             ;
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Libbrecht and Noble (2015)
                             </a>
                             ).
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F1.large.jpg?width=800&height=600&carousel=1" title="Overview of early, intermediate and late approaches for integrating multimodal data. (a) The early approach combines multiple datasets into an intermediate representation, from which predictive models can be inferred. (b) The intermediate approach jointly models the multiple datasets and their elements through an intermediate representation. (c) The late approach first builds local model(s) for each individual modality, which are then integrated into the final predictive model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Overview of early, intermediate and late approaches for integrating multimodal data.</span> (a) The early approach combines multiple datasets into an intermediate representation, from which predictive models can be inferred. (b) The intermediate approach jointly models the multiple datasets and their elements through an intermediate representation. (c) The late approach first builds local model(s) for each individual modality, which are then integrated into the final predictive model.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F1.medium.gif" width="440" height="89"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F1.medium.gif" width="440" height="89"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Fig. 1:
                              </span>
                              <span class="caption-title">
                               Overview of early, intermediate and late approaches for integrating multimodal data.
                              </span>
                              <p id="p-12" class="first-child">
                               (a) The early approach combines multiple datasets into an intermediate representation, from which predictive models can be inferred. (b) The intermediate approach jointly models the multiple datasets and their elements through an intermediate representation. (c) The late approach first builds local model(s) for each individual modality, which are then integrated into the final predictive model.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-13">
                             In view of these limitations of the early and intermediate approaches, the late approach (
                             <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                              Fig. 1c
                             </a>
                             ) offers an alternative path to improved prediction performance by first deriving specialized local predictive models from the individual modalities, and aggregating those models (
                             <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">
                              Zitnik et al. (2019)
                             </a>
                             ;
                             <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                              Gligorijević and Pržulj (2015)
                             </a>
                             ). This approach has the potential to capture maximal information in the individual modalities into the local predictors, whose aggregation then builds consensus, thus effectively utilizing both the commonalities and diversity among the modalities. However, this approach has not been systematically implemented and studied for biomedical data integration and predictive modeling problems (
                             <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">
                              Zitnik et al. (2019)
                             </a>
                             ).
                            </p>
                            <p id="p-14">
                             In this work, we propose heterogeneous ensembles as a novel systematic realization of the late integration strategy to build effective integrative predictive models from multimodal biomedical data. These ensembles can aggregate an unrestricted number and types of local models, thus providing an effective framework for aggregating diverse information captured in these models (
                             <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                              Whalen et al. (2016)
                             </a>
                             ). Significantly, this approach differs from homogeneous ensembles, such as random forest (
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              Breiman (2001)
                             </a>
                             ) and boosting (
                             <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                              Schapire and Freund (2013)
                             </a>
                             ), which typically aggregate only one type of individual models, e.g., decision trees in random forests. Furthermore, homogeneous methods learn individual models as a part of the ensemble process, and thus cannot integrate models that have been derived independently a priori. The advantages of heterogeneous ensembles have been demonstrated in several biomedical applications, such as protein function prediction (
                             <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">
                              Whalen et al. (2016)
                             </a>
                             ;
                             <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                              Wang et al. (2018)
                             </a>
                             ;
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              Stanescu and Pandey (2017)
                             </a>
                             ), enhancing the predictive power of DREAM Challenges (
                             <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                              Sieberts et al. (2016
                             </a>
                             ,
                             <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                              2021
                             </a>
                             )) and modeling infectious disease epidemics (
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Ray and Reich (2018)
                             </a>
                             ). However, these applications were limited to individual (unimodal) datasets.
                            </p>
                            <p id="p-15">
                             In the current work, we leverage the flexibility of the heterogeneous ensemble framework to advance predictive modeling from multimodal data in an approach that we name Ensemble Integration (EI;
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Fig. 2
                             </a>
                             ). Specifically, EI uses the same heterogeneous ensemble methods as used for unimodal data to integrate local models derived from individual data types. Through this process, EI is capable of incorporating both the consensus and diversity among the individual modalities.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F2.large.jpg?width=800&height=600&carousel=1" title="Overview of the Ensemble Integration (EI) framework for multimodal data. In the implementation of EI tested in this work, we used ten standard binary classification algorithms, such as support vector machine (SVM), random forest (RF) and logistic regression (LR), as implemented in Weka (Frank et al. (2005)), to derive sets of local predictive models 1, 2,…,N from the data modalities 1, 2,…,N. We then applied the stacking and ensemble selection methods to these local models to generate the EI models. These models generated prediction scores for the entities and multimodal data of interest that were evaluated to assess their performance. Finally, we used our novel interpretation method to identify the features that contributed the most substantially to the best-performing EI model’s predictions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Overview of the Ensemble Integration (EI) framework for multimodal data.</span> In the implementation of EI tested in this work, we used ten standard binary classification algorithms, such as support vector machine (SVM), random forest (RF) and logistic regression (LR), as implemented in Weka (Frank <em>et al</em>. (2005)), to derive sets of local predictive models 1, 2,…,N from the data modalities 1, 2,…,N. We then applied the stacking and ensemble selection methods to these local models to generate the EI models. These models generated prediction scores for the entities and multimodal data of interest that were evaluated to assess their performance. Finally, we used our novel interpretation method to identify the features that contributed the most substantially to the best-performing EI model’s predictions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Fig. 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F2.medium.gif" width="440" height="233"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Fig. 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F2.medium.gif" width="440" height="233"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Fig. 2:
                              </span>
                              <span class="caption-title">
                               Overview of the Ensemble Integration (EI) framework for multimodal data.
                              </span>
                              <p id="p-16" class="first-child">
                               In the implementation of EI tested in this work, we used ten standard binary classification algorithms, such as support vector machine (SVM), random forest (RF) and logistic regression (LR), as implemented in Weka (
                               <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                                Frank et al. (2005)
                               </a>
                               ), to derive sets of local predictive models 1, 2,…,N from the data modalities 1, 2,…,N. We then applied the stacking and ensemble selection methods to these local models to generate the EI models. These models generated prediction scores for the entities and multimodal data of interest that were evaluated to assess their performance. Finally, we used our novel interpretation method to identify the features that contributed the most substantially to the best-performing EI model’s predictions.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-17">
                             A challenge associated with multimodal data integration using EI is that the ensembles generated may be complex and difficult to interpret, not unlike other sophisticated machine learning-based models (
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              Doshi-Velez and Kim (2017)
                             </a>
                             ). The interpretation of these ensembles is important for understanding the rationale behind their predictions and generating trust for the user. To address this challenge, we also propose a novel interpretation framework for EI-based models built from multimodal data.
                            </p>
                            <p id="p-18">
                             We tested the prediction and interpretation capabilities of EI on two diverse and challenging problems: protein function prediction (PFP) and disease outcome (COVID-19 mortality) prediction. Specifically, we evaluated EI’s performance at predicting the functions (GO terms) of human proteins from multimodal datasets in the STRING database (
                             <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                              Szklarczyk et al. (2021)
                             </a>
                             ). We also tested the effectiveness of EI for predicting the likelihood of a COVID-19 patient’s mortality (death) from the disease using the multimodal data collected in a treating hospital’s EHRs (
                             <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                              Wynants et al. (2020)
                             </a>
                             ). We compared EI’s performance with those of established data integration methods that have been used for these problems. Finally, in addition to evaluating EI’s prediction abilities, we also used our ensemble interpretation method to reveal the key features that contributed to the EI-based predictive model of COVID-19 mortality, and verified the relevance of these features from the literature.
                            </p>
                           </div>
                           <div class="section" id="sec-6">
                            <h2 class="">
                             2 Methods
                            </h2>
                            <p id="p-19">
                             Here, we describe the methodologies and datasets used in our development and evaluation of the EI framework and other approaches. We followed the Data, Optimization, Model and Evaluation (DOME) recommendations (
                             <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                              Walsh et al. (2021)
                             </a>
                             ), as detailed in the opening subsection of Supplementary Materials.
                            </p>
                            <div id="sec-7" class="subsection">
                             <h3>
                              2.1 Ensemble Integration (EI)
                             </h3>
                             <p id="p-20">
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              shows the general EI framework for multimodal data, as well as the implementation of EI built and tested in this work. First, predictive models were trained on each data modality; we refer to these as local models below. We used ten established binary classification algorithms implemented in Weka (
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               Frank et al. (2005)
                              </a>
                              ) for training local models. These algorithms were AdaBoost, Decision Tree (J48), Gradient Boosting, K-nearest Neighbors, Support Vector Machine (SVM), Random Forest, Logistic Regression (LR), Rule-based Classification (PART), Naive Bayes and Voted Perceptron. All the algorithms were used with their default parameters in Weka, with the exception of specifying C=0.001 for SVM and M=100 for LR to control time to convergence. To handle potential imbalance between the classes being predicted, the entities belonging to the majority negative class in each modality were randomly under-sampled to balance the number of positive and negative class entities before training the local models. The corresponding test sets retained the same class ratio.
                             </p>
                             <p id="p-21">
                              Next, using the base predictions generated by the local models, EI used the following heterogeneous ensemble methods to build the integrative predictive model.
                             </p>
                             <ul class="list-unord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-22">
                                Mean aggregation, which calculates the ensemble output as the mean of the base prediction scores.
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-23">
                                The iterative ensemble selection method of
                                <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                 Caruana et al. (2004
                                </a>
                                ,
                                <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                                 2006
                                </a>
                                ) (CES), which starts with an empty set as the current ensemble. In each iteration, CES adds the local model that improves the current ensemble’s prediction performance by the largest amount. The iterative process, which samples the local models with replacement, continues until there is no further gain in performance.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-24">
                                Stacking (
                                <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                 Sesmero et al. (2015)
                                </a>
                                ), which uses the base predictions as features to train a second-level predictive model (meta-predictor) as the final ensemble. We used eight established binary classification algorithms available in Python’s sklearn library (
                                <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                 Pedregosa et al. (2011)
                                </a>
                                ) to train meta-predictors. These algorithms were AdaBoost, Decision Tree, Gradient Boosting, K-nearest Neighbors, SVM, Random Forest, LR and Naive Bayes. We also included the XGBoost classifier (
                                <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                                 Chen and Guestrin (2016)
                                </a>
                                ) as a meta-prediction algorithm. All the algorithms were executed using their default parameter settings, with the exception of the linear kernel being used with SVM to control the runtime.
                               </p>
                              </li>
                             </ul>
                             <p id="p-25">
                              Thus, each execution of EI yielded eleven models, i.e., one each based on mean aggregation and CES, and nine based on stacking. These EI models then generated the final prediction scores for the entities of interest. These scores were evaluated to assess the models’ performance.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              2.2 Interpretation of EI models
                             </h3>
                             <p id="p-26">
                              To aid the interpretation of EI models and build trust in them, we propose a novel method to identify the key features in the various data modalities that contribute the most to the model’s predictions (Supplementary Fig. 1). This method first quantifies the contributions the features in the individual modalities (local features) make to the corresponding local models. It then quantifies the contribution of each local model to the EI ensemble. Finally, it combines these contributions to determine the most important features for the EI model.
                             </p>
                             <p id="p-27">
                              Algorithmically, the method calculates the local feature ranks (LFRs) from the local models and the local model ranks (LMRs) from the EI model (
                              <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">
                               Algorithm 1
                              </a>
                              ) as follows. The LFRs for each local model are calculated based on the difference of the performance of the model when each feature is held out. This calculation is carried out using the ClassifierAttributeEval function in Weka. The calculation of LMRs depends on the type of ensemble algorithm used to build the EI model under consideration. If the model under consideration is based on mean aggregation, all the local models are assigned the same LMRs. If it is based on CES, the local models are assigned LMRs based on how many times they are included in the final ensemble. Finally, if the EI ensemble is based on stacking, LMRs are determined using the permutation importance of each constituent local model (
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               Breiman (2001)
                              </a>
                              ). This importance measure is calculated as the average change in performance of the ensemble when the local model’s output is randomly permuted a hundred times and as many ensembles are retrained. This calculation is conducted using the permutation_importance function in the sklearn library (Pedregosa et al. (2011)). LMRs are then determined based on the descending order of the permutation importance values. Due to the varying number of local models and features these ranks are calculated for, all the ranks are normalized into percentile ranks that range from 1/n to 1, where n is the total number of local models in the ensemble or the total number of features in the modality being considered.
                             </p>
                             <div id="statement-1" class="statement">
                              <span class="statement-label">
                               Algorithm 1:
                              </span>
                              <h3>
                               Calculate percentile ranks of local models in an ensemble
                              </h3>
                              <div id="F3" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F3.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F3.medium.gif" width="416" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F3.medium.gif" width="416" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure3" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-28">
                              </p>
                             </div>
                             <div id="statement-2" class="statement">
                              <span class="statement-label">
                               Algorithm 2:
                              </span>
                              <h3>
                               Calculate final ranking of features for an ensemble
                              </h3>
                              <div id="F4" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F4.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F4.medium.gif" width="440" height="390"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F4.medium.gif" width="440" height="390"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure4" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-29">
                              </p>
                             </div>
                             <p id="p-30">
                              Next, we combined these two lists of ranks to compute the feature ranking for the EI model as shown in
                              <a id="xref-statement-2-1" class="xref-statement" href="#statement-2">
                               Algorithm 2
                              </a>
                              . We first calculated the product of the LMR and LFR for each valid pair of local model and feature. We then averaged these products for each local feature across all the local models to generate its rank product scores (RPSs), which quantified the feature’s overall contribution to the EI model. All the features in all the modalities were then sorted in ascending order of RPSs to determine the final ranking of the features.
                             </p>
                             <p id="p-31">
                              Note that this interpretation method is applicable to any EI model. However, in order to focus its use in our experiments, we applied it only to the best-performing ensemble algorithm within EI for the label of interest (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ). This ensemble algorithm was used to train an EI model on the whole dataset, which was then interpreted.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              2.3 Biomedical problems used for evaluating EI
                             </h3>
                             <p id="p-32">
                              To evaluate the effectiveness of EI for biomedical problems, we tested the framework on two representative problems, one each from bioinformatics and clinical informatics. These problems were protein function prediction from STRING data (
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               Szklarczyk et al. (2021)
                              </a>
                              ), and COVID-19 mortality prediction from EHR data, respectively. Below, we provide details of these problems, as well as the associated multimodal datasets used.
                             </p>
                             <div id="sec-10" class="subsection">
                              <h4>
                               2.3.1 Protein function prediction from STRING data
                              </h4>
                              <p id="p-33">
                               As discussed in the Introduction, a variety of complementary data modalities can be used to predict the functions of proteins (
                               <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                                Pandey et al. (2006)
                               </a>
                               ), and the most accurate predictions are often obtained by integrating these modalities (
                               <a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">
                                Zitnik et al. (2019)
                               </a>
                               ). Thus, protein function prediction (PFP) is an ideal problem to test EI’s effectiveness for integrating multimodal data and producing accurate predictions. We followed the most commonly used definition of protein functions as Gene Ontology (GO) terms (
                               <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                Radivojac et al. (2013)
                               </a>
                               ). We predicted annotations of human proteins to these terms using diverse multimodal datasets from version 11.5 of the STRING database (
                               <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">
                                Szklarczyk et al. (2021)
                               </a>
                               ) released in August 2021. These datasets consist of pairwise functional associations between thousands of proteins from several species, including human. These associations are derived from the multi-omic data sources listed in
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Table 1
                               </a>
                               . Since these data sources provide complementary information about protein function (
                               <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">
                                Pandey et al. (2006)
                               </a>
                               ;
                               <a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">
                                Zitnik et al. (2019)
                               </a>
                               ;
                               <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                                Radivojac et al. (2013)
                               </a>
                               ), the associations derived from them in STRING are also multi-modal.
                              </p>
                              <div id="T1" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2664749/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2664749/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2664749" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 1:
                                </span>
                                <span class="caption-title">
                                 Modalities of STRING data used for protein function prediction.
                                </span>
                                <p id="p-34" class="first-child">
                                 For each modality of STRING, the description and number of features are included in the table.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-35">
                               We tested the effectiveness of EI for predicting protein function (GO term annotations) from these multimodal STRING datasets. For all the individual datasets, the adjacency vector of each protein was used as its feature vector for PFP model training and evaluation, since this representation has been shown to be effective for automated network-based gene/protein classification (
                               <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                                Liu et al. (2020)
                               </a>
                               ). If a certain protein was not included in any of the STRING datasets, its corresponding feature vector was assigned to be all zeros before integration and prediction. EI was executed for each GO term individually on the multimodal datasets.
                              </p>
                              <p id="p-36">
                               We compared EI’s performance with those of established network-based early integration algorithms, namely Mashup (
                               <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                                Cho et al. (2016)
                               </a>
                               ) and deepNF (
                               <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                                Gligorijević et al. (2018)
                               </a>
                               ) following the methodology described in Supplementary Fig. 2. Mashup and deepNF use graph propagation- and deep learning-based algorithms, respectively, to derive an integrated network corresponding to the input networks. In our study, these algorithms were applied to the STRING data described above to generated corresponding integrated networks. The same adjacency vector as feature vector representation was used for the integrated networks generated using these methods. This representation yielded feature vectors of lengths 800 and 1200 from the Mashup and deepNF integrated networks. Note that these networks already represent a layer of integration of the STRING data. Thus, to generate predictive models of GO term annotations from these networks that were consistent with the single layer of integration in EI (Supplementary Fig. 2), we used the local modeling algorithms listed in
                               <a id="xref-sec-7-1" class="xref-sec" href="#sec-7">
                                Section 2.1
                               </a>
                               . Finally, to assess the value of integrating multimodal data, we also predicted annotations from each of the STRING datasets individually using the same heterogeneous ensemble algorithms as included in EI (
                               <a id="xref-sec-7-2" class="xref-sec" href="#sec-7">
                                Section 2.1
                               </a>
                               ). Again, these heterogeneous ensembles represented a single layer of integration of the base models derived from the individual modalities (Supplementary Fig. 2).
                              </p>
                              <p id="p-37">
                               EI and the baseline methods were used to predict the annotations of 18,866 human proteins to 2,139 GO Molecular Function and Biological Process terms that were used to annotate at least fifty human proteins in the data released on January 13th, 2022. This lower limit was chosen to reduce the variability and enhance the reliability of the results obtained. For each GO term, the proteins annotated to it with non-IEA evidence codes were defined as positive examples. We defined negative examples as follows: a protein p was labeled as a negative example for a term t in an ontology o (e.g., Molecular Function) if p had at least one annotation to a term in o, but was not manually annotated (i.e., with a non-IEA evidence code) to t nor to its ancestors or descendants (
                               <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                                Mostafavi et al. (2008)
                               </a>
                               ). For each term, only the positively or negatively annotated proteins were used for the evaluation of EI and the baseline methods.
                              </p>
                              <p id="p-38">
                               It is noteworthy that GO terms are organized within hierarchical ontologies (
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                The Gene Ontology Consortium (2017)
                               </a>
                               ;
                               <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                                Ashburner et al. (2000)
                               </a>
                               ), with deeper terms representing more specific functions and annotating fewer proteins. These properties of a GO term, namely its depth and the number of proteins annotated with it, have been shown to substantially influence the performance of PFP methods (
                               <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                                Zhou et al. (2019)
                               </a>
                               ). Thus, to compare the performance of EI and the baseline methods more comprehensively, we also assessed how their performance varied with these properties. For this, we first compared the performances of the prediction methods across GO terms grouped by the number of human genes annotated to them. In our consideration set, there were 166 terms with more than 1000 annotations, 162 with 500 to 1000 annotations, 388 with 200 to 500 annotations, 544 with 100 to 200 annotations and 879 with 50 to 100 annotations. Since these numbers of proteins annotated to GO terms varied over several orders of magnitude, we also considered a more normalized measure of these number known as information content of a term (
                               <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">
                                Zhou et al. (2019)
                               </a>
                               ). For a term t, this quantity is defined as - log
                               <sub>
                                10
                               </sub>
                               (p(t)), where p(t) is the probability of a human protein being annotated with t. Finally, the depth of GO term t was defined as the shortest path from the root of the corresponding ontology to t. The depth and information content of GO terms were calculated using the GOATOOLS package (version 1.0.3) (
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                Klopfenstein et al. (2018)
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-11" class="subsection">
                              <h4>
                               2.3.2 COVID-19 mortality prediction from EHR data
                              </h4>
                              <p id="p-39">
                               The COVID-19 pandemic has infected over 570 million individuals and caused over 6 million deaths globally, as of July 22nd, 2022, as per the Johns Hopkins University Coronavirus Resource Center (
                               <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                                Dong et al. (2020)
                               </a>
                               ). It thus represents the most serious public health threat the world has faced in a long time. For a patient being treated for this disease, it is immensely useful for healthcare providers to have an estimate of the patient’s outcomes, since they can adapt the treatment accordingly (
                               <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">
                                Wynants et al. (2020)
                               </a>
                               ). The multimodal data collected from these patients into their EHRs, such as data collected at admission, co-morbidity information, vital signs and laboratory test measurements, represent useful sources of information that can be integrated and used for predicting these outcomes.
                              </p>
                              <p id="p-40">
                               We tested EI’s ability to address this need. Specifically, we used EI to predict the likelihood of an individual dying from COVID-19 (mortality), the most serious outcome, for patients treated at Mount Sinai between March 15 and October 1, 2020 from the data modalities in the EHR system described in
                               <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                                Table 2
                               </a>
                               (details of the features are in Supplementary Table 1). These modalities and features were selected and processed by expert clinicians and informaticians, and provided by the Mount Sinai Data Warehouse. Since vital signs and laboratory tests were measured multiple times for each patient, we used their respective first values recorded during the first 36 hours of hospitalization to enable early outcome prediction, as recommended by another study (
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                Vaid et al. (2020)
                               </a>
                               ). Only features with fewer than 30% missing values were included in the analysis. Any remaining missing values in each modality were imputed using the KNNImpute method (
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                Troyanskaya et al. (2001)
                               </a>
                               ) with K set to 5. The categorical features in all the modalities were transformed to numerical values by one-hot encoding. The continuous features were normalized into z-scores. The resultant number of features included in each modality are specified in the descriptions above. Using the above data, we tested EI for predicting if a patient died from COVID-19 (mortality) during the course of their hospitalization. This outcome was determined using the value of the Boolean “deceased” flag in the patient’s EHR. In our dataset covering 4, 783 patients, 1, 325 (27.7%) passed away from the disease.
                              </p>
                              <div id="T2" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2664750/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2664750/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2664750" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Table 2:
                                </span>
                                <span class="caption-title">
                                 Modalities of electronic health record (EHR) data used for predicting mortality due to COVID-19.
                                </span>
                                <p id="p-41" class="first-child">
                                 For each modality of EHR data, the description and number of features are included in the table.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-42">
                               Specifically, we compared EI’s performance with that of an early integration approach proposed by another study (
                               <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                                Vaid et al. (2020)
                               </a>
                               ). This approach concatenated the feature vectors from the individual modalities for a given patient, and built mortality prediction models using XGBoost (
                               <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                                Chen and Guestrin (2016)
                               </a>
                               ), which is considered the most effective method for tabular data (
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                Shwartz-Ziv and Armon (2021)
                               </a>
                               ). The default parameters specified in the Python xgboost library (
                               <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                                Chen and Guestrin (2016)
                               </a>
                               ) were used for this baseline method. Additionally, similar to the PFP experiments, we also considered heterogeneous ensembles built on the individual data modalities as alternative baselines.
                              </p>
                              <p id="p-43">
                               Finally, we used the novel method described in
                               <a id="xref-sec-8-1" class="xref-sec" href="#sec-8">
                                Section 2.2
                               </a>
                               to identify the features in the above data modalities that contributed the most to the EI predictive models for COVID-19 mortality. We also compared these features with those that contributed the most to the XGBoost model for the same outcome. For this, as in
                               <a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">
                                Vaid et al. (2020)
                               </a>
                               ’s study, we calculated the mean absolute SHAP value (
                               <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                                Lundberg et al. (2020)
                               </a>
                               ) for all the features in the XGBoost model, and ranked them in descending order of this value to identify the most important features. We then conducted the Fisher’s exact test to calculate the statistical significance of the overlap between the two feature tests.
                              </p>
                             </div>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              2.4 Evaluation methodology
                             </h3>
                             <p id="p-44">
                              All the heterogeneous ensembles, both from EI and baseline approaches, were trained and evaluated in a five-fold nested cross-validation (Nested CV) setup (
                              <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">
                               Whalen et al. (2016)
                              </a>
                              ) in the above experiments. In this setup, the whole dataset is split into five outer folds, which are further divided into inner folds. The inner folds are used for training the local models, while the outer folds are used for training and evaluating the ensembles. Nested CV helps reduce overfitting during heterogeneous ensemble learning by separating the set of examples on which the local and ensemble models are trained and evaluated (
                              <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">
                               Whalen et al. (2016)
                              </a>
                              ). The Mashup and deepNF baselines in PFP, and XGBoost baseline in the COVID-19 mortality prediction experiments were executed in the standard 5-fold cross-validation setup.
                             </p>
                             <p id="p-45">
                              All the GO terms and COVID-19 mortality outcomes predicted in our experiments were unbalanced, as is common knowledge for PFP (
                              <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                               Radivojac et al. (2013)
                              </a>
                              ;
                              <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">
                               Zhou et al. (2019)
                              </a>
                              ). To assess the performance of the prediction methods for the more challenging minority positive class in each experiment, we used F
                              <sub>
                               max
                              </sub>
                              in all our evaluations. F
                              <sub>
                               max
                              </sub>
                              is the maximum value of F-measure across all prediction score thresholds among the combinations of precision and recall for the positive class, and has been recommended for PFP assessment by the CAFA initiative (
                              <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">
                               Radivojac et al. (2013)
                              </a>
                              ;
                              <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">
                               Zhou et al. (2019)
                              </a>
                              ). We also we measured the prediction performances for both problems in terms of the precision and recall yielding the F
                              <sub>
                               max
                              </sub>
                              value reported. Furthermore, since our PFP experiments involved evaluations on over 2,000 GO terms, we also statistically compared the performances of EI and the baselines using Friedman and Nemenyi tests (
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               Demšar (2006)
                              </a>
                              ) to assess the overall performance of all the methods tested.
                             </p>
                             <p id="p-46">
                              Finally, as explained in the EI interpretation method (
                              <a id="xref-sec-8-2" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                              ), performance assessment was also needed for calculating LMRs in EI ensembles, as well as for calculating LFRs for features included in the local models. For both these calculations, we used the Area Under the Precision-Recall Curve (AUPRC), since it was the only class imbalance-aware performance measure available as an option in the sklearn and Weka functions used (
                              <a id="xref-sec-8-3" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                              ).
                             </p>
                             <p id="p-47">
                              The code implementing all the above methods and the data used are available at
                              <a href="https://github.com/GauravPandeyLab/ensemble_integration">
                               https://github.com/GauravPandeyLab/ensemble_integration
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             3 Results
                            </h2>
                            <p id="p-48">
                             Below, we describe and analyze the results obtained in our experiments on protein function and COVID-19 mortality prediction.
                            </p>
                            <div id="sec-14" class="subsection">
                             <h3>
                              3.1 Protein function prediction
                             </h3>
                             <p id="p-49">
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 3
                              </a>
                              shows the distributions of the F
                              <sub>
                               max
                              </sub>
                              scores of all the protein function prediction (PFP) approaches tested in
                              <a id="xref-sec-10-1" class="xref-sec" href="#sec-10">
                               Section 2.3.1
                              </a>
                              . For each approach, this figure shows a box plot denoting the distribution of one F
                              <sub>
                               max
                              </sub>
                              score for each GO term. This score is for the algorithm implementing the approach that performed the best for that GO term, e.g., stacking using logistic regression for Ensemble Integration for GO:0000976 (transcription cis-regulatory region binding). As explained in
                              <a id="xref-sec-10-2" class="xref-sec" href="#sec-10">
                               Section 2.3.1
                              </a>
                              , each of the prediction approaches tested here, namely EI, deepNF and Mashup with local modeling algorithms and the individual STRING modalities with heterogeneous ensembles, represented a single layer of data integration (Supplementary Fig. 2). Thus, the selection of the algorithm that yielded the best performance for each approach for each GO term is intended to present the corresponding strongest integrative predictor for each term, and thus facilitate consistency across the diverse approaches evaluated.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F5.large.jpg?width=800&height=600&carousel=1" title="The distributions of the performances of the protein function prediction approaches tested in this work. This performance was measured in terms of the Fmax score for each of the 2,139 GO terms. For EI and the individual modalities, the score for the best performing heterogeneous ensemble method for each GO term is shown here. For the Mashup and deepNF early integration methods, the score of the best local modeling algorithm for each term is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The distributions of the performances of the protein function prediction approaches tested in this work.</span> This performance was measured in terms of the <em>F</em><sub><em>max</em></sub> score for each of the 2,139 GO terms. For EI and the individual modalities, the score for the best performing heterogeneous ensemble method for each GO term is shown here. For the Mashup and deepNF early integration methods, the score of the best local modeling algorithm for each term is shown.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F5.medium.gif" width="440" height="280"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F5.medium.gif" width="440" height="280"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 3:
                               </span>
                               <span class="caption-title">
                                The distributions of the performances of the protein function prediction approaches tested in this work.
                               </span>
                               <p id="p-50" class="first-child">
                                This performance was measured in terms of the F
                                <sub>
                                 max
                                </sub>
                                score for each of the 2,139 GO terms. For EI and the individual modalities, the score for the best performing heterogeneous ensemble method for each GO term is shown here. For the Mashup and deepNF early integration methods, the score of the best local modeling algorithm for each term is shown.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-51">
                              These results show that the performance of Ensemble Integration (EI) was significantly better than those of Mashup and deepNF (Friedman-Nemenyi FDR = 9.34 × 10
                              <sup>
                               -14
                              </sup>
                              and < 2 × 10
                              <sup>
                               -16
                              </sup>
                              , respectively), as well as each of the individual STRING data modalities (Friedman-Nemenyi FDR < 2 × 10
                              <sup>
                               -16
                              </sup>
                              ). These results were generally consistent with those obtained in terms of the precision and recall yielding the above F
                              <sub>
                               max
                              </sub>
                              values (Supplementary Fig. 4).
                             </p>
                             <p id="p-52">
                              We also examined how the performance of EI, deepNF and Mashup varied with the depth (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 4a
                              </a>
                              ) and information content (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 4b
                              </a>
                              ) of the GO terms included in our evaluation. Across all depths and information content levels, EI consistently performed better than both deepNF and Mashup, exemplified by the higher median F
                              <sub>
                               max
                              </sub>
                              values across all the GO term subsets. We also assessed the performance of all the prediction approaches across GO term subsets grouped by the number of human genes annotated to them (Supplementary Fig. 3). Although the performance of all the approaches deteriorated, as expected, for terms with fewer annotations, EI generally performed well, and better than the other approaches. The only exception to this observation was the set of terms with the fewest annotations (50–100), where Mashup produced significantly more accurate predictions than EI.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F6.large.jpg?width=800&height=600&carousel=1" title="Distribution of the performances (Fmax scores) of Ensemble Integration (EI), deepNF and Mashup for GO terms at varying depths and information content levels. The depth and information content values of the 2,139 GO terms included in this experiment were calculated using the GOATOOLS package (version 1.0.3) (Klopfenstein et al. (2018))." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Distribution of the performances (<em>F</em><sub><em>max</em></sub> scores) of Ensemble Integration (EI), deepNF and Mashup for GO terms at varying depths and information content levels.</span> The depth and information content values of the 2,139 GO terms included in this experiment were calculated using the GOATOOLS package (version 1.0.3) (Klopfenstein <em>et al</em>. (2018)).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F6.medium.gif" width="315" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F6.medium.gif" width="315" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 4:
                               </span>
                               <span class="caption-title">
                                Distribution of the performances (F
                                <sub>
                                 max
                                </sub>
                                scores) of Ensemble Integration (EI), deepNF and Mashup for GO terms at varying depths and information content levels.
                               </span>
                               <p id="p-53" class="first-child">
                                The depth and information content values of the 2,139 GO terms included in this experiment were calculated using the GOATOOLS package (version 1.0.3) (
                                <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                                 Klopfenstein et al. (2018)
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-54">
                              Collectively, these results indicated that EI was able to achieve the goals of data integration and predictive modeling for PFP more effectively than other alternate approaches.
                             </p>
                             <p id="p-55">
                              Finally, to gain insight into which ensemble methods performed the best for EI, we analyzed this distribution for all the GO terms tested (Supplementary Fig. 5). Stacking with random forest and logistic regression were generally the best-performing EI methods, consistent with observations in our previous work with single datasets (
                              <a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">
                               Whalen et al. (2016)
                              </a>
                              ;
                              <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">
                               Wang et al. (2018)
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              3.2 COVID-19 mortality prediction
                             </h3>
                             <p id="p-56">
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 5
                              </a>
                              shows the distributions of the performance of the various implementations of EI and heterogeneous ensembles derived from the individual EHR data modalities for predicting the mortality over hospitalization outcome. Note that, unlike
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 3
                              </a>
                              , the performance of all the implementations of each of the approaches are shown in the corresponding box plots, since these results are only for one outcome. Also shown is the performance of the baseline XGBoost method (dotted red line). As in the PFP experiments, EI performed significantly better than the individual modalities (Wilcoxon rank-sum FDR < 0.0082). Two of the eleven EI ensembles tested performed better than XGBoost. The best-performing EI method (stacking using logistic regression) scored a 0.011 (1.66%) higher F
                              <sub>
                               max
                              </sub>
                              than XGBoost.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F7.large.jpg?width=800&height=600&carousel=1" title="The distribution of Fmax values of various ensemble models and XGBoost for predicting mortality from COVID-19 over a patient’s hospitalization. The performance distributions of Ensemble Integration (EI) and the heterogeneous ensembles derived from the individual EHR data modalities are shown as box-and-whisker plots. Each plot includes eleven ensemble models built using mean aggregation, CES, and nine stacking algorithms. The numbers in parentheses next to the names on the x-axis indicate the number of features in the corresponding modality. The dotted red line indicates the performance of the XGBoost model trained after concatenating the feature vectors in each individual modality." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">The distribution of <em>F</em><sub><em>max</em></sub> values of various ensemble models and XGBoost for predicting mortality from COVID-19 over a patient’s hospitalization.</span> The performance distributions of Ensemble Integration (EI) and the heterogeneous ensembles derived from the individual EHR data modalities are shown as box-and-whisker plots. Each plot includes eleven ensemble models built using mean aggregation, CES, and nine stacking algorithms. The numbers in parentheses next to the names on the <em>x</em>-axis indicate the number of features in the corresponding modality. The dotted red line indicates the performance of the XGBoost model trained after concatenating the feature vectors in each individual modality.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F7.medium.gif" width="440" height="334"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F7.medium.gif" width="440" height="334"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 5:
                               </span>
                               <span class="caption-title">
                                The distribution of F
                                <sub>
                                 max
                                </sub>
                                values of various ensemble models and XGBoost for predicting mortality from COVID-19 over a patient’s hospitalization.
                               </span>
                               <p id="p-57" class="first-child">
                                The performance distributions of Ensemble Integration (EI) and the heterogeneous ensembles derived from the individual EHR data modalities are shown as box-and-whisker plots. Each plot includes eleven ensemble models built using mean aggregation, CES, and nine stacking algorithms. The numbers in parentheses next to the names on the x-axis indicate the number of features in the corresponding modality. The dotted red line indicates the performance of the XGBoost model trained after concatenating the feature vectors in each individual modality.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-58">
                              We also examined the variation of precision and recall values for this best-performing EI method, the best-performing heterogeneous ensembles inferred from individual modalities and XGBoost (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. 6
                              </a>
                              ). The comparative results were generally consistent with those shown in
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 5
                              </a>
                              , with EI generally performing well and slightly better than XGBoost. EI also achieved a slightly better balance between precision (0.569) and recall (0.752) at the point where F-measure was maximized than the other approaches, including XGBoost (precision=0.529, recall=0.801).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F8.large.jpg?width=800&height=600&carousel=1" title="Precision-recall curves of representative models from EI, individual EHR modalities and XGBoost for predicting COVID-19 mortality. The values in parentheses show the Fmax value of each of these models, and the cross marker on each curve indicates the precision and recall at which the corresponding Fmax was obtained." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1636597197" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Precision-recall curves of representative models from EI, individual EHR modalities and XGBoost for predicting COVID-19 mortality.</span> The values in parentheses show the <em>F</em><sub><em>max</em></sub> value of each of these models, and the cross marker on each curve indicates the precision and recall at which the corresponding <em>F</em><sub><em>max</em></sub> was obtained.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Fig. 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F8.medium.gif" width="440" height="344"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Fig. 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F8.medium.gif" width="440" height="344"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/07/25/2020.05.29.123497/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Fig. 6:
                               </span>
                               <span class="caption-title">
                                Precision-recall curves of representative models from EI, individual EHR modalities and XGBoost for predicting COVID-19 mortality.
                               </span>
                               <p id="p-59" class="first-child">
                                The values in parentheses show the F
                                <sub>
                                 max
                                </sub>
                                value of each of these models, and the cross marker on each curve indicates the precision and recall at which the corresponding F
                                <sub>
                                 max
                                </sub>
                                was obtained.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-60">
                              Overall, these results showed the slight advantage of EI over early integration for COVID-19 mortality prediction.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              3.3 Interpretation of COVID-19 mortality prediction
                             </h3>
                             <p id="p-61">
                              Using the method described in
                              <a id="xref-sec-8-4" class="xref-sec" href="#sec-8">
                               Section 2.2
                              </a>
                              , we also identified the ten features that contributed the most to the best-performing EI model for the COVID-19 mortality outcome (
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Table 3
                              </a>
                              ). These features included five from laboratory tests, three from admission, one from vital signs and one from co-morbidities. This distribution was consistent with the observation that that laboratory tests produced the most accurate predictions across all the modalities (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 5
                              </a>
                              ).
                             </p>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2664754/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2664754/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2664754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 3:
                               </span>
                               <span class="caption-title">
                                Ten highest contribution features identified from the best-performing EI model for predicting mortality due to COVID-19.
                               </span>
                               <p id="p-62" class="first-child">
                                The table provides detailed information on the features, including their names, the modality they were included in, and their description. The features are sorted in terms of increasing rank product scores (third column), which is the EI interpretation metric our method calculates (
                                <a id="xref-sec-8-5" class="xref-sec" href="#sec-8">
                                 Section 2.2
                                </a>
                                .)
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-63">
                              We also identified the ten most contributing features in the XGBoost model (Supplementary Table 2). Four of these features, namely age at admission, minimum oxygen saturation among vital signs, and blood urea nitrogen and calcium measurements among laboratory tests, overlapped with the ten most predictive features for the EI model. This overlap was statistically significant (Fisher’s exact test p=0.0089, odds ratio=8.74).
                             </p>
                             <p id="p-64">
                              The results in the above subsections illustrate the utility of our EI framework for addressing biomedical prediction problems, as well as interpreting the predictive models generated by the framework.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-17">
                            <h2 class="">
                             4 Discussion
                            </h2>
                            <p id="p-65">
                             We proposed a novel framework named Ensemble Integration (EI) to perform data integration and predictive modeling on heterogeneous multimodal data. In contrast to the more commonly used early and intermediate data integration approaches, EI adopts the late integration approach. In EI, one or more local models are derived from the individual data modalities using appropriate algorithms. These local models are then integrated into a global predictive model using effective heterogeneous ensemble methods, such as stacking and ensemble selection. Thus, EI offers the flexibility of deriving individually effective local models from each of the data modalities, which addresses challenges related to the differing semantics of these modalities. The use of heterogeneous ensemble methods then enables the incorporation of both the consensus and diversity among the local models and individual data modalities.
                            </p>
                            <p id="p-66">
                             We tested EI for the diverse problems of protein function prediction (PFP) from multimodal STRING datasets and predicting mortality due to COVID-19 from multimodal EHR datasets. In both these experiments, we compared EI’s performance with those obtained from the individual data modalities, as well as established early integration methods, specifically deepNF and Mashup for PFP, and XGBoost for COVID-19 mortality prediction. In all these experiments, EI performed significantly better than the individual data modalities, showing that it accomplishes its data integration goals successfully. EI also performed better than the early integration approaches, most likely due to its ability to aggregate the complementary information encapsulated in the local models, which may be lost in the uniform integrated representations used by early and intermediate strategies.
                            </p>
                            <p id="p-67">
                             We also proposed a novel interpretation method for EI models that ranks the features in the individual modalities in terms of their contributions to the EI model under consideration. We tested this method on the representative EI model constructed for predicting the COVID-19 mortality outcome. We found that several of the ten most important features identified for mortality due to COVID-19 were laboratory test measurements (
                             <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">
                              Table 3
                             </a>
                             ), which was consistent with the observation that this modality yielded the most accurate predictions among all the individual modalities (
                             <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                              Fig. 5
                             </a>
                             ). In particular, we found that the measurements of blood urea nitrogen (BUN) and calcium in patients’ blood samples were key features. These findings were consistent with prior research on the relevance of these measurements to mortality due to COVID-19 (
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              Basheer et al. (2021)
                             </a>
                             ;
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              Lippi et al. (2020)
                             </a>
                             ). Patients’ age and minimum oxygen saturation were also found to be important features, again consistent with prior evidence (
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Price-Haywood et al. (2020)
                             </a>
                             ;
                             <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                              Yadaw et al. (2020)
                             </a>
                             ;
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              Berenguer et al. (2020)
                             </a>
                             ;
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              Pun et al. (2021)
                             </a>
                             ). We also analyzed the consistency of our list of most predictive features with the ten features found to be most predictive of the same outcome using
                             <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">
                              Vaid et al. (2020)
                             </a>
                             ’s XGBoost method that was considered the baseline in our prediction experiments (
                             <a id="xref-sec-15-1" class="xref-sec" href="#sec-15">
                              Section 3.2
                             </a>
                             ). We found that four features, namely age at admission, minimum oxygen saturation among vital signs, and calcium and BUN measurements among laboratory tests, were common to both lists. This overlap was statistically significant, indicating that EI can identify important features consistent with other prediction methodologies, as well as reveal novel relevant features. A deeper investigation of the predictive features we identified can shed further light on the pathophysiology of COVID-19 and mortality due to the disease.
                            </p>
                            <p id="p-68">
                             In summary, this paper presented the novel EI framework, and evidence in support of its ability to address challenging biomedical prediction problems. We also discussed challenges with the current work, and avenues for future work. Our efforts represent the first step in the systematizing and expanding the use of the late integration approach for complex, multimodal biomedical data.
                            </p>
                            <p id="p-69">
                             Our work also has some limitations, which offer avenues for future work. First, although EI is capable of integrating both structured and unstructured data modalities, as well as a variety of local models derived from these modalities, we only tested EI with structured datasets and standard classification algorithms used to derive local models from these datasets. In the future, it would be valuable to test EI with unstructured data and specialized local models as well, such as label propagation on network data (
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Cowen et al. (2017)
                             </a>
                             ), convolutional neural networks (CNNs) derived from the biomedical images (
                             <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                              Shen et al. (2017)
                             </a>
                             ) and recurrent neural networks (RNNs) trained on sequential or time series data (
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              Geraci et al. (2017)
                             </a>
                             ). Furthermore, in our experimental evaluations, we only compared EI to the more commonly used early data integration approaches, namely Mashup and deepNF for PFP and XGBoost for COVID-19 mortality prediction. These comparisons should be expanded to intermediate integration approaches as well. Finally, our PFP evaluations were based on multimodal functional associations derived from diverse types of omic data in the STRING database. It would be valuable to evaluate the ability of EI for predicting protein function directly from the raw omic data types. Such expanded applications and evaluations will enable a more comprehensive assessment of the capabilities of EI.
                            </p>
                            <p id="p-70">
                             There are some limitations with the EI interpretation method and its application as well. First, since predictive model interpretation can be subjective, and hence, not scalable (
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              Doshi-Velez and Kim (2017)
                             </a>
                             ), we only tested this method on the COVID-19 datasets and mortality outcome, not the considerably more numerous (2,139) and diverse GO terms considered in the PFP experiments. To interpret the EI models built for these terms, it would be useful to prioritize these terms based on their relevance to the biological topic of interest.
                            </p>
                            <p id="p-71">
                             Furthermore, the implementation of the proposed feature ranking method was based on AUPRC, since this was the only class imbalance-aware measure available in the Weka and sklearn functions used for determining local ranks of features and models respectively. However, since the basic principles of these rankings are general, they can be implemented with other performance measures, such as F
                             <sub>
                              max
                             </sub>
                             , as well using custom code. Also, due to the use of percentile ranks in the interpretation method, it is slightly biased in favor of modalities with larger sets of features. Thus, in addition to the clinical relevance of laboratory test measurements to monitoring patients’ COVID-19 status, this bias also played a role in these features being the highest ranked for the mortality outcome. This limitation can potentially be addressed by considering normalized versions of model and feature ranks.
                            </p>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Funding
                            </h2>
                            <p id="p-73">
                             This work was supported by NIH grant# R01HG011407-01A1.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-72">
                             We thank Akhil Vaid and Sharon Nirenberg for their technical advice on our COVID-19 data processing and experiments, and Andrew DePass and Jamie Bennett for testing the EI code. The study was enabled in part by computational resources provided by Scientific Computing at the Icahn School of Medicine at Mount Sinai, Oracle Cloud credits and related resources provided by the Oracle for Research program.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-7">
                               This version of the manuscript has been revised to provide more details and clarifications of the methods and results.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-8">
                               <a href="https://github.com/GauravPandeyLab/ensemble_integration">
                                https://github.com/GauravPandeyLab/ensemble_integration
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.1" data-doi="10.1038/75556">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ashburner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Gene ontology: tool for the unification of biology. the gene ontology consortium
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  25
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  29
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/75556&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10802651&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000086884000011&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Basheer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Clinical predictors of mortality and critical illness in patients with covid-19 pneumonia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Metabolites
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.3" data-doi="10.1016/j.cmi.2020.07.024">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berenguer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Characteristics and predictors of death among 4035 consecutively hospitalized patients with covid-19 in spain
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clin Microbiol Infect
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1525
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1536
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClin%2BMicrobiol%2BInfect%26rft.volume%253D26%26rft.spage%253D1525%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmi.2020.07.024%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cmi.2020.07.024&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boehm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Harnessing multimodal data integration to advance precision oncology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature reviews. Cancer
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Breiman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Random forests
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Machine Learning
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  45
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  5
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  32
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%26rft.volume%253D45%26rft.spage%253D5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caldera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Interactome-based approaches to human disease
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current Opinion in Systems Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  88
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  94
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BSystems%2BBiology%26rft.volume%253D3%26rft.spage%253D88%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caruana
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Ensemble selection from libraries of models
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the twenty-first international conference on Machine learning
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caruana
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Getting the most out of ensemble selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Sixth International Conference on Data Mining (ICDM’06)
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  828
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  833
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.29.123497v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guestrin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  Xgboost: A scalable tree boosting system
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, KDD ‘16
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  785
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  794
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  New York, NY, USA
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Association for Computing Machinery
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Compact integration of multi-network topology for functional analysis of genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Syst
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  540
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  548
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BSyst%26rft.volume%253D3%26rft.spage%253D540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cowen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Network propagation: a universal amplifier of genetic associations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Rev Genet
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  551
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  562
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BRev%2BGenet%26rft.volume%253D18%26rft.spage%253D551%26rft_id%253Dinfo%253Apmid%252F28607512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28607512&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Demšar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Statistical comparisons of classifiers over multiple data sets
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Machine Learning Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  30
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D7%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.13" data-doi="10.1016/S1473-3099(20)30120-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An interactive web-based dashboard to track COVID-19 in real time
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Lancet Infectious Diseases
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  533
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  534
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2BInfectious%2BDiseases%26rft.volume%253D20%26rft.spage%253D533%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252820%252930120-1%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S1473-3099(20)30120-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Doshi-Velez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Towards a rigorous science of interpretable machine learning
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv: Machine Learning
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.29.123497v3.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frank
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-source">
                                  Weka: A machine learning workbench for data mining
                                 </span>
                                 ., pages
                                 <span class="cit-fpage">
                                  1305
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1314
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer, Berlin
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.16" data-doi="10.1136/eb-2017-102688">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Geraci
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Applying deep neural networks to unstructured text notes in electronic medical records for phenotyping youth depression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Evidence-Based Mental Health
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  83
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  87
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEvidence-Based%2BMental%2BHealth%26rft_id%253Dinfo%253Adoi%252F10.1136%252Feb-2017-102688%26rft_id%253Dinfo%253Apmid%252F28739578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZWJtZW50YWwiO3M6NToicmVzaWQiO3M6NzoiMjAvMy84MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzA3LzI1LzIwMjAuMDUuMjkuMTIzNDk3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gligorijević
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pržulj
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Methods for biological data integration: perspectives and challenges
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J R Soc Interface
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  112
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gligorijević
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  deepNF: deep network fusion for protein function prediction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  3873
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3881
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.29.123497v3.19">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    W.
                                   </span>
                                   <span class="cit-name-surname">
                                    Buntine
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    M.
                                   </span>
                                   <span class="cit-name-surname">
                                    Grobelnik
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    D.
                                   </span>
                                   <span class="cit-name-surname">
                                    Mladenić
                                   </span>
                                  </span>
                                  , and
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    J.
                                   </span>
                                   <span class="cit-name-surname">
                                    Shawe-Taylor
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Greene
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cunningham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  A matrix factorization approach for integrating multiple data views
                                 </span>
                                 . In
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Buntine
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Grobelnik
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mladenić
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Shawe-Taylor
                                  </span>
                                 </span>
                                 , editors,
                                 <span class="cit-source">
                                  Machine Learning and Knowledge Discovery in Databases
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  423
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  438
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  Berlin, Heidelberg
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer Berlin Heidelberg
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.20" data-doi="10.1186/s13059-017-1215-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hasin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Multi-omics approaches to disease
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  83
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D18%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-017-1215-1%26rft_id%253Dinfo%253Apmid%252F28476144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-017-1215-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28476144&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.21" data-doi="10.1038/nrg3208">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jensen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P. B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Mining electronic health records: towards better research applications and clinical care
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  395
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  405
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DJensen%26rft.auinit1%253DP.%2BB.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D395%26rft.epage%253D405%26rft.atitle%253DMining%2Belectronic%2Bhealth%2Brecords%253A%2Btowards%2Bbetter%2Bresearch%2Bapplications%2Band%2Bclinical%2Bcare.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3208%26rft_id%253Dinfo%253Apmid%252F22549152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3208&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22549152&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klopfenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. V.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Goatools: A python library for gene ontology analyses
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific Reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  10872
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2BReports%26rft.volume%253D8%26rft.spage%253D10872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krassowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  State of the field in multi-omics research: From computational needs to data mining and sharing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Genetics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1598
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BGenetics%26rft.volume%253D11%26rft.spage%253D1598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.24" data-doi="10.1038/nrg3920">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Libbrecht
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. W.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W. S.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Machine learning applications in genetics and genomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  321
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  332
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D16%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3920%26rft_id%253Dinfo%253Apmid%252F25948244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrg3920&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25948244&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.25" data-doi="10.1177/0004563220922255">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lippi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Electrolyte imbalances in patients with severe coronavirus disease 2019 (COVID-19)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Clin Biochem
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  57
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  262
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  265
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BClin%2BBiochem%26rft.volume%253D57%26rft.spage%253D262%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0004563220922255%26rft_id%253Dinfo%253Apmid%252F32266828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1177/0004563220922255&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32266828&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.26" data-doi="10.1093/bioinformatics/btaa150">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Supervised learning is an accurate method for network-based gene classification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3457
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3465
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D36%26rft.spage%253D3457%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtaa150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btaa150&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lundberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  From local explanations to global understanding with explainable ai for trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Machine Intelligence
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  56
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  67
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMachine%2BIntelligence%26rft.volume%253D2%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.28" data-doi="10.1186/gb-2008-9-s1-s4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mostafavi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  GeneMANIA: a real-time multiple association network integration algorithm for predicting gene function
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  Suppl 1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  S4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiology%26rft.volume%253D9%26rft.spage%253DS4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-s1-s4%26rft_id%253Dinfo%253Apmid%252F18613948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-s1-s4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18613948&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.29.123497v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  Computational Approaches for Protein Function Prediction: A Survey
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Technical Report 06-028
                                 </span>
                                 ,
                                 <span class="cit-publ-name">
                                  University of Minnesota
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.30" data-doi="10.1524/auto.2011.0951">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pedregosa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Scikit-learn: Machine learning in python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Machine Learning Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  85
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2825
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2830
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMachine%2BLearning%2BResearch%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1524%252Fauto.2011.0951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1524/auto.2011.0951&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.31" data-doi="10.1056/NEJMsa2011686">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Price-Haywood
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Hospitalization and mortality among black patients and white patients with covid-19
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  N Engl J Med
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  382
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  26
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2534
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2543
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DN%2BEngl%2BJ%2BMed%26rft.volume%253D382%26rft.spage%253D2534%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMsa2011686%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJMsa2011686&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B. T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Prevalence and risk factors for delirium in critically ill patients with COVID-19 (COVID-D): a multicentre cohort study
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Lancet Respir Med
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  239
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  250
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLancet%2BRespir%2BMed%26rft.volume%253D9%26rft.spage%253D239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Radivojac
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A large-scale evaluation of computational protein function prediction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  221
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  227
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D10%26rft.spage%253D221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ray
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. L.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reich
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N. G.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Prediction of infectious disease epidemics via weighted density ensembles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D14%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.35" data-doi="10.1093/bioinformatics/btm344">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saeys
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A review of feature selection techniques in bioinformatics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2507
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2517
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm344%26rft_id%253Dinfo%253Apmid%252F17720704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm344&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17720704&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000250673800001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schapire
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. E.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Freund
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Boosting: Foundations and algorithms
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Kybernetes
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sesmero
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Generating ensembles of heterogeneous classifiers using stacked generalization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  WIREs Data Mining and Knowledge Discovery
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  21
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  34
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DWIREs%2BData%2BMining%2Band%2BKnowledge%2BDiscovery%26rft.volume%253D5%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.38" data-doi="10.1146/annurev-bioeng-071516-044442">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Deep learning in medical image analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu Rev Biomed Eng
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  221
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu%2BRev%2BBiomed%2BEng%26rft.volume%253D19%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-bioeng-071516-044442%26rft_id%253Dinfo%253Apmid%252F28301734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-bioeng-071516-044442&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28301734&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shwartz-Ziv
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Armon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  Tabular data: Deep learning is not all you need
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sieberts
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Crowdsourced assessment of common genetic contribution to predicting anti-tnf treatment response in rheumatoid arthritis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  12460
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BCommunications%26rft.volume%253D7%26rft.spage%253D12460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sieberts
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Developing better digital health measures of parkinson’s disease using free living data and a crowdsourced data analysis challenge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  medRxiv
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.42" data-doi="10.1142/9789813207813_0028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stanescu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pandey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Learning parsimonious ensembles for unbalanced computational genomics problems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Pac Symp Biocomput
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  288
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  299
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPac%2BSymp%2BBiocomput%26rft.volume%253D22%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1142%252F9789813207813_0028%26rft_id%253Dinfo%253Apmid%252F27896983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1142/9789813207813_0028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27896983&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.43" data-doi="10.1093/nar/gkaa1074">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Szklarczyk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The STRING database in 2021: customizable protein–protein networks, and functional characterization of user-uploaded gene/measurement sets
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Research
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  49
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  D1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  D605
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D612
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.volume%253D49%26rft.spage%253DD605%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkaa1074%26rft_id%253Dinfo%253Apmid%252F33237311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkaa1074&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=33237311&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.44" data-doi="10.1093/nar/gkw1108">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  The Gene Ontology Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Expansion of the gene ontology knowledgebase and resources
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .,
                                 <span class="cit-vol">
                                  45
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  D1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  D331
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D338
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1108%26rft_id%253Dinfo%253Apmid%252F27899567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkw1108&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27899567&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.45" data-doi="10.1093/bioinformatics/17.6.520">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Troyanskaya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Missing value estimation methods for DNA microarrays
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  520
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  525
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DTroyanskaya%26rft.auinit1%253DO.%26rft.volume%253D17%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D525%26rft.atitle%253DMissing%2Bvalue%2Bestimation%2Bmethods%2Bfor%2BDNA%2Bmicroarrays%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.6.520%26rft_id%253Dinfo%253Apmid%252F11395428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/17.6.520&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11395428&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000169404700005&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.46" data-doi="10.2196/24018">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vaid
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Machine learning to predict mortality and critical events in a cohort of patients with covid-19 in new york city: Model development and validation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Med Internet Res
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e24018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BMed%2BInternet%2BRes%26rft.volume%253D22%26rft.spage%253De24018%26rft_id%253Dinfo%253Adoi%252F10.2196%252F24018%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2196/24018&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Walsh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  DOME: recommendations for supervised machine learning validation in biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1122
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1127
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D18%26rft.spage%253D1122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.48" data-doi="10.1038/nmeth.2810">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Similarity network fusion for aggregating data types on a genomic scale
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  333
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  337
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D11%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2810%26rft_id%253Dinfo%253Apmid%252F24464287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.2810&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24464287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Large-scale protein function prediction using heterogeneous ensembles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  F1000Res
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whalen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Predicting protein function and other biomedical characteristics with heterogeneous ensembles
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  92
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  102
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D93%26rft.spage%253D92%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.51">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wynants
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Prediction models for diagnosis and prognosis of covid-19: systematic review and critical appraisal
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMJ
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  369
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yadaw
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Clinical features of COVID-19 mortality: development and validation of a clinical prediction model
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Lancet Digit Health
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e516
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  e525
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DLancet%2BDigit%2BHealth%26rft.volume%253D2%26rft.spage%253D516e%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F07%2F25%2F2020.05.29.123497.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.53">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.29.123497v3.54">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zitnik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Machine learning for integrating data in biology and medicine: Principles, practice, and opportunities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Information Fusion
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  71
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  91
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInformation%2BFusion%26rft.volume%253D50%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.29.123497v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.07.23.501266v2" title="Exploring Relationships between Functional Network Connectivity and Cognition with an Explainable Clustering Approach" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.10.483885v1" title="Cancer related subarachnoid hemorrhage: a multicenter retrospective study using propensity score matching analysis" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted July 25, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.29.123497v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.05.29.123497v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/07/25/2020.05.29.123497.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.29.123497v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.29.123497v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Integrating multimodal data through interpretable heterogeneous ensembles
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2662353"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-0P3HquohTqybjgX8tXzoSH866gEDPpfWD-kSpfqVgTI"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484471615"/>
                                   <input type="hidden" name="captcha_token" value="fc648e03f807045ad98ab05015c87118"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2662353--3" data-node-nid="2662353" id="node-2662353--4726181433" data-pisa="biorxiv;2020.05.29.123497v3" data-pisa-master="biorxiv;2020.05.29.123497" data-seqnum="2662353" data-apath="/biorxiv/early/2022/07/25/2020.05.29.123497.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Integrating multimodal data through interpretable heterogeneous ensembles
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Yan Chak
                                  </span>
                                  <span class="nlm-surname">
                                   Li
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Linhua
                                  </span>
                                  <span class="nlm-surname">
                                   Wang
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Jeffrey N.
                                  </span>
                                  <span class="nlm-surname">
                                   Law
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   T. M.
                                  </span>
                                  <span class="nlm-surname">
                                   Murali
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Gaurav
                                  </span>
                                  <span class="nlm-surname">
                                   Pandey
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.29.123497;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.29.123497
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&title=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&title=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&text=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&t=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&title=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&title=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&title=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2662353--3" data-node-nid="2662353" id="citation-node-2662353--42001600848" data-pisa="biorxiv;2020.05.29.123497v3" data-pisa-master="biorxiv;2020.05.29.123497" data-seqnum="2662353" data-apath="/biorxiv/early/2022/07/25/2020.05.29.123497.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Integrating multimodal data through interpretable heterogeneous ensembles
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Yan Chak
                                      </span>
                                      <span class="nlm-surname">
                                       Li
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Linhua
                                      </span>
                                      <span class="nlm-surname">
                                       Wang
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Jeffrey N.
                                      </span>
                                      <span class="nlm-surname">
                                       Law
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       T. M.
                                      </span>
                                      <span class="nlm-surname">
                                       Murali
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Gaurav
                                      </span>
                                      <span class="nlm-surname">
                                       Pandey
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.29.123497;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.29.123497
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Li%2C+Yan+Chak&rft.au=Wang%2C+Linhua&rft.au=Law%2C+Jeffrey+N.&rft.au=Murali%2C+T.+M.&rft.au=Pandey%2C+Gaurav">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2662353/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2662353/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2662353/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2662353/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2662353/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2662353/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2662353/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2662353/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2662353/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2662353/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2662353/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2662353/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&count=horizontal&via=&text=Integrating%20multimodal%20data%20through%20interpretable%20heterogeneous%20ensembles&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.29.123497v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.29.123497v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
