<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.07.21.214148v5.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.07.21.214148v5.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.07.21.214148v5.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/01/11/2020.07.21.214148.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.07.21.214148v5"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Predicting cancer prognosis and drug response from the tumor microbiome"/>
  <meta name="DC.Identifier" content="10.1101/2020.07.21.214148"/>
  <meta name="DC.Date" content="2022-01-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Tumor gene expression is predictive of patient prognosis in some cancers. However, RNA- seq and whole genome sequencing data contain not only reads from host tumor and normal tissue, but also reads from the tumor microbiome, which can be used to infer the microbial abundances in each tumor. Here, we show that tumor microbial abundances, alone or in combination with tumor gene expression data, can predict cancer prognosis and drug response to some extent – microbial abundances are significantly less predictive of prognosis than gene expression, although remarkably, similarly as predictive of drug response, but in mostly different cancer-drug combinations. Thus, it appears possible to leverage existing sequencing technology, or develop new protocols, to obtain more non-redundant information about prognosis and drug response from RNA-seq and whole genome sequencing experiments than could be obtained from tumor gene expression or genomic data alone.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Leandro C. Hermida"/>
  <meta name="DC.Contributor" content="E. Michael Gertz"/>
  <meta name="DC.Contributor" content="Eytan Ruppin"/>
  <meta name="article:published_time" content="2022-01-11"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Predicting cancer prognosis and drug response from the tumor microbiome"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Tumor gene expression is predictive of patient prognosis in some cancers. However, RNA- seq and whole genome sequencing data contain not only reads from host tumor and normal tissue, but also reads from the tumor microbiome, which can be used to infer the microbial abundances in each tumor. Here, we show that tumor microbial abundances, alone or in combination with tumor gene expression data, can predict cancer prognosis and drug response to some extent – microbial abundances are significantly less predictive of prognosis than gene expression, although remarkably, similarly as predictive of drug response, but in mostly different cancer-drug combinations. Thus, it appears possible to leverage existing sequencing technology, or develop new protocols, to obtain more non-redundant information about prognosis and drug response from RNA-seq and whole genome sequencing experiments than could be obtained from tumor gene expression or genomic data alone.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.07.21.214148v5"/>
  <meta name="citation_id" content="2020.07.21.214148v5"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.07.21.214148"/>
  <meta name="citation_num_pages" content="46"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.07.21.214148"/>
  <meta name="citation_author" content="Leandro C. Hermida"/>
  <meta name="citation_author_institution" content="Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of Maryland"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3887-518X"/>
  <meta name="citation_author" content="E. Michael Gertz"/>
  <meta name="citation_author_institution" content="Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8390-4387"/>
  <meta name="citation_author" content="Eytan Ruppin"/>
  <meta name="citation_author_institution" content="Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)"/>
  <meta name="citation_author_email" content="eytan.ruppin@nih.gov"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7862-3940"/>
  <meta name="citation_reference" content="Grossman, R. L. et al. Toward a Shared Vision for Cancer Genomic Data. New England Journal of Medicine 375, 1109–1112 (2016)."/>
  <meta name="citation_reference" content="Ahluwalia, P., Kolhe, R. & Gahlay, G. K. The clinical relevance of gene expression based prognostic signatures in colorectal cancer. Biochimica et Biophysica Acta (BBA) - Reviews on Cancer 1875, 188513 (2021)."/>
  <meta name="citation_reference" content="Brodsky, A. S. et al. Expression profiling of primary and metastatic ovarian tumors reveals differences indicative of aggressive disease. PLoS One 9, e94476 (2014)."/>
  <meta name="citation_reference" content="Liu, Y. et al. Pan-cancer analysis of clinical significance and associated molecular features of glycolysis. Bioengineered 12, 4233–4246 (2021)."/>
  <meta name="citation_reference" content="Selfors, L. M., Stover, D. G., Harris, I. S., Brugge, J. S. & Coloff, J. L. Identification of cancer genes that are independent of dominant proliferation and lineage programs. Proc Natl Acad Sci U S A 114, E11276–E11284 (2017)."/>
  <meta name="citation_reference" content="Shimoni, Y. Association between expression of random gene sets and survival is evident in multiple cancer types and may be explained by sub-classification. PLoS Comput Biol 14, e1006026 (2018)."/>
  <meta name="citation_reference" content="Shukla, S. et al. Development of an RNA-Seq Based Prognostic Signature in Lung Adenocarcinoma. J Natl Cancer Inst 109, (2017)."/>
  <meta name="citation_reference" content="Venet, D., Dumont, J. E. & Detours, V. Most random gene expression signatures are significantly associated with breast cancer outcome. PLoS Comput Biol 7, e1002240 (2011)."/>
  <meta name="citation_reference" content="Milanez-Almeida, P., Martins, A. J., Germain, R. N. & Tsang, J. S. Cancer prognosis with shallow tumor RNA sequencing. Nature Medicine 26, 188–192 (2020)."/>
  <meta name="citation_reference" content="Poore, G. D. et al. Microbiome analyses of blood and tissues suggest cancer diagnostic approach. Nature 579, 567–574 (2020)."/>
  <meta name="citation_reference" content="Simon, N., Friedman, J., Hastie, T., & Tibshirani, R. Regularization Paths for Cox’s Proportional Hazards Model via Coordinate Descent. Journal of Statistical Software 39 (5), 1–13 (2011)."/>
  <meta name="citation_reference" content="Hung, H. & Chiang, C.T. Estimation methods for time-dependent AUC models with survival data. Canadian Journal of Statistics 38 (1), 8–26 (2010)."/>
  <meta name="citation_reference" content="Lambert, J. & Chevret, S. Summary measure of discrimination in survival models based on cumulative/dynamic time-dependent ROC curves. Statistical methods in medical research 25 (5), 2088–2102 (2016)."/>
  <meta name="citation_reference" content="Ding Z et al. Evaluating the molecule-based prediction of clinical drug responses in cancer. Bioinformatics 32, (19): 2891–5 (2016)."/>
  <meta name="citation_reference" content="Guyon, I., Weston, J., Barnhill, S. & Vapnik, V. Gene Selection for Cancer Classification using Support Vector Machines. Machine Learning 46, 389–422 (2002)."/>
  <meta name="citation_reference" content="Zou H. & Hastie T. Regularization and variable selection via the elastic net. J R Statist Soc B 67 (2), 301–320 (2005)."/>
  <meta name="citation_reference" content="Ritchie, M.E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res 43 (7), e47 (2015)."/>
  <meta name="citation_reference" content="Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010)."/>
  <meta name="citation_reference" content="McCarthy, D. J., Chen, Y. & Smyth, G. K. Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation. Nucleic Acids Res 40, 4288–4297 (2012)."/>
  <meta name="citation_reference" content="Geller, L. T. et al. Potential role of intratumor bacteria in mediating tumor resistance to the chemotherapeutic drug gemcitabine. Science 357, 1156–1160 (2017)."/>
  <meta name="citation_reference" content="Pushalkar, S. et al. The Pancreatic Cancer Microbiome Promotes Oncogenesis by Induction of Innate and Adaptive Immune Suppression. Cancer Discov 8, 403–416 (2018)."/>
  <meta name="citation_reference" content="Nejman, D. et al. The human tumor microbiome is composed of tumor type–specific intracellular bacteria. Science 368, 973–980 (2020)."/>
  <meta name="citation_reference" content="Vabalas A. et al. Machine learning algorithm validation with a limited sample size. PLoS One 14 (11): e0224365 (2019)."/>
  <meta name="citation_reference" content="Lin, D. et al. Microbiome factors in HPV-driven carcinogenesis and cancers. PLoS Pathog 16 (6), e1008524 (2020)."/>
  <meta name="citation_reference" content="Zhu, H. et al. Chlamydia Trachomatis Infection-Associated Risk of Cervical Cancer: A Meta-Analysis. Medicine (Baltimore) 95 (13): e3077 (2016)."/>
  <meta name="citation_reference" content="Gnanasekar A. et al. The intratumor microbiome predicts prognosis across gender and subtypes in papillary thyroid carcinoma. Comput Struct Biotechnol J 19, 1986–1997 (2021)."/>
  <meta name="citation_reference" content="Dohlman et al. The cancer microbiome atlas: a pan-cancer comparative analysis to distinguish tissue-resident microbiota from contaminants. Cell Host Microbe 10 (2), 281–298.e5 (2021)."/>
  <meta name="citation_reference" content="Parsonnet, J. et al. Helicobacter pylori infection and the risk of gastric carcinoma. N Engl J Med 325, 1127–1131 (1991)."/>
  <meta name="citation_reference" content="Ferreira R.M. et al. Gastric microbial community profiling reveals a dysbiotic cancer-associated microbiota. Gut 67 (2), 226–236 (2018)."/>
  <meta name="citation_reference" content="Abate, G., Qureshi, S. & Mazumder, S. A. Cedecea davisae bacteremia in a neutropenic patient with acute myeloid leukemia. J Infect 63, 83–85 (2011)."/>
  <meta name="citation_reference" content="Akinosoglou, K. et al. Bacteraemia due to Cedecea davisae in a patient with sigmoid colon cancer: a case report and brief review of the literature. Diagn Microbiol Infect Dis 74, 303– 306 (2012)."/>
  <meta name="citation_reference" content="Koh, Y. R. et al. The first Korean case of Sphingobacterium spiritivorum bacteremia in a patient with acute myeloid leukemia. Ann Lab Med 33, 283–287 (2013)."/>
  <meta name="citation_reference" content="Wu, P. et al. Profiling the Urinary Microbiota in Male Patients with Bladder Cancer in China. Front Cell Infect Microbiol 8, 167 (2018)."/>
  <meta name="citation_reference" content="Coker, O. O. et al. Mucosal microbiome dysbiosis in gastric carcinogenesis. Gut 67, 1024– 1032 (2018)."/>
  <meta name="citation_reference" content="Castaño-Rodriguez N. et al. Dysbiosis of the microbiome in gastric carcinogenesis. Sci Rep 7 (1), 15957 (2015)."/>
  <meta name="citation_reference" content="Ogwang, S. et al. Bacterial Conversion of Folinic Acid Is Required for Antifolate Resistance. Journal of Biological Chemistry 286, 15377–15390 (2011)."/>
  <meta name="citation_reference" content="Eslami-S, Z., Majidzadeh-A, K., Halvaei, S., Babapirali, F. & Esmaeili, R. Microbiome and Breast Cancer: New Role for an Ancient Population. Front Oncol 10, (2020)."/>
  <meta name="citation_reference" content="Shin, H. J., Kim, D. N. & Lee, S. K. Association between Epstein-Barr virus infection and chemoresistance to docetaxel in gastric carcinoma. Mol. Cells 32, 173–179 (2011)."/>
  <meta name="citation_reference" content="Di Rienzi, S. C. et al. The human gut and groundwater harbor non-photosynthetic bacteria belonging to a new candidate phylum sibling to Cyanobacteria. eLife 2, e01102 (2013)."/>
  <meta name="citation_reference" content="Sedighi M. et al. Therapeutic bacteria to combat cancer; current advances, challenges, and opportunities. Cancer Med 8 (6), 3167–3181 (2019)."/>
  <meta name="citation_reference" content="Coley WB. Late results of the treatment of inoperable sarcoma by the mixed toxins of erysipelas and bacillus prodigiosus. Trans Southern Surg Gynecol Ass 18, 197 (1906)."/>
  <meta name="citation_reference" content="Fehleisen F. Ueber die Züchtung der Erysipelkokken auf künstlichem Nährboden und ihre Übertragbarkeit auf den Menschen. Dtsch Med Wochenschr 8, 553-554 (1882)."/>
  <meta name="citation_reference" content="Busch W. Aus der Sitzung der medicinischen Section vom 13 November 1867. Berl Klin Wochenschr 5, 137 (1868)."/>
  <meta name="citation_reference" content="Kalaora S. et al. Identification of bacteria-derived HLA-bound peptides in melanoma. Nature 592 (7852), 138–143 (2021)."/>
  <meta name="citation_reference" content="Liu, J. et al. An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics. Cell 173, 400–416.e11 (2018)."/>
  <meta name="citation_reference" content="Pedregosa et al. Scikit-learn: Machine Learning in Python. J Mach Learn Res 12, 2825–2830 (2011)."/>
  <meta name="citation_reference" content="citation_title=Machine Learning and Knowledge Discovery in Databases;citation_pages=243-259;citation_year=2015)"/>
  <meta name="citation_reference" content="Pölsterl, S., Navab, N., & Katouzian, A., An Efficient Training Algorithm for Kernel Survival Support Vector Machines. 4th Workshop on Machine Learning in Life Sciences, 23 September 2016, Riva del Garda, Italy."/>
  <meta name="citation_reference" content="Pölsterl, S. et al. Heterogeneous ensembles for predicting survival of metastatic, castrate-resistant prostate cancer patients. F1000Res 5, 2676 (2017)."/>
  <meta name="citation_reference" content="Hothorn, T., Hornik, K., Wiel, M. A. van de & Zeileis, A. Implementing a Class of Permutation Tests: The coin Package. Journal of Statistical Software 28, 1–23 (2008)."/>
  <meta name="twitter:title" content="Predicting cancer prognosis and drug response from the tumor microbiome"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Tumor gene expression is predictive of patient prognosis in some cancers. However, RNA- seq and whole genome sequencing data contain not only reads from host tumor and normal tissue, but also reads from the tumor microbiome, which can be used to infer the microbial abundances in each tumor. Here, we show that tumor microbial abundances, alone or in combination with tumor gene expression data, can predict cancer prognosis and drug response to some extent – microbial abundances are significantly less predictive of prognosis than gene expression, although remarkably, similarly as predictive of drug response, but in mostly different cancer-drug combinations. Thus, it appears possible to leverage existing sequencing technology, or develop new protocols, to obtain more non-redundant information about prognosis and drug response from RNA-seq and whole genome sequencing experiments than could be obtained from tumor gene expression or genomic data alone.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Predicting cancer prognosis and drug response from the tumor microbiome"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Tumor gene expression is predictive of patient prognosis in some cancers. However, RNA- seq and whole genome sequencing data contain not only reads from host tumor and normal tissue, but also reads from the tumor microbiome, which can be used to infer the microbial abundances in each tumor. Here, we show that tumor microbial abundances, alone or in combination with tumor gene expression data, can predict cancer prognosis and drug response to some extent – microbial abundances are significantly less predictive of prognosis than gene expression, although remarkably, similarly as predictive of drug response, but in mostly different cancer-drug combinations. Thus, it appears possible to leverage existing sequencing technology, or develop new protocols, to obtain more non-redundant information about prognosis and drug response from RNA-seq and whole genome sequencing experiments than could be obtained from tumor gene expression or genomic data alone.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-01-11"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2345108"/>
  <title>
   Predicting cancer prognosis and drug response from the tumor microbiome | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3yExMIGjweFDVjleIBuzZEX5DtrAjSMHqK-9hYiokU0"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2345108","apath":"\/biorxiv\/early\/2022\/01\/11\/2020.07.21.214148.atom","pisa":"biorxiv;2020.07.21.214148v5","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.07.21.214148v5"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.07.21.214148v5","title":"Predicting cancer prognosis and drug response from the tumor microbiome","identifier":"node\/2345108"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QH7wPbkF+zWpwx9ScLJ5S3IRlugiFtcO0Gz3IKZBwSW1+1QkfiOVaz0\\\\\\\/5dzBvrU4nz\\\\\\\/CbZe2bOXtfsGFB24TPUQCGIOyXUNXt7210HnrBvoSjDNBMdK9SFkvXiy4rWU+UNStpgxXikbuuY3TM+x5gO\\\\\\\/05oyM+wWF5uk5LNBmTUZzIWprAbr2yTGaDI7TIuDH4sBoTOj+JU+AZJydOcpLxrqYyIQqZjNhvR5mc93qhWXjqnjlfhsUzPar7qFv9luPWWQIWNdaf3OSMqiswA4UdXZy0a7Ad2x8kpMPcebjYIE0aywanbtx4yg369OgcWCzRTYbRe8WSsTh\\\\\\\/rZ1x4387LX6b3Zbc52UUe0NeTuPAYPhVplffT79sWmMo2STWi81c4SnvEZrqCNoVgyWNBfxOM9JyFJBNP81+GydWlBuO3aH1TPauJ8rAa9lGTp1EdAyCVwMlvRAEWkJdTg21J7yOjqUzzh5Mb8pIzIMhOFO8VBfFwmcZDSDFpGzS7kB15OYRK4UdQ7tIJGw3EgkdTsgWzMCb8YMEywCxjSixxyHZMMCL09OV3kfiIzzu9YvGD6cDqt6w+lj1dpRFpdTZk7O+ZYXUTNNNMh33Ev18EkZTwvP7wIL\\\\\\\/J5LxrIqI1NmA0ZdIK0Q1V7a9Vm5VNBmJ3pQFrUcSITp+xbsuHR8g166CG8\\\\\\\/UVHVoe9xWtbOwUC+UQ19n8SjDRw5vWVozoiWJMajtrfehj\\\\\\\/eTjyhmtAqq0SwQylOnl7pOK9EiygMVDbxG56Z2uBStUvI2EUpXkFY3OLmSrsyXhgsfyQtorSEK5G3xi8MjEcXik5YKRS5\\\\\\\/nlBWLV4VxYVteT8EAxfccpVQf3MmNy0YSqNwIA3zS6kMYDPdP1LxmWcm61WtOK\\\\\\\/Bw2IJo53V3M\\\\\\\/qS93kt3YS+owlyymoHtNitJuOLdETzbAkyGMq++IyqmuDUlUY4Y1o\\\\\\\/er\\\\\\\/iyEw3RjJjpBM7AmrF2fV1B\\\\\\\/98Vr5KxhdAxNXup9NksTRQvoL6RpA42xIFGWt0ImuVHY\\\\\\\/tNPb1Ncqng17fh7SmIdunj4DnqKZq1H7r0SBikDJeqzIFzOEsGtDePqIRnKdAy441QdF8nFGnOyPHDWADwpWSzEPcGooB+U\\\\\\\/R9e76TAvWLMfbkndnbtgNvjf5w5xKaGCN2DtfkB9LwOjPbUytTgN14JAI1szgkb8WvAoZJ5ceF1NMHsE\\\\\\\/18EOO0iQAV0gTgL7fxampJOTCCPZhBRVGAdDEcQ\\\\\\\/X\\\\\\\/rx6yDGKqQhsttwa\\\\\\\/orpIFc+QLeh5GEm5c8iAv1Kdyk+AkLFYibidRVw1lz5wC1fy6875Po2rHQxI\\\\\\\/I\\\\\\\/5\\\\\\\/3u59+ahzalr\\\\\\\/NgPV3dBRHuxYF6ToHAqrAaQx4ElXR5olRuXRzNjt59uJuu+\\\\\\\/oU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022bYoZOTe3UvP\\\\\\\/4V+z+kM47A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022c74c0216401c2e05206b5a48c3effb0216ab8d39dd5d515e02b2378bd53e5f03\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.07.21.214148v5.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2345108 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.07.21.214148v5.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_971604364">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_971604364" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1906473728" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-zmMllgzV0xEQNEmO2oPB5e7Eh1xRykyIZVhdgVM01EU"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2345108" data-node-nid="2345108" id="node-2345108--21627004503" data-pisa="biorxiv;2020.07.21.214148v5" data-pisa-master="biorxiv;2020.07.21.214148" data-apath="/biorxiv/early/2022/01/11/2020.07.21.214148.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Predicting cancer prognosis and drug response from the tumor microbiome
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0003-3887-518X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Leandro C.
                      </span>
                      <span class="nlm-surname">
                       Hermida
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0001-8390-4387" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       E. Michael
                      </span>
                      <span class="nlm-surname">
                       Gertz
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0002-7862-3940" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Eytan
                      </span>
                      <span class="nlm-surname">
                       Ruppin
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.07.21.214148
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2345108--21627004503" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Leandro C. Hermida
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)
                       </span>
                       , Bethesda, MD,
                       <span class="nlm-country">
                        USA
                       </span>
                       .
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of Maryland
                       </span>
                       , College Park, MD,
                       <span class="nlm-country">
                        USA
                       </span>
                       .
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Leandro%2BC.%2BHermida%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Hermida%20LC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALeandro%2BC.%2BHermida%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3887-518X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Leandro C. Hermida
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     E. Michael Gertz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)
                       </span>
                       , Bethesda, MD,
                       <span class="nlm-country">
                        USA
                       </span>
                       .
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=E.%2BMichael%2BGertz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gertz%20EM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AE.%2BMichael%2BGertz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-8390-4387" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for E. Michael Gertz
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Eytan Ruppin
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Cancer Data Science Laboratory (CDSL), National Cancer Institute (NCI), National Institutes of Health (NIH)
                       </span>
                       , Bethesda, MD,
                       <span class="nlm-country">
                        USA
                       </span>
                       .
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Eytan%2BRuppin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ruppin%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AEytan%2BRuppin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-7862-3940" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Eytan Ruppin
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:eytan.ruppin@nih.gov" class="" data-icon-position="" data-hide-link-title="0">
                        eytan.ruppin@nih.gov
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.07.21.214148v5" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.21.214148v5.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.21.214148v5.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.21.214148v5.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.21.214148v5.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.21.214148v5.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.07.21.214148v5.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2345108" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2345108/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             Tumor gene expression is predictive of patient prognosis in some cancers. However, RNA- seq and whole genome sequencing data contain not only reads from host tumor and normal tissue, but also reads from the tumor microbiome, which can be used to infer the microbial abundances in each tumor. Here, we show that tumor microbial abundances, alone or in combination with tumor gene expression data, can predict cancer prognosis and drug response to some extent – microbial abundances are significantly less predictive of prognosis than gene expression, although remarkably, similarly as predictive of drug response, but in mostly different cancer-drug combinations. Thus, it appears possible to leverage existing sequencing technology, or develop new protocols, to obtain more non-redundant information about prognosis and drug response from RNA-seq and whole genome sequencing experiments than could be obtained from tumor gene expression or genomic data alone.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-9">
                             The Cancer Genome Atlas (TCGA), available from the NCI Genomic Data Commons (GDC)
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             , provides RNA-seq and whole genomic sequencing (WGS) data for thousands of cases across dozens of cancer types. RNA-seq data is typically used to measure the expression of human genes, and there is a long history linking tumor gene expression to cancer outcomes
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              –
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . Milanez-Almeida et al.
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             recently showed that gene expression from TCGA RNA-seq data could predict overall survival (OS) or progression-free interval (PFI) better than classical clinical prognostic covariates – age at diagnosis, gender, and tumor stage. Importantly, Milanez-Almeida et al. used a data-driven machine learning (ML) based approach which selected features that were predictive of and correlated with prognosis, rather than approaches based on classical statistics or biological knowledge that chose features a priori.
                            </p>
                            <p id="p-10">
                             Research into the human tumor microbiome has been rapidly expanding, and multiple laboratories have attempted to utilize existing technologies and data to identify microbes and quantify their abundance within human tumors compared to adjacent normal tissue. RNA-seq and WGS data not only contain human sequencing reads, but also reads from the local intratumor microbiome that are typically filtered out from the data when analyzing human gene expression or genomic alterations. Poore et al.
                             <sup>
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             recently developed a computational workflow, using two orthogonal microbial detection pipelines, to estimate, decontaminate, normalize, and batch effect correct microbial abundances from human high-throughput sequencing data. They applied this workflow to create a first-of-its-kind comprehensive dataset of pan-cancer tumor microbial abundances derived from WGS or RNA-seq data for the entire TCGA cohort.
                            </p>
                            <p id="p-11">
                             Our central research questions then were, 1) does a data-driven ML approach reveal that tumor microbial abundances in TCGA data, quantified from these reads, are predictive of cancer prognosis or drug response, 2) what microbial genera are potentially predictive biomarkers of prognosis or drug response, 3) how do these models compare to equivalent models based on tumor gene expression data, and 4) does combining both microbial abundance and gene expression features produce models and select combinations of genes and microbial genera that are more predictive of prognosis or drug response than models from each individual data type? We used the processed microbial abundances directly from the Poore et al. dataset to build predictive models of prognosis and drug response for TCGA. We also used TCGA RNA-seq read counts to build equivalent predictive models for comparison. As a positive control, we also showed that our prognosis ML modeling methods, which differed somewhat from Milanez-Almeida et al.
                             <sup>
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                             </sup>
                             , identified a similar set cancers and outcomes for which gene expression was predictive of prognosis.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Tumor microbial abundances are substantially less predictive of prognosis than gene expression
                             </h3>
                             <p id="p-12">
                              An overview of the analytical workflow is presented in
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                              </strong>
                              . It has four major parts, 1) data download and preprocessing, 2) prognosis and drug response ML modeling, 3) model evaluation and scoring, and 4) further feature analysis. A more detailed technical description of the analysis pipeline and computational methods is provided in Methods.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F1.large.jpg?width=800&height=600&carousel=1" title="Analysis pipeline overview. Download and data preprocessing (left) of Poore et al. TCGA primary tumor Kraken2 Voom-SNM microbial abundances with additional filters to reduce technical variation, NCI Genomic Data Commons (GDC) harmonized TCGA primary tumor RNA-seq counts and clinical data, Liu et al. TCGA curated overall survival (OS) and progression-free interval (PFI) outcome data, and Ding et al. TCGA curated drug response clinical data. Prognosis machine learning (ML) modeling (middle) of microbial abundance, gene expression, and combined data types with clinical covariates for each cancer using penalized Cox with elastic net penalties (Coxnet) against matched clinical covariate-only models using standard Cox regression. Drug response classification ML modeling of the same data types with clinical covariates for each cancer-drug combination using three ML approaches, 1) SVM-RFE, elastic net logistic regression (LGR), and limma-trend (microbial and combined data types) or edgeR (gene expression) differential analysis feature scoring and selection with L2 penalized LGR. Matched clinical covariate-only modeling performed with L2 penalized linear SVM or LGR. ML modeling generates 100 model instances for each model from 75/25 train/test randomly shuffled and stratified dataset splits. ML model instance scoring (right top) using concordance index (C-index) and time-dependent cumulative/dynamic AUC (C/D AUC(t)) for prognosis models and area under receiver-operating characteristic curve (AUROC) and area under the precision-recall curve (AUPRC) for drug response models. Significance of model performance improvement over matched clinical covariate-only model determined by signed rank test of C-index or AUROC scores between each matched model instance for prognosis and drug response models, respectively. Feature analysis (right bottom) performed using model instance coefficients and selection frequencies. Overall feature importance ranking and significance determined by signed rank test of model instance feature coefficients shifting from zero and filtering of top features for selection frequency ≥ 20%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Analysis pipeline overview.</span> Download and data preprocessing (<strong>left</strong>) of Poore et al. TCGA primary tumor Kraken2 Voom-SNM microbial abundances with additional filters to reduce technical variation, NCI Genomic Data Commons (GDC) harmonized TCGA primary tumor RNA-seq counts and clinical data, Liu et al. TCGA curated overall survival (OS) and progression-free interval (PFI) outcome data, and Ding et al. TCGA curated drug response clinical data. Prognosis machine learning (ML) modeling (<strong>middle</strong>) of microbial abundance, gene expression, and combined data types with clinical covariates for each cancer using penalized Cox with elastic net penalties (Coxnet) against matched clinical covariate-only models using standard Cox regression. Drug response classification ML modeling of the same data types with clinical covariates for each cancer-drug combination using three ML approaches, 1) SVM-RFE, elastic net logistic regression (LGR), and limma-trend (microbial and combined data types) or edgeR (gene expression) differential analysis feature scoring and selection with L2 penalized LGR. Matched clinical covariate-only modeling performed with L2 penalized linear SVM or LGR. ML modeling generates 100 model instances for each model from 75/25 train/test randomly shuffled and stratified dataset splits. ML model instance scoring (<strong>right top</strong>) using concordance index (C-index) and time-dependent cumulative/dynamic AUC (C/D AUC(t)) for prognosis models and area under receiver-operating characteristic curve (AUROC) and area under the precision-recall curve (AUPRC) for drug response models. Significance of model performance improvement over matched clinical covariate-only model determined by signed rank test of C-index or AUROC scores between each matched model instance for prognosis and drug response models, respectively. Feature analysis (<strong>right bottom</strong>) performed using model instance coefficients and selection frequencies. Overall feature importance ranking and significance determined by signed rank test of model instance feature coefficients shifting from zero and filtering of top features for selection frequency ≥ 20%.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F1.medium.gif" width="440" height="226"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F1.medium.gif" width="440" height="226"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Analysis pipeline overview.
                               </span>
                               <p id="p-13" class="first-child">
                                Download and data preprocessing (
                                <strong>
                                 left
                                </strong>
                                ) of Poore et al. TCGA primary tumor Kraken2 Voom-SNM microbial abundances with additional filters to reduce technical variation, NCI Genomic Data Commons (GDC) harmonized TCGA primary tumor RNA-seq counts and clinical data, Liu et al. TCGA curated overall survival (OS) and progression-free interval (PFI) outcome data, and Ding et al. TCGA curated drug response clinical data. Prognosis machine learning (ML) modeling (
                                <strong>
                                 middle
                                </strong>
                                ) of microbial abundance, gene expression, and combined data types with clinical covariates for each cancer using penalized Cox with elastic net penalties (Coxnet) against matched clinical covariate-only models using standard Cox regression. Drug response classification ML modeling of the same data types with clinical covariates for each cancer-drug combination using three ML approaches, 1) SVM-RFE, elastic net logistic regression (LGR), and limma-trend (microbial and combined data types) or edgeR (gene expression) differential analysis feature scoring and selection with L2 penalized LGR. Matched clinical covariate-only modeling performed with L2 penalized linear SVM or LGR. ML modeling generates 100 model instances for each model from 75/25 train/test randomly shuffled and stratified dataset splits. ML model instance scoring (
                                <strong>
                                 right top
                                </strong>
                                ) using concordance index (C-index) and time-dependent cumulative/dynamic AUC (C/D AUC(t)) for prognosis models and area under receiver-operating characteristic curve (AUROC) and area under the precision-recall curve (AUPRC) for drug response models. Significance of model performance improvement over matched clinical covariate-only model determined by signed rank test of C-index or AUROC scores between each matched model instance for prognosis and drug response models, respectively. Feature analysis (
                                <strong>
                                 right bottom
                                </strong>
                                ) performed using model instance coefficients and selection frequencies. Overall feature importance ranking and significance determined by signed rank test of model instance feature coefficients shifting from zero and filtering of top features for selection frequency ≥ 20%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-14">
                              We built OS and PFI gene expression ML models of 32 TCGA tumor types (see
                              <strong>
                               Supplementary Data 1
                              </strong>
                              for cohort information) using the Coxnet
                              <sup>
                               <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              algorithm, which jointly selects the most predictive subset of features via cross-validation (CV) while simultaneously being able to control for prognostic clinical covariates. In our models, we included and controlled for the clinical covariates age at diagnosis, gender, and tumor stage. For comparison, we also built standard Cox regression models based on the clinical covariates alone. We evaluated the predictive performance of our models using Harrell’s concordance index (C- index), which is a metric of survival model predictive accuracy. Each model analysis generated 100 model instances and C-index scores from randomly shuffled train-test CV splits on the data. We found 33 OS and PFI models for 21 tumor types that had a mean C-index score ≥ 0.6 and significantly outperformed their corresponding clinical covariate-only models (
                              <strong>
                               <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                Fig. 2a
                               </a>
                               &
                               <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                                c
                               </a>
                               ,
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Supplementary Figs. 1a
                               </a>
                               ,
                               <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                                2a
                               </a>
                              </strong>
                              ). Our models were predictive of prognosis in 11 of the same 13 tumor types that were reported by Milanez-Almeida et al.
                              <sup>
                               <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                              </sup>
                              (
                              <strong>
                               <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                                Supplementary Table 1
                               </a>
                              </strong>
                              ). We did not analyze one tumor type that Milanez-Almeida did, acute myeloid leukemia (LAML), because Poore et al. excluded it from their analysis. Among the cancers and outcomes that Milanez-Almeida et al. analyzed, our methodology produced predictive models for four additional tumor types: breast cancer (BRCA), cervical squamous cell carcinoma (CESC), sarcoma (SARC), and uterine corpus endometrial carcinoma (UCEC), as well as quite a few predictive models for additional cancers and outcomes that were not analyzed in their study (
                              <strong>
                               <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                                Supplementary Table 1
                               </a>
                              </strong>
                              ). We also evaluated prognosis model performance by calculating the time-dependent, cumulative/dynamic area under the curve (AUC
                              <sup>
                               C/D
                              </sup>
                              (t))
                              <sup>
                               <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                               ,
                               <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                              </sup>
                              , which is an extension of the area under the receiver-operating characteristic curve (AUROC) for continuous outcomes and can provide a more detailed resolution picture of predictive performance throughout the test outcome time range compared to the C-index score. Although 33 of our OS and PFI gene expression models had a statistically significant C-index score improvement compared to clinical covariates alone, only 22 of these models showed an improvement in AUC
                              <sup>
                               C/D
                              </sup>
                              (t), where the improvement in mean AUC
                              <sup>
                               C/D
                              </sup>
                              (t) over the entire test time range after diagnosis was ≥ 0.025 (
                              <strong>
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Supplementary Figs. 1b
                               </a>
                               ,
                               <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                                2b
                               </a>
                              </strong>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F2.large.jpg?width=800&height=600&carousel=1" title="Performance of gene expression and microbial abundance prognosis prediction models where features add predictive power to clinical covariates. Mean C-index scores for (a) gene expression with clinical covariate models (orange) and (b) microbial abundance with clinical covariate models (blue) vs clinical covariate- only models (grey). Error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. (c) C-index score violin density plots for the six models where microbial abundance with clinical covariate features outperform clinical covariate-only models. Corresponding gene expression models shown for comparison. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores shown as red dots with red lines connecting the means. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. Adjusted p-values colored in red signify difference where clinical covariate-only model is better." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of gene expression and microbial abundance prognosis prediction models where features add predictive power to clinical covariates.</span> Mean C-index scores for <strong>(a)</strong> gene expression with clinical covariate models (orange) and <strong>(b)</strong> microbial abundance with clinical covariate models (blue) vs clinical covariate- only models (grey). Error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. <strong>(c)</strong> C-index score violin density plots for the six models where microbial abundance with clinical covariate features outperform clinical covariate-only models. Corresponding gene expression models shown for comparison. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores shown as red dots with red lines connecting the means. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. Adjusted p-values colored in red signify difference where clinical covariate-only model is better.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F2.medium.gif" width="316" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F2.medium.gif" width="316" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Performance of gene expression and microbial abundance prognosis prediction models where features add predictive power to clinical covariates.
                               </span>
                               <p id="p-15" class="first-child">
                                Mean C-index scores for
                                <strong>
                                 (a)
                                </strong>
                                gene expression with clinical covariate models (orange) and
                                <strong>
                                 (b)
                                </strong>
                                microbial abundance with clinical covariate models (blue) vs clinical covariate- only models (grey). Error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001.
                                <strong>
                                 (c)
                                </strong>
                                C-index score violin density plots for the six models where microbial abundance with clinical covariate features outperform clinical covariate-only models. Corresponding gene expression models shown for comparison. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores shown as red dots with red lines connecting the means. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. Adjusted p-values colored in red signify difference where clinical covariate-only model is better.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              We applied Coxnet
                              <sup>
                               <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              using the same methodology to build prognosis models using the microbial abundance estimates provided by Poore et al.
                              <sup>
                               <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                10
                               </a>
                              </sup>
                              . We found six microbial abundance models that had a mean C-index score ≥ 0.6 and significantly outperformed their corresponding clinical covariate-only models (
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Fig. 2b
                               </a>
                               &
                               <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                                c
                               </a>
                               ,
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Supplementary Fig. 3a
                               </a>
                              </strong>
                              ). We found that in only two of the six models, microbial abundances outperformed clinical covariates alone in terms of AUC
                              <sup>
                               C/D
                              </sup>
                              (t), where the improvement in mean AUC
                              <sup>
                               C/D
                              </sup>
                              (t) over the entire test time range after diagnosis was ≥ 0.025 (
                              <strong>
                               <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                                Supplementary Fig. 3b
                               </a>
                              </strong>
                              ). In adrenocortical carcinoma (ACC), microbial features predicted OS significantly better than clinical prognostic covariates starting at approximately 6 years after diagnosis. In CESC, microbial abundances predicted OS better than clinical covariates from approximately 6 months to 10 years after diagnosis. Overall, we found that tumor microbial abundances from Poore et al. were only marginally predictive of prognosis across the TCGA cohort, and that gene expression was a significantly more powerful predictor of prognosis (
                              <strong>
                               <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                                Fig. 2
                               </a>
                               ,
                               <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                                Supplementary Figs. 1
                               </a>
                               -
                               <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                                3
                               </a>
                              </strong>
                              ).
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Tumor microbial abundances are predictive of chemotherapy drug response in some cancers and in mostly different cancer-drug combinations than gene expression
                             </h3>
                             <p id="p-17">
                              We next asked whether tumor microbial abundances from pre-treatment biopsies could predict drug response better than the clinical covariates age at diagnosis, gender, and tumor stage alone. TCGA drug response clinical data were obtained from Ding et al.
                              <sup>
                               <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                              </sup>
                              as described in Methods. Cases with complete response (CR) or partial response (PR) were labeled as responders and those with stable disease (SD) or progressive disease (PD) as non-responders. Thirty TCGA cancer-drug combinations met our minimum dataset size thresholds (see
                              <strong>
                               Supplementary Data 1
                              </strong>
                              for cohort information). We built drug response models using three different ML methods: 1) a variant of the linear support vector machine recursive feature elimination (SVM-RFE) algorithm
                              <sup>
                               <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                              </sup>
                              that we developed, 2) logistic regression (LGR) with elastic net
                              <sup>
                               <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              (L1 + L2) penalties and embedded feature selection, and 3) logistic regression with an L2 penalty and limma
                              <sup>
                               <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              (for microbial abundance and combined data type datasets) or edgeR
                              <sup>
                               <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                               ,
                               <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                                19
                               </a>
                              </sup>
                              (for RNA-seq count datasets) differential abundance/expression feature scoring and wrapper selection methods (see Methods for details). All three ML methods unconditionally included the clinical covariates – age at diagnosis, gender, and tumor stage – in the model (bypassing feature selection) while selecting the most predictive subset of microbial abundance or gene expression features. For comparison, we built standard linear SVM or LGR models using the clinical covariates alone. We evaluated the predictive performance of drug response models using AUROC. Each analysis generated 100 model instances, AUROC, and area under the precision-recall curve (AUPRC) scores from randomly shuffled train-test CV splits on the data.
                             </p>
                             <p id="p-18">
                              We found five microbial abundance cancer-drug combinations that had a mean AUROC score ≥ 0.6 and performed better than clinical covariates alone in at least two out of three ML methods (
                              <strong>
                               <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                              </strong>
                              ). Three of these cancer-drug combinations involved stomach adenocarcinoma (STAD). We performed the same drug response modeling using TCGA gene expression data and here we found six cancer-drug response combinations that had a mean AUROC score ≥ 0.6 and significantly outperformed their corresponding clinical covariate-only models in at least two out of three ML methods (
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                              </strong>
                              ). Only one cancer-drug combination, SARC docetaxel, overlapped between the microbial abundance and gene expression drug response model results, suggesting that tumor microbial abundances have independent predictive power. Even though one of our thresholds for a significant drug response model was a mean AUROC score ≥ 0.6, the 11 total significant models that we found from both data types each had a mean AUROC > 0.7. We also found there was considerable overlap in the selected microbial abundance and gene expression features reported by each ML method (
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig. 5a
                               </a>
                               & c
                              </strong>
                              ) and frequently found a significant correlation between the feature importance rankings reported by each ML method when comparing the two most significant methods in each cancer-drug combination (
                              <strong>
                               <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                Fig. 5b
                               </a>
                               & d
                              </strong>
                              ). These results suggest that our significant drug response models and their inferred important features are not the result a specific ML modeling methodology. Overall, our results support the notion that the tumor microbiome may contain information that is predictive of drug response in some cancers, consistent with recent reports
                              <sup>
                               <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                                20
                               </a>
                               ,
                               <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                                21
                               </a>
                              </sup>
                              .
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F3.large.jpg?width=800&height=600&carousel=1" title="Performance of microbial abundance drug response prediction models in the five cancer-drug combinations where models performed better than clinical covariates alone. (a) Mean AUROC scores for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) and (b) mean AUROC scores for each ML method. In both (a) and (b) error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. (c) Violin density plots of AUROC scores for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines. (d) Mean ROC (blue) and (e) precision-recall (PR) curves (purple) for microbial abundance with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in (c). In (c-e) results for the modeling method that had the most significant Wilcoxon signed-rank test are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of microbial abundance drug response prediction models in the five cancer-drug combinations where models performed better than clinical covariates alone.</span> <strong>(a)</strong> Mean AUROC scores for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) and <strong>(b)</strong> mean AUROC scores for each ML method. In both <strong>(a)</strong> and <strong>(b)</strong> error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. <strong>(c)</strong> Violin density plots of AUROC scores for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines. <strong>(d)</strong> Mean ROC (blue) and <strong>(e)</strong> precision-recall (PR) curves (purple) for microbial abundance with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in <strong>(c)</strong>. In <strong>(c-e)</strong> results for the modeling method that had the most significant Wilcoxon signed-rank test are shown.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F3.medium.gif" width="388" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F3.medium.gif" width="388" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Performance of microbial abundance drug response prediction models in the five cancer-drug combinations where models performed better than clinical covariates alone.
                               </span>
                               <p id="p-19" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Mean AUROC scores for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) and
                                <strong>
                                 (b)
                                </strong>
                                mean AUROC scores for each ML method. In both
                                <strong>
                                 (a)
                                </strong>
                                and
                                <strong>
                                 (b)
                                </strong>
                                error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001.
                                <strong>
                                 (c)
                                </strong>
                                Violin density plots of AUROC scores for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines.
                                <strong>
                                 (d)
                                </strong>
                                Mean ROC (blue) and
                                <strong>
                                 (e)
                                </strong>
                                precision-recall (PR) curves (purple) for microbial abundance with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in
                                <strong>
                                 (c)
                                </strong>
                                . In
                                <strong>
                                 (c-e)
                                </strong>
                                results for the modeling method that had the most significant Wilcoxon signed-rank test are shown.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F4.large.jpg?width=800&height=600&carousel=1" title="Performance of gene expression drug response prediction models in the six cancer-drug combinations where models performed better than clinical covariates alone. (a) Mean AUROC scores for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) and (b) mean AUROC scores for each ML method. In both (a) and (b) error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. (c) Violin density plots of AUROC scores for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines. (d) Mean ROC (orange) and (e) precision-recall (PR) curves (green) for gene expression with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p- values shown at top of violin plots in (c). In (c-e) results for the modeling method that had the most significant Wilcoxon signed-rank test are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of gene expression drug response prediction models in the six cancer-drug combinations where models performed better than clinical covariates alone.</span> <strong>(a)</strong> Mean AUROC scores for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) and <strong>(b)</strong> mean AUROC scores for each ML method. In both <strong>(a)</strong> and <strong>(b)</strong> error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001. <strong>(c)</strong> Violin density plots of AUROC scores for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines. <strong>(d)</strong> Mean ROC (orange) and <strong>(e)</strong> precision-recall (PR) curves (green) for gene expression with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p- values shown at top of violin plots in <strong>(c)</strong>. In <strong>(c-e)</strong> results for the modeling method that had the most significant Wilcoxon signed-rank test are shown.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F4.medium.gif" width="417" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F4.medium.gif" width="417" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Performance of gene expression drug response prediction models in the six cancer-drug combinations where models performed better than clinical covariates alone.
                               </span>
                               <p id="p-20" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Mean AUROC scores for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) and
                                <strong>
                                 (b)
                                </strong>
                                mean AUROC scores for each ML method. In both
                                <strong>
                                 (a)
                                </strong>
                                and
                                <strong>
                                 (b)
                                </strong>
                                error bars denote standard deviations. Significance: * ≤ 0.01, ** ≤ 0.001, *** ≤ 0.0001.
                                <strong>
                                 (c)
                                </strong>
                                Violin density plots of AUROC scores for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean AUROC scores are shown as red dots connected by red lines.
                                <strong>
                                 (d)
                                </strong>
                                Mean ROC (orange) and
                                <strong>
                                 (e)
                                </strong>
                                precision-recall (PR) curves (green) for gene expression with clinical covariate models vs clinical covariate-only models (grey). Mean AUROC and AUPRC scores shown in legends and shaded areas denote standard deviations. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p- values shown at top of violin plots in
                                <strong>
                                 (c)
                                </strong>
                                . In
                                <strong>
                                 (c-e)
                                </strong>
                                results for the modeling method that had the most significant Wilcoxon signed-rank test are shown.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F5.large.jpg?width=800&height=600&carousel=1" title="Comparison of drug response model top-ranked selected features by each ML method. For each drug response model, we selected the two best ML methods by significance for the prediction improvement over their respective clinical covariate-only model. (a, c) Venn diagrams for microbial abundance (a) or gene expression (c) models comparing the number of features individually selected by each ML method, and the intersection of the two ML methods. (b, d) Spearman rank correlation plots for microbial abundance (b) or gene expression (d) models showing that the median rank of features (among the 100 model instances in which the feature was selected) often correlated between the two most significant ML methods. The best method is shown on the x-axis, the second best on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Comparison of drug response model top-ranked selected features by each ML method.</span> For each drug response model, we selected the two best ML methods by significance for the prediction improvement over their respective clinical covariate-only model. <strong>(a, c)</strong> Venn diagrams for microbial abundance <strong>(a)</strong> or gene expression <strong>(c)</strong> models comparing the number of features individually selected by each ML method, and the intersection of the two ML methods. <strong>(b, d)</strong> Spearman rank correlation plots for microbial abundance <strong>(b)</strong> or gene expression <strong>(d)</strong> models showing that the median rank of features (among the 100 model instances in which the feature was selected) often correlated between the two most significant ML methods. The best method is shown on the x-axis, the second best on the y-axis.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F5.medium.gif" width="440" height="409"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F5.medium.gif" width="440" height="409"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Comparison of drug response model top-ranked selected features by each ML method.
                               </span>
                               <p id="p-21" class="first-child">
                                For each drug response model, we selected the two best ML methods by significance for the prediction improvement over their respective clinical covariate-only model.
                                <strong>
                                 (a, c)
                                </strong>
                                Venn diagrams for microbial abundance
                                <strong>
                                 (a)
                                </strong>
                                or gene expression
                                <strong>
                                 (c)
                                </strong>
                                models comparing the number of features individually selected by each ML method, and the intersection of the two ML methods.
                                <strong>
                                 (b, d)
                                </strong>
                                Spearman rank correlation plots for microbial abundance
                                <strong>
                                 (b)
                                </strong>
                                or gene expression
                                <strong>
                                 (d)
                                </strong>
                                models showing that the median rank of features (among the 100 model instances in which the feature was selected) often correlated between the two most significant ML methods. The best method is shown on the x-axis, the second best on the y-axis.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Combining tumor microbial abundance and gene expression features adds a modest predictive improvement in some cancers
                             </h3>
                             <p id="p-22">
                              Finally, we investigated if models built from combining microbial abundance and gene expression features would result in an improvement in predictive power over their corresponding single data type models. Combining data types resulted in a modest predictive improvement in only three prognosis models: SARC OS, STAD PFI, and thymoma (THYM) OS (
                              <strong>
                               <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                Supplementary Fig. 4a
                               </a>
                              </strong>
                              ). Although this improvement was not statistically significant in terms of C-index score, the AUC
                              <sup>
                               C/D
                              </sup>
                              (t) metric showed a clear improvement in prognostic predictive power for these models, where the improvement in mean AUC
                              <sup>
                               C/D
                              </sup>
                              (t) over the entire time range after diagnosis was ≥ 0.025 compared to their respective single data type models. We also found five combined data type drug response models which performed significantly better than clinical covariates alone, although none of these models reached statistical significance when compared to their respective single data type models in terms of improvement in AUROC score, but one of these models, for BLCA cisplatin, did show an improvement in AUROC ≥ 0.025 compared to its corresponding single data type models (
                              <strong>
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Supplementary Fig. 4b-c
                               </a>
                              </strong>
                              ).
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Evaluating the robustness of drug response models
                             </h3>
                             <p id="p-23">
                              Some of the TCGA drug response cohorts used in our study were of limited size and this could have an impact on the robustness of our analysis (see
                              <strong>
                               Supplementary Data 1
                              </strong>
                              for cohort size information). To study this issue further, we evaluated the significance of model scores using a class label permutation test. We shuffled dataset class labels 1000 times and each time ran the outer CV procedure on the permuted dataset, where for each CV iteration we fit a model instance and calculated an AUROC score. We then calculated a p-value from the fraction of permuted scores that were greater than or equal to the true score. Three of the five microbial abundance drug response models that were reported above to have perform significantly better than clinical covariates alone had a permutation test p-value < 0.05 and the remaining two, for stomach adenocarcinoma (STAD) cisplatin and oxaliplatin, had p-values < 0.08 (
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Fig. 6a
                               </a>
                              </strong>
                              ). Permutation test scores and significance for microbial abundance models were similar regardless of the modeling method used (
                              <strong>
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                Supplementary Fig. 5a
                               </a>
                              </strong>
                              ). Five of the six gene expression drug response models that performed significantly better than clinical covariates alone had a permutation test p-value < 0.05 (
                              <strong>
                               <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                Fig. 6c
                               </a>
                              </strong>
                              ). Again here, permutation test scores and significance were similar regardless of the modeling method used (
                              <strong>
                               <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                Supplementary Fig. 6a
                               </a>
                              </strong>
                              ). The testicular germ cell tumor (TGCT) bleomycin gene expression model did not quite reach significance, though it is worth mentioning that for the edgeR feature selection and L2 logistic regression modeling method it was close (p = 0.077).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F6.large.jpg?width=800&height=600&carousel=1" title="Evaluation of drug response model robustness. Model significance and robustness was further evaluated using a class label permutation test and examination of the effect feature selection had on model performance. Results for the modeling method which had the most significant Wilcoxon signed-rank test are shown. (a, c) Permutation test result histograms and significance for microbial abundance (a) or gene expression (c) models showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. (b, d) Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances for microbial abundance (b) or gene expression (d) models. Shaded areas denote standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Evaluation of drug response model robustness.</span> Model significance and robustness was further evaluated using a class label permutation test and examination of the effect feature selection had on model performance. Results for the modeling method which had the most significant Wilcoxon signed-rank test are shown. <strong>(a, c)</strong> Permutation test result histograms and significance for microbial abundance <strong>(a)</strong> or gene expression <strong>(c)</strong> models showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. <strong>(b, d)</strong> Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances for microbial abundance <strong>(b)</strong> or gene expression <strong>(d)</strong> models. Shaded areas denote standard deviations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F6.medium.gif" width="440" height="355"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F6.medium.gif" width="440" height="355"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Evaluation of drug response model robustness.
                               </span>
                               <p id="p-24" class="first-child">
                                Model significance and robustness was further evaluated using a class label permutation test and examination of the effect feature selection had on model performance. Results for the modeling method which had the most significant Wilcoxon signed-rank test are shown.
                                <strong>
                                 (a, c)
                                </strong>
                                Permutation test result histograms and significance for microbial abundance
                                <strong>
                                 (a)
                                </strong>
                                or gene expression
                                <strong>
                                 (c)
                                </strong>
                                models showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram.
                                <strong>
                                 (b, d)
                                </strong>
                                Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances for microbial abundance
                                <strong>
                                 (b)
                                </strong>
                                or gene expression
                                <strong>
                                 (d)
                                </strong>
                                models. Shaded areas denote standard deviations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              We further evaluated the robustness of our significant drug response models by examining the effect that the number of selected features had on model performance. During the hyperparameter grid search and tuning that occurred in the nested inner CV during each model instance fitting, scores for every combination of hyperparameter setting and inner CV train/validation fold were saved (see Methods for full details). We plotted how these scores were affected by the hyperparameters that controlled feature selection. Our decision to conservatively limit the feature selection search space in our drug response models to a maximum of 400 best scoring features, to reduce model complexity and the possibility of overfitting, appeared sufficient, as scores for our significant models reached a maximum or leveled off well within this search range (
                              <strong>
                               <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                                Fig. 6b
                               </a>
                               & d
                              </strong>
                              ). In the five microbial abundance models, predictive power was driven by a small number of features in three models, where selecting more features did not contribute to additional predictive power or it added noise (
                              <strong>
                               <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                                Fig. 6b
                               </a>
                              </strong>
                              ). Even in the remaining two models, most of the predictive power was driven by the top 50 to 100 features. In the six gene expression models, this finding was even more stark, where all the predictive power was achieved by a small number of features in each model (
                              <strong>
                               <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                                Fig. 6d
                               </a>
                              </strong>
                              ). In all the significant models from both data types, the variance in scores was not significantly affected by the number of selected features and feature-to-sample ratio within our chosen hyperparameter search range. As with the permutation test results, we found the effect that the number of selected features had on model performance was similar regardless of the feature selection or modeling method used (
                              <strong>
                               <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                Supplementary Fig. 5b
                               </a>
                               ,
                               <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                6b
                               </a>
                              </strong>
                              ). In summary, these two comprehensive analyses suggest that the significant cancer-drug response combinations found in this study and the most important features inferred from their models represent a potentially real and robust biological signal.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Feature analysis reveals a wide range of predictive microbial genera
                             </h3>
                             <p id="p-26">
                              To learn more about the most predictive features, we determined the top microbial genera and top genes (
                              <strong>
                               Supplementary Data 2
                              </strong>
                              ) selected by each of the significantly predictive microbial abundance and gene expression models, respectively, according to their selection frequency and model coefficients across the 100 model instances from each analysis. There were 428 distinct microbial genera appearing in at least one prognosis or drug response model. Of these 428 genera, 160 were individually significantly predictive of prognosis or drug response by a Wilcoxon test, indicating that the other genera were significantly predictive in combination. The median number of genera selected per model was 52, with a minimum of 3 (BRCA docetaxel) and a maximum of 78 (STAD cisplatin). Of the 428 genera, 95 were selected in more than one model and only 13 were selected in more than two models. This is consistent with the observation of Nejman et al.
                              <sup>
                               <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                22
                               </a>
                              </sup>
                              that the tumor microbiome is tumor type specific. The predictive genera we found span all non-eukaryotic domains of life, in total encompassing 365 bacterial, 17 archaeal, and 46 viral genera (
                              <strong>
                               Supplementary Data 2
                              </strong>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-27">
                             In summary, we find that the microbial abundance estimates generated by Poore et al.
                             <sup>
                              <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             are predictive of cancer patient prognosis and response to chemotherapy in a subset of tumor types, survival outcomes, and treatments. Machine learning methods, such as those applied in this study, are not able to infer causality, but only inform on the positive or negative predictive associations covariates have with the response variable. The potential causal role that those covariates may play in determining patient prognosis or drug response can only be ascertained via dedicated mechanistic studies. Overall, in terms of the number of significant models, based on their cross-validated C-index or AUROC scores and improvement over clinical covariates alone, the tumor microbiome is considerably less predictive than the tumor human transcriptome at predicting patient prognosis, but notably, performs similarly to gene expression at predicting chemotherapy response and in mostly different cancer-drug combinations. Our investigation motivates future studies investigating the role of the tumor microbiome in predicting the response to targeted therapies and immunotherapies.
                            </p>
                            <p id="p-28">
                             There are also some limitations to our current study. As we described previously, some TCGA drug response cohorts were of limited size or had relatively few responder or non-responder cases within these cohorts and this could have an impact on the interpretability of the results. Vabalas et al.
                             <sup>
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                             </sup>
                             conducted a literature review of ML algorithm validation of high-dimensional biological data models with limited sample size and performed their own independent simulation analyses evaluating different techniques. They found that, consistent with previous literature, nested CV was the optimal validation method and gives unbiased performance estimates regardless of sample size. They also found that performing feature selection and other model development steps (e.g., normalization, outlier removal) fully within the inner nested CV is essential to avoid overfitting and to produce unbiased results, and that hyperparameter tuning should ideally also be performed in nested fashion. Finally, they found that performing an adequate number of CV folds was important to reduce bias. Our analyses have followed their observations and recommendations, employing them at every level of model development and evaluation, including additional techniques not reviewed in their work (see Methods for full details).
                            </p>
                            <p id="p-29">
                             There are further limitations to this study inherited from limitations in the data, originally raised by Poore et al.
                             <sup>
                              <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             First, the study was retrospective, using existing data from the TCGA. As such, it did not involve any specific protocols to capture microbial reads or to control for contamination. Second, decontamination of such retrospective data is a highly involved and dataset-specific process, which they made great effort to validate. Poore et al. conclude from this validation that the retrospective study of TCGA was successful, and that similar retrospective studies would be valuable. A third point, which they touch on briefly, is that the protocols that were used have limitations with respect to capturing microbial reads and cannot distinguish if the source of microbial reads is intracellular or extracellular, or alive or dead when the sample was taken. Poore et al. suggest, correctly we believe, that additional protocols need to be developed for prospective studies.
                            </p>
                            <p id="p-30">
                             Accepting the limitations of the study, we observed certain trends. Proteobacteria and Firmicutes were the most frequent phyla identified as predictive features (
                             <strong>
                              Supplementary Data 2
                             </strong>
                             ), followed by Actinobacteria and Bacteroidetes. Among viruses, Herpesvirales were the most frequent. More microbial genera were negatively predictive of drug response or prognosis than those positively predictive (negative for 306/537 features; two-sided binomial test p-value = 0.0014). Firmicutes reversed this trend, being more often positively predictive (positive for 49/82 features, two-sided Fisher’s exact test p-value = 0. 0.0036;
                             <strong>
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Supplementary Table 2
                              </a>
                             </strong>
                             ).
                            </p>
                            <p id="p-31">
                             Further examining the predictive features of our significant models and their cancer types, we found that several genera of Firmicutes were predictive of OS in CESC, including genera of Lactobacillales were found to be negatively predictive of survival. We also found that the genus Chlamydia had an even stronger negatively predictive association with OS in CESC. Notably, though CESC is known to often arise from HPV infection, the presence of other microbial species, in particular the genera Chlamydia and Lactobacillales, have been reported in the literature to be associated with the risk of developing CESC
                             <sup>
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ,
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-32">
                             Our prognosis analysis results were different than two recent reports
                             <sup>
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ,
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                             </sup>
                             which found some intratumor microbes that were potentially correlated with prognosis in three TCGA cancers. We did not find that the Poore et al. tumor microbial abundances estimated from TCGA were predictive of OS or PFI in these three cancers using our data-driven, regularized ML computational approach. A few important possible reasons for this difference in results are that different source data and methods were used to perform prognosis analysis compared to these studies. Gnanasekar et al.
                             <sup>
                              <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                             </sup>
                             analyzed the THCA cohort by tumor subtype, they used harmonized and normalized GDC TCGA data instead of legacy TCGA followed by normalization and batch effect correction as in Poore et al., they only used RNA-seq data instead of WGS and RNA-seq data, they applied different methods for extraction of microbial reads and decontamination, and finally they did not perform any direct analysis of correlation of their derived microbial abundances with survival outcomes. Dohlman et al.
                             <sup>
                              <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                             </sup>
                             analyzed colorectal cancers (colon (COAD) and rectum (READ) adenocarcinomas) also using harmonized and normalized GDC TCGA data, they used WGS and whole exome sequencing (WXS) data instead of WGS and RNA-seq, they also used different methods for extraction and decontamination of microbial reads, and finally they also applied classical univariate statistics on their entire data to infer correlation with overall survival (OS). While we believe the use of harmonized GDC TCGA data is superior to legacy TCGA, Poore et al. applied robust computational methods to remove technical variation from legacy TCGA data and validated that their approach was effective. We also applied additional filters of TCGA samples to further remove technical variation. We also believe that, in general, applying classical univariate statistics on the entire data to find correlations has the potential to overfit the specific dataset and it does not consider the multivariate nature of high-dimensional biological data like intratumor microbial abundances. A data-centric, multivariate, and regularized ML approach focused on fitting models on training data and evaluating on unseen test data has potential to generalize better and discover whether features are potentially predictive of and correlated with the response variable, such as survival outcomes or drug response.
                            </p>
                            <p id="p-33">
                             Looking at our drug response model results, in STAD, tumor microbial abundances were predictive of response to three different drugs: cisplatin, leucovorin, and oxaliplatin. The genus Helicobacter was a quantified microbial abundance feature in the Poore et al. dataset although notably, even though it is well established that patients infected with H. pylori have an increased risk of developing gastric cancer
                             <sup>
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                             </sup>
                             , Helicobacter was not identified as a predictive feature of drug response in our STAD models. This finding is in line with recent research indicating reduced microbial diversity, decreased abundance of H. pylori, and enrichment of other mostly commensal bacterial genera in gastric carcinoma
                             <sup>
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                             </sup>
                             . Instead, in STAD we found that known opportunistic bacteria Cedecea and Sphingobacterium were both strongly negatively predictive of leucovorin response, Sphingobacterium was strongly negatively predictive of cisplatin response, and the opportunistic bacteria Rouxiella was strongly negative predictive of oxaliplatin response. Cedecea and Sphingobacterium have been implicated in bacteremia in immunocompromised individuals in rare cases, including cancer
                             <sup>
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              –
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                             </sup>
                             . As dysbiosis is frequent in stomach cancer
                             <sup>
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ,
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                             </sup>
                             , and considering the mechanism of action of leucovorin, it may be of interest to study whether organisms from these two genera may sequester or prevent the bacterial production of folinic acid
                             <sup>
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-34">
                             We found three microbial genera whose abundances were strongly associated with breast cancer response to doxetaxel. Indeed, the involvement of the microbiome in breast cancer (BRCA)
                             <sup>
                              <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ,
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                             </sup>
                             has recently received considerable attention. In BRCA, we found that the genus containing Epstein-Barr virus (EBV) was negatively associated with response to docetaxel, which is concordant with previous findings that EBV is associated with chemoresistance to docetaxel in gastric cancer
                             <sup>
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                             </sup>
                             . Interestingly, Cyanobacteria were predictive features in several cancers in our study and we identified a genus of Cyanobacteria as predictive of response to docetaxel in BRCA. Notably, the presence of Cyanobacteria in BRCA was recently confirmed by Nejman et al.
                             <sup>
                              <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                             </sup>
                             by 16S-rRNA sequencing. While the genus we identified, Raphidiopsis, a planktonic Cyanobacteria that produces toxins harmful to human health and found in freshwater, is possibly a taxonomic identification error in the original microbial abundance estimates, our findings may point to a related genus under the recently discovered clade Melainabacteria of Cyanobacteria
                             <sup>
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                             </sup>
                             , which is present in humans. Though Melainabacteria are difficult to culture, we believe that confirmation of the relationship between BRCA to response to docetaxel and Melainabacteria should be tested, and a first step would be to confirm our computationally derived findings in a dedicated 16S-rRNA analysis.
                            </p>
                            <p id="p-35">
                             Interestingly, in sarcoma (SARC), among the most predictive microbial features we found the genus Lactococcus to be positively associated with response to docetaxel. Lactococcus contains species that can sometimes cause opportunistic infections in humans, as Lactococcus are similar to Streptococcus and formerly belonged to that genus. The result that this genus was positively associated with response in our model initially appeared counterintuitive, although while the use of therapeutic bacteria as antitumor agents has not been an extensively studied field, there have been some limited findings in the literature that suggest the use of bacteriotherapy as anticancer agents
                             <sup>
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                             </sup>
                             . Historically, the intentional use of the toxins of various Streptococcus species showing significant antitumor activity in SARC has been documented
                             <sup>
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              –
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                             </sup>
                             . One possible testable explanation for some microbes being strongly positive predictive of docetaxel response in our model is that they might produce some extracellular products or toxins that could work as an adjuvant to the chemotherapy.
                            </p>
                            <p id="p-36">
                             In summary, while these findings and others reported in this study are computationally derived associations, we believe that they can serve as leads for further experimental studies of the role of microbial species in modulating patients survival and drug response, potentially by metabolizing drug levels in the tumor microenvironment as suggested above, or by altering the immune response, either by changing the levels of specific immunometabolites or by having the tumors present specific bacterial antigens
                             <sup>
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                             </sup>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Data retrieval and processing
                             </h3>
                             <p id="p-37">
                              Normalized and batch effect corrected microbial abundance data for 32 TCGA tumor types were downloaded from the online data repository referenced in Poore et al.
                              <sup>
                               <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">
                                10
                               </a>
                              </sup>
                              (
                              <a href="ftp://ftp.microbio.me/pub/cancer_microbiome_analysis">
                               ftp://ftp.microbio.me/pub/cancer_microbiome_analysis
                              </a>
                              ). Specifically, the “Kraken-TCGA-Voom-SNM-Plate-Center-Filtering-Data.csv” microbial abundance data file and adjoining “Metadata-TCGA-Kraken-17625-Samples.csv” metadata file were used as the starting input for further data processing.
                             </p>
                             <p id="p-38">
                              We first filtered the data for primary tumor samples (TCGA “Primary Tumor” or “Additional - New Primary” sample types). Poore et al. generated microbial abundances from all the available WGS and RNA-seq data in legacy TCGA (after some quality filters), which frequently contained replicate WGS and RNA-seq data for the same case and sample type. It was common in legacy TCGA to increase WGS sequencing coverage by performing an additional sequencing run from the same sample and these secondary runs typically had a much lower number of reads and coverage compared to their corresponding primary sequencing runs. When comparing the normalized and batch effect corrected read counts between these WGS runs, we found that microbial abundance data which came from lower coverage secondary runs could be substantially different from abundances derived from the larger primary sequencing runs. Therefore, we excluded microbial abundance data which came from secondary runs. In addition, legacy TCGA commonly contained data for the same samples analyzed using different computational pipeline versions. We excluded replicate microbial abundance data from older TCGA analysis pipeline versions if a replicate from a newer version existed. After the above filters, the Poore et al. data went from 17,625 samples and 10,183 unique cases to 12,111 samples and 9,812 unique cases (comprising of 1,944 WGS samples from 1,904 unique cases and 10,167 RNA-seq samples from 9,745 unique cases).
                             </p>
                             <p id="p-39">
                              TCGA gender, age at diagnosis, and tumor stage demographic and clinical data and as well as primary tumor RNA-seq read count data for the 32 TCGA tumor types included in our study were obtained from the NCI Genomic Data Commons (GDC Data Release v29.0) using the R Bioconductor package GenomicDataCommons. TCGA GENCODE v22 gene annotations were obtained from the GDC data portal and Ensembl Gene v98 using the R package rtracklayer and R Bioconductor packages AnnotationHub and ensembldb. The downloaded GDC primary tumor cohort with RNA-seq read count data comprised of 9,735 samples from 9,680 unique cases. There were 68 cases at the GDC which had missing age of diagnosis but existing values in the Poore et al. data and we chose not to exclude these data and used the Poore et al. age of diagnosis values for these cases. TCGA curated survival phenotypic data
                              <sup>
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                              </sup>
                              were obtained from UCSC Xena. Cases which had both missing overall survival (OS) and progressive-free interval (PFI) outcome data were excluded from survival modeling.
                             </p>
                             <p id="p-40">
                              TCGA curated drug response clinical data were compiled from Ding et al.
                              <sup>
                               <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                                14
                               </a>
                              </sup>
                              Our drug response models used the following binary classification targets: complete response (CR) and partial response (PR) were labeled as responders and stable disease (SD) and progressive disease (PD) as non-responders. All TCGA samples with drug response phenotypic data were from pre- treatment biopsies. Due to the limited cancer-drug combination cohort sizes in TCGA, we modeled each drug individually, even if a patient received multiple drugs concurrently. If the same drug was given at multiple timepoints to a patient, we only considered their first drug response. We considered cancer-drug combinations that contained a minimum of 18 cases and at least 4 cases per response binary class, except for STAD oxaliplatin, where we allowed a minimum of 14 cases so that the gene expression dataset could be included. In total, we analyzed 30 cancer-drug combinations which had paired microbial abundance and gene expression data that met the above thresholds. Combined feature microbial abundance and gene expression datasets were created by joining data from each individual dataset which had matching TCGA sample UUIDs. For some TCGA cases, data existed from multiple different aliquots per sample or multiple technical runs per aliquot, therefore in these cases all combinations were joined at the sample UUID level. Cross-validation sampling probability weights as well as model and scoring sample weights were applied to account and adjust for any imbalance caused by the process.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              ML modeling
                             </h3>
                             <p id="p-41">
                              Machine learning (ML) models were built using the scikit-learn
                              <sup>
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                46
                               </a>
                              </sup>
                              and scikit-survival libraries
                              <sup>
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                               –
                               <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                49
                               </a>
                              </sup>
                              . Custom extensions to scikit-learn and scikit-survival were developed to add new methods and functionalities required by this project. Survival models were built using Coxnet – regularized Cox regression with elastic net penalties
                              <sup>
                               <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              . Coxnet models controlled for gender, age at diagnosis, and tumor stage clinical prognostic covariates by including them as unpenalized features in the model (Coxnet penalty factor = 0). Drug response classification models were built using three different ML methods: 1) a variant of the linear support vector machine recursive feature elimination (SVM-RFE) algorithm
                              <sup>
                               <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                                15
                               </a>
                              </sup>
                              that we developed with a number of additional features and better performance than the scikit-learn built-in version, 2) logistic regression (LGR) with elastic net
                              <sup>
                               <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              (L1 + L2) penalties and embedded feature selection, and 3) LGR with an L2 penalty and limma
                              <sup>
                               <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                                17
                               </a>
                              </sup>
                              (for tumor microbial and combination datasets) or edgeR
                              <sup>
                               <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">
                                18
                               </a>
                               ,
                               <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                                19
                               </a>
                              </sup>
                              (for RNA-seq count datasets) differential abundance/expression feature scoring inside a k-best wrapper feature selection method around the learning algorithm. Limma differential abundance analysis was run inside the ML pipeline with default parameters except for fitting an intensity-dependent trend to the prior variances and running a robust empirical Bayes procedure (eBayes function parameters trend = TRUE and robust = TRUE). edgeR differential expression analysis was run inside the ML pipeline with default parameters except for enabling robust estimation of the negative binomial dispersion (calcDispersions function robust = TRUE) and robust estimation of the prior quasi-likelihood (QL) dispersion (glmQLFit function robust = TRUE). Both limma and edgeR methods scored and ranked features by differential abundance/expression p-value.
                             </p>
                             <p id="p-42">
                              All three drug response ML methods unconditionally included the same three clinical covariates in the model as in the prognosis models by having them bypass feature selection in the ML pipeline, though in drug response models, clinical covariates were modeled as L2 penalized features. In SVM-RFE, clinical covariate features bypassed recursive feature elimination but were always included at each RFE recursive feature elimination model fitting step as well as final model refitting. To the best of our knowledge, no available comprehensive ML library in python or R currently provides an elastic net LGR algorithm with the functionality to specify features that can bypass embedded feature selection and be modeled with an L2 penalty (setting the R glmnet penalty factor, for example, does not provide this functionality as it is not a penalty factor per regularization term but a factor applied to the sum of both L1 and L2 terms). In order to develop this functionality for our study, our elastic net LGR model pipeline was designed as a two-level LGR, 1) an elastic net LGR and embedded feature selection on only microbial abundance or gene expression features with clinical covariates bypassing this step, followed by 2) an L2 penalized LGR on features selected by the elastic net LGR step and the clinical covariates. We know this design does not likely produce the exact same model settings and results of a single-level elastic net LGR algorithm with the functionality we needed, if such an implementation it existed, though we tested every drug response model through an ML pipeline with elastic net LGR and no clinical feature selection bypass and found that model predictive performance, feature coefficients and signs, and feature importance rankings were similar to our two-level ML pipeline setup.
                             </p>
                             <p id="p-43">
                              Gender was one-hot encoded and tumor stage ordinal encoded by major stage. In the final cohort included in our prognosis and drug response models, 3363 out of 9708 tumor microbial abundance cases (34.64%) and 3244 out of 9484 gene expression cases (34.21%) had tumor stage “not reported” or BRCA stage “X”. Since missing tumor stage metadata is so prevalent in TCGA, we took the approach of including these in our study and modeled missing tumor stage with as neutral an ordinal encoding as possible. Looking at the distribution of reported major tumor stages in our cohort, we determined that encoding missing data as an ordinal between tumor stage II and III was as close to the middle of the distribution of stages in TCGA as we could possibly achieve with ordinal encoding.
                             </p>
                             <p id="p-44">
                              All prognosis and drug response models included the previously described feature selection as well as normalization and transformation steps integrated into the ML modeling pipeline using an extended version of the scikit-learn Pipeline framework. Each cancer, data type, and survival or drug response target type combination was modeled individually using a nested cross-validation (CV) strategy to perform model selection and evaluation on held-out test data. Training data splits always underwent feature selection, normalization, and transformation through the ML pipeline independently from held-out test or validation data splits before learning. Models built using gene expression read count data included edgeR low count filtering, weighted trimmed mean of M-values (TMM) normalization, and log counts per million (CPM) transformation steps within the ML pipeline. These were developed and integrated into our scikit-learn-based framework via R and rpy2. All models also included standardization of features within the ML pipeline just before learning. During prediction, held-out test or validation data were feature selected, normalized, and transformed through the ML pipeline using the parameters learned from the training data at each pipeline step before model prediction and scoring. Hyperparameter search and optimization of all model pipeline steps was performed in nested fashion within the inner nested CV. All cross-validation iterators kept replicate sample data per case grouped together such that data would only reside in either the train or test split during each CV iteration.
                             </p>
                             <p id="p-45">
                              Survival models used a stratified and randomly shuffled outer CV with 75% train and 25% test split sizes that was repeated 100 times. The CV procedure stratified the splits on event status. Each training set from the outer CV was used to perform hyperparameter tuning and model selection by optimizing Harrell’s concordance index (C-index) over a stratified, randomly shuffled, 4-fold inner CV on the training set repeated 5 times. A few cancer datasets contained fewer than four uncensored cases which required reducing the number of inner CV folds for these models such that at least one case per fold was uncensored. The data derived from Poore et al. often included more than one sample per case, and an unequal number of samples between cases, therefore requiring either ML model sample weighting or CV random sampling per case. The Coxnet implementation in scikit-survival does not currently support sample weights, therefore our custom outer CV iterator randomly sampled one replicate sample per case during each iteration, using a sampling procedure with probability weights that balanced the probability that a replicate WGS- or RNA-seq-based sample was selected during each CV iteration. Model selection grid search was performed on the following hyperparameters: elastic net penalty L1 ratios 0.1, 0.3, 0.5, 0.7, 0.8, 0.9, 0.95, 0.99, and 1, and for each L1 ratio a default alpha path of 100 alphas using an alpha min ratio of 10
                              <sup>
                               −2
                              </sup>
                              . Alpha is the constant multiplier of the penalty terms in the Coxnet objective function. Optimal alpha and L1 ratio settings were determined via inner CV and a model with these settings was then refit on the entire outer CV train data split. Model performance was evaluated in both inner and outer CV on each held-out validation or test data split, respectively, by generating model test predicted risk scores and using these scores to directly calculate C-index scores. We also evaluated and compared model predictive performance for each test data split survival time period by calculating time-dependent cumulative/dynamic AUCs
                              <sup>
                               <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                               ,
                               <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                                13
                               </a>
                              </sup>
                              .
                             </p>
                             <p id="p-46">
                              Drug response models used a stratified, randomly shuffled, 4-fold outer CV that was repeated 25 times (i.e., 100 model instances). Each training set from the outer CV was used to perform hyperparameter tuning and model selection by optimizing the area under the receiver-operating characteristic curve (AUROC) over a stratified, randomly shuffled, 3-fold inner CV repeated 5 times. Case replicate sample weights were provided to SVM-RFE and LGR learning algorithms and all model selection and evaluation scoring methods. Class weights were provided to SVM-RFE and LGR learning algorithms to adjust for any class imbalance. Model selection grid search was performed on the following hyperparameters: L2 penalized SVM and LGR C regularization parameter from a range of 10
                              <sup>
                               −5
                              </sup>
                              to 10
                              <sup>
                               3
                              </sup>
                              , elastic net LGR L1 ratios of 0.1, 0.3, 0.5, 0.7, 0.8, 0.9, 0.95, 0.99 and 1, elastic net LGR C regularization parameter from a range of 10
                              <sup>
                               -2
                              </sup>
                              to 10
                              <sup>
                               3
                              </sup>
                              (microbial abundance) or from 10
                              <sup>
                               -2
                              </sup>
                              to 10
                              <sup>
                               1
                              </sup>
                              (gene expression and combined data type), and finally RFE, elastic net LGR, and limma and edgeR feature scorer k-best feature selection search range from 1 to 400 top scoring microbial abundance, gene expression, or combined data type features. SVM-RFE models performed a feature elimination procedure of the one worst feature per recursive step for microbial abundance models (which started with 1287 features in the Poore et al. data) and 5% of worst remaining features per recursive step until 1300 features were reached followed by the one worst feature per recursive step for gene expression (starting with 60,483 features in GENCODE v22) and combined data type models (starting with 61,770 features). Optimized hyperparameter settings were determined via inner CV and a model with the optimized settings was then refit on the entire outer CV train data split. Model performance was evaluated in both inner and outer CV on each held-out validation or test data split, respectively, by AUROC, average precision (AVPRE) or area under precision-recall curve (AUPRC), and balanced accuracy (BCR). AUROC was used to evaluate and select the best model and optimized hyperparameter settings from the grid search.
                             </p>
                             <p id="p-47">
                              Gender, age at diagnosis, and tumor stage clinical covariate-only survival models were built using standard unpenalized Cox regression. Clinical covariate-only drug response models were built using L2 penalized linear SVM or LGR. Models included standardization of features as part of the ML pipeline. Models were trained and tested using the same outer CV iterators and train/test data splits as their corresponding microbial abundance, gene expression, or combination data type models. To test whether a Coxnet, SVM-RFE, or LGR microbial abundance or gene expression model was significantly better than their corresponding Cox, linear SVM, or LGR clinical covariate-only model, respectively, a two-sided Wilcoxon signed-rank test was performed between the 100 pairs of C-index or AUROC scores between both models. All raw p-values generated from the signed-rank test across survival or drug response analyses from the same data type were adjusted for multiple testing using the Benjamini-Hochberg (BH) procedure to control the false discovery rate (FDR), and a threshold FDR ≤ 0.01 was used to determine statistical significance. To test whether a combined data type model was significantly better than its corresponding microbial abundance or gene expression model, a two-sided Dunn test was performed between all three groups of data type model scores. Each Dunn test raw p-value was adjusted for multiple testing using the Benjamini-Hochberg (BH) procedure to control the false discovery rate (FDR), and a threshold FDR ≤ 0.05 was used to determine statistical significance.
                             </p>
                             <p id="p-48">
                              Permutation tests were performed by shuffling dataset class labels 1000 times and each time running the outer CV procedure on the permuted dataset, where for each CV iteration we fit a model instance and calculated an AUROC score, totaling 100,000 fits and scores for each model. Permutation mean AUROC scores were compared to the true mean AUROC score for the model and a p-value was calculated from the fraction of permutation mean scores that were greater than or equal to the true mean score. A p-value ≤ 0.05 was used to determine statistical significance. The Freedman-Draconis rule was used in permutation test histogram plots to compute the bin width. Analysis of the effect of number of selected features on model performance was performed via the hyperparameter grid search and tuning that occurred in the nested inner CV during each model instance fitting, where scores for every combination of hyperparameter setting and inner CV train/validation fold were saved for all model instances and used for plotting.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Microbial abundance model feature analysis
                             </h3>
                             <p id="p-49">
                              For each analysis, 100 prognosis or drug response model instances were generated from the outer CV procedure. Each model instance selected a subset of features that performed best during CV and the model algorithm learned coefficients (or weights) for each feature. To select microbial genera for downstream investigation from the feature results across all these model instances, we proceeded as follows. First, we applied a two-sided Wilcoxon signed-rank test that the mean feature coefficient rank generated by the model is shifted away from zero, and thus that the genus is identifiably positively or negatively associated with survival or drug response. For all Wilcoxon tests, we used the package coin
                              <sup>
                               <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                50
                               </a>
                              </sup>
                              , which allows exact calculation of p-values. Coefficients were ignored when a genus was assigned a zero coefficient or absent from a model. Second, within each model, all coefficients, ignoring the results of the Wilcoxon test, were ranked by absolute magnitude. We then kept genera that were among the top 50 features in at least 20% of the models and for which the Holm-adjusted, two-sided Wilcoxon signed-rank test p-value was ≤ 0.01. Having a Coxnet feature coefficient equal to zero or feature being absent from an SVM-RFE or LGR model was not strong enough evidence that the genus has no effect, but rather that one or more features with stronger effect were chosen. Thus, we ignored genera with a zero coefficient or absent from a model when computing mean coefficient weight and Wilcoxon statistics on the means.
                             </p>
                             <p id="p-50">
                              For the drug response models, where three ML methods were tested, we noted the features selected by individual models and the median rank the feature attained in the instances in which it appeared, but further filtered the features to account for the consensus between ML models. We kept features selected in any two ML model methods that individually met our criteria for inclusion, ignoring features in ML models that did not meet these criteria. We then computed the Spearman correlation between the median ranks attained by the features.
                             </p>
                             <p id="p-51">
                              For each selected microbial feature, we tested whether it was a significantly univariate feature of survival or drug response. This is a strictly different question than whether the coefficient of a feature has consistent sign – sign may be consistent when used in combination with other features, but the feature may not be individually predictive. For drug response models, we divided individuals into responders and non-responders, and for survival data we divided individuals whose survival time was greater or less than the censored median, ignoring those who were lost to follow up before median time. For each cancer-test type pair, we applied a two-sided Wilcoxon rank-sum test. We applied a Benjamini-Hochberg multiple hypothesis correction for each cancer-test type pair and report the false discovery rate in
                              <strong>
                               Supplementary Data 2
                              </strong>
                              .
                             </p>
                             <p id="p-52">
                              We analyzed the distribution of features, selected by the rules described above, that had positive or negative signs for their mean coefficient. We used a two-sided binomial test to show that selected features had significantly more negative the positive mean coefficients. We used a two-sided Fisher’s exact test to determine if selected genera belonging to Firmicutes had a statistically significant difference in the breakdown between positive and negative mean coefficients than selected features as a whole.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Data availability
                            </h2>
                            <p id="p-53">
                             All results generated from this work are available under
                             <a href="https://doi.org/10.5281/zenodo.5221525">
                              https://doi.org/10.5281/zenodo.5221525
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-14">
                            <h2 class="">
                             Code availability
                            </h2>
                            <p id="p-54">
                             All code and data used to produce this work are available under
                             <a href="https://github.com/ruppinlab/tcga-microbiome-prediction">
                              https://github.com/ruppinlab/tcga-microbiome-prediction
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-16">
                            <h2 class="">
                             Author information
                            </h2>
                            <div class="section" id="sec-17">
                             <h3>
                              Contributions
                             </h3>
                             <p id="p-56">
                              L.C.H., E.M.G., and E.R. designed the study. L.C.H. and E.M.G. performed all computational analyses and results interpretation. L.C.H., E.M.G., and E.R. wrote the paper.
                             </p>
                            </div>
                            <div class="section" id="sec-18">
                             <h3>
                              Corresponding authors
                             </h3>
                             <p id="p-57">
                              Correspondence to Eytan Ruppin.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             Ethics declaration
                            </h2>
                            <div class="section" id="sec-20">
                             <h3>
                              Competing interests
                             </h3>
                             <p id="p-58">
                              All authors declare that they have no competing interests.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Supplementary Tables
                            </h2>
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2349474/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup last">
                                 <a href="/highwire/markup/2349474/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 1.
                              </span>
                              <p id="p-59" class="first-child">
                               Prognosis model results comparison to Milanez-Almeida et al.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T2" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2349480/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2349480/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2349480" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplementary Table 2.
                              </span>
                              <p id="p-60" class="first-child">
                               Per cancer, the number of times genera from the phylum Firmicutes were found among the selected features, whether positively or negatively associated with drug response or prognosis.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-22">
                            <h2 class="">
                             Supplementary Figures
                            </h2>
                            <div id="F7" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F7.large.jpg?width=800&height=600&carousel=1" title="Performance of gene expression overall survival (OS) models in the 16 tumor types where gene expression adds predictive power to clinical covariates. (a) C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. (b) Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range after diagnosis shown as a horizontal dotted line and in legends and shaded areas denote standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of gene expression overall survival (OS) models in the 16 tumor types where gene expression adds predictive power to clinical covariates.</span> <strong>(a)</strong> C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. <strong>(b)</strong> Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range after diagnosis shown as a horizontal dotted line and in legends and shaded areas denote standard deviations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F7.medium.gif" width="408" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F7.medium.gif" width="408" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 1.
                              </span>
                              <span class="caption-title">
                               Performance of gene expression overall survival (OS) models in the 16 tumor types where gene expression adds predictive power to clinical covariates.
                              </span>
                              <p id="p-61" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top.
                               <strong>
                                (b)
                               </strong>
                               Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range after diagnosis shown as a horizontal dotted line and in legends and shaded areas denote standard deviations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F8" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F8.large.jpg?width=800&height=600&carousel=1" title="Performance of gene expression progression-free interval (PFI) models in the 17 tumor types where gene expression adds predictive power to clinical covariates. (a) C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. (b) Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of gene expression progression-free interval (PFI) models in the 17 tumor types where gene expression adds predictive power to clinical covariates.</span> <strong>(a)</strong> C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top. <strong>(b)</strong> Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F8.medium.gif" width="407" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F8.medium.gif" width="407" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 2.
                              </span>
                              <span class="caption-title">
                               Performance of gene expression progression-free interval (PFI) models in the 17 tumor types where gene expression adds predictive power to clinical covariates.
                              </span>
                              <p id="p-62" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               C-index score density distributions for gene expression with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. Significance for the prediction improvement over clinical covariate-only models was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top.
                               <strong>
                                (b)
                               </strong>
                               Time-dependent, cumulative/dynamic AUCs for gene expression with clinical covariate models (orange) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F9" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F9.large.jpg?width=800&height=600&carousel=1" title="Performance of microbial abundance prognosis models in the five tumor types where microbial abundance features add predictive power to clinical covariates. (a) C-index score density distributions for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. (b) Time-dependent, cumulative/dynamic AUCs for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote standard deviations. Significance was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in (a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of microbial abundance prognosis models in the five tumor types where microbial abundance features add predictive power to clinical covariates.</span> <strong>(a)</strong> C-index score density distributions for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red. <strong>(b)</strong> Time-dependent, cumulative/dynamic AUCs for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote standard deviations. Significance was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in <strong>(a)</strong>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F9.medium.gif" width="440" height="182"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F9.medium.gif" width="440" height="182"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 3.
                              </span>
                              <span class="caption-title">
                               Performance of microbial abundance prognosis models in the five tumor types where microbial abundance features add predictive power to clinical covariates.
                              </span>
                              <p id="p-63" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               C-index score density distributions for microbial abundance with clinical covariate models vs clinical covariate-only models. Lines connecting points (light grey) represent score pairs from same train-test split on the data. Mean C-index scores and connecting lines shown in red.
                               <strong>
                                (b)
                               </strong>
                               Time-dependent, cumulative/dynamic AUCs for microbial abundance with clinical covariate models (blue) vs clinical covariate-only models (grey) following years after diagnosis. Mean AUCs across entire test time range shown as a horizontal dotted line and in legends and shaded areas denote standard deviations. Significance was calculated using a two-sided Wilcoxon signed-rank test and adjusted for multiple testing using the Benjamini-Hochberg method with adjusted p-values shown at top of violin plots in
                               <strong>
                                (a)
                               </strong>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F10.large.jpg?width=800&height=600&carousel=1" title="Performance of combined microbial abundance and gene expression models where combining both data types adds to predictive power. (a) Prognosis model C-index score violin density distributions (left) between microbial abundance, gene expression, and combined data type models. Mean scores shown in red. Significance was calculated using a two-sided Dunn test and p-values were adjusted for multiple testing using the Benjamini-Hochberg method. Model C-index score violin density distributions (middle) for combined data type with clinical covariate models vs clinical covariate-only models. Time-dependent, cumulative/dynamic AUCs (right) for combined data type (purple), microbial abundance (blue), and gene expression (orange) models following years after diagnosis. (b) ML method drug response model mean AUROC scores where combining data types performed better than clinical covariates alone. For the combined data type drug response model closest to reach significant improvement over respective single data type models (c) AUROC score violin density distributions (left) comparing each data type model, AUROC score violin density distributions (middle left) comparing combined data type and clinical covariate model to clinical covariate-only model, and mean ROC and PR curves (right) for each data type model. Mean AUROC and AUPRC scores shown in panel legends and shaded areas denote standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of combined microbial abundance and gene expression models where combining both data types adds to predictive power.</span> <strong>(a)</strong> Prognosis model C-index score violin density distributions (left) between microbial abundance, gene expression, and combined data type models. Mean scores shown in red. Significance was calculated using a two-sided Dunn test and p-values were adjusted for multiple testing using the Benjamini-Hochberg method. Model C-index score violin density distributions (middle) for combined data type with clinical covariate models vs clinical covariate-only models. Time-dependent, cumulative/dynamic AUCs (right) for combined data type (purple), microbial abundance (blue), and gene expression (orange) models following years after diagnosis. <strong>(b)</strong> ML method drug response model mean AUROC scores where combining data types performed better than clinical covariates alone. For the combined data type drug response model closest to reach significant improvement over respective single data type models <strong>(c)</strong> AUROC score violin density distributions (left) comparing each data type model, AUROC score violin density distributions (middle left) comparing combined data type and clinical covariate model to clinical covariate-only model, and mean ROC and PR curves (right) for each data type model. Mean AUROC and AUPRC scores shown in panel legends and shaded areas denote standard deviations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F10.medium.gif" width="332" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F10.medium.gif" width="332" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 4.
                              </span>
                              <span class="caption-title">
                               Performance of combined microbial abundance and gene expression models where combining both data types adds to predictive power.
                              </span>
                              <p id="p-64" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Prognosis model C-index score violin density distributions (left) between microbial abundance, gene expression, and combined data type models. Mean scores shown in red. Significance was calculated using a two-sided Dunn test and p-values were adjusted for multiple testing using the Benjamini-Hochberg method. Model C-index score violin density distributions (middle) for combined data type with clinical covariate models vs clinical covariate-only models. Time-dependent, cumulative/dynamic AUCs (right) for combined data type (purple), microbial abundance (blue), and gene expression (orange) models following years after diagnosis.
                               <strong>
                                (b)
                               </strong>
                               ML method drug response model mean AUROC scores where combining data types performed better than clinical covariates alone. For the combined data type drug response model closest to reach significant improvement over respective single data type models
                               <strong>
                                (c)
                               </strong>
                               AUROC score violin density distributions (left) comparing each data type model, AUROC score violin density distributions (middle left) comparing combined data type and clinical covariate model to clinical covariate-only model, and mean ROC and PR curves (right) for each data type model. Mean AUROC and AUPRC scores shown in panel legends and shaded areas denote standard deviations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F11.large.jpg?width=800&height=600&carousel=1" title="Evaluation of microbial abundance drug response model robustness. (a) Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. (b) Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Evaluation of microbial abundance drug response model robustness.</span> <strong>(a)</strong> Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. <strong>(b)</strong> Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F11.medium.gif" width="337" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F11.medium.gif" width="337" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 5.
                              </span>
                              <span class="caption-title">
                               Evaluation of microbial abundance drug response model robustness.
                              </span>
                              <p id="p-65" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram.
                               <strong>
                                (b)
                               </strong>
                               Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F12.large.jpg?width=800&height=600&carousel=1" title="Evaluation of gene expression drug response model robustness. (a) Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. (b) Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-64894631" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Evaluation of gene expression drug response model robustness.</span> <strong>(a)</strong> Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram. <strong>(b)</strong> Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplementary Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F12.medium.gif" width="372" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplementary Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F12.medium.gif" width="372" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplementary Figure 6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/01/11/2020.07.21.214148/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplementary Figure 6.
                              </span>
                              <span class="caption-title">
                               Evaluation of gene expression drug response model robustness.
                              </span>
                              <p id="p-66" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Class label permutation test result histograms and significance showing the distribution of permutation mean AUROC scores. True mean AUROC score shown as dotted vertical grey line and kernel density estimate shown as a curve over the histogram.
                               <strong>
                                (b)
                               </strong>
                               Curves showing the effect that model hyperparameters which control the number of selected features had on mean AUROC and average precision (AVPRE) scores during hyperparameter grid search across all 100 model instances. Shaded areas denote standard deviations
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgements
                            </h2>
                            <p id="p-55">
                             The results shown here are in whole or part based upon data generated by the TCGA Research Network (
                             <a href="https://www.cancer.gov/tcga">
                              https://www.cancer.gov/tcga
                             </a>
                             ). This research was supported by the Intramural Research Program of the National Institutes of Health, National Cancer Institute and by NIH grant 1ZIABC011803-03. The authors would like to personally thank Christopher Buck from the NCI, Pedro Milanez-Almeida and John Tsang from NIH NIAID, and Alejandro Schäffer, Welles Robinson, Fiorella Schischlik, Sanju Sinha, and Sanna Madan from NCI CDSL for their assistance in this project. This study utilized the high-performance computational capabilities of the HPC Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD (
                             <a href="https://hpc.nih.gov">
                              https://hpc.nih.gov
                             </a>
                             ). The authors would like to thank Richard Lehr, Tim Miller, Wolfgang Resch, and Steve Fellini for their assistance in running this analysis on the NIH HPC Biowulf cluster. The authors would also like to thank Joel Nothman, Andreas Mueller, and Adrin Jalali from the scikit-learn core development team and scikit-survival author Sebastian Pölsterl for their assistance with developing extensions to their libraries.
                             <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                              Figure 1
                             </a>
                             embedded image credits (all CC 3.0, BSD 3-Clause, or Apache 2.0 license): scikit-learn.org, scikit-survival.readthedocs.io, eli5.readthedocs.io, gdc.cancer.gov, github.com/Bioconductor/BiocStickers, Fiorella Schischlik, wikipedia.org.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-others" id="fn-1">
                              <p id="p-1">
                               <a class="rev-xref" href="#xref-fn-1-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                †
                               </span>
                               Equally contributing first authors
                              </p>
                             </li>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-5">
                               Further revised and expanded manuscript and figures
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-3">
                              <p id="p-6">
                               <a href="https://doi.org/10.5281/zenodo.5221525">
                                https://doi.org/10.5281/zenodo.5221525
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-4">
                              <p id="p-7">
                               <a href="https://doi.org/10.5281/zenodo.5838055">
                                https://doi.org/10.5281/zenodo.5838055
                               </a>
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-5">
                              <p id="p-8">
                               <a href="https://github.com/ruppinlab/tcga-microbiome-prediction">
                                https://github.com/ruppinlab/tcga-microbiome-prediction
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.1" data-doi="10.1056/NEJMp1607591">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grossman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. L.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Toward a Shared Vision for Cancer Genomic Data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  New England Journal of Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  375
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1109
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1112
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.volume%253D375%26rft.spage%253D1109%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJMp1607591%26rft_id%253Dinfo%253Apmid%252F27653561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJMp1607591&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27653561&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ahluwalia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolhe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gahlay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The clinical relevance of gene expression based prognostic signatures in colorectal cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochimica et Biophysica Acta (BBA) - Reviews on Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  1875
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  188513
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa%2B%2528BBA%2529%2B-%2BReviews%2Bon%2BCancer%26rft.volume%253D1875%26rft.spage%253D188513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.3" data-doi="10.1371/journal.pone.0094476">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brodsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Expression profiling of primary and metastatic ovarian tumors reveals differences indicative of aggressive disease
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e94476
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De94476%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0094476%26rft_id%253Dinfo%253Apmid%252F24732363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0094476&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24732363&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Pan-cancer analysis of clinical significance and associated molecular features of glycolysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioengineered
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4233
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4246
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioengineered%26rft.volume%253D12%26rft.spage%253D4233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.5" data-doi="10.1073/pnas.1714877115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Selfors
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stover
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I. S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brugge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coloff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Identification of cancer genes that are independent of dominant proliferation and lineage programs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E11276
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E11284
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1714877115%26rft_id%253Dinfo%253Apmid%252F29229826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE0LzUyL0UxMTI3NiI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzExLzIwMjAuMDcuMjEuMjE0MTQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shimoni
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Association between expression of random gene sets and survival is evident in multiple cancer types and may be explained by sub-classification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e1006026
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shukla
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Development of an RNA-Seq Based Prognostic Signature in Lung Adenocarcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Natl Cancer Inst
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Venet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dumont
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Detours
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Most random gene expression signatures are significantly associated with breast cancer outcome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e1002240
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Milanez-Almeida
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Germain
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R. N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tsang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J. S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cancer prognosis with shallow tumor RNA sequencing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  188
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  192
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMedicine%26rft.volume%253D26%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.10" data-doi="10.1038/s41586-020-2095-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Poore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Microbiome analyses of blood and tissues suggest cancer diagnostic approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  579
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  567
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  574
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D579%26rft.spage%253D567%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-020-2095-1%26rft_id%253Dinfo%253Apmid%252F32214244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-020-2095-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32214244&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tibshirani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Regularization Paths for Cox’s Proportional Hazards Model via Coordinate Descent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Statistical Software
                                 </abbr>
                                 <span class="cit-vol">
                                  39
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  13
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D39%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Estimation methods for time-dependent AUC models with survival data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Canadian Journal of Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  38
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  8
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  26
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCanadian%2BJournal%2Bof%2BStatistics%26rft.volume%253D38%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.13" data-doi="10.1177/0962280213515571">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chevret
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Summary measure of discrimination in survival models based on cumulative/dynamic time-dependent ROC curves
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Statistical methods in medical research
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  2088
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2102
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStatistical%2Bmethods%2Bin%2Bmedical%2Bresearch%26rft_id%253Dinfo%253Adoi%252F10.1177%252F0962280213515571%26rft_id%253Dinfo%253Apmid%252F24395866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1177/0962280213515571&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24395866&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.14" data-doi="10.1093/bioinformatics/btw344">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ding
                                  </span>
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Evaluating the molecule-based prediction of clinical drug responses in cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 , (
                                 <span class="cit-issue">
                                  19
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2891
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw344%26rft_id%253Dinfo%253Apmid%252F27354694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw344&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27354694&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.15">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guyon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weston
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barnhill
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vapnik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Gene Selection for Cancer Classification using Support Vector Machines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Machine Learning
                                 </abbr>
                                 <span class="cit-vol">
                                  46
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  389
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  422
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMachine%2BLearning%26rft.volume%253D46%26rft.spage%253D389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.16" data-doi="10.1111/j.1467-9868.2005.00503.x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zou
                                  </span>
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastie
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Regularization and variable selection via the elastic net
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J R Statist Soc B
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  301
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BR%2BStatist%2BSoc%2BB%26rft.volume%253D67%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ritchie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.E.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al
                                 </span>
                                 <span class="cit-article-title">
                                  . limma powers differential expression analyses for RNA-sequencing and microarray studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  43
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ),
                                 <span class="cit-issue">
                                  e47
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18. in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.18" data-doi="10.1093/bioinformatics/btp616">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCarthy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smyth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  edgeR: a Bioconductor package for differential expression analysis of digital gene expression data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  139
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  140
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19910308&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000273116100025&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19. in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.19" data-doi="10.1093/nar/gks042">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCarthy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smyth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G. K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4288
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4297
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks042%26rft_id%253Dinfo%253Apmid%252F22287627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gks042&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22287627&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000304535500013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20. in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.20" data-doi="10.1126/science.aah5043">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Geller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L. T.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Potential role of intratumor bacteria in mediating tumor resistance to the chemotherapeutic drug gemcitabine
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  357
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1156
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1160
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGeller%26rft.auinit1%253DL.%2BT.%26rft.volume%253D357%26rft.issue%253D6356%26rft.spage%253D1156%26rft.epage%253D1160%26rft.atitle%253DPotential%2Brole%2Bof%2Bintratumor%2Bbacteria%2Bin%2Bmediating%2Btumor%2Bresistance%2Bto%2Bthe%2Bchemotherapeutic%2Bdrug%2Bgemcitabine%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aah5043%26rft_id%253Dinfo%253Apmid%252F28912244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNTcvNjM1Ni8xMTU2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMTEvMjAyMC4wNy4yMS4yMTQxNDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.21" data-doi="10.1158/2159-8290.CD-17-1134">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pushalkar
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The Pancreatic Cancer Microbiome Promotes Oncogenesis by Induction of Innate and Adaptive Immune Suppression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Discov
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  403
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  416
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BDiscov%26rft_id%253Dinfo%253Adoi%252F10.1158%252F2159-8290.CD-17-1134%26rft_id%253Dinfo%253Apmid%252F29567829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiY2FuZGlzYyI7czo1OiJyZXNpZCI7czo3OiI4LzQvNDAzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMTEvMjAyMC4wNy4yMS4yMTQxNDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference 22. in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.22" data-doi="10.1126/science.aay9189">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nejman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The human tumor microbiome is composed of tumor type–specific intracellular bacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  368
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  973
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  980
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNejman%26rft.auinit1%253DD.%26rft.volume%253D368%26rft.issue%253D6494%26rft.spage%253D973%26rft.epage%253D980%26rft.atitle%253DThe%2Bhuman%2Btumor%2Bmicrobiome%2Bis%2Bcomposed%2Bof%2Btumor%2Btype-specific%2Bintracellular%2Bbacteria%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aay9189%26rft_id%253Dinfo%253Apmid%252F32467386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNjgvNjQ5NC85NzMiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8xMS8yMDIwLjA3LjIxLjIxNDE0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.23" data-doi="10.1371/journal.pone.0224365">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vabalas
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Machine learning algorithm validation with a limited sample size
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e0224365
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D14%26rft.spage%253De0224365%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0224365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0224365&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.24" data-doi="10.1371/journal.ppat.1008524">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Microbiome factors in HPV-driven carcinogenesis and cancers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Pathog
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1008524
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog%26rft.volume%253D16%26rft.spage%253De1008524%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1008524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1008524&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.25" data-doi="10.1097/MD.0000000000003077">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Chlamydia Trachomatis Infection-Associated Risk of Cervical Cancer: A Meta-Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Medicine
                                 </abbr>
                                 (Baltimore)
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  13
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e3077
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMedicine%26rft.volume%253D95%26rft.spage%253De3077%26rft_id%253Dinfo%253Adoi%252F10.1097%252FMD.0000000000003077%26rft_id%253Dinfo%253Apmid%252F27043670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1097/MD.0000000000003077&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27043670&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gnanasekar
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The intratumor microbiome predicts prognosis across gender and subtypes in papillary thyroid carcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Comput Struct Biotechnol J
                                 </abbr>
                                 <span class="cit-vol">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1986
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1997
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DComput%2BStruct%2BBiotechnol%2BJ%26rft.volume%253D19%26rft.spage%253D1986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference 27. in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Dohlman
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The cancer microbiome atlas: a pan-cancer comparative analysis to distinguish tissue-resident microbiota from contaminants
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Host Microbe
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  281
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  298
                                 </span>
                                 .e5 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D10%26rft.spage%253D281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference 28. in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.28" data-doi="10.1056/NEJM199110173251603">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parsonnet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Helicobacter pylori infection and the risk of gastric carcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  N Engl J Med
                                 </abbr>
                                 <span class="cit-vol">
                                  325
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1127
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1131
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DParsonnet%26rft.auinit1%253DJ.%26rft.volume%253D325%26rft.issue%253D16%26rft.spage%253D1127%26rft.epage%253D1131%26rft.atitle%253DHelicobacter%2Bpylori%2Binfection%2Band%2Bthe%2Brisk%2Bof%2Bgastric%2Bcarcinoma%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJM199110173251603%26rft_id%253Dinfo%253Apmid%252F1891020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1056/NEJM199110173251603&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1891020&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991GK53800003&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.29" data-doi="10.1136/gutjnl-2017-314205">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferreira
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Gastric microbial community profiling reveals a dysbiotic cancer-associated microbiota
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gut
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  226
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  236
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGut%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fgutjnl-2017-314205%26rft_id%253Dinfo%253Apmid%252F29102920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ3V0am5sIjtzOjU6InJlc2lkIjtzOjg6IjY3LzIvMjI2IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDEvMTEvMjAyMC4wNy4yMS4yMTQxNDguYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abate
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qureshi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mazumder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cedecea davisae bacteremia in a neutropenic patient with acute myeloid leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Infect
                                 </abbr>
                                 <span class="cit-vol">
                                  63
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  83
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  85
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Binfection%26rft.stitle%253DJ%2BInfect%26rft.aulast%253DAbate%26rft.auinit1%253DG.%26rft.volume%253D63%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D85%26rft.atitle%253DCedecea%2Bdavisae%2Bbacteremia%2Bin%2Ba%2Bneutropenic%2Bpatient%2Bwith%2Bacute%2Bmyeloid%2Bleukemia.%26rft_id%253Dinfo%253Apmid%252F21571373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21571373&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.31" data-doi="10.1016/j.diagmicrobio.2012.06.019">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Akinosoglou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Bacteraemia due to Cedecea davisae in a patient with sigmoid colon cancer: a case report and brief review of the literature
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Diagn Microbiol Infect Dis
                                 </abbr>
                                 <span class="cit-vol">
                                  74
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  303
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  306
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDiagnostic%2Bmicrobiology%2Band%2Binfectious%2Bdisease%26rft.stitle%253DDiagn%2BMicrobiol%2BInfect%2BDis%26rft.aulast%253DAkinosoglou%26rft.auinit1%253DK.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D303%26rft.epage%253D306%26rft.atitle%253DBacteraemia%2Bdue%2Bto%2BCedecea%2Bdavisae%2Bin%2Ba%2Bpatient%2Bwith%2Bsigmoid%2Bcolon%2Bcancer%253A%2Ba%2Bcase%2Breport%2Band%2Bbrief%2Breview%2Bof%2Bthe%2Bliterature.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.diagmicrobio.2012.06.019%26rft_id%253Dinfo%253Apmid%252F22884845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.diagmicrobio.2012.06.019&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22884845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.32" data-doi="10.3343/alm.2013.33.4.283">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y. R.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The first Korean case of Sphingobacterium spiritivorum bacteremia in a patient with acute myeloid leukemia
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Ann Lab Med
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  283
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  287
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnn%2BLab%2BMed%26rft.volume%253D33%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.3343%252Falm.2013.33.4.283%26rft_id%253Dinfo%253Apmid%252F23826566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.3343/alm.2013.33.4.283&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23826566&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Profiling the Urinary Microbiota in Male Patients with Bladder Cancer in China
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Cell Infect Microbiol
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  167
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference 34. in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.34" data-doi="10.1136/gutjnl-2017-314281">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O. O.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Mucosal microbiome dysbiosis in gastric carcinogenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Gut
                                 </abbr>
                                 <span class="cit-vol">
                                  67
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1024
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1032
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGut%26rft_id%253Dinfo%253Adoi%252F10.1136%252Fgutjnl-2017-314281%26rft_id%253Dinfo%253Apmid%252F28765474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ3V0am5sIjtzOjU6InJlc2lkIjtzOjk6IjY3LzYvMTAyNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAxLzExLzIwMjAuMDcuMjEuMjE0MTQ4LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference 35. in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Castaño-Rodriguez
                                  </span>
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Dysbiosis of the microbiome in gastric carcinogenesis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci Rep
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  15957
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci%2BRep%26rft.volume%253D7%26rft.spage%253D15957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.36" data-doi="10.1074/jbc.M111.231076">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ogwang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Bacterial Conversion of Folinic Acid Is Required for Antifolate Resistance
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Biological Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  286
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  15377
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  15390
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.231076%26rft_id%253Dinfo%253Apmid%252F21372133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMTcvMTUzNzciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMS8xMS8yMDIwLjA3LjIxLjIxNDE0OC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eslami-S
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Majidzadeh-A
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Halvaei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Babapirali
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Esmaeili
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Microbiome and Breast Cancer: New Role for an Ancient Population
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Front Oncol
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 , (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.38" data-doi="10.1007/s10059-011-0066-y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H. J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D. N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Association between Epstein-Barr virus infection and chemoresistance to docetaxel in gastric carcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cells
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  173
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  179
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecules%2Band%2Bcells%26rft.stitle%253DMol%2BCells%26rft.aulast%253DShin%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D32%26rft.issue%253D2%26rft.spage%253D173%26rft.epage%253D179%26rft.atitle%253DAssociation%2Bbetween%2BEpstein-Barr%2Bvirus%2Binfection%2Band%2Bchemoresistance%2Bto%2Bdocetaxel%2Bin%2Bgastric%2Bcarcinoma.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10059-011-0066-y%26rft_id%253Dinfo%253Apmid%252F21626300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10059-011-0066-y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21626300&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Di Rienzi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The human gut and groundwater harbor non-photosynthetic bacteria belonging to a new candidate phylum sibling to Cyanobacteria
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e01102
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sedighi
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Therapeutic bacteria to combat cancer; current advances, challenges, and opportunities
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer Med
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  3167
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3181
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%2BMed%26rft.volume%253D8%26rft.spage%253D3167%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coley
                                  </span>
                                  <span class="cit-name-given-names">
                                   WB
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Late results of the treatment of inoperable sarcoma by the mixed toxins of erysipelas and bacillus prodigiosus
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trans Southern Surg Gynecol Ass
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  197
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1906
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrans%2BSouthern%2BSurg%2BGynecol%2BAss%26rft.volume%253D18%26rft.spage%253D197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.42" data-doi="10.1055/s-0029-1196806">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fehleisen
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Ueber die Züchtung der Erysipelkokken auf künstlichem Nährboden und ihre Übertragbarkeit auf den Menschen
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Dtsch Med Wochenschr
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  553
                                 </span>
                                 -
                                 <span class="cit-lpage">
                                  554
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1882
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDtsch%2BMed%2BWochenschr%26rft.volume%253D8%26rft.spage%253D553%26rft_id%253Dinfo%253Adoi%252F10.1055%252Fs-0029-1196806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1055/s-0029-1196806&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Busch
                                  </span>
                                  <span class="cit-name-given-names">
                                   W
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Aus der Sitzung der medicinischen Section vom 13 November 1867
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Berl Klin Wochenschr
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  137
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1868
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBerl%2BKlin%2BWochenschr%26rft.volume%253D5%26rft.spage%253D137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kalaora
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Identification of bacteria-derived HLA-bound peptides in melanoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  592
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7852
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  138
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  143
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D592%26rft.spage%253D138%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.45" data-doi="10.1016/j.cell.2018.02.052">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  An Integrated TCGA Pan-Cancer Clinical Data Resource to Drive High-Quality Survival Outcome Analytics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  173
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  400
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  416
                                 </span>
                                 .e11 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D173%26rft.spage%253D400%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2018.02.052%26rft_id%253Dinfo%253Apmid%252F29625055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2018.02.052&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29625055&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.46" data-doi="10.1016/j.cpc.2010.04.018">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  Pedregosa
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scikit-learn: Machine Learning in Python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Mach Learn Res
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2825
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2830
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BMach%2BLearn%2BRes%26rft.volume%253D12%26rft.spage%253D2825%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cpc.2010.04.018%26rft_id%253Dinfo%253Apmid%252F23755062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cpc.2010.04.018&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23755062&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F01%2F11%2F2020.07.21.214148.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.07.21.214148v5.47">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-surname">
                                    Appice
                                   </span>
                                   ,
                                   <span class="cit-name-given-names">
                                    A
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pölsterl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navab
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katouzian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-chapter-title">
                                  Fast Training of Support Vector Machines for Survival Analysis
                                 </span>
                                 . in
                                 <span class="cit-source">
                                  Machine Learning and Knowledge Discovery in Databases
                                 </span>
                                 (eds.
                                 <span class="cit-ed">
                                  <span class="cit-name-surname">
                                   Appice
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 )
                                 <span class="cit-fpage">
                                  243
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  259
                                 </span>
                                 (
                                 <span class="cit-publ-name">
                                  Springer International Publishing
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015)
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.21.214148v5.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pölsterl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Navab
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 , &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katouzian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .,
                                 <span class="cit-article-title">
                                  An Efficient Training Algorithm for Kernel Survival Support Vector Machines
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  4th Workshop on Machine Learning in Life Sciences
                                 </abbr>
                                 , 23 September 2016, Riva del Garda, Italy.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pölsterl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Heterogeneous ensembles for predicting survival of metastatic, castrate-resistant prostate cancer patients
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  . F1000Res
                                 </abbr>
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2676
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253D.%2BF1000Res%26rft.volume%253D5%26rft.spage%253D2676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.21.214148v5.50" data-doi="10.18637/jss.v028.i08">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hothorn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hornik
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. A. van de
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zeileis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Implementing a Class of Permutation Tests: The coin Package
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Statistical Software
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D28%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v028.i08%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.18637/jss.v028.i08&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.07.21.214148v5" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.07.26.453861v2" title="The Alzheimer’s gene SORL1 is a regulator of endosomal traffic and recycling in human neurons" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.07.30.454453v3" title="Anaplerotic flux into the Calvin-Benson cycle. Hydrogen isotope evidence for in vivo occurrence in C3 metabolism" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted January 11, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.07.21.214148v5.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.07.21.214148v5.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/01/11/2020.07.21.214148.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.07.21.214148v5.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.07.21.214148v5" class="active" data-icon-position="" data-hide-link-title="0">
                                   Predicting cancer prognosis and drug response from the tumor microbiome
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2345108"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-3cKKwkdbYgHPs395SZuY1qs7jhDrz_jNLXa0-CE81GA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484474507"/>
                                   <input type="hidden" name="captcha_token" value="ab813f9dfd1b55bbc73f815dce0e4054"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2345108--3" data-node-nid="2345108" id="node-2345108--41767802662" data-pisa="biorxiv;2020.07.21.214148v5" data-pisa-master="biorxiv;2020.07.21.214148" data-seqnum="2345108" data-apath="/biorxiv/early/2022/01/11/2020.07.21.214148.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Predicting cancer prognosis and drug response from the tumor microbiome
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Leandro C.
                                  </span>
                                  <span class="nlm-surname">
                                   Hermida
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   E. Michael
                                  </span>
                                  <span class="nlm-surname">
                                   Gertz
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Eytan
                                  </span>
                                  <span class="nlm-surname">
                                   Ruppin
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.07.21.214148;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.07.21.214148
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&title=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&title=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&text=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&t=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&title=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&title=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&title=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2345108--5" data-node-nid="2345108" id="citation-node-2345108--6752816363" data-pisa="biorxiv;2020.07.21.214148v5" data-pisa-master="biorxiv;2020.07.21.214148" data-seqnum="2345108" data-apath="/biorxiv/early/2022/01/11/2020.07.21.214148.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Predicting cancer prognosis and drug response from the tumor microbiome
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Leandro C.
                                      </span>
                                      <span class="nlm-surname">
                                       Hermida
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       E. Michael
                                      </span>
                                      <span class="nlm-surname">
                                       Gertz
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Eytan
                                      </span>
                                      <span class="nlm-surname">
                                       Ruppin
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.07.21.214148;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.07.21.214148
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Hermida%2C+Leandro+C.&rft.au=Gertz%2C+E.+Michael&rft.au=Ruppin%2C+Eytan">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2345108/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2345108/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2345108/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2345108/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2345108/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2345108/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2345108/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2345108/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2345108/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2345108/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2345108/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2345108/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&count=horizontal&via=&text=Predicting%20cancer%20prognosis%20and%20drug%20response%20from%20the%20tumor%20microbiome&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.07.21.214148v5&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.07.21.214148v5" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/cancer-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Cancer Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
