<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.05.06.058180v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.05.06.058180v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.05.06.058180v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-3.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/11/11/2020.05.06.058180.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.05.06.058180v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Cancer phylogenetic tree inference at scale from 1000s of single cell genomes"/>
  <meta name="DC.Identifier" content="10.1101/2020.05.06.058180"/>
  <meta name="DC.Date" content="2022-11-11"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at <https://github.com/UBC-Stat-ML/sitkatree.git>.

### Competing Interest Statement

Dr. Sohrab P. Shah and Dr. Samuel Aparicio are founders, shareholders, and consultants of Canexia Health Inc."/>
  <meta name="DC.Contributor" content="Sohrab Salehi"/>
  <meta name="DC.Contributor" content="Fatemeh Dorri"/>
  <meta name="DC.Contributor" content="Kevin Chern"/>
  <meta name="DC.Contributor" content="Farhia Kabeer"/>
  <meta name="DC.Contributor" content="Nicole Rusk"/>
  <meta name="DC.Contributor" content="Tyler Funnell"/>
  <meta name="DC.Contributor" content="Marc J Williams"/>
  <meta name="DC.Contributor" content="Daniel Lai"/>
  <meta name="DC.Contributor" content="Mirela Andronescu"/>
  <meta name="DC.Contributor" content="Kieran R. Campbell"/>
  <meta name="DC.Contributor" content="Andrew McPherson"/>
  <meta name="DC.Contributor" content="Samuel Aparicio"/>
  <meta name="DC.Contributor" content="Andrew Roth"/>
  <meta name="DC.Contributor" content="Sohrab Shah"/>
  <meta name="DC.Contributor" content="Alexandre Bouchard-Côté"/>
  <meta name="article:published_time" content="2022-11-11"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Cancer phylogenetic tree inference at scale from 1000s of single cell genomes"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at https://github.com/UBC-Stat-ML/sitkatree.git.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.05.06.058180v3"/>
  <meta name="citation_id" content="2020.05.06.058180v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.05.06.058180"/>
  <meta name="citation_num_pages" content="51"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.05.06.058180"/>
  <meta name="citation_author" content="Sohrab Salehi"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author" content="Fatemeh Dorri"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of British Columbia"/>
  <meta name="citation_author" content="Kevin Chern"/>
  <meta name="citation_author_institution" content="Department of Statistics, University of British Columbia"/>
  <meta name="citation_author" content="Farhia Kabeer"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia"/>
  <meta name="citation_author" content="Nicole Rusk"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author" content="Tyler Funnell"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author" content="Marc J Williams"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5524-4174"/>
  <meta name="citation_author" content="Daniel Lai"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia"/>
  <meta name="citation_author_institution" content="Department of Molecular Oncology, BC Cancer Research Centre"/>
  <meta name="citation_author" content="Mirela Andronescu"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia"/>
  <meta name="citation_author_institution" content="Department of Molecular Oncology, BC Cancer Research Centre"/>
  <meta name="citation_author" content="Kieran R. Campbell"/>
  <meta name="citation_author_institution" content="Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital"/>
  <meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto"/>
  <meta name="citation_author_institution" content="Department of Statistical Sciences, University of Toronto"/>
  <meta name="citation_author" content="Andrew McPherson"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author" content="Samuel Aparicio"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia"/>
  <meta name="citation_author_institution" content="Department of Molecular Oncology, BC Cancer Research Centre"/>
  <meta name="citation_author" content="Andrew Roth"/>
  <meta name="citation_author_institution" content="Department of Computer Science, University of British Columbia"/>
  <meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, University of British Columbia"/>
  <meta name="citation_author_institution" content="Department of Molecular Oncology, BC Cancer Research Centre"/>
  <meta name="citation_author" content="Sohrab Shah"/>
  <meta name="citation_author_institution" content="Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center"/>
  <meta name="citation_author" content="Alexandre Bouchard-Côté"/>
  <meta name="citation_author_institution" content="Department of Statistics, University of British Columbia"/>
  <meta name="citation_author_email" content="bouchard@stat.ubc.ca"/>
  <meta name="citation_reference" content="E. Laks et al. “Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing”. In: Cell 179.5 (2019), 1207–1221.e22."/>
  <meta name="citation_reference" content="M. Pellegrino et al. “High-throughput single-cell DNA sequencing of acute myeloid leukemia tumors with droplet micro?uidics”. In: Genome research 28.9 (2018), pp. 1345–1352."/>
  <meta name="citation_reference" content="T. Baslan et al. “Genome-wide copy number analysis of single cells”. In: Nature pro-tocols 7.6 (2012), pp. 1024–1041."/>
  <meta name="citation_reference" content="C. Gawad, W. Koh, and S. R. Quake. “Single-cell genome sequencing: current state of the science”. In: Nature Reviews Genetics 17.3 (2016), p. 175."/>
  <meta name="citation_reference" content="S. Salehi et al. “Clonal fitness inferred from time-series modelling of single-cell can-cer genomes”. In: Nature 595.7868 (2021), pp. 585–590."/>
  <meta name="citation_reference" content="J. J. Quinn et al. “Single-cell lineages reveal the rates, routes, and drivers of metas-tasis in cancer xenografts”. In: Science 371.6532 (2021), eabc1944."/>
  <meta name="citation_reference" content="C. Abbosh et al. “Phylogenetic ctDNA analysis depicts early-stage lung cancer evo-lution”. In: Nature 545.7655 (2017), pp. 446–451."/>
  <meta name="citation_reference" content="Y. Wang et al. “Clonal evolution in breast cancer revealed by single nucleus genome sequencing”. In: Nature 512.7513 (2014), pp. 155–160."/>
  <meta name="citation_reference" content="M. L. Leung et al. “Single-cell DNA sequencing reveals a late-dissemination model in metastatic colorectal cancer”. In: Genome research 27.8 (2017), pp. 1287–1299."/>
  <meta name="citation_reference" content="C. Yu et al. “Discovery of biclonal origin and a novel oncogene SLC12A5 in colon cancer by single-cell sequencing”. In: Cell research 24.6 (2014), pp. 701–712."/>
  <meta name="citation_reference" content="R. Schwartz and A. A. Schäffer. “The evolution of tumour phylogenetics: principles and practice”. In: Nature Reviews Genetics 18.4 (2017), pp. 213–229."/>
  <meta name="citation_reference" content="K. Jahn, J. Kuipers, and N. Beerenwinkel. “Tree inference for single-cell data”. In: Genome Biology 17 (2016), p. 86."/>
  <meta name="citation_reference" content="E. M. Ross and F. Markowetz. “OncoNEM: inferring tumor evolution from single-cell sequencing data”. In: Genome Biology 17 (2016), p. 69."/>
  <meta name="citation_reference" content="C. A Miller et al. “SciClone: inferring clonal architecture and tracking the spatial and temporal patterns of tumor evolution”. In: PLoS Comput Biol 10.8 (2014), e1003665."/>
  <meta name="citation_reference" content="J. Singer et al. “Single-cell mutation identification via phylogenetic inference”. In: Nature communications 9.1 (2018), pp. 1–8."/>
  <meta name="citation_reference" content="H. Zahn et al. “Scalable whole-genome single-cell library preparation without pream-plification”. In: Nature Methods 14.2 (2017), pp. 167–173."/>
  <meta name="citation_reference" content="T. L. Kaufmann et al. “MEDICC2: whole-genome doubling aware copy-number phy-logenies for cancer evolution”. In: bioRxiv (2021)."/>
  <meta name="citation_reference" content="G. Satas et al. “Scarlet: Single-cell tumor phylogeny inference with copy-number constrained mutation losses”. In: Cell Systems 10.4 (2020), pp. 323–332."/>
  <meta name="citation_reference" content="F. Wang et al. “MEDALT: single-cell copy number lineage tracing enabling gene dis-covery”. In: Genome biology 22.1 (2021), pp. 1–22."/>
  <meta name="citation_reference" content="X. Xu et al. “Single-Cell Exome Sequencing Reveals Single-Nucleotide Mutation Characteristics of a Kidney Tumor”. In: Cell 148.5 (2012), pp. 886–895."/>
  <meta name="citation_reference" content="M. K. Kuhner and J. Felsenstein. “A simulation comparison of phylogeny algorithms under equal and unequal evolutionary rates”. In: Molecular Biology and Evolution 11.3 (1994), pp. 459–468."/>
  <meta name="citation_reference" content="S. Guindon and O. Gascuel. “A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood”. In: Systematic Biology 52.5 (2003), pp. 696–704."/>
  <meta name="citation_reference" content="T. L. Williams and B. M. E. Moret. “An investigation of phylogenetic likelihood meth-ods”. In: Third IEEE Symposium on Bioinformatics and Bioengineering, 2003. Pro-ceedings. 2003, pp. 79–86."/>
  <meta name="citation_reference" content="A. Som. “ML or NJ-MCL? A comparison between two robust phylogenetic methods”. In: Computational Biology and Chemistry 33.5 (2009), pp. 373–378."/>
  <meta name="citation_reference" content="S. Malikic et al. “Integrative inference of subclonal tumour evolution from single-cell and bulk sequencing data”. In: Nature communications 10.1 (2019), pp. 1–12."/>
  <meta name="citation_reference" content="J. Ma et al. “The infinite sites model of genome evolution”. In: Proceedings of the National Academy of Sciences 105.38 (2008), pp. 14254–14261."/>
  <meta name="citation_reference" content="C. D. Greenman et al. “Estimation of rearrangement phylogeny for cancer genomes”. In: Genome Research 22.2 (2012), pp. 346–361."/>
  <meta name="citation_reference" content="H. Zafar et al. “SiFit: inferring tumor trees from single-cell sequencing data under finite-sites models”. In: Genome biology 18.1 (2017), pp. 1–20."/>
  <meta name="citation_reference" content="J. Househam et al. “Phenotypic plasticity and genetic control in colorectal cancer evolution”. In: Nature (2022), pp. 1–10."/>
  <meta name="citation_reference" content="P. Eirew et al. “Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution”. In: Nature 518.7539 (2015), pp. 422–426."/>
  <meta name="citation_reference" content="K. Yi and Y. Seok Ju. “Patterns and mechanisms of structural variations in human cancer”. In: Experimental & Molecular Medicine 50.8 (2018), p. 98."/>
  <meta name="citation_reference" content="S. Mishra and J. R. Whetstine. “Different Facets of Copy Number Changes: Per-manent, Transient, and Adaptive”. In: Molecular and Cellular Biology 36.7 (2016), pp. 1050–1063."/>
  <meta name="citation_reference" content="D.B. Wilson. “Generating Random Spanning Trees More Quickly Than the Cover Time”. In: Proceedings of the Twenty-eighth Annual ACM Symposium on Theory of Computing. STOC ‘96. New York, NY, USA: ACM, 1996, pp. 296–303. ISBN: 978-0-89791-785-8."/>
  <meta name="citation_reference" content="Y. W. Teh et al. “Dirichlet Process.” In: Encyclopedia of machine learning 1063 (2010), pp. 280–287."/>
  <meta name="citation_reference" content="A. Bouchard-Côté et al. “Blang: Probabilitistic Programming for Combinatorial Spaces”. In: Journal of Statistical Software 103 (2022), pp. 1–98."/>
  <meta name="citation_reference" content="S. Geman and D. Geman. “Stochastic relaxation, Gibbs distributions, and the Bayesian restoration of images”. In: IEEE Transactions on pattern analysis and ma-chine intelligence 6 (1984), pp. 721–741."/>
  <meta name="citation_reference" content="R.M. Neal. “Slice sampling”. In: The Annals of Statistics 31.3 (2003), pp. 705–767."/>
  <meta name="citation_reference" content="C. Robert. The Bayesian Choice -From Decision-Theoretic Foundations | Christian Robert | Springer. 2007."/>
  <meta name="citation_reference" content="D. F. Robinson and L. R. Foulds. “Comparison of phylogenetic trees”. In: Mathemat-ical biosciences 53.1-2 (1981), pp. 131–147."/>
  <meta name="citation_reference" content="R. Desper and O. Gascuel. “Fast and accurate phylogeny reconstruction algorithms based on the minimum-evolution principle”. In: International Workshop on Algorithms in Bioinformatics. Springer. 2002, pp. 357–374."/>
  <meta name="citation_reference" content="X. F Mallory et al. “Methods for copy number aberration detection from single-cell DNA-sequencing data”. In: Genome biology 21.1 (2020), pp. 1–22."/>
  <meta name="citation_reference" content="R. Sainudiin and A. Véber. “A Beta-splitting model for evolutionary trees”. In: Royal Society open science 3.5 (2016), p. 160016."/>
  <meta name="citation_reference" content="M. G. B. Blum and O. François. “Which random processes describe the tree of life? A large-scale study of phylogenetic tree imbalance”. In: Systematic Biology 55.4 (2006), pp. 685–691."/>
  <meta name="citation_reference" content="D. Aldous. “Probability distributions on cladograms”. In: Random discrete structures. Springer, 1996, pp. 1–18."/>
  <meta name="citation_reference" content="R. Gao et al. “Punctuated copy number evolution and clonal stasis in triple-negative breast cancer”. In: Nature genetics 48.10 (2016), p. 1119."/>
  <meta name="citation_reference" content="J. F. C. Kingman. “The coalescent”. In: Stochastic processes and their applications 13.3 (1982), pp. 235–248."/>
  <meta name="citation_reference" content="K.P. Schliep. “phangorn: phylogenetic analysis in R”. In: Bioinformatics 27.4 (2011), pp. 592–593."/>
  <meta name="citation_reference" content="P. R. Staab and D. Metzler. “Coala: an R framework for coalescent simulation”. In: Bioinformatics 32.12 (2016), pp. 1903–1904."/>
  <meta name="citation_reference" content="W. J. Youden. “Index for rating diagnostic tests”. In: Cancer 3.1 (1950), pp. 32–35."/>
  <meta name="twitter:title" content="Cancer phylogenetic tree inference at scale from 1000s of single cell genomes"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at <https://github.com/UBC-Stat-ML/sitkatree.git>.

### Competing Interest Statement

Dr. Sohrab P. Shah and Dr. Samuel Aparicio are founders, shareholders, and consultants of Canexia Health Inc."/>
  <meta name="og-title" property="og:title" content="Cancer phylogenetic tree inference at scale from 1000s of single cell genomes"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at <https://github.com/UBC-Stat-ML/sitkatree.git>.

### Competing Interest Statement

Dr. Sohrab P. Shah and Dr. Samuel Aparicio are founders, shareholders, and consultants of Canexia Health Inc."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-11-11"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2844645"/>
  <title>
   Cancer phylogenetic tree inference at scale from 1000s of single cell genomes | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8EpxO4vrKVGco3xZpo3hoGUiYQy2rxetaoUuTjpcOHs"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2844645","apath":"\/biorxiv\/early\/2022\/11\/11\/2020.05.06.058180.atom","pisa":"biorxiv;2020.05.06.058180v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.05.06.058180v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.05.06.058180v3","title":"Cancer phylogenetic tree inference at scale from 1000s of single cell genomes","identifier":"node\/2844645"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QbAt2Q4LWrCpNzBWcT6mZgGQ8KkUjLgNBHY7dY6FChIS79\\\\\\\/Z68TQQBdKsxW6ZaBaWZR3kslnpcZ3vdv24O1\\\\\\\/\\\\\\\/wmuN2inDyK1BYgIcK74uZ7Lr6CBumel7XK9ZJOtUWEem1OvD1wHaSTf+lTdQk8jUdeB6rnR8LLhEt42BHs09rHbnPogrYV4t4ocOb3F9tzu68kyb\\\\\\\/3CJODUaf7dWcxGgMQ9V4DUxa7x403mjzet2n1wHBhGm9LamRTB1jt57YqI3Q\\\\\\\/ZP7CPzOS1mWur3XvOeeYLwtpNH23PdQzTiBdbRFVVcbfcxNnNxLFo5395OmNZZeUHBiInKmrPAgbC0z4xlJCvsP2SzSOKUpXLBgf61mTfP3iy6RU0NxOTw0kbyrkyPsi2rnOwOgYIftAxOn3BPToTiOV9a1fndJkVET1QpN9uFckeP9XHXh8WiXAOurchL5kSxO0IAfcjQ9Wo9fmDpROpIyOZ0o0EiYAuqtyi4tc4JNabZETIYlq+fGaTZOPam1W0FPiPEp74Kv4\\\\\\\/2vEw201U38QL6G8QA99O\\\\\\\/r+XcjQ3wv6SSJOS4wI\\\\\\\/rxjjE5PZ1IeP6EwrJdVCM4iXU3yHyVFwESMVL9soA+7eluB5\\\\\\\/Hzr9UqNO\\\\\\\/yMQo0ZrOJRglilO3BQe5LgrXtLP1IuB70hKyR5BdieKaz1je33Dvb5PFEDtKLff5K67D6zLa5vXfVgCiq0YDqpoL8QzNDL6jI\\\\\\\/oRA\\\\\\\/hrNdbCdpUUN4rnJIk5Xcz2zRL1Ku9Yc0qMW5awY9fOqrFvWJyzC0t402sgbzDvj841IjrCizfOQVdH1FH7nK0VYsef2YtmCMq6TZpmcs1\\\\\\\/c5NucDjzVtSa+PEQxvlw3bnt1wPCCbB1JK\\\\\\\/Cdl9L1RUFKwdoQ2aMm\\\\\\\/4dOquqrBK\\\\\\\/69JX89usDrpsFRUJjk\\\\\\\/0FbHv5FpAxEqj7PFEEAG\\\\\\\/ocKTePgofo23SXdt9jVdsvj\\\\\\\/VN15odSxtwlW81pYmn6hDIwNcjQFrSxjskyS1ypwA3ZTpPnnP9Qp8e1OUoVeEazoKYqp+jbHKggg9d4+lE4XTWkv7AVFMYTJI0L1Ib8WVF4yAWvxIqloVxAtzT1C5ueuvpX7E7\\\\\\\/hDybn4xLbB45aIDE2ViPfCmwDgUr\\\\\\\/e6bQtejyVUTi5RzyJyTFlVsUWXCwUOxs2K48EvVGScbxQh6k\\\\\\\/PBWj0q+R\\\\\\\/0srh3526gERY9OT2HBst0DnV+emRcXZBJksZ+MdMYxBduvoAslNFuKmNeRYRFfkGeUc8kOsREmAGoUVlZDhsT9ByaOSIYr5bv65VwwIfxEPC2m14zn8juSLiavua8gOrvehPZ0RysJRp98daUn2zD1rJE4+J2jUKZhHhJvtmkPssIOTtEkgD4cO2UsT0cS1qjFZixhOAaOY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022elEOB2+1KDUb1hidcj0mow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00226bca2e625d76589abb5fc963eeda74828f417266cb46ae1ad193a67937a6187b\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.05.06.058180v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2844645 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.05.06.058180v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1415319768">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1415319768" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_848093927" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-7O0HcpDHbg8Bq0oh9US5PK8aMPgih1U60v99b2FXj9w"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2844645" data-node-nid="2844645" id="node-2844645--2660706185" data-pisa="biorxiv;2020.05.06.058180v3" data-pisa-master="biorxiv;2020.05.06.058180" data-apath="/biorxiv/early/2022/11/11/2020.05.06.058180.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Cancer phylogenetic tree inference at scale from 1000s of single cell genomes
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Sohrab
                      </span>
                      <span class="nlm-surname">
                       Salehi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Fatemeh
                      </span>
                      <span class="nlm-surname">
                       Dorri
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Kevin
                      </span>
                      <span class="nlm-surname">
                       Chern
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Farhia
                      </span>
                      <span class="nlm-surname">
                       Kabeer
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Nicole
                      </span>
                      <span class="nlm-surname">
                       Rusk
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Tyler
                      </span>
                      <span class="nlm-surname">
                       Funnell
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0001-5524-4174" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Marc J
                      </span>
                      <span class="nlm-surname">
                       Williams
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="7">
                      <span class="nlm-given-names">
                       Daniel
                      </span>
                      <span class="nlm-surname">
                       Lai
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="8">
                      <span class="nlm-given-names">
                       Mirela
                      </span>
                      <span class="nlm-surname">
                       Andronescu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="9">
                      <span class="nlm-given-names">
                       Kieran R.
                      </span>
                      <span class="nlm-surname">
                       Campbell
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="10">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       McPherson
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="11">
                      <span class="nlm-given-names">
                       Samuel
                      </span>
                      <span class="nlm-surname">
                       Aparicio
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="12">
                      <span class="nlm-given-names">
                       Andrew
                      </span>
                      <span class="nlm-surname">
                       Roth
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="13">
                      <span class="nlm-given-names">
                       Sohrab
                      </span>
                      <span class="nlm-surname">
                       Shah
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="14">
                      <span class="nlm-given-names">
                       Alexandre
                      </span>
                      <span class="nlm-surname">
                       Bouchard-Côté
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.05.06.058180
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2844645--2660706185" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Sohrab Salehi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sohrab%2BSalehi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Salehi%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASohrab%2BSalehi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Fatemeh Dorri
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of British Columbia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Fatemeh%2BDorri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dorri%20F&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AFatemeh%2BDorri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Kevin Chern
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Statistics, University of British Columbia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kevin%2BChern%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Chern%20K&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKevin%2BChern%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Farhia Kabeer
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Pathology and Laboratory Medicine, University of British Columbia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Farhia%2BKabeer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kabeer%20F&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AFarhia%2BKabeer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Nicole Rusk
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Nicole%2BRusk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Rusk%20N&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ANicole%2BRusk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Tyler Funnell
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Tyler%2BFunnell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Funnell%20T&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ATyler%2BFunnell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Marc J Williams
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Marc%2BJ%2BWilliams%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Williams%20MJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMarc%2BJ%2BWilliams%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-5524-4174" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Marc J Williams
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Daniel Lai
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Pathology and Laboratory Medicine, University of British Columbia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular Oncology, BC Cancer Research Centre
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Daniel%2BLai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lai%20D&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ADaniel%2BLai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Mirela Andronescu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Pathology and Laboratory Medicine, University of British Columbia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular Oncology, BC Cancer Research Centre
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mirela%2BAndronescu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Andronescu%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMirela%2BAndronescu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Kieran R. Campbell
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Lunenfeld-Tanenbaum Research Institute, Mount Sinai Hospital
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular Genetics, University of Toronto
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        Department of Statistical Sciences, University of Toronto
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kieran%2BR.%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Campbell%20KR&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AKieran%2BR.%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-10">
                    <div class="author-tooltip-name">
                     Andrew McPherson
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BMcPherson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=McPherson%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BMcPherson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-11">
                    <div class="author-tooltip-name">
                     Samuel Aparicio
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Pathology and Laboratory Medicine, University of British Columbia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular Oncology, BC Cancer Research Centre
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Samuel%2BAparicio%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Aparicio%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASamuel%2BAparicio%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-12">
                    <div class="author-tooltip-name">
                     Andrew Roth
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Computer Science, University of British Columbia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Pathology and Laboratory Medicine, University of British Columbia
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular Oncology, BC Cancer Research Centre
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Andrew%2BRoth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Roth%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAndrew%2BRoth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-13">
                    <div class="author-tooltip-name">
                     Sohrab Shah
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Computational Oncology, Department of Epidemiology and Biostatistics, Memorial Sloan Kettering Cancer Center
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sohrab%2BShah%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Shah%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASohrab%2BShah%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-14">
                    <div class="author-tooltip-name">
                     Alexandre Bouchard-Côté
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Statistics, University of British Columbia
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alexandre%2BBouchard-C%C3%B4t%C3%A9%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bouchard-C%C3%B4t%C3%A9%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlexandre%2BBouchard-C%25C3%25B4t%25C3%25A9%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:bouchard@stat.ubc.ca" class="" data-icon-position="" data-hide-link-title="0">
                        bouchard@stat.ubc.ca
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.05.06.058180v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.058180v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.058180v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.058180v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.05.06.058180v3.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.05.06.058180v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2844645" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2844645/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-3">
                             A new generation of scalable single cell whole genome sequencing (scWGS) methods allows unprecedented high resolution measurement of the evolutionary dynamics of cancer cell populations. Phylogenetic reconstruction is central to identifying sub-populations and distinguishing the mutational processes that gave rise to them. Existing phylogenetic tree building models do not scale to the tens of thousands of high resolution genomes achievable with current scWGS methods. We constructed a phylogenetic model and associated Bayesian inference procedure, sitka, specifically for scWGS data. The method is based on a novel phylogenetic encoding of copy number (CN) data, the sitka transformation, that simplifies the site dependencies induced by rearrangements while still forming a sound foundation to phylogenetic inference. The sitka transformation allows us to design novel scalable Markov chain Monte Carlo (MCMC) algorithms. Moreover, we introduce a novel point mutation calling method that incorporates the CN data and the underlying phylogenetic tree to overcome the low per-cell coverage of scWGS. We demonstrate our method on three single cell datasets, including a novel PDX series, and analyse the topological properties of the inferred trees. Sitka is freely available at
                             <a href="https://github.com/UBC-Stat-ML/sitkatree.git">
                              https://github.com/UBC-Stat-ML/sitkatree.git
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             1 Introduction
                            </h2>
                            <p id="p-6">
                             A main challenge in investigating cancer evolution is the need to resolve the subpopulation structure of a heterogeneous tumour sample. Advances in next generation scWGS have enabled more accurate, quantitative measurements of tumours as they evolve [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ,
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ,
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ,
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ]. Phylogenetic reconstruction is central to identifying clones in longitudinal xenoengraftment [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ,
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ] as well as patients [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ], and has been used to approximate the rate and timing of mutation [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ] to determine the origins and clonality of metastasis [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             Single cell cancer phylogenetics is an evolving field. Multiple approaches, spanning different study designs and data sources are reviewed in [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. Many phylogenetic inference methods such as Scite, SciΦ, OncoNEM and SciClone use the often-made infinite site model assumption and consider point mutations as input or assume a small number of leaf nodes [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ]. However, emerging single cell platforms produce up to thousands of single cell genomes and are suitable for determining copy number aberrations (CNA) [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. Compared to phylogenetic methods based on point mutations, fewer can build phylogenies from large scale CN data. Recent related work includes MEDALT [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], which models single-cell copy number lineages using a spanning tree over cells rather than a phylogeny; and SCARLET [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], which proposes a point mutation-based phylogeny inference procedure that calibrates mutation losses with copy number profiles. Distance based and agglomerative clustering methods such as neighbour joining [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ] are often used to elucidate hierarchical structures over cells, in particular, in the context of CNA, see [
                             <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], however, distance-based methods tend to produce less accurate tree reconstructions [
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ,
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ,
                             <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ].
                            </p>
                            <p id="p-8">
                             Single cell whole genome DNA sequencing provides low, yet uniform coverage [
                             <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. That is, the sequenced reads cover for each single cell about 0.1 per cent of the genome. In this setting, most point mutations are not observed in most single cells making calling SNVs difficult. However, it is possible to identify the relative copy number for segments of the genome. These copy number events can be used as phylogenetic markers.
                            </p>
                            <p id="p-9">
                             We describe sitka, a phylogenetic model and an associated Bayesian inference procedure designed specifically for inference based on CN information extracted from scWGS data (see
                             <strong>
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                             </strong>
                             ). Our method addresses two key challenges: first, each CNA event typically affects a large number of genomic sites, breaking the independence assumptions required by existing phylogenetic methods [
                             <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                              25
                             </a>
                             ,
                             <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ]; second, while detailed modelling of dependent evolutionary processes is in principle possible, they entail computational requirements incompatible with the scale of modern scWGS data [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ]. To confront these two difficulties, sitka uses a novel phylogenetic encoding of CN data, providing a statistical-computational trade-off by simplifying the site dependencies induced by rearrangements, while still forming a sound foundation to phylogenetic inference. Based on this encoding, we propose an innovative phylogenetic tree exploration move which makes the cost of Markov chain Monte Carlo (MCMC) iterations bounded by O(|C |+ |L |), where |C |is the number of cells and L is the number of loci. In contrast, existing off-the-shelf likelihood-based methods incur an iteration cost of O(|C| |L|) [
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ,
                             <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ]. Moreover, the novel move considers an exponential number of neighbouring trees whereas off-the-shelf moves consider a polynomial size set of neighbours.
                            </p>
                            <p id="p-10">
                             Sitka’s workflow proceeds by partitioning the CN information extracted from scWGS data of each cell into bins of fixed size (500Kb) with an integer CN state associated with each bin. This input data is then transformed into a binary format that captures CN changes, but not their direction or magnitude. Conditional on this binary encoding, sitka then yields an approximate posterior distribution on compatible phylogenetic trees.
                            </p>
                            <p id="p-11">
                             Potential applications of sitka include lineage tracing and subclonal structure identification. In lineage tracing the goal is to relate single cells to their ancestors based on genomic markers. This is especially useful in experimental designs where multiple samples from the same subject exist, e.g., multi-region or timeseries studies [
                             <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]. In subclonal structure identification, the topology of the inferred phylogeny can be used to make inferences about the evolutionary forces acting on the trees [
                             <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                              29
                             </a>
                             ].
                            </p>
                            <p id="p-12">
                             We compare sitka with other tree inference methods on three real-world datasets, including triple negative breast cancer patient derived xenograft samples, high grade serous ovarian primary and matched relapse samples. Since the true phylogeny is unknown, we design a phylogenetic goodness-of-fit framework to quantitatively assess the performance of our method and to visualize reconstruction confidence as well as violations of our assumptions.
                            </p>
                            <p id="p-13">
                             We use the sitka inferred trees to analyse the topological properties of the real-world datasets. Finally, we introduce a model extension that enables the placement of single nucleotide variants (SNV) with high levels of missingness on a tree inferred from the CN data.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             2 Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              2.1 Sitka: scalable single cell phylogenetic tree inference
                             </h3>
                             <p id="p-14">
                              <strong>
                               <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                              </strong>
                              shows the workflow of the sitka method. Sitka is based on a transformation of single cell copy number matrices retaining only presence or absence of changes in copy number profiles between contiguous genomic bins. This transformation allows us to approximate a complex evolutionary process (integer-valued copy numbers, prone to a high degree of homoplasy and dense dependence structure across sites) using a probabilistic version of a perfect phylogeny (see
                              <strong>
                               <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                Fig. 1
                               </a>
                              </strong>
                              ). We leverage the special structure created by the change point transformation to build a special purpose MCMC kernel, which has better computational scalability per move compared to classical phylogenetic kernels (Methods
                              <a id="xref-sec-22-1" class="xref-sec" href="#sec-22">
                               section 9.4.3
                              </a>
                              ).
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F1.large.jpg?width=800&height=600&carousel=1" title="Description of the process involved in the construction of markers, the input to the sitka model. A bin is a contiguous set of genomic positions. Each pair of consecutive bins (e.g., bins 1 and 2 in (a)) is associated with a marker (e.g., marker 1) that measures for each individual cell, whether there is a difference between the CNA states of the two bins. (a) The observed CNA matrix for a subset of bins on a chromosome. The rows are sequenced single cells, and the columns are bins. The CN states are colour-coded. (b) The three markers shown are associated with the four bins. Each marker records the presence (black) or absence (white) of a CN state change between a pair of consecutive bins. Note that in the CNA matrix, there is a CN change at row 3 from bin 1 to bin 2 (CN state 3 to 6). This is reflected in the marker matrix, at row 3 of marker 1 with a black square. There are no changes between bins 2 and 3 across any rows in the CNA matrix. This is reflected in marker 2 comprising all white squares. (c) For visualisation purposes, the CNA matrix can be interlaced with the marker matrix to more clearly show where the CNA changes occur. Each column of the marker matrix is inserted between the associated pair of columns in the CNA matrix. The resulting matrix is an example of an augmented view that combines data from two or more sources (here the CNA matrix and the marker matrix). In an augmented view, we call columns from each source a channel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Description of the process involved in the construction of <em>markers</em>, the input to the sitka model.</span> A <em>bin</em> is a contiguous set of genomic positions. Each pair of consecutive bins (e.g., bins 1 and 2 in (<strong>a</strong>)) is associated with a <em>marker</em> (e.g., marker 1) that measures for each individual cell, whether there is a difference between the CNA states of the two bins. (<strong>a</strong>) The observed CNA matrix for a subset of bins on a chromosome. The rows are sequenced single cells, and the columns are bins. The CN states are colour-coded. (<strong>b</strong>) The three markers shown are associated with the four bins. Each marker records the presence (black) or absence (white) of a CN state change between a pair of consecutive bins. Note that in the CNA matrix, there is a CN change at row 3 from bin 1 to bin 2 (CN state 3 to 6). This is reflected in the marker matrix, at row 3 of marker 1 with a black square. There are no changes between bins 2 and 3 across any rows in the CNA matrix. This is reflected in marker 2 comprising all white squares. (<strong>c</strong>) For visualisation purposes, the CNA matrix can be interlaced with the marker matrix to more clearly show where the CNA changes occur. Each column of the marker matrix is inserted between the associated pair of columns in the CNA matrix. The resulting matrix is an example of an <em>augmented</em> view that combines data from two or more sources (here the CNA matrix and the marker matrix). In an augmented view, we call columns from each source a <em>channel</em>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F1.medium.gif" width="440" height="214"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F1.medium.gif" width="440" height="214"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Description of the process involved in the construction of markers, the input to the sitka model.
                               </span>
                               <p id="p-15" class="first-child">
                                A bin is a contiguous set of genomic positions. Each pair of consecutive bins (e.g., bins 1 and 2 in (
                                <strong>
                                 a
                                </strong>
                                )) is associated with a marker (e.g., marker 1) that measures for each individual cell, whether there is a difference between the CNA states of the two bins. (
                                <strong>
                                 a
                                </strong>
                                ) The observed CNA matrix for a subset of bins on a chromosome. The rows are sequenced single cells, and the columns are bins. The CN states are colour-coded. (
                                <strong>
                                 b
                                </strong>
                                ) The three markers shown are associated with the four bins. Each marker records the presence (black) or absence (white) of a CN state change between a pair of consecutive bins. Note that in the CNA matrix, there is a CN change at row 3 from bin 1 to bin 2 (CN state 3 to 6). This is reflected in the marker matrix, at row 3 of marker 1 with a black square. There are no changes between bins 2 and 3 across any rows in the CNA matrix. This is reflected in marker 2 comprising all white squares. (
                                <strong>
                                 c
                                </strong>
                                ) For visualisation purposes, the CNA matrix can be interlaced with the marker matrix to more clearly show where the CNA changes occur. Each column of the marker matrix is inserted between the associated pair of columns in the CNA matrix. The resulting matrix is an example of an augmented view that combines data from two or more sources (here the CNA matrix and the marker matrix). In an augmented view, we call columns from each source a channel.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F2.large.jpg?width=800&height=600&carousel=1" title="Pathological tree reconstruction under default observation prior. (a) The true tree reconstruction in a simple example with a balanced phylogeny with two clades of size two, and two unique markers, coloured red and blue, that distinguish the left and right clades respectively. (b) The binarised input matrix corresponding to the four cells at the two markers. The desired observation error rates should be zero and the latent and observed marker matrices should match exactly, as the perfect phylogeny assumption holds. If the observation error parameters are set to one, that is  and , then the latent marker matrix with all entries flipped as shown in (c) will have an equal likelihood under this setting as the desired latent matrix has when error rates are set to zero. (d) The incorrect tree reconstruction where the left and right clades are erroneously assigned to the blue and red markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Pathological tree reconstruction under default observation prior.</span> (<strong>a</strong>) The true tree reconstruction in a simple example with a balanced phylogeny with two clades of size two, and two unique markers, coloured red and blue, that distinguish the left and right clades respectively. (<strong>b</strong>) The binarised input matrix corresponding to the four cells at the two markers. The desired observation error rates should be zero and the latent and observed marker matrices should match exactly, as the perfect phylogeny assumption holds. If the observation error parameters are set to one, that is  and , then the latent marker matrix with all entries flipped as shown in (<strong>c</strong>) will have an equal likelihood under this setting as the desired latent matrix has when error rates are set to zero. (<strong>d</strong>) The incorrect tree reconstruction where the left and right clades are erroneously assigned to the blue and red markers.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F2.medium.gif" width="440" height="119"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F2.medium.gif" width="440" height="119"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                Pathological tree reconstruction under default observation prior.
                               </span>
                               <p id="p-16" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) The true tree reconstruction in a simple example with a balanced phylogeny with two clades of size two, and two unique markers, coloured red and blue, that distinguish the left and right clades respectively. (
                                <strong>
                                 b
                                </strong>
                                ) The binarised input matrix corresponding to the four cells at the two markers. The desired observation error rates should be zero and the latent and observed marker matrices should match exactly, as the perfect phylogeny assumption holds. If the observation error parameters are set to one, that is
                                <span class="inline-formula" id="inline-formula-1">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/F2/embed/inline-graphic-1.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/F2/embed/inline-graphic-1.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and
                                <span class="inline-formula" id="inline-formula-2">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/F2/embed/inline-graphic-2.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/F2/embed/inline-graphic-2.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                , then the latent marker matrix with all entries flipped as shown in (
                                <strong>
                                 c
                                </strong>
                                ) will have an equal likelihood under this setting as the desired latent matrix has when error rates are set to zero. (
                                <strong>
                                 d
                                </strong>
                                ) The incorrect tree reconstruction where the left and right clades are erroneously assigned to the blue and red markers.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F3.large.jpg?width=800&height=600&carousel=1" title="Workflow of sitka. (a) Sitka takes copy number calls data from a heterogeneous single-cell population. The cells (rows of the copy number matrix) are randomly sorted. (b) A lossy binary transformation is applied to obtain markers data. (Methods section 9.2 and Fig. 1). Note that each single-cell is now represented by the presence or absence of CN changes between consecutive bins. (c) The boundary conditions are smoothed to account for cell-specific marker misalignment. (Methods section 9.3) to correct for this marker misalignment. Note how the columns in the inset in panel-c are less noisy than their counterpart in panel-b. (d) A subset of markers present in at least 5 percent of the cells are chosen for input to the tree inference algorithm. (e) An MCMC algorithm efficiently explores the tree space. (f) An example of an edge-insertion. The two insets are zoomed in from panel-e. Each inset depicts a subtree, where red diamonds and blue circles denote marker nodes and single-cells respectively. Also see Supplementary Fig. 1 for details. (g) The indicator matrix of all post-burn-in MCMC trees are averaged to generate a matrix indicating the posterior probability of a cell being attached to a marker (Methods section 9.4.5). (h) The copy number data in (a) is sorted according to the inferred consensus tree, shown on the left of the matrix. (i) The inset shows the tuple of marker columns in the context of the copy number calls, namely inf. (inferred markers, i.e., latent state xc,l), post. (posterior probability of the latent state xc,l), and obs. (observed markers), interlaced with the CN columns (similar to Supplementary Fig. 1). The results are from the SA535 dataset, a triple negative breast cancer patient derived xenograft sample (Methods section 2.2).)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Workflow of sitka.</span> (<strong>a</strong>) Sitka takes copy number calls data from a heterogeneous single-cell population. The cells (rows of the copy number matrix) are randomly sorted. (<strong>b</strong>) A lossy binary transformation is applied to obtain markers data. (Methods section 9.2 and <strong>Fig. 1</strong>). Note that each single-cell is now represented by the presence or absence of CN changes between consecutive bins. (<strong>c</strong>) The boundary conditions are smoothed to account for cell-specific marker misalignment. (Methods section 9.3) to correct for this marker misalignment. Note how the columns in the inset in panel-<strong>c</strong> are less noisy than their counterpart in panel-<strong>b</strong>. (<strong>d</strong>) A subset of markers present in at least 5 percent of the cells are chosen for input to the tree inference algorithm. (<strong>e</strong>) An MCMC algorithm efficiently explores the tree space. (<strong>f</strong>) An example of an edge-insertion. The two insets are zoomed in from panel-<strong>e</strong>. Each inset depicts a subtree, where red diamonds and blue circles denote marker nodes and single-cells respectively. Also see <strong>Supplementary Fig. 1</strong> for details. (<strong>g</strong>) The indicator matrix of all post-burn-in MCMC trees are averaged to generate a matrix indicating the posterior probability of a cell being attached to a marker (Methods section 9.4.5). (<strong>h</strong>) The copy number data in (<strong>a</strong>) is sorted according to the inferred consensus tree, shown on the left of the matrix. (<strong>i</strong>) The inset shows the tuple of marker columns in the context of the copy number calls, namely <strong>inf</strong>. (inferred markers, i.e., latent state <em>x</em><sub><em>c,l</em></sub>), <strong>post</strong>. (posterior probability of the latent state <em>x</em><sub><em>c,l</em></sub>), and <strong>obs</strong>. (observed markers), interlaced with the CN columns (similar to <strong>Supplementary Fig. 1</strong>). The results are from the <em>SA</em>535 dataset, a triple negative breast cancer patient derived xenograft sample (Methods section 2.2).)</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F3.medium.gif" width="398" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F3.medium.gif" width="398" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                Workflow of sitka.
                               </span>
                               <p id="p-17" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) Sitka takes copy number calls data from a heterogeneous single-cell population. The cells (rows of the copy number matrix) are randomly sorted. (
                                <strong>
                                 b
                                </strong>
                                ) A lossy binary transformation is applied to obtain markers data. (Methods
                                <a id="xref-sec-15-1" class="xref-sec" href="#sec-15">
                                 section 9.2
                                </a>
                                and
                                <strong>
                                 <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                                  Fig. 1
                                 </a>
                                </strong>
                                ). Note that each single-cell is now represented by the presence or absence of CN changes between consecutive bins. (
                                <strong>
                                 c
                                </strong>
                                ) The boundary conditions are smoothed to account for cell-specific marker misalignment. (Methods
                                <a id="xref-sec-16-1" class="xref-sec" href="#sec-16">
                                 section 9.3
                                </a>
                                ) to correct for this marker misalignment. Note how the columns in the inset in panel-
                                <strong>
                                 c
                                </strong>
                                are less noisy than their counterpart in panel-
                                <strong>
                                 b
                                </strong>
                                . (
                                <strong>
                                 d
                                </strong>
                                ) A subset of markers present in at least 5 percent of the cells are chosen for input to the tree inference algorithm. (
                                <strong>
                                 e
                                </strong>
                                ) An MCMC algorithm efficiently explores the tree space. (
                                <strong>
                                 f
                                </strong>
                                ) An example of an edge-insertion. The two insets are zoomed in from panel-
                                <strong>
                                 e
                                </strong>
                                . Each inset depicts a subtree, where red diamonds and blue circles denote marker nodes and single-cells respectively. Also see
                                <strong>
                                 Supplementary Fig. 1
                                </strong>
                                for details. (
                                <strong>
                                 g
                                </strong>
                                ) The indicator matrix of all post-burn-in MCMC trees are averaged to generate a matrix indicating the posterior probability of a cell being attached to a marker (Methods
                                <a id="xref-sec-24-1" class="xref-sec" href="#sec-24">
                                 section 9.4.5
                                </a>
                                ). (
                                <strong>
                                 h
                                </strong>
                                ) The copy number data in (
                                <strong>
                                 a
                                </strong>
                                ) is sorted according to the inferred consensus tree, shown on the left of the matrix. (
                                <strong>
                                 i
                                </strong>
                                ) The inset shows the tuple of marker columns in the context of the copy number calls, namely
                                <strong>
                                 inf
                                </strong>
                                . (inferred markers, i.e., latent state x
                                <sub>
                                 c,l
                                </sub>
                                ),
                                <strong>
                                 post
                                </strong>
                                . (posterior probability of the latent state x
                                <sub>
                                 c,l
                                </sub>
                                ), and
                                <strong>
                                 obs
                                </strong>
                                . (observed markers), interlaced with the CN columns (similar to
                                <strong>
                                 Supplementary Fig. 1
                                </strong>
                                ). The results are from the SA535 dataset, a triple negative breast cancer patient derived xenograft sample (Methods
                                <a id="xref-sec-4-1" class="xref-sec" href="#sec-4">
                                 section 2.2
                                </a>
                                ).)
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-18">
                              We visualise the input data to sitka in a colour-coded matrix exemplified in
                              <strong>
                               <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                                Supplementary Fig. 1-a
                               </a>
                              </strong>
                              . Each row in the matrix corresponds to an individual cell that has been sequenced in a single-cell platform. Each column in the matrix is a locus that is represented by a bin (a contiguous set of genomic positions). We assume that the integer copy number of each bin has been estimated as a preprocessing step, e.g., using a hidden Markov model [
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ]. In
                              <strong>
                               <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                                Supplementary Fig. 1-a
                               </a>
                              </strong>
                              the copy number state is encoded by the colour of each entry in the matrix.
                             </p>
                             <p id="p-19">
                              The output of sitka includes two types of directed rooted trees. Type I is the tree used for MCMC sampling in the inference procedure, and type II, which is derived from type I, is used in visualisation (
                              <strong>
                               <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                                Fig. 4-a-c
                               </a>
                              </strong>
                              ). The set of nodes in a type I tree is given by the union of the cells, the CN change points (markers) under study, and a root node v
                              <sup>
                               *
                              </sup>
                              . The topology of a type I tree bears the following phylogenetic interpretation: given a cell c in the tree, c is hypothesized to harbour the markers in the shortest path between c and the root node v
                              <sup>
                               *
                              </sup>
                              , and only those markers. We enforce the constraint that all cells are leaf nodes, while markers can be either internal or leaf nodes. Markers placed at the leaves are interpreted as outliers, for example measured CN change points that are false positives.
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F4.large.jpg?width=800&height=600&carousel=1" title="Results over real-datasets and benchmarking against baseline methods. (a), (b), and (c) show the consensus tree and marker-space matrix for the OV A, SA501, and SA535 datasets respectively. (d) Comparison to baseline methods.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Results over real-datasets and benchmarking against baseline methods.</span> (<strong>a</strong>), (<strong>b</strong>), and (<strong>c</strong>) show the consensus tree and marker-space matrix for the <em>OV A, SA</em>501, and <em>SA</em>535 datasets respectively. (<strong>d</strong>) Comparison to baseline methods.)</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F4.medium.gif" width="440" height="337"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F4.medium.gif" width="440" height="337"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                Results over real-datasets and benchmarking against baseline methods.
                               </span>
                               <p id="p-20" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ), (
                                <strong>
                                 b
                                </strong>
                                ), and (
                                <strong>
                                 c
                                </strong>
                                ) show the consensus tree and marker-space matrix for the OV A, SA501, and SA535 datasets respectively. (
                                <strong>
                                 d
                                </strong>
                                ) Comparison to baseline methods.)
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-21">
                              To convert a type I tree to a type II tree, we remove from the type I tree all marker nodes that are leaf nodes, i.e., markers that are not present in any cells. We also collapse into a single node, the list of connected marker nodes that have exactly one descendent (i.e., chains).
                              <strong>
                               <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                Fig. 5
                               </a>
                              </strong>
                              shows a small type I tree, its transformation to a type II tree and the respective marker matrix. We visualise the input matrix and the estimated tree simultaneously by sorting the individual cells (rows of the matrix) such that they line up with the position of the corresponding leaves of the tree.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F5.large.jpg?width=800&height=600&carousel=1" title="Visualisation of a small tree and its corresponding marker matrix. A small type I tree t (a), its transformation into a type II tree (b), and the corresponding marker matrix x = (xc,l) (c). The red nodes in (a) correspond from top to bottom to markers 2, 3, 1 in this order. Given a tree t, the latent marker matrix x is a deterministic function x = x(t). Note that the clade comprising single-cells 3 and 4 has support in both markers 1 and 3. For clarity, we do not visualise type I trees, but plot their transformation, i.e., type II trees as follows. We remove from the type I tree all marker nodes that have xc,l = 0 for all single-cells c. Lists of connected edges that have exactly one descendent (i.e., chains) are also collapsed into a single edge, e.g., the edge corresponding to markers 2 and 3 are collapsed into one edge (since marker 2 has only one descendent, namely single-cell 2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Visualisation of a small tree and its corresponding marker matrix.</span> A small type I tree <em>t</em> (<strong>a</strong>), its transformation into a type II tree (<strong>b</strong>), and the corresponding marker matrix <em>x</em> = (<em>x</em><sub><em>c,l</em></sub>) (<strong>c</strong>). The red nodes in (<strong>a</strong>) correspond from top to bottom to markers 2, 3, 1 in this order. Given a tree <em>t</em>, the latent marker matrix <em>x</em> is a deterministic function <em>x</em> = <em>x</em>(<em>t</em>). Note that the clade comprising single-cells 3 and 4 has support in both markers 1 and 3. For clarity, we do not visualise type I trees, but plot their transformation, i.e., type II trees as follows. We remove from the type I tree all marker nodes that have <em>x</em><sub><em>c,l</em></sub> = 0 for all single-cells <em>c</em>. Lists of connected edges that have exactly one descendent (i.e., chains) are also collapsed into a single edge, e.g., the edge corresponding to markers 2 and 3 are collapsed into one edge (since marker 2 has only one descendent, namely single-cell 2).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F5.medium.gif" width="440" height="153"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F5.medium.gif" width="440" height="153"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Visualisation of a small tree and its corresponding marker matrix.
                               </span>
                               <p id="p-22" class="first-child">
                                A small type I tree t (
                                <strong>
                                 a
                                </strong>
                                ), its transformation into a type II tree (
                                <strong>
                                 b
                                </strong>
                                ), and the corresponding marker matrix x = (x
                                <sub>
                                 c,l
                                </sub>
                                ) (
                                <strong>
                                 c
                                </strong>
                                ). The red nodes in (
                                <strong>
                                 a
                                </strong>
                                ) correspond from top to bottom to markers 2, 3, 1 in this order. Given a tree t, the latent marker matrix x is a deterministic function x = x(t). Note that the clade comprising single-cells 3 and 4 has support in both markers 1 and 3. For clarity, we do not visualise type I trees, but plot their transformation, i.e., type II trees as follows. We remove from the type I tree all marker nodes that have x
                                <sub>
                                 c,l
                                </sub>
                                = 0 for all single-cells c. Lists of connected edges that have exactly one descendent (i.e., chains) are also collapsed into a single edge, e.g., the edge corresponding to markers 2 and 3 are collapsed into one edge (since marker 2 has only one descendent, namely single-cell 2).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-23">
                              Sitka uses change points as phylogenetic traits modelled using a relaxation of the perfect phylogeny assumption. For a phylogenetic tree, the perfect phylogeny assumption holds if and only if for all markers l, l changes at most once from its ancestral state over the tree. Change points arising from non-overlapping CNA events (i.e., such that the genomic locations affected by the CN event do not intersect) preserve the perfect phylogeny assumption.
                              <strong>
                               <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                Fig. 6
                               </a>
                              </strong>
                              shows examples of overlapping CNA events and their effect on markers. The two scenarios that can lead to the violation of the perfect phylogeny assumption are (i) when a CNA gain event is followed by an overlapping loss event or (ii) when a loss event is followed by an overlapping loss event, and the second event removes either end-point of the first event. For both (i) and (ii), a violation occurs only when the second overlapping event hits the same copy as the first event.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F6.large.jpg?width=800&height=600&carousel=1" title="Perfect phylogeny and effects of overlapping CNA events. The effects of overlapping CNA events on the perfect phylogeny assumption. A segment of a chromosome with five consecutive bins and their four corresponding markers are shown. Each panel follows the CN states interlaced with markers for a cell at the ancestral state (top), after a CNA event (middle), and after a second overlapping CNA event (bottom). The numbers in the CNA squares show the integer CN state (e.g., the ancestral state has two copies of the 5-bins long segment). (a) Two overlapping CNA gains maintain the perfect phylogeny assumption. If the infinite site assumption holds, it is unlikely for the end-points of the two gain events to exactly match. The same argument holds for a CNA loss followed by a CNA gain event. Note that in these cases, once a change point is acquired, it is not lost. (b) If a loss event is followed by another loss event in which either end-points of the first event is removed, the perfect phylogeny assumption will be violated (e.g., marker 3 is lost after the second loss event). Note that a violation does not occur if the loss events hit different copies of a segment. (c) Similarly, if a gain event is followed by a loss event, only if the latter erases the end-points of the former is the perfect phylogeny violated. Note how marker 2 and marker 3 are lost after the second CNA event." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Perfect phylogeny and effects of overlapping CNA events.</span> The effects of overlapping CNA events on the perfect phylogeny assumption. A segment of a chromosome with five consecutive bins and their four corresponding markers are shown. Each panel follows the CN states interlaced with markers for a cell at the ancestral state (top), after a CNA event (middle), and after a second overlapping CNA event (bottom). The numbers in the CNA squares show the integer CN state (e.g., the ancestral state has two copies of the 5-bins long segment). (<strong>a</strong>) Two overlapping CNA gains maintain the perfect phylogeny assumption. If the infinite site assumption holds, it is unlikely for the end-points of the two gain events to exactly match. The same argument holds for a CNA loss followed by a CNA gain event. Note that in these cases, once a change point is acquired, it is not lost. (<strong>b</strong>) If a loss event is followed by another loss event in which either end-points of the first event is removed, the perfect phylogeny assumption will be violated (e.g., marker 3 is lost after the second loss event). Note that a violation does not occur if the loss events hit different copies of a segment. (<strong>c</strong>) Similarly, if a gain event is followed by a loss event, only if the latter erases the end-points of the former is the perfect phylogeny violated. Note how marker 2 and marker 3 are lost after the second CNA event.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F6.medium.gif" width="440" height="349"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F6.medium.gif" width="440" height="349"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Perfect phylogeny and effects of overlapping CNA events.
                               </span>
                               <p id="p-24" class="first-child">
                                The effects of overlapping CNA events on the perfect phylogeny assumption. A segment of a chromosome with five consecutive bins and their four corresponding markers are shown. Each panel follows the CN states interlaced with markers for a cell at the ancestral state (top), after a CNA event (middle), and after a second overlapping CNA event (bottom). The numbers in the CNA squares show the integer CN state (e.g., the ancestral state has two copies of the 5-bins long segment). (
                                <strong>
                                 a
                                </strong>
                                ) Two overlapping CNA gains maintain the perfect phylogeny assumption. If the infinite site assumption holds, it is unlikely for the end-points of the two gain events to exactly match. The same argument holds for a CNA loss followed by a CNA gain event. Note that in these cases, once a change point is acquired, it is not lost. (
                                <strong>
                                 b
                                </strong>
                                ) If a loss event is followed by another loss event in which either end-points of the first event is removed, the perfect phylogeny assumption will be violated (e.g., marker 3 is lost after the second loss event). Note that a violation does not occur if the loss events hit different copies of a segment. (
                                <strong>
                                 c
                                </strong>
                                ) Similarly, if a gain event is followed by a loss event, only if the latter erases the end-points of the former is the perfect phylogeny violated. Note how marker 2 and marker 3 are lost after the second CNA event.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              Imposing a perfect phylogeny on the observed change points is restrictive, as we expect both violations of the assumptions (e.g., due to homoplasy), and measurement noise. To address this we use an observation model (Methods
                              <a id="xref-sec-18-1" class="xref-sec" href="#sec-18">
                               section 9.4.1
                              </a>
                              ) which assigns positive probability to arbitrary deviations from the perfect phylogeny assumption, while encouraging configurations where few markers and cells are involved in violations. Subsequently we impose the perfect phylogeny assumption on a latent maker matrix defined as follows. Given a type I tree t, the latent marker matrix x is a deterministic function x = x(t). We compute x : t → {0, 1 }
                              <sup>
                               C×L
                              </sup>
                              by setting x
                              <sub>
                               c,l
                              </sub>
                              = 1 if the single-cell c is a descendent of the marker node l in tree t, and otherwise x
                              <sub>
                               c,l
                              </sub>
                              = 0. We use y
                              <sub>
                               c,l
                              </sub>
                              to refer to the observed change point l in individual cell c (Methods
                              <a id="xref-sec-18-2" class="xref-sec" href="#sec-18">
                               section 9.4.1
                              </a>
                              ).
                             </p>
                             <p id="p-26">
                              Synthetic experiments show that sitka’s performance decreases roughly linearly as a function of the rate of the key types of expected violation of the perfect phylogeny assumption (
                              <strong>
                               <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                Fig. 7-a
                               </a>
                              </strong>
                              ,
                              <strong>
                               b
                              </strong>
                              , Methods
                              <a id="xref-sec-26-1" class="xref-sec" href="#sec-26">
                               section 9.5
                              </a>
                              ).
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F7.large.jpg?width=800&height=600&carousel=1" title="Synthetic experiments and an application to point mutation placement. (a) RF distance of sitka tree estimate to the best-possible tree. The first plot holds pis constant at zero. The second plot holds ploss constant at 0. The third plot varies pis = ploss jointly. (b) Estimation of violation rates in real data and a set of synthetic data. (c) Over 20,000 SNV’s with high levels of missingness are placed on a backbone tree inferred from the CNA data for SA535." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Synthetic experiments and an application to point mutation placement.</span> (<strong>a</strong>) RF distance of sitka tree estimate to the best-possible tree. The first plot holds <em>p</em><sub><em>is</em></sub> constant at zero. The second plot holds <em>p</em><sub><em>loss</em></sub> constant at 0. The third plot varies <em>p</em><sub><em>is</em></sub> = <em>p</em><sub><em>loss</em></sub> jointly. <strong>(b)</strong> Estimation of violation rates in real data and a set of synthetic data. (<strong>c)</strong> Over 20,000 SNV’s with high levels of missingness are placed on a backbone tree inferred from the CNA data for <em>SA</em>535.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F7.medium.gif" width="440" height="296"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F7.medium.gif" width="440" height="296"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Synthetic experiments and an application to point mutation placement.
                               </span>
                               <p id="p-27" class="first-child">
                                (
                                <strong>
                                 a
                                </strong>
                                ) RF distance of sitka tree estimate to the best-possible tree. The first plot holds p
                                <sub>
                                 is
                                </sub>
                                constant at zero. The second plot holds p
                                <sub>
                                 loss
                                </sub>
                                constant at 0. The third plot varies p
                                <sub>
                                 is
                                </sub>
                                = p
                                <sub>
                                 loss
                                </sub>
                                jointly.
                                <strong>
                                 (b)
                                </strong>
                                Estimation of violation rates in real data and a set of synthetic data. (
                                <strong>
                                 c)
                                </strong>
                                Over 20,000 SNV’s with high levels of missingness are placed on a backbone tree inferred from the CNA data for SA535.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              2.2 Performance of sitka relative to alternative approaches
                             </h3>
                             <p id="p-28">
                              We compare the performance of sitka to alternative approaches (Supplementary section 2) on three scWGS datasets introduced here (
                              <strong>
                               <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                                Fig. 4-a-c
                               </a>
                              </strong>
                              ). The first dataset, SA535, is generated for this project and contains 679 cells from three passages of a triple negative breast cancer (TNBC) patient derived xenograft sample. Passages X1, X5, and X8 had 62, 369, and 231 cells post quality filtering (Methods
                              <a id="xref-sec-14-1" class="xref-sec" href="#sec-14">
                               section 9.1
                              </a>
                              ) respectively. We also include 17 mostly diploid control cells. These cells are combined to generate the input to the analysis pipeline (
                              <strong>
                               <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                                Supplementary Fig. 2
                               </a>
                              </strong>
                              ). The second dataset, labelled OV A, consists of cells from three samples taken from a patient with high grade serous (HGS) ovarian cancer. The first sample, SA1090, was from an ascites pre-treatment, while SA922 was from an ascites post-treatment. The third sample, SA921, was taken from the ovary. See
                              <strong>
                               <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                                Supplementary Fig. 3
                               </a>
                              </strong>
                              for the tree and the CNA profile heatmap for this dataset. The final dataset, SA501 [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ], is another TNBC xenograft tumour from 6 untreated passages, namely X2, X5, X6, X8, X11, and X15. After filtering, 515, 236, 328, 189, 836, and 308 cells remain in each passage respectively (for a total of 2,412 cells, see
                              <strong>
                               <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                                Supplementary Fig. 4
                               </a>
                              </strong>
                              ).
                              <strong>
                               Supplementary Table
                              </strong>
                              1 shows the attrition after each step of filtering cells per passage in each dataset.
                             </p>
                             <p id="p-29">
                              To evaluate inferred trees from sitka and other tree reconstruction methods, we use a goodness of fit performance metric, which compares the compatibility of observed CN change points with a given phylogeny using Youden’s J index (Methods
                              <a id="xref-sec-30-1" class="xref-sec" href="#sec-30">
                               section 9.6
                              </a>
                              ,
                              <strong>
                               <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                                Fig. 4-d
                               </a>
                              </strong>
                              ). Sitka has the highest Youden’s index across all three datasets.
                              <kbd>
                               UPGMA
                              </kbd>
                              and
                              <kbd>
                               WPGMA
                              </kbd>
                              perform similarly on SA501 and SA535.
                              <kbd>
                               UPGMA
                              </kbd>
                              performs slightly better than
                              <kbd>
                               WPGMA
                              </kbd>
                              on the OV A dataset.
                              <kbd>
                               HDBSCAN
                              </kbd>
                              has a close but slightly smaller Youden’s index than
                              <kbd>
                               UPGMA
                              </kbd>
                              over the SA535 and OV A datasets, but performs marginally better on SA501. NJ trails
                              <kbd>
                               WPGMA
                              </kbd>
                              on SA501 and the OV A datasets, and has the lowest Youden’s index on SA535. MrBayes performs well on the smallest dataset, SA535, with
                              <kbd>
                               MrBayes-np2
                              </kbd>
                              and
                              <kbd>
                               MrBayes-np8
                              </kbd>
                              performing similar to
                              <kbd>
                               WPGMA
                              </kbd>
                              , and
                              <kbd>
                               MrBayesWithBinaryInput
                              </kbd>
                              having achieved the second highest Youden’s index. On the OV A data,
                              <kbd>
                               MrBayesWithBinaryInput
                              </kbd>
                              and
                              <kbd>
                               MrBayes-np2
                              </kbd>
                              trail behind
                              <kbd>
                               medicc2
                              </kbd>
                              and
                              <kbd>
                               MEDALT
                              </kbd>
                              , while
                              <kbd>
                               MEDALTWithBinaryInput
                              </kbd>
                              has the lowest Youden’s index among all methods on all datasets.
                              <kbd>
                               MrBayesWithBinaryInput
                              </kbd>
                              and
                              <kbd>
                               MrBayes-np2
                              </kbd>
                              trail behind NJ over the SA501 dataset.
                              <kbd>
                               medicc2
                              </kbd>
                              and
                              <kbd>
                               MEDALT
                              </kbd>
                              without binary input, ran with default settings, did not yield a result given our available computational budget, with the former not finishing after several days, and the latter running out of memory (144 GB). Following [
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ], we run
                              <kbd>
                               MrBayes
                              </kbd>
                              for 10,000,000 generations.
                              <kbd>
                               MrBayes-np8
                              </kbd>
                              had completed only 278,000 iterations running on SA501 after several days. The results in this comparison suggest that sitka performs better than the baseline methods. Running sitka on the real-world datasets took on average 22.3, 46.6, and 12.9 hours for the OV A, SA501, and SA535 datasets respectively, on a Linux workstation with 72 Intel Xeon Platinum 8272CL 2.60GHz CPU processors and 144 GB of memory.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              2.3 Single cell resolution phylogenetic inference in PDX
                             </h3>
                             <p id="p-30">
                              Here we analyse the foregoing three multi-sample datasets. To visualise the tree inference results we arrange the inferred consensus tree t (Methods
                              <a id="xref-sec-24-2" class="xref-sec" href="#sec-24">
                               section 9.4.5
                              </a>
                              ) and the cell-by-bin CN matrix side by side where the rows of the matrix correspond to the position of individual cells on the tree and the markers are arranged by their genomic position (
                              <strong>
                               <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                                Fig. 3-h
                               </a>
                              </strong>
                              ).
                              <strong>
                               <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                                Fig. 4
                               </a>
                              </strong>
                              -
                              <strong>
                               a-c
                              </strong>
                              shows examples of the multi-channel visualisation where each marker is represented by a tuple of three different data-types or channels, namely: (i) the latent markers induced by the consensus tree, x(t); (ii) the matrix of marginal posterior probability that cell c is a descendent of marker l, computed via the average
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              (
                              <strong>
                               <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                                Fig. 3-g
                               </a>
                              </strong>
                              , Methods
                              <a id="xref-sec-24-3" class="xref-sec" href="#sec-24">
                               section 9.4.5
                              </a>
                              ); and (iii) the sitka transformed input data y
                              <sub>
                               c,l
                              </sub>
                              .
                             </p>
                             <p id="p-31">
                              We use this view to assess potential discrepancies between the input data and the inferred tree. In most cells and markers (as quantified in
                              <strong>
                               <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                                Supplementary Fig. 9.6
                               </a>
                              </strong>
                              ), the observed data is in close agreement with the inferred tree. In the following we provide some examples of disagreements. Consider first the ChrX in the OV 2295 dataset (
                              <strong>
                               <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                                Fig. 4-a
                               </a>
                              </strong>
                              ). ChrX has a long orange band (inferred marker in channel (i)) not matched by a black band (observed marker in channel (iii)) suggesting that a perfect phylogeny violation has occurred. The pattern in this marker is consistent with the presence of an ancestral event followed by a deletion. In
                              <strong>
                               <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                                Fig. 4-b
                               </a>
                              </strong>
                              , a set of diploid cells are attached to the root of the tree. These are control cells included in the experiment and correspond to a region in the bottom of the matrix with no inferred markers (orange bands) and almost no observed markers (black bands). In this dataset, there are change points where the observed marker has a high density (black band), but the tree is reconstructed with the marker absent (no matching orange band). Examples can be found in Chr1, Chr7 and Chr16. One possible explanation could be that the end-points of each event were detected as slightly shifted across cells. For instance, in
                              <strong>
                               <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                                Supplementary Fig. 4
                               </a>
                              </strong>
                              there are two bins with an amplification (CN state equal to three) in Chr1p where cells that harbour a mutation in the first bin appear not to have a mutation in the second bin, suggesting that the same event was called in the first bin in some cells, and in the second bin in others. An alternative hypothesis is that the cells in this dataset have a mutator phenotype that promotes CN mutations in these bins.
                             </p>
                             <p id="p-32">
                              <strong>
                               <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                                Supplementary Fig. 5
                               </a>
                              </strong>
                              shows the distribution of mismatch rates for each dataset, defined as the fraction of times that the observed and inferred markers do not match, i.e.,
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              for l ϵ L (corresponding to the black and orange bands in
                              <strong>
                               <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                                Fig. 4-a
                               </a>
                              </strong>
                              ). In OV 2295, 41 markers (11%) have a mismatch rate of over 50%, where marker chr15_67000001_67500000 has the highest mismatch rate at 70%. In SA501, 30 markers (11%) have a mismatch rate of over 50%, 13 of which (5%) have a mismatch rate of over 75%. SA535 has the lowest maximum mismatch rate at 49% (marker 15_72000001_72500000).
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              2.4 Placement of SNVs using the CNA inferred tree
                             </h3>
                             <p id="p-33">
                              To determine the presence or absence of SNVs in cells using data with high levels of missingness, we develop an extension of sitka, the sitka-snv model. Given single cell level variant read counts, the model incorporates CN data to place SNVs on the sitka-inferred phylogenetic tree. This backbone CN tree provides a principled way to pool statistical strength across groups of single cells sequenced at low coverage, including data from the DLP+ platform [
                              <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              ]. The output of the sitka-snv model is an extended tree that has marker nodes that comprise SNVs in addition to the original CNAs.
                             </p>
                             <p id="p-34">
                              The SNVs are added to the existing CNA-based tree with the computational complexity of O(|C| + |L |) per SNV.
                              <strong>
                               <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                Fig. 7-c
                               </a>
                              </strong>
                              shows the result of SNV placement with the number of variant reads in SA535, corresponding to the tree shown in
                              <strong>
                               <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                                Fig. 4-c
                               </a>
                               .
                               <a id="xref-fig-23-1" class="xref-fig" href="#F23">
                                Supplementary Figs. 14
                               </a>
                               ,
                               <a id="xref-fig-24-1" class="xref-fig" href="#F24">
                                15
                               </a>
                              </strong>
                              , and
                              <strong>
                               <a id="xref-fig-25-1" class="xref-fig" href="#F25">
                                16
                               </a>
                              </strong>
                              show the number of variant reads and the matching SNV call probabilities for the SA535, OV A and SA501 datasets respectively. Sitka and sitka-snv provide a comprehensive genomic analysis tool for large scale low-coverage scWGS.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-7">
                            <h2 class="">
                             3 Discussion
                            </h2>
                            <p id="p-35">
                             Our method ignores certain pairwise dependencies induced by copy number change events having two end-points (except in cases where one of the end points is the end of the chromosome, e.g., whole-chromosome-arm events). This artificial duplication of the events having two input end-points can lead to the method being overconfident, i.e., outputting credible intervals that are smaller than expected. This is partly a reason for focusing more on point estimates (consensus trees) in this work, which we expect are less affected by this phenomenon (see
                             <a id="xref-sec-28-1" class="xref-sec" href="#sec-28">
                              Section 9.5.2
                             </a>
                             ).
                            </p>
                            <p id="p-36">
                             In the present study we use data in which the genome of the single cells CNA profiles are partitioned into bins of a fixed size (500Kb), each assigned a constant integer CN state. The relatively large size is due to the low coverage inherent to the scWGS platform, but it implies that the same bin may harbour multiple CNA events. Biological processes that result in complex DNA rearrangements could further increase the probability of having two hits in one bin [
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              31
                             </a>
                             ,
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              32
                             </a>
                             ]. Post hoc inspection is necessary to rule out large violations of our assumptions. This highlights the importance of our goodness-of-fit and visualisation methods as they help detecting such violations.
                            </p>
                            <p id="p-37">
                             We note that we lose information when applying the sitka transformation. This transformation is necessary for the computational feasibility of the likelihood. In absence of this relaxation, the computational complexity of each iteration of the MCMC algorithm may no longer be bounded by O(|C| + |L|). Indeed, the approach to efficiently compute the likelihood depends on binary latent variables with specific perfect phylogeny assumptions, and it is not clear how to generalize this calculation to models that keep track of the evolution of more detailed CN state information along the phylogeny.
                            </p>
                            <p id="p-38">
                             Structural variations such as chromothripsis, that affect multiple segments of the genome at the same time, make it difficult to determine the rate of CNA events and suggest that CNA events may not be suitable molecular clocks to estimate branch lengths. One possible remedy is to first infer the tree topology via markers based on CNA events and then conditioned on this topology, add SNVs to the tree. The number of SNVs on each edge of the tree may be used to inform branch lengths.
                            </p>
                            <p id="p-39">
                             Our preprocessing pipeline excludes multiple cells from the analysis (see
                             <strong>
                              Supplementary Table
                             </strong>
                             1). We filter out a fraction of cells to remove contaminated cells, either doublets (DNA material from two cells that is inadvertently merged) or mouse cells (in our real world datasets, we study human tumours that were transplanted into mice), cells with too many erroneous sequencing artefacts, and cycling cells (in the process of replicating their DNA). Removing a portion of the sequenced cells will decrease the statistical power to determine the subclonal structure of the population—an important application of this work—, and may bias the sampling against clones that have a higher division rate. We expect this will be an intrinsic limitation to any scWGS phylogenetic methods and this motivates the design of improved classification methods detecting cell cycling from genomic and imaging data.
                            </p>
                            <p id="p-40">
                             We developed two main variants of our Bayesian models, one with error rate parameters shared by all loci (global) and one with locus-specific (local) error rates. In our simulation experiments, the global and local parameterizations performed similarly. Based on the similar performance of these two models and the fact that the global parameterization is computationally cheaper, we recommend the use of the global parameterization by default.
                            </p>
                            <p id="p-41">
                             Evaluating the performance of a phylogenetic reconstruction method on real-world datasets is difficult, mainly due to a paucity of ground truth. One promising area of research is the use of CRISPR-Cas9 based lineage tracing [
                             <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. In absence of ground truth data, we developed a goodness-of-fit framework that to our knowledge enables a first of a kind benchmarking of phylogenetic inference methods over real-world scWGS CAN datasets.
                            </p>
                            <p id="p-42">
                             Phylogenetic tree reconstruction is a principled way to identify subpopulations in a heterogeneous single-cell population. This in turn enables the use of population genetics models that track the abundance of subpopulations over multiple timepoints [
                             <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ] and to make inferences about the evolutionary forces acting on each clone. Further study with timeseries modelling will provide insight into therapeutic strategies promoting early intervention, drug combinations and evolution-aware approaches to clinical management.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             5 Funding
                            </h2>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             6 Author Contributions
                            </h2>
                            <p id="p-44">
                             SS, FD: computational method development, data analysis, manuscript writing; KC: data analysis, manuscript writing; KRC, AR: method development; FK, data generation; DL, MA, AM, MW, TF: computational biology, data analysis; NR: manuscript editing; SA: data generation and oversight; SPS: method development and oversight; ABC: project conception and oversight, statistical inference method development, manuscript writing, senior responsible author.
                            </p>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             7 Competing Interests
                            </h2>
                            <p id="p-45">
                             S.P.S. and S.A. are founders, shareholders, and consultants of Imagia Canexia Health Inc.
                            </p>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             8 Code availability
                            </h2>
                            <p id="p-46">
                             Sitka is available at
                             <a href="https://github.com/UBC-Stat-ML/sitkatree.git">
                              https://github.com/UBC-Stat-ML/sitkatree.git
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             9 Methods
                            </h2>
                            <div class="section" id="sec-14">
                             <h3>
                              9.1 Pre-processing
                             </h3>
                             <p id="p-47">
                              The raw data contain cells that are either contaminated (e.g., contains biological material from mice) or have undesired sequencing artefacts. These include cells that were captured for DNA sequencing when undergoing mitosis. Since the sitka model does not account for such phenomena, the filtering is an important step.
                              <strong>
                               Supplementary Fig. 12
                              </strong>
                              shows the steps taken from pulling the raw data to the CNA integer matrix ready for sitka transformation (details in the Supplementary Information). Briefly, we remove control cells, cells with highly-noisy CN calls, and cells that have very few mapped reads. We also remove copy number bins that lie in difficult to sequence regions of the genome (bins with lowmappability). Finally, we drop cells that, based on their CNA profile, are suspected to be cycling cells.
                             </p>
                            </div>
                            <div class="section" id="sec-15">
                             <h3>
                              9.2 The sitka transformation
                             </h3>
                             <p id="p-48">
                              To obtain the C × L
                              <sub>
                               Markers
                              </sub>
                              phylogenetic markers matrix y that comprises the input to the sitka model, we apply a lossy transformation to the C × L
                              <sub>
                               Bins
                              </sub>
                              CNA matrix a that involves computing the change in copy number state between two consecutive bins.
                              <strong>
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Supplementary Fig. 1
                               </a>
                              </strong>
                              shows a small CNA matrix and its corresponding transformation into the marker matrix. For brevity, in what follows we assume that only one chromosome is used, so that L
                              <sub>
                               Bins
                              </sub>
                              = L and L
                              <sub>
                               Markers
                              </sub>
                              = L
                              <sub>
                               Bins
                              </sub>
                              − 1. In practice, we use all available chromosomes, and L
                              <sub>
                               Markers
                              </sub>
                              = L
                              <sub>
                               Bins
                              </sub>
                              − N
                              <sub>
                               Chr
                              </sub>
                              where N
                              <sub>
                               Chr
                              </sub>
                              denotes the total number of chromosomes used.
                             </p>
                             <p id="p-49">
                              Given a filtered cell-by-locus matrix a, we sort bins by their genomic position. Then in each chromosome, we compute markers as the binarised difference between consecutive bins. In other words, y = (y
                              <sub>
                               c,l
                              </sub>
                              <sup>
                               ′
                              </sup>
                              ) and l′ ∈ {1,…,L − 1}, and
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where
                              <strong>
                               1
                              </strong>
                              (x) is the indicator function.
                             </p>
                            </div>
                            <div class="section" id="sec-16">
                             <h3>
                              9.3 Fixing jitter and selection of phylogenetic markers
                             </h3>
                             <p id="p-50">
                              The copy numbers available to us in this work are estimated independently for each cell. This is one reason why the start position (bin) of the same CN change event may be slightly different across cells, generating some jitter. We address this by enumerating each change point column in order of decreasing density (where the density of column l is given by ∑
                              <sub>
                               c ∈C
                              </sub>
                              y
                              <sub>
                               c,l
                              </sub>
                              / |C |) and merging the column with its k = 2 immediate neighbours (see Algorithm 1 for details). An example of the result of the jitter correction heuristic is shown in
                              <strong>
                               <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                              </strong>
                              panel
                              <strong>
                               c
                              </strong>
                              . To speed-up computation, only a subset of markers present in at least a minimum number of cells are chosen for phylogenetic inference. That is, we removed columns l in y with relative density ∑
                              <sub>
                               c ∈C
                              </sub>
                              y
                              <sub>
                               c,l
                              </sub>
                              / |C| less than a threshold, set to 5%. Larger values of this threshold may lead to less resolved clades in the inferred tree.
                             </p>
                             <div id="statement-1" class="statement">
                              <span class="statement-label">
                               Algorithm 1
                              </span>
                              <p id="p-51">
                               JitterFix
                              </p>
                              <div id="F8" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F8.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F8.medium.gif" width="440" height="141"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F8.medium.gif" width="440" height="141"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure8" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-52">
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-17">
                             <h3>
                              9.4 The sitka model
                             </h3>
                             <div class="section" id="sec-18">
                              <h4>
                               9.4.1 Model description
                              </h4>
                              <p id="p-53">
                               The sitka model starts with the perfect phylogeny assumption for the latent variables x
                               <sub>
                                c,l
                               </sub>
                               but allows deviation from it via allowing noisy observations y
                               <sub>
                                c,l
                               </sub>
                               . In a perfect phylogeny model, each phylogenetic trait arises only once on the rooted tree topology and all cells descending from that position will inherit that trait and no deletions are allowed.
                              </p>
                              <p id="p-54">
                               Let C and L denote the sets of cells and loci respectively.
                              </p>
                              <p id="p-55">
                               We posit an observation probability model p(y |x, θ), where θ are model parameters described shortly, and both x and y are cell by locus matrices, the former being latent (derived from the unobserved tree via x = x(t)), while the latter is the matrix obtained from the sitka transformation. To model errors in copy number calls as well as perfect phylogeny violations, we introduce false positive and negative rate parameters r
                               <sup>
                                FP
                               </sup>
                               ∈ (0, 1) and r
                               <sup>
                                FN
                               </sup>
                               ∈ (0, 1) respectively, and an error matrix
                               <span class="disp-formula" id="disp-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               from which we set:
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               We define two type of models, differing in the choice of functions
                               <span class="inline-formula" id="inline-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-5.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-5.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and dimensionality of θ: one based on global error parameters, and one based on locus-specific error parameters.
                              </p>
                              <p id="p-56">
                               For the global parameterization,
                               <span class="inline-formula" id="inline-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-6.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-6.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and the false positive and false negative functions are given by
                               <span class="inline-formula" id="inline-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-7.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-7.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-57">
                               For the locus-specific error model, we set the error rates to be locus-dependent:
                               <span class="inline-formula" id="inline-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . With this extra flexibility, the model can discount the effect of a trait violating the perfect phylogeny assumption, by setting high error rates for the trait’s locus.
                              </p>
                              <p id="p-58">
                               The two parameterizations are compared in the Supplementary Information. We use the global parameterization by default unless mentioned otherwise.
                              </p>
                              <p id="p-59">
                               In both the global and locus-specific parameterizations, we need to construct a prior distribution p(θ) over the error parameters. Using a uniform prior distribution with support on [0, 1] can lead to pathological cases as shown in
                               <strong>
                                <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                                 Fig. 2
                                </a>
                               </strong>
                               . To avoid that, we use the following prior distributions on the two types of error:
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               We use
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               in our experiments involving synthetic data. For experiments on real world data, we use
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               as default. When the model is misspecified from an overly conservative bound, the trace, and thus posterior distribution, collapses to the boundaries. For example, when using a false positive rate of 0.1 for synthetic data, the resulting approximate marginal posterior of the false positive rate corresponds to a near-point mass at 0.1. We did not observe such boundary collapse on the real datasets studied in this work.
                              </p>
                              <p id="p-60">
                               Next, we describe the prior p(t) on phylogenies using a two-step generative process:
                              </p>
                              <div class="section" id="sec-19">
                               <h5>
                                Sampling a mutation tree
                               </h5>
                               <p id="p-61">
                                let 𝒱
                                <sup>
                                 m
                                </sup>
                                = L ∪v
                                <sup>
                                 *
                                </sup>
                                denote a vertex set composed of one vertex for each of the |L| loci plus one artificial root node v
                                <sup>
                                 *
                                </sup>
                                . The artificial root node induces an implicit notion of direction on the edges, viewing them as pointing away from v
                                <sup>
                                 *
                                </sup>
                                . Let 𝒯;
                                <sup>
                                 m
                                </sup>
                                denote the set of trees t
                                <sup>
                                 m
                                </sup>
                                spanning 𝒱
                                <sup>
                                 m
                                </sup>
                                . The interpretation of t
                                <sup>
                                 m
                                </sup>
                                is as follows: there is a directed path from vertex/locus l to l′ in t
                                <sup>
                                 m
                                </sup>
                                if and only if the trait indexed by l is hypothesized to have emerged in a cell which is ancestral to the cell in which l′ emerged. Pick one element t
                                <sup>
                                 m
                                </sup>
                                ∈ 𝒯
                                <sup>
                                 m
                                </sup>
                                .
                               </p>
                              </div>
                              <div class="section" id="sec-20">
                               <h5>
                                Sampling cell assignments
                               </h5>
                               <p id="p-62">
                                assign each cell to a vertex in t
                                <sup>
                                 m
                                </sup>
                                . The interpretation of assigning cell c to locus l is that among the traits under study, c is hypothesized to possess only the traits visited by the shortest path from v
                                <sup>
                                 *
                                </sup>
                                to l in t
                                <sub>
                                 m
                                </sub>
                                . If a cell c is assigned to v
                                <sup>
                                 *
                                </sup>
                                , the interpretation is that c is hypothesized to possess none of the traits under study.
                               </p>
                               <p id="p-63">
                                The number of possible trees obtained from this two-step sampling process is:
                                <span class="disp-formula" id="disp-formula-5">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-13.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-13.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where we use Cayley’s formula to compute |𝒯
                                <sup>
                                 m
                                </sup>
                                |. Hence the uniform prior probability mass function over the possible outputs of this two-step sampling process is given by:
                                <span class="disp-formula" id="disp-formula-6">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-14.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-14.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                where 𝒯 is the set of all perfect phylogenetic trees that result from the two step generative process described above. Simulation from the prior can be performed using Wilson’s algorithm [
                                <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                 33
                                </a>
                                ], followed by independent categorical sampling to simulate the cell assignments.
                               </p>
                               <p id="p-64">
                                This simple prior has a useful property: if a collection of say two splits are supported by m
                                <sub>
                                 1
                                </sub>
                                and m
                                <sub>
                                 2
                                </sub>
                                traits, then the prior probability for an additional trait to support the first versus second split is proportional to (m
                                <sub>
                                 1
                                </sub>
                                + 1, m
                                <sub>
                                 2
                                </sub>
                                + 1). Therefore, there is a “rich gets richer” behaviour built-in into the prior, which is viewed as useful in many Bayesian non-parametric models [
                                <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                                 34
                                </a>
                                ]. More precisely, this “rich gets richer” behaviour emerges when grouping trees into equivalence classes and looking at the induced prior on these equivalence classes, i.e., the distribution obtained by summing the prior over the trees in the equivalence class. Specifically, consider the equivalence relation such that two type I trees t, t′ are in the same equivalence class if and only if f (t) = f (t′), where f (·) consists in transforming t into a type II tree while annotating each edge by the number of events on that edge. Since there are different numbers of type I trees in different equivalent classes, this means that the induced prior on these equivalence classes is non-uniform.
                               </p>
                              </div>
                             </div>
                             <div class="section" id="sec-21">
                              <h4>
                               9.4.2 Inference
                              </h4>
                              <p id="p-65">
                               The posterior distribution,
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is approximated using MCMC. Two MCMC moves are used, described in the next two sections. The posterior distribution is summarized using a Bayes estimator described in
                               <a id="xref-sec-24-4" class="xref-sec" href="#sec-24">
                                Section 9.4.5
                               </a>
                               . The model is implemented in the Blang probabilistic programming language [
                               <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                                35
                               </a>
                               ].
                              </p>
                             </div>
                             <div class="section" id="sec-22">
                              <h4>
                               9.4.3 MCMC tree exploration move
                              </h4>
                              <p id="p-66">
                               Sitka uses a tree sampling move to efficiently explore, at each MCMC iteration, the posterior distribution in a large neighbourhood of a given tree. Given a tree t and locus l, we define a neighbourhood N
                               <sup>
                                l
                               </sup>
                               (t) ⊂ 𝒯 by removing l from t, and considering all possible ways to reattach l and hence defining a neighbourhood of phylogenetic trees (we also implemented a separate move reattaching cell nodes instead of locus nodes, its derivation follows similar lines as the move described in this section). The process of removing l is called an edge-contraction (removing an edge after connecting its two end-points) while the process of adding back a locus is called an edge-insertion. An edge insertion (see
                               <strong>
                                <a id="xref-fig-10-3" class="xref-fig" href="#F10">
                                 Supplementary Fig. 1
                                </a>
                               </strong>
                               for a visualization) can be described as follows:
                              </p>
                              <ol class="list-ord" id="list-1">
                               <li id="list-item-1">
                                <p id="p-67">
                                 Pick a non-cell vertex v, i.e., an element from the set R = {v
                                 <sup>
                                  *
                                 </sup>
                                 } ∪L\ {l} where v
                                 <sup>
                                  *
                                 </sup>
                                 is the root node.
                                </p>
                               </li>
                               <li id="list-item-2">
                                <p id="p-68">
                                 Pick any subset of v’s descendent subtrees and disconnect them from v.
                                </p>
                               </li>
                               <li id="list-item-3">
                                <p id="p-69">
                                 Add a new node l under v and move the selected nodes from step 2 above and attach them to l.
                                </p>
                               </li>
                              </ol>
                              <p id="p-70">
                               In the following, we derive the probability distributions to be used in steps 1 and 2 above that lead to a Gibbs sampling algorithm [
                               <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                                36
                               </a>
                               ]. The Gibbs sampler first selects a locus l from a fixed distribution (a tuning parameter), which we take for simplicity as being uniform over the |L| loci.
                              </p>
                              <p id="p-71">
                               After having sampled l, we partition N
                               <sup>
                                l
                               </sup>
                               (t
                               <sub>
                                \l
                               </sub>
                               ) into blocks corresponding to the choice of node v made in Step 1,
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . The Gibbs conditional probabilities required in step 1 above are of the form:
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where:
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and t
                               <sub>
                                \l
                               </sub>
                               denotes the tree obtained after performing an edge contraction, where the contracted edge is between l and the parent node of l. To compute ρ
                               <sub>
                                v
                               </sub>
                               efficiently, we start with the following likelihood recursion for all vertex v in t
                               <sub>
                                \l
                               </sub>
                               . First, for all vertices c corresponding to a cell and b ∈ {0, 1}, define:
                               <span class="disp-formula" id="disp-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Next, we perform the following bottom-up recursion for all subtrees of t
                               <sub>
                                \l
                               </sub>
                               : for all v ∈ R, b ∈ {0, 1},
                               <span class="disp-formula" id="disp-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where children(v) denotes the list of children of vertex v.
                              </p>
                              <p id="p-72">
                               We can now return to the problem of computing
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . First, observe that the sum in
                               <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                                Equation (2
                               </a>
                               ) can be re-indexed by a bit vector
                               <strong>
                                b
                               </strong>
                               = (b
                               <sub>
                                1
                               </sub>
                               , b
                               <sub>
                                2
                               </sub>
                               ,…, b
                               <sub>
                                k
                               </sub>
                               ), b
                               <sub>
                                v
                               </sub>
                               ″ ∈ {0, 1} of length equal to k = children(v). Each bit b
                               <sub>
                                v
                               </sub>
                               ″ is equal to one if children v″ is to be moved into a child of v′ (refer to
                               <strong>
                                <a id="xref-fig-10-4" class="xref-fig" href="#F10">
                                 Supplementary Fig. 1
                                </a>
                               </strong>
                               ), and zero if it is to stay as a child of v. For each possible assignment, we obtain a tree
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and its probability can be decomposed into factors corresponding to cells that are descendant of v (denoted C
                               <sub>
                                v
                               </sub>
                               , solid red thick line under the tree of
                               <strong>
                                <a id="xref-fig-10-5" class="xref-fig" href="#F10">
                                 Supplementary Fig. 1-B
                                </a>
                               </strong>
                               ) and those that are not (denoted C
                               <sub>
                                \v
                               </sub>
                               , dashed green thick line under the tree of
                               <strong>
                                <a id="xref-fig-10-6" class="xref-fig" href="#F10">
                                 Supplementary Fig. 1-B
                                </a>
                               </strong>
                               ).
                              </p>
                              <p id="p-73">
                               The product of the likelihood factors corresponding to cells that are not descendants of v (“outside product”) does not depend on the choice of the bit vector. This outside product can be obtained as follows:
                               <span class="disp-formula" id="disp-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Note that this assumes
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . As a workaround to cases where there are structural zeros, we recommend injecting small numerical values if
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (we used 10
                               <sup>
                                −6
                               </sup>
                               in our implementation).
                              </p>
                              <p id="p-74">
                               For the cells under v, we now have to take into account whether they are selected under the newly introduced locus or not. More precisely, for each of the children v
                               <sub>
                                1
                               </sub>
                               , v
                               <sub>
                                2
                               </sub>
                               ,…, v
                               <sub>
                                k
                               </sub>
                               , we have to take into account the value of the bit vector b = (b
                               <sub>
                                1
                               </sub>
                               , b
                               <sub>
                                2
                               </sub>
                               ,…, b
                               <sub>
                                k
                               </sub>
                               ). The sum over possible assignments written naively has a number of terms which is exponential in k, but can be rewritten into a product over k factors:
                               <span class="disp-formula" id="disp-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Putting it all together, we obtain for some constants K
                               <sub>
                                i
                               </sub>
                               independent of v:
                               <span class="disp-formula" id="disp-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Putting these together we can compute the probabilities required in step 1 above:
                               <span class="disp-formula" id="disp-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Once v is sampled, we choose a subset of its children to move to v′ by sampling k independent Bernoulli random variables with the i-th one having bias
                               <span class="disp-formula" id="disp-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and selecting children with corresponding Bernoulli realisations of 1.
                              </p>
                             </div>
                             <div class="section" id="sec-23">
                              <h4>
                               9.4.4 MCMC parameter exploration move
                              </h4>
                              <p id="p-75">
                               To resample the parameters θ we condition on the tree t, and hence on the hidden state matrix x, and update θ in a Metropolis-within-Gibbs framework. There are two different samplers depending on whether the global or locus-specific parameterization is used. We start with describing the former.
                              </p>
                              <p id="p-76">
                               We compute two sufficient statistics from the matrix x (i) the number of false positive instances, n
                               <sup>
                                FP
                               </sup>
                               , and (ii) the number of false negative instances, n
                               <sup>
                                FN
                               </sup>
                               ,
                               <span class="disp-formula" id="disp-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Based on these cached statistics, we obtain:
                               <span class="disp-formula" id="disp-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where the the number of positive n
                               <sup>
                                P
                               </sup>
                               and negative n
                               <sup>
                                N
                               </sup>
                               instances in the data can be precomputed,
                               <span class="disp-formula" id="disp-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Based on the above expression, which can be evaluated in O(1) once the statistics are computed, we then use a slice sampling algorithm to update the parameters [
                               <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                37
                               </a>
                               ].
                              </p>
                              <p id="p-77">
                               The sampler for the locus-specific parameterization is very similar. The main difference is that we compute the statistics for each locus l:
                               <span class="disp-formula" id="disp-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Then a slice sampling move is applied to each locus-specific parameter.
                              </p>
                             </div>
                             <div class="section" id="sec-24">
                              <h4>
                               9.4.5 Posterior summarization
                              </h4>
                              <p id="p-78">
                               To summarize the posterior distribution using a point estimate, we approximate the Bayes estimator [
                               <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                                38
                               </a>
                               ] by minimising the Bayes risk for a loss function
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               encoding the cost of selecting an “action” a when the true tree is t′ and the true parameter is θ′:
                               <span class="disp-formula" id="disp-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Here, an “action” consists in selecting a consensus tree t. Moreover, the loss function we consider only depends on the true tree t′ and not on the true parameter θ′, so we write this loss function as L(t, t′), simplifying the above equation into:
                               <span class="disp-formula" id="disp-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               One default choice for L(t, t′) is the L
                               <sub>
                                1
                               </sub>
                               metric on the matrices of induced indicators x(t), x(t′):
                               <span class="disp-formula" id="disp-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               It is useful to define the marginal indicators m
                               <sub>
                                c,l
                               </sub>
                               that can be conceptualised as the posterior probability of cell c to have trait l:
                               <span class="disp-formula" id="disp-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-33.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-33.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Using the MCMC samples t
                               <sup>
                                1
                               </sup>
                               , t
                               <sup>
                                2
                               </sup>
                               ,…, t
                               <sup>
                                N
                               </sup>
                               , we obtain a Monte Carlo approximation:
                               <span class="disp-formula" id="disp-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-34.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-34.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with probability one.
                              </p>
                              <p id="p-79">
                               <strong>
                                <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                                 Fig. 3-g
                                </a>
                               </strong>
                               shows an example of the matrix m each element of which is one of the approximated
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . We can now write the objective function of
                               <a id="xref-disp-formula-23-1" class="xref-disp-formula" href="#disp-formula-23">
                                Equation (7)
                               </a>
                               via the above marginal indicators:
                               <span class="disp-formula" id="disp-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-35.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-35.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               We use a greedy algorithm to approximately minimize
                               <a id="xref-disp-formula-27-1" class="xref-disp-formula" href="#disp-formula-27">
                                Equation (8)
                               </a>
                               . We start with a star tree with leaves C rooted at v
                               <sup>
                                *
                               </sup>
                               and add loci from L one by one from a locus queue sorted by priority score. The priority score of each locus l is computed as
                               <span class="disp-formula" id="disp-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="disp-formula" id="disp-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The quantities in the priority queue can be computed as in
                               <a id="xref-sec-22-2" class="xref-sec" href="#sec-22">
                                Section 9.4.3
                               </a>
                               . We take the result of the minimization of the Bayes risk as the consensus tree τ
                               <sub>
                                Consensus
                               </sub>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-25">
                              <h4>
                               9.4.6 Consensus tree and CNA heatmap visualisation
                              </h4>
                              <p id="p-80">
                               To visualize the consensus tree, we collapse the chains (sequence of loci having only one child) as well as remove the subtrees containing no cells. We align the leaves of the tree which correspond to cells after collapsing to the rows of a cell-locus matrix.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              9.5 Synthetic experiments
                             </h3>
                             <div class="section" id="sec-27">
                              <h4>
                               9.5.1 Benchmarking
                              </h4>
                              <p id="p-81">
                               To assess the performance of sitka against alternative approaches, we ran inference on 90 simulated datasets of varying characteristics. We will refer to this set of datasets as S90; its simulation procedure is described in
                               <a id="xref-sec-29-1" class="xref-sec" href="#sec-29">
                                Section 9.5.3
                               </a>
                               . For each dataset in S90, we scored each method by computing the Robinson-Foulds (RF) [
                               <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                                39
                               </a>
                               ] distance between the simulated tree and the inferred tree. The scores were normalized within each dataset by dividing each method’s score by the worst performing method’s score (note that the set of methods includes sampling a tree uniformly at random; the motivation of this normalization is to correct for the intrinsic difficulty of datasets).
                              </p>
                              <p id="p-82">
                               We compared sitka against the following baseline methods: UPGMA, WPGMA, NJ, HDBSCAN, and balanced and ordinary least-squares minimum-evolution methods (BME, OME respectively) of [
                               <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                                40
                               </a>
                               ]. We also report the score of a uniformly random bifurcating tree, Uniform, to help interpret the absolute scores. Each method was given raw data from S90, as well as input identical to that of sitka, i.e., filtered binary marker data. Sitka’s inference settings are summarized in
                               <strong>
                                Supplementary Table 2
                               </strong>
                               .
                              </p>
                              <p id="p-83">
                               Baseline methods performed significantly worse with sitka’s input and are thus omitted from the following summary. Sitka’s normalized RF score (0.57 ± 0.04) outperformed all baseline methods, the next best performer was BME (0.91 ± 0.07). Sitka ranked first in all 90 but one set of data, where it ranked third for one dataset of size 500 × 800. These results are summarized in
                               <strong>
                                <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                                 Supplementary Fig. 8
                                </a>
                               </strong>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-28">
                              <h4>
                               9.5.2 Exploratory experiments within sitka
                              </h4>
                              <p id="p-84">
                               To explore the effectiveness of global versus local (locus-specific) parameterization (
                               <a id="xref-sec-18-3" class="xref-sec" href="#sec-18">
                                Section 9.4.1
                               </a>
                               ), and the posterior summarization method (
                               <a id="xref-sec-24-5" class="xref-sec" href="#sec-24">
                                Section 9.4.5
                               </a>
                               ), we ran inference on 10 synthetic datasets. We will refer to this set of datasets as S10; its simulation procedure is described in
                               <a id="xref-sec-29-2" class="xref-sec" href="#sec-29">
                                Section 9.5.3
                               </a>
                               . Inference settings are summarized in
                               <strong>
                                Supplementary Table 2
                               </strong>
                               .
                              </p>
                              <p id="p-85">
                               RF distances from the best-possible tree were computed as a metric. The best-possible tree is defined as the perfect phylogenetic tree constructed from the noiseless synthetic, unviolated cell-locus matrix data. Note that the best possible tree is derived from the true tree (i.e., the one used in the first step of the generating process), but in general can be different. To understand why, note that the tree generation process will simulate on each edge of the true tree a Poisson distributed number of evolutionary events (
                               <a id="xref-sec-29-3" class="xref-sec" href="#sec-29">
                                Section 9.5.3
                               </a>
                               ). As a result, some edges can have zero associated evolutionary events. This means that even if we turned off all observation noise it would not be possible to recover these zero-event edges, they are in a sense unidentifiable. The process of producing the best possible tree essentially consists in collapsing these zero-event edges, hence forming a multifurcating reference tree.
                              </p>
                              <p id="p-86">
                               For a baseline with which to compare the greedy estimator (GE) of
                               <a id="xref-sec-24-6" class="xref-sec" href="#sec-24">
                                Section 9.4.5
                               </a>
                               , consider the trace search estimator (TSE). The TSE is defined as a tree in the sampler trace that minimizes the sample L
                               <sub>
                                1
                               </sub>
                               distance (
                               <a id="xref-sec-24-7" class="xref-sec" href="#sec-24">
                                Section 9.4.5
                               </a>
                               ). Formally,
                               <span class="disp-formula" id="disp-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-38.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-38.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where {t′} denotes the set of trees that were sampled during the MCMC procedure.
                              </p>
                              <p id="p-87">
                               The GE method outperformed the TSE method under both the global and local models. This suggests the proposed GE can, informally, harness more information from the posterior. Under the TSE, the global model (0.44 ± 0.09; mean normalized RF score ± standard error) outperformed the local model (0.71 ± 0.06). This observation suggests that the local parameterization has a strong influence on the trace (in tree space) of our sampler, as the TSE is essentially a search over the posterior sample. Under the GE, the global model (0.31 ± 0.07) and local model (0.30 ± 0.07) performed evenly well. This observation suggests that the choice of parameterization does not heavily influence the information contained in the marginal posterior over trees. Ultimately this experiment suggests that the GE summarizes the marginal posterior sufficiently well such that the global model, the simpler model of the two, suffices for reconstructing phylogenies and should be the preferred model. A summarizing plot is shown in
                               <strong>
                                Supplementary Fig. 9
                               </strong>
                               .
                              </p>
                              <p id="p-88">
                               In our next synthetic experiment, we aimed to study the effects of perfect phylogeny assumption violations on the reconstruction of trees, and attempted to draw connections to real world data. The two violations considered are infinite sites and loss violations, described in
                               <a id="xref-sec-29-4" class="xref-sec" href="#sec-29">
                                Section 9.5.3
                               </a>
                               . Inference was performed on 130 datasets (S130). Inference settings are summarized in
                               <strong>
                                Supplementary Table 2
                               </strong>
                               , and the simulation procedure for S130 is described in
                               <a id="xref-sec-29-5" class="xref-sec" href="#sec-29">
                                Section 9.5.3
                               </a>
                               .
                              </p>
                              <p id="p-89">
                               The experiment results are summarized in
                               <strong>
                                <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                                 Fig. 7-a
                                </a>
                               </strong>
                               . Holding one violation rate fixed at zero and varying the other, we observed linear effects for both types of violations. The results suggest sitka is more robust to infinite sites violations, with estimated effects to be 0.31 ± 0.07 (normalized RF distance ±standard error), which is much less than loss violations (0.47 ± 0.07). When varied together, the linear effects were estimated to be 0.25 ±0.04, 0.38 ±0.04 respectively. In an attempt to draw connections to real datasets, we developed a heuristic method to obtain a rough estimate of both violation rates. The estimated rates obtained on real data were all less than 0.25 (the estimation heuristic is described below;
                               <strong>
                                <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                                 Fig. 7-b
                                </a>
                               </strong>
                               ).
                              </p>
                              <p id="p-90">
                               We now describe the heuristic we used to obtain rough estimates of the rates of the two types of violation. Given the inferred tree and its corresponding marker matrix x (as in
                               <a id="xref-sec-18-4" class="xref-sec" href="#sec-18">
                                Section 9.4.1
                               </a>
                               ), and the sitka-transformed marker matrix y (as in
                               <a id="xref-sec-15-2" class="xref-sec" href="#sec-15">
                                Section 9.2
                               </a>
                               ), define the difference matrix z := x− y, i.e., z has entries z
                               <sub>
                                i,j
                               </sub>
                               = x
                               <sub>
                                i,j
                               </sub>
                               (t) − y
                               <sub>
                                i,j
                               </sub>
                               , where t is the consensus reconstruction. To motivate how we can detect violations of the Infinite Site assumption (IS, i.e., genomic bins in which more than one events occur), and losses, refer to 10, and notice that these violations tend to leave a distinctive pattern on the difference of the two matrices x (positive entries shown in black in the Figure) and y (positive entries shown in orange in the Figure). Specifically, define z
                               <sub>
                                Loss
                               </sub>
                               with entries
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , and similarly z
                               <sub>
                                IS
                               </sub>
                               with entries
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Given an integer-valued threshold E
                               <sub>
                                v
                               </sub>
                               > 0, we say a column or trait l in z
                               <sub>
                                v
                               </sub>
                               (for v ∈ {Loss, IS}) has a violation if there exists an island of size at least as large as E
                               <sub>
                                v
                               </sub>
                               . An island of size s in column l is defined to be any sequence of row indices i, i + 1,…,i + s such that
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are, not necessarily the same, 0 or undefined. Finally, the proportion of columns with a given type of violation, loss or infinite sites, is taken to be the violation rate estimate.
                              </p>
                              <p id="p-91">
                               We also performed experiments to compare our full Bayesian analysis to Maximum Like-lihood estimation (MLE). To do so, we generated data as follows: we first sampled a tree generated uniformly over topologies; second, we generated synthetic data according to y
                               <sub>
                                c,l
                               </sub>
                               |x
                               <sub>
                                c,l
                               </sub>
                               ∼ Normal(x
                               <sub>
                                c,l
                               </sub>
                               , σ
                               <sup>
                                2
                               </sup>
                               ), varying σ
                               <sup>
                                2
                               </sup>
                               to control the amount of noise. We generate matrices of size |C | = 1000 and |L| = 50 for σ
                               <sup>
                                2
                               </sup>
                               ∈ {1/10, 2/10,…, 5/10}. In these experiments we provide the well-specified noise model to both inference methods. We approximate the MLE using a greedy scheme with the same structure as the one described in
                               <a id="xref-sec-24-8" class="xref-sec" href="#sec-24">
                                Section 9.4.5
                               </a>
                               . The results are shown in
                               <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                                Figure 8
                               </a>
                               . The Bayesian methods outperform the greedy maximum likelihood heuristic by a large margin.
                              </p>
                              <div id="F9" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F9.large.jpg?width=800&height=600&carousel=1" title="Synthetic data results comparing our Bayesian estimator to a Maximum Like-lihood Estimator (MLE). Boxes from left to right show different amounts of noise in the synthetic data generation, corresponding to values for σ2. The y axis measures the L1 tree distances loss, normalized by |C||L|." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Synthetic data results comparing our Bayesian estimator to a Maximum Like-lihood Estimator (MLE). Boxes from left to right show different amounts of noise in the synthetic data generation, corresponding to values for <em>σ</em><sup>2</sup>. The <em>y</em> axis measures the L1 tree distances loss, normalized by |<em>C</em>||<em>L</em>|.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F9.medium.gif" width="440" height="152"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F9.medium.gif" width="440" height="152"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure 8.
                                </span>
                                <p id="p-92" class="first-child">
                                 Synthetic data results comparing our Bayesian estimator to a Maximum Like-lihood Estimator (MLE). Boxes from left to right show different amounts of noise in the synthetic data generation, corresponding to values for σ
                                 <sup>
                                  2
                                 </sup>
                                 . The y axis measures the L1 tree distances loss, normalized by |C||L|.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                              <p id="p-93">
                               Finally, we investigate the impact of ignoring pairwise dependencies between the two end points of CNA events. We first make the observation that if we subset the sitka markers to keep only those where the copy number is increasing from left to right, we retain only one end point of each paired event. This creates a smaller set of independent markers L′ ⊂ L. We can compute one sitka tree t based on all L loci (which includes ignored pairwise dependencies), and one sitka tree t′ based on L′ (a smaller set of independent loci). We can then inspect the proportion of identical entries in the matrices x(t′) compared to x(t), the latter subsetted to the columns in L′.
                              </p>
                              <p id="p-94">
                               We performed the experiment described above on the S90 datasets (described in 9.5.3), and three noise regimes with increasing amounts of noise injected: (I) where step (ii) in 9.5.3 is skipped; (II) uniform noise parameters FPR and FNR drawn from uniform distributions on the intervals (0.0005, 0.005), (0.005, 0.015) respectively, doubling noise parameters drawn from a uniform distribution on (0.015, 0.035) distribution, jitter noise parameters drawn from a uniform distribution on (0.15, 0.35); (III) uniform noise parameters FPR and FNR drawn from uniform distributions on the intervals (0.001, 0.01), (0.01, 0.03) respectively, doubling noise parameters drawn from a uniform distribution on (0.03, 0.07) distribution, jitter noise parameters drawn from a uniform distribution on (0.3, 0.7). All results are averaged over 15 datasets.
                              </p>
                              <p id="p-95">
                               In all three regimes we observed a large overlap between t and t′, but this overlap is negatively correlated with noise: in regime (I) we observed a mean overlap of 0.99 (sd 0.004); in regime (II), a mean overlap of 0.97 (sd 0.009); in regime (III), a mean overlap of 0.76 (sd 0.18). The results support that in a low to moderate noise regime, it is reasonable to ignore violation of pairwise dependencies for the purpose of point estimation (consensus tree construction). In the higher noise regime, it may be advantageous to build the two trees t and t′. We expect neither to systematically outperform the other, the trade-off being that t is built from more data but with independence violations, whereas t
                               <sup>
                                !
                               </sup>
                               is built from less data but without independence assumption violations. Our goodness-of-fit tests can be used to select one of these two trees for final output.
                              </p>
                             </div>
                             <div class="section" id="sec-29">
                              <h4>
                               9.5.3 Data simulation
                              </h4>
                              <p id="p-96">
                               Datasets in S90 were generated in two steps: (i) simulate a cell tree and its corresponding CNA data, and (ii) inject noise into the CNA data from step one.
                              </p>
                              <p id="p-97">
                               In the first step we used the simulator of [
                               <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ] to generate trees along with CNAs, where leaf nodes represent observed cells and internal nodes represent latent ancestral cells, i.e., unobserved cells. An edge in the tree represents an ancestral relationship between the respective cells.
                              </p>
                              <p id="p-98">
                               The simulator of [
                               <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ] itself consists of two parts, which we briefly describe as follows. First, the simulator samples a tree based on a generalization of the Blum-François Beta-splitting (GBFBS) model [
                               <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ,
                               <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                43
                               </a>
                               ], which is inspired by the Beta-splitting model of [
                               <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                44
                               </a>
                               ].
                               <sup>
                                <a id="xref-fn-3-1" class="xref-fn" href="#fn-3">
                                 1
                                </a>
                               </sup>
                               The Betasplitting model is particularly well-suited for generating a wide range of topologies, varying from balanced to imbalanced tree structures. Second, given a tree, CNAs are simulated on the edges of the tree where the number and size of CNAs are drawn from Poisson and exponential distributions respectively. The simulator also accounts for clonal whole chromosome amplification events, motivated by punctuated evolution models [
                               <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                45
                               </a>
                               ].
                              </p>
                              <p id="p-99">
                               The second step of our synthetic data simulation process, independent of [
                               <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ], injects noise into a cell by locus input CNA matrix y, and outputs a noisy matrix of the same size. Three types of noise were employed, namely, uniform noise, jitter noise, and a doubling noise.
                              </p>
                              <p id="p-100">
                               The uniform noise is parameterized by false positive (FPR) and false negative (FNR) rate parameters. For each element of the input matrix y
                               <sub>
                                ij
                               </sub>
                               , add an integer N
                               <sub>
                                ij
                               </sub>
                               ∼ Binomial(y
                               <sub>
                                ij
                               </sub>
                               , FNR) or subtract an integer M
                               <sub>
                                ij
                               </sub>
                               ∼ Binomial(1, FPR).
                              </p>
                              <p id="p-101">
                               The doubling noise is parameterized by a probability p
                               <sub>
                                d
                               </sub>
                               : for each row of the CNA matrix y, draw a factor K where K − 1 ∼ Binomial(1, p
                               <sub>
                                d
                               </sub>
                               ), which is then multiplied to the row of the CNA matrix as noise. This procedure effectively, on average, doubles the copy number values for p
                               <sub>
                                d
                               </sub>
                               proportion of cells in the sample.
                              </p>
                              <p id="p-102">
                               The jitter noise is parameterized by a probability p
                               <sub>
                                j
                               </sub>
                               . First, map the CNA matrix to its marker matrix. Then for each marker, the locus corresponding to the marker is randomly duplicated to the previous bin(s), or the next bin(s). The number of bins J to be overwritten — zero, one, or two — is drawn from a Binomial(2, p
                               <sub>
                                j
                               </sub>
                               ) distribution.
                              </p>
                              <p id="p-103">
                               Datasets in S90 were of sizes {500, 1000, 1500, 2000, 2500, 3000} cells by (approximately) 400, 600, 800 markers. For each combination of sizes, we generated five datasets based on different random seeds and parameters to make a total of 6 × 3 × 5 = 90 datasets. The approximate number of markers is the target number of markers after correcting for jitter and filtering.
                               <strong>
                                <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                                 Supplementary Fig. 7
                                </a>
                               </strong>
                               shows the CNA profiles of a subset of simulated data.
                              </p>
                              <p id="p-104">
                               To describe the simulation parameters used for S90, we follow the terminologies and notation used in [
                               <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ]. For generating trees, the α and β values parameterize the generalized Beta-splitting model. We used a symmetric parameterization of α = β ∈ {−0.9, −0.83, −0.7, −0.48, −0.1}. For generating CNA data, the mean number of CNA to be added to a branch in the tree was chosen to generate data with approximately the number of desired markers post filtering and jitter-fixing. The multiplier of the mean CNA on the root was set to 8, the whole amplification rate (rate of an allele chosen to be amplified) was set to 0.5. The remaining parameters used default settings. See [
                               <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">
                                41
                               </a>
                               ] for a more thorough description of parameters.
                              </p>
                              <p id="p-105">
                               For injecting noise, we drew the uniform noise parameters FPR and FNR from uniform distributions on the intervals (0.001, 0.01), (0.01, 0.03) respectively. The doubling noise parameters p
                               <sub>
                                d
                               </sub>
                               were drawn from a Uniform(0.03, 0.07) distribution. The jitter noise parameters p
                               <sub>
                                j
                               </sub>
                               were drawn from a Uniform(0.3, 0.7) distribution.
                              </p>
                              <p id="p-106">
                               Datasets in S10 and S130 were also generated in two steps: (i) simulate a cell tree and its corresponding binary marker data satisfying perfect phylogeny assumptions, and (ii) inject noise and/or violations into the the binary marker data from step one.
                              </p>
                              <p id="p-107">
                               In the first step, a tree is generated via Kingman’s coalescent [
                               <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                46
                               </a>
                               ].
                               <sup>
                                <a id="xref-fn-4-1" class="xref-fn" href="#fn-4">
                                 2
                                </a>
                               </sup>
                               Briefly, we sample a coalescent tree for the set of cells C by uniformly selecting pairs of cells c
                               <sub>
                                i
                               </sub>
                               , c
                               <sub>
                                j
                               </sub>
                               ∈ C to coalesce backwards in time. The waiting time, or the branch length, between each event is exponentially distributed. Conditionally on the coalescent tree and given a set of loci L, we simulate a |C| ×|L| marker matrix y. Every entry y
                               <sub>
                                i,j
                               </sub>
                               is initialized to 0. Then for each column l, we select a subset of cells C′ from C to set y
                               <sub>
                                i,l
                               </sub>
                               to 1, for all i ∈ C′. The subset of cells is sampled by choosing a branch on the tree with probability proportional to the branch length, and selecting all cells descendant from the selected branch. In essence, we are simulating the number of events via a Poisson process, and directly mapping these events to the cell-locus marker matrix. The above concludes the data generation procedure satisfying perfect phylogeny assumptions.
                              </p>
                              <p id="p-108">
                               In the second step of S10’s simulator, we injected artificial noise by introducing standard false positive and negative values into y. This concludes S10’s simulator. The simulator for S130 has an additional sampling step for controlling the degree of perfect phylogeny violations. We considered two types of violations: (i) the loss of markers along a tree’s branches, and (ii) the violation of the infinite sites (IS) assumption, that is, the occurrence of multiple distinct events in the same locus.
                              </p>
                              <p id="p-109">
                               The procedure for simulating loss of marker events can be described as follows. First, randomly select a locus l, then identify the most recent common ancestor a for the set of cells {i : y
                               <sub>
                                i,l
                               </sub>
                               = 1}. Given a, sample a cell d descendant of a (including a). Finally, the loss event is simulated by reverting y
                               <sub>
                                i,l
                               </sub>
                               to 0, for all i descendant of, and including, d.
                              </p>
                              <p id="p-110">
                               IS model violations were simulated as follows. Uniformly sample a pair of loci (j, k), and merge y
                               <sub>
                                ·,j
                               </sub>
                               , y
                               <sub>
                                ·,k
                               </sub>
                               into one column, yielding a cell-locus matrix of size one less than the original size. However, to maintain control over |L|, datasets in S130 were simulated with |L| + N
                               <sub>
                                IS
                               </sub>
                               loci such that after simulating IS violations, we recover a matrix of size |C| × |L|, where N
                               <sub>
                                IS
                               </sub>
                               is the number of IS violations.
                              </p>
                              <p id="p-111">
                               The total number of loss and infinite sites violation events (N
                               <sub>
                                Loss
                               </sub>
                               , N
                               <sub>
                                IS
                               </sub>
                               ) were drawn from binomial distributions with probability p
                               <sub>
                                Loss
                               </sub>
                               , p
                               <sub>
                                IS
                               </sub>
                               respectively (and size L). As a final step, false positives and negatives were artificially injected.
                              </p>
                              <p id="p-112">
                               For both S10 and S130, datasets of size |C| ×|L| = 500 × 100 with FNR and FPR both set to 0.002 were generated. For S130, the unordered pair (p
                               <sub>
                                Loss
                               </sub>
                               , p
                               <sub>
                                IS
                               </sub>
                               ) were set to values in {(0, 0), (0.1, 0.1),…, (0.4, 0.4)} ∪ {(0, 0.1), (0, 0.2), (0, 0.3), (0, 0.4)}. For each configuration of simulation parameters, 10 different seeds were used to generate a total of 10 and 130 datasets for S10 and S130 respectively.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-30">
                             <h3>
                              9.6 Goodness-of-fit
                             </h3>
                             <p id="p-113">
                              To evaluate the goodness-of-fit of inferred trees on real data, we suggest a test comparing the posterior distribution over entries of the matrix x with the data y.
                             </p>
                             <p id="p-114">
                              Since we will assess the goodness-of-fit of not only our method but also different baseline, we start by explaining how we can generalize the notion of the x matrix used in our method to other tree reconstruction methods. To do so, consider an inferred rooted tree, τ, and define a matrix-value function g(τ) as follows. If τ is a tree inferred from sitka, set g(τ) = x(τ). For trees inferred from baseline methods, we proceed as follows. Let τ denote a rooted tree, u, one of its unlabelled internal nodes, and c one of its leaves. Let clade(u) denote the clade corresponding to u, i.e., the set of leaves descendent from u. We define g
                              <sub>
                               c,u
                              </sub>
                              (τ) =
                              <strong>
                               1
                              </strong>
                              [c ∈ clade(τ)].
                             </p>
                             <p id="p-115">
                              In general the inferred trees from the baseline methods do not have named internal nodes, nor do they have the same number of internal nodes as the number of loci L. Therefore we do not know which locus in the inferred tree τ corresponds to which locus in the matrix y. We note that this is not the case with trees inferred from sitka where the internal nodes of the tree correspond to the columns of the induced genotype matrix g. As a result, for methods other than sitka, for each column in the input data matrix, we pick a clade in τ that has the highest prediction accuracy for the entries in that column.
                             </p>
                             <p id="p-116">
                              More precisely, for each method, we report Youden’s J index [
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               49
                              </a>
                              ] which is equal to the sum of the sensitivity and specificity minus 1. We now define a binary classification counts matrix function h, i.e., a function which, for two vectors w and z of length C, forms the confusion matrix:
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For example h
                              <sub>
                               0,0
                              </sub>
                              (w, z) would count the number of times both elements of w and z were equal to zero (or true negative). We define accuracy for a given confusion matrix h = h(w, z) as
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              We further define sensitivity and specificity as
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For a given tree τ and its corresponding genotype matrix g = g(τ) we compute the Youden’s score as follows:
                             </p>
                             <ol class="list-ord" id="list-2">
                              <li id="list-item-4">
                               <p id="p-117">
                                for all locus l in y,
                                <span class="inline-formula" id="inline-formula-25">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-25.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-25.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                ,
                               </p>
                              </li>
                              <li id="list-item-5">
                               <p id="p-118">
                                h
                                <sub>
                                 τ
                                </sub>
                                := ∑
                                <sub>
                                 l
                                </sub>
                                ′
                                <sub>
                                 ∈columns(g)
                                </sub>
                                h
                                <sub>
                                 l
                                </sub>
                                ′
                               </p>
                              </li>
                              <li id="list-item-6">
                               <p id="p-119">
                                youden
                                <sub>
                                 τ
                                </sub>
                                := youden(h
                                <sub>
                                 τ
                                </sub>
                                ).
                               </p>
                              </li>
                             </ol>
                             <p id="p-120">
                              That is for each locus in y, we take the clade that among all possible clades in τ maximizes the accuracy in predicting which cells are present in the l-th column of y. We then sum over all these scores to compute a confusion matrix for τ and use this agglomerative matrix to compute the Youden’s score for the tree. We use the delta method to calculate confidence intervals. Recall that the delta method is concerned with the asymptotic behaviour of a distribution for a function ψ of an asymptotically Gaussian random vector. For a sequence of random vectors X
                              <sub>
                               n
                              </sub>
                              for which
                              <span class="inline-formula" id="inline-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we have that
                              <span class="inline-formula" id="inline-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-27.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-27.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . In this context we use the identify
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <strong>
                               <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                                Fig. 4-d
                               </a>
                              </strong>
                              shows the Youden’s score and its 95% confidence interval for sitka and 6 baseline methods on 3 different real-world datasets. Sitka has a higher score than all competing methods.
                             </p>
                            </div>
                            <div class="section" id="sec-31">
                             <h3>
                              9.7 Application: assignment of single nucleotide variants
                             </h3>
                             <p id="p-121">
                              Here we posit an observation probability model for adding single nucleotide variant (SNV) data to an existing phylogenetic tree.
                             </p>
                             <p id="p-122">
                              For locus l in cell c, let
                              <span class="inline-formula" id="inline-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              denote the observed SNV data where the total number of reads, the number of reads with a variant allele, and the corresponding copy number are indicated by d
                              <sub>
                               c,l
                              </sub>
                              , ν
                              <sub>
                               c,l
                              </sub>
                              , and c
                              <sub>
                               c,l
                              </sub>
                              respectively.
                             </p>
                             <p id="p-123">
                              We use
                              <span class="inline-formula" id="inline-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to denote an indicator variable taking the value one if and only if an ancestor of cell c harboured a single nucleotide alteration event at locus l. This variable is unobserved and the focus of inference in this section. As in the sitka model, we assume a perfect phylogeny structure on these indicator variables, and add an error model to relate
                              <span class="inline-formula" id="inline-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              to the observed data while allowing violations of the perfect phylogeny assumption and measurement noise. In the context of single nucleotide data, this is similar to [
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                              ]. The parameters of the error model are denoted θ
                              <sup>
                               SNV
                              </sup>
                              = (E
                              <sub>
                               FP
                              </sub>
                              , E
                              <sub>
                               FN
                              </sub>
                              ), where E
                              <sub>
                               FP
                              </sub>
                              and E
                              <sub>
                               FN
                              </sub>
                              are false positive rate and false negative rates, respectively. Define:
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where d
                              <sub>
                               c,l
                              </sub>
                              and c
                              <sub>
                               c,l
                              </sub>
                              are given inputs. The likelihood probability of cell node c is denoted by
                              <span class="inline-formula" id="inline-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where b ∈ {0, 1}. For
                              <span class="inline-formula" id="inline-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              reflects the likelihood of cell c being mutated at locus l; and for
                              <span class="inline-formula" id="inline-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              reflects the likelihood of cell c not being mutated at locus l. For
                              <span class="inline-formula" id="inline-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , we set
                              <span class="inline-formula" id="inline-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-124">
                              The probability
                              <span class="inline-formula" id="inline-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is obtained by marginalizing a mixture of binomial distributions depending on all possible genotype states of locus l at cell c. Given the copy number c
                              <sub>
                               c,l
                              </sub>
                              , the possible genotype states are = 𝒢 {A … A, AA … B, A … BB, …, B … B}, where each element has a length equal to c
                              <sub>
                               c,l
                              </sub>
                              . For example, the genotype AAB refers to a genotype with one variant allele B and two reference alleles A. For each genotype state g
                              <sub>
                               i
                              </sub>
                              , where i indexes the elements of
                              <strong>
                               G
                              </strong>
                              , the mean parameter of the corresponding binomial distribution is denoted by
                              <span class="inline-formula" id="inline-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              :
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where ℬ (g
                              <sub>
                               i
                              </sub>
                              ) represents the number of variant alleles of genotype g
                              <sub>
                               i
                              </sub>
                              . Therefore, for b = 1,
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The value of p(g
                              <sub>
                               i
                              </sub>
                              ) equals
                              <span class="inline-formula" id="inline-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , and ϵ
                              <sub>
                               FN
                              </sub>
                              represents the error due to mutation loss or tree errors.
                             </p>
                             <p id="p-125">
                              If the mutation status of cell c at locus l is a wildtype (i.e., mutation is not present), then the possible genotype states should not have any variant allele. The only possible genotype state is {A … A}. The mean parameter of the binomial distribution equals ϵ
                              <sub>
                               FP
                              </sub>
                              (false positive rate). Therefore,
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              With the proposed probability model for SNVs, we can incorporate both SNV data and CNA data to infer the underlying tree phylogeny in the sitka model. Therefore,
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where C and L are the disjoint set of cells and loci, respectively. In this section, the loci set L includes both CNA and SNV traits.
                             </p>
                             <p id="p-126">
                              Assume now that we seek to add one locus to an existing tree. We proceed similarly to
                              <a id="xref-sec-22-3" class="xref-sec" href="#sec-22">
                               Section 9.4.3
                              </a>
                              .
                              <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                               Equation (4)
                              </a>
                              can be rewritten in the following form:
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Where
                              <span class="inline-formula" id="inline-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , for b ∈ {0, 1} is:
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For v ∈ R = {v
                              <sup>
                               *
                              </sup>
                              } ∪ L\{l}, and b ∈ {0, 1}, the value of
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              For the cell nodes that are the leaves of the tree
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <div class="section" id="sec-32">
                              <h4>
                               9.7.1 Detection of SNVs for individual cells
                              </h4>
                              <p id="p-127">
                               Given a fixed CNA tree (denoted by t) and the read counts data (y
                               <sup>
                                SNV
                               </sup>
                               denoted by y for simplicity), here the goal is to calculate the posterior distribution of
                               <span class="inline-formula" id="inline-formula-42">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-42.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-42.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , the mutation status of locus l at cell c, which we denote by x
                               <sub>
                                c,l
                               </sub>
                               for simplicity.
                              </p>
                              <p id="p-128">
                               The joint probability distribution of x
                               <sub>
                                c,l
                               </sub>
                               , y and t can be written as:
                               <span class="disp-formula" id="disp-formula-44">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-53.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-53.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-45">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-54.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-54.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where R is the set of all loci nodes in the tree (including the root) excluding locus l. The joint probability distribution is calculated as
                               <span class="disp-formula" id="disp-formula-46">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-55.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-55.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The set 𝒫 (c, t) denotes all nodes on the shortest path from cell c to the root of the tree (including the root and excluding the cell c node). An example of the path on an imaginary tree is depicted in
                               <strong>
                                Supplementary Fig. 11
                               </strong>
                               . The nodes coloured in green belong to (c, t). Therefore, the posterior probability distribution of x
                               <sub>
                                c,l
                               </sub>
                               = 1 yields
                               <span class="disp-formula" id="disp-formula-47">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-56.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-56.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Rewriting
                               <a id="xref-disp-formula-47-1" class="xref-disp-formula" href="#disp-formula-47">
                                Equation (20)
                               </a>
                               assuming uniform probability distribution for p(t ′) yields:
                               <span class="disp-formula" id="disp-formula-48">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-57.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-57.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where N denotes the set of all trait nodes, C denotes the set of all cell nodes, C
                               <sub>
                                v
                               </sub>
                               denotes the cells that are a descendant of node v, and C
                               <sub>
                                \v
                               </sub>
                               denotes the cells that are a not descendant of node v. The product of the likelihood contributions for non-descendant nodes can be calculated by taking the product of
                               <span class="inline-formula" id="inline-formula-43">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-43.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-43.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               for all cells, divided by the ones that are descendant of v:
                               <span class="disp-formula" id="disp-formula-49">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-58.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-58.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Therefore:
                               <span class="disp-formula" id="disp-formula-50">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-59.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-59.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               The likelihood contribution of descendant cells can be re-indexed by a binary vector
                               <strong>
                                b
                               </strong>
                               = (b
                               <sub>
                                1
                               </sub>
                               , b
                               <sub>
                                2
                               </sub>
                               ,…, b
                               <sub>
                                k
                               </sub>
                               ), where b
                               <sub>
                                i
                               </sub>
                               ∈ {0, 1}, and b
                               <sub>
                                i
                               </sub>
                               = 1 if the child v is to be moved into a child of the node l. The value of k denotes the number of children of v. The i
                               <sup>
                                *
                               </sup>
                               th child of v which is on the path from node v to cell c is called
                               <span class="inline-formula" id="inline-formula-44">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-44.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-44.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . This implies
                               <span class="inline-formula" id="inline-formula-45">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-45.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/inline-graphic-45.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (See
                               <strong>
                                Supplementary Fig. 11
                               </strong>
                               ). Therefore:
                               <span class="disp-formula" id="disp-formula-51">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-60.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-60.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Rewriting
                               <a id="xref-disp-formula-50-1" class="xref-disp-formula" href="#disp-formula-50">
                                Equation (21)
                               </a>
                               using
                               <a id="xref-disp-formula-51-1" class="xref-disp-formula" href="#disp-formula-51">
                                Equation (22)
                               </a>
                               yields:
                               <span class="disp-formula" id="disp-formula-52">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-61.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-61.gif"/>
                                 </noscript>
                                </span>
                               </span>
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-33">
                             <h3>
                              9.8 Computational complexity of the SNV calling algorithm
                             </h3>
                             <p id="p-129">
                              The computational complexity of
                              <a id="xref-disp-formula-52-1" class="xref-disp-formula" href="#disp-formula-52">
                               Equation (23)
                              </a>
                              is O(| C | · | L |) with C the number of cells and | L | the number of loci. In order to reduce the complexity of calculating p(x
                              <sub>
                               c,l
                              </sub>
                              = 1 y, t) for each locus and cell, 𝒫 ′ (c, t) is defined to denote the nodes sitting on the path from root to cell c, excluding the root node and including the cell c node. Then,
                              <span class="disp-formula" id="disp-formula-53">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-62.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-62.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Therefore,
                              <span class="disp-formula" id="disp-formula-54">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-63.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/11/11/2020.05.06.058180/embed/graphic-63.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Calculating p(x
                              <sub>
                               c,l
                              </sub>
                              = 1|y, t) with a recursive approach reduces the complexity from O(|C||L|) to O(|C| + |L|), where as in the last section L is the union of SNV and CNA loci.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-34">
                            <h2 class="">
                             List of Supplementary Figures
                            </h2>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F10.large.jpg?width=800&height=600&carousel=1" title="(a) Reading from left to right: the interpretation of removing a column in the matrix x is to perform contraction of an edge corresponding to a locus shown in bold. Reading from right to left: the interpretation of inserting back a column while assigning new binary values is an edge insertion. The circled node v refers to Step 1. The subtrees in bold refer to those selected in Step 2. The edge in bold, the one introduced in Step 3. (b) Decomposition used for the recursion of Section 9.4.3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">(<strong>a</strong>) Reading from left to right: the interpretation of removing a column in the matrix <em>x</em> is to perform contraction of an edge corresponding to a locus shown in bold. Reading from right to left: the interpretation of inserting back a column while assigning new binary values is an edge insertion. The circled node <em>v</em> refers to Step 1. The subtrees in bold refer to those selected in Step 2. The edge in bold, the one introduced in Step 3. (<strong>b</strong>) Decomposition used for the recursion of Section 9.4.3.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F10.medium.gif" width="440" height="109"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F10.medium.gif" width="440" height="109"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 1.
                              </span>
                              <p id="p-130" class="first-child">
                               (
                               <strong>
                                a
                               </strong>
                               ) Reading from left to right: the interpretation of removing a column in the matrix x is to perform contraction of an edge corresponding to a locus shown in bold. Reading from right to left: the interpretation of inserting back a column while assigning new binary values is an edge insertion. The circled node v refers to Step 1. The subtrees in bold refer to those selected in Step 2. The edge in bold, the one introduced in Step 3. (
                               <strong>
                                b
                               </strong>
                               ) Decomposition used for the recursion of
                               <a id="xref-sec-22-4" class="xref-sec" href="#sec-22">
                                Section 9.4.3
                               </a>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F11.large.jpg?width=800&height=600&carousel=1" title="Phylogenetic tree and CNA profile heatmap for the SA535 dataset. The rows of the heatmap are sorted according to the placement of cells on the phylogenetic tree. The columns of the heatmap are sorted by their genomic position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">Phylogenetic tree and CNA profile heatmap for the SA535 dataset. The rows of the heatmap are sorted according to the placement of cells on the phylogenetic tree. The columns of the heatmap are sorted by their genomic position.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F11.medium.gif" width="440" height="179"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F11.medium.gif" width="440" height="179"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 2.
                              </span>
                              <p id="p-131" class="first-child">
                               Phylogenetic tree and CNA profile heatmap for the SA535 dataset. The rows of the heatmap are sorted according to the placement of cells on the phylogenetic tree. The columns of the heatmap are sorted by their genomic position.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F12.large.jpg?width=800&height=600&carousel=1" title="Phylogenetic tree and CNA profile heatmap for the OV A dataset. The nearly diploid cells with the loss of heterozygosity on chromosome X are from SA1090. The cells with an amplification on chromosome 22 are from SA922. The rest belong to SA921." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Phylogenetic tree and CNA profile heatmap for the <em>OV A</em> dataset. The nearly diploid cells with the loss of heterozygosity on chromosome X are from SA1090. The cells with an amplification on chromosome 22 are from SA922. The rest belong to SA921.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F12.medium.gif" width="440" height="179"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F12.medium.gif" width="440" height="179"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 3." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 3.
                              </span>
                              <p id="p-132" class="first-child">
                               Phylogenetic tree and CNA profile heatmap for the OV A dataset. The nearly diploid cells with the loss of heterozygosity on chromosome X are from SA1090. The cells with an amplification on chromosome 22 are from SA922. The rest belong to SA921.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F13.large.jpg?width=800&height=600&carousel=1" title="Phylogenetic tree and CNA profile heatmap for the SA501 dataset. Note that the diploid cells at the bottom of the heatmap are control cells that were included in the experiment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">Phylogenetic tree and CNA profile heatmap for the SA501 dataset. Note that the diploid cells at the bottom of the heatmap are control cells that were included in the experiment.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F13.medium.gif" width="440" height="179"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F13.medium.gif" width="440" height="179"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 4." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 4.
                              </span>
                              <p id="p-133" class="first-child">
                               Phylogenetic tree and CNA profile heatmap for the SA501 dataset. Note that the diploid cells at the bottom of the heatmap are control cells that were included in the experiment.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F14.large.jpg?width=800&height=600&carousel=1" title="the distribution of mismatch rate defined as the fraction of cells that have a mismatch between the inferred and jitter-fixed value of a marker." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">the distribution of mismatch rate defined as the fraction of cells that have a mismatch between the inferred and jitter-fixed value of a marker.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F14.medium.gif" width="440" height="146"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F14.medium.gif" width="440" height="146"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 5." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 5.
                              </span>
                              <p id="p-134" class="first-child">
                               the distribution of mismatch rate defined as the fraction of cells that have a mismatch between the inferred and jitter-fixed value of a marker.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F15.large.jpg?width=800&height=600&carousel=1" title="(a) Tree imbalance index where zero indicates that the tree is consistent with one simulated from a Yule model (completely balanced) and positive values indicate deviation from the Yule model (more imbalanced). For ease of plotting, each balance index is normalised by the absolute value of the maximum estimated statistic among all samples. Cumulatively adding more timepoints (b), or for the maximal subtree comprising cells of a specific timepoint (c)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><strong>(a)</strong> Tree imbalance index where zero indicates that the tree is consistent with one simulated from a Yule model (completely balanced) and positive values indicate deviation from the Yule model (more imbalanced). For ease of plotting, each balance index is normalised by the absolute value of the maximum estimated statistic among all samples. Cumulatively adding more timepoints (<strong>b</strong>), or for the maximal subtree comprising cells of a specific timepoint (<strong>c</strong>).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F15.medium.gif" width="440" height="219"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F15.medium.gif" width="440" height="219"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 6." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 6.
                              </span>
                              <p id="p-135" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Tree imbalance index where zero indicates that the tree is consistent with one simulated from a Yule model (completely balanced) and positive values indicate deviation from the Yule model (more imbalanced). For ease of plotting, each balance index is normalised by the absolute value of the maximum estimated statistic among all samples. Cumulatively adding more timepoints (
                               <strong>
                                b
                               </strong>
                               ), or for the maximal subtree comprising cells of a specific timepoint (
                               <strong>
                                c
                               </strong>
                               ).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F16.large.jpg?width=800&height=600&carousel=1" title="Synthetic datasets simulated from the GRFBS model. The number of cells, number of loci, and β values are are (500 × 800, − 0.83), (1000 × 400, − 0.1), (2000 ×800, − 0.9), (3000 ×400, − 0.48) for the upper left, upper right, lower left, and lower right figures respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Synthetic datasets simulated from the GRFBS model. The number of cells, number of loci, and <em>β</em> values are are (500 × 800, − 0.83), (1000 × 400, − 0.1), (2000 ×800, − 0.9), (3000 ×400, − 0.48) for the upper left, upper right, lower left, and lower right figures respectively.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F16.medium.gif" width="440" height="299"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F16.medium.gif" width="440" height="299"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 7." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 7.
                              </span>
                              <p id="p-136" class="first-child">
                               Synthetic datasets simulated from the GRFBS model. The number of cells, number of loci, and β values are are (500 × 800, − 0.83), (1000 × 400, − 0.1), (2000 ×800, − 0.9), (3000 ×400, − 0.48) for the upper left, upper right, lower left, and lower right figures respectively.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F17" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F17.large.jpg?width=800&height=600&carousel=1" title="Tree reconstruction evaluation using a normalized Robinson–Foulds metric on synthetic datasets from S90, simulated from Beta-splitting processes. Here normalization is done by dividing the RF distance of each inference method by the worst performer per dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Tree reconstruction evaluation using a normalized Robinson–Foulds metric on synthetic datasets from <em>S</em>90, simulated from Beta-splitting processes. Here normalization is done by dividing the RF distance of each inference method by the worst performer per dataset.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F17.medium.gif" width="440" height="439"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F17.medium.gif" width="440" height="439"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 8." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 8.
                              </span>
                              <p id="p-137" class="first-child">
                               Tree reconstruction evaluation using a normalized Robinson–Foulds metric on synthetic datasets from S90, simulated from Beta-splitting processes. Here normalization is done by dividing the RF distance of each inference method by the worst performer per dataset.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F18" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F18.large.jpg?width=800&height=600&carousel=1" title="A model and estimator comparison based on tree reconstruction accuracy for datasets from S10. For each dataset, inference was performed on both the globally- and locally-parameterized model. Both the greedy and trace search estimates were computed for each inference result." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A model and estimator comparison based on tree reconstruction accuracy for datasets from <em>S</em>10. For each dataset, inference was performed on both the globally- and locally-parameterized model. Both the greedy and trace search estimates were computed for each inference result.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F18.medium.gif" width="440" height="260"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 9." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F18.medium.gif" width="440" height="260"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 9." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 9.
                              </span>
                              <p id="p-138" class="first-child">
                               A model and estimator comparison based on tree reconstruction accuracy for datasets from S10. For each dataset, inference was performed on both the globally- and locally-parameterized model. Both the greedy and trace search estimates were computed for each inference result.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F19" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F19.large.jpg?width=800&height=600&carousel=1" title="A locus in SA535 that violates the perfect phylogeny assumption. The method is robust to this violation and correctly allocates the other relations (positive x entries shown in black in the matrix; positive y entries shown in orange; posterior marginals m in shades of red). (a) the part of the tree where the violation occurs, magnified in (b), where the short band of orange not matched with a black band is indicative of the violation. (c) the corresponding tree showing the two points of the tree where the two events collocated in the same bin occurred in the tree. The sister clades of a,b and d suggest that this is a case of two insertion events rather than a loss." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">A locus in <em>SA</em>535 that violates the perfect phylogeny assumption. The method is robust to this violation and correctly allocates the other relations (positive <em>x</em> entries shown in black in the matrix; positive <em>y</em> entries shown in orange; posterior marginals <em>m</em> in shades of red). <strong>(a)</strong> the part of the tree where the violation occurs, magnified in <strong>(b)</strong>, where the short band of orange not matched with a black band is indicative of the violation. <strong>(c)</strong> the corresponding tree showing the two points of the tree where the two events collocated in the same bin occurred in the tree. The sister clades of a,b and d suggest that this is a case of two insertion events rather than a loss.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F19.medium.gif" width="440" height="130"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 10." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F19.medium.gif" width="440" height="130"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 10." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 10.
                              </span>
                              <p id="p-139" class="first-child">
                               A locus in SA535 that violates the perfect phylogeny assumption. The method is robust to this violation and correctly allocates the other relations (positive x entries shown in black in the matrix; positive y entries shown in orange; posterior marginals m in shades of red).
                               <strong>
                                (a)
                               </strong>
                               the part of the tree where the violation occurs, magnified in
                               <strong>
                                (b)
                               </strong>
                               , where the short band of orange not matched with a black band is indicative of the violation.
                               <strong>
                                (c)
                               </strong>
                               the corresponding tree showing the two points of the tree where the two events collocated in the same bin occurred in the tree. The sister clades of a,b and d suggest that this is a case of two insertion events rather than a loss.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F20" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F20.large.jpg?width=800&height=600&carousel=1" title="A schematic view of the underlying tree inferred from CNA and SNV loci across multiple cells. Black and white nodes represent cells and loci, respectively. The grey triangle represents a subtree rooted at a node. It includes all of the nodes and edges in the subtree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">A schematic view of the underlying tree inferred from CNA and SNV loci across multiple cells. Black and white nodes represent cells and loci, respectively. The grey triangle represents a subtree rooted at a node. It includes all of the nodes and edges in the subtree.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F20.medium.gif" width="440" height="422"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 11." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F20.medium.gif" width="440" height="422"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 11." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 11.
                              </span>
                              <p id="p-140" class="first-child">
                               A schematic view of the underlying tree inferred from CNA and SNV loci across multiple cells. Black and white nodes represent cells and loci, respectively. The grey triangle represents a subtree rooted at a node. It includes all of the nodes and edges in the subtree.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F21" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F21.large.jpg?width=800&height=600&carousel=1" title="Filtering the CNA data for tree inference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">Filtering the CNA data for tree inference.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F21.medium.gif" width="167" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 12." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F21.medium.gif" width="167" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 12." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 12.
                              </span>
                              <p id="p-141" class="first-child">
                               Filtering the CNA data for tree inference.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F22" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F22.large.jpg?width=800&height=600&carousel=1" title="An example of replicating cells. Note the scattered localised deletions. This heatmap is from a HER2+ PDX line. These late replicating cells form a finger like clade in the tree. The top inset shows chromosome 4 while the bottom inset spans chromosomes 7 and 8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">An example of replicating cells. Note the scattered localised deletions. This heatmap is from a HER2+ PDX line. These late replicating cells form a <em>finger</em> like clade in the tree. The top inset shows chromosome 4 while the bottom inset spans chromosomes 7 and 8.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F22.medium.gif" width="440" height="164"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 13." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F22.medium.gif" width="440" height="164"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 13." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 13.
                              </span>
                              <p id="p-142" class="first-child">
                               An example of replicating cells. Note the scattered localised deletions. This heatmap is from a HER2+ PDX line. These late replicating cells form a finger like clade in the tree. The top inset shows chromosome 4 while the bottom inset spans chromosomes 7 and 8.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F23" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F23.large.jpg?width=800&height=600&carousel=1" title="SNV variant reads data and SNV call probabilities for SA535 dataset beside the underlying phylogenetic tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">SNV variant reads data and SNV call probabilities for SA535 dataset beside the underlying phylogenetic tree.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F23.medium.gif" width="440" height="387"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 14." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F23.medium.gif" width="440" height="387"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 14." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 14.
                              </span>
                              <p id="p-143" class="first-child">
                               SNV variant reads data and SNV call probabilities for SA535 dataset beside the underlying phylogenetic tree.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F24" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F24.large.jpg?width=800&height=600&carousel=1" title="SNV variant reads data and SNV call probabilities for OVA dataset beside the underlying phylogenetic tree.." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">SNV variant reads data and SNV call probabilities for OVA dataset beside the underlying phylogenetic tree..</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F24.medium.gif" width="440" height="388"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 15." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F24.medium.gif" width="440" height="388"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 15." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 15.
                              </span>
                              <p id="p-144" class="first-child">
                               SNV variant reads data and SNV call probabilities for OVA dataset beside the underlying phylogenetic tree..
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F25" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F25.large.jpg?width=800&height=600&carousel=1" title="SNV variant reads data and SNV call probabilities for SA501 dataset beside the underlying phylogenetic tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-818011257" data-figure-caption='<div class="highwire-markup">SNV variant reads data and SNV call probabilities for SA501 dataset beside the underlying phylogenetic tree.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Supplemental Figure 16." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F25.medium.gif" width="440" height="391"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Supplemental Figure 16." src="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F25.medium.gif" width="440" height="391"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Supplemental Figure 16." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/11/11/2020.05.06.058180/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Supplemental Figure 16.
                              </span>
                              <p id="p-145" class="first-child">
                               SNV variant reads data and SNV call probabilities for SA501 dataset beside the underlying phylogenetic tree.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-146">
                             <strong>
                              Supplemental Figure 1
                             </strong>
                             . (
                             <strong>
                              a
                             </strong>
                             ) Reading from left to right: the interpretation of removing a column in the matrix x is to perform contraction of an edge corresponding to a locus shown in bold. Reading from right to left: the interpretation of inserting back a column while assigning new binary values is an edge insertion. The circled node v refers to Step 1. The subtrees in bold refer to those selected in Step 2. The edge in bold, the one introduced in Step 3. (
                             <strong>
                              b
                             </strong>
                             ) Decomposition used for the recursion of
                             <a id="xref-sec-22-5" class="xref-sec" href="#sec-22">
                              Section 9.4.3
                             </a>
                             .
                            </p>
                            <p id="p-147">
                             <strong>
                              Supplemental Figure 2
                             </strong>
                             . Phylogenetic tree and CNA profile heatmap for the SA535 dataset
                            </p>
                            <p id="p-148">
                             <strong>
                              Supplemental Figure 3
                             </strong>
                             . Phylogenetic tree and CNA profile heatmap for the OV A
                            </p>
                            <p id="p-149">
                             <strong>
                              Supplemental Figure 4
                             </strong>
                             . Phylogenetic tree and CNA profile heatmap for the SA501 dataset
                            </p>
                            <p id="p-150">
                             <strong>
                              Supplemental Figure 5
                             </strong>
                             . The distribution of mismatch rate
                            </p>
                            <p id="p-151">
                             <strong>
                              Supplemental Figure 6
                             </strong>
                             . The tree topology balance
                            </p>
                            <p id="p-152">
                             <strong>
                              Supplemental Figure 7
                             </strong>
                             . A subset of synthetic datasets used for benchmarking
                            </p>
                            <p id="p-153">
                             <strong>
                              Supplemental Figure 8
                             </strong>
                             . Tree reconstruction evaluation on synthetic datasets.
                            </p>
                            <p id="p-154">
                             <strong>
                              Supplemental Figure 9
                             </strong>
                             . Synthetic experiment comparing global versus local parameterizations.
                            </p>
                            <p id="p-155">
                             <strong>
                              Supplemental Figure 10
                             </strong>
                             . A locus in SA535 that violates the perfect phylogeny assumption.
                            </p>
                            <p id="p-156">
                             <strong>
                              Supplemental Figure 11
                             </strong>
                             . A schematic representation of CNA and SNV tree
                            </p>
                            <p id="p-157">
                             <strong>
                              Supplemental Figure 12
                             </strong>
                             . Filtering the CNA data for tree inference.
                            </p>
                            <p id="p-158">
                             <strong>
                              Supplemental Figure 13
                             </strong>
                             . An example of replicating cells
                            </p>
                            <p id="p-159">
                             <strong>
                              Supplemental Figure 14
                             </strong>
                             . SNV variant reads data and SNV call probabilities for SA535 dataset beside the underlying phylogenetic tree.
                            </p>
                            <p id="p-160">
                             <strong>
                              Supplemental Figure 15
                             </strong>
                             . SNV variant reads data and SNV call probabilities for OVA dataset beside the underlying phylogenetic tree..
                            </p>
                            <p id="p-161">
                             <strong>
                              Supplemental Figure 16
                             </strong>
                             . SNV variant reads data and SNV call probabilities for SA501 dataset beside the underlying phylogenetic tree.
                            </p>
                            <div id="T1" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2847338/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup last">
                                 <a href="/highwire/markup/2847338/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplemental Table 1.
                              </span>
                              <p id="p-162" class="first-child">
                               Summary of real-world datasets used.
                               <kbd>
                                final
                               </kbd>
                               is the final number of cells after all filters except for
                               <kbd>
                                !lmr
                               </kbd>
                               are applied.
                               <kbd>
                                final
                               </kbd>
                               additionally filters out lmr cells, those that have total mapped reads fewer than 500,000. Abbreviations used are tp: time point; qual. : quality;
                               <kbd>
                                !sphase
                               </kbd>
                               : not sphase;
                               <kbd>
                                !lmr
                               </kbd>
                               : not low mapped reads.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="T2" class="table pos-float">
                             <div class="table-inline table-callout-links">
                              <div class="callout">
                               <span>
                                View this table:
                               </span>
                               <ul class="callout-links">
                                <li class="view-inline first">
                                 <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2847352/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                  View inline
                                 </a>
                                </li>
                                <li class="view-popup">
                                 <a href="/highwire/markup/2847352/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                  View popup
                                 </a>
                                </li>
                                <li class="download-ppt last">
                                 <a href="/highwire/powerpoint/2847352" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                  Download powerpoint
                                 </a>
                                </li>
                               </ul>
                              </div>
                             </div>
                             <div class="table-caption">
                              <span class="table-label">
                               Supplemental Table 2.
                              </span>
                              <p id="p-163" class="first-child">
                               Inference settings used for each dataset.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             4 Acknowledgements
                            </h2>
                            <p id="p-43">
                             This project was generously supported by the BC Cancer Foundation at BC Cancer and Cycle for Survival supporting Memorial Sloan Kettering Cancer Center. SPS holds the Nicholls Biondi Chair in Computational Oncology and is a Susan G. Komen Scholar (#GC233085). SA holds the Nan and Lorraine Robertson Chair in Breast Cancer and is a Canada Research Chair in Molecular Oncology (950-230610). Additional funding provided by the Terry Fox Research Institute Grant 1082, Canadian Cancer Society Research Institute Impact program Grant 705617, CIHR Grant FDN-148429, Breast Cancer Research Foundation award (BCRF-18-180, BCRF-19-180 and BCRF-20-180), MSK Cancer Center Support Grant/Core Grant (P30 CA008748), National Institutes of Health Grant (1RM1 HG011014-01), CCSRI Grant (#705636), the Cancer Research UK Grand Challenge Program, Canada Foundation for Innovation (40044) to SA, SPS and ABC. We extend our gratitude to Sarah P. Otto for her helpful comments on a draft of this manuscript.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-2">
                              <p id="p-5">
                               Improvements to the writing, performing new comparisons with additional recently developed methods, new experiments comparing MLE and Bayesian methods, as well as new experiments investigating the effect of ignoring independence across sites.
                              </p>
                             </li>
                             <li class="fn" id="fn-3">
                              <p id="p-164">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 1
                                </sup>
                               </span>
                               Authors of [
                               <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                                42
                               </a>
                               ] argue the GBFBS model is capable of realizing topologies comparable to that of the original Beta-splitting model. When β = α → −1, trees are totally imbalanced; when β = α → ∞, trees are perfectly balanced.
                              </p>
                             </li>
                             <li class="fn" id="fn-4">
                              <p id="p-165">
                               <a class="rev-xref" href="#xref-fn-4-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 2
                                </sup>
                               </span>
                               We used the R packages [
                               <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                47
                               </a>
                               ,
                               <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                48
                               </a>
                               ] for simulation.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Laks
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Clonal Decomposition and DNA Replication States Defined by Scaled Single-Cell Genome Sequencing
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  179
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  1207
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1221.e22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCell%26rft.volume%253D179%26rft.spage%253D1207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Pellegrino
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  High-throughput single-cell DNA sequencing of acute myeloid leukemia tumors with droplet micro?uidics
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1345
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1352
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2Bresearch%26rft.volume%253D28%26rft.spage%253D1345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                  <span class="cit-name-surname">
                                   Baslan
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Genome-wide copy number analysis of single cells
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature pro-tocols
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1024
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1041
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2Bpro-tocols%26rft.volume%253D7%26rft.spage%253D1024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gawad
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Koh
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Quake
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Single-cell genome sequencing: current state of the science
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature Reviews Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  175
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2BReviews%2BGenetics%26rft.volume%253D17%26rft.spage%253D175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Salehi
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Clonal fitness inferred from time-series modelling of single-cell can-cer genomes
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  595
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7868
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  585
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  590
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%26rft.volume%253D595%26rft.spage%253D585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Quinn
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Single-cell lineages reveal the rates, routes, and drivers of metas-tasis in cancer xenografts
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Science
                                 </abbr>
                                 <span class="cit-vol">
                                  371
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6532
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  eabc1944
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BScience%26rft.volume%253D371%26rft.spage%253D1944eabc%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Abbosh
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Phylogenetic ctDNA analysis depicts early-stage lung cancer evo-lution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  545
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7655
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  446
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  451
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%26rft.volume%253D545%26rft.spage%253D446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Clonal evolution in breast cancer revealed by single nucleus genome sequencing
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  512
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7513
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  155
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  160
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%26rft.volume%253D512%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Leung
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Single-cell DNA sequencing reveals a late-dissemination model in metastatic colorectal cancer
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome research
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1287
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1299
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2Bresearch%26rft.volume%253D27%26rft.spage%253D1287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Discovery of biclonal origin and a novel oncogene SLC12A5 in colon cancer by single-cell sequencing
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Cell research
                                 </abbr>
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  701
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  712
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCell%2Bresearch%26rft.volume%253D24%26rft.spage%253D701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schwartz
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A. A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schäffer
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  The evolution of tumour phylogenetics: principles and practice
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature Reviews Genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  229
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2BReviews%2BGenetics%26rft.volume%253D18%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Jahn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kuipers
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                  <span class="cit-name-surname">
                                   Beerenwinkel
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Tree inference for single-cell data
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  86
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2BBiology%26rft.volume%253D17%26rft.spage%253D86%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E. M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ross
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Markowetz
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  OncoNEM: inferring tumor evolution from single-cell sequencing data
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  69
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2BBiology%26rft.volume%253D17%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. A
                                  </span>
                                  <span class="cit-name-surname">
                                   Miller
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  SciClone: inferring clonal architecture and tracking the spatial and temporal patterns of tumor evolution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: PLoS Comput Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ),
                                 <span class="cit-fpage">
                                  e1003665
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BPLoS%2BComput%2BBiol%26rft.volume%253D10%26rft.spage%253D1003665e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.15" data-doi="10.1038/s41467-017-02554-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Single-cell mutation identification via phylogenetic inference
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2Bcommunications%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-02554-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-017-02554-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zahn
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Scalable whole-genome single-cell library preparation without pream-plification
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  167
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  173
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2BMethods%26rft.volume%253D14%26rft.spage%253D167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.17">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kaufmann
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  MEDICC2: whole-genome doubling aware copy-number phy-logenies for cancer evolution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                  <span class="cit-name-surname">
                                   Satas
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Scarlet: Single-cell tumor phylogeny inference with copy-number constrained mutation losses
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Cell Systems
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  323
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  332
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCell%2BSystems%26rft.volume%253D10%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.19">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  MEDALT: single-cell copy number lineage tracing enabling gene dis-covery
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2Bbiology%26rft.volume%253D22%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Single-Cell Exome Sequencing Reveals Single-Nucleotide Mutation Characteristics of a Kidney Tumor
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  148
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  886
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  895
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCell%26rft.volume%253D148%26rft.spage%253D886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kuhner
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Felsenstein
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A simulation comparison of phylogeny algorithms under equal and unequal evolutionary rates
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Molecular Biology and Evolution
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1994
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  459
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  468
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BMolecular%2BBiology%2Band%2BEvolution%26rft.volume%253D11%26rft.spage%253D459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Guindon
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gascuel
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Systematic Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  696
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  704
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BSystematic%2BBiology%26rft.volume%253D52%26rft.spage%253D696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   T. L.
                                  </span>
                                  <span class="cit-name-surname">
                                   Williams
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   B. M. E.
                                  </span>
                                  <span class="cit-name-surname">
                                   Moret
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  An investigation of phylogenetic likelihood meth-ods
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Third IEEE Symposium on Bioinformatics and Bioengineering, 2003. Pro-ceedings
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  79
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  86
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Som
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  ML or NJ-MCL? A comparison between two robust phylogenetic methods
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Computational Biology and Chemistry
                                 </abbr>
                                 <span class="cit-vol">
                                  33
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  373
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  378
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BComputational%2BBiology%2Band%2BChemistry%26rft.volume%253D33%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.25">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Malikic
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Integrative inference of subclonal tumour evolution from single-cell and bulk sequencing data
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2Bcommunications%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  The infinite sites model of genome evolution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  105
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  38
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  14254
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14261
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft.volume%253D105%26rft.spage%253D14254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C. D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Greenman
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Estimation of rearrangement phylogeny for cancer genomes
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome Research
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  346
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  361
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2BResearch%26rft.volume%253D22%26rft.spage%253D346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                  <span class="cit-name-surname">
                                   Zafar
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  SiFit: inferring tumor trees from single-cell sequencing data under finite-sites models
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  20
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2Bbiology%26rft.volume%253D18%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Househam
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Phenotypic plasticity and genetic control in colorectal cancer evolution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Eirew
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  518
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7539
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  422
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  426
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%26rft.volume%253D518%26rft.spage%253D422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                  <span class="cit-name-surname">
                                   Yi
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                  <span class="cit-name-surname">
                                   Seok Ju
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Patterns and mechanisms of structural variations in human cancer
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Experimental & Molecular Medicine
                                 </abbr>
                                 <span class="cit-vol">
                                  50
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  8
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  98
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BExperimental%2B%2526%2BMolecular%2BMedicine%26rft.volume%253D50%26rft.spage%253D98%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Mishra
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Whetstine
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Different Facets of Copy Number Changes: Per-manent, Transient, and Adaptive
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Molecular and Cellular Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1050
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1063
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BMolecular%2Band%2BCellular%2BBiology%26rft.volume%253D36%26rft.spage%253D1050%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.058180v3.33">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Wilson
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  Generating Random Spanning Trees More Quickly Than the Cover Time
                                 </span>
                                 ”.
                                 <span class="cit-source">
                                  In: Proceedings of the Twenty-eighth Annual ACM Symposium on Theory of Computing. STOC ‘
                                 </span>
                                 <span class="cit-vol">
                                  96
                                 </span>
                                 .
                                 <span class="cit-publ-loc">
                                  New York, NY, USA: ACM
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  296
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  303
                                 </span>
                                 . ISBN:
                                 <span class="cit-isbn">
                                  978-0-89791-785-8
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BProceedings%2Bof%2Bthe%2BTwenty-eighth%2BAnnual%2BACM%2BSymposium%2Bon%2BTheory%2Bof%2BComputing.%2BSTOC%2B%253F%26rft.volume%253D96%26rft.spage%253D296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.34">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Y. W.
                                  </span>
                                  <span class="cit-name-surname">
                                   Teh
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Dirichlet Process
                                 </span>
                                 .”
                                 <abbr class="cit-jnl-abbrev">
                                  In: Encyclopedia of machine learning
                                 </abbr>
                                 <span class="cit-vol">
                                  1063
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  280
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  287
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BEncyclopedia%2Bof%2Bmachine%2Blearning%26rft.volume%253D1063%26rft.spage%253D280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Bouchard-Côté
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Blang: Probabilitistic Programming for Combinatorial Spaces
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Journal of Statistical Software
                                 </abbr>
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  98
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D103%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                  <span class="cit-name-surname">
                                   Geman
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Geman
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Stochastic relaxation, Gibbs distributions, and the Bayesian restoration of images
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: IEEE Transactions on pattern analysis and ma-chine intelligence
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1984
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  721
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  741
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BIEEE%2BTransactions%2Bon%2Bpattern%2Banalysis%2Band%2Bma-chine%2Bintelligence%26rft.volume%253D6%26rft.spage%253D721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.37">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                  <span class="cit-name-surname">
                                   Neal
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Slice sampling
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: The Annals of Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  31
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  705
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  767
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BThe%2BAnnals%2Bof%2BStatistics%26rft.volume%253D31%26rft.spage%253D705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.058180v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Robert
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  The Bayesian Choice -From Decision-Theoretic Foundations | Christian Robert |
                                 </span>
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D. F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   L. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Foulds
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Comparison of phylogenetic trees
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Mathemat-ical biosciences
                                 </abbr>
                                 <span class="cit-vol">
                                  53
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1-2
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1981
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  131
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  147
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BMathemat-ical%2Bbiosciences%26rft.volume%253D53%26rft.spage%253D131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.058180v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Desper
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gascuel
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  Fast and accurate phylogeny reconstruction algorithms based on the minimum-evolution principle
                                 </span>
                                 ”.
                                 <span class="cit-source">
                                  In: International Workshop on Algorithms in Bioinformatics
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 .
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  374
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.41" data-doi="10.1186/S13059-020-02000-8/FIGURES/6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   X. F
                                  </span>
                                  <span class="cit-name-surname">
                                   Mallory
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Methods for copy number aberration detection from single-cell DNA-sequencing data
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Genome biology
                                 </abbr>
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BGenome%2Bbiology%26rft.volume%253D21%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252FS13059-020-02000-8%252FFIGURES%252F6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/S13059-020-02000-8/FIGURES/6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Sainudiin
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                  <span class="cit-name-surname">
                                   Véber
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  A Beta-splitting model for evolutionary trees
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Royal Society open science
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  160016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BRoyal%2BSociety%2Bopen%2Bscience%26rft.volume%253D3%26rft.spage%253D160016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M. G. B.
                                  </span>
                                  <span class="cit-name-surname">
                                   Blum
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                  <span class="cit-name-surname">
                                   François
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Which random processes describe the tree of life? A large-scale study of phylogenetic tree imbalance
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Systematic Biology
                                 </abbr>
                                 <span class="cit-vol">
                                  55
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  685
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  691
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BSystematic%2BBiology%26rft.volume%253D55%26rft.spage%253D685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.05.06.058180v3.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Aldous
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-chapter-title">
                                  Probability distributions on cladograms
                                 </span>
                                 ”.
                                 <span class="cit-source">
                                  In: Random discrete structures
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1996
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 “
                                 <span class="cit-article-title">
                                  Punctuated copy number evolution and clonal stasis in triple-negative breast cancer
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Nature genetics
                                 </abbr>
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), p.
                                 <span class="cit-fpage">
                                  1119
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BNature%2Bgenetics%26rft.volume%253D48%26rft.spage%253D1119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   J. F. C.
                                  </span>
                                  <span class="cit-name-surname">
                                   Kingman
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  The coalescent
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Stochastic processes and their applications
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  235
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BStochastic%2Bprocesses%2Band%2Btheir%2Bapplications%26rft.volume%253D13%26rft.spage%253D235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   K.P.
                                  </span>
                                  <span class="cit-name-surname">
                                   Schliep
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  phangorn: phylogenetic analysis in R
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  592
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  593
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BBioinformatics%26rft.volume%253D27%26rft.spage%253D592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P. R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Staab
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                  <span class="cit-name-surname">
                                   Metzler
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Coala: an R framework for coalescent simulation
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  1903
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1904
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BBioinformatics%26rft.volume%253D32%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.05.06.058180v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   W. J.
                                  </span>
                                  <span class="cit-name-surname">
                                   Youden
                                  </span>
                                 </span>
                                 . “
                                 <span class="cit-article-title">
                                  Index for rating diagnostic tests
                                 </span>
                                 ”.
                                 <abbr class="cit-jnl-abbrev">
                                  In: Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 .
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1950
                                 </span>
                                 ), pp.
                                 <span class="cit-fpage">
                                  32
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  35
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DIn%253A%2BCancer%26rft.volume%253D3%26rft.spage%253D32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.05.06.058180v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.12.07.471678v4" title="Rapid bacterial and fungal successional dynamics in first year after Chaparral wildfire" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.07.09.499435v2" title="Structure-Preserving Visualization for Single-cell RNA-Seq Profiles Using Deep Manifold Transformation with Batch-Correction" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted November 11, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.05.06.058180v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.05.06.058180v3.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.05.06.058180v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.05.06.058180v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Cancer phylogenetic tree inference at scale from 1000s of single cell genomes
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2844645"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-Fj5UNK6GArUUxDPDl6K7PwgUrdGa8CY6IHwh5urNHnM"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484470315"/>
                                   <input type="hidden" name="captcha_token" value="e92899e13fe140ac67d1a1d5f20595be"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2844645--3" data-node-nid="2844645" id="node-2844645--41347331275" data-pisa="biorxiv;2020.05.06.058180v3" data-pisa-master="biorxiv;2020.05.06.058180" data-seqnum="2844645" data-apath="/biorxiv/early/2022/11/11/2020.05.06.058180.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Cancer phylogenetic tree inference at scale from 1000s of single cell genomes
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Sohrab
                                  </span>
                                  <span class="nlm-surname">
                                   Salehi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Fatemeh
                                  </span>
                                  <span class="nlm-surname">
                                   Dorri
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Kevin
                                  </span>
                                  <span class="nlm-surname">
                                   Chern
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Farhia
                                  </span>
                                  <span class="nlm-surname">
                                   Kabeer
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Nicole
                                  </span>
                                  <span class="nlm-surname">
                                   Rusk
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Tyler
                                  </span>
                                  <span class="nlm-surname">
                                   Funnell
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Marc J
                                  </span>
                                  <span class="nlm-surname">
                                   Williams
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Daniel
                                  </span>
                                  <span class="nlm-surname">
                                   Lai
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Mirela
                                  </span>
                                  <span class="nlm-surname">
                                   Andronescu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Kieran R.
                                  </span>
                                  <span class="nlm-surname">
                                   Campbell
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="10">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   McPherson
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="11">
                                  <span class="nlm-given-names">
                                   Samuel
                                  </span>
                                  <span class="nlm-surname">
                                   Aparicio
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="12">
                                  <span class="nlm-given-names">
                                   Andrew
                                  </span>
                                  <span class="nlm-surname">
                                   Roth
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="13">
                                  <span class="nlm-given-names">
                                   Sohrab
                                  </span>
                                  <span class="nlm-surname">
                                   Shah
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="14">
                                  <span class="nlm-given-names">
                                   Alexandre
                                  </span>
                                  <span class="nlm-surname">
                                   Bouchard-Côté
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.05.06.058180;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.05.06.058180
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&title=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&title=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&text=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&t=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&title=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&title=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&title=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2844645--5" data-node-nid="2844645" id="citation-node-2844645--6482634670" data-pisa="biorxiv;2020.05.06.058180v3" data-pisa-master="biorxiv;2020.05.06.058180" data-seqnum="2844645" data-apath="/biorxiv/early/2022/11/11/2020.05.06.058180.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Cancer phylogenetic tree inference at scale from 1000s of single cell genomes
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Sohrab
                                      </span>
                                      <span class="nlm-surname">
                                       Salehi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Fatemeh
                                      </span>
                                      <span class="nlm-surname">
                                       Dorri
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Kevin
                                      </span>
                                      <span class="nlm-surname">
                                       Chern
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Farhia
                                      </span>
                                      <span class="nlm-surname">
                                       Kabeer
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Nicole
                                      </span>
                                      <span class="nlm-surname">
                                       Rusk
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Tyler
                                      </span>
                                      <span class="nlm-surname">
                                       Funnell
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Marc J
                                      </span>
                                      <span class="nlm-surname">
                                       Williams
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Daniel
                                      </span>
                                      <span class="nlm-surname">
                                       Lai
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Mirela
                                      </span>
                                      <span class="nlm-surname">
                                       Andronescu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Kieran R.
                                      </span>
                                      <span class="nlm-surname">
                                       Campbell
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="10">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       McPherson
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="11">
                                      <span class="nlm-given-names">
                                       Samuel
                                      </span>
                                      <span class="nlm-surname">
                                       Aparicio
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="12">
                                      <span class="nlm-given-names">
                                       Andrew
                                      </span>
                                      <span class="nlm-surname">
                                       Roth
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="13">
                                      <span class="nlm-given-names">
                                       Sohrab
                                      </span>
                                      <span class="nlm-surname">
                                       Shah
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="14">
                                      <span class="nlm-given-names">
                                       Alexandre
                                      </span>
                                      <span class="nlm-surname">
                                       Bouchard-Côté
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.05.06.058180;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.05.06.058180
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Salehi%2C+Sohrab&rft.au=Dorri%2C+Fatemeh&rft.au=Chern%2C+Kevin&rft.au=Kabeer%2C+Farhia&rft.au=Rusk%2C+Nicole&rft.au=Funnell%2C+Tyler&rft.au=Williams%2C+Marc+J&rft.au=Lai%2C+Daniel&rft.au=Andronescu%2C+Mirela&rft.au=Campbell%2C+Kieran+R.&rft.au=McPherson%2C+Andrew&rft.au=Aparicio%2C+Samuel&rft.au=Roth%2C+Andrew&rft.au=Shah%2C+Sohrab&rft.au=Bouchard-C%C3%B4t%C3%A9%2C+Alexandre">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2844645/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2844645/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2844645/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2844645/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2844645/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2844645/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2844645/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2844645/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2844645/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2844645/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2844645/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2844645/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&count=horizontal&via=&text=Cancer%20phylogenetic%20tree%20inference%20at%20scale%20from%201000s%20of%20single%20cell%20genomes&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.05.06.058180v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.05.06.058180v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
