<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.11.09.373795v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.11.09.373795v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.11.09.373795v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-2.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/05/18/2020.11.09.373795.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.11.09.373795v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="TiPS: rapidly simulating trajectories and phylogenies from compartmental models"/>
  <meta name="DC.Identifier" content="10.1101/2020.11.09.373795"/>
  <meta name="DC.Date" content="2022-05-18"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="We introduce TiPS, an R package to generate trajectories and phylogenetic trees associated with a compartmental model. Trajectories are simulated using Gillespie’s exact or approximate stochastic simulation algorithm, or a newly-proposed mixed version of the two. Phylogenetic trees are simulated from a trajectory under a backwards-in-time approach (i.e. coalescent). TiPS is based on the Rcpp package, allowing to combine the flexibility of R for model definition and the speed of C++ for simulations execution. The model is defined in R with a set of reactions, which allow capturing heterogeneity in life cycles or any sort of population structure. TiPS converts the model into C++ code and compiles it into a simulator that is interfaced in R via a function. Furthermore, the package allows defining time periods in which the model’s parameters can take different values. This package is particularly well suited for population genetics and phylodynamics studies that need to generate a large number of phylogenies used for population dynamics studies. This package is available on the CRAN at <https://cran.r-project.org/package=TiPS>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Gonché Danesh"/>
  <meta name="DC.Contributor" content="Emma Saulnier"/>
  <meta name="DC.Contributor" content="Olivier Gascuel"/>
  <meta name="DC.Contributor" content="Marc Choisy"/>
  <meta name="DC.Contributor" content="Samuel Alizon"/>
  <meta name="article:published_time" content="2022-05-18"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="TiPS: rapidly simulating trajectories and phylogenies from compartmental models"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>We introduce TiPS, an R package to generate trajectories and phylogenetic trees associated with a compartmental model. Trajectories are simulated using Gillespie’s exact or approximate stochastic simulation algorithm, or a newly-proposed mixed version of the two. Phylogenetic trees are simulated from a trajectory under a backwards-in-time approach (i.e. coalescent). TiPS is based on the Rcpp package, allowing to combine the flexibility of R for model definition and the speed of C++ for simulations execution. The model is defined in R with a set of reactions, which allow capturing heterogeneity in life cycles or any sort of population structure. TiPS converts the model into C++ code and compiles it into a simulator that is interfaced in R <i>via</i> a function. Furthermore, the package allows defining time periods in which the model’s parameters can take different values. This package is particularly well suited for population genetics and phylodynamics studies that need to generate a large number of phylogenies used for population dynamics studies. This package is available on the CRAN at https://cran.r-project.org/package=TiPS.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.11.09.373795v2"/>
  <meta name="citation_id" content="2020.11.09.373795v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.11.09.373795"/>
  <meta name="citation_num_pages" content="31"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.11.09.373795"/>
  <meta name="citation_author" content="Gonché Danesh"/>
  <meta name="citation_author_institution" content="MIVEGEC, CNRS, IRD, Université de Montpellier"/>
  <meta name="citation_author_email" content="gonche.danesh@gmail.com"/>
  <meta name="citation_author" content="Emma Saulnier"/>
  <meta name="citation_author_institution" content="MIVEGEC, CNRS, IRD, Université de Montpellier"/>
  <meta name="citation_author" content="Olivier Gascuel"/>
  <meta name="citation_author_institution" content="ISYEB, UMR 7205, MNHN, CNRS"/>
  <meta name="citation_author" content="Marc Choisy"/>
  <meta name="citation_author_institution" content="Centre for Tropical Medicine and Global Health, Nuffield Department of Medicine, University of Oxford"/>
  <meta name="citation_author_institution" content="Oxford University Clinical Research Unit"/>
  <meta name="citation_author" content="Samuel Alizon"/>
  <meta name="citation_author_institution" content="MIVEGEC, CNRS, IRD, Université de Montpellier"/>
  <meta name="citation_author_institution" content="Center for Interdisciplinary Research in Biology (CIRB), College de France, CNRS, INSERM, Université PSL"/>
  <meta name="citation_reference" content="Otto SP, Day T. A biologist’s guide to mathematical modeling in ecology and evolution. Princeton University Press; 2007."/>
  <meta name="citation_reference" content="Keeling MJ, Rohani P. Modeling infectious diseases in humans and animals. Princeton University Press; 2008."/>
  <meta name="citation_reference" content="Lenormand T, Roze D, Rousset F. Stochasticity in evolution. Trends in Ecology and Evolution. 2009;24(3):157– 165."/>
  <meta name="citation_reference" content="Grenfell BT, Pybus OG, Gog JR, Wood JLN, Janet M, Mumford JA, et al. Unifying the Epidemiological and Evolutionary Dynamics of Pathogens. Science. 2004;303(5656):327–332."/>
  <meta name="citation_reference" content="Volz EM, Koelle K, Bedford T. Viral phylodynamics. PLoS Comput Biol. 2013;9(3):e1002947."/>
  <meta name="citation_reference" content="Frost SD, Pybus OG, Gog JR, Viboud C, Bonhoeffer S, Bedford T. Eight challenges in phylodynamic inference. Epidemics. 2015;10:88–92."/>
  <meta name="citation_reference" content="Ratmann O, Donker G, Meijer A, Fraser C, Koelle K. Phylodynamic inference and model assessment with approximate bayesian computation: influenza as a case study. PLoS Comput Biol. 2012;8(12):e1002835."/>
  <meta name="citation_reference" content="Gascuel F, Ferrière R, Aguilée R, Lambert A. How Ecology and Landscape Dynamics Shape Phylogenetic Trees. Systematic Biology. 2015;64(4):590–607."/>
  <meta name="citation_reference" content="Saulnier E, Gascuel O, Alizon S. Inferring epidemiological parameters from phylogenies using regression-ABC: A comparative study. PLOS Computational Biology. 2017 Mar;13(3):e1005416."/>
  <meta name="citation_reference" content="Gillespie DT. A general method for numerically simulating the stochastic time evolution of coupled chemical reactions. Journal of Computational Physics. 1976;22(4):403–434."/>
  <meta name="citation_reference" content="Kurtz TG. Solutions of ordinary differential equations as limits of pure jump markov processes. Journal of Applied Probability. 1970;7(1):49–58."/>
  <meta name="citation_reference" content="Pineda-Krch M. GillespieSSA: implementing the stochastic simulation algorithm in R. Journal of Statistical Software. 2008;25(12):1–18."/>
  <meta name="citation_reference" content="Johnson P. adaptivetau: efficient stochastic simulations in R. R Package Version. 2014;."/>
  <meta name="citation_reference" content="Bjørnstad ON. Epidemics. Models and Data using R. Springer; 2018."/>
  <meta name="citation_reference" content="Pennell M, Eastman J, Slater G, Brown J, Uyeda J, Fitzjohn R, et al. geiger v2.0: an expanded suite of methods for fitting macroevolutionary models to phylogenetic trees. Bioinformatics. 2014;30:2216–2218."/>
  <meta name="citation_reference" content="Revell LJ. phytools: An R package for phylogenetic comparative biology (and other things). Methods in Ecology and Evolution. 2012;3:217–223."/>
  <meta name="citation_reference" content="Paradis E, Schliep K. ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R. Bioinformatics. 2019;35:526–528."/>
  <meta name="citation_reference" content="Volz EM. Complex population dynamics and the coalescent under neutrality. Genetics. 2012;190(1):187–201."/>
  <meta name="citation_reference" content="Vaughan TG, Drummond AJ. A stochastic simulator of birth-death master equations with application to phylodynamics. Molecular Biology and Evolution. 2013;30(6):1480–1493."/>
  <meta name="citation_reference" content="Bouckaert R, Heled J, Kühnert D, Vaughan T, Wu CH, Xie D, et al. BEAST 2: A Software Platform for Bayesian Evolutionary Analysis. PLoS Computational Biology. 2014;10(4):1–6."/>
  <meta name="citation_reference" content="Eddelbuettel D, Francois R. Rcpp: Seamless R and C++ Integration. J Stat Software. 2011;40(1):1–18."/>
  <meta name="citation_reference" content="Gillespie DT. Approximate accelerated stochastic simulation of chemically reacting systems. Journal of Chemical Physics. 2001;115(4):1716–1733."/>
  <meta name="citation_reference" content="Cao Y, Gillespie DT, Petzold LR. The slow-scale stochastic simulation algorithm. The Journal of Chemical Physics. 2005;122(1):014116. Available from: https://doi.org/10.1063/1.1824902."/>
  <meta name="citation_reference" content="Cao Y, Gillespie DT, Petzold LR. Adaptive explicit-implicit tau-leaping method with automatic tau selection. Journal of Chemical Physics. 2007;126(22):1–10."/>
  <meta name="citation_reference" content="Kingman JFC. The coalescent. Stochastic processes and their applications. 1982;13(3):235–248."/>
  <meta name="citation_reference" content="Danesh G, Virlogeux V, Ramière C, Charre C, Cotte L, Alizon S. Quantifying transmission dynamics of acute hepatitis C virus infections in a heterogeneous population using sequence data. PLoS Pathogens. 2021;17(9):e1009916."/>
  <meta name="citation_reference" content="Hartfield M, Alizon S. Epidemiological feedbacks affect evolutionary emergence of pathogens. Am Nat. 2014;183(4):E105–E117."/>
  <meta name="citation_reference" content="Alizon S. Superspreading genomes. Science. 2021;371(6529):574–575."/>
  <meta name="citation_reference" content="Cumming DH, Fenton MB, Rautenbach IL, Taylor RD, Cumming GS, Cumming MS, et al. Elephants, woodlands and biodiversity in southern Africa. South African Journal of Science. 1997;93(5):231–236."/>
  <meta name="citation_reference" content="Whyte I, van Aarde R, Pimm SL. Managing the elephants of Kruger National Park. Animal Conservation. 1998;1(2):77–83."/>
  <meta name="twitter:title" content="TiPS: rapidly simulating trajectories and phylogenies from compartmental models"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="We introduce TiPS, an R package to generate trajectories and phylogenetic trees associated with a compartmental model. Trajectories are simulated using Gillespie’s exact or approximate stochastic simulation algorithm, or a newly-proposed mixed version of the two. Phylogenetic trees are simulated from a trajectory under a backwards-in-time approach (i.e. coalescent). TiPS is based on the Rcpp package, allowing to combine the flexibility of R for model definition and the speed of C++ for simulations execution. The model is defined in R with a set of reactions, which allow capturing heterogeneity in life cycles or any sort of population structure. TiPS converts the model into C++ code and compiles it into a simulator that is interfaced in R via a function. Furthermore, the package allows defining time periods in which the model’s parameters can take different values. This package is particularly well suited for population genetics and phylodynamics studies that need to generate a large number of phylogenies used for population dynamics studies. This package is available on the CRAN at <https://cran.r-project.org/package=TiPS>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="TiPS: rapidly simulating trajectories and phylogenies from compartmental models"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="We introduce TiPS, an R package to generate trajectories and phylogenetic trees associated with a compartmental model. Trajectories are simulated using Gillespie’s exact or approximate stochastic simulation algorithm, or a newly-proposed mixed version of the two. Phylogenetic trees are simulated from a trajectory under a backwards-in-time approach (i.e. coalescent). TiPS is based on the Rcpp package, allowing to combine the flexibility of R for model definition and the speed of C++ for simulations execution. The model is defined in R with a set of reactions, which allow capturing heterogeneity in life cycles or any sort of population structure. TiPS converts the model into C++ code and compiles it into a simulator that is interfaced in R via a function. Furthermore, the package allows defining time periods in which the model’s parameters can take different values. This package is particularly well suited for population genetics and phylodynamics studies that need to generate a large number of phylogenies used for population dynamics studies. This package is available on the CRAN at <https://cran.r-project.org/package=TiPS>.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-05-18"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2550658"/>
  <title>
   TiPS: rapidly simulating trajectories and phylogenies from compartmental models | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"9nchUMDUV8Oak2L1Ne32HCDmeikbrn5yyiLJKCqHpAE"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2550658","apath":"\/biorxiv\/early\/2022\/05\/18\/2020.11.09.373795.atom","pisa":"biorxiv;2020.11.09.373795v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.11.09.373795v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.11.09.373795v2","title":"TiPS: rapidly simulating trajectories and phylogenies from compartmental models","identifier":"node\/2550658"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kRjkqmh42LWuqCHeV\\\\\\\/EA5aGTJvYqmynNnHkxrTHc1y\\\\\\\/c6K+hXD6CQk9CoMJrAg3FH\\\\\\\/dppZVRlyLpO\\\\\\\/0SCecyW8r2FFVWSVCQ9\\\\\\\/KCk9CnBxir7JumRenmeojOlBWggxA4pM+hwYbwdS\\\\\\\/KGZ8fCook2kkiSPh1UhhhDQ5FA45umcnP+A7C\\\\\\\/67o0pJf63AvXCQQHIltCJoJu6xfmPdiz5a70LuJrsDTkXZesz8SF\\\\\\\/GMP6gJKUpaQUTnJrsDHRN0cdBYq+JZ9mRX0D6SUKpA6rHZ1daqQCcxXGnYIsMSCIxjcPCPwlwURAnv+Nz\\\\\\\/I9CiS81KZl45r4dc+XncG6cjAAeOJOZ2tIRkQw4m8\\\\\\\/BJe7s7Ra4Qh006SEMxu0jJZYLxYDOcuCWPyosho8cUqqkBQV3aGnqcf0JX2gQvKmjslLdh3T1AjU+Ec1QnooB23O1dgQ6Vjx3hhgYODo4VXKN5tdwZ5Qc43S1p\\\\\\\/yLgxZcl9rBT8phpWbCkpYo0tQUiSBpM9oUC5zIn6rIaXaAaIoLkGaWjJi2zwvdTfBxALBdfyilUJtXJe7vipnmlYq2u1RATGrM74kYeoYDdOxKTip7gnEanKVD\\\\\\\/vbG\\\\\\\/AdTJwIRwsJDA5fiM6tAW0srlesg2cnDKxY0laCit\\\\\\\/16d6u7G38LGOv2dwM52zH0gtNbrSGMM1dBSthnb+I5WK9Mu8g1WrXPkWyXyakT5HQDLReOio0NLBLIjfR29VRAiVV41l7Vv\\\\\\\/kzhFDZIJA9dk3qzKPY5LHm\\\\\\\/+Ty7yWLHU0\\\\\\\/3M3RuQN6WziTUIGn6gdl1YCArkGz7pCwzo5z0LkS\\\\\\\/G2u7sfnFGY9azOZfUPVUo2zle+jsaZ5RU1FETiDYtWpknWiBbdC\\\\\\\/E+tpHanC0N5eGxmrgmDC+HokQXxNDA0jHJCMWMEnVn2BTzAB1zcqOcuVc9VL+7rV4m0+QhIU0Y+E3jjvtFSj71e0pI9Nf3oktC8EM8dw531UYOpTeZwf1pZyGfbIl\\\\\\\/r5gkhKJVkX+EZ8YwTeYP42HBunFWqgJngxPRJsc83VbZa7hSvOyD5TBYxzfnuKU+30iIXut8Fxeivf3Cf3QmKxRrd1STnuSYTdIIKaNbY5HSdVeGYv47ctpn6xYLYK5hKARlwssMAH1OF7Oz8s3tP46w7x0s9AO1EloguexCfNmJ1OK5t+jTQgL0M+3z\\\\\\\/EXV\\\\\\\/wxF6lCsCDixFCqfM3ihPheQWo+JSC2zGvs3Qbfopez9gKsZo1wyTfCnWb7vmlX+orN7FzRZHkSusMSSdLiuSFV6U4KUBxhqkaaEVDQtqD9D6HF83bRsyyJDvD10Zz8W4hE16BKUukxck5lpO\\\\\\\/b5JDxg9tEClLboboDudvYMbtzY1LntYXBNNvMAL8tTJEKOMZcaRn6b8=\\\u0022,\\\u0022iv\\\u0022:\\\u00225Uuo0kZZbiD\\\\\\\/qubwg114IA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00220c406e249590c317f3a38161d06e6dc9ea1ae6c107f749b18ab8246489665441\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.11.09.373795v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2550658 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.11.09.373795v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_834990180">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_834990180" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1864928836" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-DzSz04EMVgUPgcnU_R2LacRP6i8K1ytnozv2orOknSI"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2550658" data-node-nid="2550658" id="node-2550658--220263218" data-pisa="biorxiv;2020.11.09.373795v2" data-pisa-master="biorxiv;2020.11.09.373795" data-apath="/biorxiv/early/2022/05/18/2020.11.09.373795.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    TiPS: rapidly simulating trajectories and phylogenies from compartmental models
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Gonché
                      </span>
                      <span class="nlm-surname">
                       Danesh
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Emma
                      </span>
                      <span class="nlm-surname">
                       Saulnier
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Olivier
                      </span>
                      <span class="nlm-surname">
                       Gascuel
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="3">
                      <span class="nlm-given-names">
                       Marc
                      </span>
                      <span class="nlm-surname">
                       Choisy
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Samuel
                      </span>
                      <span class="nlm-surname">
                       Alizon
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.11.09.373795
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2550658--220263218" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Gonché Danesh
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        MIVEGEC, CNRS, IRD, Université de Montpellier
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Gonch%C3%A9%2BDanesh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Danesh%20G&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AGonch%25C3%25A9%2BDanesh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:gonche.danesh@gmail.com" class="" data-icon-position="" data-hide-link-title="0">
                        gonche.danesh@gmail.com
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Emma Saulnier
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        MIVEGEC, CNRS, IRD, Université de Montpellier
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Emma%2BSaulnier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Saulnier%20E&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AEmma%2BSaulnier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Olivier Gascuel
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        ISYEB, UMR 7205, MNHN, CNRS
                       </span>
                       , SU, UA, Paris,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Olivier%2BGascuel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Gascuel%20O&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AOlivier%2BGascuel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Marc Choisy
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Centre for Tropical Medicine and Global Health, Nuffield Department of Medicine, University of Oxford
                       </span>
                       ,
                       <span class="nlm-country">
                        UK
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Oxford University Clinical Research Unit
                       </span>
                       , Ho Chi Minh City,
                       <span class="nlm-country">
                        Vietnam
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Marc%2BChoisy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Choisy%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AMarc%2BChoisy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Samuel Alizon
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        MIVEGEC, CNRS, IRD, Université de Montpellier
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Center for Interdisciplinary Research in Biology (CIRB), College de France, CNRS, INSERM, Université PSL
                       </span>
                       , Paris,
                       <span class="nlm-country">
                        France
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Samuel%2BAlizon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Alizon%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ASamuel%2BAlizon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.11.09.373795v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2550658" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2550658/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.09.373795v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2550658" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2550658/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.09.373795v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2550658" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2550658/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.11.09.373795v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2550658" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2550658/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.11.09.373795v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2550658" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2550658/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             We introduce
                             <kbd>
                              TiPS
                             </kbd>
                             , an R package to generate trajectories and phylogenetic trees associated with a compartmental model. Trajectories are simulated using Gillespie’s exact or approximate stochastic simulation algorithm, or a newly-proposed mixed version of the two. Phylogenetic trees are simulated from a trajectory under a backwards-in-time approach (i.e. coalescent).
                             <kbd>
                              TiPS
                             </kbd>
                             is based on the
                             <kbd>
                              Rcpp
                             </kbd>
                             package, allowing to combine the flexibility of R for model definition and the speed of C++ for simulations execution. The model is defined in R with a set of reactions, which allow capturing heterogeneity in life cycles or any sort of population structure.
                             <kbd>
                              TiPS
                             </kbd>
                             converts the model into C++ code and compiles it into a simulator that is interfaced in R via a function. Furthermore, the package allows defining time periods in which the model’s parameters can take different values. This package is particularly well suited for population genetics and phylodynamics studies that need to generate a large number of phylogenies used for population dynamics studies. This package is available on the CRAN at
                             <a href="https://cran.r-project.org/package=TiPS">
                              https://cran.r-project.org/package=TiPS
                             </a>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-4">
                             Stochastic population dynamics simulations are routinely used in ecology and epidemiology to generate trajectories (i.e. time series of population sizes) and genealogies that capture the relatedness between individuals [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             –
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. The increasing amount of genetic data is fuelling interest in linking population dynamics and genealogies because the former can leave footprints in individuals’ genomes [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             –
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ]. Such phylodynamics studies involve computer-intensive methods that can require the simulation of many trajectories and genealogies [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             –
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ].
                            </p>
                            <p id="p-5">
                             A common method to simulate population dynamics trajectories is Gillespie’s exact stochastic simulation algorithm (SSA) [
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ], which is rooted in probability theory [
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. In the R software environment, it is implemented in packages such as
                             <kbd>
                              Gillespi
                             </kbd>
                             eSSA [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ],
                             <kbd>
                              adaptivetau
                             </kbd>
                             [
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ], or
                             <kbd>
                              epimdr
                             </kbd>
                             [
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ]. In this same environment,
                             <kbd>
                              geiger
                             </kbd>
                             [
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ],
                             <kbd>
                              phytools
                             </kbd>
                             [
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ],
                             <kbd>
                              ape
                             </kbd>
                             [
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ], and
                             <kbd>
                              TreeSim
                             </kbd>
                             can simulate phylogenies using a birth-death model. However, few software packages simulate both trajectories and genealogies. One exception is
                             <kbd>
                              rcolgem
                             </kbd>
                             , updated to
                             <kbd>
                              phydynR
                             </kbd>
                             , in R [
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ], which involves coalescent processes. Another exception is the software package
                             <kbd>
                              MASTER
                             </kbd>
                             [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ] in the BEAST2 platform [
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ].
                            </p>
                            <p id="p-6">
                             We introduce
                             <kbd>
                              TiPS
                             </kbd>
                             , a flexible and easy-to-use R package to rapidly simulate population trajectories and phylogenies using a backwards-in-time, i.e. coalescent, process with either pre-defined sampling dates or a stochastic sampling scheme. We also introduce a new approximate version of the Gillespie algorithm to increase calculation speed. A brief benchmarking analysis shows that
                             <kbd>
                              TiPS
                             </kbd>
                             is faster than
                             <kbd>
                              adaptivetau
                             </kbd>
                             to simulate trajectories, especially for large populations (
                             <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                              Figure 3a
                             </a>
                             ). It is also at least one order of magnitude faster than
                             <kbd>
                              phydynR
                             </kbd>
                             to simulate phylogenies (
                             <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                              Figure 3b
                             </a>
                             ).
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Package overview
                             </h3>
                             <p id="p-7">
                              <kbd>
                               TiPS
                              </kbd>
                              generates two types of stochastic simulation outputs: population dynamics trajectories and phylogenies. These are obtained using a continuous-time individual-based model defined in R as a system of reactions. The model is first transcribed in C++ and then compiled, before being linked back to a simulating function in R thanks to the
                              <kbd>
                               Rcpp
                              </kbd>
                              package [
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ]. The general structure of the pipeline is illustrated in
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              .
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F1.large.jpg?width=800&height=600&carousel=1" title="General structure of the TiPS pipeline. The equations and outputs correspond to the SIR epidemiological model [2]. The functions of the R package are in blue. The simulator of trajectories, which is built as a function, is in orange. The variable traj, in red, is the output trajectory of class simutraj. It can be plotted using our plot method. TiPS used the simulated trajectory and 100 sampling dates that we generated (variable sampling_dates in green) to simulate a sampled phylogeny. The output simulated phylogeny is a phylo class object from the ape R package [17], which can be used for plotting." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">General structure of the <kbd>TiPS</kbd> pipeline.</span> The equations and outputs correspond to the <em>SIR</em> epidemiological model [2]. The functions of the R package are in blue. The simulator of trajectories, which is built as a function, is in orange. The variable <em>traj</em>, in red, is the output trajectory of class <em>simutraj</em>. It can be plotted using our <em>plot</em> method. <kbd>TiPS</kbd> used the simulated trajectory and 100 sampling dates that we generated (variable <em>sampling_dates</em> in green) to simulate a sampled phylogeny. The output simulated phylogeny is a <em>phylo</em> class object from the <kbd>ape</kbd> R package [17], which can be used for plotting.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F1.medium.gif" width="440" height="391"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F1.medium.gif" width="440" height="391"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <span class="caption-title">
                                General structure of the
                                <kbd>
                                 TiPS
                                </kbd>
                                pipeline.
                               </span>
                               <p id="p-8" class="first-child">
                                The equations and outputs correspond to the SIR epidemiological model [
                                <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                                 2
                                </a>
                                ]. The functions of the R package are in blue. The simulator of trajectories, which is built as a function, is in orange. The variable traj, in red, is the output trajectory of class simutraj. It can be plotted using our plot method.
                                <kbd>
                                 TiPS
                                </kbd>
                                used the simulated trajectory and 100 sampling dates that we generated (variable sampling_dates in green) to simulate a sampled phylogeny. The output simulated phylogeny is a phylo class object from the
                                <kbd>
                                 ape
                                </kbd>
                                R package [
                                <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                                 17
                                </a>
                                ], which can be used for plotting.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Model description
                             </h3>
                             <p id="p-9">
                              <kbd>
                               TiPS
                              </kbd>
                              simulates trajectories from compartmental models. These models divide the population (animals, cells, etc) into distinct categories (geographic, clinical state, etc) or so-called compartments in which the sub-population behaves uniformly. The population, in these models, may progress between the different compartments.
                             </p>
                             <p id="p-10">
                              Here, we illustrate the use of
                              <kbd>
                               TiPS
                              </kbd>
                              with the SIR epidemiological model, where individuals can have three clinical states: susceptible (S), infected (I), and removed (R) [
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ]. The model can be captured with a system of two individual-based reactions:
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where β and γ are the transmission and recovery rates. The rate of occurrence of each reaction is indicated above the transition arrow and the corresponding population-based system of ordinary differential equations (ODE) is shown in the top-right panel of
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              .
                             </p>
                             <p id="p-11">
                              A simulating function is generated from the individual-based reactions of the model of interest. These are entered by the user as a string vector (top left box of
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Simulating trajectories
                             </h3>
                             <p id="p-12">
                              The simulation function takes as arguments a named numeric vector that contains the initial number of individuals in each compartment, a named list with the parameter values, a vector of the time limits of the simulation, and the type of algorithm to use for the simulation. Users can also enter a vector of breakpoints, which allows parameter values to vary over time. These breakpoints are indicated in the time limits vector, and the corresponding parameter values are ordered chronologically in the named list of parameter values.
                             </p>
                             <p id="p-13">
                              Three simulation algorithms are implemented in the
                              <kbd>
                               Rcpp
                              </kbd>
                              simulating function:
                             </p>
                             <ol class="list-ord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-14">
                                Gillespie’s Direct Algorithm (GDA, the default option) [
                                <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                                 10
                                </a>
                                ] is an exact algorithm that simulates the time until the next event δ
                                <sub>
                                 t
                                </sub>
                                by assuming that waiting times are exponentially distributed. A limitation is that its computational complexity scales linearly with the number of simulated events.
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-15">
                                Gillespie’s Tau-Leap Algorithm (GTA) [
                                <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                                 22
                                </a>
                                ] is an approximate algorithm that introduces a fixed time step τ during which the number of events of each type is assumed to be Poisson distributed. This algorithm is limited in terms of computation time if the time step is small compared to the rate at which events occur.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-16">
                                The Mixed Simulation Algorithm (MSA) is a new algorithm we introduce that switches from GDA to GTA depending on the respective values of δ
                                <sub>
                                 t
                                </sub>
                                and τ. The algorithm switches from GDA to GTA if 10 successive estimations of δ
                                <sub>
                                 t
                                </sub>
                                are shorter than τ/10. The algorithm switches back to GDA if the total number of realised events is lower than the number of possible events. The MSA shares similarities with the slow-scale stochastic simulation algorithm [
                                <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                                 23
                                </a>
                                ] or the adaptive explicit-implicit tau-leaping method for an optimised tau-leap selection [
                                <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                                 24
                                </a>
                                ].
                               </p>
                              </li>
                             </ol>
                             <p id="p-17">
                              The output of a trajectory simulation is a named list containing the simulated trajectory and some details about the model and the simulation, such as the reactions of the model, the parameter values, the time range, the algorithm used to perform simulations, the time-step in case the algorithm is the GTA or the MSA, and, for reproducibility purposes, the random seed used to initialise a pseudorandom number generator.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Simulating phylogenies
                             </h3>
                             <p id="p-18">
                              A phylogeny is the representation of the evolutionary history and relationships between genes, organisms, or groups of organisms. The root of the tree represents the ancestor of all lineages, and the leaves represent the most recent descendants of that ancestor.
                              <kbd>
                               TiPS
                              </kbd>
                              simulates binary phylogenies that are rooted in time. We distinguish the biological process from the observation process. The former corresponds to a sequence of individual birth, migration, and death events. The latter corresponds to sampling or other events that may interrupt the biological process (e.g. sterilisation or treatment).
                             </p>
                             <p id="p-19">
                              <kbd>
                               TiPS
                              </kbd>
                              uses a coalescent approach [
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ] to simulate phylogenetic trees based on trajectories, which correspond to a list of dated events (or ‘reactions’) and sampling dates (e.g. based on observed data). Each node in the simulated phylogenetic tree is associated with a state, i.e. a compartment name, and its height, i.e. its distance to the root.
                             </p>
                             <p id="p-20">
                              Individuals who can contribute directly to the phylogeny are referred to as ‘demes’ and the compartment to which they belong as a ‘deme compartment’. Individuals in non-deme compartments are not observed in the phylogenetic process. For infection phylodynamics, when simulating a phylogenetic tree, we trace back the history of sampled infections. In the SIR model, for example, these are only found in the I compartment (S and R are non-deme compartments). In an ecological context, under a predatorprey model, for example, tracking the history of predator and prey individuals means that both are deme compartments.
                             </p>
                             <p id="p-21">
                              <kbd>
                               TiPS
                              </kbd>
                              can simulate the phylogeny of the full trajectory or that of a sampled phylogeny if sampling dates are provided. These dates can either be provided by the user as a vector (bottom left panel of
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              ) or generated during the trajectory simulation by adding a sampling reaction in the model. Since parameter values can vary with time,
                              <kbd>
                               TiPS
                              </kbd>
                              can reproduce temporal variations in sampling intensity. If sampled individuals can belong to more than one deme compartment, the user can choose between defining the state associated with each sampling event or defining a proportion of sampling events associated with each state. In the last case,
                              <kbd>
                               TiPS
                              </kbd>
                              randomly associates a state to each sampling date.
                             </p>
                             <p id="p-22">
                              After this pre-processing, the sampling dates are organised as a named list containing a vector of decimal dates (column name ‘Date’) and a vector containing the reactions indicating the state of individuals to sample (column name ‘Reaction’).
                              <kbd>
                               TiPS
                              </kbd>
                              then incorporates these pieces of information into the recorded trajectory (containing also dates and reactions) in chronological order.
                             </p>
                             <p id="p-23">
                              The simulation of a phylogeny (sampled or not) starts from the most recent sampling date (or the most recent death event) and progresses through the simulated trajectory backwards-in-time. When simulating a phylogenetic tree, each event in the trajectory is defined as one of the following types of tree reactions:
                             </p>
                             <ul class="list-none" id="list-2">
                              <li id="list-item-4">
                               <p id="p-24">
                                <span class="list-label">
                                 -
                                </span>
                                births: these reactions correspond to the creation of a new deme individual;
                               </p>
                              </li>
                              <li id="list-item-5">
                               <p id="p-25">
                                <span class="list-label">
                                 -
                                </span>
                                migrations: these correspond to the displacement of a deme individual from its deme compartment to another;
                               </p>
                              </li>
                              <li id="list-item-6">
                               <p id="p-26">
                                <span class="list-label">
                                 -
                                </span>
                                death: these correspond to the removal of a deme individual from its deme compartment or the displacement of a deme individual from its deme compartment to a non-deme compartment;
                               </p>
                              </li>
                              <li id="list-item-7">
                               <p id="p-27">
                                <span class="list-label">
                                 -
                                </span>
                                samplings: these correspond to the sampling of deme individuals.
                               </p>
                              </li>
                             </ul>
                             <p id="p-28">
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table 1
                              </a>
                              illustrates demes and reaction types for specific ecological and epidemiological models.
                             </p>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2553268/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2553268/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2553268" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table 1:
                               </span>
                               <span class="caption-title">
                                Description of demes and reactions for an ecological and an epidemiological model.
                               </span>
                               <p id="p-29" class="first-child">
                                ‘ODEs’ stands for ordinary differential equations.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-30">
                              Each of these four types of reactions can result in a modification in the simulated tree. A forward-in-time birth event can be represented by a coalescence between two lineages under a backwards-in-time process (or a branching under a forward-in-time process). A sampling event interrupting a biological process is represented as an external node (or leaf) in the phylogeny, and a death event, if observed, is also represented as a leaf. A migration event of an individual from one deme compartment to another will change and update the state of its corresponding lineage.
                             </p>
                             <p id="p-31">
                              In some cases, especially with the tau-leap method, more than one event may occur on the same date (e.g. three new births). To determine the number of events that lead to a change in the phylogeny (e.g. a branching), we draw a number from a hypergeometric distribution, which is appropriate since it describes the number of successful events (k) when drawing n times (without replacement) from a total population of size N that contains K elements corresponding to a ‘success’. For example, when a birth event in the trajectory is encountered in the phylogeny simulation, the algorithm draws, using the hypergeometric distribution, the number of sampled child lineages and the number of sampled parent lineages to coalesce into one sampled parent lineage. If at time t, the number of sampled individuals is small, all birth events at time t may not lead to an observed branching in the tree. Similarly, when a migration event is encountered, the algorithm draws the number of sampled lineages associated with a change in state.
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2
                              </a>
                              illustrates how the tree is simulated from the trajectory and further details can be found in the Appendix.
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F2.large.jpg?width=800&height=600&carousel=1" title="Tree simulation for a SIR model. the trajectory shows the series of epidemiological events and sampling events. the phylogeny represents the epidemiological history of the individuals carrying sampled viruses in solid lines. The rest of the history is shown in dashed lines. In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Tree simulation for a SIR model.</span> the trajectory shows the series of epidemiological events and sampling events. the phylogeny represents the epidemiological history of the individuals carrying sampled viruses in solid lines. The rest of the history is shown in dashed lines. In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F2.medium.gif" width="440" height="354"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F2.medium.gif" width="440" height="354"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2:
                               </span>
                               <span class="caption-title">
                                Tree simulation for a SIR model.
                               </span>
                               <p id="p-32" class="first-child">
                                the trajectory shows the series of epidemiological events and sampling events. the phylogeny represents the epidemiological history of the individuals carrying sampled viruses in solid lines. The rest of the history is shown in dashed lines. In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-33">
                              The output simulated phylogeny is an R object of class phylo as defined in the
                              <kbd>
                               ape
                              </kbd>
                              package [
                              <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ].
                             </p>
                             <p id="p-34">
                              An illustration of how to simulate a phylogeny can be found in the ‘Simulating phylogenies’ box in
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Figure 1
                              </a>
                              and in the Appendix for a logistic growth ecological model.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Benchmarking
                             </h3>
                             <p id="p-35">
                              To evaluate
                              <kbd>
                               TiPS
                              </kbd>
                              ’s performances, we designed a benchmarking analysis on both modules of the software package, i.e. the trajectory and the phylogeny simulators, comparing with existing
                              <kbd>
                               R
                              </kbd>
                              packages (
                              <kbd>
                               adaptivetau
                              </kbd>
                              and
                              <kbd>
                               phydynR
                              </kbd>
                              ).
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table S1
                              </a>
                              summarises the main features of the approaches used.
                             </p>
                             <p id="p-36">
                              We first evaluated the computational speed and the accuracy of the trajectories (i.e. populating dynamics). For five initial population sizes and three different
                              <kbd>
                               R
                              </kbd>
                              packages, we simulated 10, 000 trajectories of the epidemiological Susceptible-Infected-Recovered (SIR) model and measured the execution time.
                             </p>
                             <p id="p-37">
                              We then compared the time to simulate phylogenies under an SIR model. We varied sampling proportions in order to obtain target phylogenies of various sizes (10, 100, 500, 1000, and 1500 leaves). We simulated 1,000 phylogenies under each sampling scheme using
                              <kbd>
                               phydynR
                              </kbd>
                              and our package.
                             </p>
                             <p id="p-38">
                              Furthermore, assuming a more detailed epidemiological model with two host types, as described in [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ], we simulated an epidemiological trajectory using the tau-leap algorithm, and a complete phylogeny, such that each end of infection event corresponded to a leaf in the tree. The simulation generated a full transmission chain corresponding to a phylogeny with 154, 507 leaves. From this complete phylogeny, we generated 10 sub-trees by randomly sampling and keeping 1, 000 leaves for each sub-tree. We then used
                              <kbd>
                               TiPS
                              </kbd>
                              and
                              <kbd>
                               phydynR
                              </kbd>
                              to simulate 1, 000 phylogenies with each package under the same epidemiological model with the same parameter values. Note that the 1, 000 dates of each target sub-tree were imposed when simulating these phylogenies using a backwards-in-time approach. To compare the 2, 000 simulated phylogenies with the target one, we computed 60 summary statistics for each of them using the methods described in [
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ].
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-8">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-39">
                             In
                             <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                              Figure 3(a)
                             </a>
                             , we show the median execution time for one trajectory simulation and the 50% interquartile envelope.
                             <kbd>
                              TiPS
                             </kbd>
                             and
                             <kbd>
                              adaptivetau
                             </kbd>
                             rely on Gillespie’s Direct method (GDA), whereas
                             <kbd>
                              phydynR
                             </kbd>
                             uses the Euler-Maruyama integration (EMI). As expected, the population size, and, hence, the number of events per unit of time, increases the execution time for GDA-based packages but not for the EMI-based package. However,
                             <kbd>
                              TiPS
                             </kbd>
                             remains faster than the other two software packages for large populations (10
                             <sup>
                              7
                             </sup>
                             individuals). In
                             <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                              Figure 3(b)
                             </a>
                             , we perform the same simulations using approximations of the Gillespie algorithm with fixed time steps. The computational speed of this GTA implemented in
                             <kbd>
                              TiPS
                             </kbd>
                             and
                             <kbd>
                              adaptivetau
                             </kbd>
                             is comparable and much faster than the GDA. Our new MSA algorithm outperforms existing methods and improves computational speed compared to our GTA, especially for small population sizes.
                            </p>
                            <div id="F3" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F3.large.jpg?width=800&height=600&carousel=1" title="Benchmark analyses of trajectories simulations. a) Median computation speed and 50% confidence intervals (CI) for one simulation using GDA or EMI methods, b) median computation speed and 50% CI for one simulation using approximating GDA methods, and c) mean resulting trajectories for prevalence time series and the 90% CI. GDA stands for Gillespie’s Direct Method and EMI for Euler-Maruyama Integration. 10, 000 trajectories simulations were performed under an epidemiological SIR model, for five initial population sizes N varying from 103 to 107 and with parameter values ℛ 0 = 2, γ = 1/3 and ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Benchmark analyses of trajectories simulations.</span> a) Median computation speed and 50% confidence intervals (CI) for one simulation using GDA or EMI methods, b) median computation speed and 50% CI for one simulation using approximating GDA methods, and c) mean resulting trajectories for prevalence time series and the 90% CI. GDA stands for Gillespie’s Direct Method and EMI for Euler-Maruyama Integration. 10, 000 trajectories simulations were performed under an epidemiological SIR model, for five initial population sizes <em>N</em> varying from 10<sup>3</sup> to 10<sup>7</sup> and with parameter values <em>ℛ</em> <sub>0</sub> = 2, <em>γ</em> = 1/3 and .</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F3.medium.gif" width="440" height="141"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F3.medium.gif" width="440" height="141"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 3:
                              </span>
                              <span class="caption-title">
                               Benchmark analyses of trajectories simulations.
                              </span>
                              <p id="p-40" class="first-child">
                               a) Median computation speed and 50% confidence intervals (CI) for one simulation using GDA or EMI methods, b) median computation speed and 50% CI for one simulation using approximating GDA methods, and c) mean resulting trajectories for prevalence time series and the 90% CI. GDA stands for Gillespie’s Direct Method and EMI for Euler-Maruyama Integration. 10, 000 trajectories simulations were performed under an epidemiological SIR model, for five initial population sizes N varying from 10
                               <sup>
                                3
                               </sup>
                               to 10
                               <sup>
                                7
                               </sup>
                               and with parameter values ℛ
                               <sub>
                                0
                               </sub>
                               = 2, γ = 1/3 and
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/F3/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/F3/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-41">
                             In
                             <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                              Figure 3(c)
                             </a>
                             , we show the deterministic trajectory, and, for each algorithm used, the mean simulated trajectory and its 90% confidence envelope. Among the packages studied,
                             <kbd>
                              TiPS
                             </kbd>
                             (in blue) is the one that yields the trajectories that are the closest to the deterministic prediction (in black). Note that there is a temporal shift for all the stochastic simulations, with a more rapid increase in population size compared to the deterministic model. This comes from the fact that stochasticity tends to disfavour trajectories that spread more slowly than average because they are more likely to go extinct. This known population genetics effect has been described in epidemiology models [
                             <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                              27
                             </a>
                             ].
                            </p>
                            <p id="p-42">
                             We then analyse the median time executions of simulations of phylogenies for each tool and each sampling scheme (
                             <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                              Figure 4(a)
                             </a>
                             ).
                             <kbd>
                              TiPS
                             </kbd>
                             ’s median simulation time is several orders of magnitude faster than that of the other method, with a more pronounced advantage for large phylogenies.
                            </p>
                            <div id="F4" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F4.large.jpg?width=800&height=600&carousel=1" title="Benchmarking analysis of phylogenies simulations. a) Median computation speed for simulating one phylogeny of a given size and b) First two axes of a Principal Component Analysis (PCA) of phylogenies summary statistics. In a, the shaded area shows the 95% confidence intervals. In b, the cross shows the target phylogeny. The three colors represent three analyses using different target sub-trees. The phylogenies summary statistics used for the PCA are the same as in [9]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Benchmarking analysis of phylogenies simulations.</span> a) Median computation speed for simulating one phylogeny of a given size and b) First two axes of a Principal Component Analysis (PCA) of phylogenies summary statistics. In a, the shaded area shows the 95% confidence intervals. In b, the cross shows the target phylogeny. The three colors represent three analyses using different target sub-trees. The phylogenies summary statistics used for the PCA are the same as in [9].</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F4.medium.gif" width="440" height="147"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F4.medium.gif" width="440" height="147"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 4:
                              </span>
                              <span class="caption-title">
                               Benchmarking analysis of phylogenies simulations.
                              </span>
                              <p id="p-43" class="first-child">
                               a) Median computation speed for simulating one phylogeny of a given size and b) First two axes of a Principal Component Analysis (PCA) of phylogenies summary statistics. In a, the shaded area shows the 95% confidence intervals. In b, the cross shows the target phylogeny. The three colors represent three analyses using different target sub-trees. The phylogenies summary statistics used for the PCA are the same as in [
                               <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                               ].
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-44">
                             Regarding the accuracy of the simulations, we see in
                             <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                              Figure 4(b)
                             </a>
                             that the cross, which indicates the projection of the summary statistics from the target phylogeny, is contained in the envelope containing 90% of the phylogenies simulated using
                             <kbd>
                              TiPS
                             </kbd>
                             but not of those simulated using
                             <kbd>
                              phydynR
                             </kbd>
                             , and that for three different target phylogenies used represented by distinct colors. We observed the same results for seven other target phylogenies used. This means that the target phylogeny cannot be distinguished from phylogenies simulated using the same model with the same parameters using our package. The discrepancy observed for
                             <kbd>
                              phydynR
                             </kbd>
                             could originate from the trajectory, which strongly differs from the deterministic prediction (
                             <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                              Figure 3c
                             </a>
                             ). Furthermore,
                             <kbd>
                              TiPS
                             </kbd>
                             is favoured in this analysis because the same algorithm (Gillespie’s Tau-Leap) was used to simulate the target phylogeny and the 1, 000 sub-trees.
                             <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                              Supplementary Figure S7
                             </a>
                             shows the distributions of summary statistics computed from the phylogenies simulated using
                             <kbd>
                              TiPS
                             </kbd>
                             and
                             <kbd>
                              phydynR
                             </kbd>
                             for each analysis using a different target tree.
                            </p>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-45">
                             We developed a flexible R package to rapidly simulate trajectories and phylogenies from compartmental models. Its structure allows the user to include several sources of heterogeneity between individuals or between populations, e.g. different life stages or metapopulation structures. The simulation of the phylogeny relies on the trajectory and involves a coalescent approach.
                            </p>
                            <p id="p-46">
                             Our benchmarking analyses show that
                             <kbd>
                              TiPS
                             </kbd>
                             is comparable to or outperforms existing
                             <kbd>
                              R
                             </kbd>
                             packages in terms of speed when generating numerous trajectories or phylogenies. The same is true for the accuracy of the simulation outputs.
                            </p>
                            <p id="p-47">
                             The first asset of this software package is its flexibility since it can readily be used for any compartmental model. A second asset is its computation speed as it can simulate trajectories in a matter of milliseconds on a regular desktop computer. These properties have already been used for infection phylodynamics studies involving Approximate Bayesian Computing (ABC) [
                             <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">
                              26
                             </a>
                             ] or to illustrate the effect of superspreading events [
                             <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                              28
                             </a>
                             ].
                            </p>
                            <p id="p-48">
                             Beyond epidemiology, this software package can be used more broadly to simulate population dynamics and associated genealogies. Some future extensions of
                             <kbd>
                              TiPS
                             </kbd>
                             will consist in introducing non-Markovian dynamics, simulating multifurcating phylogenies and implement other optimised stochastic simulation algorithms [
                             <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                              23
                             </a>
                             ,
                             <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">
                              24
                             </a>
                             ].
                            </p>
                           </div>
                           <div class="section" id="sec-10">
                            <h2 class="">
                             Supplementary Information
                            </h2>
                            <div class="section" id="sec-11">
                             <h3>
                              S1 Phylogenies simulation algorithm
                             </h3>
                             <p id="p-49">
                              <kbd>
                               TiPS
                              </kbd>
                              simulates rooted and binary phylogenies, meaning that every internal node has exactly two daughter nodes. The root of the tree represents the ancestor of all lineages, and the tips of the branches (or leaves) represent the most recent descendants of that ancestor. A coalescence between two lineages using a backwards-in-time process is equivalent to a branching event in the phylogeny in the forward-in-time process. Under the hypothesis of neutral evolution that phylodynamics relies on, a branching event can represent, for example, a viral transmission event, and a leaf can represent the end of viral infection. Note that we will refer to a node’s height as its distance to the root.
                             </p>
                             <p id="p-50">
                              In this section, we distinguish two types of compartments: the deme compartments, where individuals contribute directly or indirectly to the phylogeny, and the non-deme compartments that cannot be placed in the genealogical process. For example, in a SIR epidemiological model, the deme and non-deme compartments would be respectively the I compartment and the S and R compartments. Each deme compartment is denoted X
                              <sub>
                               i
                              </sub>
                              , with i ranging from 1 to the number of deme compartments in the model. Sampled individuals belong to the sub-compartment
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and are all associated with a leaf in the tree. We also introduce
                              <span class="inline-formula" id="inline-formula-3">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-3.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-3.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , the sub-compartment of
                              <span class="inline-formula" id="inline-formula-4">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-4.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-4.gif"/>
                                </noscript>
                               </span>
                              </span>
                              corresponding to individuals in
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              that have not yet been but may be sampled in a backwards-in-time process. The discrete size of compartments
                              <span class="inline-formula" id="inline-formula-6">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-6.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-6.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-7">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-7.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-7.gif"/>
                                </noscript>
                               </span>
                              </span>
                              at time t are denoted as
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , respectively. A non-deme compartment is denoted Z and its discrete size |Z|.
                             </p>
                             <p id="p-51">
                              The tree simulation starts from the last (i.e. most recent) sampling date and progresses through the simulated trajectory backwards-in-time. The number of events that lead to a change in the phylogeny is drawn from a hypergeometric distribution. The hypergeometric distribution is appropriate as it describes the number of events k from a sample n drawn from a total population N without replacement. Each of the four types of reactions (sampling, birth, death, and migration) can result in a modification in the simulated tree : a new external node (or leaf) or the coalescence of two lineages.
                             </p>
                             <div class="section" id="sec-12">
                              <h4>
                               Sampling event
                              </h4>
                              <p id="p-52">
                               We define a sampling event as an event that interrupts the biological process of an individual in the population of interest, and a re-sampling event as an observation event. In an ecological context, the sterilisation of an animal would be a sampling event whereas marking the animal would be a re-sampling event. In an epidemiological context, that is when tracing back the history of pathogen spread, a sampling event usually corresponds to a host individual being detected. Usually, this is assumed to coincide with the end of the infectious period, because the individual will be isolated or use adequate protection to prevent further transmission. However, if this assumption does not apply, it is possible to treat the event as a re-sampling event, such that the host individual will continue to transmit the pathogen to other individuals.
                              </p>
                              <p id="p-53">
                               When we know the sampling dates, we know the number of sampling events occurring at time
                               <span class="inline-formula" id="inline-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and, therefore, the number of tips to create with height t. However, since we allow sampled lineages to continue to have an offspring, we need to determine if the nodes we sampled at time t are associated with re-sampling events or not (i.e. if they should be linked to a node that has already been sampled after time t in our coalescent approach). The number of re-sampling (n
                               <sub>
                                RS
                               </sub>
                               ) and sampling events (n
                               <sub>
                                S
                               </sub>
                               ) at time t is governed by the following relationships:
                               <span class="disp-formula" id="disp-formula-3">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-4">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-9.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-9.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where
                               <span class="inline-formula" id="inline-formula-11">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is the total number of phylogeny nodes generated at t. Using the hypergeometric distribution, we compute the number of re-sampling events n
                               <sub>
                                res
                               </sub>
                               if we sample
                               <span class="inline-formula" id="inline-formula-12">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               times without replacement in a sample of size
                               <span class="inline-formula" id="inline-formula-13">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               containing
                               <span class="inline-formula" id="inline-formula-14">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               individuals. The number of ‘classical’ samplings, n
                               <sub>
                                S
                               </sub>
                               , is the difference between n
                               <sub>
                                RS
                               </sub>
                               and the number of tips to be generated
                               <span class="inline-formula" id="inline-formula-15">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-54">
                               In the case of a re-sampling, we randomly pick a node from
                               <span class="inline-formula" id="inline-formula-16">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-16.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-16.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (the pool of individuals who have not yet been sampled), update its height to t, and link it to a node from
                               <span class="inline-formula" id="inline-formula-17">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-17.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-17.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               (the pool that has been sampled). Each re-sampling event decreases
                               <span class="inline-formula" id="inline-formula-18">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-18.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-18.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               by one. In the case of a ‘classical’ sampling event, a new node with height t is created in
                               <span class="inline-formula" id="inline-formula-19">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-19.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-19.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-20">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-20.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-20.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               increases by one.
                              </p>
                             </div>
                             <div class="section" id="sec-13">
                              <h4>
                               Birth event
                              </h4>
                              <p id="p-55">
                               A birth reaction can be written as follows: X
                               <sub>
                                i
                               </sub>
                               → X
                               <sub>
                                j
                               </sub>
                               + X
                               <sub>
                                i
                               </sub>
                               where i and j range from 1 to the number of deme compartments. The birth reaction can also be written as Z + X
                               <sub>
                                i
                               </sub>
                               → X
                               <sub>
                                j
                               </sub>
                               + X
                               <sub>
                                i
                               </sub>
                               if it includes a non-deme individual. In this section, we will refer to a donor lineage as the parent lineage and a recipient lineage as the child lineage. A birth reaction leads to one of two types of modification in the tree: a coalescence or an ‘invisible’ coalescence. A coalescence corresponds to the coalescence of two sampled lineages from two individuals, the recipient (or child) and the donor (or parent), into one sampled individual lineage (the donor). An ‘invisible’ coalescence is defined as the coalescence of the sampled lineage of the recipient
                               <span class="inline-formula" id="inline-formula-21">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-21.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-21.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and an unsampled lineage of the donor
                               <span class="inline-formula" id="inline-formula-22">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-22.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-22.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               into one unsampled lineage
                               <span class="inline-formula" id="inline-formula-23">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-23.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-23.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               .
                              </p>
                              <p id="p-56">
                               We first need to determine the number of recipient lineages n
                               <sub>
                                recipient
                               </sub>
                               . Again, assuming a hypergeometric distribution, we compute the number of recipient lineages n
                               <sub>
                                recipient
                               </sub>
                               if we sample n
                               <sub>
                                rep
                               </sub>
                               times without replacement in a sample of size |X
                               <sub>
                                j
                               </sub>
                               | containing
                               <span class="inline-formula" id="inline-formula-24">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-24.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-24.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               individuals (see
                               <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">
                                equation S2a
                               </a>
                               ). If there are no recipient lineages (n
                               <sub>
                                recipient
                               </sub>
                               = 0), there is no coalescence or invisible coalescence and, therefore, no change in the tree. Otherwise, we need to determine the number of donor lineages n
                               <sub>
                                donor
                               </sub>
                               . Since the sampling is performed without replacement, the total sample size is updated to |X
                               <sub>
                                j
                               </sub>
                               | − n
                               <sub>
                                rep
                               </sub>
                               and the number of daughter lineages Y represented by nodes that are available for the coalescence becomes
                               <span class="inline-formula" id="inline-formula-25">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-25.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-25.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Thus, we compute the number of donor lineages n
                               <sub>
                                donor
                               </sub>
                               if we sample n
                               <sub>
                                rep
                               </sub>
                               times without replacement in a sample of size |X
                               <sub>
                                j
                               </sub>
                               | − n
                               <sub>
                                rep
                               </sub>
                               containing
                               <span class="inline-formula" id="inline-formula-26">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-26.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-26.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               individuals with i = j
                               <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">
                                S2b
                               </a>
                               , or in a sample of size |X
                               <sub>
                                j
                               </sub>
                               | containing
                               <span class="inline-formula" id="inline-formula-27">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-27.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-27.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               individuals with i ≠ j
                               <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">
                                S2c
                               </a>
                               . Mathematically, we can write:
                               <span class="disp-formula" id="disp-formula-5">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-10.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-10.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-6">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-11.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-11.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-7">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-12.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-12.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Knowing the number of recipient lineages, we need to determine the number of coalescences (n
                               <sub>
                                C
                               </sub>
                               ), i.e. the number of lineages among the n
                               <sub>
                                donor
                               </sub>
                               sampled donor lineages that will coalesce with the recipient ones. The number of visible coalescences is drawn from a hypergeometric law where we sample n
                               <sub>
                                recipient
                               </sub>
                               times in a sample of total size n
                               <sub>
                                rep
                               </sub>
                               containing n
                               <sub>
                                donor
                               </sub>
                               sampled donor lineages
                               <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">
                                S3a
                               </a>
                               . The number of invisible coalescences, n
                               <sub>
                                IC
                               </sub>
                               , is the number of remaining recipient lineages that have not coalesced with sampled donor lineages
                               <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">
                                S3b
                               </a>
                               .
                               <span class="disp-formula" id="disp-formula-8">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-13.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-13.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               <span class="disp-formula" id="disp-formula-9">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-14.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-14.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               Upon a coalescence event,
                               <span class="inline-formula" id="inline-formula-28">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-28.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-28.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is decreased by one, a node is randomly picked in
                               <span class="inline-formula" id="inline-formula-29">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-29.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-29.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , its height is updated to t, and it is linked to another node that is removed from
                               <span class="inline-formula" id="inline-formula-30">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-30.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-30.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . For an invisible coalescence event,
                               <span class="inline-formula" id="inline-formula-31">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-31.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-31.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               is decreased by one and both
                               <span class="inline-formula" id="inline-formula-32">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-32.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-32.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and
                               <span class="inline-formula" id="inline-formula-33">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-33.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-33.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               are increased by one. In this case, a new node from X″ is created with height t and linked to a node randomly picked in
                               <span class="inline-formula" id="inline-formula-34">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-34.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-34.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . In both cases, |X
                               <sub>
                                i
                               </sub>
                               | is decreased by one (which is already known from the trajectory).
                              </p>
                             </div>
                             <div class="section" id="sec-14">
                              <h4>
                               Death event
                              </h4>
                              <p id="p-57">
                               By default, a death reaction does not require any tree modification. However, if we want to simulate a full tree instead of a sampled one, the sampling events will correspond to the death events, which will therefore lead to the addition of a node. In this case, the number of sampled death events is n
                               <sub>
                                SD
                               </sub>
                               = n
                               <sub>
                                rep
                               </sub>
                               .
                              </p>
                             </div>
                             <div class="section" id="sec-15">
                              <h4>
                               Migration event
                              </h4>
                              <p id="p-58">
                               Only migration events involving deme individuals can lead to a modification in the tree. These migration reactions can be written as X
                               <sub>
                                i
                               </sub>
                               → X
                               <sub>
                                j
                               </sub>
                               , with j ≠ i. We assume that the number of migrations that lead to a tree modification is given by the following hypergeometric distribution:
                               <span class="disp-formula" id="disp-formula-10">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-15.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-15.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               A migration increases
                               <span class="inline-formula" id="inline-formula-35">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-35.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-35.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               and decrease
                               <span class="inline-formula" id="inline-formula-36">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-36.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-36.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               by one. A new node is created in
                               <span class="inline-formula" id="inline-formula-37">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-37.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-37.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               with height t and linked to a node randomly picked in
                               <span class="inline-formula" id="inline-formula-38">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-38.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-38.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , which is then removed from
                               <span class="inline-formula" id="inline-formula-39">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-39.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-39.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               . Furthermore, X
                               <sub>
                                i
                               </sub>
                               is incremented by one and X
                               <sub>
                                j
                               </sub>
                               is decremented by one.
                              </p>
                             </div>
                            </div>
                            <div class="section" id="sec-16">
                             <h3>
                              S2 Application to an epidemiological SI
                              <sub>
                               a
                              </sub>
                              I
                              <sub>
                               c
                              </sub>
                              R model
                             </h3>
                             <p id="p-59">
                              We illustrate the functioning of TiPS using an SI
                              <sub>
                               a
                              </sub>
                              I
                              <sub>
                               c
                              </sub>
                              R epidemiological compartmental model, where individuals can be susceptible (with density S), infected in acute phase (I
                              <sub>
                               a
                              </sub>
                              ), infected in chronic phase (I
                              <sub>
                               c
                              </sub>
                              ), and removed (R). The corresponding ODE system is
                              <span class="disp-formula" id="disp-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where β is the infectious contact rate, γ the recovery rate, α the virulence, and 1/σ the expected duration of the acute phase.
                             </p>
                             <p id="p-60">
                              The model can be described as an individual-based model using a system of reactions:
                              <span class="disp-formula" id="disp-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-20.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-20.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-21.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-21.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-22.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-22.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-23.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-23.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-24.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-24.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the rate of occurrence of each reaction is indicated above the transition arrow.
                             </p>
                             <p id="p-61">
                              Reactions
                              <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">
                               S6a
                              </a>
                              and
                              <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">
                               S6b
                              </a>
                              are birth reactions,
                              <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">
                               S6c
                              </a>
                              and
                              <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">
                               S6d
                              </a>
                              are migration reactions, and finally
                              <a id="xref-disp-formula-19-1" class="xref-disp-formula" href="#disp-formula-19">
                               S6e
                              </a>
                              is a death reaction event.
                             </p>
                             <p id="p-62">
                              TiPS will first build and generate a function to simulate trajectories using this system of reactions. Population dynamics are simulated by providing parameter values using one of the algorithm described in the main text. In the following, we focus on the model captured by system of
                              <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">
                               equations S6
                              </a>
                              .
                             </p>
                             <p id="p-63">
                              In this approach, under this epidemiological model, we trace back the epidemiological history of the sampled virus. Hence, the deme compartments, i.e. the ones sampled, where the virus is present and then contributing to the phylogeny, are I
                              <sub>
                               a
                              </sub>
                              and I
                              <sub>
                               c
                              </sub>
                              .
                             </p>
                             <p id="p-64">
                              Once the trajectory is simulated, to simulate a sampled phylogeny, TiPS requires the sampling dates and the proportion of the sampling dates to be associated with each type of deme. Let us assume, for example, that 15% of the sampling dates are associated with the I
                              <sub>
                               a
                              </sub>
                              deme and 85% with the I
                              <sub>
                               c
                              </sub>
                              deme compartment. TiPS randomly assigns each sampling date to a deme compartment based on these ratios and adds the dates to the list of events in the simulated trajectory (see
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Supplementary Figure S5
                              </a>
                              ). The R code to build the simulator, simulate a trajectory and a phylogeny are shown in
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Supplementary Figure S1
                              </a>
                              .
                             </p>
                             <p id="p-65">
                              Deme compartments (I
                              <sub>
                               a
                              </sub>
                              and I
                              <sub>
                               c
                              </sub>
                              ) can be composed of individuals represented by nodes in the simulated tree (belonging to the sub-compartments
                              <span class="inline-formula" id="inline-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-40.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-40.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-41.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-41.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ). These individuals represented by nodes can also be still unsampled (belonging to
                              <span class="inline-formula" id="inline-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-42.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-42.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              ).
                             </p>
                             <p id="p-66">
                              TiPS starts the simulation of the phylogeny from the most recent sampling event and follows the trajectory backwards in time.
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F5.large.jpg?width=800&height=600&carousel=1" title="Simulating a trajectory and a phylogeny using TiPS. The equations and outputs correspond to the SIaIcR model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable traj in red is the output trajectory. The phylogeny is plotted using using the plot method of the phylo class. [17]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Simulating a trajectory and a phylogeny using TiPS.</span> The equations and outputs correspond to the <em>SI</em><sub><em>a</em></sub><em>I</em><sub><em>c</em></sub><em>R</em> model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable <em>traj</em> in red is the output trajectory. The phylogeny is plotted using using the plot method of the <em>phylo</em> class. [17].</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F5.medium.gif" width="440" height="344"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F5.medium.gif" width="440" height="344"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1:
                               </span>
                               <span class="caption-title">
                                Simulating a trajectory and a phylogeny using TiPS.
                               </span>
                               <p id="p-67" class="first-child">
                                The equations and outputs correspond to the SI
                                <sub>
                                 a
                                </sub>
                                I
                                <sub>
                                 c
                                </sub>
                                R model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable traj in red is the output trajectory. The phylogeny is plotted using using the plot method of the phylo class. [
                                <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">
                                 17
                                </a>
                                ].
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-68">
                              If the backward step in the trajectory leads to one or multiple migration events, i.e., in this model, from the acute phase compartment (I
                              <sub>
                               a
                              </sub>
                              ) to the chronic infectious phase compartment (I
                              <sub>
                               c
                              </sub>
                              ) as in reaction
                              <a id="xref-disp-formula-17-2" class="xref-disp-formula" href="#disp-formula-17">
                               S6c
                              </a>
                              , the number of tree modifications is given by the following hypergeometric distribution:
                              <span class="disp-formula" id="disp-formula-20">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-26.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-26.gif"/>
                                </noscript>
                               </span>
                              </span>
                              An illustration of the tree update is shown in
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Supplementary Figure S2
                              </a>
                              .
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F6.large.jpg?width=800&height=600&carousel=1" title="Migration event in the SIaIcR model. The evolution of a pathogen lineage in an individual during the acute phase of its infection is represented in red. The individual then enters the chronic phase of his infection and the pathogen lineage continues to evolve (represented in blue branches). The solid branch represents the evolution of the sampled pathogen lineage and the dashed branch represents the evolution of an unsampled pathogen lineage. The unsampled lineage is eventually removed and not represented in the final simulated phylogeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Migration event in the <em>SI</em><sub><em>a</em></sub><em>I</em><sub><em>c</em></sub><em>R</em> model.</span> The evolution of a pathogen lineage in an individual during the acute phase of its infection is represented in red. The individual then enters the chronic phase of his infection and the pathogen lineage continues to evolve (represented in blue branches). The solid branch represents the evolution of the sampled pathogen lineage and the dashed branch represents the evolution of an unsampled pathogen lineage. The unsampled lineage is eventually removed and not represented in the final simulated phylogeny.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F6.medium.gif" width="440" height="240"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F6.medium.gif" width="440" height="240"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2:
                               </span>
                               <span class="caption-title">
                                Migration event in the SI
                                <sub>
                                 a
                                </sub>
                                I
                                <sub>
                                 c
                                </sub>
                                R model.
                               </span>
                               <p id="p-69" class="first-child">
                                The evolution of a pathogen lineage in an individual during the acute phase of its infection is represented in red. The individual then enters the chronic phase of his infection and the pathogen lineage continues to evolve (represented in blue branches). The solid branch represents the evolution of the sampled pathogen lineage and the dashed branch represents the evolution of an unsampled pathogen lineage. The unsampled lineage is eventually removed and not represented in the final simulated phylogeny.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-70">
                              If the backward step in the trajectory leads to a birth reaction, two tree modifications are possible: a coalescence and an invisible coalescence. Here, we consider the donor as the host transmitting the pathogen, and the recipient the one that has been infected. In this model, there are two different demes and two different birth reactions (see reactions
                              <a id="xref-disp-formula-15-3" class="xref-disp-formula" href="#disp-formula-15">
                               S6a
                              </a>
                              and
                              <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">
                               S6b
                              </a>
                              ). Given the birth reaction
                              <a id="xref-disp-formula-15-4" class="xref-disp-formula" href="#disp-formula-15">
                               S6a
                              </a>
                              , the number of coalescences n
                              <sub>
                               C
                              </sub>
                              and invisible coalescences n
                              <sub>
                               IC
                              </sub>
                              leading to tree modifications are governed by the following relationships:
                              <span class="disp-formula" id="disp-formula-21">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-28.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-28.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-22">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-29.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-29.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-23">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-30.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-30.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-24">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-31.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-31.gif"/>
                                </noscript>
                               </span>
                              </span>
                              Given birth reaction
                              <a id="xref-disp-formula-16-3" class="xref-disp-formula" href="#disp-formula-16">
                               S6b
                              </a>
                              , the number of coalescences and invisible coalescences leading to tree modifications are governed by the following relationships:
                              <span class="disp-formula" id="disp-formula-25">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-32.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-32.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-26">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-33.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-33.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-27">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-34.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-34.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-28">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-35.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-35.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Supplementary Figure S3
                              </a>
                              illustrates possible tree updates.
                             </p>
                             <p id="p-71">
                              This model features two types of sampling reactions: the sampling of an I
                              <sub>
                               a
                              </sub>
                              individual and the sampling of an I
                              <sub>
                               c
                              </sub>
                              individual. If the backward step in the trajectory leads to one or multiple sampling reactions of I
                              <sub>
                               a
                              </sub>
                              individuals, the number of re-samplings
                              <span class="inline-formula" id="inline-formula-44">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-45">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and classical samplings
                              <span class="inline-formula" id="inline-formula-46">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              and
                              <span class="inline-formula" id="inline-formula-47">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/inline-graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              are governed by the relationships
                              <a id="xref-disp-formula-29-1" class="xref-disp-formula" href="#disp-formula-29">
                               S10a
                              </a>
                              and
                              <a id="xref-disp-formula-30-1" class="xref-disp-formula" href="#disp-formula-30">
                               S10b
                              </a>
                              , respectively. An illustration of the possible modifications in the tree are shown in
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Supplementary Figure S4
                              </a>
                              . Note that the user can allow for re-sampling or not.
                              <span class="disp-formula" id="disp-formula-29">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-36.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-36.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-30">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-37.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-37.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-31">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-38.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-38.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-32">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-39.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-39.gif"/>
                                </noscript>
                               </span>
                              </span>
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F7.large.jpg?width=800&height=600&carousel=1" title="Births events in the SIaIcR model. Each transmission event from a living infectious individual to a susceptible individual can be represented by a branching. We show the donnor lineage on the right side of the branching and the recipient pathogen lineage (i.e. the newly-infected individual) on the left side. Dots correspond to nodes in the resulting phylogeny. Color and branch line codes are identical to Figure S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Births events in the <em>SI</em><sub><em>a</em></sub><em>I</em><sub><em>c</em></sub><em>R</em> model.</span> Each transmission event from a living infectious individual to a susceptible individual can be represented by a branching. We show the donnor lineage on the right side of the branching and the recipient pathogen lineage (i.e. the newly-infected individual) on the left side. Dots correspond to nodes in the resulting phylogeny. Color and branch line codes are identical to Figure S2.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F7.medium.gif" width="440" height="121"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F7.medium.gif" width="440" height="121"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S3:
                               </span>
                               <span class="caption-title">
                                Births events in the SI
                                <sub>
                                 a
                                </sub>
                                I
                                <sub>
                                 c
                                </sub>
                                R model.
                               </span>
                               <p id="p-72" class="first-child">
                                Each transmission event from a living infectious individual to a susceptible individual can be represented by a branching. We show the donnor lineage on the right side of the branching and the recipient pathogen lineage (i.e. the newly-infected individual) on the left side. Dots correspond to nodes in the resulting phylogeny. Color and branch line codes are identical to
                                <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                                 Figure S2
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F8.large.jpg?width=800&height=600&carousel=1" title="Samplings in the SIaIcR model. Colors, branches, and dots code are identical to Figure S3. Each sampling event leads to the addition of a node. Re-sampling events (left side of the figure) occur when the pathogen lineage has already been sampled but the individual currently carrying it has never been sampled (individuals can only be sampled once but can transmit after sampling). Otherwise, we have a classical sampling event (right side of the figure), i.e. the pathogen has not been sampled yet." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Samplings in the <em>SI</em><sub><em>a</em></sub><em>I</em><sub><em>c</em></sub><em>R</em> model.</span> Colors, branches, and dots code are identical to Figure S3. Each sampling event leads to the addition of a node. Re-sampling events (left side of the figure) occur when the pathogen lineage has already been sampled but the individual currently carrying it has never been sampled (individuals can only be sampled once but can transmit after sampling). Otherwise, we have a classical sampling event (right side of the figure), i.e. the pathogen has not been sampled yet.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F8.medium.gif" width="440" height="345"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F8.medium.gif" width="440" height="345"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S4:
                               </span>
                               <span class="caption-title">
                                Samplings in the SI
                                <sub>
                                 a
                                </sub>
                                I
                                <sub>
                                 c
                                </sub>
                                R model.
                               </span>
                               <p id="p-73" class="first-child">
                                Colors, branches, and dots code are identical to
                                <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                                 Figure S3
                                </a>
                                . Each sampling event leads to the addition of a node. Re-sampling events (left side of the figure) occur when the pathogen lineage has already been sampled but the individual currently carrying it has never been sampled (individuals can only be sampled once but can transmit after sampling). Otherwise, we have a classical sampling event (right side of the figure), i.e. the pathogen has not been sampled yet.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F9.large.jpg?width=800&height=600&carousel=1" title="Tree simulation. We represent the epidemiological history of the individuals carrying sampled viruses in solid lines and the rest in dashed lines. Colors are identical to Figure S3. In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side. All possible tree modifications are represented in this figure: (1) Coalescence; (2) Migration; (3) Re-sampling; (4) Invisible Coalescence; (5) Sampling." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Tree simulation.</span> We represent the epidemiological history of the individuals carrying sampled viruses in solid lines and the rest in dashed lines. Colors are identical to Figure S3. In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side. All possible tree modifications are represented in this figure: (1) Coalescence; (2) Migration; (3) Re-sampling; (4) Invisible Coalescence; (5) Sampling.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F9.medium.gif" width="440" height="323"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F9.medium.gif" width="440" height="323"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S5:
                               </span>
                               <span class="caption-title">
                                Tree simulation.
                               </span>
                               <p id="p-74" class="first-child">
                                We represent the epidemiological history of the individuals carrying sampled viruses in solid lines and the rest in dashed lines. Colors are identical to
                                <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                                 Figure S3
                                </a>
                                . In this representation, at each birth event (branching), the donor is deviated to the right side and the recipient to the left side. All possible tree modifications are represented in this figure: (1) Coalescence; (2) Migration; (3) Re-sampling; (4) Invisible Coalescence; (5) Sampling.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-17">
                             <h3>
                              S3 Application to a logistic growth model
                             </h3>
                             <p id="p-75">
                              In this section we use the African Savannah elephant (loxodonta africana) population in the Kruger National Park as an example that has demonstrated an important increase in the last two decades. A study has shown that a high number of the elephants can damage the Park’s ecosystem [
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ]. To prevent that, until the early 90’s, a solution was the culling of elephants. Due to ethical issues, another solution has been proposed where female elephants would be on contraceptives [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ].
                             </p>
                             <p id="p-76">
                              Here we use a logistic growth model to study the population dynamics. The ODE system is:
                              <span class="disp-formula" id="disp-formula-33">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-43.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-43.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-34">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-44.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-44.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where N is the elephant population density, K is the carrying capacity of the environment, and r is the intrinsic growth rate of the population where r = b − d with b the birth rate and d the death rate.
                             </p>
                             <p id="p-77">
                              The model can be described as an individual-based model using a system of reactions:
                              <span class="disp-formula" id="disp-formula-35">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-45.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-45.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-36">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-46.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-46.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where the rate of occurrence of each reaction is indicated above the transition arrow. Reaction
                              <a id="xref-disp-formula-35-1" class="xref-disp-formula" href="#disp-formula-35">
                               S13a
                              </a>
                              is a birth reaction and
                              <a id="xref-disp-formula-36-1" class="xref-disp-formula" href="#disp-formula-36">
                               S13b
                              </a>
                              a death reaction.
                             </p>
                             <p id="p-78">
                              TiPS allows to simulate a phylogeny without sampling dates, where events interrupting the biological process, such as deaths events or here sterilisation events, simulated in the trajectory are represented as leaves in the phylogeny. To illustrate this module of the tool, we add a sterilisation rate to the model to simulate the events in the trajectory. The system of reactions becomes:
                              <span class="disp-formula" id="disp-formula-37">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-47.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-47.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-38">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-48.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-48.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-39">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-49.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-49.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where reaction
                              <a id="xref-disp-formula-39-1" class="xref-disp-formula" href="#disp-formula-39">
                               S14c
                              </a>
                              is the sterilisation reaction with a rate of its occurrence indicated above the transition error. Using this system of reactions, TiPS will first build and generate a function to simulate trajectories. Population dynamics are simulated by providing parameter values using one of the algorithm described in the main text. The R code to build the simulator, simulate a trajectory and a phylogeny are shown in
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Supplementary Figure S6
                              </a>
                              .
                             </p>
                             <p id="p-79">
                              We assume that a sampling event is an event that interrupts a biological process. In this example, where no sampling dates are required, the death events and the elephant sterilisation events will be considered as the sampling events and will be represented as leaves in the simulated phylogeny.
                             </p>
                             <p id="p-80">
                              We introduce here the compartments N, N′ and N′′ where N′′ ⊆ N′ ⊆ N. All the elephants are in compartment N, the sampled elephants are sub-compartment N and the elephants that in N that have not yet been but may be sampled are in sub-compartment N′′.
                             </p>
                             <p id="p-81">
                              TiPS starts the simulation of the phylogeny from the most recent sampling event and follows the trajectory backwards-in-time.
                             </p>
                             <p id="p-82">
                              When the backward step in the trajectory at time t leads to n death or sterilisation events, n new nodes are created with height t in N′ and |N′ | increases by one. Note that, each node is labelled with the corresponding reaction, so we can distinguish and visualise them when plotting the phylogeny.
                             </p>
                             <p id="p-83">
                              If the backward step in the trajectory leads to a birth reaction, two tree modifications are possible: a coalescence and an invisible coalescence. A coalescence in the phylogeny corresponds to the coalescence of two sampled lineages each representing an elephant, into one sampled individual (the donor). An ‘invisible’ coalescence is the coalescence of the sampled lineage of the recipient individual (N′) and an unsampled lineage of the donor individual (N′′) into one unsampled lineage (the donor, N′′).
                             </p>
                             <p id="p-84">
                              Given the birth reaction
                              <a id="xref-disp-formula-37-1" class="xref-disp-formula" href="#disp-formula-37">
                               S14a
                              </a>
                              , the number of coalescences n
                              <sub>
                               C
                              </sub>
                              and invisible coalescences n
                              <sub>
                               IC
                              </sub>
                              leading to tree modifications are governed by the following relationships:
                              <span class="disp-formula" id="disp-formula-40">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-50.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-50.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-41">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-51.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-51.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-42">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-52.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-52.gif"/>
                                </noscript>
                               </span>
                              </span>
                              <span class="disp-formula" id="disp-formula-43">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-53.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/05/18/2020.11.09.373795/embed/graphic-53.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where n
                              <sub>
                               rep
                              </sub>
                              is the number of birth events in the trajectory at time t of the trajectory, n
                              <sub>
                               recipient
                              </sub>
                              is the number of recipient lineages and n
                              <sub>
                               donor
                              </sub>
                              the number of donor lineages. Upon a coalescence, |N′ | is decreased by one, a node is randomly picked from N′ with height t and is linked to another node that is removed from N′. For an invisible coalescence, |N′ | is decreased by one both and N′′ and N′ are increased by one. A new node from N′′ is created with height t ans is linked to a node randomly picked in N′. In both cases, |N | is decreased by one as recorded already in the simulated trajectory.
                             </p>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F10.large.jpg?width=800&height=600&carousel=1" title="Simulating a trajectory and a phylogeny using TiPS given a logistic growth model. The equations and outputs correspond to the logistic growth model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable traj in red is the output trajectory. The phylogeny is plotted using a function from the ape R package. [17]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Simulating a trajectory and a phylogeny using TiPS given a logistic growth model.</span> The equations and outputs correspond to the logistic growth model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable <em>traj</em> in red is the output trajectory. The phylogeny is plotted using a function from the ape R package. [17].</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F10.medium.gif" width="438" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F10.medium.gif" width="438" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S6:
                               </span>
                               <span class="caption-title">
                                Simulating a trajectory and a phylogeny using TiPS given a logistic growth model.
                               </span>
                               <p id="p-85" class="first-child">
                                The equations and outputs correspond to the logistic growth model. The functions of the R package are in blue. The simulator of trajectories built as a function is in orange. The variable traj in red is the output trajectory. The phylogeny is plotted using a function from the ape R package. [
                                <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">
                                 17
                                </a>
                                ].
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-18">
                             <h3>
                              S4 Benchmarking
                             </h3>
                             <div class="section" id="sec-19">
                              <h4>
                               S4.1 Benchmarking: trajectories
                              </h4>
                              <p id="p-86">
                               To evaluate our simulator we performed a benchmarking analysis on both modules of the tool, i.e. the trajectory and the phylogeny simulators, using two existing
                               <kbd>
                                R
                               </kbd>
                               packages.
                               <kbd>
                                adpativetau
                               </kbd>
                               performs simulations of trajectories of continuous-time Markov processes by using Gillespie’s stochastic simulation algorithms.
                               <kbd>
                                phydynR
                               </kbd>
                               is a package for phylodynamic inference using population genetic models and performs simulations of trajectories as well using the Euler-Maruyama integration method and provides methods for simulating trees conditional on a demographic process. The different algorithms of each package are presented in
                               <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                Table S1
                               </a>
                               .
                              </p>
                              <div id="T2" class="table pos-float">
                               <div class="table-inline table-callout-links">
                                <div class="callout">
                                 <span>
                                  View this table:
                                 </span>
                                 <ul class="callout-links">
                                  <li class="view-inline first">
                                   <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2553261/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                    View inline
                                   </a>
                                  </li>
                                  <li class="view-popup">
                                   <a href="/highwire/markup/2553261/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                    View popup
                                   </a>
                                  </li>
                                  <li class="download-ppt last">
                                   <a href="/highwire/powerpoint/2553261" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                    Download powerpoint
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                               <div class="table-caption">
                                <span class="table-label">
                                 Supplementary Table S1:
                                </span>
                                <span class="caption-title">
                                 Main algorithms of the
                                 <kbd>
                                  R
                                 </kbd>
                                 packages compared.
                                </span>
                                <p id="p-87" class="first-child">
                                 GDA stands for Gillespie’s Direct Algorithm, GTA for Gillespie’s Tau-Leap Algorithm, and MSA for Mixed simulation algorithm.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div class="section" id="sec-20">
                              <h4>
                               S4.2 Benchmarking: phylogenies
                              </h4>
                              <p id="p-88">
                               To evaluate the accuracy of phylogeny simulations, we generated 10 different sub-trees of 1, 000 leaves, using detailed epidemiological model with two host types, as described in [
                               <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">
                                26
                               </a>
                               ]. We then used
                               <kbd>
                                TiPS
                               </kbd>
                               and
                               <kbd>
                                phydynR
                               </kbd>
                               to simulate 1, 000 phylogenies with each package under the same epidemiological model with the same parameter values, imposing the 1, 000 dates of each target sub-tree under a backwards-in-time approach. To compare the simulated phylogenies with the target one, we computed summary statistics for each of them using the methods described in [
                               <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                               ].
                               <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                                Supplementary Figure S7
                               </a>
                               shows the distributions of summary statistics computed from the phylogenies simulated using
                               <kbd>
                                TiPS
                               </kbd>
                               (in red) and using
                               <kbd>
                                phydynR
                               </kbd>
                               (in orange) for each analysis using a different target tree (in black).
                              </p>
                              <div id="F11" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F11.large.jpg?width=800&height=600&carousel=1" title="Distributions of summary statistics. The summary statistics are grouped into three families : branch lengths (denoted BL), tree topology (TOPO) and Lineage-Throuht-Time plot (LTT). The dots represent the median and the horizontal lines represent the 95% HPD. Red distributions correspond to the summary statistics computed from the phylogenies simulated using TiPS, orange distributions correspond to the summary statistics computed from the phylogenies simulated using phydynR, for each target tree analysis. Black dots represent the values of summary statistics computed from each target tree." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-857318795" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Distributions of summary statistics.</span> The summary statistics are grouped into three families : branch lengths (denoted BL), tree topology (TOPO) and Lineage-Throuht-Time plot (LTT). The dots represent the median and the horizontal lines represent the 95% HPD. Red distributions correspond to the summary statistics computed from the phylogenies simulated using <kbd>TiPS</kbd>, orange distributions correspond to the summary statistics computed from the phylogenies simulated using <kbd>phydynR</kbd>, for each target tree analysis. Black dots represent the values of summary statistics computed from each target tree.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F11.medium.gif" width="440" height="326"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F11.medium.gif" width="440" height="326"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/05/18/2020.11.09.373795/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure S7:
                                </span>
                                <span class="caption-title">
                                 Distributions of summary statistics.
                                </span>
                                <p id="p-89" class="first-child">
                                 The summary statistics are grouped into three families : branch lengths (denoted BL), tree topology (TOPO) and Lineage-Throuht-Time plot (LTT). The dots represent the median and the horizontal lines represent the 95% HPD. Red distributions correspond to the summary statistics computed from the phylogenies simulated using
                                 <kbd>
                                  TiPS
                                 </kbd>
                                 , orange distributions correspond to the summary statistics computed from the phylogenies simulated using
                                 <kbd>
                                  phydynR
                                 </kbd>
                                 , for each target tree analysis. Black dots represent the values of summary statistics computed from each target tree.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.09.373795v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Otto
                                  </span>
                                  <span class="cit-name-given-names">
                                   SP
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Day
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  A biologist’s guide to mathematical modeling in ecology and evolution
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Princeton University Press
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.09.373795v2.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keeling
                                  </span>
                                  <span class="cit-name-given-names">
                                   MJ
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rohani
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-source">
                                  Modeling infectious diseases in humans and animals
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Princeton University Press
                                 </span>
                                 ;
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenormand
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roze
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rousset
                                  </span>
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Stochasticity in evolution
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends in Ecology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  24
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  157
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  165
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D24%26rft.spage%253D157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.4" data-doi="10.1126/science.1090727">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grenfell
                                  </span>
                                  <span class="cit-name-given-names">
                                   BT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pybus
                                  </span>
                                  <span class="cit-name-given-names">
                                   OG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gog
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wood
                                  </span>
                                  <span class="cit-name-given-names">
                                   JLN
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janet
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mumford
                                  </span>
                                  <span class="cit-name-given-names">
                                   JA
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Unifying the Epidemiological and Evolutionary Dynamics of Pathogens
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2004
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  303
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5656
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  327
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  332
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGrenfell%26rft.auinit1%253DB.%2BT.%26rft.volume%253D303%26rft.issue%253D5656%26rft.spage%253D327%26rft.epage%253D332%26rft.atitle%253DUnifying%2Bthe%2BEpidemiological%2Band%2BEvolutionary%2BDynamics%2Bof%2BPathogens%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090727%26rft_id%253Dinfo%253Apmid%252F14726583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDMvNTY1Ni8zMjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8xOC8yMDIwLjExLjA5LjM3Mzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.09.373795v2.5" data-doi="10.1371/journal.pcbi.1002947">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Volz
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koelle
                                  </span>
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bedford
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Viral phylodynamics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1002947
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D9%26rft.spage%253De1002947%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002947%26rft_id%253Dinfo%253Apmid%252F23555203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002947&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23555203&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.6" data-doi="10.1016/j.epidem.2014.09.001">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frost
                                  </span>
                                  <span class="cit-name-given-names">
                                   SD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pybus
                                  </span>
                                  <span class="cit-name-given-names">
                                   OG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gog
                                  </span>
                                  <span class="cit-name-given-names">
                                   JR
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Viboud
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bonhoeffer
                                  </span>
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bedford
                                  </span>
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Eight challenges in phylodynamic inference
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Epidemics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  88
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  92
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEpidemics%26rft.volume%253D10%26rft.spage%253D88%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.epidem.2014.09.001%26rft_id%253Dinfo%253Apmid%252F25843391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.epidem.2014.09.001&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25843391&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.7" data-doi="10.1371/journal.pcbi.1002835">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ratmann
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Donker
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meijer
                                  </span>
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fraser
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koelle
                                  </span>
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Phylodynamic inference and model assessment with approximate bayesian computation: influenza as a case study
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Comput Biol
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1002835
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput%2BBiol%26rft.volume%253D8%26rft.spage%253De1002835%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002835%26rft_id%253Dinfo%253Apmid%252F23300420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002835&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23300420&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.11.09.373795v2.8" data-doi="10.1093/sysbio/syv014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gascuel
                                  </span>
                                  <span class="cit-name-given-names">
                                   F
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ferrière
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aguilée
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambert
                                  </span>
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  How Ecology and Landscape Dynamics Shape Phylogenetic Trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systematic Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  590
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  607
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystematic%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fsysbio%252Fsyv014%26rft_id%253Dinfo%253Apmid%252F25771083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/sysbio/syv014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25771083&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saulnier
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gascuel
                                  </span>
                                  <span class="cit-name-given-names">
                                   O
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alizon
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Inferring epidemiological parameters from phylogenies using regression-ABC: A comparative study
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS Computational Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 <span class="cit-month">
                                  Mar
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1005416
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BComputational%2BBiology%26rft.volume%253D13%26rft.spage%253D1005416e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.10" data-doi="10.1016/0021-9991(76)90041-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  <span class="cit-name-given-names">
                                   DT
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A general method for numerically simulating the stochastic time evolution of coupled chemical reactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Computational Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1976
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  403
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  434
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BComputational%2BPhysics%26rft.volume%253D22%26rft.spage%253D403%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0021-9991%252876%252990041-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0021-9991(76)90041-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1976CQ87900001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.11" data-doi="10.2307/3212147">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kurtz
                                  </span>
                                  <span class="cit-name-given-names">
                                   TG
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Solutions of ordinary differential equations as limits of pure jump markov processes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Applied Probability
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1970
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  49
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  58
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BProbability%26rft.volume%253D7%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3212147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/3212147&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1970F968800004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.12" data-doi="10.18637/JSS.V025.I10">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pineda-Krch
                                  </span>
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  GillespieSSA: implementing the stochastic simulation algorithm in R
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Statistical Software
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BStatistical%2BSoftware%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252FJSS.V025.I10%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.18637/JSS.V025.I10&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  adaptivetau: efficient stochastic simulations in R
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  R Package Version
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;.
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.11.09.373795v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bjørnstad
                                  </span>
                                  <span class="cit-name-given-names">
                                   ON
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Epidemics
                                 </span>
                                 .
                                 <span class="cit-source">
                                  Models and Data using R
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer; 2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.15" data-doi="10.1093/bioinformatics/btu181">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pennell
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eastman
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Slater
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brown
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Uyeda
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fitzjohn
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  geiger v2.0: an expanded suite of methods for fitting macroevolutionary models to phylogenetic trees
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  2216
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2218
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu181%26rft_id%253Dinfo%253Apmid%252F24728855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu181&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24728855&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Revell
                                  </span>
                                  <span class="cit-name-given-names">
                                   LJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  phytools: An R package for phylogenetic comparative biology (and other things)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods in Ecology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  217
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  223
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2BEcology%2Band%2BEvolution%26rft.volume%253D3%26rft.spage%253D217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.17" data-doi="10.1093/bioinformatics/bty633">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paradis
                                  </span>
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schliep
                                  </span>
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  526
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  528
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D35%26rft.spage%253D526%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty633%26rft_id%253Dinfo%253Apmid%252F30016406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty633&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30016406&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.18" data-doi="10.1534/genetics.111.134627">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Volz
                                  </span>
                                  <span class="cit-name-given-names">
                                   EM
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Complex population dynamics and the coalescent under neutrality
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genetics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  190
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  187
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  201
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.134627%26rft_id%253Dinfo%253Apmid%252F22042576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzEvMTg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDUvMTgvMjAyMC4xMS4wOS4zNzM3OTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.19" data-doi="10.1093/molbev/mst057">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vaughan
                                  </span>
                                  <span class="cit-name-given-names">
                                   TG
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Drummond
                                  </span>
                                  <span class="cit-name-given-names">
                                   AJ
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A stochastic simulator of birth-death master equations with application to phylodynamics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular Biology and Evolution
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  30
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1480
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1493
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst057%26rft_id%253Dinfo%253Apmid%252F23505043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/molbev/mst057&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23505043&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F05%2F18%2F2020.11.09.373795.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000318812000023&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bouckaert
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heled
                                  </span>
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kühnert
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vaughan
                                  </span>
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  <span class="cit-name-given-names">
                                   CH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  BEAST 2: A Software Platform for Bayesian Evolutionary Analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Computational Biology
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BComputational%2BBiology%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eddelbuettel
                                  </span>
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Francois
                                  </span>
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Rcpp: Seamless R and C++ Integration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Stat Software
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BStat%2BSoftware%26rft.volume%253D40%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.22" data-doi="10.1063/1.1378322">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  <span class="cit-name-given-names">
                                   DT
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Approximate accelerated stochastic simulation of chemically reacting systems
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2001
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1716
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1733
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BPhysics%26rft.volume%253D115%26rft.spage%253D1716%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.1378322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.1378322&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000169800000010&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.23" data-doi="10.1063/1.1824902">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cao
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  <span class="cit-name-given-names">
                                   DT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petzold
                                  </span>
                                  <span class="cit-name-given-names">
                                   LR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The slow-scale stochastic simulation algorithm
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  122
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  014116
                                 </span>
                                 . Available from:
                                 <a href="https://doi.org/10.1063/1.1824902">
                                  https://doi.org/10.1063/1.1824902
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BChemical%2BPhysics%26rft.volume%253D122%26rft.spage%253D014116%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.1824902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.1824902&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.24" data-doi="10.1063/1.2746330">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cao
                                  </span>
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gillespie
                                  </span>
                                  <span class="cit-name-given-names">
                                   DT
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Petzold
                                  </span>
                                  <span class="cit-name-given-names">
                                   LR
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Adaptive explicit-implicit tau-leaping method with automatic tau selection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Chemical Physics
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  126
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  22
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  10
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BChemical%2BPhysics%26rft.volume%253D126%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1063%252F1.2746330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1063/1.2746330&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.25" data-doi="10.1016/0304-4149(82)90011-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingman
                                  </span>
                                  <span class="cit-name-given-names">
                                   JFC
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The coalescent
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Stochastic processes and their applications
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  235
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DStochastic%2Bprocesses%2Band%2Btheir%2Bapplications%26rft.volume%253D13%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4149%252882%252990011-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0304-4149(82)90011-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danesh
                                  </span>
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Virlogeux
                                  </span>
                                  <span class="cit-name-given-names">
                                   V
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ramière
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Charre
                                  </span>
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cotte
                                  </span>
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alizon
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Quantifying transmission dynamics of acute hepatitis C virus infections in a heterogeneous population using sequence data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Pathogens
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  9
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  e1009916
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathogens%26rft.volume%253D17%26rft.spage%253De1009916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.27" data-doi="10.1086/674795">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartfield
                                  </span>
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alizon
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Epidemiological feedbacks affect evolutionary emergence of pathogens
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am Nat
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  183
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  E105
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E117
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAm%2BNat%26rft.volume%253D183%26rft.spage%253DE105%26rft_id%253Dinfo%253Adoi%252F10.1086%252F674795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1086/674795&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.28" data-doi="10.1126/science.abg0100">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alizon
                                  </span>
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Superspreading genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  371
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6529
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  574
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  575
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAlizon%26rft.auinit1%253DS.%26rft.volume%253D371%26rft.issue%253D6529%26rft.spage%253D574%26rft.epage%253D575%26rft.atitle%253DSuperspreading%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.abg0100%26rft_id%253Dinfo%253Apmid%252F33542127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNzEvNjUyOS81NzQiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wNS8xOC8yMDIwLjExLjA5LjM3Mzc5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   DH
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fenton
                                  </span>
                                  <span class="cit-name-given-names">
                                   MB
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rautenbach
                                  </span>
                                  <span class="cit-name-given-names">
                                   IL
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Taylor
                                  </span>
                                  <span class="cit-name-given-names">
                                   RD
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   GS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cumming
                                  </span>
                                  <span class="cit-name-given-names">
                                   MS
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Elephants, woodlands and biodiversity in southern Africa
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  South African Journal of Science
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  93
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  231
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  236
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSouth%2BAfrican%2BJournal%2Bof%2BScience%26rft.volume%253D93%26rft.spage%253D231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1997XP49200013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.11.09.373795v2.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whyte
                                  </span>
                                  <span class="cit-name-given-names">
                                   I
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   van Aarde
                                  </span>
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pimm
                                  </span>
                                  <span class="cit-name-given-names">
                                   SL
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Managing the elephants of Kruger National Park
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Animal Conservation
                                 </abbr>
                                 .
                                 <span class="cit-pub-date">
                                  1998
                                 </span>
                                 ;
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  77
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  83
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnimal%2BConservation%26rft.volume%253D1%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.11.09.373795v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.04.26.489486v3" title="Single-cell tracking data aimed for big data analyses" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.03.25.485836v2" title="Facultative parthenogenesis in an asexual stick insect: re-evolution of sex or vestigial sexual capacity?" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted May 18, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.11.09.373795v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.11.09.373795v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.11.09.373795v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   TiPS: rapidly simulating trajectories and phylogenies from compartmental models
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2550658"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-5K0S1Jwqo0Zs8A7YYei2NQx07yNi3C_rq2QCfimdTZQ"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484484804"/>
                                   <input type="hidden" name="captcha_token" value="2de5a2748a8a1b4d0d225641b1d447b2"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2550658--3" data-node-nid="2550658" id="node-2550658--41824985214" data-pisa="biorxiv;2020.11.09.373795v2" data-pisa-master="biorxiv;2020.11.09.373795" data-seqnum="2550658" data-apath="/biorxiv/early/2022/05/18/2020.11.09.373795.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 TiPS: rapidly simulating trajectories and phylogenies from compartmental models
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Gonché
                                  </span>
                                  <span class="nlm-surname">
                                   Danesh
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Emma
                                  </span>
                                  <span class="nlm-surname">
                                   Saulnier
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Olivier
                                  </span>
                                  <span class="nlm-surname">
                                   Gascuel
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Marc
                                  </span>
                                  <span class="nlm-surname">
                                   Choisy
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Samuel
                                  </span>
                                  <span class="nlm-surname">
                                   Alizon
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.11.09.373795;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.11.09.373795
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&title=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&title=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&text=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&t=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&title=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&title=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&title=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2550658--5" data-node-nid="2550658" id="citation-node-2550658--634326655" data-pisa="biorxiv;2020.11.09.373795v2" data-pisa-master="biorxiv;2020.11.09.373795" data-seqnum="2550658" data-apath="/biorxiv/early/2022/05/18/2020.11.09.373795.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     TiPS: rapidly simulating trajectories and phylogenies from compartmental models
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Gonché
                                      </span>
                                      <span class="nlm-surname">
                                       Danesh
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Emma
                                      </span>
                                      <span class="nlm-surname">
                                       Saulnier
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Olivier
                                      </span>
                                      <span class="nlm-surname">
                                       Gascuel
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Marc
                                      </span>
                                      <span class="nlm-surname">
                                       Choisy
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Samuel
                                      </span>
                                      <span class="nlm-surname">
                                       Alizon
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.11.09.373795;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.11.09.373795
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Danesh%2C+Gonch%C3%A9&rft.au=Saulnier%2C+Emma&rft.au=Gascuel%2C+Olivier&rft.au=Choisy%2C+Marc&rft.au=Alizon%2C+Samuel">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2550658/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2550658/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2550658/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2550658/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2550658/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2550658/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2550658/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2550658/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2550658/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2550658/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2550658/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2550658/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&count=horizontal&via=&text=TiPS%3A%20rapidly%20simulating%20trajectories%20and%20phylogenies%20from%20compartmental%20models&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.11.09.373795v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.11.09.373795v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/bioinformatics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Bioinformatics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
