<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.09.07.285114v4.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.09.07.285114v4.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.09.07.285114v4.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/09/2020.09.07.285114.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.09.07.285114v4"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers"/>
  <meta name="DC.Identifier" content="10.1101/2020.09.07.285114"/>
  <meta name="DC.Date" content="2022-03-09"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="By using TGIRT-seq, we identified >8,500 short full-length excised linear intron (FLEXI) RNAs in human cells. Subsets of FLEXIs accumulated in a cell-type specific manner, and ∼200 corresponded to agotrons or mirtrons or encoded snoRNAs. Analysis of CLIP-seq datasets identified potential interactions between FLEXIs and >100 different RNA-binding proteins (RBPs), 53 of which had binding sites in ≥30 different FLEXIs. In addition to proteins that function in RNA splicing, these 53 RBPs included transcription factors, chromatin remodeling proteins, and cellular growth regulators that impacted FLEXI host gene alternative splicing and/or mRNA levels in knockdown datasets. We computationally identified six groups of RBPs whose binding sites were enriched in different subsets of FLEXIs: AGO1-4 and DICER associated with agotrons and mirtrons; AATF, DKC1, NOLCI, and SMNDC1 associated with snoRNA-encoding FLEXIs; two different combinations of alternative splicing factors found in stress granules; and two novel RBP-intron combinations, one including LARP4 and PABC4, which function together in the cytoplasm to regulate ribosomal protein translation. Our results suggest a model in which proteins involved in transcriptional regulation, alternative splicing, or post-splicing secondary functions bind and stabilize cell-type specific subsets of FLEXIs that perform different biological functions and have potential utility as biomarkers.

### Competing Interest Statement

Thermostable group II intron reverse transcriptase (TGIRT) enzymes and methods for their use are the subject of patents and patent applications that have been licensed by the University of Texas and East Tennessee State University to InGex, LLC. A.M.L., some former and present members of the Lambowitz laboratory, and the University of Texas are minority equity holders in InGex and receive royalty payments from the sale of TGIRT enzymes and kits and from the sublicensing of intellectual property by InGex to other companies. A.M.L., J.Y., H.X. and D.C.W. are inventors on a patent application filed by the University of Texas at Austin for the use of full-length excised intron RNAs and intron RNA fragments as biomarkers. S.W., E.A.F.K. and M.A. have no competing interests."/>
  <meta name="DC.Contributor" content="Jun Yao"/>
  <meta name="DC.Contributor" content="Shelby Winans"/>
  <meta name="DC.Contributor" content="Hengyi Xu"/>
  <meta name="DC.Contributor" content="Douglas C. Wu"/>
  <meta name="DC.Contributor" content="Elizabeth A. Ferrick-Kiddie"/>
  <meta name="DC.Contributor" content="Manuel Ares"/>
  <meta name="DC.Contributor" content="Alan M. Lambowitz"/>
  <meta name="article:published_time" content="2022-03-09"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers"/>
  <meta name="citation_abstract" lang="en" content="<h3>ABSTRACT</h3>
<p>By using TGIRT-seq, we identified &gt;8,500 short full-length excised linear intron (FLEXI) RNAs in human cells. Subsets of FLEXIs accumulated in a cell-type specific manner, and ∼200 corresponded to agotrons or mirtrons or encoded snoRNAs. Analysis of CLIP-seq datasets identified potential interactions between FLEXIs and &gt;100 different RNA-binding proteins (RBPs), 53 of which had binding sites in ≥30 different FLEXIs. In addition to proteins that function in RNA splicing, these 53 RBPs included transcription factors, chromatin remodeling proteins, and cellular growth regulators that impacted FLEXI host gene alternative splicing and/or mRNA levels in knockdown datasets. We computationally identified six groups of RBPs whose binding sites were enriched in different subsets of FLEXIs: AGO1-4 and DICER associated with agotrons and mirtrons; AATF, DKC1, NOLCI, and SMNDC1 associated with snoRNA-encoding FLEXIs; two different combinations of alternative splicing factors found in stress granules; and two novel RBP-intron combinations, one including LARP4 and PABC4, which function together in the cytoplasm to regulate ribosomal protein translation. Our results suggest a model in which proteins involved in transcriptional regulation, alternative splicing, or post-splicing secondary functions bind and stabilize cell-type specific subsets of FLEXIs that perform different biological functions and have potential utility as biomarkers.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.09.07.285114v4"/>
  <meta name="citation_id" content="2020.09.07.285114v4"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.09.07.285114"/>
  <meta name="citation_num_pages" content="110"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.09.07.285114"/>
  <meta name="citation_author" content="Jun Yao"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1232-1587"/>
  <meta name="citation_author" content="Shelby Winans"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0434-8074"/>
  <meta name="citation_author" content="Hengyi Xu"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9809-4882"/>
  <meta name="citation_author" content="Douglas C. Wu"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_institution" content="Invitae Corporation"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6179-3110"/>
  <meta name="citation_author" content="Elizabeth A. Ferrick-Kiddie"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0088-5315"/>
  <meta name="citation_author" content="Manuel Ares"/>
  <meta name="citation_author_institution" content="Department of Molecular, Cell, and Developmental Biology University of California"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2552-9168"/>
  <meta name="citation_author" content="Alan M. Lambowitz"/>
  <meta name="citation_author_institution" content="Departments of Molecular Biosciences and Oncology University of Texas at"/>
  <meta name="citation_author_institution" content="TX 78712"/>
  <meta name="citation_author_email" content="lambowitz@austin.utexas.edu"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6036-2423"/>
  <meta name="citation_reference" content="Abebrese, E.L., Ali, S.H., Arnold, Z.R., Andrews, V.M., Armstrong, K., Burns, L., Crowder, H.R., Day, R.T., Jr., Hsu, D.G., Jarrell, K., et al. (2017). Identification of human short introns. PLOS ONE 12, e0175393. 10.1371/journal.pone.0175393."/>
  <meta name="citation_reference" content="Alawi, F., and Lin, P. (2010). Loss of dyskerin reduces the accumulation of a subset of H/ACA snoRNA-derived miRNA. Cell Cycle 9, 2467–2469. 10.4161/cc.9.12.11922."/>
  <meta name="citation_reference" content="Bailey, T.L., Boden, M., Buske, F.A., Frith, M., Grant, C.E., Clementi, L., Ren, J., Li, W.W., and Noble, W.S. (2009). MEME Suite: tools for motif discovery and searching. Nucleic Acids Res. 37, W202–W208. 10.1093/nar/gkp335."/>
  <meta name="citation_reference" content="Berezikov, E., Chung, W.-J., Willis, J., Cuppen, E., and Lai, E.C. (2007). Mammalian mirtron genes. Mol. Cell 28, 328–336. 10.1016/j.molcel.2007.09.028."/>
  <meta name="citation_reference" content="Burset, M., Seledtsov, I.A., and Solovyev, V.V. (2000). Analysis of canonical and non-canonical splice sites in mammalian genomes. Nucleic Acids Res. 28, 4364–4375. 10.1093/nar/28.21.4364."/>
  <meta name="citation_reference" content="Chapman, K.B., and Boeke, J.D. (1991). Isolation and characterization of the gene encoding yeast debranching enzyme. Cell 65, 483–492. 10.1016/0092-8674(91)90466-C."/>
  <meta name="citation_reference" content="Defays, D. (1977). An efficient algorithm for a complete link method. The Computer Journal 20, 364–366. 10.1093/comjnl/20.4.364."/>
  <meta name="citation_reference" content="Dodd, D.W., Gagnon, K.T., and Corey, D.R. (2013). Digital quantitation of potential therapeutic target RNAs. Nucleic Acid Ther 23, 188–194. 10.1089/nat.2013.0427."/>
  <meta name="citation_reference" content="Farrell, M.J., Dobson, A.T., and Feldman, L.T. (1991). Herpes simplex virus latency-associated transcript is a stable intron. Proc. Natl. Acad. Sci. U.S.A. 88, 790–794. 10.1073/pnas.88.3.790."/>
  <meta name="citation_reference" content="Gao, K., Masuda, A., Matsuura, T., and Ohno, K. (2008). Human branch point consensus sequence is yUnAy. Nucleic Acids Res. 36, 2257–2267. 10.1093/nar/gkn073."/>
  <meta name="citation_reference" content="Gardner, E.J., Nizami, Z.F., Talbot, C.C., and Gall, J.G. (2012). Stable intronic sequence RNA (sisRNA), a new class of noncoding RNA from the oocyte nucleus of Xenopus tropicalis. Genes Dev. 26, 2550–2559. 10.1101/gad.202184.112."/>
  <meta name="citation_reference" content="Gower, J.C. (1971). A General Coefficient of Similarity and Some of Its Properties. Biometrics 27, 857. 10.2307/2528823."/>
  <meta name="citation_reference" content="Grant, C.E., Bailey, T.L., and Noble, W.S. (2011). FIMO: scanning for occurrences of a given motif. Bioinformatics 27, 1017–1018. 10.1093/bioinformatics/btr064."/>
  <meta name="citation_reference" content="Gu, Z., Eils, R., and Schlesner, M. (2016). Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics 32, 2847–2849. 10.1093/bioinformatics/btw313."/>
  <meta name="citation_reference" content="Hafner, M., Landthaler, M., Burger, L., Khorshid, M., Hausser, J., Berninger, P., Rothballer, A., Ascano, M., Jr., Jungkamp, A.-C., Munschauer, M., et al. (2010). Transcriptome-wide identification of RNA-binding protein and microRNA target sites by PAR-CLIP. Cell 141, 129–141. 10.1016/j.cell.2010.03.009."/>
  <meta name="citation_reference" content="Hansen, K.D., Irizarry, R.A., and Wu, Z. (2012). Removing technical variability in RNA-seq data using conditional quantile normalization. Biostatistics 13, 204–216. 10.1093/biostatistics/kxr054."/>
  <meta name="citation_reference" content="Hansen, T.B., Venø, M.T., Jensen, T.I., Schaefer, A., Damgaard, C.K., and Kjems, J. (2016). Argonaute-associated short introns are a novel class of gene regulators. Nat Commun 7, 11538. 10.1038/ncomms11538."/>
  <meta name="citation_reference" content="Huang, D.W., Sherman, B.T., and Lempicki, R.A. (2009). Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat. Protoc. 4, 44–57. 10.1038/nprot.2008.211."/>
  <meta name="citation_reference" content="Kaiser, R.W.J., Ignarski, M., Van Nostrand, E.L., Frese, C.K., Jain, M., Cukoski, S., Heinen, H., Schaechter, M., Seufert, L., Bunte, K., et al. (2019). A protein-RNA interaction atlas of the ribosome biogenesis factor AATF. Sci. Rep. 9, 11071. 10.1038/s41598-019-47552-3."/>
  <meta name="citation_reference" content="Katibah, G.E., Qin, Y., Sidote, D.J., Yao, J., Lambowitz, A.M., and Collins, K. (2014). Broad and adaptable RNA structure recognition by the human interferon-induced tetratricopeptide repeat protein IFIT5. Proc. Natl. Acad. Sci. U.S.A. 111, 12025–12030. 10.1073/pnas.1412842111."/>
  <meta name="citation_reference" content="Kim, D., Paggi, J.M., Park, C., Bennett, C., and Salzberg, S.L. (2019). Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat. Biotechnol. 37, 907–915. 10.1038/s41587-019-0201-4."/>
  <meta name="citation_reference" content="Kim, P., Yang, M., Yiya, K., Zhao, W., and Zhou, X. (2020). ExonSkipDB: functional annotation of exon skipping event in human. Nucleic Acids Res. 48, D896–D907. 10.1093/nar/gkz917."/>
  <meta name="citation_reference" content="Kobak, D., and Berens, P. (2019). The art of using t-SNE for single-cell transcriptomics. Nat Commun 10, 5416. 10.1038/s41467-019-13056-x."/>
  <meta name="citation_reference" content="Kulesza, C.A., and Shenk, T. (2006). Murine cytomegalovirus encodes a stable intron that facilitates persistent replication in the mouse. Proc. Natl. Acad. Sci. U.S.A. 103, 18302–18307. 10.1073/pnas.0608718103."/>
  <meta name="citation_reference" content="Langmead, B., and Salzberg, S.L. (2012). Fast gapped-read alignment with Bowtie 2. Nat. Methods 9, 357–359. 10.1038/nmeth.1923."/>
  <meta name="citation_reference" content="Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R. (2009). The sequence alignment/map format and SAMtools. Bioinformatics 25, 2078–2079. 10.1093/bioinformatics/btp352."/>
  <meta name="citation_reference" content="Liberzon, A., Birger, C., Thorvaldsdóttir, H., Ghandi, M., Mesirov, J.P., and Tamayo, P. (2015). The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst. 1, 417–425. 10.1016/j.cels.2015.12.004."/>
  <meta name="citation_reference" content="Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17, pp. 10–12."/>
  <meta name="citation_reference" content="Massenet, S., Bertrand, E., and Verheggen, C. (2017). Assembly and trafficking of box C/D and H/ACA snoRNPs. RNA Biol 14, 680–692. 10.1080/15476286.2016.1243646."/>
  <meta name="citation_reference" content="Mattijssen, S., Arimbasseri, A.G., Iben, J.R., Gaidamakov, S., Lee, J., Hafner, M., and Maraia, R.J. (2017). LARP4 mRNA codon-tRNA match contributes to LARP4 activity for ribosomal protein mRNA poly(A) tail length protection. eLife 6, e28889. 10.7554/eLife.28889."/>
  <meta name="citation_reference" content="Morgan, J.T., Fink, G.R., and Bartel, D.P. (2019). Excised linear introns regulate growth in yeast. Nature 565, 606–611. 10.1038/s41586-018-0828-1."/>
  <meta name="citation_reference" content="Moss, W.N., and Steitz, J.A. (2013). Genome-wide analyses of Epstein-Barr virus reveal conserved RNA structures and a novel stable intronic sequence RNA. BMC Genomics 14, 543. 10.1186/1471-2164-14-543."/>
  <meta name="citation_reference" content="Nunes, C., Mestre, I., Marcelo, A., Koppenol, R., Matos, C.A., and Nóbrega, C. (2019). MSGP: the first database of the protein components of the mammalian stress granules. Database 2019, baz031. 10.1093/database/baz031."/>
  <meta name="citation_reference" content="Okamura, K., Hagen, J.W., Duan, H., Tyler, D.M., and Lai, E.C. (2007). The mirtron pathway generates microRNA-class regulatory RNAs in Drosophila. Cell 130, 89–100. 10.1016/j.cell.2007.06.028."/>
  <meta name="citation_reference" content="Parenteau, J., Maignon, L., Berthoumieux, M., Catala, M., Gagnon, V., and Abou Elela, S. (2019). Introns are mediators of cell response to starvation. Nature 565, 612–617. 10.1038/s41586-018-0859-7."/>
  <meta name="citation_reference" content="Patro, R., Duggal, G., Love, M.I., Irizarry, R.A., and Kingsford, C. (2017). Salmon provides fast and bias-aware quantification of transcript expression. Nature Methods 14, 417–419. 10.1038/nmeth.4197."/>
  <meta name="citation_reference" content="Pek, J.W., Osman, I., Tay, M.L.-I., and Zheng, R.T. (2015). Stable intronic sequence RNAs have possible regulatory roles in Drosophila melanogaster. J. Cell Biol. 211, 243–251. 10.1083/jcb.201507065."/>
  <meta name="citation_reference" content="Qin, Y., Yao, J., Wu, D.C., Nottingham, R.M., Mohr, S., Hunicke-Smith, S., and Lambowitz, A.M. (2016). High-throughput sequencing of human plasma RNA by using thermostable group II intron reverse transcriptases. RNA 22, 111–128. 10.1261/rna.054809.115."/>
  <meta name="citation_reference" content="Quinlan, A.R. (2014). BEDTools: the swiss-army tool for genome feature analysis. Curr. Protoc. Bioinformatics 47, 11.12.11–34. 10.1002/0471250953.bi1112s47."/>
  <meta name="citation_reference" content="Risso, D., Perraudeau, F., Gribkova, S., Dudoit, S., and Vert, J.-P. (2018). A general and flexible method for signal extraction from single-cell RNA-seq data. Nat Commun 9, 284. 10.1038/s41467-017-02554-5."/>
  <meta name="citation_reference" content="Robinson, J.T., Thorvaldsdóttir, H., Winckler, W., Guttman, M., Lander, E.S., Getz, G., and Mesirov, J.P. (2011). Integrative genomics viewer."/>
  <meta name="citation_reference" content="Ruby, J.G., Jan, C.H., and Bartel, D.P. (2007). Intronic microRNA precursors that bypass Drosha processing. Nature 448, 83–86. 10.1038/nature05983."/>
  <meta name="citation_reference" content="Rybak-Wolf, A., Jens, M., Murakawa, Y., Herzog, M., Landthaler, M., and Rajewsky, N. (2014). A variety of Dicer substrates in human and C. elegans. Cell 159, 1153–1167. 10.1016/j.cell.2014.10.040."/>
  <meta name="citation_reference" content="Saini, H., Bicknell, A.A., Eddy, S.R., and Moore, M.J. (2019). Free circular introns with an unusual branchpoint in neuronal projections. eLife 8, e47809. 10.7554/eLife.47809."/>
  <meta name="citation_reference" content="Savage, K.I., Gorski, J.J., Barros, E.M., Irwin, G.W., Manti, L., Powell, A.J., Pellagatti, A., Lukashchuk, N., McCance, D.J., McCluggage, W.G., et al. (2014). Identification of a BRCA1- mRNA splicing complex required for efficient DNA repair and maintenance of genomic stability. Mol Cell 54, 445–459. 10.1016/j.molcel.2014.03.021."/>
  <meta name="citation_reference" content="Sharbati, S., Sharbati, J., Hoeke, L., Bohmer, M., and Einspanier, R. (2012). Quantification and accurate normalisation of small RNAs through new custom RT-qPCR arrays demonstrates Salmonella-induced microRNAs in human monocytes. BMC Genomics 13, 23. 10.1186/1471-2164-13-23."/>
  <meta name="citation_reference" content="Shen, S., Park, J.W., Lu, Z.-x., Lin, L., Henry, M.D., Wu, Y.N., Zhou, Q., and Xing, Y. (2014). rMATS: Robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Proceedings of the National Academy of Sciences 111, E5593–E5601. 10.1073/pnas.1419161111."/>
  <meta name="citation_reference" content="Sheth, N., Roca, X., Hastings, M.L., Roeder, T., Krainer, A.R., and Sachidanandam, R. (2006). Comprehensive splice-site analysis using comparative genomics. Nucleic Acids Res. 34, 3955–3967. 10.1093/nar/gkl556."/>
  <meta name="citation_reference" content="Shurtleff, M.J., Yao, J., Qin, Y., Nottingham, R.M., Temoche-Diaz, M.M., Schekman, R., and Lambowitz, A.M. (2017). Broad role for YBX1 in defining the small noncoding RNA composition of exosomes. Proc. Natl. Acad. Sci. U.S.A. 114, E8987–E8995. 10.1073/pnas.1712108114."/>
  <meta name="citation_reference" content="Siepel, A., Bejerano, G., Pedersen, J.S., Hinrichs, A.S., Hou, M., Rosenbloom, K., Clawson, H., Spieth, J., Hillier, L.W., Richards, S., et al. (2005). Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Research 15, 1034–1050. 10.1101/gr.3715005."/>
  <meta name="citation_reference" content="Singh, R., and Reddy, R. (1989). Gamma-monomethyl phosphate: a cap structure in spliceosomal U6 small nuclear RNA. Proc Natl Acad Sci U S A 86, 8280–8283. 10.1073/pnas.86.21.8280."/>
  <meta name="citation_reference" content="Talhouarne, G.J.S., and Gall, J.G. (2018). Lariat intronic RNAs in the cytoplasm of vertebrate cells. Proc. Natl. Acad. Sci. U.S.A. 115, E7970–E7977. 10.1073/pnas.1808816115."/>
  <meta name="citation_reference" content="The ENCODE Project Consortium (2012). An integrated encyclopedia of DNA elements in the human genome. Nature 489, 57–74. 10.1038/nature11247."/>
  <meta name="citation_reference" content="The UniProt Consortium (2018). UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506–D515. 10.1093/nar/gky1049."/>
  <meta name="citation_reference" content="citation_pages=327-368;citation_year=2006"/>
  <meta name="citation_reference" content="Van Nostrand, E.L., Pratt, G.A., Shishkin, A.A., Gelboin-Burkhart, C., Fang, M.Y., Sundararaman, B., Blue, S.M., Nguyen, T.B., Surka, C., Elkins, K., et al. (2016). Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP). Nat. Methods 13, 508–514. 10.1038/nmeth.3810."/>
  <meta name="citation_reference" content="Van Nostrand, E.L., Pratt, G.A., Yee, B.A., Wheeler, E.C., Blue, S.M., Mueller, J., Park, S.S., Garcia, K.E., Gelboin-Burkhart, C., Nguyen, T.B., et al. (2020). Principles of RNA processing from analysis of enhanced CLIP maps for 150 RNA binding proteins. Genome Biol. 21, 90. 10.1186/s13059-020-01982-9."/>
  <meta name="citation_reference" content="Vohhodina, J., Barros, E.M., Savage, A.L., Liberante, F.G., Manti, L., Bankhead, P., Cosgrove, N., Madden, A.F., Harkin, D.P., and Savage, K.I. (2017). The RNA processing factors THRAP3 and BCLAF1 promote the DNA damage response through selective mRNA splicing and nuclear export. Nucleic Acids Res 45, 12816–12833. 10.1093/nar/gkx1046."/>
  <meta name="citation_reference" content="Wang, C., Query, C.C., and Meier, U.T. (2002). Immunopurified small nucleolar ribonucleoprotein particles pseudouridylate rRNA independently of their association with phosphorylated Nopp140. Mol Cell Biol 22, 8457–8466. 10.1128/mcb.22.24.8457-8466.2002."/>
  <meta name="citation_reference" content="Wen, J., Ladewig, E., Shenker, S., Mohammed, J., and Lai, E.C. (2015). Analysis of nearly one thousand mammalian mirtrons reveals novel features of Dicer substrates. PLoS Computat. Biol. 11, e1004441. 10.1371/journal.pcbi.1004441."/>
  <meta name="citation_reference" content="Wilkinson, M.E., Charenton, C., and Nagai, K. (2020). RNA splicing by the spliceosome. Annu. Rev. Biochem. 89, 359–388. 10.1146/annurev-biochem-091719-064225."/>
  <meta name="citation_reference" content="Xu, H., Yao, J., Wu, D.C., and Lambowitz, A.M. (2019). Improved TGIRT-seq methods for comprehensive transcriptome profiling with decreased adapter dimer formation and bias correction. Sci. Rep. 9, 7953. 10.1038/s41598-019-44457-z."/>
  <meta name="citation_reference" content="Yao, J., Wu, D.C., Nottingham, R.M., and Lambowitz, A.M. (2020). Identification of protein-protected mRNA fragments and structured excised intron RNAs in human plasma by TGIRT-seq peak calling. eLife 9, e60743. 10.7554/eLife.60743."/>
  <meta name="citation_reference" content="Youn, J.-Y., Dyakov, B.J.A., Zhang, J., Knight, J.D.R., Vernon, R.M., Forman-Kay, J.D., and Gingras, A.-C. (2019). Properties of stress granule and P-body proteomes. Mol. Cell 76, 286–294. https://doi.org/10.1016/j.molcel.2019.09.014."/>
  <meta name="citation_reference" content="Zhang, Y., Zhang, X.-O., Chen, T., Xiang, J.-F., Yin, Q.-F., Xing, Y.-H., Zhu, S., Yang, L., and Chen, L.-L. (2013). Circular intronic long noncoding RNAs. Mol. Cell 51, 792–806. 10.1016/j.molcel.2013.08.017."/>
  <meta name="citation_reference" content="Zhao, M., Kim, P., Mitra, R., Zhao, J., and Zhao, Z. (2016). TSGene 2.0: an updated literature-based knowledgebase for tumor suppressor genes. Nucleic Acids Res. 44, D1023–D1031. 10.1093/nar/gkv1268."/>
  <meta name="twitter:title" content="Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="By using TGIRT-seq, we identified >8,500 short full-length excised linear intron (FLEXI) RNAs in human cells. Subsets of FLEXIs accumulated in a cell-type specific manner, and ∼200 corresponded to agotrons or mirtrons or encoded snoRNAs. Analysis of CLIP-seq datasets identified potential interactions between FLEXIs and >100 different RNA-binding proteins (RBPs), 53 of which had binding sites in ≥30 different FLEXIs. In addition to proteins that function in RNA splicing, these 53 RBPs included transcription factors, chromatin remodeling proteins, and cellular growth regulators that impacted FLEXI host gene alternative splicing and/or mRNA levels in knockdown datasets. We computationally identified six groups of RBPs whose binding sites were enriched in different subsets of FLEXIs: AGO1-4 and DICER associated with agotrons and mirtrons; AATF, DKC1, NOLCI, and SMNDC1 associated with snoRNA-encoding FLEXIs; two different combinations of alternative splicing factors found in stress granules; and two novel RBP-intron combinations, one including LARP4 and PABC4, which function together in the cytoplasm to regulate ribosomal protein translation. Our results suggest a model in which proteins involved in transcriptional regulation, alternative splicing, or post-splicing secondary functions bind and stabilize cell-type specific subsets of FLEXIs that perform different biological functions and have potential utility as biomarkers.

### Competing Interest Statement

Thermostable group II intron reverse transcriptase (TGIRT) enzymes and methods for their use are the subject of patents and patent applications that have been licensed by the University of Texas and East Tennessee State University to InGex, LLC. A.M.L., some former and present members of the Lambowitz laboratory, and the University of Texas are minority equity holders in InGex and receive royalty payments from the sale of TGIRT enzymes and kits and from the sublicensing of intellectual property by InGex to other companies. A.M.L., J.Y., H.X. and D.C.W. are inventors on a patent application filed by the University of Texas at Austin for the use of full-length excised intron RNAs and intron RNA fragments as biomarkers. S.W., E.A.F.K. and M.A. have no competing interests."/>
  <meta name="og-title" property="og:title" content="Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="By using TGIRT-seq, we identified >8,500 short full-length excised linear intron (FLEXI) RNAs in human cells. Subsets of FLEXIs accumulated in a cell-type specific manner, and ∼200 corresponded to agotrons or mirtrons or encoded snoRNAs. Analysis of CLIP-seq datasets identified potential interactions between FLEXIs and >100 different RNA-binding proteins (RBPs), 53 of which had binding sites in ≥30 different FLEXIs. In addition to proteins that function in RNA splicing, these 53 RBPs included transcription factors, chromatin remodeling proteins, and cellular growth regulators that impacted FLEXI host gene alternative splicing and/or mRNA levels in knockdown datasets. We computationally identified six groups of RBPs whose binding sites were enriched in different subsets of FLEXIs: AGO1-4 and DICER associated with agotrons and mirtrons; AATF, DKC1, NOLCI, and SMNDC1 associated with snoRNA-encoding FLEXIs; two different combinations of alternative splicing factors found in stress granules; and two novel RBP-intron combinations, one including LARP4 and PABC4, which function together in the cytoplasm to regulate ribosomal protein translation. Our results suggest a model in which proteins involved in transcriptional regulation, alternative splicing, or post-splicing secondary functions bind and stabilize cell-type specific subsets of FLEXIs that perform different biological functions and have potential utility as biomarkers.

### Competing Interest Statement

Thermostable group II intron reverse transcriptase (TGIRT) enzymes and methods for their use are the subject of patents and patent applications that have been licensed by the University of Texas and East Tennessee State University to InGex, LLC. A.M.L., some former and present members of the Lambowitz laboratory, and the University of Texas are minority equity holders in InGex and receive royalty payments from the sale of TGIRT enzymes and kits and from the sublicensing of intellectual property by InGex to other companies. A.M.L., J.Y., H.X. and D.C.W. are inventors on a patent application filed by the University of Texas at Austin for the use of full-length excised intron RNAs and intron RNA fragments as biomarkers. S.W., E.A.F.K. and M.A. have no competing interests."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-09"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2437873"/>
  <title>
   Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ifNk_KLqii6TePxL-nqJULHd28XmtD4k8WSAXw-O_k4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2437873","apath":"\/biorxiv\/early\/2022\/03\/09\/2020.09.07.285114.atom","pisa":"biorxiv;2020.09.07.285114v4","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.09.07.285114v4"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.09.07.285114v4","title":"Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers","identifier":"node\/2437873"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wnLlB8JIMigWQ\\\\\\\/ztiYaNEDOrf1yMj8rhnuHiYd4qvmT8pk6RESwNVnb1EFJyZiUxW2NTTWuodoPd2VkUwcrDPqPwcFjtwTw59ueOZfAEtbJCgq\\\\\\\/N32ND3IoEu4NPrWH8gksca\\\\\\\/Ltnj9WUo+q8YFTrO3ZVoAVXgaVgvu3wkK5LEJ4twW0z7Z9XCPwcKACWRkMdL9U8nvaRkZtN0Alvea\\\\\\\/AqMkjpTfHXXUQ2HM2UQw6Dzhe4XBh\\\\\\\/gJjFa84abA2CzAv8jH6YhO0y8T\\\\\\\/hdxFTi0OqbuBbfMDXdp3R0ntLogeoqUoHLE1vUdffQ+Kwa0XFgifQPsqCmQVVAW1VKKiZ1rlQCfuo\\\\\\\/Kzjje104BDYnITA95IVe3wWLo8+sIct7k+8bim6rnHFP7bgy6+7oFnNhGzj\\\\\\\/i1HcOLXGEQxdV+4gIKOgnrpl4HKzMTT2Ehg4S3Tl3yYWwC4Uwrr\\\\\\\/+U7rTviAO6fvC6rQ8T8FMbu5+DOZpdHEec0ks9bafekN0VOE1teu6dtLJduGA5cZPiAAtDcGRdUOMzP18U9adTf4RDl7xfNSfyNErbKUXV8v9XW7PncDe3rI5QNDFdu6956P87NggDdpn6D6WDImusZcyJlWfGzR2\\\\\\\/1AQW1S+BSBP72\\\\\\\/6X60LnH+YhHHFb1M0y24GQ2tOV6Y1cO5lXm+qSj2tQXs4Iqqz2MW6ovo2Z4MiRnfzAoZBQ8O41paqhpmh2aQtYsu7wInNpwvXnN8Wkmx9g3EAWLRwU9NwJdz6enCa+fjdx8WJAKr4rpD+k1bxflhB0CumEkcm4TTynKc+YRMLGuDZLI+txN4NKrw+XRCqpQlns05faE7KvEAxDr7HIFeUMrgg812TVV\\\\\\\/zvL2n7+qaeQhlC9bIqa9cHS5r166NEy0Sh3qLzGziIDW4UOEw6A3wXLsPfwcGRW4k9\\\\\\\/lZrpNLtses9fmz33qjnYuZEZb0tAqn1C6kPpjcRmd3DNdYYcM9iho3OdGxPWo2g5ANJKT0P7DA5L\\\\\\\/u558aN6w3KnGOTElU0mcEDQX5v\\\\\\\/HKOnnLWYQyNIL8UXvUYmyxzMRDcv4Jm7mqxsCKMgeEh+qe6bV08YAoPmreiOeHCXCr5GWw6wOpe\\\\\\\/pKA1ILeS\\\\\\\/IIfzYqnCPgbyRoDNLYymPqGtEw4nGsnlHDINQw9lsKmLfUFc\\\\\\\/+TR1\\\\\\\/bLphVubEcXR8WlaGWVFpNOBBgnJaUkxFyt77gnExuZ9URhnOddB9Jd4L6uI8CfYKJme9QW6M2aav7IzyKNJig5cBt4tFOzp2WoBSn73H8yk+ZhW9P84TL9Xcknfm5uMV0QW4jkLdpbc5NSIgJqbwifYyYxZoP8xRKWBT73gfvi1\\\\\\\/qgEPoQGraEjGlaO4zBnBa9di+hs\\\\\\\/qlEMlxkR4IsXxyGUr4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022L5f41K+BFbz8nzaVTzoWdg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022c3a6c864ad854f881c0dad1c76cdd91f6bd2d8e0f65a51f98796775076202927\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.09.07.285114v4.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2437873 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.09.07.285114v4.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_1015013715">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_1015013715" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1641842608" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-8Liz-fn_u2Q0ST8p1ZTDoMjHcJdoEawSrvaWxUAUmMk"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2437873" data-node-nid="2437873" id="node-2437873--21477285591" data-pisa="biorxiv;2020.09.07.285114v4" data-pisa-master="biorxiv;2020.09.07.285114" data-apath="/biorxiv/early/2022/03/09/2020.09.07.285114.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0002-1232-1587" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Jun
                      </span>
                      <span class="nlm-surname">
                       Yao
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1">
                      <a href="http://orcid.org/0000-0003-0434-8074" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Shelby
                      </span>
                      <span class="nlm-surname">
                       Winans
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2">
                      <a href="http://orcid.org/0000-0001-9809-4882" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Hengyi
                      </span>
                      <span class="nlm-surname">
                       Xu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0001-6179-3110" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Douglas C.
                      </span>
                      <span class="nlm-surname">
                       Wu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0003-0088-5315" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Elizabeth A.
                      </span>
                      <span class="nlm-surname">
                       Ferrick-Kiddie
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0002-2552-9168" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Manuel
                      </span>
                      <span class="nlm-surname">
                       Ares
                      </span>
                      <span class="nlm-suffix">
                       Jr.
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0001-6036-2423" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Alan M.
                      </span>
                      <span class="nlm-surname">
                       Lambowitz
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.09.07.285114
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2437873--21477285591" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Jun Yao
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Jun%2BYao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Yao%20J&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AJun%2BYao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-1232-1587" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Jun Yao
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Shelby Winans
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Shelby%2BWinans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Winans%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AShelby%2BWinans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0434-8074" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Shelby Winans
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Hengyi Xu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Hengyi%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Xu%20H&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AHengyi%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-9809-4882" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Hengyi Xu
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Douglas C. Wu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Invitae Corporation
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Douglas%2BC.%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wu%20DC&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ADouglas%2BC.%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-6179-3110" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Douglas C. Wu
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Elizabeth A. Ferrick-Kiddie
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Elizabeth%2BA.%2BFerrick-Kiddie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ferrick-Kiddie%20EA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AElizabeth%2BA.%2BFerrick-Kiddie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-0088-5315" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Elizabeth A. Ferrick-Kiddie
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Manuel Ares
                     <span class="nlm-suffix">
                      Jr.
                     </span>
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Department of Molecular, Cell, and Developmental Biology University of California
                       </span>
                       , Santa Cruz Santa Cruz, California 95064
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Manuel%2BAres%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ares%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AManuel%2BAres%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-2552-9168" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Manuel Ares
                       <span class="nlm-suffix">
                        Jr.
                       </span>
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Alan M. Lambowitz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Departments of Molecular Biosciences and Oncology University of Texas at
                       </span>
                       Austin Austin
                       <span class="nlm-institution">
                        TX 78712
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Alan%2BM.%2BLambowitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lambowitz%20AM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAlan%2BM.%2BLambowitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-6036-2423" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Alan M. Lambowitz
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lambowitz@austin.utexas.edu" class="" data-icon-position="" data-hide-link-title="0">
                        lambowitz@austin.utexas.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.09.07.285114v4" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.07.285114v4.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.07.285114v4.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.07.285114v4.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.09.07.285114v4.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.09.07.285114v4.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2437873" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2437873/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             ABSTRACT
                            </h2>
                            <p id="p-2">
                             By using TGIRT-seq, we identified >8,500 short full-length excised linear intron (FLEXI) RNAs in human cells. Subsets of FLEXIs accumulated in a cell-type specific manner, and ∼200 corresponded to agotrons or mirtrons or encoded snoRNAs. Analysis of CLIP-seq datasets identified potential interactions between FLEXIs and >100 different RNA-binding proteins (RBPs), 53 of which had binding sites in ≥30 different FLEXIs. In addition to proteins that function in RNA splicing, these 53 RBPs included transcription factors, chromatin remodeling proteins, and cellular growth regulators that impacted FLEXI host gene alternative splicing and/or mRNA levels in knockdown datasets. We computationally identified six groups of RBPs whose binding sites were enriched in different subsets of FLEXIs: AGO1-4 and DICER associated with agotrons and mirtrons; AATF, DKC1, NOLCI, and SMNDC1 associated with snoRNA-encoding FLEXIs; two different combinations of alternative splicing factors found in stress granules; and two novel RBP-intron combinations, one including LARP4 and PABC4, which function together in the cytoplasm to regulate ribosomal protein translation. Our results suggest a model in which proteins involved in transcriptional regulation, alternative splicing, or post-splicing secondary functions bind and stabilize cell-type specific subsets of FLEXIs that perform different biological functions and have potential utility as biomarkers.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             INTRODUCTION
                            </h2>
                            <p id="p-4">
                             Most protein-coding genes in eukaryotes consist of coding regions (exons) separated by non-coding regions (introns), which are removed by RNA splicing to produce mRNAs. RNA splicing is performed by the spliceosome, a large ribonucleoprotein complex that catalyzes transesterification reactions yielding ligated exons and an excised intron lariat RNA, whose 5’ end is linked to a branch-point nucleotide near its 3’ end by a 2’,5’-phosphodiester bond (
                             <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">
                              Wilkinson et al., 2020
                             </a>
                             ). After splicing, this bond is typically hydrolyzed by a dedicated debranching enzyme (DBR1) to produce a linear intron RNA, which is rapidly degraded by cellular ribonucleases (
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              Chapman and Boeke, 1991
                             </a>
                             ). In a few cases, excised intron RNAs persist after excision, either as branched circular RNAs (lariats whose tails have been removed) or as unbranched linear RNAs, with some found to contribute to cellular or viral regulatory processes (
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              Farrell et al., 1991
                             </a>
                             ;
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              Gardner et al., 2012
                             </a>
                             ;
                             <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                              Kulesza and Shenk, 2006
                             </a>
                             ;
                             <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                              Morgan et al., 2019
                             </a>
                             ;
                             <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                              Moss and Steitz, 2013
                             </a>
                             ;
                             <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                              Pek et al., 2015
                             </a>
                             ;
                             <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                              Saini et al., 2019
                             </a>
                             ;
                             <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                              Talhouarne and Gall, 2018
                             </a>
                             ;
                             <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">
                              Zhang et al., 2013
                             </a>
                             ). The latter include a group of yeast introns that accumulate as unbranched linear RNAs and contribute to cell growth regulation and stress responses, possibly by sequestering spliceosomal proteins in stationary phase or other stress conditions (
                             <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                              Morgan et al., 2019
                             </a>
                             ;
                             <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                              Parenteau et al., 2019
                             </a>
                             ). Other examples are mirtrons, structured excised intron RNAs that are debranched by DBR1 and processed by DICER into miRNAs (
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              Berezikov et al., 2007
                             </a>
                             ;
                             <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                              Okamura et al., 2007
                             </a>
                             ;
                             <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                              Ruby et al., 2007
                             </a>
                             ), and agotrons, structured excised linear intron RNAs that bind AGO2 and can repress target mRNAs in a miRNA-like manner (
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              Hansen et al., 2016
                             </a>
                             ).
                            </p>
                            <p id="p-5">
                             Recently, we identified 44 different short (≤300 nt) full-length excised linear intron (FLEXI) RNAs in human plasma by using Thermostable Group II Intron Reverse Transcriptase sequencing (TGIRT-seq), which gives continuous end-to-end sequence reads of structured RNAs (
                             <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">
                              Yao et al., 2020
                             </a>
                             ). FLEXI RNAs were identified by peak calling as discrete blocks of continuous full- length intron reads that began and ended within 3 nt of the annotated splice sites. Subsets of plasma FLEXIs corresponded to annotated agotrons or pre-miRNAs of annotated mirtrons (denoted mirtron pre-miRNAs) (
                             <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">
                              Yao et al., 2020
                             </a>
                             ). Almost all (>95%) of the plasma FLEXIs had stable predicted secondary structures (≤-30 kcal/mol), and ∼50% contained an annotated binding site for one or more RNA-binding proteins (RBPs), which could afford protection from highly active RNases in plasma. These findings suggested that FLEXI RNAs might have utility as stable RNA biomarkers in liquid biopsies (
                             <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">
                              Yao et al., 2020
                             </a>
                             ).
                            </p>
                            <p id="p-6">
                             Despite the precedents of agotrons and mirtrons, human excised linear intron RNAs have remained largely unexplored. Here, we used TGIRT-seq to search for FLEXIs in different human cells and tissues. We identified >8,500 different FLEXI RNAs, subsets of which accumulated in a cell-type specific manner and have potential utility as biomarkers. The identified FLEXIs included agotrons, mirtron pre-miRNAs, and snoRNA-encoding introns, as well computationally identified subsets that were enriched in binding sites for transcription factors, chromatin remodeling proteins and proteins that regulate cell growth and different steps in gene expression. Our findings suggest novel biological functions for FLEXI RNAs and provide a road map for further analyses of FLEXI-RBP interactions.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             RESULTS and DISCUSSION
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Identification of FLEXI RNAs in cellular RNA preparations
                             </h3>
                             <p id="p-7">
                              A search of the human genome (Ensembl GRCh38 Release 93 annotations;
                              <a href="http://www.en-sembl.org">
                               http://www.en-sembl.org
                              </a>
                              ) identified 51,645 short introns (≤300 nt) in 12,020 different genes that could potentially give rise to FLEXI RNAs. To determine which short introns could be detected as FLEXI RNAs in human cells, we carried out TGIRT-seq of rRNA-depleted, unfragmented HEK-293T, K562, and HeLa S3 whole-cell RNAs, as well as Universal Human Reference RNA (UHRR), a commercial mixture of total RNAs from ten human cell lines (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ). TGIRT-seq is well suited for the detection of excised linear intron RNAs as the high processivity and strand displacement activity of TGIRTs together with reverse transcription at elevated temperatures enable continuous full-length sequence reads of structured small non-coding RNAs (sncRNAs), which are intractable to reverse transcription by the retroviral RTs used in other RNA-seq methods (
                              <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                               Katibah et al., 2014
                              </a>
                              ;
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               Qin et al., 2016
                              </a>
                              ;
                              <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                               Shurtleff et al., 2017
                              </a>
                              ). In the TGIRT-seq datasets of unfragmented cellular RNAs obtained for this study, most of the reads corresponded to full-length mature tRNAs, Y RNAs, snoRNAs, and snRNAs (
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig. S1
                              </a>
                              and S2). Further, because the variety of TGIRT enzyme used in this study has difficulty reading through long stretches of poly(A), reads from mature polyadenylated mRNAs comprised a low percentage of total reads (0.7-5.3%;
                              <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                               Fig. S1
                              </a>
                              ), leading to the enrichment of reads mapping to excised intron RNAs.
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F9.large.jpg?width=800&height=600&carousel=1" title="TGIRT-seq of rRNA-depleted unfragmented cellular RNAs. (A) Stacked bar graphs showing the percentage of reads mapping to different categories of annotated genomic features in combined datasets for the indicated samples. Genomic features follow Ensembl GRCh38 Release 93 annotations. rRNA includes cellular and mitochondrial (Mt) rRNAs, and protein coding includes all transcripts of protein-coding genes from both the nuclear and Mt genomes. (B) Stacked bar graphs showing the percentage of bases mapping to different regions of the sense strand of protein-coding genes. CDS, coding sequences; intergenic, regions upstream or downstream of transcription start and stop sites annotated in RefSeq; intron, intronic regions; UTR, 5’- or 3’-untranslated regions. Healthy and Cancer indicate tumor and neighboring healthy tissues from breast cancer patients A and B (see Fig. 8)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">TGIRT-seq of rRNA-depleted unfragmented cellular RNAs.</span> (<em>A</em>) Stacked bar graphs showing the percentage of reads mapping to different categories of annotated genomic features in combined datasets for the indicated samples. Genomic features follow Ensembl GRCh38 Release 93 annotations. rRNA includes cellular and mitochondrial (Mt) rRNAs, and protein coding includes all transcripts of protein-coding genes from both the nuclear and Mt genomes. (<em>B</em>) Stacked bar graphs showing the percentage of bases mapping to different regions of the sense strand of protein-coding genes. CDS, coding sequences; intergenic, regions upstream or downstream of transcription start and stop sites annotated in RefSeq; intron, intronic regions; UTR, 5’- or 3’-untranslated regions. Healthy and Cancer indicate tumor and neighboring healthy tissues from breast cancer patients A and B (see Fig. 8).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F9.medium.gif" width="440" height="337"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F9.medium.gif" width="440" height="337"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure S1.
                               </span>
                               <span class="caption-title">
                                TGIRT-seq of rRNA-depleted unfragmented cellular RNAs.
                               </span>
                               <p id="p-60" class="first-child">
                                (A) Stacked bar graphs showing the percentage of reads mapping to different categories of annotated genomic features in combined datasets for the indicated samples. Genomic features follow Ensembl GRCh38 Release 93 annotations. rRNA includes cellular and mitochondrial (Mt) rRNAs, and protein coding includes all transcripts of protein-coding genes from both the nuclear and Mt genomes. (B) Stacked bar graphs showing the percentage of bases mapping to different regions of the sense strand of protein-coding genes. CDS, coding sequences; intergenic, regions upstream or downstream of transcription start and stop sites annotated in RefSeq; intron, intronic regions; UTR, 5’- or 3’-untranslated regions. Healthy and Cancer indicate tumor and neighboring healthy tissues from breast cancer patients A and B (see
                                <a id="xref-fig-8-7" class="xref-fig" href="#F8">
                                 Fig. 8
                                </a>
                                ).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F1.large.jpg?width=800&height=600&carousel=1" title="Characteristics of FLEXI RNAs in human cells and plasma. (A) UpSet plots of FLEXI RNAs and their host genes identified in unfragmented RNA preparations from the indicated samples. Different FLEXIs from the same host gene were aggregated into one entry for that gene. (B) Numbers and percentages of identified FLEXIs that correspond to annotated agotrons, pre-miRNA of annotated mirtrons, or encode annotated snoRNAs in each of the samples. “Total” indicates the total number of different FLEXIs in each category in this collection of samples, and GRCh38 indicates the number of annotated short introns (≤300 nt) in each category in the GRCh38 human genome reference sequence. “Agotron and Mirtron” indicates the number of FLEXIs annotated as both an agotron and mirtron, and “Agotron or Mirtron” indicates the number and percentages of FLEXIs annotated as either an agotron or mirtron. The number and percentages of FLEXIs that contain an embedded snoRNAs and the number of those snoRNAs that are small Cajal body-specific snoRNAs (scaRNAs) are also indicated. (C) 5’- and 3’-splice site (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of FLEXIs corresponding to GC-AG and GU-AG U2-type spliceosomal introns in a merged dataset for the four cellular RNA samples compared to a literature consensus for human U2-type spliceosomal introns (Gao et al., 2008; Sheth et al., 2006) and to consensus sequences for other Ensemble GRCh38-annotated U2-type short (≤300 nt) and long (>300 nt) introns. The number of introns for each consensus sequence is indicated to the right. (D) Density distribution plots of length, GC content, and minimum free energy (MFE) for the most stable RNA secondary structure predicted by RNAfold for FLEXIs detected in a merged dataset for the four cellular RNA samples (black) or human plasma (red) compared to other GRCh38-annotated short introns (≤300 nt, dashed gray line). (E) Density distribution plots of the abundance of different categories of FLEXIs in different cellular RNA samples color coded as indicated below the plots. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (gray, arrows) in the same samples are shown for comparison. Red dotted lines separate low abundance FLEXIs (" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Characteristics of FLEXI RNAs in human cells and plasma.</span> (<em>A</em>) UpSet plots of FLEXI RNAs and their host genes identified in unfragmented RNA preparations from the indicated samples. Different FLEXIs from the same host gene were aggregated into one entry for that gene. <em>(B</em>) Numbers and percentages of identified FLEXIs that correspond to annotated agotrons, pre-miRNA of annotated mirtrons, or encode annotated snoRNAs in each of the samples. “Total” indicates the total number of different FLEXIs in each category in this collection of samples, and GRCh38 indicates the number of annotated short introns (≤300 nt) in each category in the GRCh38 human genome reference sequence. “Agotron and Mirtron” indicates the number of FLEXIs annotated as both an agotron and mirtron, and “Agotron or Mirtron” indicates the number and percentages of FLEXIs annotated as either an agotron or mirtron. The number and percentages of FLEXIs that contain an embedded snoRNAs and the number of those snoRNAs that are small Cajal body-specific snoRNAs (scaRNAs) are also indicated. <em>(C</em>) 5’- and 3’-splice site (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of FLEXIs corresponding to GC-AG and GU-AG U2-type spliceosomal introns in a merged dataset for the four cellular RNA samples compared to a literature consensus for human U2-type spliceosomal introns (Gao et al., 2008; Sheth et al., 2006) and to consensus sequences for other Ensemble GRCh38-annotated U2-type short (≤300 nt) and long (>300 nt) introns. The number of introns for each consensus sequence is indicated to the right. (<em>D</em>) Density distribution plots of length, GC content, and minimum free energy (MFE) for the most stable RNA secondary structure predicted by RNAfold for FLEXIs detected in a merged dataset for the four cellular RNA samples (black) or human plasma (red) compared to other GRCh38-annotated short introns (≤300 nt, dashed gray line). (<em>E</em>) Density distribution plots of the abundance of different categories of FLEXIs in different cellular RNA samples color coded as indicated below the plots. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (gray, arrows) in the same samples are shown for comparison. Red dotted lines separate low abundance FLEXIs (<0.002 RPM) from higher abundance FLEXIs (≥0.002 RPM). (<em>F</em>) Density distribution plots of phastCons scores of different categories of FLEXIs detected in the cellular and plasma RNA samples compared to other annotated short introns (≤300 nt). PhastCons scores were the average score across all intron bases calculated from multiple sequences alignment of 27 primates, including humans plus mouse, dog, and armadillo.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F1.medium.gif" width="417" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F1.medium.gif" width="417" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <span class="caption-title">
                                Characteristics of FLEXI RNAs in human cells and plasma.
                               </span>
                               <p id="p-8" class="first-child">
                                (A) UpSet plots of FLEXI RNAs and their host genes identified in unfragmented RNA preparations from the indicated samples. Different FLEXIs from the same host gene were aggregated into one entry for that gene. (B) Numbers and percentages of identified FLEXIs that correspond to annotated agotrons, pre-miRNA of annotated mirtrons, or encode annotated snoRNAs in each of the samples. “Total” indicates the total number of different FLEXIs in each category in this collection of samples, and GRCh38 indicates the number of annotated short introns (≤300 nt) in each category in the GRCh38 human genome reference sequence. “Agotron and Mirtron” indicates the number of FLEXIs annotated as both an agotron and mirtron, and “Agotron or Mirtron” indicates the number and percentages of FLEXIs annotated as either an agotron or mirtron. The number and percentages of FLEXIs that contain an embedded snoRNAs and the number of those snoRNAs that are small Cajal body-specific snoRNAs (scaRNAs) are also indicated. (C) 5’- and 3’-splice site (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of FLEXIs corresponding to GC-AG and GU-AG U2-type spliceosomal introns in a merged dataset for the four cellular RNA samples compared to a literature consensus for human U2-type spliceosomal introns (
                                <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                                 Gao et al., 2008
                                </a>
                                ;
                                <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                 Sheth et al., 2006
                                </a>
                                ) and to consensus sequences for other Ensemble GRCh38-annotated U2-type short (≤300 nt) and long (>300 nt) introns. The number of introns for each consensus sequence is indicated to the right. (D) Density distribution plots of length, GC content, and minimum free energy (MFE) for the most stable RNA secondary structure predicted by RNAfold for FLEXIs detected in a merged dataset for the four cellular RNA samples (black) or human plasma (red) compared to other GRCh38-annotated short introns (≤300 nt, dashed gray line). (E) Density distribution plots of the abundance of different categories of FLEXIs in different cellular RNA samples color coded as indicated below the plots. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (gray, arrows) in the same samples are shown for comparison. Red dotted lines separate low abundance FLEXIs (<0.002 RPM) from higher abundance FLEXIs (≥0.002 RPM). (F) Density distribution plots of phastCons scores of different categories of FLEXIs detected in the cellular and plasma RNA samples compared to other annotated short introns (≤300 nt). PhastCons scores were the average score across all intron bases calculated from multiple sequences alignment of 27 primates, including humans plus mouse, dog, and armadillo.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-9">
                              To identify FLEXIs in the TGIRT-seq datasets, we compiled the coordinates of all short introns (≤300 nt) in Ensembl GRCh38 Release 93 annotations into a BED file and searched for the intersections with our data. As done previously for FLEXIs in plasma, we defined FLEXI reads as continuous intron reads that began and ended within 3 nt of the annotated splice sites (
                              <a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">
                               Yao et al., 2020
                              </a>
                              ). This definition enabled us to include full-length intron reads with small numbers of extra non-templated nucleotides added by cellular processes or during TGIRT-seq library preparations, but excluded long continuous intron reads that may be biologically significant but did not satisfy the strict definition of a FLEXI at either end. The 300-nt size cutoff was based on the size distribution of FLEXIs found in human plasma (
                              <a id="xref-ref-63-5" class="xref-bibr" href="#ref-63">
                               Yao et al., 2020
                              </a>
                              ) and encompasses 85-92% of the FLEXIs detected in the different RNA preparations, including the most abundant FLEXI RNAs (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Fig. S3A
                              </a>
                              ). The searches were done by using combined datasets obtained from multiple technical replicate libraries totaling 666 to 768 million mapped paired-end reads for each cellular RNA sample, with subsequent comparisons to biological replicates for four of the cell types (
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              ). In addition to human cellular RNAs, we searched for FLEXIs in remapped datasets of human plasma RNAs from healthy individuals (
                              <a id="xref-ref-63-6" class="xref-bibr" href="#ref-63">
                               Yao et al., 2020
                              </a>
                              ).
                             </p>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F10.large.jpg?width=800&height=600&carousel=1" title="Integrative Genomics Viewer (IGV) screenshots showing coverage tracts and read alignments for sncRNAs detected in TGIRT-seq datasets of unfragmented cellular RNAs. The name of the sncRNA is shown at the top with its length indicated in parentheses and the arrow indicating the 5’ to 3’ orientation of the RNA. Coverage tracts (gray) and read alignments for combined technical replicates of each cellular RNA sample type color coded as indicated in the Figure are shown below. Reads were down sampled to a maximum of 100 for display in IGV. Gray in the coverage tracts indicates bases in the read that match the reference base, and other colors indicate bases in the read that do not match the reference base (red, thymidine; green, adenosine; blue, cytidine; and brown, guanosine). Misincorporation at known sites of tRNA post-transcriptionally modified bases are highlighted in the alignments: m1A58: 1-methyladenosine at position 58; I: inosine. CCA indicates the post-transcriptionally added 3’ CCA sequence of tRNAs mapped against a reference set of mature tRNA sequences." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Integrative Genomics Viewer (IGV) screenshots showing coverage tracts and read alignments for sncRNAs detected in TGIRT-seq datasets of unfragmented cellular RNAs. The name of the sncRNA is shown at the top with its length indicated in parentheses and the arrow indicating the 5’ to 3’ orientation of the RNA. Coverage tracts (gray) and read alignments for combined technical replicates of each cellular RNA sample type color coded as indicated in the Figure are shown below. Reads were down sampled to a maximum of 100 for display in IGV. Gray in the coverage tracts indicates bases in the read that match the reference base, and other colors indicate bases in the read that do not match the reference base (red, thymidine; green, adenosine; blue, cytidine; and brown, guanosine). Misincorporation at known sites of tRNA post-transcriptionally modified bases are highlighted in the alignments: m<sup>1</sup>A58: 1-methyladenosine at position 58; I: inosine. CCA indicates the post-transcriptionally added 3’ CCA sequence of tRNAs mapped against a reference set of mature tRNA sequences.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F10.medium.gif" width="300" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F10.medium.gif" width="300" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2.
                               </span>
                               <p id="p-61" class="first-child">
                                Integrative Genomics Viewer (IGV) screenshots showing coverage tracts and read alignments for sncRNAs detected in TGIRT-seq datasets of unfragmented cellular RNAs. The name of the sncRNA is shown at the top with its length indicated in parentheses and the arrow indicating the 5’ to 3’ orientation of the RNA. Coverage tracts (gray) and read alignments for combined technical replicates of each cellular RNA sample type color coded as indicated in the Figure are shown below. Reads were down sampled to a maximum of 100 for display in IGV. Gray in the coverage tracts indicates bases in the read that match the reference base, and other colors indicate bases in the read that do not match the reference base (red, thymidine; green, adenosine; blue, cytidine; and brown, guanosine). Misincorporation at known sites of tRNA post-transcriptionally modified bases are highlighted in the alignments: m
                                <sup>
                                 1
                                </sup>
                                A58: 1-methyladenosine at position 58; I: inosine. CCA indicates the post-transcriptionally added 3’ CCA sequence of tRNAs mapped against a reference set of mature tRNA sequences.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F11.large.jpg?width=800&height=600&carousel=1" title="Characteristics of FLEXI RNAs. (A) Density plots showing the distribution of length of FLEXIs (≤300 nt; red) and long FLEXIs (>300 nt; blue) in combined datasets for each of the four cellular RNA samples. (B) Density plots showing the length distribution of all reads of any length mapping within different categories of FLEXI RNAs in combined datasets for each of the four cellular RNA samples compared to those mapping to other Ensemble GRCh38-annotated short introns (≤300 nt) in the same combined datasets. Separate plots color coded by category as shown in the Figure are overlaid for FLEXI RNAs containing annotated binding sites for AGO1-4, DICER, five core spliceosome proteins (AQR, BUD13, EFTUD2, PRPF8 and SF3B4), other annotated RBPs, FLEXIs without (w/o) annotated RBP-binding sites, and other GRCh38annotated short introns (≤300 nt). Relative length was calculated from the read spans of TGIRTseq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs were removed prior to calculating the distributions to avoid interference from mature snoRNA reads. (C) Three-dimensional bar chart showing the percentage of FLEXI RNA reads ending at different positions around the intron-exon junctions in each of combined datasets for each of the four cellular RNA samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Characteristics of FLEXI RNAs.</span> (<em>A</em>) Density plots showing the distribution of length of FLEXIs (≤300 nt; red) and long FLEXIs (>300 nt; blue) in combined datasets for each of the four cellular RNA samples. (<em>B</em>) Density plots showing the length distribution of all reads of any length mapping within different categories of FLEXI RNAs in combined datasets for each of the four cellular RNA samples compared to those mapping to other Ensemble GRCh38-annotated short introns (≤300 nt) in the same combined datasets. Separate plots color coded by category as shown in the Figure are overlaid for FLEXI RNAs containing annotated binding sites for AGO1-4, DICER, five core spliceosome proteins (AQR, BUD13, EFTUD2, PRPF8 and SF3B4), other annotated RBPs, FLEXIs without (w/o) annotated RBP-binding sites, and other GRCh38annotated short introns (≤300 nt). Relative length was calculated from the read spans of TGIRTseq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs were removed prior to calculating the distributions to avoid interference from mature snoRNA reads. <em>(C)</em> Three-dimensional bar chart showing the percentage of FLEXI RNA reads ending at different positions around the intron-exon junctions in each of combined datasets for each of the four cellular RNA samples.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F11.medium.gif" width="388" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F11.medium.gif" width="388" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S3.
                               </span>
                               <span class="caption-title">
                                Characteristics of FLEXI RNAs.
                               </span>
                               <p id="p-62" class="first-child">
                                (A) Density plots showing the distribution of length of FLEXIs (≤300 nt; red) and long FLEXIs (>300 nt; blue) in combined datasets for each of the four cellular RNA samples. (B) Density plots showing the length distribution of all reads of any length mapping within different categories of FLEXI RNAs in combined datasets for each of the four cellular RNA samples compared to those mapping to other Ensemble GRCh38-annotated short introns (≤300 nt) in the same combined datasets. Separate plots color coded by category as shown in the Figure are overlaid for FLEXI RNAs containing annotated binding sites for AGO1-4, DICER, five core spliceosome proteins (AQR, BUD13, EFTUD2, PRPF8 and SF3B4), other annotated RBPs, FLEXIs without (w/o) annotated RBP-binding sites, and other GRCh38annotated short introns (≤300 nt). Relative length was calculated from the read spans of TGIRTseq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs were removed prior to calculating the distributions to avoid interference from mature snoRNA reads. (C) Three-dimensional bar chart showing the percentage of FLEXI RNA reads ending at different positions around the intron-exon junctions in each of combined datasets for each of the four cellular RNA samples.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2442229/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup last">
                                  <a href="/highwire/markup/2442229/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S1.
                               </span>
                               <span class="caption-title">
                                Summary of datasets obtained in this study
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-10">
                              By using this approach, we identified 8,144 different FLEXI RNAs originating from 3,743 different protein-coding genes, lncRNA genes, or pseudogenes (collectively denoted FLEXI host genes). The distribution of these FLEXI RNAs and their host genes between the different cellular RNA samples and plasma is shown in UpSet plots in
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              . Sixty five of the FLEXIs identified in the four cellular samples and plasma corresponded to annotated agotrons (
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ) and 114 corresponded to pre-miRNAs of an annotated mirtron (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ), both of which were shown previously to be debranched linear intron RNAs (
                              <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                               Berezikov et al., 2007
                              </a>
                              ;
                              <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ). The proportion of FLEXI RNAs corresponding to annotated agotrons or mirtron premRNAs was higher in plasma (15.6%) than in cells (1.5-2.2%; p <1x10
                              <sup>
                               -15
                              </sup>
                              by Fisher’s exact test,
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ), possibly reflecting preferential cellular export and/or greater stability of structured RNAs in plasma (
                              <a id="xref-ref-63-7" class="xref-bibr" href="#ref-63">
                               Yao et al., 2020
                              </a>
                              ). A small number of FLEXI RNAs found in cells but not plasma (43, 0.5% of the total) encoded snoRNAs, all of which were also detected as mature snoRNAs in the same samples (
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ).
                             </p>
                             <p id="p-11">
                              In IGV alignments of reads mapping to a protein-coding gene, FLEXIs stand out from spliced exon and intron-exon junction reads as columns of continuous intron reads that began and ended within 3 nt of the annotated splice sites (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              ). The IGV alignments for 4I_JUP, an annotated agotron (
                              <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ), provide an example of an intron for which a high proportion of intron reads (94%) satisfied the conservative definition of a FLEXI in HeLa S3 cells, but had 3’ truncations that resulted in lower proportions of the reads (2-3%) being counted as FLEXIs in the other three cellular RNA samples (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              , top left). Some FLEXIs showed a propensity for indels at runs of U residues downstream of the putative branch-point nucleotide, but no indication of impediments or misincorporations at the putative branch-point nucleotide expected for a lariat RNA (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2A
                              </a>
                              and
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              ).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F2.large.jpg?width=800&height=600&carousel=1" title="FLEXI RNAs are detected by TGIRT-seq as continuous full-length, end-to-end reads of excised intron RNAs. (A) IGV screenshots showing read alignments for FLEXI RNAs in different cellular RNA samples. Gene names are at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing gene annotations (exons, thick bars; introns, thin lines) with read alignments shown below. Reads are color coded by cell type as shown in the Figure. The length of the FLEXI that is the focus of each panel is shown above the intron annotation, and the number of reads assigned to that FLEXI (continuous reads that begin and end within 3 nt of the annotated 5’- and 3’-splice sites) compared to total reads for that intron are shown within the read alignments. Reads were down sampled to a maximum of 100 for display in IGV. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. (B) Histogram of the length distribution of reads of any length mapping within introns identified as FLEXI RNAs in a merged dataset for the four cellular RNA samples (red) compared to those mapping within other Ensemble GRCh38-annotated short introns (≤300 nt) in the same samples (blue). Relative length was calculated from the read spans of TGIRT-seq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs or scaRNAs were removed for the comparisons to avoid interference from mature snoRNA reads. (C) Three-dimensional bar chart showing the percentage of FLEXI RNA reads in a merged dataset for the cellular RNA samples ending at different positions around the intron-exon junctions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">FLEXI RNAs are detected by TGIRT-seq as continuous full-length, end-to-end reads of excised intron RNAs.</span> <em>(A)</em> IGV screenshots showing read alignments for FLEXI RNAs in different cellular RNA samples. Gene names are at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing gene annotations (exons, thick bars; introns, thin lines) with read alignments shown below. Reads are color coded by cell type as shown in the Figure. The length of the FLEXI that is the focus of each panel is shown above the intron annotation, and the number of reads assigned to that FLEXI (continuous reads that begin and end within 3 nt of the annotated 5’- and 3’-splice sites) compared to total reads for that intron are shown within the read alignments. Reads were down sampled to a maximum of 100 for display in IGV. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. <em>(B)</em> Histogram of the length distribution of reads of any length mapping within introns identified as FLEXI RNAs in a merged dataset for the four cellular RNA samples (red) compared to those mapping within other Ensemble GRCh38-annotated short introns (≤300 nt) in the same samples (blue). Relative length was calculated from the read spans of TGIRT-seq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs or scaRNAs were removed for the comparisons to avoid interference from mature snoRNA reads. <em>(C)</em> Three-dimensional bar chart showing the percentage of FLEXI RNA reads in a merged dataset for the cellular RNA samples ending at different positions around the intron-exon junctions.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F2.medium.gif" width="440" height="390"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F2.medium.gif" width="440" height="390"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <span class="caption-title">
                                FLEXI RNAs are detected by TGIRT-seq as continuous full-length, end-to-end reads of excised intron RNAs.
                               </span>
                               <p id="p-12" class="first-child">
                                (A) IGV screenshots showing read alignments for FLEXI RNAs in different cellular RNA samples. Gene names are at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing gene annotations (exons, thick bars; introns, thin lines) with read alignments shown below. Reads are color coded by cell type as shown in the Figure. The length of the FLEXI that is the focus of each panel is shown above the intron annotation, and the number of reads assigned to that FLEXI (continuous reads that begin and end within 3 nt of the annotated 5’- and 3’-splice sites) compared to total reads for that intron are shown within the read alignments. Reads were down sampled to a maximum of 100 for display in IGV. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. (B) Histogram of the length distribution of reads of any length mapping within introns identified as FLEXI RNAs in a merged dataset for the four cellular RNA samples (red) compared to those mapping within other Ensemble GRCh38-annotated short introns (≤300 nt) in the same samples (blue). Relative length was calculated from the read spans of TGIRT-seq reads normalized to the length of the corresponding intron. Introns encoding embedded snoRNAs or scaRNAs were removed for the comparisons to avoid interference from mature snoRNA reads. (C) Three-dimensional bar chart showing the percentage of FLEXI RNA reads in a merged dataset for the cellular RNA samples ending at different positions around the intron-exon junctions.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F3.large.jpg?width=800&height=600&carousel=1" title="FLEXI RNA profiles are cell-type specific due to differences in host gene transcription, alternative splicing, and intron RNA turnover. (A) Scatter plots comparing log2- transformed RPM of FLEXI RNA reads (top row) and all reads that mapped to FLEXI host genes (bottom row) in pairwise comparisons of cellular RNA samples. Relatively abundant FLEXIs detected at ≥0.01 RPM in all replicates of one or both of the compared cell types and their host genes are marked in red and blue for the samples compared on the y-axis and x-axis, respectively. r is the Pearson correlation coefficient (B) IGV screenshots showing examples of cell-type specific differences in FLEXI RNA abundance due to differences in alternative splicing, splicing efficiency and/or differential stability of FLEXI RNAs encoded by the same host gene. Gene maps for different RNA isoforms generated by alternative splicing of FLEXI RNAs are shown at the bottom with highlighted introns arbitrarily named A to F from the 5’ to 3’ end of the gene. Reads for different cellular RNA samples are color coded as indicated at the top right. Reads mapping to exons or non-FLEXI introns were omitted for clarity. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. Reads were down sampled to a maximum of 100 for display in IGV. (C) Characteristics of highly conserved FLEXIs in a merged dataset for the four cellular RNA samples. FLEXIs were divided into three groups based on phastCons scores for comparisons (300 nt). RI indicates FLEXIs that are retained in some annotated alternatively spliced transcripts. In frame RI indicates retained FLEXIs that have reading frames in frame with that of the flanking exons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">FLEXI RNA profiles are cell-type specific due to differences in host gene transcription, alternative splicing, and intron RNA turnover.</span> (<em>A</em>) Scatter plots comparing log<sub>2</sub>- transformed RPM of FLEXI RNA reads (top row) and all reads that mapped to FLEXI host genes (bottom row) in pairwise comparisons of cellular RNA samples. Relatively abundant FLEXIs detected at ≥0.01 RPM in all replicates of one or both of the compared cell types and their host genes are marked in red and blue for the samples compared on the y-axis and x-axis, respectively. <em>r</em> is the Pearson correlation coefficient <em>(B</em>) IGV screenshots showing examples of cell-type specific differences in FLEXI RNA abundance due to differences in alternative splicing, splicing efficiency and/or differential stability of FLEXI RNAs encoded by the same host gene. Gene maps for different RNA isoforms generated by alternative splicing of FLEXI RNAs are shown at the bottom with highlighted introns arbitrarily named A to F from the 5’ to 3’ end of the gene. Reads for different cellular RNA samples are color coded as indicated at the top right. Reads mapping to exons or non-FLEXI introns were omitted for clarity. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. Reads were down sampled to a maximum of 100 for display in IGV. (<em>C</em>) Characteristics of highly conserved FLEXIs in a merged dataset for the four cellular RNA samples. FLEXIs were divided into three groups based on phastCons scores for comparisons (<0.75, ≥0.75, and ≥0.99). The number of FLEXIs in each group is shown to the left, and the numbers and percentages of those FLEXIs in different categories are shown to the right. Alt. Spl. indicates FLEXIs that are alternatively spliced to generate different protein isoforms based on Ensemble GRCh38-annotation. Shared SS indicates FLEXIs that share a 5’- or 3’-splice site with a longer intron (>300 nt). RI indicates FLEXIs that are retained in some annotated alternatively spliced transcripts. In frame RI indicates retained FLEXIs that have reading frames in frame with that of the flanking exons.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F3.medium.gif" width="266" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F3.medium.gif" width="266" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <span class="caption-title">
                                FLEXI RNA profiles are cell-type specific due to differences in host gene transcription, alternative splicing, and intron RNA turnover.
                               </span>
                               <p id="p-13" class="first-child">
                                (A) Scatter plots comparing log
                                <sub>
                                 2
                                </sub>
                                - transformed RPM of FLEXI RNA reads (top row) and all reads that mapped to FLEXI host genes (bottom row) in pairwise comparisons of cellular RNA samples. Relatively abundant FLEXIs detected at ≥0.01 RPM in all replicates of one or both of the compared cell types and their host genes are marked in red and blue for the samples compared on the y-axis and x-axis, respectively. r is the Pearson correlation coefficient (B) IGV screenshots showing examples of cell-type specific differences in FLEXI RNA abundance due to differences in alternative splicing, splicing efficiency and/or differential stability of FLEXI RNAs encoded by the same host gene. Gene maps for different RNA isoforms generated by alternative splicing of FLEXI RNAs are shown at the bottom with highlighted introns arbitrarily named A to F from the 5’ to 3’ end of the gene. Reads for different cellular RNA samples are color coded as indicated at the top right. Reads mapping to exons or non-FLEXI introns were omitted for clarity. Mismatched nucleotides in boxes at the 5’ end of RNA reads are due to non-templated nucleotide addition (NTA) to the 3’ end of cDNAs during TGIRT-seq library preparation. Reads were down sampled to a maximum of 100 for display in IGV. (C) Characteristics of highly conserved FLEXIs in a merged dataset for the four cellular RNA samples. FLEXIs were divided into three groups based on phastCons scores for comparisons (<0.75, ≥0.75, and ≥0.99). The number of FLEXIs in each group is shown to the left, and the numbers and percentages of those FLEXIs in different categories are shown to the right. Alt. Spl. indicates FLEXIs that are alternatively spliced to generate different protein isoforms based on Ensemble GRCh38-annotation. Shared SS indicates FLEXIs that share a 5’- or 3’-splice site with a longer intron (>300 nt). RI indicates FLEXIs that are retained in some annotated alternatively spliced transcripts. In frame RI indicates retained FLEXIs that have reading frames in frame with that of the flanking exons.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-14">
                              A histogram of the length distribution of all reads that mapped within an intron identified as a FLEXI in any of the four cellular RNA samples showed a peak at 90-100% intron length with only a smattering corresponding to shorter fragments (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ). By contrast, reads mapping within other short introns (≤300 nt) annotated in GRCh38 corresponded to heterogeneously sized RNA fragments, as expected for rapid intron turnover after splicing (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ). Further, most of the FLEXI reads began and ended directly at the splice sites (87% for the 5’-splice site and 58% for the 3’-splice site), with <10% extending a short distance into a flanking exon (
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig. 2C
                              </a>
                              ). Similar FLEXI length distributions were seen in each of the four individual cellular RNA sample types, including for different subsets of FLEXIs described further below, except those containing a binding site for DICER, which showed additional discrete peaks corresponding to shorter intron fragments that may be DICER cleavage products (
                              <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                               Fig. S3B
                              </a>
                              and C). RT-qPCR assays for 7 relatively abundant FLEXIs, including an annotated agotron (4I_JUP), confirmed that they were sensitive to digestion by Terminator (Lucigen), a commercial 5’ to 3’ exonuclease that degrades linear RNAs but not circular or lariat RNAs, which have blocked 5’ ends (
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Fig. S4
                              </a>
                              ). Collectively, these findings show that TGIRT-seq can be used to identify large numbers of short full-length excised linear intron RNAs in human cells.
                             </p>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F12.large.jpg?width=800&height=600&carousel=1" title="FLEXI RNAs are susceptible to 5’ to 3’ exonuclease digestion. Unfragmented HEK-293T and UHHR RNAs (500 ng) or three synthetic oligonucleotide controls corresponding to FLEXI 3I_RAN (100 ng; linear with or without a 5’ phosphate or circular generated by T4 RNA Ligase I (New England Biolabs)) were treated with Terminator exonuclease (Lucigen) or mock treated. Reverse transcription with Maxima H reverse transcriptase (ThermoFisher) was then performed using gene-specific primers near the 3’ end of the RNA for 7 different FLEXIs, 3 sncRNAs, and the 3I_RAN FLEXI oligonucleotide controls, the latter using the same primer used for 3I_RAN FLEXI in unfragmented cellular RNA (Table S4). Quantitative PCR (qPCR) was done using SYBR Green with oligonucleotide primers listed in Table S4. Data are reported for three replicates as a percent of FLEXI RNA remaining after exonuclease treatment relative to a mock-treated RNA sample, run in parallel. Fold change was calculated using the delta Ct method. U6, which has a 5’ γ-monomethyl phosphate cap (Singh and Reddy, 1989), was insensitive to the exonuclease treatment, as expected. Linear forms of the oligonucleotide control were sensitive to exonuclease treatment whereas the circular form was less sensitive. Sensitivity of the linear oligonucleotide with a 5’-phosphate could reflect incomplete phosphorylation of the synthetic oligonucleotide or reported off target activity in the standard terminator buffer, Buffer A, which is the recommended buffer in the manufacturer’s instructions for achieving maximal efficiency of Terminator digestion. The sensitivity of one replicate of the circular oligonucleotide to Terminator treatment might reflect incomplete circularization of the RNA used for that replicate. PCR specificity was confirmed by gel extracting UHHR qPCR amplicons followed by TOPOTA cloning and Sanger sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">FLEXI RNAs are susceptible to 5’ to 3’ exonuclease digestion.</span> Unfragmented HEK-293T and UHHR RNAs (500 ng) or three synthetic oligonucleotide controls corresponding to FLEXI 3I_RAN (100 ng; linear with or without a 5’ phosphate or circular generated by T4 RNA Ligase I (New England Biolabs)) were treated with Terminator exonuclease (Lucigen) or mock treated. Reverse transcription with Maxima H reverse transcriptase (ThermoFisher) was then performed using gene-specific primers near the 3’ end of the RNA for 7 different FLEXIs, 3 sncRNAs, and the 3I_RAN FLEXI oligonucleotide controls, the latter using the same primer used for 3I_RAN FLEXI in unfragmented cellular RNA (Table S4). Quantitative PCR (qPCR) was done using SYBR Green with oligonucleotide primers listed in Table S4. Data are reported for three replicates as a percent of FLEXI RNA remaining after exonuclease treatment relative to a mock-treated RNA sample, run in parallel. Fold change was calculated using the delta Ct method. U6, which has a 5’ γ-monomethyl phosphate cap (Singh and Reddy, 1989), was insensitive to the exonuclease treatment, as expected. Linear forms of the oligonucleotide control were sensitive to exonuclease treatment whereas the circular form was less sensitive. Sensitivity of the linear oligonucleotide with a 5’-phosphate could reflect incomplete phosphorylation of the synthetic oligonucleotide or reported off target activity in the standard terminator buffer, Buffer A, which is the recommended buffer in the manufacturer’s instructions for achieving maximal efficiency of Terminator digestion. The sensitivity of one replicate of the circular oligonucleotide to Terminator treatment might reflect incomplete circularization of the RNA used for that replicate. PCR specificity was confirmed by gel extracting UHHR qPCR amplicons followed by TOPOTA cloning and Sanger sequencing.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F12.medium.gif" width="440" height="213"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F12.medium.gif" width="440" height="213"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S4.
                               </span>
                               <span class="caption-title">
                                FLEXI RNAs are susceptible to 5’ to 3’ exonuclease digestion.
                               </span>
                               <p id="p-63" class="first-child">
                                Unfragmented HEK-293T and UHHR RNAs (500 ng) or three synthetic oligonucleotide controls corresponding to FLEXI 3I_RAN (100 ng; linear with or without a 5’ phosphate or circular generated by T4 RNA Ligase I (New England Biolabs)) were treated with Terminator exonuclease (Lucigen) or mock treated. Reverse transcription with Maxima H reverse transcriptase (ThermoFisher) was then performed using gene-specific primers near the 3’ end of the RNA for 7 different FLEXIs, 3 sncRNAs, and the 3I_RAN FLEXI oligonucleotide controls, the latter using the same primer used for 3I_RAN FLEXI in unfragmented cellular RNA (
                                <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">
                                 Table S4
                                </a>
                                ). Quantitative PCR (qPCR) was done using SYBR Green with oligonucleotide primers listed in
                                <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">
                                 Table S4
                                </a>
                                . Data are reported for three replicates as a percent of FLEXI RNA remaining after exonuclease treatment relative to a mock-treated RNA sample, run in parallel. Fold change was calculated using the delta Ct method. U6, which has a 5’ γ-monomethyl phosphate cap (
                                <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                 Singh and Reddy, 1989
                                </a>
                                ), was insensitive to the exonuclease treatment, as expected. Linear forms of the oligonucleotide control were sensitive to exonuclease treatment whereas the circular form was less sensitive. Sensitivity of the linear oligonucleotide with a 5’-phosphate could reflect incomplete phosphorylation of the synthetic oligonucleotide or reported off target activity in the standard terminator buffer, Buffer A, which is the recommended buffer in the manufacturer’s instructions for achieving maximal efficiency of Terminator digestion. The sensitivity of one replicate of the circular oligonucleotide to Terminator treatment might reflect incomplete circularization of the RNA used for that replicate. PCR specificity was confirmed by gel extracting UHHR qPCR amplicons followed by TOPOTA cloning and Sanger sequencing.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              FLEXI RNAs have distinctive structural characteristics
                             </h3>
                             <p id="p-15">
                              Most of the 8,082 FLEXIs identified in the cellular RNA datasets have sequence characteristics of major U2-type spliceosomal introns (8,004 with GU-AG splice sites and 78 with GC-AG splice sites;
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1C
                              </a>
                              ), with only 36 having sequence characteristics of minor U12-type spliceosomal introns (34 with GU-AG and 2 with AU-AC splice sites;
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Fig. S5A
                              </a>
                              ), and 23 having non-canonical splice sites (e.g., AU-AG and AU-AU;
                              <a id="xref-fig-13-2" class="xref-fig" href="#F13">
                               Fig. S5B
                              </a>
                              ) (
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               Burset et al., 2000
                              </a>
                              ;
                              <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">
                               Sheth et al., 2006
                              </a>
                              ). The splice-site and branch-point (BP) consensus sequences of FLEXI RNAs matched literature consensus sequences for human introns (
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               Burset et al., 2000
                              </a>
                              ;
                              <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">
                               Gao et al., 2008
                              </a>
                              ), possibly with a higher propensity for C residues upstream of the branch-point A and 3’-splice site (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1C
                              </a>
                              for U2-type FLEXIs and
                              <a id="xref-fig-13-3" class="xref-fig" href="#F13">
                               Fig. S5A
                              </a>
                              for U12-type FLEXIs) (
                              <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">
                               Gao et al., 2008
                              </a>
                              ). Although hydrolytic splicing, which would directly produce an excised linear intron RNA, cannot be excluded, their largely canonical branch-point sequences suggests that most if not all FLEXIs were excised as lariat RNAs and debranched after splicing, as found previously for mirtron pre-miRNAs and agotrons (
                              <a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ;
                              <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">
                               Okamura et al., 2007
                              </a>
                              ;
                              <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">
                               Ruby et al., 2007
                              </a>
                              ).
                             </p>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F13.large.jpg?width=800&height=600&carousel=1" title="FLEXI RNAs corresponding to U12-type introns or with non-GU-AG 5’- and 3’-splice sites. (A) 5’- and 3’-splice sites (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of GU-AG and AU-AC U12-type FLEXIs detected in combined datasets for each of the four cellular RNA samples (HEK-293T, HeLa S3, K-562, and UHHR) compared to a literature consensus sequence for human U12-type introns (top) (Burset et al., 2000; Gao et al., 2008) and consensus sequences for other Ensemble GRCh38-annotated U12-type short (≤300 nt) and long (>300 nt) introns in the same combined datasets (below). The number of introns for each consensus sequence is indicated to the right of the sequence. (B) IGV screenshots showing examples of coverage tracts and read alignments for FLEXI RNAs having U12-type or non-canonical 5’- and 3’-splice sites. Dinucleotides at the 5’- and 3’-ends of the intron are indicated at the upper left. The host gene name and genomic coordinates of the FLEXI are shown at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing the genomic sequence and gene annotations for different transcript isoforms (exons, thick bars; introns, thin lines). Read alignments for FLEXI RNAs are shown below the tracts and are color coded by sample type as indicated in the key at the upper right of the panel. Reads were down sampled to a maximum of 100 for display in IGV." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">FLEXI RNAs corresponding to U12-type introns or with non-GU-AG 5’- and 3’-splice sites.</span> (<em>A</em>) 5’- and 3’-splice sites (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of GU-AG and AU-AC U12-type FLEXIs detected in combined datasets for each of the four cellular RNA samples (HEK-293T, HeLa S3, K-562, and UHHR) compared to a literature consensus sequence for human U12-type introns (top) (Burset et al., 2000; Gao et al., 2008) and consensus sequences for other Ensemble GRCh38-annotated U12-type short (≤300 nt) and long (>300 nt) introns in the same combined datasets (below). The number of introns for each consensus sequence is indicated to the right of the sequence. (<em>B</em>) IGV screenshots showing examples of coverage tracts and read alignments for FLEXI RNAs having U12-type or non-canonical 5’- and 3’-splice sites. Dinucleotides at the 5’- and 3’-ends of the intron are indicated at the upper left. The host gene name and genomic coordinates of the FLEXI are shown at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing the genomic sequence and gene annotations for different transcript isoforms (exons, thick bars; introns, thin lines). Read alignments for FLEXI RNAs are shown below the tracts and are color coded by sample type as indicated in the key at the upper right of the panel. Reads were down sampled to a maximum of 100 for display in IGV.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F13.medium.gif" width="298" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F13.medium.gif" width="298" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S5.
                               </span>
                               <span class="caption-title">
                                FLEXI RNAs corresponding to U12-type introns or with non-GU-AG 5’- and 3’-splice sites.
                               </span>
                               <p id="p-64" class="first-child">
                                (A) 5’- and 3’-splice sites (5’SS and 3’SS, respectively) and branch-point (BP) consensus sequences of GU-AG and AU-AC U12-type FLEXIs detected in combined datasets for each of the four cellular RNA samples (HEK-293T, HeLa S3, K-562, and UHHR) compared to a literature consensus sequence for human U12-type introns (top) (
                                <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                                 Burset et al., 2000
                                </a>
                                ;
                                <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">
                                 Gao et al., 2008
                                </a>
                                ) and consensus sequences for other Ensemble GRCh38-annotated U12-type short (≤300 nt) and long (>300 nt) introns in the same combined datasets (below). The number of introns for each consensus sequence is indicated to the right of the sequence. (B) IGV screenshots showing examples of coverage tracts and read alignments for FLEXI RNAs having U12-type or non-canonical 5’- and 3’-splice sites. Dinucleotides at the 5’- and 3’-ends of the intron are indicated at the upper left. The host gene name and genomic coordinates of the FLEXI are shown at the top with the arrow below indicating the 5’ to 3’ orientation of the encoded RNA followed by tracts showing the genomic sequence and gene annotations for different transcript isoforms (exons, thick bars; introns, thin lines). Read alignments for FLEXI RNAs are shown below the tracts and are color coded by sample type as indicated in the key at the upper right of the panel. Reads were down sampled to a maximum of 100 for display in IGV.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              We used density plots to compare the cumulative distribution of different properties of the >8,000 FLEXIs detected in the combined TGIRT-seq datasets for human cell lines (K-562, HEK-293T, HeLa S3, and UHHR) and plasma with those of other short introns in the same datasets (
                              <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ). The plots showed that FLEXI RNAs differed significantly from other short introns in being skewed toward higher GC contents and lower minimum free energies (MFEs; ΔG) for the most stable secondary structure predicted by RNAfold (p≤0.001 for both by Wilcoxon signed-rank test;
                              <a id="xref-fig-1-9" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ), features that could contribute to their stability as FLEXIs. These differences were most pronounced for the FLEXIs detected in human plasma, which were a relatively homogeneous subset with peaks at 90-nt length, 70% GC content, and -40 kcal/mole MFE (
                              <a id="xref-fig-1-10" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ). By comparison, the density plots for the FLEXIs detected in cells showed similar peaks but larger shoulders extending to longer lengths, lower GC contents, and less stable predicted secondary structures (≥-25 kcal/mole;
                              <a id="xref-fig-1-11" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ), again suggesting preferential export or greater resistance to plasma nucleases of shorter, more stably structured FLEXI RNAs.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Abundance of FLEXI RNAs
                             </h3>
                             <p id="p-17">
                              Density plots comparing the abundances (RPM) of FLEXIs in the cellular RNA samples (
                              <a id="xref-fig-1-12" class="xref-fig" href="#F1">
                               Fig. 1E
                              </a>
                              ) showed that they could be divided into two categories: one corresponding to low abundance FLEXIs (<0.002 RPM), which likely include transient intron RNA turnover intermediates, and the other corresponding to FLEXI RNAs that accumulate to higher abundance in different cell types (peaks between 0.002 and 0.1 RPM with a tail extending to 6.9 RPM) overlapping the lower end of the abundance distribution for snoRNAs (dashed yellow line). In HeLa S3 cells, the peak of more abundant FLEXIs was predominant, and agotrons comprised a higher proportion of more abundant FLEXIs than in the other cell types (
                              <a id="xref-fig-1-13" class="xref-fig" href="#F1">
                               Fig. 1E
                              </a>
                              ). The relatively abundant FLEXIs included 3,363 detected at ≥0.01 RPM in one or more of the cellular RNA samples.
                             </p>
                             <p id="p-18">
                              To estimate copy number per cell values of FLEXI RNAs, we used sncRNAs of known cellular abundance in the TGIRT-seq datasets to produce a linear regression model for the relationship between sequence-bias corrected log
                              <sub>
                               10
                              </sub>
                              RPM values and log
                              <sub>
                               10
                              </sub>
                              transformed sncRNA copy number per cell values from the literature (
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Fig. S6A
                              </a>
                              ;
                              <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">
                               Table S2
                              </a>
                              ). By using this method, we estimated that moderately abundant FLEXIs (0.01-0.99 RPM) were present at 15 to 239 copies per cell, and the most abundant FLEXIs (≥1 RPM) were present at 363 to 1,863 copies per cell (see linear regression plots in
                              <a id="xref-fig-14-2" class="xref-fig" href="#F14">
                               Fig. S6A
                              </a>
                              ). Based on the linear regression model, FLEXIs previously annotated as agotrons were present at 1 to 1,335 copies per cell; mirtron pre-miRNA FLEXIs at 1 to 225 copies per cell; and snoRNA-encoding FLEXIs at 1 to 265 copies per cell, where 1 is the minimum copy number cell value that could be estimated from the linear regression. Copy per cell values determined by droplet digital PCR (ddPCR) for a sampling of FLEXIs were generally of the same order of magnitude as the TGIRT-seq quantitation, but varied with the choice of sncRNA standard, with the most abundant FLEXIs estimated to be present at 3,000 to 4,000 copies per cell similar to that of U7 RNA when the latter was used as a standard (
                              <a id="xref-fig-14-3" class="xref-fig" href="#F14">
                               Fig. S6B
                              </a>
                              ).
                             </p>
                             <div id="F14" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F14.large.jpg?width=800&height=600&carousel=1" title="Estimates of FLEXI RNA abundance using a linear regression model based on sncRNA standards in TGIRT-seq datasets and droplet digital PCR. (A) Linear regression of molecule per cell values based on RPM values of sncRNAs of known cellular abundance in TGIRT-seq datasets. The scatter plots show the relationship between log10 transformed copy number per cell values for sncRNAs reported in the literature (Tycowski et al., 2006) and log10 transformed RPM of sncRNAs in combined datasets for each of the four cellular RNA samples before (top panels) or after (bottom) bias-correction of TGIRT-seq reads using a random forest regression model (Xu et al., 2019). The linear regression was plotted as a light blue line with 95% confidence intervals shown as dashed blue lines. sncRNA used in the linear regression were major spliceosomal snRNAs U1, U2, U4, U5, and U6; minor spliceosomal snRNAs U4ATAC and U6ATAC; U7, 7SL and 7SK RNA; MRP and RNase P RNA (Table S2) (Tycowski et al., 2006). Pearson (r) correlation coefficients are shown in the upper left of each panel. A test of the linear regression model (lrm) for three sncRNAs (U7, SNORD14B, and SNORD44) used as ddPCR standards below gave copy number per cell estimates consistent with literature values (Tycowski et al., 2006), but with copy number per cell values varying between different cell types (see Table at the bottom of panel A). (B) Abundance estimates for FLEXI RNAs based on the TGIRT-seq linear regression model and droplet digital PCR (ddPCR). The left-hand columns show RPM and copy number per cell values of FLEXIs estimated using the linear regression model in cellular RNA samples. The right-hand columns show copy number per cell values based on ddPCR abundance relative to three different sncRNA standards run in parallel (U7, SNORD14B, and SNORD44). For ddPCR, reverse transcription was performed on DNasetreated, size-selected (≤200 nt) RNA preparations using Maxima H reverse transcriptase with a gene-specific primer at the 3’ end of the putative FLEXI or sncRNA. ddPCR was then performed using primer sets within the FLEXI sequence or spanning the 5’exon-FLEXI junction, with 5’exon-FLEXI junction signals acting as a control to detect amplification of any RNAs with retained FLEXI introns, which was subtracted from the FLEXI signal when applicable. Copy number per cell values for FLEXIs were calculated based on copy number per cell values for U7, SNORD14B, and SNORD44 in each cellular RNA sample, calculated using the linear regression model without or with TGIRT-seq bias correction (see Table at the bottom of panel A). The ddPCR values relative to U7 RNA, which has the closest cellular abundance to FLEXIs, showed general agreement with the copy number per cell values estimated from the linear regression model without TGIRT-seq bias correction, while the other sncRNA controls, predicted to be in much higher abundance than FLEXIs, gave FLEXI quantifications closer to the bias-corrected linear regression model. Primer amplification specificity was verified by gel extraction of a representative set of matched qPCR amplicons, which were then Sanger sequenced directly or after TOPO-TA cloning (ThermoFisher). The combined TGIRT-seq and ddPCR approach gives greater confidence in the identity and abundance of FLEXIs. Although ddPCR offers a vast improvement in precision compared to qPCR, it is still subject to PCR artifacts and primer-design constraints. Using multiple sncRNA controls concurrently allows for an estimate range for FLEXI abundance, which can minimize the impact of inaccuracies in quantifying any individual sncRNA." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Estimates of FLEXI RNA abundance using a linear regression model based on sncRNA standards in TGIRT-seq datasets and droplet digital PCR. (<em>A</em>) Linear regression of molecule per cell values based on RPM values of sncRNAs of known cellular abundance in TGIRT-seq datasets. The scatter plots show the relationship between log<sub>10</sub> transformed copy number per cell values for sncRNAs reported in the literature (Tycowski et al., 2006) and log<sub>10</sub> transformed RPM of sncRNAs in combined datasets for each of the four cellular RNA samples before (top panels) or after (bottom) bias-correction of TGIRT-seq reads using a random forest regression model (Xu et al., 2019). The linear regression was plotted as a light blue line with 95% confidence intervals shown as dashed blue lines. sncRNA used in the linear regression were major spliceosomal snRNAs U1, U2, U4, U5, and U6; minor spliceosomal snRNAs U4ATAC and U6ATAC; U7, 7SL and 7SK RNA; MRP and RNase P RNA (Table S2) (Tycowski et al., 2006). Pearson (<em>r</em>) correlation coefficients are shown in the upper left of each panel. A test of the linear regression model (lrm) for three sncRNAs (U7, SNORD14B, and SNORD44) used as ddPCR standards below gave copy number per cell estimates consistent with literature values (Tycowski et al., 2006), but with copy number per cell values varying between different cell types (see Table at the bottom of panel A). (<em>B</em>) Abundance estimates for FLEXI RNAs based on the TGIRT-seq linear regression model and droplet digital PCR (ddPCR). The left-hand columns show RPM and copy number per cell values of FLEXIs estimated using the linear regression model in cellular RNA samples. The right-hand columns show copy number per cell values based on ddPCR abundance relative to three different sncRNA standards run in parallel (U7, SNORD14B, and SNORD44). For ddPCR, reverse transcription was performed on DNasetreated, size-selected (≤200 nt) RNA preparations using Maxima H reverse transcriptase with a gene-specific primer at the 3’ end of the putative FLEXI or sncRNA. ddPCR was then performed using primer sets within the FLEXI sequence or spanning the 5’exon-FLEXI junction, with 5’exon-FLEXI junction signals acting as a control to detect amplification of any RNAs with retained FLEXI introns, which was subtracted from the FLEXI signal when applicable. Copy number per cell values for FLEXIs were calculated based on copy number per cell values for U7, SNORD14B, and SNORD44 in each cellular RNA sample, calculated using the linear regression model without or with TGIRT-seq bias correction (see Table at the bottom of panel A). The ddPCR values relative to U7 RNA, which has the closest cellular abundance to FLEXIs, showed general agreement with the copy number per cell values estimated from the linear regression model without TGIRT-seq bias correction, while the other sncRNA controls, predicted to be in much higher abundance than FLEXIs, gave FLEXI quantifications closer to the bias-corrected linear regression model. Primer amplification specificity was verified by gel extraction of a representative set of matched qPCR amplicons, which were then Sanger sequenced directly or after TOPO-TA cloning (ThermoFisher). The combined TGIRT-seq and ddPCR approach gives greater confidence in the identity and abundance of FLEXIs. Although ddPCR offers a vast improvement in precision compared to qPCR, it is still subject to PCR artifacts and primer-design constraints. Using multiple sncRNA controls concurrently allows for an estimate range for FLEXI abundance, which can minimize the impact of inaccuracies in quantifying any individual sncRNA.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F14.medium.gif" width="340" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S6." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F14.medium.gif" width="340" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S6.
                               </span>
                               <p id="p-65" class="first-child">
                                Estimates of FLEXI RNA abundance using a linear regression model based on sncRNA standards in TGIRT-seq datasets and droplet digital PCR. (A) Linear regression of molecule per cell values based on RPM values of sncRNAs of known cellular abundance in TGIRT-seq datasets. The scatter plots show the relationship between log
                                <sub>
                                 10
                                </sub>
                                transformed copy number per cell values for sncRNAs reported in the literature (
                                <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">
                                 Tycowski et al., 2006
                                </a>
                                ) and log
                                <sub>
                                 10
                                </sub>
                                transformed RPM of sncRNAs in combined datasets for each of the four cellular RNA samples before (top panels) or after (bottom) bias-correction of TGIRT-seq reads using a random forest regression model (
                                <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">
                                 Xu et al., 2019
                                </a>
                                ). The linear regression was plotted as a light blue line with 95% confidence intervals shown as dashed blue lines. sncRNA used in the linear regression were major spliceosomal snRNAs U1, U2, U4, U5, and U6; minor spliceosomal snRNAs U4ATAC and U6ATAC; U7, 7SL and 7SK RNA; MRP and RNase P RNA (
                                <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">
                                 Table S2
                                </a>
                                ) (
                                <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">
                                 Tycowski et al., 2006
                                </a>
                                ). Pearson (r) correlation coefficients are shown in the upper left of each panel. A test of the linear regression model (lrm) for three sncRNAs (U7, SNORD14B, and SNORD44) used as ddPCR standards below gave copy number per cell estimates consistent with literature values (
                                <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">
                                 Tycowski et al., 2006
                                </a>
                                ), but with copy number per cell values varying between different cell types (see Table at the bottom of panel A). (B) Abundance estimates for FLEXI RNAs based on the TGIRT-seq linear regression model and droplet digital PCR (ddPCR). The left-hand columns show RPM and copy number per cell values of FLEXIs estimated using the linear regression model in cellular RNA samples. The right-hand columns show copy number per cell values based on ddPCR abundance relative to three different sncRNA standards run in parallel (U7, SNORD14B, and SNORD44). For ddPCR, reverse transcription was performed on DNasetreated, size-selected (≤200 nt) RNA preparations using Maxima H reverse transcriptase with a gene-specific primer at the 3’ end of the putative FLEXI or sncRNA. ddPCR was then performed using primer sets within the FLEXI sequence or spanning the 5’exon-FLEXI junction, with 5’exon-FLEXI junction signals acting as a control to detect amplification of any RNAs with retained FLEXI introns, which was subtracted from the FLEXI signal when applicable. Copy number per cell values for FLEXIs were calculated based on copy number per cell values for U7, SNORD14B, and SNORD44 in each cellular RNA sample, calculated using the linear regression model without or with TGIRT-seq bias correction (see Table at the bottom of panel A). The ddPCR values relative to U7 RNA, which has the closest cellular abundance to FLEXIs, showed general agreement with the copy number per cell values estimated from the linear regression model without TGIRT-seq bias correction, while the other sncRNA controls, predicted to be in much higher abundance than FLEXIs, gave FLEXI quantifications closer to the bias-corrected linear regression model. Primer amplification specificity was verified by gel extraction of a representative set of matched qPCR amplicons, which were then Sanger sequenced directly or after TOPO-TA cloning (ThermoFisher). The combined TGIRT-seq and ddPCR approach gives greater confidence in the identity and abundance of FLEXIs. Although ddPCR offers a vast improvement in precision compared to qPCR, it is still subject to PCR artifacts and primer-design constraints. Using multiple sncRNA controls concurrently allows for an estimate range for FLEXI abundance, which can minimize the impact of inaccuracies in quantifying any individual sncRNA.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2442240/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2442240/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2442240" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S2.
                               </span>
                               <span class="caption-title">
                                Abundance of sncRNAs (RPM) detected by TGIRT-seq in cellular RNA samples compared to literature copy number per cell values for these RNAs
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Abundant FLEXI RNAs have cell-type specific expression patterns
                             </h3>
                             <p id="p-19">
                              Scatter plots of FLEXIs detected in pairwise comparisons of different cell lines identified numerous relatively abundant FLEXI RNAs (≥0.01 RPM) that were reproducibly detected and differentially expressed between the two compared cell types (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              and
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               Fig. S7
                              </a>
                              ). Further, the expression patterns of these relatively abundant FLEXIs were more discriminatory between cell types than those of their corresponding host genes, indicating that FLEXI RNA abundance is controlled independently of host gene mRNA levels (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              ). IGV alignments showed that factors that may contribute to the more discriminatory expression patterns of FLEXIs included celltype specific differences in alternative splicing, splicing efficiency, and intron RNA turnover, all of which can result in differences in the relative abundance of different FLEXIs transcribed from the same host gene (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              ).
                             </p>
                             <div id="F15" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F15.large.jpg?width=800&height=600&carousel=1" title="Scatter plots comparing log2-transformed RPM of relatively abundant (≥0.01 RPM) FLEXI RNAs in biological replicates (Bio 1 and Bio 2) for four different cellular RNA samples. Bio 1 is a merged dataset for all technical replicates for the indicated cellular RNA samples obtained for this study, and Bio 2 is a merged dataset for all technical replicates of different RNA preparations from the same cell types obtained for other studies (including a different lot number for UHHR; see Table S1). FLEXIs that were detected in all Bio 1 technical replicates are shown as red dots. r is the Pearson correlation coefficient." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Scatter plots comparing log<em><sub>2</sub></em>-transformed RPM of relatively abundant (≥0.01 RPM) FLEXI RNAs in biological replicates (Bio 1 and Bio 2) for four different cellular RNA samples. Bio 1 is a merged dataset for all technical replicates for the indicated cellular RNA samples obtained for this study, and Bio 2 is a merged dataset for all technical replicates of different RNA preparations from the same cell types obtained for other studies (including a different lot number for UHHR; see Table S1). FLEXIs that were detected in all Bio 1 technical replicates are shown as red dots. <em>r</em> is the Pearson correlation coefficient.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F15.medium.gif" width="401" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S7." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F15.medium.gif" width="401" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S7.
                               </span>
                               <p id="p-66" class="first-child">
                                Scatter plots comparing log
                                <sub>
                                 2
                                </sub>
                                -transformed RPM of relatively abundant (≥0.01 RPM) FLEXI RNAs in biological replicates (Bio 1 and Bio 2) for four different cellular RNA samples. Bio 1 is a merged dataset for all technical replicates for the indicated cellular RNA samples obtained for this study, and Bio 2 is a merged dataset for all technical replicates of different RNA preparations from the same cell types obtained for other studies (including a different lot number for UHHR; see
                                <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">
                                 Table S1
                                </a>
                                ). FLEXIs that were detected in all Bio 1 technical replicates are shown as red dots. r is the Pearson correlation coefficient.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              The cell-type specific expression patterns of FLEXI RNAs were confirmed by principal component analysis (PCA), t-SNE (
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               Kobak and Berens, 2019
                              </a>
                              ) and ZINB-WaVE (
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               Risso et al., 2018
                              </a>
                              ), which showed clustering of the FLEXI RNA expression profiles in the cellular RNA datasets obtained for this study (including for MDA-MB-231 cells analyzed below) with those in TGIRT-seq datasets for additional biological and technical replicates for four of the cellular RNA sample types obtained from other studies (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Fig. S8A
                              </a>
                              ,
                              <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              ). The clustering of cell-type specific expression patterns improved when restricted to more abundant FLEXIs detected at ≥0.01 RPM, but only slightly compared to those for all FLEXIs and did not improve further with higher abundance cutoffs (
                              <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                               Fig. S8B
                              </a>
                              ). This likely reflects that sample comparisons using the above algorithms are dominated by higher abundance FLEXIs. Collectively, these findings indicate that relatively abundant FLEXI RNAs have cell-type specific expression patterns.
                             </p>
                             <div id="F16" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F16.large.jpg?width=800&height=600&carousel=1" title="Clustering analysis of detected FLEXI RNAs. (A) and (B) PCA, PCA-initialized t- SNE, and ZINB-WaVE analysis for FLEXI RNAs detected at ≥1 read and ≥0.01 RPM, respectively, in datasets for biological and technical replicates of rRNA-depleted unfragmented cellular RNAs (Table S1). t-SNE (Kobak and Berens, 2019) and ZINB-WaVE (Risso et al., 2018) are algorithms widely used for the analysis of single cell RNA-seq datasets with zero-inflated counts. Top rows are clustering of technical replicates from difference cell types used in this study. Middle and bottom rows show clustering of datasets obtained for this study (circle, biological replicate 1; Bio 1) and biological replicate datasets for rRNA-depleted unfragmented HEK-293T, K- 562, MDA-MB-231, and UHRR RNAs obtained for other studies (triangle, biological replicate 2; Bio 2) (see Table S1), before and after batch effect correction by ZINB-WaVE, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Clustering analysis of detected FLEXI RNAs.</span> (<em>A</em>) and (<em>B</em>) PCA, PCA-initialized t- SNE, and ZINB-WaVE analysis for FLEXI RNAs detected at ≥1 read and ≥0.01 RPM, respectively, in datasets for biological and technical replicates of rRNA-depleted unfragmented cellular RNAs (Table S1). t-SNE (Kobak and Berens, 2019) and ZINB-WaVE (Risso et al., 2018) are algorithms widely used for the analysis of single cell RNA-seq datasets with zero-inflated counts. Top rows are clustering of technical replicates from difference cell types used in this study. Middle and bottom rows show clustering of datasets obtained for this study (circle, biological replicate 1; Bio 1) and biological replicate datasets for rRNA-depleted unfragmented HEK-293T, K- 562, MDA-MB-231, and UHRR RNAs obtained for other studies (triangle, biological replicate 2; Bio 2) (see Table S1), before and after batch effect correction by ZINB-WaVE, respectively.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F16.medium.gif" width="273" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S8." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F16.medium.gif" width="273" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S8.
                               </span>
                               <span class="caption-title">
                                Clustering analysis of detected FLEXI RNAs.
                               </span>
                               <p id="p-67" class="first-child">
                                (A) and (B) PCA, PCA-initialized t- SNE, and ZINB-WaVE analysis for FLEXI RNAs detected at ≥1 read and ≥0.01 RPM, respectively, in datasets for biological and technical replicates of rRNA-depleted unfragmented cellular RNAs (
                                <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">
                                 Table S1
                                </a>
                                ). t-SNE (
                                <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                                 Kobak and Berens, 2019
                                </a>
                                ) and ZINB-WaVE (
                                <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">
                                 Risso et al., 2018
                                </a>
                                ) are algorithms widely used for the analysis of single cell RNA-seq datasets with zero-inflated counts. Top rows are clustering of technical replicates from difference cell types used in this study. Middle and bottom rows show clustering of datasets obtained for this study (circle, biological replicate 1; Bio 1) and biological replicate datasets for rRNA-depleted unfragmented HEK-293T, K- 562, MDA-MB-231, and UHRR RNAs obtained for other studies (triangle, biological replicate 2; Bio 2) (see
                                <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">
                                 Table S1
                                </a>
                                ), before and after batch effect correction by ZINB-WaVE, respectively.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Evolutionary conservation of FLEXIs
                             </h3>
                             <p id="p-21">
                              PhastCons scores provide a measure of evolutionary conservation, with higher phastCons scores indicating a higher probability that a sequence is conserved and functionally important across a range of organisms (
                              <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                               Siepel et al., 2005
                              </a>
                              ). Density plots of phastCons scores calculated across 27 primates including humans plus mouse, dog, and armadillo showed that most FLEXIs, including those corresponding to mirtron pre-miRNAs or agotrons, were poorly conserved (peaks at 0.06-0.09 compared to 0.02 for other annotated short introns in GRCh38), but with tails extending to higher phastCons scores. FLEXIs encoding snoRNAs had higher phastCons scores (four at ≥0.5) than did other FLEXIs, likely indicating conservation of the snoRNA component (
                              <a id="xref-fig-1-14" class="xref-fig" href="#F1">
                               Fig. 1F
                              </a>
                              , yellow line). The relatively low phastCons scores of most FLEXIs could reflect recent acquisition and/or rapid sequence divergence in the human lineage.
                             </p>
                             <p id="p-22">
                              Although most FLEXI RNAs were poorly conserved, 2% (158) of the FLEXIs that were not annotated as mirtrons, agotrons or encoding snoRNAs had phastCons scores ≥0.75, possibly reflecting an evolutionarily conserved sequence-dependent function. Compared to those with lower phastCons scores, these more highly conserved FLEXIs were enriched in introns that were alternatively spliced (69% compared to 35%), shared a 5’- or 3’-splice site with a longer intron (44% compared to 18%), corresponded to a retained intron (47% compared to 25%), or contained an in-frame protein-coding sequence that would be expressed if all or part of the intron was retained within a mRNA (20% compared to 7.8%;
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              ). The proportion of FLEXIs with these characteristics was even higher for the 44 most highly conserved FLEXIs (phastCons scores ≥0.99;
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              ). A FLEXI in the human EIF1 gene with phastCons score = 1.00 resulted from acquisition of a 3’-splice site in its highly conserved 3’ UTR leading to expression of a novel human-specific EIF1 isoform (chr17:41,690,818-41,690,902; Supplemental File) (
                              <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                               Kim et al., 2020
                              </a>
                              ). These findings suggest that in most if not all cases the sequence conservation of FLEXIs with high phastCons scores reflects evolutionary constraints on different protein isoforms produced by alternative splicing.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              FLEXIs contain binding sites for >100 different RBPs with diverse cellular functions
                             </h3>
                             <p id="p-23">
                              To explore possible biological functions of FLEXIs, we searched for RBPs that have binding sites in FLEXI RNAs in CLIP-seq datasets (ENCODE high confidence eCLIP datasets for K562 and HepG2 cells (
                              <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">
                               Van Nostrand et al., 2016
                              </a>
                              ) and AGO1-4 and DICER PAR-CLIP datasets for HEK-293 cells (
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               Hafner et al., 2010
                              </a>
                              ;
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               Rybak-Wolf et al., 2014
                              </a>
                              )). More than half of the detected FLEXIs (4,505; 55%) contained a CLIP-seq-identified binding site for one or more of 126 different RBPs, with 53 RBPs (51 in the eCLIP datasets plus DICER and AGO1-4 in the PAR-CLIP datasets) having binding sites in ≥30 different FLEXI RNAs (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ,
                              <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                               Fig. S9A
                              </a>
                              ). These 53 RBPs included spliceosomal proteins and other proteins that function in RNA splicing; DICER, AGO1-4 and other proteins that have miRNA-related functions; and a surprising number of proteins that function in cellular regulatory processes, including transcriptional regulation, chromatin assembly and disassembly, cellular growth regulation, and stress responses (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              and
                              <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">
                               Table S3
                              </a>
                              ). Although cross-linking does not distinguish whether the RBP was bound to the intron before or after splicing, examination of sequence reads in the ENCODE eCLIP datasets showed that all of the RBPs that have a binding site in ≥30 different FLEXIs were cross-linked to ≥95% full-length intron RNAs corresponding to identified FLEXIs, with 32 showing a ≥2-fold enrichment of such reads compared to the control antibody datasets (
                              <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                               Fig. S10
                              </a>
                              ). This finding indicates that whether or not the RBP bound to the intron before or after splicing, some fraction of each RBP remained bound to an excised intron RNA and may have protected it from RNase I digestion used for processing the cross-linked RNAs for sequencing in the ENCODE datasets.
                             </p>
                             <div id="F17" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F17.large.jpg?width=800&height=600&carousel=1" title="Bar graphs showing the number of FLEXIs and other classes of introns in a merged dataset for the four cellular samples that have a CLIP-seq identified RBP-binding site for the indicated RBP. (A) FLEXI RNAs. (B) Other Ensembl GRCh38-annotated short introns (≤300 nt); (C) Ensembl GRCh38-annotated long introns (>300 nt). Bars are color coded by RBP function as shown at the top. Asterisks in panels B and C indicate the 53 proteins identified as binding 30 or more different FLEXIs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Bar graphs showing the number of FLEXIs and other classes of introns in a merged dataset for the four cellular samples that have a CLIP-seq identified RBP-binding site for the indicated RBP. (<em>A</em>) FLEXI RNAs. (<em>B</em>) Other Ensembl GRCh38-annotated short introns (≤300 nt); (<em>C</em>) Ensembl GRCh38-annotated long introns (>300 nt). Bars are color coded by RBP function as shown at the top. Asterisks in panels B and C indicate the 53 proteins identified as binding 30 or more different FLEXIs.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S9." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F17.medium.gif" width="440" height="311"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S9." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F17.medium.gif" width="440" height="311"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S9.
                               </span>
                               <p id="p-68" class="first-child">
                                Bar graphs showing the number of FLEXIs and other classes of introns in a merged dataset for the four cellular samples that have a CLIP-seq identified RBP-binding site for the indicated RBP. (A) FLEXI RNAs. (B) Other Ensembl GRCh38-annotated short introns (≤300 nt); (C) Ensembl GRCh38-annotated long introns (>300 nt). Bars are color coded by RBP function as shown at the top. Asterisks in panels B and C indicate the 53 proteins identified as binding 30 or more different FLEXIs.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F18" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F18.large.jpg?width=800&height=600&carousel=1" title="Enrichment of FLEXI reads in ENCODE eCLIP datasets. eCLIP data were obtained from ENCODE for all RBPs that bind ≥30 FLEXIs, and ≥95% full-length intron reads were identified by intersecting the mapped eCLIP reads with FLEXI coordinates. Data are shown as an enrichment of ≥95% full-length intron reads for each RBP, calculated as a ratio of reads in the eCLIP versus the control antibody dataset (**** p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Enrichment of FLEXI reads in ENCODE eCLIP datasets.</span> eCLIP data were obtained from ENCODE for all RBPs that bind ≥30 FLEXIs, and ≥95% full-length intron reads were identified by intersecting the mapped eCLIP reads with FLEXI coordinates. Data are shown as an enrichment of ≥95% full-length intron reads for each RBP, calculated as a ratio of reads in the eCLIP versus the control antibody dataset (**** p<0.0001, * p<0.05, Fisher’s least significant difference test). The dashed line indicates two-fold enrichment. Names of proteins are color coded by protein function as indicated in the Figure.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S10." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F18.medium.gif" width="440" height="242"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S10." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F18.medium.gif" width="440" height="242"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S10.
                               </span>
                               <span class="caption-title">
                                Enrichment of FLEXI reads in ENCODE eCLIP datasets.
                               </span>
                               <p id="p-69" class="first-child">
                                eCLIP data were obtained from ENCODE for all RBPs that bind ≥30 FLEXIs, and ≥95% full-length intron reads were identified by intersecting the mapped eCLIP reads with FLEXI coordinates. Data are shown as an enrichment of ≥95% full-length intron reads for each RBP, calculated as a ratio of reads in the eCLIP versus the control antibody dataset (**** p<0.0001, * p<0.05, Fisher’s least significant difference test). The dashed line indicates two-fold enrichment. Names of proteins are color coded by protein function as indicated in the Figure.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T3" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2442223/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup last">
                                  <a href="/highwire/markup/2442223/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S3.
                               </span>
                               <span class="caption-title">
                                Proteins that bind 30 or more different FLEXIs
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T4" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2442227/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2442227/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2442227" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S4.
                               </span>
                               <span class="caption-title">
                                Oligonucleotides used for construction of TGIRT-seq libraries
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="T5" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2442215/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup last">
                                  <a href="/highwire/markup/2442215/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S5.
                               </span>
                               <span class="caption-title">
                                Oligonucleotides used for droplet digital PCR (ddPCR) and Terminator 5’ to 3’ exonuclease digestion assays.
                               </span>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F4.large.jpg?width=800&height=600&carousel=1" title="FLEXI interactions with RNA-binding proteins. (A) Bar graph showing the number of different cellular FLEXI RNAs that have a CLIP-seq identified binding site for the 53 RBPs that have binding sites in ≥30 different FLEXI RNAs in a merged dataset for the four cellular RNA samples. An extended bar graph for all 126 RBPs that have a CLIP-seq identified binding site in a FLEXI RNA is shown in Fig. S9A. The grids below the bar graph show RBP function and cellular localization information from (Van Nostrand et al., 2020) supplemented by the mammalian RNA granule and stress granule protein databases (Nunes et al., 2019; Youn et al., 2019) and the UniProt database (The UniProt Consortium, 2018), followed by RBP-binding site and protein function information obtained in this study from analysis of CLIP-seq and knockdown datasets: RNA-binding sites: I, within the intron; SS-I, at or adjacent to the 5’- and/or 3’- splice site within the intron; SS-E, adjacent to splice site in flanking exons (Fig. 5C and Fig. S13). mRNA levels in RBP knockdown datasets: light blue and red, significant bias toward higher (log2FC>0) or lower (log2FC300 nt) or other short introns (≤300 nt) annotated in GRCh38. RBPs whose binding sites were ≥1% of the total RBP-binding sites and significantly different between the compared groups (p ≤0.05 calculated by Fisher’s exact test) are indicated by the name of the protein color coded by protein function as indicated at the top of Figure. For each panel, the box in the left-hand scatter plot delineates a region that is expanded in the right-hand scatter plot. The circle in the right-hand scatter plot of panel C delineates 12 additional RBPs whose binding sites meet the abundance and significance criteria for enrichment in long introns compared to FLEXIs but were too numerous to name in the Figure (FAM120A, GTF2F1, HNRNPC, HNRNPK, HNRNPL, HNRNPM, ILF3, MATR3, NONO, PCBP2, SUGP2, and TARDBP)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">FLEXI interactions with RNA-binding proteins.</span> (<em>A</em>) Bar graph showing the number of different cellular FLEXI RNAs that have a CLIP-seq identified binding site for the 53 RBPs that have binding sites in ≥30 different FLEXI RNAs in a merged dataset for the four cellular RNA samples. An extended bar graph for all 126 RBPs that have a CLIP-seq identified binding site in a FLEXI RNA is shown in Fig. S9A. The grids below the bar graph show RBP function and cellular localization information from (Van Nostrand et al., 2020) supplemented by the mammalian RNA granule and stress granule protein databases (Nunes et al., 2019; Youn et al., 2019) and the UniProt database (The UniProt Consortium, 2018), followed by RBP-binding site and protein function information obtained in this study from analysis of CLIP-seq and knockdown datasets: RNA-binding sites: I, within the intron; SS-I, at or adjacent to the 5’- and/or 3’- splice site within the intron; SS-E, adjacent to splice site in flanking exons (Fig. 5C and Fig. S13). mRNA levels in RBP knockdown datasets: light blue and red, significant bias toward higher (log<sub>2</sub>FC>0) or lower (log<sub>2</sub>FC<0) mRNA levels, respectively, compared to all significantly changed genes; gray, significant change in FLEXI host gene mRNA expression levels, but not significantly biased in either direction (Fig. 5A and Fig. S11). Alternative Splicing (Alt. Spl.) in RBP knockdown datasets: RI, significant increase in retained introns; SE, significant increase in skipped exons; Fig. 5B and Fig. S12). Lower GC%: RBPs whose binding sites were significantly enriched in FLEXIs having lower GC content than other FLEXIs (Fig. S15). phastCons scores ≥0.75: RBPs whose binding sites were significantly enriched in evolutionarily conserved FLEXIs with phastCons ≥0.75 (Fig. 6D). (<em>B</em> and <em>C</em>) Scatter plots comparing the abundance (percentage) of binding sites for different RBPs in FLEXI RNAs with those in longer introns (>300 nt) or other short introns (≤300 nt) annotated in GRCh38. RBPs whose binding sites were ≥1% of the total RBP-binding sites and significantly different between the compared groups (p ≤0.05 calculated by Fisher’s exact test) are indicated by the name of the protein color coded by protein function as indicated at the top of Figure. For each panel, the box in the left-hand scatter plot delineates a region that is expanded in the right-hand scatter plot. The circle in the right-hand scatter plot of panel C delineates 12 additional RBPs whose binding sites meet the abundance and significance criteria for enrichment in long introns compared to FLEXIs but were too numerous to name in the Figure (FAM120A, GTF2F1, HNRNPC, HNRNPK, HNRNPL, HNRNPM, ILF3, MATR3, NONO, PCBP2, SUGP2, and TARDBP).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F4.medium.gif" width="440" height="428"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F4.medium.gif" width="440" height="428"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <span class="caption-title">
                                FLEXI interactions with RNA-binding proteins.
                               </span>
                               <p id="p-24" class="first-child">
                                (A) Bar graph showing the number of different cellular FLEXI RNAs that have a CLIP-seq identified binding site for the 53 RBPs that have binding sites in ≥30 different FLEXI RNAs in a merged dataset for the four cellular RNA samples. An extended bar graph for all 126 RBPs that have a CLIP-seq identified binding site in a FLEXI RNA is shown in
                                <a id="xref-fig-17-2" class="xref-fig" href="#F17">
                                 Fig. S9A
                                </a>
                                . The grids below the bar graph show RBP function and cellular localization information from (
                                <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">
                                 Van Nostrand et al., 2020
                                </a>
                                ) supplemented by the mammalian RNA granule and stress granule protein databases (
                                <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                                 Nunes et al., 2019
                                </a>
                                ;
                                <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">
                                 Youn et al., 2019
                                </a>
                                ) and the UniProt database (
                                <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">
                                 The UniProt Consortium, 2018
                                </a>
                                ), followed by RBP-binding site and protein function information obtained in this study from analysis of CLIP-seq and knockdown datasets: RNA-binding sites: I, within the intron; SS-I, at or adjacent to the 5’- and/or 3’- splice site within the intron; SS-E, adjacent to splice site in flanking exons (
                                <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                                 Fig. 5C
                                </a>
                                and
                                <a id="xref-fig-21-1" class="xref-fig" href="#F21">
                                 Fig. S13
                                </a>
                                ). mRNA levels in RBP knockdown datasets: light blue and red, significant bias toward higher (log
                                <sub>
                                 2
                                </sub>
                                FC>0) or lower (log
                                <sub>
                                 2
                                </sub>
                                FC<0) mRNA levels, respectively, compared to all significantly changed genes; gray, significant change in FLEXI host gene mRNA expression levels, but not significantly biased in either direction (
                                <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                                 Fig. 5A
                                </a>
                                and
                                <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                                 Fig. S11
                                </a>
                                ). Alternative Splicing (Alt. Spl.) in RBP knockdown datasets: RI, significant increase in retained introns; SE, significant increase in skipped exons;
                                <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                                 Fig. 5B
                                </a>
                                and
                                <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                                 Fig. S12
                                </a>
                                ). Lower GC%: RBPs whose binding sites were significantly enriched in FLEXIs having lower GC content than other FLEXIs (
                                <a id="xref-fig-23-1" class="xref-fig" href="#F23">
                                 Fig. S15
                                </a>
                                ). phastCons scores ≥0.75: RBPs whose binding sites were significantly enriched in evolutionarily conserved FLEXIs with phastCons ≥0.75 (
                                <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                                 Fig. 6D
                                </a>
                                ). (B and C) Scatter plots comparing the abundance (percentage) of binding sites for different RBPs in FLEXI RNAs with those in longer introns (>300 nt) or other short introns (≤300 nt) annotated in GRCh38. RBPs whose binding sites were ≥1% of the total RBP-binding sites and significantly different between the compared groups (p ≤0.05 calculated by Fisher’s exact test) are indicated by the name of the protein color coded by protein function as indicated at the top of Figure. For each panel, the box in the left-hand scatter plot delineates a region that is expanded in the right-hand scatter plot. The circle in the right-hand scatter plot of panel C delineates 12 additional RBPs whose binding sites meet the abundance and significance criteria for enrichment in long introns compared to FLEXIs but were too numerous to name in the Figure (FAM120A, GTF2F1, HNRNPC, HNRNPK, HNRNPL, HNRNPM, ILF3, MATR3, NONO, PCBP2, SUGP2, and TARDBP).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F19" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-19.large.jpg?width=800&height=600&carousel=1" title="Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets. The plots show -log10 transformed adjusted p-values versus log2 transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log2FC>0) or down (log2FC" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets.</span> The plots show -log<sub>10</sub> transformed adjusted p-values versus log<sub>2</sub> transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log<sub>2</sub>FC>0) or down (log<sub>2</sub>FC<0) regulated host genes containing a FLEXI with a binding site for the RBP to that in all significantly changed FLEXI host genes). RBP knock- downs that resulted in significant changes in mRNA levels from host genes containing a FLEXI with a binding site for the knocked down RNA but no significant directional bias are indicated by a bidirectional arrow next to the RBP name. Datasets that were not available for an RBP in one of the two cell lines were left as a blank space.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-19.medium.gif" width="354" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-19.medium.gif" width="354" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-20.large.jpg?width=800&height=600&carousel=1" title="Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets. The plots show -log10 transformed adjusted p-values versus log2 transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log2FC>0) or down (log2FC" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets.</span> The plots show -log<sub>10</sub> transformed adjusted p-values versus log<sub>2</sub> transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log<sub>2</sub>FC>0) or down (log<sub>2</sub>FC<0) regulated host genes containing a FLEXI with a binding site for the RBP to that in all significantly changed FLEXI host genes). RBP knock- downs that resulted in significant changes in mRNA levels from host genes containing a FLEXI with a binding site for the knocked down RNA but no significant directional bias are indicated by a bidirectional arrow next to the RBP name. Datasets that were not available for an RBP in one of the two cell lines were left as a blank space.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-20.medium.gif" width="357" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-20.medium.gif" width="357" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-21.large.jpg?width=800&height=600&carousel=1" title="Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets. The plots show -log10 transformed adjusted p-values versus log2 transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log2FC>0) or down (log2FC" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets.</span> The plots show -log<sub>10</sub> transformed adjusted p-values versus log<sub>2</sub> transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log<sub>2</sub>FC>0) or down (log<sub>2</sub>FC<0) regulated host genes containing a FLEXI with a binding site for the RBP to that in all significantly changed FLEXI host genes). RBP knock- downs that resulted in significant changes in mRNA levels from host genes containing a FLEXI with a binding site for the knocked down RNA but no significant directional bias are indicated by a bidirectional arrow next to the RBP name. Datasets that were not available for an RBP in one of the two cell lines were left as a blank space.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-21.medium.gif" width="354" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-21.medium.gif" width="354" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F19/graphic-21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S11.
                               </span>
                               <span class="caption-title">
                                Volcano plots showing gene expression changes in FLEXI host genes in RBP knockdown versus control datasets.
                               </span>
                               <p id="p-70" class="first-child">
                                The plots show -log
                                <sub>
                                 10
                                </sub>
                                transformed adjusted p-values versus log
                                <sub>
                                 2
                                </sub>
                                transformed fold changes for ENSEMBL-annotated genes in knockdown versus control datasets for the indicated RBPs in K-562 and HepG2 cells (GENCODE). Host genes of FLEXI RNAs are shown as red dots. Other genes with or without significant expression changes (adjusted p-value ≤0.05) are shown as black or gray dots, respectively. RBP knockdown that had a significant bias towards increased or decreased mRNA levels in the group of host genes encoding FLEXIs with a binding site for the RBP are indicated by up (light blue) or down (red) arrows next to the RBP name (p-value ≤0.05 determined by Fisher’s exact test comparing the ratio of significantly up (log
                                <sub>
                                 2
                                </sub>
                                FC>0) or down (log
                                <sub>
                                 2
                                </sub>
                                FC<0) regulated host genes containing a FLEXI with a binding site for the RBP to that in all significantly changed FLEXI host genes). RBP knock- downs that resulted in significant changes in mRNA levels from host genes containing a FLEXI with a binding site for the knocked down RNA but no significant directional bias are indicated by a bidirectional arrow next to the RBP name. Datasets that were not available for an RBP in one of the two cell lines were left as a blank space.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F20" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-22.large.jpg?width=800&height=600&carousel=1" title="Splicing changes at or adjacent to introns that give rise to FLEXI RNAs in RBP knockdown versus control datasets. Splicing changes were calculated using rMATS (Shen et al., 2014). Density plots show inclusion of retained introns (RI) or skipped exons (SE) for bound FLEXIs (red), unbound FLEXIs (blue), and other short introns (black) for RBP knock- downs that caused a significant change (p≤0.05) in splicing at or adjacent to bound FLEXIs compared to the other categories of introns. In the RI plots, shifts to the right or left indicate an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov-Smirnov test. Names of RBP are color coded by protein function: red, RNA splicing-related function; orange, miRNA-related function; black, other function. Datasets that were not available for a specific RBP in one of the two cell lines were left as a blank space." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Splicing changes at or adjacent to introns that give rise to FLEXI RNAs in RBP knockdown versus control datasets.</span> Splicing changes were calculated using rMATS (Shen et al., 2014). Density plots show inclusion of retained introns (RI) or skipped exons (SE) for bound FLEXIs (red), unbound FLEXIs (blue), and other short introns (black) for RBP knock- downs that caused a significant change (p≤0.05) in splicing at or adjacent to bound FLEXIs compared to the other categories of introns. In the RI plots, shifts to the right or left indicate an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov-Smirnov test. Names of RBP are color coded by protein function: red, RNA splicing-related function; orange, miRNA-related function; black, other function. Datasets that were not available for a specific RBP in one of the two cell lines were left as a blank space.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-22.medium.gif" width="440" height="329"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S12." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-22.medium.gif" width="440" height="329"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-23.large.jpg?width=800&height=600&carousel=1" title="Splicing changes at or adjacent to introns that give rise to FLEXI RNAs in RBP knockdown versus control datasets. Splicing changes were calculated using rMATS (Shen et al., 2014). Density plots show inclusion of retained introns (RI) or skipped exons (SE) for bound FLEXIs (red), unbound FLEXIs (blue), and other short introns (black) for RBP knock- downs that caused a significant change (p≤0.05) in splicing at or adjacent to bound FLEXIs compared to the other categories of introns. In the RI plots, shifts to the right or left indicate an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov-Smirnov test. Names of RBP are color coded by protein function: red, RNA splicing-related function; orange, miRNA-related function; black, other function. Datasets that were not available for a specific RBP in one of the two cell lines were left as a blank space." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Splicing changes at or adjacent to introns that give rise to FLEXI RNAs in RBP knockdown versus control datasets.</span> Splicing changes were calculated using rMATS (Shen et al., 2014). Density plots show inclusion of retained introns (RI) or skipped exons (SE) for bound FLEXIs (red), unbound FLEXIs (blue), and other short introns (black) for RBP knock- downs that caused a significant change (p≤0.05) in splicing at or adjacent to bound FLEXIs compared to the other categories of introns. In the RI plots, shifts to the right or left indicate an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov-Smirnov test. Names of RBP are color coded by protein function: red, RNA splicing-related function; orange, miRNA-related function; black, other function. Datasets that were not available for a specific RBP in one of the two cell lines were left as a blank space.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S12." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-23.medium.gif" width="345" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S12." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-23.medium.gif" width="345" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F20/graphic-23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S12.
                               </span>
                               <span class="caption-title">
                                Splicing changes at or adjacent to introns that give rise to FLEXI RNAs in RBP knockdown versus control datasets.
                               </span>
                               <p id="p-71" class="first-child">
                                Splicing changes were calculated using rMATS (
                                <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">
                                 Shen et al., 2014
                                </a>
                                ). Density plots show inclusion of retained introns (RI) or skipped exons (SE) for bound FLEXIs (red), unbound FLEXIs (blue), and other short introns (black) for RBP knock- downs that caused a significant change (p≤0.05) in splicing at or adjacent to bound FLEXIs compared to the other categories of introns. In the RI plots, shifts to the right or left indicate an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov-Smirnov test. Names of RBP are color coded by protein function: red, RNA splicing-related function; orange, miRNA-related function; black, other function. Datasets that were not available for a specific RBP in one of the two cell lines were left as a blank space.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              RBPs that bind FLEXIs also bind other short and long introns
                             </h3>
                             <p id="p-25">
                              All 53 proteins in the ENCODE eCLIP and PAR-CLIP datasets that have cross-link identified binding sites in ≥30 different FLEXIs also bind thousands of other short and long introns (
                              <a id="xref-fig-17-3" class="xref-fig" href="#F17">
                               Fig. S9B
                              </a>
                              and C). Scatter plots showed that RBPs whose binding sites were significantly over-represented in FLEXIs compared to long introns (significant enrichment defined as ≥1% of total RBPbinding sites; p≤0.05, Fisher’s exact test) included spliceosomal proteins, as well as AGO1-4, DICER, DDX24, GRWD1 (a histone-binding protein that functions in ribosome assembly and regulates chromatin dynamics), RBM22 (a spliceosomal component that may play a role in regulating cell division), UCHL5 (ubiquitin carboxy-terminal hydrolase), and ZNF622 (a putative transcriptional activator that is a positive regulator of apoptosis;
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                              ), with AGO1-4, DICER, and RBM22 also significantly over-represented in FLEXIs compared to other short introns (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4C
                              </a>
                              ). The scatter plots also identified numerous proteins whose binding sites were over-represented in other short or long introns compared to FLEXIs (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4B and C
                              </a>
                              ). The over-representation of spliceosomal protein-binding sites in FLEXI RNAs may reflect that most FLEXIs dissociate more slowly from spliceosomes than do long introns, providing greater opportunity for cross-linking. If so, the abundance of FLEXIs as a whole might be sufficiently high to affect intracellular levels of spliceosomal proteins in response to stimuli that globally affect FLEXI RNA turnover, as found for the collective of yeast linear introns that accumulate under stress conditions (
                              <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">
                               Morgan et al., 2019
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              FLEXI RBP knockdowns impact FLEXI host gene mRNA levels and alternative splicing
                             </h3>
                             <p id="p-26">
                              ENCODE shRNA knockdown datasets for K-562 and/or HepG2 cells were available for all 51 of the RBPs identified in the ENCODE eCLIP datasets as having a binding site in ≥30 different FLEXI RNAs (
                              <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                               The ENCODE Project Consortium, 2012
                              </a>
                              ;
                              <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">
                               Van Nostrand et al., 2016
                              </a>
                              ). By analyzing gene counts in the knockdown datasets, we identified 17 RBPs whose knockdown resulted in significant changes in mRNA expression levels from a group of host genes whose FLEXIs contained a binding site for the knocked down RBP compared to three groups composed of the same number of randomly selected genes (p≤0.05 by analysis of variance (ANOVA) test;
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              and
                              <a id="xref-fig-19-2" class="xref-fig" href="#F19">
                               Fig. S11
                              </a>
                              ; results summarized in grids below the bar graph in
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). For GKPOW, LARP4, RPS3, TIAL1, U2AF2 and UCHL5, these changes were significantly biased toward decreased mRNA levels, while for DDX55, RBM22, and ZNF622, the changes were significantly biased toward increased mRNA levels (p ≤0.05, Fisher’s exact test based on comparison to all significantly changed genes;
                              <a id="xref-fig-19-3" class="xref-fig" href="#F19">
                               Fig. S11
                              </a>
                              ). For the remaining RBPs who knockdowns caused significant changes in FLEXI host gene mRNA levels, the changes were not significantly biased in either direction (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              and
                              <a id="xref-fig-19-4" class="xref-fig" href="#F19">
                               Fig. S11
                              </a>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F5.large.jpg?width=800&height=600&carousel=1" title="Impact of FLEXI interactions with RNA-binding proteins on FLEXI host gene mRNA abundance and alternative splicing. (A). Number of FLEXI host genes with significantly changed mRNA levels, including both under- or overexpressed genes, in RBP knockdown versus control datasets. RNA-seq datasets from RBP knockdown (shRNA or CRISPR) and control datasets for K-562 and HepG2 cells were obtained from ENCODE for all RBPs that bind ≥30 different FLEXIs. Changes in mRNA abundance for host genes with FLEXIs that contain a binding site for the knocked down RBP compared to three randomly chosen genes sets with the same number of genes were calculated from the knockdown datasets by using DESeq2, and the significance of the changes was determined by Analysis of Variance (ANOVA): **** p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Impact of FLEXI interactions with RNA-binding proteins on FLEXI host gene mRNA abundance and alternative splicing.</span> (<em>A</em>). Number of FLEXI host genes with significantly changed mRNA levels, including both under- or overexpressed genes, in RBP knockdown versus control datasets. RNA-seq datasets from RBP knockdown (shRNA or CRISPR) and control datasets for K-562 and HepG2 cells were obtained from ENCODE for all RBPs that bind ≥30 different FLEXIs. Changes in mRNA abundance for host genes with FLEXIs that contain a binding site for the knocked down RBP compared to three randomly chosen genes sets with the same number of genes were calculated from the knockdown datasets by using DESeq2, and the significance of the changes was determined by Analysis of Variance (ANOVA): **** p<0.0001, *** p<0.001, **p<0.01, *p≤0.05. Volcano plots showing the magnitude and direction of changes for all RBPs whose knockdowns caused significant changes in FLEXI host gene mRNA levels are shown in Fig. S11. (<em>B</em>) Alternative splicing changes involving FLEXI introns in RBP knockdowns versus control datasets calculated using rMATS (Shen et al., 2014). Density plots show the inclusion level difference for retained introns (RI) or skipped exons (SE) for FLEXIs that contain a binding site for the indicated RBP (denoted bound FLEXIs, red), FLEXIs that do not contain a binding site for the indicated RBP (denoted unbound FLEXIs, blue), and other short introns (black). In the RI plots, shifts to the right or left represent an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov- Smirnov test. Plots for all RBP knockdowns that caused a significant change (p≤0.05) involving a bound FLEXI are shown in Fig. S12. (<em>C</em>) Density plots comparing the location of ENCODE eCLIP identified RBP-binding sites within FLEXIs to those in other short or long introns. RBP- binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools, and plotted as the mid-point of the annotated RBP-binding sites normalize as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. In each panel, binding-site distributions are shown for all FLEXIs (red), all other short introns (≤300 nt; blue), and all longer introns (>300 nt; black) bound by the indicated RBP. Peak positions in the density plots for FLEXI RNAs are indicated by red dashed. Density plots of the mid-points of RBP-binding sites for all 51 ENCODE eCLIP RBPs that bind ≥30 different FLEXIs are shown in Fig. S13.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F5.medium.gif" width="440" height="353"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F5.medium.gif" width="440" height="353"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <span class="caption-title">
                                Impact of FLEXI interactions with RNA-binding proteins on FLEXI host gene mRNA abundance and alternative splicing.
                               </span>
                               <p id="p-27" class="first-child">
                                (A). Number of FLEXI host genes with significantly changed mRNA levels, including both under- or overexpressed genes, in RBP knockdown versus control datasets. RNA-seq datasets from RBP knockdown (shRNA or CRISPR) and control datasets for K-562 and HepG2 cells were obtained from ENCODE for all RBPs that bind ≥30 different FLEXIs. Changes in mRNA abundance for host genes with FLEXIs that contain a binding site for the knocked down RBP compared to three randomly chosen genes sets with the same number of genes were calculated from the knockdown datasets by using DESeq2, and the significance of the changes was determined by Analysis of Variance (ANOVA): **** p<0.0001, *** p<0.001, **p<0.01, *p≤0.05. Volcano plots showing the magnitude and direction of changes for all RBPs whose knockdowns caused significant changes in FLEXI host gene mRNA levels are shown in
                                <a id="xref-fig-19-5" class="xref-fig" href="#F19">
                                 Fig. S11
                                </a>
                                . (B) Alternative splicing changes involving FLEXI introns in RBP knockdowns versus control datasets calculated using rMATS (
                                <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                 Shen et al., 2014
                                </a>
                                ). Density plots show the inclusion level difference for retained introns (RI) or skipped exons (SE) for FLEXIs that contain a binding site for the indicated RBP (denoted bound FLEXIs, red), FLEXIs that do not contain a binding site for the indicated RBP (denoted unbound FLEXIs, blue), and other short introns (black). In the RI plots, shifts to the right or left represent an increase or decrease in retained introns, respectively, and in the SE plots, shifts to the left or right indicate an increase or decrease in skipped exons, respectively. Statistical significance was calculated by Kolmogorov- Smirnov test. Plots for all RBP knockdowns that caused a significant change (p≤0.05) involving a bound FLEXI are shown in
                                <a id="xref-fig-20-2" class="xref-fig" href="#F20">
                                 Fig. S12
                                </a>
                                . (C) Density plots comparing the location of ENCODE eCLIP identified RBP-binding sites within FLEXIs to those in other short or long introns. RBP- binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools, and plotted as the mid-point of the annotated RBP-binding sites normalize as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. In each panel, binding-site distributions are shown for all FLEXIs (red), all other short introns (≤300 nt; blue), and all longer introns (>300 nt; black) bound by the indicated RBP. Peak positions in the density plots for FLEXI RNAs are indicated by red dashed. Density plots of the mid-points of RBP-binding sites for all 51 ENCODE eCLIP RBPs that bind ≥30 different FLEXIs are shown in
                                <a id="xref-fig-21-2" class="xref-fig" href="#F21">
                                 Fig. S13
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F21" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-24.large.jpg?width=800&height=600&carousel=1" title="Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns. (A) RBPs with an RNA splicing-related function. (B) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (C) RBPs that bind snoRNAs. (D) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns.</span> (<em>A</em>) RBPs with an RNA splicing-related function. (<em>B</em>) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (<em>C</em>) RBPs that bind snoRNAs. (<em>D</em>) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-24.medium.gif" width="339" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-24.medium.gif" width="339" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-25.large.jpg?width=800&height=600&carousel=1" title="Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns. (A) RBPs with an RNA splicing-related function. (B) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (C) RBPs that bind snoRNAs. (D) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns.</span> (<em>A</em>) RBPs with an RNA splicing-related function. (<em>B</em>) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (<em>C</em>) RBPs that bind snoRNAs. (<em>D</em>) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-25.medium.gif" width="319" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-25.medium.gif" width="319" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-26.large.jpg?width=800&height=600&carousel=1" title="Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns. (A) RBPs with an RNA splicing-related function. (B) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (C) RBPs that bind snoRNAs. (D) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns.</span> (<em>A</em>) RBPs with an RNA splicing-related function. (<em>B</em>) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (<em>C</em>) RBPs that bind snoRNAs. (<em>D</em>) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-26.medium.gif" width="440" height="422"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-26.medium.gif" width="440" height="422"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F21/graphic-26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S13.
                               </span>
                               <span class="caption-title">
                                Density plots showing the location of eCLIP-identified RBP-binding sites within FLEXIs and other classes of introns.
                               </span>
                               <p id="p-72" class="first-child">
                                (A) RBPs with an RNA splicing-related function. (B) RBPs with either a miRNA-related function or miRNA- and splicing-related functions. (C) RBPs that bind snoRNAs. (D) RBPs with other functions. RBP-binding sites overlapping with introns were identified by intersecting annotated RBP-binding sites with IDR analysis from the eCLIP datasets with the intron coordinates using BedTools and plotted as the mid-point of the annotated RBP-binding sites normalized as a percentage of intron length, with 0% and 100% corresponding to the 5’ and 3’ end of the intron, respectively. For each RBP, binding-site distributions are shown for FLEXIs (red), other short introns (≤300 nt; blue), and long introns (>300 nt; black). Vertical red dashed lines indicate the position of peaks in the density plots for FLEXI RNAs. Names of RBP are color coded by protein function: RNA splicing-related function, red; miRNA-related function, orange; both RNA splicing- and miRNA-related functions, blue; other functions, black. Blank spaces were left in cases where datasets were not available for an RBP in one of the two cell lines used to obtain the eCLIP datasets.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-28">
                              Knockdowns of 27 RBPs resulted in significant changes in alternative splicing (skipped exons (SE) or retained introns (RI)) of FLEXIs that contained a binding site for that RBP, but not FLEXIs that lacked such a binding site nor other short introns that were not detected as FLEXIs (
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              ,
                              <a id="xref-fig-20-3" class="xref-fig" href="#F20">
                               Fig. S12
                              </a>
                              , results summarized in
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ; p≤0.05, Kolmogorov-Smirnov test). In addition to proteins that function in RNA splicing, these included 17 proteins with no previously known RNA splicing-related function (AATF, DKC1, GRWD1, IGF2BP1, LARP4, LIN28B, LSM11, METAP2, NOLC1, PABPC4, RBM15, RBM22, RPS3, SUB1, UCHL5, XRN2, and YBX3). Knockdown of 9 of these proteins also affected mRNA levels of FLEXI host genes (see grids in
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). Collectively, knockdown of 34 of the 51 RBPs identified in the ENCODE eCLIP datasets as cross-linking to ≥30 FLEXIs resulted in significant changes in mRNA expression levels and/or alternative splicing of host genes for bound FLEXIs, indicating functional interactions with introns that give rise to FLEXI RNAs.
                             </p>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Mapping of RBP-binding sites in intron RNAs
                             </h3>
                             <p id="p-29">
                              To gain insight into how RBP binding might affect the production and function of FLEXI RNAs, we collected CLIP-seq reads containing intron sequences for the 53 RBPs that have a binding site in ≥30 FLEXIs in the ENCODE eCLIP and AGO1-4 and DICER PAR-CLIP datasets and mapped their binding sites in FLEXIs and in other categories of introns (
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              and
                              <a id="xref-fig-21-3" class="xref-fig" href="#F21">
                               Fig. S13
                              </a>
                              , results summarized in
                              <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). We found that 37 of these proteins bind at or near the 5’- and 3’- splice sites (SS), while 16 have binding sites more uniformly distributed throughout the body of the intron (I). For those proteins that bind at or near the splice sites, the midpoint of the annotated binding site peak in the FLEXI RNA was either at the splice site or immediately adjacent to it within the intron (SS-I) or flanking exon (SS-E). Notably, the RBPs whose binding sites were at or near the 5’- and 3’-splice sites included not only splicing factors, but also many proteins whose primary function is not RNA splicing, including AGO1-4, BCLAF1 (BCL2-2-associated transcription factor), DDX3X, DDX24, DDX55, FXR2, G3BP1, GEMIN5, GPKOW, GRWD1, IGF2BP, LARP4, LIN28B, LSM11, METAP2, NCBP2, PABPC4, PABPN1, PUM1, RPS3, SND1, SRSF1, SUB1 (Transcriptional Regulator), UCHL5, YBX3, ZNF622, and ZNF800). The RBPs that have binding sites more uniformly distributed throughout the FLEXI included AATF (Apoptosis Antagonizing Transcription Factor), DICER, DKC1 (Dyskerin), EFTUD2, KHSRP, NOLC1, PCBP1, PCBP2, PRPF4, RBFOX2, RBM5, RBM15, RBM22, SMNDC1, TIA1, and XRN2, a nuclear 5’ to 3’ exonuclease that degrades excised linear intron RNAs (
                              <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              and
                              <a id="xref-fig-21-4" class="xref-fig" href="#F21">
                               Fig. S13
                              </a>
                              ). In some cases, RBPs whose binding sites were localized near the 5’- and 3’-splice sites in long introns had binding sites more centrally localized in FLEXIs (e.g., RBM5;
                              <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              ; RBM15, TIA1, TIAL1;
                              <a id="xref-fig-21-5" class="xref-fig" href="#F21">
                               Fig. S13
                              </a>
                              ). The RBP-binding site locations suggest possible mechanisms by which individual RBPs may affect the production and/or stability of FLEXI RNAs.
                             </p>
                            </div>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Identification of RBP-binding sites enriched in agotrons, mirtrons, snoRNA-encoding FLEXIs, and evolutionarily conserved FLEXIs
                             </h3>
                             <p id="p-30">
                              Although FLEXIs as a whole are enriched in binding sites for spliceosomal proteins (
                              <a id="xref-fig-4-10" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                              ), it remained possible that subsets of FLEXI RNAs might be enriched in binding sites for RBPs that perform specialized biological functions. Such FLEXIs seemed likely to include agotrons and mirtrons, which bind AGO1-4 and DICER, proteins that recognize branch-point occluding secondary structures whose formation may require full or partial dissociation of the spliceosome (
                              <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">
                               Berezikov et al., 2007
                              </a>
                              ;
                              <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ). Consistent with this prediction, scatter plots confirmed that FLEXIs annotated as agotrons (n=61) or mirtrons (n=110) in the cellular RNA samples were significantly enriched in binding sites for AGO1-4 and DICER relative to those for spliceosomal or other proteins (significant enrichment defined as ≥2% of total RBP-binding sites; p ≤0.05 by Fisher’s exact test;
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 6A and B
                              </a>
                              , left panel). Density plots showed that the cellular FLEXI RNAs annotated as agotrons were a smaller, more homogeneously sized GC-rich subset of FLEXIs than were pre-miRNAs of annotated mirtrons (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig. 6A and B
                              </a>
                              , right panel).
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F6.large.jpg?width=800&height=600&carousel=1" title="Enrichment of RBP-binding sites for different RNA-binding proteins in structurally distinct subsets of FLEXI RNAs. Each panel shows scatter plots identifying RBPs whose binding sites are enriched in different subsets of FLEXI RNAs (left) and density plots comparing the characteristics of those FLEXIs to all other FLEXIs (right) in a merged dataset for the cellular RNA samples. (A) FLEXIs annotated as agotrons; (B) FLEXIs corresponding to pre-miRNAs of annotated mirtrons; (C) FLEXIs encoding an annotated snoRNA. Further analysis showed that that AATF cross-linked only to FLEXIs encoding C/D-box snoRNAs, while DKC1, which is a core H/ACA-box snoRNA binding protein (Massenet et al., 2017), cross-linked to FLEXIs encoding both H/ACA- and C/D-box snoRNAs. These cross-linking patterns raise the possibly that binding of DKC1 to FLEXIs encoding C/D-box snoRNAs is mediated by its interaction with AATF. (D) conserved FLEXIs with phastCons score ≥0.75; (E-J) FLEXIs bound by different bioinformatically identified clusters of RBPs whose binding sites were enriched in different subsets of FLEXI RNAs (see Fig. 7). The number of different FLEXIs comprising each subset is indicated in parentheses at the top. RBP-binding site annotations were from the ENCODE 150 RBP eCLIP dataset with irreproducible discovery rate (IDR) and AGO1-4 and DICER PAR- CLIP datasets. In the scatter plots, RBP-binding sites whose abundance was ≥2% of all RBP- binding sites and significantly different between the compared subset and all other FLEXIs (adjusted p≤0.05; calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled with the name of the RBP color coded by protein function as indicated at the bottom of the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset (red) to those of all other detected FLEXIs (black). p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Enrichment of RBP-binding sites for different RNA-binding proteins in structurally distinct subsets of FLEXI RNAs.</span> Each panel shows scatter plots identifying RBPs whose binding sites are enriched in different subsets of FLEXI RNAs (left) and density plots comparing the characteristics of those FLEXIs to all other FLEXIs (right) in a merged dataset for the cellular RNA samples. (<em>A</em>) FLEXIs annotated as agotrons; (<em>B</em>) FLEXIs corresponding to pre-miRNAs of annotated mirtrons; (<em>C</em>) FLEXIs encoding an annotated snoRNA. Further analysis showed that that AATF cross-linked only to FLEXIs encoding C/D-box snoRNAs, while DKC1, which is a core H/ACA-box snoRNA binding protein (Massenet et al., 2017), cross-linked to FLEXIs encoding both H/ACA- and C/D-box snoRNAs. These cross-linking patterns raise the possibly that binding of DKC1 to FLEXIs encoding C/D-box snoRNAs is mediated by its interaction with AATF. (<em>D</em>) conserved FLEXIs with phastCons score ≥0.75; (<em>E-J</em>) FLEXIs bound by different bioinformatically identified clusters of RBPs whose binding sites were enriched in different subsets of FLEXI RNAs (see Fig. 7). The number of different FLEXIs comprising each subset is indicated in parentheses at the top. RBP-binding site annotations were from the ENCODE 150 RBP eCLIP dataset with irreproducible discovery rate (IDR) and AGO1-4 and DICER PAR- CLIP datasets. In the scatter plots, RBP-binding sites whose abundance was ≥2% of all RBP- binding sites and significantly different between the compared subset and all other FLEXIs (adjusted p≤0.05; calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled with the name of the RBP color coded by protein function as indicated at the bottom of the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset (red) to those of all other detected FLEXIs (black). p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation). Independently generated scatter plots and density plots for each individual RBP that binds ≥30 different FLEXI RNAs are shown in Fig. S15.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F6.medium.gif" width="440" height="305"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F6.medium.gif" width="440" height="305"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <span class="caption-title">
                                Enrichment of RBP-binding sites for different RNA-binding proteins in structurally distinct subsets of FLEXI RNAs.
                               </span>
                               <p id="p-31" class="first-child">
                                Each panel shows scatter plots identifying RBPs whose binding sites are enriched in different subsets of FLEXI RNAs (left) and density plots comparing the characteristics of those FLEXIs to all other FLEXIs (right) in a merged dataset for the cellular RNA samples. (A) FLEXIs annotated as agotrons; (B) FLEXIs corresponding to pre-miRNAs of annotated mirtrons; (C) FLEXIs encoding an annotated snoRNA. Further analysis showed that that AATF cross-linked only to FLEXIs encoding C/D-box snoRNAs, while DKC1, which is a core H/ACA-box snoRNA binding protein (
                                <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                                 Massenet et al., 2017
                                </a>
                                ), cross-linked to FLEXIs encoding both H/ACA- and C/D-box snoRNAs. These cross-linking patterns raise the possibly that binding of DKC1 to FLEXIs encoding C/D-box snoRNAs is mediated by its interaction with AATF. (D) conserved FLEXIs with phastCons score ≥0.75; (E-J) FLEXIs bound by different bioinformatically identified clusters of RBPs whose binding sites were enriched in different subsets of FLEXI RNAs (see
                                <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                                 Fig. 7
                                </a>
                                ). The number of different FLEXIs comprising each subset is indicated in parentheses at the top. RBP-binding site annotations were from the ENCODE 150 RBP eCLIP dataset with irreproducible discovery rate (IDR) and AGO1-4 and DICER PAR- CLIP datasets. In the scatter plots, RBP-binding sites whose abundance was ≥2% of all RBP- binding sites and significantly different between the compared subset and all other FLEXIs (adjusted p≤0.05; calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled with the name of the RBP color coded by protein function as indicated at the bottom of the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset (red) to those of all other detected FLEXIs (black). p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation). Independently generated scatter plots and density plots for each individual RBP that binds ≥30 different FLEXI RNAs are shown in
                                <a id="xref-fig-23-2" class="xref-fig" href="#F23">
                                 Fig. S15
                                </a>
                                .
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-32">
                              Similar analyses showed that the FLEXIs encoding annotated snoRNAs were significantly enriched in bindings sites for DKC1 (Dyskerin) and NOLC1 (Nucleolar and Coiled Body Phosphoprotein 1, alias Nopp140), which have snoRNA-related functions (
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               Alawi and Lin, 2010
                              </a>
                              ;
                              <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">
                               Wang et al., 2002
                              </a>
                              ), along with several other proteins, including SMNDC1 (Survival of Motor Neuron Domain Containing 1, alias SPF30), which is better known for binding snRNAs, and AATF (Apoptosis Antagonizing Transcription Factor) (
                              <a id="xref-fig-6-4" class="xref-fig" href="#F6">
                               Fig. 6C
                              </a>
                              ). The latter protein was found previously to bind both DKC1 transcripts and snoRNAs, potentially linking the ribosome biosynthesis function of snoRNAs to the regulation of cell proliferation, apoptosis, and p53 activity (
                              <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                               Kaiser et al., 2019
                              </a>
                              ) (see legend
                              <a id="xref-fig-6-5" class="xref-fig" href="#F6">
                               Fig. 6C
                              </a>
                              for differences in RBP-binding sites in FLEXIs encoding H/ACA-box and CD-box snoRNAs).
                             </p>
                             <p id="p-33">
                              Finally, the 158 highly conserved FLEXIs (phastCons scores ≥ 0.75), which included a high proportion of alternatively spliced introns (see above), were significantly enriched in binding sites for splicing factor U2AF2 and splicing regulator TIAL1 as well as the transcriptional regulators BCLAF1 and ZNF622, potentially linking alternative splicing of these introns to transcriptional and other cell regulatory pathways (
                              <a id="xref-fig-6-6" class="xref-fig" href="#F6">
                               Fig. 6D
                              </a>
                              ;
                              <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">
                               Table S3
                              </a>
                              ), a role previously suggested for BCLAF1 (
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               Savage et al., 2014
                              </a>
                              ;
                              <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">
                               Vohhodina et al., 2017
                              </a>
                              ). Collectively, the above findings show that specific subsets of FLEXI RNAs are enriched in binding sites for RBPs that preform specialized biological functions relative to those for spliceosomal proteins and led us to wonder whether such enrichment might be a distinguishing feature of RBPs that bind or remain bound to excised intron RNAs after complete or partial dissociation from the spliceosome.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Computational identification of RBPs whose binding sites are co-enriched in different subsets of FLEXI RNAs
                             </h3>
                             <p id="p-34">
                              Encouraged by the above findings, we attempted to mathematically identify other RBPs whose binding sites were co-enriched in different subsets of FLEXIs RNAs without a priori assumptions about the function of the protein or bound FLEXIs. To do so, we first extracted subsets of FLEXIs with a CLIP-seq identified binding site for 47 of the 53 RBPs that bind ≥30 different FLEXIs in the combined CLIP-seq datasets, excluding six core spliceosomal proteins (AQR, BUD13, EFTUD2, PPIG, PRPF8, and SF3B4), whose abundant binding sites would otherwise complicate the analysis. We then identified which of the 126 RBPs that bind FLEXIs were coenriched in each of the 47 subsets by comparing the frequencies of different RBP-binding sites in each subset to those in all FLEXs (significant enrichment defined as abundance ≥2%, adjusted p- value ≤0.05). RBPs whose binding sites were significantly over- or under-represented in the tested subset and those that did not meet these criteria were key coded, and hierarchal clustering was performed using the Gower’s distance between the 47 subsets (
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               Gower, 1971
                              </a>
                              ). The results were displayed as a dendrogram heat map for FLEXIs in each of the four cellular RNA datasets (HEK-293T, K-562, HeLa S3, and UHHR;
                              <a id="xref-fig-22-1" class="xref-fig" href="#F22">
                               Fig. S14
                              </a>
                              ).
                             </p>
                             <div id="F22" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F22.large.jpg?width=800&height=600&carousel=1" title="Hierarchical clustering of RBPs based on patterns of over- and under-represented RBP-binding sites in FLEXI RNAs. Subsets of FLEXIs that have a CLIP-seq identified binding site for each of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs in merged TGIRT-seq datasets were extracted for each of the cellular RNA samples. Contingency tables comparing the frequency of RBP-binding sites in each of the 47 subsets in each of the cellular RNA samples were then generated for all 126 RBPs identified as having a binding site in a FLEXI RNA (124 RBPs from ENCODE eCLIP dataset plus AGO1-4 and DICER from PAR- CLIP dataset). Multiple binding sites for the same RBP in the same FLEXI were counted as one binding site, and p-values were calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure. RBP-binding sites were then key-coded as not significantly different or as significantly over- or under-represented (≥2% abundance, p≤0.05) in the tested subset of FLEXIs compared to all FLEXIs, and matrices of the Gower’s distance between the 47 subsets in each of the cellular RNA samples were calculated using the key-coded information and used as input for hierarchical clustering by the complete linkage clustering method (Defays, 1977). The results were displayed as a heat map to identify subsets of FLEXIs showing similar patterns of significantly over- and under-represented RBP-binding sites in each of the cellular RNA samples, with the color scale based on -log10 transformed adjusted p-values. Significantly co-enriched RBPs are indicated by an X in the heatmap box, and clusters of co-enriched RBPs are delineated in larger boxes. Proteins names are color coded by function as indicated in the Figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Hierarchical clustering of RBPs based on patterns of over- and under-represented RBP-binding sites in FLEXI RNAs.</span> Subsets of FLEXIs that have a CLIP-seq identified binding site for each of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs in merged TGIRT-seq datasets were extracted for each of the cellular RNA samples. Contingency tables comparing the frequency of RBP-binding sites in each of the 47 subsets in each of the cellular RNA samples were then generated for all 126 RBPs identified as having a binding site in a FLEXI RNA (124 RBPs from ENCODE eCLIP dataset plus AGO1-4 and DICER from PAR- CLIP dataset). Multiple binding sites for the same RBP in the same FLEXI were counted as one binding site, and p-values were calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure. RBP-binding sites were then key-coded as not significantly different or as significantly over- or under-represented (≥2% abundance, p≤0.05) in the tested subset of FLEXIs compared to all FLEXIs, and matrices of the Gower’s distance between the 47 subsets in each of the cellular RNA samples were calculated using the key-coded information and used as input for hierarchical clustering by the complete linkage clustering method (Defays, 1977). The results were displayed as a heat map to identify subsets of FLEXIs showing similar patterns of significantly over- and under-represented RBP-binding sites in each of the cellular RNA samples, with the color scale based on -log<sub>10</sub> transformed adjusted p-values. Significantly co-enriched RBPs are indicated by an X in the heatmap box, and clusters of co-enriched RBPs are delineated in larger boxes. Proteins names are color coded by function as indicated in the Figure.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S14." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F22.medium.gif" width="369" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S14." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F22.medium.gif" width="369" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S14." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S14.
                               </span>
                               <span class="caption-title">
                                Hierarchical clustering of RBPs based on patterns of over- and under-represented RBP-binding sites in FLEXI RNAs.
                               </span>
                               <p id="p-73" class="first-child">
                                Subsets of FLEXIs that have a CLIP-seq identified binding site for each of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs in merged TGIRT-seq datasets were extracted for each of the cellular RNA samples. Contingency tables comparing the frequency of RBP-binding sites in each of the 47 subsets in each of the cellular RNA samples were then generated for all 126 RBPs identified as having a binding site in a FLEXI RNA (124 RBPs from ENCODE eCLIP dataset plus AGO1-4 and DICER from PAR- CLIP dataset). Multiple binding sites for the same RBP in the same FLEXI were counted as one binding site, and p-values were calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure. RBP-binding sites were then key-coded as not significantly different or as significantly over- or under-represented (≥2% abundance, p≤0.05) in the tested subset of FLEXIs compared to all FLEXIs, and matrices of the Gower’s distance between the 47 subsets in each of the cellular RNA samples were calculated using the key-coded information and used as input for hierarchical clustering by the complete linkage clustering method (
                                <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                                 Defays, 1977
                                </a>
                                ). The results were displayed as a heat map to identify subsets of FLEXIs showing similar patterns of significantly over- and under-represented RBP-binding sites in each of the cellular RNA samples, with the color scale based on -log
                                <sub>
                                 10
                                </sub>
                                transformed adjusted p-values. Significantly co-enriched RBPs are indicated by an X in the heatmap box, and clusters of co-enriched RBPs are delineated in larger boxes. Proteins names are color coded by function as indicated in the Figure.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-35">
                              We thus identified six clusters of RBPs whose binding sites were co-enriched in different subsets of FLEXI RNAs (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). Each RBP cluster consists of two or more proteins whose binding sites were significantly enriched in the subset of FLEXIs comprising the cluster in all four of the cellular RNA datasets plus additional proteins that met the abundance and significance criteria for association with the cluster in some cell types but not others. Scatter plots confirmed that binding sites for the identified proteins were significantly enriched relative to those for most or all of the six core spliceosomal proteins in the subsets of FLEXIs comprising the clusters (
                              <a id="xref-fig-6-7" class="xref-fig" href="#F6">
                               Fig. 6E-J
                              </a>
                              ). Independently generated scatter plots for the subset of FLEXIs with a binding site for each individual RBP in the cluster confirmed the co-enrichment of binding sites for other RBPs in the same cluster (
                              <a id="xref-fig-23-3" class="xref-fig" href="#F23">
                               Fig. S15
                              </a>
                              ).
                             </p>
                             <div id="F23" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-28.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-28.medium.gif" width="312" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-28.medium.gif" width="312" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-28.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-28.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-29.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-29.medium.gif" width="306" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-29.medium.gif" width="306" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-29.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-29.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-30.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-30.medium.gif" width="310" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-30.medium.gif" width="310" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-30.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-30.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-31.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-31.medium.gif" width="298" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-31.medium.gif" width="298" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-31.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-31.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-32.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-32.medium.gif" width="309" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-32.medium.gif" width="309" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-32.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-32.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-33.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-33.medium.gif" width="315" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-33.medium.gif" width="315" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-33.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-33.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-34.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-34.medium.gif" width="314" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-34.medium.gif" width="314" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-34.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-34.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-35.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-35.medium.gif" width="316" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-35.medium.gif" width="316" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-35.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-35.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-36.large.jpg?width=800&height=600&carousel=1" title="Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup">Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-36.medium.gif" width="371" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-36.medium.gif" width="371" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                 <ul class="highwire-figure-links inline">
                                  <li class="download-fig first">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-36.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                    Download figure
                                   </a>
                                  </li>
                                  <li class="new-tab last">
                                   <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F23/graphic-36.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                    Open in new tab
                                   </a>
                                  </li>
                                 </ul>
                                </div>
                               </div>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S15.
                               </span>
                               <p id="p-74" class="first-child">
                                Scatter plots and density plots comparing RBP-binding sites and characteristics of subsets of FLEXIs bound by each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. Plots are shown for each protein in Clusters I to VI, followed by the remainder of the 47 RBPs that have binding sites in ≥30 different FLEXI RNAs. In the scatter plots, RBP- binding sites whose relative abundance was significantly over-represented or under-represented in the tested subset of FLEXI compared to other FLEXIs (≥2% abundance, p≤0.05 calculated by Fisher’s exact test and adjusted by the Benjamini-Hochberg procedure) are labeled by the name of the protein color coded by protein function as indicated in the Figure. The density distribution plots compare the length, GC content, and MFE for the most stable secondary structure predicted by RNAfold for the FLEXI RNAs comprising the subset of FLEXIs containing a binding site for the (red) compared to all other FLEXIs (black). The number of FLEXIs comprising each subset is indicated in parentheses next to the name of the RBP. p-values are shown at the top left of those density plots in which the distribution for the subset of FLEXIs differed significantly from other FLEXIs (p<0.01 and FDR≤0.05 as determined by 1,000 Monte-Carlo simulation).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F7.large.jpg?width=800&height=600&carousel=1" title="Identification of clusters of RBPs whose binding sites were co-enriched in subsets of FLEXI RNAs. (A) Hierarchical clustering of FLEXI RNAs based on patterns of over- and under-represented RBP-binding sites. Hierarchical clustering was done as described in Results and Methods and in Fig. S14. Clusters I to VI are delineated at the top of the Figure, and the names of the RBPs comprising the cluster are shown at the bottom of the Figure. The proteins that met the abundance and significance criteria for association with the clusters in HEK-293T, HeLa S3, K-562, and UHHR are indicated by filled boxes at the top. The grids below summarize RBP- binding site and protein function information obtained in this study from the analysis of public CLIP-seq and knockdown datasets followed by protein function and cellular localization information from (Van Nostrand et al., 2020) supplemented by the mammalian RNA granule and stress granule protein databases (Nunes et al., 2019; Youn et al., 2019) and the UniProt database (The UniProt Consortium, 2018) (see legend Fig. 4A for details). (B) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed with DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Identification of clusters of RBPs whose binding sites were co-enriched in subsets of FLEXI RNAs.</span> (<em>A</em>) Hierarchical clustering of FLEXI RNAs based on patterns of over- and under-represented RBP-binding sites. Hierarchical clustering was done as described in Results and Methods and in Fig. S14. Clusters I to VI are delineated at the top of the Figure, and the names of the RBPs comprising the cluster are shown at the bottom of the Figure. The proteins that met the abundance and significance criteria for association with the clusters in HEK-293T, HeLa S3, K-562, and UHHR are indicated by filled boxes at the top. The grids below summarize RBP- binding site and protein function information obtained in this study from the analysis of public CLIP-seq and knockdown datasets followed by protein function and cellular localization information from (Van Nostrand et al., 2020) supplemented by the mammalian RNA granule and stress granule protein databases (Nunes et al., 2019; Youn et al., 2019) and the UniProt database (The UniProt Consortium, 2018) (see legend Fig. 4A for details). (<em>B</em>) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed with DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F7.medium.gif" width="357" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F7.medium.gif" width="357" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <span class="caption-title">
                                Identification of clusters of RBPs whose binding sites were co-enriched in subsets of FLEXI RNAs.
                               </span>
                               <p id="p-36" class="first-child">
                                (A) Hierarchical clustering of FLEXI RNAs based on patterns of over- and under-represented RBP-binding sites. Hierarchical clustering was done as described in Results and Methods and in
                                <a id="xref-fig-22-2" class="xref-fig" href="#F22">
                                 Fig. S14
                                </a>
                                . Clusters I to VI are delineated at the top of the Figure, and the names of the RBPs comprising the cluster are shown at the bottom of the Figure. The proteins that met the abundance and significance criteria for association with the clusters in HEK-293T, HeLa S3, K-562, and UHHR are indicated by filled boxes at the top. The grids below summarize RBP- binding site and protein function information obtained in this study from the analysis of public CLIP-seq and knockdown datasets followed by protein function and cellular localization information from (
                                <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">
                                 Van Nostrand et al., 2020
                                </a>
                                ) supplemented by the mammalian RNA granule and stress granule protein databases (
                                <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">
                                 Nunes et al., 2019
                                </a>
                                ;
                                <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">
                                 Youn et al., 2019
                                </a>
                                ) and the UniProt database (
                                <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">
                                 The UniProt Consortium, 2018
                                </a>
                                ) (see legend
                                <a id="xref-fig-4-11" class="xref-fig" href="#F4">
                                 Fig. 4A
                                </a>
                                for details). (B) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed with DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-37">
                              Clusters V and VI (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ) recapitulated and extended the findings above for agotron/mirtron and snoRNA-encoding FLEXIs. Cluster VI consists of AGO1-4 and DICER, which bind overlapping subsets of 479 GC-rich FLEXIs (
                              <a id="xref-fig-6-8" class="xref-fig" href="#F6">
                               Fig. 6J
                              </a>
                              ). Notably, only 23 of the 250 FLEXIs that have a binding site for AGO1-4 in the AGO1-4 PAR-CLIP dataset corresponded to an annotated agotron (
                              <a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ), and only 44 of the 308 FLEXIs that have a binding site for DICER in the DICER PAR-CLIP dataset corresponded to the pre-miRNA of an annotated mirtron (
                              <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">
                               Wen et al., 2015
                              </a>
                              ). The other FLEXIs bound by these proteins could be unannotated agotrons or mirtrons, could be cleaved by DICER to yield unannotated short regulatory RNAs, or could reflect binding of AGO1-4 and DICER to structured introns that could evolve into agotrons or mirtrons, a possibility suggested previously for mirtrons (
                              <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">
                               Ruby et al., 2007
                              </a>
                              ).
                             </p>
                             <p id="p-38">
                              Cluster V consists of four proteins identified above as binding snoRNA-encoding FLEXIs (DKC1, NOLC1, AATF, and SMDC1), with the first three satisfying the abundance and significance criteria for association with this cluster in all four of the cellular RNA samples and SMDC1 in only two of the four cellular RNA samples (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). Collectively, these four proteins have binding sites in 160 FLEXIs, only 43 of which ENCODE annotated snoRNAs. The FLEXIs bound by these proteins are longer and have lower GC% content than other FLEXIs, with many having stable predicted secondary structures that might include features recognized by the snoRNA- and snRNA-binding proteins comprising the clusters (
                              <a id="xref-fig-6-9" class="xref-fig" href="#F6">
                               Fig. 6I
                              </a>
                              ).
                             </p>
                             <p id="p-39">
                              Cluster III and IV each consist of two splicing-related proteins (splicing regulators TIA1 and TIAL1 for cluster III and splicing factors U2AF1 an U2AF2 for cluster IV), whose binding sites were significantly enriched relative to those for core spliceosomal proteins in the FLEXIs comprising the cluster in all four cell types, with splicing regulator KHSRP satisfying the abundance and significance criteria for association with this cluster only in HEK-293T cells (
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). The RBPs in clusters III and IV have binding sites in 100 and 427 FLEXIs, respectively, which collectively have significantly lower GC content than other FLEXIs (
                              <a id="xref-fig-6-10" class="xref-fig" href="#F6">
                               Fig. 6G and H
                              </a>
                              ). Four of these proteins (TIA1, TIAL1, U2AF1, and U2AF2) were identified above as impacting FLEXI host gene mRNAs expression levels and/or alternative splicing of bound FLEXIs in knockdown datasets, and three (TIAL1, U2AF1, and U2AF2) were found to have binding sites significantly enriched in FLEXIs with high phastCons scores, many of which are alternatively spliced to generate different protein isoforms. Four of the five proteins in these clusters (TIA1, TIAL1, U2AF1, and KHSRP;
                              <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ) have been found associated with stress granules (
                              <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">
                               Nunes et al., 2019
                              </a>
                              ), possibly reflecting a post-splicing destination for FLEXIs bound by these proteins.
                             </p>
                             <p id="p-40">
                              Clusters I and II were the largest in terms of numbers of proteins and most interesting in revealing novel RBP-intron associations. Cluster I is comprised of two RBPs (LARP4 and PABPC4), which satisfy the abundance and significance criteria for association with the cluster in all four cellular RNA samples, and six other proteins: (Transcription Regulator SUB1, DDX3X, RPS3, NCBP2, DDX55, and Methionine Aminopeptidase 2 (METAP2)), which satisfy these criteria in HEK-293T cells but not in some of the other cell types (
                              <a id="xref-fig-7-7" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). These eight proteins have binding sites in 386 FLEXIs, with all eight binding at or near the 5’- and 3’-splice sites and knock down of seven impacting expression of FLEXI hosts genes and/or alternative splicing of bound FLEXI RNAs (
                              <a id="xref-fig-5-10" class="xref-fig" href="#F5">
                               Fig. 5
                              </a>
                              ,
                              <a id="xref-fig-7-8" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              , and Figs. S11-S13). LARP4 and PABP were found previously to function together to regulate the stability and translation of ribosomal protein mRNAs (
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               Mattijssen et al., 2017
                              </a>
                              ), and the remaining six proteins also have mRNA synthesis or translation-related functions and/or could link these functions to other cellular regulatory processes (
                              <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">
                               Table S3
                              </a>
                              ). All of the RBPs in this cluster except for SUB1 are present in the cytoplasm, consistent with a translation-related function (
                              <a id="xref-fig-7-9" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ).
                             </p>
                             <p id="p-41">
                              Finally, cluster II is comprised of five proteins (BCLAF1, UCHL5, ZNF622, TRA2A, ZNF800) that satisfy the abundance and significance criteria for association with this cluster in all of the cellular RNA sample types, two other proteins (GRWD1 and PUM1) that satisfy these criteria in three of the four cellular RNA sample types, and two (DDX24 and FXR2) that satisfy these criteria in only one of the cellular RNA sample types (
                              <a id="xref-fig-7-10" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). Collectively, these nine proteins have binding sites in 976 FLEXIs, with all nine binding near the 5’- and 3’-splice sites. Three of these proteins (UCHL5, ZNF622, and GRWD1) impacted mRNA expression levels of FLEXI host genes or alternative splicing at bound FLEXIs in knockdown datasets, and two others (BCLAF1 and ZNF622) have binding sites enriched in FLEXIs with high phastCons scores, many of which are alternatively spliced introns (
                              <a id="xref-fig-6-11" class="xref-fig" href="#F6">
                               Fig. 6D
                              </a>
                              ). Eight of these proteins are present in the cytoplasm, with three (UCHL5, ZNF622, and TRA2A) distinguished from other RBPs in the clusters as being associated with the cytoskeleton, microtubules, actin, or the cell cortex.
                             </p>
                             <p id="p-42">
                              Simulations using ten different subsets of 2,000 randomly selected other (i.e., non-FLEXI) short or long introns identified many of the same protein clusters, but in most cases less clearly defined than those identified for FLEXIs and with additional protein clusters not seen for FLEXIs (
                              <a id="xref-fig-24-1" class="xref-fig" href="#F24">
                               Fig. S16A
                              </a>
                              ). Cluster IV (U2AF1/U2AF2) was found in all simulations for other short and long intron, while cluster III (TIA1/TIAL1) was found in all simulations for non-FLEXI short introns, but only 40% of the simulations for long introns. Clusters I and II were found in ≥90% of the simulations for other short and long introns, but with only two proteins found consistently in each cluster (LARP4 and PABPC4 for cluster I and UCHL5 and ZNF622 for cluster 2). Three of the cluster V proteins (AATF/DKC1/NOLC1) were found together in 90% of simulations for other short introns but in only 50% of simulations for long introns, while cluster I (AGO1-4/DICER) did not appear in any simulations for non-FLEXI short introns and in only 40% of the simulations for long introns. Four additional protein clusters (PCBP1/PCBP2, METAP2/YBX3, LSM11/PABPN1, RBFOX2/PHRP4) were found in ≥60% of simulations for other short introns but less frequently or not at all in FLEXIs or long introns. These findings suggest both common and different protein associations for the three classes of introns. The clearer definition of the six clusters identified for FLEXIs may reflect that the proteins in these clusters bind together in the same short, relatively stable excised intron RNAs rather than to fragments of more rapidly turning over introns.
                             </p>
                             <div id="F24" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F24.large.jpg?width=800&height=600&carousel=1" title="Identification of clusters of RBPs whose binding sites are enriched in randomly selected subsets of other short and long introns. (A) Hierarchical clustering was carried out as in Fig. 7 for ten sets of 2,000 randomly selected other (non-FLEXI) short introns and long introns (Sim1-10). The clusters of RBPs enriched in subsets of FLEXI RNAs in HEK-293T, HeLa S3, K-562, and UHHR are shown at the top for comparison. (B) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed using DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Identification of clusters of RBPs whose binding sites are enriched in randomly selected subsets of other short and long introns. (<em>A</em>) Hierarchical clustering was carried out as in Fig. 7 for ten sets of 2,000 randomly selected other (non-FLEXI) short introns and long introns (Sim1-10). The clusters of RBPs enriched in subsets of FLEXI RNAs in HEK-293T, HeLa S3, K-562, and UHHR are shown at the top for comparison. (<em>B</em>) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed using DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S16." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F24.medium.gif" width="440" height="435"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S16." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F24.medium.gif" width="440" height="435"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S16." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S16.
                               </span>
                               <p id="p-75" class="first-child">
                                Identification of clusters of RBPs whose binding sites are enriched in randomly selected subsets of other short and long introns. (A) Hierarchical clustering was carried out as in
                                <a id="xref-fig-7-17" class="xref-fig" href="#F7">
                                 Fig. 7
                                </a>
                                for ten sets of 2,000 randomly selected other (non-FLEXI) short introns and long introns (Sim1-10). The clusters of RBPs enriched in subsets of FLEXI RNAs in HEK-293T, HeLa S3, K-562, and UHHR are shown at the top for comparison. (B) A heatmap of GO terms enriched in host genes of FLEXI RNAs containing binding sites for different RBPs. GO enrichment analysis was performed using DAVID bioinformatics tools, using all FLEXI host genes as the background. Similar results were obtained using all ENSEMBL-annotated genes as background (not shown).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              FLEXI RNAs associated with RBP clusters originate from host genes with related biological functions
                             </h3>
                             <p id="p-43">
                              To determine if the host genes encoding FLEXIs in the six identified clusters might be functionally related, we analyzed gene ontology (GO) terms of host genes bound by the same RBPs using DAVID bioinformatic tools and displayed the results as a heat map for each of the RBPs (
                              <a id="xref-fig-7-11" class="xref-fig" href="#F7">
                               Fig. 7B
                              </a>
                              ). Overall, the heat map for GO terms related to host gene function was remarkably congruent with the identified RBP clusters. The host genes encoding FLEXIs bound by AGO1-4 and DICER (Cluster VI) encode proteins involved in translation and focal adhesion as well as structural components of the ribosome and proteins found in nuclear speckles, while those encoding FLEXIs in cluster II, III, and IV encode proteins involved in RNA splicing, gene expression and mRNA processing, with the host genes for FLEXIs bound by some proteins cluster II also involved in cell-cell adhesion. The host genes encoding FLEXIs bound by the snoRNA-related proteins in cluster V have functions related to ribosomes, translation, and rRNA processing, with the host genes for FLEXIs bound SMNDC1 enriched in additional GO terms. Notably, the GO terms for the host genes encoding FLEXIs in clusters I, which have binding sites LARP4 and PABPC4, which function together in regulating the stability and translation of ribosomal protein mRNAs, largely overlap those for the host genes for the snoRNA-related FLEXIs in cluster V and were similarly enriched in genes encoding ribosomal proteins (12% and 10.6% compared to 1.4% in all FLEXIs; p<0.001), suggesting that cluster I FLEXIs may function in a parallel pathway for regulating ribosome biogenesis. Collectively, these findings indicate that the host genes encoding FLEXI RNAs that bind the same subsets of RBPs are functionally related and could be coordinately regulated to produce different subsets FLEXI RNPs.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              FLEXI as potential cancer biomarkers
                             </h3>
                             <p id="p-44">
                              The cell type-specific expression patterns of FLEXI RNAs suggested that they might be useful as RNA biomarkers. As an initial test of this possibility, we compared FLEXI RNAs in matched tumor and neighboring healthy tissue from two breast cancer patients (patients A and B; PR
                              <sup>
                               +
                              </sup>
                              , ER
                              <sup>
                               +
                              </sup>
                              , HER2
                              <sup>
                               -
                              </sup>
                              and PR unknown, ER
                              <sup>
                               -
                              </sup>
                              , HER2
                              <sup>
                               -
                              </sup>
                              , respectively) and two breast cancer cell lines (MDA-MB- 231 and MCF7). UpSet plots for the relatively abundant FLEXIs (≥0.01 RPM) showed hundreds of differences in FLEXI RNAs and FLEXI host genes between the cancer and healthy samples (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. 8A
                              </a>
                              ). Further, scatter plots comparing the FLEXI RNAs detected in the matched healthy and tumor samples from patients A and B identified multiple candidate FLEXI RNA biomarkers (5 and 43 in patients A and B, respectively) that were detected at ≥0.01 RPM in all replicate libraries from the cancer patient tissue, but not detected in the matched healthy tissue (red dots and protein names in red in
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Fig. 8B
                              </a>
                              ). As for the previous comparisons between different cellular RNA samples (
                              <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              ), the scatter plots showed a wider spectrum of differences than did those for mRNAs from the same host genes (
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Fig. 8B
                              </a>
                              ). Notably, FLEXI RNAs were more abundant in the highly metastatic MDA-MB-231 cells than in MCF-7 cells, with density plot profiles suggesting upregulation of snoRNA-encoding FLEXIs and agotrons in MDA-MB-231 (
                              <a id="xref-fig-25-1" class="xref-fig" href="#F25">
                               Fig. S17
                              </a>
                              ).
                             </p>
                             <div id="F25" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F25.large.jpg?width=800&height=600&carousel=1" title="Density distribution plots of the abundance of different categories of FLEXI RNAs in MDA-MB-231 and MCF-7 cells. FLEXI RNA abundance (RPM) was calculated from combined datasets for technical replicates of each sample. Different categories of FLEXI RNAs are color coded as indicated in the Figure. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (arrows) in the same samples are shown for comparison." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Density distribution plots of the abundance of different categories of FLEXI RNAs in MDA-MB-231 and MCF-7 cells.</span> FLEXI RNA abundance (RPM) was calculated from combined datasets for technical replicates of each sample. Different categories of FLEXI RNAs are color coded as indicated in the Figure. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (arrows) in the same samples are shown for comparison.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S17." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F25.medium.gif" width="440" height="155"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S17." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F25.medium.gif" width="440" height="155"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S17." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S17.
                               </span>
                               <span class="caption-title">
                                Density distribution plots of the abundance of different categories of FLEXI RNAs in MDA-MB-231 and MCF-7 cells.
                               </span>
                               <p id="p-76" class="first-child">
                                FLEXI RNA abundance (RPM) was calculated from combined datasets for technical replicates of each sample. Different categories of FLEXI RNAs are color coded as indicated in the Figure. The abundance distribution of annotated mature snoRNAs (dashed yellow line), and the abundance of different sncRNAs (arrows) in the same samples are shown for comparison.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F8.large.jpg?width=800&height=600&carousel=1" title="FLEXI RNAs in breast cancer tumors and cell lines. (A) UpSet plots of FLEXI RNAs detected at ≥0.01 RPM and their host genes in unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A (PR+, ER+, HER2-) and B (PR unknown, ER-, HER2-) and breast cancer cell lines MDA-MB-231 and MCF7. Different FLEXI RNAs from the same host gene were aggregated into one entry for that gene. FLEXI RNAs and FLEXI host genes are listed below some sample groups in descending order of RPM with the range of RPM indicated below each column. (B) Scatter plots comparing log2 transformed RPM of FLEXI RNA reads in unfragmented RNA preparations (left plot in each pair) and all reads mapping to FLEXI host genes (right plot in each pair) in chemically fragmented RNA preparations from cancer and healthy breast tissues from patients A and B. FLEXI RNAs present at ≥0.01 RPM and detected in all replicate libraries from the cancer tissue but not in matched healthy tissue are indicated in red with up to the 10 most abundant FLEXIs satisfying these criteria listed to the right. An additional 33 FLEXIs meeting these criteria in patient B are listed in the Supplemental file. In panels A and B, FLEXIs originating from annotated oncogenes and oncogenes that encode FLEXI RNAs are indicated with an asterisk. (C) GO enrichment analysis of genes encoding detected FLEXI RNAs in 50 hallmark gene sets (MSigDB) in combined patient A + B healthy tissue and cancer sample datasets. Names of pathways significantly enriched (p≤0.05) in all or at least one cancer sample are in red and orange, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">FLEXI RNAs in breast cancer tumors and cell lines.</span> (<em>A</em>) UpSet plots of FLEXI RNAs detected at ≥0.01 RPM and their host genes in unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A (PR<sup>+</sup>, ER<sup>+</sup>, HER2<sup>-</sup>) and B (PR unknown, ER<sup>-</sup>, HER2<sup>-</sup>) and breast cancer cell lines MDA-MB-231 and MCF7. Different FLEXI RNAs from the same host gene were aggregated into one entry for that gene. FLEXI RNAs and FLEXI host genes are listed below some sample groups in descending order of RPM with the range of RPM indicated below each column. (<em>B</em>) Scatter plots comparing log<sub>2</sub> transformed RPM of FLEXI RNA reads in unfragmented RNA preparations (left plot in each pair) and all reads mapping to FLEXI host genes (right plot in each pair) in chemically fragmented RNA preparations from cancer and healthy breast tissues from patients A and B. FLEXI RNAs present at ≥0.01 RPM and detected in all replicate libraries from the cancer tissue but not in matched healthy tissue are indicated in red with up to the 10 most abundant FLEXIs satisfying these criteria listed to the right. An additional 33 FLEXIs meeting these criteria in patient B are listed in the Supplemental file. In panels A and B, FLEXIs originating from annotated oncogenes and oncogenes that encode FLEXI RNAs are indicated with an asterisk. (<em>C</em>) GO enrichment analysis of genes encoding detected FLEXI RNAs in 50 hallmark gene sets (MSigDB) in combined patient A + B healthy tissue and cancer sample datasets. Names of pathways significantly enriched (p≤0.05) in all or at least one cancer sample are in red and orange, respectively.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F8.medium.gif" width="372" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F8.medium.gif" width="372" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8.
                               </span>
                               <span class="caption-title">
                                FLEXI RNAs in breast cancer tumors and cell lines.
                               </span>
                               <p id="p-45" class="first-child">
                                (A) UpSet plots of FLEXI RNAs detected at ≥0.01 RPM and their host genes in unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A (PR
                                <sup>
                                 +
                                </sup>
                                , ER
                                <sup>
                                 +
                                </sup>
                                , HER2
                                <sup>
                                 -
                                </sup>
                                ) and B (PR unknown, ER
                                <sup>
                                 -
                                </sup>
                                , HER2
                                <sup>
                                 -
                                </sup>
                                ) and breast cancer cell lines MDA-MB-231 and MCF7. Different FLEXI RNAs from the same host gene were aggregated into one entry for that gene. FLEXI RNAs and FLEXI host genes are listed below some sample groups in descending order of RPM with the range of RPM indicated below each column. (B) Scatter plots comparing log
                                <sub>
                                 2
                                </sub>
                                transformed RPM of FLEXI RNA reads in unfragmented RNA preparations (left plot in each pair) and all reads mapping to FLEXI host genes (right plot in each pair) in chemically fragmented RNA preparations from cancer and healthy breast tissues from patients A and B. FLEXI RNAs present at ≥0.01 RPM and detected in all replicate libraries from the cancer tissue but not in matched healthy tissue are indicated in red with up to the 10 most abundant FLEXIs satisfying these criteria listed to the right. An additional 33 FLEXIs meeting these criteria in patient B are listed in the Supplemental file. In panels A and B, FLEXIs originating from annotated oncogenes and oncogenes that encode FLEXI RNAs are indicated with an asterisk. (C) GO enrichment analysis of genes encoding detected FLEXI RNAs in 50 hallmark gene sets (MSigDB) in combined patient A + B healthy tissue and cancer sample datasets. Names of pathways significantly enriched (p≤0.05) in all or at least one cancer sample are in red and orange, respectively.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-46">
                              FLEXI host genes in all four cancer samples were significantly enriched (p≤0.05) for hallmark gene sets (
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               Liberzon et al., 2015
                              </a>
                              ) that are dysregulated in many cancers, including glycolysis, G2M checkpoint, UV response up, and PI3K/AKT/MTOR signaling;
                              <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                               Fig. 8C
                              </a>
                              , pathway names in red). Gene sets that were significantly enriched in one or more of the cancer samples but not in the healthy controls included mitotic spindle, MYC targets V1 and V2, estrogen response early and late, androgen response, oxidative phosphorylation, mTORC1 signaling, apical junction, and cholesterol homeostasis (
                              <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                               Fig. 8C
                              </a>
                              , pathway names in orange).
                             </p>
                             <p id="p-47">
                              Only a small number of the FLEXIs identified as potential biomarkers in the UpSet and scatter plots corresponded to annotated oncogenes (asterisks in
                              <a id="xref-fig-8-6" class="xref-fig" href="#F8">
                               Fig. 8A and B
                              </a>
                              ). Additional UpSet plots for FLEXI RNAs originating from known oncogenes identified 169 that were up upregulated and 81 that were downregulated ≥2-fold in one or more of the cancer samples compared to the healthy controls (
                              <a id="xref-fig-26-1" class="xref-fig" href="#F26">
                               Fig. S18A
                              </a>
                              and B). Similar patterns of up- and down regulation in the cancer samples were seen for tumor suppressor genes (
                              <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">
                               Zhao et al., 2016
                              </a>
                              ) (
                              <a id="xref-fig-26-2" class="xref-fig" href="#F26">
                               Fig. S18C
                              </a>
                              and D). These findings reflect that FLEXI RNA abundance is dictated not only by transcription but also by alternative splicing and intron RNA turnover. They do not preclude oncogene or tumor suppressor gene FLEXIs from serving as biomarkers for a specific cancer so long as they are reproducibly up or downregulated in that cancer. However, most of the FLEXIs that best discriminated between the cancer and healthy samples arose from genes that were strongly up or downregulated in response to oncogenesis but not oncogenes that drive this process.
                             </p>
                             <div id="F26" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F26.large.jpg?width=800&height=600&carousel=1" title="Oncogene FLEXI RNAs in breast cancer tumors and cell lines. (A and B) UpSet plots of upregulated (A) and downregulated (B) oncogene FLEXI RNAs in rRNA-depleted unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A and B and breast cancer cell lines MDA-MB-231 and MCF7. (C and D) UpSet plots of upregulated (C) and downregulated (D) tumor suppressor gene (TSG) FLEXI RNAs in the same unfragmented RNA preparations. Up and down regulated FLEXI RNAs were defined as those with an RPM- fold change ≥2. The most abundant oncogene or tumor suppressor gene FLEXI RNAs (up to a limit of 10) are listed below some sample groups, with the range of RPM values indicated in parentheses at the bottom. Different FLEXIs originating from the FASN oncogene (highlighted in red) showed notably different patterns of up- or down-regulation in the same cancer samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-568647098" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Oncogene FLEXI RNAs in breast cancer tumors and cell lines. (<em>A</em> and <em>B</em>) UpSet plots of upregulated (A) and downregulated (B) oncogene FLEXI RNAs in rRNA-depleted unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A and B and breast cancer cell lines MDA-MB-231 and MCF7. (<em>C</em> and <em>D</em>) UpSet plots of upregulated (C) and downregulated (D) tumor suppressor gene (TSG) FLEXI RNAs in the same unfragmented RNA preparations. Up and down regulated FLEXI RNAs were defined as those with an RPM- fold change ≥2. The most abundant oncogene or tumor suppressor gene FLEXI RNAs (up to a limit of 10) are listed below some sample groups, with the range of RPM values indicated in parentheses at the bottom. Different FLEXIs originating from the <em>FASN</em> oncogene (highlighted in red) showed notably different patterns of up- or down-regulation in the same cancer samples.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S18." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F26.medium.gif" width="440" height="407"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S18." src="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F26.medium.gif" width="440" height="407"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S18." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/09/2020.09.07.285114/F26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S18.
                               </span>
                               <p id="p-77" class="first-child">
                                Oncogene FLEXI RNAs in breast cancer tumors and cell lines. (A and B) UpSet plots of upregulated (A) and downregulated (B) oncogene FLEXI RNAs in rRNA-depleted unfragmented RNA preparations from matched cancer/healthy breast tissues from patients A and B and breast cancer cell lines MDA-MB-231 and MCF7. (C and D) UpSet plots of upregulated (C) and downregulated (D) tumor suppressor gene (TSG) FLEXI RNAs in the same unfragmented RNA preparations. Up and down regulated FLEXI RNAs were defined as those with an RPM- fold change ≥2. The most abundant oncogene or tumor suppressor gene FLEXI RNAs (up to a limit of 10) are listed below some sample groups, with the range of RPM values indicated in parentheses at the bottom. Different FLEXIs originating from the FASN oncogene (highlighted in red) showed notably different patterns of up- or down-regulation in the same cancer samples.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Summary, Perspectives, and Limitations
                             </h3>
                             <p id="p-48">
                              Here, we used TGIRT-seq of rRNA-depleted, unfragmented RNA preparations to identify thousands of FLEXI RNAs in human cells and tissues. In total, including the initial cellular and plasma RNA samples and subsequent cancer samples, we identified 8,687 different FLEXIs expressed from 3,923 host genes, representing ∼17% of the 51,645 short introns (≤300 nt) annotated in Ensembl GRCh38 Release 93 annotations. That FLEXIs are linear RNAs was inferred from the characteristics of TGIRT-seq reads (
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ) and confirmed by digestion with a 5’ to 3’ exonuclease (
                              <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                               Fig. S4
                              </a>
                              ). Most FLEXIs have high GC content and stable predicted secondary structures, which may contribute to their stability as FLEXIs (
                              <a id="xref-fig-1-15" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ). In addition to low abundance FLEXIs, which likely include transient intron RNA turnover intermediates, we identified subsets of FLEXIs in each cell type that accumulated to relatively high abundance, including many with abundances of hundreds to thousands of copies per cell (
                              <a id="xref-fig-1-16" class="xref-fig" href="#F1">
                               Fig. 1E
                              </a>
                              and
                              <a id="xref-fig-14-4" class="xref-fig" href="#F14">
                               Fig. S6
                              </a>
                              ). These relatively abundant FLEXIs had reproducible cell-type specific expression patterns (
                              <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                               Fig. 3
                              </a>
                              ,
                              <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                               Fig. S7
                              </a>
                              , and
                              <a id="xref-fig-16-3" class="xref-fig" href="#F16">
                               Fig. S8
                              </a>
                              ), suggesting that FLEXIs may have utility as RNA biomarkers in cells as well as plasma.
                             </p>
                             <p id="p-49">
                              By searching CLIP-seq datasets, we identified 126 different RBPs that have annotated binding sites in FLEXI RNAs, including 53 RBPs that have binding sites in ≥30 different FLEXIs on which we focused further analysis (
                              <a id="xref-fig-4-12" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). These 53 RPBs included spliceosomal proteins and other proteins that function in RNA splicing, AGO1-4 and DICER and other proteins that have a miRNA-related functions, and a surprising number of proteins that function in cell regulatory processes, including transcription factors (AATF, BCLAF1, and SUB1, a chromatin remodeling protein (GRWD1), proteins that function in translational regulation (LARP4, METAP2, PABC4), and cellular growth regulators (DDX3 and IGF2B1;
                              <a id="xref-fig-4-13" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ;
                              <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">
                               Table S3
                              </a>
                              ). Although CLIP-seq does not distinguish whether an RBP bound to an intron before or after splicing, all of the RBPs identified as having binding sites in ≥30 FLEXI RNAs in the ENCODE eCLIP datasets were associated with ≥95% full-length reads corresponding to identified FLEXIs, despite the use of RNase I in processing the cross-linked RNAs for sequencing (
                              <a id="xref-fig-4-14" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              and
                              <a id="xref-fig-18-2" class="xref-fig" href="#F18">
                               Fig. S10
                              </a>
                              ). The ability RBPs to protect short excised intron RNAs from nucleases that function in intron turnover presumably contributes to their accumulation as FLEXIs.
                             </p>
                             <p id="p-50">
                              Addressing the biological significance of FLEXI-RBP interactions by analysis of ENCODE knockdown datasets, we found that knockdown of 34 of the RBPs that have binding sites in ≥30 different FLEXIs resulted in significant changes in alternative splicing of bound FLEXIs and/or
                             </p>
                             <p id="p-51">
                              mRNA levels of FLEXIs that contained a binding site for that RBP relative to control groups (
                              <a id="xref-fig-4-15" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ,
                              <a id="xref-fig-5-11" class="xref-fig" href="#F5">
                               Fig. 5A and B
                              </a>
                              ,
                              <a id="xref-fig-19-6" class="xref-fig" href="#F19">
                               Fig. S11
                              </a>
                              , and
                              <a id="xref-fig-20-4" class="xref-fig" href="#F20">
                               Fig. S12
                              </a>
                              ). Surprisingly 17 of the 27 RBPs whose knockdown impacted alternative splicing of bound FLEXIs had no previously known RNA splicing- related function, including the cell regulatory proteins AATF, RPS3, SUB1, GRWD1, IGF2B, LARP4, PABC4, and XRN2 a nuclear 5’ to 3’ exonuclease that degrades linear RNAs, (see Results for complete list). The involvement of transcription, translation, and cell growth regulatory proteins in RNA splicing provides a means of linking splicing to other cellular regulatory pathways, a role suggested previously for BCLAF1 (
                              <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">
                               Savage et al., 2014
                              </a>
                              ;
                              <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">
                               Vohhodina et al., 2017
                              </a>
                              ) and for binding of AATF to snoRNAs (
                              <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">
                               Kaiser et al., 2019
                              </a>
                              ).
                             </p>
                             <p id="p-52">
                              By mapping RBP-binding sites in CLIP-seq datasets, we found that 37 of the 53 RBPs that bind ≥30 FLEXI RNAs bind at or near the 5’- and 3’-splice sites, while 16 have binding sites more uniformly distributed throughout the body of the intron (
                              <a id="xref-fig-5-12" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              and
                              <a id="xref-fig-21-6" class="xref-fig" href="#F21">
                               Fig. S13
                              </a>
                              ). The binding of RBPs to the 5’- and 3’-splice sites in a pre-mRNA could be facilitated for FLEXIs that fold into a stable secondary structure that brings the 5’- and 3’-splice sites into proximity, as suggested for the binding of the spliceosome to the 5’- and 3’-spice sites of human short introns (
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               Abebrese et al., 2017
                              </a>
                              ). Alternatively, some RBPs may bind to the exposed 5’ and 3’ ends of the FLEXI RNA after complete or partial spliceosome dissociation. In either case, RBPs bound simultaneously to the 5’- and 3’-ends of an excised intron would effectively circularize the intron RNA, protecting it from exonucleases that function in intron RNA turnover and potentially contributing to its accumulation as a FLEXI.
                             </p>
                             <p id="p-53">
                              Compared to other short and long introns, FLEXIs as a whole were enriched in binding sites for core spliceosomal proteins (
                              <a id="xref-fig-4-16" class="xref-fig" href="#F4">
                               Fig. 4B and C
                              </a>
                              ), possibly reflecting that most FLEXIs dissociate more slowly from the spliceosome than other introns, providing greater opportunity for crosslinking. However, we also found that subsets of FLEXIs with known or hypothesized biological functions were enriched in binding sites for RBPs related to those functions, possibly reflecting that these RBPs remained bound to the excised intron RNA after dissociation from the spliceosome. These subsets of FLEXIs included agotrons and mirtrons, which were enriched in binding sites for AGO1-4 and DICER (
                              <a id="xref-fig-6-12" class="xref-fig" href="#F6">
                               Fig. 6A and B
                              </a>
                              ); snoRNA-encoding FLEXIs, which were enriched in proteins with snoRNA-related functions (
                              <a id="xref-fig-6-13" class="xref-fig" href="#F6">
                               Fig. 6C
                              </a>
                              ); and evolutionarily conserved FLEXIs, which were enriched in binding sites for transcription factors and splicing regulators that may control the production of different protein isoforms from FLEXI host genes (
                              <a id="xref-fig-6-14" class="xref-fig" href="#F6">
                               Fig. 6D
                              </a>
                              ).
                             </p>
                             <p id="p-54">
                              Building on these findings, we used a computational method with no a priori assumptions about RBP or FLEXI function to identify six clusters of proteins whose binding sites were coenriched relative to those for spliceosomal and other proteins in different subsets of FLEXI RNAs (
                              <a id="xref-fig-7-12" class="xref-fig" href="#F7">
                               Fig. 7
                              </a>
                              and
                              <a id="xref-fig-22-3" class="xref-fig" href="#F22">
                               Fig. S14
                              </a>
                              ). This computational method recapitulated the findings that agotrons and mirtrons are enriched in binding sites for AGO1-4 and DICER and that snoRNA-encoding FLEXIs are enriched in binding sites for snoRNA related proteins, but also identified many additional FLEXIs bound by these proteins that were not previously annotated as agotrons, mirtrons, or snoRNA-encoding introns. The computational method also identified two sets of splicing factors and regulators that were co-enriched in different subsets of FLEXIs, and two groups of RBPs that may relate to novel functions of FLEXI RNAs (
                              <a id="xref-fig-7-13" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). Particularly suggestive of the latter was cluster I, which included two proteins, LARP4 and PABC4, which were known to function together to regulate the stability and translation of ribosomal protein mRNAs (
                              <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                               Mattijssen et al., 2017
                              </a>
                              ), along with additional proteins that have translation-related functions (NCBP2, METAP2, and RPS3), or that could link these functions to transcriptional regulation of other cell regulatory pathways (SUB1, RPS3, DDX3X, and DDX55). Further, the host genes encoding the FLEXIs in cluster I have GO terms similar to those for the snoRNA-related FLEXIs in cluster V (
                              <a id="xref-fig-7-14" class="xref-fig" href="#F7">
                               Fig. 7B
                              </a>
                              ) suggesting that the FLEXIs and RBPs of cluster I may function similarly to snoRNAs in a feedback loop regulating ribosome assembly or function (
                              <a id="xref-fig-7-15" class="xref-fig" href="#F7">
                               Fig. 7B
                              </a>
                              ). Along the same lines, cluster II included three proteins (UCHL5, ZNF622, and TRA2A) that were distinguished from other proteins associated with FLEXI/RBP clusters as being localized to ER, cytoskeleton, and cell cortex, possibly reflecting a cytoskeleton or membrane-associated function, and clusters III and IV consisted of five splicing factors or regulators, four of which localize to stress granules (TIA1, TIAL1, U2AF1, and KHSRP) (
                              <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">
                               Nunes et al., 2019
                              </a>
                              ), possibly a post-splicing destination for the bound FLEXIs (
                              <a id="xref-fig-7-16" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              ). Collectively, these findings suggest that some FLEXIs may function as scaffold to co-localize or co-regulate multiple proteins with related biological functions.
                             </p>
                             <p id="p-55">
                              Our computational identification of intron-RBP associations is undoubtedly incomplete, as it depends upon assigning a relatively arbitrary significance criterion for association of an RBP with the cluster and would miss proteins that are not efficiently cross-linked to FLEXI RNAs. Minimally, however, our findings demonstrate that different subsets of RBPs bind specifically to distinct subsets of FLEXIs and set the stage for more detailed analysis of these RBP-intron associations.
                             </p>
                             <p id="p-56">
                              Finally, the relatively abundant FLEXIs that have cell-type specific expression patterns may have utility as cellular RNA biomarkers. In addition to being linked to the transcription of thousands of protein-coding and lncRNA genes, FLEXI RNA levels also reflect differences in alternative splicing and intron RNA stability, providing higher resolution of cellular differences than mRNAs transcribed from the same gene. Because many FLEXIs are predicted to fold into stable secondary structures, their initial identification may be best done by TGIRT-seq, which yields full-length, end-to-end sequence reads of structured RNAs, possibly utilizing size-selected small RNA preparations in which FLEXIs are likely to be enriched. Once identified, the validation of candidate FLEXI biomarkers and their routine monitoring would best be done by methods such as RT-qPCR, microarrays, other hybridization-based assays, or targeted RNA-seq that can give rapid quantitative read outs in targeted RNA panels for different diseases.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Data access
                             </h3>
                             <p id="p-57">
                              TGIRT-seq datasets have been deposited in the Sequence Read Archive (SRA) under accession numbers PRJNA648481 and PRJNA640428. A gene counts table, dataset metadata file, FLEXI metadata file, RBP annotation file, and scripts used for data processing and plotting have been deposited in GitHub:
                              <a href="https://github.com/reykeryao/FLEXI">
                               https://github.com/reykeryao/FLEXI
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-19">
                            <h2 class="">
                             COMPETING INTEREST STATEMENT
                            </h2>
                            <p id="p-58">
                             Thermostable group II intron reverse transcriptase (TGIRT) enzymes and methods for their use are the subject of patents and patent applications that have been licensed by the University of Texas and East Tennessee State University to InGex, LLC. A.M.L., some former and present members of the Lambowitz laboratory, and the University of Texas are minority equity holders in InGex and receive royalty payments from the sale of TGIRT-enzymes and kits and from the sublicensing of intellectual property by InGex to other companies. A.M.L., J.Y., H.X. and D.C.W. are inventors on a patent application filed by the University of Texas at Austin for the use of full-length excised intron RNAs and intron RNA fragments as biomarkers. S.W., E.F.K. and M.A. have no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Methods
                            </h2>
                            <div class="section" id="sec-22">
                             <h3>
                              DNA and RNA oligonucleotides
                             </h3>
                             <p id="p-78">
                              DNA and RNA oligonucleotides were purchased from Integrated DNA Technologies (IDT) in RNase-free, HPLC-purified form. The DNA and RNA oligonucleotides used for TGIRT-seq are shown in
                              <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">
                               Table S4
                              </a>
                              . R2R DNA oligonucleotides with 3’ A, C, G, and T residues were handmixed in equimolar amounts prior to annealing to the R2 RNA oligonucleotide to generate RNA/DNA heteroduplexes with a 1-nt 3’-DNA overhang for RNA-seq adapter addition by TGIRT template switching.
                             </p>
                             <p id="p-79">
                              The oligonucleotides used Terminator exonuclease and ddPCR experiments are listed in
                              <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">
                               Table S5
                              </a>
                              and include reverse transcriptase primers, PCR primers, and the 3I_RAN FLEXI synthetic oligonucleotide controls with and without a 5’ phosphate. The sequence specificity of all primer sets (
                              <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">
                               Table S5
                              </a>
                              ) was confirmed by gel purifying a representative subset of qPCR amplicons and either directly Sanger sequencing or by TOPO-TA cloning (ThermoFisher) followed by Sanger sequencing. A circular form of the 3I_RAN FLEXI synthetic oligonucleotide control was generated by incubating the 5’phosphorylated-3I_RAN FLEXI oligonucleotide (500 ng) with T4 RNA Ligase I (10 U; New England Biolabs) for 2 h at 25 °C and 2 min at 95 °C. The circularized oligonucleotide was cleaned up with an RNA Clean & Concentrator kit (Zymo Research), using 8 sample volumes of ethanol added during purification (denoted 8X ethanol method) to maximize the recovery of small RNAs concurrently with the longer RNAs.
                             </p>
                            </div>
                            <div class="section" id="sec-23">
                             <h3>
                              RNA preparations
                             </h3>
                             <p id="p-80">
                              Universal Human Reference RNA (UHRR) was purchased from Agilent (product name: Quantitative PCR Human Reference Total RNA), and HeLa S3 and MCF-7 RNAs were purchased from Thermo Fisher. RNAs from matched frozen healthy/tumor tissues of breast cancer patients were purchased from Origene (500 ng; Patient A: PR
                              <sup>
                               +
                              </sup>
                              , ER
                              <sup>
                               +
                              </sup>
                              , HER2
                              <sup>
                               -
                              </sup>
                              , CR562524/CR543839; Patient B: PR unknown, ER
                              <sup>
                               -
                              </sup>
                              , HER2
                              <sup>
                               -
                              </sup>
                              , CR560540/CR532030).
                             </p>
                             <p id="p-81">
                              K-562, HEK-293T/17, and MDA-MB-231 RNAs were isolated from cultured cells by using a mirVana miRNA Isolation Kit (Thermo Fisher). K-562 cells (ATCC CTL-243) were maintained in Iscove’s Modified Dulbecco’s Medium (IMDM) + 4 mM L-glutamine and 25 mM HEPES; Thermo Fisher) supplemented with 10% Fetal Bovine Serum (FBS; Gemini Bio-Products), and approximately 2 x 10
                              <sup>
                               6
                              </sup>
                              cells were used for RNA extraction. HEK-293T/17 cells (ATCC CRL-11268) were maintained in Dulbecco’s Modified Eagle Medium (DMEM) + 4.5 g/L D-glucose, 4 mM L-glutamine, and 1 mM sodium pyruvate, supplemented with 10% FBS, and approximately 4 x 10
                              <sup>
                               6
                              </sup>
                              cells were used for RNA extraction. MDA-MB-231 cells (ATCC HTB-26) were maintained in DMEM + 4.5 g/L D-glucose and 4 mM L-glutamine (Thermo Fisher) supplemented with 10% FBS and 1X PSQ (Penicillin, Streptomycin, and Glutamine: Thermo Fisher), and approximately 4 x 10
                              <sup>
                               6
                              </sup>
                              cells were used for RNA extraction. All cells were maintained at 37 °C in a humidified 5% CO
                              <sub>
                               2
                              </sub>
                              atmosphere.
                             </p>
                             <p id="p-82">
                              For RNA isolation, cells were harvested by centrifugation (after trypsinization for HEK- 293T/17 and MDA-MB-231 cells) at 300 x g for 10 min at 4 °C and washed twice by centrifugation with cold Dulbecco’s Phosphate Buffered Saline (Thermo Fisher). The indicated number of cells (see above) was then resuspended in 600 µL of mirVana Lysis Buffer and RNA was isolated according to the kit manufacturer’s protocol with elution in a final volume of 100 µL.
                             </p>
                            </div>
                            <div class="section" id="sec-24">
                             <h3>
                              Terminator exonuclease assay
                             </h3>
                             <p id="p-83">
                              Unfragmented HEK-293T and UHHR total RNAs (500 ng) or synthetic 3I_RAN FLEXI oligonucleotide controls (100 ng; oligonucleotide with or without a 5’phosphate or circularized oligonucleotide) were treated with 1 U Terminator exonuclease (Lucigen) or mock treated in the manufacturer’s Buffer A for 1 h at 37 °C followed by 1 h at 30 °C. The terminator reactions were cleaned up with the RNA Clean & Concentrator kit (8X ethanol method; Zymo Research). The manufacturer’s instructions note that the standard buffer (Buffer A) is the preferred reaction buffer for maximal exonuclease activity, but that non-5’-monophosphate-specific activity, including activity with RNAs having a 5’ hydroxyl activity, might also result. Reverse transcription was performed by using a gene specific primer (1 μM final concentration) located near the 3’ end of the FLEXI intron with 100 U Maxima H reverse transcriptase (ThermoFisher) in the manufacturer’s buffer and incubated at 25 °C for 10 min, 50 °C for 30 min, 65 °C for 30 min and 85 °C for 5 min. qPCR reactions were carried out with 1 μL cDNA and PowerUp SYBR Green master mix (Life Technologies) using a ViiA 7 Real Time PCR system (ThermoFisher). Delta CT values were calculated to obtain a fold change of RNA levels in the treated compared to the mock treated sample. The primer sequences and 3I_RAN FLEXI synthetic oligonucleotide control sequences are shown in
                              <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">
                               Table S5
                              </a>
                              .
                             </p>
                            </div>
                            <div class="section" id="sec-25">
                             <h3>
                              Droplet digital PCR
                             </h3>
                             <p id="p-84">
                              Unfragmented HeLa S3, UHRR or HEK-293T RNAs (5 μg) were DNased with 2 U TURBO DNase (Invitrogen) and purified using an RNA Clean & Concentrator kit (Zymo Research) following the manufacturer’s small-size selection method, to separate RNAs ≤200 nt. The input of the size-selected RNAs for reverse transcription reactions were scaled to have a similar SNORD14B control amplification when compared with total RNA inputs (HEK-293T and HeLa S3, 5 ng; UHHR, 1 ng). The RNAs were reverse transcribed by using 1 μM final concentration of a gene specific primer (
                              <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">
                               Table S5
                              </a>
                              ) located at the 3’ end of the FLEXI intron with 100 U Maxima H Reverse Transcriptase (ThermoFisher) in the manufacturer’s buffer and incubated at 25 °C for 10 min, 50 °C for 30 min, 65 °C for 30 min and 85 °C for 5 min. Matched control samples containing the small size-selected RNA, but omitting the RT (RT-) were performed for every reaction. Droplet digital PCR (ddPCR) was performed on 1 μL cDNA using QX200 ddPCR EvaGreen Supermix (Biorad) according to manufacturer’s protocol with the QX200 Droplet Digital PCR system (BioRad). Primer sets (
                              <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">
                               Table S5
                              </a>
                              ) were designed to encompass either the FLEXI intron sequence or to span the upstream 5’exon-FLEXI junction in order to quantify the FLEXI RNA compared to any precursor or mature mRNA with retained FLEXI introns. The same genespecific-generated cDNA was used as the PCR template for quantifying both the FLEXI and 5’ exon-FLEXI junctions. Due to the short size of the FLEXI sequences, primer design was constrained to the best available primer location that maximized length and sequence specificity. As a result of the primer constraints, some primers amplified a smaller region of the FLEXI sequence than the TGIRT-seq criteria (i.e., 5’ and 3’ ends within 3 nt of the annotated splice sites), which could contribute to differences between ddPCR and TGIRT-seq quantification. 5’-exon- FLEXI junction primer pairs used a different reverse primer than the FLEXI, that was inset closer to the 5’exon-FLEXI junction, so that the size of the amplicon would be more similar to the size of the compared FLEXI amplicon. As a consequence of these constraints, some primer pairs amplified background non-specific products in addition to verified products. For each reaction replicate, any RT- signal was subtracted, and three background-subtracted reaction replicates were averaged for each primer pair. The majority of the 5’exon-FLEXI junction reactions either gave no product or amplified non-specific products. However, a few 5’exon-FLEXI junction reactions produced validated products sequenced by direct Sanger sequencing and/or TOPOTA cloning and Sanger sequencing of gel extracted, matched qPCR amplicons (HEK-293T, RPS2; HeLa S3, ACTB; UHRR, POLG and ACTB). These sequence-validated 5’exon-FLEXI junction ddPCR signals were subtracted from the FLEXI ddPCR signals to obtain the final ddPCR estimate for abundance of the FLEXI. FLEXI copies per reaction were then normalized to the controls performed in parallel: U7 (chosen for similar abundance to FLEXIs), SNORD14B (
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               Dodd et al., 2013
                              </a>
                              ), and SNORD44 (
                              <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                               Sharbati et al., 2012
                              </a>
                              ). Finally, the normalized FLEXI copies per reaction were converted to copy number per cell estimates by multiplying by respective control values generated by the linear regression model without or with TGIRT-seq bias correction.
                             </p>
                            </div>
                            <div class="section" id="sec-26">
                             <h3>
                              TGIRT-seq
                             </h3>
                             <p id="p-85">
                              To remove residual DNA, UHRR and HeLa S3 RNAs (1 μg) and patients A and B healthy and cancer tissue RNAs (500 ng) were incubated with 20 U exonuclease I (Lucigen) and 2 U Baseline-ZERO DNase (Lucigen) in Baseline-ZERO DNase Buffer for 30 min at 37 °C, and K562, MDA-MB-231 and HEK-293T cell RNAs (5 μg) were incubated with 2 U TURBO DNase for 30 min at 37 °C (ThermoFisher). After DNA digestion, RNA was cleaned up with an RNA Clean & Concentrator kit (8X ethanol method; Zymo Research). The eluted RNAs were ribodepleted by using the rRNA removal section of a TruSeq Stranded Total RNA Library Prep Human/Mouse/Rat kit (Illumina), with the supernatant from the magnetic-bead separation cleaned-up by using a Zymo RNA Clean & Concentrator kit with 8X ethanol. After checking RNA concentration and length by using an Agilent 2100 Bioanalyzer with a 6000 RNA Pico chip, RNAs were aliquoted into ∼20 ng portions and stored at -80 °C until use.
                             </p>
                             <p id="p-86">
                              For the preparation of samples containing chemically fragmented long RNAs, RNA preparations were treated with exonuclease I and Baseline-Zero DNase to remove residual DNA and ribodepleted, as described above. The supernatant from the magnetic-bead separation after ribodepletion was then cleaned-up with a Zymo RNA Clean & Concentrator kit using the manufacturer’s two-fraction protocol, which separates RNAs into long and short RNA fractions (200- nt cut-off). The long RNAs were then fragmented to 70-100 nt by using an NEBNext Magnesium RNA Fragmentation Module (94 °C for 7 min; New England Biolabs). After clean-up by using a Zymo RNA Clean & Concentrator kit (8X ethanol protocol), the fragmented long RNAs were combined with the unfragmented short RNAs and treated with T4 polynucleotide kinase (Epicentre) to remove 3’ phosphates (
                              <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">
                               Xu et al., 2019
                              </a>
                              ), followed by clean-up using a Zymo RNA Clean & Concentrator kit (8X ethanol protocol). After confirming the RNA fragment size range and RNA concentration by using an Agilent 2100 Bioanalyzer with a 6000 RNA Pico chip, the RNA was aliquoted into 4 ng portions for storage in -80 °C.
                             </p>
                             <p id="p-87">
                              TGIRT-seq libraries were prepared as described (
                              <a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">
                               Xu et al., 2019
                              </a>
                              ) using 20-50 ng of ribodepleted unfragmented cellular RNA or 4-10 ng of ribodepleted chemically fragmented cellular RNA. Template-switching and reverse transcription reactions were done with 1 μM TGIRT-III (InGex) and 100 nM pre-annealed R2 RNA/R2R DNA starter duplex in 20 μl of reaction medium containing 450 mM NaCl, 5 mM MgCl
                              <sub>
                               2
                              </sub>
                              , 20 mM Tris-HCl, pH 7.5 and 5 mM DTT. Reactions were set up with all components except dNTPs, pre-incubated for 30 min at room temperature, a step that increases the efficiency of RNA-seq adapter addition by TGIRT template switching, and initiated by adding dNTPs (final concentrations 1 mM each of dATP, dCTP, dGTP, and dTTP). The reactions were incubated for 15 min at 60 °C and then terminated by adding 1 μl 5 M NaOH to degrade RNA and heating at 95 °C for 5 min followed by neutralization with 1 μl 5 M HCl and one round of MinElute column clean-up (Qiagen). The R1R DNA adapter was adenylated by using a 5’ DNA Adenylation kit (New England Biolabs) and then ligated to the 3’ end of the cDNA by using thermostable 5’ App DNA/RNA Ligase (New England Biolabs) for 2 h at 65 °C. The ligated products were purified by using a MinElute Reaction Cleanup Kit and amplified by PCR with Phusion High-Fidelity DNA polymerase (Thermo Fisher Scientific): denaturation at 98 °C for 5 sec followed by 12 cycles of 98 °C 5 sec, 60 °C 10 sec, 72 °C 15 sec and then held at 4 °C until removed. The PCR products were cleaned up by using Agencourt AMPure XP beads (1.4X volume; Beckman Coulter) and sequenced on an Illumina NextSeq 500 to obtain 2 x 75 nt paired-end reads or on an Illumina NovaSeq 6000 to obtain 2 x 150 nt paired- end reads at the Genome Sequence and Analysis Facility of the University of Texas at Austin.
                             </p>
                             <p id="p-88">
                              TGIRT-seq datasets of commercial human plasma pooled from multiple healthy individuals were obtained in a previous study (
                              <a id="xref-ref-63-8" class="xref-bibr" href="#ref-63">
                               Yao et al., 2020
                              </a>
                              ) and deposited in the National Center for Biotechnology Information Sequence Read Archive under accession number PRJNA640428.
                             </p>
                             <p id="p-89">
                              TGIRT-seq datasets for biological replicates of unfragmented cellular RNAs from HEK-293T were previously deposited in the National Center for Biotechnology Information Sequence Read Archive under accession number PRJNA389469 (samples WC and YC). TGIRT-seq datasets for biological replicates of unfragmented cellular RNAs from K-562, MDA-MB-231 and UHRR were previously deposited in the National Center for Biotechnology Information Sequence Read Archive under accession number PRJNA722757 (K-562 cells, samples UFC1-7; MDA-MB-231, samples MDA1-5; and UHRR, samples UHRR0-9).
                             </p>
                            </div>
                            <div class="section" id="sec-27">
                             <h3>
                              Sequencing data processing
                             </h3>
                             <p id="p-90">
                              Illumina TruSeq adapters and PCR primer sequences were trimmed from the reads with Cutadapt v2.8 (sequencing quality score cut-off at 20; p-value <0.01) (
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               Martin, 2011
                              </a>
                              ) and reads <15-nt after trimming were discarded. To minimize mismapping, we used a sequential mapping strategy. First, reads were mapped to the human mitochondrial genome (Ensembl GRCh38 Release 93) and the Escherichia coli genome (GeneBank: NC_000913) using HISAT2 v2.1.0 (
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               Kim et al., 2019
                              </a>
                              ) with customized settings (-k 10 --rfg 1,3 --rdg 1,3 --mp 4,2 --no-mixed --no-discordant -- no-spliced-alignment) to filter out reads derived from mitochondrial and E. coli RNAs (denoted Pass 1). Unmapped read from Pass1 were then mapped to a collection of customized references sequences for human sncRNAs (miRNA, tRNA, Y RNA, Vault RNA, 7SL RNA, 7SK RNA) and rRNAs (2.2-kb 5S rRNA repeats from the 5S rRNA cluster on chromosome 1 (1q42, Gene- Bank: X12811) and 43-kb 45S rRNA containing 5.8S, 18S and 28S rRNAs from clusters on chromosomes 13,14,15, 21, and 22 (GeneBank: U13369), using HISAT2 with the following settings -k 20 --rdg 1,3 --rfg 1,3 --mp 2,1 --no-mixed --no-discordant --no-spliced-alignment --norc (denoted Pass 2). Unmapped reads from Pass 2 were then mapped to the human genome reference sequence (Ensembl GRCh38 Release 93) using HISAT2 with settings optimized for nonspliced mapping (-k 10 --rdg 1,3 --rfg 1,3 --mp 4,2 --no-mixed --no-discordant --no-spliced- alignment) (denoted Pass 3) followed by splice aware mapping (-k 10 --rdg 1,3 --rfg 1,3 --mp 4,2 --no-mixed --no-discordant --dta) (denoted Pass 4). Finally, the remaining unmapped reads were mapped to Ensembl GRCh38 Release 93 by Bowtie 2 v2.2.5 (
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               Langmead and Salzberg, 2012
                              </a>
                              ) using local alignment (-k 10 --rdg 1,3 --rfg 1,3 --mp 4 --ma 1 --no-mixed --no-discordant --very- sensitive-local) to improve mapping rates for reads containing post-transcriptionally added 5’ or 3’ nucleotides (poly(A) or poly(U)), short untrimmed adapter sequences, and non-templated nucleotides added to the 3’ end of the cDNAs by TGIRT-III during TGIRT-seq library preparation (denoted Pass 5). For reads that map to multiple genomic loci with the same mapping score in passes 3 to 5, the alignment with the shortest distance between the two paired ends (i.e., the shortest read span) was selected. In the case of ties (i.e., reads with the same mapping score and read span), reads mapping to a chromosome were selected over reads mapping to scaffold sequences, and in other cases, the read was assigned randomly to one of the tied choices. The filtered multiply mapped reads were then combined with the uniquely mapped reads from Passes 3- 5 by using SAMtools v1.10 (
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               Li et al., 2009
                              </a>
                              ).
                             </p>
                             <p id="p-91">
                              To generate counts for different genomic features, the mapped reads were intersected with Ensembl GRCh38 Release 93 gene annotations plus the RNY5 gene and its 10 pseudogenes, which are not annotated in this release. Coverage of each feature was calculated by BEDTools v2.29.2 (
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               Quinlan, 2014
                              </a>
                              ). To avoid miscounting reads with embedded sncRNAs that were not filtered out in Pass 2 (e.g., snoRNAs), reads were first intersected with sncRNA annotations and the remaining reads were then intersected with the annotations for protein-coding genes RNAs, lincRNAs, antisense RNAs, and other lncRNAs to get the read count for each annotated feature. Coverage plots and read alignments were created by using Integrative Genomics Viewer v2.6.2 (IGV) (
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               Robinson et al., 2011
                              </a>
                              ). Genes with >100 mapped reads were down sampled to 100 mapped reads in IGV for visualization.
                             </p>
                            </div>
                            <div class="section" id="sec-28">
                             <h3>
                              FLEXI identification and characterization
                             </h3>
                             <p id="p-92">
                              To identify short introns that could give rise to FLEXI RNAs, intron annotations were extracted from Ensemble GRCh38 Release 93 gene annotations using a customized script and filtered to remove introns >300 nt as well as duplicate intron annotations from different mRNA isoforms. To identify FLEXI RNAs, mapped reads were intersected with the short intron annotations using BEDTools, and read pairs (Read 1 and Read 2) ending at or within 3 nucleotides of annotated 5’- and 3’-splice sites were identified as corresponding to FLEXI RNAs. Exon-junction reads were defined as those in which one of the paired-end read mapped within the intron and the other within a flanking exon.
                             </p>
                             <p id="p-93">
                              UpSet plots of FLEXI RNAs from different sample types were plotted by using the ComplexHeatmap package v2.2.0 in R (
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               Gu et al., 2016
                              </a>
                              ). For UpSet plots of FLEXI host genes, FLEXI RNAs were aggregated by Ensemble ID, and different FLEXI RNAs from the same gene were combined into one entry. Density distribution plots and scatter plots were plotted by using R.
                             </p>
                             <p id="p-94">
                              FLEXI RNAs corresponding to annotated mirtrons or agotrons were identified by intersecting FLEXI RNA coordinates with those of annotated mirtrons (
                              <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">
                               Wen et al., 2015
                              </a>
                              ) and agotrons (
                              <a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">
                               Hansen et al., 2016
                              </a>
                              ). FLEXI RNAs containing embedded snoRNAs were identified by intersecting the FLEXI RNA coordinates with those of annotated snoRNA and scaRNA from Ensembl GRCh38 annotations.
                             </p>
                             <p id="p-95">
                              5’- and 3’-splice sites (SS) and branch-point (BP) consensus sequences of human U2- and U12-type spliceosomal introns were obtained from previous publications (
                              <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">
                               Gao et al., 2008
                              </a>
                              ;
                              <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">
                               Sheth et al., 2006
                              </a>
                              ). FLEXI RNAs corresponding to U12-type introns were identified by searching for: (i) FLEXI RNAs with AU-AC ends and (ii) the 5’-splice site consensus sequence of U12-type introns with GU-AG ends (
                              <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">
                               Sheth et al., 2006
                              </a>
                              ) using FIMO (
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               Grant et al., 2011
                              </a>
                              ) with the following settings: FIMO --text --norc <GU_AG_U12_5SS motif file> <sequence file>. Splice-site consensus sequences of FLEXI RNAs were calculated from nucleotides frequencies at the 5’ and 3’ ends of the introns. The U2-type intron branch-point consensus sequence of intron RNAs was identified by searching for U2-type branch-point consensus using FIMO (
                              <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                               Grant et al., 2011
                              </a>
                              ). The U12-type intron branch-point consensus was determined by searching for motifs enriched within 40 nt of the 3’ end of the introns using MEME (
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               Bailey et al., 2009
                              </a>
                              ) with settings: meme <sequence file> -rna -oc <output folder> -mod anr -nmotifs 100 -minw 6 -minsites 100 -markov order 1 -evt 0.05. The branch-point consensus sequence of U12-type FLEXI RNAs (2 with AU-AC ends and 34 with GU-AG matching the 5’ sequence of GU-AG U12-type introns) was determined by manual sequence alignment and calculation of nucleotide frequencies. Motif logos were plotted from the nucleotide frequency tables of each motif using scripts from MEME suite (
                              <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                               Bailey et al., 2009
                              </a>
                              ).
                             </p>
                            </div>
                            <div class="section" id="sec-29">
                             <h3>
                              Clustering analysis
                             </h3>
                             <p id="p-96">
                              PCA analysis of cell-type specific FLEXI RNA profiles in replicate cellular RNA datasets was plotted using R, and PCA initialized t-SNE and ZINB-WaVE analyses of these datasets were plotted using the Rtsne (
                              <a href="https://github.com/jkrijthe/Rtsne">
                               https://github.com/jkrijthe/Rtsne
                              </a>
                              ) and zinbwave (
                              <a href="https://github.com/drisso/zinbwave">
                               https://github.com/drisso/zinbwave
                              </a>
                              ) packages in R (
                              <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">
                               Risso et al., 2018
                              </a>
                              ). Batch effect correction was done by using ZINB-WaVE to include the batch information as a sample-level covariate. The normalized counts outputted by ZINB-WaVE were used for PCA analysis, and the W matrix outputted by ZINB-WaVE was used for visualization directly or by t-SNE.
                             </p>
                            </div>
                            <div class="section" id="sec-30">
                             <h3>
                              RBP-binding site analysis
                             </h3>
                             <p id="p-97">
                              FLEXI RNAs containing RBP-binding sites were identified by intersecting the FLEXI RNA coordinates with ENCODE annotated binding sites of 150 RBPs from the eCLIP-seq with IDR analysis (
                              <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">
                               Van Nostrand et al., 2016
                              </a>
                              ) and crosslink-centered regions (CCRs) identified from the DICER PAR-CLIP (
                              <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">
                               Rybak-Wolf et al., 2014
                              </a>
                              ) and AGO1-4 PAR-CLIP (
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               Hafner et al., 2010
                              </a>
                              ) datasets by using BEDTools. Multiple binding sites for the same RBP in an intron were collapsed and counted as one site for that RBP. Functional annotations, localization patterns, and predicted RNA-binding domains of RBPs in the ENCODE eCLIP dataset were based on
                              <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">
                               Table S2
                              </a>
                              of (
                              <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">
                               Van Nostrand et al., 2020
                              </a>
                              ). RBPs found in stress granules were as annotated in the RNA Granule and Mammalian Stress Granules Proteome (MSGP) databases (
                              <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">
                               Nunes et al., 2019
                              </a>
                              ;
                              <a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">
                               Youn et al., 2019
                              </a>
                              ). The functional annotations, localization patterns, and RNA-binding domains of AGO1-4 and DICER were from the UniProt database (
                              <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">
                               The UniProt Consortium, 2018
                              </a>
                              ). Near full length FLEXI or other short intron reads were identified in ENCODE eCLIP datasets by intersecting mapped eCLIP reads with FLEXI or other short intron coordinates using BedTools.
                             </p>
                             <p id="p-98">
                              Gene expression changes in RBP knockdown datasets were calculated from changes in mRNA abundance measured by using DESeq2 after GC correction with Salmon (
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               Patro et al., 2017
                              </a>
                              ) and CQN (
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               Hansen et al., 2012
                              </a>
                              ). The number of significant gene expression changes (DESeq2-calculated adjusted p-value<0.05) for FLEXI host genes was compared to three gene sets containing the same number of randomly selected genes, and ANOVA was used to identify knockdowns that resulted in significantly more gene expression changes among FLEXI host genes than expected by random chance. To identify FLEXI host genes whose changes in mRNA levels were significantly biased in one direction, the numbers of upregulated and downregulated host genes containing a FLEXI with a binding site for the knocked-down RBP (defined as log
                              <sub>
                               2
                              </sub>
                              - transformed fold change>0, adjusted p-value<0.05) were compared to those for all significantly changed genes by Fisher’s exact test, and those with p-value <0.05 for difference in one or the other direction were considered significantly biased in that direction. Volcano plots were generating by plotting log
                              <sub>
                               2
                              </sub>
                              transformed fold changes in the mRNA levels between the knockdown and control datasets versus -log
                              <sub>
                               10
                              </sub>
                              adjusted p-values.
                             </p>
                             <p id="p-99">
                              Changes in alternative splicing were calculated by using rMATS (
                              <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">
                               Shen et al., 2014
                              </a>
                              ). Coordinates of adjacent introns or of retained introns from splicing change files (MATS output) were intersected with coordinates of bound FLEXIs (i.e., FLEXIs that contain a binding site for a RBP), unbound FLEXIs (i.e., FLEXIs that do not contain a binding site for the same RBP), and other short introns, and the results were displayed as a density plot (Seaborn distplot) showing the inclusion level difference for the same intron between the knockdown and control datatsets.
                             </p>
                             <p id="p-100">
                              To map RBP-binding sites within introns, RBP-binding sites from ENCODE eCLIP datasets were intersected with FLEXI, other short intron, or long intron coordinates (BedTools). Any binding site overlapping with any portion of the intron was used. The location of the binding site with respect to normalized intron length was calculated from the distance between the mid-point of the annotated binding site and the beginning or end of the intron. Data were displayed as a density plot of the mid-point of the binding sites normalized by intron length.
                             </p>
                             <p id="p-101">
                              For hierarchical clustering analysis of the RBP-binding site co-enrichment, we first extracted subsets of FLEXIs that have a binding site for each of 47 RBPs that have binding sites in ≥30 different FLEXI RNAs from datasets for each of the cellular RNA samples. We then identified over- and under-represented RBPs in each of the 47 subsets for each cellular RNA sample by generating a contingency table comparing the frequency of binding sites for the 126 RBPs that have an identified binding site in a FLEXI RNA (
                              <a id="xref-fig-17-4" class="xref-fig" href="#F17">
                               Fig. S9
                              </a>
                              ) in the tested subset to its frequency in all FLEXIs. Multiple binding sites for the same RBP in the same FLEXI were counted as one binding site. p-values were calculated using Fisher’s exact test and then adjusted by the Benjamini-Hochberg procedure and log
                              <sub>
                               10
                              </sub>
                              transformed. RBP binding sites were then key-coded as not significantly different or as significantly over- or under-represented (≥2% abundance, p≤0.05) in the tested subset of FLEXIs compared to all FLEXIs, and matrices of the Gower’s distance (
                              <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                               Gower, 1971
                              </a>
                              ) were calculated using the key-coded information from the above analyses and used as input for hierarchical clustering by the complete linkage clustering method (
                              <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">
                               Defays, 1977
                              </a>
                              ). The results were displayed as a heat map plotted using the R package cluster and ComplexHeatmap, respectively.
                             </p>
                             <p id="p-102">
                              GO enrichment analysis of host genes encoding FLEXIs bound by different RBPs was performed by using DAVID bioinformatics tools (
                              <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                               Huang et al., 2009
                              </a>
                              ) with all FLEXI host genes as the background. Hierarchical clustering was performed based on p-values for GO term enrichment of FLEXI host genes bound by the same RBPs using the Seaborn ClusterMap package in Python (
                              <a href="https://zenodo.org">
                               https://zenodo.org
                              </a>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             ACKNOWLEDGMENTS
                            </h2>
                            <p id="p-59">
                             We thank Marta Mastroianni and Ryan Nottingham (University of Texas at Austin) for cell culture and RNA preparations, Blerta Xhemalce (University of Texas at Austin) for a fresh stock of MDA-MB-231 cells, Camilo Molina Romero and Kavyaa Choudhary (University of Texas at Austin) for help with validating amplicons for terminator and digital PCR analysis, and Jim Manley (Columbia), Richard Maraia (NIH/NICHD), Carla Van Den Berg (University of Texas at Austin), and Sandra Wolin (NIH/NCI) for comments on the manuscript. The authors acknowledge the Texas Advanced Computing Center (TACC;
                             <a href="http://www.tacc.utexas.edu">
                              http://www.tacc.utexas.edu
                             </a>
                             ) at the University of Texas at Austin for providing high performance computing resources that have contributed to the research results reported in this paper (URL:
                             <a href="http://www.tacc.utexas.edu">
                              http://www.tacc.utexas.edu
                             </a>
                             ). This work was supported by NIH grants R01 GM37949 and R35 GM136216 to AML. MA was supported by NIH grant R01 GM040478.
                            </p>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             REFERENCES
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.1" data-doi="10.1371/journal.pone.0175393">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abebrese
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ali
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arnold
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Andrews
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Armstrong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burns
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Crowder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Day
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.T.
                                  </span>
                                  ,
                                  <span class="cit-name-suffix">
                                   Jr.
                                  </span>
                                  .
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hsu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jarrell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Identification of human short introns
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLOS ONE
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e0175393
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1371/journal.pone.0175393
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLOS%2BONE%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0175393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0175393&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.2" data-doi="10.4161/cc.9.12.11922">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alawi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Loss of dyskerin reduces the accumulation of a subset of H/ACA snoRNA-derived miRNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Cycle
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2467
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2469
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.4161/cc.9.12.11922
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2Bcycle%2B%2528Georgetown%252C%2BTex.%2529%26rft.stitle%253DCell%2BCycle%26rft.aulast%253DAlawi%26rft.auinit1%253DF.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D2467%26rft.epage%253D2469%26rft.atitle%253DLoss%2Bof%2Bdyskerin%2Breduces%2Bthe%2Baccumulation%2Bof%2Ba%2Bsubset%2Bof%2BH%252FACA%2BsnoRNA-derived%2BmiRNA.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fcc.9.12.11922%26rft_id%253Dinfo%253Apmid%252F20519940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.4161/cc.9.12.11922&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20519940&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.3" data-doi="10.1093/nar/gkp335">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Buske
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clementi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.W.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  MEME Suite: tools for motif discovery and searching
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  W202
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  W208
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkp335
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp335%26rft_id%253Dinfo%253Apmid%252F19458158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkp335&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19458158&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000267889100037&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.4" data-doi="10.1016/j.molcel.2007.09.028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berezikov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.-J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Willis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cuppen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Mammalian mirtron genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  328
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  336
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.molcel.2007.09.028
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBerezikov%26rft.auinit1%253DE.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D328%26rft.epage%253D336%26rft.atitle%253DMammalian%2Bmirtron%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.09.028%26rft_id%253Dinfo%253Apmid%252F17964270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.09.028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17964270&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000250739800016&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.5" data-doi="10.1093/nar/28.21.4364">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burset
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seledtsov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Solovyev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.V
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Analysis of canonical and non-canonical splice sites in mammalian genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  28
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4364
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4375
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/28.21.4364
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBurset%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D21%26rft.spage%253D4364%26rft.epage%253D4375%26rft.atitle%253DAnalysis%2Bof%2Bcanonical%2Band%2Bnon-canonical%2Bsplice%2Bsites%2Bin%2Bmammalian%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.21.4364%26rft_id%253Dinfo%253Apmid%252F11058137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/28.21.4364&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11058137&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000165263900040&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.6" data-doi="10.1016/0092-8674(91)90466-C">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chapman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Boeke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Isolation and characterization of the gene encoding yeast debranching enzyme
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  65
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  483
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  492
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/0092-8674(91)90466-C
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChapman%26rft.auinit1%253DK.%2BB.%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D483%26rft.epage%253D492%26rft.atitle%253DIsolation%2Band%2Bcharacterization%2Bof%2Bthe%2Bgene%2Bencoding%2Byeast%2Bdebranching%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990466-C%26rft_id%253Dinfo%253Apmid%252F1850323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90466-C&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=1850323&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1991FK18200015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.7" data-doi="10.1093/comjnl/20.4.364">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Defays
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1977
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An efficient algorithm for a complete link method
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Computer Journal
                                 </abbr>
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  364
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  366
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/comjnl/20.4.364
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DThe%2BComputer%2BJournal%26rft.stitle%253DThe%2BComputer%2BJournal%26rft.aulast%253DDefays%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D364%26rft.epage%253D366%26rft.atitle%253DAn%2Befficient%2Balgorithm%2Bfor%2Ba%2Bcomplete%2Blink%2Bmethod%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fcomjnl%252F20.4.364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/comjnl/20.4.364&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.8" data-doi="10.1089/nat.2013.0427">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dodd
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gagnon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Corey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Digital quantitation of potential therapeutic target RNAs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acid Ther
                                 </abbr>
                                 <span class="cit-vol">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  188
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  194
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1089/nat.2013.0427
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcid%2BTher%26rft.volume%253D23%26rft.spage%253D188%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fnat.2013.0427%26rft_id%253Dinfo%253Apmid%252F23656494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1089/nat.2013.0427&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23656494&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.9" data-doi="10.1073/pnas.88.3.790">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Farrell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dobson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feldman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.T
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1991
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Herpes simplex virus latency-associated transcript is a stable intron
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  88
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  790
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  794
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.88.3.790
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFarrell%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D88%26rft.issue%253D3%26rft.spage%253D790%26rft.epage%253D794%26rft.atitle%253DHerpes%2Bsimplex%2Bvirus%2Blatency-associated%2Btranscript%2Bis%2Ba%2Bstable%2Bintron.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.3.790%26rft_id%253Dinfo%253Apmid%252F1846963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4OC8zLzc5MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA5LzIwMjAuMDkuMDcuMjg1MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.10" data-doi="10.1093/nar/gkn073">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Masuda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Matsuura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ohno
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Human branch point consensus sequence is yUnAy
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2257
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2267
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkn073
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn073%26rft_id%253Dinfo%253Apmid%252F18285363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkn073&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18285363&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000255152200021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.11" data-doi="10.1101/gad.202184.112">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gardner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nizami
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Talbot
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Stable intronic sequence RNA (sisRNA), a new class of noncoding RNA from the oocyte nucleus of Xenopus tropicalis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genes Dev
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2550
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2559
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1101/gad.202184.112
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DGardner%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D26%26rft.issue%253D22%26rft.spage%253D2550%26rft.epage%253D2559%26rft.atitle%253DStable%2Bintronic%2Bsequence%2BRNA%2B%2528sisRNA%2529%252C%2Ba%2Bnew%2Bclass%2Bof%2Bnoncoding%2BRNA%2Bfrom%2Bthe%2Boocyte%2Bnucleus%2Bof%2BXenopus%2Btropicalis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.202184.112%26rft_id%253Dinfo%253Apmid%252F23154985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI2LzIyLzI1NTAiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8wOS8yMDIwLjA5LjA3LjI4NTExNC5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.12" data-doi="10.2307/2528823">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gower
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1971
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A General Coefficient of Similarity and Some of Its Properties
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biometrics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  857
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.2307/2528823
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D27%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2307/2528823&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1971L269000004&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.13" data-doi="10.1093/bioinformatics/btr064">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bailey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Noble
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  FIMO: scanning for occurrences of a given motif
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1017
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1018
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/bioinformatics/btr064
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr064%26rft_id%253Dinfo%253Apmid%252F21330290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr064&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21330290&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000289162000022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.14" data-doi="10.1093/bioinformatics/btw313">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eils
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schlesner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Complex heatmaps reveal patterns and correlations in multidimensional genomic data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2847
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2849
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/bioinformatics/btw313
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw313%26rft_id%253Dinfo%253Apmid%252F27207943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw313&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27207943&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.15" data-doi="10.1016/j.cell.2010.03.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landthaler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Khorshid
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hausser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berninger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rothballer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ascano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                  ,
                                  <span class="cit-name-suffix">
                                   Jr.
                                  </span>
                                  .
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jungkamp
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.-C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Munschauer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Transcriptome-wide identification of RNA-binding protein and microRNA target sites by PAR-CLIP
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  141
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  129
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  141
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.cell.2010.03.009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHafner%26rft.auinit1%253DM.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D141%26rft.atitle%253DTranscriptome-wide%2Bidentification%2Bof%2BRNA-binding%2Bprotein%2Band%2BmicroRNA%2Btarget%2Bsites%2Bby%2BPAR-CLIP.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.009%26rft_id%253Dinfo%253Apmid%252F20371350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20371350&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000276211100018&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.16" data-doi="10.1093/biostatistics/kxr054">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irizarry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Removing technical variability in RNA-seq data using conditional quantile normalization
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  204
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  216
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/biostatistics/kxr054
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxr054%26rft_id%253Dinfo%253Apmid%252F22285995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxr054&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22285995&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000301293800002&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.17" data-doi="10.1038/ncomms11538">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hansen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Venø
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jensen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schaefer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Damgaard
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.K.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kjems
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Argonaute-associated short introns are a novel class of gene regulators
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  11538
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/ncomms11538
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D7%26rft.spage%253D11538%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/ncomms11538&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.18" data-doi="10.1038/nprot.2008.211">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sherman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.T.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lempicki
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Protoc
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  44
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  57
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nprot.2008.211
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DHuang%26rft.auinit1%253Dd.%2Ba.%2B%2B.%2BW.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253D44%26rft.epage%253D57%26rft.atitle%253DSystematic%2Band%2Bintegrative%2Banalysis%2Bof%2Blarge%2Bgene%2Blists%2Busing%2BDAVID%2Bbioinformatics%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2008.211%26rft_id%253Dinfo%253Apmid%252F19131956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nprot.2008.211&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19131956&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000265781800006&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.19" data-doi="10.1038/s41598-019-47552-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaiser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.W.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ignarski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Nostrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frese
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jain
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cukoski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Heinen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schaechter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Seufert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bunte
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A protein-RNA interaction atlas of the ribosome biogenesis factor AATF
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  11071
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41598-019-47552-3
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-47552-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-019-47552-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.20" data-doi="10.1073/pnas.1412842111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Katibah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sidote
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Collins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Broad and adaptable RNA structure recognition by the human interferon-induced tetratricopeptide repeat protein IFIT5
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  12025
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12030
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.1412842111
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1412842111%26rft_id%253Dinfo%253Apmid%252F25092312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzMzLzEyMDI1IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.21" data-doi="10.1038/s41587-019-0201-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Paggi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bennett
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.L
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Biotechnol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  907
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  915
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41587-019-0201-4
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol%26rft.volume%253D37%26rft.spage%253D907%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41587-019-0201-4%26rft_id%253Dinfo%253Apmid%252F31375807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41587-019-0201-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31375807&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.22" data-doi="10.1093/nar/gkz917">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yiya
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  ExonSkipDB: functional annotation of exon skipping event in human
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  48
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D896
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D907
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkz917
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D48%26rft.spage%253DD896%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkz917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkz917&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.23" data-doi="10.1038/s41467-019-13056-x">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kobak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berens
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The art of using t-SNE for single-cell transcriptomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  5416
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41467-019-13056-x
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-13056-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-019-13056-x&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.24" data-doi="10.1073/pnas.0608718103">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kulesza
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Murine cytomegalovirus encodes a stable intron that facilitates persistent replication in the mouse
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  103
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  18302
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  18307
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.0608718103
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608718103%26rft_id%253Dinfo%253Apmid%252F17105807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzQ4LzE4MzAyIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.25" data-doi="10.1038/nmeth.1923">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Langmead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Salzberg
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.L
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Fast gapped-read alignment with Bowtie 2
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  359
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nmeth.1923
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22388286&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000302218500017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.26" data-doi="10.1093/bioinformatics/btp352">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Handsaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wysoker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fennell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Homer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abecasis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Durbin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The sequence alignment/map format and SAMtools
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  25
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2078
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2079
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/bioinformatics/btp352
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19505943&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000268808600014&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.27" data-doi="10.1016/j.cels.2015.12.004">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liberzon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Birger
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thorvaldsdóttir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghandi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mesirov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.P.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tamayo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The Molecular Signatures Database (MSigDB) hallmark gene set collection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell Syst
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  1
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  417
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  425
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.cels.2015.12.004
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%2BSyst%26rft.volume%253D1%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2015.12.004%26rft_id%253Dinfo%253Apmid%252F26771021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cels.2015.12.004&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26771021&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.28" data-doi="10.14806/EJ.17.1.200">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Martin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Cutadapt removes adapter sequences from high-throughput sequencing reads
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  EMBnet.journal
                                 </abbr>
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 , pp.
                                 <span class="cit-fpage">
                                  10
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DEMBnet.journal%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252FEJ.17.1.200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.14806/EJ.17.1.200&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.29" data-doi="10.1080/15476286.2016.1243646">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Massenet
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bertrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verheggen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Assembly and trafficking of box C/D and H/ACA snoRNPs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  RNA Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  680
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  692
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1080/15476286.2016.1243646
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol%26rft.volume%253D14%26rft.spage%253D680%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1243646%26rft_id%253Dinfo%253Apmid%252F27715451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1243646&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27715451&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.30" data-doi="10.7554/eLife.28889">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mattijssen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arimbasseri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Iben
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gaidamakov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hafner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maraia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.J
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  LARP4 mRNA codon-tRNA match contributes to LARP4 activity for ribosomal protein mRNA poly(A) tail length protection
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e28889
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.7554/eLife.28889
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253D2173%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.28889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.28889&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.31" data-doi="10.1038/s41586-018-0828-1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Morgan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fink
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Excised linear introns regulate growth in yeast
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  565
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  606
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  611
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41586-018-0828-1
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D565%26rft.spage%253D606%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0828-1%26rft_id%253Dinfo%253Apmid%252F30651636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0828-1&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30651636&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.32" data-doi="10.1186/1471-2164-14-543">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moss
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.N.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Genome-wide analyses of Epstein-Barr virus reveal conserved RNA structures and a novel stable intronic sequence RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  543
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1186/1471-2164-14-543
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D543%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-543%26rft_id%253Dinfo%253Apmid%252F23937650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-543&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23937650&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.33" data-doi="10.1093/database/baz031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nunes
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mestre
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marcelo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koppenol
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Matos
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nóbrega
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  MSGP: the first database of the protein components of the mammalian stress granules
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Database
                                 </abbr>
                                 <span class="cit-vol">
                                  2019
                                 </span>
                                 , baz031.
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/database/baz031
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDatabase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdatabase%252Fbaz031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/database/baz031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.34" data-doi="10.1016/j.cell.2007.06.028">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Okamura
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hagen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tyler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The mirtron pathway generates microRNA-class regulatory RNAs in Drosophila
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  130
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  89
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  100
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.cell.2007.06.028
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DOkamura%26rft.auinit1%253DK.%26rft.volume%253D130%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D100%26rft.atitle%253DThe%2Bmirtron%2Bpathway%2Bgenerates%2BmicroRNA-class%2Bregulatory%2BRNAs%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.06.028%26rft_id%253Dinfo%253Apmid%252F17599402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.06.028&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17599402&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000248587000013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.35" data-doi="10.1038/s41586-018-0859-7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Parenteau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maignon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Berthoumieux
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Catala
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gagnon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abou Elela
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Introns are mediators of cell response to starvation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  565
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  612
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  617
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41586-018-0859-7
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D565%26rft.spage%253D612%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0859-7%26rft_id%253Dinfo%253Apmid%252F30651641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0859-7&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30651641&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.36" data-doi="10.1038/nmeth.4197">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Patro
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duggal
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Love
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irizarry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.A.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kingsford
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Salmon provides fast and bias-aware quantification of transcript expression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  417
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  419
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nmeth.4197
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BMethods%26rft.volume%253D14%26rft.spage%253D417%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4197%26rft_id%253Dinfo%253Apmid%252F28263959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.4197&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28263959&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.37" data-doi="10.1083/jcb.201507065">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Osman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.L.-I.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zheng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.T
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Stable intronic sequence RNAs have possible regulatory roles in Drosophila melanogaster
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Cell Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  211
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  243
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  251
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1083/jcb.201507065
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201507065%26rft_id%253Dinfo%253Apmid%252F26504165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIxMS8yLzI0MyI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA5LzIwMjAuMDkuMDcuMjg1MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.38" data-doi="10.1261/rna.054809.115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nottingham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mohr
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hunicke-Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  High-throughput sequencing of human plasma RNA by using thermostable group II intron reverse transcriptases
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  RNA
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  111
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  128
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1261/rna.054809.115
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.054809.115%26rft_id%253Dinfo%253Apmid%252F26554030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIyLzEvMTExIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.39" data-doi="10.1002/0471250953.bi1112s47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Quinlan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  BEDTools: the swiss-army tool for genome feature analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Curr. Protoc. Bioinformatics
                                 </abbr>
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  11.12.11
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  34
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1002/0471250953.bi1112s47
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D47%26rft.spage%253D111211%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi1112s47%26rft_id%253Dinfo%253Apmid%252F25199790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/0471250953.bi1112s47&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25199790&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.40" data-doi="10.1038/s41467-017-02554-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Risso
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Perraudeau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gribkova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dudoit
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A general and flexible method for signal extraction from single-cell RNA-seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  284
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41467-017-02554-5
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-017-02554-5%26rft_id%253Dinfo%253Apmid%252F29348443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-017-02554-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=29348443&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thorvaldsdóttir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winckler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guttman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Getz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mesirov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                 <abbr class="cit-jnl-abbrev">
                                  Integrative genomics viewer
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.42" data-doi="10.1038/nature05983">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruby
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.H.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bartel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.P
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Intronic microRNA precursors that bypass Drosha processing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  448
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  83
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  86
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nature05983
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRuby%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D448%26rft.issue%253D7149%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DIntronic%2BmicroRNA%2Bprecursors%2Bthat%2Bbypass%2BDrosha%2Bprocessing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05983%26rft_id%253Dinfo%253Apmid%252F17589500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature05983&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17589500&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000247720900042&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.43" data-doi="10.1016/j.cell.2014.10.040">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rybak-Wolf
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jens
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Murakawa
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Herzog
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Landthaler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rajewsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  A variety of Dicer substrates in human and C. elegans
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  159
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1153
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1167
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.cell.2014.10.040
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D159%26rft.spage%253D1153%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.10.040%26rft_id%253Dinfo%253Apmid%252F25416952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.10.040&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25416952&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.44" data-doi="10.7554/eLife.47809">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bicknell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eddy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Moore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Free circular introns with an unusual branchpoint in neuronal projections
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e47809
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.7554/eLife.47809
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.47809%26rft_id%253Dinfo%253Apmid%252F31697236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.47809&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31697236&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.45" data-doi="10.1016/j.molcel.2014.03.021">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Savage
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gorski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barros
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Irwin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Manti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Powell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pellagatti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lukashchuk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCance
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCluggage
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Identification of a BRCA1- mRNA splicing complex required for efficient DNA repair and maintenance of genomic stability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  445
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  459
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.molcel.2014.03.021
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol%2BCell%26rft.volume%253D54%26rft.spage%253D445%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.03.021%26rft_id%253Dinfo%253Apmid%252F24746700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.03.021&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24746700&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.46" data-doi="10.1186/1471-2164-13-23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharbati
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharbati
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hoeke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bohmer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Einspanier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Quantification and accurate normalisation of small RNAs through new custom RT-qPCR arrays demonstrates Salmonella-induced microRNAs in human monocytes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  BMC Genomics
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  23
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1186/1471-2164-13-23
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSharbati%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.spage%253D23%26rft.epage%253D23%26rft.atitle%253DQuantification%2Band%2Baccurate%2Bnormalisation%2Bof%2Bsmall%2BRNAs%2Bthrough%2Bnew%2Bcustom%2BRT-qPCR%2Barrays%2Bdemonstrates%2BSalmonella-induced%2BmicroRNAs%2Bin%2Bhuman%2Bmonocytes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-23%26rft_id%253Dinfo%253Apmid%252F22248082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-23&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22248082&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-2" title="View reference  in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.47" data-doi="10.1073/pnas.1419161111">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.-x.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Henry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  rMATS: Robust and flexible detection of differential alternative splicing from replicate RNA-Seq data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E5593
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E5601
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.1419161111
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1419161111%26rft_id%253Dinfo%253Apmid%252F25480548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzUxL0U1NTkzIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.48" data-doi="10.1093/nar/gkl556">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sheth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roca
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hastings
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roeder
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krainer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sachidanandam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Comprehensive splice-site analysis using comparative genomics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3955
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3967
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkl556
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl556%26rft_id%253Dinfo%253Apmid%252F16914448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkl556&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16914448&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000240583800017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.49" data-doi="10.1073/pnas.1712108114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shurtleff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nottingham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Temoche-Diaz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schekman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Broad role for YBX1 in defining the small noncoding RNA composition of exosomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E8987
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E8995
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.1712108114
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1712108114%26rft_id%253Dinfo%253Apmid%252F29073095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzQzL0U4OTg3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.50" data-doi="10.1101/gr.3715005">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Siepel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bejerano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pedersen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinrichs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosenbloom
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clawson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Spieth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hillier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Richards
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Research
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1034
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1050
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1101/gr.3715005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3715005%26rft_id%253Dinfo%253Apmid%252F16024819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvMTAzNCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA5LzIwMjAuMDkuMDcuMjg1MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.51" data-doi="10.1073/pnas.86.21.8280">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  1989
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Gamma-monomethyl phosphate: a cap structure in spliceosomal U6 small nuclear RNA
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  86
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  8280
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8283
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.86.21.8280
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSingh%26rft.auinit1%253DR.%26rft.volume%253D86%26rft.issue%253D21%26rft.spage%253D8280%26rft.epage%253D8283%26rft.atitle%253DGamma-monomethyl%2Bphosphate%253A%2Ba%2Bcap%2Bstructure%2Bin%2Bspliceosomal%2BU6%2Bsmall%2Bnuclear%2BRNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.21.8280%26rft_id%253Dinfo%253Apmid%252F2813391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODYvMjEvODI4MCI7czo0OiJhdG9tIjtzOjQ4OiIvYmlvcnhpdi9lYXJseS8yMDIyLzAzLzA5LzIwMjAuMDkuMDcuMjg1MTE0LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.52" data-doi="10.1073/pnas.1808816115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Talhouarne
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.J.S.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.G
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Lariat intronic RNAs in the cytoplasm of vertebrate cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U.S.A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E7970
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  E7977
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1073/pnas.1808816115
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1808816115%26rft_id%253Dinfo%253Apmid%252F30082412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzM0L0U3OTcwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.53" data-doi="10.1038/nature11247">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  The ENCODE Project Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  An integrated encyclopedia of DNA elements in the human genome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  489
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  57
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  74
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nature11247
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature11247&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22955616&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000308347000039&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.54" data-doi="10.1093/nar/gky1049">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth cit-collab">
                                  The UniProt Consortium
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  UniProt: a worldwide hub of protein knowledge
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D506
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D515
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gky1049
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D47%26rft.spage%253DD506%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1049%26rft_id%253Dinfo%253Apmid%252F30395287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gky1049&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30395287&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.09.07.285114v4.55">
                               <div class="cit-metadata">
                                <ol class="duplicate">
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    The RNA
                                   </span>
                                   <span class="cit-name-surname">
                                    World
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    Third
                                   </span>
                                   <span class="cit-name-surname">
                                    Edition
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    R.F.
                                   </span>
                                   <span class="cit-name-surname">
                                    Gesteland
                                   </span>
                                  </span>
                                  ,
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    T.R.
                                   </span>
                                   <span class="cit-name-surname">
                                    Cech
                                   </span>
                                  </span>
                                  , and
                                 </li>
                                 <li>
                                  <span class="cit-ed">
                                   <span class="cit-name-given-names">
                                    J.F.
                                   </span>
                                   <span class="cit-name-surname">
                                    Atkins
                                   </span>
                                  </span>
                                 </li>
                                </ol>
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tycowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kolev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Conrad
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fok
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 ).
                                 <span class="cit-chapter-title">
                                  The ever-growing world of small nuclear ribonucleoproteins
                                 </span>
                                 . In
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   The RNA
                                  </span>
                                  <span class="cit-name-surname">
                                   World
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   Third
                                  </span>
                                  <span class="cit-name-surname">
                                   Edition
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   R.F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Gesteland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   T.R.
                                  </span>
                                  <span class="cit-name-surname">
                                   Cech
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-ed">
                                  <span class="cit-name-given-names">
                                   J.F.
                                  </span>
                                  <span class="cit-name-surname">
                                   Atkins
                                  </span>
                                 </span>
                                 , eds. (
                                 <span class="cit-publ-name">
                                  Cold Spring Harbor Laboratory Press
                                 </span>
                                 ,
                                 <span class="cit-publ-loc">
                                  NY
                                 </span>
                                 .), pp.
                                 <span class="cit-fpage">
                                  327
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  368
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.56" data-doi="10.1038/nmeth.3810">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Nostrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pratt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shishkin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gelboin-Burkhart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sundararaman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blue
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Surka
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elkins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  508
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  514
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/nmeth.3810
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D13%26rft.spage%253D508%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3810%26rft_id%253Dinfo%253Apmid%252F27018577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3810&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=27018577&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.57" data-doi="10.1186/s13059-020-01982-9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Van Nostrand
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pratt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wheeler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Blue
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mueller
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Garcia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gelboin-Burkhart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nguyen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Principles of RNA processing from analysis of enhanced CLIP maps for 150 RNA binding proteins
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Genome Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  21
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  90
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1186/s13059-020-01982-9
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-020-01982-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1186/s13059-020-01982-9&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.58" data-doi="10.1093/nar/gkx1046">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vohhodina
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barros
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Savage
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liberante
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Manti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bankhead
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cosgrove
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Madden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harkin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.P.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Savage
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.I
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  The RNA processing factors THRAP3 and BCLAF1 promote the DNA damage response through selective mRNA splicing and nuclear export
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 <span class="cit-vol">
                                  45
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  12816
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  12833
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkx1046
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft.volume%253D45%26rft.spage%253D12816%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkx1046&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.59" data-doi="10.1128/MCB.22.24.8457-8466.2002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Query
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Meier
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   U.T
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Immunopurified small nucleolar ribonucleoprotein particles pseudouridylate rRNA independently of their association with phosphorylated Nopp140
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol Cell Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  8457
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  8466
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1128/mcb.22.24.8457-8466.2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWang%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D24%26rft.spage%253D8457%26rft.epage%253D8466%26rft.atitle%253DImmunopurified%2BSmall%2BNucleolar%2BRibonucleoprotein%2BParticles%2BPseudouridylate%2BrRNA%2BIndependently%2Bof%2BTheir%2BAssociation%2Bwith%2BPhosphorylated%2BNopp140%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.24.8457-8466.2002%26rft_id%253Dinfo%253Apmid%252F12446766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yNC84NDU3IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDMvMDkvMjAyMC4wOS4wNy4yODUxMTQuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.60" data-doi="10.1371/journal.pcbi.1004441">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ladewig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mohammed
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Analysis of nearly one thousand mammalian mirtrons reveals novel features of Dicer substrates. PLoS Computat
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biol
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e1004441
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1371/journal.pcbi.1004441
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiol%26rft.volume%253D11%26rft.spage%253D1004441e%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004441&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.61" data-doi="10.1146/annurev-biochem-091719-064225">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilkinson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Charenton
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nagai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  RNA splicing by the spliceosome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Annu. Rev. Biochem
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  89
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  359
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  388
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1146/annurev-biochem-091719-064225
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BBiochem%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-091719-064225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-091719-064225&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.62" data-doi="10.1038/s41598-019-44457-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Improved TGIRT-seq methods for comprehensive transcriptome profiling with decreased adapter dimer formation and bias correction
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Sci. Rep
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  7953
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1038/s41598-019-44457-z
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSci.%2BRep%26rft.volume%253D9%26rft.spage%253D7953%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-019-44457-z%26rft_id%253Dinfo%253Apmid%252F31138886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-019-44457-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=31138886&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.63" data-doi="10.7554/eLife.60743">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nottingham
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lambowitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Identification of protein-protected mRNA fragments and structured excised intron RNAs in human plasma by TGIRT-seq peak calling
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  eLife
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e60743
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.7554/eLife.60743
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D9%26rft.spage%253D60743%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.60743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.7554/eLife.60743&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.64" data-doi="10.1016/j.molcel.2019.09.014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Youn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dyakov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.J.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Knight
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.D.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vernon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Forman-Kay
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.D.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gingras
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.-C
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Properties of stress granule and P-body proteomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  76
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  286
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  294
                                 </span>
                                 .
                                 <a href="https://doi.org/10.1016/j.molcel.2019.09.014">
                                  https://doi.org/10.1016/j.molcel.2019.09.014
                                 </a>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D76%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2019.09.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2019.09.014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.65" data-doi="10.1016/j.molcel.2013.08.017">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.-O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xiang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.-F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.-F.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xing
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.-H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.-L
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  Circular intronic long noncoding RNAs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Mol. Cell
                                 </abbr>
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  792
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  806
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1016/j.molcel.2013.08.017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D51%26rft.spage%253D792%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.08.017%26rft_id%253Dinfo%253Apmid%252F24035497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.08.017&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24035497&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000330191200009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label ref-label-empty">
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.09.07.285114v4.66" data-doi="10.1093/nar/gkv1268">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mitra
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z
                                  </span>
                                 </span>
                                 . (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                 <span class="cit-article-title">
                                  TSGene 2.0: an updated literature-based knowledgebase for tumor suppressor genes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nucleic Acids Res
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  44
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  D1023
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  D1031
                                 </span>
                                 .
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  <span class="cit-pub-id-scheme-doi">
                                   doi:
                                  </span>
                                  10.1093/nar/gkv1268
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1268%26rft_id%253Dinfo%253Apmid%252F26590405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/nar/gkv1268&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26590405&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F09%2F2020.09.07.285114.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.09.07.285114v4" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.10.22.464987v2" title="Filopodia-mediated basement membrane assembly at pre-invasive tumor boundaries" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.28.481967v2" title="Multivariate genetic analysis of personality and cognitive traits reveals abundant pleiotropy and improves prediction" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 09, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.09.07.285114v4.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.09.07.285114v4.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.09.07.285114v4.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.09.07.285114v4" class="active" data-icon-position="" data-hide-link-title="0">
                                   Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2437873"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-q4_InCqr0QPtB_iF5pgF0MQWMBEkoUDksO8zADXgXq4"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484478588"/>
                                   <input type="hidden" name="captcha_token" value="727c8fd72a008502d1582f5c4022a932"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2437873--3" data-node-nid="2437873" id="node-2437873--41227983804" data-pisa="biorxiv;2020.09.07.285114v4" data-pisa-master="biorxiv;2020.09.07.285114" data-seqnum="2437873" data-apath="/biorxiv/early/2022/03/09/2020.09.07.285114.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Jun
                                  </span>
                                  <span class="nlm-surname">
                                   Yao
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Shelby
                                  </span>
                                  <span class="nlm-surname">
                                   Winans
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Hengyi
                                  </span>
                                  <span class="nlm-surname">
                                   Xu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Douglas C.
                                  </span>
                                  <span class="nlm-surname">
                                   Wu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Elizabeth A.
                                  </span>
                                  <span class="nlm-surname">
                                   Ferrick-Kiddie
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Manuel
                                  </span>
                                  <span class="nlm-surname">
                                   Ares
                                  </span>
                                  <span class="nlm-suffix">
                                   Jr.
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Alan M.
                                  </span>
                                  <span class="nlm-surname">
                                   Lambowitz
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.09.07.285114;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.09.07.285114
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&title=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&title=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&text=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&t=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&title=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&title=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&title=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2437873--5" data-node-nid="2437873" id="citation-node-2437873--61237401917" data-pisa="biorxiv;2020.09.07.285114v4" data-pisa-master="biorxiv;2020.09.07.285114" data-seqnum="2437873" data-apath="/biorxiv/early/2022/03/09/2020.09.07.285114.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Human cells contain myriad excised linear intron RNAs with links to gene regulation and potential utility as biomarkers
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Jun
                                      </span>
                                      <span class="nlm-surname">
                                       Yao
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Shelby
                                      </span>
                                      <span class="nlm-surname">
                                       Winans
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Hengyi
                                      </span>
                                      <span class="nlm-surname">
                                       Xu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Douglas C.
                                      </span>
                                      <span class="nlm-surname">
                                       Wu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Elizabeth A.
                                      </span>
                                      <span class="nlm-surname">
                                       Ferrick-Kiddie
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Manuel
                                      </span>
                                      <span class="nlm-surname">
                                       Ares
                                      </span>
                                      <span class="nlm-suffix">
                                       Jr.
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Alan M.
                                      </span>
                                      <span class="nlm-surname">
                                       Lambowitz
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.09.07.285114;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.09.07.285114
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Yao%2C+Jun&rft.au=Winans%2C+Shelby&rft.au=Xu%2C+Hengyi&rft.au=Wu%2C+Douglas+C.&rft.au=Ferrick-Kiddie%2C+Elizabeth+A.&rft.au=Ares%2C+Manuel&rft.au=Lambowitz%2C+Alan+M.">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2437873/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2437873/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2437873/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2437873/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2437873/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2437873/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2437873/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2437873/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2437873/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2437873/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2437873/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2437873/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&count=horizontal&via=&text=Human%20cells%20contain%20myriad%20excised%20linear%20intron%20RNAs%20with%20links%20to%20gene%20regulation%20and%20potential%20utility%20as%20biomarkers&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.09.07.285114v4&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.09.07.285114v4" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genomics" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genomics
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23507)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11824)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11440)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15189)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47703)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
