<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.07.30.223149v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.07.30.223149v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.07.30.223149v2.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/embed/graphic-8.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/02/14/2020.07.30.223149.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.07.30.223149v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations"/>
  <meta name="DC.Identifier" content="10.1101/2020.07.30.223149"/>
  <meta name="DC.Date" content="2022-02-14"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Cells sense, manipulate and respond to their mechanical microenvironment in a plethora of physiological processes, yet the understanding of how cells transmit, receive and interpret environmental cues to communicate with distant cells is severely limited due to lack of tools to quantitatively infer the complex tangle of dynamic cell-cell interactions in complicated environments. We present a computational method to systematically infer and quantify long-range mechanical cell-cell communication through the extracellular matrix (cell-ECM-cell communication) by correlating ECM remodeling fluctuations in between communicating cells and demonstrating that these fluctuations contain sufficient information to define unique signatures that robustly distinguish between different pairs of communicating cells. We demonstrate our method with finite element simulations and live 3D imaging of fibroblasts and cancer cells embedded in fibrin gels. While previous studies relayed on the formation of a visible fibrous ‘band’ extending between cells to inform on mechanical communication, our method detected mechanical propagation even in cases where visible bands never formed. We revealed that while contractility is required, band formation is not necessary, for cell-ECM-cell communication, and that mechanical signals propagate from one cell to another even upon massive reduction in their contractility. Our method sets the stage to measure the fundamental aspects of intercellular long-range mechanical communication in physiological contexts and may provide a new functional readout for high content 3D image-based screening. The ability to infer cell-ECM-cell communication using standard confocal microscopy holds the promise for wide use and democratizing the method.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="DC.Contributor" content="Assaf Nahum"/>
  <meta name="DC.Contributor" content="Yoni Koren"/>
  <meta name="DC.Contributor" content="Bar Ergaz"/>
  <meta name="DC.Contributor" content="Sari Natan"/>
  <meta name="DC.Contributor" content="Shahar Goren"/>
  <meta name="DC.Contributor" content="Avraham Kolel"/>
  <meta name="DC.Contributor" content="Sankar Jagadeeshan"/>
  <meta name="DC.Contributor" content="Moshe Elkabets"/>
  <meta name="DC.Contributor" content="Ayelet Lesman"/>
  <meta name="DC.Contributor" content="Assaf Zaritsky"/>
  <meta name="article:published_time" content="2022-02-14"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Cells sense, manipulate and respond to their mechanical microenvironment in a plethora of physiological processes, yet the understanding of how cells transmit, receive and interpret environmental cues to communicate with distant cells is severely limited due to lack of tools to quantitatively infer the complex tangle of dynamic cell-cell interactions in complicated environments. We present a computational method to systematically infer and quantify long-range mechanical cell-cell communication through the extracellular matrix (cell-ECM-cell communication) by correlating ECM remodeling fluctuations in between communicating cells and demonstrating that these fluctuations contain sufficient information to define unique signatures that robustly distinguish between different pairs of communicating cells. We demonstrate our method with finite element simulations and live 3D imaging of fibroblasts and cancer cells embedded in fibrin gels. While previous studies relayed on the formation of a visible fibrous ‘band’ extending between cells to inform on mechanical communication, our method detected mechanical propagation even in cases where visible bands never formed. We revealed that while contractility is required, band formation is not necessary, for cell-ECM-cell communication, and that mechanical signals propagate from one cell to another even upon massive reduction in their contractility. Our method sets the stage to measure the fundamental aspects of intercellular long-range mechanical communication in physiological contexts and may provide a new functional readout for high content 3D image-based screening. The ability to infer cell-ECM-cell communication using standard confocal microscopy holds the promise for wide use and democratizing the method.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.07.30.223149v2"/>
  <meta name="citation_id" content="2020.07.30.223149v2"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.07.30.223149"/>
  <meta name="citation_num_pages" content="69"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.07.30.223149"/>
  <meta name="citation_author" content="Assaf Nahum"/>
  <meta name="citation_author_institution" content="Department of Software and Information Systems Engineering, Ben-Gurion University of the Negev"/>
  <meta name="citation_author" content="Yoni Koren"/>
  <meta name="citation_author_institution" content="School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author" content="Bar Ergaz"/>
  <meta name="citation_author_institution" content="School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author" content="Sari Natan"/>
  <meta name="citation_author_institution" content="School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9419-3808"/>
  <meta name="citation_author" content="Shahar Goren"/>
  <meta name="citation_author_institution" content="Department of Biomedical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author" content="Avraham Kolel"/>
  <meta name="citation_author_institution" content="School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author" content="Sankar Jagadeeshan"/>
  <meta name="citation_author_institution" content="The Shraga Segal Dept. of Microbiology, Immunology and Genetics, Faculty of Health Sciences, Ben-Gurion University of the Negev"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3778-8757"/>
  <meta name="citation_author" content="Moshe Elkabets"/>
  <meta name="citation_author_institution" content="The Shraga Segal Dept. of Microbiology, Immunology and Genetics, Faculty of Health Sciences, Ben-Gurion University of the Negev"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3634-9098"/>
  <meta name="citation_author" content="Ayelet Lesman"/>
  <meta name="citation_author_institution" content="School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University"/>
  <meta name="citation_author_email" content="assafza@bgu.ac.il"/>
  <meta name="citation_author_email" content="ayeletlesman@tauex.tau.ac.il"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2511-4381"/>
  <meta name="citation_author" content="Assaf Zaritsky"/>
  <meta name="citation_author_institution" content="Department of Software and Information Systems Engineering, Ben-Gurion University of the Negev"/>
  <meta name="citation_author_email" content="assafza@bgu.ac.il"/>
  <meta name="citation_author_email" content="ayeletlesman@tauex.tau.ac.il"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1477-5478"/>
  <meta name="citation_reference" content="Korff, T. & Augustin, H.G. Tensional forces in fibrillar extracellular matrices control directional capillary sprouting. J Cell Sci 112, 3249–3258 (1999)."/>
  <meta name="citation_reference" content="Reinhart-King, C.A., Dembo, M. & Hammer, D.A. Cell-cell mechanical communication through compliant substrates. Biophysical journal 95, 6044–6051 (2008)."/>
  <meta name="citation_reference" content="Winer, J.P., Oake, S. & Janmey, P.A. Non-linear elasticity of extracellular matrices enables contractile cells to communicate local position and orientation. PLoS One 4, e6382 (2009)."/>
  <meta name="citation_reference" content="Ma, X., Schickel, M.E., Stevenson, M.D., Sarang-Sieminski, A.L., Gooch, K.J., Ghadiali, S.N. & Hart, R.T. Fibers in the extracellular matrix enable long-range stress transmission between cells. Biophysical journal 104, 1410–1418 (2013)."/>
  <meta name="citation_reference" content="Shi, Q., Ghosh, R.P., Engelke, H., Rycroft, C.H., Cassereau, L., Sethian, J.A., Weaver, V.M. & Liphardt, J.T. Rapid disorganization of mechanically interacting systems of mammary acini. Proceedings of the National Academy of Sciences 111, 658–663 (2014)."/>
  <meta name="citation_reference" content="Notbohm, J., Lesman, A., Rosakis, P., Tirrell, D.A. & Ravichandran, G. Microbuckling of fibrin provides a mechanism for cell mechanosensing. Journal of The Royal Society Interface 12, 20150320 (2015)."/>
  <meta name="citation_reference" content="Nitsan, I., Drori, S., Lewis, Y.E., Cohen, S. & Tzlil, S. Mechanical communication in cardiac cell synchronized beating. Nature Physics (2016)."/>
  <meta name="citation_reference" content="Ronceray, P., Broedersz, C.P. & Lenz, M. Fiber networks amplify active stress. Proc Natl Acad Sci U S A 113, 2827–2832 (2016)."/>
  <meta name="citation_reference" content="Kim, J., Feng, J., Jones, C.A., Mao, X., Sander, L.M., Levine, H. & Sun, B. Stress-induced plasticity of dynamic collagen networks. Nature communications 8, 842 (2017)."/>
  <meta name="citation_reference" content="Han, Y.L., Ronceray, P., Xu, G., Malandrino, A., Kamm, R.D., Lenz, M., Broedersz, C.P. & Guo, M. Cell contraction induces long-ranged stress stiffening in the extracellular matrix. Proc Natl Acad Sci U S A 115, 4075–4080 (2018)."/>
  <meta name="citation_reference" content="Ban, E., Franklin, J.M., Nam, S., Smith, L.R., Wang, H., Wells, R.G., Chaudhuri, O., Liphardt, J.T. & Shenoy, V.B. Mechanisms of Plastic Deformation in Collagen Networks Induced by Cellular Forces. Biophysical journal 114, 450–461 (2018)."/>
  <meta name="citation_reference" content="Goren, S., Koren, Y., Xu, X. & Lesman, A. Elastic Anisotropy Governs the Range of Cell-Induced Displacements. Biophysical journal 118, 1152–1164 (2020)."/>
  <meta name="citation_reference" content="Natan, S., Koren, Y., Shelah, O., Goren, S. & Lesman, A. Long-range mechanical coupling of cells in 3D fibrin gels. Molecular biology of the cell, mbc. E20–01-0079 (2020)."/>
  <meta name="citation_reference" content="Davidson, C.D., DePalma, S.J., Wang, W.Y., Kamen, J.L., Jayco, D.K.P. & Baker, B.M. Mechanical intercellular communication via matrix-borne cell force transmission during vascular network formation. bioRxiv, 2021.2008.2017.456669 (2021)."/>
  <meta name="citation_reference" content="Doyle, A.D. & Yamada, K.M. Mechanosensing via cell-matrix adhesions in 3D microenvironments. Exp Cell Res 343, 60–66 (2016)."/>
  <meta name="citation_reference" content="Pakshir, P., Alizadehgiashi, M., Wong, B., Coelho, N.M., Chen, X., Gong, Z., Shenoy, V.B., McCulloch, C.A. & Hinz, B. Dynamic fibroblast contractions attract remote macrophages in fibrillar collagen matrix. Nat Commun 10, 1850 (2019)."/>
  <meta name="citation_reference" content="Stopak, D. & Harris, A.K. Connective tissue morphogenesis by fibroblast traction: I. Tissue culture observations. Developmental biology 90, 383–398 (1982)."/>
  <meta name="citation_reference" content="Davis, G.E. & Camarillo, C.W. Regulation of endothelial cell morphogenesis by integrins, mechanical forces, and matrix guidance pathways. Exp Cell Res 216, 113–123 (1995)."/>
  <meta name="citation_reference" content="Guo, C.L., Ouyang, M., Yu, J.Y., Maslov, J., Price, A. & Shen, C.Y. Long-range mechanical force enables self-assembly of epithelial tubular patterns. Proc Natl Acad Sci U S A 109, 5576–5582 (2012)."/>
  <meta name="citation_reference" content="Duscher, D., Maan, Z.N., Wong, V.W., Rennert, R.C., Januszyk, M., Rodrigues, M., Hu, M., Whitmore, A.J., Whittam, A.J. & Longaker, M.T. Mechanotransduction and fibrosis. Journal of biomechanics 47, 1997–2005 (2014)."/>
  <meta name="citation_reference" content="Han, W., Chen, S., Yuan, W., Fan, Q., Tian, J., Wang, X., Chen, L., Zhang, X., Wei, W., Liu, R., Qu, J., Jiao, Y., Austin, R.H. & Liu, L. Oriented collagen fibers direct tumor cell intravasation. Proc Natl Acad Sci U S A 113, 11208–11213 (2016)."/>
  <meta name="citation_reference" content="Ahmadzadeh, H., Webster, M.R., Behera, R., Jimenez Valencia, A.M., Wirtz, D., Weeraratna, A.T. & Shenoy, V.B. Modeling the two-way feedback between contractility and matrix realignment reveals a nonlinear mode of cancer cell invasion. Proc Natl Acad Sci U S A 114, E1617–e1626 (2017)."/>
  <meta name="citation_reference" content="Long, Y., Niu, Y., Liang, K. & Du, Y. Mechanical communication in fibrosis progression. Trends Cell Biol 32, 70–90 (2022)."/>
  <meta name="citation_reference" content="Wells, R.G. Tissue mechanics and fibrosis. Biochimica et Biophysica Acta (BBA)-Molecular Basis of Disease 1832, 884–890 (2013)."/>
  <meta name="citation_reference" content="Conklin, M.W., Eickhoff, J.C., Riching, K.M., Pehlke, C.A., Eliceiri, K.W., Provenzano, P.P., Friedl, A. & Keely, P.J. Aligned collagen is a prognostic signature for survival in human breast carcinoma. Am J Pathol 178, 1221–1232 (2011)."/>
  <meta name="citation_reference" content="Zemel, A. & Kemkemer, R. Cell–Matrix and Cell–Cell Mechanical Interactions. Cell and Matrix Mechanics, 136–165 (2014)."/>
  <meta name="citation_reference" content="Wang, H., Abhilash, A.S., Chen, C.S., Wells, R.G. & Shenoy, V.B. Long-range force transmission in fibrous matrices enabled by tension-driven alignment of fibers. Biophysical journal 107, 2592–2603 (2014)."/>
  <meta name="citation_reference" content="Clark, A.G., Maitra, A., Jacques, C., Simon, A., Pérez-González, C., Trepat, X., Voituriez, R. & Vignjevic, D.M. Viscoelastic relaxation of collagen networks provides a self-generated directional cue during collective migration. bioRxiv (2020)."/>
  <meta name="citation_reference" content="Wershof, E., Park, D., Barry, D.J., Jenkins, R.P., Rullan, A., Wilkins, A., Schlegelmilch, K., Roxanis, I., Anderson, K.I., Bates, P.A. & Sahai, E. A FIJI macro for quantifying pattern in extracellular matrix. Life Sci Alliance 4 (2021)."/>
  <meta name="citation_reference" content="Sopher, R.S., Tokash, H., Natan, S., Sharabi, M., Shelah, O., Tchaicheeyan, O. & Lesman, A. Nonlinear Elasticity of the ECM Fibers Facilitates Efficient Intercellular Communication. Biophysical journal 115, 1357–1370 (2018)."/>
  <meta name="citation_reference" content="Mann, A., Sopher, R.S., Goren, S., Shelah, O., Tchaicheeyan, O. & Lesman, A. Force chains in cell-cell mechanical communication. J R Soc Interface 16, 20190348 (2019)."/>
  <meta name="citation_reference" content="Mohammadi, H., Janmey, P.A. & McCulloch, C.A. Lateral boundary mechanosensing by adherent cells in a collagen gel system. Biomaterials 35, 1138–1149 (2014)."/>
  <meta name="citation_reference" content="Vicente-Manzanares, M., Ma, X., Adelstein, R.S. & Horwitz, A.R. Non-muscle myosin II takes centre stage in cell adhesion and migration. Nature reviews Molecular cell biology 10, 778–790 (2009)."/>
  <meta name="citation_reference" content="Lim, J.I., Sabouri-Ghomi, M., Machacek, M., Waterman, C.M. & Danuser, G. Protrusion and actin assembly are coupled to the organization of lamellar contractile structures. Experimental cell research 316, 2027–2041 (2010)."/>
  <meta name="citation_reference" content="Ng, M.R., Besser, A., Brugge, J.S. & Danuser, G. Mapping the dynamics of force transduction at cell–cell junctions of epithelial clusters. Elife 3, e03282 (2015)."/>
  <meta name="citation_reference" content="Zaritsky, A., Tseng, Y.-Y., Rabadán, M.A., Krishna, S., Overholtzer, M., Danuser, G. & Hall, A. Diverse roles of guanine nucleotide exchange factors in regulating collective cell migration. The Journal of cell biology, jcb. 201609095 (2017)."/>
  <meta name="citation_reference" content="Machacek, M., Hodgson, L., Welch, C., Elliott, H., Pertz, O., Nalbant, P., Abell, A., Johnson, G.L., Hahn, K.M. & Danuser, G. Coordination of Rho GTPase activities during cell protrusion. Nature 461, 99–103 (2009)."/>
  <meta name="citation_reference" content="Discher, D.E., Janmey, P. & Wang, Y.L. Tissue cells feel and respond to the stiffness of their substrate. Science 310, 1139–1143 (2005)."/>
  <meta name="citation_reference" content="Bray, M.-A., Singh, S., Han, H., Davis, C.T., Borgeson, B., Hartland, C., Kost-Alimova, M., Gustafsdottir, S.M., Gibson, C.C. & Carpenter, A.E. Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes. Nature protocols 11, 1757 (2016)."/>
  <meta name="citation_reference" content="McQuin, C., Goodman, A., Chernyshev, V., Kamentsky, L., Cimini, B.A., Karhohs, K.W., Doan, M., Ding, L., Rafelski, S.M., Thirstrup, D., Wiegraebe, W., Singh, S., Becker, T., Caicedo, J.C. & Carpenter, A.E. CellProfiler 3.0: Next-generation image processing for biology. PLoS Biol 16, e2005970 (2018)."/>
  <meta name="citation_reference" content="Edmondson, R., Broglie, J.J., Adcock, A.F. & Yang, L. Three-dimensional cell culture systems and their applications in drug discovery and cell-based biosensors. Assay Drug Dev Technol 12, 207–218 (2014)."/>
  <meta name="citation_reference" content="Desai, S.S., Tung, J.C., Zhou, V.X., Grenert, J.P., Malato, Y., Rezvani, M., Español-Suñer, R., Willenbring, H., Weaver, V.M. & Chang, T.T. Physiological ranges of matrix rigidity modulate primary mouse hepatocyte function in part through hepatocyte nuclear factor 4 alpha. Hepatology 64, 261–275 (2016)."/>
  <meta name="citation_reference" content="Kang, H., Wen, Q., Janmey, P.A., Tang, J.X., Conti, E. & MacKintosh, F.C. Nonlinear elasticity of stiff filament networks: strain stiffening, negative normal stress, and filament alignment in fibrin gels. J Phys Chem B 113, 3799–3805 (2009)."/>
  <meta name="citation_reference" content="Wade, R.J. & Burdick, J.A. Engineering ECM signals into biomaterials. Materials Today 15, 454–459 (2012)."/>
  <meta name="citation_reference" content="Liang, L., Jones, C., Chen, S., Sun, B. & Jiao, Y. Heterogeneous force network in 3D cellularized collagen networks. Phys Biol 13, 066001 (2016)."/>
  <meta name="citation_reference" content="Vader, D., Kabla, A., Weitz, D. & Mahadevan, L. Strain-induced alignment in collagen gels. PLoS One 4, e5902 (2009)."/>
  <meta name="citation_reference" content="Storm, C., Pastore, J.J., MacKintosh, F.C., Lubensky, T.C. & Janmey, P.A. Nonlinear elasticity in biological gels. Nature 435, 191–194 (2005)."/>
  <meta name="citation_reference" content="Friedmann-Morvinski, D., Bushong, E.A., Ke, E., Soda, Y., Marumoto, T., Singer, O., Ellisman, M.H. & Verma, I.M. Dedifferentiation of neurons and astrocytes by oncogenes can induce gliomas in mice. Science 338, 1080–1084 (2012)."/>
  <meta name="citation_reference" content="Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J.Y., White, D.J., Hartenstein, V., Eliceiri, K., Tomancak, P. & Cardona, A. Fiji: an open-source platform for biological-image analysis. Nat Methods 9, 676–682 (2012)."/>
  <meta name="citation_reference" content="Cordelieres, F. & Jackson, J. 3D objects counter, ImageJ: Image Processing and Analysis in Java. Available at rsb. info. nih. gov/ij/plugins/track/objects. html (verified 5 January 2011) (2005)."/>
  <meta name="citation_reference" content="Kwiatkowski, D., Phillips, P.C., Schmidt, P. & Shin, Y. Testing the null hypothesis of stationarity against the alternative of a unit root: How sure are we that economic time series have a unit root? Journal of econometrics 54, 159–178 (1992)."/>
  <meta name="citation_reference" content="Cheung, Y.-W. & Lai, K.S. Lag order and critical values of the augmented Dickey–Fuller test. Journal of Business & Economic Statistics 13, 277–280 (1995)."/>
  <meta name="citation_reference" content="Steinwachs, J., Metzner, C., Skodzek, K., Lang, N., Thievessen, I., Mark, C., Münster, S., Aifantis, K.E. & Fabry, B. Three-dimensional force microscopy of cells in biopolymer networks. Nat Methods 13, 171–176 (2016)."/>
  <meta name="twitter:title" content="Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Cells sense, manipulate and respond to their mechanical microenvironment in a plethora of physiological processes, yet the understanding of how cells transmit, receive and interpret environmental cues to communicate with distant cells is severely limited due to lack of tools to quantitatively infer the complex tangle of dynamic cell-cell interactions in complicated environments. We present a computational method to systematically infer and quantify long-range mechanical cell-cell communication through the extracellular matrix (cell-ECM-cell communication) by correlating ECM remodeling fluctuations in between communicating cells and demonstrating that these fluctuations contain sufficient information to define unique signatures that robustly distinguish between different pairs of communicating cells. We demonstrate our method with finite element simulations and live 3D imaging of fibroblasts and cancer cells embedded in fibrin gels. While previous studies relayed on the formation of a visible fibrous ‘band’ extending between cells to inform on mechanical communication, our method detected mechanical propagation even in cases where visible bands never formed. We revealed that while contractility is required, band formation is not necessary, for cell-ECM-cell communication, and that mechanical signals propagate from one cell to another even upon massive reduction in their contractility. Our method sets the stage to measure the fundamental aspects of intercellular long-range mechanical communication in physiological contexts and may provide a new functional readout for high content 3D image-based screening. The ability to infer cell-ECM-cell communication using standard confocal microscopy holds the promise for wide use and democratizing the method.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-title" property="og:title" content="Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Cells sense, manipulate and respond to their mechanical microenvironment in a plethora of physiological processes, yet the understanding of how cells transmit, receive and interpret environmental cues to communicate with distant cells is severely limited due to lack of tools to quantitatively infer the complex tangle of dynamic cell-cell interactions in complicated environments. We present a computational method to systematically infer and quantify long-range mechanical cell-cell communication through the extracellular matrix (cell-ECM-cell communication) by correlating ECM remodeling fluctuations in between communicating cells and demonstrating that these fluctuations contain sufficient information to define unique signatures that robustly distinguish between different pairs of communicating cells. We demonstrate our method with finite element simulations and live 3D imaging of fibroblasts and cancer cells embedded in fibrin gels. While previous studies relayed on the formation of a visible fibrous ‘band’ extending between cells to inform on mechanical communication, our method detected mechanical propagation even in cases where visible bands never formed. We revealed that while contractility is required, band formation is not necessary, for cell-ECM-cell communication, and that mechanical signals propagate from one cell to another even upon massive reduction in their contractility. Our method sets the stage to measure the fundamental aspects of intercellular long-range mechanical communication in physiological contexts and may provide a new functional readout for high content 3D image-based screening. The ability to infer cell-ECM-cell communication using standard confocal microscopy holds the promise for wide use and democratizing the method.

### Competing Interest Statement

The authors have declared no competing interest."/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-02-14"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2401796"/>
  <title>
   Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8NyPGe0WzGntZ-9a_1t0vTuFe2OJBZMw4iNlBoq7iGA"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2401796","apath":"\/biorxiv\/early\/2022\/02\/14\/2020.07.30.223149.atom","pisa":"biorxiv;2020.07.30.223149v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.07.30.223149v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.07.30.223149v2","title":"Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations","identifier":"node\/2401796"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pbMm7FFqtAlNhOJON8baA5hEZXOuTLVmDV\\\\\\\/Hgh6AKlM\\\\\\\/\\\\\\\/A\\\\\\\/xIR4xiZxuXtuv5YRGaPu2hfPxhQMNSzn8BQXgT6JeOPeL8jK6M35Re11mz9KyUZXqmwSh6d1cybaUYlj42VtFNZI4QQF2+p2\\\\\\\/IDUQRNKaLoDaAo1h5xAbLGy0EJwRs\\\\\\\/RzaDwxJpOQXWplHojxow+kktlgT59MncNoWpoK\\\\\\\/ZO2FiADERREfkkbS35v5TQvqjMtkyRV0AVnuDwO1acjCEwHul8GIa6Tc73jyI73Dxwakl\\\\\\\/QJXNC3QjDbXnlHrA4DI8Zz4A9KMeuxNAjBrmvRRwy86hI3ce7K8uKcrRVJU7wIwow0ookCEMGcwueiTJ47b4BO3C46Nnn9Ek7vd\\\\\\\/vu4SUIsnQRodhRMcwWW7ey5GP\\\\\\\/8tQ1YhoAdtxAQDNe84dEOx7egSRCxv9WvCmUnQ8fNikr5+xcwSULqi86qsXmkzJWPyR+y+Fv0WYWLCrLRiZE+nVrvhNzWbAeWIrg1e9VDeYoBShpVHsCdwO6pMOIDvzDyAV8yM\\\\\\\/bPYQ7iqQJkAKsuuAyWfcwKMhHlUUtjpZA7\\\\\\\/1qLFR+HwdkFhmFNg+0g2futQ74QBxUKG7acC3IEvnYU2GjG7B+4q33U9i3PJ63MVZJuj\\\\\\\/vsiY3WFGK+EzFA01BErOlGUtazGGIg2mtscnJf3DcCweII0GPecwT+g69CTYPufjT5yruQK3TXBGJTF7yCmNV\\\\\\\/ojzK05NrCz7XhPQXOubaVt0YGc795Zr\\\\\\\/eiwCzgE1MYP496CBYcYdvHinUiN1OeuquHVvFkqpLIdARc6R9BmgyDkJzUZ1yyH0NiNVBOTAIV8kVp90d28Yy7Zo7P0rD3J7He38qZ+tUKdn1yWR25QbUuHGvIR0CB6tqGsryZcTdDMaeW32LtDX5Htdtwr4ToUoLDC1MrisVvdhbZt6TaM4lF\\\\\\\/EfxCVkS5hFMaYqyMyMgxZtuH06pJiMj8MgHY45+HSak\\\\\\\/6ghojZx04frNIK57aF5rh4M7JzTyxREamnhfnttbXH+1h0OhRs+byQVnwX61IQyHK96+St7Nksp3BqncqrtPYMRKZ29cnyI2jiYUMoV89Ba0I+jca7o8tiTk3dBp3s5lNbUkmpAmGC\\\\\\\/fJrmmwUk47ID4+72XR3rKjV7P6tRNflFd7gYPD0gf260rXuPErVFMM+bFOG9AlNfzBot9\\\\\\\/3dzfIE4\\\\\\\/ezDixrbJGThUqVQDVlNLht1aXfxwLrQMyvZv3AxXEBp96S43IL0sN99w2FusawutuPVfeZFZOjE73jZQVttV6iXaCB+7t6EZBuNpRJSD4FWcUkhSuXdA3jZhEIm\\\\\\\/kIbpkG+GbDd0sUFbeMcoVjalNlapoXyz3rnPN3U1h0GuUd4qsw2iY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NVofHCcV0DNgQxArP+cpvg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00221b4250c39d38c561596aae574ae61af59301f9e21dc5cc2ca6e11b5900969ffe\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.07.30.223149v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2401796 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.07.30.223149v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_856358071">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_856358071" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1254471230" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-yYe7bmliyASysFC4uDwYCEGEhv5bMnOmbpwuXd8pNXo"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2401796" data-node-nid="2401796" id="node-2401796--22065467713" data-pisa="biorxiv;2020.07.30.223149v2" data-pisa-master="biorxiv;2020.07.30.223149" data-apath="/biorxiv/early/2022/02/14/2020.07.30.223149.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Assaf
                      </span>
                      <span class="nlm-surname">
                       Nahum
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Yoni
                      </span>
                      <span class="nlm-surname">
                       Koren
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Bar
                      </span>
                      <span class="nlm-surname">
                       Ergaz
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0001-9419-3808" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Sari
                      </span>
                      <span class="nlm-surname">
                       Natan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="4">
                      <span class="nlm-given-names">
                       Shahar
                      </span>
                      <span class="nlm-surname">
                       Goren
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="5">
                      <span class="nlm-given-names">
                       Avraham
                      </span>
                      <span class="nlm-surname">
                       Kolel
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0002-3778-8757" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Sankar
                      </span>
                      <span class="nlm-surname">
                       Jagadeeshan
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-3634-9098" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Moshe
                      </span>
                      <span class="nlm-surname">
                       Elkabets
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8">
                      <a href="http://orcid.org/0000-0003-2511-4381" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Ayelet
                      </span>
                      <span class="nlm-surname">
                       Lesman
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9">
                      <a href="http://orcid.org/0000-0002-1477-5478" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Assaf
                      </span>
                      <span class="nlm-surname">
                       Zaritsky
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.07.30.223149
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2401796--22065467713" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Assaf Nahum
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Software and Information Systems Engineering, Ben-Gurion University of the Negev
                       </span>
                       , Beer-Sheva 84105,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Assaf%2BNahum%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Nahum%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAssaf%2BNahum%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Yoni Koren
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Yoni%2BKoren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Koren%20Y&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AYoni%2BKoren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Bar Ergaz
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Bar%2BErgaz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ergaz%20B&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3ABar%2BErgaz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Sari Natan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sari%2BNatan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Natan%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASari%2BNatan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0001-9419-3808" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Sari Natan
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Shahar Goren
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Biomedical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Shahar%2BGoren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Goren%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AShahar%2BGoren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Avraham Kolel
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Avraham%2BKolel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Kolel%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAvraham%2BKolel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Sankar Jagadeeshan
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        The Shraga Segal Dept. of Microbiology, Immunology and Genetics, Faculty of Health Sciences, Ben-Gurion University of the Negev
                       </span>
                       , Beer-Sheva 84105,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Sankar%2BJagadeeshan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jagadeeshan%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASankar%2BJagadeeshan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-3778-8757" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Sankar Jagadeeshan
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Moshe Elkabets
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        The Shraga Segal Dept. of Microbiology, Immunology and Genetics, Faculty of Health Sciences, Ben-Gurion University of the Negev
                       </span>
                       , Beer-Sheva 84105,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Moshe%2BElkabets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Elkabets%20M&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMoshe%2BElkabets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-3634-9098" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Moshe Elkabets
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Ayelet Lesman
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        School of Mechanical Engineering, Faculty of Engineering, Tel-Aviv University
                       </span>
                       ,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Ayelet%2BLesman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lesman%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAyelet%2BLesman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0003-2511-4381" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Ayelet Lesman
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:assafza@bgu.ac.il" class="" data-icon-position="" data-hide-link-title="0">
                        assafza@bgu.ac.il
                       </a>
                       <a href="mailto:ayeletlesman@tauex.tau.ac.il" class="" data-icon-position="" data-hide-link-title="0">
                        ayeletlesman@tauex.tau.ac.il
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-9">
                    <div class="author-tooltip-name">
                     Assaf Zaritsky
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Department of Software and Information Systems Engineering, Ben-Gurion University of the Negev
                       </span>
                       , Beer-Sheva 84105,
                       <span class="nlm-country">
                        Israel
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Assaf%2BZaritsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zaritsky%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AAssaf%2BZaritsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-1477-5478" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Assaf Zaritsky
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:assafza@bgu.ac.il" class="" data-icon-position="" data-hide-link-title="0">
                        assafza@bgu.ac.il
                       </a>
                       <a href="mailto:ayeletlesman@tauex.tau.ac.il" class="" data-icon-position="" data-hide-link-title="0">
                        ayeletlesman@tauex.tau.ac.il
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.07.30.223149v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.223149v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.223149v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.223149v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.223149v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.07.30.223149v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.07.30.223149v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2401796" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2401796/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Cells sense, manipulate and respond to their mechanical microenvironment in a plethora of physiological processes, yet the understanding of how cells transmit, receive and interpret environmental cues to communicate with distant cells is severely limited due to lack of tools to quantitatively infer the complex tangle of dynamic cell-cell interactions in complicated environments. We present a computational method to systematically infer and quantify long-range mechanical cell-cell communication through the extracellular matrix (cell-ECM-cell communication) by correlating ECM remodeling fluctuations in between communicating cells and demonstrating that these fluctuations contain sufficient information to define unique signatures that robustly distinguish between different pairs of communicating cells. We demonstrate our method with finite element simulations and live 3D imaging of fibroblasts and cancer cells embedded in fibrin gels. While previous studies relayed on the formation of a visible fibrous ‘band’ extending between cells to inform on mechanical communication, our method detected mechanical propagation even in cases where visible bands never formed. We revealed that while contractility is required, band formation is not necessary, for cell-ECM-cell communication, and that mechanical signals propagate from one cell to another even upon massive reduction in their contractility. Our method sets the stage to measure the fundamental aspects of intercellular long-range mechanical communication in physiological contexts and may provide a new functional readout for high content 3D image-based screening. The ability to infer cell-ECM-cell communication using standard confocal microscopy holds the promise for wide use and democratizing the method.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             Many types of cells apply considerable traction forces on their surrounding matrix leading to ECM deformations that can propagate to large distances of tens of cell diameters away
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              –
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             . These deformations can form a visible fibrous band of aligned and dense fibers coupling the communicating cells mechanically, influencing the cells’ internal molecular state
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             and active response
                             <sup>
                              <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                             </sup>
                             . This long-range mechanical intercellular communication through the ECM (termed here cell-ECM-cell communication) was observed for cells embedded in synthetic
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ,
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                             </sup>
                             and biopolymer fibrous matrices
                             <sup>
                              <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                              ,
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                              –
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                              ,
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              –
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             and are thought to play a key role in tissue regeneration and disease progression
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ,
                              <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">
                               16
                              </a>
                              –
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                             </sup>
                             . In-vivo, fiber alignment bands can serve as ECM ‘tracks’ for cell migration with potential roles in wound healing, cancer metastasis and fibrosis
                             <sup>
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ,
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-7">
                             Quantification of mechanical cell-ECM-cell communication has been typically achieved by measuring the density, alignment or displacement of the remodeled ECM between the cell pairs
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                              ,
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ,
                              <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              –
                              <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                              ,
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              –
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                             </sup>
                             . While current measurements characterize the formation of the visible fibrous band extending between communicating cells, they lack the sensitivity to measure the dynamic reciprocal mechanical information transfer between the cells. Thus, current methods are limited in measuring potential cell-ECM-cell communication in the absence of visible bands, hampering our ability to distinguish which cells are actually communicating from the many cells that have the potential to communicate. Bridging this gap will enable tackling long-standing open questions in how tissues develop and diseases progress by enabling us to identify which cells are communicating with each other, and to what extent, in complex environments.
                            </p>
                            <p id="p-8">
                             Here, we present a new computational method to quantify the transmitted ECM signal in between communicating cells by correlating temporal fluctuations of the remodeled matrix. Computational simulations and 3D live imaging of fibroblasts and cancer cells embedded in fibrin gels demonstrate the power of our method in identifying unique ECM remodeling signatures that allow to robustly distinguish between different pairs of communicating cells. Using this method, we were able to measure communication between cell pairs that do not form a visible ‘band’ of densified ECM, and after partial depletion of contractility upon Myosin-II inhibition. These results imply that mechanical signals propagate from one cell to another even in low cell contractility levels.
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Results
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Analysis of ECM density between pairs of communicating cells using finite element simulations
                             </h3>
                             <p id="p-9">
                              During cell-cell mechanical communication, every localized fibrous region in between the communicating cells is affected by two components: the contractile force of the adjacent cell (referred as ‘cell-ECM’ interaction), and the force transmitted from the second distant cell (referred as ‘cell-ECM-cell’ communication) (
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              ). To quantitatively characterize the independent contribution of each of these two components, we simulated contracting cells embedded within 2D fibrous networks using finite element discrete modeling, quantified the change in the ECM density between pairs of communicating cells and compared it to single cells interacting with the ECM. While these simulations do not reflect the true complexity of biological systems, they capture the essence of the mechanical elements of cell contractility and force propagation in fibrous nonlinear elastic networks (Methods). Thus, these simulations serve as a minimal mechanical model that enables us to independently tune mechanical parameters (e.g., cell-cell distance and cell contractility), and quantitatively infer their effect on ECM remodeling. These simulations will be used to ask whether ECM remodeling can robustly encode unique ECM signature in communicating cell pairs, forming the basis for our method.
                             </p>
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F1.large.jpg?width=800&height=600&carousel=1" title="Quantifying ECM densification in simulated cell-ECM-cell mechanical communication. (A) Fibers in between pairs of cells (red and purple circles) are remodeled by the integrated mechanical forces that both cells exert on the ECM. Colored arrows depict the magnitude of the force experienced in a specific location in the fibrous gel that are generated by the two cells. (B) Quantitative visualization of representative simulated cell-ECM interactions (left) and cell-ECM-cell communication (right) at the onset (“begin”) and after (“end”) 50% cell contraction. Color code encodes the ECM density in z-score, the number of standard deviation above background levels. Scale bar is one cell diameter. (C) Quantifying fiber densification as a function of the distance around simulated single cells (“single cells”) and as a function of the distance between cell pairs (“pair distance”). Single cells (N = 7). Pair distance of 4 (N = 19), 5 (N = 20 pairs), 7 (N = 20 pairs), or 9 (N = 19 pairs) cell diameters. Note that window distance is bounded in lower pair distances. Mean and standard deviation are displayed for each distance. (D) A dense fibrous “band” between cell pairs is formed by a combination of cell-ECM interactions and cell-ECM-cell communication." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantifying ECM densification in simulated cell-ECM-cell mechanical communication. (<strong>A</strong>) Fibers in between pairs of cells (red and purple circles) are remodeled by the integrated mechanical forces that both cells exert on the ECM. Colored arrows depict the magnitude of the force experienced in a specific location in the fibrous gel that are generated by the two cells. (<strong>B</strong>) Quantitative visualization of representative simulated cell-ECM interactions (left) and cell-ECM-cell communication (right) at the onset (“begin”) and after (“end”) 50% cell contraction. Color code encodes the ECM density in z-score, the number of standard deviation above background levels. Scale bar is one cell diameter. (<strong>C</strong>) Quantifying fiber densification as a function of the distance around simulated single cells (“single cells”) and as a function of the distance between cell pairs (“pair distance”). Single cells (N = 7). Pair distance of 4 (N = 19), 5 (N = 20 pairs), 7 (N = 20 pairs), or 9 (N = 19 pairs) cell diameters. Note that window distance is bounded in lower pair distances. Mean and standard deviation are displayed for each distance. (<strong>D</strong>) A dense fibrous “band” between cell pairs is formed by a combination of cell-ECM interactions and cell-ECM-cell communication.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F1.medium.gif" width="401" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F1.medium.gif" width="401" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1.
                               </span>
                               <p id="p-10" class="first-child">
                                Quantifying ECM densification in simulated cell-ECM-cell mechanical communication. (
                                <strong>
                                 A
                                </strong>
                                ) Fibers in between pairs of cells (red and purple circles) are remodeled by the integrated mechanical forces that both cells exert on the ECM. Colored arrows depict the magnitude of the force experienced in a specific location in the fibrous gel that are generated by the two cells. (
                                <strong>
                                 B
                                </strong>
                                ) Quantitative visualization of representative simulated cell-ECM interactions (left) and cell-ECM-cell communication (right) at the onset (“begin”) and after (“end”) 50% cell contraction. Color code encodes the ECM density in z-score, the number of standard deviation above background levels. Scale bar is one cell diameter. (
                                <strong>
                                 C
                                </strong>
                                ) Quantifying fiber densification as a function of the distance around simulated single cells (“single cells”) and as a function of the distance between cell pairs (“pair distance”). Single cells (N = 7). Pair distance of 4 (N = 19), 5 (N = 20 pairs), 7 (N = 20 pairs), or 9 (N = 19 pairs) cell diameters. Note that window distance is bounded in lower pair distances. Mean and standard deviation are displayed for each distance. (
                                <strong>
                                 D
                                </strong>
                                ) A dense fibrous “band” between cell pairs is formed by a combination of cell-ECM interactions and cell-ECM-cell communication.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-11">
                              To enable quantitative comparison between simulations, we normalized the fiber density to its z-score - the number of standard deviations away from the mean background fiber density at regions that were not influenced by the cells (Methods). In simulations of single contractile cells, we found that the regions next to the cell’s boundary became denser following the application of 50% cell contraction (
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Fig. 1B
                              </a>
                              ). Upon the presence of a second cell, the overall densification was governed by the integrated contractile activity of both cells (
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Fig. 1A
                              </a>
                              ). This dual contribution led to the formation of a band of increased density along the connecting axis between the cells (
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Fig. 1B-C
                              </a>
                              ). While for single cells the fiber densification faded to the background level at a characteristic distance of approximately 2 cell diameters, the bands between pairs of cells were characterized with increased fiber density that extended further away from the cells and were increased for cell pairs that were located closer to one another (smaller pair distance,
                              <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                               Fig. 1C
                              </a>
                              ). The increased fiber density between cell pairs in comparison to single cells could potentially be used to quantitatively decouple the contribution of the cell-autonomous and cell-cell communication components that together form the visible band between the cells (
                              <a id="xref-fig-1-6" class="xref-fig" href="#F1">
                               Fig. 1D
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Distinguishing between communicating versus non-communicating simulated cell pairs using temporal correlation of local ECM remodeling fluctuations
                             </h3>
                             <p id="p-12">
                              Given that local ECM regions located along a band experience forces exerted by both cells and the discrepancy between ECM remodeling by a single and a pair of cells (
                              <a id="xref-fig-1-7" class="xref-fig" href="#F1">
                               Fig. 1
                              </a>
                              ), we hypothesized that local ECM remodeling fluctuations contain sufficient information to distinguish communicating cells from cells that are not communicating with one another. To test this hypothesis, we simulated the temporal process that led to the final remodeling. We consecutively applied 1% cell contraction for 50 steps, reaching 50% cell contraction. For each step, we recorded the fiber density between communicating cells. These iterative simulations captured the temporal dynamics of force propagation between the cells during cell-ECM-cell communication. As expected, the fiber density close to the cells’ edge gradually increased over time (
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Fig. 2A-B
                              </a>
                              , Video S1).
                             </p>
                             <div id="F2" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F2.large.jpg?width=800&height=600&carousel=1" title="Using temporal correlations of ECM remodeling fluctuations to distinguish between communicating and non-communicating simulated cell pairs. (A-B). Quantitative visualization and quantification of the dynamics of simulated cell-ECM-cell communication. (A) Representative simulation visualization at the onset (top), 25% cell contraction (middle), and 50% cell contraction (bottom). (B) Quantifying fiber densification as a function of simulated contraction steps in between cell pairs. N = 20 cell pairs. Color code (left) and y-axis (right) encodes the ECM density in z-score, the number of standard deviation above background levels. The pair distance between simulated cell centers was set to 7 cell diameters. (C) Schematic sketch of comparing ECM remodeling fluctuations of communicating versus non-communicating cell pairs. The correlation between quantification windows of communicating pairs (upper in green) is evaluated in relation to the correlation between two cells from two different communicating pairs (lower in orange and purple). (D) Distinguishing between communicating and non-communicating simulated cell pairs correlations of the fluctuations of the second derivative of fiber density with different levels of contraction heterogeneity. Each data point records the correlation between a simulated communicating cell pair, each box plots record the corresponding distribution of correlations between (the many) pairs of non-communicating cells. Mean contraction of 1% and standard deviation (heterogeneity) of 0%, 0.25%, 0.5%, and 0.75%. Three statistical tests are performed according to the following order. (1) Wilcoxon sum-rank testing the null hypothesis that the two correlation distributions of communicating and non-communicating cell pairs are originating from the same distribution. Wilcoxon sign-rank testing the null hypothesis that the correlation distributions of (2) communicating or (3) non-communicating are distributed around a mean 0. Std. = 0%: N = 20, p-values: (1) not significant, (2) not significant, (3) not significant. Std. = 0.25%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.5%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.75%: N = 20, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. (E) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the pair distance. Pair distance: 4 (N = 20 pairs, p-value < 0.0001), 5 (N = 19 pairs, p-value < 0.0001), 7 (N = 19 pairs, p-value < 0.01) and 9 (N = 19 pairs, p-value < 0.01)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Using temporal correlations of ECM remodeling fluctuations to distinguish between communicating and non-communicating simulated cell pairs. (<strong>A-B</strong>). Quantitative visualization and quantification of the dynamics of simulated cell-ECM-cell communication. (<strong>A</strong>) Representative simulation visualization at the onset (top), 25% cell contraction (middle), and 50% cell contraction (bottom). (<strong>B</strong>) Quantifying fiber densification as a function of simulated contraction steps in between cell pairs. N = 20 cell pairs. Color code (left) and y-axis (right) encodes the ECM density in z-score, the number of standard deviation above background levels. The pair distance between simulated cell centers was set to 7 cell diameters. (<strong>C</strong>) Schematic sketch of comparing ECM remodeling fluctuations of communicating versus non-communicating cell pairs. The correlation between quantification windows of communicating pairs (upper in green) is evaluated in relation to the correlation between two cells from two different communicating pairs (lower in orange and purple). (<strong>D</strong>) Distinguishing between communicating and non-communicating simulated cell pairs correlations of the fluctuations of the second derivative of fiber density with different levels of contraction heterogeneity. Each data point records the correlation between a simulated communicating cell pair, each box plots record the corresponding distribution of correlations between (the many) pairs of non-communicating cells. Mean contraction of 1% and standard deviation (heterogeneity) of 0%, 0.25%, 0.5%, and 0.75%. Three statistical tests are performed according to the following order. (1) Wilcoxon sum-rank testing the null hypothesis that the two correlation distributions of communicating and non-communicating cell pairs are originating from the same distribution. Wilcoxon sign-rank testing the null hypothesis that the correlation distributions of (2) communicating or (3) non-communicating are distributed around a mean 0. Std. = 0%: N = 20, p-values: (1) not significant, (2) not significant, (3) not significant. Std. = 0.25%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.5%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.75%: N = 20, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. (<strong>E</strong>) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the pair distance. Pair distance: 4 (N = 20 pairs, p-value < 0.0001), 5 (N = 19 pairs, p-value < 0.0001), 7 (N = 19 pairs, p-value < 0.01) and 9 (N = 19 pairs, p-value < 0.01).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F2.medium.gif" width="440" height="404"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F2.medium.gif" width="440" height="404"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 2.
                               </span>
                               <p id="p-13" class="first-child">
                                Using temporal correlations of ECM remodeling fluctuations to distinguish between communicating and non-communicating simulated cell pairs. (
                                <strong>
                                 A-B
                                </strong>
                                ). Quantitative visualization and quantification of the dynamics of simulated cell-ECM-cell communication. (
                                <strong>
                                 A
                                </strong>
                                ) Representative simulation visualization at the onset (top), 25% cell contraction (middle), and 50% cell contraction (bottom). (
                                <strong>
                                 B
                                </strong>
                                ) Quantifying fiber densification as a function of simulated contraction steps in between cell pairs. N = 20 cell pairs. Color code (left) and y-axis (right) encodes the ECM density in z-score, the number of standard deviation above background levels. The pair distance between simulated cell centers was set to 7 cell diameters. (
                                <strong>
                                 C
                                </strong>
                                ) Schematic sketch of comparing ECM remodeling fluctuations of communicating versus non-communicating cell pairs. The correlation between quantification windows of communicating pairs (upper in green) is evaluated in relation to the correlation between two cells from two different communicating pairs (lower in orange and purple). (
                                <strong>
                                 D
                                </strong>
                                ) Distinguishing between communicating and non-communicating simulated cell pairs correlations of the fluctuations of the second derivative of fiber density with different levels of contraction heterogeneity. Each data point records the correlation between a simulated communicating cell pair, each box plots record the corresponding distribution of correlations between (the many) pairs of non-communicating cells. Mean contraction of 1% and standard deviation (heterogeneity) of 0%, 0.25%, 0.5%, and 0.75%. Three statistical tests are performed according to the following order. (1) Wilcoxon sum-rank testing the null hypothesis that the two correlation distributions of communicating and non-communicating cell pairs are originating from the same distribution. Wilcoxon sign-rank testing the null hypothesis that the correlation distributions of (2) communicating or (3) non-communicating are distributed around a mean 0. Std. = 0%: N = 20, p-values: (1) not significant, (2) not significant, (3) not significant. Std. = 0.25%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.5%: N = 19, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. Std. = 0.75%: N = 20, p-values: (1) < 0.0001, (2) < 0.001, (3) not significant. (
                                <strong>
                                 E
                                </strong>
                                ) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the pair distance. Pair distance: 4 (N = 20 pairs, p-value < 0.0001), 5 (N = 19 pairs, p-value < 0.0001), 7 (N = 19 pairs, p-value < 0.01) and 9 (N = 19 pairs, p-value < 0.01).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-14">
                              We expected that the temporal correlation of ECM remodeling fluctuations between communicating cell pairs will exceed that of non-communicating cell pairs (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2C
                              </a>
                              ). In simulations of pairs of both communicating and non-communicating cells, we found positive correlations throughout the simulation in the fiber density dynamics and in its temporal derivative, i.e., change in fiber density over time (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. S1A
                              </a>
                              ). The correlation in fiber density is attributed to the monotonic increase in ECM densification between communicating cells (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2B
                              </a>
                              ). The correlation in the temporal derivative of fiber density is attributed to an association between fiber density and the change in fiber density (
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Fig. S1B
                              </a>
                              ): the ECM became denser over time, leading to increased temporal derivatives. However, these general trends lead to high correlations between any two cells regardless of whether they communicate with each other, and confound the unique fluctuations that may be attributed to a specific pair of communicating cells. Thus, we aimed at overcoming this confounder by dampening correlations between non-communicating cells toward zero. This was achieved by detrending, i.e., removing the temporal trends via a second temporal derivative. Although the correlations between non-communicating cell pairs were lost with a second temporal derivative, the correlations between communicating pairs were also lost and we were unable to distinguish between communicating and non-communicating cell pairs (
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Fig. S1A
                              </a>
                              ).
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F8.large.jpg?width=800&height=600&carousel=1" title="Temporal correlations of ECM remodeling fluctuations in communicating and non-communicating simulated cell pairs. In these simulations each cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. (A) Correlations for fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) for communicating and non-communicating cell pairs. N = 20 cell pairs at a pair distance of 5 cell diameters. All p-values not significant. (B) Two-dimensional distribution of fiber density and change in fiber density measured in consecutive contraction steps. Quantification windows adjacent to each cell along the band. N = 120 cell pairs (N = 20 for distances of 5, 7, 9, 12, 15 and 17 cell diameters). Constant cell contraction of 1% between consecutive time frames. Pearson correlation coefficient = 0.59, p-value < 0.0001. (C) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the window distance. Pair distance = 5 (N = 19 pairs). All p-values < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Temporal correlations of ECM remodeling fluctuations in communicating and non-communicating simulated cell pairs. In these simulations each cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. (<strong>A</strong>) Correlations for fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) for communicating and non-communicating cell pairs. N = 20 cell pairs at a pair distance of 5 cell diameters. All p-values not significant. (<strong>B</strong>) Two-dimensional distribution of fiber density and change in fiber density measured in consecutive contraction steps. Quantification windows adjacent to each cell along the band. N = 120 cell pairs (N = 20 for distances of 5, 7, 9, 12, 15 and 17 cell diameters). Constant cell contraction of 1% between consecutive time frames. Pearson correlation coefficient = 0.59, p-value < 0.0001. (<strong>C</strong>) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the window distance. Pair distance = 5 (N = 19 pairs). All p-values < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F8.medium.gif" width="440" height="405"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F8.medium.gif" width="440" height="405"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S1:
                               </span>
                               <p id="p-103" class="first-child">
                                Temporal correlations of ECM remodeling fluctuations in communicating and non-communicating simulated cell pairs. In these simulations each cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. (
                                <strong>
                                 A
                                </strong>
                                ) Correlations for fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) for communicating and non-communicating cell pairs. N = 20 cell pairs at a pair distance of 5 cell diameters. All p-values not significant. (
                                <strong>
                                 B
                                </strong>
                                ) Two-dimensional distribution of fiber density and change in fiber density measured in consecutive contraction steps. Quantification windows adjacent to each cell along the band. N = 120 cell pairs (N = 20 for distances of 5, 7, 9, 12, 15 and 17 cell diameters). Constant cell contraction of 1% between consecutive time frames. Pearson correlation coefficient = 0.59, p-value < 0.0001. (
                                <strong>
                                 C
                                </strong>
                                ) Correlation of communicating versus non-communicating cell pairs using the second derivative of fiber density dynamics as a function of the window distance. Pair distance = 5 (N = 19 pairs). All p-values < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-15">
                              We suspected that the inability to distinguish between communicating and non-communicating simulated cell pairs was due to lack of variability in ECM remodeling over time. Every simulated cell contracted in the exact same magnitude as any other simulated cell leading to nearly exact time-dependent remodeling of the adjacent ECM regime and thus masking the unique ECM fluctuation patterns that may exist between communicating partners. Thus, our hypothesis was that variability in the cells’ contraction activity (an inherent property of cells) will lead to variability in the local ECM remodeling around each cell, leading to a unique communication signature between partners. In other words, we thought that heterogeneity in ECM remodeling would enable us to decouple the general ECM remodeling that occurs between any random pair of cells from the specific signal that is transmitted between communicating partners. Indeed, when introducing heterogeneity in simulated cell contraction, we distinguished communicating from non-communicating simulated cell pairs while maintaining the non-communicating correlations around zero (
                              <a id="xref-fig-2-4" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                              ). Cell heterogeneity was included in the simulations by drawing the instantaneous contraction rate of each cell independently from a normal distribution with mean (μ) contraction of 1% and a standard deviation (σ) in the range of 0-0.75% (std.) (Methods). Even a minimal standard deviation of 0.25% in cell contraction was sufficient to make a clear distinction between communicating and non-communicating cell pairs, which improved with increasing standard deviations (
                              <a id="xref-fig-2-5" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                              ). This distinction was negatively correlated to the pair distance - as cells were placed further apart, it became more difficult to distinguish between communicating and non-communicating cell pairs (
                              <a id="xref-fig-2-6" class="xref-fig" href="#F2">
                               Fig. 2E
                              </a>
                              ). Moreover, this distinction was improved when moving the ECM quantification window along the band, toward the communication partner (increasing the window distance), which increased the communication partner’s influence (
                              <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                               Fig. S1D
                              </a>
                              ). Altogether, these results established the notion that temporal correlation of local ECM remodeling fluctuations can distinguish between communicating and non-communicating simulated cell pairs, and that contractile heterogeneity is required to make this distinction.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Analysis of 3D ECM density between pairs of communicating fibroblast cells
                             </h3>
                             <p id="p-16">
                              To test the simulated results of correlation in ECM remodeling as a marker of mechanical cell-ECM-cell communication, we used 3D time-lapse confocal microscopy to image live NIH/3T3 fibroblast cells (GFP-Actin) embedded in 3D fluorescently labeled fibrin gels, where fibrin intensity was used as a proxy of fiber density (Methods). To visualize and measure fiber intensity in between cell pairs in 3D, we transformed the microscopy axes to a new 3D coordinate system that is aligned around the connecting axis between the cells’ centers (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3A
                              </a>
                              ,
                              <a id="xref-fig-9-1" class="xref-fig" href="#F9">
                               Fig. S2
                              </a>
                              , Video S2, Methods) and performed z-score fiber intensity normalization in respect to the local background (Methods). While single cells did not show visually apparent fiber densification beyond regions close to the cell, approximately 60% of all cell pairs formed a visible band of increased density extending along the connecting axis between the cells (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3B
                              </a>
                              ) that showed increased mechanical coupling for cells pairs that were closer to one another (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3C
                              </a>
                              ,
                              <a id="xref-fig-10-1" class="xref-fig" href="#F10">
                               Fig. S3
                              </a>
                              ). The fiber density close to the cells’ edge gradually increased over time in pairs of communicating cells but remained constant in single cells (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3D
                              </a>
                              , Video S3 versus Video S4). The higher fiber intensity at the onset of imaging was attributed to the time (approximately 30 minutes) that passed from setting up the experiment until the onset of imaging (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3D
                              </a>
                              , z-score of approximately 3 standard deviations above the background intensity). During this time, cells have already deformed the fibers around them. These results conclude that the formation of dense fibrous bands between fibroblasts cell pairs embedded in 3D fibrin gels are indicative of a mechanical coupling in concurrence with previous studies
                              <sup>
                               <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                                5
                               </a>
                               ,
                               <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                               –
                               <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                                11
                               </a>
                              </sup>
                              .
                             </p>
                             <div id="F9" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F9.large.jpg?width=800&height=600&carousel=1" title="ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the side view (top) and top view (bottom). Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup">ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the side view (top) and top view (bottom). Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black).</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F9.medium.gif" width="287" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F9.medium.gif" width="287" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S2:
                               </span>
                               <p id="p-104" class="first-child">
                                ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the side view (top) and top view (bottom). Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F10" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F10.large.jpg?width=800&height=600&carousel=1" title="The distance between pairs of communicating cells is negatively correlated to the density of fibrous regions further away from the cell. I.e., as the quantification region moves away from one cell, the effect of the other cell drops as a function of the pair distance, implying a stronger mechanical interaction with the other cell for closer pairs. (A) Schematic sketch. Pair distance (purple) between cell pairs at the onset of imaging. Window distance (cyan) is measured from the boundaries of one cell toward the other cell in the pair, along the axis (in 3D) defined by the pair. (B) Correlation between the window distance and the correlation between pair distance and fiber density across cell pairs. N = 15 window distances. Pearson correlation coefficient = -0.76, p-value < 0.001. Red asterisks reflect a significance level below 0.05. Insets: two significant correlations between fiber density pair distance for window distance of 1 (top) and 2.6 (right) cell diameters. N = 38 (top) and 10 (right) communicating cell pairs. Pearson correlation coefficient = -0.37 (top) and -0.71 (right), p-value = 0.02 for both. The reduced number of cell pairs (right) stems from overlapping of the quantification regions due to short pair distances. Note that the color coding (purple, cyan, green) is matching the sketch in panel A. (C) Summary table for the data in panel B. Bold rows with red asterisk reflect correlations with significance levels below 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">The distance between pairs of communicating cells is negatively correlated to the density of fibrous regions further away from the cell. I.e., as the quantification region moves away from one cell, the effect of the other cell drops as a function of the pair distance, implying a stronger mechanical interaction with the other cell for closer pairs. (<strong>A</strong>) Schematic sketch. Pair distance (purple) between cell pairs at the onset of imaging. Window distance (cyan) is measured from the boundaries of one cell toward the other cell in the pair, along the axis (in 3D) defined by the pair. (<strong>B</strong>) Correlation between the window distance and the correlation between pair distance and fiber density across cell pairs. N = 15 window distances. Pearson correlation coefficient = -0.76, p-value < 0.001. Red asterisks reflect a significance level below 0.05. Insets: two significant correlations between fiber density pair distance for window distance of 1 (top) and 2.6 (right) cell diameters. N = 38 (top) and 10 (right) communicating cell pairs. Pearson correlation coefficient = -0.37 (top) and -0.71 (right), p-value = 0.02 for both. The reduced number of cell pairs (right) stems from overlapping of the quantification regions due to short pair distances. Note that the color coding (purple, cyan, green) is matching the sketch in panel A. (<strong>C</strong>) Summary table for the data in panel B. Bold rows with red asterisk reflect correlations with significance levels below 0.05.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F10.medium.gif" width="440" height="390"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F10.medium.gif" width="440" height="390"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S3:
                               </span>
                               <p id="p-105" class="first-child">
                                The distance between pairs of communicating cells is negatively correlated to the density of fibrous regions further away from the cell. I.e., as the quantification region moves away from one cell, the effect of the other cell drops as a function of the pair distance, implying a stronger mechanical interaction with the other cell for closer pairs. (
                                <strong>
                                 A
                                </strong>
                                ) Schematic sketch. Pair distance (purple) between cell pairs at the onset of imaging. Window distance (cyan) is measured from the boundaries of one cell toward the other cell in the pair, along the axis (in 3D) defined by the pair. (
                                <strong>
                                 B
                                </strong>
                                ) Correlation between the window distance and the correlation between pair distance and fiber density across cell pairs. N = 15 window distances. Pearson correlation coefficient = -0.76, p-value < 0.001. Red asterisks reflect a significance level below 0.05. Insets: two significant correlations between fiber density pair distance for window distance of 1 (top) and 2.6 (right) cell diameters. N = 38 (top) and 10 (right) communicating cell pairs. Pearson correlation coefficient = -0.37 (top) and -0.71 (right), p-value = 0.02 for both. The reduced number of cell pairs (right) stems from overlapping of the quantification regions due to short pair distances. Note that the color coding (purple, cyan, green) is matching the sketch in panel A. (
                                <strong>
                                 C
                                </strong>
                                ) Summary table for the data in panel B. Bold rows with red asterisk reflect correlations with significance levels below 0.05.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F3.large.jpg?width=800&height=600&carousel=1" title="Quantifying ECM densification during cell-ECM-cell mechanical communication with 3D live imaging of communicating fibroblast cells. (A) ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the top-side view. Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black). Side view and top view are illustrated in Fig. S2. (B) Representative images of a single cell (left) and a cell pair (right) at the onset (top), after (slightly over) two hours (middle), and (slightly over) four hours of live cell imaging (bottom). The initial distance between the cell centers in the pair was ∼117 μm (∼7.8 cell diameters assuming mean fibroblast diameter of 15 μm). White dashed line represents the connecting axis between the cells. White dashed rectangle next to the cell boundaries represents one quantification window (see Methods for details). Scale bar = 15 μm. (C) Quantifying cell-ECM-cell communication as a function of distance between cell pairs. Single cells (N = 7), Pair distance of 4-6 (N = 9 pairs), 6-8 (N = 13 pairs) and 8-10 (N = 5 pairs) cell diameters (assuming cell diameter of 15 μm). (D) Quantifying fiber densification dynamics near a single cell and “band” formation in between pairs of cells. N = 7 single cell, N = 13 cell pairs at pair distances of 6-8 cell diameters (90-120 μm)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantifying ECM densification during cell-ECM-cell mechanical communication with 3D live imaging of communicating fibroblast cells. (<strong>A</strong>) ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the top-side view. Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black). Side view and top view are illustrated in Fig. S2. (<strong>B</strong>) Representative images of a single cell (left) and a cell pair (right) at the onset (top), after (slightly over) two hours (middle), and (slightly over) four hours of live cell imaging (bottom). The initial distance between the cell centers in the pair was ∼117 μm (∼7.8 cell diameters assuming mean fibroblast diameter of 15 μm). White dashed line represents the connecting axis between the cells. White dashed rectangle next to the cell boundaries represents one quantification window (see Methods for details). Scale bar = 15 μm. (<strong>C</strong>) Quantifying cell-ECM-cell communication as a function of distance between cell pairs. Single cells (N = 7), Pair distance of 4-6 (N = 9 pairs), 6-8 (N = 13 pairs) and 8-10 (N = 5 pairs) cell diameters (assuming cell diameter of 15 μm). (<strong>D</strong>) Quantifying fiber densification dynamics near a single cell and “band” formation in between pairs of cells. N = 7 single cell, N = 13 cell pairs at pair distances of 6-8 cell diameters (90-120 μm).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F3.medium.gif" width="440" height="348"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F3.medium.gif" width="440" height="348"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3.
                               </span>
                               <p id="p-17" class="first-child">
                                Quantifying ECM densification during cell-ECM-cell mechanical communication with 3D live imaging of communicating fibroblast cells. (
                                <strong>
                                 A
                                </strong>
                                ) ECM remodeling dynamics are quantified along the connecting axis between the cells: illustration of the top-side view. Microscopy axes (X’, Y’, Z’) marked with dashed arrows, transformed visualization and quantification axes (connecting axis, XY, Z) marked with solid arrows. The cuboid region in-between the cell pair (brown), has the width of a cell diameter, and is used for quantification along the connecting axis. The length of the cuboid region is the pair distance. The cuboid left and right sides were parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black). Side view and top view are illustrated in
                                <a id="xref-fig-9-2" class="xref-fig" href="#F9">
                                 Fig. S2
                                </a>
                                . (
                                <strong>
                                 B
                                </strong>
                                ) Representative images of a single cell (left) and a cell pair (right) at the onset (top), after (slightly over) two hours (middle), and (slightly over) four hours of live cell imaging (bottom). The initial distance between the cell centers in the pair was ∼117 μm (∼7.8 cell diameters assuming mean fibroblast diameter of 15 μm). White dashed line represents the connecting axis between the cells. White dashed rectangle next to the cell boundaries represents one quantification window (see Methods for details). Scale bar = 15 μm. (
                                <strong>
                                 C
                                </strong>
                                ) Quantifying cell-ECM-cell communication as a function of distance between cell pairs. Single cells (N = 7), Pair distance of 4-6 (N = 9 pairs), 6-8 (N = 13 pairs) and 8-10 (N = 5 pairs) cell diameters (assuming cell diameter of 15 μm). (
                                <strong>
                                 D
                                </strong>
                                ) Quantifying fiber densification dynamics near a single cell and “band” formation in between pairs of cells. N = 7 single cell, N = 13 cell pairs at pair distances of 6-8 cell diameters (90-120 μm).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Temporal correlation of local ECM remodeling fluctuations defines a signature for cell-ECM-cell communication
                             </h3>
                             <p id="p-18">
                              Rather than comparing communicating versus non-communicating cell pairs (
                              <a id="xref-fig-2-7" class="xref-fig" href="#F2">
                               Fig. 2D
                              </a>
                              ), definitive quantification of cell-ECM-cell communication lies in the ability to distinguish between pairs of communicating cells, i.e., whether the ECM fluctuations of one cells pair has a unique signature that can be distinguished from that of a different pair of communicating cells. We tackled this challenge by testing whether the correlation between a simulated pair of communicating cells surpassed the correlation between one cell from that pair and another cell from a different simulated pair of communicating cells in a different fibrous network (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F4.large.jpg?width=800&height=600&carousel=1" title="Distinction between pairs of communicating cells with same-versus-different pair analysis. (A) Schematic sketch of the same-versus-different pair analysis. The correlation between quantification windows of communicating pairs (“same”, green) is evaluated in relation to the correlation between one cell from that pair and another cell from a different communicating pair (“different”, purple). (B) Same-versus-different pair analysis in simulations. Each data point records the correlation between the “same” and “different” cell pairs using the second derivative of fiber density dynamics. Mean contraction of 1% and standard deviation of 0.5%. All combinations of “same”/”different” were considered. Left: Pair distances: 4 (N = 20), 5 (N = 19), 7 (N = 19) and 9 (N = 19) cell diameters. “Same” pair had a higher correlation than “different” pair in 78% of the matched correlations. Right: “Same” - “different” correlation distributions for different pair distances: 4 (N = 20 pairs), 5 (N = 19 pairs), 7 (N = 19 pairs), and 9 (N = 19 pairs) cell diameters. 92% (pair distance = 4), 79% (pair distance = 5), 70% (pair distance = 7) and 68% (pair distance = 9) of data points were positive, implying that the “same” correlation is higher than the “different” correlation. Wilcoxon signed-rank testing the null hypothesis that the data are distributed around a mean 0: p-value < 0.0001 for all pair distances. (C) Schematic sketch of a cell, the quantification window and the normalization border. The normalization border is composed of two parts, above and below the quantification window, each with a width of approximately a cell diameter (15 μm), height of 0.5 cell size and with a gap of 0.25 from the quantification window (full information in Methods). (D) Same-versus-different pair analysis in experiments. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between cell pairs with a visible band, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics. Top: Representative pair of communication cells shown after 255 minutes of imaging. Scale bar = 15 μm. Line connects the cell centers in the XY/Z space. For the displayed pair the Pearson correlation coefficient was 0.66 (Z offset = 0) and 0.71 (Z offset = 0.5). The same image is also shown in Fig. 3B. Bottom left: Pair distance = 60-150 μm (∼4-10 cell diameters). N = 48 cell pairs, all combinations of “same”/”different” were considered. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations. Bottom right: “Same” - “different” correlation distributions for different pair distances: 4-6 (N = 18 pairs), 6-8 (N = 19 pairs), and 8-10 (N = 11 pairs) cell diameters. “Same” pair had a higher correlation than “different” pair in 93% (pair distance 4-6), 96% (pair distance 6-8) and 91% (pair distance 8-10). Wilcoxon signed-rank test p-value < 0.0001 for all pair distances. (E) Same-versus-different pair analysis for cell triplets. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics. Left: Triplet of cells communicating with one another after 255 minutes of live cell imaging. Scale bars = 15 μm. Middle: Same-versus-different pair analysis for the cell triplets from the left panel. Pair distances approximately 7.3 (Cells 1 vs. 2), 6.3 (Cells 1 vs. 3) and 4.9 (Cells 2 vs. 3) cell diameters. “Same” pair had a higher correlation than the “different” pair in 7/8 (Cells 1 & 2), 8/8 (Cells 1 & 3) and 8/8 (Cells 2 & 3) of the matched correlations. Wilcoxon signed-rank test p-value < 0.05. Right: Multiple triplets. N = 3 triplets color coded in shades of green, blue and orange. Triplet #3 was presented in the previous panels (matched shades of orange between panels). 92% of matched same minus different correlations were positive. Wilcoxon signed-rank p-value < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Distinction between pairs of communicating cells with same-versus-different pair analysis. (<strong>A</strong>) Schematic sketch of the <em>same-versus-different pair analysis</em>. The correlation between quantification windows of communicating pairs (“same”, green) is evaluated in relation to the correlation between one cell from that pair and another cell from a different communicating pair (“different”, purple). (<strong>B</strong>) Same-versus-different pair analysis in simulations. Each data point records the correlation between the “same” and “different” cell pairs using the second derivative of fiber density dynamics. Mean contraction of 1% and standard deviation of 0.5%. All combinations of “same”/”different” were considered. Left: Pair distances: 4 (N = 20), 5 (N = 19), 7 (N = 19) and 9 (N = 19) cell diameters. “Same” pair had a higher correlation than “different” pair in 78% of the matched correlations. Right: “Same” - “different” correlation distributions for different pair distances: 4 (N = 20 pairs), 5 (N = 19 pairs), 7 (N = 19 pairs), and 9 (N = 19 pairs) cell diameters. 92% (pair distance = 4), 79% (pair distance = 5), 70% (pair distance = 7) and 68% (pair distance = 9) of data points were positive, implying that the “same” correlation is higher than the “different” correlation. Wilcoxon signed-rank testing the null hypothesis that the data are distributed around a mean 0: p-value < 0.0001 for all pair distances. (<strong>C</strong>) Schematic sketch of a cell, the quantification window and the normalization border. The normalization border is composed of two parts, above and below the quantification window, each with a width of approximately a cell diameter (15 μm), height of 0.5 cell size and with a gap of 0.25 from the quantification window (full information in Methods). (<strong>D</strong>) Same-versus-different pair analysis in experiments. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between cell pairs with a visible band, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics. Top: Representative pair of communication cells shown after 255 minutes of imaging. Scale bar = 15 μm. Line connects the cell centers in the XY/Z space. For the displayed pair the Pearson correlation coefficient was 0.66 (Z offset = 0) and 0.71 (Z offset = 0.5). The same image is also shown in Fig. 3B. Bottom left: Pair distance = 60-150 μm (∼4-10 cell diameters). N = 48 cell pairs, all combinations of “same”/”different” were considered. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations. Bottom right: “Same” - “different” correlation distributions for different pair distances: 4-6 (N = 18 pairs), 6-8 (N = 19 pairs), and 8-10 (N = 11 pairs) cell diameters. “Same” pair had a higher correlation than “different” pair in 93% (pair distance 4-6), 96% (pair distance 6-8) and 91% (pair distance 8-10). Wilcoxon signed-rank test p-value < 0.0001 for all pair distances. (<strong>E</strong>) Same-versus-different pair analysis for cell triplets. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics. Left: Triplet of cells communicating with one another after 255 minutes of live cell imaging. Scale bars = 15 μm. Middle: Same-versus-different pair analysis for the cell triplets from the left panel. Pair distances approximately 7.3 (Cells 1 vs. 2), 6.3 (Cells 1 vs. 3) and 4.9 (Cells 2 vs. 3) cell diameters. “Same” pair had a higher correlation than the “different” pair in 7/8 (Cells 1 & 2), 8/8 (Cells 1 & 3) and 8/8 (Cells 2 & 3) of the matched correlations. Wilcoxon signed-rank test p-value < 0.05. Right: Multiple triplets. N = 3 triplets color coded in shades of green, blue and orange. Triplet #3 was presented in the previous panels (matched shades of orange between panels). 92% of matched same minus different correlations were positive. Wilcoxon signed-rank p-value < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F4.medium.gif" width="368" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F4.medium.gif" width="368" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4.
                               </span>
                               <p id="p-19" class="first-child">
                                Distinction between pairs of communicating cells with same-versus-different pair analysis. (
                                <strong>
                                 A
                                </strong>
                                ) Schematic sketch of the same-versus-different pair analysis. The correlation between quantification windows of communicating pairs (“same”, green) is evaluated in relation to the correlation between one cell from that pair and another cell from a different communicating pair (“different”, purple). (
                                <strong>
                                 B
                                </strong>
                                ) Same-versus-different pair analysis in simulations. Each data point records the correlation between the “same” and “different” cell pairs using the second derivative of fiber density dynamics. Mean contraction of 1% and standard deviation of 0.5%. All combinations of “same”/”different” were considered. Left: Pair distances: 4 (N = 20), 5 (N = 19), 7 (N = 19) and 9 (N = 19) cell diameters. “Same” pair had a higher correlation than “different” pair in 78% of the matched correlations. Right: “Same” - “different” correlation distributions for different pair distances: 4 (N = 20 pairs), 5 (N = 19 pairs), 7 (N = 19 pairs), and 9 (N = 19 pairs) cell diameters. 92% (pair distance = 4), 79% (pair distance = 5), 70% (pair distance = 7) and 68% (pair distance = 9) of data points were positive, implying that the “same” correlation is higher than the “different” correlation. Wilcoxon signed-rank testing the null hypothesis that the data are distributed around a mean 0: p-value < 0.0001 for all pair distances. (
                                <strong>
                                 C
                                </strong>
                                ) Schematic sketch of a cell, the quantification window and the normalization border. The normalization border is composed of two parts, above and below the quantification window, each with a width of approximately a cell diameter (15 μm), height of 0.5 cell size and with a gap of 0.25 from the quantification window (full information in Methods). (
                                <strong>
                                 D
                                </strong>
                                ) Same-versus-different pair analysis in experiments. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between cell pairs with a visible band, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics.
                                <span class="underline">
                                 Top
                                </span>
                                : Representative pair of communication cells shown after 255 minutes of imaging. Scale bar = 15 μm. Line connects the cell centers in the XY/Z space. For the displayed pair the Pearson correlation coefficient was 0.66 (Z offset = 0) and 0.71 (Z offset = 0.5). The same image is also shown in
                                <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                                 Fig. 3B
                                </a>
                                .
                                <span class="underline">
                                 Bottom left
                                </span>
                                : Pair distance = 60-150 μm (∼4-10 cell diameters). N = 48 cell pairs, all combinations of “same”/”different” were considered. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations.
                                <span class="underline">
                                 Bottom right
                                </span>
                                : “Same” - “different” correlation distributions for different pair distances: 4-6 (N = 18 pairs), 6-8 (N = 19 pairs), and 8-10 (N = 11 pairs) cell diameters. “Same” pair had a higher correlation than “different” pair in 93% (pair distance 4-6), 96% (pair distance 6-8) and 91% (pair distance 8-10). Wilcoxon signed-rank test p-value < 0.0001 for all pair distances. (
                                <strong>
                                 E
                                </strong>
                                ) Same-versus-different pair analysis for cell triplets. Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics.
                                <span class="underline">
                                 Left
                                </span>
                                : Triplet of cells communicating with one another after 255 minutes of live cell imaging. Scale bars = 15 μm.
                                <span class="underline">
                                 Middle
                                </span>
                                : Same-versus-different pair analysis for the cell triplets from the left panel. Pair distances approximately 7.3 (Cells 1 vs. 2), 6.3 (Cells 1 vs. 3) and 4.9 (Cells 2 vs. 3) cell diameters. “Same” pair had a higher correlation than the “different” pair in 7/8 (Cells 1 & 2), 8/8 (Cells 1 & 3) and 8/8 (Cells 2 & 3) of the matched correlations. Wilcoxon signed-rank test p-value < 0.05.
                                <span class="underline">
                                 Right
                                </span>
                                : Multiple triplets. N = 3 triplets color coded in shades of green, blue and orange. Triplet #3 was presented in the previous panels (matched shades of orange between panels). 92% of matched same minus different correlations were positive. Wilcoxon signed-rank p-value < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              Specifically, we compared the correlation in the second temporal derivative of the local ECM density dynamics between quantification windows adjacent to each cell pair, located in the ‘same’ pair versus a cell in a ‘different’ pair, and accordingly coined the term same-versus-different pair analysis (schematic in
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4A
                              </a>
                              ). Having the “same” correlation exceeding the “different” correlation implies that the correlation between a communicating cell pair is not merely an effect of a similar ECM densification pattern that is common to any pair of communication cells, but rather is indicative of communication unique for the “same” cell pair at test. We considered all possible ordered combinations of triplets of cells that include a pair of communicating cells and a third cell that takes part in a different communicating cell pair. In simulated cells, a “same” pair had a higher correlation than a “different” pair in 92% of the matched correlations for pair distance of 4 cell-diameters, and this correlation gradually reduced with increased pair distance (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4B
                              </a>
                              ). To assess the validity of the method for fibrous ECM network with varying mechanical properties we also simulated a network with a relative linear-elastic behavior until 15% strain (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Fig. S4A
                              </a>
                              ). Same-versus-different analysis was improved for the stiffening model (
                              <a id="xref-fig-11-2" class="xref-fig" href="#F11">
                               Fig. S4B
                              </a>
                              ), implicating the role of strain stiffening in long-range mechanical communication, in line with our previous studies
                              <sup>
                               <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                                12
                               </a>
                               ,
                               <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                                30
                               </a>
                               ,
                               <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                                31
                               </a>
                              </sup>
                              . Altogether, simulated cells communicating with one-another were more synchronized in their ECM remodeling fluctuations, and same-versus-different analysis could distinguish between different pairs of communicating cells.
                             </p>
                             <div id="F11" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F11.large.jpg?width=800&height=600&carousel=1" title="Comparison of fibrous material models with linear and stiffening behavior. (A) Stress-strain curves representing the mechanical behavior of the fibers used in the simulations. Stiffening model is based on 2.4 mg/ml collagen, linear model is based on 4 mg/ml fibrin 53. (B) Same network correlations minus different network correlations as a function of the cell-to-cell pair distance (second derivative of fiber density, cell heterogeneity of mean contraction of 1% and standard deviation of 0.5%). Correlations were calculated for cell-to-cell pair distances of 4 (20 pairs for the stiffening model and 10 pairs for linear model), 5 (19 pairs for the stiffening model and 10 pairs for the linear model), 7 (19 pairs for the stiffening and 10 pairs for the linear), and 9 (19 pairs for the stiffening model and 10 pairs for the linear model) cell diameters. For ‘stiffening’, “same” > “different” in 72% of the matched same-versus-different pair analysis (92%, 79%, 70% and 68% for pair distances of 4,5,7 and 9 cell diameters, correspondingly), whereas in for ‘linear’, “same” > “different” in 61% of the matched same-versus-different pair analysis (76%, 68%, 71% and 43% for pair distances of 4,5,7 and 9 cell diameters, correspondingly)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Comparison of fibrous material models with linear and stiffening behavior. (<strong>A</strong>) Stress-strain curves representing the mechanical behavior of the fibers used in the simulations. Stiffening model is based on 2.4 mg/ml collagen, linear model is based on 4 mg/ml fibrin <sup>53</sup>. (<strong>B</strong>) Same network correlations minus different network correlations as a function of the cell-to-cell pair distance (second derivative of fiber density, cell heterogeneity of mean contraction of 1% and standard deviation of 0.5%). Correlations were calculated for cell-to-cell pair distances of 4 (20 pairs for the stiffening model and 10 pairs for linear model), 5 (19 pairs for the stiffening model and 10 pairs for the linear model), 7 (19 pairs for the stiffening and 10 pairs for the linear), and 9 (19 pairs for the stiffening model and 10 pairs for the linear model) cell diameters. For ‘stiffening’, “same” > “different” in 72% of the matched same-versus-different pair analysis (92%, 79%, 70% and 68% for pair distances of 4,5,7 and 9 cell diameters, correspondingly), whereas in for ‘linear’, “same” > “different” in 61% of the matched same-versus-different pair analysis (76%, 68%, 71% and 43% for pair distances of 4,5,7 and 9 cell diameters, correspondingly).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F11.medium.gif" width="440" height="196"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F11.medium.gif" width="440" height="196"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S4:
                               </span>
                               <p id="p-106" class="first-child">
                                Comparison of fibrous material models with linear and stiffening behavior. (
                                <strong>
                                 A
                                </strong>
                                ) Stress-strain curves representing the mechanical behavior of the fibers used in the simulations. Stiffening model is based on 2.4 mg/ml collagen, linear model is based on 4 mg/ml fibrin
                                <sup>
                                 <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">
                                  53
                                 </a>
                                </sup>
                                . (
                                <strong>
                                 B
                                </strong>
                                ) Same network correlations minus different network correlations as a function of the cell-to-cell pair distance (second derivative of fiber density, cell heterogeneity of mean contraction of 1% and standard deviation of 0.5%). Correlations were calculated for cell-to-cell pair distances of 4 (20 pairs for the stiffening model and 10 pairs for linear model), 5 (19 pairs for the stiffening model and 10 pairs for the linear model), 7 (19 pairs for the stiffening and 10 pairs for the linear), and 9 (19 pairs for the stiffening model and 10 pairs for the linear model) cell diameters. For ‘stiffening’, “same” > “different” in 72% of the matched same-versus-different pair analysis (92%, 79%, 70% and 68% for pair distances of 4,5,7 and 9 cell diameters, correspondingly), whereas in for ‘linear’, “same” > “different” in 61% of the matched same-versus-different pair analysis (76%, 68%, 71% and 43% for pair distances of 4,5,7 and 9 cell diameters, correspondingly).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-21">
                              Non-genetic phenotypic cell contractile heterogeneity, which we artificially introduced to simulations, and was necessary to identify cell-ECM-cell communication, is an inherent cellular property. Thus, we aimed at extending our simulation results of distinguishing between pairs of communicating cells to experimental data. The first temporal derivative was sufficient to remove non-stationarity effects in ECM remodeling fluctuations of single and pairs of communicating fibroblast cells (
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Fig. S5
                              </a>
                              ), so we could avoid the second derivative in the correlation analysis of experimental data. Unlike simulations, all cells in a single experiment were embedded in the same fibrous gel possibly inducing spatial ECM correlations in nearby regions that do not necessarily involve cell-ECM-cell communication. Thus, we had to control for spurious correlations that could originate from the proximity between the ECM regions in the same network. This was achieved by including an additional step in the quantification where we corrected the ECM remodeling fluctuations in the quantification window in respect to local regions close to that window to reduce local temporal artifacts that may lead to erroneous correlations (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4C
                              </a>
                              , Methods).
                             </p>
                             <div id="F12" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F12.large.jpg?width=800&height=600&carousel=1" title="Detrending fiber density of single cell and cell pairs. Shown are correlations between fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) in time. (A-C) Cell pairs. Pairs distance 60-150 μm (4-10 cell diameters). N = 42 cell pairs. (A) Wilcoxon signed-rank testing the null hypothesis that the correlations are distributed around a mean = 0: p-value < 0.0001 (D), not significant (D’, D’’). (B) Kwiatkowski–Phillips–Schmidt–Shin (KPSS) testing the null hypothesis that the data is stationary. Each data point is the KPSS test p-value rejecting the null hypothesis that the time series is stationary, with p-value < 0.05. The null hypothesis is rejected for 99% of the raw and for 100% of first or second derivative time series. (C) Augmented Dickey Fuller test (ADF) testing the null hypothesis that the data is non-stationary. Each data point is the ADF test p-value rejecting the null hypothesis that the time series is not stationary, with p-value < 0.05. The null hypothesis is rejected for 24% of the raw time series, 46% for the first derivative and 55% for the second derivative. (D-F) Single cells. N = 7 cells. Correlations are computed between all possible pairs of single cells, total of 21 correlations. (D) Wilcoxon signed-rank p-value not significant (D, D’, D’’). (E) KPSS is not rejected for all raw, first, or second derivative time series. (F) ADF is rejected for 29% of the raw time series, 71% for the first derivative and 57% for the second derivative." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Detrending fiber density of single cell and cell pairs. Shown are correlations between fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) in time. (<strong>A-C</strong>) Cell pairs. Pairs distance 60-150 μm (4-10 cell diameters). N = 42 cell pairs. (<strong>A</strong>) Wilcoxon signed-rank testing the null hypothesis that the correlations are distributed around a mean = 0: p-value < 0.0001 (D), not significant (D’, D’’). (<strong>B</strong>) Kwiatkowski–Phillips–Schmidt–Shin (KPSS) testing the null hypothesis that the data is stationary. Each data point is the KPSS test p-value rejecting the null hypothesis that the time series is stationary, with p-value < 0.05. The null hypothesis is rejected for 99% of the raw and for 100% of first or second derivative time series. (<strong>C</strong>) Augmented Dickey Fuller test (ADF) testing the null hypothesis that the data is non-stationary. Each data point is the ADF test p-value rejecting the null hypothesis that the time series is not stationary, with p-value < 0.05. The null hypothesis is rejected for 24% of the raw time series, 46% for the first derivative and 55% for the second derivative. (<strong>D-F</strong>) Single cells. N = 7 cells. Correlations are computed between all possible pairs of single cells, total of 21 correlations. (<strong>D</strong>) Wilcoxon signed-rank p-value not significant (D, D’, D’’). (<strong>E</strong>) KPSS is not rejected for all raw, first, or second derivative time series. (<strong>F</strong>) ADF is rejected for 29% of the raw time series, 71% for the first derivative and 57% for the second derivative.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F12.medium.gif" width="440" height="327"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F12.medium.gif" width="440" height="327"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S5:
                               </span>
                               <p id="p-107" class="first-child">
                                Detrending fiber density of single cell and cell pairs. Shown are correlations between fiber density (D), first temporal derivative (D’) and second temporal derivative (D’’) in time. (
                                <strong>
                                 A-C
                                </strong>
                                ) Cell pairs. Pairs distance 60-150 μm (4-10 cell diameters). N = 42 cell pairs. (
                                <strong>
                                 A
                                </strong>
                                ) Wilcoxon signed-rank testing the null hypothesis that the correlations are distributed around a mean = 0: p-value < 0.0001 (D), not significant (D’, D’’). (
                                <strong>
                                 B
                                </strong>
                                ) Kwiatkowski–Phillips–Schmidt–Shin (KPSS) testing the null hypothesis that the data is stationary. Each data point is the KPSS test p-value rejecting the null hypothesis that the time series is stationary, with p-value < 0.05. The null hypothesis is rejected for 99% of the raw and for 100% of first or second derivative time series. (
                                <strong>
                                 C
                                </strong>
                                ) Augmented Dickey Fuller test (ADF) testing the null hypothesis that the data is non-stationary. Each data point is the ADF test p-value rejecting the null hypothesis that the time series is not stationary, with p-value < 0.05. The null hypothesis is rejected for 24% of the raw time series, 46% for the first derivative and 55% for the second derivative. (
                                <strong>
                                 D-F
                                </strong>
                                ) Single cells. N = 7 cells. Correlations are computed between all possible pairs of single cells, total of 21 correlations. (
                                <strong>
                                 D
                                </strong>
                                ) Wilcoxon signed-rank p-value not significant (D, D’, D’’). (
                                <strong>
                                 E
                                </strong>
                                ) KPSS is not rejected for all raw, first, or second derivative time series. (
                                <strong>
                                 F
                                </strong>
                                ) ADF is rejected for 29% of the raw time series, 71% for the first derivative and 57% for the second derivative.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-22">
                              We performed the same-versus-different pair analysis, but were able to quantitatively distinguish between different pairs of communicating fibroblast cells only after shifting the quantification window 7.5 μm above or below the axis connecting the communicating cells (
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4D
                              </a>
                              ,
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Fig. S6
                              </a>
                              ). A “same” pair had a higher correlation than its corresponding “different” pair in 94% of the matched correlations, for various pair distances (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig. 4D
                              </a>
                              ,
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               Fig. S7A-C
                              </a>
                              ) and even for different cell pairs within triplets of cells (
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Fig. 4E
                              </a>
                              ).
                             </p>
                             <div id="F13" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F13.large.jpg?width=800&height=600&carousel=1" title="Enhanced identification of communicating cells by shifting the quantification window’s location. Same-versus-different pair analysis for different locations of the quantification window. Pair distance was 60-150 μm (∼4-10) cell diameters, window distance = 0, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/”different” were considered. Slightly different numbers of cell pairs per quantification window location stems from discarding pairs where the quantification windows exceeded beyond the image boundaries. See Fig. 7A for axes schematics. Top panel (Z = 0.5, XY = 0): N = 48 cell pairs. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. Left panel (Z = 0, XY = -0.5): N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 50% of the matched correlations. Wilcoxon signed-rank p-value not significant. Middle panel (Z = 0, XY = 0): N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 52% of the matched correlations. Wilcoxon signed-rank p-value < 0.001. Right panel (Z = 0, XY = 0.5): N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 56% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. Bottom panel (Z = -0.5, XY = 0): N = 36 cell pairs. “Same” pair had a higher correlation than “different” pair in 88% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Enhanced identification of communicating cells by shifting the quantification window’s location. Same-versus-different pair analysis for different locations of the quantification window. Pair distance was 60-150 μm (∼4-10) cell diameters, window distance = 0, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/”different” were considered. Slightly different numbers of cell pairs per quantification window location stems from discarding pairs where the quantification windows exceeded beyond the image boundaries. See Fig. 7A for axes schematics. <strong>Top panel (Z = 0.5, XY = 0)</strong>: N = 48 cell pairs. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. <strong>Left panel (Z = 0, XY = -0.5)</strong>: N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 50% of the matched correlations. Wilcoxon signed-rank p-value not significant. <strong>Middle panel (Z = 0, XY = 0)</strong>: N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 52% of the matched correlations. Wilcoxon signed-rank p-value < 0.001. <strong>Right panel (Z = 0, XY = 0.5)</strong>: N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 56% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. <strong>Bottom panel (Z = -0.5, XY = 0)</strong>: N = 36 cell pairs. “Same” pair had a higher correlation than “different” pair in 88% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F13.medium.gif" width="431" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F13.medium.gif" width="431" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S6:
                               </span>
                               <p id="p-108" class="first-child">
                                Enhanced identification of communicating cells by shifting the quantification window’s location. Same-versus-different pair analysis for different locations of the quantification window. Pair distance was 60-150 μm (∼4-10) cell diameters, window distance = 0, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/”different” were considered. Slightly different numbers of cell pairs per quantification window location stems from discarding pairs where the quantification windows exceeded beyond the image boundaries. See
                                <a id="xref-fig-7-7" class="xref-fig" href="#F7">
                                 Fig. 7A
                                </a>
                                for axes schematics.
                                <strong>
                                 Top panel (Z = 0.5, XY = 0)
                                </strong>
                                : N = 48 cell pairs. “Same” pair had a higher correlation than “different” pair in 94% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.
                                <strong>
                                 Left panel (Z = 0, XY = -0.5)
                                </strong>
                                : N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 50% of the matched correlations. Wilcoxon signed-rank p-value not significant.
                                <strong>
                                 Middle panel (Z = 0, XY = 0)
                                </strong>
                                : N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 52% of the matched correlations. Wilcoxon signed-rank p-value < 0.001.
                                <strong>
                                 Right panel (Z = 0, XY = 0.5)
                                </strong>
                                : N = 42 cell pairs. “Same” pair had a higher correlation than “different” pair in 56% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.
                                <strong>
                                 Bottom panel (Z = -0.5, XY = 0)
                                </strong>
                                : N = 36 cell pairs. “Same” pair had a higher correlation than “different” pair in 88% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F14" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F14.large.jpg?width=800&height=600&carousel=1" title="Identification of communicating cells in different pair distances. Same-versus-different pair analysis. Quantification windows were placed 7.5 μm above the connecting axis between the cells (see Fig. 5E for schematics). Correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered. Partitioning the data to cell pairs intervals of 4-6, 6-8, or 8-10 cell diameters was performed under the assumption of a mean fibroblast diameter of 15 μm. (A) Pair distance 60-90 μm (4-6 cell diameters). N = 18 cell pairs. “Same” pair had a higher correlation than “different” pair in 93% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (B) Pair distance 90-120 μm (6-8 cell diameters). N = 19 cell pairs. “Same” pair had a higher correlation than “different” pair in 96% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (C) Pair distance 120-150 μm (8-10). N = 11 cell pairs. “Same” pair had a higher correlation than “different” pair in 91% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Identification of communicating cells in different pair distances. Same-versus-different pair analysis. Quantification windows were placed 7.5 μm above the connecting axis between the cells (see Fig. 5E for schematics). Correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered. Partitioning the data to cell pairs intervals of 4-6, 6-8, or 8-10 cell diameters was performed under the assumption of a mean fibroblast diameter of 15 μm. (<strong>A</strong>) Pair distance 60-90 μm (4-6 cell diameters). N = 18 cell pairs. “Same” pair had a higher correlation than “different” pair in 93% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (<strong>B</strong>) Pair distance 90-120 μm (6-8 cell diameters). N = 19 cell pairs. “Same” pair had a higher correlation than “different” pair in 96% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (<strong>C</strong>) Pair distance 120-150 μm (8-10). N = 11 cell pairs. “Same” pair had a higher correlation than “different” pair in 91% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F14.medium.gif" width="440" height="164"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F14.medium.gif" width="440" height="164"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S7:
                               </span>
                               <p id="p-109" class="first-child">
                                Identification of communicating cells in different pair distances. Same-versus-different pair analysis. Quantification windows were placed 7.5 μm above the connecting axis between the cells (see
                                <a id="xref-fig-5-12" class="xref-fig" href="#F5">
                                 Fig. 5E
                                </a>
                                for schematics). Correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered. Partitioning the data to cell pairs intervals of 4-6, 6-8, or 8-10 cell diameters was performed under the assumption of a mean fibroblast diameter of 15 μm. (
                                <strong>
                                 A
                                </strong>
                                ) Pair distance 60-90 μm (4-6 cell diameters). N = 18 cell pairs. “Same” pair had a higher correlation than “different” pair in 93% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 B
                                </strong>
                                ) Pair distance 90-120 μm (6-8 cell diameters). N = 19 cell pairs. “Same” pair had a higher correlation than “different” pair in 96% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 C
                                </strong>
                                ) Pair distance 120-150 μm (8-10). N = 11 cell pairs. “Same” pair had a higher correlation than “different” pair in 91% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-23">
                              To control for potential masking of cell-ECM-cell communication by correlated non-communication related local ECM remodeling we devised a computational control that we term real-versus-fake pair analysis. We created new pairs where each is composed of one real cell and another fake cell located in the exact same distance as the matched pair of communicating cells, and away from other cells (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5A
                              </a>
                              , Methods). ECM remodeling correlation between a pair of communicating cells (“real-real”) was higher than its corresponding “real-fake” pair in 86% of the matched correlations providing a standardized internal control and further validation that our method truly captures cell-ECM-cell communication (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5B
                              </a>
                              ).
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F5.large.jpg?width=800&height=600&carousel=1" title="Quantitative identification of unique ECM remodeling signatures of communication partners by quantifying the confounding factor of non-communication-related local deformations in the ECM and matchmaking between communication partners. (A) Schematic sketch of the real-versus-fake pair analysis. “Fake” cell (cyan) coordinates were located on the circumference of a circle (dashed purple), with a radius r from the corresponding “real” communicating cell (dark green), and with the maximal distance (d, orange) in respect to all other real cells (light green). (B) Correlations between ECM fluctuations of pairs of communicating cell pairs (“real-real”, x-axis) versus the correlation between a cell and non-cellular-related local deformations in the ECM (“real-fake”, y-axis). Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the communicating cells, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics. N = 48 “real-real” cells at cell pair distances ranging 4-10 cell diameters. “Real-real” pair had a higher correlation than “real-fake” pair in 86% of cases. Wilcoxon sign-rank p-value < 0.0001. (C) Schematic sketch of the matchmaking analysis. ECM remodeling dynamics of a given cell (left) was correlated with 49 other cells (with repeats – see Methods) from an experiment (right). The rank of the correlation with the true communication partner (i.e., the position in the sorted list of all correlations with other cells, marked in green) was recorded. This process repeated for all cells. (D-G) Distributions of the correlation rank with the true communication partner for cell pairs with a visible band. Pair distance of 60-150 μm (∼4-10 cell diameters). Correlations were calculated using the first derivative of the ECM intensity over time. “Correct match” refers to a correlation rank of 1. (D) Random matching: simulation of arbitrary matching leads to correct matching probability of 0.02 (N = 50 cells). (E) Quantification window 0.5 cell diameters (7.5 μm) above the connecting axis between the cells: correct matching probability of 0.69 (selecting from N = 96 cells), compared to 0.02 for random matching (panel D). (F) Quantification window at the connecting axis between the cells: correct matching probability of 0.1 (selecting from N = 84 cells), compared to 0.02 for random matching. (G) ECM regions (without cells) located close to each other (“fake-following” pairs, see Fig. S8 for schematics): correct matching probability of 0.02 (N = 50 regions), compared to 0.02 for random matching." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Quantitative identification of unique ECM remodeling signatures of communication partners by quantifying the confounding factor of non-communication-related local deformations in the ECM and matchmaking between communication partners. (<strong>A</strong>) Schematic sketch of the real-versus-fake pair analysis. “Fake” cell (cyan) coordinates were located on the circumference of a circle (dashed purple), with a radius <em>r</em> from the corresponding “real” communicating cell (dark green), and with the maximal distance (<em>d</em>, orange) in respect to all other real cells (light green). (<strong>B</strong>) Correlations between ECM fluctuations of pairs of communicating cell pairs (“real-real”, x-axis) versus the correlation between a cell and non-cellular-related local deformations in the ECM (“real-fake”, y-axis). Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the communicating cells, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics. N = 48 “real-real” cells at cell pair distances ranging 4-10 cell diameters. “Real-real” pair had a higher correlation than “real-fake” pair in 86% of cases. Wilcoxon sign-rank p-value < 0.0001. (<strong>C</strong>) Schematic sketch of the matchmaking analysis. ECM remodeling dynamics of a given cell (left) was correlated with 49 other cells (with repeats – see Methods) from an experiment (right). The rank of the correlation with the true communication partner (i.e., the position in the sorted list of all correlations with other cells, marked in green) was recorded. This process repeated for all cells. (<strong>D-G</strong>) Distributions of the correlation rank with the true communication partner for cell pairs with a visible band. Pair distance of 60-150 μm (∼4-10 cell diameters). Correlations were calculated using the first derivative of the ECM intensity over time. “Correct match” refers to a correlation rank of 1. (<strong>D</strong>) Random matching: simulation of arbitrary matching leads to correct matching probability of 0.02 (N = 50 cells). (<strong>E</strong>) Quantification window 0.5 cell diameters (7.5 μm) above the connecting axis between the cells: correct matching probability of 0.69 (selecting from N = 96 cells), compared to 0.02 for random matching (panel D). (<strong>F</strong>) Quantification window at the connecting axis between the cells: correct matching probability of 0.1 (selecting from N = 84 cells), compared to 0.02 for random matching. (<strong>G</strong>) ECM regions (without cells) located close to each other (“fake-following” pairs, see Fig. S8 for schematics): correct matching probability of 0.02 (N = 50 regions), compared to 0.02 for random matching.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F5.medium.gif" width="419" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F5.medium.gif" width="419" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5.
                               </span>
                               <p id="p-24" class="first-child">
                                Quantitative identification of unique ECM remodeling signatures of communication partners by quantifying the confounding factor of non-communication-related local deformations in the ECM and matchmaking between communication partners. (
                                <strong>
                                 A
                                </strong>
                                ) Schematic sketch of the real-versus-fake pair analysis. “Fake” cell (cyan) coordinates were located on the circumference of a circle (dashed purple), with a radius r from the corresponding “real” communicating cell (dark green), and with the maximal distance (d, orange) in respect to all other real cells (light green). (
                                <strong>
                                 B
                                </strong>
                                ) Correlations between ECM fluctuations of pairs of communicating cell pairs (“real-real”, x-axis) versus the correlation between a cell and non-cellular-related local deformations in the ECM (“real-fake”, y-axis). Quantification windows were placed ∼0.5 cell diameter (7.5 μm) above the connecting axis between the communicating cells, no offset in X/Y-axis. Correlations were calculated using the first derivative of fiber density dynamics. N = 48 “real-real” cells at cell pair distances ranging 4-10 cell diameters. “Real-real” pair had a higher correlation than “real-fake” pair in 86% of cases. Wilcoxon sign-rank p-value < 0.0001. (
                                <strong>
                                 C
                                </strong>
                                ) Schematic sketch of the matchmaking analysis. ECM remodeling dynamics of a given cell (left) was correlated with 49 other cells (with repeats – see Methods) from an experiment (right). The rank of the correlation with the true communication partner (i.e., the position in the sorted list of all correlations with other cells, marked in green) was recorded. This process repeated for all cells. (
                                <strong>
                                 D-G
                                </strong>
                                ) Distributions of the correlation rank with the true communication partner for cell pairs with a visible band. Pair distance of 60-150 μm (∼4-10 cell diameters). Correlations were calculated using the first derivative of the ECM intensity over time. “Correct match” refers to a correlation rank of 1. (
                                <strong>
                                 D
                                </strong>
                                ) Random matching: simulation of arbitrary matching leads to correct matching probability of 0.02 (N = 50 cells). (
                                <strong>
                                 E
                                </strong>
                                ) Quantification window 0.5 cell diameters (7.5 μm) above the connecting axis between the cells: correct matching probability of 0.69 (selecting from N = 96 cells), compared to 0.02 for random matching (panel D). (
                                <strong>
                                 F
                                </strong>
                                ) Quantification window at the connecting axis between the cells: correct matching probability of 0.1 (selecting from N = 84 cells), compared to 0.02 for random matching. (
                                <strong>
                                 G
                                </strong>
                                ) ECM regions (without cells) located close to each other (“fake-following” pairs, see
                                <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                                 Fig. S8
                                </a>
                                for schematics): correct matching probability of 0.02 (N = 50 regions), compared to 0.02 for random matching.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F15" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F15.large.jpg?width=800&height=600&carousel=1" title="Controlling for potential masking of cell-ECM-cell communication by local ECM remodeling fluctuations. (A) Schematic sketch of a “fake-following” pair (cyan) mimicking a real cell pair (green) by repeating the cells’ shifts in X and Y axes (green shadow) in a fixed distance from the real pair (cyan shadow). (B) Quantification of same-versus-different analysis for “fake-following” pairs. N = 25 fake cell pairs. “Same” pair had a higher correlation than “different” pair in 46% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (C-F) Cell-ECM-cell communication is more prominent than the masking by local ECM remodeling fluctuations. Same-vs-different pair analysis of cell pairs and their corresponding “fake-follower” cell pair. Pair distance 60-150 μm, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered for both the cell pairs and the “fake” cell pairs independently. Left panels compare the distribution of same-versus-different pair analysis of cell pairs (“real”) versus “fake-follower” pairs (“fake”). Right panels compare matched same-vs-different pair analysis for each cell pair (“real”) and its corresponding “fake-follower” pair (“fake”). Panels C-D and E-F are the same analysis with low (15 minutes) versus high (5 minutes) temporal resolution correspondingly. In panels C and E the quantification window is located at the focal plane between the cell pair, while in panels D and F 7.5 μm above the focal plain between the cell pair. (C) Quantification window at the focal plane between the cell pair. Temporal resolution = 15 minutes. N = 20 matched pairs. Left: “same” > “different” in 54% (“real”) and 62% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 43% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (D) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 15 minutes. N = 25 matched pairs. Left: “same” > “different” in 98% (“real”) and 46% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 90% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (E) Quantification window at the focal plane between the cell pair. Temporal resolution = 5 minutes. N = 19 matched pairs. Left: “same” > “different” in 56% (“real”) and 57% (“fake”) of the same-vs-different pair analysis. Wilcoxon signed-rank p-values < 0.001. Right: “real” > “fake” in 54% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.05. (F) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 5 minutes. N = 14 matched pairs. (G) “Same” > “different” in 100% (“real”) and 67% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. (H) “Real” > “fake” in 100% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Controlling for potential masking of cell-ECM-cell communication by local ECM remodeling fluctuations. (<strong>A</strong>) Schematic sketch of a “fake-following” pair (cyan) mimicking a real cell pair (green) by repeating the cells’ shifts in X and Y axes (green shadow) in a fixed distance from the real pair (cyan shadow). (<strong>B</strong>) Quantification of same-versus-different analysis for “fake-following” pairs. N = 25 fake cell pairs. “Same” pair had a higher correlation than “different” pair in 46% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (<strong>C-F</strong>) Cell-ECM-cell communication is more prominent than the masking by local ECM remodeling fluctuations. Same-vs-different pair analysis of cell pairs and their corresponding “fake-follower” cell pair. Pair distance 60-150 μm, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered for both the cell pairs and the “fake” cell pairs independently. Left panels compare the distribution of same-versus-different pair analysis of cell pairs (“real”) versus “fake-follower” pairs (“fake”). Right panels compare matched same-vs-different pair analysis for each cell pair (“real”) and its corresponding “fake-follower” pair (“fake”). Panels C-D and E-F are the same analysis with low (15 minutes) versus high (5 minutes) temporal resolution correspondingly. In panels C and E the quantification window is located at the focal plane between the cell pair, while in panels D and F 7.5 μm above the focal plain between the cell pair. (<strong>C</strong>) Quantification window at the focal plane between the cell pair. Temporal resolution = 15 minutes. N = 20 matched pairs. Left: “same” > “different” in 54% (“real”) and 62% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 43% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (<strong>D</strong>) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 15 minutes. N = 25 matched pairs. Left: “same” > “different” in 98% (“real”) and 46% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 90% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (<strong>E</strong>) Quantification window at the focal plane between the cell pair. Temporal resolution = 5 minutes. N = 19 matched pairs. Left: “same” > “different” in 56% (“real”) and 57% (“fake”) of the same-vs-different pair analysis. Wilcoxon signed-rank p-values < 0.001. Right: “real” > “fake” in 54% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.05. (<strong>F</strong>) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 5 minutes. N = 14 matched pairs. (<strong>G</strong>) “Same” > “different” in 100% (“real”) and 67% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. (<strong>H</strong>) “Real” > “fake” in 100% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F15.medium.gif" width="440" height="430"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F15.medium.gif" width="440" height="430"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S8:
                               </span>
                               <p id="p-110" class="first-child">
                                Controlling for potential masking of cell-ECM-cell communication by local ECM remodeling fluctuations. (
                                <strong>
                                 A
                                </strong>
                                ) Schematic sketch of a “fake-following” pair (cyan) mimicking a real cell pair (green) by repeating the cells’ shifts in X and Y axes (green shadow) in a fixed distance from the real pair (cyan shadow). (
                                <strong>
                                 B
                                </strong>
                                ) Quantification of same-versus-different analysis for “fake-following” pairs. N = 25 fake cell pairs. “Same” pair had a higher correlation than “different” pair in 46% of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 C-F
                                </strong>
                                ) Cell-ECM-cell communication is more prominent than the masking by local ECM remodeling fluctuations. Same-vs-different pair analysis of cell pairs and their corresponding “fake-follower” cell pair. Pair distance 60-150 μm, correlations were calculated using the first derivative of fiber density dynamics. All combinations of “same”/“different” were considered for both the cell pairs and the “fake” cell pairs independently. Left panels compare the distribution of same-versus-different pair analysis of cell pairs (“real”) versus “fake-follower” pairs (“fake”). Right panels compare matched same-vs-different pair analysis for each cell pair (“real”) and its corresponding “fake-follower” pair (“fake”). Panels C-D and E-F are the same analysis with low (15 minutes) versus high (5 minutes) temporal resolution correspondingly. In panels C and E the quantification window is located at the focal plane between the cell pair, while in panels D and F 7.5 μm above the focal plain between the cell pair. (
                                <strong>
                                 C
                                </strong>
                                ) Quantification window at the focal plane between the cell pair. Temporal resolution = 15 minutes. N = 20 matched pairs. Left: “same” > “different” in 54% (“real”) and 62% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 43% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 D
                                </strong>
                                ) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 15 minutes. N = 25 matched pairs. Left: “same” > “different” in 98% (“real”) and 46% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. Right: “real” > “fake” in 90% of the matched same-versus-different pair analysis. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 E
                                </strong>
                                ) Quantification window at the focal plane between the cell pair. Temporal resolution = 5 minutes. N = 19 matched pairs. Left: “same” > “different” in 56% (“real”) and 57% (“fake”) of the same-vs-different pair analysis. Wilcoxon signed-rank p-values < 0.001. Right: “real” > “fake” in 54% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.05. (
                                <strong>
                                 F
                                </strong>
                                ) Quantification window 7.5 μm above focal plane between the cell pair. Temporal resolution = 5 minutes. N = 14 matched pairs. (
                                <strong>
                                 G
                                </strong>
                                ) “Same” > “different” in 100% (“real”) and 67% (“fake”) of the same-versus-different pair analysis. Wilcoxon signed-rank p-values < 0.0001. (
                                <strong>
                                 H
                                </strong>
                                ) “Real” > “fake” in 100% of the matched same-vs-different pair analysis observations. Wilcoxon signed-rank p-value < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              Given our ability to distinguish one pair from a different pair of communicating cells, we wondered whether we can match a cell to its true communication partner when considering all other cells in the experiment (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5C
                              </a>
                              , Video S5, Methods). For this “matchmaking” analysis, we considered the true communication partner as the one that is connected with a visible band. For each cell, in every cell pair with a visible band, we recorded the first derivative of its fiber density dynamics, correlated it to all other cells in the gel, and reported the rank of its true matching partner. With 50 potential partners, the expected random probability of identifying the communication partner is 1/50 = 0.02 (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5D
                              </a>
                              ). Using the correlation-based matching, the probability of identifying the true communication partner was 0.69 (
                              <a id="xref-fig-5-5" class="xref-fig" href="#F5">
                               Fig. 5E
                              </a>
                              ). This accuracy of identifying the matching partner dropped to 0.1 when considering quantification regions along the connecting axis between the cells (i.e., no offset in the z-axis,
                              <a id="xref-fig-5-6" class="xref-fig" href="#F5">
                               Fig. 5F
                              </a>
                              ) and was not attributed to correlated non-communication related local ECM remodeling as verified by careful analysis that considered ECM regions (without cells) located close to each other (“fake” pairs,
                              <a id="xref-fig-5-7" class="xref-fig" href="#F5">
                               Fig. 5G
                              </a>
                              ,
                              <a id="xref-fig-15-2" class="xref-fig" href="#F15">
                               Fig. S8
                              </a>
                              , Methods).
                             </p>
                             <p id="p-26">
                              To support the generalization of our method in measuring cell-ECM-cell communication in cell systems beyond fibroblasts we performed experimental validation with a different cell model of murine cancer Hras-transformed mutated cells (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Fig. S9
                              </a>
                              , Methods). Faster imaging (5 minutes per frame) provided a more sensitive readout for cell-ECM-cell communication, reaching perfect accuracy in same-versus-different, real-versus-fake and matchmaking analyses, thus enhancing the capacity to distinguish pairs of communicating cells (
                              <a id="xref-fig-17-1" class="xref-fig" href="#F17">
                               Fig. S10
                              </a>
                              ). However, in considering the inherent tradeoff between temporal resolution and the number of cell pairs we can image in a single experiment with our microscope we prioritize the latter for purposes of collecting sufficient statistics for the rest of this study.
                             </p>
                             <div id="F16" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F16.large.jpg?width=800&height=600&carousel=1" title="Cell-ECM-cell communication in Murine cancer Hras mutated cells. Same-versus-different: N = 48, 94% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 48, 86% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 96, correct matching probability = 72%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup">Cell-ECM-cell communication in Murine cancer Hras mutated cells. Same-versus-different: N = 48, 94% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 48, 86% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 96, correct matching probability = 72%.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F16.medium.gif" width="440" height="121"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F16.medium.gif" width="440" height="121"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S9:
                               </span>
                               <p id="p-111" class="first-child">
                                Cell-ECM-cell communication in Murine cancer Hras mutated cells. Same-versus-different: N = 48, 94% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 48, 86% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 96, correct matching probability = 72%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F17" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F17.large.jpg?width=800&height=600&carousel=1" title="Sensitivity of same-versus-different pair analysis to the temporal resolution. Pair distance 60-150 μm, quantification windows were placed 7.5 μm above the connecting axis between the cells (see Fig. 7E for schematics). Correlations were calculated using the first derivative of fiber density dynamics. (A) Identification of communicating cells in higher temporal resolution of 5 minutes per frame. Same-versus-different pair analysis. All combinations of “same”/“different” were considered. Pairs with a visible band. N = 14 cell pairs. “Same” pair had a higher correlation than “different” pair in all (100%) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (B) Sensitivity of same-versus-different pair analysis to the temporal resolution. The experimental temporal resolution was artificially down-sampled to create multiple time series with reduced time resolutions with different starting times. All possible time intervals under a given temporal resolution (different starting time) were considered leading to multiple recorded observations for a single pair of communicating cells (Methods). Left: Low temporal resolution experiments. Original time resolution was 15 minutes per frame. N = 2 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N = 48 pairs (15/30/45 minutes), N = 43 pairs (60 minutes) and N = 14 pairs (75 minutes). “Same” pair had a higher correlation than “different” pair in 94% (15 minutes), 87% (30 minutes), 77% (45 minutes), 68% (60 minutes) and 67% (75 minutes) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001 for all temporal resolutions. Right: High temporal resolution experiments (5 minutes per frame). N = 3 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N total pairs = 14 pairs (5/10/15 minutes, 100/99/99% “same” > different” correspondingly), N = 13 pairs (20 minutes, 99% “same” > different”), N = 12 pairs (25 minutes, 99% “same” > different”), N = 9 pairs (30/35/40 minutes, 99/98/98% “same” > different” correspondingly), N = 8 pairs (45/50/55 minutes, 98/96/97% correspondingly), N = 7 pairs (60/65/70/75 minutes, 95/92/92/91% “same” > different” correspondingly), n = 6 pairs (80/85 minutes, 91/86% “same” > different” correspondingly). (C) Real-versus-fake for high temporal resolution: N = 13, 100% “real” > “fake”, p-value < 0.0001. (D) Matchmaking analysis for high temporal resolution: N = 28 cells, correct matching probability = 100%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Sensitivity of same-versus-different pair analysis to the temporal resolution. Pair distance 60-150 μm, quantification windows were placed 7.5 μm above the connecting axis between the cells (see Fig. 7E for schematics). Correlations were calculated using the first derivative of fiber density dynamics. (<strong>A</strong>) Identification of communicating cells in higher temporal resolution of 5 minutes per frame. Same-versus-different pair analysis. All combinations of “same”/“different” were considered. Pairs with a visible band. N = 14 cell pairs. “Same” pair had a higher correlation than “different” pair in all (100%) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (<strong>B</strong>) Sensitivity of same-versus-different pair analysis to the temporal resolution. The experimental temporal resolution was artificially down-sampled to create multiple time series with reduced time resolutions with different starting times. All possible time intervals under a given temporal resolution (different starting time) were considered leading to multiple recorded observations for a single pair of communicating cells (Methods). Left: Low temporal resolution experiments. Original time resolution was 15 minutes per frame. N = 2 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N = 48 pairs (15/30/45 minutes), N = 43 pairs (60 minutes) and N = 14 pairs (75 minutes). “Same” pair had a higher correlation than “different” pair in 94% (15 minutes), 87% (30 minutes), 77% (45 minutes), 68% (60 minutes) and 67% (75 minutes) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001 for all temporal resolutions. Right: High temporal resolution experiments (5 minutes per frame). N = 3 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N total pairs = 14 pairs (5/10/15 minutes, 100/99/99% “same” > different” correspondingly), N = 13 pairs (20 minutes, 99% “same” > different”), N = 12 pairs (25 minutes, 99% “same” > different”), N = 9 pairs (30/35/40 minutes, 99/98/98% “same” > different” correspondingly), N = 8 pairs (45/50/55 minutes, 98/96/97% correspondingly), N = 7 pairs (60/65/70/75 minutes, 95/92/92/91% “same” > different” correspondingly), n = 6 pairs (80/85 minutes, 91/86% “same” > different” correspondingly). (<strong>C</strong>) Real-versus-fake for high temporal resolution: N = 13, 100% “real” > “fake”, p-value < 0.0001. (<strong>D</strong>) Matchmaking analysis for high temporal resolution: N = 28 cells, correct matching probability = 100%.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F17.medium.gif" width="440" height="351"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F17.medium.gif" width="440" height="351"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S10:
                               </span>
                               <p id="p-112" class="first-child">
                                Sensitivity of same-versus-different pair analysis to the temporal resolution. Pair distance 60-150 μm, quantification windows were placed 7.5 μm above the connecting axis between the cells (see
                                <a id="xref-fig-7-8" class="xref-fig" href="#F7">
                                 Fig. 7E
                                </a>
                                for schematics). Correlations were calculated using the first derivative of fiber density dynamics. (
                                <strong>
                                 A
                                </strong>
                                ) Identification of communicating cells in higher temporal resolution of 5 minutes per frame. Same-versus-different pair analysis. All combinations of “same”/“different” were considered. Pairs with a visible band. N = 14 cell pairs. “Same” pair had a higher correlation than “different” pair in all (100%) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001. (
                                <strong>
                                 B
                                </strong>
                                ) Sensitivity of same-versus-different pair analysis to the temporal resolution. The experimental temporal resolution was artificially down-sampled to create multiple time series with reduced time resolutions with different starting times. All possible time intervals under a given temporal resolution (different starting time) were considered leading to multiple recorded observations for a single pair of communicating cells (Methods).
                                <span class="underline">
                                 Left
                                </span>
                                : Low temporal resolution experiments. Original time resolution was 15 minutes per frame. N = 2 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N = 48 pairs (15/30/45 minutes), N = 43 pairs (60 minutes) and N = 14 pairs (75 minutes). “Same” pair had a higher correlation than “different” pair in 94% (15 minutes), 87% (30 minutes), 77% (45 minutes), 68% (60 minutes) and 67% (75 minutes) of the matched correlations. Wilcoxon signed-rank p-value < 0.0001 for all temporal resolutions.
                                <span class="underline">
                                 Right
                                </span>
                                : High temporal resolution experiments (5 minutes per frame). N = 3 experiments, same-versus-different pair analysis was performed independently for each experiment and the results were pooled together without mixing pairs from different experiments. N total pairs = 14 pairs (5/10/15 minutes, 100/99/99% “same” > different” correspondingly), N = 13 pairs (20 minutes, 99% “same” > different”), N = 12 pairs (25 minutes, 99% “same” > different”), N = 9 pairs (30/35/40 minutes, 99/98/98% “same” > different” correspondingly), N = 8 pairs (45/50/55 minutes, 98/96/97% correspondingly), N = 7 pairs (60/65/70/75 minutes, 95/92/92/91% “same” > different” correspondingly), n = 6 pairs (80/85 minutes, 91/86% “same” > different” correspondingly). (
                                <strong>
                                 C
                                </strong>
                                ) Real-versus-fake for high temporal resolution: N = 13, 100% “real” > “fake”, p-value < 0.0001. (
                                <strong>
                                 D
                                </strong>
                                ) Matchmaking analysis for high temporal resolution: N = 28 cells, correct matching probability = 100%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-27">
                              Altogether, we defined three computational measurements providing us with systematic means to decipher cell-ECM-cell communication: (1) Same-versus-different – quantifying the ability to distinguish between ECM remolding of one pair of communicating cells from a different pair of communicating cells, (2) Real-versus-fake – quantifying the confounding factor of non-communication-related local deformations in the ECM, and (3) Matchmaking – quantitative identification of unique ECM remodeling signatures of communication partners. True communication is characterized by high values in all three measurements.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Experimental controls validate ECM remodeling correlations as a measurement for cell-ECM-cell communication
                             </h3>
                             <p id="p-28">
                              As negative controls we included a new set of experiments. Severe reduction in all three communication readouts was measured between pairs of fluorescent beads in size comparable to cells, and in experiments with dead cells (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6A-B
                              </a>
                              ), compared to live fibroblasts (
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Figs. 4
                              </a>
                              -
                              <a id="xref-fig-5-8" class="xref-fig" href="#F5">
                               5
                              </a>
                              ) and cancer cells (
                              <a id="xref-fig-16-2" class="xref-fig" href="#F16">
                               Fig. S9
                              </a>
                              ). To include background ECM remodeling resulting from contraction of other live cells, we co-cultured live and dead cells within the same gel. In the presence of live cells, no communication was measured between pairs of dead cells (
                              <a id="xref-fig-6-2" class="xref-fig" href="#F6">
                               Fig. 6C
                              </a>
                              ) and between live-dead cell pairs (
                              <a id="xref-fig-6-3" class="xref-fig" href="#F6">
                               Fig. 6D
                              </a>
                              ) - a validation that our measurement for cell-ECM-cell communication is robust to local ECM deformations originating from contractile activity of the two partners and not from other cells in the gel or remote sensing of physical boundaries through the ECM (e.g.,
                              <sup>
                               <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                                32
                               </a>
                              </sup>
                              ) correspondingly. Summary of all measurements over all experimental conditions are reported in Table S1.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F6.large.jpg?width=800&height=600&carousel=1" title="Experimental controls measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (A) Experiments with fluorescent beads. Same-versus-different: N = 25, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 25, 52% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 50, correct matching probability = 14%. (B-D) Experimental controls with live and dead fibroblast cells. (B) Dead cells. Same-versus-different: N = 35, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 29, 60% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 70, correct matching probability = 17%. (C) Dead cells with the presence of live cells. Same-versus-different: N = 17, 55% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 17, 54% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 34, correct matching probability = 15%. (D) Live and dead cells. Same-versus-different: N = 17, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 11, 62% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 22, correct matching probability = 14%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Experimental controls measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (<strong>A</strong>) Experiments with fluorescent beads. Same-versus-different: N = 25, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 25, 52% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 50, correct matching probability = 14%. (<strong>B-D</strong>) Experimental controls with live and dead fibroblast cells. (<strong>B</strong>) Dead cells. Same-versus-different: N = 35, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 29, 60% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 70, correct matching probability = 17%. (<strong>C</strong>) Dead cells with the presence of live cells. Same-versus-different: N = 17, 55% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 17, 54% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 34, correct matching probability = 15%. (<strong>D</strong>) Live and dead cells. Same-versus-different: N = 17, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 11, 62% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 22, correct matching probability = 14%.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F6.medium.gif" width="339" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F6.medium.gif" width="339" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6.
                               </span>
                               <p id="p-29" class="first-child">
                                Experimental controls measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (
                                <strong>
                                 A
                                </strong>
                                ) Experiments with fluorescent beads. Same-versus-different: N = 25, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 25, 52% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 50, correct matching probability = 14%. (
                                <strong>
                                 B-D
                                </strong>
                                ) Experimental controls with live and dead fibroblast cells. (
                                <strong>
                                 B
                                </strong>
                                ) Dead cells. Same-versus-different: N = 35, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 29, 60% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 70, correct matching probability = 17%. (
                                <strong>
                                 C
                                </strong>
                                ) Dead cells with the presence of live cells. Same-versus-different: N = 17, 55% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 17, 54% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 34, correct matching probability = 15%. (
                                <strong>
                                 D
                                </strong>
                                ) Live and dead cells. Same-versus-different: N = 17, 61% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 11, 62% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 22, correct matching probability = 14%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Decoupling band formation and communication sensing
                             </h3>
                             <p id="p-30">
                              Our finding that cell-ECM-cell communication can be identified quantitatively only when shifting the quantification window away from the connecting axis between the cell centers, encouraged us to systematically measure the relations between cell-ECM-cell communication and band formation in experiments. We analyzed different ECM quantification windows in the 3D space around the cell by shifting them laterally (perpendicular, in the XY axis) and axially (up and down, in the Z-axis) in relation to the connecting axis (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 7A
                              </a>
                              , Methods). For each shifted window we measured the final normalized fiber density after 255 minutes and also, independently, performed same-versus-different pair analysis for the corresponding windows over time. While the fiber density was maximal along the connecting axis, the discrimination between different pairs of communicating cells was optimized above or below (axially) the connecting axis (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 7B-D
                              </a>
                              ), and excluded the possibility that this was an artifact of saturated pixels along the dense fibrous band (
                              <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                               Fig. S11
                              </a>
                              ). These results were consistent for faster imaging experiments (
                              <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                               Fig. S12
                              </a>
                              ) establishing that communication sensing, i.e., our method’s ability to identify cell-ECM-cell communication, is decoupled from band formation.
                             </p>
                             <div id="F18" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F18.large.jpg?width=800&height=600&carousel=1" title="Association between cell pair correlations and the fraction of saturated pixels in the region used to measure cell-cell communication. For both panels, pair distances range between 4 and 10 cell diameters (60-150 μm, assuming mean fibroblast diameter of 15 μm). Correlations were calculated using the first derivative of fiber density dynamics. (A) Saturation-independent correlations along the connecting axis between the cells (Pearson correlation coefficient p-value > 0.05), N = 42 cell pairs. Image saturation does not hamper the ability to identify cell-ECM-cell communication along the dense fibrous band. (B) Image saturation slightly reduces the ability to identify cell-ECM-cell communication 0.5 cell diameters (7.5 μm) above the connecting axis between the cells. N = 48 cell pairs, Pearson correlation coefficient = -0.18, p-value < 0.0001." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Association between cell pair correlations and the fraction of saturated pixels in the region used to measure cell-cell communication. For both panels, pair distances range between 4 and 10 cell diameters (60-150 μm, assuming mean fibroblast diameter of 15 μm). Correlations were calculated using the first derivative of fiber density dynamics. (<strong>A</strong>) Saturation-independent correlations along the connecting axis between the cells (Pearson correlation coefficient p-value > 0.05), N = 42 cell pairs. Image saturation does not hamper the ability to identify cell-ECM-cell communication along the dense fibrous band. (<strong>B</strong>) Image saturation slightly reduces the ability to identify cell-ECM-cell communication 0.5 cell diameters (7.5 μm) above the connecting axis between the cells. N = 48 cell pairs, Pearson correlation coefficient = -0.18, p-value < 0.0001.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F18.medium.gif" width="440" height="194"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S11:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F18.medium.gif" width="440" height="194"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S11:
                               </span>
                               <p id="p-113" class="first-child">
                                Association between cell pair correlations and the fraction of saturated pixels in the region used to measure cell-cell communication. For both panels, pair distances range between 4 and 10 cell diameters (60-150 μm, assuming mean fibroblast diameter of 15 μm). Correlations were calculated using the first derivative of fiber density dynamics. (
                                <strong>
                                 A
                                </strong>
                                ) Saturation-independent correlations along the connecting axis between the cells (Pearson correlation coefficient p-value > 0.05), N = 42 cell pairs. Image saturation does not hamper the ability to identify cell-ECM-cell communication along the dense fibrous band. (
                                <strong>
                                 B
                                </strong>
                                ) Image saturation slightly reduces the ability to identify cell-ECM-cell communication 0.5 cell diameters (7.5 μm) above the connecting axis between the cells. N = 48 cell pairs, Pearson correlation coefficient = -0.18, p-value < 0.0001.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F19" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F19.large.jpg?width=800&height=600&carousel=1" title="Spatial decoupling of band formation and cell-ECM-cell communication in cell pairs with a visible band imaged in high temporal resolution of 5 minutes per frame. Pair distance of 60-150 μm. (A) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (B) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Spatial decoupling of band formation and cell-ECM-cell communication in cell pairs with a visible band imaged in high temporal resolution of 5 minutes per frame. Pair distance of 60-150 μm. (<strong>A</strong>) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (<strong>B</strong>) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F19.medium.gif" width="440" height="189"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S12:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F19.medium.gif" width="440" height="189"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S12:
                               </span>
                               <p id="p-114" class="first-child">
                                Spatial decoupling of band formation and cell-ECM-cell communication in cell pairs with a visible band imaged in high temporal resolution of 5 minutes per frame. Pair distance of 60-150 μm. (
                                <strong>
                                 A
                                </strong>
                                ) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (
                                <strong>
                                 B
                                </strong>
                                ) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F7.large.jpg?width=800&height=600&carousel=1" title="Spatial decoupling of band formation and cell-ECM-cell communication. (A) Cell pair axes schematic sketch. Z-axis (cyan) and XY-axis (purple) are perpendicular to each other and to the connecting axis between the cells’ centers (black). (B-D) Mean fiber density and cell-ECM-cell communication (same-versus-different analysis) for shifted quantification windows in cell pairs with a visible band. Pair distances of 60-150 μm (∼4-10, cell diameters). The dashed vertical line in panels B-C is used as a line profile in panel D. (B) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (C) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05. (D) Fiber density and cell-ECM-cell communication along the axial line-scan (offset in XY axis = 0). Peaks in cell-ECM-cell communication appear above (and below) the connecting axis between the cells, where fiber density is maximal on the connecting axis. (E) Schematic sketch of the optimal quantification window location to quantify cell-ECM-cell communication. Offset in Z axis = 0.5. Offset in XY axis = 0. (F-I) Cell-ECM-cell communication for cell pairs with no visible band. Quantification windows were placed 7.5 μm (∼0.5 cell diameter) above the connecting axis between the cells. (F) Representative cell pair with no visible band at the onset (“begin”) and after (“end”) 255 minutes of cell imaging. Scale bar = 15 μm. (G) Quantification of the dynamics of the first derivative of ECM intensity in the cell pair in panel F implicating the absence of a band. Left/right - quantification windows adjacent to cell boundaries, “middle” - quantification window in between the cells centers at each time frame. Pearson correlation coefficient between left and right cells inner regions = 0.7, p-value < 0.0001. (H-I) Cell pairs with no visible band with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (H) Standard time resolution of 15 minutes per frame. Same-versus-different: N = 14, 80% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 15, 70% “real” > “fake”, p-value < 0.01. Matchmaking analysis: N = 28, correct matching probability = 46%. (I) Rapid time resolution of 5 minutes per frame. Same-versus-different: N = 56, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 56, 94% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 112, correct matching probability = 87%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Spatial decoupling of band formation and cell-ECM-cell communication. (<strong>A</strong>) Cell pair axes schematic sketch. Z-axis (cyan) and XY-axis (purple) are perpendicular to each other and to the connecting axis between the cells’ centers (black). (<strong>B-D</strong>) Mean fiber density and cell-ECM-cell communication (same-versus-different analysis) for shifted quantification windows in cell pairs with a visible band. Pair distances of 60-150 μm (∼4-10, cell diameters). The dashed vertical line in panels B-C is used as a line profile in panel D. (<strong>B</strong>) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (<strong>C</strong>) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05. (<strong>D</strong>) Fiber density and cell-ECM-cell communication along the axial line-scan (offset in XY axis = 0). Peaks in cell-ECM-cell communication appear above (and below) the connecting axis between the cells, where fiber density is maximal on the connecting axis. (<strong>E</strong>) Schematic sketch of the optimal quantification window location to quantify cell-ECM-cell communication. Offset in Z axis = 0.5. Offset in XY axis = 0. (<strong>F-I</strong>) Cell-ECM-cell communication for cell pairs with no visible band. Quantification windows were placed 7.5 μm (∼0.5 cell diameter) above the connecting axis between the cells. (<strong>F</strong>) Representative cell pair with no visible band at the onset (“begin”) and after (“end”) 255 minutes of cell imaging. Scale bar = 15 μm. (<strong>G</strong>) Quantification of the dynamics of the first derivative of ECM intensity in the cell pair in panel F implicating the absence of a band. Left/right - quantification windows adjacent to cell boundaries, “middle” - quantification window in between the cells centers at each time frame. Pearson correlation coefficient between left and right cells inner regions = 0.7, p-value < 0.0001. (<strong>H-I</strong>) Cell pairs with no visible band with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (<strong>H</strong>) Standard time resolution of 15 minutes per frame. Same-versus-different: N = 14, 80% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 15, 70% “real” > “fake”, p-value < 0.01. Matchmaking analysis: N = 28, correct matching probability = 46%. (<strong>I</strong>) Rapid time resolution of 5 minutes per frame. Same-versus-different: N = 56, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 56, 94% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 112, correct matching probability = 87%.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F7.medium.gif" width="316" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7." src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F7.medium.gif" width="316" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7." data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7.
                               </span>
                               <p id="p-31" class="first-child">
                                Spatial decoupling of band formation and cell-ECM-cell communication. (
                                <strong>
                                 A
                                </strong>
                                ) Cell pair axes schematic sketch. Z-axis (cyan) and XY-axis (purple) are perpendicular to each other and to the connecting axis between the cells’ centers (black). (
                                <strong>
                                 B-D
                                </strong>
                                ) Mean fiber density and cell-ECM-cell communication (same-versus-different analysis) for shifted quantification windows in cell pairs with a visible band. Pair distances of 60-150 μm (∼4-10, cell diameters). The dashed vertical line in panels B-C is used as a line profile in panel D. (
                                <strong>
                                 B
                                </strong>
                                ) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (
                                <strong>
                                 C
                                </strong>
                                ) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05. (
                                <strong>
                                 D
                                </strong>
                                ) Fiber density and cell-ECM-cell communication along the axial line-scan (offset in XY axis = 0). Peaks in cell-ECM-cell communication appear above (and below) the connecting axis between the cells, where fiber density is maximal on the connecting axis. (
                                <strong>
                                 E
                                </strong>
                                ) Schematic sketch of the optimal quantification window location to quantify cell-ECM-cell communication. Offset in Z axis = 0.5. Offset in XY axis = 0. (
                                <strong>
                                 F-I
                                </strong>
                                ) Cell-ECM-cell communication for cell pairs with no visible band. Quantification windows were placed 7.5 μm (∼0.5 cell diameter) above the connecting axis between the cells. (
                                <strong>
                                 F
                                </strong>
                                ) Representative cell pair with no visible band at the onset (“begin”) and after (“end”) 255 minutes of cell imaging. Scale bar = 15 μm. (
                                <strong>
                                 G
                                </strong>
                                ) Quantification of the dynamics of the first derivative of ECM intensity in the cell pair in panel F implicating the absence of a band. Left/right - quantification windows adjacent to cell boundaries, “middle” - quantification window in between the cells centers at each time frame. Pearson correlation coefficient between left and right cells inner regions = 0.7, p-value < 0.0001. (
                                <strong>
                                 H-I
                                </strong>
                                ) Cell pairs with no visible band with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (
                                <strong>
                                 H
                                </strong>
                                ) Standard time resolution of 15 minutes per frame. Same-versus-different: N = 14, 80% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 15, 70% “real” > “fake”, p-value < 0.01. Matchmaking analysis: N = 28, correct matching probability = 46%. (
                                <strong>
                                 I
                                </strong>
                                ) Rapid time resolution of 5 minutes per frame. Same-versus-different: N = 56, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 56, 94% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 112, correct matching probability = 87%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-32">
                              Previous studies analyzed the intercellular band as a measure of mechanical communication. The decoupling between band formation and communication sensing led us to examine whether our method can measure communication for cell pairs that did not form a visible band between the cells (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 7E-G
                              </a>
                              ). Indeed, same-versus-different, real-versus-fake and matchmaking analyses identified cell-ECM-cell communication for band-less pairs, slightly above or below the connecting axis, implying that communication is present even when a band is not visible to the naked-eye (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Fig. 7H
                              </a>
                              ,
                              <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                               Fig. S13
                              </a>
                              , Video S6). Faster imaging identified more prominently cell-ECM-cell communication for cell pairs without a visible band (
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Fig. 7I
                              </a>
                              ).
                             </p>
                             <div id="F20" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F20.large.jpg?width=800&height=600&carousel=1" title="Spatial decoupling of band formation and cell-ECM-cell communication in pairs with no visible band imaged in high temporal resolution of 5 minutes. Pair distance of 60-150 μm. (A) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (B) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “Same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Spatial decoupling of band formation and cell-ECM-cell communication in pairs with no visible band imaged in high temporal resolution of 5 minutes. Pair distance of 60-150 μm. (<strong>A</strong>) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (<strong>B</strong>) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “Same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S13:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F20.medium.gif" width="440" height="189"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S13:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F20.medium.gif" width="440" height="189"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S13:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S13:
                               </span>
                               <p id="p-115" class="first-child">
                                Spatial decoupling of band formation and cell-ECM-cell communication in pairs with no visible band imaged in high temporal resolution of 5 minutes. Pair distance of 60-150 μm. (
                                <strong>
                                 A
                                </strong>
                                ) Fiber density. Mean fiber density for systematic offsets in Z and XY axes. (
                                <strong>
                                 B
                                </strong>
                                ) Cell-ECM-cell communication. Mean fraction of higher “same”, correlation between communicating pairs, versus “different”, correlation between one cell from that pair and another cell from a different communicating pair for systematic offsets in Z and XY axes. Red ‘x’ marked that the null hypothesis that “Same” - “different” correlations are distributed around zero was not rejected with p-value ≤ 0.05.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-33">
                              We conclude that our method’s ability to measure communication is improved when correlating quantification windows located slightly away from the connecting axis in-between the communicating cells, while the densest fiber band is formed directly along the connecting axis and that cell-ECM-cell communication exists even when no visible band is formed between the cell pair. These results challenge the current notion that band formation is the hallmark of cell-cell communication in fibrous gels.
                             </p>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Higher signal-to-noise ratio off the connecting axis supports sensitive measurement of cell-ECM-cell communication
                             </h3>
                             <p id="p-34">
                              We hypothesized that cell-ECM-cell communication is best measured slightly away (in the Z-axis) from the connecting axis between the cells because the higher ECM intensities along the fibrous band confound our ability to sensitively measure the ECM fluctuations as a signature of cell-ECM-cell communication. To test this hypothesis, we calculated the association between fiber density and the sensitivity in measuring communication via same-versus-different pair analysis by correlating these parameters across different locations of the quantification window. This analysis demonstrated a sweet spot for measuring cell-ECM-cell communication (
                              <a id="xref-fig-21-1" class="xref-fig" href="#F21">
                               Fig. S14A
                              </a>
                              ). Regions with low fiber density, that were mostly located away from the connecting axis, as well as denser band regions, had low same-versus-different discrimination. Analysis of the temporal change in fiber density revealed that while the band intensity is maximal along the connecting axis between the cells, the magnitude of the changes in ECM intensity is similar along the connecting axis and slightly above it (
                              <a id="xref-fig-21-2" class="xref-fig" href="#F21">
                               Fig. S14B
                              </a>
                              ). Because the communicating signal is measured as the change in ECM intensity relative to the background intensity (
                              <a id="xref-fig-4-9" class="xref-fig" href="#F4">
                               Fig. 4C
                              </a>
                              ), the ‘signal to noise ratio’ of the measurable communication signal is higher off the connecting axis, making it a more sensitive measurement for cell-ECM-cell communication.
                             </p>
                             <div id="F21" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F21.large.jpg?width=800&height=600&carousel=1" title="Higher ECM intensities along the fibrous band confound our ability to sensitively measure the ECM fluctuations as a signature of cell-ECM-cell communication. (A) Association between fiber density and our ability to identify cell-ECM-cell communication with same-versus-different pair analysis. Each data point represents the mean fiber density in the quantification window (x-axis) and the fraction of cell pairs with “same” correlation higher than “different” correlation (y-axis) for different locations of the quantification window. The data here was derived from matched bins in Fig. 7B (x-axis) and Fig. 7C (y-axis) in a region that spans 1 cell diameter away (15 µm) from the quantification plane defined adjacent to the cells in the connecting axis between the communicating cells: all bins in the range -1≤XY≤1 and - 1≤Z≤1. Pearson correlation coefficient = -0.26, p-value < 0.0001. Note the stiff drop in cell-ECM-cell communication around fiber density of ∼3-4 z-scores. (B) The band intensity is maximal along the connecting axis between the cells. The magnitude of the changes in ECM intensity is similar along the connecting axis and slightly above it, where communication is optimal. Top: imaging at temporal resolution of 15 minutes per frame (left-most and right most are duplication of Fig. 7B and Fig. 7C correspondingly), bottom: imaging at temporal resolution of 5 minutes per frame (left-most and right most are duplication of Fig. S12A and Fig. S12B correspondingly)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Higher ECM intensities along the fibrous band confound our ability to sensitively measure the ECM fluctuations as a signature of cell-ECM-cell communication. (<strong>A</strong>) Association between fiber density and our ability to identify cell-ECM-cell communication with same-versus-different pair analysis. Each data point represents the mean fiber density in the quantification window (x-axis) and the fraction of cell pairs with “same” correlation higher than “different” correlation (y-axis) for different locations of the quantification window. The data here was derived from matched bins in Fig. 7B (x-axis) and Fig. 7C (y-axis) in a region that spans 1 cell diameter away (15 µm) from the quantification plane defined adjacent to the cells in the connecting axis between the communicating cells: all bins in the range -1≤XY≤1 and - 1≤Z≤1. Pearson correlation coefficient = -0.26, p-value < 0.0001. Note the stiff drop in cell-ECM-cell communication around fiber density of ∼3-4 z-scores. (<strong>B</strong>) The band intensity is maximal along the connecting axis between the cells. The magnitude of the changes in ECM intensity is similar along the connecting axis and slightly above it, where communication is optimal. Top: imaging at temporal resolution of 15 minutes per frame (left-most and right most are duplication of Fig. 7B and Fig. 7C correspondingly), bottom: imaging at temporal resolution of 5 minutes per frame (left-most and right most are duplication of Fig. S12A and Fig. S12B correspondingly).</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S14:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F21.medium.gif" width="386" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S14:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F21.medium.gif" width="386" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S14:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S14:
                               </span>
                               <p id="p-116" class="first-child">
                                Higher ECM intensities along the fibrous band confound our ability to sensitively measure the ECM fluctuations as a signature of cell-ECM-cell communication. (
                                <strong>
                                 A
                                </strong>
                                ) Association between fiber density and our ability to identify cell-ECM-cell communication with same-versus-different pair analysis. Each data point represents the mean fiber density in the quantification window (x-axis) and the fraction of cell pairs with “same” correlation higher than “different” correlation (y-axis) for different locations of the quantification window. The data here was derived from matched bins in
                                <a id="xref-fig-7-9" class="xref-fig" href="#F7">
                                 Fig. 7B
                                </a>
                                (x-axis) and
                                <a id="xref-fig-7-10" class="xref-fig" href="#F7">
                                 Fig. 7C
                                </a>
                                (y-axis) in a region that spans 1 cell diameter away (15 µm) from the quantification plane defined adjacent to the cells in the connecting axis between the communicating cells: all bins in the range -1≤XY≤1 and - 1≤Z≤1. Pearson correlation coefficient = -0.26, p-value < 0.0001. Note the stiff drop in cell-ECM-cell communication around fiber density of ∼3-4 z-scores. (
                                <strong>
                                 B
                                </strong>
                                ) The band intensity is maximal along the connecting axis between the cells. The magnitude of the changes in ECM intensity is similar along the connecting axis and slightly above it, where communication is optimal. Top: imaging at temporal resolution of 15 minutes per frame (left-most and right most are duplication of
                                <a id="xref-fig-7-11" class="xref-fig" href="#F7">
                                 Fig. 7B
                                </a>
                                and
                                <a id="xref-fig-7-12" class="xref-fig" href="#F7">
                                 Fig. 7C
                                </a>
                                correspondingly), bottom: imaging at temporal resolution of 5 minutes per frame (left-most and right most are duplication of
                                <a id="xref-fig-19-2" class="xref-fig" href="#F19">
                                 Fig. S12A
                                </a>
                                and
                                <a id="xref-fig-19-3" class="xref-fig" href="#F19">
                                 Fig. S12B
                                </a>
                                correspondingly).
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              The role of Myosin II contractility on cell-ECM-cell communication
                             </h3>
                             <p id="p-35">
                              Given that we could measure cell-ECM-cell communication even without a visible band forming between the cells, and considering the fact that this type of long-range communication is inherently mechanical, we next asked what is the role of contractility in cell-ECM-cell communication. Contractility inhibition with a dosage of 85 nM of Blebbistatin, a Myosin II inhibitor, revealed that cell-ECM-cell communication can be measured even when contractility is partially inhibited and a visible band between the cells practically never forms (
                              <a id="xref-fig-22-1" class="xref-fig" href="#F22">
                               Fig. S15A
                              </a>
                              ). Further increasing the Blebbistatin dosage to 150 nM verified that contractility is required for this mode of mechanical communication (
                              <a id="xref-fig-22-2" class="xref-fig" href="#F22">
                               Fig. S15B
                              </a>
                              ). Cumulatively, these results suggest that mechanical signals propagate from one cell to another even upon massive reduction in their contractility and thus cells can mechanically communicate even with reduced contractility levels that are not sufficient to form a visible band.
                             </p>
                             <div id="F22" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F22.large.jpg?width=800&height=600&carousel=1" title="Contractility inhibition experiments measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (A) 85 nM Blebbistatin. Same-versus-different: N = 42, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 42, 90% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 84, correct matching probability = 79%. (B) 150 nM Blebbistatin. Same-versus-different: N = 34, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 34, 50% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 68, correct matching probability = 13%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Contractility inhibition experiments measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (<strong>A</strong>) 85 nM Blebbistatin. Same-versus-different: N = 42, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 42, 90% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 84, correct matching probability = 79%. (<strong>B</strong>) 150 nM Blebbistatin. Same-versus-different: N = 34, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 34, 50% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 68, correct matching probability = 13%.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S15:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F22.medium.gif" width="440" height="315"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S15:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F22.medium.gif" width="440" height="315"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S15:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S15:
                               </span>
                               <p id="p-117" class="first-child">
                                Contractility inhibition experiments measured with same-versus-different analysis (first column), real-versus-fake analysis (middle column), and match making analysis (right column). (
                                <strong>
                                 A
                                </strong>
                                ) 85 nM Blebbistatin. Same-versus-different: N = 42, 95% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 42, 90% “real” > “fake”, p-value < 0.0001. Matchmaking analysis: N = 84, correct matching probability = 79%. (
                                <strong>
                                 B
                                </strong>
                                ) 150 nM Blebbistatin. Same-versus-different: N = 34, 66% “same” > “different, p-value < 0.0001. Real-versus-fake: N = 34, 50% “real” > “fake”, p-value not significant. Matchmaking analysis: N = 68, correct matching probability = 13%.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-11" class="subsection">
                             <h3>
                              Identifying leader and follower in pairs of simulated communicating cells
                             </h3>
                             <p id="p-36">
                              We next asked if we could use our approach to quantify asymmetric interactions between the communicating partners. More specifically, can we identify which cell in a communicating pair is more dominant or influential? Our quantitative interpretation of “influential” is that past ECM-remodeling fluctuations of one cell are predictive of the future ECM-remodeling of its communicating partner. Such temporal order defines a leader-follower relation. We performed simulations where a “follower” cell “imitates” the previous contraction of its influential “leader cell”. In other words, the contraction of both cells are determined by the leader cell with a time lag of one step (
                              <a id="xref-fig-23-1" class="xref-fig" href="#F23">
                               Fig. S16A
                              </a>
                              ). The simplest approach to quantitatively identify a temporal order is by cross-correlation with time lags, where the correlation between two time-series is calculated for a given lag, and the time-lag that leads to the maximal correlation determines the temporal order. This analysis successfully identified which simulated cell was the leader and which one was the follower with an accurate lag time (
                              <a id="xref-fig-23-2" class="xref-fig" href="#F23">
                               Fig. S16B
                              </a>
                              ,
                              <a id="xref-fig-24-1" class="xref-fig" href="#F24">
                               Fig. S17A
                              </a>
                              ). To evaluate the robustness of using correlations to identify leader-follower relations we simulated cell pairs where the follower contraction was composed of an independent component, and a component that is dependent on its leader: Contraction
                              <sub>
                               follower
                              </sub>
                              (t) = (1 – α) * N(μ,σ) + α * Contraction
                              <sub>
                               leader
                              </sub>
                              (t – 1), where t > 1, 0 ≤ α ≤ 1, μ = 1, σ = 0.5 are the mean and standard deviation correspondingly, and Contraction
                              <sub>
                               leader
                              </sub>
                              is drawn from the normal distribution N(μ,σ). The term α indicates the “followership” magnitude, higher values of α imply that the follower is more influenced by its leader contraction. The correlation increased with increased influence of the leader cells (
                              <a id="xref-fig-23-3" class="xref-fig" href="#F23">
                               Fig. S16C
                              </a>
                              ) and the maximal cross correlation occurred at the correct lag, accurately identifying the leader/follower roles for all simulated pairs for α = 0.5, where the follower cell contraction is determined with equal contribution from its intrinsic “decision” and the extrinsic influence by its leader (
                              <a id="xref-fig-23-4" class="xref-fig" href="#F23">
                               Fig. S16D
                              </a>
                              ,
                              <a id="xref-fig-24-2" class="xref-fig" href="#F24">
                               Fig. S17B
                              </a>
                              ). In a second validation we tested whether we can identify leader/follower when the follower contracts more than its leader. This leads to increased ECM remodeling that might propagate to the leader cell and mask its influence on the follower. We set α to 1 (follower is copycatting the leader’s contraction), and introduced β ≥ 1 - the fold increase of the follower’s contraction: Contraction
                              <sub>
                               follower
                              </sub>
                              (t) = β * Contraction
                              <sub>
                               leader
                              </sub>
                              (t – 1). The correlation was nearly identical for β ≤ (20% increase in follower contraction), and the first mistaken prediction of the follower/follower assignment occurred for β = 1.2, for 1 out of 7 simulated pairs (
                              <a id="xref-fig-23-5" class="xref-fig" href="#F23">
                               Fig. S16E
                              </a>
                              ,
                              <a id="xref-fig-24-3" class="xref-fig" href="#F24">
                               Fig. S17C
                              </a>
                              ). We further validated these results by pairing leaders or followers to cells from other simulated communicating pairs to create artificial pairs of cells that did not interact with one another (
                              <a id="xref-fig-25-1" class="xref-fig" href="#F25">
                               Fig. S18
                              </a>
                              ). However, cross-correlation analysis did not identify leader-follower relations in experimental data (
                              <a id="xref-fig-26-1" class="xref-fig" href="#F26">
                               Fig. S19
                              </a>
                              ). This inability to identify leader-follower relations in experimental data could be attributed to lack of sufficient temporal resolution – the response of the follower cell may occur in time scales faster than the 5 minutes temporal resolution in this study. Another alternative is that our method is not sufficiently sensitive to measure the subtle ECM fluctuations that distinguish leader from follower cells. There is always the possibility that fibroblast cells do not form leader-follower communication patterns, perhaps due to insufficient heterogeneity that can be resolved by assessment of cells pairs from mixed genetic backgrounds. These possibilities are left to be explored in future studies. Cumulatively, these results verified that cross-correlation of ECM remodeling fluctuations can robustly identify leader and follower cells in simulated communicating cells but not in our experimental data.
                             </p>
                             <div id="F23" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F23.large.jpg?width=800&height=600&carousel=1" title="Identifying leader-follower relation in communicating cell pairs. (A) Schematic sketch of the simulation. Right cell (red, follower) mimics the contraction of the left cell (purple, leader) at the previous time point (Methods). (B-E) Simulated cell pair cross correlation for different time lags. N = 13 cell pairs. Pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (B) Distribution of cell pair correlations as a function of the time lag. (C-D) Identifying leaders and followers as a function of varying values of α. Increased α values imply an increased contribution of the extrinsic component - the influence by the leader cell. (C) Distribution of cell pair correlation with time lag = 1 as a function of α values. (D) Fraction of cell pairs where the leader (time lag 1 or 2), follower (time lag -2 or -1) or none (time lag 0) lead to maximal cell pair correlation, and is thus identified as leader, as a function of α values. (E) Distribution of cell pair correlation with time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Identifying leader-follower relation in communicating cell pairs. (<strong>A</strong>) Schematic sketch of the simulation. Right cell (red, follower) mimics the contraction of the left cell (purple, leader) at the previous time point (Methods). (<strong>B-E</strong>) Simulated cell pair cross correlation for different time lags. N = 13 cell pairs. Pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (<strong>B</strong>) Distribution of cell pair correlations as a function of the time lag. (<strong>C-D</strong>) Identifying leaders and followers as a function of varying values of α. Increased α values imply an increased contribution of the extrinsic component - the influence by the leader cell. (<strong>C</strong>) Distribution of cell pair correlation with time lag = 1 as a function of α values. (<strong>D</strong>) Fraction of cell pairs where the leader (time lag 1 or 2), follower (time lag -2 or -1) or none (time lag 0) lead to maximal cell pair correlation, and is thus identified as leader, as a function of α values. (<strong>E</strong>) Distribution of cell pair correlation with time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S16:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F23.medium.gif" width="440" height="360"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S16:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F23.medium.gif" width="440" height="360"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S16:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S16:
                               </span>
                               <p id="p-118" class="first-child">
                                Identifying leader-follower relation in communicating cell pairs. (
                                <strong>
                                 A
                                </strong>
                                ) Schematic sketch of the simulation. Right cell (red, follower) mimics the contraction of the left cell (purple, leader) at the previous time point (Methods). (
                                <strong>
                                 B-E
                                </strong>
                                ) Simulated cell pair cross correlation for different time lags. N = 13 cell pairs. Pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (
                                <strong>
                                 B
                                </strong>
                                ) Distribution of cell pair correlations as a function of the time lag. (
                                <strong>
                                 C-D
                                </strong>
                                ) Identifying leaders and followers as a function of varying values of α. Increased α values imply an increased contribution of the extrinsic component - the influence by the leader cell. (
                                <strong>
                                 C
                                </strong>
                                ) Distribution of cell pair correlation with time lag = 1 as a function of α values. (
                                <strong>
                                 D
                                </strong>
                                ) Fraction of cell pairs where the leader (time lag 1 or 2), follower (time lag -2 or -1) or none (time lag 0) lead to maximal cell pair correlation, and is thus identified as leader, as a function of α values. (
                                <strong>
                                 E
                                </strong>
                                ) Distribution of cell pair correlation with time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F24" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F24.large.jpg?width=800&height=600&carousel=1" title="Leader-follower lag time analysis in simulated cell pairs. Cell pair cross correlation for different time lags. N = 13 cell pairs. Pairs distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (A) Distribution of time lags that lead to maximal cell pair correlation. (B) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal cell pair correlation as a function of α values. Increased α implies an increased contribution of the extrinsic component - the influence by the leader cell. Note that α = 0 implies no leader effect and thus the fraction is expected to be random. (C) Fraction of cell pairs where the simulated time lag leads to maximal cell pair correlation as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Leader-follower lag time analysis in simulated cell pairs. Cell pair cross correlation for different time lags. N = 13 cell pairs. Pairs distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (<strong>A</strong>) Distribution of time lags that lead to maximal cell pair correlation. (<strong>B</strong>) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal cell pair correlation as a function of α values. Increased α implies an increased contribution of the extrinsic component - the influence by the leader cell. Note that α = 0 implies no leader effect and thus the fraction is expected to be random. (<strong>C</strong>) Fraction of cell pairs where the simulated time lag leads to maximal cell pair correlation as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S17:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F24.medium.gif" width="440" height="145"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S17:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F24.medium.gif" width="440" height="145"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S17:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S17:
                               </span>
                               <p id="p-119" class="first-child">
                                Leader-follower lag time analysis in simulated cell pairs. Cell pair cross correlation for different time lags. N = 13 cell pairs. Pairs distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contraction follower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. Time lag is applied to the follower’s time series before correlation is calculated. (
                                <strong>
                                 A
                                </strong>
                                ) Distribution of time lags that lead to maximal cell pair correlation. (
                                <strong>
                                 B
                                </strong>
                                ) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal cell pair correlation as a function of α values. Increased α implies an increased contribution of the extrinsic component - the influence by the leader cell. Note that α = 0 implies no leader effect and thus the fraction is expected to be random. (
                                <strong>
                                 C
                                </strong>
                                ) Fraction of cell pairs where the simulated time lag leads to maximal cell pair correlation as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F25" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F25.large.jpg?width=800&height=600&carousel=1" title="Cross correlation analysis of artificial pairing of simulated leader/follower cells that did not interact with one another as a control. Cross correlation for different time lags between a cell from one communicating pair and another cell from a different communicating pair. N = 13 cell pairs, pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contractionfollower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. The time lag is applied to the follower’s time series before the correlation was calculated. (A) Distribution of correlations as a function of the time lag. (B) Distribution of time lags that lead to maximal correlation. (C) Distribution of the correlation for time lag = 1 as a function of α values. Increased α implies an increased contribution of the extrinsic component for the communicating cell paris before the artificial pairing to cells that never communicated with one another. (D) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal correlation as a function of α values. (E) Distribution of the correlation for time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1. (F) Fraction of cell pairs where the simulated time lag leads to maximal correlation as a function of β values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Cross correlation analysis of artificial pairing of simulated leader/follower cells that did not interact with one another as a control. Cross correlation for different time lags between a cell from one communicating pair and another cell from a different communicating pair. N = 13 cell pairs, pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contractionfollower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. The time lag is applied to the follower’s time series before the correlation was calculated. (<strong>A</strong>) Distribution of correlations as a function of the time lag. (<strong>B</strong>) Distribution of time lags that lead to maximal correlation. (<strong>C</strong>) Distribution of the correlation for time lag = 1 as a function of α values. Increased α implies an increased contribution of the extrinsic component for the communicating cell paris before the artificial pairing to cells that never communicated with one another. (<strong>D</strong>) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal correlation as a function of α values. (<strong>E</strong>) Distribution of the correlation for time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. <em>α</em> value was set to 1. (F) Fraction of cell pairs where the simulated time lag leads to maximal correlation as a function of β values.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S18:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F25.medium.gif" width="294" height="440"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S18:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F25.medium.gif" width="294" height="440"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F25.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S18:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F25.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S18:
                               </span>
                               <p id="p-120" class="first-child">
                                Cross correlation analysis of artificial pairing of simulated leader/follower cells that did not interact with one another as a control. Cross correlation for different time lags between a cell from one communicating pair and another cell from a different communicating pair. N = 13 cell pairs, pair distance = 7 cell diameters. The leader cell draws its contraction in each step independently from a normal distribution with mean of 1% and standard deviation of 0.5%. The follower cell draws its “intrinsic” component (contractionfollower) from the same distribution. The second derivative of the fiber density dynamics was used for correlation. The time lag is applied to the follower’s time series before the correlation was calculated. (
                                <strong>
                                 A
                                </strong>
                                ) Distribution of correlations as a function of the time lag. (
                                <strong>
                                 B
                                </strong>
                                ) Distribution of time lags that lead to maximal correlation. (
                                <strong>
                                 C
                                </strong>
                                ) Distribution of the correlation for time lag = 1 as a function of α values. Increased α implies an increased contribution of the extrinsic component for the communicating cell paris before the artificial pairing to cells that never communicated with one another. (
                                <strong>
                                 D
                                </strong>
                                ) Fraction of cell pairs where the simulated time lag (that was set to one step) leads to maximal correlation as a function of α values. (
                                <strong>
                                 E
                                </strong>
                                ) Distribution of the correlation for time lag = 1 as a function of β values. Increased β implies an increased contraction by the follower cell. α value was set to 1. (F) Fraction of cell pairs where the simulated time lag leads to maximal correlation as a function of β values.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <div id="F26" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F26.large.jpg?width=800&height=600&carousel=1" title="Cross correlation analysis did not identify leader-follower relations in experimental data. N = 19 communicating cell pairs with visible bands. Temporal resolution = 5 minutes between consecutive time frames. Pair distance of 60-150 μm. Quantification windows were placed 7.5 μm above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics. Time lags were recorded with their absolute value (symmetry argument: no preference to a specific cell as in the simulations). (A-B) Cell pair cross correlation for different time lags. (A) Distribution of cell pair correlations as a function of the time lag. (B) Distribution of time lags that lead to maximal cell pair correlation. (C-D) Cross correlation for different time lags between one cell from a communicating pair and another cell from a different communicating pair as a control. (C) Distribution of correlations as a function of the time lag. (D) Distribution of time lags that lead to maximal correlation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Cross correlation analysis did not identify leader-follower relations in experimental data. N = 19 communicating cell pairs with visible bands. Temporal resolution = 5 minutes between consecutive time frames. Pair distance of 60-150 μm. Quantification windows were placed 7.5 μm above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics. Time lags were recorded with their absolute value (symmetry argument: no preference to a specific cell as in the simulations). (<strong>A-B</strong>) Cell pair cross correlation for different time lags. (<strong>A</strong>) Distribution of cell pair correlations as a function of the time lag. (<strong>B</strong>) Distribution of time lags that lead to maximal cell pair correlation. (<strong>C-D</strong>) Cross correlation for different time lags between one cell from a communicating pair and another cell from a different communicating pair as a control. (<strong>C</strong>) Distribution of correlations as a function of the time lag. (<strong>D</strong>) Distribution of time lags that lead to maximal correlation.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure S19:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F26.medium.gif" width="440" height="422"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure S19:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F26.medium.gif" width="440" height="422"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F26.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S19:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F26.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure S19:
                               </span>
                               <p id="p-121" class="first-child">
                                Cross correlation analysis did not identify leader-follower relations in experimental data. N = 19 communicating cell pairs with visible bands. Temporal resolution = 5 minutes between consecutive time frames. Pair distance of 60-150 μm. Quantification windows were placed 7.5 μm above the connecting axis between the cells. Correlations were calculated using the first derivative of fiber density dynamics. Time lags were recorded with their absolute value (symmetry argument: no preference to a specific cell as in the simulations). (
                                <strong>
                                 A-B
                                </strong>
                                ) Cell pair cross correlation for different time lags. (
                                <strong>
                                 A
                                </strong>
                                ) Distribution of cell pair correlations as a function of the time lag. (
                                <strong>
                                 B
                                </strong>
                                ) Distribution of time lags that lead to maximal cell pair correlation. (
                                <strong>
                                 C-D
                                </strong>
                                ) Cross correlation for different time lags between one cell from a communicating pair and another cell from a different communicating pair as a control. (
                                <strong>
                                 C
                                </strong>
                                ) Distribution of correlations as a function of the time lag. (
                                <strong>
                                 D
                                </strong>
                                ) Distribution of time lags that lead to maximal correlation.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-37">
                             Our work proposes a systematic computational method to quantify cell-ECM-cell interactions and demonstrated its applicability in simulations and experiments. The method quantifies the local fiber remodeling dynamics between pairs of communicating cells in 2D (simulations) and 3D (experiments) by applying the following key steps. Normalization in relation to the background to enable robust comparison across experiments; Subtracting background deformations to avoid masking of the communication signal; Detrending to avoid spurious correlations; Systematic evaluation of the location of the quantification window to extract the most informative signal; Measurement of the temporal correlation as a quantitative readout for cell-ECM-cell communication. Our method provides technical advances that will open the door for cell biologists and biophysicists to decipher how cells process mechanical information transmitted through the microenvironment.
                            </p>
                            <p id="p-38">
                             We combined finite element simulations and 3D live cell imaging experiments. Our minimal model, although not reflecting the true complexity of the biological system, captures the essence of the mechanical elements of cell contractility and force propagation in fibrous nonlinear elastic networks and thus serves as a viable tool to control various parameters independently to test and verify the sensitivity and robustness of our approach. For example, examining the minimal cell contractile heterogeneity required to effectively quantify cell-ECM-cell communication (
                             <a id="xref-fig-2-8" class="xref-fig" href="#F2">
                              Fig. 2D
                             </a>
                             ( and measure its decay as a function of the distance between the cells)
                             <a id="xref-fig-4-10" class="xref-fig" href="#F4">
                              Fig. 4B
                             </a>
                             ), assessing the effect of changing ECM material from linear to nonlinear behavior (
                             <a id="xref-fig-11-3" class="xref-fig" href="#F11">
                              Fig. S4
                             </a>
                             ) and testing the method’s sensitivity to cell autonomously and contraction magnitude that may mask the capability to identify leader-follower relations (
                             <a id="xref-fig-23-6" class="xref-fig" href="#F23">
                              Fig. S16B-E
                             </a>
                             ).
                            </p>
                            <p id="p-39">
                             We found that heterogeneity in cell contractility created unique temporal patterns of ECM fluctuations that were necessary to quantitatively identify cell-ECM-cell communication, allowing us to distinguish between different pairs of communicating cells using temporal correlation of local ECM remodeling fluctuations. We devised two measurements for cell-ECM-cell communication (same-versus-different and matchmaking) and a third measurement to decouple non-communication-related local ECM deformations from real communication (real-versus-fake). These in silico controls combined with experimental validations verified that our method can sensitively and robustly measure cell-ECM-cell communication in fibroblast and cancer cells (
                             <a id="xref-fig-5-9" class="xref-fig" href="#F5">
                              Fig. 5
                             </a>
                             ,
                             <a id="xref-fig-16-3" class="xref-fig" href="#F16">
                              Fig. S9
                             </a>
                             ).
                            </p>
                            <p id="p-40">
                             Until now, the consensus in the field was that the existence of a visible fibrous band is indicative of cell-ECM-cell communication. Our method does not rely on the existence of a band, rather it correlates local ECM deformations as a measure for “communication”, the unique mechanical signal propagating from one cell to its communication partner. We demonstrate the sensitivity of correlating ECM fluctuations by establishing that the formation of a denser ECM region between the cells is not required for cell-ECM-cell communication, thus decoupling fiber densification and long-range mechanical intercellular communication. In fact, we find that the fibrous band is a confounder to measure the unique communication signature between a pair of cells (
                             <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                              Fig. 7
                             </a>
                             ), probably due to lower ‘signal to noise ratio’ (
                             <a id="xref-fig-21-3" class="xref-fig" href="#F21">
                              Fig. S14
                             </a>
                             ). Moreover, dose-dependent Blebbistatin experiments revealed that while cellular contractile force is required for communication, cells are still able to communicate even after substantial reduction of their contractility, where fibrous bands never form (
                             <a id="xref-fig-22-3" class="xref-fig" href="#F22">
                              Fig. S15
                             </a>
                             ). These results align with previous studies suggesting that myosin II– mediated contractility acts as an inhibitor for cell-to-cell mechanical communication by arresting passive mechanical force transduction through the cellular cortex during collective cell migration
                             <sup>
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              –
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-41">
                             We aimed at identifying leader-follower relationships within pairs of communicating cells. We simulated a situation where one cell influenced its communicating partner, determining its future contractions and demonstrated that our method can robustly identify the leader and follower cells from the ECM-remodeling fluctuations. This was even possible in challenging scenarios where followers were only partially influenced by their leader or contracted up to 20% more than their leader (
                             <a id="xref-fig-23-7" class="xref-fig" href="#F23">
                              Fig. S16C-E
                             </a>
                             ). These results set theoretical limitations on our method’s capacity to decouple cell contractility from leadership status due to increased ECM remodeling that can propagate to the leader cell and mask its influence on the follower. We were not able to identify leader-follower relations in our experimental data, either because of lack of sensitivity in the given experimental conditions (e.g., time resolution, subtle “leadership” signal in cell pairs from the same genetic background) or because this mode of matrix-mediated mechanical communication does not yield leader-follower pairs in fibroblasts. However, our computational model of follower-leaders can motivate future experiments to investigate the notion of leader-follower relations between cells in long-range mechanical cell-cell communication, for example under conditions as faster imaging and/or co-culturing cells in different molecular/functional states or from different genetic backgrounds.
                            </p>
                            <p id="p-42">
                             We used standard confocal imaging in this study, which is inherently limited in its temporal resolution, axial resolution and size of field of view. Despite these limitations, we succeeded to quantitatively characterize intercellular mechanical communication through fibrous environments, being able to near perfectly match all communication partners, with tens of potential candidates for each match, only from ECM remodeling fluctuations. This performance, especially given the standard microscopy that is available in almost any academic institute, highlights the potential of democratizing cell-ECM-cell communication quantification.
                            </p>
                            <p id="p-43">
                             Our proof of principle study is an enabler of mechanistic understanding of long-range cell-cell mechanical communication, and sets the ground for potential applications. Having the ability to quantitatively measure cell-ECM-cell communication, could enable to systematically probe the cellular and molecular players for this mode of mechanical communication, naturally starting with the actomyosin cytoskeleton and adhesion complexes
                             <sup>
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             . We expect that our method could be generalized to non-fibrous environments, such as synthetic (e.g., poly-acrylamide hydrogels) or biological (e.g., Matrigel) hydrogels, by embedding tracer particles that will enable the indirect measurement of local ECM remodeling fluctuations. Another exciting venue is deciphering the role of long-range intercellular communication in the more complex and physiological relevant microenvironments in vivo. A third extension of our method would be to apply it to other modes of cell-cell communication. For example, using intracellular molecular fluctuations as the functional readout to infer cell-cell communication, an approach previously taken in
                             <sup>
                              <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                             </sup>
                             to reveal signaling ordering at the intracellular scale.
                            </p>
                            <p id="p-44">
                             One application where the ability to precisely measure cell-cell communication may be useful is tissue engineering. By controlling the patterning of multiple cell types one could optimize tissue formation according to cell type specific communication capabilities
                             <sup>
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                             </sup>
                             . Another application is high content 3D image-based screening. Image-based phenotypic screening is traditionally applied with 2D imaging, quantifying single cell morphology and distributions of intracellular fluorescent intensities, and is applied for multiple applications including identification and characterization of small molecules in drug discovery
                             <sup>
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                             </sup>
                             . Recent tools enable high-content 3D image-based cell phenotyping
                             <sup>
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                             </sup>
                             , providing a more physiologically relevant context for in vivo follow-up studies
                             <sup>
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                             </sup>
                             . The interplay between 3D cell morphology, the interactions with the environment and the mechanical communication with other cells will likely provide important complementary functional readouts for 3D image-based phenotypic cell phenotyping that are not accessible with current methods. Such phenotyping could be very useful for applications where the cellular microenvironment and the interactions between cells and the ECM are established hallmarks, such as in cancer or fibrosis
                             <sup>
                              <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                               21
                              </a>
                              ,
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              –
                              <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ,
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                             </sup>
                             .
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Computational Modeling
                             </h3>
                             <div id="sec-15" class="subsection">
                              <h4>
                               Finite element simulations of cells contraction in fibrous networks
                              </h4>
                              <p id="p-45">
                               We used our previously described computational finite element model of one or two cells contracting in two-dimensional fibrous networks
                               <sup>
                                <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">
                                 12
                                </a>
                                ,
                                <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">
                                 30
                                </a>
                                ,
                                <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">
                                 31
                                </a>
                               </sup>
                               . We used the finite element software Abaqus/CAE 2017 (Dassault Systèmes Simulia) to model the network mechanics and perform the simulations. The software’s standard\implicit solver was used in all simulations. The cells were modeled as circular voids in the fiber networks. The fibers were represented by discrete one-dimensional elements connected by nodes (
                               <a id="xref-fig-27-1" class="xref-fig" href="#F27">
                                Fig. S20A
                               </a>
                               ), randomly distributed to set an isotropic and homogeneous network structure prior to cell contraction (see next). The output of each finite-element simulation included the information regarding each network element’s location and dynamics.
                              </p>
                              <div id="F27" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F27.large.jpg?width=800&height=600&carousel=1" title="Methods. (A) Components of a finite-element simulation network. Fibers are represented by discrete one-dimensional elements (red) connected by two nodes (blue). (B) Finite-element network dimensions. The simulated fibrous network was defined as a circular arena, of radius 2 AU, that is composed of connected elements. One or two circular voids, of radius 0.08 AU (green), define the cell/s. The center of the network is called the origin (purple). Single cells centers are located in the origin. Cell pairs centers are located along the X axis in predefined distance from the origin. Ratio of cell radius and the network radius in the depiction does not resemble the actual proportion between the two. (C) Transformation of cell pairs images. First transformation (top-left to bottom-left) aligns the cell centers to a common Y’-axis. Images are in top-view perspective. We use X’ and Z’ axes for cell pair visualization (top-right). The second transformation (top-right to bottom-right) aligns the cell centers to a common Z’-axis. Images are in side-view perspective. The transformed image (bottom-right) is used for cell pair quantification. (D) Transformation of single cell images. Illustration is in top-view perspective. First transformation (blue) in 0°, 45°, 90° or 135° in the XY axis. Second transformation (green) in 0°, 45°, 90° or 135° in the Z’ axis. (E) Quantifying fiber (simulation elements) intersection with the quantification windows in simulations. The quantification window is illustrated with a dashed box. To be considered, a fiber must be either exclusively inside the quantification window or crossing the window’s boundaries. The sub-fiber within the window’s boundaries (green) are included while sub-fibers external to the quantification window (red) are excluded from the quantification. (F) The quantification windows for simulated single cells are placed in four directions at 90 degrees radial intervals. (G) Quantification windows in simulations used for computing the normalization parameters. Quantification window (blue as an example) is the size of a cell diameter by a cell diameter (0.08 x 0.08 AU). All quantification windows used for this computation (cyan) are located inside the network boundaries and outside the cells with a step resolution (overlap) of 0.02 (AU) in each of the axes. (H) Quantification windows in experiments used for computing the normalization parameters. Quantification window (blue as an example) length is 15 μm, approximately the cell diameter in 3D. All quantification windows used for this computation (cyan) are located exclusively inside the image boundaries in all axes and without intersection with bounding boxes (orange) around the cells, with a step resolution (overlap) of 1/10 of the image axis length for each axis. (I) An example of overlapping quantification windows between a pair of communicating cells. Images are in a side-view average projection perspective. Quantification windows are marked in white dashed boxes. At the onset of imaging (top) the quantification did not overlap, however after 3.5 hours of live cell imaging (bottom) the quantification windows did overlap (orange area). Scale bar = 15 μm. (J) An example of a quantification window extending beyond the image boundaries at the onset of imaging. Image is in a side-view perspective with averaged pixel intensities visualization after the transformation. Quantification windows are marked in white dashed boxes. Red area represents the sub-window outside the image boundaries. Scale bar = 15 μm. (K) Valid (“V”) and invalid (“X”) quantification windows (see Methods for definition) over time of three different cells. Blue rectangle represents the longest shared valid temporal sequence for cell #1 and cell #2 and the green rectangle for cell #2 and cell #3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-709650196" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml">Methods. (<strong>A</strong>) Components of a finite-element simulation network. Fibers are represented by discrete one-dimensional elements (red) connected by two nodes (blue). (<strong>B</strong>) Finite-element network dimensions. The simulated fibrous network was defined as a circular arena, of radius 2 AU, that is composed of connected elements. One or two circular voids, of radius 0.08 AU (green), define the cell/s. The center of the network is called the origin (purple). Single cells centers are located in the origin. Cell pairs centers are located along the X axis in predefined distance from the origin. Ratio of cell radius and the network radius in the depiction does not resemble the actual proportion between the two. (<strong>C</strong>) Transformation of cell pairs images. First transformation (top-left to bottom-left) aligns the cell centers to a common Y’-axis. Images are in top-view perspective. We use X’ and Z’ axes for cell pair visualization (top-right). The second transformation (top-right to bottom-right) aligns the cell centers to a common Z’-axis. Images are in side-view perspective. The transformed image (bottom-right) is used for cell pair quantification. (<strong>D</strong>) Transformation of single cell images. Illustration is in top-view perspective. First transformation (blue) in 0°, 45°, 90° or 135° in the XY axis. Second transformation (green) in 0°, 45°, 90° or 135° in the Z’ axis. (<strong>E</strong>) Quantifying fiber (simulation elements) intersection with the quantification windows in simulations. The quantification window is illustrated with a dashed box. To be considered, a fiber must be either exclusively inside the quantification window or crossing the window’s boundaries. The sub-fiber within the window’s boundaries (green) are included while sub-fibers external to the quantification window (red) are excluded from the quantification. (<strong>F</strong>) The quantification windows for simulated single cells are placed in four directions at 90 degrees radial intervals. (<strong>G</strong>) Quantification windows in simulations used for computing the normalization parameters. Quantification window (blue as an example) is the size of a cell diameter by a cell diameter (0.08 x 0.08 AU). All quantification windows used for this computation (cyan) are located inside the network boundaries and outside the cells with a step resolution (overlap) of 0.02 (AU) in each of the axes. (<strong>H</strong>) Quantification windows in experiments used for computing the normalization parameters. Quantification window (blue as an example) length is 15 μm, approximately the cell diameter in 3D. All quantification windows used for this computation (cyan) are located exclusively inside the image boundaries in all axes and without intersection with bounding boxes (orange) around the cells, with a step resolution (overlap) of 1/10 of the image axis length for each axis. (<strong>I</strong>) An example of overlapping quantification windows between a pair of communicating cells. Images are in a side-view average projection perspective. Quantification windows are marked in white dashed boxes. At the onset of imaging (top) the quantification did not overlap, however after 3.5 hours of live cell imaging (bottom) the quantification windows did overlap (orange area). Scale bar = 15 μm. (<strong>J</strong>) An example of a quantification window extending beyond the image boundaries at the onset of imaging. Image is in a side-view perspective with averaged pixel intensities visualization after the transformation. Quantification windows are marked in white dashed boxes. Red area represents the sub-window outside the image boundaries. Scale bar = 15 μm. (<strong>K</strong>) Valid (“V”) and invalid (“X”) quantification windows (see Methods for definition) over time of three different cells. Blue rectangle represents the longest shared valid temporal sequence for cell #1 and cell #2 and the green rectangle for cell #2 and cell #3.</div></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure S20:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F27.medium.gif" width="309" height="440"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure S20:" src="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F27.medium.gif" width="309" height="440"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F27.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S20:" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/02/14/2020.07.30.223149/F27.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-caption">
                                <span class="fig-label">
                                 Figure S20:
                                </span>
                                <p id="p-122" class="first-child">
                                 Methods. (
                                 <strong>
                                  A
                                 </strong>
                                 ) Components of a finite-element simulation network. Fibers are represented by discrete one-dimensional elements (red) connected by two nodes (blue). (
                                 <strong>
                                  B
                                 </strong>
                                 ) Finite-element network dimensions. The simulated fibrous network was defined as a circular arena, of radius 2 AU, that is composed of connected elements. One or two circular voids, of radius 0.08 AU (green), define the cell/s. The center of the network is called the origin (purple). Single cells centers are located in the origin. Cell pairs centers are located along the X axis in predefined distance from the origin. Ratio of cell radius and the network radius in the depiction does not resemble the actual proportion between the two. (
                                 <strong>
                                  C
                                 </strong>
                                 ) Transformation of cell pairs images. First transformation (top-left to bottom-left) aligns the cell centers to a common Y’-axis. Images are in top-view perspective. We use X’ and Z’ axes for cell pair visualization (top-right). The second transformation (top-right to bottom-right) aligns the cell centers to a common Z’-axis. Images are in side-view perspective. The transformed image (bottom-right) is used for cell pair quantification. (
                                 <strong>
                                  D
                                 </strong>
                                 ) Transformation of single cell images. Illustration is in top-view perspective. First transformation (blue) in 0°, 45°, 90° or 135° in the XY axis. Second transformation (green) in 0°, 45°, 90° or 135° in the Z’ axis. (
                                 <strong>
                                  E
                                 </strong>
                                 ) Quantifying fiber (simulation elements) intersection with the quantification windows in simulations. The quantification window is illustrated with a dashed box. To be considered, a fiber must be either exclusively inside the quantification window or crossing the window’s boundaries. The sub-fiber within the window’s boundaries (green) are included while sub-fibers external to the quantification window (red) are excluded from the quantification. (
                                 <strong>
                                  F
                                 </strong>
                                 ) The quantification windows for simulated single cells are placed in four directions at 90 degrees radial intervals. (
                                 <strong>
                                  G
                                 </strong>
                                 ) Quantification windows in simulations used for computing the normalization parameters. Quantification window (blue as an example) is the size of a cell diameter by a cell diameter (0.08 x 0.08 AU). All quantification windows used for this computation (cyan) are located inside the network boundaries and outside the cells with a step resolution (overlap) of 0.02 (AU) in each of the axes. (
                                 <strong>
                                  H
                                 </strong>
                                 ) Quantification windows in experiments used for computing the normalization parameters. Quantification window (blue as an example) length is 15 μm, approximately the cell diameter in 3D. All quantification windows used for this computation (cyan) are located exclusively inside the image boundaries in all axes and without intersection with bounding boxes (orange) around the cells, with a step resolution (overlap) of 1/10 of the image axis length for each axis. (
                                 <strong>
                                  I
                                 </strong>
                                 ) An example of overlapping quantification windows between a pair of communicating cells. Images are in a side-view average projection perspective. Quantification windows are marked in white dashed boxes. At the onset of imaging (top) the quantification did not overlap, however after 3.5 hours of live cell imaging (bottom) the quantification windows did overlap (orange area). Scale bar = 15 μm. (
                                 <strong>
                                  J
                                 </strong>
                                 ) An example of a quantification window extending beyond the image boundaries at the onset of imaging. Image is in a side-view perspective with averaged pixel intensities visualization after the transformation. Quantification windows are marked in white dashed boxes. Red area represents the sub-window outside the image boundaries. Scale bar = 15 μm. (
                                 <strong>
                                  K
                                 </strong>
                                 ) Valid (“V”) and invalid (“X”) quantification windows (see Methods for definition) over time of three different cells. Blue rectangle represents the longest shared valid temporal sequence for cell #1 and cell #2 and the green rectangle for cell #2 and cell #3.
                                </p>
                                <div class="sb-div caption-clear">
                                </div>
                               </div>
                              </div>
                             </div>
                             <div id="sec-16" class="subsection">
                              <h4>
                               Fiber network architecture
                              </h4>
                              <p id="p-46">
                               We used Matlab R2018b to construct the network geometry and architecture as previously described
                               <sup>
                                <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">
                                 12
                                </a>
                               </sup>
                               . The networks were designed to optimize the fiber orientations distribution toward uniformity (i.e., isotropic) and toward homogeneous fiber density. Briefly, we devised a random process to create network geometries, as we previously reported in
                               <sup>
                                <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">
                                 12
                                </a>
                               </sup>
                               . The process starts from uniformly scattering nodes in a circular domain. The nodes were then connected by fiber elements by considering an objective cost function which controls the fiber length, fiber connectivity (i.e., the mean number of fibers intersected at each node) and the angle between fibers connected at each node. The network coordinates spanned from -2 to 2 (AU) in X- and Y-axis (
                               <a id="xref-fig-27-2" class="xref-fig" href="#F27">
                                Fig. S20B
                               </a>
                               ). Cell centers were located along the X-axis, with a cell diameter of 0.08 (
                               <a id="xref-fig-27-3" class="xref-fig" href="#F27">
                                Fig. S20B
                               </a>
                               ). The mean fiber thickness was 0.2 AU, and mean fiber length was 20 AU, fitting typical fiber density-to-length ratio for collagen\fibrin gels
                               <sup>
                                <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                                 43
                                </a>
                                –
                                <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                                 45
                                </a>
                               </sup>
                               . The cell diameter before contraction was set to 0.08 AU, so that the cell diameter/mean fiber length ratio was 4:1, a typical ratio for fibroblast cells embedded in fibrin gel
                               <sup>
                                <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                               </sup>
                               . The average connectivity of the network was set to eight, to balance the tradeoff between the finite element software numerical stability and physiological relevance.
                              </p>
                             </div>
                             <div id="sec-17" class="subsection">
                              <h4>
                               The mechanical properties of the simulated fiber networks
                              </h4>
                              <p id="p-47">
                               Fibers were connected to one another by nodes, which acted as a freely rotating hinge, allowing for a rotation of the fibers without resistance. The fibers were modeled as linear truss elements, undergoing uniaxial tension or compression, without bending. They were characterized by nonlinear behavior typical to ECM fibers (such as collagen), including buckling under compression
                               <sup>
                                <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                                 6
                                </a>
                                ,
                                <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                                 8
                                </a>
                               </sup>
                               and stiffening under tension
                               <sup>
                                <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                 4
                                </a>
                               </sup>
                               . We represented the buckling of the fibers by an elastic modulus which was ten times smaller at compressive strains exceeding 2% relative to the elastic modulus at small strains (−2% to 2%). Stiffening was achieved by an exponential increase in the elastic modulus for tensile strains larger than 2%
                               <sup>
                                <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">
                                 12
                                </a>
                                ,
                                <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">
                                 44
                                </a>
                                ,
                                <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                 46
                                </a>
                               </sup>
                               . In all simulations, the outer boundary of the network was fixed for translations and rotations.
                              </p>
                              <p id="p-48">
                               To assess the validity of the method for fibrous ECM networks with varying mechanical properties, we performed additional simulations in which we tested the effect of changing the stiffening behavior to account for a material with a relative linear behavior until 15% strain, representative of fibrin gels (
                               <a id="xref-fig-11-4" class="xref-fig" href="#F11">
                                Fig. S4A
                               </a>
                               ), based on, for example
                               <sup>
                                <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                                 47
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-18" class="subsection">
                              <h4>
                               Simulating cell-ECM-cell communication
                              </h4>
                              <p id="p-49">
                               Cell contraction was modeled by applying a boundary condition of radial isotropic contractile displacements to all nodes constituting the cell boundaries, reaching up to 50% contraction of the cell radius. To simulate time we consecutively applied 1% cell contraction for 50 steps, reaching a final 50% cell contraction. Heterogeneity in cell contraction was implemented by applying cell contraction selected from a normal distribution with a mean (μ) of 1% and varying standard deviations (σ) of 0, 0.25, 0.5 or 0.75, in each simulation step. In all simulations, the network size was set to 50 cell diameters and the cells were placed in its center to prevent boundary effects.
                              </p>
                             </div>
                             <div id="sec-19" class="subsection">
                              <h4>
                               The role of the simulations as a minimal mechanical model for extensive assessment of our method
                              </h4>
                              <p id="p-50">
                               The simulations capture the essence of the material mechanical properties, mechanical elements of cell contractility and force propagation in fibrous nonlinear elastic networks. However, the representation of cells as circular cavities that contract uniformly (with unrealistic high contraction) in 2D without responding to the propagated mechanical signal, is far from realistic biology. Still, this minimal implementation captures the essence of the experimental system in terms of its mechanical aspects (contractile units in fibrous nonlinear networks) and thus serves as a “clean” mechanical framework to examine a process that is inherently mechanical (propagation of forces between cells). Thus, these simulations allow us to obtain proof-of-concept to the ability of a correlation-based method to detect a mechanical signal that propagates in the ECM between two contractile elements. Within this framework, the simulations unambiguously demonstrate the power of correlation in detecting the propagation of a mechanical signal from one cell to the other, as well as the effect of cell-cell distance, cells contraction mismatch and the ability to detect leader-follows. Revisiting the minimal model to include components that are more realistic is not central to the current methodology study, and will be addressed in future studies.
                              </p>
                             </div>
                             <div id="sec-20" class="subsection">
                              <h4>
                               Leader-follower simulations
                              </h4>
                              <p id="p-51">
                               These simulations were implemented such that one cell (the “follower”) contracted with a dependency on the other cell (the “leader”). The leader contraction at each time point was selected from a normal distribution N(μ = 1%, σ = 0.5%). In the first time step, the follower contraction was drawn independently, from the same normal distribution. From the second step and onwards, the follower contraction was composed of two components:
                              </p>
                              <p id="p-52">
                               Contraction
                               <sub>
                                follower
                               </sub>
                               (t) = (1 – α) * N(μ,σ) + α * Contraction
                               <sub>
                                leader
                               </sub>
                               (t – 1),
                              </p>
                              <p id="p-53">
                               where Contraction
                               <sub>
                                follower
                               </sub>
                               (t) is the follower intrinsic contraction at time step t drawn from N(μ = 1%, σ = 0.5%) and Contraction
                               <sub>
                                leader
                               </sub>
                               (t – 1) is the leader’s contraction at the previous time step drawn from the same distribution. α is a constant that varies between 0 and 1 and defines the level of leader-follower dependency: α = 0 is the case of independently contracting cells and α = 1 is the case where the follower cell repeats the leader’s contraction with 1 frame delay (see
                               <a id="xref-fig-24-4" class="xref-fig" href="#F24">
                                Fig. S17A
                               </a>
                               ). In the second leader-follower simulation, the follower contracts according to the following equation: Contraction
                               <sub>
                                follower
                               </sub>
                               (t) = β * Contraction
                               <sub>
                                leader
                               </sub>
                               (t – 1) where β > 1, thus imitating the leader with a certain augmentation.
                              </p>
                             </div>
                            </div>
                            <div id="sec-21" class="subsection">
                             <h3>
                              Experiments
                             </h3>
                             <div id="sec-22" class="subsection">
                              <h4>
                               Cell culture and chemical reagents
                              </h4>
                              <p id="p-54">
                               Swiss 3T3 fibroblasts stably expressed GFP-actin (obtained as gifts from S. Fraser, University of Southern California, Los Angeles, CA) were cultured in DMEM supplemented with 10% fetal bovine serum, nonessential amino acids, sodium pyruvate, L-glutamine, 100 units/ml penicillin, 100 µg/ml streptomycin, and 100 µg/ml Neomycin, in a 37◦C humid incubator.
                              </p>
                              <p id="p-55">
                               Murine Hras mutated-transform cells have been generated by infecting normal epithelial cells of the tongue with mutated-HRAS and shTP53 as described in
                               <sup>
                                <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                 48
                                </a>
                               </sup>
                               .
                              </p>
                              <p id="p-56">
                               Fluorescent beads experiments: 10–14 µm polystyrene beads (SPHERO™, Spherotech, Inc., Lake Forest, IL, USA) were embedded in 20 µl fibrin gels (10µl thrombin, 10µl fibrinogen) in concentration of 8 × 103.
                              </p>
                              <p id="p-57">
                               Dead cells experiments: Dead cells were prepared by placing cells in a hot water bath at 65 ◦C for 30 min, 8 × 103 dead cells were mixed in 20 µl of fibrin (10µl thrombin, 10µl fibrinogen). Co-culture live and dead cells within the same gel: 4 × 103 dead cells were mixed with 4 × 103 live cells. After gel polymerization, ethidium homodimer (5µM, red fluorescence) was added to the medium and incubated for 30 minutes at 20–25°C. This resulted in staining the nuclei of dead cells.
                              </p>
                              <p id="p-58">
                               Blebbistatin experiments: 2 ml medium containing 85 µM or 150 µM blebbistatin were added to cover the polymerized cellular gel.
                              </p>
                             </div>
                             <div id="sec-23" class="subsection">
                              <h4>
                               Fibrinogen labeling
                              </h4>
                              <p id="p-59">
                               Alexa Fluor 546 carboxylic acid, succinimidyl ester (Invitrogen) was mixed with fibrinogen solution in a 7.5:1 molar ratio for 1 hour at room temperature and then filtered through a HiTrap desalting column (GE Healthcare) packed with Sephadex G-25 resin, to separate the unreacted dye.
                              </p>
                             </div>
                             <div id="sec-24" class="subsection">
                              <h4>
                               3D fibrin gel preparation
                              </h4>
                              <p id="p-60">
                               GFP-Actin 3T3 fibroblast cells (8×103 cells) were mixed with 10 µl of a 20 U/ml thrombin solution (Omrix Biopharmaceuticals). Then, 10 µl of a 10 mg/ml fluorescently labeled fibrinogen (Omrix Biopharmaceuticals) suspension was placed in a 35-mm cover-slip bottom dish (MatTek Corporation) and mixed gently with the 10 µl cells suspended in thrombin. The resulting fibrin gel was placed in the incubator for 20 min to polymerize, after which, a warm medium was added to cover the gel. The fibrin gels had an approximate shape of half a sphere, attached to the bottom surface of a cover slip, with a gel height of about 2-3 mm, and cells were fully embedded in the 3D gel. From this point on, the gels including the cells embedded in them, were maintained in 37°C 5% CO
                               <sup>
                                <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                                 2
                                </a>
                               </sup>
                               .
                              </p>
                             </div>
                             <div id="sec-25" class="subsection">
                              <h4>
                               Time-lapse confocal microscopy
                              </h4>
                              <p id="p-61">
                               Pairs of cells were imaged with a Zeiss 880 confocal microscope, equipped with a 40X NA=1.1 water immersion lens (Zeiss) and a 30mW argon laser was used to image both the cells (GFP-Actin) and the fluorescently labeled-fibrin matrix with excitation wavelength of 488, and a separated emission spectrum for each. Throughout imaging, the gels with the contained cells were maintained in a 37°C 5% CO
                               <sup>
                                <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                                 2
                                </a>
                               </sup>
                               incubation chamber. Confocal z-stacks were acquired every 5-15 min for about 6 hours from cell seeding. We manually validated that the imaged cells did not undergo division to avoid the enhanced contraction during division. Confocal imaging inherently includes a tradeoff between the temporal resolution, the axial resolution and the number of imaged locations: higher temporal/axial resolution leads to the lower numbers of locations imaged leading to smaller numbers of communicating cell pairs per experiment. Experiments were imaged in three settings: (1) Temporal resolution of 15 minutes, with 21 locations of images in resolution of 512X512 pixels (0.41X0.41X2 µm in X’Y’Z’) and 36 Z-slices. (2) Temporal resolution of 5 minutes, with 9 locations in each and resolution of 256X256 pixels (0.83X0.83X2 µm in X’Y’Z’) and 40 Z-slices. (3) Temporal resolution of 21 minutes, with 7 locations of images in resolution of 512X512 pixels (0.41X0.41X0.53 µm in X’Y’Z’) and 187 Z-slices, where X’, Y’ and Z’ are the microscopy axes. The spatial image resolution used in our study is not sufficient to localize individual fibers, but fine resolution is not required for our method, which uses the time series of the fluorescent intensity accumulated in a quantification window. We prioritized imaging large field of views over better spatial or temporal resolution to increase the number of cell pairs we can image in a single experiment and thus enhancing our statistical power. Moreover, the use of relatively low temporal and spatial resolutions are in fact an advantage. The ability to robustly detect cell-ECM-cell communication even at lower resolutions is yet another indication for our method’s sensitivity and the wide availability of standard confocal imaging in practically any academic institute further highlights the potential of democratizing cell-ECM-cell communication quantification.
                              </p>
                             </div>
                            </div>
                            <div id="sec-26" class="subsection">
                             <h3>
                              Image analysis and quantifications
                             </h3>
                             <div id="sec-27" class="subsection">
                              <h4>
                               Preprocessing live imaging data
                              </h4>
                              <p id="p-62">
                               We first applied Fiji’s
                               <sup>
                                <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                 49
                                </a>
                               </sup>
                               “Bleach Correction” on the raw fiber channel with the “Histogram Matching” option. On the actin (cell) channel we applied a median filter (radius = 2) followed by a Gaussian blurring filter (sigma = 2), before segmenting the cells over time using the “3D Objects Counter” Fiji’s plugin
                               <sup>
                                <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                 50
                                </a>
                               </sup>
                               (Threshold = 15, Size filter > 400). The cell’s center coordinates in 3D for each time frame was recorded and used for cell tracking. Custom Python code was used for the cell tracking, by identification of cells to track in the first time frame and simply assigning the nearest cell (in 3D Euclidean distance) in the next time frame to construct the trajectory. Shorter trajectories were recorded for cells that moved beyond the field of view during imaging. This simple approach was sufficient thanks to the sparsity of the cell seeding.
                              </p>
                             </div>
                             <div id="sec-28" class="subsection">
                              <h4>
                               Transforming 3D images for visualization and quantification
                              </h4>
                              <p id="p-63">
                               To visualize and quantify the 3D band between a cell pair we transformed the image to a new coordinate system that is defined in relation to the spatial relation between the pair. We transformed the image from the microscopy axes (denoted X’,Y’,Z’) to the following three new axes. The connecting axis, defined by the line connecting the cells’ centers (
                               <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                                Fig. 3A
                               </a>
                               ,
                               <a id="xref-fig-9-3" class="xref-fig" href="#F9">
                                Fig. S2
                               </a>
                               and Video S2 black line). The Z axis, parallel to the microscopy axial plane (Z’) and perpendicular to the connecting axis (
                               <a id="xref-fig-3-8" class="xref-fig" href="#F3">
                                Fig. 3A
                               </a>
                               ,
                               <a id="xref-fig-9-4" class="xref-fig" href="#F9">
                                Fig. S2
                               </a>
                               and Video S2 cyan line). The XY axis, perpendicular to the connecting axis and to the Z axis (
                               <a id="xref-fig-3-9" class="xref-fig" href="#F3">
                                Fig. 3A
                               </a>
                               ,
                               <a id="xref-fig-9-5" class="xref-fig" href="#F9">
                                Fig. S2
                               </a>
                               and Video S2 purple line). For visualization, we used the new 2D axis defined by XY and Z (
                               <a id="xref-fig-27-4" class="xref-fig" href="#F27">
                                Fig. S20C
                               </a>
                               ). We used Fiji’s
                               <sup>
                                <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">
                                 49
                                </a>
                               </sup>
                               “Reslice” function (default “Output spacing”, “Slice count” according to the fibroblast diameter of 15 µm) to slice the images from top to bottom in the XY axis perpendicular to the connecting axis between the cells, with a width of 15 µm, interpolating axial pixel values to match the spatial resolution in XY using bilinear interpolation. Finally, we averaged the pixel intensities across the slices using Fiji’s “Z project” (Projection type = “Average intensity”) to create the 2D visualization (for example see the bottom of
                               <a id="xref-fig-1-8" class="xref-fig" href="#F1">
                                Fig. 1E
                               </a>
                               ). To visualize single cells, we picked an arbitrary XY axis (either 0°, 45°, 90° or 135°) with the same axial axis (Z). These visualizations were used for all experiments and all manual annotations (identifying imaging artifacts, and cell pairs with/without a visible band).
                              </p>
                              <p id="p-64">
                               We implemented custom Python code to quantify the ECM density between a pair of communicating cells. First, we transformed the 3D axes to XY slices and Z, replicating the visualization without the Z-interpolation and without the slices averaging. Second, we performed another transformation, rotating the image onto the connecting axis between the cells to reach a common Z-axis. This transformation generates a 3D image where the Z-axis is perpendicular to the XY-axis that is perpendicular to the connecting axis between the cell’s centers. This property allows us to move axially in relation to the 3D line connecting the cells. The whole process is depicted in
                               <a id="xref-fig-27-5" class="xref-fig" href="#F27">
                                Fig. S20C
                               </a>
                               .
                              </p>
                              <p id="p-65">
                               The second transformation rotated the original X’, Y’ and Z’ axes thus defining a new coordinate system, where the transformed pixel size (“resolution”) in the new connecting axis and in the Z axis are a weighted combination of the original microscopy resolution in X’, Y’ and Z’.
                               <span class="disp-formula" id="disp-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/graphic-8.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/graphic-8.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               where θ is the calculated rotation angle between the connecting axis and the microscopy lateral plane before the rotation. The XY axis resolution remains unchanged (X’,Y’
                               <sub>
                                resolution
                               </sub>
                               ). These new resolutions were used to calculate the size and location of the quantification window (see below).
                              </p>
                              <p id="p-66">
                               Single cells do not have a preferred axis in 3D since they do not have a communicating partner. Thus, to quantify ECM density near single cells we sampled around each cell in 32 different orientations in 3D. 16 transformations were defined using all paired combinations of four angles (0°, 45°, 90° and 135°), each transformation pair was applied similarly to the transformations in cell pairs. For example, the pair <45,135> implies first rotating the image in 45° in X’,Y’ (blue arrow in
                               <a id="xref-fig-27-6" class="xref-fig" href="#F27">
                                Fig. S20D
                               </a>
                               ) followed by a 135° rotation in Z’ (green arrow in
                               <a id="xref-fig-27-7" class="xref-fig" href="#F27">
                                Fig. S20D
                               </a>
                               ). These 16 transformations were used in two directions along the rotated axes leading to 32 orientations for quantification (see below).
                              </p>
                             </div>
                             <div id="sec-29" class="subsection">
                              <h4>
                               Manual filtering of defected image frames
                              </h4>
                              <p id="p-67">
                               A small fraction of frames in a few experiment locations had imaging-related artifacts that hampered our ability to accurately segment the cell and quantify ECM densities. These artifacts included incorrect cell segmentation, dark areas due to imaging malfunctioning of the microscope and “light waves” (Video S7) that may have been the result of an air or water bubble trapped in the lenese’s immersion oil. To include these experiments in our analysis we manually identified and recorded defected frames that had these artifacts and considered only valid time frames (without this artifact), when computing fiber density and correlations.
                              </p>
                             </div>
                             <div id="sec-30" class="subsection">
                              <h4>
                               Manual annotation of cell pairs with or without a visible band of increased density
                              </h4>
                              <p id="p-68">
                               For analysis we considered cell pairs with pair distance ranging at 60-150 μm (4-10 cell diameters, assuming fibroblast diameter of 15μm). Based on previous studies that focused solely on cell pairs with a visible band of increased density between them
                               <sup>
                                <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">
                                 6
                                </a>
                                ,
                                <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">
                                 9
                                </a>
                                ,
                                <a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">
                                 11
                                </a>
                                ,
                                <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">
                                 46
                                </a>
                               </sup>
                               , we partitioned our dataset to cell pairs that formed and ones that did not form a visible band of denser fibers between the cells. This partition was performed manually by visual assessment of the pixel intensity along the full length of the connecting axis between the cells at the end of imaging. Visually apparent bands appeared in approximately 60% of the imaged cell pairs.
                              </p>
                             </div>
                             <div id="sec-31" class="subsection">
                              <h4>
                               Quantification window size
                              </h4>
                              <p id="p-69">
                               To quantify the local ECM density we used a quantification window of the size of a cell diameter in all axes, in 2D simulations (0.08 AU) or 3D in experiments (15 μm). This size was set to optimize the tradeoff between including sufficient data versus too much irrelevant data within the window. The number of pixels defining the quantification window (cell diameter in simulations, 15 μm in experiments) were calculated according to the transformed image resolutions. The same scale was also used upon shifting the quantification windows.
                              </p>
                             </div>
                             <div id="sec-32" class="subsection">
                              <h4>
                               Quantifying local fiber density in simulations
                              </h4>
                              <p id="p-70">
                               The local fiber density was calculated as the accumulated fiber volume within the quantification window. We assume that the fiber volume is preserved even when the fiber is deformed. However, this property does not hold in the simulated 2D representation of the fibers where their buckling property reduces the simulated fiber lengths. This is an inherent limitation of simulating a 3D process in 2D. To overcome this limitation we normalize each fiber to its initial length before summing the fiber length in the quantification window. More specifically, we considered two scenarios (
                               <a id="xref-fig-27-8" class="xref-fig" href="#F27">
                                Fig. S20E
                               </a>
                               ). (1) For the case where the fiber was located exclusively within the quantification window, its length at the onset of the simulation was used for quantification. (2) For the case where the fiber was not located exclusively within the quantification window (i.e., crossing the window boundaries), we used only the sub-fiber within the quantification window while adjusting to the full fiber length at the onset of the simulation: Fiber
                               <sub>
                                Volume
                               </sub>
                               (t) =
                               <span class="inline-formula" id="inline-formula-1">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/inline-graphic-1.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/inline-graphic-1.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , where Fiber
                               <sub>
                                Volume
                               </sub>
                               (t) is the fiber volume at time t, Fiber
                               <sub>
                                InnerLength
                               </sub>
                               (t) is the length of the sub-fiber within the quantification window at a time t, Fiber
                               <sub>
                                Length
                               </sub>
                               (t
                               <sub>
                                0
                               </sub>
                               ) is the overall fiber length at the onset of simulation and Fiber
                               <sub>
                                Length
                               </sub>
                               (t) is the overall fiber length at time t.
                              </p>
                              <p id="p-71">
                               The fiber density within a quantification window was defined as the accumulated fiber volume within it. For single cells the mean fiber density of four windows, above, below, to the left and to the right of the cell was recorded (
                               <a id="xref-fig-27-9" class="xref-fig" href="#F27">
                                Fig. S20F
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-33" class="subsection">
                              <h4>
                               Quantifying local fiber density in experiments
                              </h4>
                              <p id="p-72">
                               The mean fluorescent fibrin channel intensity was used as a proxy of fiber density within the transformed 3D quantification windows (see earlier). Quantification windows with over 5% of pixels extending beyond the image boundaries were marked as “invalid” and were excluded from further analysis. Quantification windows for single cells were calculated similarly to simulations, but in 3D, averaging the mean intensity in 32 orientations (see earlier).
                              </p>
                             </div>
                             <div id="sec-34" class="subsection">
                              <h4>
                               Normalizing the local fiber density in simulations and experiments
                              </h4>
                              <p id="p-73">
                               To enable quantitative comparison across experiments and between simulations and experiments, we normalized the fiber density to its z-score -the number of standard deviations away from the mean background fiber density at quantification windows that were not influenced by the cells. Background quantification windows were defined for every location at the onset of simulation/imaging before (simulation) or where minimal (experiments) ECM remodeling occured. To calculate the mean background fiber density we considered all quantification windows that did not intersect with the quantification window around the cells’ center (
                               <a id="xref-fig-27-10" class="xref-fig" href="#F27">
                                Fig. S20G
                               </a>
                               - simulations, windows step resolution = 0.02 cell diameter in each axis;
                               <a id="xref-fig-27-11" class="xref-fig" href="#F27">
                                Fig. S20H
                               </a>
                               - experiments, windows step resolution = 1/10 of the image axis length for each axis). The mean (μ) and the standard deviation (σ) of all background quantification windows was calculated per simulation/location and was used to normalize each quantification window according to the z-score measure,
                               <span class="inline-formula" id="inline-formula-2">
                                <span class="highwire-responsive-lazyload">
                                 <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/inline-graphic-2.gif"/>
                                 <noscript>
                                  <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/02/14/2020.07.30.223149/embed/inline-graphic-2.gif"/>
                                 </noscript>
                                </span>
                               </span>
                               , i.e., the variation from the mean background intensity in units of standard deviation, where FiberDensity is the fiber density quantification before normalization. This measure could be pooled and compared across locations, experiments and could even be used to compare simulations to experiments.
                              </p>
                             </div>
                             <div id="sec-35" class="subsection">
                              <h4>
                               Extracting local ECM density over time
                              </h4>
                              <p id="p-74">
                               After performing the image transformation (see above), the quantification windows were placed adjacent to cell boundaries in 2D (simulations) or 3D (experiments) within the cuboid along the connecting axis between the cells, see
                               <a id="xref-fig-9-6" class="xref-fig" href="#F9">
                                Fig. S2
                               </a>
                               and Video S2. This location was updated at each time frame according to the current cell boundary positions that changed due to cell contraction (simulations) or motion (experiments). For example, this tracking corrected for axial drifting of cells toward the glass due to the contraction of the entire gel induced by cell forces. Shifts in the quantification windows were performed relative to this position. Offset in the Z axis translated to a quantification window placed above/below the pair connected axis and perpendicular to the XY axis. Offset in the XY axis translated to a quantification window placed to left/right in XY without changing the Z position. Shifting the quantification window toward the communication partner or away from a single cell (window distance > 0) was performed on the connecting axis between the cells (
                               <a id="xref-fig-10-2" class="xref-fig" href="#F10">
                                Fig. S3A
                               </a>
                               ), or the axis defined by the image transformation angle in single cells. For a given time-lapse sequence we recorded and normalized the fiber density within the corresponding quantification windows over time.
                              </p>
                              <p id="p-75">
                               We marked quantification windows within the time series as invalid if one of the following criteria holds: (1) Frames with annotated imaging artifacts within the corresponding imaging locations (bad cell segmentation, malfunctioning of the imaging microscope and “light waves”, Video S7). (2) Overlapping quantification windows, in future corresponding time frames in the time series (
                               <a id="xref-fig-27-12" class="xref-fig" href="#F27">
                                Fig. S20I
                               </a>
                               ). (3) Quantification windows with over 5% of pixels extending beyond the image boundaries (
                               <a id="xref-fig-27-13" class="xref-fig" href="#F27">
                                Fig. S20J
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-36" class="subsection">
                              <h4>
                               Eliminating imaging artifacts between time frames to avoid correlations not related to cell-ECM-cell communication
                              </h4>
                              <p id="p-76">
                               Cells from the same experiment were embedded in the same fibrous gel inducing local ECM correlations that are not related to cell-ECM-cell communication. To eliminate these artifacts the local ECM density in each quantification window was normalized by two other windows (termed together “normalization border”) located above and below (in the Z-axis) the quantification window (see
                               <a id="xref-fig-4-11" class="xref-fig" href="#F4">
                                Fig. 4C
                               </a>
                               ). Each normalization border height was 0.5 cell diameter and located with a gap of 0.25 cell diameter above or below the quantification window. These parameters were set to optimize the tradeoff between reducing erroneous correlations induced by local correlated ECM fluctuations near the quantification windows and maintaining the ECM fluctuations necessary for measuring cell-ECM-cell communication. Specifically, the parameters were determined to maximize same-versus-different analysis in live cells while minimizing it for dead cells. The normalized and corrected ECM density time series were used for all correlation-based analyses.
                              </p>
                             </div>
                             <div id="sec-37" class="subsection">
                              <h4>
                               Correlation-based analyses
                              </h4>
                              <p id="p-77">
                               For correlation-based analysis we considered the longest sub-sequences with continuous valid time frames and mutual timestamps (
                               <a id="xref-fig-27-14" class="xref-fig" href="#F27">
                                Fig. S20K
                               </a>
                               ). We considered only cell pairs with mutual sub-sequences of at least 15 (temporal resolution = 15 minutes) or 50 (temporal resolution = 5 minutes) time frames. Correlation was calculated for the second derivative (simulations) or first derivative (experiments) of the fiber density dynamics according to stationarity criteria to avoid high correlations stemming from the monotonic increase of the fiber density (simulations and experiments) and its derivative (simulations) (
                               <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                                Fig. S1
                               </a>
                               ,
                               <a id="xref-fig-12-2" class="xref-fig" href="#F12">
                                Fig. S5
                               </a>
                               ). We determined the first/second derivative detrending according to two stationarity tests Kwiatkowski–Phillips–Schmidt–Shin (KPSS)
                               <sup>
                                <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">
                                 51
                                </a>
                               </sup>
                               and Augmented Dickey Fuller (ADF)
                               <sup>
                                <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">
                                 52
                                </a>
                               </sup>
                               . The null hypothesis in the KPSS test is time series stationarity, while in the ADF test is time series non-stationarity. Temporal correlations were calculated using Pearson correlation on the derived time series.
                              </p>
                             </div>
                             <div id="sec-38" class="subsection">
                              <h4>
                               Same-versus-different pair analysis
                              </h4>
                              <p id="p-78">
                               To establish that cell-ECM-cell communication of one cell pair can be distinguished from a second cell pair we tested whether the correlation between a cell pair (“same” pair) surpassed the correlation between one cell from that pair and another cell from a different cell pair (“different” pair) (
                               <a id="xref-fig-4-12" class="xref-fig" href="#F4">
                                Fig. 4A
                               </a>
                               ). This comparison was termed same-versus-different pair analysis. In this analysis, we considered all combinations of triplets of cells in an experiment, that included one communicating cell pair and another cell that takes part in another communicating pair (
                               <a id="xref-fig-4-13" class="xref-fig" href="#F4">
                                Fig. 4A
                               </a>
                               ). The quantification window of each cell in the analysis was always located in relation to its communication partner (
                               <a id="xref-fig-4-14" class="xref-fig" href="#F4">
                                Fig. 4A
                               </a>
                               ).
                              </p>
                             </div>
                             <div id="sec-39" class="subsection">
                              <h4>
                               Real-versus-fake pair analysis
                              </h4>
                              <p id="p-79">
                               To control for misinterpreting non-communication related local ECM remodeling correlations as cell-ECM-cell communication we devised a standardized internal computational control that measures whether the correlation between a cell pair (“real” pair) surpassed the correlation between one cell from that pair and another “fake” cell (forming together a “fake” pair). The “fake” cell location was determined by following two constraints. The distance between the “fake” and its “real” partner is equal to the distance between the communicating cell pair, while maximizing the distance to other real cells. The first constraint was defined to allow comparable distance between the “real” and “fake” cells pairs. The second constraint was defined to minimize the effect of other cells in the vicinity. See depiction in
                               <a id="xref-fig-5-10" class="xref-fig" href="#F5">
                                Fig. 5A
                               </a>
                               .
                              </p>
                             </div>
                             <div id="sec-40" class="subsection">
                              <h4>
                               Matchmaking analysis
                              </h4>
                              <p id="p-80">
                               For each cell that takes part in a cell pair we tested our ability to identify its matched communication partner from all the other cells in that experiment (
                               <a id="xref-fig-5-11" class="xref-fig" href="#F5">
                                Fig. 5C
                               </a>
                               ). This was performed by ranking the ECM remodeling fluctuations (i.e., first derivative of the fiber density dynamics) correlations between the cell at test to all other potential communication partners and recording the ranking (i.e., the position in the sorted list of all correlations with other cells) of the true communication partner. The cells that were considered as potential communication partners were determined based on the evaluation at test, for example considering only pairs with bands, pairs without bands, or all pairs regardless of having bands. To make this analysis independent of the number of cells in an experiment, we randomly selected, with repetition, 49 potential communication partners with an expected random probability of identifying the communication partner of 1/50 = 0.02. The probability of identifying the true communication partner was recorded at the matchmaking score.
                              </p>
                             </div>
                             <div id="sec-41" class="subsection">
                              <h4>
                               Controlling for potential masking of cell-ECM-cell communication by local ECM remodeling fluctuations
                              </h4>
                              <p id="p-81">
                               To further verify that our results in the same-versus-different pair and matchmaking analyses were not merely an artifact of local ECM correlations in the fibrous network, we compared the correlation of communicating cell pairs to ECM remodeling correlations in quantification windows that were placed in cell-free, fibrous areas. The intuition behind this control experiment was to consider correlations in quantification windows that measure the proximity-component, or the local mutual ECM fluctuations, without the influence of the communicating cells. These controls were performed by manually annotating “fake” cell pairs, and analyzing them while “following” the corresponding “real” cells (
                               <a id="xref-fig-15-3" class="xref-fig" href="#F15">
                                Fig. S8A
                               </a>
                               , cyan cells). Specifically, the quantification windows’ locations followed their corresponding cell pair’s motion by shifting in X’ and Y’ while maintaining a fixed distance from the real pair.
                              </p>
                             </div>
                             <div id="sec-42" class="subsection">
                              <h4>
                               Assessing sensitivity to temporal resolution
                              </h4>
                              <p id="p-82">
                               To examine the sensitivity of our method to the temporal resolution we performed same-versus-different pair analysis for down-sampled time series. For the sake of completeness, all possible starting time frames were considered when setting the first time frame for sampling. For example, when down-sampling the temporal resolution from 5 to 15 minutes per frame, there are 15/5 = 3 possible starting time frames: t
                               <sub>
                                0
                               </sub>
                               , t
                               <sub>
                                1
                               </sub>
                               and t
                               <sub>
                                2
                               </sub>
                               . When choosing t
                               <sub>
                                1
                               </sub>
                               , for example, the first 3 sampled time frames are t
                               <sub>
                                1,
                               </sub>
                               t
                               <sub>
                                4
                               </sub>
                               , t
                               <sub>
                                7
                               </sub>
                               . Time series shorter than 5 (temporal resolution = 15 minutes) or 15 (temporal resolution = 5 minutes) time frames were excluded from further correlation-based analysis. The initial chosen time frame for sampling was set for all same-versus-different computations of that time frame. For each temporal resolution, we pooled all same-versus-different results, which included multiple time series per down-sampled time series as described above. See
                               <a id="xref-fig-17-2" class="xref-fig" href="#F17">
                                Fig. S10B
                               </a>
                               for full assessment.
                              </p>
                             </div>
                             <div id="sec-43" class="subsection">
                              <h4>
                               Pooling data across experiments for statistical assessment
                              </h4>
                              <p id="p-83">
                               Each experiment was analyzed independently to avoid erroneous relations stemming from correlating two cells in different fibrous networks. Such correlations would be inherently lower when correlating ECM fluctuations between different networks versus in the same fibrous network, due to global network remodeling. Thus, correlating ECM fluctuations between different experiments will lead to erroneously optimistic results, which we avoided here by considering all possible combinations of “same”/”different”, “real”/”fake”, and matchmaking for each experiment independently. After analyzing each experiment independently we pooled all the results across experiments for statistical assessment. The non-parametric Wilcoxon signed-rank test was used for statistical analysis testing the null hypothesis that a distribution, such as of “same”-”different” correlations, was distributed around zero.
                              </p>
                              <p id="p-84">
                               Summary of all measurements over all experimental conditions are reported in Table S1.
                              </p>
                             </div>
                             <div id="sec-44" class="subsection">
                              <h4>
                               Leader-follower analysis with cross-correlation
                              </h4>
                              <p id="p-85">
                               Finite-element simulations with a predefined leader and follower were examined using cross-correlation analysis, measuring the correlation between two time series under different time lags. We generated simulations where one cell in a pair was predetermined as the “leader” and its communication partner as the “follower”. The “follower” cell “imitated” the “leader” cell contraction in the previous time step, thus the contraction of the “follower” cell is lagging one simulation round behind the “leader”. The magnitude of influence that the leader had on the following was defined with the parameter α: Contraction
                               <sub>
                                follower
                               </sub>
                               (t) = (1 – α) * N(μ,σ) + α * Contraction
                               <sub>
                                leader
                               </sub>
                               (t – 1), wheret > 1, α = 0, 0.25, 0.5, 0.75 or 1, μ = 1, σ = 0.5 and Contraction
                               <sub>
                                leader
                               </sub>
                               was drawn from the normal distribution N(μ,σ). The parameter α indicates the “followership” magnitude, higher values of α imply that the follower is more influenced by its leader contraction.
                              </p>
                              <p id="p-86">
                               In a second simulation we tested whether our approach can identify “leader” and “follower” even when the “follower” contracted more than the “leader”, but was still copycating the leader’s contraction with a time lag of 1 simulation round. The fold increase of the “follower’s” contraction was defined using a parameter β: Contraction
                               <sub>
                                follower
                               </sub>
                               (t) = β * Contraction
                               <sub>
                                leader
                               </sub>
                               (t – 1), where β = 1, 1.05, 1.1 or 1.2.
                              </p>
                              <p id="p-87">
                               Cross-correlation analysis was performed by comparing different time-lags to the cells’ time series and evaluating in relation to the simulation ground truth. Cross-correlation was also performed on experimental data, where ground truth was not available, but did not identify any leader/follower relations, either due to lacking temporal resolution or lack of leaders/followers in our dataset.
                              </p>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-45">
                            <h2 class="">
                             Data
                            </h2>
                            <p id="p-88">
                             See Tables S2-3 for detailed information regarding all simulated and experimental data in this work.
                            </p>
                           </div>
                           <div class="section" id="sec-46">
                            <h2 class="">
                             Software and data availability
                            </h2>
                            <p id="p-89">
                             Processed data and source code are publicly available at
                             <a href="https://github.com/assafna/cell-ecm-project">
                              https://github.com/assafna/cell-ecm-project
                             </a>
                             . The data include the processed ECM remodeling fluctuation time series for each cell, in the key simulations and experiments. The source code to perform all analyses presented here is included.
                            </p>
                           </div>
                           <div class="section" id="sec-48">
                            <h2 class="">
                             Author Contribution
                            </h2>
                            <p id="p-91">
                             AZ and AL conceived the study. SN designed the experimental assay. SN and BE performed all experiments. YK performed all simulations. SG designed the simulated fibrous networks. SG and ME provided the cancer cell line. AN developed analytic tools, analyzed, and interpreted the data. AN, AL and AZ drafted the manuscript. AK assisted with the microscopy experiments. AL and AZ mentored the authors. All authors wrote and edited the manuscript and approved its content.
                            </p>
                           </div>
                           <div class="section" id="sec-49">
                            <h2 class="">
                             Competing Financial Interests
                            </h2>
                            <p id="p-92">
                             The authors declare that they have no competing interests.
                            </p>
                           </div>
                           <div class="section" id="sec-50">
                            <h2 class="">
                             Supplementary tables legends
                            </h2>
                            <p id="p-93">
                             <strong>
                              Table S1:
                             </strong>
                             Summary of experimental results, measuring cell-ECM-cell communication in different cell systems, imaging and experimental conditions. Same-versus-different, real-versus-fake and matchmaking results were recorded for each condition.
                            </p>
                            <p id="p-94">
                             <strong>
                              Table S2:
                             </strong>
                             Simulation data table. These data include single cells and cell pairs, heterogeneity and leader / follower in pair distances 5, 7 and 9 cell diameters.
                            </p>
                            <p id="p-95">
                             <strong>
                              Table S3:
                             </strong>
                             Experimental data table. Experimental data was partitioned to different categories as followed. (1) Whether the experiment involved single cell or cell pairs, (2) Which cell line (fibroblasts/cancer) or beads were used, (3) Whether cells were treated with Blebbistatin or whether the (control) experiment included dead cells, (4) Are the cells “real” or is it a “fake” cells analysis, (5) Whether the cell pairs had a dense fibrous band between them, (6) What was the temporal resolution (in minutes), (7) What was the pair distance between the cells (4-6, 6-8 and 8-10 cell diameters, where cell diameter is estimated at ∼15µm), (8) What was the Blebbistatin dosage (in nM), (9) Whether the experiment consisted exclusively dead cells, (10) What was the number (N) of observations (mostly cell pairs) in the category.
                            </p>
                           </div>
                           <div class="section" id="sec-51">
                            <h2 class="">
                             Supplementary video legends
                            </h2>
                            <p id="p-96">
                             <strong>
                              Video S1
                             </strong>
                             : Finite-element simulation of a cell pair contracting in a 2D fibrous network. Pair distance = 7 cell diameters. Both cell contracts 1% for 50 steps (Methods).
                            </p>
                            <p id="p-97">
                             <strong>
                              Video S2
                             </strong>
                             : 3D representation of a cell pair quantification axes. Microscopy axes in the left bottom corner: X’ (dark green), Y’ (orange) and Z’ (cyan). Cells (green) are in different Z’ coordinates. Connecting axis (black) is between the cell’s centers. Quantification windows (yellow) are adjacent to each cell along the connecting axis (black) between the cell’s centers, left and right sides are parallel to the microscopy axial plane. The Z-axis (cyan) and the XY-axis (purple) are perpendicular to each other and to the connecting axis (black). The video demonstrates offsets between -1 and 1 cell diameters of the quantification windows in the Z (cyan) and XY (purple) axes.
                            </p>
                            <p id="p-98">
                             <strong>
                              Video S3
                             </strong>
                             : Time-lapse confocal imaging of a single fibroblast embedded in a 3D fibrin gel. XY/Z average projection visualization of a single cell (Methods). Time resolution: 21 minutes. Scale bar = 15 μm.
                            </p>
                            <p id="p-99">
                             <strong>
                              Video S4
                             </strong>
                             : Time-lapse confocal imaging of a pair of fibroblast cells embedded in a 3D fibrin gel forming a band of dense fibers in the connecting axis between the cells. XY/Z average projection visualization of a pair of cells (Methods). Pair distance: 117 μm. Time resolution: 15 minutes. Scale bar = 15 μm.
                            </p>
                            <p id="p-100">
                             <strong>
                              Video S5
                             </strong>
                             : Matchmaking between communication partners. Images are in a side-view perspective. Cell pairs are with visible bands. Quantification windows (white dashed rectangles) are adjacent to each cell along the connecting axis (black) between the cell’s centers. The video demonstrates the analysis of a cell by correlating it’s ECM remodeling dynamics with all other candidates. Correlation results are for demonstration purpose only.
                            </p>
                            <p id="p-101">
                             <strong>
                              Video S6
                             </strong>
                             : Time-lapse confocal imaging of a pair of fibroblast cells embedded in 3D fibrin gel not forming a band of dense fibers in the connecting axis between the cells. Pair distance: 110 μm. Time resolution: 15 minutes. Scale bar = 15 μm.
                            </p>
                            <p id="p-102">
                             <strong>
                              Video S7
                             </strong>
                             : Time lapse imaging of a cell pair with a manually identified illumination artifact in several time frames that were excluded from analysis.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Funding and Acknowledgements
                            </h2>
                            <p id="p-90">
                             This work was supported by the Israeli Council for Higher Education (CHE) via Data Science Research Center, Ben-Gurion University of the Negev, Israel (to AZ), by the Wellcome Leap Delta Tissue program (to AZ), the Israel Science Foundation (1474/16, to AL), the Israel Science Foundation-Israeli Centers for Research Excellence (1902/12, to AL), and the Zimin Institute for Engineering Solutions Advancing Better Lives (to AL). We thank Oren Tchaicheeyan, Nir Gov and Meghan Driscoll for critically reading the manuscript. We thank Jean-Yves Tinevez and Dmitry Ershov from Institut Pasteur for training AN in image acquisition and analysis, this training was made possible thanks to NEUBIAS (COST Action, CA15124) Short Term Scientific Mission (STSM).
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-4">
                               1. Substantial reorganization of the manuscript structure focusing on the methodological contribution and validations.
2. Systematic decoupling of cell-cell communication from the confounding factor of non-communication-related local deformations in the ECM.
3. Experimental validations.
4. Assessment of the role of Myosin II contractility on cell-cell communication with Blebbistatin experiments.
5. Emphasizing the role of the simulations as a minimal mechanical model for extensive assessment of our method.
                              </p>
                             </li>
                             <li class="fn-dataset fn-group-external-links" id="fn-2">
                              <p id="p-5">
                               <a href="https://github.com/assafna/cell-ecm-project">
                                https://github.com/assafna/cell-ecm-project
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.1">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Korff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Augustin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tensional forces in fibrillar extracellular matrices control directional capillary sprouting
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Cell Sci
                                 </abbr>
                                 <span class="cit-vol">
                                  112
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3249
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3258
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1999
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DKorff%26rft.auinit1%253DT.%26rft.volume%253D112%26rft.issue%253D19%26rft.spage%253D3249%26rft.epage%253D3258%26rft.atitle%253DTensional%2Bforces%2Bin%2Bfibrillar%2Bextracellular%2Bmatrices%2Bcontrol%2Bdirectional%2Bcapillary%2Bsprouting%26rft_id%253Dinfo%253Apmid%252F10504330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjExMi8xOS8zMjQ5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMC4wNy4zMC4yMjMxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.2" data-doi="10.1529/biophysj.107.127662">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Reinhart-King
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Dembo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hammer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cell-cell mechanical communication through compliant substrates
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  95
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  6044
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  6051
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2008
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.volume%253D95%26rft.issue%253D12%26rft.spage%253D6044%26rft.epage%253D6051%26rft.atitle%253DCell-cell%2Bmechanical%2Bcommunication%2Bthrough%2Bcompliant%2Bsubstrates.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.107.127662%26rft_id%253Dinfo%253Apmid%252F18775964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1529/biophysj.107.127662&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18775964&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000261559300048&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Winer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Oake
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janmey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Non-linear elasticity of extracellular matrices enables contractile cells to communicate local position and orientation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  e6382
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.4" data-doi="10.1016/j.bpj.2013.02.017">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schickel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stevenson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sarang-Sieminski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gooch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghadiali
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.N.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fibers in the extracellular matrix enable long-range stress transmission between cells
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1410
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1418
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2Bjournal%26rft.volume%253D104%26rft.spage%253D1410%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2013.02.017%26rft_id%253Dinfo%253Apmid%252F23561517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2013.02.017&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23561517&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.5" data-doi="10.1073/pnas.1311312110">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ghosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Engelke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rycroft
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cassereau
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sethian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weaver
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liphardt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Rapid disorganization of mechanically interacting systems of mammary acini
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 <span class="cit-vol">
                                  111
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  658
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  663
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1311312110%26rft_id%253Dinfo%253Apmid%252F24379367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTEvMi82NTgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIwLjA3LjMwLjIyMzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Notbohm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lesman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rosakis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tirrell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ravichandran
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Microbuckling of fibrin provides a mechanism for cell mechanosensing
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of The Royal Society Interface
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  20150320
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nitsan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Drori
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lewis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cohen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tzlil
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanical communication in cardiac cell synchronized beating
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Physics
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.8" data-doi="10.1073/pnas.1514208113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ronceray
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Broedersz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fiber networks amplify active stress
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2827
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2832
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1514208113%26rft_id%253Dinfo%253Apmid%252F26921325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzExLzI4MjciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIwLjA3LjMwLjIyMzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Feng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sander
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Levine
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Stress-induced plasticity of dynamic collagen networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 ,
                                 <span class="cit-issue">
                                  842
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.10" data-doi="10.1073/pnas.1722619115">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ronceray
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malandrino
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lenz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Broedersz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cell contraction induces long-ranged stress stiffening in the extracellular matrix
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  4075
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4080
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1722619115%26rft_id%253Dinfo%253Apmid%252F29618614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzE2LzQwNzUiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIwLjA3LjMwLjIyMzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.11">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ban
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Franklin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wells
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chaudhuri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liphardt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanisms of Plastic Deformation in Collagen Networks Induced by Cellular Forces
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  450
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  461
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2Bjournal%26rft.volume%253D114%26rft.spage%253D450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lesman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Elastic Anisotropy Governs the Range of Cell-Induced Displacements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1152
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1164
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2Bjournal%26rft.volume%253D118%26rft.spage%253D1152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Natan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Koren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shelah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lesman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-range mechanical coupling of cells in 3D fibrin gels
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Molecular biology of the cell
                                 </abbr>
                                 , mbc.
                                 <span class="cit-fpage">
                                  E20
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  01-0079
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davidson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   DePalma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jayco
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.K.P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Mechanical intercellular communication via matrix-borne cell force transmission during vascular network formation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 , 2021.2008.2017.456669 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.15" data-doi="10.1016/j.yexcr.2015.10.033">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Doyle
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yamada
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanosensing via cell-matrix adhesions in 3D microenvironments
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Exp Cell Res
                                 </abbr>
                                 <span class="cit-vol">
                                  343
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  60
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  66
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExp%2BCell%2BRes%26rft.volume%253D343%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2015.10.033%26rft_id%253Dinfo%253Apmid%252F26524505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2015.10.033&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26524505&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.16" data-doi="10.1038/s41467-019-09709-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pakshir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Alizadehgiashi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Coelho
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCulloch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hinz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Dynamic fibroblast contractions attract remote macrophages in fibrillar collagen matrix
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Commun
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1850
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D10%26rft.spage%253D1850%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-019-09709-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-019-09709-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.17" data-doi="10.1016/0012-1606(82)90388-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Stopak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Harris
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.K
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Connective tissue morphogenesis by fibroblast traction: I. Tissue culture observations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Developmental biology
                                 </abbr>
                                 <span class="cit-vol">
                                  90
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  383
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  398
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1982
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DStopak%26rft.auinit1%253DD.%26rft.volume%253D90%26rft.issue%253D2%26rft.spage%253D383%26rft.epage%253D398%26rft.atitle%253DConnective%2Btissue%2Bmorphogenesis%2Bby%2Bfibroblast%2Btraction.%2BI.%2BTissue%2Bculture%2Bobservations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252882%252990388-8%26rft_id%253Dinfo%253Apmid%252F7075867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0012-1606(82)90388-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7075867&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1982NH74100017&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               18.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.18" data-doi="10.1006/excr.1995.1015">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Camarillo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.W
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Regulation of endothelial cell morphogenesis by integrins, mechanical forces, and matrix guidance pathways
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Exp Cell Res
                                 </abbr>
                                 <span class="cit-vol">
                                  216
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  113
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  123
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DDavis%26rft.auinit1%253DG.%2BE.%26rft.volume%253D216%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D123%26rft.atitle%253DRegulation%2Bof%2Bendothelial%2Bcell%2Bmorphogenesis%2Bby%2Bintegrins%252C%2Bmechanical%2Bforces%252C%2Band%2Bmatrix%2Bguidance%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fexcr.1995.1015%26rft_id%253Dinfo%253Apmid%252F7813611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1006/excr.1995.1015&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=7813611&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1995QB47300015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               19.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.19" data-doi="10.1073/pnas.1114781109">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Guo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ouyang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maslov
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Price
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-range mechanical force enables self-assembly of epithelial tubular patterns
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  5576
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  5582
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1114781109%26rft_id%253Dinfo%253Apmid%252F22427356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzE1LzU1NzYiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIwLjA3LjMwLjIyMzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               20.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.20" data-doi="10.1016/j.jbiomech.2014.03.031">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Duscher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Z.N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rennert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Januszyk
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rodrigues
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whitmore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Whittam
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Longaker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanotransduction and fibrosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of biomechanics
                                 </abbr>
                                 <span class="cit-vol">
                                  47
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1997
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2005
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiomechanics%26rft.volume%253D47%26rft.spage%253D1997%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiomech.2014.03.031%26rft_id%253Dinfo%253Apmid%252F24709567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jbiomech.2014.03.031&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24709567&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               21.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference 21. in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.21" data-doi="10.1073/pnas.1610347113">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yuan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tian
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wei
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Qu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Austin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Oriented collagen fibers direct tumor cell intravasation
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  11208
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11213
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1610347113%26rft_id%253Dinfo%253Apmid%252F27663743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQwLzExMjA4IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMC4wNy4zMC4yMjMxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               22.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.22" data-doi="10.1073/pnas.1617037114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ahmadzadeh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Webster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Behera
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jimenez Valencia
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wirtz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weeraratna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.T.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Modeling the two-way feedback between contractility and matrix realignment reveals a nonlinear mode of cancer cell invasion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc Natl Acad Sci U S A
                                 </abbr>
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  E1617
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  e1626
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProc%2BNatl%2BAcad%2BSci%2BU%2BS%2BA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1617037114%26rft_id%253Dinfo%253Apmid%252F28196892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzkvRTE2MTciO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMi8xNC8yMDIwLjA3LjMwLjIyMzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               23.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference 23. in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Long
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Niu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Du
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mechanical communication in fibrosis progression
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Trends Cell Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  32
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  70
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  90
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2022
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DTrends%2BCell%2BBiol%26rft.volume%253D32%26rft.spage%253D70%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               24.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference 24. in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.24">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wells
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.G.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Tissue mechanics and fibrosis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biochimica et Biophysica Acta (BBA)-Molecular Basis of Disease
                                 </abbr>
                                 <span class="cit-vol">
                                  1832
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  884
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  890
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa%2B%2528BBA%2529-Molecular%2BBasis%2Bof%2BDisease%26rft.volume%253D1832%26rft.spage%253D884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               25.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference 25. in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.25" data-doi="10.1016/j.ajpath.2010.11.076">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Conklin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eickhoff
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Riching
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pehlke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eliceiri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Provenzano
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedl
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Keely
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Aligned collagen is a prognostic signature for survival in human breast carcinoma
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am J Pathol
                                 </abbr>
                                 <span class="cit-vol">
                                  178
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1221
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1232
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2BOf%2BPathology%26rft.stitle%253DAm.%2BJ.%2BPathol.%26rft.aulast%253DConklin%26rft.auinit1%253DM.%2BW.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1221%26rft.epage%253D1232%26rft.atitle%253DAligned%2Bcollagen%2Bis%2Ba%2Bprognostic%2Bsignature%2Bfor%2Bsurvival%2Bin%2Bhuman%2Bbreast%2Bcarcinoma.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajpath.2010.11.076%26rft_id%253Dinfo%253Apmid%252F21356373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ajpath.2010.11.076&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21356373&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000288185600029&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               26.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference 26. in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zemel
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kemkemer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cell–Matrix and Cell–Cell Mechanical Interactions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cell and Matrix Mechanics
                                 </abbr>
                                 ,
                                 <span class="cit-fpage">
                                  136
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  165
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               27.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.27" data-doi="10.1016/j.bpj.2014.09.044">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abhilash
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wells
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shenoy
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Long-range force transmission in fibrous matrices enabled by tension-driven alignment of fibers
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  107
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2592
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2603
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2Bjournal%26rft.volume%253D107%26rft.spage%253D2592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2014.09.044%26rft_id%253Dinfo%253Apmid%252F25468338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2014.09.044&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25468338&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               28.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.28">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Clark
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.G.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Maitra
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jacques
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pérez-González
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Trepat
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Voituriez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vignjevic
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.M.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Viscoelastic relaxation of collagen networks provides a self-generated directional cue during collective migration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               29.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference 29. in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wershof
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Park
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Barry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jenkins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rullan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wilkins
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schlegelmilch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Roxanis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bates
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sahai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A FIJI macro for quantifying pattern in extracellular matrix
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Life Sci Alliance
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               30.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference 30. in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.30" data-doi="10.1016/j.bpj.2018.07.036">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sopher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tokash
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Natan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sharabi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shelah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tchaicheeyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lesman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nonlinear Elasticity of the ECM Fibers Facilitates Efficient Intercellular Communication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biophysical journal
                                 </abbr>
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1357
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1370
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiophysical%2Bjournal%26rft.volume%253D115%26rft.spage%253D1357%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2018.07.036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.bpj.2018.07.036&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               31.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference 31. in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mann
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sopher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goren
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shelah
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tchaicheeyan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lesman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Force chains in cell-cell mechanical communication
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J R Soc Interface
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  20190348
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ%2BR%2BSoc%2BInterface%26rft.volume%253D16%26rft.spage%253D20190348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               32.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference 32. in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.32" data-doi="10.1016/j.biomaterials.2013.10.059">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mohammadi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janmey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McCulloch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Lateral boundary mechanosensing by adherent cells in a collagen gel system
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biomaterials
                                 </abbr>
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1138
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1149
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiomaterials%26rft.volume%253D35%26rft.spage%253D1138%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.biomaterials.2013.10.059%26rft_id%253Dinfo%253Apmid%252F24215732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.biomaterials.2013.10.059&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24215732&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               33.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference 33. in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.33" data-doi="10.1038/nrm2786">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vicente-Manzanares
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Adelstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Horwitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.R
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Non-muscle myosin II takes centre stage in cell adhesion and migration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature reviews Molecular cell biology
                                 </abbr>
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  778
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  790
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DVicente-Manzanares%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D778%26rft.epage%253D790%26rft.atitle%253DNon-muscle%2Bmyosin%2BII%2Btakes%2Bcentre%2Bstage%2Bin%2Bcell%2Badhesion%2Band%2Bmigration.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2786%26rft_id%253Dinfo%253Apmid%252F19851336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrm2786&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19851336&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000271071200012&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               34.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.34" data-doi="10.1016/j.yexcr.2010.04.011">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lim
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sabouri-Ghomi
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Machacek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Waterman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danuser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Protrusion and actin assembly are coupled to the organization of lamellar contractile structures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Experimental cell research
                                 </abbr>
                                 <span class="cit-vol">
                                  316
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2027
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2041
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DExperimental%2Bcell%2Bresearch%26rft.stitle%253DExp%2BCell%2BRes%26rft.aulast%253DLim%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D316%26rft.issue%253D13%26rft.spage%253D2027%26rft.epage%253D2041%26rft.atitle%253DProtrusion%2Band%2Bactin%2Bassembly%2Bare%2Bcoupled%2Bto%2Bthe%2Borganization%2Bof%2Blamellar%2Bcontractile%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2010.04.011%26rft_id%253Dinfo%253Apmid%252F20406634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2010.04.011&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=20406634&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000279288100001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               35.
                              </span>
                              <div class="cit ref-cit ref-journal no-rev-xref" id="cit-2020.07.30.223149v2.35">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Besser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Brugge
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.S.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danuser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mapping the dynamics of force transduction at cell–cell junctions of epithelial clusters
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Elife
                                 </abbr>
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e03282
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D3%26rft.spage%253De03282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               36.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference 36. in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zaritsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tseng
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.-Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rabadán
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Krishna
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Overholtzer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danuser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hall
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Diverse roles of guanine nucleotide exchange factors in regulating collective cell migration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  The Journal of cell biology, jcb
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  201609095
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               37.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference 37. in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.37" data-doi="10.1038/nature08242">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Machacek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hodgson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Welch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Elliott
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pertz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Nalbant
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Abell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Johnson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hahn
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Danuser
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Coordination of Rho GTPase activities during cell protrusion
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  461
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  99
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  103
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMachacek%26rft.auinit1%253DM.%26rft.volume%253D461%26rft.issue%253D7260%26rft.spage%253D99%26rft.epage%253D103%26rft.atitle%253DCoordination%2Bof%2BRho%2BGTPase%2Bactivities%2Bduring%2Bcell%2Bprotrusion.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08242%26rft_id%253Dinfo%253Apmid%252F19693013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature08242&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19693013&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000269478800038&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               38.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference 38. in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.38" data-doi="10.1126/science.1116995">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Discher
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janmey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Tissue cells feel and respond to the stiffness of their substrate
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  310
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1139
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1143
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDischer%26rft.auinit1%253DD.%2BE.%26rft.volume%253D310%26rft.issue%253D5751%26rft.spage%253D1139%26rft.epage%253D1143%26rft.atitle%253DTissue%2BCells%2BFeel%2Band%2BRespond%2Bto%2Bthe%2BStiffness%2Bof%2BTheir%2BSubstrate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116995%26rft_id%253Dinfo%253Apmid%252F16293750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1MS8xMTM5IjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMC4wNy4zMC4yMjMxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               39.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference 39. in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bray
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.-A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Han
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Davis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Borgeson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartland
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kost-Alimova
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gustafsdottir
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gibson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carpenter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Cell Painting, a high-content image-based assay for morphological profiling using multiplexed fluorescent dyes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature protocols
                                 </abbr>
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1757
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bprotocols%26rft.volume%253D11%26rft.spage%253D1757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               40.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference 40. in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.40" data-doi="10.1371/JOURNAL.PBIO.2005970">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   McQuin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Goodman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chernyshev
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kamentsky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cimini
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Karhohs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Doan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ding
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rafelski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thirstrup
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wiegraebe
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Becker
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Caicedo
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Carpenter
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.E
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  CellProfiler 3.0: Next-generation image processing for biology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  16
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e2005970
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol%26rft.volume%253D16%26rft.spage%253De2005970%26rft_id%253Dinfo%253Adoi%252F10.1371%252FJOURNAL.PBIO.2005970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/JOURNAL.PBIO.2005970&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               41.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference 41. in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.41" data-doi="10.1089/adt.2014.573">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Edmondson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Broglie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Adcock
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Three-dimensional cell culture systems and their applications in drug discovery and cell-based biosensors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Assay Drug Dev Technol
                                 </abbr>
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  207
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  218
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAssay%2BDrug%2BDev%2BTechnol%26rft.volume%253D12%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fadt.2014.573%26rft_id%253Dinfo%253Apmid%252F24831787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1089/adt.2014.573&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24831787&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               42.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference 42. in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Desai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Grenert
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.P.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Malato
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rezvani
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Español-Suñer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Willenbring
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weaver
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.T
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Physiological ranges of matrix rigidity modulate primary mouse hepatocyte function in part through hepatocyte nuclear factor 4 alpha
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Hepatology
                                 </abbr>
                                 <span class="cit-vol">
                                  64
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  261
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  275
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHepatology%26rft.volume%253D64%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               43.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference 43. in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Q.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janmey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.X.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Conti
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacKintosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.C
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nonlinear elasticity of stiff filament networks: strain stiffening, negative normal stress, and filament alignment in fibrin gels
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J Phys Chem B
                                 </abbr>
                                 <span class="cit-vol">
                                  113
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  3799
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  3805
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.stitle%253DJournal%2Bof%2BPhysical%2BChemistry.%2BB%252C%2BCondensed%2BMatter%252C%2BMaterials%252C%2BSurfaces%252C%2BInterfaces%2Band%2BBiophysical%26rft.aulast%253DKang%26rft.auinit1%253DH.%26rft.volume%253D113%26rft.issue%253D12%26rft.spage%253D3799%26rft.epage%253D3805%26rft.atitle%253DNonlinear%2Belasticity%2Bof%2Bstiff%2Bfilament%2Bnetworks%253A%2Bstrain%2Bstiffening%252C%2Bnegative%2Bnormal%2Bstress%252C%2Band%2Bfilament%2Balignment%2Bin%2Bfibrin%2Bgels.%26rft_id%253Dinfo%253Apmid%252F19243107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19243107&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               44.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference 44. in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wade
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   R.J.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Burdick
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Engineering ECM signals into biomaterials
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Materials Today
                                 </abbr>
                                 <span class="cit-vol">
                                  15
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  454
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  459
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMaterials%2BToday%26rft.volume%253D15%26rft.spage%253D454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               45.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference 45. in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jones
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Sun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jiao
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Heterogeneous force network in 3D cellularized collagen networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Phys Biol
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  066001
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhys%2BBiol%26rft.volume%253D13%26rft.spage%253D066001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               46.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference 46. in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.46" data-doi="10.1371/journal.pone.0005902">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Vader
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kabla
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Weitz
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mahadevan
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   L
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Strain-induced alignment in collagen gels
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PLoS One
                                 </abbr>
                                 <span class="cit-vol">
                                  4
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  e5902
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVader%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D6%26rft.spage%253De5902%26rft.epage%253De5902%26rft.atitle%253DStrain-induced%2Balignment%2Bin%2Bcollagen%2Bgels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005902%26rft_id%253Dinfo%253Apmid%252F19529768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005902&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=19529768&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               47.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference 47. in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.47" data-doi="10.1038/nature03521">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Storm
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pastore
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   MacKintosh
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lubensky
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Janmey
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.A
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Nonlinear elasticity in biological gels
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  435
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  191
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  194
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DStorm%26rft.auinit1%253DC.%26rft.volume%253D435%26rft.issue%253D7039%26rft.spage%253D191%26rft.epage%253D194%26rft.atitle%253DNonlinear%2Belasticity%2Bin%2Bbiological%2Bgels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03521%26rft_id%253Dinfo%253Apmid%252F15889088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature03521&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=15889088&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000229021100036&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               48.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference 48. in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.48" data-doi="10.1126/science.1226929">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Friedmann-Morvinski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bushong
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ke
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Soda
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Marumoto
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Singer
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   O.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ellisman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.H.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Verma
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.M
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Dedifferentiation of neurons and astrocytes by oncogenes can induce gliomas in mice
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Science
                                 </abbr>
                                 <span class="cit-vol">
                                  338
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1080
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1084
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1226929%26rft_id%253Dinfo%253Apmid%252F23087000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzgvNjExMC8xMDgwIjtzOjQ6ImF0b20iO3M6NDg6Ii9iaW9yeGl2L2Vhcmx5LzIwMjIvMDIvMTQvMjAyMC4wNy4zMC4yMjMxNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               49.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference 49. in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.49" data-doi="10.1038/nmeth.2019">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schindelin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Arganda-Carreras
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Frise
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kaynig
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Longair
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Pietzsch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   T.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Preibisch
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Rueden
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Saalfeld
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmid
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tinevez
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.Y.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   White
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hartenstein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   V.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Eliceiri
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Tomancak
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cardona
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Fiji: an open-source platform for biological-image analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  676
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  682
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchindelin%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D676%26rft.epage%253D682%26rft.atitle%253DFiji%253A%2Ban%2Bopen-source%2Bplatform%2Bfor%2Bbiological-image%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2019%26rft_id%253Dinfo%253Apmid%252F22743772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.2019&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22743772&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000305942200021&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               50.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference 50. in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cordelieres
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   F.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Jackson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  3D objects counter, ImageJ: Image Processing and Analysis in Java
                                 </span>
                                 . Available at rsb. info. nih. gov/ij/plugins/track/objects. html (verified 5 January 2011) (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               51.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference 51. in text" id="ref-51">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.51" data-doi="10.1016/0304-4076(92)90104-Y">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Kwiatkowski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   D.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Phillips
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Schmidt
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   P.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Shin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Testing the null hypothesis of stationarity against the alternative of a unit root: How sure are we that economic time series have a unit root?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of econometrics
                                 </abbr>
                                 <span class="cit-vol">
                                  54
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  159
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  178
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1992
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Beconometrics%26rft.volume%253D54%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4076%252892%252990104-Y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/0304-4076(92)90104-Y&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=A1992JW38400007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               52.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference 52. in text" id="ref-52">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.52">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Cheung
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.-W.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lai
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.S
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Lag order and critical values of the augmented Dickey–Fuller test
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Business & Economic Statistics
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  277
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  280
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1995
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBusiness%2B%2526%2BEconomic%2BStatistics%26rft.volume%253D13%26rft.spage%253D277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               53.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference 53. in text" id="ref-53">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.07.30.223149v2.53" data-doi="10.1038/nmeth.3685">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Steinwachs
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Metzner
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Skodzek
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   N.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Thievessen
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mark
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Münster
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Aifantis
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.E.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Fabry
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Three-dimensional force microscopy of cells in biopolymer networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat Methods
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  171
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  176
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat%2BMethods%26rft.volume%253D13%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3685%26rft_id%253Dinfo%253Apmid%252F26641311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nmeth.3685&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26641311&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F02%2F14%2F2020.07.30.223149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.07.30.223149v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2022.02.14.480411v1" title="Genetic load in incomplete lupus erythematosus" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.14.480315v1" title="Protein undernutrition reduces the efficacy of praziquantel in a murine model of Schistosoma mansoni infection" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted February 14, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.07.30.223149v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2020.07.30.223149v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/02/14/2020.07.30.223149.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.07.30.223149v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.07.30.223149v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2401796"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-MgT9G-jIZhL9WWY5-ATGg1zOGIoNFvyoNnEg3mNNgXU"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484475183"/>
                                   <input type="hidden" name="captcha_token" value="96f44f893973caa57f5717076f1a1476"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2401796--3" data-node-nid="2401796" id="node-2401796--41051548917" data-pisa="biorxiv;2020.07.30.223149v2" data-pisa-master="biorxiv;2020.07.30.223149" data-seqnum="2401796" data-apath="/biorxiv/early/2022/02/14/2020.07.30.223149.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Assaf
                                  </span>
                                  <span class="nlm-surname">
                                   Nahum
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Yoni
                                  </span>
                                  <span class="nlm-surname">
                                   Koren
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Bar
                                  </span>
                                  <span class="nlm-surname">
                                   Ergaz
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Sari
                                  </span>
                                  <span class="nlm-surname">
                                   Natan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Shahar
                                  </span>
                                  <span class="nlm-surname">
                                   Goren
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Avraham
                                  </span>
                                  <span class="nlm-surname">
                                   Kolel
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Sankar
                                  </span>
                                  <span class="nlm-surname">
                                   Jagadeeshan
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Moshe
                                  </span>
                                  <span class="nlm-surname">
                                   Elkabets
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Ayelet
                                  </span>
                                  <span class="nlm-surname">
                                   Lesman
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="9">
                                  <span class="nlm-given-names">
                                   Assaf
                                  </span>
                                  <span class="nlm-surname">
                                   Zaritsky
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.07.30.223149;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.07.30.223149
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&title=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&title=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&text=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&t=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&title=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&title=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&title=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2401796--5" data-node-nid="2401796" id="citation-node-2401796--6271442675" data-pisa="biorxiv;2020.07.30.223149v2" data-pisa-master="biorxiv;2020.07.30.223149" data-seqnum="2401796" data-apath="/biorxiv/early/2022/02/14/2020.07.30.223149.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Inference of long-range cell-cell mechanical communication from ECM remodeling fluctuations
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Assaf
                                      </span>
                                      <span class="nlm-surname">
                                       Nahum
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Yoni
                                      </span>
                                      <span class="nlm-surname">
                                       Koren
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Bar
                                      </span>
                                      <span class="nlm-surname">
                                       Ergaz
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Sari
                                      </span>
                                      <span class="nlm-surname">
                                       Natan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Shahar
                                      </span>
                                      <span class="nlm-surname">
                                       Goren
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Avraham
                                      </span>
                                      <span class="nlm-surname">
                                       Kolel
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Sankar
                                      </span>
                                      <span class="nlm-surname">
                                       Jagadeeshan
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Moshe
                                      </span>
                                      <span class="nlm-surname">
                                       Elkabets
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Ayelet
                                      </span>
                                      <span class="nlm-surname">
                                       Lesman
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="9">
                                      <span class="nlm-given-names">
                                       Assaf
                                      </span>
                                      <span class="nlm-surname">
                                       Zaritsky
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.07.30.223149;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.07.30.223149
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Nahum%2C+Assaf&rft.au=Koren%2C+Yoni&rft.au=Ergaz%2C+Bar&rft.au=Natan%2C+Sari&rft.au=Goren%2C+Shahar&rft.au=Kolel%2C+Avraham&rft.au=Jagadeeshan%2C+Sankar&rft.au=Elkabets%2C+Moshe&rft.au=Lesman%2C+Ayelet&rft.au=Zaritsky%2C+Assaf">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2401796/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2401796/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2401796/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2401796/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2401796/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2401796/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2401796/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2401796/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2401796/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2401796/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2401796/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2401796/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&count=horizontal&via=&text=Inference%20of%20long-range%20cell-cell%20mechanical%20communication%20from%20ECM%20remodeling%20fluctuations&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.07.30.223149v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.07.30.223149v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/cell-biology" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Cell Biology
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
