<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.medrxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2021.07.12.21260400v2.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2021.07.12.21260400v2.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2021.07.12.21260400v2.ppt"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/medrxiv/early/2022/03/16/2021.07.12.21260400.atom"/>
  <meta name="HW.pisa" content="medrxiv;2021.07.12.21260400v2"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Set-based rare variant association tests for biobank scale sequencing data sets"/>
  <meta name="DC.Identifier" content="10.1101/2021.07.12.21260400"/>
  <meta name="DC.Date" content="2022-03-16"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/"/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="UK Biobank has released the large-scale whole-exome sequencing (WES) data, but the best practices remain unclear for rare variant tests, and an existing approach, SAIGE-GENE, can have inflated type I error rates with high computation costs. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency compared to SAIGE-GENE. In the analysis of UKBB WES data of 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene-phenotype associations. In addition, we showed that incorporating multiple MAF cutoffs and functional annotations can help identify novel gene-phenotype associations.

### Competing Interest Statement

B.M.N. is a member of Deep Genomics Scientific Advisory Board, has received travel expenses from Illumina, and also serves as a consultant for Avanir and Trigeminal solutions. K.J.K is a consultant for Vor Biopharma.

### Funding Statement

SL was supported by Brain Pool Plus (BP+, Brain Pool+) Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Science and ICT (2020H1D3A2A03100666, S.L). WB and ZZ were supported by NIH R01 HG008773. WZ was supported by the National Human Genome Research Institute of the National Institutes of Health under award number T32HG010464. We thank Dr. Alkes Price for the constructive comments and suggestions.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

This research has been conducted using the UK Biobank Resource under application number 45227.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SAIGE-GENE+ is implemented as an open-source R package available at <https://github.com/saigegit/SAIGE>. SAIGE-GENE is available at <https://github.com/weizhouUMICH/SAIGE/master>. The summary statistics and QQ plots for 30 quantitative phenotypes and 141 binary phenotypes in UK Biobank by SAIGE-GENE+ are currently available for public download at [https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1] 

[https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1]

 [1]: https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt"/>
  <meta name="DC.Contributor" content="Wei Zhou"/>
  <meta name="DC.Contributor" content="Wenjian Bi"/>
  <meta name="DC.Contributor" content="Zhangchen Zhao"/>
  <meta name="DC.Contributor" content="Kushal K. Dey"/>
  <meta name="DC.Contributor" content="Karthik A. Jagadeesh"/>
  <meta name="DC.Contributor" content="Konrad J. Karczewski"/>
  <meta name="DC.Contributor" content="Mark J. Daly"/>
  <meta name="DC.Contributor" content="Benjamin M. Neale"/>
  <meta name="DC.Contributor" content="Seunggeun Lee"/>
  <meta name="article:published_time" content="2022-03-16"/>
  <meta name="citation_title" content="Set-based rare variant association tests for biobank scale sequencing data sets"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>UK Biobank has released the large-scale whole-exome sequencing (WES) data, but the best practices remain unclear for rare variant tests, and an existing approach, SAIGE-GENE, can have inflated type I error rates with high computation costs. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency compared to SAIGE-GENE. In the analysis of UKBB WES data of 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene-phenotype associations. In addition, we showed that incorporating multiple MAF cutoffs and functional annotations can help identify novel gene-phenotype associations.</p>"/>
  <meta name="citation_journal_title" content="medRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory Press"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="medrxiv;2021.07.12.21260400v2"/>
  <meta name="citation_id" content="2021.07.12.21260400v2"/>
  <meta name="citation_public_url" content="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2"/>
  <meta name="citation_abstract_html_url" content="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2.abstract"/>
  <meta name="citation_full_html_url" content="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2.full"/>
  <meta name="citation_pdf_url" content="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2021.07.12.21260400"/>
  <meta name="citation_num_pages" content="11"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_firstpage" content="2021.07.12.21260400"/>
  <meta name="citation_author" content="Wei Zhou"/>
  <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital"/>
  <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_email" content="wzhou@broadinstitute.org"/>
  <meta name="citation_author_email" content="lee7801@snu.ac.kr"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7719-0859"/>
  <meta name="citation_author" content="Wenjian Bi"/>
  <meta name="citation_author_institution" content="Department of Medical Genetics, School of Basic Medical Sciences, Peking University"/>
  <meta name="citation_author_institution" content="Center for Statistical Genetics, University of Michigan School of Public Health"/>
  <meta name="citation_author_institution" content="Department of Biostatistics, University of Michigan School of Public Health"/>
  <meta name="citation_author" content="Zhangchen Zhao"/>
  <meta name="citation_author_institution" content="Center for Statistical Genetics, University of Michigan School of Public Health"/>
  <meta name="citation_author_institution" content="Department of Biostatistics, University of Michigan School of Public Health"/>
  <meta name="citation_author" content="Kushal K. Dey"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, Harvard T. H. Chan School of Public Health"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3520-2345"/>
  <meta name="citation_author" content="Karthik A. Jagadeesh"/>
  <meta name="citation_author_institution" content="Department of Epidemiology, Harvard T. H. Chan School of Public Health"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0957-812X"/>
  <meta name="citation_author" content="Konrad J. Karczewski"/>
  <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital"/>
  <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2878-4671"/>
  <meta name="citation_author" content="Mark J. Daly"/>
  <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital"/>
  <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland, Helsinki Institute of Life Sciences, University of Helsinki"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0949-8752"/>
  <meta name="citation_author" content="Benjamin M. Neale"/>
  <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital"/>
  <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_institution" content="Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1513-6077"/>
  <meta name="citation_author" content="Seunggeun Lee"/>
  <meta name="citation_author_institution" content="Center for Statistical Genetics, University of Michigan School of Public Health"/>
  <meta name="citation_author_institution" content="Department of Biostatistics, University of Michigan School of Public Health"/>
  <meta name="citation_author_institution" content="Graduate School of Data Science, Seoul National University"/>
  <meta name="citation_author_email" content="wzhou@broadinstitute.org"/>
  <meta name="citation_author_email" content="lee7801@snu.ac.kr"/>
  <meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8097-3878"/>
  <meta name="citation_reference" content="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)."/>
  <meta name="citation_reference" content="Li, X. et al. Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale. Nat. Genet. 52, 969–983 (2020)."/>
  <meta name="citation_reference" content="Liu, Y. & Xie, J. Cauchy combination test: a powerful test with analytic p-value calculation under arbitrary dependency structures. J. Am. Stat. Assoc. 115, 393–402 (2020)."/>
  <meta name="citation_reference" content="Zhou, W. et al. Scalable generalized linear mixed model for region-based association tests in large biobanks and cohorts. Nat. Genet. 52, 634–639 (2020)."/>
  <meta name="citation_reference" content="Wu, M. C. et al. Rare-variant association testing for sequencing data with the sequence kernel association test. Am. J. Hum. Genet. 89, 82–93 (2011)."/>
  <meta name="citation_reference" content="Lee, S., Wu, M. C. & Lin, X. Optimal tests for rare variant effects in sequencing association studies. Biostatistics 13, 762–775 (2012)."/>
  <meta name="citation_reference" content="Mbatchou, J. et al. Computationally efficient whole-genome regression for quantitative and binary traits. Nat. Genet. 53, 1097–1103 (2021)."/>
  <meta name="citation_reference" content="Karczewski, K. J. et al. Systematic single-variant and gene-based association testing of 3,700 phenotypes in 281,850 UK Biobank exomes. bioRxiv (2021) doi:10.1101/2021.06.19.21259117."/>
  <meta name="citation_reference" content="Povysil, G. et al. Rare-variant collapsing analyses for complex traits: guidelines and applications. Nat. Rev. Genet. 20, 747–759 (2019)."/>
  <meta name="citation_reference" content="Liu, Y. et al. ACAT: A Fast and Powerful p Value Combination Method for Rare-Variant Analysis in Sequencing Studies. Am. J. Hum. Genet. 104, 410–421 (2019)."/>
  <meta name="citation_reference" content="Klein, A. P. et al. Genome-wide meta-analysis identifies five new susceptibility loci for pancreatic cancer. Nat. Commun. 9, 556 (2018)."/>
  <meta name="citation_reference" content="Wedell, A., Ritzén, E. M., Haglund-Stengler, B. & Luthman, H. Steroid 21-hydroxylase deficiency: three additional mutated alleles and establishment of phenotype-genotype relationships of common mutations. Proc. Natl. Acad. Sci. U. S. A. 89, 7232–7236 (1992)."/>
  <meta name="citation_reference" content="Ruth, K. S. et al. Using human genetics to understand the disease impacts of testosterone in men and women. Nat. Med. 26, 252–258 (2020)."/>
  <meta name="citation_reference" content="Chambers, I. et al. Nanog safeguards pluripotency and mediates germline development. Nature 450, 1230–1234 (2007)."/>
  <meta name="citation_reference" content="Hart, A. H. et al. The pluripotency homeobox gene NANOG is expressed in human germ cell tumors. Cancer 104, 2092–2098 (2005)."/>
  <meta name="citation_reference" content="Wang, K., Li, M. & Hakonarson, H. ANNOVAR: Functional annotation of genetic variants from next-generation sequencing data."/>
  <meta name="citation_reference" content="Gagliano Taliun, S. A. et al. Exploring and visualizing large-scale genetic associations by using PheWeb. Nat. Genet. 52, 550–552 (2020)."/>
  <meta name="twitter:title" content="Set-based rare variant association tests for biobank scale sequencing data sets"/>
  <meta name="twitter:site" content="@medrxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="twitter:description" content="UK Biobank has released the large-scale whole-exome sequencing (WES) data, but the best practices remain unclear for rare variant tests, and an existing approach, SAIGE-GENE, can have inflated type I error rates with high computation costs. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency compared to SAIGE-GENE. In the analysis of UKBB WES data of 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene-phenotype associations. In addition, we showed that incorporating multiple MAF cutoffs and functional annotations can help identify novel gene-phenotype associations.

### Competing Interest Statement

B.M.N. is a member of Deep Genomics Scientific Advisory Board, has received travel expenses from Illumina, and also serves as a consultant for Avanir and Trigeminal solutions. K.J.K is a consultant for Vor Biopharma.

### Funding Statement

SL was supported by Brain Pool Plus (BP+, Brain Pool+) Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Science and ICT (2020H1D3A2A03100666, S.L). WB and ZZ were supported by NIH R01 HG008773. WZ was supported by the National Human Genome Research Institute of the National Institutes of Health under award number T32HG010464. We thank Dr. Alkes Price for the constructive comments and suggestions.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

This research has been conducted using the UK Biobank Resource under application number 45227.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SAIGE-GENE+ is implemented as an open-source R package available at <https://github.com/saigegit/SAIGE>. SAIGE-GENE is available at <https://github.com/weizhouUMICH/SAIGE/master>. The summary statistics and QQ plots for 30 quantitative phenotypes and 141 binary phenotypes in UK Biobank by SAIGE-GENE+ are currently available for public download at [https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1] 

[https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1]

 [1]: https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt"/>
  <meta name="og-title" property="og:title" content="Set-based rare variant association tests for biobank scale sequencing data sets"/>
  <meta name="og-url" property="og:url" content="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2"/>
  <meta name="og-site-name" property="og:site_name" content="medRxiv"/>
  <meta name="og-description" property="og:description" content="UK Biobank has released the large-scale whole-exome sequencing (WES) data, but the best practices remain unclear for rare variant tests, and an existing approach, SAIGE-GENE, can have inflated type I error rates with high computation costs. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency compared to SAIGE-GENE. In the analysis of UKBB WES data of 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene-phenotype associations. In addition, we showed that incorporating multiple MAF cutoffs and functional annotations can help identify novel gene-phenotype associations.

### Competing Interest Statement

B.M.N. is a member of Deep Genomics Scientific Advisory Board, has received travel expenses from Illumina, and also serves as a consultant for Avanir and Trigeminal solutions. K.J.K is a consultant for Vor Biopharma.

### Funding Statement

SL was supported by Brain Pool Plus (BP+, Brain Pool+) Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Science and ICT (2020H1D3A2A03100666, S.L). WB and ZZ were supported by NIH R01 HG008773. WZ was supported by the National Human Genome Research Institute of the National Institutes of Health under award number T32HG010464. We thank Dr. Alkes Price for the constructive comments and suggestions.

### Author Declarations

I confirm all relevant ethical guidelines have been followed, and any necessary IRB and/or ethics committee approvals have been obtained.

Yes

The details of the IRB/oversight body that provided approval or exemption for the research described are given below:

This research has been conducted using the UK Biobank Resource under application number 45227.

I confirm that all necessary patient/participant consent has been obtained and the appropriate institutional forms have been archived, and that any patient/participant/sample identifiers included were not known to anyone (e.g., hospital staff, patients or participants themselves) outside the research group so cannot be used to identify individuals.

Yes

I understand that all clinical trials and any other prospective interventional studies must be registered with an ICMJE-approved registry, such as ClinicalTrials.gov. I confirm that any such study reported in the manuscript has been registered and the trial registration ID is provided (note: if posting a prospective study registered retrospectively, please provide a statement in the trial ID field explaining why the study was not registered in advance).

Yes

I have followed all appropriate research reporting guidelines and uploaded the relevant EQUATOR Network research reporting checklist(s) and other pertinent material as supplementary files, if applicable.

Yes

SAIGE-GENE+ is implemented as an open-source R package available at <https://github.com/saigegit/SAIGE>. SAIGE-GENE is available at <https://github.com/weizhouUMICH/SAIGE/master>. The summary statistics and QQ plots for 30 quantitative phenotypes and 141 binary phenotypes in UK Biobank by SAIGE-GENE+ are currently available for public download at [https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1] 

[https://storage.googleapis.com/leelabsg/saige-gene/reformat\_all\_withPhenoDetails.txt][1]

 [1]: https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.medrxiv.org/sites/default/files/images/medrxiv_logo_homepage7-5-small-test-up.png"/>
  <meta name="citation_date" content="2022-03-16"/>
  <meta name="description" content="medRxiv - The Preprint Server for Health Sciences"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2"/>
  <link rel="shortlink" href="https://www.medrxiv.org/node/490258"/>
  <title>
   Set-based rare variant association tests for biobank scale sequencing data sets | medRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__MF4P_i8lKWP7DQR7GmjDAVTRBKLnzxvs1Dvton-Awi0__yi5OflfckG_G8-Li4dDQQ3wO9cGIcdPx_YQ7ZZDuj5w__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.medrxiv.org/sites/default/files/advagg_css/css__5mxrBK4DuyvC2MR5irx_TzVn9c8vr4vIjmBXjunHfxE__rtQnXv8GOc-AsynbhzSlQBD_2lVxAkB6XEFUCK72qXc__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.css" media="all"/>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-141606356-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"eAwpWRnVRlmQ2F6X-27bibZ-6EjT9OnVJE3QLTvxjN4"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490258","apath":"\/medrxiv\/early\/2022\/03\/16\/2021.07.12.21260400.atom","pisa":"medrxiv;2021.07.12.21260400v2","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"medrxiv;2021.07.12.21260400v2"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"medRxiv","url":"https:\/\/www.medrxiv.org\/content\/10.1101\/2021.07.12.21260400v2","title":"Set-based rare variant association tests for biobank scale sequencing data sets","identifier":"node\/490258"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QMx+6krJZL2mHvf\\\\\\\/CcAVCxohUyaL7demf+J\\\\\\\/D1P6CXtXcVEck7UMQ9d5IgwdLuqKp\\\\\\\/a7DghvBBbB2jEam4W4cdiwXTpa+kxr4TtvM8sCf\\\\\\\/bo9YEiVnbSd8gzRaQKPfKcxzSchGldk+5j2lHkTojgukNHJwNqUXaaDPpHl+CJYKn6RDSoRZEujaTJ5D18pBvSn5plGEEPwyU5JoAhF\\\\\\\/9GsQpZZAHcKkdON29Xi18KEO5xumdsTRHiV1Fa5bcl0oXezfjRCVfDLqEMXGo+JOfU6y6XLHM+LhdgUSGY51TUpxTwFAgOSB2jcHKXzvSZGOd3tWcjl\\\\\\\/ms4Y6jlyH5\\\\\\\/M8XGbrycCsYcjJ4+CAHX5j9s0MXD2\\\\\\\/zaDYFErN\\\\\\\/8E0ExQfCitRx76tFuCUBjUmOIPr8ZwsOEj1P6ll9Hl3ocoLoMz6Uml372HURoiZwN9TqsCSPHC60QS18xg79w2XfGURODC0aHUjnDd8iyRwXidGpKxu0iX7dOusW9X7\\\\\\\/eh5zMB65u+8rv0eegYUYh0zCGASok+3FcUJBO5ruq\\\\\\\/EigTDnEi3ysy8rcHCxOEPs9XE8tWh6LWpKaMcbsHj5cxuSjKq8XWyIzVtN93EsJ91+SmH01m4ecxPBZ6LGK74rO1S1fQ\\\\\\\/t3uQb47MxEcLOJdhhJwaoDxFvGrc9hC7+ocOaPJXSb7Xh93LxD1zms4wwo1suT\\\\\\\/IGmft\\\\\\\/gHv37fQ\\\\\\\/jBmhYauUwFUS3ElbpXiMAa9WdobDkTO6l8NBF32rVi06qjt3qtMTFEOInaMc4WyjOP23xeEUaf\\\\\\\/7YUW0at2cyfcZTSKYf6qeLLa8fKkywnGdtgS6RkMP\\\\\\\/ptJzAbBK9yASuPm0pufm6rwMl23BC\\\\\\\/5k3CjaJrEOFKBKqSApajxkd7Ix6XFlWMU6o8jIdRfGR4Nohf54SVA8KwJqsFwPpLjebts1RLUMp2qhPRs24+sZe4\\\\\\\/CNo59OLHPSp9f8pcZHQbNDXQQhhXdnWlq7hW575cHDmyMUIM1WJ6RB4zzncSTbfGUNprAL+fa3HfnsRIA2oMjSwhAxlLRHhCT3hNP5vvCtBK42CmEzMMNQlK4EcMezijXU2uW9V5Fz8lhK7PTydYYKsRETATiBHderACgviSVeG7EifrCv12h1sCOFy2ptNb3uK7BynFb8KtkUvatITZhA3oDY200vj0NBOJTii\\\\\\\/3w3bA+oLVh2KAF59xjmxVdXdwntWPD7DtqL9munn\\\\\\\/DDY5eMDMAES3g+xWTZwRNTbltsR91OtZBMMoOMEzvXDsRz+jSoUPKngiXoKo9gkAYRUtye9KBv2Faq1QYALT399+P1E2KA3qcbNC\\\\\\\/KIEXDJgfSbO5KovewxwvWzESF4FZ2oYCahFA06ZEglopZANeiSH+kOOttsdao=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Fgzj1i\\\\\\\/4iYm8ccwzA6xvaQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u0022f01329db85216732faa980a40d99e2512a969cf07bfa3cf549990361ec6711c8\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2021.07.12.21260400v2.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.medrxiv.org\/sites\/default\/files\/medrxiv_internal_logo.png"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"medrxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-490258 node-type-highwire-article context-content hw-default-jcode-medrxiv hw-article-type-article">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P4HH5NV" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-P4HH5NV');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="medRxiv" src="https://www.medrxiv.org/sites/default/files/medrxiv_internal_logo.png"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/content/about-medrxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2021.07.12.21260400v2.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_837890992">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_837890992" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_2010801726" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-v2aP4f6KN0T5axCxcQQOYlJDgnKspaFdz3j47a0DOEo"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node490258" data-node-nid="490258" id="node-490258--21434075444" data-pisa="medrxiv;2021.07.12.21260400v2" data-pisa-master="medrxiv;2021.07.12.21260400" data-apath="/medrxiv/early/2022/03/16/2021.07.12.21260400.atom" data-hw-author-tooltip-instance="">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-medrxiv-article-top clearfix has-author-tooltip">
                   <h1 class="highwire-cite-title" id="page-title">
                    Set-based rare variant association tests for biobank scale sequencing data sets
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0">
                      <a href="http://orcid.org/0000-0001-7719-0859" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Wei
                      </span>
                      <span class="nlm-surname">
                       Zhou
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Wenjian
                      </span>
                      <span class="nlm-surname">
                       Bi
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Zhangchen
                      </span>
                      <span class="nlm-surname">
                       Zhao
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3">
                      <a href="http://orcid.org/0000-0002-3520-2345" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Kushal K.
                      </span>
                      <span class="nlm-surname">
                       Dey
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4">
                      <a href="http://orcid.org/0000-0002-0957-812X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Karthik A.
                      </span>
                      <span class="nlm-surname">
                       Jagadeesh
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5">
                      <a href="http://orcid.org/0000-0003-2878-4671" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Konrad J.
                      </span>
                      <span class="nlm-surname">
                       Karczewski
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6">
                      <a href="http://orcid.org/0000-0002-0949-8752" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Mark J.
                      </span>
                      <span class="nlm-surname">
                       Daly
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7">
                      <a href="http://orcid.org/0000-0003-1513-6077" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Benjamin M.
                      </span>
                      <span class="nlm-surname">
                       Neale
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8">
                      <a href="http://orcid.org/0000-0002-8097-3878" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon">
                       <span class="hw-icon-orcid hw-icon-color-orcid">
                       </span>
                       <span class="title element-invisible">
                        View ORCID Profile
                       </span>
                      </a>
                      <span class="nlm-given-names">
                       Seunggeun
                      </span>
                      <span class="nlm-surname">
                       Lee
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2021.07.12.21260400
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-490258--21434075444" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Wei Zhou
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Analytic and Translational Genetics Unit, Massachusetts General Hospital
                       </span>
                       , Boston, Massachusetts,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Program in Medical and Population Genetics, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wei%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhou%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AWei%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0001-7719-0859" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Wei Zhou
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:wzhou@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        wzhou@broadinstitute.org
                       </a>
                       <a href="mailto:lee7801@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">
                        lee7801@snu.ac.kr
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Wenjian Bi
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        4
                       </span>
                       <span class="nlm-institution">
                        Department of Medical Genetics, School of Basic Medical Sciences, Peking University
                       </span>
                       , Beijing,
                       <span class="nlm-country">
                        China
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Center for Statistical Genetics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Department of Biostatistics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Wenjian%2BBi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Bi%20W&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AWenjian%2BBi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Zhangchen Zhao
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Center for Statistical Genetics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Department of Biostatistics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Zhangchen%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Zhao%20Z&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AZhangchen%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-3">
                    <div class="author-tooltip-name">
                     Kushal K. Dey
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Department of Epidemiology, Harvard T. H. Chan School of Public Health
                       </span>
                       , Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Kushal%2BK.%2BDey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Dey%20KK&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKushal%2BK.%2BDey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-3520-2345" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Kushal K. Dey
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-4">
                    <div class="author-tooltip-name">
                     Karthik A. Jagadeesh
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        7
                       </span>
                       <span class="nlm-institution">
                        Department of Epidemiology, Harvard T. H. Chan School of Public Health
                       </span>
                       , Boston, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Karthik%2BA.%2BJagadeesh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Jagadeesh%20KA&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKarthik%2BA.%2BJagadeesh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0957-812X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Karthik A. Jagadeesh
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-5">
                    <div class="author-tooltip-name">
                     Konrad J. Karczewski
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Analytic and Translational Genetics Unit, Massachusetts General Hospital
                       </span>
                       , Boston, Massachusetts,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Program in Medical and Population Genetics, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Konrad%2BJ.%2BKarczewski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Karczewski%20KJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AKonrad%2BJ.%2BKarczewski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-2878-4671" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Konrad J. Karczewski
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-6">
                    <div class="author-tooltip-name">
                     Mark J. Daly
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Analytic and Translational Genetics Unit, Massachusetts General Hospital
                       </span>
                       , Boston, Massachusetts,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Program in Medical and Population Genetics, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        8
                       </span>
                       <span class="nlm-institution">
                        Institute for Molecular Medicine Finland, Helsinki Institute of Life Sciences, University of Helsinki
                       </span>
                       , Helsinki,
                       <span class="nlm-country">
                        Finland
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Mark%2BJ.%2BDaly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Daly%20MJ&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3AMark%2BJ.%2BDaly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0002-0949-8752" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Mark J. Daly
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-7">
                    <div class="author-tooltip-name">
                     Benjamin M. Neale
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Analytic and Translational Genetics Unit, Massachusetts General Hospital
                       </span>
                       , Boston, Massachusetts,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Program in Medical and Population Genetics, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Stanley Center for Psychiatric Research, Broad Institute of Harvard and MIT
                       </span>
                       , Cambridge, MA,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Benjamin%2BM.%2BNeale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Neale%20BM&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ABenjamin%2BM.%2BNeale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link last">
                      <a href="http://orcid.org/0000-0003-1513-6077" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Benjamin M. Neale
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-8">
                    <div class="author-tooltip-name">
                     Seunggeun Lee
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        5
                       </span>
                       <span class="nlm-institution">
                        Center for Statistical Genetics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        6
                       </span>
                       <span class="nlm-institution">
                        Department of Biostatistics, University of Michigan School of Public Health
                       </span>
                       , Ann Arbor, MI,
                       <span class="nlm-country">
                        USA
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        9
                       </span>
                       <span class="nlm-institution">
                        Graduate School of Data Science, Seoul National University
                       </span>
                       , Seoul,
                       <span class="nlm-country">
                        Korea
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Seunggeun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Lee%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ASeunggeun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-orcid-link">
                      <a href="http://orcid.org/0000-0002-8097-3878" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       ORCID record for Seunggeun Lee
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:wzhou@broadinstitute.org" class="" data-icon-position="" data-hide-link-title="0">
                        wzhou@broadinstitute.org
                       </a>
                       <a href="mailto:lee7801@snu.ac.kr" class="" data-icon-position="" data-hide-link-title="0">
                        lee7801@snu.ac.kr
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2021.07.12.21260400v2" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.07.12.21260400v2.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.07.12.21260400v2.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.07.12.21260400v2.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.07.12.21260400v2.supplementary-material" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="supplementary-material" data-url-enabled="1">
                     Supplementary material
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_data/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2021.07.12.21260400v2.external-links" class="panels-ajax-tab-tab" data-panel-name="article_tab_data_code" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="external-links" data-url-enabled="1">
                     Data/Code
                    </a>
                    <a href="/panels_ajax_tab/article_tab_data_code/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2021.07.12.21260400v2.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490258" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:490258/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.medrxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             UK Biobank has released the large-scale whole-exome sequencing (WES) data, but the best practices remain unclear for rare variant tests, and an existing approach, SAIGE-GENE, can have inflated type I error rates with high computation costs. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency compared to SAIGE-GENE. In the analysis of UKBB WES data of 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene-phenotype associations. In addition, we showed that incorporating multiple MAF cutoffs and functional annotations can help identify novel gene-phenotype associations.
                            </p>
                           </div>
                           <div class="section" id="sec-1">
                            <h2 class="">
                             Main
                            </h2>
                            <p id="p-16">
                             UK Biobank (UKBB) recently released the whole-exome sequencing (WES) data of most of their participants
                             <sup>
                              <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                               1
                              </a>
                             </sup>
                             , allowing for studying rare variant associations for complex traits and diseases. However, the best practices remain unclear for set-based rare variant tests in large-scale biobanks. A common practice is to test for associations with all rare (minor allele frequency, MAF ≤1%,) loss-of-function (LoF) and missense variants, but this approach can lose power if association signals are enriched in very rare variants or certain functional annotation classes. To improve power, researchers can restrict the test for rarer variants such as variants with MAF ≤0.1% or MAF ≤0.01%. Another approach is incorporating functional annotations. Since multiple variant sets with different MAF cutoffs and functional annotations exist, tests should be done multiple times for each gene or region and results should be aggregated using the minimum p-value or the Cauchy combination
                             <sup>
                              <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                              ,
                              <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                               3
                              </a>
                             </sup>
                             .
                            </p>
                            <p id="p-17">
                             Currently, SAIGE-GENE
                             <sup>
                              <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                               4
                              </a>
                             </sup>
                             is the only method developed to conduct the non-burden type set-based rare variant tests, such as SKAT
                             <sup>
                              <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                             </sup>
                             and SKAT-O
                             <sup>
                              <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                             </sup>
                             , for binary phenotypes with unbalanced case-control ratios in biobank-scale data. For example, in our evaluation, the most recent set-based test, STAAR
                             <sup>
                              <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             , cannot control for type I error rates in the presence of case-control imbalance (
                             <strong>
                              Supplementary Figure 1
                             </strong>
                             ). Burden tests can be done with single variant test methods and implemented in software packages including REGENIE2
                             <sup>
                              <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                               7
                              </a>
                             </sup>
                             , but the Burden test can suffer from a lower power compared to the SKAT-O test that is more robust to proportion of causal variants and direction of causal effects
                             <sup>
                              <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">
                               6
                              </a>
                             </sup>
                             , which we also observed in simulation studies
                             <strong>
                              (Methods, Supplementary Note A, Supplementary Figure 2)
                             </strong>
                             . SAIGE-GENE has been recently used for exome-wide association analysis for 3,700 phenotypes of 281,850 individuals in the UKBB WES data
                             <sup>
                              <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                             </sup>
                             . In the evaluation of the method using the UKBB WES data with 160K white British samples, we have found that all the tests (Burden, SKAT and SKAT-O) in SAIGE-GENE performed well when testing all rare variants with MAF 1% (
                             <strong>
                              <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                             </strong>
                             ), but inflation was observed in SKAT and SKAT-O tests in SAIGE-GENE when restricting to variants with MAF ≤0.1% or MAF≤0.01% and the case-control rates were more unbalanced than 1:30 (
                             <strong>
                              <a id="xref-fig-1-2" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              , Supplementary Figure 3)
                             </strong>
                             . To examine whether the inflation is due to inflated type I error rates or polygenicity of the phenotypes, we carried out type I error simulation studies with various case-control ratios (
                             <strong>
                              Supplementary Note B
                             </strong>
                             and
                             <strong>
                              Methods
                             </strong>
                             ), and observed the same inflation (
                             <strong>
                              Supplementary Figure 4
                             </strong>
                             ). This suggests that SKAT and SKAT-O in SAIGE-GENE can suffer from inflated type I error rates.
                            </p>
                            <div id="F1" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F1.large.jpg?width=800&height=600&carousel=1" title="Quantile-quantile (Q-Q) plots for Burden, SKAT3 and SKAT-O4 for four exemplary binary phenotypes in the UKBB WES data using A. SAIGE-GENE and B. SAIGE-GENE+. The tests were performed for 18,372 genes with missense and loss-of-function (LoF) variants with three different maximum MAF cutoffs: 1%, 0.1%, and 0.01%. Names of genes reaching the exome-wide significant threshold (p-value < 2.5×10−6) in SAIGE-GENE+ are annotated in the plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099239991" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Quantile-quantile (Q-Q) plots for Burden, SKAT<sup>3</sup> and SKAT-O<sup>4</sup> for four exemplary binary phenotypes in the UKBB WES data using A. SAIGE-GENE and B. SAIGE-GENE+.</span> The tests were performed for 18,372 genes with missense and loss-of-function (LoF) variants with three different maximum MAF cutoffs: 1%, 0.1%, and 0.01%. Names of genes reaching the exome-wide significant threshold (p-value < 2.5×10<sup>−6</sup>) in SAIGE-GENE+ are annotated in the plots.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F1.medium.gif" width="440" height="428"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 1." src="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F1.medium.gif" width="440" height="428"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                              <span class="fig-label">
                               Figure 1.
                              </span>
                              <span class="caption-title">
                               Quantile-quantile (Q-Q) plots for Burden, SKAT
                               <sup>
                                <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">
                                 3
                                </a>
                               </sup>
                               and SKAT-O
                               <sup>
                                <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">
                                 4
                                </a>
                               </sup>
                               for four exemplary binary phenotypes in the UKBB WES data using A. SAIGE-GENE and B. SAIGE-GENE+.
                              </span>
                              <p id="p-18" class="first-child">
                               The tests were performed for 18,372 genes with missense and loss-of-function (LoF) variants with three different maximum MAF cutoffs: 1%, 0.1%, and 0.01%. Names of genes reaching the exome-wide significant threshold (p-value < 2.5×10
                               <sup>
                                −6
                               </sup>
                               ) in SAIGE-GENE+ are annotated in the plots.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-19">
                             In addition, the computation cost is not low enough to test for the multiple variant sets. For example, to test the largest gene (TTN) with 16,227 variants in the UKBB WES data with 3 different MAF cutoffs (MAF≤1%, ≤0.1%, ≤0.01%) and 3 different annotations (LoF only, LoF+missense, and LoF+missense+synonymous), SAIGE-GENE required 164 CPU hours and 65 Gigabytes (Gb) of memory (
                             <strong>
                              Supplementary Table 1
                             </strong>
                             ).
                            </p>
                            <p id="p-20">
                             To address these issues, we propose SAIGE-GENE+. Although SAIGE-GENE uses various approaches, including saddlepoint approximation and exact resampling, to adjust for unbalanced case-control ratios, these approaches cannot fully address the imbalance and sparsity in the data (
                             <strong>
                              <a id="xref-fig-1-3" class="xref-fig" href="#F1">
                               Figure 1A
                              </a>
                              , Supplementary Figure 4A
                             </strong>
                             ). In order to reduce the sparsity, prior to the set-based association tests, SAIGE-GENE+ collapses ultra-rare variants with MAC ≤10 in the “absence and presence” way that has been commonly adapted in the association analysis of ultra-rare variants
                             <sup>
                              <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                               9
                              </a>
                              ,
                              <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                               10
                              </a>
                             </sup>
                             by assuming those variants have the same effect direction on the testing phenotype (
                             <strong>
                              Methods)
                             </strong>
                             . We observed that the inflation of SKAT and SKAT-O has been substantially reduced in SAIGE-GENE+ and all tests have well controlled type I errors in both simulated (
                             <strong>
                              Supplementary Figure 4B
                             </strong>
                             ) and the UKBB WES data (
                             <strong>
                              <a id="xref-fig-1-4" class="xref-fig" href="#F1">
                               Figure 1B
                              </a>
                             </strong>
                             ) for four exemplary phenotypes with case-control ratios 1:32 to 1:267. The genomic control inflation factors also became closer to one (
                             <strong>
                              Supplementary Figure 3)
                             </strong>
                             .
                            </p>
                            <p id="p-21">
                             Collapsing ultra-rare variants in SAIGE-GENE+ decreases the number of variants in each gene or region (
                             <strong>
                              Supplementary Figure 5
                             </strong>
                             ), leading to reduced computation time and memory usage (
                             <strong>
                              <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                               Figure 2A
                              </a>
                              , Supplementary Table 1, Supplementary Figure 6
                             </strong>
                             ). To further reduce computational cost, SAIGE-GENE+ reads in genotypes or dosages for all genetic markers only once, followed by conducting multiple association tests corresponding to different MAF cutoffs and annotations. The computation time of SAIGE-GENE+ for conducting all Burden, SKAT, and SKAT-O was 1,407 times decreased (9,851 mins vs. 7 mins) and the memory usage dropped from 65Gb to 2.1Gb compared to SAIGE-GENE when testing the largest gene TTN (16,227 LoF+missense+synonymous variants) for its association with the basal metabolic rate (
                             <strong>
                              Supplementary Table 1
                             </strong>
                             ). To test 18,372 genes with 150,000 samples using SKAT-O tests with three MAF cutoff 1%, 0.1%, and 0.01% and three different variant annotations: LoF only, LoF+missense, and LoF+missense+synonymous, SAIGE-GENE+ costs 78.6 CPU hours (18.8 CPU hours for fitting the null mixed model using a full GRM as Step 1 and 59.8 CPU hours for association tests as Step 2) and maximum 4.8Gb memory (4.8Gb for Step 1 and 2 Gb for Step 2) (
                             <strong>
                              Supplementary Tables 2 and 3, Supplementary Figure 7
                             </strong>
                             ). In addition, when a sparse GRM instead of a full GRM is used in Step 1 in SAIGE-GENE+, the time cost and memory usage dramatically dropped for fitting the null mixed model (< 1 min and 0.61Gb) (
                             <strong>
                              Supplementary Table 2, Supplementary Note C
                             </strong>
                             ). We also compared the computation time of SAIGE-GENE+ and REGENIE2 (
                             <strong>
                              Supplementary Table 2
                             </strong>
                             and
                             <strong>
                              3, Supplementary Note D, Supplementary Figure 7
                             </strong>
                             ).
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F2.large.jpg?width=800&height=600&carousel=1" title="Performance of SAIGE-GENE+ in UK Biobank WES data A. Computation time and memory of the gene-based tests (Step2, See Method) in SAIGE-GENE and SAIGE-GENE+ for four example genes with different numbers of variants. The SKAT-O tests were conducted with three maximum MAF cutoffs: 1%, 0.1%, and 0.01% and three variant annotations: LoF only, LoF + missense, and LoF + missense +synonymous and combined using the Cauchy combination. The plots are in the log10-log10 scale. The details of the numbers and genes are presented in the Supplementary Table 1. B. Most significant variant sets across the three different MAF cutoffs: 1%, 0.1%, and 0.01% and three functional annotations: LoF (L) only, LoF + missense (M+L), and LoF + missense +synonymous (S+M+L). Distribution of variant sets with the smallest p-values, among 551 significant gene-phenotype associations identified by SAIGE-GENE+ in the analyses of 30 quantitative traits and 141 binary traits in the UKBB WES data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1099239991" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Performance of SAIGE-GENE+ in UK Biobank WES data</span> <strong>A. Computation time and memory of the gene-based tests (Step2, See Method) in SAIGE-GENE and SAIGE-GENE+ for four example genes with different numbers of variants</strong>. The SKAT-O tests were conducted with three maximum MAF cutoffs: 1%, 0.1%, and 0.01% and three variant annotations: LoF only, LoF + missense, and LoF + missense +synonymous and combined using the Cauchy combination. The plots are in the log10-log10 scale. The details of the numbers and genes are presented in the <strong>Supplementary Table 1</strong>. <strong>B. Most significant variant sets across the three different MAF cutoffs: 1%</strong>, <em>0</em>.<strong>1%, and 0.01% and three functional annotations: LoF (L) only, LoF + missense (M+L), and LoF + missense +synonymous (S+M+L)</strong>. Distribution of variant sets with the smallest p-values, among 551 significant gene-phenotype associations identified by SAIGE-GENE+ in the analyses of 30 quantitative traits and 141 binary traits in the UKBB WES data.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F2.medium.gif" width="435" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 2." src="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F2.medium.gif" width="435" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2." data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.medrxiv.org/content/medrxiv/early/2022/03/16/2021.07.12.21260400/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 2.
                              </span>
                              <span class="caption-title">
                               Performance of SAIGE-GENE+ in UK Biobank WES data
                              </span>
                              <p id="p-22" class="first-child">
                               <strong>
                                A. Computation time and memory of the gene-based tests (Step2, See Method) in SAIGE-GENE and SAIGE-GENE+ for four example genes with different numbers of variants
                               </strong>
                               . The SKAT-O tests were conducted with three maximum MAF cutoffs: 1%, 0.1%, and 0.01% and three variant annotations: LoF only, LoF + missense, and LoF + missense +synonymous and combined using the Cauchy combination. The plots are in the log10-log10 scale. The details of the numbers and genes are presented in the
                               <strong>
                                Supplementary Table 1
                               </strong>
                               .
                              </p>
                              <p id="p-23">
                               <strong>
                                B. Most significant variant sets across the three different MAF cutoffs: 1%
                               </strong>
                               , 0.
                               <strong>
                                1%, and 0.01% and three functional annotations: LoF (L) only, LoF + missense (M+L), and LoF + missense +synonymous (S+M+L)
                               </strong>
                               . Distribution of variant sets with the smallest p-values, among 551 significant gene-phenotype associations identified by SAIGE-GENE+ in the analyses of 30 quantitative traits and 141 binary traits in the UKBB WES data.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-24">
                             By collapsing ultra-rare variants, SAIGE-GENE+ can have more significant p-values for several well-known gene-phenotype associations than SAIGE-GENE, even though the latter has inflated type I errors. We applied SAIGE-GENE and SAIGE-GENE+ to 37 self-reported binary phenotypes in the UKBB WES data using three different maximum MAF cutoffs: 1%, 0.1%, and 0.01% to 18,372 gene including all missense and LoF variants. We have observed 27 SKAT-O tests with more significant p-values in SAIGE-GENE+ than in SAIGE-GENE (
                             <strong>
                              Supplementary Table 4
                             </strong>
                             ). For example, BRCA2 for breast cancer with MAF ≤0.1% had a p-value 7.62×10
                             <sup>
                              −8
                             </sup>
                             in SAIGE-GENE+ and 1.65×10
                             <sup>
                              −3
                             </sup>
                             in SAIGE-GENE. Similarly, we observed the gene GCK for diabetes with maximum MAF ≤0.1% had a more significant p-value (1.22×10
                             <sup>
                              −13
                             </sup>
                             ) in SAIGE-GENE+ than in SAIGE-GENE (p-value = 4.06×10
                             <sup>
                              −6
                             </sup>
                             ). More detailed discussion can be found in
                             <strong>
                              Supplementary Note E
                             </strong>
                             .
                            </p>
                            <p id="p-25">
                             We evaluated the powers of SAIGE-GENE+ and SAIGE-GENE through extensive simulation studies based on the real genotypes of 10 genes in the UKBB WES data (
                             <strong>
                              Supplementary Table 5, Supplementary Note A, Methods
                             </strong>
                             ) for simulated binary traits. Three scenarios with different settings of proportions of causal variants across the multiple functional annotations and different settings of absolute effect sizes for causal variants were used. For each scenario, two settings of effect directions were used: 1. All causal variants increased disease risk; 2. 100% of LoF, 80% of missense, and 50% of synonymous causal variants increased disease risk and the other causal variants decreased disease risk (
                             <strong>
                              Supplementary Table 6
                             </strong>
                             ). The prevalence of the binary phenotypes was set to be 10%, under which both SAIGE-GENE and SAIGE-GENE+ have well controlled type I error rates for Burden, SKAT, and SKAT-O tests. We observed that in all scenarios, SAIGE-GENE+ had higher or similar empirical power than SAIGE-GENE (
                             <strong>
                              Supplementary Figure 8)
                             </strong>
                             with the increased median Chi-square statistics (
                             <strong>
                              Supplementary Table 7
                             </strong>
                             ). Our results also demonstrated as previously reported that the SKAT-O test can have higher power than the Burden test with more significant p-values and higher median Chi-square statistics (
                             <strong>
                              Supplementary Figure 2, Supplementary Table 7
                             </strong>
                             ), while the Burden test p-values from SAIGE-GENE+ are highly concordant to p-values from REGENIE2 (Pearson’s correlation R
                             <sup>
                              <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">
                               2
                              </a>
                             </sup>
                             = 0.99 for -log
                             <sub>
                              10
                             </sub>
                             (p-value)) (
                             <strong>
                              Supplementary Figure 9
                             </strong>
                             ). In addition, the simulation results suggested incorporating multiple functional annotations (LoF, LoF+Missense, LoF+Missense+Synonymous) and maximum MAF cutoffs (0.01%, 0.1%, and 1%) can have an increased power compared to only using a single maximum MAF cutoffs (1%) on one set of function annotation (LoF+Missense+Synonymous) (
                             <strong>
                              Supplementary Figure 10, Supplementary Table 7)
                             </strong>
                             .
                            </p>
                            <p id="p-26">
                             We applied SAIGE-GENE+ to analyze 18,372 genes in the UKBB 160K white British sample data using three different MAF cutoffs: 1%, 0.1%, and 0.01% and three different variant annotations: LoF only, LoF+missense, and LoF+missense +synonymous for 30 quantitative and 141 binary traits. The nine SKAT-O test p-values were then combined using Cauchy combination or the minimum p-value approach (
                             <strong>
                              Methods)
                             </strong>
                             . 465 gene-phenotype associations were significant at the exome-wide significance threshold (p-value ≤2.5×10
                             <sup>
                              −6
                             </sup>
                             ) for 27 quantitative traits (
                             <strong>
                              Supplementary Table 8
                             </strong>
                             ) and 86 gene-phenotype associations for 51 binary traits (
                             <strong>
                              Supplementary Table 9
                             </strong>
                             ). Since the expected number of p-values < 2.5×10
                             <sup>
                              −6
                             </sup>
                             under no association across all the phenotypes is 7.85, the false discovery rate is 0.014. Known genes BRCA1, BRCA2, CHEK2, PALB2, and SAMHD1 were significant for breast cancer, CNTNAP3B, CDKN2A and MITF for melanoma, IL33 for asthma, GCK for type 2 diabetes, and LDLR for high cholesterol and ischemic heart disease. For quantitative traits, MC4R and GIPR were significant for body mass index, CETP, LIPG, and LPL for the HDL cholesterol, LDLR and PCSK9 for LDL, and MEPE for heel bone mineral density. We also identified potentially novel gene-phenotype associations. For binary traits, a pancreatic cancer susceptibility gene NOC2L
                             <sup>
                              <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                             </sup>
                             was significant for hypovolemia (p-value = 9.68×10
                             <sup>
                              −7
                             </sup>
                             ), CYP21A2, known for the 21-hydroxylase deficient congenital adrenal hyperplasia (CAH)
                             <sup>
                              <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                               12
                              </a>
                             </sup>
                             , for allergy/adverse effect of penicillin (p-value = 3.63×10
                             <sup>
                              −7
                             </sup>
                             ). Also for quantitative traits, CHEK2, known to be associated with breast cancer and sex hormone-binding globulin measurement
                             <sup>
                              <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                               13
                              </a>
                             </sup>
                             , was significant for age at menopause (p-value = 4.51×10
                             <sup>
                              −17
                             </sup>
                             ), IGLL5, which encodes an immunoglobulin lambda-like polypeptide, for the lymphocyte count (p-value = 1.50×10
                             <sup>
                              −12
                             </sup>
                             ), and NANOG, which mediates germline development
                             <sup>
                              <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                               14
                              </a>
                             </sup>
                             and is highly expressed in embryonic carcinoma
                             <sup>
                              <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                             </sup>
                             , for age at first live birth (p-value = 2.01 ×10
                             <sup>
                              −6
                             </sup>
                             ).
                            </p>
                            <p id="p-27">
                             Using lower MAF cutoffs may help identify novel associations in which the association signals are highly enriched in the rarer variant. For example, the association between PDCD1LG2, which encodes Programmed Cell Death 1 Ligand, and chronic lymphoid leukemia became significant when tests were restricted to variants with MAF ≤0.01% and 0.1% (p-value= 7.5×10
                             <sup>
                              −7
                             </sup>
                             ) compared to testing all variants with MAF ≤1% (p-value = 5.4×10
                             <sup>
                              −4
                             </sup>
                             ) (
                             <strong>
                              Supplementary Table 10)
                             </strong>
                             . The underlying reason for this observation could be that the associations are largely enriched in the rarer variants in the gene, e.g. the most significant variant is the low frequency variant rs7854303 (9:5557672, MAF=3.4×10
                             <sup>
                              −4
                             </sup>
                             ) (see the Phe Web browser). Using a higher MAC cutoff 1% will include many non-causal variants, which decreases the power of the set-based association tests. In addition, including lower MAF cutoff sets helped to replicate known associations including MLH1 for colorectal cancer and CDKN2A for melanoma (
                             <strong>
                              Supplementary Table 10)
                             </strong>
                             . Due to the multiple comparison burden, including lower MAF cutoff sets can make marginally significant associations non-significant. For 141 binary phenotypes, 17 out of 92 (18.4%) associations were further identified with lower MAF cutoff sets, while 9 (9.8%) associations became non-significant (
                             <strong>
                              Supplementary Figure 11A, Supplementary Table 10
                             </strong>
                             ). For 30 quantitative traits, 28 out of 465 (6%) associations were further identified, while 53 (11.4%) associations became non-significant (
                             <strong>
                              Supplementary Figure 11A, Supplementary Table 11
                             </strong>
                             ), suggesting that restricting association tests to rarer variants has a gain for binary phenotypes. In functional annotation categories, 184 associations were identified when the tests were conducted on LoF variants only, including LoF+missense sets identified 299 additional associations, and when LoF+missense+synonymous sets were also included, 91 more associations were identified (
                             <strong>
                              Supplementary Figure 11B
                             </strong>
                             ). These results are consistent with the observation in the simulation studies that empirical power increased when incorporating multiple functional annotation combinations and maximum MAF cutoffs (
                             <strong>
                              Supplementary Figure 10
                             </strong>
                             ). We also investigated among the 551 significant gene-phenotype associations, which variant set had the smallest p-value (
                             <strong>
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Figure 2B
                              </a>
                             </strong>
                             ). Interestingly, among sets with MAF ≤ 0.01%, LoF variant sets generally had the smallest p-values, while when the MAF cutoff ≤ 1%, LoF+missense+synonymous sets generally had the smallest p-values.
                            </p>
                            <p id="p-28">
                             In summary, our results demonstrated that by incorporating multiple MAF cutoffs and functional annotations the exome-wide rare-variant association tests can help identify novel gene-phenotype associations and our SAIGE-GENE+ can facilitate this. Moreover, SAIGE-GENE+ has an option to allow users to fit the null generalized linear mixed model with a sparse genetic relationship matrix (GRM) to further reduce the computation burden (
                             <strong>
                              Supplementary Note C and Supplementary Figure 12 to 14
                             </strong>
                             ).
                            </p>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-3" class="subsection">
                             <h3>
                              Collapsing ultra-rare variants
                             </h3>
                             <p id="p-29">
                              Ultra-rare variants with MAC ≤10 were collapsed to a single marker (
                              <strong>
                               Supplementary Figure 15
                              </strong>
                              ). More specifically, if no per-marker weights are provided by the user, all ultra-rare variants will be collapsed to a new variant in the “absence and presence” way
                              <sup>
                               <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">
                                9
                               </a>
                              </sup>
                              . The dosage for each sample was assigned as the maximum dosage value among all ultra-rare variants carried by the sample, if any. Then the weights of the collapsed variant and non ultra-rare variants (MAC > 10) are calculated based on their MAF from beta distribution Beta(MAF, a
                              <sub>
                               1
                              </sub>
                              , a
                              <sub>
                               2
                              </sub>
                              ). By default, same as the setting of the SKAT-O test, a
                              <sub>
                               1
                              </sub>
                              = 1 and a
                              <sub>
                               1
                              </sub>
                              = 25 are used. If the per-marker weights are provided by the user, the ultra-rare variants will be collapsed to a new variant whose dosages are the maximum values among the weighted dosages.
                             </p>
                            </div>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Aggregating multiple tests
                             </h3>
                             <p id="p-30">
                              For each gene or region, p-values of multiple testing sets corresponding to multiple maximum MAF cutoffs and functional annotations were aggregated using the Cauchy combination
                              <sup>
                               <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ,
                               <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                              </sup>
                              . Note that the Cauchy combination does not work when any p-value is 1. Thus, we used the minimum p-value with the Bonferroni correction to combine multiple tests when at least one test had p-value = 1.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Type I error evaluation
                             </h3>
                             <p id="p-31">
                              To evaluate the type I error control of SAIGE-GENE and SAIGE-GENE+, we simulated binary phenotypes under the null hypothesis of no genetic effects based on the observed genotypes by WES of the 166,955 samples with white British ancestry in the UK Biobank (
                              <strong>
                               Supplementary Note B
                              </strong>
                              ). We conducted gene-based tests for 7,932 genes on the even chromosomes with missense and LoF variants using three different maximum MAF cutoffs: 1%, 0.1%, and 0.01%. In total, 158,640 gene-based tests were conducted for each maximum MAF cutoff for SAIGE-GENE and SAIGE-GENE+, respectively, and the Q-Q plots were shown in
                              <strong>
                               Supplementary Figure 4
                              </strong>
                              . Our simulation results suggest that SAIGE-GENE+ has well controlled type I errors with case-control ratios < 1:100 when testing variants with maximum minor allele frequency (MAF) = 0.01% (
                              <strong>
                               Supplementary Figure 4B
                              </strong>
                              ).
                             </p>
                             <p id="p-32">
                              We also evaluated the type I error control of SAIGE-GENE, SAIGE-GENE+ and STAAR (
                              <strong>
                               <a id="xref-fig-1-5" class="xref-fig" href="#F1">
                                Figure 1
                               </a>
                               , Supplementary Figure 1
                              </strong>
                              ) in real data. We applied the methods to four exemplary self-reported binary phenotypes with various case control ratios in the UKBB WES data to 18,372 genes including all LoF and missense variants using three different maximum MAF cutoffs: 1%, 0.1%, and 0.01%. For STAAR, we used the relative coefficient cutoff 0.05 for the sparse GRM to fit the null models.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Power evaluation
                             </h3>
                             <p id="p-33">
                              To evaluate the power of SAIGE-GENE+ and SAIGE-GENE, we simulated binary phenotypes based on the genotypes of ten genes in the WES data of 166,955 samples with white British ancestry in the UK Biobank (
                              <strong>
                               Supplementary Table 5
                              </strong>
                              ). The selected genes showed significant gene-phenotype associations (
                              <strong>
                               Supplementary Table 4
                              </strong>
                              ) and had wide ranges of the number of rare variants from 2901 (APOB) to 107 (GPSM3). The prevalence of the phenotype was set to be 10%, under which, both SAIGE-GENE and SAIGE-GENE+ have well controlled type I error rates for Burden, SKAT, and SKAT-O tests (
                              <strong>
                               Supplementary Figure 3
                              </strong>
                              ). Three scenarios with different settings of proportions of causal variants and magnitudes of effect sizes for causal variants were used:1) low proportion of causal variants and low effect sizes, 2) low proportion of causal variants and high effect sizes, and 3) high proportion of causal variants and high effect sizes (
                              <strong>
                               Supplementary Table 6
                              </strong>
                              ). More details about the simulation settings are described in
                              <strong>
                               Supplementary Note A
                              </strong>
                              . Our simulation results suggest that SAIGE-GENE+ has higher or similar empirical power than SAIGE-GENE (
                              <strong>
                               Supplementary Figure 8
                              </strong>
                              and
                              <strong>
                               Supplementary Table 7
                              </strong>
                              ).
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              UK Biobank WES data analysis
                             </h3>
                             <p id="p-34">
                              We applied SAIGE-GENE+ to analyze 18,372 genes in the UKBB WES data of 166,955 white British samples for 30 quantitative traits and 141 binary traits. Three different maximum MAF cutoffs: 1%, 0.1%, and 0.01% and three different variant annotations: LoF only, LoF + missense, and LoF + missense +synonymous were applied followed by aggregating the multiple SKAT-O tests using the Cauchy combination
                              <sup>
                               <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">
                                2
                               </a>
                               ,
                               <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">
                                3
                               </a>
                              </sup>
                              for each gene. Variants were annotated using ANNOVAR
                              <sup>
                               <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                                16
                               </a>
                              </sup>
                              . The LoF variants include those annotated as frameshift deletion, frameshift insertion, non-frameshift deletion, non-frameshift insertion, splicing, stop gain, and stop loss. Sex, age when attended assessment center, and first four PCs that were estimated using all samples with White British ancestry were adjusted in all tests. 250,656 pruned markers with MAF ≥ 1%, which were pruned from the directly genotyped markers using the following parameters, were used to construct GRM: window size of 500 base pairs (bp), step-size of 50 bp, and pairwise r2 < 0.2. We used the relative coefficient cutoff 0.05 for the sparse GRM for the variance ratio estimation after fitting the null models. The model was fitted with leave-one-chromosome-out (LOCO) to avoid proximal contamination.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Computation cost evaluation
                             </h3>
                             <p id="p-35">
                              Benchmarking was performed on randomly sub-sampled UK Biobank WES data (up to 150,000 samples) with White British participants for glaucoma (1,741 cases and 162,408 controls). We report the medians of five runs for run times and memory with samples randomly selected from the full sample set using five different sampling seeds. SAIGE-GENE and SAIGE-GENE+ use a two-step approach. Step1 estimates the model parameters (i.e. variance component and fixed effect coefficients) in the null model and Step2 conducts set-based association tests. SAIGE-GENE+ runs Step 1 with all covariates as offset, which leads to a decrease of the computation time (
                              <strong>
                               Supplementary Table 12
                              </strong>
                              ). We then compared computation time and memory usages of Step2 (
                              <strong>
                               <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                                Figure 2A
                               </a>
                               , Supplementary Table 1, Supplementary Figure 6
                              </strong>
                              ). Note that model parameters need to be estimated only once for each phenotype and can be used genome-wide regardless of MAF cutoffs and functional annotations. The computation cost of Step1 in SAIGE-GENE+ was given in
                              <strong>
                               Supplementary Figure 7A
                              </strong>
                              and
                              <strong>
                               Supplementary Table 2
                              </strong>
                              . SAIGE-GENE+ has an option to use a sparse genetic relationship matrix (GRM), which further reduces computation cost in Step1 (
                              <strong>
                               Supplementary Note C
                              </strong>
                              ).
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-9">
                            <h2 class="">
                             Code and data availability
                            </h2>
                            <p id="p-36">
                             SAIGE-GENE+ is implemented as an open-source R package available at
                             <a href="https://github.com/saigegit/SAIGE">
                              https://github.com/saigegit/SAIGE
                             </a>
                             . SAIGE-GENE is available at
                             <a href="https://github.com/weizhouUMICH/SAIGE/master">
                              https://github.com/weizhouUMICH/SAIGE/master
                             </a>
                             . REGENIE (version 2.2.4) was downloaded from
                             <a href="https://github.com/rgcgithub/regenie">
                              https://github.com/rgcgithub/regenie
                             </a>
                             . STAAR (version 0.9.5) was downloaded from
                             <a href="https://github.com/xihaoli/STAAR">
                              https://github.com/xihaoli/STAAR
                             </a>
                             The PheWeb
                             <sup>
                              <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                             </sup>
                             like visual server for 30 quantitative and 141 binary phenotypes of UK Biobank WES data analysis results are currently available at
                             <a href="https://ukb-200kexome.leelabsg.org">
                              https://ukb-200kexome.leelabsg.org
                             </a>
                            </p>
                           </div>
                           <div class="section data-availability" id="sec-11">
                            <h2 class="">
                             Data Availability
                            </h2>
                            <p id="p-37">
                             SAIGE-GENE+ is implemented as an open-source R package available at
                             <a href="https://github.com/saigegit/SAIGE">
                              https://github.com/saigegit/SAIGE
                             </a>
                             . SAIGE-GENE is available at
                             <a href="https://github.com/weizhouUMICH/SAIGE/master">
                              https://github.com/weizhouUMICH/SAIGE/master
                             </a>
                             . The summary statistics and QQ plots for 30 quantitative phenotypes and 141 binary phenotypes in UK Biobank by SAIGE-GENE+ are currently available for public download at
                             <a href="https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt">
                              https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt
                             </a>
                            </p>
                            <p id="p-38">
                             <a href="https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt">
                              https://storage.googleapis.com/leelabsg/saige-gene/reformat_all_withPhenoDetails.txt
                             </a>
                            </p>
                           </div>
                           <div class="section" id="sec-12">
                            <h2 class="">
                             Competing financial interests statement
                            </h2>
                            <p id="p-40">
                             B.M.N. is a member of Deep Genomics Scientific Advisory Board, has received travel expenses from Illumina, and also serves as a consultant for Avanir and Trigeminal solutions. K.J.K is a consultant for Vor Biopharma.
                            </p>
                           </div>
                           <div class="section" id="sec-13">
                            <h2 class="">
                             Author contributions
                            </h2>
                            <p id="p-41">
                             W.Z., W.B., Z.Z., and S.L. designed experiments. W.Z., W.B., Z.Z., performed experiments and analyzed the UK Biobank data. W.Z. implemented the software with input from W.B. and Z.Z.. Helpful advice was provided by K.K.D, K.A.J., K.J.K., B.M.N., and M.J.D.. W.Z., W.B., Z.Z., and S.L. wrote the manuscript with input from all co-authors.
                            </p>
                           </div>
                           <div class="section ack" id="ack-1">
                            <h2 class="">
                             Acknowledgments
                            </h2>
                            <p id="p-39">
                             This research has been conducted using the UK Biobank Resource under application number 45227. SL was supported by Brain Pool Plus (BP+, Brain Pool+) Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Science and ICT (2020H1D3A2A03100666, S.L). WB and ZZ were supported by NIH R01 HG008773. WZ was supported by the National Human Genome Research Institute of the National Institutes of Health under award number T32HG010464. We thank Dr. Alkes Price for the constructive comments and suggestions.
                            </p>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-15">
                               We carried out extensive power simulation studies using UK-Biobank WES genotype data and different scenarios of the proportion of the causal variants, effect sizes, and effect direction.
We thoroughly investigated the software code, removed any redundant code and debug outputs.The computation performance has been updated as the newer version of our software has largely improved the computation efficiency.
We demonstrated how the functional variant weights are collapsed for ultra-rare variants.
We have also added more details to explain why different variant frequency thresholds can achieve power improvement.
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               1.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1. in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.1" data-doi="10.1038/s41586-018-0579-z">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Bycroft
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The UK Biobank resource with deep phenotyping and genomic data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  562
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  203
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  209
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D562%26rft.spage%253D203%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0579-z%26rft_id%253Dinfo%253Apmid%252F30305743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41586-018-0579-z&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30305743&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               2.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2. in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.2" data-doi="10.1038/s41588-020-0676-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  969
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  983
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D52%26rft.spage%253D969%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-020-0676-4%26rft_id%253Dinfo%253Apmid%252F32839606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41588-020-0676-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32839606&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               3.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3. in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Xie
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Cauchy combination test: a powerful test with analytic p-value calculation under arbitrary dependency structures
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  J. Am. Stat. Assoc
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  393
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  402
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc%26rft.volume%253D115%26rft.spage%253D393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               4.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4. in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.4" data-doi="10.1038/s41588-020-0621-6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Zhou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   W.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scalable generalized linear mixed model for region-based association tests in large biobanks and cohorts
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  634
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  639
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D52%26rft.spage%253D634%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-020-0621-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41588-020-0621-6&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               5.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5. in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.5" data-doi="10.1016/j.ajhg.2011.05.029">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Rare-variant association testing for sequencing data with the sequence kernel association test
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am. J. Hum. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  89
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  82
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  93
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2011
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DM.%2BC.%26rft.volume%253D89%26rft.issue%253D1%26rft.spage%253D82%26rft.epage%253D93%26rft.atitle%253DRare-variant%2Bassociation%2Btesting%2Bfor%2Bsequencing%2Bdata%2Bwith%2Bthe%2Bsequence%2Bkernel%2Bassociation%2Btest.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.05.029%26rft_id%253Dinfo%253Apmid%252F21737059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.05.029&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=21737059&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               6.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6. in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.6" data-doi="10.1093/biostatistics/kxs014">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M. C.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   X.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Optimal tests for rare variant effects in sequencing association studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Biostatistics
                                 </abbr>
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  762
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  775
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBiostatistics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxs014%26rft_id%253Dinfo%253Apmid%252F22699862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxs014&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22699862&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000308885800015&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               7.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7. in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Mbatchou
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Computationally efficient whole-genome regression for quantitative and binary traits
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  53
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1097
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1103
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D53%26rft.spage%253D1097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               8.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8. in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.8" data-doi="10.1101/2021.06.19.21259117">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Karczewski
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. J.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Systematic single-variant and gene-based association testing of 3,700 phenotypes in 281,850 UK Biobank exomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  bioRxiv
                                 </abbr>
                                 (
                                 <span class="cit-pub-date">
                                  2021
                                 </span>
                                 )
                                 <span class="cit-pub-id-sep cit-pub-id-doi-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  doi:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-doi">
                                  10.1101/2021.06.19.21259117
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F2021.06.19.21259117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1101/2021.06.19.21259117&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               9.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9. in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.9">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Povysil
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   G.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Rare-variant collapsing analyses for complex traits: guidelines and applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Rev. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  20
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  747
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  759
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet%26rft.volume%253D20%26rft.spage%253D747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               10.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10. in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.10" data-doi="10.1016/j.ajhg.2019.01.002">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   Y.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  ACAT: A Fast and Powerful p Value Combination Method for Rare-Variant Analysis in Sequencing Studies
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Am. J. Hum. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  410
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  421
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet%26rft.volume%253D104%26rft.spage%253D410%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2019.01.002%26rft_id%253Dinfo%253Apmid%252F30849328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2019.01.002&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=30849328&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               11.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11. in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.11" data-doi="10.1038/s41467-018-02942-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Klein
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. P.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Genome-wide meta-analysis identifies five new susceptibility loci for pancreatic cancer
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Commun
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  556
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun%26rft.volume%253D9%26rft.spage%253D556%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-02942-5%26rft_id%253Dinfo%253Apmid%252Fhttp%253A%252F%252Fwww.n%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41467-018-02942-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=http://www.n&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               12.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12. in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.12" data-doi="10.1073/pnas.89.15.7232">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wedell
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ritzén
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   E. M.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Haglund-Stengler
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   B.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Luthman
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <span class="cit-article-title">
                                  Steroid 21-hydroxylase deficiency: three additional mutated alleles and establishment of phenotype-genotype relationships of common mutations
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proc. Natl. Acad. Sci. U. S. A
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  89
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  7232
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  7236
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  1992
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWedell%26rft.auinit1%253DA.%26rft.volume%253D89%26rft.issue%253D15%26rft.spage%253D7232%26rft.epage%253D7236%26rft.atitle%253DSteroid%2B21-hydroxylase%2Bdeficiency%253A%2Bthree%2Badditional%2Bmutated%2Balleles%2Band%2Bestablishment%2Bof%2Bphenotype-genotype%2Brelationships%2Bof%2Bcommon%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.15.7232%26rft_id%253Dinfo%253Apmid%252F1496017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODkvMTUvNzIzMiI7czo0OiJhdG9tIjtzOjUwOiIvbWVkcnhpdi9lYXJseS8yMDIyLzAzLzE2LzIwMjEuMDcuMTIuMjEyNjA0MDAuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               13.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13. in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Ruth
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K. S.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Using human genetics to understand the disease impacts of testosterone in men and women
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Med
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  26
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  252
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  258
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BMed%26rft.volume%253D26%26rft.spage%253D252%26rft_id%253Dinfo%253Apmid%252F32042192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=32042192&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               14.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14. in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.14" data-doi="10.1038/nature06403">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Chambers
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   I.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Nanog safeguards pluripotency and mediates germline development
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature
                                 </abbr>
                                 <span class="cit-vol">
                                  450
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  1230
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1234
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DChambers%26rft.auinit1%253DI.%26rft.volume%253D450%26rft.issue%253D7173%26rft.spage%253D1230%26rft.epage%253D1234%26rft.atitle%253DNanog%2Bsafeguards%2Bpluripotency%2Band%2Bmediates%2Bgermline%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06403%26rft_id%253Dinfo%253Apmid%252F18097409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nature06403&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18097409&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000251786200050&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               15.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15. in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.15" data-doi="10.1002/cncr.21435">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hart
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   A. H.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The pluripotency homeobox gene NANOG is expressed in human germ cell tumors
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Cancer
                                 </abbr>
                                 <span class="cit-vol">
                                  104
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  2092
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2098
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCancer%26rft.stitle%253DCancer%26rft.aulast%253DHart%26rft.auinit1%253DA.%2BH.%26rft.volume%253D104%26rft.issue%253D10%26rft.spage%253D2092%26rft.epage%253D2098%26rft.atitle%253DThe%2Bpluripotency%2Bhomeobox%2Bgene%2BNANOG%2Bis%2Bexpressed%2Bin%2Bhuman%2Bgerm%2Bcell%2Btumors.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcncr.21435%26rft_id%253Dinfo%253Apmid%252F16206293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1002/cncr.21435&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=16206293&link_type=MED&atom=%2Fmedrxiv%2Fearly%2F2022%2F03%2F16%2F2021.07.12.21260400.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000233103000007&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               16.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16. in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   K.
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   M.
                                  </span>
                                 </span>
                                 &
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Hakonarson
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   H.
                                  </span>
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  ANNOVAR: Functional annotation of genetic variants from next-generation sequencing data
                                 </abbr>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               17.
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17. in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2021.07.12.21260400v2.17" data-doi="10.1038/s41588-020-0622-5">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-surname">
                                   Gagliano Taliun
                                  </span>
                                  ,
                                  <span class="cit-name-given-names">
                                   S. A.
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Exploring and visualizing large-scale genetic associations by using PheWeb
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nat. Genet
                                 </abbr>
                                 .
                                 <span class="cit-vol">
                                  52
                                 </span>
                                 ,
                                 <span class="cit-fpage">
                                  550
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  552
                                 </span>
                                 (
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 ).
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet%26rft.volume%253D52%26rft.spage%253D550%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-020-0622-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41588-020-0622-5&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://medRxiv.disqus.com/?url=https%3A%2F%2Fwww.medrxiv.org%2Fcontent%2F10.1101%2F2021.07.12.21260400v2" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.10.26.21265510v2" title="Virtual reality intervention alleviates dyspnea in patients recovering from COVID pneumonia" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2021.10.01.21264368v2" title="Neural Correlates of Eye Contact and Social Function in Autism Spectrum Disorder" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 16, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2021.07.12.21260400v2.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-biorxiv-supplementary-fragment">
                       <div class="pane-content">
                        <p>
                         <a href="/content/10.1101/2021.07.12.21260400v2.supplementary-material">
                          <i class="icon-file">
                          </i>
                          Supplementary Material
                         </a>
                        </p>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/early/2022/03/16/2021.07.12.21260400.external-links" target="_self" class="link-icon">
                         <span class="icon-file">
                         </span>
                         <span class="title">
                          Data/Code
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2021.07.12.21260400v2.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about medRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2021.07.12.21260400v2" class="active" data-icon-position="" data-hide-link-title="0">
                                   Set-based rare variant association tests for biobank scale sequencing data sets
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from medRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the medRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/490258"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-Ss6E3dd6G2d2dEn8uKf0S8GUDmCjr20MqOWIaXFEDlc"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="277157956"/>
                                   <input type="hidden" name="captcha_token" value="2ed262a789ea4a993143dbe5ad525e2f"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node490258--3" data-node-nid="490258" id="node-490258--45819645" data-pisa="medrxiv;2021.07.12.21260400v2" data-pisa-master="medrxiv;2021.07.12.21260400" data-seqnum="490258" data-apath="/medrxiv/early/2022/03/16/2021.07.12.21260400.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Set-based rare variant association tests for biobank scale sequencing data sets
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Wei
                                  </span>
                                  <span class="nlm-surname">
                                   Zhou
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Wenjian
                                  </span>
                                  <span class="nlm-surname">
                                   Bi
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Zhangchen
                                  </span>
                                  <span class="nlm-surname">
                                   Zhao
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="3">
                                  <span class="nlm-given-names">
                                   Kushal K.
                                  </span>
                                  <span class="nlm-surname">
                                   Dey
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="4">
                                  <span class="nlm-given-names">
                                   Karthik A.
                                  </span>
                                  <span class="nlm-surname">
                                   Jagadeesh
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="5">
                                  <span class="nlm-given-names">
                                   Konrad J.
                                  </span>
                                  <span class="nlm-surname">
                                   Karczewski
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="6">
                                  <span class="nlm-given-names">
                                   Mark J.
                                  </span>
                                  <span class="nlm-surname">
                                   Daly
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="7">
                                  <span class="nlm-given-names">
                                   Benjamin M.
                                  </span>
                                  <span class="nlm-surname">
                                   Neale
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="8">
                                  <span class="nlm-given-names">
                                   Seunggeun
                                  </span>
                                  <span class="nlm-surname">
                                   Lee
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 medRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2021.07.12.21260400;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2021.07.12.21260400
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&title=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&title=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&text=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&t=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&title=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&title=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets&summary=&source=medRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&title=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.medrxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node490258--5" data-node-nid="490258" id="citation-node-490258--6707879329" data-pisa="medrxiv;2021.07.12.21260400v2" data-pisa-master="medrxiv;2021.07.12.21260400" data-seqnum="490258" data-apath="/medrxiv/early/2022/03/16/2021.07.12.21260400.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Set-based rare variant association tests for biobank scale sequencing data sets
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Wei
                                      </span>
                                      <span class="nlm-surname">
                                       Zhou
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Wenjian
                                      </span>
                                      <span class="nlm-surname">
                                       Bi
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Zhangchen
                                      </span>
                                      <span class="nlm-surname">
                                       Zhao
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="3">
                                      <span class="nlm-given-names">
                                       Kushal K.
                                      </span>
                                      <span class="nlm-surname">
                                       Dey
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="4">
                                      <span class="nlm-given-names">
                                       Karthik A.
                                      </span>
                                      <span class="nlm-surname">
                                       Jagadeesh
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="5">
                                      <span class="nlm-given-names">
                                       Konrad J.
                                      </span>
                                      <span class="nlm-surname">
                                       Karczewski
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="6">
                                      <span class="nlm-given-names">
                                       Mark J.
                                      </span>
                                      <span class="nlm-surname">
                                       Daly
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="7">
                                      <span class="nlm-given-names">
                                       Benjamin M.
                                      </span>
                                      <span class="nlm-surname">
                                       Neale
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="8">
                                      <span class="nlm-given-names">
                                       Seunggeun
                                      </span>
                                      <span class="nlm-surname">
                                       Lee
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     medRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2021.07.12.21260400;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2021.07.12.21260400
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=medRxiv&rft.au=Zhou%2C+Wei&rft.au=Bi%2C+Wenjian&rft.au=Zhao%2C+Zhangchen&rft.au=Dey%2C+Kushal+K.&rft.au=Jagadeesh%2C+Karthik+A.&rft.au=Karczewski%2C+Konrad+J.&rft.au=Daly%2C+Mark+J.&rft.au=Neale%2C+Benjamin+M.&rft.au=Lee%2C+Seunggeun">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/490258/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/490258/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/490258/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/490258/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/490258/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/490258/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/490258/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/490258/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/490258/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/490258/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/490258/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/490258/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&count=horizontal&via=&text=Set-based%20rare%20variant%20association%20tests%20for%20biobank%20scale%20sequencing%20data%20sets&counturl=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.medrxiv.org/content/10.1101/2021.07.12.21260400v2&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.medrxiv.org/content/10.1101/2021.07.12.21260400v2" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/genetic-and-genomic-medicine" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Genetic and Genomic Medicine
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/addiction-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Addiction Medicine
                          </a>
                          <span class="article-count">
                           (218)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/allergy-and-immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Allergy and Immunology
                          </a>
                          <span class="article-count">
                           (496)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/anesthesia" class="" data-icon-position="" data-hide-link-title="0">
                           Anesthesia
                          </a>
                          <span class="article-count">
                           (106)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cardiovascular-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Cardiovascular Medicine
                          </a>
                          <span class="article-count">
                           (1121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dentistry-and-oral-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Dentistry and Oral Medicine
                          </a>
                          <span class="article-count">
                           (198)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/dermatology" class="" data-icon-position="" data-hide-link-title="0">
                           Dermatology
                          </a>
                          <span class="article-count">
                           (141)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/emergency-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Emergency Medicine
                          </a>
                          <span class="article-count">
                           (275)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/endocrinology-including-diabetes-mellitus-and-metabolic-disease" class="" data-icon-position="" data-hide-link-title="0">
                           Endocrinology (including Diabetes Mellitus and Metabolic Disease)
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (9829)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/forensic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Forensic Medicine
                          </a>
                          <span class="article-count">
                           (5)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/gastroenterology" class="" data-icon-position="" data-hide-link-title="0">
                           Gastroenterology
                          </a>
                          <span class="article-count">
                           (484)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetic-and-genomic-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Genetic and Genomic Medicine
                          </a>
                          <span class="article-count">
                           (2344)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/geriatric-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Geriatric Medicine
                          </a>
                          <span class="article-count">
                           (225)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-economics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Economics
                          </a>
                          <span class="article-count">
                           (465)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-informatics" class="" data-icon-position="" data-hide-link-title="0">
                           Health Informatics
                          </a>
                          <span class="article-count">
                           (1574)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-policy" class="" data-icon-position="" data-hide-link-title="0">
                           Health Policy
                          </a>
                          <span class="article-count">
                           (738)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/health-systems-and-quality-improvement" class="" data-icon-position="" data-hide-link-title="0">
                           Health Systems and Quality Improvement
                          </a>
                          <span class="article-count">
                           (609)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hematology" class="" data-icon-position="" data-hide-link-title="0">
                           Hematology
                          </a>
                          <span class="article-count">
                           (239)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/hiv-aids" class="" data-icon-position="" data-hide-link-title="0">
                           HIV/AIDS
                          </a>
                          <span class="article-count">
                           (510)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/infectious_diseases" class="" data-icon-position="" data-hide-link-title="0">
                           Infectious Diseases (except HIV/AIDS)
                          </a>
                          <span class="article-count">
                           (11686)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/intensive-care-and-critical-care-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Intensive Care and Critical Care Medicine
                          </a>
                          <span class="article-count">
                           (617)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-education" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Education
                          </a>
                          <span class="article-count">
                           (242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/medical-ethics" class="" data-icon-position="" data-hide-link-title="0">
                           Medical Ethics
                          </a>
                          <span class="article-count">
                           (67)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nephrology" class="" data-icon-position="" data-hide-link-title="0">
                           Nephrology
                          </a>
                          <span class="article-count">
                           (259)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neurology" class="" data-icon-position="" data-hide-link-title="0">
                           Neurology
                          </a>
                          <span class="article-count">
                           (2170)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nursing" class="" data-icon-position="" data-hide-link-title="0">
                           Nursing
                          </a>
                          <span class="article-count">
                           (134)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/nutrition" class="" data-icon-position="" data-hide-link-title="0">
                           Nutrition
                          </a>
                          <span class="article-count">
                           (340)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/obstetrics-and-gynecology" class="" data-icon-position="" data-hide-link-title="0">
                           Obstetrics and Gynecology
                          </a>
                          <span class="article-count">
                           (427)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/occupational-and-environmental-health" class="" data-icon-position="" data-hide-link-title="0">
                           Occupational and Environmental Health
                          </a>
                          <span class="article-count">
                           (521)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/oncology" class="" data-icon-position="" data-hide-link-title="0">
                           Oncology
                          </a>
                          <span class="article-count">
                           (1188)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ophthalmology" class="" data-icon-position="" data-hide-link-title="0">
                           Ophthalmology
                          </a>
                          <span class="article-count">
                           (368)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/orthopedics" class="" data-icon-position="" data-hide-link-title="0">
                           Orthopedics
                          </a>
                          <span class="article-count">
                           (129)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/otolaryngology" class="" data-icon-position="" data-hide-link-title="0">
                           Otolaryngology
                          </a>
                          <span class="article-count">
                           (221)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pain-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Pain Medicine
                          </a>
                          <span class="article-count">
                           (148)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/palliative-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Palliative Medicine
                          </a>
                          <span class="article-count">
                           (50)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (315)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pediatrics" class="" data-icon-position="" data-hide-link-title="0">
                           Pediatrics
                          </a>
                          <span class="article-count">
                           (702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-therapeutics" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Therapeutics
                          </a>
                          <span class="article-count">
                           (304)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/primary-care-research" class="" data-icon-position="" data-hide-link-title="0">
                           Primary Care Research
                          </a>
                          <span class="article-count">
                           (269)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/psychiatry-and-clinical-psychology" class="" data-icon-position="" data-hide-link-title="0">
                           Psychiatry and Clinical Psychology
                          </a>
                          <span class="article-count">
                           (2201)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/public-and-global-health" class="" data-icon-position="" data-hide-link-title="0">
                           Public and Global Health
                          </a>
                          <span class="article-count">
                           (4705)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/radiology-and-imaging" class="" data-icon-position="" data-hide-link-title="0">
                           Radiology and Imaging
                          </a>
                          <span class="article-count">
                           (786)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rehabilitation-medicine-and-physical-therapy" class="" data-icon-position="" data-hide-link-title="0">
                           Rehabilitation Medicine and Physical Therapy
                          </a>
                          <span class="article-count">
                           (462)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/respiratory-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Respiratory Medicine
                          </a>
                          <span class="article-count">
                           (626)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/rheumatology" class="" data-icon-position="" data-hide-link-title="0">
                           Rheumatology
                          </a>
                          <span class="article-count">
                           (276)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sexual-and-reproductive-health" class="" data-icon-position="" data-hide-link-title="0">
                           Sexual and Reproductive Health
                          </a>
                          <span class="article-count">
                           (227)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/sports-medicine" class="" data-icon-position="" data-hide-link-title="0">
                           Sports Medicine
                          </a>
                          <span class="article-count">
                           (214)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/surgery" class="" data-icon-position="" data-hide-link-title="0">
                           Surgery
                          </a>
                          <span class="article-count">
                           (254)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Toxicology
                          </a>
                          <span class="article-count">
                           (43)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/transplantation" class="" data-icon-position="" data-hide-link-title="0">
                           Transplantation
                          </a>
                          <span class="article-count">
                           (121)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/urology" class="" data-icon-position="" data-hide-link-title="0">
                           Urology
                          </a>
                          <span class="article-count">
                           (94)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__llgGUcz2-k3Jmyyyog-_4yI1wJfGWwx0eQ2lbnM9IIo__iVohOHEOxt0s_XFW8FExNR9Y7ZodYkDjuSAmPcXloXM__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript" src="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom_medrxiv.js">
  </script>
  <script type="text/javascript" src="https://www.medrxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__yDRjOTiBnSnu2tzh2xbnVTigKcZk0CFMqnf6m8vMg3M.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
