<html lang="en" dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML">
 <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
  <link rel="dns-prefetch" href="//d33xdlntwy0kbs.cloudfront.net"/>
  <link rel="dns-prefetch" href="//www.google.com"/>
  <link rel="dns-prefetch" href="//scholar.google.com"/>
  <link rel="dns-prefetch" href="//www.google-analytics.com"/>
  <link rel="dns-prefetch" href="//stats.g.doubleclick.net"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <link rel="shortcut icon" href="https://www.biorxiv.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon"/>
  <link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10.1101/2020.06.18.159913v3.full.pdf"/>
  <link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10.1101/2020.06.18.159913v3.full.txt"/>
  <link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10.1101/2020.06.18.159913v3.ppt"/>
  <meta name="article_thumbnail" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-1.gif"/>
  <meta name="type" content="article"/>
  <meta name="category" content="article"/>
  <meta name="HW.identifier" content="/biorxiv/early/2022/03/17/2020.06.18.159913.atom"/>
  <meta name="HW.pisa" content="biorxiv;2020.06.18.159913v3"/>
  <meta name="DC.Format" content="text/html"/>
  <meta name="DC.Language" content="en"/>
  <meta name="DC.Title" content="Brainprints: identifying individuals from magnetoencephalograms"/>
  <meta name="DC.Identifier" content="10.1101/2020.06.18.159913"/>
  <meta name="DC.Date" content="2022-03-17"/>
  <meta name="DC.Publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="DC.Rights" content="© 2022, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission."/>
  <meta name="DC.AccessRights" content="restricted"/>
  <meta name="DC.Description" content="Magnetoencephalography (MEG) is used to study a wide variety of cognitive processes. Increasingly, researchers are adopting principles of open science and releasing their MEG data. While essential for reproducibility, sharing MEG data has unforeseen privacy risks. Individual differences may make a participant identifiable from their anonymized recordings. However, our ability to identify individuals based on these individual differences has not yet been assessed. Here, we propose interpretable MEG features to characterize individual difference. We term these features brainprints (brain fingerprints). We show through several datasets that brainprints accurately identify individuals across days, tasks, and even between MEG and Electroencephalography (EEG). Furthermore, we identify consistent brainprint components that are important for identification. We study the dependence of identifiability on the amount of data available. We also relate identifiability to the level of preprocessing, the experimental task. Our findings reveal specific aspects of individual variability in MEG. They also raise concerns about unregulated sharing of brain data, even if anonymized.

![Figure 1:][1]</img>

Figure 1: Graphical abstract.
Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing.



### Competing Interest Statement

The authors have declared no competing interest.

 [1]: pending:yes"/>
  <meta name="DC.Contributor" content="Shenghao Wu"/>
  <meta name="DC.Contributor" content="Aaditya Ramdas"/>
  <meta name="DC.Contributor" content="Leila Wehbe"/>
  <meta name="article:published_time" content="2022-03-17"/>
  <meta name="article:section" content="New Results"/>
  <meta name="citation_title" content="Brainprints: identifying individuals from magnetoencephalograms"/>
  <meta name="citation_abstract" lang="en" content="<h3>Abstract</h3>
<p>Magnetoencephalography (MEG) is used to study a wide variety of cognitive processes. Increasingly, researchers are adopting principles of open science and releasing their MEG data. While essential for reproducibility, sharing MEG data has unforeseen privacy risks. Individual differences may make a participant identifiable from their anonymized recordings. However, our ability to identify individuals based on these individual differences has not yet been assessed. Here, we propose interpretable MEG features to characterize individual difference. We term these features brainprints (brain fingerprints). We show through several datasets that brainprints accurately identify individuals across days, tasks, and even between MEG and Electroencephalography (EEG). Furthermore, we identify consistent brainprint components that are important for identification. We study the dependence of identifiability on the amount of data available. We also relate identifiability to the level of preprocessing, the experimental task. Our findings reveal specific aspects of individual variability in MEG. They also raise concerns about unregulated sharing of brain data, even if anonymized.</p>"/>
  <meta name="citation_journal_title" content="bioRxiv"/>
  <meta name="citation_publisher" content="Cold Spring Harbor Laboratory"/>
  <meta name="citation_publication_date" content="2022/01/01"/>
  <meta name="citation_mjid" content="biorxiv;2020.06.18.159913v3"/>
  <meta name="citation_id" content="2020.06.18.159913v3"/>
  <meta name="citation_public_url" content="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3"/>
  <meta name="citation_abstract_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3.abstract"/>
  <meta name="citation_full_html_url" content="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3.full"/>
  <meta name="citation_pdf_url" content="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913.full.pdf"/>
  <meta name="citation_doi" content="10.1101/2020.06.18.159913"/>
  <meta name="citation_num_pages" content="53"/>
  <meta name="citation_article_type" content="Article"/>
  <meta name="citation_section" content="New Results"/>
  <meta name="citation_firstpage" content="2020.06.18.159913"/>
  <meta name="citation_author" content="Shenghao Wu"/>
  <meta name="citation_author_institution" content="Neuroscience Institute, Carnegie Mellon University"/>
  <meta name="citation_author_institution" content="Machine Learning Department, Carnegie Mellon University"/>
  <meta name="citation_author" content="Aaditya Ramdas"/>
  <meta name="citation_author_institution" content="Machine Learning Department, Carnegie Mellon University"/>
  <meta name="citation_author_institution" content="Department of Statistics and Data Science, Carnegie Mellon University"/>
  <meta name="citation_author" content="Leila Wehbe"/>
  <meta name="citation_author_institution" content="Neuroscience Institute, Carnegie Mellon University"/>
  <meta name="citation_author_institution" content="Machine Learning Department, Carnegie Mellon University"/>
  <meta name="citation_author_email" content="lwehbe@cmu.edu"/>
  <meta name="citation_reference" content="Thomas Landrain, Morgan Meyer, Ariel Martin Perez, and Remi Sussan. Do-it-yourself biology: challenges and promises for an open science and technology movement. Systems and synthetic biology, 7(3):115–126, 2013."/>
  <meta name="citation_reference" content="Krzysztof Gorgolewski, Oscar Esteban, Gunnar Schaefer, Brian Wandell, and Russell Poldrack. Openneuro—a free online platform for sharing and analysis of neuroimaging data. Organization for human brain mapping. Vancouver, Canada, 1677(2), 2017."/>
  <meta name="citation_reference" content="Latanya Sweeney. Simple demographics often identify people uniquely. Health (San Francisco), 671:1–34, 2000."/>
  <meta name="citation_reference" content="Halbert L Dunn. Record linkage. American Journal of Public Health and the Nations Health, 36(12):1412–1416, 1946."/>
  <meta name="citation_reference" content="David C Van Essen, Stephen M Smith, Deanna M Barch, Timothy EJ Behrens, Essa Yacoub, Kamil Ugurbil, Wu-Minn HCP Consortium, et al. The wu-minn human connectome project: an overview. Neuroimage, 80:62–79, 2013."/>
  <meta name="citation_reference" content="Jaewoo Lee and Chris Clifton. Differential identifiability. In Proceedings of the 18th ACM SIGKDD international conference on Knowledge discovery and data mining, pages 1041–1049, 2012."/>
  <meta name="citation_reference" content="Craig Gentry. Fully homomorphic encryption using ideal lattices. In Proceedings of the forty-first annual ACM symposium on Theory of computing, pages 169–178, 2009."/>
  <meta name="citation_reference" content="Giles L Colclough, Mark W Woolrich, PK Tewarie, Matthew J Brookes, Andrew J Quinn, and Stephen M Smith. How reliable are meg resting-state connectivity metrics? Neuroimage, 138:284–293, 2016."/>
  <meta name="citation_reference" content="Katherine S Button, John PA Ioannidis, Claire Mokrysz, Brian A Nosek, Jonathan Flint, Emma SJ Robinson, and Marcus R Munafò. Power failure: why small sample size undermines the reliability of neuroscience. Nature Reviews Neuroscience, 14(5):365–376, 2013."/>
  <meta name="citation_reference" content="Russell A Poldrack, Chris I Baker, Joke Durnez, Krzysztof J Gorgolewski, Paul M Matthews, Marcus R Munafò, Thomas E Nichols, Jean-Baptiste Poline, Edward Vul, and Tal Yarkoni. Scanning the horizon: towards transparent and reproducible neuroimaging research. Nature reviews neuroscience, 18(2):115, 2017."/>
  <meta name="citation_reference" content="Vincent Wens, Mathieu Bourguignon, Serge Goldman, Brice Marty, Marc Op De Beeck, Catherine Clumeck, Alison Mary, Philippe Peigneux, Patrick Van Bogaert, Matthew J Brookes, et al. Inter-and intra-subject variability of neuromagnetic resting state networks. Brain topography, 27(5):620–634, 2014."/>
  <meta name="citation_reference" content="M Poulos, M Rangoussi, N Alexandris, and A Evangelou. Person identification from the eeg using nonlinear signal classification. Methods of information in Medicine, 41(01):64–75, 2002."/>
  <meta name="citation_reference" content="Lan Ma, James W Minett, Thierry Blu, and William SY Wang. Resting state eeg-based biometrics for individual identification using convolutional neural networks. In 2015 37th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC), pages 2848–2851. IEEE, 2015."/>
  <meta name="citation_reference" content="Xiang Zhang, Lina Yao, Salil S Kanhere, Yunhao Liu, Tao Gu, and Kaixuan Chen. Mindid: Person identification from brain waves through attention-based recurrent neural network. Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technologies, 2(3):1–23, 2018."/>
  <meta name="citation_reference" content="Emily S Finn, Xilin Shen, Dustin Scheinost, Monica D Rosenberg, Jessica Huang, Marvin M Chun, Xenophon Papademetris, and R Todd Constable. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nature neuroscience, 18(11):1664, 2015."/>
  <meta name="citation_reference" content="Oscar Miranda-Dominguez, Brian D Mills, Samuel D Carpenter, Kathleen A Grant, Christopher D Kroenke, Joel T Nigg, and Damien A Fair. Connectotyping: model based fingerprinting of the functional connectome. PloS one, 9(11), 2014."/>
  <meta name="citation_reference" content="M Demuru, AA Gouw, A Hillebrand, CJ Stam, BW Van Dijk, P Scheltens, BM Tijms, E Konijnenberg, M Ten Kate, A Den Braber, et al. Functional and effective whole brain connectivity using magnetoencephalography to identify monozygotic twin pairs. Scientific reports, 7(1):1–11, 2017."/>
  <meta name="citation_reference" content="Andrew James Anderson, Kelsey McDermott, Brian Rooks, Kathi L Heffner, David Dodell-Feder, and Feng V Lin. Decoding individual identity from brain activity elicited in imagining common experiences. Nature communications, 11(1):1–14, 2020."/>
  <meta name="citation_reference" content="Christian Wachinger, Polina Golland, William Kremen, Bruce Fischl, Martin Reuter, Alzheimer’s Disease Neuroimaging Initiative, et al. Brainprint: A discriminative characterization of brain morphology. NeuroImage, 109:232–248, 2015."/>
  <meta name="citation_reference" content="Blair C Armstrong, Maria V Ruiz-Blondet, Negin Khalifian, Kenneth J Kurtz, Zhanpeng Jin, and Sarah Laszlo. Brainprint: Assessing the uniqueness, collectability, and permanence of a novel method for erp biometrics. Neurocomputing, 166:59–67, 2015."/>
  <meta name="citation_reference" content="Christian Wachinger, Polina Golland, and Martin Reuter. Brainprint: Identifying subjects by their brain. In International Conference on Medical Image Computing and Computer-Assisted Intervention, pages 41–48. Springer, 2014."/>
  <meta name="citation_reference" content="Debadatta Dash, Paul Ferrari, and Jun Wang. Spatial and spectral fingerprint in the brain: Speaker identification from single trial meg signals. In INTERSPEECH, pages 1203–1207, 2019."/>
  <meta name="citation_reference" content="JK Rowling. of Book: Harry Potter and the Sorcerer’s Stone. Year: Scholastic Press/1997, 1997."/>
  <meta name="citation_reference" content="Andy Liaw, Matthew Wiener, et al. Classification and regression by randomforest. R news, 2(3):18–22, 2002."/>
  <meta name="citation_reference" content="Sofie S Meyer, James Bonaiuto, Mark Lim, Holly Rossiter, Sheena Waters, David Bradbury, Sven Bestmann, Matthew Brookes, Martina F Callaghan, Nikolaus Weiskopf, et al. Flexible head-casts for high spatial precision meg. Journal of neuroscience methods, 276:38–45, 2017."/>
  <meta name="citation_reference" content="H-RM Tan, Joachim Gross, and PJ Uhlhaas. Meg—measured auditory steadystate oscillations show high test–retest reliability: A sensor and source-space analysis. Neuroimage, 122:417–426, 2015."/>
  <meta name="citation_reference" content="Douglas Owen Cheyne and Paul Ferrari. Meg studies of motor cortex gamma oscillations: evidence for a gamma “fingerprint” in the brain? Frontiers in human neuroscience, 7:575, 2013."/>
  <meta name="citation_reference" content="Mark D Vida, Adrian Nestor, David C Plaut, and Marlene Behrmann. Spatiotemporal dynamics of similarity-based neural representations of facial identity. Proceedings of the National Academy of Sciences, 114(2):388–393, 2017."/>
  <meta name="citation_reference" content="Enrico Amico and Joaquín Goñi. The quest for identifiability in human functional connectomes. Scientific reports, 8(1):1–14, 2018."/>
  <meta name="citation_reference" content="Sheng Zhang and Chiang-Shan R Li. Task-related, low-frequency task-residual, and resting state activity in the default mode network brain regions. Frontiers in psychology, 3:172, 2012."/>
  <meta name="citation_reference" content="Omer Grigg and Cheryl L Grady. Task-related effects on the temporal and spatial dynamics of resting-state functional connectivity in the default network. PloS one, 5(10), 2010."/>
  <meta name="citation_reference" content="Ying Yang, Michael J Tarr, Robert E Kass, and Elissa M Aminoff. Exploring spatiotemporal neural dynamics of the human visual cortex. Human brain mapping, 40(14):4213–4238, 2019."/>
  <meta name="citation_reference" content="Tony Ro, Bruno Breitmeyer, Philip Burton, Neel S Singhal, and David Lane. Feedback contributions to visual awareness in human occipital cortex. Current biology, 13(12):1038–1041, 2003."/>
  <meta name="citation_reference" content="Oliver Jakobs, Robert Langner, Svenja Caspers, Christian Roski, Edna C Cieslik, Karl Zilles, Angela R Laird, Peter T Fox, and Simon B Eickhoff. Across-study and within-subject functional connectivity of a right temporo-parietal junction subregion involved in stimulus–context integration. Neuroimage, 60(4):2389–2398, 2012."/>
  <meta name="citation_reference" content="Riitta Salmelin. Clinical neurophysiology of language: the meg approach. Clinical Neurophysiology, 118(2):237–254, 2007."/>
  <meta name="citation_reference" content="Pravat K Mandal, Anwesha Banerjee, Manjari Tripathi, and Ankita Sharma. A comprehensive review of magnetoencephalography (meg) studies for brain functionality in healthy aging and alzheimer’s disease (ad). Frontiers in Computational Neuroscience, 12:60, 2018."/>
  <meta name="citation_reference" content="Alexandre Gramfort, Martin Luessi, Eric Larson, Denis A Engemann, Daniel Strohmeier, Christian Brodbeck, Lauri Parkkonen, and Matti S Hämäläinen. Mne software for processing meg and eeg data. Neuroimage, 86:446–460, 2014."/>
  <meta name="citation_reference" content="Pilar Garcés, María Carmen Martín-Buro, and Fernando Maestú. Quantifying the test-retest reliability of magnetoencephalography resting-state functional connectivity. Brain connectivity, 6(6):448–460, 2016."/>
  <meta name="citation_reference" content="Kristina T Legget, Allison K Hild, Sarah E Steinmetz, Steven T Simon, and Donald C Rojas. Meg and eeg demonstrate similar test-retest reliability of the 40 hz auditory steady-state response. International Journal of Psychophysiology, 114:16–23, 2017."/>
  <meta name="citation_reference" content="María Carmen Martín-Buro, Pilar Garcés, and Fernando Maestú. Test-retest reliability of resting-state magnetoencephalography power in sensor and source space. Human brain mapping, 37(1):179–190, 2016."/>
  <meta name="citation_reference" content="Corinna Cortes, Giulia DeSalvo, and Mehryar Mohri. Learning with rejection. In International Conference on Algorithmic Learning Theory, pages 67–82. Springer, 2016."/>
  <meta name="citation_reference" content="Manasij Venkatesh, Joseph Jaja, and Luiz Pessoa. Comparing functional connectivity matrices: A geometry-aware approach applied to participant identification. NeuroImage, 207:116398, 2020."/>
  <meta name="citation_reference" content="Brian Kulis et al. Metric learning: A survey. Foundations and trends in machine learning, 5(4):287–364, 2012."/>
  <meta name="citation_reference" content="Qiang Yang, Yang Liu, Tianjian Chen, and Yongxin Tong. Federated machine learning: Concept and applications. ACM Transactions on Intelligent Systems and Technology (TIST), 10(2):1–19, 2019."/>
  <meta name="citation_reference" content="Yegang Hu, Chunli Yin, Jicong Zhang, and Yuping Wang. Partial least square aided beamforming algorithm in magnetoencephalography source imaging. Frontiers in neuroscience, 12:616, 2018."/>
  <meta name="citation_reference" content="Pauli Virtanen, Ralf Gommers, Travis E Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, et al. Scipy 1.0: fundamental algorithms for scientific computing in python. Nature methods, 17(3):261–272, 2020."/>
  <meta name="citation_reference" content="Samu Taulu and Matti Kajola. Presentation of electromagnetic multichannel data: the signal space separation method. Journal of Applied Physics, 97(12):124905, 2005."/>
  <meta name="citation_reference" content="Samu Taulu and Juha Simola. Spatiotemporal signal space separation method for rejecting nearby interference in meg measurements. Physics in Medicine & Biology, 51(7):1759, 2006."/>
  <meta name="citation_reference" content="Mikko A Uusitalo and Risto J Ilmoniemi. Signal-space projection method for separating meg or eeg into components. Medical and Biological Engineering and Computing, 35(2):135–140, 1997."/>
  <meta name="twitter:title" content="Brainprints: identifying individuals from magnetoencephalograms"/>
  <meta name="twitter:site" content="@biorxivpreprint"/>
  <meta name="twitter:card" content="summary"/>
  <meta name="twitter:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="twitter:description" content="Magnetoencephalography (MEG) is used to study a wide variety of cognitive processes. Increasingly, researchers are adopting principles of open science and releasing their MEG data. While essential for reproducibility, sharing MEG data has unforeseen privacy risks. Individual differences may make a participant identifiable from their anonymized recordings. However, our ability to identify individuals based on these individual differences has not yet been assessed. Here, we propose interpretable MEG features to characterize individual difference. We term these features brainprints (brain fingerprints). We show through several datasets that brainprints accurately identify individuals across days, tasks, and even between MEG and Electroencephalography (EEG). Furthermore, we identify consistent brainprint components that are important for identification. We study the dependence of identifiability on the amount of data available. We also relate identifiability to the level of preprocessing, the experimental task. Our findings reveal specific aspects of individual variability in MEG. They also raise concerns about unregulated sharing of brain data, even if anonymized.

![Figure 1:][1]</img>

Figure 1: Graphical abstract.
Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing.



### Competing Interest Statement

The authors have declared no competing interest.

 [1]: pending:yes"/>
  <meta name="og-title" property="og:title" content="Brainprints: identifying individuals from magnetoencephalograms"/>
  <meta name="og-url" property="og:url" content="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3"/>
  <meta name="og-site-name" property="og:site_name" content="bioRxiv"/>
  <meta name="og-description" property="og:description" content="Magnetoencephalography (MEG) is used to study a wide variety of cognitive processes. Increasingly, researchers are adopting principles of open science and releasing their MEG data. While essential for reproducibility, sharing MEG data has unforeseen privacy risks. Individual differences may make a participant identifiable from their anonymized recordings. However, our ability to identify individuals based on these individual differences has not yet been assessed. Here, we propose interpretable MEG features to characterize individual difference. We term these features brainprints (brain fingerprints). We show through several datasets that brainprints accurately identify individuals across days, tasks, and even between MEG and Electroencephalography (EEG). Furthermore, we identify consistent brainprint components that are important for identification. We study the dependence of identifiability on the amount of data available. We also relate identifiability to the level of preprocessing, the experimental task. Our findings reveal specific aspects of individual variability in MEG. They also raise concerns about unregulated sharing of brain data, even if anonymized.

![Figure 1:][1]</img>

Figure 1: Graphical abstract.
Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing.



### Competing Interest Statement

The authors have declared no competing interest.

 [1]: pending:yes"/>
  <meta name="og-type" property="og:type" content="article"/>
  <meta name="og-image" property="og:image" content="https://www.biorxiv.org/sites/default/files/images/biorxiv_logo_homepage7-5-small.png"/>
  <meta name="citation_date" content="2022-03-17"/>
  <meta name="description" content="bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution"/>
  <meta name="generator" content="Drupal 7 (http://drupal.org)"/>
  <link rel="canonical" href="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3"/>
  <link rel="shortlink" href="https://www.biorxiv.org/node/2452346"/>
  <title>
   Brainprints: identifying individuals from magnetoencephalograms | bioRxiv
  </title>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__7SC0i-kTgUlQGKuqbmyS18Sez8FDO-aG9FSHkGrLGl8__9Xf-fJpmF9h8tvWywQTaOzJzFLt4sEfx108Wh1gYmm0__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__JbFqFjYGp4Zx8gvmj6v5YmfNmbiFphGHPblyC9bfG5Y__OScmsb_1nSVmm_Ax3cJ5Rq7p081PahYkvF_YWQd5GtE__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
.panels-flexible-new .panels-flexible-region{padding:0}.panels-flexible-new .panels-flexible-region-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-region-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-region-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-column{padding:0}.panels-flexible-new .panels-flexible-column-inside{padding-right:.5em;padding-left:.5em}.panels-flexible-new .panels-flexible-column-inside-first{padding-left:0}.panels-flexible-new .panels-flexible-column-inside-last{padding-right:0}.panels-flexible-new .panels-flexible-row{padding:0 0 .5em;margin:0}.panels-flexible-new .panels-flexible-row-last{padding-bottom:0}.panels-flexible-column-new-main{float:left;width:99.0000%}.panels-flexible-new-inside{padding-right:0}.panels-flexible-new{width:auto}.panels-flexible-region-new-center{float:left;width:99.0000%}.panels-flexible-row-new-main-row-inside{padding-right:0}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__PWuQ_RYRTJ4BLEKsbWQeHysPg0gOQ3571ruQa_rXvAo__cj81oxtSGUe-AIO3tlkSBJPNpKP5ZgLdB8f8awBQsWU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <style type="text/css" media="all">
   /* &lt;![CDATA[ */
#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}
/* ]]&gt; */
  </style>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__Wnlyen9qEpwh_Qaf9okEu4QdVGM0BDothxeqA6Nbvo8__EJmw6SZD9bYoS8jocCpPYS3JFRURpdzmuvJoAUNiI-g__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__fse0T7HwKBxFeJ5ZSFW6P7GHsblWECJwe3f8Eo0Byb8__GnHmvNTOB0-xZRDC4PFtbOee_g-W6li7rpWRuc9O4B4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://www.biorxiv.org/sites/default/files/advagg_css/css__g_BFrOIH2jRr6w2kKikVPe0XcLWrQStWL-Z3vs6TbJQ__5c1ZiN_pM9tjxvpyA9VpEacScF0S_W4R222pC-s_-Pk__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.css" media="all"/>
  <link type="text/css" rel="stylesheet" href="https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css" media="all"/>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BKYqkKToQ7EjirB7eIdMEH5521EU3da9IpoOs8Ex2XI__aSjVoX8giBmLhN2EbCgIGQJNu89Mh5aVu1LvI_gkJ7Y__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__4Cn2dxvNlsJ-sHe6QOTLREaQvcqb0Yh0Zm9tTOHtQow__JeZEUjzbaj_yX6UjCI8eBbXy_J64ZVuoWmc2fSpLZHo__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl=en&render=explicit&onload=drupalRecaptchaOnload">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__dGWpV57YWu3sX6UOe04RMH-iP9jSkEP7Ajt0caYXZZk__u2clNMaOO3LCM0tpFNpsQhnHrJ4_SUXG1gFb00eB1iU__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&amp;&amp;g(l.readyState)&amp;&amp;(u.r=o=1,!q&amp;&amp;h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&amp;&amp;m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&amp;&amp;y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&amp;&amp;(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&amp;&amp;(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&amp;&amp;h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&amp;&amp;!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&amp;&amp;"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&amp;&amp;!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&amp;&amp;(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f&lt;d;f++)g=a[f].split("="),(e=z[g.shift()])&amp;&amp;(c=e(c,g));for(f=0;f&lt;b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&amp;&amp;(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&amp;&amp;"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&amp;&amp;f.load(function(){k(),e&amp;&amp;e(i.origUrl,h,g),j&amp;&amp;j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&amp;&amp;b++;return b}(),a)a.hasOwnProperty(n)&amp;&amp;(!c&amp;&amp;!--m&amp;&amp;(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&amp;&amp;a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&amp;&amp;l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&amp;&amp;c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i&lt;a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&amp;&amp;h(j,l);else Object(a)===a&amp;&amp;h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&amp;&amp;b.addEventListener&amp;&amp;(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&amp;&amp;g(k.readyState)&amp;&amp;(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var _prum=[['id', '612e3d94173350001100005d'], ['mark', 'firstbyte',
      (new Date()).getTime()]]; (function() {
      var s=document.getElementsByTagName('script')[0],
      p=document.createElement('script');
      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-45638026-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NLv5i-3GSzSseLOfsMVDpopto1ZjqkoN81KYxqUUVkc"},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"2452346","apath":"\/biorxiv\/early\/2022\/03\/17\/2020.06.18.159913.atom","pisa":"biorxiv;2020.06.18.159913v3","processed":["highwire_math"],"markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"biorxiv;2020.06.18.159913v3"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqus":{"domain":"biorxivstage","url":"https:\/\/www.biorxiv.org\/content\/10.1101\/2020.06.18.159913v3","title":"Brainprints: identifying individuals from magnetoencephalograms","identifier":"node\/2452346"},"panels_ajax_pane":{"new-28877068-b9cb-4641-830b-b6b4638c98bb":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DNnmn7QkncZtOF3C3VB4eaEP9Ax5euZnEBvHvqZSRVMeXRY\\\\\\\/LYgF8YI6lNaE3cYEeEAPC\\\\\\\/MRGf9TBa\\\\\\\/7Za76NPIS+adrBkmghBB7fgCEu+gXnafgWIIExtUN6NirZYPBa5+9ZdTFwjwj3HLvt\\\\\\\/hxY4BSUcE9Nxqc0zgpZVLflGmh6YP5rm7fuyxRhnv9Y9yF+FBh9qmD7jCnxYdhyfNUKhu+i21ma79Uxg5NFpLi2NHy1n5Wy75WNjgv834d+T1DLH0idhWIaRnPG6TbNpQszE6ScZK4iE5nccBKEbL4F4zCpD\\\\\\\/HPEGWA9JnN5NwtQKLkt6\\\\\\\/W3IAD+aD8lqFzL7wrrWCiAB1JgHaP8cJyHIAjwPS1vRHOAt2xPAQ2eptAah5R+K38bbBHl\\\\\\\/C17aaUCAobcGb35UcqCR34F8CsPKUtlOzvXQpDTGCJmX14Vr707TYTDES1Vnb9TRax1nHmMLbPXuyNZsD9fertPPLgw8xgj\\\\\\\/IB8HQFFMruCS5k7NadNKtxH9z5qgzobIPjGHipfoWTBebe32l4sFBKjDzQBsnnQXEFJ8XCXDquc5MyW1k\\\\\\\/yZXjjskY\\\\\\\/jpLgwxWmvN\\\\\\\/WcwLU3jR3ZCx7z1RP+t3tIjnuKuUzBDIAehl3qqtRzvNamPtzlM2HeAzOt9KoK2HAs6KWDv9ZIAVtK+MFHBv66MYb98dYSvwzxMc1kdCCTo1bDtCuchz6kBl9PCCQKBqpmrP3FCrckC3Ip9FUFnif21NHGXds4krdK+MTiQBMXVaE6qzCLBhFbufjzbsnuOzB54DEndHccqIQHwOWxj7xEsO\\\\\\\/mvnu5qvePMgPcyVd5kn2+R0fULRc4yYI\\\\\\\/xqF3aEB8vvdUA771CEFa5gHcZASOQVQ5MCeSEsiJwQsgxpH\\\\\\\/oeuLBfLYr8Lt53cOlICLMtmUDkFzDhqQgqWp9taeHShabzHwYq+1U+gI1zY2uzVFoP0J5okpl5biIy4MbuotN6qytfuQvhUfOCr4fJPVEv86G5\\\\\\\/smesCQnsppuYpvNxeAp7smo8d2JfaUqTjhd4Dkhd+wIe2TX0hAlDce4J9RyGByJi1U0THfbMqP+T+Bbuok9rQeywbTwdfPOcVhl2kswpjrVomLPBQa2cSFcEzjsBq9p6IbZnDSOqZM3ltaMFEaKvFUElBv6FNyyJffM1cy9I2q8rARNACi2XQsmNYvHKQRY\\\\\\\/BsbyrHJ63hXWFFDdd4mZvHfHVJkWGL0pieouEQLuhC9WoUzilYu9JpVl+SJNYppUa\\\\\\\/CPRUXk31rnDaHSaPl3aFn9al5zZmD8cXST\\\\\\\/8f7tEXAYxWsNr4f0ZYOwvy\\\\\\\/Igkfe2+rvEPnuFeNnUpCFKV5g0DCH1btPeGHKJbbnYZ81eXzMv3\\\\\\\/57+5pFlJBItNo6glE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022zCp2JjzYwA\\\\\\\/pCejYBuyL1Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022f9d8845712f72c9fee5db7d9ff111eab\\\u0022}\u0022,\u0022hmac\u0022:\u00229b2d02b85260ad7d5fd6c69f16238a72d40eb3ebaa163c812be30cc36dc56ea7\u0022}"},"urlIsAjaxTrusted":{"\/content\/10.1101\/2020.06.18.159913v3.full":true},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"https:\/\/www.biorxiv.org\/sites\/default\/files\/bioRxiv_article.jpg"},"highwire_list_expand":{"is_collapsed":"1"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"eu_cookie_compliance":{"popup_enabled":1,"popup_agreed_enabled":0,"popup_hide_agreed":0,"popup_clicking_confirmation":1,"popup_scrolling_confirmation":false,"popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","use_mobile_message":false,"mobile_popup_html_info":"\u003Cdiv\u003E\n  \u003Cdiv class =\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n          \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022agree-button eu-cookie-compliance-default-button\u0022\u003EContinue\u003C\/button\u003E\n                    \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button\u0022\u003EFind out more\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","mobile_breakpoint":"768","popup_html_agreed":"\u003Cdiv\u003E\n  \u003Cdiv class=\u0022popup-content agreed\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Ch2\u003EThank you for accepting cookies\u003C\/h2\u003E\u003Cp\u003EYou can now hide this message or find out more about cookies.\u003C\/p\u003E    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022hide-popup-button eu-cookie-compliance-hide-button\u0022\u003EHide\u003C\/button\u003E\n              \u003Cbutton type=\u0022button\u0022 class=\u0022find-more-button eu-cookie-compliance-more-button-thank-you\u0022 \u003EMore info\u003C\/button\u003E\n          \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E","popup_use_bare_css":false,"popup_height":"auto","popup_width":"100%","popup_delay":1000,"popup_link":"\/help\/cookie-policy","popup_link_new_window":1,"popup_position":null,"popup_language":"en","store_consent":false,"better_support_for_screen_readers":0,"reload_page":0,"domain":"","popup_eu_only_js":0,"cookie_lifetime":"365","cookie_session":false,"disagree_do_not_show_popup":0,"method":"default","whitelisted_cookies":"","withdraw_markup":"\u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-tab\u0022\u003EPrivacy settings\u003C\/button\u003E\n\u003Cdiv class=\u0022eu-cookie-withdraw-banner\u0022\u003E\n  \u003Cdiv class=\u0022popup-content info\u0022\u003E\n    \u003Cdiv id=\u0022popup-text\u0022\u003E\n      \u003Cp\u003E\u0026lt;h2\u0026gt;We use cookies on this site to enhance your user experience\u0026lt;\/h2\u0026gt;\u0026lt;p\u0026gt;You have given your consent for us to set cookies.\u0026lt;\/p\u0026gt;\u003C\/p\u003E\n    \u003C\/div\u003E\n    \u003Cdiv id=\u0022popup-buttons\u0022\u003E\n      \u003Cbutton type=\u0022button\u0022 class=\u0022eu-cookie-withdraw-button\u0022\u003EWithdraw consent\u003C\/button\u003E\n    \u003C\/div\u003E\n  \u003C\/div\u003E\n\u003C\/div\u003E\n","withdraw_enabled":false},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1,"trackUrlFragments":1},"jnl_biorxiv_styles":{"defaultJCode":"biorxiv"},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--&gt;&lt;!]]&gt;
  </script>
 </head>
 <body class="html not-front not-logged-in page-node page-node- page-node-2452346 node-type-highwire-article context-content hw-default-jcode-biorxiv hw-article-type-article hw-article-category-new-results">
  <noscript>
   <iframe src="//www.googletagmanager.com/ns.html?id=GTM-M677548" height="0" width="0" style="display:none;visibility:hidden">
   </iframe>
  </noscript>
  <script type="text/javascript">
   (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&amp;l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');
  </script>
  <div id="skip-link">
   <a href="#main-content" class="element-invisible element-focusable">
    Skip to main content
   </a>
  </div>
  <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
   <header id="section-header" class="section section-header">
    <div id="zone-branding" class="zone zone-branding clearfix print-display-block container-30">
     <div class="grid-15 prefix-1 region region-branding print-display-block" id="region-branding">
      <div class="region-inner region-branding-inner">
       <div class="branding-data clearfix">
        <div class="logo-img">
         <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0">
          <img alt="bioRxiv" src="https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg"/>
         </a>
        </div>
       </div>
      </div>
     </div>
     <div class="grid-11 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
      <div class="region-inner region-branding-second-inner">
       <div class="block block-system block-menu block-main-menu block-system-main-menu odd block-without-title" id="block-system-main-menu">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <nav class="menubar-nav">
           <ul class="menu" role="menu">
            <li class="first leaf" role="menuitem">
             <a href="/" title="" class="" data-icon-position="" data-hide-link-title="0">
              Home
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/about-biorxiv" class="" data-icon-position="" data-hide-link-title="0">
              About
             </a>
            </li>
            <li class="leaf" role="menuitem">
             <a href="/submit-a-manuscript" class="" data-icon-position="" data-hide-link-title="0">
              Submit
             </a>
            </li>
            <li class="last leaf" role="menuitem">
             <a href="/content/alertsrss" title="" class="" data-icon-position="" data-hide-link-title="0">
              ALERTS / RSS
             </a>
            </li>
           </ul>
          </nav>
         </div>
        </div>
       </div>
       <div class="block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title" id="block-panels-mini-biorxiv-search-box">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_search_box">
           <div class="panel-panel panel-col">
            <div>
             <div class="panel-pane pane-highwire-seach-quicksearch">
              <div class="pane-content">
               <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10.1101/2020.06.18.159913v3.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8">
                <div>
                 <div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
                  <label class="element-invisible" for="search_rightsidebar_keywords_27254823">
                   Search for this keyword
                  </label>
                  <input placeholder="Search" type="text" id="search_rightsidebar_keywords_27254823" name="keywords" value="" size="60" maxlength="128" class="form-text"/>
                 </div>
                 <div class="button-wrapper button-mini">
                  <span class="icon-search">
                  </span>
                  <input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="search_rightsidebar_submit_1099129511" name="op" value="Search" class="form-submit"/>
                 </div>
                 <input type="hidden" name="form_build_id" value="form-6MtTPrAXKB1lGZIJDMTUDPtieb14a7q73vuhDs9JLAc"/>
                 <input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0"/>
                </div>
               </form>
              </div>
             </div>
             <div class="panel-separator">
             </div>
             <div class="panel-pane pane-custom pane-2 advanced-search-link">
              <div class="pane-content">
               <a href="/search">
                Advanced Search
               </a>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
    <div id="zone-header" class="zone zone-header clearfix container-30">
    </div>
   </header>
   <section id="section-content" class="section section-content">
    <div id="zone-content" class="zone zone-content clearfix container-30">
     <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
      <div class="region-inner region-content-inner">
       <a id="main-content">
       </a>
       <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
        <div class="block-inner clearfix">
         <div class="content clearfix">
          <div class="panel-display panels-960-layout jcore-2col-layout">
           <div class="panel-row-wrapper clearfix">
            <div class="main-content-wrapper grid-17 suffix-1 alpha">
             <div class="panel-panel panel-region-content">
              <div class="inside">
               <div class="panel-pane pane-highwire-article-citation">
                <div class="pane-content">
                 <div class="highwire-article-citation highwire-citation-type-highwire-article node2452346" data-node-nid="2452346" id="node-2452346--21077588945" data-pisa="biorxiv;2020.06.18.159913v3" data-pisa-master="biorxiv;2020.06.18.159913" data-apath="/biorxiv/early/2022/03/17/2020.06.18.159913.atom" data-hw-author-tooltip-instance="highwire_author_tooltip">
                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-top clearfix has-author-tooltip">
                   <span class="biorxiv-article-type">
                    New Results
                   </span>
                   <h1 class="highwire-cite-title" id="page-title">
                    Brainprints: identifying individuals from magnetoencephalograms
                   </h1>
                   <div class="highwire-cite-authors">
                    <span class="highwire-citation-authors">
                     <span class="highwire-citation-author first" data-delta="0">
                      <span class="nlm-given-names">
                       Shenghao
                      </span>
                      <span class="nlm-surname">
                       Wu
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="1">
                      <span class="nlm-given-names">
                       Aaditya
                      </span>
                      <span class="nlm-surname">
                       Ramdas
                      </span>
                     </span>
                     ,
                     <span class="highwire-citation-author" data-delta="2">
                      <span class="nlm-given-names">
                       Leila
                      </span>
                      <span class="nlm-surname">
                       Wehbe
                      </span>
                     </span>
                    </span>
                   </div>
                   <div class="highwire-cite-metadata">
                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                     <span class="label">
                      doi:
                     </span>
                     https://doi.org/10.1101/2020.06.18.159913
                    </span>
                   </div>
                  </div>
                  <div id="hw-article-author-popups-node-2452346--21077588945" style="display: none;">
                   <div class="author-tooltip-0">
                    <div class="author-tooltip-name">
                     Shenghao Wu
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Neuroscience Institute, Carnegie Mellon University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Machine Learning Department, Carnegie Mellon University
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Shenghao%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wu%20S&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AShenghao%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-1">
                    <div class="author-tooltip-name">
                     Aaditya Ramdas
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Machine Learning Department, Carnegie Mellon University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        3
                       </span>
                       <span class="nlm-institution">
                        Department of Statistics and Data Science, Carnegie Mellon University
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Aaditya%2BRamdas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Ramdas%20A&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link last">
                      <a href="/search/author1%3AAaditya%2BRamdas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                    </ul>
                   </div>
                   <div class="author-tooltip-2">
                    <div class="author-tooltip-name">
                     Leila Wehbe
                    </div>
                    <div class="author-tooltip-affiliation">
                     <span class="author-tooltip-text">
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        1
                       </span>
                       <span class="nlm-institution">
                        Neuroscience Institute, Carnegie Mellon University
                       </span>
                      </div>
                      <div class="author-affiliation">
                       <span class="nlm-sup">
                        2
                       </span>
                       <span class="nlm-institution">
                        Machine Learning Department, Carnegie Mellon University
                       </span>
                      </div>
                     </span>
                    </div>
                    <ul class="author-tooltip-find-more">
                     <li class="author-tooltip-gs-link first">
                      <a href="/lookup/google-scholar?link_type=googlescholar&gs_type=author&author%5B0%5D=Leila%2BWehbe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on Google Scholar
                      </a>
                     </li>
                     <li class="author-tooltip-pubmed-link">
                      <a href="/lookup/external-ref?access_num=Wehbe%20L&link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">
                       Find this author on PubMed
                      </a>
                     </li>
                     <li class="author-site-search-link">
                      <a href="/search/author1%3ALeila%2BWehbe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">
                       Search for this author on this site
                      </a>
                     </li>
                     <li class="author-corresp-email-link last">
                      <span>
                       For correspondence:
                       <a href="mailto:lwehbe@cmu.edu" class="" data-icon-position="" data-hide-link-title="0">
                        lwehbe@cmu.edu
                       </a>
                      </span>
                     </li>
                    </ul>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs">
                <div class="pane-content">
                 <div class="item-list">
                  <ul class="tabs inline panels-ajax-tab">
                   <li class="first">
                    <a href="/content/10.1101/2020.06.18.159913v3" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:2452346" data-trigger="" data-url-enabled="1">
                     Abstract
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_art/node:2452346/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.18.159913v3.full-text" class="panels-ajax-tab-tab" data-panel-name="article_tab_full_text" data-target-id="highwire_article_tabs" data-entity-context="node:2452346" data-trigger="full-text" data-url-enabled="1">
                     Full Text
                    </a>
                    <a href="/panels_ajax_tab/article_tab_full_text/node:2452346/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.18.159913v3.article-info" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:2452346" data-trigger="article-info" data-url-enabled="1">
                     Info/History
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_info/node:2452346/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li>
                    <a href="/content/10.1101/2020.06.18.159913v3.article-metrics" class="panels-ajax-tab-tab" data-panel-name="article_tab_metrics" data-target-id="highwire_article_tabs" data-entity-context="node:2452346" data-trigger="article-metrics" data-url-enabled="1">
                     Metrics
                    </a>
                    <a href="/panels_ajax_tab/article_tab_metrics/node:2452346/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                   <li class="last">
                    <a href="/content/10.1101/2020.06.18.159913v3.full.pdf+html" class="panels-ajax-tab-tab" data-panel-name="biorxiv_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:2452346" data-trigger="full.pdf+html" data-url-enabled="1">
                     <i class="icon-file-alt">
                     </i>
                     Preview PDF
                    </a>
                    <a href="/panels_ajax_tab/biorxiv_tab_pdf/node:2452346/1" rel="nofollow" style="display:none" class="js-crawler-link">
                    </a>
                   </li>
                  </ul>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-panel-tabs-container">
                <div class="pane-content">
                 <div data-panels-ajax-tab-preloaded="article_tab_full_text" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container">
                  <div class="panels-ajax-tab-loading" style="display:none">
                   <img class="loading" src="https://www.biorxiv.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading"/>
                  </div>
                  <div class="panels-ajax-tab-wrap-article_tab_full_text">
                   <div class="panel-display panel-1col clearfix">
                    <div class="panel-panel panel-col">
                     <div>
                      <div class="panel-pane pane-highwire-markup">
                       <div class="pane-content">
                        <div class="highwire-markup">
                         <div xmlns="http://www.w3.org/1999/xhtml" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" class="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                          <div class="article fulltext-view">
                           <span class="highwire-journal-article-marker-start">
                           </span>
                           <div class="section abstract" id="abstract-1">
                            <h2 class="">
                             Abstract
                            </h2>
                            <p id="p-2">
                             Magnetoencephalography (MEG) is used to study a wide variety of cognitive processes. Increasingly, researchers are adopting principles of open science and releasing their MEG data. While essential for reproducibility, sharing MEG data has unforeseen privacy risks. Individual differences may make a participant identifiable from their anonymized recordings. However, our ability to identify individuals based on these individual differences has not yet been assessed. Here, we propose interpretable MEG features to characterize individual difference. We term these features brainprints (brain fingerprints). We show through several datasets that brainprints accurately identify individuals across days, tasks, and even between MEG and Electroencephalography (EEG). Furthermore, we identify consistent brainprint components that are important for identification. We study the dependence of identifiability on the amount of data available. We also relate identifiability to the level of preprocessing, the experimental task. Our findings reveal specific aspects of individual variability in MEG. They also raise concerns about unregulated sharing of brain data, even if anonymized.
                            </p>
                            <div id="sec-1" class="subsection">
                             <div id="F1" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F1.large.jpg?width=800&height=600&carousel=1" title="Graphical abstract. Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Graphical abstract.</span> Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing.</div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F1.medium.gif" width="440" height="135"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F1.medium.gif" width="440" height="135"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml">
                               <span class="fig-label">
                                Figure 1:
                               </span>
                               <span class="caption-title">
                                Graphical abstract.
                               </span>
                               <p id="p-3" class="first-child">
                                Identifying which subject a segment of MEG data belongs to is strikingly easy when other data from the same session is available for every subject. We propose three types of interpretable features that can also be used to identify individuals across sessions with high accuracy. Identifiability of individuals is influenced by factors such as resting state vs. task state, components of each feature, the sample size and the level of preprocessing. Our results reveal aspects of individual variability in MEG signals and highlight privacy risks associated with MEG data sharing.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-2">
                            <h2 class="">
                             Introduction
                            </h2>
                            <p id="p-6">
                             The open science movement is a result of the increasing awareness of the importance of sharing data and code to promote scientific reproducibility [
                             <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">
                              1
                             </a>
                             ]. Public repositories enable researchers to share their neuroimaging data (fMRI, EEG, MEG, etc.) while making sure to censor out individual information [
                             <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">
                              2
                             </a>
                             ]. However, data anonymization does not always preserve privacy [
                             <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">
                              3
                             </a>
                             ]. Combining different types of information using methods such as record linkage approaches [
                             <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">
                              4
                             </a>
                             ] may cause serious privacy violations. This problem is exacerbated when multiple datasets that happen to contain the same individual are available, which is rather common in neuroimaging (e.g. [
                             <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">
                              5
                             </a>
                             ]). Hence it is natural to ask if anonymized individuals can be identified from neuroimaging datasets and if so to what degree. Specifically, we ask: do individuals have a brainprint, a brain-activity analog of a fingerprint? If there is evidence for a brainprint, then researchers may be warned about how easily individual information can be inferred, and it may cause them (and the field) to act with more caution when publishing neuroimaging data online. For instance, it may pave the way for the adoption of more sophisticated data-release mechanisms like differential privacy [
                             <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">
                              6
                             </a>
                             ] and homomorphic encryption [
                             <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">
                              7
                             </a>
                             ].
                            </p>
                            <p id="p-7">
                             Assume there are two multi-individual neuroimaging datasets with overlapping participants: a “source” dataset and a “target” dataset. The question of interest is: can we accurately decide which individual in the source dataset corresponds to the individual in the target set? In other words, is there individual identifiability between the two datasets? The aforementioned question could arise naturally in practice: it is very common for university labs to recruit their own lab members for preliminary studies; these are anonymously released with an associated publication. Assume that one year later, lab member A relocates to city B, and privately volunteers for a study by a public hospital that tracks the effect of a drug (or some intervention) on patients in early stages of early-onset Alzheimer’s, while collecting MEG data. If this data is also anonymously released at a future point, brainprints could plausibly be used to detect a common participant, thus identifying that A has Alzheimer’s because only one member of the lab moved to city B. This would already be a gross unintended violation of privacy, but one can further imagine that an insurance company uses this to prove that a condition was pre-existing at the time of the first scan (before the individual themselves knew), or use it to decide individual-level pricing.
                            </p>
                            <p id="p-8">
                             If high individual identifiability exists even if the source and target set were recorded in separate sessions for each individual, there might be essential differences in the patterns of the data among individuals which is preserved across scanning sessions. Namely, individual identifiability might be related to variability in brain structure or function (or other individual characteristics such as head size). In multi-individual, multi-session neuroimaging data, there exists “within-session” variability across individuals in the same session and “cross-session” variability of the same individual cross sessions [
                             <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ]. For simplicity, consider the four scenarios in
                             <a id="xref-fig-2-1" class="xref-fig" href="#F2">
                              Fig. 2
                             </a>
                             . Low variability in both within-session (individuals are similar) and cross-session (an individual’s data is consistent across session) is likely to promote statistical power for detecting average group effects with fixed sample size, thereby facilitating reproducibility [
                             <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">
                              9
                             </a>
                             ,
                             <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">
                              10
                             </a>
                             ]. High cross-session and low within-session variability (e.g. individual 1’s data in session 1 is very different from their data for session 2, but somehow very similar to individual 2’s data in session 1) may indicate session-specific artifacts (e.g. the scanner was faulty during the recording of session 1 for all individuals). High cross- and within-session variability makes data unreliable. Finally, high within-session (individuals are different from each other) and low cross-session variability (individuals are similar to themselves) leads to individual identifiability. Individual identifiability in turn indicates consistent individual differences, which in themselves are an important topic of scientific enquiry [
                             <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ]. Understanding sources of consistent variability can help learn the underpinnings of disease or more generally to map the relationship of brain structure and activity to individual behavioral characteristics.
                            </p>
                            <div id="F2" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F2.large.jpg?width=800&height=600&carousel=1" title="Individual identifiability is a function of individual and session variability in neuroimaging. Consider repeating an experiment in multiple sessions for a group of individuals. Cross-session variability refers to the change in the recorded data for the same individual across sessions, while within-session variability refers to differences in a single session’s recorded data across individuals (keeping all other variables, including the stimulus, unchanged). The ideal conditions for the scientific discovery of an effect shared by the group is low within-session and low cross-session variability. However, the combination of low within-session and high cross-session variability indicates an artifact or a confound in the experiment design (e.g. each month, one session is recorded for all individuals and the instrument has a drift over time). High within-session variability paired with low cross-session variability leads to individual identifiability with the individual’s data acting like a stable signature that differentiates them from others. Finally, high within-session and cross-session variability lead to unreliable data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Individual identifiability is a function of individual and session variability in neuroimaging.</span> Consider repeating an experiment in multiple sessions for a group of individuals. Cross-session variability refers to the change in the recorded data for the same individual across sessions, while within-session variability refers to differences in a single session’s recorded data across individuals (keeping all other variables, including the stimulus, unchanged). The ideal conditions for the scientific discovery of an effect shared by the group is low within-session and low cross-session variability. However, the combination of low within-session and high cross-session variability indicates an artifact or a confound in the experiment design (e.g. each month, one session is recorded for all individuals and the instrument has a drift over time). High within-session variability paired with low cross-session variability leads to individual identifiability with the individual’s data acting like a stable signature that differentiates them from others. Finally, high within-session and cross-session variability lead to unreliable data.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure 2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F2.medium.gif" width="440" height="430"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure 2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F2.medium.gif" width="440" height="430"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure 2:
                              </span>
                              <span class="caption-title">
                               Individual identifiability is a function of individual and session variability in neuroimaging.
                              </span>
                              <p id="p-9" class="first-child">
                               Consider repeating an experiment in multiple sessions for a group of individuals. Cross-session variability refers to the change in the recorded data for the same individual across sessions, while within-session variability refers to differences in a single session’s recorded data across individuals (keeping all other variables, including the stimulus, unchanged). The ideal conditions for the scientific discovery of an effect shared by the group is low within-session and low cross-session variability. However, the combination of low within-session and high cross-session variability indicates an artifact or a confound in the experiment design (e.g. each month, one session is recorded for all individuals and the instrument has a drift over time). High within-session variability paired with low cross-session variability leads to individual identifiability with the individual’s data acting like a stable signature that differentiates them from others. Finally, high within-session and cross-session variability lead to unreliable data.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <p id="p-10">
                             Similar individual identification problems have been studied using EEG and fMRI for the purpose of biometric authentication and to investigate individual differences [
                             <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">
                              12
                             </a>
                             ,
                             <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">
                              13
                             </a>
                             ,
                             <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ,
                             <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">
                              16
                             </a>
                             ,
                             <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">
                              17
                             </a>
                             ,
                             <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">
                              18
                             </a>
                             ]. The term ‘brainprint’ has also been previously used to represent brain-specific information, such as morphology and event related potential biometrics [
                             <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">
                              19
                             </a>
                             ,
                             <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">
                              20
                             </a>
                             ,
                             <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">
                              21
                             </a>
                             ], that can be used to identify individuals. Individual identification with MEG data, however, has not been fully explored. Due to availability of MEG datasets, only single-trial MEG data has been studied for person identification [
                             <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">
                              22
                             </a>
                             ]. Other MEG studies focusing on variability of individual data [
                             <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">
                              8
                             </a>
                             ,
                             <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">
                              11
                             </a>
                             ] may not make connections with individual identifiability. Cross-modality identifiability has also not been explored.
                            </p>
                            <p id="p-11">
                             In this paper, we argue that individuals can be easily identified with MEG data. We measure identifiability as identification accuracy with three interpretable MEG features on multiple public and private MEG datasets. We show that identifiability is not a product of environmental artifacts and specific features have a consistent performance between task and resting state data. We further demonstrate that identifiability is preserved even between MEG and EEG datasets. We dissect the contribution of each features into sub-features to understand what may be leading to the high identifiability. Factors such as the amount of data and level of preprocessing are also shown to have influence on identifiability. Our analysis not only confirms the worrisome potential of privacy being compromised by released MEG data via extracting simple features but also leverage the interpretability of the features to explain the underlying mechanism for the high identifiability, thereby relating it to individual variability.
                            </p>
                           </div>
                           <div class="section" id="sec-3">
                            <h2 class="">
                             Results
                            </h2>
                            <p id="p-12">
                             In the result section, we first confirm the existence of the fingerprint using data of a single session, of multiple-sessions, of multiple-tasks, and even of multiple recording modalities. We use machine learning tools as well as interpretable features to show that identification is easy when the MEG sessions were collected on a single visit. We then show that the proposed features also achieve high accuracy on datasets of multiple visits to the scanner, and some features are even consistent on datasets between different tasks and from imaging recording modalities. We finally show which components of each feature is important for individual identification, and that sample size and level of preprocessing will also affect identification accuracy.
                            </p>
                            <div id="sec-4" class="subsection">
                             <h3>
                              Within-session identification is surprisingly easy
                             </h3>
                             <p id="p-13">
                              To measure identifiability, we consider the test accuracy of a classifier trained to identify participants from their MEG recording. We first focus on within session identifiability. In this context, we assume that each participant undergoes one session. A classifier is trained on a subset of the session, in which each trial is labeled with the identity of the participant it corresponds to. In our framework, we refer to the training set as the source set. Then, on held-out test data, the classifier predicts which participant is associated with each test trial. We refer to the test set as the target set. As an example, we investigated individual identifiability on a MEG dataset of eight participants during a reading task. Participants were asked to read a chapter of Harry Potter [
                              <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] while each word was presented for 0.5 s on a screen. There were 5176 trials (words) for each individual. The data was recorded using the Elekta Neuromag system. The Harry Potter (HP) data is a single-session dataset: the data for each individual were collected on a single visit of the MEG scanner. Hence the source and target set are non-overlapping subsets of that single session. We preprocessed and downsampled the data from 1000 Hz to 200 Hz so that there are 100 time points for each word. We trained a random forest classifier [
                              <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">
                               24
                              </a>
                              ] using the MEG recording of all channels at a randomly selected time point, a flattened vector representing the snapshot of the topographic map (topomap) of the brain activity (see Methods and
                              <a id="xref-fig-12-1" class="xref-fig" href="#F12">
                               Fig. S3
                              </a>
                              ). Random forest is a powerful classifier that uses a majority vote of a number of decision trees to predict the label associated with a given feature. Under this setting, we are asking if there is any individual-specific information contained in the topomap, the basic element of MEG recording. We split the dataset into 10 non-overlapping folds and used one as the target (testing) set and the other nine as the source (training) set. This 10-fold cross-validation scheme yielded a high identification accuracy (0.94) while the chance accuracy is only 0.125. We also repeated the analysis by only sampling one topomap from each trial to deflate possible statistical dependency and still obtained an accuracy of 0.923. This surprisingly high accuracy on merely 0.05 s of MEG data suggests the existence of strong patterns detected by the random forest classifier. This strong pattern may be contained on the transient spatial distribution of an individual’s MEG activity and is strongly distinctive of an individual. This high accuracy with the limited amount of information used suggests that within-session identification is a strikingly easy task.
                             </p>
                            </div>
                            <div id="sec-5" class="subsection">
                             <h3>
                              Interpretable MEG features yield high identification accuracy
                             </h3>
                             <p id="p-14">
                              The random forest classifier may not enclose enough information to explain the high identifiability of the HP data because of the black-box nature of the algorithm. The topomap mainly contains the spatial information: how heterogeneous the amplitude of the signal is across channels at a certain time point. High identifiability may also be attained using temporal and frequency information. We proposed three interpretable features for individual identification to further disseminate the individual-specific information. These features are interpretable because they bear biological meanings and hence can be used to interpret the high identification accuracy. The three features were based on n randomly selected trials (words) which have the shape [102 channels, 100 time points, n trials] (
                              <a id="xref-fig-3-1" class="xref-fig" href="#F3">
                               Fig. 3(a)
                              </a>
                              ).
                              <strong>
                               sp
                              </strong>
                              (
                              <a id="xref-fig-3-2" class="xref-fig" href="#F3">
                               Fig. 3(b)
                              </a>
                              ) is the spatial correlation between different sensors which may be related to individual-specific correlated activities between areas of the brain or the anatomy of the individual (e.g. brain size) [
                              <a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">
                               25
                              </a>
                              ].
                              <strong>
                               tp
                              </strong>
                              (
                              <a id="xref-fig-3-3" class="xref-fig" href="#F3">
                               Fig. 3(c)
                              </a>
                              ) is the temporal correlation between the time points into a trial. A high value in the
                              <strong>
                               tp
                              </strong>
                              matrix indicates highly synchronous brain signals between two temporal points, which might be related to participant specific stimulus processing latencies. A relevant study shows that the temporal change of brain activities in auditory steady-state responses are different between individuals [
                              <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">
                               26
                              </a>
                              ].
                              <strong>
                               fq
                              </strong>
                              (
                              <a id="xref-fig-3-4" class="xref-fig" href="#F3">
                               Fig. 3(d)
                              </a>
                              ) represents the distribution of the power intensity of signal frequency. Individual differences might also manifest as differences in the power distribution along frequency bands [
                              <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">
                               27
                              </a>
                              ,
                              <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">
                               22
                              </a>
                              ].
                             </p>
                             <div id="F3" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F3.large.jpg?width=800&height=600&carousel=1" title="High within-session identification accuracy on HP data with three interpretable features. (a) Shape of the HP data before featurization. The HP data consists of participants reading a book chapter one word at a time for 0.5s each. The data is resampled to have the dimension [102 channels, 100 time points, n trials] where each trial corresponds to one word and n to the number of words. (b) The spatial correlation feature sp is a 102 × 102 Pearson’s correlation coefficient matrix computed across the time points and trials. (c) The temporal correlation feature tp is a 100 × 100 Pearson’s correlation matrix computed across the channels and trials. (d) The frequency feature fq is a vector in ℝ51 where 51 is the number of frequency bands. The power at each band was averaged across channels and trials. (e) Identification accuracy with the three features. The accuracy was averaged across 100 identification runs of 8 individuals. The red dashed line represents the chance level (= 0.125). The error bars are the standard errors across individuals and identification runs and are invisible since they are all zeros." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">High within-session identification accuracy on HP data with three interpretable features.</span> <strong>(a)</strong> Shape of the HP data before featurization. The HP data consists of participants reading a book chapter one word at a time for 0.5s each. The data is resampled to have the dimension [102 channels, 100 time points, <em>n</em> trials] where each trial corresponds to one word and <em>n</em> to the number of words. <strong>(b)</strong> The spatial correlation feature <strong>sp</strong> is a 102 × 102 Pearson’s correlation coefficient matrix computed across the time points and trials. <strong>(c)</strong> The temporal correlation feature <strong>tp</strong> is a 100 × 100 Pearson’s correlation matrix computed across the channels and trials. <strong>(d)</strong> The frequency feature <strong>fq</strong> is a vector in ℝ<sup>51</sup> where 51 is the number of frequency bands. The power at each band was averaged across channels and trials. <strong>(e)</strong> Identification accuracy with the three features. The accuracy was averaged across 100 identification runs of 8 individuals. The red dashed line represents the chance level (= 0.125). The error bars are the standard errors across individuals and identification runs and are invisible since they are all zeros.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F3.medium.gif" width="440" height="100"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F3.medium.gif" width="440" height="100"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 3:
                               </span>
                               <span class="caption-title">
                                High within-session identification accuracy on HP data with three interpretable features.
                               </span>
                               <p id="p-15" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Shape of the HP data before featurization. The HP data consists of participants reading a book chapter one word at a time for 0.5s each. The data is resampled to have the dimension [102 channels, 100 time points, n trials] where each trial corresponds to one word and n to the number of words.
                                <strong>
                                 (b)
                                </strong>
                                The spatial correlation feature
                                <strong>
                                 sp
                                </strong>
                                is a 102 × 102 Pearson’s correlation coefficient matrix computed across the time points and trials.
                                <strong>
                                 (c)
                                </strong>
                                The temporal correlation feature
                                <strong>
                                 tp
                                </strong>
                                is a 100 × 100 Pearson’s correlation matrix computed across the channels and trials.
                                <strong>
                                 (d)
                                </strong>
                                The frequency feature
                                <strong>
                                 fq
                                </strong>
                                is a vector in ℝ
                                <sup>
                                 51
                                </sup>
                                where 51 is the number of frequency bands. The power at each band was averaged across channels and trials.
                                <strong>
                                 (e)
                                </strong>
                                Identification accuracy with the three features. The accuracy was averaged across 100 identification runs of 8 individuals. The red dashed line represents the chance level (= 0.125). The error bars are the standard errors across individuals and identification runs and are invisible since they are all zeros.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-16">
                              We used the 1-Nearest Neighbor (1NN) identification procedure, similar to Finn et al. [
                              <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ], to test if the three features are brainprints for the within-session identification task. For a given feature such as
                              <strong>
                               sp
                              </strong>
                              , the feature was computed on the source set using n randomly sampled trials (n = 300 for the HP data). Target set features were also computed in the same way (but unlabeled) with the same number of trials. The 1NN classifier simply assigned each target feature to the participant with the closest source feature (we used correlation to measure distance). The aforementioned matching process was repeated for 100 runs to account for the variance of the feature on the sampled trials, and the accuracy was averaged across these 100 runs. The simplicity of this 1NN classifier optimizes the interpretability of the result. With n = 300 trials all three features achieve perfect identification accuracy (
                              <a id="xref-fig-3-5" class="xref-fig" href="#F3">
                               Fig. 3(e)
                              </a>
                              , the accuracy for
                              <strong>
                               sp, tp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              is 1 ± 0, mean ± SE, p < 0.0002, see Supplement B for how we computed the p-values). In fact, the high identifiability can be attained with as few as n = 100 trials (
                              <a id="xref-fig-11-1" class="xref-fig" href="#F11">
                               Fig. S2(a)
                              </a>
                              ). The high identifiability with
                              <strong>
                               sp, tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              suggests they are brainprints, at least for identifying individuals within a session. Therefore, multiple features capturing different aspects of the MEG activity can be used for identifying individuals.
                             </p>
                            </div>
                            <div id="sec-6" class="subsection">
                             <h3>
                              Cross-session identification confirms the existence of brainprints
                             </h3>
                             <p id="p-17">
                              The high within-session identification accuracy suggests
                              <strong>
                               sp, tp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              are individual-specific within a session. Artifacts such as environmental noise and equipment configurations, however, might be the main contributing factor to within-session identification accuracy. Hence, we examined the consistency of the three features when the same type of task data was collected from each individual on multiple sessions. This setting tests if the features are preserved over time, i.e. if they are indeed brainprints and not mere artifacts. If the identifiability is significantly lower on multi-session datasets, the high identifiability on the HP data may be a mere result of session-specific artifacts, since the recording session for each individual is performed on different days. If high cross-session identifiability is observed,
                              <strong>
                               sp, tp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              can be considered genuine brainprints because they are unique to individual and invariant between sessions. This would also suggest low cross-session and high within-session variability (
                              <a id="xref-fig-2-2" class="xref-fig" href="#F2">
                               Fig. 2
                              </a>
                              ).
                             </p>
                             <p id="p-18">
                              We tested the three features on two multi-session datasets: FST [
                              <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], a four-session dataset where four individuals were shown pictures of familiar and unfamiliar faces with 1464 trials and SEN, a three-session dataset where four individuals were shown sentences with 3575 trials. Both recordings were recorded with the Elekta Neuromag sytem and were preprocessed and downsampled from 1000 Hz to 200 Hz so that there were 100 time points in one picture/sentence which we considered as one trial (see Methods). Since each individual has recordings conducted on different days, we set the target and source data to be from different sessions (
                              <a id="xref-fig-4-1" class="xref-fig" href="#F4">
                               Fig. 4(a)
                              </a>
                              ), to test the role of environmental artifacts and further confirm the existence of the brainprints. In addition to identification accuracy which is binary on one matching procedure, we used a continuous version, the rank accuracy, which captures more information in a failure case where an individual is misidentified. Rank accuracy captures the rank of the correct assignment out of all possible assignments; it is 1 if the target feature of each individual have the largest similarity to the source features for that individual, and is
                              <span class="inline-formula" id="inline-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-1.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-1.gif"/>
                                </noscript>
                               </span>
                              </span>
                              if the similarity is the smallest. The chance rank accuracy is
                              <span class="inline-formula" id="inline-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-2.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-2.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . In addition to the identification and rank accuracy, we also used a metric, differential identifiability [
                              <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ] which measures the similarity between the features of the same individual as compared to that of other individuals (see Methods and
                              <a id="xref-fig-18-1" class="xref-fig" href="#F18">
                               Fig. S9
                              </a>
                              ).
                             </p>
                             <div id="F4" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F4.large.jpg?width=800&height=600&carousel=1" title="Cross-session identification on FST and SEN data confirms existence of brainprints. (a) Schema of the cross-session identification task. For one identification run, the features of each individual are computed using randomly sampled trials (N = 300) from both the source and target session. Target session features are then classified by selecting the individual with the largest similarity score in the source session. (b) Heat maps of the cross-session identification accuracy using the three features on FST data. Each grid represents the average accuracy across 4 individuals and 100 identification runs. The within-session accuracy (diagonal entries) are computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. (c) Average cross-session identification accuracy and rank accuracy for each feature on FST data. Within-session accuracy (diagonal entries in (b)) were excluded in computation. Error bars are the SEs across cross-sessions (N = 12), individuals (N = 4), and identification runs (N = 100) and are invisible due to small values. Red dashed lines are the chance level for the identification accuracy (= 0.25) and rank accuracy (= 0.625). (d) Identification and rank accuracy on FST data by individual. Within-session accuracy were excluded in computation. Error bars are the SEs across cross-sessions (N = 12) and and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in (c). (e)–(g), same as (b)–(d) but on SEN data with the same number of individuals and identification runs (N = 4 and N = 100) but different number of cross-sessions (N = 6). The high identification accuracy with the three features on multi-session datasets confirms these features can be brainprints for individual identification." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Cross-session identification on FST and SEN data confirms existence of brainprints.</span> <strong>(a)</strong> Schema of the cross-session identification task. For one identification run, the features of each individual are computed using randomly sampled trials (<em>N</em> = 300) from both the source and target session. Target session features are then classified by selecting the individual with the largest similarity score in the source session. <strong>(b)</strong> Heat maps of the cross-session identification accuracy using the three features on FST data. Each grid represents the average accuracy across 4 individuals and 100 identification runs. The within-session accuracy (diagonal entries) are computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. <strong>(c)</strong> Average cross-session identification accuracy and rank accuracy for each feature on FST data. Within-session accuracy (diagonal entries in <strong>(b)</strong>) were excluded in computation. Error bars are the SEs across cross-sessions (<em>N</em> = 12), individuals (<em>N</em> = 4), and identification runs (<em>N</em> = 100) and are invisible due to small values. Red dashed lines are the chance level for the identification accuracy (= 0.25) and rank accuracy (= 0.625). <strong>(d)</strong> Identification and rank accuracy on FST data by individual. Within-session accuracy were excluded in computation. Error bars are the SEs across cross-sessions (<em>N</em> = 12) and and identification runs (<em>N</em> = 100) and are invisible due to small values. The red dashed lines are the same as in <strong>(c). (e)–(g)</strong>, same as <strong>(b)–(d)</strong> but on SEN data with the same number of individuals and identification runs (<em>N</em> = 4 and <em>N</em> = 100) but different number of cross-sessions (<em>N</em> = 6). The high identification accuracy with the three features on multi-session datasets confirms these features can be brainprints for individual identification.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F4.medium.gif" width="440" height="277"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F4.medium.gif" width="440" height="277"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 4:
                               </span>
                               <span class="caption-title">
                                Cross-session identification on FST and SEN data confirms existence of brainprints.
                               </span>
                               <p id="p-19" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Schema of the cross-session identification task. For one identification run, the features of each individual are computed using randomly sampled trials (N = 300) from both the source and target session. Target session features are then classified by selecting the individual with the largest similarity score in the source session.
                                <strong>
                                 (b)
                                </strong>
                                Heat maps of the cross-session identification accuracy using the three features on FST data. Each grid represents the average accuracy across 4 individuals and 100 identification runs. The within-session accuracy (diagonal entries) are computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage.
                                <strong>
                                 (c)
                                </strong>
                                Average cross-session identification accuracy and rank accuracy for each feature on FST data. Within-session accuracy (diagonal entries in
                                <strong>
                                 (b)
                                </strong>
                                ) were excluded in computation. Error bars are the SEs across cross-sessions (N = 12), individuals (N = 4), and identification runs (N = 100) and are invisible due to small values. Red dashed lines are the chance level for the identification accuracy (= 0.25) and rank accuracy (= 0.625).
                                <strong>
                                 (d)
                                </strong>
                                Identification and rank accuracy on FST data by individual. Within-session accuracy were excluded in computation. Error bars are the SEs across cross-sessions (N = 12) and and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in
                                <strong>
                                 (c). (e)–(g)
                                </strong>
                                , same as
                                <strong>
                                 (b)–(d)
                                </strong>
                                but on SEN data with the same number of individuals and identification runs (N = 4 and N = 100) but different number of cross-sessions (N = 6). The high identification accuracy with the three features on multi-session datasets confirms these features can be brainprints for individual identification.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-20">
                              Both
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              achieve almost perfect average identification and rank accuracy on both FST and SEN data whereas
                              <strong>
                               sp
                              </strong>
                              achieved lower but still well above-chance accuracy (
                              <a id="xref-fig-4-2" class="xref-fig" href="#F4">
                               Fig. 4(c,f)
                              </a>
                              ). The high cross-session identification accuracy of
                              <strong>
                               sp, tp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              confirms that it is reasonable to call them brainprints for individual identification in MEG. The lower identification accuracy for
                              <strong>
                               sp
                              </strong>
                              is due to low accuracy on a two of the individuals (
                              <a id="xref-fig-4-3" class="xref-fig" href="#F4">
                               Fig. 4(d,g)
                              </a>
                              ) in both datasets. This is also confirmed using the confusion matrices (
                              <a id="xref-fig-16-1" class="xref-fig" href="#F16">
                               Fig. S7
                              </a>
                              However, identification accuracy of these individuals is not consistently low across all session pairs (
                              <a id="xref-fig-4-4" class="xref-fig" href="#F4">
                               Fig. 4(b,e)
                              </a>
                              ) indicating that
                              <strong>
                               sp
                              </strong>
                              only perform worse for these subjects between certain sessions.
                             </p>
                             <p id="p-21">
                              For SEN data, the MEG recording of two subjects were taken on the same day for session 1 and 2. Since the identification accuracy of
                              <strong>
                               sp
                              </strong>
                              corresponding to these two pair of sessions (1 vs 2 and 2 vs 1) does not yield higher accuracy than the average (the mean identification accuracy between these two session pairs is 0.655, lower than 0.72, the mean across all cross-session pairs), the accuracy for
                              <strong>
                               sp
                              </strong>
                              is not inflated due to this issue with duplicated recording times. In line with the results on the HP dataset,
                              <strong>
                               sp, tp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              are the brainprints that are consistent even between recording sessions with
                              <strong>
                               tp, fq
                              </strong>
                              leading to higher identifiability.
                             </p>
                            </div>
                            <div id="sec-7" class="subsection">
                             <h3>
                              Spatial brainprints are consistent across resting-state and tasks
                             </h3>
                             <p id="p-22">
                              The high performance and interpretability of the brainprints make it enticing to study the factors and the underlying mechanism for identification. We looked at the performance of these features between two sessions of different types collected on the same day to test their consistency between different brain states. We compared the features using the Human Connectome Project (HCP) MEG data [
                              <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ] between a resting-state session (422 trials in average) in which individuals (N = 77) rest and do not perform a task and a task-MEG session (372 trials in average) where these same individuals view images and perform a working memory task. The dataset was recorded using the MAGNES 3600 system. We preprocessed and downsampled the data from 1024 Hz to 200 Hz and there were 500 time points in one trial in the WM data which corresponds to 2.5 s after the onset of the stimulus (see Methods). For the resting dataset, we simply reshape the recording into consecutive blocks similar to the WM dataset and performed the same analysis.
                             </p>
                             <p id="p-23">
                              Consistent with the cross-session results in
                              <a id="xref-fig-4-5" class="xref-fig" href="#F4">
                               Fig. 4
                              </a>
                              ,
                              <strong>
                               sp
                              </strong>
                              yields a high identification accuracy (
                              <a id="xref-fig-4-6" class="xref-fig" href="#F4">
                               Fig. 4(b)
                              </a>
                              , 0.77 ± 0.0034, mean ± SE, p < 0.0002), well above the 0.013 random baseline. This suggests that the spatial fingerprint is consistent between different brain states which confirms a similar finding in fMRI [
                              <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ]. The by-individual identification accuracy (
                              <a id="xref-fig-5-1" class="xref-fig" href="#F5">
                               Fig. 5(c)
                              </a>
                              ) shows that there is a small subset of individuals whose accuracy is below random, which may be due to the lack of head position correction in the HCP collection protocol.
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              do not perform as well as
                              <strong>
                               sp
                              </strong>
                              , suggesting that the temporal rhythm and frequency involved might be different between resting-state and task [
                              <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">
                               30
                              </a>
                              ,
                              <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">
                               31
                              </a>
                              ].
                             </p>
                             <div id="F5" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F5.large.jpg?width=800&height=600&carousel=1" title="Consistent sp for cross-task identification on Human Connectome Project data. (a) Heat maps of the cross-task identification accuracy using the three features on HP data. Both resting and working memory (WM) data were recorded on the same day. For one identification run, the features of each individual were computed using randomly sampled trials (N = 200) from both the source and target session. Each grid represents the average accuracy across 77 individuals and 100 identification runs. The within-task accuracy (diagonal entries) were computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. (b) Average cross-task identification accuracy and rank accuracy for each feature on HCP data. Within-task accuracy (resting vs. resting, WM vs. WM) are excluded in computation. Error bars are the SEs across cross-task sessions (N = 2), individuals (N = 77), and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy  and rank accuracy . (c) Identification (upper three rows) and rank (lower three rows) accuracy on HP data by individual. Within-task accuracy were excluded in computation. Error bars are the SEs across cross-task sessions (N = 2) and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in (b). These results indicate that sp is consistent even when performing different tasks (resting vs WM) in the source and target session." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Consistent sp for cross-task identification on Human Connectome Project data.</span> <strong>(a)</strong> Heat maps of the cross-task identification accuracy using the three features on HP data. Both resting and working memory (WM) data were recorded on the same day. For one identification run, the features of each individual were computed using randomly sampled trials (<em>N</em> = 200) from both the source and target session. Each grid represents the average accuracy across 77 individuals and 100 identification runs. The within-task accuracy (diagonal entries) were computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. <strong>(b)</strong> Average cross-task identification accuracy and rank accuracy for each feature on HCP data. Within-task accuracy (resting vs. resting, WM vs. WM) are excluded in computation. Error bars are the SEs across cross-task sessions (<em>N</em> = 2), individuals (<em>N</em> = 77), and identification runs (<em>N</em> = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy  and rank accuracy . <strong>(c)</strong> Identification (upper three rows) and rank (lower three rows) accuracy on HP data by individual. Within-task accuracy were excluded in computation. Error bars are the SEs across cross-task sessions (<em>N</em> = 2) and identification runs (<em>N</em> = 100) and are invisible due to small values. The red dashed lines are the same as in <strong>(b)</strong>. These results indicate that <strong>sp</strong> is consistent even when performing different tasks (resting vs WM) in the source and target session.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F5.medium.gif" width="440" height="91"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F5.medium.gif" width="440" height="91"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 5:
                               </span>
                               <span class="caption-title">
                                Consistent sp for cross-task identification on Human Connectome Project data.
                               </span>
                               <p id="p-24" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Heat maps of the cross-task identification accuracy using the three features on HP data. Both resting and working memory (WM) data were recorded on the same day. For one identification run, the features of each individual were computed using randomly sampled trials (N = 200) from both the source and target session. Each grid represents the average accuracy across 77 individuals and 100 identification runs. The within-task accuracy (diagonal entries) were computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage.
                                <strong>
                                 (b)
                                </strong>
                                Average cross-task identification accuracy and rank accuracy for each feature on HCP data. Within-task accuracy (resting vs. resting, WM vs. WM) are excluded in computation. Error bars are the SEs across cross-task sessions (N = 2), individuals (N = 77), and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy
                                <span class="inline-formula" id="inline-formula-3">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F5/embed/inline-graphic-3.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F5/embed/inline-graphic-3.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and rank accuracy
                                <span class="inline-formula" id="inline-formula-4">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F5/embed/inline-graphic-4.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F5/embed/inline-graphic-4.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                                <strong>
                                 (c)
                                </strong>
                                Identification (upper three rows) and rank (lower three rows) accuracy on HP data by individual. Within-task accuracy were excluded in computation. Error bars are the SEs across cross-task sessions (N = 2) and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in
                                <strong>
                                 (b)
                                </strong>
                                . These results indicate that
                                <strong>
                                 sp
                                </strong>
                                is consistent even when performing different tasks (resting vs WM) in the source and target session.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-25">
                              The rank accuracy of
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              (
                              <a id="xref-fig-5-2" class="xref-fig" href="#F5">
                               Fig. 5(b)
                              </a>
                              , 0.82 ± 0.0017 and 0.85 ± 0.0016, mean ± SE, p < 0.0002 for all) are much higher than the baseline (= 0.506). The majority of the individuals also have higher rank accuracy than baseline for
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              (
                              <a id="xref-fig-5-3" class="xref-fig" href="#F5">
                               Fig. 5(c)
                              </a>
                              ). The higher rank accuracy suggests that
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              may still contain individual-specific information but are not strong enough to achieve a high identification accuracy. Since the individuals perform different tasks on the source and target session, the rank accuracy indicates the potential consistent brainprint the generalizes beyond the task. It is noticeable that for the HCP dataset, the recording sessions of one individual were recorded on the same day. Hence one may exercise caution when extend the conclusions to cross-session datasets.
                             </p>
                            </div>
                            <div id="sec-8" class="subsection">
                             <h3>
                              Temporal and frequency brainprints are consistent across modalities
                             </h3>
                             <p id="p-26">
                              So far we have verified that the brainprints are consistent across visits, and even between resting and tasks. It would be a stronger piece of evidence if we show that brainprints can identify individuals during two visits to different centers with different recording modalities. We looked at MEG and EEG (electroencephalography) data of 14 participants viewing scene images (362 trials for each individual, one trial lasted 1 s), obtained from [
                              <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">
                               32
                              </a>
                              ]. Both MEG and EEG were recorded for the exact same stimuli, but on different days for each participant, making it an ideal testbed to verify the consistency of brainprints across different imaging modalities. We downsampled the MEG and EEG data from 1000 Hz and 512 Hz to 110 Hz so that there were 110 time points per trial. Since the spatial arrangements of MEG and EEG are different, we only tested the accuracy using
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              .
                             </p>
                             <p id="p-27">
                              Both features yield well above-chance identification accuracy and rank accuracy (
                              <a id="xref-fig-6-1" class="xref-fig" href="#F6">
                               Fig. 6
                              </a>
                              , identification accuracy for
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              is 0.45 and 0.46 whereas the chance accuracy is
                              <span class="inline-formula" id="inline-formula-5">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-5.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-5.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , same conclusion for the rank accuracy). This constitutes strong evidence that the frequency and temporal information of an individual’s response to stimuli are preserved even when different imaging modalities are used. The consistency also indicates that, at least for the temporal and frequency feature, the high accuracy is due to the individual-specific responses despite the possibility of different artifacts induced by MEG and EEG machines.
                             </p>
                             <div id="F6" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F6.large.jpg?width=800&height=600&carousel=1" title="Consistent tp and fq for cross-modality identification on MEG-EEG data. (a) Heat maps of the cross-modality identification accuracy using the two features on MEG-EEG data. MEG and EEG data for the same individual were recorded on different days. For one identification run, the features of each individual were computed using randomly sampled trials (N = 200) from both the source and target session. Each grid represents the average accuracy across 14 individuals and 100 identification runs. The within-task accuracy (diagonal entries) was computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. (b) Average cross-modality identification accuracy and rank accuracy for each feature. Within-modality accuracy (MEG vs. MEG, EEG vs. EEG) were excluded in the computation. Error bars are the SEs across cross-modality sessions (N = 2), individuals (N = 14), and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy  and rank accuracy . (c) Identification (upper two rows) and rank (lower two rows) accuracy on MEG-EEG data by individual. Within-modality accuracy is excluded in the computation. Error bars are the SEs across cross-modality sessions (N = 2) and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in (b). These results indicate that tp and fq are consistent even when different neuroimaging modalities were used in the source and target session." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Consistent tp and fq for cross-modality identification on MEG-EEG data.</span> <strong>(a)</strong> Heat maps of the cross-modality identification accuracy using the two features on MEG-EEG data. MEG and EEG data for the same individual were recorded on different days. For one identification run, the features of each individual were computed using randomly sampled trials (<em>N</em> = 200) from both the source and target session. Each grid represents the average accuracy across 14 individuals and 100 identification runs. The within-task accuracy (diagonal entries) was computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage. <strong>(b)</strong> Average cross-modality identification accuracy and rank accuracy for each feature. Within-modality accuracy (MEG vs. MEG, EEG vs. EEG) were excluded in the computation. Error bars are the SEs across cross-modality sessions (<em>N</em> = 2), individuals (<em>N</em> = 14), and identification runs (<em>N</em> = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy  and rank accuracy . <strong>(c)</strong> Identification (upper two rows) and rank (lower two rows) accuracy on MEG-EEG data by individual. Within-modality accuracy is excluded in the computation. Error bars are the SEs across cross-modality sessions (<em>N</em> = 2) and identification runs (<em>N</em> = 100) and are invisible due to small values. The red dashed lines are the same as in <strong>(b)</strong>. These results indicate that <strong>tp</strong> and <strong>fq</strong> are consistent even when different neuroimaging modalities were used in the source and target session.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F6.medium.gif" width="440" height="94"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F6.medium.gif" width="440" height="94"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 6:
                               </span>
                               <span class="caption-title">
                                Consistent tp and fq for cross-modality identification on MEG-EEG data.
                               </span>
                               <p id="p-28" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Heat maps of the cross-modality identification accuracy using the two features on MEG-EEG data. MEG and EEG data for the same individual were recorded on different days. For one identification run, the features of each individual were computed using randomly sampled trials (N = 200) from both the source and target session. Each grid represents the average accuracy across 14 individuals and 100 identification runs. The within-task accuracy (diagonal entries) was computed using the same source-target splitting procedure as on the Harry Potter data to avoid data leakage.
                                <strong>
                                 (b)
                                </strong>
                                Average cross-modality identification accuracy and rank accuracy for each feature. Within-modality accuracy (MEG vs. MEG, EEG vs. EEG) were excluded in the computation. Error bars are the SEs across cross-modality sessions (N = 2), individuals (N = 14), and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the chance level for the identification accuracy
                                <span class="inline-formula" id="inline-formula-6">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F6/embed/inline-graphic-6.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F6/embed/inline-graphic-6.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                and rank accuracy
                                <span class="inline-formula" id="inline-formula-7">
                                 <span class="highwire-responsive-lazyload">
                                  <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F6/embed/inline-graphic-7.gif"/>
                                  <noscript>
                                   <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/F6/embed/inline-graphic-7.gif"/>
                                  </noscript>
                                 </span>
                                </span>
                                .
                                <strong>
                                 (c)
                                </strong>
                                Identification (upper two rows) and rank (lower two rows) accuracy on MEG-EEG data by individual. Within-modality accuracy is excluded in the computation. Error bars are the SEs across cross-modality sessions (N = 2) and identification runs (N = 100) and are invisible due to small values. The red dashed lines are the same as in
                                <strong>
                                 (b)
                                </strong>
                                . These results indicate that
                                <strong>
                                 tp
                                </strong>
                                and
                                <strong>
                                 fq
                                </strong>
                                are consistent even when different neuroimaging modalities were used in the source and target session.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div id="sec-9" class="subsection">
                             <h3>
                              Not every part of a brainprint is equally important
                             </h3>
                             <p id="p-29">
                              What contributes to the high identifiability of the three brainprints? Understanding the relative contribution of the components of brainprints could help understand individual identifiability and variability. We divided the three brainprints into sub-features and looked at their identification accuracy to see which components contain the most individual-specific information.
                              <strong>
                               sp
                              </strong>
                              was divided into correlations between groups of sensors: Left Occipital (LO), Right Occipital (RO), Left Parietal (LP), Right Parietal (RP), Left Temporal (LT), Right Temporal (RT), Left Frontal (LF), Right Frontal (RF).
                              <strong>
                               tp
                              </strong>
                              was divided into correlations between time intervals.
                              <strong>
                               fq
                              </strong>
                              was divided into frequencies within a sliding window. We use the SEN and FST dataset to focus on cross-session patterns.
                             </p>
                             <p id="p-30">
                              For both SEN and FST, the correlations between sensors within Left Occipital (LO) and between LO and Right Parietal (RP) yielded high accuracy (
                              <a id="xref-fig-7-1" class="xref-fig" href="#F7">
                               Fig. 7(a)
                              </a>
                              , inset, and
                              <a id="xref-fig-19-1" class="xref-fig" href="#F19">
                               Fig. S10
                              </a>
                              ). LO is involved in visual processing [
                              <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">
                               33
                              </a>
                              ] and RP is involved in sensory integration [
                              <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">
                               34
                              </a>
                              ], both of which are functions recruited by the experimental task. Due to the nature of the sampled signal and the physical properties of the skull, each MEG sensor samples coarsely from the brain, making it hard to say whether MEG spatial correlation effectively corresponds to functional connectivity, especially for nearby sensors [
                              <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ]. However, the fact that correlations between faraway groups of sensors, for example, LT and RT, still have good accuracy suggesting it may be due to actual functional correlation between these areas, but it could still be the case that it is the difference in skull shapes that contributes to the high
                              <strong>
                               sp
                              </strong>
                              accuracy.
                             </p>
                             <div id="F7" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F7.large.jpg?width=800&height=600&carousel=1" title="Identification accuracy of components of the features. See Fig. S10 for a,b on FST data. (a) Identification accuracy of the sub-features of sp on SEN data. Each grid represents the identification accuracy using the corresponding entries of sp averaged across cross-sessions (N = 6), individuals (N = 4), and identification runs (N = 100). Inset is the plot of the sensor group layout and edges correspond to the sensor group pair with over 0.7 accuracy for both FST and SEN. The topomap was plotted using the python MNE package [37]. (b) Identification accuracy of the sub-features of tp on SEN data. Each grid represents the identification accuracy using the corresponding entries of tp averaged across the same dimensions as in a. Inset is an example MEG signal of one individual averaged across channels (N = 102) and trials (N = 1000). Arrows correspond to the entries of the heatmap with over 0.9 accuracy for both FST and SEN. (c) Identification accuracy of the sub-features of fq on SEN (upper plot) and FST (lower plot) data. Each dot represents the identification accuracy using the corresponding entries of tp averaged across cross-sessions (N = 6 for SEN and 12 for FST), individuals (N = 4), and identification runs (N = 100). Accuracy values of f larger than 60 Hz were truncated since the curve became flat. Error bars are SE across cross-sessions, individuals, and identification runs and are invisible due to small values. The curve peaks at f = 6 Hz for SEN and f = 8Hz for FST. The accuracy of some components of a feature is consistently higher than the rest on both datasets, indicating that some parts of a certain feature may be more important in identifying individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Identification accuracy of components of the features.</span> See Fig. S10 for <strong>a,b</strong> on FST data. <strong>(a)</strong> Identification accuracy of the sub-features of <strong>sp</strong> on SEN data. Each grid represents the identification accuracy using the corresponding entries of <strong>sp</strong> averaged across cross-sessions (<em>N</em> = 6), individuals (<em>N</em> = 4), and identification runs (<em>N</em> = 100). Inset is the plot of the sensor group layout and edges correspond to the sensor group pair with over 0.7 accuracy for both FST and SEN. The topomap was plotted using the python MNE package [37]. <strong>(b)</strong> Identification accuracy of the sub-features of <strong>tp</strong> on SEN data. Each grid represents the identification accuracy using the corresponding entries of <strong>tp</strong> averaged across the same dimensions as in <strong>a</strong>. Inset is an example MEG signal of one individual averaged across channels (<em>N</em> = 102) and trials (<em>N</em> = 1000). Arrows correspond to the entries of the heatmap with over 0.9 accuracy for both FST and SEN. <strong>(c)</strong> Identification accuracy of the sub-features of <strong>fq</strong> on SEN (upper plot) and FST (lower plot) data. Each dot represents the identification accuracy using the corresponding entries of <strong>tp</strong> averaged across cross-sessions (<em>N</em> = 6 for SEN and 12 for FST), individuals (<em>N</em> = 4), and identification runs (<em>N</em> = 100). Accuracy values of <em>f</em> larger than 60 Hz were truncated since the curve became flat. Error bars are SE across cross-sessions, individuals, and identification runs and are invisible due to small values. The curve peaks at <em>f</em> = 6 Hz for SEN and <em>f</em> = 8Hz for FST. The accuracy of some components of a feature is consistently higher than the rest on both datasets, indicating that some parts of a certain feature may be more important in identifying individuals.</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F7.medium.gif" width="440" height="171"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F7.medium.gif" width="440" height="171"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 7:
                               </span>
                               <span class="caption-title">
                                Identification accuracy of components of the features.
                               </span>
                               <p id="p-31" class="first-child">
                                See
                                <a id="xref-fig-19-2" class="xref-fig" href="#F19">
                                 Fig. S10
                                </a>
                                for
                                <strong>
                                 a,b
                                </strong>
                                on FST data.
                                <strong>
                                 (a)
                                </strong>
                                Identification accuracy of the sub-features of
                                <strong>
                                 sp
                                </strong>
                                on SEN data. Each grid represents the identification accuracy using the corresponding entries of
                                <strong>
                                 sp
                                </strong>
                                averaged across cross-sessions (N = 6), individuals (N = 4), and identification runs (N = 100). Inset is the plot of the sensor group layout and edges correspond to the sensor group pair with over 0.7 accuracy for both FST and SEN. The topomap was plotted using the python MNE package [
                                <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">
                                 37
                                </a>
                                ].
                                <strong>
                                 (b)
                                </strong>
                                Identification accuracy of the sub-features of
                                <strong>
                                 tp
                                </strong>
                                on SEN data. Each grid represents the identification accuracy using the corresponding entries of
                                <strong>
                                 tp
                                </strong>
                                averaged across the same dimensions as in
                                <strong>
                                 a
                                </strong>
                                . Inset is an example MEG signal of one individual averaged across channels (N = 102) and trials (N = 1000). Arrows correspond to the entries of the heatmap with over 0.9 accuracy for both FST and SEN.
                                <strong>
                                 (c)
                                </strong>
                                Identification accuracy of the sub-features of
                                <strong>
                                 fq
                                </strong>
                                on SEN (upper plot) and FST (lower plot) data. Each dot represents the identification accuracy using the corresponding entries of
                                <strong>
                                 tp
                                </strong>
                                averaged across cross-sessions (N = 6 for SEN and 12 for FST), individuals (N = 4), and identification runs (N = 100). Accuracy values of f larger than 60 Hz were truncated since the curve became flat. Error bars are SE across cross-sessions, individuals, and identification runs and are invisible due to small values. The curve peaks at f = 6 Hz for SEN and f = 8Hz for FST. The accuracy of some components of a feature is consistently higher than the rest on both datasets, indicating that some parts of a certain feature may be more important in identifying individuals.
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-32">
                              For both SEN and FST, the super-diagonal of the heat map for temporal sub-features (
                              <a id="xref-fig-7-2" class="xref-fig" href="#F7">
                               Fig. 7(b)
                              </a>
                              and
                              <a id="xref-fig-19-3" class="xref-fig" href="#F19">
                               Fig. S10
                              </a>
                              ) had high accuracy. The super-diagonal entries correspond to the cross-correlation of the MEG signal between two consecutive segments of 0.05 s. Hence the rhythm of the signal within a short segment of time contributes to identifiability, which can also be seen from the banded structure of
                              <strong>
                               tp
                              </strong>
                              (
                              <a id="xref-fig-2-3" class="xref-fig" href="#F2">
                               Fig. 2(c)
                              </a>
                              ). Moreover, the correlations between fourth and fifth 0.05 s yield considerably high accuracy on both datasets
                              <strong>
                               tp
                              </strong>
                              (
                              <a id="xref-fig-7-3" class="xref-fig" href="#F7">
                               Fig. 7(b)
                              </a>
                              inset). These time periods overlap with the time we expect the brain is processing word and picture stimuli[
                              <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">
                               35
                              </a>
                              ].
                             </p>
                             <p id="p-33">
                              The power intensity of frequencies between 4 and 13 Hz yields the highest accuracy on both SEN and FST data (
                              <a id="xref-fig-7-4" class="xref-fig" href="#F7">
                               Fig. 7(c)
                              </a>
                              ), the peak is 6 Hz for SEN and 8 Hz for FST. These peaks roughly corresponds to the Theta and Alpha frequency band which are related to the resting state, memory, and mental coordination [
                              <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ]. The accuracy is also moderately high on part of Beta band (14-31 Hz) where attention and concentration are recruited [
                              <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">
                               36
                              </a>
                              ]. We also grouped the frequencies into canonical frequency bands and discovered a similar pattern (
                              <a id="xref-fig-20-1" class="xref-fig" href="#F20">
                               Fig. S11
                              </a>
                              ).
                             </p>
                            </div>
                            <div id="sec-10" class="subsection">
                             <h3>
                              Identifiability changes with data size and preprocessing
                             </h3>
                             <p id="p-34">
                              The last dimensions that we investigate is the dependence of individual identification on the amount of available data and on the level of data preprocessing.
                             </p>
                             <p id="p-35">
                              We look at the identification accuracy using the three brainprints while increasing the sample size n. The identification accuracy increases with the amount of data used for computing
                              <strong>
                               sp, fp
                              </strong>
                              , and
                              <strong>
                               fq
                              </strong>
                              (
                              <a id="xref-fig-8-1" class="xref-fig" href="#F8">
                               Fig. 8(a)
                              </a>
                              ) as the sampling variance becomes smaller. In general, with 50 s of data, the brainprints perform well on cross-session identification of the same task.
                              <strong>
                               sp
                              </strong>
                              becomes reasonably accurate on the HCP dataset with 100 trials corresponding to 250 s of recording, possibly because more trials are required to accurately compute features that are distinguishable within a larger pool of individuals. For FST and SEN, the identification accuracy of
                              <strong>
                               sp
                              </strong>
                              saturates at fewer number of trials than
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              . It is possible that
                              <strong>
                               sp
                              </strong>
                              requires fewer trials to be estimated robustly.
                             </p>
                             <div id="F8" class="fig pos-float type-figure odd">
                              <div class="highwire-figure">
                               <div class="fig-inline-img-wrapper">
                                <div class="fig-inline-img">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F8.large.jpg?width=800&height=600&carousel=1" title="Factors affecting identification accuracy. (a) Identification accuracy with respect to the number of trials (sample size) used for the featurization of FST, SEN, and HCP data. Each dot represents the identification accuracy averaged across individuals, identification runs, and cross-sessions (or cross-task sessions) excluding the within-session or within-task results. Error bars are the SEs across the corresponding cross-sessions (or cross-task sessions), individuals, and identification runs of each dataset and are invisible due to small values. (b)-(c) Identification (b) and rank (c) accuracy of the three features computed on raw and fully preprocessed FST and SEN data. The same color represents the same feature as in (a). For (b), the identification accuracy across sessions (N = 12 for FST and = 6 for SEN) and individuals (N = 4) were averaged with respect to identification runs (N = 100) and were put into one vector (of N = 48 entries for FST and 24 entries for SEN) for each feature and level of preprocessing. The heights of the bar plots are the mean of the corresponding vector. A two-sided unpaired t-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the sp feature for SEN. For (c), the rank accuracy were put into one vector in the same way as in (b). The heights of the bar plots are the mean of the corresponding vector and the error bars are its SE A two-sided unpaired t-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the sp feature for SEN" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Factors affecting identification accuracy.</span> <strong>(a)</strong> Identification accuracy with respect to the number of trials (sample size) used for the featurization of FST, SEN, and HCP data. Each dot represents the identification accuracy averaged across individuals, identification runs, and cross-sessions (or cross-task sessions) excluding the within-session or within-task results. Error bars are the SEs across the corresponding cross-sessions (or cross-task sessions), individuals, and identification runs of each dataset and are invisible due to small values. <strong>(b)-(c)</strong> Identification <strong>(b)</strong> and rank <strong>(c)</strong> accuracy of the three features computed on raw and fully preprocessed FST and SEN data. The same color represents the same feature as in <strong>(a)</strong>. For <strong>(b)</strong>, the identification accuracy across sessions (<em>N</em> = 12 for FST and = 6 for SEN) and individuals (<em>N</em> = 4) were averaged with respect to identification runs (<em>N</em> = 100) and were put into one vector (of <em>N</em> = 48 entries for FST and 24 entries for SEN) for each feature and level of preprocessing. The heights of the bar plots are the mean of the corresponding vector. A two-sided unpaired <em>t</em>-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the <strong>sp</strong> feature for SEN. For <strong>(c)</strong>, the rank accuracy were put into one vector in the same way as in <strong>(b)</strong>. The heights of the bar plots are the mean of the corresponding vector and the error bars are its SE A two-sided unpaired <em>t</em>-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the <strong>sp</strong> feature for SEN</div></div>' data-icon-position="" data-hide-link-title="0">
                                  <span class="hw-responsive-img">
                                   <img class="highwire-fragment fragment-image lazyload" alt="Figure 8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F8.medium.gif" width="440" height="283"/>
                                   <noscript>
                                    <img class="highwire-fragment fragment-image" alt="Figure 8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F8.medium.gif" width="440" height="283"/>
                                   </noscript>
                                  </span>
                                 </a>
                                </div>
                               </div>
                               <ul class="highwire-figure-links inline">
                                <li class="download-fig first">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8:" data-icon-position="" data-hide-link-title="0">
                                  Download figure
                                 </a>
                                </li>
                                <li class="new-tab last">
                                 <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                  Open in new tab
                                 </a>
                                </li>
                               </ul>
                              </div>
                              <div class="fig-caption">
                               <span class="fig-label">
                                Figure 8:
                               </span>
                               <span class="caption-title">
                                Factors affecting identification accuracy.
                               </span>
                               <p id="p-36" class="first-child">
                                <strong>
                                 (a)
                                </strong>
                                Identification accuracy with respect to the number of trials (sample size) used for the featurization of FST, SEN, and HCP data. Each dot represents the identification accuracy averaged across individuals, identification runs, and cross-sessions (or cross-task sessions) excluding the within-session or within-task results. Error bars are the SEs across the corresponding cross-sessions (or cross-task sessions), individuals, and identification runs of each dataset and are invisible due to small values.
                                <strong>
                                 (b)-(c)
                                </strong>
                                Identification
                                <strong>
                                 (b)
                                </strong>
                                and rank
                                <strong>
                                 (c)
                                </strong>
                                accuracy of the three features computed on raw and fully preprocessed FST and SEN data. The same color represents the same feature as in
                                <strong>
                                 (a)
                                </strong>
                                . For
                                <strong>
                                 (b)
                                </strong>
                                , the identification accuracy across sessions (N = 12 for FST and = 6 for SEN) and individuals (N = 4) were averaged with respect to identification runs (N = 100) and were put into one vector (of N = 48 entries for FST and 24 entries for SEN) for each feature and level of preprocessing. The heights of the bar plots are the mean of the corresponding vector. A two-sided unpaired t-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the
                                <strong>
                                 sp
                                </strong>
                                feature for SEN. For
                                <strong>
                                 (c)
                                </strong>
                                , the rank accuracy were put into one vector in the same way as in
                                <strong>
                                 (b)
                                </strong>
                                . The heights of the bar plots are the mean of the corresponding vector and the error bars are its SE A two-sided unpaired t-test was performed on the vectors of the same feature and dataset between the raw and preprocessed data. The p-values for all pairs are less than 0.05, except for the
                                <strong>
                                 sp
                                </strong>
                                feature for SEN
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                             <p id="p-37">
                              Preprocessing may also affect identification accuracy. We compared the difference in the identification and rank accuracy between the raw and preprocessed data (
                              <a id="xref-fig-8-2" class="xref-fig" href="#F8">
                               Fig. 8 (b,c)
                              </a>
                              ). The changes in accuracy were all statistically significant (
                              <a id="xref-fig-8-3" class="xref-fig" href="#F8">
                               Fig. 8(b,c)
                              </a>
                              )) when the raw data was preprocessed for all the three features except for the
                              <strong>
                               sp
                              </strong>
                              feature for SEN. For both FST and SEN, preprocessing yielded better accuracy for
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              . However, for
                              <strong>
                               sp
                              </strong>
                              , the results point in opposite directions: preprocessing increases identifiability for FST and decreases it for SEN (though with little statistical significance). There was one difference in the preprocessing pipeline for both datasets: FST preprocessing did not include head position correction due to a lack of head position recordings whereas SEN does. Head position correction might be changing the signal in inhomogeneous ways thereby undermining the identifiability with
                              <strong>
                               sp
                              </strong>
                              . We also found that head size and average movement have a weak correlation with identification accuracy in the HCP data, (not statistically significant after multiple comparison correction), shown in
                              <a id="xref-fig-21-1" class="xref-fig" href="#F21">
                               Fig. S12
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-11">
                            <h2 class="">
                             Discussion
                            </h2>
                            <p id="p-38">
                             An individual can be identified with a number of differential characteristics, including their real ‘fingerprints’. Existing studies have suggested the existence a fingerprint in brain signals (e.g. [
                             <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">
                              14
                             </a>
                             ,
                             <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">
                              15
                             </a>
                             ]). In this paper, we argued that such brainprints also exist in MEG data and, in fact, there are multiple of them that capture different information from the MEG data. We showed that these brainprints are likely not by-products of environmental artifacts and may pertain to the underlying brain response to stimuli. These analyses, apart from adding to the existing evidence of the brainprints, may bear alarming meanings in privacy issues and provoke thoughts on how scientific conclusions based on multiple individuals have to be examined carefully given these consistent individual-specific features.
                            </p>
                            <p id="p-39">
                             In this section, we first discuss the implications of these results in detail, following the same order of the previous section. We then mention limitations and potential improvement to our analysis of brainprints.
                            </p>
                            <div id="sec-12" class="subsection">
                             <h3>
                              Within-session identifiability
                             </h3>
                             <p id="p-40">
                              Using the HP data, we showed that both random forest classification with topomaps and 1NN classification with certain interpretable features can be used to correctly identify individuals when the data is collected on a single session. The high accuracy based on merely 0.5 s of data for
                              <strong>
                               sp
                              </strong>
                              and 25 s for
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              is striking since small amounts of data usually leads to inaccurate estimates of these features, unless the underlying patterns are strong. The easy task of identifying individuals on single-session dataset points to strong individual-specific patterns which may or may not be brain-activity related.
                             </p>
                            </div>
                            <div id="sec-13" class="subsection">
                             <h3>
                              Uniqueness of brainprints
                             </h3>
                             <p id="p-41">
                              The three features we proposed may not be the only characteristics of MEG data that can be used for individual identification. However, these features represent fundamental aspects of MEG data (and even time series in general) hence they may be a vital first step to understand brainprints. Specifically, we propose the temporal feature,
                              <strong>
                               tp
                              </strong>
                              , because of the high temporal-resolution of MEG data. This feature may have not been used for other types of neuroimaging datasets, suggesting that different features may be informative depending on the nature of dataset of interest.
                             </p>
                            </div>
                            <div id="sec-14" class="subsection">
                             <h3>
                              Cross-session identifiability
                             </h3>
                             <p id="p-42">
                              The high cross-session identification accuracy using
                              <strong>
                               sp
                              </strong>
                              confirms it is a brainprint, and supports the previous literature on the similar features in fMRI and EEG [
                              <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ,
                              <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">
                               38
                              </a>
                              ]. The higher accuracy by
                              <strong>
                               sp, tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              suggest that multiple aspects of the individual activity captured by MEG may be used for identification. The generally lower accuracy from
                              <strong>
                               sp
                              </strong>
                              might be the result of the change of alignment of sensors for each individual. However, since necessary steps have taken in the preprocessing pipeline to align the sensors (Supplement A) and each MEG sensor measures brain activity from a non-trivially large area, it remains unclear if the issue is the alignment. Another interpretation of this result is that the temporal and frequency information is more consistent for an individual across time and the spatial information may slowly evolve over time (e.g. when the individual slowly moves during the recording).
                             </p>
                             <p id="p-43">
                              Some source-target session pairs have lower identification accuracy than others for
                              <strong>
                               sp
                              </strong>
                              (
                              <a id="xref-fig-4-7" class="xref-fig" href="#F4">
                               Fig. 4(b,e)
                              </a>
                              ) and the identification accuracy is not necessarily reciprocal, for example, 0.76 vs 1 (mean, session 3 as source, session 2 as target vs session 2 as source, session 3 as target) in FST. The lower accuracy of
                              <strong>
                               sp
                              </strong>
                              of on specific source-target sessions of specific individuals suggests that the identifiability of
                              <strong>
                               sp
                              </strong>
                              may not be uniform over time and individuals.
                             </p>
                             <p id="p-44">
                              The three highly identifiable features on FST and SEN represent an alarming message for experimentalists to consider before releasing MEG data. The existence of brainprints are also examples of certain functions of the MEG data with high cross-individual variability preserved across sessions, which has been widely discussed on various types of neuroimaging data [
                              <a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">
                               8
                              </a>
                              ,
                              <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">
                               39
                              </a>
                              ,
                              <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">
                               11
                              </a>
                              ,
                              <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">
                               40
                              </a>
                              ,
                              <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">
                               41
                              </a>
                              ]. For example, the high accuracy with
                              <strong>
                               tp
                              </strong>
                              suggests the existence of individual variability in their temporal response to the same stimuli. Understanding brainprints will facilitate the understanding of the underlying anatomical and functional variability between individuals.
                             </p>
                            </div>
                            <div id="sec-15" class="subsection">
                             <h3>
                              Cross-task identifiability
                             </h3>
                             <p id="p-45">
                              The consistent performance of
                              <strong>
                               sp
                              </strong>
                              on the HCP data is in line with a previous study on fMRI of overlapping individuals that the spatial connectome is preserved between tasks [
                              <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">
                               15
                              </a>
                              ]. The rank accuracy of
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              on HCP data indicates the potential of these two features to be consistent within individuals (
                              <a id="xref-fig-5-4" class="xref-fig" href="#F5">
                               Fig. 5(c)
                              </a>
                              ) because the majority of individuals still have higher than chance rank accuracy than identification accuracy. The current underperformance of these two features, as expected, is likely due to the different temporal dynamics between the resting and task data. This difference may be eliminated by removing the trial part from the task MEG, focusing on inter-trial intervals or baseline periods, and hence boost the identification accuracy of
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              . More complicated matching method may be proposed to further boost the performance of these two brainprints. The within-task identification accuracy (
                              <a id="xref-fig-4-8" class="xref-fig" href="#F4">
                               Fig. 4(a)
                              </a>
                              ), on the other hand, is still high for all features. With the large pool of participants, the high accuracy confirms the strong individual-specific information contained in the three features within a certain task.
                             </p>
                            </div>
                            <div id="sec-16" class="subsection">
                             <h3>
                              Cross-modality identifiability
                             </h3>
                             <p id="p-46">
                              We observed that both the temporal and frequency features can be used to identify participants across modalities (MEG and EEG) with accuracy much higher than chance. This further supports the interpretation that the temporal and frequency brainprints are capturing idiosyncrasies that are specific to the time-course of how a stimulus is processed (both in amplitude and in frequency), outside possible artifacts induced by MEG and EEG machines.
                             </p>
                            </div>
                            <div id="sec-17" class="subsection">
                             <h3>
                              Interpretability of brainprints
                             </h3>
                             <p id="p-47">
                              For the three brainprints, higher accuracy seems to be associated with the components of features with more stimuli-driven activity: the occipital lobe, the time around the stimulus, and frequency bands the with highest power intensities (
                              <a id="xref-fig-13-1" class="xref-fig" href="#F13">
                               Fig. S4
                              </a>
                              ,
                              <a id="xref-fig-14-1" class="xref-fig" href="#F14">
                               S5
                              </a>
                              ,
                              <a id="xref-fig-15-1" class="xref-fig" href="#F15">
                               S6
                              </a>
                              ). Indeed, MEG signal is most sensitive to transient, coordinated firings of many neurons that happen after stimulus onset. This commonality indicates the possibility that higher accuracy is related to event-related signals, which in turn suggests that identifiability might be caused by different individuals responding differently to the stimulus. This dependence on stimulus may explain the low accuracy with
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              on HCP data and also suggest that the identifiability originates from brain-related activities instead of session- and individual-specific artifacts.
                             </p>
                             <p id="p-48">
                              However, these accuracy patterns of specific components of a feature could also be explained by a signal-to-noise ratio argument: regions, time-points, or frequencies related to stimulus processing correspond to parts of the underlying brain signal with higher amplitudes (while the ambient noise amplitude is constant). It might be that the increase in signal magnitude make the (spatial, temporal or frequency) activity patterns that are specific to a individual more detectable by increasing their amplitude relative to the ambient noise, even if these patterns are not inherently related to stimulus processing and are just consistent features of a individual’s brain activity.
                             </p>
                            </div>
                            <div id="sec-18" class="subsection">
                             <h3>
                              Sample size and level of preprocessing
                             </h3>
                             <p id="p-49">
                              <strong>
                               sp
                              </strong>
                              accuracy tends to saturate with fewer number of trials than the other two features on FST and SEN data but with more trials on HCP data (
                              <a id="xref-fig-8-4" class="xref-fig" href="#F8">
                               Fig. 8(a)
                              </a>
                              ). This difference is likely due to the difference in the maximum accuracy a feature can attain: in HCP data,
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              has much lower maximum accuracy and will reach the peak with smaller number of trials. In FST and SEN data, the spatial pattern may require fewer trials to estimate accurately, as compared to the temporal and frequency features.
                             </p>
                             <p id="p-50">
                              The artifact removal and temporal filtering in the preprocessing pipeline might have prevented session-specific noise from contaminating individual-specific features, resulting in higher accuracy for
                              <strong>
                               tp
                              </strong>
                              and
                              <strong>
                               fq
                              </strong>
                              . The seemingly contradictory accuracy on
                              <strong>
                               sp
                              </strong>
                              does not justify our results: identifiability using
                              <strong>
                               sp
                              </strong>
                              increases after prepossessing when not performing head position correction but decreases when performing it. On the one hand, it is expected that head position correction would improve identifiability by recentering each individual’s data to the same position in each session. On the other hand, head position correction may remove individual-specific information such as the head shape, causing the decrease in the accuracy of
                              <strong>
                               sp
                              </strong>
                              . Future work and analysis of additional datasets are required to investigate this result. The difference in the accuracy between raw and preprocessed data suggests, for example, encrypting the data with session-specific noise may lower identification accuracy.
                             </p>
                            </div>
                            <div id="sec-19" class="subsection">
                             <h3>
                              Limitations
                             </h3>
                             <p id="p-51">
                              Due to the limited availability of multi-session MEG data, more experiments are needed to generalize our findings to a larger population and other tasks. For example, the cross-session identifiability results depend on 4 subjects and may suffer from high variance. A larger population (with multiple sessions per participant) may benefit the interpretation of brainprints and eventually attribute the high identifiability of certain features to the underlying brain mechanism.
                             </p>
                             <p id="p-52">
                              Throughout the paper, we assumed both the target and source datasets had the same pool of participants in the scope of this paper. If we don’t know if one individual from the target set is included in the source set, other classification methods which allow for abstaining from classification (e.g. [
                              <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">
                               42
                              </a>
                              ]) may be used to account for the case when no label in the source set can be assigned to the individual. This situation is an example of a more realistic identification problem because an individual’s participation in multiple MEG studies is usually unknown to the public.
                             </p>
                            </div>
                            <div id="sec-20" class="subsection">
                             <h3>
                              Future solutions
                             </h3>
                             <p id="p-53">
                              More complicated features can be proposed which combine the spatial, temporal, and frequency information to improve identifiability. For example, functional connectivity at different frequency bands has been used to identify twins from other participants [
                              <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">
                               17
                              </a>
                              ]. New feature similarity function that focuses on the structure of the correlation matrices may also be used to improve accuracy [
                              <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">
                               43
                              </a>
                              ]. Metric learning [
                              <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">
                               44
                              </a>
                              ] can also be used to learn the similarity function in a supervised manner which may boost performance with sufficient labeled MEG data.
                             </p>
                             <p id="p-54">
                              On the other hand, given the high identification accuracy with brainprints in this study, privacy-preserving algorithms need to be proposed to account for this privacy issue. Federated learning [
                              <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">
                               45
                              </a>
                              ] may be a promising framework as data collected from multiple sessions and sites can be analyzed together without revealing critical information of each specific dataset.
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-21">
                            <h2 class="">
                             Methods
                            </h2>
                            <div id="sec-22" class="subsection">
                             <h3>
                              Within- vs cross- session
                             </h3>
                             <p id="p-55">
                              We called a pair of source and target sets “within-session” if, for each individual, both datasets were collected in the same visit to the scanner. For example, two blocks of a resting-state recording of a participant collected on the same day are within-session. If the two datasets are collected on different days for each individual, they are “cross-session”. For example, a resting state recording on day 1 and another resting-state recording on day 2 are cross-session. Individuals with within-session data may be easier to identify since the source and target data were collected under almost the same environment.
                             </p>
                            </div>
                            <div id="sec-23" class="subsection">
                             <h3>
                              Within-session data
                             </h3>
                             <p id="p-56">
                              The Harry Potter dataset was recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland) at the Brain Mapping Center at the University of Pittsburgh. Individuals were asked to read a chapter of Harry Potter [
                              <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">
                               23
                              </a>
                              ] while each word was presented for 0.5 s on a screen. There were 5176 trials (words) for each one of the eight individuals. There were 306 sensors at 102 locations where each location has one magnetometer and two planar gradiometers whose signal was averaged. The sampling frequency of the data was 1000 Hz which was further downsampled to 200 Hz. Details about the preprocessing of all the datasets in this paper can be found in Supplement A. The data was parsed into trials where each trial corresponds to the MEG recording when an individual was reading a word. Specifically, the trials of individual k is
                              <span class="inline-formula" id="inline-formula-8">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-8.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-8.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where I
                              <sub>
                               k
                              </sub>
                              is the number of trials for individual k, 102 represents the number of spatial channels, and 100 represents the number of temporal points in the trial. Since the recording of each individual was collected in one session, we simply split the data into a target and source dataset for the within-session identification task.
                             </p>
                            </div>
                            <div id="sec-24" class="subsection">
                             <h3>
                              Cross-session data
                             </h3>
                             <p id="p-57">
                              We considered the following two datasets which have recordings on multiple days:
                             </p>
                             <p id="p-58">
                              2- FST data [
                              <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">
                               28
                              </a>
                              ], shared online:
                              <sup>
                               <a id="xref-fn-2-1" class="xref-fn" href="#fn-2">
                                1
                               </a>
                              </sup>
                              individuals saw faces with each face appearing on the screen. There were 1464 trials for each individual. Each trial lasted 0.5 s.
                             </p>
                             <p id="p-59">
                              There were 4 individuals and 4 sessions. The sampling frequency was 1000 Hz and was downsampled to 200 Hz. Intervals between consecutive sessions were several days.
                             </p>
                             <p id="p-60">
                              2- SEN data (unpublished anonymized citation): individuals read sentences. Each trial lasted 0.5 s. There were 4 individuals and 3 sessions. There were 3575 trials for each individual. The sampling frequency was 1000 Hz and was downsampled to 200 Hz. Intervals between consecutive sessions ranged from days to weeks. In this dataset, two sessions for two individuals were recorded at the same day.
                             </p>
                             <p id="p-61">
                              The 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland) at the Brain Mapping Center at the University of Pittsburgh was used to obtain both recordings. The shape of one trial of the two datasets is 102 channels by 100 time points, the same as the Harry Potter data. We used 300 trials to create features for each run of identification. For the within-session identification (diagonal entries of
                              <a id="xref-fig-3-6" class="xref-fig" href="#F3">
                               Fig. 3(b,c)
                              </a>
                              ), we split the recording for each individual into non-overlapping source and target set before featurization.
                             </p>
                            </div>
                            <div id="sec-25" class="subsection">
                             <h3>
                              Task vs resting data
                             </h3>
                             <p id="p-62">
                              We looked at the Human Connectome Project data
                              <sup>
                               <a id="xref-fn-3-1" class="xref-fn" href="#fn-3">
                                2
                               </a>
                              </sup>
                              [
                              <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">
                               5
                              </a>
                              ]. The recording was obtained using the whole head MAGNES 3600 (4D Neuroimaging, San Diego, CA) system located at the Saint Louis University (SLU) medical campus during a single-day visit for each individual. There were two sessions, one resting-state recording and one working-memory (WM) task recording where the stimuli were images for the participants to remember. The number of trials for each individual varied because different trials were removed for each individual in the preprocessing step due to signal quality. There were 422 trials in average for the resting dataset and 372 trials in average for the WM dataset. Each trial of the WM corresponded to the 2.5 s of the recording after the onset of the stimulus. The two datasets had 77 individuals in common and we only looked at these individuals. There were 146 channels (after bad-channel removal) and the signal was downsampled to 200 Hz. The two sessions were collected on the same day with a break of several hours. We used 200 trials for featurization for each run of identification due to fewer number of total trials as compared to the aforementioned datasets.
                             </p>
                            </div>
                            <div id="sec-26" class="subsection">
                             <h3>
                              MEG vs EEG data
                             </h3>
                             <p id="p-63">
                              We looked at the scene viewing data
                              <sup>
                               <a id="xref-fn-4-1" class="xref-fn" href="#fn-4">
                                3
                               </a>
                              </sup>
                              . The dataset includes both MEG and EEG recordings of 14 participants viewing 362 scene images (trials), and the MEG and EEG sessions were recorded separately on different days. MEG data was recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland) at the Brain Mapping Center at the University of Pittsburgh. EEG data was recorded using a 128-channel whole-head system (ActiveTwo, Biosemi, Amsterdam, Netherlands) at the EEG laboratory of the Psychology Department at Carnegie Mellon University. Each image was shown for 3 − 6 repetitions and the signal was averaged across these repetitions for each image. MEG data was recorded at 1000 Hz and was downsampled to 110 Hz. EEG data was recorded at 512 Hz and was downsampled to 110 Hz. Each trial corresponds to the 1 second of the stimulus presentation. The shape of one trial is 102 channels by 110 time points for MEG (averaged across three sensors for the 102 sensor locations) and 128 channels by 110 for EEG. We used 200 trials to create features for each of the 100 identification runs. For the within-session identification (diagonal entries of
                              <a id="xref-fig-7-5" class="xref-fig" href="#F7">
                               Fig. 7(a)
                              </a>
                              ), we split the recording for each individual into non-overlapping source and target set before featurization.
                             </p>
                            </div>
                            <div id="sec-27" class="subsection">
                             <h3>
                              Random forest identification with raw features
                             </h3>
                             <p id="p-64">
                              We trained a random forest classifier with 256 estimators by first concatenating all the trials of each individual along the time dimension, resulting in
                              <strong>
                               X
                              </strong>
                              <sub>
                               i
                              </sub>
                              ∈ ℝ
                              <sup>
                               102×N
                              </sup>
                              , i = 1, ⋯, 8 where N = 100×5176 is the total number of time points for each individual. We then randomly selected 10000 topomaps from each individual and obtained {
                              <strong>
                               x
                              </strong>
                              <sub>
                               ij
                              </sub>
                              ∈ ℝ
                              <sup>
                               102
                              </sup>
                              , i = 1, ⋯, 8, j = 1, ⋯, 10000} as the training dataset where each sample is a flattened vector with 102 entries corresponding to the signal across all channels at one time point (or, the topomap). The training label is y
                              <sub>
                               ij
                              </sub>
                              = i. A 10-fold cross validation was used to compute the classification accuracy. Data was z-scored by channel separately on training and testing data to avoid data leakage. To further avoid dependency between time points, we repeated the analysis while only sampling one topomap per each trial: for each individual, we randomly sampled 5000 trials without replacement and randomly sampled one topomap from each one of the 5000 trials, leading to 5000 randomly sampled topomaps for each individual.
                             </p>
                            </div>
                            <div id="sec-28" class="subsection">
                             <h3>
                              Interpretable MEG features
                             </h3>
                             <p id="p-65">
                              Let X ∈ ℝ
                              <sup>
                               102×100×n
                              </sup>
                              represent the recording used for featurization, with 102 channels, 100 time points, and n randomly sampled trials. The three features are defined as follows:
                             </p>
                             <ol class="list-ord" id="list-1">
                              <li id="list-item-1">
                               <p id="p-66">
                                Spatial correlation (
                                <strong>
                                 sp
                                </strong>
                                ): Pearson correlation between channels averaged over time. X was reshaped into ℝ
                                <sup>
                                 102×100n
                                </sup>
                                before the correlations between rows of the reshaped matrix were computed.
                               </p>
                              </li>
                              <li id="list-item-2">
                               <p id="p-67">
                                Temporal correlation (
                                <strong>
                                 tp
                                </strong>
                                ): Pearson correlation between time points averaged over channels. X was reshaped into ℝ
                                <sup>
                                 100×102n
                                </sup>
                                before the correlations between rows of the reshaped matrix were computed.
                               </p>
                              </li>
                              <li id="list-item-3">
                               <p id="p-68">
                                Frequency (
                                <strong>
                                 fq
                                </strong>
                                ): power spectrum averaged over channels. Power spectrum of X(i, :, j) was computed using a Tukey window with shape parameter of 0.25 and window size of 100 time points for i = 1, ⋯, 102, j = 1, ⋯, n. The final power spectrum was obtained by averaging across i, j.
                               </p>
                              </li>
                             </ol>
                            </div>
                            <div id="sec-29" class="subsection">
                             <h3>
                              Identification using 1NN
                             </h3>
                             <p id="p-69">
                              We performed R = 100 identification runs. In identification run r, we randomly split the Harry Potter dataset into non-overlapping source and target set, z-scored the source and target by channel separately, and computed the feature
                              <span class="inline-formula" id="inline-formula-9">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              averaged over n = 300 randomly sampled trials using data α ∈ {target, source} for individual i and F ∈ {
                              <strong>
                               sp, tp, fq
                              </strong>
                              }. The features from the target to the source set were matched with a labeling with replacement protocol :
                              <span class="disp-formula" id="disp-formula-1">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/graphic-9.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/graphic-9.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where K = 8 is the total number of individuals and m(·, ·) is the similarity function measuring the similarity between the two features. We used Pearson correlation as our similarity function. The identification accuracy for individual i and feature F is
                              <span class="inline-formula" id="inline-formula-10">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              The averaged identification accuracy for feature F is
                              <span class="inline-formula" id="inline-formula-11">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-11.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-11.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The random baseline is
                              <span class="inline-formula" id="inline-formula-12">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-12.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-12.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                             <p id="p-70">
                              When the source set and target set were from the same session, we split the dataset into non-overlapping sets as we did in the within-session identification. We did not split data when the source and target data are from different sessions since there is no potential data leakage. We z-scored the data by channel on the source and target separately.
                             </p>
                            </div>
                            <div id="sec-30" class="subsection">
                             <h3>
                              Rank accuracy
                             </h3>
                             <p id="p-71">
                              The rank accuracy of individual i on one run of identification (suppressing notations of feature F and run r) is defined as
                              <span class="inline-formula" id="inline-formula-13">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-13.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-13.gif"/>
                                </noscript>
                               </span>
                              </span>
                              where K is the number of individuals, rank
                              <span class="inline-formula" id="inline-formula-14">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-14.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-14.gif"/>
                                </noscript>
                               </span>
                              </span>
                              is over
                              <span class="inline-formula" id="inline-formula-15">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-15.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-15.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . The rank accuracy equals to 1 if the feature of the same individual has the largest similarity between the source and target sets among all K individuals, and is
                              <span class="inline-formula" id="inline-formula-16">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-16.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-16.gif"/>
                                </noscript>
                               </span>
                              </span>
                              if the similarity is the smallest. The rank accuracy captures more information in a failure case where an individual is misidentified. The random baseline for the rank accuracy is
                              <span class="inline-formula" id="inline-formula-17">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-17.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-17.gif"/>
                                </noscript>
                               </span>
                              </span>
                              .
                             </p>
                            </div>
                            <div id="sec-31" class="subsection">
                             <h3>
                              Differential identifiability
                             </h3>
                             <p id="p-72">
                              We also adopted a type of accuracy called differential identifiability [
                              <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">
                               29
                              </a>
                              ] to better understand the robustness of the identification. For one identification run, let C ∈ ℝ
                              <sup>
                               K×K
                              </sup>
                              denote the correlation matrix between the source and target features where K is the number of individuals. The differential identifiability for this identification run was calculated as:
                              <span class="disp-formula" id="disp-formula-2">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/graphic-10.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/graphic-10.gif"/>
                                </noscript>
                               </span>
                              </span>
                              A high differential identifiability would indicate that the features for the same individual (diagonal entries of C) are much more similar as compared to features between different individuals (off-diagonal entries of C). The chance differential identifiability is 0 where there is no difference between the diagonal and off-diagonal entries if C.
                             </p>
                            </div>
                            <div id="sec-32" class="subsection">
                             <h3>
                              Sub-features
                             </h3>
                             <p id="p-73">
                              Each feature was decomposed as follows:
                             </p>
                             <ol class="list-ord" id="list-2">
                              <li id="list-item-4">
                               <p id="p-74">
                                <strong>
                                 sp
                                </strong>
                                : The sensors were partitioned into 8 subgroups according to the map in
                                <a id="xref-fig-1-1" class="xref-fig" href="#F1">
                                 Figure 1
                                </a>
                                of [
                                <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">
                                 46
                                </a>
                                ]: Left Frontal (LF), Right Frontal (RF), Left Temporal (LT), Right Temporal (RT), Left Parietal (LP), Right Parietal (RP), Left Occipital (LO), Right Occipital (LO). Each subfeature was the rows and columns of the spatial correlation matrix corresponding to the sensors in one of the eight groups: letΣ
                                <sub>
                                 s
                                </sub>
                                ∈ ℝ
                                <sup>
                                 102×102
                                </sup>
                                be the spatial correlation matrix, then the subfeature corresponding to the correlation between RT and LT, for example, isΣ
                                <sub>
                                 s
                                </sub>
                                (ind
                                <sub>
                                 RT
                                </sub>
                                , ind
                                <sub>
                                 LT
                                </sub>
                                ) where ind
                                <sub>
                                 RT
                                </sub>
                                is the set of channel indices in the RT group and ind
                                <sub>
                                 LT
                                </sub>
                                corresponds to the LT group.
                               </p>
                              </li>
                              <li id="list-item-5">
                               <p id="p-75">
                                <strong>
                                 tp
                                </strong>
                                : The 100 temporal points were divided into 10 consecutive segments containing 10 time points. Each subfeature was the rows and columns of the temporal correlation matrix corresponding to one of the ten segments: letΣ
                                <sub>
                                 t
                                </sub>
                                ∈ ℝ
                                <sup>
                                 100×100
                                </sup>
                                be the spatial correlation matrix, then subfeature corresponding to the correlation between the first and second time segment, for example, isΣ
                                <sub>
                                 t
                                </sub>
                                (1 : 10, 11 : 20).
                               </p>
                              </li>
                              <li id="list-item-6">
                               <p id="p-76">
                                <strong>
                                 fq
                                </strong>
                                : Each subfeature was the segment of the frequency feature vector corresponding to [f, f + 10] Hz where f ∈ {0, 2,⋯, 90} Hz.
                               </p>
                              </li>
                             </ol>
                            </div>
                            <div id="sec-33" class="subsection">
                             <h3>
                              Raw vs preprocessed data
                             </h3>
                             <p id="p-77">
                              In
                              <a id="xref-fig-8-5" class="xref-fig" href="#F8">
                               Fig. 8(b)
                              </a>
                              , we compared the identification accuracy between the raw and preprocessed data for FST and SEN dataset. The details of the full preprocessing pipeline is included in the supplement. In FST dataset, for a given feature and dataset, there were 48 matching results (12 cross-session comparisons ×4 individuals averaged across 100 identification runs) where each one corresponds to the result of deciding which individual from the source session matches the one individual from the target session. An two-sided unpaired t-test was performed to determine if there is a significant difference in the identification accuracy between the raw and preprocessed data. In SEN dataset, for a given feature and dataset, there were 24 matching results (6 cross-session comparisons ×4 individuals averaged across 100 identification runs) instead. The analysis was performed in a similar way for the rank accuracy in In
                              <a id="xref-fig-8-6" class="xref-fig" href="#F8">
                               Fig. 8(c)
                              </a>
                              .
                             </p>
                            </div>
                           </div>
                           <div class="section" id="sec-34">
                            <h2 class="">
                             Supplementary Material
                            </h2>
                            <div class="section" id="sec-35">
                             <h3>
                              A. Data preprocessing
                             </h3>
                             <p id="p-78">
                              Here we list the preprocessing steps applied to the four types of datasets: Harry Potter (HP), SEN, FST, and Human Connectome Project (HCP). For the EEG and MEG data, please refer to this website for the details of preprocessing:
                              <a href="https://figshare.com/articles/dataset/MEG_EEG_data_viewing_scene_pictures/16766938">
                               https://figshare.com/articles/dataset/MEG_EEG_data_viewing_scene_pictures/16766938
                              </a>
                              . A summary is listed in
                              <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">
                               Table S1
                              </a>
                              . For all datasets, we used an order 8 Chebyshev type I anti-aliasing filter in Python Scipy package [
                              <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">
                               47
                              </a>
                              ] for downsampling. For any within-session identification task, data was z-scored within its corresponding type of dataset (target vs source). Some steps of preprocessing were performed using the python MNE package [
                              <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">
                               37
                              </a>
                              ].
                             </p>
                             <ol class="list-ord" id="list-3">
                              <li id="list-item-7">
                               <p id="p-79">
                                <strong>
                                 HP/SEN
                                </strong>
                                : The 306-channel Elekta Neuromag system was used for the recording. Source-space separation (SSS) along with Maxwell filtering and their temporal extension (tSSS) [
                                <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">
                                 48
                                </a>
                                ,
                                <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">
                                 49
                                </a>
                                ] were used for bad channel correction, head position correction, and electromagnetic artifacts removal. Empty room artifacts were removed. 1 ∼ 150 Hz bandpass filter and 60 & 120 Hz notch filter were used to remove line noise. Heartbeats and eyeblinks artifacts were removed with signal-space projection (SSP) [
                                <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">
                                 50
                                </a>
                                ]. The data was downsampled to 200 Hz and z-scored by channel within each individual and session.
                               </p>
                              </li>
                              <li id="list-item-8">
                               <p id="p-80">
                                <strong>
                                 FST
                                </strong>
                                (preprocessing pipeline was included in the source code): The 306-channel Elekta Neuromag system was used for the recording. Source-space separation (SSS) along with Maxwell filtering and their temporal extension (tSSS) were used for bad channel correction and electromagnetic artifacts removal. Empty room artifacts were removed. We didn’t perform head position correction since there was no head position data. 1 ∼ 150 Hz Bandpass filter and 60 & 120 Hz Notch filter were used to remove line noise. Heartbeats and eyeblinks artifacts were also removed with SSP. The data was downsampled to 200 Hz and z-scored by channel within each individual and session.
                               </p>
                              </li>
                              <li id="list-item-9">
                               <p id="p-81">
                                <strong>
                                 HCP
                                </strong>
                                : Both resting and WM datasets were already preprocessed and downloaded from the HCP database
                                <sup>
                                 <a id="xref-fn-5-1" class="xref-fn" href="#fn-5">
                                  4
                                 </a>
                                </sup>
                                . The details of the preprocessing pipeline can be found at
                                <a href="https://www.humanconnectome.org/storage/app/media/documentation/s1200/HCP_S1200_Release_Reference_Manual.pdf">
                                 https://www.humanconnectome.org/storage/app/media/documentation/s1200/HCP_S1200_Release_Reference_Manual.pdf
                                </a>
                                . MAGNES 3600 (4D Neuroimaging, San Diego, CA) system was used for the recording. For WM data, we looked at the TIM partition which corresponds to −1.5 ∼ 2.5 s relative to the onset of the image. For both resting and WM data, the sampling frequency of the preprocessed data is 508.63 Hz, and 2 s of data were selected from each trial. This corresponds to the whole 1018 time points in the resting data and [763 : 1780]-th time point for the WM data (corresponding to 0 ∼ 2 s relative to the onset of the image). The 2 s data was then downsampled to 101.73 Hz. Data was z-scored by channel within each individual and each data type (resting and WM). We looked at the 146 channels which were marked “good” among all the 77 overlapping individual between resting and WM.
                               </p>
                              </li>
                             </ol>
                             <div id="T1" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2455540/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2455540/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2455540" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S1:
                               </span>
                               <p id="p-82" class="first-child">
                                Summary of the preprocessing stpes for HP, SEN, FST, and HCP data
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                            <div class="section" id="sec-36">
                             <h3>
                              B. Statistical significance of the results
                             </h3>
                             <p id="p-83">
                              The identification and rank accuracy were averaged across subjects, identification runs, and session pairs. The reported accuracies, being so large, are both statistically and practically significant, and are nearly impossible to attribute to random chance, but accurately quantifying the uncertainty is challenging in our setup. Since featurization for each session of each subject was done before the matching, there is some weak dependence on the accuracy between subjects, session, and identification runs. This dependence makes it hard to analytically obtain a p-value for the accuracy. One numerical alternative is to permute the original recording within each session across subjects before performing matching, but this is computationally expensive as it involves loading and computing large chunks of data 1000s of times. Hence we provide below a (natural, but approximate) permutation-based method for a p-value to test the null that the match is a random guess.
                             </p>
                             <p id="p-84">
                              Let
                              <strong>
                               y
                              </strong>
                              <sup>
                               i
                              </sup>
                              denote the true labels of session i. Note that
                              <strong>
                               y
                              </strong>
                              <sup>
                               i
                              </sup>
                              = [1, 2, 3, 4]
                              <sup>
                               T
                              </sup>
                              for any session.
                             </p>
                             <p id="p-85">
                              The permutation test is performed as follows:
                             </p>
                             <div id="statement-1" class="statement">
                              <span class="statement-label">
                               Algorithm 1:
                              </span>
                              <h3>
                               Null distribution for the identification/rank accuracy
                              </h3>
                              <div id="F9" class="fig pos-float type-figure odd">
                               <div class="highwire-figure">
                                <div class="fig-inline-img-wrapper">
                                 <div class="fig-inline-img">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F9.large.jpg?width=800&height=600&carousel=1" title="" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"></div>' data-icon-position="" data-hide-link-title="0">
                                   <span class="hw-responsive-img">
                                    <img class="highwire-fragment fragment-image lazyload" alt="Figure" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F9.medium.gif" width="440" height="120"/>
                                    <noscript>
                                     <img class="highwire-fragment fragment-image" alt="Figure" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F9.medium.gif" width="440" height="120"/>
                                    </noscript>
                                   </span>
                                  </a>
                                 </div>
                                </div>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure9" data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <p id="p-86">
                              </p>
                             </div>
                             <p id="p-87">
                              To calculate the p-value, we simply compute
                              <span class="inline-formula" id="inline-formula-18">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-18.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-18.gif"/>
                                </noscript>
                               </span>
                              </span>
                              , where a is the observed average accuracy of a feature across subjects, sessions, and identification runs.
                              <a id="xref-statement-1-1" class="xref-statement" href="#statement-1">
                               Algorithm 1
                              </a>
                              permutes the labels for each session independently but the permutation remains unchanged for the same source-target pair across identification runs. We summarize the p-values for the identification and rank accuracy of three features on the FST, SEN, and HCP data using T = 4999 permutation runs. For all the p-values, since we have not encountered any a
                              <sub>
                               t
                              </sub>
                              that exceeds the accuracy number, their values are simply
                              <span class="inline-formula" id="inline-formula-19">
                               <span class="highwire-responsive-lazyload">
                                <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-19.gif"/>
                                <noscript>
                                 <img class="highwire-embed" alt="Embedded Image" src="https://www.biorxiv.org/sites/default/files/highwire/biorxiv/early/2022/03/17/2020.06.18.159913/embed/inline-graphic-19.gif"/>
                                </noscript>
                               </span>
                              </span>
                              . We emphasize that even though these p-values are technically only approximate due to some weak dependence, the fact that we did not see a single permutation which achieved a higher accuracy than ours should convince even rigorous skeptics that it is nearly impossible to explain away our accuracies to chance.
                             </p>
                             <div id="T2" class="table pos-float">
                              <div class="table-inline table-callout-links">
                               <div class="callout">
                                <span>
                                 View this table:
                                </span>
                                <ul class="callout-links">
                                 <li class="view-inline first">
                                  <a href="##" class="table-expand-inline" data-table-url="/highwire/markup/2455535/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&table-expand-inline=1" data-icon-position="" data-hide-link-title="0">
                                   View inline
                                  </a>
                                 </li>
                                 <li class="view-popup">
                                  <a href="/highwire/markup/2455535/expansion?width=1000&height=500&iframe=true&postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">
                                   View popup
                                  </a>
                                 </li>
                                 <li class="download-ppt last">
                                  <a href="/highwire/powerpoint/2455535" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">
                                   Download powerpoint
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                              <div class="table-caption">
                               <span class="table-label">
                                Table S2:
                               </span>
                               <p id="p-88" class="first-child">
                                Statistical significance for the accuracy numbers
                               </p>
                               <div class="sb-div caption-clear">
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section" id="sec-37">
                            <h2 class="">
                             C. Supplementary figures
                            </h2>
                            <div id="F10" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F10.large.jpg?width=800&height=600&carousel=1" title="Layout of the sensors for HP,FST, SEN, and MEG/EEG(MEG) data (306-channel Elekta Neuromag system). The channel numbers are consistent with the channel id of sp (if specified). Inset is the partitioning of the sensors same as Fig. 7(a) of the main text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Layout of the sensors for HP,FST, SEN, and MEG/EEG(MEG) data (306-channel Elekta Neuromag system).</span> The channel numbers are consistent with the channel id of <strong>sp</strong> (if specified). Inset is the partitioning of the sensors same as Fig. 7(a) of the main text.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S1:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F10.medium.gif" width="440" height="429"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S1:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F10.medium.gif" width="440" height="429"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S1:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S1:
                              </span>
                              <span class="caption-title">
                               Layout of the sensors for HP,FST, SEN, and MEG/EEG(MEG) data (306-channel Elekta Neuromag system).
                              </span>
                              <p id="p-89" class="first-child">
                               The channel numbers are consistent with the channel id of
                               <strong>
                                sp
                               </strong>
                               (if specified). Inset is the partitioning of the sensors same as
                               <a id="xref-fig-7-6" class="xref-fig" href="#F7">
                                Fig. 7(a)
                               </a>
                               of the main text.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F11" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F11.large.jpg?width=800&height=600&carousel=1" title="Identification accuracy of sp,tp, and fq on the Harry Potter data. (a) Related to Fig. 3. Each dot was averaged across individuals (= 8) and identification runs (= 100). Error bars are the SEs across individuals and identification runs and are invisible due to small values. Each trial is 0.5 s in length. The trends for tp and fq are similar to that of the cross-session data (SEN and FST). sp requires as few as one trial to achieve a perfect accuracy. This indicates strong spatial patterns in the HP data which are specific to each individual. This is expected since HP does not have more than one session, and the identification accuracy for sp may be lower if there are multiple sessions in HP data, similar to what we have observed on FST and SEN data. (b) Identification accuracy per individual for the three features on HP data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Identification accuracy of sp,tp, and fq on the Harry Potter data.</span> <strong>(a)</strong> Related to Fig. 3. Each dot was averaged across individuals (= 8) and identification runs (= 100). Error bars are the SEs across individuals and identification runs and are invisible due to small values. Each trial is 0.5 s in length. The trends for <strong>tp</strong> and <strong>fq</strong> are similar to that of the cross-session data (SEN and FST). <strong>sp</strong> requires as few as one trial to achieve a perfect accuracy. This indicates strong spatial patterns in the HP data which are specific to each individual. This is expected since HP does not have more than one session, and the identification accuracy for <strong>sp</strong> may be lower if there are multiple sessions in HP data, similar to what we have observed on FST and SEN data. <strong>(b)</strong> Identification accuracy per individual for the three features on HP data.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S2:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F11.medium.gif" width="440" height="187"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S2:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F11.medium.gif" width="440" height="187"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S2:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S2:
                              </span>
                              <span class="caption-title">
                               Identification accuracy of sp,tp, and fq on the Harry Potter data.
                              </span>
                              <p id="p-90" class="first-child">
                               <strong>
                                (a)
                               </strong>
                               Related to
                               <a id="xref-fig-3-7" class="xref-fig" href="#F3">
                                Fig. 3
                               </a>
                               . Each dot was averaged across individuals (= 8) and identification runs (= 100). Error bars are the SEs across individuals and identification runs and are invisible due to small values. Each trial is 0.5 s in length. The trends for
                               <strong>
                                tp
                               </strong>
                               and
                               <strong>
                                fq
                               </strong>
                               are similar to that of the cross-session data (SEN and FST).
                               <strong>
                                sp
                               </strong>
                               requires as few as one trial to achieve a perfect accuracy. This indicates strong spatial patterns in the HP data which are specific to each individual. This is expected since HP does not have more than one session, and the identification accuracy for
                               <strong>
                                sp
                               </strong>
                               may be lower if there are multiple sessions in HP data, similar to what we have observed on FST and SEN data.
                               <strong>
                                (b)
                               </strong>
                               Identification accuracy per individual for the three features on HP data.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F12" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-16.large.jpg?width=800&height=600&carousel=1" title="Example topomaps of HP, FST, and SEN data. We plotted the topomap of these three datasets because they were all recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland). Each topomap represents the spatial distribution of the brain activity at t = 100 ms into a trial, averaged across all trials in a session for each subject. The minimum and maximum of the colormap were specific to each topomap and for convenience, we use one colorbar to represent the scale of the color. For the multi-session FST and SEN data, the spatial distribution of the signal of an individual is more similar across sessions (within a column) as compared to that of other individuals (between columns)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Example topomaps of HP, FST, and SEN data.</span> We plotted the topomap of these three datasets because they were all recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland). Each topomap represents the spatial distribution of the brain activity at <em>t</em> = 100 ms into a trial, averaged across all trials in a session for each subject. The minimum and maximum of the colormap were specific to each topomap and for convenience, we use one colorbar to represent the scale of the color. For the multi-session FST and SEN data, the spatial distribution of the signal of an individual is more similar across sessions (within a column) as compared to that of other individuals (between columns).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-16.medium.gif" width="440" height="435"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-16.medium.gif" width="440" height="435"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-17.large.jpg?width=800&height=600&carousel=1" title="Example topomaps of HP, FST, and SEN data. We plotted the topomap of these three datasets because they were all recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland). Each topomap represents the spatial distribution of the brain activity at t = 100 ms into a trial, averaged across all trials in a session for each subject. The minimum and maximum of the colormap were specific to each topomap and for convenience, we use one colorbar to represent the scale of the color. For the multi-session FST and SEN data, the spatial distribution of the signal of an individual is more similar across sessions (within a column) as compared to that of other individuals (between columns)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Example topomaps of HP, FST, and SEN data.</span> We plotted the topomap of these three datasets because they were all recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland). Each topomap represents the spatial distribution of the brain activity at <em>t</em> = 100 ms into a trial, averaged across all trials in a session for each subject. The minimum and maximum of the colormap were specific to each topomap and for convenience, we use one colorbar to represent the scale of the color. For the multi-session FST and SEN data, the spatial distribution of the signal of an individual is more similar across sessions (within a column) as compared to that of other individuals (between columns).</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S3:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-17.medium.gif" width="440" height="429"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S3:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-17.medium.gif" width="440" height="429"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-17.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F12/graphic-17.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S3:
                              </span>
                              <span class="caption-title">
                               Example topomaps of HP, FST, and SEN data.
                              </span>
                              <p id="p-91" class="first-child">
                               We plotted the topomap of these three datasets because they were all recorded using the 306-channel whole-head MEG system (Elekta Neuromag, Helsinki, Finland). Each topomap represents the spatial distribution of the brain activity at t = 100 ms into a trial, averaged across all trials in a session for each subject. The minimum and maximum of the colormap were specific to each topomap and for convenience, we use one colorbar to represent the scale of the color. For the multi-session FST and SEN data, the spatial distribution of the signal of an individual is more similar across sessions (within a column) as compared to that of other individuals (between columns).
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F13" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F13.large.jpg?width=800&height=600&carousel=1" title="Example sp (spatial connectivity) of FST data. Each heatmap represents a 102 × 102 spatial correlation matrix, where each entry represents the correlation between the brain activity on the corresponding channels. For better illustration we clipped the correlation into [0,1]. The general patterns of the correlation matrices are similar to each other. Some subsets of the heatmap, for example, the bottom-right corner, the top-left corner, and the red rectangle areas are more consistent within a individual and different between individuals. This suggests that only the interactions among a subset of sensors are individual-specific. The red rectangle areas, in particular, roughly correspond to the correlations within the left occipital (LO) lobe which yields the highest identification accuracy on both FST and SEN data (see Fig. 7(a) and Fig. S10). More complicated comparison algorithms may be proposed to focus on these specific subsets to improve the identification accuracy." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Example sp (spatial connectivity) of FST data.</span> Each heatmap represents a 102 × 102 spatial correlation matrix, where each entry represents the correlation between the brain activity on the corresponding channels. For better illustration we clipped the correlation into [0,1]. The general patterns of the correlation matrices are similar to each other. Some subsets of the heatmap, for example, the bottom-right corner, the top-left corner, and the red rectangle areas are more consistent within a individual and different between individuals. This suggests that only the interactions among a subset of sensors are individual-specific. The red rectangle areas, in particular, roughly correspond to the correlations within the left occipital (LO) lobe which yields the highest identification accuracy on both FST and SEN data (see Fig. 7(a) and Fig. S10). More complicated comparison algorithms may be proposed to focus on these specific subsets to improve the identification accuracy.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S4:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F13.medium.gif" width="440" height="410"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S4:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F13.medium.gif" width="440" height="410"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S4:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S4:
                              </span>
                              <span class="caption-title">
                               Example sp (spatial connectivity) of FST data.
                              </span>
                              <p id="p-92" class="first-child">
                               Each heatmap represents a 102 × 102 spatial correlation matrix, where each entry represents the correlation between the brain activity on the corresponding channels. For better illustration we clipped the correlation into [0,1]. The general patterns of the correlation matrices are similar to each other. Some subsets of the heatmap, for example, the bottom-right corner, the top-left corner, and the red rectangle areas are more consistent within a individual and different between individuals. This suggests that only the interactions among a subset of sensors are individual-specific. The red rectangle areas, in particular, roughly correspond to the correlations within the left occipital (LO) lobe which yields the highest identification accuracy on both FST and SEN data (see
                               <a id="xref-fig-7-7" class="xref-fig" href="#F7">
                                Fig. 7(a)
                               </a>
                               and
                               <a id="xref-fig-19-4" class="xref-fig" href="#F19">
                                Fig. S10
                               </a>
                               ). More complicated comparison algorithms may be proposed to focus on these specific subsets to improve the identification accuracy.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F14" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F14.large.jpg?width=800&height=600&carousel=1" title="Example tp (temporal connectivity) of FST data. Each heatmap represents a 100 × 100 temporal correlation matrix, where each entry represents the correlation between the brain activity on the corresponding time points. For better illustration we clipped the correlation into [0,1]. The banded structure of the matrices are preserved for the same individual across sessions, and are different between individuals in terms of the number of bands and the relative locations of the bands. The banded structure indicates that there are stronger correlations of the signal with itself at certain lags. In other words, looking at the auto-correlation of the signal or even cross-correlation between different channels may reveal interesting results about the temporal dynamics of the brain activities. The individual-specific band structures also confirm the findings in Fig. 7(b) that correlations of the signal with itself at certain lags are best able to identify individuals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Example tp (temporal connectivity) of FST data.</span> Each heatmap represents a 100 × 100 temporal correlation matrix, where each entry represents the correlation between the brain activity on the corresponding time points. For better illustration we clipped the correlation into [0,1]. The banded structure of the matrices are preserved for the same individual across sessions, and are different between individuals in terms of the number of bands and the relative locations of the bands. The banded structure indicates that there are stronger correlations of the signal with itself at certain lags. In other words, looking at the auto-correlation of the signal or even cross-correlation between different channels may reveal interesting results about the temporal dynamics of the brain activities. The individual-specific band structures also confirm the findings in Fig. 7(b) that correlations of the signal with itself at certain lags are best able to identify individuals.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S5:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F14.medium.gif" width="440" height="410"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S5:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F14.medium.gif" width="440" height="410"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S5:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S5:
                              </span>
                              <span class="caption-title">
                               Example tp (temporal connectivity) of FST data.
                              </span>
                              <p id="p-93" class="first-child">
                               Each heatmap represents a 100 × 100 temporal correlation matrix, where each entry represents the correlation between the brain activity on the corresponding time points. For better illustration we clipped the correlation into [0,1]. The banded structure of the matrices are preserved for the same individual across sessions, and are different between individuals in terms of the number of bands and the relative locations of the bands. The banded structure indicates that there are stronger correlations of the signal with itself at certain lags. In other words, looking at the auto-correlation of the signal or even cross-correlation between different channels may reveal interesting results about the temporal dynamics of the brain activities. The individual-specific band structures also confirm the findings in
                               <a id="xref-fig-7-8" class="xref-fig" href="#F7">
                                Fig. 7(b)
                               </a>
                               that correlations of the signal with itself at certain lags are best able to identify individuals.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F15" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F15.large.jpg?width=800&height=600&carousel=1" title="Example fq (frequency) of FST data. Each plot represents power (averaged across channels) vs. frequencies (Hz), where the range of frequencies is [0, 100] with a 2 Hz increment. For all individuals, there are two peaks in the power spectrum. The two peaks correspond to around 5 and 10 Hz. The relative height of the two peaks as well as the shape of the curve near the two peaks are consistently unique to an individual across sessions and different across individuals. There are also small peaks near 20 Hz for some individuals. These frequencies with higher amplitudes seem to align with the results shown in Fig. 7(c) where the frequency band near 10 Hz yields the highest identification accuracy. Hence the components of fq associated with more stimuli-driven activity or larger signal-to-noise ratio seem to yield better results." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Example fq (frequency) of FST data.</span> Each plot represents power (averaged across channels) vs. frequencies (Hz), where the range of frequencies is [0, 100] with a 2 Hz increment. For all individuals, there are two peaks in the power spectrum. The two peaks correspond to around 5 and 10 Hz. The relative height of the two peaks as well as the shape of the curve near the two peaks are consistently unique to an individual across sessions and different across individuals. There are also small peaks near 20 Hz for some individuals. These frequencies with higher amplitudes seem to align with the results shown in Fig. 7(c) where the frequency band near 10 Hz yields the highest identification accuracy. Hence the components of <strong>fq</strong> associated with more stimuli-driven activity or larger signal-to-noise ratio seem to yield better results.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S6:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F15.medium.gif" width="439" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S6:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F15.medium.gif" width="439" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F15.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S6:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F15.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S6:
                              </span>
                              <span class="caption-title">
                               Example fq (frequency) of FST data.
                              </span>
                              <p id="p-94" class="first-child">
                               Each plot represents power (averaged across channels) vs. frequencies (Hz), where the range of frequencies is [0, 100] with a 2 Hz increment. For all individuals, there are two peaks in the power spectrum. The two peaks correspond to around 5 and 10 Hz. The relative height of the two peaks as well as the shape of the curve near the two peaks are consistently unique to an individual across sessions and different across individuals. There are also small peaks near 20 Hz for some individuals. These frequencies with higher amplitudes seem to align with the results shown in
                               <a id="xref-fig-7-9" class="xref-fig" href="#F7">
                                Fig. 7(c)
                               </a>
                               where the frequency band near 10 Hz yields the highest identification accuracy. Hence the components of
                               <strong>
                                fq
                               </strong>
                               associated with more stimuli-driven activity or larger signal-to-noise ratio seem to yield better results.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F16" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F16.large.jpg?width=800&height=600&carousel=1" title="Confusion matrices for HP, FST, SEN, and MEG/EEG data. For the HP, FST, SEN, and MEG/EEG datasets, we constructed the confusion matrices from the 100 identification runs of 2 cross-sessions (HP), 12 cross-sessions (FST), 6 cross-sessions (SEN), and 2 cross-modality (MEG/EEG). The numbers were normalized by the sum of each row and represent the percentage of true labels predicted as the corresponding label on the horizontal axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Confusion matrices for HP, FST, SEN, and MEG/EEG data.</span> For the HP, FST, SEN, and MEG/EEG datasets, we constructed the confusion matrices from the 100 identification runs of 2 cross-sessions (HP), 12 cross-sessions (FST), 6 cross-sessions (SEN), and 2 cross-modality (MEG/EEG). The numbers were normalized by the sum of each row and represent the percentage of true labels predicted as the corresponding label on the horizontal axis.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S7:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F16.medium.gif" width="329" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S7:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F16.medium.gif" width="329" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F16.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S7:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F16.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S7:
                              </span>
                              <span class="caption-title">
                               Confusion matrices for HP, FST, SEN, and MEG/EEG data.
                              </span>
                              <p id="p-95" class="first-child">
                               For the HP, FST, SEN, and MEG/EEG datasets, we constructed the confusion matrices from the 100 identification runs of 2 cross-sessions (HP), 12 cross-sessions (FST), 6 cross-sessions (SEN), and 2 cross-modality (MEG/EEG). The numbers were normalized by the sum of each row and represent the percentage of true labels predicted as the corresponding label on the horizontal axis.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F17" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-22.large.jpg?width=800&height=600&carousel=1" title="Confusion matrices for HCP data. For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Confusion matrices for HCP data.</span> For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-22.medium.gif" width="440" height="379"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-22.medium.gif" width="440" height="379"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-22.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-22.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-23.large.jpg?width=800&height=600&carousel=1" title="Confusion matrices for HCP data. For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Confusion matrices for HCP data.</span> For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-23.medium.gif" width="440" height="382"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-23.medium.gif" width="440" height="382"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-23.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-23.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-24.large.jpg?width=800&height=600&carousel=1" title="Confusion matrices for HCP data. For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Confusion matrices for HCP data.</span> For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S8:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-24.medium.gif" width="440" height="382"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S8:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-24.medium.gif" width="440" height="382"/>
                                  </noscript>
                                 </span>
                                </a>
                                <ul class="highwire-figure-links inline">
                                 <li class="download-fig first">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-24.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">
                                   Download figure
                                  </a>
                                 </li>
                                 <li class="new-tab last">
                                  <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F17/graphic-24.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                   Open in new tab
                                  </a>
                                 </li>
                                </ul>
                               </div>
                              </div>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S8:
                              </span>
                              <span class="caption-title">
                               Confusion matrices for HCP data.
                              </span>
                              <p id="p-96" class="first-child">
                               For the HCP datasets, we constructed the confusion matrices for the three features from the 100 identification runs of 2 cross-sessions. The numbers were normalized by the sum of each row and represent the percentage of true label predicted as the corresponding label on the horizontal axis.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F18" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F18.large.jpg?width=800&height=600&carousel=1" title="Differential identifiability of the four datasets. We computed the differential identifiability of the FST, SEN, HCP, and MEG-EEG datasets. The random baseline is 0 (red dashed line). The error bar represents the SE across cross-sessions and identification runs and is invisible on some datasets. The differential identifiability for all the features considered is much larger than the random baseline on all the datasets. For FST and SEN, even though sp has lower identification accuracy than the other two features, their differential identifiability is still higher. This phenomenon is flipped for the HCP and MEG-EEG datasets. This might be due to the fact that many components of tp and fq are similar for the same task (i.e. in FST and SEN), leading to smaller deferential identifiability than sp." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Differential identifiability of the four datasets.</span> We computed the differential identifiability of the FST, SEN, HCP, and MEG-EEG datasets. The random baseline is 0 (red dashed line). The error bar represents the SE across cross-sessions and identification runs and is invisible on some datasets. The differential identifiability for all the features considered is much larger than the random baseline on all the datasets. For FST and SEN, even though <strong>sp</strong> has lower identification accuracy than the other two features, their differential identifiability is still higher. This phenomenon is flipped for the HCP and MEG-EEG datasets. This might be due to the fact that many components of <strong>tp</strong> and <strong>fq</strong> are similar for the same task (i.e. in FST and SEN), leading to smaller deferential identifiability than <strong>sp</strong>.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S9:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F18.medium.gif" width="440" height="95"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S9:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F18.medium.gif" width="440" height="95"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F18.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S9:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F18.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S9:
                              </span>
                              <span class="caption-title">
                               Differential identifiability of the four datasets.
                              </span>
                              <p id="p-97" class="first-child">
                               We computed the differential identifiability of the FST, SEN, HCP, and MEG-EEG datasets. The random baseline is 0 (red dashed line). The error bar represents the SE across cross-sessions and identification runs and is invisible on some datasets. The differential identifiability for all the features considered is much larger than the random baseline on all the datasets. For FST and SEN, even though
                               <strong>
                                sp
                               </strong>
                               has lower identification accuracy than the other two features, their differential identifiability is still higher. This phenomenon is flipped for the HCP and MEG-EEG datasets. This might be due to the fact that many components of
                               <strong>
                                tp
                               </strong>
                               and
                               <strong>
                                fq
                               </strong>
                               are similar for the same task (i.e. in FST and SEN), leading to smaller deferential identifiability than
                               <strong>
                                sp
                               </strong>
                               .
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F19" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F19.large.jpg?width=800&height=600&carousel=1" title="Identification accuracy with sub features for a: sp and b: tp in FST data. Related to Fig. 7. For both FST and SEN, the within-LO and LO-RP correlations yield high identification accuracy. Similarly, for both FST and SEN, the super-diagonal and the correlations between the fourth and fifth 0.05 s yield high accuracy. The consistency of the results on the two datasets suggest that our conclusions regarding the importance of sub-features are not due to experiment-specific artifacts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Identification accuracy with sub features for a: sp and b: tp in FST data.</span> Related to Fig. 7. For both FST and SEN, the within-LO and LO-RP correlations yield high identification accuracy. Similarly, for both FST and SEN, the super-diagonal and the correlations between the fourth and fifth 0.05 s yield high accuracy. The consistency of the results on the two datasets suggest that our conclusions regarding the importance of sub-features are not due to experiment-specific artifacts.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S10:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F19.medium.gif" width="440" height="186"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S10:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F19.medium.gif" width="440" height="186"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F19.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S10:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F19.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S10:
                              </span>
                              <span class="caption-title">
                               Identification accuracy with sub features for a: sp and b: tp in FST data.
                              </span>
                              <p id="p-98" class="first-child">
                               Related to
                               <a id="xref-fig-7-10" class="xref-fig" href="#F7">
                                Fig. 7
                               </a>
                               . For both FST and SEN, the within-LO and LO-RP correlations yield high identification accuracy. Similarly, for both FST and SEN, the super-diagonal and the correlations between the fourth and fifth 0.05 s yield high accuracy. The consistency of the results on the two datasets suggest that our conclusions regarding the importance of sub-features are not due to experiment-specific artifacts.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F20" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F20.large.jpg?width=800&height=600&carousel=1" title="Identification accuracy with respect to frequency bands. Related to Fig. 7. We plotted the identification accuracy with respect to different frequency bands to understand their significance on identifiability. Due to the limitation of sampling frequency, we grouped the frequencies according to this modified range: Delta (0 ∼4 Hz), Theta (4 ∼8 Hz), Alpha (8 ∼14 Hz), Beta (14 ∼32 Hz), and Gamma (32 ∼100 Hz). The different peaks for SEN and FST datasets might be due to the borderline difference between Alpha and Beta bands." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><span xmlns="http://www.w3.org/1999/xhtml" class="caption-title">Identification accuracy with respect to frequency bands.</span> Related to Fig. 7. We plotted the identification accuracy with respect to different frequency bands to understand their significance on identifiability. Due to the limitation of sampling frequency, we grouped the frequencies according to this modified range: Delta (0 ∼4 Hz), Theta (4 ∼8 Hz), Alpha (8 ∼14 Hz), Beta (14 ∼32 Hz), and Gamma (32 ∼100 Hz). The different peaks for SEN and FST datasets might be due to the borderline difference between Alpha and Beta bands.</div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S11:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F20.medium.gif" width="440" height="137"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S11:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F20.medium.gif" width="440" height="137"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F20.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S11:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F20.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S11:
                              </span>
                              <span class="caption-title">
                               Identification accuracy with respect to frequency bands.
                              </span>
                              <p id="p-99" class="first-child">
                               Related to
                               <a id="xref-fig-7-11" class="xref-fig" href="#F7">
                                Fig. 7
                               </a>
                               . We plotted the identification accuracy with respect to different frequency bands to understand their significance on identifiability. Due to the limitation of sampling frequency, we grouped the frequencies according to this modified range: Delta (0 ∼4 Hz), Theta (4 ∼8 Hz), Alpha (8 ∼14 Hz), Beta (14 ∼32 Hz), and Gamma (32 ∼100 Hz). The different peaks for SEN and FST datasets might be due to the borderline difference between Alpha and Beta bands.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                            <div id="F21" class="fig pos-float type-figure odd">
                             <div class="highwire-figure">
                              <div class="fig-inline-img-wrapper">
                               <div class="fig-inline-img">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F21.large.jpg?width=800&height=600&carousel=1" title="Effect of head size and average movement on identifiability for the HCP data. For the HCP dataset, we looked at how identifiability may change with respect to the size of the individual’s skull and average movement, which were both recorded for each experiment session. We plotted the identification accuracy and rank accuracy with respect to these metrics and computed the statistical significance at the level of individuals (Pearson’s r). It can be seen that the average movement may have a positive impact on identification using sp possibly due to the fact that more movement may induce more individual-specific information. Bigger head size may lead to higher accuracy using tp possibly due to the more distant brain areas recorded and the more accurate average response to the stimuli. The fq is not affected by either metric because it is a purely spectral feature and may not be easily influenced by these spatial-related metrics. However, we do ask for caution in interpreting these results because these correlations are not statistically significant after multiple comparison corrections." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-315543812" data-figure-caption='<div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml"><span class="caption-title">Effect of head size and average movement on identifiability for the HCP data.</span> For the HCP dataset, we looked at how identifiability may change with respect to the size of the individual’s skull and average movement, which were both recorded for each experiment session. We plotted the identification accuracy and rank accuracy with respect to these metrics and computed the statistical significance at the level of individuals (Pearson’s <em>r</em>). It can be seen that the average movement may have a positive impact on identification using <strong>sp</strong> possibly due to the fact that more movement may induce more individual-specific information. Bigger head size may lead to higher accuracy using <strong>tp</strong> possibly due to the more distant brain areas recorded and the more accurate average response to the stimuli. The <strong>fq</strong> is not affected by either metric because it is a purely spectral feature and may not be easily influenced by these spatial-related metrics. However, we do ask for caution in interpreting these results because these correlations are not statistically significant after multiple comparison corrections.</div></div>' data-icon-position="" data-hide-link-title="0">
                                 <span class="hw-responsive-img">
                                  <img class="highwire-fragment fragment-image lazyload" alt="Figure S12:" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F21.medium.gif" width="309" height="440"/>
                                  <noscript>
                                   <img class="highwire-fragment fragment-image" alt="Figure S12:" src="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F21.medium.gif" width="309" height="440"/>
                                  </noscript>
                                 </span>
                                </a>
                               </div>
                              </div>
                              <ul class="highwire-figure-links inline">
                               <li class="download-fig first">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F21.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure S12:" data-icon-position="" data-hide-link-title="0">
                                 Download figure
                                </a>
                               </li>
                               <li class="new-tab last">
                                <a href="https://www.biorxiv.org/content/biorxiv/early/2022/03/17/2020.06.18.159913/F21.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">
                                 Open in new tab
                                </a>
                               </li>
                              </ul>
                             </div>
                             <div class="fig-caption">
                              <span class="fig-label">
                               Figure S12:
                              </span>
                              <span class="caption-title">
                               Effect of head size and average movement on identifiability for the HCP data.
                              </span>
                              <p id="p-100" class="first-child">
                               For the HCP dataset, we looked at how identifiability may change with respect to the size of the individual’s skull and average movement, which were both recorded for each experiment session. We plotted the identification accuracy and rank accuracy with respect to these metrics and computed the statistical significance at the level of individuals (Pearson’s r). It can be seen that the average movement may have a positive impact on identification using
                               <strong>
                                sp
                               </strong>
                               possibly due to the fact that more movement may induce more individual-specific information. Bigger head size may lead to higher accuracy using
                               <strong>
                                tp
                               </strong>
                               possibly due to the more distant brain areas recorded and the more accurate average response to the stimuli. The
                               <strong>
                                fq
                               </strong>
                               is not affected by either metric because it is a purely spectral feature and may not be easily influenced by these spatial-related metrics. However, we do ask for caution in interpreting these results because these correlations are not statistically significant after multiple comparison corrections.
                              </p>
                              <div class="sb-div caption-clear">
                              </div>
                             </div>
                            </div>
                           </div>
                           <div class="section fn-group" id="fn-group-1">
                            <h2>
                             Footnotes
                            </h2>
                            <ul>
                             <li class="fn-update fn-group-summary-of-updates" id="fn-1">
                              <p id="p-5">
                               Figures, text, results.
                              </p>
                             </li>
                             <li class="fn" id="fn-2">
                              <p id="p-101">
                               <a class="rev-xref" href="#xref-fn-2-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 1
                                </sup>
                               </span>
                               <a href="https://figshare.com/articles/FST_raw_data/4233107">
                                https://figshare.com/articles/FST_raw_data/4233107
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-3">
                              <p id="p-102">
                               <a class="rev-xref" href="#xref-fn-3-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 2
                                </sup>
                               </span>
                               <a href="https://www.humanconnectome.org/study/hcp-young-adult">
                                https://www.humanconnectome.org/study/hcp-young-adult
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-4">
                              <p id="p-103">
                               <a class="rev-xref" href="#xref-fn-4-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 3
                                </sup>
                               </span>
                               <a href="https://figshare.com/articles/dataset/MEG_EEG_data_viewing_scene_pictures/16766938">
                                https://figshare.com/articles/dataset/MEG_EEG_data_viewing_scene_pictures/16766938
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-5">
                              <p id="p-104">
                               <a class="rev-xref" href="#xref-fn-5-1">
                                ↵
                               </a>
                               <span class="fn-label">
                                <sup>
                                 4
                                </sup>
                               </span>
                               <a href="https://www.humanconnectome.org/study/hcp-young-adult">
                                https://www.humanconnectome.org/study/hcp-young-adult
                               </a>
                              </p>
                             </li>
                             <li class="fn" id="fn-6">
                              <p id="p-105">
                               <span class="fn-label">
                                <sup>
                                 5
                                </sup>
                               </span>
                               No continuous recording of head position was available in HCP data
                              </p>
                             </li>
                             <li class="fn" id="fn-7">
                              <p id="p-106">
                               <span class="fn-label">
                                <sup>
                                 6
                                </sup>
                               </span>
                               page 68 of
                               <a href="https://www.humanconnectome.org/storage/app/media/documentation/s1200/HCP_S1200_Release_Reference_Manual.pdf">
                                https://www.humanconnectome.org/storage/app/media/documentation/s1200/HCP_S1200_Release_Reference_Manual.pdf
                               </a>
                              </p>
                             </li>
                            </ul>
                           </div>
                           <div class="section ref-list" id="ref-list-1">
                            <h2 class="">
                             References
                            </h2>
                            <ol class="cit-list ref-use-labels">
                             <li>
                              <span class="ref-label">
                               [1].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference [1] in text" id="ref-1">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.1" data-doi="10.1007/s11693-013-9116-4">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas
                                  </span>
                                  <span class="cit-name-surname">
                                   Landrain
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Morgan
                                  </span>
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ariel Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Perez
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Remi
                                  </span>
                                  <span class="cit-name-surname">
                                   Sussan
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Do-it-yourself biology: challenges and promises for an open science and technology movement
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Systems and synthetic biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  115
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  126
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DSystems%2Band%2Bsynthetic%2Bbiology%26rft.volume%253D7%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11693-013-9116-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s11693-013-9116-4&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [2].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference [2] in text" id="ref-2">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.2">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Krzysztof
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorgolewski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Oscar
                                  </span>
                                  <span class="cit-name-surname">
                                   Esteban
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Gunnar
                                  </span>
                                  <span class="cit-name-surname">
                                   Schaefer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Wandell
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Russell
                                  </span>
                                  <span class="cit-name-surname">
                                   Poldrack
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Openneuro—a free online platform for sharing and analysis of neuroimaging data. Organization for human brain mapping
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Vancouver, Canada
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  1677
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [3].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference [3] in text" id="ref-3">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Latanya
                                  </span>
                                  <span class="cit-name-surname">
                                   Sweeney
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Simple demographics often identify people uniquely
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Health (San Francisco)
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  671
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  34
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2000
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHealth%2B%2528San%2BFrancisco%2529%26rft.volume%253D671%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [4].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference [4] in text" id="ref-4">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.4" data-doi="10.2105/AJPH.36.12.1412">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Halbert L
                                  </span>
                                  <span class="cit-name-surname">
                                   Dunn
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Record linkage
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  American Journal of Public Health and the Nations Health
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  36
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1412
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1416
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1946
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bpublic%2Bhealth%2Band%2Bthe%2Bnation%2527s%2Bhealth%26rft.stitle%253DAm%2BJ%2BPublic%2BHealth%2BNations%2BHealth%26rft.aulast%253DDunn%26rft.auinit1%253DH.%2BL.%26rft.volume%253D36%26rft.issue%253D12%26rft.spage%253D1412%26rft.epage%253D1416%26rft.atitle%253DRecord%2BLinkage.%26rft_id%253Dinfo%253Adoi%252F10.2105%252FAJPH.36.12.1412%26rft_id%253Dinfo%253Apmid%252F18016455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.2105/AJPH.36.12.1412&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=18016455&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [5].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference [5] in text" id="ref-5">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.5" data-doi="10.1016/j.neuroimage.2013.05.041">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David C
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Essen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephen M
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Deanna M
                                  </span>
                                  <span class="cit-name-surname">
                                   Barch
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Timothy EJ
                                  </span>
                                  <span class="cit-name-surname">
                                   Behrens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Essa
                                  </span>
                                  <span class="cit-name-surname">
                                   Yacoub
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kamil
                                  </span>
                                  <span class="cit-name-surname">
                                   Ugurbil
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth cit-collab">
                                  Wu-Minn HCP Consortium
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  The wu-minn human connectome project: an overview
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  80
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  62
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  79
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D80%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2013.05.041%26rft_id%253Dinfo%253Apmid%252F23684880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2013.05.041&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23684880&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000322416000009&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [6].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference [6] in text" id="ref-6">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.6">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jaewoo
                                  </span>
                                  <span class="cit-name-surname">
                                   Lee
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chris
                                  </span>
                                  <span class="cit-name-surname">
                                   Clifton
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Differential identifiability
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Proceedings of the 18th ACM SIGKDD international conference on Knowledge discovery and data mining
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  1041
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1049
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [7].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference [7] in text" id="ref-7">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.7">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Craig
                                  </span>
                                  <span class="cit-name-surname">
                                   Gentry
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Fully homomorphic encryption using ideal lattices
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In Proceedings of the forty-first annual ACM symposium on Theory of computing
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  169
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  178
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2009
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [8].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference [8] in text" id="ref-8">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.8" data-doi=":10.1016/j.neuroimage.2016.05.070">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Giles L
                                  </span>
                                  <span class="cit-name-surname">
                                   Colclough
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark W
                                  </span>
                                  <span class="cit-name-surname">
                                   Woolrich
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   PK
                                  </span>
                                  <span class="cit-name-surname">
                                   Tewarie
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew J
                                  </span>
                                  <span class="cit-name-surname">
                                   Brookes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew J
                                  </span>
                                  <span class="cit-name-surname">
                                   Quinn
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Stephen M
                                  </span>
                                  <span class="cit-name-surname">
                                   Smith
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  How reliable are meg resting-state connectivity metrics?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  138
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  284
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  293
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D138%26rft.spage%253D284%26rft_id%253Dinfo%253Adoi%252F%253A10.1016%252Fj.neuroimage.2016.05.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=:10.1016/j.neuroimage.2016.05.070&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [9].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference [9] in text" id="ref-9">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.9" data-doi="10.1038/nrn3475">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Katherine S
                                  </span>
                                  <span class="cit-name-surname">
                                   Button
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   John PA
                                  </span>
                                  <span class="cit-name-surname">
                                   Ioannidis
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Claire
                                  </span>
                                  <span class="cit-name-surname">
                                   Mokrysz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian A
                                  </span>
                                  <span class="cit-name-surname">
                                   Nosek
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Flint
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Emma SJ
                                  </span>
                                  <span class="cit-name-surname">
                                   Robinson
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marcus R
                                  </span>
                                  <span class="cit-name-surname">
                                   Munafò
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Power failure: why small sample size undermines the reliability of neuroscience
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature Reviews Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  14
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  365
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  376
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2BReviews%2BNeuroscience%26rft.volume%253D14%26rft.spage%253D365%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn3475%26rft_id%253Dinfo%253Apmid%252F23571845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrn3475&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=23571845&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [10].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference [10] in text" id="ref-10">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.10" data-doi="10.1038/nrn.2016.167">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Russell A
                                  </span>
                                  <span class="cit-name-surname">
                                   Poldrack
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chris I
                                  </span>
                                  <span class="cit-name-surname">
                                   Baker
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joke
                                  </span>
                                  <span class="cit-name-surname">
                                   Durnez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Krzysztof J
                                  </span>
                                  <span class="cit-name-surname">
                                   Gorgolewski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul M
                                  </span>
                                  <span class="cit-name-surname">
                                   Matthews
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marcus R
                                  </span>
                                  <span class="cit-name-surname">
                                   Munafò
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thomas E
                                  </span>
                                  <span class="cit-name-surname">
                                   Nichols
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jean-Baptiste
                                  </span>
                                  <span class="cit-name-surname">
                                   Poline
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edward
                                  </span>
                                  <span class="cit-name-surname">
                                   Vul
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tal
                                  </span>
                                  <span class="cit-name-surname">
                                   Yarkoni
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Scanning the horizon: towards transparent and reproducible neuroimaging research
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature reviews neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  115
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Breviews%2Bneuroscience%26rft.volume%253D18%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrn.2016.167%26rft_id%253Dinfo%253Apmid%252F28053326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nrn.2016.167&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=28053326&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [11].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference [11] in text" id="ref-11">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.11" data-doi="10.1007/s10548-014-0364-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Vincent
                                  </span>
                                  <span class="cit-name-surname">
                                   Wens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mathieu
                                  </span>
                                  <span class="cit-name-surname">
                                   Bourguignon
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Serge
                                  </span>
                                  <span class="cit-name-surname">
                                   Goldman
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brice
                                  </span>
                                  <span class="cit-name-surname">
                                   Marty
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marc
                                  </span>
                                  <span class="cit-name-surname">
                                   Op De Beeck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Catherine
                                  </span>
                                  <span class="cit-name-surname">
                                   Clumeck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alison
                                  </span>
                                  <span class="cit-name-surname">
                                   Mary
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philippe
                                  </span>
                                  <span class="cit-name-surname">
                                   Peigneux
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Patrick
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Bogaert
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew J
                                  </span>
                                  <span class="cit-name-surname">
                                   Brookes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Inter-and intra-subject variability of neuromagnetic resting state networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brain topography
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  27
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  5
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  620
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  634
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBrain%2Btopography%26rft.volume%253D27%26rft.spage%253D620%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10548-014-0364-8%26rft_id%253Dinfo%253Apmid%252F24777562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1007/s10548-014-0364-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24777562&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [12].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference [12] in text" id="ref-12">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.12">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Poulos
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Rangoussi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   N
                                  </span>
                                  <span class="cit-name-surname">
                                   Alexandris
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                  <span class="cit-name-surname">
                                   Evangelou
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Person identification from the eeg using nonlinear signal classification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Methods of information in Medicine
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  41
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  01
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  64
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  75
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMethods%2Bof%2Binformation%2Bin%2Bmedicine%26rft.stitle%253DMethods%2BInf%2BMed%26rft.aulast%253DPoulos%26rft.auinit1%253DM.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D64%26rft.epage%253D75%26rft.atitle%253DPerson%2Bidentification%2Bfrom%2Bthe%2BEEG%2Busing%2Bnonlinear%2Bsignal%2Bclassification.%26rft_id%253Dinfo%253Apmid%252F11933767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=11933767&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000174503800013&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [13].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference [13] in text" id="ref-13">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.18.159913v3.13">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lan
                                  </span>
                                  <span class="cit-name-surname">
                                   Ma
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James W
                                  </span>
                                  <span class="cit-name-surname">
                                   Minett
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Thierry
                                  </span>
                                  <span class="cit-name-surname">
                                   Blu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William SY
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Resting state eeg-based biometrics for individual identification using convolutional neural networks
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In 2015 37th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC)
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  2848
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2851
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  IEEE
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [14].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference [14] in text" id="ref-14">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.14">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xiang
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lina
                                  </span>
                                  <span class="cit-name-surname">
                                   Yao
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Salil S
                                  </span>
                                  <span class="cit-name-surname">
                                   Kanhere
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yunhao
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tao
                                  </span>
                                  <span class="cit-name-surname">
                                   Gu
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kaixuan
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mindid: Person identification from brain waves through attention-based recurrent neural network
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technologies
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BACM%2Bon%2BInteractive%252C%2BMobile%252C%2BWearable%2Band%2BUbiquitous%2BTechnologies%26rft.volume%253D2%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [15].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference [15] in text" id="ref-15">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.15" data-doi="10.1038/nn.4135">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Emily S
                                  </span>
                                  <span class="cit-name-surname">
                                   Finn
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xilin
                                  </span>
                                  <span class="cit-name-surname">
                                   Shen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Dustin
                                  </span>
                                  <span class="cit-name-surname">
                                   Scheinost
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Monica D
                                  </span>
                                  <span class="cit-name-surname">
                                   Rosenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jessica
                                  </span>
                                  <span class="cit-name-surname">
                                   Huang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marvin M
                                  </span>
                                  <span class="cit-name-surname">
                                   Chun
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Xenophon
                                  </span>
                                  <span class="cit-name-surname">
                                   Papademetris
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   R
                                  </span>
                                  <span class="cit-name-surname">
                                   Todd Constable
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  18
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1664
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bneuroscience%26rft.volume%253D18%26rft.spage%253D1664%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnn.4135%26rft_id%253Dinfo%253Apmid%252F26457551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/nn.4135&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=26457551&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [16].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference [16] in text" id="ref-16">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.16">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Oscar
                                  </span>
                                  <span class="cit-name-surname">
                                   Miranda-Dominguez
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian D
                                  </span>
                                  <span class="cit-name-surname">
                                   Mills
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Samuel D
                                  </span>
                                  <span class="cit-name-surname">
                                   Carpenter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kathleen A
                                  </span>
                                  <span class="cit-name-surname">
                                   Grant
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christopher D
                                  </span>
                                  <span class="cit-name-surname">
                                   Kroenke
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joel T
                                  </span>
                                  <span class="cit-name-surname">
                                   Nigg
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Damien A
                                  </span>
                                  <span class="cit-name-surname">
                                   Fair
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Connectotyping: model based fingerprinting of the functional connectome
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  9
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  11
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [17].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference [17] in text" id="ref-17">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.17" data-doi="10.1038/s41598-017-07117-8">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Demuru
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   AA
                                  </span>
                                  <span class="cit-name-surname">
                                   Gouw
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                  <span class="cit-name-surname">
                                   Hillebrand
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   CJ
                                  </span>
                                  <span class="cit-name-surname">
                                   Stam
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   BW
                                  </span>
                                  <span class="cit-name-surname">
                                   Van Dijk
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   P
                                  </span>
                                  <span class="cit-name-surname">
                                   Scheltens
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   BM
                                  </span>
                                  <span class="cit-name-surname">
                                   Tijms
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   E
                                  </span>
                                  <span class="cit-name-surname">
                                   Konijnenberg
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   M
                                  </span>
                                  <span class="cit-name-surname">
                                   Ten Kate
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   A
                                  </span>
                                  <span class="cit-name-surname">
                                   Den Braber
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Functional and effective whole brain connectivity using magnetoencephalography to identify monozygotic twin pairs
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  11
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-07117-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1038/s41598-017-07117-8&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [18].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference [18] in text" id="ref-18">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.18">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andrew James
                                  </span>
                                  <span class="cit-name-surname">
                                   Anderson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kelsey
                                  </span>
                                  <span class="cit-name-surname">
                                   McDermott
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Rooks
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kathi L
                                  </span>
                                  <span class="cit-name-surname">
                                   Heffner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Dodell-Feder
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Feng V
                                  </span>
                                  <span class="cit-name-surname">
                                   Lin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Decoding individual identity from brain activity elicited in imagining common experiences
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature communications
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  11
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bcommunications%26rft.volume%253D11%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [19].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference [19] in text" id="ref-19">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.19" data-doi="10.1016/j.neuroimage.2015.01.032">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Wachinger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Polina
                                  </span>
                                  <span class="cit-name-surname">
                                   Golland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   William
                                  </span>
                                  <span class="cit-name-surname">
                                   Kremen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bruce
                                  </span>
                                  <span class="cit-name-surname">
                                   Fischl
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Reuter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth cit-collab">
                                  Alzheimer’s Disease Neuroimaging Initiative
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Brainprint: A discriminative characterization of brain morphology
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NeuroImage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  109
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  232
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  248
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroImage%26rft.volume%253D109%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2015.01.032%26rft_id%253Dinfo%253Apmid%252F25613439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2015.01.032&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=25613439&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [20].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference [20] in text" id="ref-20">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.20">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Blair C
                                  </span>
                                  <span class="cit-name-surname">
                                   Armstrong
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Maria V
                                  </span>
                                  <span class="cit-name-surname">
                                   Ruiz-Blondet
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Negin
                                  </span>
                                  <span class="cit-name-surname">
                                   Khalifian
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kenneth J
                                  </span>
                                  <span class="cit-name-surname">
                                   Kurtz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Zhanpeng
                                  </span>
                                  <span class="cit-name-surname">
                                   Jin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sarah
                                  </span>
                                  <span class="cit-name-surname">
                                   Laszlo
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Brainprint: Assessing the uniqueness, collectability, and permanence of a novel method for erp biometrics
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neurocomputing
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  166
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  59
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  67
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeurocomputing%26rft.volume%253D166%26rft.spage%253D59%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [21].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference [21] in text" id="ref-21">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.18.159913v3.21">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Wachinger
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Polina
                                  </span>
                                  <span class="cit-name-surname">
                                   Golland
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Reuter
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Brainprint: Identifying subjects by their brain
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In International Conference on Medical Image Computing and Computer-Assisted Intervention
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  41
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  48
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [22].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference [22] in text" id="ref-22">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.22">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Debadatta
                                  </span>
                                  <span class="cit-name-surname">
                                   Dash
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferrari
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jun
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Spatial and spectral fingerprint in the brain: Speaker identification from single trial meg signals
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  In INTERSPEECH
                                 </abbr>
                                 , pages
                                 <span class="cit-fpage">
                                  1203
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1207
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [23].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference [23] in text" id="ref-23">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.18.159913v3.23">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   JK
                                  </span>
                                  <span class="cit-name-surname">
                                   Rowling
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-source">
                                  of Book: Harry Potter and the Sorcerer’s Stone
                                 </span>
                                 . Year:
                                 <span class="cit-publ-name">
                                  Scholastic Press/1997
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [24].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference [24] in text" id="ref-24">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.24" data-doi="10.1159/000323281">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Andy
                                  </span>
                                  <span class="cit-name-surname">
                                   Liaw
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew
                                  </span>
                                  <span class="cit-name-surname">
                                   Wiener
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Classification and regression by randomforest
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  R news
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  2
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  18
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  22
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2002
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DR%2Bnews%26rft.volume%253D2%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000323281%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1159/000323281&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [25].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference [25] in text" id="ref-25">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.25" data-doi="10.1016/j.jneumeth.2016.11.009">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sofie S
                                  </span>
                                  <span class="cit-name-surname">
                                   Meyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   James
                                  </span>
                                  <span class="cit-name-surname">
                                   Bonaiuto
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark
                                  </span>
                                  <span class="cit-name-surname">
                                   Lim
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Holly
                                  </span>
                                  <span class="cit-name-surname">
                                   Rossiter
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sheena
                                  </span>
                                  <span class="cit-name-surname">
                                   Waters
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Bradbury
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sven
                                  </span>
                                  <span class="cit-name-surname">
                                   Bestmann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matthew
                                  </span>
                                  <span class="cit-name-surname">
                                   Brookes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martina F
                                  </span>
                                  <span class="cit-name-surname">
                                   Callaghan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Nikolaus
                                  </span>
                                  <span class="cit-name-surname">
                                   Weiskopf
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Flexible head-casts for high spatial precision meg
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of neuroscience methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  276
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  38
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  45
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bneuroscience%2Bmethods%26rft.volume%253D276%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jneumeth.2016.11.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.jneumeth.2016.11.009&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [26].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference [26] in text" id="ref-26">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.26">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   H-RM
                                  </span>
                                  <span class="cit-name-surname">
                                   Tan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joachim
                                  </span>
                                  <span class="cit-name-surname">
                                   Gross
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   PJ
                                  </span>
                                  <span class="cit-name-surname">
                                   Uhlhaas
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Meg—measured auditory steadystate oscillations show high test–retest reliability: A sensor and source-space analysis
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  122
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  417
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  426
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D122%26rft.spage%253D417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [27].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference [27] in text" id="ref-27">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.27">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Douglas Owen
                                  </span>
                                  <span class="cit-name-surname">
                                   Cheyne
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Paul
                                  </span>
                                  <span class="cit-name-surname">
                                   Ferrari
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Meg studies of motor cortex gamma oscillations: evidence for a gamma “fingerprint” in the brain?
                                 </span>
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in human neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  7
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  575
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2013
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bhuman%2Bneuroscience%26rft.volume%253D7%26rft.spage%253D575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [28].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference [28] in text" id="ref-28">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.28" data-doi="10.1073/pnas.1614763114">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mark D
                                  </span>
                                  <span class="cit-name-surname">
                                   Vida
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Adrian
                                  </span>
                                  <span class="cit-name-surname">
                                   Nestor
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David C
                                  </span>
                                  <span class="cit-name-surname">
                                   Plaut
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Marlene
                                  </span>
                                  <span class="cit-name-surname">
                                   Behrmann
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Spatiotemporal dynamics of similarity-based neural representations of facial identity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Proceedings of the National Academy of Sciences
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  388
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  393
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BNational%2BAcademy%2Bof%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1614763114%26rft_id%253Dinfo%253Apmid%252F28028220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTQvMi8zODgiO3M6NDoiYXRvbSI7czo0ODoiL2Jpb3J4aXYvZWFybHkvMjAyMi8wMy8xNy8yMDIwLjA2LjE4LjE1OTkxMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink">
                                 <span>
                                  <span class="cit-reflinks-abstract">
                                   Abstract
                                  </span>
                                  <span class="cit-sep cit-reflinks-variant-name-sep">
                                   /
                                  </span>
                                  <span class="cit-reflinks-full-text">
                                   <span class="free-full-text">
                                    FREE
                                   </span>
                                   Full Text
                                  </span>
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [29].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference [29] in text" id="ref-29">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.29">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Enrico
                                  </span>
                                  <span class="cit-name-surname">
                                   Amico
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joaquín
                                  </span>
                                  <span class="cit-name-surname">
                                   Goñi
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  The quest for identifiability in human functional connectomes
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Scientific reports
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  8
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  14
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DScientific%2Breports%26rft.volume%253D8%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [30].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference [30] in text" id="ref-30">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.30">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sheng
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chiang-Shan R
                                  </span>
                                  <span class="cit-name-surname">
                                   Li
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Task-related, low-frequency task-residual, and resting state activity in the default mode network brain regions
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in psychology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  3
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  172
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bpsychology%26rft.volume%253D3%26rft.spage%253D172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [31].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference [31] in text" id="ref-31">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.31">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Omer
                                  </span>
                                  <span class="cit-name-surname">
                                   Grigg
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Cheryl L
                                  </span>
                                  <span class="cit-name-surname">
                                   Grady
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Task-related effects on the temporal and spatial dynamics of resting-state functional connectivity in the default network
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  PloS one
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  10
                                 </span>
                                 ),
                                 <span class="cit-pub-date">
                                  2010
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [32].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference [32] in text" id="ref-32">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.32">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ying
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Michael J
                                  </span>
                                  <span class="cit-name-surname">
                                   Tarr
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert E
                                  </span>
                                  <span class="cit-name-surname">
                                   Kass
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Elissa M
                                  </span>
                                  <span class="cit-name-surname">
                                   Aminoff
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Exploring spatiotemporal neural dynamics of the human visual cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Human brain mapping
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  40
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  14
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  4213
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  4238
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHuman%2Bbrain%2Bmapping%26rft.volume%253D40%26rft.spage%253D4213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [33].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference [33] in text" id="ref-33">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.33" data-doi="10.1016/S0960-9822(03)00337-3">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tony
                                  </span>
                                  <span class="cit-name-surname">
                                   Ro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Bruno
                                  </span>
                                  <span class="cit-name-surname">
                                   Breitmeyer
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Philip
                                  </span>
                                  <span class="cit-name-surname">
                                   Burton
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Neel S
                                  </span>
                                  <span class="cit-name-surname">
                                   Singhal
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Lane
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Feedback contributions to visual awareness in human occipital cortex
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Current biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  13
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1038
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  1041
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2003
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DRo%26rft.auinit1%253DT.%26rft.volume%253D13%26rft.issue%253D12%26rft.spage%253D1038%26rft.epage%253D1041%26rft.atitle%253DFeedback%2Bcontributions%2Bto%2Bvisual%2Bawareness%2Bin%2Bhuman%2Boccipital%2Bcortex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252803%252900337-3%26rft_id%253Dinfo%253Apmid%252F12814549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/S0960-9822(03)00337-3&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=12814549&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000183588000022&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [34].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference [34] in text" id="ref-34">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.34" data-doi="10.1016/j.neuroimage.2012.02.037">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Oliver
                                  </span>
                                  <span class="cit-name-surname">
                                   Jakobs
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Robert
                                  </span>
                                  <span class="cit-name-surname">
                                   Langner
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Svenja
                                  </span>
                                  <span class="cit-name-surname">
                                   Caspers
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Roski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Edna C
                                  </span>
                                  <span class="cit-name-surname">
                                   Cieslik
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Karl
                                  </span>
                                  <span class="cit-name-surname">
                                   Zilles
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Angela R
                                  </span>
                                  <span class="cit-name-surname">
                                   Laird
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Peter T
                                  </span>
                                  <span class="cit-name-surname">
                                   Fox
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Simon B
                                  </span>
                                  <span class="cit-name-surname">
                                   Eickhoff
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Across-study and within-subject functional connectivity of a right temporo-parietal junction subregion involved in stimulus–context integration
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  60
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  2389
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  2398
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D60%26rft.spage%253D2389%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2012.02.037%26rft_id%253Dinfo%253Apmid%252F22387170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2012.02.037&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=22387170&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000302926600046&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [35].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference [35] in text" id="ref-35">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.35" data-doi="10.1016/j.clinph.2006.07.316">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Riitta
                                  </span>
                                  <span class="cit-name-surname">
                                   Salmelin
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Clinical neurophysiology of language: the meg approach
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Clinical Neurophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  118
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  237
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  254
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2007
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DClinical%2Bneurophysiology%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BFederation%2Bof%2BClinical%2BNeurophysiology%26rft.stitle%253DClin%2BNeurophysiol%26rft.aulast%253DSalmelin%26rft.auinit1%253DR.%26rft.volume%253D118%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D254%26rft.atitle%253DClinical%2Bneurophysiology%2Bof%2Blanguage%253A%2Bthe%2BMEG%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.clinph.2006.07.316%26rft_id%253Dinfo%253Apmid%252F17008126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.clinph.2006.07.316&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=17008126&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000244135200001&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [36].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference [36] in text" id="ref-36">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.36">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pravat K
                                  </span>
                                  <span class="cit-name-surname">
                                   Mandal
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Anwesha
                                  </span>
                                  <span class="cit-name-surname">
                                   Banerjee
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Manjari
                                  </span>
                                  <span class="cit-name-surname">
                                   Tripathi
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ankita
                                  </span>
                                  <span class="cit-name-surname">
                                   Sharma
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  A comprehensive review of magnetoencephalography (meg) studies for brain functionality in healthy aging and alzheimer’s disease (ad)
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in Computational Neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  60
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2BComputational%2BNeuroscience%26rft.volume%253D12%26rft.spage%253D60%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [37].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference [37] in text" id="ref-37">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.37" data-doi="10.1016/j.neuroimage.2013.10.027">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Alexandre
                                  </span>
                                  <span class="cit-name-surname">
                                   Gramfort
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Martin
                                  </span>
                                  <span class="cit-name-surname">
                                   Luessi
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Eric
                                  </span>
                                  <span class="cit-name-surname">
                                   Larson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Denis A
                                  </span>
                                  <span class="cit-name-surname">
                                   Engemann
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Daniel
                                  </span>
                                  <span class="cit-name-surname">
                                   Strohmeier
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Christian
                                  </span>
                                  <span class="cit-name-surname">
                                   Brodbeck
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Lauri
                                  </span>
                                  <span class="cit-name-surname">
                                   Parkkonen
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matti S
                                  </span>
                                  <span class="cit-name-surname">
                                   Hämäläinen
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Mne software for processing meg and eeg data
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Neuroimage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  86
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  446
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  460
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2014
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroimage%26rft.volume%253D86%26rft.spage%253D446%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuroimage.2013.10.027%26rft_id%253Dinfo%253Apmid%252F24161808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=10.1016/j.neuroimage.2013.10.027&link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref">
                                 <span>
                                  CrossRef
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=24161808&link_type=MED&atom=%2Fbiorxiv%2Fearly%2F2022%2F03%2F17%2F2020.06.18.159913.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline">
                                 <span>
                                  PubMed
                                 </span>
                                </a>
                                <a href="/lookup/external-ref?access_num=000330335300046&link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience">
                                 <span>
                                  Web of Science
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [38].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference [38] in text" id="ref-38">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.38">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pouya
                                  </span>
                                  <span class="cit-name-surname">
                                   Bashivan
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Irina
                                  </span>
                                  <span class="cit-name-surname">
                                   Rish
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mohammed
                                  </span>
                                  <span class="cit-name-surname">
                                   Yeasin
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Noel
                                  </span>
                                  <span class="cit-name-surname">
                                   Codella
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Learning representations from eeg with deep recurrent-convolutional neural networks
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  arXiv preprint
                                 </abbr>
                                 <span class="cit-pub-id-sep cit-pub-id-arxiv-sep">
                                 </span>
                                 <span class="cit-pub-id-scheme">
                                  arXiv:
                                 </span>
                                 <span class="cit-pub-id cit-pub-id-arxiv">
                                  1511.06448
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2015
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [39].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference [39] in text" id="ref-39">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.39">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pilar
                                  </span>
                                  <span class="cit-name-surname">
                                   Garcés
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   María
                                  </span>
                                  <span class="cit-name-surname">
                                   Carmen Martín-Buro
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fernando
                                  </span>
                                  <span class="cit-name-surname">
                                   Maestú
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Quantifying the test-retest reliability of magnetoencephalography resting-state functional connectivity
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Brain connectivity
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  6
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  6
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  448
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  460
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DBrain%2Bconnectivity%26rft.volume%253D6%26rft.spage%253D448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [40].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference [40] in text" id="ref-40">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.40">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Kristina T
                                  </span>
                                  <span class="cit-name-surname">
                                   Legget
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Allison K
                                  </span>
                                  <span class="cit-name-surname">
                                   Hild
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Sarah E
                                  </span>
                                  <span class="cit-name-surname">
                                   Steinmetz
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Steven T
                                  </span>
                                  <span class="cit-name-surname">
                                   Simon
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Donald C
                                  </span>
                                  <span class="cit-name-surname">
                                   Rojas
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Meg and eeg demonstrate similar test-retest reliability of the 40 hz auditory steady-state response
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  International Journal of Psychophysiology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  114
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  16
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  23
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2017
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BPsychophysiology%26rft.volume%253D114%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [41].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference [41] in text" id="ref-41">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.41">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   María
                                  </span>
                                  <span class="cit-name-surname">
                                   Carmen Martín-Buro
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pilar
                                  </span>
                                  <span class="cit-name-surname">
                                   Garcés
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Fernando
                                  </span>
                                  <span class="cit-name-surname">
                                   Maestú
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Test-retest reliability of resting-state magnetoencephalography power in sensor and source space
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Human brain mapping
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  37
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  1
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  179
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  190
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DHuman%2Bbrain%2Bmapping%26rft.volume%253D37%26rft.spage%253D179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [42].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference [42] in text" id="ref-42">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-book" id="cit-2020.06.18.159913v3.42">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Corinna
                                  </span>
                                  <span class="cit-name-surname">
                                   Cortes
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Giulia
                                  </span>
                                  <span class="cit-name-surname">
                                   DeSalvo
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mehryar
                                  </span>
                                  <span class="cit-name-surname">
                                   Mohri
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-chapter-title">
                                  Learning with rejection
                                 </span>
                                 .
                                 <span class="cit-source">
                                  In International Conference on Algorithmic Learning Theory
                                 </span>
                                 , pages
                                 <span class="cit-fpage">
                                  67
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  82
                                 </span>
                                 .
                                 <span class="cit-publ-name">
                                  Springer
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2016
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [43].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference [43] in text" id="ref-43">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.43">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Manasij
                                  </span>
                                  <span class="cit-name-surname">
                                   Venkatesh
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Joseph
                                  </span>
                                  <span class="cit-name-surname">
                                   Jaja
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Luiz
                                  </span>
                                  <span class="cit-name-surname">
                                   Pessoa
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Comparing functional connectivity matrices: A geometry-aware approach applied to participant identification
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  NeuroImage
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  207
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  116398
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNeuroImage%26rft.volume%253D207%26rft.spage%253D116398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [44].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference [44] in text" id="ref-44">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.44">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Brian
                                  </span>
                                  <span class="cit-name-surname">
                                   Kulis
                                  </span>
                                 </span>
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Metric learning: A survey
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Foundations and trends in machine learning
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  5
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  4
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  287
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  364
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2012
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFoundations%2Band%2Btrends%2Bin%2Bmachine%2Blearning%26rft.volume%253D5%26rft.spage%253D287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [45].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference [45] in text" id="ref-45">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.45">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Qiang
                                  </span>
                                  <span class="cit-name-surname">
                                   Yang
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yang
                                  </span>
                                  <span class="cit-name-surname">
                                   Liu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tianjian
                                  </span>
                                  <span class="cit-name-surname">
                                   Chen
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yongxin
                                  </span>
                                  <span class="cit-name-surname">
                                   Tong
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Federated machine learning: Concept and applications
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  ACM Transactions on Intelligent Systems and Technology (TIST)
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  10
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  19
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2019
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DACM%2BTransactions%2Bon%2BIntelligent%2BSystems%2Band%2BTechnology%2B%2528TIST%2529%26rft.volume%253D10%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [46].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference [46] in text" id="ref-46">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.46">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yegang
                                  </span>
                                  <span class="cit-name-surname">
                                   Hu
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Chunli
                                  </span>
                                  <span class="cit-name-surname">
                                   Yin
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jicong
                                  </span>
                                  <span class="cit-name-surname">
                                   Zhang
                                  </span>
                                 </span>
                                 , and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Yuping
                                  </span>
                                  <span class="cit-name-surname">
                                   Wang
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Partial least square aided beamforming algorithm in magnetoencephalography source imaging
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Frontiers in neuroscience
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  12
                                 </span>
                                 :
                                 <span class="cit-fpage">
                                  616
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2018
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DFrontiers%2Bin%2Bneuroscience%26rft.volume%253D12%26rft.spage%253D616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [47].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference [47] in text" id="ref-47">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.47">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pauli
                                  </span>
                                  <span class="cit-name-surname">
                                   Virtanen
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Ralf
                                  </span>
                                  <span class="cit-name-surname">
                                   Gommers
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Travis E
                                  </span>
                                  <span class="cit-name-surname">
                                   Oliphant
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matt
                                  </span>
                                  <span class="cit-name-surname">
                                   Haberland
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Tyler
                                  </span>
                                  <span class="cit-name-surname">
                                   Reddy
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   David
                                  </span>
                                  <span class="cit-name-surname">
                                   Cournapeau
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Evgeni
                                  </span>
                                  <span class="cit-name-surname">
                                   Burovski
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Pearu
                                  </span>
                                  <span class="cit-name-surname">
                                   Peterson
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Warren
                                  </span>
                                  <span class="cit-name-surname">
                                   Weckesser
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Jonathan
                                  </span>
                                  <span class="cit-name-surname">
                                   Bright
                                  </span>
                                 </span>
                                 ,
                                 <span class="cit-etal">
                                  et al.
                                 </span>
                                 <span class="cit-article-title">
                                  Scipy 1.0: fundamental algorithms for scientific computing in python
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Nature methods
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  17
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  3
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  261
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  272
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2020
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DNature%2Bmethods%26rft.volume%253D17%26rft.spage%253D261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [48].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference [48] in text" id="ref-48">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.48">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Samu
                                  </span>
                                  <span class="cit-name-surname">
                                   Taulu
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Matti
                                  </span>
                                  <span class="cit-name-surname">
                                   Kajola
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Presentation of electromagnetic multichannel data: the signal space separation method
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Journal of Applied Physics
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  97
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  12
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  124905
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2005
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BPhysics%26rft.volume%253D97%26rft.spage%253D124905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [49].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference [49] in text" id="ref-49">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.49">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Samu
                                  </span>
                                  <span class="cit-name-surname">
                                   Taulu
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Juha
                                  </span>
                                  <span class="cit-name-surname">
                                   Simola
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Spatiotemporal signal space separation method for rejecting nearby interference in meg measurements
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Physics in Medicine & Biology
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  51
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  7
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  1759
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  2006
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DPhysics%2Bin%2BMedicine%2B%2526%2BBiology%26rft.volume%253D51%26rft.spage%253D1759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                             <li>
                              <span class="ref-label">
                               [50].
                              </span>
                              <a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference [50] in text" id="ref-50">
                               ↵
                              </a>
                              <div class="cit ref-cit ref-journal" id="cit-2020.06.18.159913v3.50">
                               <div class="cit-metadata">
                                <cite>
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Mikko A
                                  </span>
                                  <span class="cit-name-surname">
                                   Uusitalo
                                  </span>
                                 </span>
                                 and
                                 <span class="cit-auth">
                                  <span class="cit-name-given-names">
                                   Risto J
                                  </span>
                                  <span class="cit-name-surname">
                                   Ilmoniemi
                                  </span>
                                 </span>
                                 .
                                 <span class="cit-article-title">
                                  Signal-space projection method for separating meg or eeg into components
                                 </span>
                                 .
                                 <abbr class="cit-jnl-abbrev">
                                  Medical and Biological Engineering and Computing
                                 </abbr>
                                 ,
                                 <span class="cit-vol">
                                  35
                                 </span>
                                 (
                                 <span class="cit-issue">
                                  2
                                 </span>
                                 ):
                                 <span class="cit-fpage">
                                  135
                                 </span>
                                 –
                                 <span class="cit-lpage">
                                  140
                                 </span>
                                 ,
                                 <span class="cit-pub-date">
                                  1997
                                 </span>
                                 .
                                </cite>
                               </div>
                               <div class="cit-extra">
                                <a href="{openurl}?query=rft.jtitle%253DMedical%2Band%2BBiological%2BEngineering%2Band%2BComputing%26rft.volume%253D35%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url">
                                 <span>
                                  OpenUrl
                                 </span>
                                </a>
                               </div>
                              </div>
                             </li>
                            </ol>
                           </div>
                           <span class="highwire-journal-article-marker-end">
                           </span>
                          </div>
                          <span class="related-urls">
                          </span>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-disqus-comment">
                <div class="pane-content">
                 <div id="disqus_thread">
                  <noscript>
                   <p>
                    <a href="http://biorxivstage.disqus.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2020.06.18.159913v3" class="" data-icon-position="" data-hide-link-title="0">
                     View the discussion thread.
                    </a>
                   </p>
                  </noscript>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-back-to-top">
                <div class="pane-content">
                 <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0">
                  <span class="icon-chevron-up">
                  </span>
                  Back to top
                 </a>
                </div>
               </div>
              </div>
             </div>
            </div>
            <div class="sidebar-right-wrapper grid-10 omega">
             <div class="panel-panel panel-region-sidebar-right">
              <div class="inside">
               <div class="panel-pane pane-highwire-node-pager">
                <div class="pane-content">
                 <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager">
                  <span class="pager-prev">
                   <a href="/content/10.1101/2021.11.17.469067v2" title="LiveCellMiner: A New Tool to Analyze Mitotic Progression" rel="prev" class="pager-link-prev link-icon">
                    <span class="icon-circle-arrow-left">
                    </span>
                    <span class="title">
                     Previous
                    </span>
                   </a>
                  </span>
                  <span class="pager-next">
                   <a href="/content/10.1101/2022.02.23.481342v2" title="Fate-resolved gene regulatory signatures of individual B lymphocytes in the early stages of Epstein-Barr Virus infection" rel="next" class="pager-link-next link-icon-right link-icon">
                    <span class="title">
                     Next
                    </span>
                    <span class="icon-circle-arrow-right">
                    </span>
                   </a>
                  </span>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-custom pane-1">
                <div class="pane-content">
                 Posted March 17, 2022.
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-art-tools">
                <div class="pane-content">
                 <div id="mini-panel-biorxiv_art_tools" class="highwire-2col-stacked panel-display">
                  <div class="panel-row-wrapper clearfix">
                   <div class="content-left-wrapper content-column">
                    <div class="panel-panel panel-region-content-left">
                     <div class="inside">
                      <div class="panel-pane pane-highwire-variant-link">
                       <div class="pane-content">
                        <a href="/content/10.1101/2020.06.18.159913v3.full.pdf" target="_self" class="article-dl-pdf-link link-icon">
                         <span class="icon-external-link-sign">
                         </span>
                         <span class="title">
                          Download PDF
                         </span>
                        </a>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                   <div class="content-right-wrapper content-column">
                    <div class="panel-panel panel-region-content-right">
                     <div class="inside">
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-art-email">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="Email this Article" data-icon-position="" data-hide-link-title="0">
                           <i class="icon-envelope">
                           </i>
                           Email
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_art_email">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-block pane-forward-form pane-forward">
                              <div class="pane-content">
                               <form action="/content/10.1101/2020.06.18.159913v3.full" method="post" id="forward-form" accept-charset="UTF-8">
                                <div>
                                 <div id="edit-instructions" class="form-item form-item-label-before form-type-item">
                                  <p>
                                   Thank you for your interest in spreading the word about bioRxiv.
                                  </p>
                                  <p>
                                   NOTE: Your email address is requested solely to identify you as the sender of this article.
                                  </p>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-email">
                                  <label for="edit-email">
                                   Your Email
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textfield form-item-name">
                                  <label for="edit-name">
                                   Your Name
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required"/>
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-recipients">
                                  <label for="edit-recipients">
                                   Send To
                                   <span class="form-required" title="This field is required.">
                                    *
                                   </span>
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea>
                                  </div>
                                  <div class="description">
                                   Enter multiple addresses on separate lines or separate them with commas.
                                  </div>
                                 </div>
                                 <div id="edit-page" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-page">
                                   You are going to email the following
                                  </label>
                                  <a href="/content/10.1101/2020.06.18.159913v3" class="active" data-icon-position="" data-hide-link-title="0">
                                   Brainprints: identifying individuals from magnetoencephalograms
                                  </a>
                                 </div>
                                 <div id="edit-subject" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-subject">
                                   Message Subject
                                  </label>
                                  (Your Name) has forwarded a page to you from bioRxiv
                                 </div>
                                 <div id="edit-body" class="form-item form-item-label-before form-type-item">
                                  <label for="edit-body">
                                   Message Body
                                  </label>
                                  (Your Name) thought you would like to see this page from the bioRxiv website.
                                 </div>
                                 <div class="form-item form-item-label-before form-type-textarea form-item-message">
                                  <label for="edit-message--2">
                                   Your Personal Message
                                  </label>
                                  <div class="form-textarea-wrapper resizable">
                                   <textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea>
                                  </div>
                                 </div>
                                 <input type="hidden" name="path" value="node/2452346"/>
                                 <input type="hidden" name="path_cid" value=""/>
                                 <input type="hidden" name="forward_footer" value=" "/>
                                 <input type="hidden" name="form_build_id" value="form-54Z34MS8RMgnku-6IYUhiadlh_C4wEG3rzfuk55H5QA"/>
                                 <input type="hidden" name="form_id" value="forward_form"/>
                                 <fieldset class="captcha form-wrapper">
                                  <legend>
                                   <span class="fieldset-legend">
                                    CAPTCHA
                                   </span>
                                  </legend>
                                  <div class="fieldset-wrapper">
                                   <div class="fieldset-description">
                                    This question is for testing whether or not you are a human visitor and to prevent automated spam submissions.
                                   </div>
                                   <input type="hidden" name="captcha_sid" value="484472955"/>
                                   <input type="hidden" name="captcha_token" value="2ac4c6784e1bac18d3f4b22e98c58b0d"/>
                                   <input type="hidden" name="captcha_response" value="Google no captcha"/>
                                   <div class="g-recaptcha" data-sitekey="6LfnJVIUAAAAAE-bUOMg0MJGki4lqSvDmhJp19fN" data-theme="light" data-type="image">
                                   </div>
                                  </div>
                                 </fieldset>
                                 <div class="form-actions form-wrapper" id="edit-actions">
                                  <input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit"/>
                                 </div>
                                </div>
                               </form>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
                       <div class="pane-content">
                        <a href="/" class="link-icon">
                         <span class="icon-share-alt">
                         </span>
                         <span class="title">
                          Share
                         </span>
                        </a>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-panels-mini pane-biorxiv-share highwire_clipboard_form_ajax_shareit">
                       <div class="pane-content">
                        <div class="panel-display omega-12-onecol" id="mini-panel-biorxiv_share">
                         <div class="panel-panel grid-12 panel-region-preface">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-citation">
                            <div class="pane-content">
                             <div class="highwire-article-citation highwire-citation-type-highwire-article node2452346--3" data-node-nid="2452346" id="node-2452346--4405983950" data-pisa="biorxiv;2020.06.18.159913v3" data-pisa-master="biorxiv;2020.06.18.159913" data-seqnum="2452346" data-apath="/biorxiv/early/2022/03/17/2020.06.18.159913.atom">
                              <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                               <div class="highwire-cite-title">
                                <div class="highwire-cite-title">
                                 Brainprints: identifying individuals from magnetoencephalograms
                                </div>
                               </div>
                               <div class="highwire-cite-authors">
                                <span class="highwire-citation-authors">
                                 <span class="highwire-citation-author first" data-delta="0">
                                  <span class="nlm-given-names">
                                   Shenghao
                                  </span>
                                  <span class="nlm-surname">
                                   Wu
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="1">
                                  <span class="nlm-given-names">
                                   Aaditya
                                  </span>
                                  <span class="nlm-surname">
                                   Ramdas
                                  </span>
                                 </span>
                                 ,
                                 <span class="highwire-citation-author" data-delta="2">
                                  <span class="nlm-given-names">
                                   Leila
                                  </span>
                                  <span class="nlm-surname">
                                   Wehbe
                                  </span>
                                 </span>
                                </span>
                               </div>
                               <div class="highwire-cite-metadata">
                                <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                 bioRxiv
                                </span>
                                <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                 2020.06.18.159913;
                                </span>
                                <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                 <span class="doi_label">
                                  doi:
                                 </span>
                                 https://doi.org/10.1101/2020.06.18.159913
                                </span>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-content">
                          <div class="inside">
                           <div class="panel-pane pane-highwire-article-clipboard-copy">
                            <div class="pane-content">
                             <div class="clipboard-copy">
                              <span class="label-url">
                               <label for="dynamic">
                                Share This Article:
                               </label>
                              </span>
                              <span class="input-text-url">
                               <input type="text" id="dynamic" value="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3" size="50"/>
                              </span>
                              <span class="copy-button button">
                               <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">
                                Copy
                               </button>
                              </span>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                         <div class="panel-panel grid-12 panel-region-postscript">
                          <div class="inside">
                           <div class="panel-pane pane-service-links">
                            <div class="pane-content">
                             <div class="service-links">
                              <a href="http://digg.com/submit?phase=2&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&title=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo"/>
                              </a>
                              <a href="http://reddit.com/submit?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&title=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo"/>
                              </a>
                              <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&text=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo"/>
                              </a>
                              <a href="http://www.facebook.com/sharer.php?u=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&t=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo"/>
                              </a>
                              <a href="http://www.google.com/bookmarks/mark?op=add&bkmk=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&title=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo"/>
                              </a>
                              <a href="http://www.linkedin.com/shareArticle?mini=true&url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&title=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms&summary=&source=bioRxiv" id="linkedin" title="Publish this post to LinkedIn" class="service-links-linkedin" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/linkedin-32px.png" alt="LinkedIn logo"/>
                              </a>
                              <a href="http://www.mendeley.com/import/?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&title=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                               <img src="https://www.biorxiv.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo"/>
                              </a>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                      <div class="panel-separator">
                      </div>
                      <div class="panel-pane pane-minipanel-dialog-link pane-biorxiv-cite-tool">
                       <div class="pane-content">
                        <div class="minipanel-dialog-wrapper">
                         <div class="minipanel-dialog-link-link">
                          <a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title="Citation Tools">
                           <span class="icon-globe">
                           </span>
                           <span class="title">
                            Citation Tools
                           </span>
                          </a>
                         </div>
                         <div class="minipanel-dialog-link-mini" style="display:none">
                          <div class="panel-display panel-1col clearfix" id="mini-panel-biorxiv_cite_tool">
                           <div class="panel-panel panel-col">
                            <div>
                             <div class="panel-pane pane-highwire-citation-export">
                              <div class="pane-content">
                               <div class="highwire-citation-export">
                                <div class="highwire-citation-info">
                                 <div class="highwire-article-citation highwire-citation-type-highwire-article cite-tool-node2452346--5" data-node-nid="2452346" id="citation-node-2452346--6396257876" data-pisa="biorxiv;2020.06.18.159913v3" data-pisa-master="biorxiv;2020.06.18.159913" data-seqnum="2452346" data-apath="/biorxiv/early/2022/03/17/2020.06.18.159913.atom">
                                  <div class="highwire-cite highwire-cite-highwire-article highwire-citation-biorxiv-article-pap-list clearfix">
                                   <div class="highwire-cite-title">
                                    <div class="highwire-cite-title">
                                     Brainprints: identifying individuals from magnetoencephalograms
                                    </div>
                                   </div>
                                   <div class="highwire-cite-authors">
                                    <span class="highwire-citation-authors">
                                     <span class="highwire-citation-author first" data-delta="0">
                                      <span class="nlm-given-names">
                                       Shenghao
                                      </span>
                                      <span class="nlm-surname">
                                       Wu
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="1">
                                      <span class="nlm-given-names">
                                       Aaditya
                                      </span>
                                      <span class="nlm-surname">
                                       Ramdas
                                      </span>
                                     </span>
                                     ,
                                     <span class="highwire-citation-author" data-delta="2">
                                      <span class="nlm-given-names">
                                       Leila
                                      </span>
                                      <span class="nlm-surname">
                                       Wehbe
                                      </span>
                                     </span>
                                    </span>
                                   </div>
                                   <div class="highwire-cite-metadata">
                                    <span class="highwire-cite-metadata-journal highwire-cite-metadata">
                                     bioRxiv
                                    </span>
                                    <span class="highwire-cite-metadata-pages highwire-cite-metadata">
                                     2020.06.18.159913;
                                    </span>
                                    <span class="highwire-cite-metadata-doi highwire-cite-metadata">
                                     <span class="doi_label">
                                      doi:
                                     </span>
                                     https://doi.org/10.1101/2020.06.18.159913
                                    </span>
                                   </div>
                                  </div>
                                 </div>
                                </div>
                                <div class="highwire-citation-formats">
                                 <h2>
                                  Citation Manager Formats
                                 </h2>
                                 <div class="highwire-citation-formats-links">
                                  <span class="Z3988" title="ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&rft.spage&rft.epage&rft.atitle=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms&rft.volume&rft.issue&rft.date=2022-01-01%2000%3A00%3A00&rft.stitle&rft.jtitle=bioRxiv&rft.au=Wu%2C+Shenghao&rft.au=Ramdas%2C+Aaditya&rft.au=Wehbe%2C+Leila">
                                  </span>
                                  <ul class="hw-citation-links inline button button-alt button-grid clearfix">
                                   <li class="bibtext first">
                                    <a href="/highwire/citation/2452346/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     BibTeX
                                    </a>
                                   </li>
                                   <li class="bookends">
                                    <a href="/highwire/citation/2452346/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Bookends
                                    </a>
                                   </li>
                                   <li class="easybib">
                                    <a href="/highwire/citation/2452346/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EasyBib
                                    </a>
                                   </li>
                                   <li class="endnote-tagged">
                                    <a href="/highwire/citation/2452346/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote (tagged)
                                    </a>
                                   </li>
                                   <li class="endnote-8-xml">
                                    <a href="/highwire/citation/2452346/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     EndNote 8 (xml)
                                    </a>
                                   </li>
                                   <li class="medlars">
                                    <a href="/highwire/citation/2452346/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Medlars
                                    </a>
                                   </li>
                                   <li class="mendeley">
                                    <a href="/highwire/citation/2452346/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Mendeley
                                    </a>
                                   </li>
                                   <li class="papers">
                                    <a href="/highwire/citation/2452346/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Papers
                                    </a>
                                   </li>
                                   <li class="refworks-tagged">
                                    <a href="/highwire/citation/2452346/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RefWorks Tagged
                                    </a>
                                   </li>
                                   <li class="reference-manager">
                                    <a href="/highwire/citation/2452346/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Ref Manager
                                    </a>
                                   </li>
                                   <li class="ris">
                                    <a href="/highwire/citation/2452346/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     RIS
                                    </a>
                                   </li>
                                   <li class="zotero last">
                                    <a href="/highwire/citation/2452346/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">
                                     Zotero
                                    </a>
                                   </li>
                                  </ul>
                                 </div>
                                </div>
                               </div>
                              </div>
                             </div>
                            </div>
                           </div>
                          </div>
                         </div>
                        </div>
                       </div>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-service-links">
                <div class="pane-content">
                 <div class="service-links">
                  <div class="item-list">
                   <ul>
                    <li class="first">
                     <a href="http://twitter.com/share?url=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&count=horizontal&via=&text=Brainprints%3A%20identifying%20individuals%20from%20magnetoencephalograms&counturl=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3" class="twitter-share-button service-links-twitter-widget" id="twitter_widget" title="Tweet This" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Tweet Widget
                      </span>
                     </a>
                    </li>
                    <li>
                     <a href="http://www.facebook.com/plugins/like.php?href=https%3A//www.biorxiv.org/content/10.1101/2020.06.18.159913v3&layout=button_count&show_faces=false&action=like&colorscheme=light&width=100&height=21&font=&locale=" id="facebook_like" title="I Like it" class="service-links-facebook-like" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Facebook Like
                      </span>
                     </a>
                    </li>
                    <li class="last">
                     <a href="https://www.biorxiv.org/content/10.1101/2020.06.18.159913v3" id="google_plus_one" title="Plus it" class="service-links-google-plus-one" rel="nofollow" data-icon-position="" data-hide-link-title="0">
                      <span class="element-invisible">
                       Google Plus One
                      </span>
                     </a>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-highwire-article-collections">
                <h2 class="pane-title">
                 Subject Area
                </h2>
                <div class="pane-content">
                 <div class="highwire-list-wrapper highwire-article-collections">
                  <div class="highwire-list">
                   <ul class="highwire-article-collection-term-list">
                    <li class="first last odd">
                     <span class="highwire-article-collection-term">
                      <a href="/collection/neuroscience" class="highlight" data-icon-position="" data-hide-link-title="0">
                       Neuroscience
                       <i class="icon-caret-right">
                       </i>
                      </a>
                     </span>
                    </li>
                   </ul>
                  </div>
                 </div>
                </div>
               </div>
               <div class="panel-separator">
               </div>
               <div class="panel-pane pane-panels-mini pane-biorxiv-subject-collections block-style-col2">
                <div class="pane-content">
                 <div class="panel-flexible panels-flexible-new clearfix" id="mini-panel-biorxiv_subject_collections">
                  <div class="panel-flexible-inside panels-flexible-new-inside">
                   <div class="panels-flexible-region panels-flexible-region-new-center panels-flexible-region-first panels-flexible-region-last">
                    <div class="inside panels-flexible-region-inside panels-flexible-region-new-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last">
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-table-title" id="biorxiv-subject-areas-table-title">
                        <div class="snippet-content">
                         <b>
                          Subject Areas
                         </b>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-snippet">
                      <div class="pane-content">
                       <div class="snippet biorxiv-subject-areas-view-papers" id="biorxiv-subject-areas-view-papers">
                        <div class="snippet-content">
                         <a href="/content/early/recent">
                          <strong>
                           All Articles
                          </strong>
                         </a>
                        </div>
                       </div>
                      </div>
                     </div>
                     <div class="panel-separator">
                     </div>
                     <div class="panel-pane pane-highwire-subject-collections">
                      <div class="pane-content">
                       <ul id="collection" class="collection highwire-list-expand">
                        <li class="outer collection depth-2 child first">
                         <div class="data-wrapper">
                          <a href="/collection/animal-behavior-and-cognition" class="" data-icon-position="" data-hide-link-title="0">
                           Animal Behavior and Cognition
                          </a>
                          <span class="article-count">
                           (4123)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biochemistry" class="" data-icon-position="" data-hide-link-title="0">
                           Biochemistry
                          </a>
                          <span class="article-count">
                           (8837)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioengineering" class="" data-icon-position="" data-hide-link-title="0">
                           Bioengineering
                          </a>
                          <span class="article-count">
                           (6539)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/bioinformatics" class="" data-icon-position="" data-hide-link-title="0">
                           Bioinformatics
                          </a>
                          <span class="article-count">
                           (23503)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/biophysics" class="" data-icon-position="" data-hide-link-title="0">
                           Biophysics
                          </a>
                          <span class="article-count">
                           (11822)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cancer-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cancer Biology
                          </a>
                          <span class="article-count">
                           (9242)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/cell-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Cell Biology
                          </a>
                          <span class="article-count">
                           (13358)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/clinical-trials" class="" data-icon-position="" data-hide-link-title="0">
                           Clinical Trials
                          </a>
                          <span class="article-count">
                           (138)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/developmental-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Developmental Biology
                          </a>
                          <span class="article-count">
                           (7456)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/ecology" class="" data-icon-position="" data-hide-link-title="0">
                           Ecology
                          </a>
                          <span class="article-count">
                           (11438)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/epidemiology" class="" data-icon-position="" data-hide-link-title="0">
                           Epidemiology
                          </a>
                          <span class="article-count">
                           (2066)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/evolutionary-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Evolutionary Biology
                          </a>
                          <span class="article-count">
                           (15187)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genetics" class="" data-icon-position="" data-hide-link-title="0">
                           Genetics
                          </a>
                          <span class="article-count">
                           (10460)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/genomics" class="" data-icon-position="" data-hide-link-title="0">
                           Genomics
                          </a>
                          <span class="article-count">
                           (14058)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/immunology" class="" data-icon-position="" data-hide-link-title="0">
                           Immunology
                          </a>
                          <span class="article-count">
                           (9197)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/microbiology" class="" data-icon-position="" data-hide-link-title="0">
                           Microbiology
                          </a>
                          <span class="article-count">
                           (22243)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/molecular-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Molecular Biology
                          </a>
                          <span class="article-count">
                           (8840)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/neuroscience" class="" data-icon-position="" data-hide-link-title="0">
                           Neuroscience
                          </a>
                          <span class="article-count">
                           (47702)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/paleontology" class="" data-icon-position="" data-hide-link-title="0">
                           Paleontology
                          </a>
                          <span class="article-count">
                           (352)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pathology" class="" data-icon-position="" data-hide-link-title="0">
                           Pathology
                          </a>
                          <span class="article-count">
                           (1433)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/pharmacology-and-toxicology" class="" data-icon-position="" data-hide-link-title="0">
                           Pharmacology and Toxicology
                          </a>
                          <span class="article-count">
                           (2493)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/physiology" class="" data-icon-position="" data-hide-link-title="0">
                           Physiology
                          </a>
                          <span class="article-count">
                           (3742)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/plant-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Plant Biology
                          </a>
                          <span class="article-count">
                           (8103)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/scientific-communication-and-education" class="" data-icon-position="" data-hide-link-title="0">
                           Scientific Communication and Education
                          </a>
                          <span class="article-count">
                           (1439)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/synthetic-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Synthetic Biology
                          </a>
                          <span class="article-count">
                           (2230)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child">
                         <div class="data-wrapper">
                          <a href="/collection/systems-biology" class="" data-icon-position="" data-hide-link-title="0">
                           Systems Biology
                          </a>
                          <span class="article-count">
                           (6050)
                          </span>
                         </div>
                        </li>
                        <li class="outer collection depth-2 child last">
                         <div class="data-wrapper">
                          <a href="/collection/zoology" class="" data-icon-position="" data-hide-link-title="0">
                           Zoology
                          </a>
                          <span class="article-count">
                           (1259)
                          </span>
                         </div>
                        </li>
                       </ul>
                      </div>
                     </div>
                    </div>
                   </div>
                  </div>
                 </div>
                </div>
               </div>
              </div>
             </div>
            </div>
           </div>
          </div>
         </div>
        </div>
       </div>
      </div>
     </div>
    </div>
   </section>
  </div>
  <div class="region region-page-bottom" id="region-page-bottom">
   <div class="region-inner region-page-bottom-inner">
   </div>
  </div>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__VNH5GD8Zz7g6_hGCOZjVjdMndxxC6naiExSSJKX3k_A__kzmEdtbqCsJx5Z9yg2Qy0PptB__ufXm-TxUOl0KZzUw__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js" defer="defer">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__N7ERJBYsOWyRnJgyoM125_Aiez2MOJGaUofG1JdWWBg__2cpwCQ7-xzTVeVvg_KOzwA1jka23oWApDPpgjoZKDCY__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript" src="//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js">
  </script>
  <script type="text/javascript" src="https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__9MyDscWUfs9SQBQB2wQzcEwFsvz4ksctP725Io-yBDU.js">
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
function euCookieComplianceLoadScripts() {}
//--&gt;&lt;!]]&gt;
  </script>
  <script type="text/javascript">
   &lt;!--//--&gt;&lt;![CDATA[//&gt;&lt;!--
var eu_cookie_compliance_cookie_name = "";
//--&gt;&lt;!]]&gt;
  </script>
 </body>
</html>
